<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for ORFik on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><B>[<A href="..#O">O</A>]</B></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-31 05:44:39 -0000 (Wed, 31 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4219</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for ORFik on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the ORFik package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/ORFik.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard timeout">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1421</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">ORFik</A>&nbsp;1.21.1</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/ORFik">landing page</A>)</SPAN><BR>Haakon Tjeldnes<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-29&nbsp;10:19:22&nbsp;-0000&nbsp;(Mon,&nbsp;29&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/ORFik</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">a33d060</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-05-04&nbsp;20:52:55&nbsp;-0000&nbsp;(Thu,&nbsp;04&nbsp;May&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph TIMEOUT">&nbsp;&nbsp;TIMEOUT&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: ORFik</TD></TR>
<TR><TD><B>Version</B>: 1.21.1</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:ORFik.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings ORFik_1.21.1.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-30 14:23:47 -0000 (Tue, 30 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-30 15:03:47 -0000 (Tue, 30 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 2400.0 seconds</TD></TR>
<TR><TD><B>RetCode</B>: None</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph TIMEOUT">&nbsp;&nbsp;TIMEOUT&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: ORFik.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:ORFik.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings ORFik_1.21.1.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/ORFik.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘ORFik/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘ORFik’ version ‘1.21.1’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘ORFik’ can be installed ... OK
* used C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04.1) 11.3.0’
* checking C++ specification ... NOTE
  Specified C++11: please drop specification unless essential
* checking installed package size ... NOTE
  installed size is  7.2Mb
  sub-directories of 1Mb or more:
    R         1.0Mb
    extdata   1.6Mb
    libs      2.8Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... NOTE
Unexported objects imported by &#x27;:::&#x27; calls:
  ‘GenomicFeatures:::.merge_seqinfo_and_infer_missing_seqlengths’
  ‘IRanges:::regroupBySupergroup’ ‘S4Vectors:::normarg_mcols’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
DEG.plot.static: no visible binding for global variable ‘Regulation’
DEG.plot.static: no visible binding for global variable ‘meanCounts’
DEG.plot.static: no visible binding for global variable ‘LFC’
DTEG.plot: no visible binding for global variable ‘Regulation’
DTEG.plot: no visible binding for global variable ‘rna’
DTEG.plot: no visible binding for global variable ‘rfp’
QCplots: no visible binding for global variable ‘leaders’
QCplots: no visible binding for global variable ‘trailers’
QCstats.plot: no visible binding for global variable ‘variable’
QCstats.plot: no visible binding for global variable ‘sample_total’
QCstats.plot: no visible binding for global variable ‘value’
QCstats.plot: no visible global function definition for ‘.’
QCstats.plot: no visible binding for global variable ‘sample_id’
QCstats.plot: no visible binding for global variable ‘percentage’
QCstats.plot: no visible binding for global variable
  ‘perc_of_counts_per_sample’
QCstats.plot: no visible binding for global variable ‘read length’
RiboQC.plot: no visible binding for global variable ‘variable’
RiboQC.plot: no visible binding for global variable ‘sample_total’
RiboQC.plot: no visible binding for global variable ‘value’
RiboQC.plot: no visible global function definition for ‘.’
RiboQC.plot: no visible binding for global variable ‘sample_id’
RiboQC.plot: no visible binding for global variable ‘percentage’
RiboQC.plot: no visible binding for global variable ‘percent’
RiboQC.plot: no visible binding for global variable ‘frame’
STAR.multiQC: no visible binding for global variable ‘sample_id’
STAR.multiQC: no visible binding for global variable ‘value’
TOP.Motif.ecdf: no visible binding for global variable ‘seq1’
TOP.Motif.ecdf: no visible binding for global variable ‘TOP’
alignmentFeatureStatistics : &lt;anonymous&gt;: no visible binding for global
  variable ‘percentage_mrna_aligned’
alignmentFeatureStatistics : &lt;anonymous&gt;: no visible binding for global
  variable ‘mRNA’
alignmentFeatureStatistics : &lt;anonymous&gt;: no visible binding for global
  variable ‘percentage_tx_aligned’
alignmentFeatureStatistics : &lt;anonymous&gt;: no visible binding for global
  variable ‘Transcript’
alignmentFeatureStatistics : &lt;anonymous&gt;: no visible binding for global
  variable ‘ratio_cds_mrna’
alignmentFeatureStatistics : &lt;anonymous&gt;: no visible binding for global
  variable ‘CDS’
alignmentFeatureStatistics : &lt;anonymous&gt;: no visible binding for global
  variable ‘ratio_cds_leader’
alignmentFeatureStatistics : &lt;anonymous&gt;: no visible binding for global
  variable ‘LEADERS’
allFeaturesHelper: no visible binding for global variable ‘te’
allFeaturesHelper: no visible binding for global variable ‘fpkmRFP’
allFeaturesHelper: no visible binding for global variable ‘fpkmRNA’
allFeaturesHelper: no visible binding for global variable ‘countRFP’
allFeaturesHelper: no visible binding for global variable ‘entropyRFP’
allFeaturesHelper: no visible binding for global variable
  ‘disengagementScores’
allFeaturesHelper: no visible binding for global variable ‘RRS’
allFeaturesHelper: no visible binding for global variable ‘RSS’
allFeaturesHelper: no visible binding for global variable ‘ORFScores’
allFeaturesHelper: no visible binding for global variable ‘ioScore’
allFeaturesHelper: no visible binding for global variable
  ‘startCodonCoverage’
allFeaturesHelper: no visible binding for global variable
  ‘startRegionRelative’
allFeaturesHelper: no visible binding for global variable ‘kozak’
allFeaturesHelper: no visible binding for global variable ‘StartCodons’
allFeaturesHelper: no visible binding for global variable ‘StopCodons’
allFeaturesHelper: no visible binding for global variable
  ‘fractionLengths’
allFeaturesHelper: no visible binding for global variable ‘distORFCDS’
allFeaturesHelper: no visible binding for global variable ‘inFrameCDS’
allFeaturesHelper: no visible binding for global variable
  ‘isOverlappingCds’
allFeaturesHelper: no visible binding for global variable ‘rankInTx’
appendZeroes: no visible binding for global variable ‘frame’
appendZeroes: no visible binding for global variable ‘position’
appendZeroes: no visible binding for global variable ‘count’
artificial.orfs: no visible binding for global variable ‘random’
artificial.orfs: no visible binding for global variable ‘pick’
artificial.orfs: no visible global function definition for ‘.’
codon_usage: no visible binding for global variable ‘merged’
codon_usage: no visible binding for global variable ‘AA’
codon_usage: no visible binding for global variable ‘codon’
codon_usage_plot: no visible binding for global variable ‘seqs’
collapse.by.scores: no visible global function definition for ‘.’
collapse.fastq.internal: no visible binding for global variable ‘N’
cor_plot: no visible binding for global variable ‘Var2’
cor_plot: no visible binding for global variable ‘Var1’
cor_plot: no visible binding for global variable ‘Cor’
cor_table: no visible binding for global variable ‘Var1’
cor_table: no visible binding for global variable ‘Var2’
cor_table: no visible binding for global variable ‘Cor’
coverageHeatMap: no visible binding for global variable ‘position’
coverageHeatMap: no visible binding for global variable ‘fraction’
coverageScorings: no visible binding for global variable ‘count’
coverageScorings: no visible binding for global variable ‘zscore’
coverageScorings: no visible binding for global variable ‘windowMean’
coverageScorings: no visible binding for global variable ‘windowSD’
coverageScorings: no visible global function definition for ‘.’
coverageScorings: no visible binding for global variable ‘gene_sum’
coverageScorings: no visible binding for global variable ‘fraction’
coverage_random_access_file: no visible binding for global variable
  ‘position’
coverage_random_access_file: no visible binding for global variable
  ‘frame’
coverage_to_dt: no visible binding for global variable ‘frame’
coverage_to_dt: no visible binding for global variable ‘position’
detectRibosomeShifts: no visible global function definition for ‘.’
detectRibosomeShifts: no visible binding for global variable ‘size’
detectRibosomeShifts: no visible binding for global variable ‘fraction’
detectRibosomeShifts: no visible binding for global variable ‘pShifted’
detectRibosomeShifts: no visible binding for global variable
  ‘sum.count’
detectRibosomeShifts: no visible binding for global variable ‘count’
detectRibosomeShifts: no visible binding for global variable
  ‘frac.score’
entropy: no visible binding for global variable ‘Hx’
entropy: no visible binding for global variable ‘codonSums’
entropy: no visible global function definition for ‘.’
fetch_xml_attributes: no visible binding for global variable
  ‘..to_keep’
filterExtremePeakGenes: no visible binding for global variable ‘count’
filterExtremePeakGenes: no visible binding for global variable
  ‘median_per_gene’
filterExtremePeakGenes: no visible global function definition for ‘.’
filterTranscripts: no visible binding for global variable ‘utr5_len’
filterTranscripts: no visible binding for global variable ‘utr3_len’
filter_empty_runs: no visible binding for global variable ‘spots’
findNGSPairs: no visible global function definition for ‘.’
findNGSPairs: no visible binding for global variable ‘forward’
findPeaksPerGene: no visible binding for global variable ‘sum_per_gene’
findPeaksPerGene: no visible binding for global variable ‘count’
findPeaksPerGene: no visible binding for global variable
  ‘mean_per_gene’
findPeaksPerGene: no visible binding for global variable ‘sd_per_gene’
findPeaksPerGene: no visible binding for global variable ‘zscore’
findPeaksPerGene: no visible binding for global variable ‘gene_id’
find_url_ebi_safe: no visible binding for global variable
  ‘run_accession’
flankPerGroup: no visible global function definition for ‘.’
flankPerGroup: no visible binding for global variable ‘group_name’
floss: no visible binding for global variable ‘ORFGrouping’
floss: no visible binding for global variable ‘widths’
floss: no visible global function definition for ‘.’
floss: no visible binding for global variable ‘CDSGrouping’
floss: no visible binding for global variable ‘fraction.x’
floss: no visible binding for global variable ‘fraction.y’
gSort: no visible binding for global variable ‘grnames’
geneToSymbol: no visible binding for global variable ‘ensembl_gene_id’
geneToSymbol: no visible binding for global variable
  ‘external_gene_name’
geneToSymbol: no visible binding for global variable
  ‘ensembl_transcript_id’
geneToSymbol: no visible binding for global variable ‘uniprotswissprot’
getNGenesCoverage: no visible global function definition for ‘.’
getNGenesCoverage: no visible binding for global variable ‘fraction’
get_phix_genome: no visible binding for global variable ‘phix.url’
initiationScore: no visible global function definition for ‘.’
initiationScore: no visible binding for global variable ‘dif’
initiationScore: no visible binding for global variable ‘fraction’
initiationScore: no visible binding for global variable ‘difPer’
isPeriodic: no visible binding for global variable ‘spec’
kozakHeatmap: no visible global function definition for ‘.’
kozakHeatmap: no visible binding for global variable ‘variable’
kozakHeatmap: no visible binding for global variable ‘value’
kozakHeatmap: no visible binding for global variable
  ‘count_seq_pos_with_count’
kozakHeatmap: no visible binding for global variable ‘median_score’
kozak_IR_ranking: no visible global function definition for ‘.’
kozak_IR_ranking: no visible binding for global variable ‘IR’
kozak_IR_ranking: no visible binding for global variable
  ‘upstream_kozak_strength’
kozak_IR_ranking: no visible binding for global variable ‘count’
kozak_IR_ranking: no visible binding for global variable ‘mean_IR’
list.genomes: no visible binding for global variable ‘STAR_index’
longestORFs: no visible global function definition for ‘.’
metaWindow: no visible binding for global variable ‘position’
metaWindow: no visible binding for global variable ‘frame’
metadata.autnaming: no visible binding for global variable
  ‘LIBRARYTYPE’
metadata.autnaming: no visible binding for global variable
  ‘LibraryStrategy’
orfFrameDistributions : &lt;anonymous&gt;: no visible binding for global
  variable ‘fraction’
orfFrameDistributions: no visible binding for global variable ‘percent’
orfFrameDistributions: no visible binding for global variable
  ‘fraction’
orfFrameDistributions: no visible binding for global variable
  ‘percent_length’
orfFrameDistributions: no visible global function definition for ‘.’
orfFrameDistributions: no visible binding for global variable
  ‘best_frame’
orfScore: no visible binding for global variable ‘frame’
orfScore: no visible binding for global variable ‘frame_one_RP’
orfScore: no visible binding for global variable ‘frame_two_RP’
pSitePlot: no visible binding for global variable ‘count’
pSitePlot: no visible binding for global variable ‘frame’
pSitePlot: no visible binding for global variable ‘position’
pcaExperiment: no visible binding for global variable ‘PC1’
pcaExperiment: no visible binding for global variable ‘PC2’
readLengthTable: no visible binding for global variable
  ‘counts_per_sample’
readLengthTable: no visible binding for global variable
  ‘perc_of_counts_per_sample’
regionPerReadLength : &lt;anonymous&gt;: no visible binding for global
  variable ‘fraction’
remakeTxdbExonIds: no visible global function definition for ‘.’
remakeTxdbExonIds: no visible binding for global variable ‘chr’
removeTxdbExons: no visible binding for global variable ‘exon_rank’
removeTxdbExons: no visible binding for global variable ‘ranks’
ribo_fft: no visible binding for global variable ‘fraction’
ribo_fft_plot: no visible binding for global variable ‘periods’
ribo_fft_plot: no visible binding for global variable ‘amplitude’
rnaNormalize: no visible binding for global variable ‘feature’
scaledWindowPositions: no visible binding for global variable
  ‘scalingFactor’
scaledWindowPositions: no visible binding for global variable
  ‘position’
scaledWindowPositions: no visible global function definition for ‘.’
scoreSummarizedExperiment: no visible global function definition for
  ‘rowSums2’
seq_usage: no visible binding for global variable ‘variable’
seq_usage: no visible binding for global variable ‘codon_sum’
seq_usage: no visible global function definition for ‘.’
seq_usage: no visible binding for global variable ‘gene_sum’
seq_usage: no visible binding for global variable
  ‘N_AA_of_type_per_gene’
seq_usage: no visible binding for global variable ‘as_prob_normalized’
seq_usage: no visible binding for global variable ‘N_total’
seq_usage: no visible binding for global variable ‘N’
seq_usage: no visible binding for global variable ‘mean_txNorm’
seq_usage: no visible binding for global variable ‘sum_txNorm’
seq_usage: no visible binding for global variable ‘dispersion’
seq_usage: no visible binding for global variable ‘dispersion_txNorm’
seq_usage: no visible binding for global variable ‘var_txNorm’
seq_usage: no visible binding for global variable ‘mean_percentage’
seq_usage: no visible binding for global variable ‘mean_txNorm_prob’
seq_usage: no visible binding for global variable
  ‘mean_txNorm_percentage’
seq_usage: no visible binding for global variable
  ‘relative_to_max_score’
shiftPlots : &lt;anonymous&gt;: no visible binding for global variable
  ‘frame’
shiftPlots : &lt;anonymous&gt;: no visible binding for global variable
  ‘position’
te.plot: no visible global function definition for ‘rowMin’
te.plot: no visible binding for global variable ‘variable’
te.plot: no visible binding for global variable ‘LFC_TE’
te.plot: no visible binding for global variable ‘rfp_log2’
te.plot: no visible binding for global variable ‘rna_log2’
te.plot: no visible binding for global variable ‘rna_log10’
te.table: no visible global function definition for ‘rowMin’
te.table: no visible binding for global variable ‘variable’
te.table: no visible binding for global variable ‘TE_log2’
te.table: no visible binding for global variable ‘rfp_log2’
te.table: no visible binding for global variable ‘rna_log2’
te_rna.plot: no visible binding for global variable ‘subtitle’
te_rna.plot: no visible binding for global variable ‘rna_log10’
te_rna.plot: no visible binding for global variable ‘TE_log2’
topMotif: no visible binding for global variable ‘seq1’
topMotif: no visible binding for global variable ‘seq2’
topMotif: no visible binding for global variable ‘seq3’
topMotif: no visible binding for global variable ‘seq4’
topMotif: no visible binding for global variable ‘seq5’
transcriptWindow: no visible binding for global variable ‘fractions’
transcriptWindow: no visible binding for global variable ‘feature’
windowCoveragePlot: no visible binding for global variable ‘feature’
windowCoveragePlot: no visible binding for global variable ‘fraction’
windowCoveragePlot: no visible binding for global variable
  ‘fraction_min’
windowCoveragePlot: no visible binding for global variable ‘position’
collapseDuplicatedReads,GAlignmentPairs: no visible global function
  definition for ‘.’
collapseDuplicatedReads,GAlignmentPairs: no visible binding for global
  variable ‘start1’
collapseDuplicatedReads,GAlignmentPairs: no visible binding for global
  variable ‘start2’
collapseDuplicatedReads,GAlignmentPairs: no visible binding for global
  variable ‘cigar1’
collapseDuplicatedReads,GAlignmentPairs: no visible binding for global
  variable ‘cigar2’
collapseDuplicatedReads,GAlignments: no visible global function
  definition for ‘.’
collapseDuplicatedReads,GRanges: no visible binding for global variable
  ‘size’
collapseDuplicatedReads,GRanges: no visible global function definition
  for ‘.’
collapseDuplicatedReads,data.table: no visible global function
  definition for ‘.’
collapseDuplicatedReads,data.table: no visible binding for global
  variable ‘size’
design,experiment: no visible binding for global variable ‘..formula’
seqlevels,experiment: no visible binding for global variable ‘df’
show,covRleList: no visible global function definition for ‘head’
Undefined global functions or variables:
  . ..formula ..to_keep AA CDS CDSGrouping Cor Hx IR LEADERS LFC LFC_TE
  LIBRARYTYPE LibraryStrategy N N_AA_of_type_per_gene N_total
  ORFGrouping ORFScores PC1 PC2 RRS RSS Regulation STAR_index
  StartCodons StopCodons TE_log2 TOP Transcript Var1 Var2 amplitude
  as_prob_normalized best_frame chr cigar1 cigar2 codon codonSums
  codon_sum count countRFP count_seq_pos_with_count counts_per_sample
  df dif difPer disengagementScores dispersion dispersion_txNorm
  distORFCDS ensembl_gene_id ensembl_transcript_id entropyRFP exon_rank
  external_gene_name feature forward fpkmRFP fpkmRNA frac.score
  fraction fraction.x fraction.y fractionLengths fraction_min fractions
  frame frame_one_RP frame_two_RP gene_id gene_sum grnames group_name
  head inFrameCDS ioScore isOverlappingCds kozak leaders mRNA
  meanCounts mean_IR mean_per_gene mean_percentage mean_txNorm
  mean_txNorm_percentage mean_txNorm_prob median_per_gene median_score
  merged pShifted perc_of_counts_per_sample percent percent_length
  percentage percentage_mrna_aligned percentage_tx_aligned periods
  phix.url pick position random rankInTx ranks ratio_cds_leader
  ratio_cds_mrna read length relative_to_max_score rfp rfp_log2 rna
  rna_log10 rna_log2 rowMin rowSums2 run_accession sample_id
  sample_total scalingFactor sd_per_gene seq1 seq2 seq3 seq4 seq5 seqs
  size spec spots start1 start2 startCodonCoverage startRegionRelative
  subtitle sum.count sum_per_gene sum_txNorm te trailers
  uniprotswissprot upstream_kozak_strength utr3_len utr5_len value
  var_txNorm variable widths windowMean windowSD zscore
Consider adding
  importFrom(&quot;base&quot;, &quot;length&quot;)
  importFrom(&quot;graphics&quot;, &quot;frame&quot;)
  importFrom(&quot;stats&quot;, &quot;df&quot;)
  importFrom(&quot;utils&quot;, &quot;head&quot;)
to your NAMESPACE file.
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... OK
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                              user system elapsed
codon_usage_exp             10.811  0.460  11.121
findUORFs_exp                6.830  0.224   7.055
computeFeatures              5.974  0.336   6.310
shiftFootprintsByExperiment  1.403  0.112   6.042
find_url_ebi                 0.362  0.020  12.060
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘Annotation_Alignment.Rmd’ using ‘UTF-8’... OK
  ‘Importing_Data.Rmd’ using ‘UTF-8’... OK
  ‘ORFikExperiment.Rmd’ using ‘UTF-8’... OK
  ‘ORFikOverview.Rmd’ using ‘UTF-8’... OK
  ‘Ribo-seq_pipeline.Rmd’ using ‘UTF-8’... OK
  ‘Working_with_transcripts.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ...</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">ORFik.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD INSTALL ORFik
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0/site-library’
* installing *source* package ‘ORFik’ ...
** using staged installation
** libs
using C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04.1) 11.3.0’
using C++11
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c findORFsHelpers.cpp -o findORFsHelpers.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c findOrfs.cpp -o findOrfs.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c findOrfsFasta.cpp -o findOrfsFasta.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c pmapFromTranscripts.cpp -o pmapFromTranscripts.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c pmapToTranscripts.cpp -o pmapToTranscripts.o
g++ -std=gnu++11 -shared -L/home/biocbuild/R/R-4.3.0/lib -L/usr/local/lib -o ORFik.so RcppExports.o findORFsHelpers.o findOrfs.o findOrfsFasta.o pmapFromTranscripts.o pmapToTranscripts.o -L/home/biocbuild/R/R-4.3.0/lib -lR
installing to /home/biocbuild/R/R-4.3.0/site-library/00LOCK-ORFik/00new/ORFik/libs
** R
** inst
** byte-compile and prepare package for lazy loading
Creating a new generic function for ‘symbols’ in package ‘ORFik’
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (ORFik)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">ORFik.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(ORFik)
Loading required package: IRanges
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following object is masked from &#x27;package:utils&#x27;:

    findMatches

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: GenomicRanges
Loading required package: GenomeInfoDb
Loading required package: GenomicAlignments
Loading required package: SummarizedExperiment
Loading required package: MatrixGenerics
Loading required package: matrixStats

Attaching package: &#x27;MatrixGenerics&#x27;

The following objects are masked from &#x27;package:matrixStats&#x27;:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars

Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


Attaching package: &#x27;Biobase&#x27;

The following object is masked from &#x27;package:MatrixGenerics&#x27;:

    rowMedians

The following objects are masked from &#x27;package:matrixStats&#x27;:

    anyMissing, rowMedians

Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: Rsamtools

Attaching package: &#x27;ORFik&#x27;

The following object is masked from &#x27;package:graphics&#x27;:

    symbols

&gt; 
&gt; test_check(&quot;ORFik&quot;)
Regulation
No change 
        6 
     widths     group.size   
 Min.   : 6   Min.   :1.000  
 1st Qu.: 9   1st Qu.:1.000  
 Median :12   Median :1.000  
 Mean   :12   Mean   :1.667  
 3rd Qu.:15   3rd Qu.:2.000  
 Max.   :18   Max.   :3.000  
[1] &quot;CAGE_Mutant_r1&quot;
[1] &quot;CAGE_Mutant_r2&quot;
[1] &quot;CAGE_WT_r1&quot;
[1] &quot;CAGE_WT_r2&quot;
[1] &quot;PAS_Mutant_r1&quot;
[1] &quot;PAS_Mutant_r2&quot;
[1] &quot;PAS_WT_r1&quot;
[1] &quot;PAS_WT_r2&quot;
[1] &quot;RFP_Mutant_r1&quot;
[1] &quot;RFP_Mutant_r2&quot;
[1] &quot;RFP_WT_r1&quot;
[1] &quot;RFP_WT_r2&quot;
[1] &quot;RNA_Mutant_r1&quot;
[1] &quot;RNA_Mutant_r2&quot;
[1] &quot;RNA_WT_r1&quot;
[1] &quot;RNA_WT_r2&quot;
Error in x$.self$finalize() : attempt to apply non-function
[1] &quot;RFP&quot;
[1] &quot;RFP&quot;
[1] &quot;RFP&quot;
[1] &quot;RFP&quot;
[ FAIL 0 | WARN 0 | SKIP 0 | PASS 411 ]
Warning message:
call dbDisconnect() when finished working with a connection 
&gt; 
&gt; proc.time()
   user  system elapsed 
111.264   7.058 117.409 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">ORFik.Rcheck/ORFik-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>DEG.analysis</TD><TD style="text-align: right;">0.789</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.824</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DEG.plot.static</TD><TD style="text-align: right;">0.460</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.481</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DEG_model</TD><TD style="text-align: right;">1.482</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">1.506</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DEG_model_results</TD><TD style="text-align: right;">0.496</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.496</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DTEG.analysis</TD><TD style="text-align: right;">0.452</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">0.492</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DTEG.plot</TD><TD style="text-align: right;">0.419</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">0.463</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ORFik.template.experiment</TD><TD style="text-align: right;">0.447</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">0.494</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ORFik.template.experiment.zf</TD><TD style="text-align: right;">0.063</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.079</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ORFikQC</TD><TD style="text-align: right;">0.436</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.440</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>QCreport</TD><TD style="text-align: right;">0.425</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.425</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>QCstats</TD><TD style="text-align: right;">0.437</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.438</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>QCstats.plot</TD><TD style="text-align: right;">0.436</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.436</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>RiboQC.plot</TD><TD style="text-align: right;">0.435</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.442</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>STAR.align.folder</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>STAR.align.single</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>STAR.index</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>STAR.install</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>STAR.remove.crashed.genome</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TOP.Motif.ecdf</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>artificial.orfs</TD><TD style="text-align: right;">0.448</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.448</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>asTX</TD><TD style="text-align: right;">0.870</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.877</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>assignTSSByCage</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bamVarName</TD><TD style="text-align: right;">0.928</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.932</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>codon_usage</TD><TD style="text-align: right;">3.727</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">3.792</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>codon_usage_exp</TD><TD style="text-align: right;">10.811</TD><TD style="text-align: right;"> 0.460</TD><TD style="text-align: right;">11.121</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>codon_usage_plot</TD><TD style="text-align: right;">4.173</TD><TD style="text-align: right;">0.573</TD><TD style="text-align: right;">4.599</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>collapse.by.scores</TD><TD style="text-align: right;">0.132</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.148</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>collapse.fastq</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>collapseDuplicatedReads-GAlignmentPairs-method</TD><TD style="text-align: right;">0.080</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.097</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>collapseDuplicatedReads-GAlignments-method</TD><TD style="text-align: right;">0.031</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.039</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>collapseDuplicatedReads-GRanges-method</TD><TD style="text-align: right;">0.031</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.039</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>collapseDuplicatedReads-data.table-method</TD><TD style="text-align: right;">0.030</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>collapseDuplicatedReads</TD><TD style="text-align: right;">0.033</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>combn.pairs</TD><TD style="text-align: right;">0.495</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.502</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>computeFeatures</TD><TD style="text-align: right;">5.974</TD><TD style="text-align: right;">0.336</TD><TD style="text-align: right;">6.310</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>computeFeaturesCage</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>config</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>config.exper</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>config.save</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>convertLibs</TD><TD style="text-align: right;">0.447</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">0.479</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>convertToOneBasedRanges</TD><TD style="text-align: right;">0.172</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.180</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>convert_bam_to_ofst</TD><TD style="text-align: right;">0.199</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.212</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>convert_to_bigWig</TD><TD style="text-align: right;">1.103</TD><TD style="text-align: right;">0.100</TD><TD style="text-align: right;">1.202</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>convert_to_covRle</TD><TD style="text-align: right;">0.595</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">0.639</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>convert_to_covRleList</TD><TD style="text-align: right;">0.952</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.980</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>countOverlapsW</TD><TD style="text-align: right;">0.096</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.105</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>countTable</TD><TD style="text-align: right;">0.505</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">0.550</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>countTable_regions</TD><TD style="text-align: right;">0.470</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.494</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>covRle</TD><TD style="text-align: right;">0.071</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.074</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>covRleFromGR</TD><TD style="text-align: right;">0.116</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.129</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>covRleList</TD><TD style="text-align: right;">0.025</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>coverageHeatMap</TD><TD style="text-align: right;">1.625</TD><TD style="text-align: right;">0.084</TD><TD style="text-align: right;">1.710</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>coveragePerTiling</TD><TD style="text-align: right;">0.941</TD><TD style="text-align: right;">0.072</TD><TD style="text-align: right;">1.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>coverageScorings</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>create.experiment</TD><TD style="text-align: right;">0.474</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.481</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>defineTrailer</TD><TD style="text-align: right;">0.207</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.223</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>design-experiment-method</TD><TD style="text-align: right;">0.503</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.540</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>detectRibosomeShifts</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>disengagementScore</TD><TD style="text-align: right;">0.619</TD><TD style="text-align: right;">0.025</TD><TD style="text-align: right;">0.643</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>distToCds</TD><TD style="text-align: right;">0.300</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.308</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>distToTSS</TD><TD style="text-align: right;">0.268</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.269</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>download.SRA</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>download.SRA.metadata</TD><TD style="text-align: right;">0.780</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">4.805</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>entropy</TD><TD style="text-align: right;">1.099</TD><TD style="text-align: right;">0.104</TD><TD style="text-align: right;">1.202</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>experiment-class</TD><TD style="text-align: right;">0.506</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.526</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>export.bed12</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.027</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>export.bigWig</TD><TD style="text-align: right;">0.041</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>export.fstwig</TD><TD style="text-align: right;">0.076</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.081</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>export.ofst-GAlignmentPairs-method</TD><TD style="text-align: right;">0.094</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.094</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>export.ofst-GAlignments-method</TD><TD style="text-align: right;">0.084</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.088</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>export.ofst-GRanges-method</TD><TD style="text-align: right;">0.086</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.086</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>export.ofst</TD><TD style="text-align: right;">0.085</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.084</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>export.wiggle</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>extendLeaders</TD><TD style="text-align: right;">1.376</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">1.440</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>extendTrailers</TD><TD style="text-align: right;">0.849</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.877</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filepath</TD><TD style="text-align: right;">0.507</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">0.539</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filterTranscripts</TD><TD style="text-align: right;">1.725</TD><TD style="text-align: right;">0.060</TD><TD style="text-align: right;">1.785</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fimport</TD><TD style="text-align: right;">0.385</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.400</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findFa</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findMapORFs</TD><TD style="text-align: right;">0.707</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.732</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findORFs</TD><TD style="text-align: right;">0.206</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.226</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findORFsFasta</TD><TD style="text-align: right;">0.087</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.091</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findPeaksPerGene</TD><TD style="text-align: right;">1.342</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">1.385</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findUORFs</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findUORFs_exp</TD><TD style="text-align: right;">6.830</TD><TD style="text-align: right;">0.224</TD><TD style="text-align: right;">7.055</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>find_url_ebi</TD><TD style="text-align: right;"> 0.362</TD><TD style="text-align: right;"> 0.020</TD><TD style="text-align: right;">12.060</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>firstEndPerGroup</TD><TD style="text-align: right;">0.083</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.087</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>firstExonPerGroup</TD><TD style="text-align: right;">0.093</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.098</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>firstStartPerGroup</TD><TD style="text-align: right;">0.083</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.088</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fix_malformed_gff</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>flankPerGroup</TD><TD style="text-align: right;">0.124</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.132</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>floss</TD><TD style="text-align: right;">0.367</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.374</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fpkm</TD><TD style="text-align: right;">0.149</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.153</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fractionLength</TD><TD style="text-align: right;">0.071</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.075</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fread.bed</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>gcContent</TD><TD style="text-align: right;">0.504</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.528</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>geneToSymbol</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getGenomeAndAnnotation</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_bioproject_candidates</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_genome_fasta</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_genome_gtf</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_noncoding_rna</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_phix_genome</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_silva_rRNA</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>groupGRangesBy</TD><TD style="text-align: right;">0.078</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.078</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>groupings</TD><TD style="text-align: right;">0.054</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.054</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>heatMapRegion</TD><TD style="text-align: right;">0.526</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.542</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import.ofst</TD><TD style="text-align: right;">0.087</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.088</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>initiationScore</TD><TD style="text-align: right;">1.333</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.333</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>insideOutsideORF</TD><TD style="text-align: right;">1.101</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">1.121</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>install.fastp</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>install.sratoolkit</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>isInFrame</TD><TD style="text-align: right;">0.301</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.301</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>isOverlapping</TD><TD style="text-align: right;">0.307</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.307</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>kozakHeatmap</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>kozakSequenceScore</TD><TD style="text-align: right;">0.710</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.714</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>lastExonEndPerGroup</TD><TD style="text-align: right;">0.08</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.08</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>lastExonPerGroup</TD><TD style="text-align: right;">0.094</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.094</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>lastExonStartPerGroup</TD><TD style="text-align: right;">0.075</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.079</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>libraryTypes</TD><TD style="text-align: right;">0.488</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.512</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>list.experiments</TD><TD style="text-align: right;">0.293</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.294</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>list.genomes</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loadRegion</TD><TD style="text-align: right;">1.979</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">2.043</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loadRegions</TD><TD style="text-align: right;">0.904</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.928</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loadTranscriptType</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loadTxdb</TD><TD style="text-align: right;">0.148</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.156</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>longestORFs</TD><TD style="text-align: right;">0.182</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.182</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeORFNames</TD><TD style="text-align: right;">0.109</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.117</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeSummarizedExperimentFromBam</TD><TD style="text-align: right;">0.488</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.496</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeTxdbFromGenome</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mergeFastq</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mergeLibs</TD><TD style="text-align: right;">0.457</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.474</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>metaWindow</TD><TD style="text-align: right;">0.277</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.280</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>numExonsPerGroup</TD><TD style="text-align: right;">0.051</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.051</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>optimizedTranscriptLengths</TD><TD style="text-align: right;">1.157</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">1.181</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>orfFrameDistributions</TD><TD style="text-align: right;">0.975</TD><TD style="text-align: right;">0.257</TD><TD style="text-align: right;">1.533</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>orfScore</TD><TD style="text-align: right;">1.228</TD><TD style="text-align: right;">0.337</TD><TD style="text-align: right;">1.151</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>organism-experiment-method</TD><TD style="text-align: right;">0.461</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">0.509</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>outputLibs</TD><TD style="text-align: right;">1.332</TD><TD style="text-align: right;">0.561</TD><TD style="text-align: right;">1.508</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pSitePlot</TD><TD style="text-align: right;">0.441</TD><TD style="text-align: right;">0.112</TD><TD style="text-align: right;">0.553</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pcaExperiment</TD><TD style="text-align: right;">1.482</TD><TD style="text-align: right;">0.292</TD><TD style="text-align: right;">1.775</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pmapFromTranscriptF</TD><TD style="text-align: right;">0.116</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">0.148</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pmapToTranscriptF</TD><TD style="text-align: right;">1.121</TD><TD style="text-align: right;">0.072</TD><TD style="text-align: right;">1.193</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rankOrder</TD><TD style="text-align: right;">0.100</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.108</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>read.experiment</TD><TD style="text-align: right;">0.466</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.482</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readBam</TD><TD style="text-align: right;">0.233</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">0.277</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readWidths</TD><TD style="text-align: right;">0.02</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.02</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>reassignTSSbyCage</TD><TD style="text-align: right;">0.797</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">0.829</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>reassignTxDbByCage</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>reduceKeepAttr</TD><TD style="text-align: right;">0.167</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.179</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>regionPerReadLength</TD><TD style="text-align: right;">0.407</TD><TD style="text-align: right;">0.246</TD><TD style="text-align: right;">0.901</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>remove.experiments</TD><TD style="text-align: right;">0.655</TD><TD style="text-align: right;">0.255</TD><TD style="text-align: right;">0.547</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ribo_fft</TD><TD style="text-align: right;">3.558</TD><TD style="text-align: right;">0.420</TD><TD style="text-align: right;">3.977</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ribo_fft_plot</TD><TD style="text-align: right;">3.387</TD><TD style="text-align: right;">0.175</TD><TD style="text-align: right;">3.563</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ribosomeReleaseScore</TD><TD style="text-align: right;">0.255</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.263</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ribosomeStallingScore</TD><TD style="text-align: right;">0.400</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.411</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>save.experiment</TD><TD style="text-align: right;">0.451</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.467</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>scaledWindowPositions</TD><TD style="text-align: right;">0.418</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.429</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>seqnamesPerGroup</TD><TD style="text-align: right;">0.079</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.083</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>shiftFootprints</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>shiftFootprintsByExperiment</TD><TD style="text-align: right;">1.403</TD><TD style="text-align: right;">0.112</TD><TD style="text-align: right;">6.042</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>shiftPlots</TD><TD style="text-align: right;">3.966</TD><TD style="text-align: right;">0.724</TD><TD style="text-align: right;">0.075</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>shifts.load</TD><TD style="text-align: right;">0.431</TD><TD style="text-align: right;">0.072</TD><TD style="text-align: right;">0.503</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>simpleLibs</TD><TD style="text-align: right;">0.448</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">0.488</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sortPerGroup</TD><TD style="text-align: right;">0.315</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">0.358</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>startCodons</TD><TD style="text-align: right;">0.652</TD><TD style="text-align: right;">0.081</TD><TD style="text-align: right;">0.733</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>startDefinition</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>startRegion</TD><TD style="text-align: right;">0.697</TD><TD style="text-align: right;">0.055</TD><TD style="text-align: right;">0.753</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>startRegionCoverage</TD><TD style="text-align: right;">0.497</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">0.529</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>startSites</TD><TD style="text-align: right;">0.387</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.395</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>stopCodons</TD><TD style="text-align: right;">0.630</TD><TD style="text-align: right;">0.031</TD><TD style="text-align: right;">0.662</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>stopDefinition</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>stopRegion</TD><TD style="text-align: right;">0.694</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.702</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>stopSites</TD><TD style="text-align: right;">0.368</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.368</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>strandBool</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>strandPerGroup</TD><TD style="text-align: right;">0.076</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.076</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>subsetToFrame</TD><TD style="text-align: right;">0.017</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>te.plot</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>te.table</TD><TD style="text-align: right;">0.459</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.472</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>te_rna.plot</TD><TD style="text-align: right;">0.426</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">0.438</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>tile1</TD><TD style="text-align: right;">0.262</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.270</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>topMotif</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>transcriptWindow</TD><TD style="text-align: right;">1.238</TD><TD style="text-align: right;">0.055</TD><TD style="text-align: right;">1.294</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>translationalEff</TD><TD style="text-align: right;">0.207</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.235</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>trimming.table</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>txNames</TD><TD style="text-align: right;">0.055</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.055</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>txNamesToGeneNames</TD><TD style="text-align: right;">1.114</TD><TD style="text-align: right;">0.031</TD><TD style="text-align: right;">1.147</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>uORFSearchSpace</TD><TD style="text-align: right;">0.735</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">0.767</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>uniqueGroups</TD><TD style="text-align: right;">0.199</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.211</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>uniqueOrder</TD><TD style="text-align: right;">0.253</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.253</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>unlistGrl</TD><TD style="text-align: right;">0.05</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.05</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>widthPerGroup</TD><TD style="text-align: right;">0.054</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.054</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>windowCoveragePlot</TD><TD style="text-align: right;">0.788</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.804</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>windowPerGroup</TD><TD style="text-align: right;">0.494</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.494</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>windowPerReadLength</TD><TD style="text-align: right;">1.127</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">1.159</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
