<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for Repitools on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><B>[<A href="..#R">R</A>]</B></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-10 10:04:39 -0000 (Wed, 10 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-03-12&nbsp;r83975)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">6211</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for Repitools on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the Repitools package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/Repitools.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1654</B>/2194</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">Repitools</A>&nbsp;1.47.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/Repitools">landing page</A>)</SPAN><BR>Mark Robinson<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-08&nbsp;19:11:19&nbsp;-0000&nbsp;(Mon,&nbsp;08&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/Repitools</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">75707ac</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:20:10&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: Repitools</TD></TR>
<TR><TD><B>Version</B>: 1.47.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:Repitools.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings Repitools_1.47.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-10 02:05:21 -0000 (Wed, 10 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-10 02:16:20 -0000 (Wed, 10 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 658.9 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: Repitools.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 1</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:Repitools.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings Repitools_1.47.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/Repitools.Rcheck’
* using R Under development (unstable) (2023-03-12 r83975)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘Repitools/DESCRIPTION’ ... OK
* this is package ‘Repitools’ version ‘1.47.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘Repitools’ can be installed ... OK
* used C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... WARNING
checkRd: (5) AdjustedCopyEstimate-class.Rd:16-34: \item in \describe must have non-empty label
checkRd: (5) BayMethList-class.Rd:58-88: \item in \describe must have non-empty label
checkRd: (5) ClusteredScoresList-class.Rd:29-50: \item in \describe must have non-empty label
checkRd: (5) ClusteredScoresList-class.Rd:57-60: \item in \describe must have non-empty label
checkRd: (5) ClusteredScoresList-class.Rd:61-64: \item in \describe must have non-empty label
checkRd: (5) ClusteredScoresList-class.Rd:65-68: \item in \describe must have non-empty label
checkRd: (5) ClusteredScoresList-class.Rd:75-78: \item in \describe must have non-empty label
checkRd: (5) CopyEstimate-class.Rd:14-23: \item in \describe must have non-empty label
checkRd: (5) GCAdjustParams-class.Rd:14-37: \item in \describe must have non-empty label
checkRd: (5) ScoresList-class.Rd:24-27: \item in \describe must have non-empty label
checkRd: (5) ScoresList-class.Rd:28-31: \item in \describe must have non-empty label
checkRd: (5) ScoresList-class.Rd:32-35: \item in \describe must have non-empty label
checkRd: (5) ScoresList-class.Rd:42-45: \item in \describe must have non-empty label
checkRd: (5) ScoresList-class.Rd:46-49: \item in \describe must have non-empty label
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... OK
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                    user system elapsed
methylEst         83.829 12.777  30.141
empBayes          44.441  8.862  24.448
sequenceCalc      11.748  0.708  12.455
cpgDensityCalc    11.593  0.844  12.449
BayMethList-class  8.511  0.460   8.973
determineOffset    7.528  0.388   7.917
maskOut            5.482  0.456   5.938
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘tests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘Repitools_vignette.Rnw’ using ‘UTF-8’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 WARNING
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/Repitools.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">Repitools.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL Repitools
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library’
* installing *source* package ‘Repitools’ ...
** using staged installation
** libs
using C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG   -I/usr/local/include   -D R_NO_REMAP -I. -fPIC  -g -O2  -c const.c -o const.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG   -I/usr/local/include   -D R_NO_REMAP -I. -fPIC  -g -O2  -c hyp2f1.c -o hyp2f1.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG   -I/usr/local/include   -D R_NO_REMAP -I. -fPIC  -g -O2  -c mtherr.c -o mtherr.o
gcc -shared -L/usr/local/lib -o Repitools.so const.o hyp2f1.o mtherr.o
installing to /home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/00LOCK-Repitools/00new/Repitools/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (Repitools)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">Repitools.Rcheck/tests/tests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; require(&quot;Repitools&quot;)
Loading required package: Repitools
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

&gt; require(&quot;BSgenome.Hsapiens.UCSC.hg18&quot;)
Loading required package: BSgenome.Hsapiens.UCSC.hg18
Loading required package: BSgenome
Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
Loading required package: GenomicRanges
Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: rtracklayer

Attaching package: &#x27;rtracklayer&#x27;

The following object is masked from &#x27;package:Repitools&#x27;:

    blocks

&gt; options(warn = -1)
&gt; 
&gt; probes &lt;- data.frame(chr = c(&quot;chr1&quot;, &quot;chr9&quot;, &quot;chrY&quot;, &quot;chr1&quot;, &quot;chr21&quot;, &quot;chr6&quot;, &quot;chr6&quot;, &quot;chr2&quot;, &quot;chrX&quot;, &quot;chr11&quot;), position = c(10000, 5500, 100000, 11000, 20000000, 500100, 499900, 700000, 9900, 90000), strand = c(&#x27;+&#x27;, &#x27;+&#x27;, &#x27;+&#x27;, &#x27;+&#x27;, &#x27;-&#x27;, &#x27;-&#x27;, &#x27;-&#x27;, &#x27;+&#x27;, &#x27;-&#x27;, &#x27;+&#x27;), stringsAsFactors = FALSE)
&gt; genes &lt;- data.frame(chr = c(&quot;chr1&quot;, &quot;chr9&quot;, &quot;chr11&quot;, &quot;chr1&quot;, &quot;chr11&quot;, &quot;chr6&quot;, &quot;chr6&quot;, &quot;chr22&quot;, &quot;chrY&quot;, &quot;chr21&quot;), start = c(10000, 7900, 950000, 10500, 74000000, 450000, 5000000, 44000000, 1500, 9800000), end = c(12500, 9500, 1000000, 14500, 75000000, 500000, 9000000, 45000000, 3000, 10000000), strand = c(&#x27;+&#x27;, &#x27;-&#x27;, &#x27;-&#x27;, &#x27;+&#x27;, &#x27;-&#x27;, &#x27;-&#x27;, &#x27;-&#x27;, &#x27;+&#x27;, &#x27;+&#x27;, &#x27;-&#x27;), name = paste(&quot;Gene&quot;, 1:10), stringsAsFactors = FALSE)
&gt; 
&gt; crossMatch &lt;- annotationLookup(probes, genes, 5000, 5000)
Processing mapping between probes and features.
Mapping done.
&gt; correctCrossMatch &lt;- list(indexes = list(`Gene 1` = as.integer(c(1, 4)), `Gene 2` = as.integer(2), `Gene 3` = integer(), `Gene 4` = as.integer(c(1, 4)), `Gene 5` = integer(), `Gene 6` = as.integer(c(6, 7)), `Gene 7` = integer(), `Gene 8` = integer(), `Gene 9` = integer(), `Gene 10` = integer()), offsets = list(`Gene 1` = as.integer(c(0, 1000)), `Gene 2` = as.integer(4000), `Gene 3` = numeric(), `Gene 4` = as.integer(c(-500, 500)), `Gene 5` = numeric(), `Gene 6` = as.integer(c(-100, 100)), `Gene 7` = numeric(), `Gene 8` = numeric(), `Gene 9` = numeric(), `Gene 10` = numeric()))
&gt; names(correctCrossMatch$offsets$`Gene 1`) &lt;- c(1, 4)
&gt; names(correctCrossMatch$offsets$`Gene 2`) &lt;- c(2)
&gt; names(correctCrossMatch$offsets$`Gene 4`) &lt;- c(1, 4)
&gt; names(correctCrossMatch$offsets$`Gene 6`) &lt;- c(6, 7)
&gt; names(correctCrossMatch$offsets$`Gene 8`) &lt;- character()
&gt; names(correctCrossMatch$offsets$`Gene 9`) &lt;- character()
&gt; 
&gt; if(!isTRUE(all.equal(crossMatch, correctCrossMatch))) 
+ 	stop(&quot;Error in annotationLookup function.&quot;)
&gt; cat(&quot;anontationLookup tested fine.\n&quot;)
anontationLookup tested fine.
&gt; 
&gt; lookupTable &lt;- makeWindowLookupTable(crossMatch$indexes, crossMatch$offsets, starts = seq(-5000, 4900, 100), ends = seq(-4900, 5000, 100))
&gt; correctLookupTable &lt;- matrix(NA, nrow = 10, ncol = 100, dimnames = list(genes$names, seq(-4950, 4950, 100)))
&gt; correctLookupTable[1, c(50, 51)] &lt;- 1
&gt; correctLookupTable[1, c(60, 61)] &lt;- 4
&gt; correctLookupTable[2, c(90, 91)] &lt;- 2
&gt; correctLookupTable[4, c(45, 46)] &lt;- 1
&gt; correctLookupTable[4, c(55, 56)] &lt;- 4
&gt; correctLookupTable[6, c(49, 50)] &lt;- 6
&gt; correctLookupTable[6, c(51, 52)] &lt;- 7
&gt; 
&gt; if(!all(lookupTable == correctLookupTable, na.rm = TRUE))
+     stop(&quot;Error in makeWindowLookupTable function&quot;)
&gt; cat(&quot;makeWindowLookupTable tested fine.\n&quot;)
makeWindowLookupTable tested fine.
&gt; 
&gt; cpgDensity &lt;- cpgDensityCalc(genes, organism = Hsapiens, window = 500, w.function=&quot;linear&quot;)
CpG density calculated for a sample.
&gt; if(!isTRUE(all.equal(cpgDensity, c(5.784, 7.620, 5.828, 2.928, 2.080, 1.252, 0.000, 7.404, 3.928, 0.000))))
+     stop(&quot;cpgDensityCalc not working for window = 500, scaling = linear&quot;)
&gt; cpgDensity &lt;- cpgDensityCalc(genes, window = 100, w.function = &quot;log&quot;, organism = Hsapiens)
CpG density calculated for a sample.
&gt; if(!isTRUE(all.equal(round(cpgDensity, 3), c(2.424, 1.882, 1.436, 0.084, 0.379, 0.000, 0.000, 0.263, 1.392, 0.000))))
+     stop(&quot;cpgDensityCalc not working for window = 100, scaling = log&quot;)
&gt; cpgDensity &lt;- cpgDensityCalc(genes, window = 1000, w.function = &quot;exp&quot;, organism = Hsapiens)
CpG density calculated for a sample.
&gt; if(!isTRUE(all.equal(round(cpgDensity, 3), c(4.874, 5.835, 4.999, 2.239, 1.567, 0.851, 0.054, 5.589, 3.229,0.062))))
+     stop(&quot;cpgDensityCalc not working for window = 1000, scaling = exp&quot;)
&gt; cpgDensity &lt;- cpgDensityCalc(genes, window = 500, w.function = &quot;none&quot;, organism = Hsapiens)
CpG density calculated for a sample.
&gt; if(!isTRUE(all.equal(cpgDensity, c(11, 14, 16, 6, 4, 2, 0, 15, 9, 0))))
+     stop(&quot;cpgDensityCalc not working for window = 500, scaling = none&quot;)
&gt; cat(&quot;cpgDensityCalc tested fine.\n&quot;)
cpgDensityCalc tested fine.
&gt; 
&gt; GCpercent &lt;- gcContentCalc(genes, Hsapiens, 500)
Calculating GC content.
&gt; if(!isTRUE(all.equal(GCpercent, c(0.504, 0.586, 0.560, 0.470, 0.540, 0.304, 0.356, 0.638, 0.444, 0.388))))
+     stop(&quot;Error in gcContentCalc function&quot;)
&gt; cat(&quot;gcContentCalc tested fine.\n&quot;)
gcContentCalc tested fine.
&gt; 
&gt; findsCount &lt;- sequenceCalc(genes, Hsapiens, 500, pattern = &quot;AATT&quot;)
&gt; if(!isTRUE(all.equal(findsCount, c(1, 1, 0, 2, 1, 8, 2, 0, 4, 10))))
+     stop(&quot;Error in sequenceCalc function counting task&quot;)
&gt; 
&gt; findsPlaces &lt;- sequenceCalc(genes, Hsapiens, 500, pattern = &quot;AATT&quot;, positions = TRUE)
&gt; correctPlaces &lt;- list(-62, 181, NULL, c(-140, -98), 231, c(-219, -146, -88, -12, 12, 182, 209, 214), c(-61, 60), NULL, c(-115, -30, 11, 80), c(-238, -228, -202, -189, -177, -106, -21, 148, 158, 238))
&gt; if(!isTRUE(all.equal(findsPlaces, correctPlaces)))
+     stop(&quot;Error in sequenceCalc function positions task&quot;)
&gt; cat(&quot;sequenceCalc tested fine.\n&quot;)
sequenceCalc tested fine.
&gt; cat(&quot;All tests passed.\n&quot;)
All tests passed.
&gt; 
&gt; proc.time()
   user  system elapsed 
115.519   6.423 121.951 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">Repitools.Rcheck/Repitools-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>BAM2GRanges</TD><TD style="text-align: right;">0.173</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.176</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>BayMethList-class</TD><TD style="text-align: right;">8.511</TD><TD style="text-align: right;">0.460</TD><TD style="text-align: right;">8.973</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GCadjustCopy</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GCbiasPlots</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>QdnaData</TD><TD style="text-align: right;">0.107</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.110</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>abcdDNA</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>absoluteCN</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annoDF2GR</TD><TD style="text-align: right;">0.025</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annoGR2DF</TD><TD style="text-align: right;">0.149</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.157</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotationBlocksCounts</TD><TD style="text-align: right;">0.167</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.194</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotationBlocksLookup</TD><TD style="text-align: right;">0.063</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.075</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotationCounts</TD><TD style="text-align: right;">0.156</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.165</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotationLookup</TD><TD style="text-align: right;">0.134</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.139</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>binPlots</TD><TD style="text-align: right;">1.637</TD><TD style="text-align: right;">0.076</TD><TD style="text-align: right;">1.712</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>blocksStats</TD><TD style="text-align: right;">0.238</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.242</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>checkProbes</TD><TD style="text-align: right;">0.156</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.156</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>chromosomeCNplots</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>clusterPlots</TD><TD style="text-align: right;">0.953</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.969</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cpgDensityCalc</TD><TD style="text-align: right;">11.593</TD><TD style="text-align: right;"> 0.844</TD><TD style="text-align: right;">12.449</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cpgDensityPlot</TD><TD style="text-align: right;">4.159</TD><TD style="text-align: right;">0.132</TD><TD style="text-align: right;">4.291</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>determineOffset</TD><TD style="text-align: right;">7.528</TD><TD style="text-align: right;">0.388</TD><TD style="text-align: right;">7.917</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>empBayes</TD><TD style="text-align: right;">44.441</TD><TD style="text-align: right;"> 8.862</TD><TD style="text-align: right;">24.448</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>enrichmentCalc</TD><TD style="text-align: right;">1.373</TD><TD style="text-align: right;">0.192</TD><TD style="text-align: right;">1.566</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>enrichmentPlot</TD><TD style="text-align: right;">2.481</TD><TD style="text-align: right;">0.368</TD><TD style="text-align: right;">2.850</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>featureBlocks</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>featureScores</TD><TD style="text-align: right;">0.694</TD><TD style="text-align: right;">0.168</TD><TD style="text-align: right;">0.862</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findClusters</TD><TD style="text-align: right;">2.108</TD><TD style="text-align: right;">0.308</TD><TD style="text-align: right;">2.416</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>gcContentCalc</TD><TD style="text-align: right;">3.193</TD><TD style="text-align: right;">0.344</TD><TD style="text-align: right;">3.537</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genQC</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genomeBlocks</TD><TD style="text-align: right;">0.074</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.085</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getProbePositionsDf</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getSampleOffsets</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>hyper</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loadPairFile</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loadSampleDirectory</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeWindowLookupTable</TD><TD style="text-align: right;">0.084</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.084</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mappabilityCalc</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>maskOut</TD><TD style="text-align: right;">5.482</TD><TD style="text-align: right;">0.456</TD><TD style="text-align: right;">5.938</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mergeReplicates</TD><TD style="text-align: right;">1.528</TD><TD style="text-align: right;">0.124</TD><TD style="text-align: right;">1.651</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>methylEst</TD><TD style="text-align: right;">83.829</TD><TD style="text-align: right;">12.777</TD><TD style="text-align: right;">30.141</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>multiHeatmap</TD><TD style="text-align: right;">0.051</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.051</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotClusters</TD><TD style="text-align: right;">0.173</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.176</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotQdnaByCN</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>processNDF</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>profilePlots</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>regionStats</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>relativeCN</TD><TD style="text-align: right;">0.099</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.105</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sequenceCalc</TD><TD style="text-align: right;">11.748</TD><TD style="text-align: right;"> 0.708</TD><TD style="text-align: right;">12.455</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>setCNVOffsets</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>summarizeScores</TD><TD style="text-align: right;">0.735</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">0.778</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>writeWig</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
