<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for GenomicAlignments on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><B>[<A href="..#G">G</A>]</B></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-10 10:04:31 -0000 (Wed, 10 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-03-12&nbsp;r83975)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">6211</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for GenomicAlignments on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the GenomicAlignments package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/GenomicAlignments.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>789</B>/2194</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">GenomicAlignments</A>&nbsp;1.37.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/GenomicAlignments">landing page</A>)</SPAN><BR>Hervé Pagès<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-08&nbsp;19:11:19&nbsp;-0000&nbsp;(Mon,&nbsp;08&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/GenomicAlignments</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">4237abd</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:29:36&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: GenomicAlignments</TD></TR>
<TR><TD><B>Version</B>: 1.37.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:GenomicAlignments.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings GenomicAlignments_1.37.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-09 16:46:25 -0000 (Tue, 09 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-09 16:55:10 -0000 (Tue, 09 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 524.7 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: GenomicAlignments.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:GenomicAlignments.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings GenomicAlignments_1.37.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/GenomicAlignments.Rcheck’
* using R Under development (unstable) (2023-03-12 r83975)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘GenomicAlignments/DESCRIPTION’ ... OK
* this is package ‘GenomicAlignments’ version ‘1.37.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Depends: includes the non-default packages:
  &#x27;BiocGenerics&#x27;, &#x27;S4Vectors&#x27;, &#x27;IRanges&#x27;, &#x27;GenomeInfoDb&#x27;,
  &#x27;GenomicRanges&#x27;, &#x27;SummarizedExperiment&#x27;, &#x27;Biostrings&#x27;, &#x27;Rsamtools&#x27;
Adding so many packages to the search path is excessive and importing
selectively is preferable.
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘GenomicAlignments’ can be installed ... OK
* used C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... NOTE
Packages listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘methods’ ‘BiocGenerics’ ‘S4Vectors’ ‘IRanges’ ‘GenomicRanges’ ‘Biostrings’ ‘Rsamtools’
A package should be listed in only one of these fields.
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... NOTE
Problems with news in ‘NEWS’:
  Cannot process chunk/lines:
    No NEW FEATURES or SIGNIFICANT USER-VISIBLE CHANGES or BUG FIXES since
  Cannot process chunk/lines:
    version 1.18.0
  Cannot process chunk/lines:
    No NEW FEATURES or SIGNIFICANT USER-VISIBLE CHANGES or BUG FIXES since
  Cannot process chunk/lines:
    version 1.16.0
  Cannot process chunk/lines:
    The first version of GenomicAlignments was included in Bioconductor 2.14. 
  Cannot process chunk/lines:
    The package was created from existing code in IRanges, ShortRead, 
  Cannot process chunk/lines:
    Rsamtools and GenomicRanges. 
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
&#x27;:::&#x27; calls which should be &#x27;::&#x27;:
  ‘S4Vectors:::makeClassinfoRowForCompactPrinting’
  ‘S4Vectors:::makePrettyMatrixForCompactPrinting’
  See the note in ?`:::` about the use of this operator.
Unexported objects imported by &#x27;:::&#x27; calls:
  ‘Rsamtools:::.BamViews_delegate’ ‘Rsamtools:::.findMateWithinGroups’
  ‘Rsamtools:::.load_bamcols_from_scanBam_res’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... WARNING
checkRd: (5) GAlignmentPairs-class.Rd:90-94: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:102-133: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:134-137: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:138-143: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:144-160: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:161-172: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:173-179: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:180-183: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:184-188: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:189-193: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:194-203: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:204-210: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:211-216: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:217-222: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:223-232: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:240-244: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:252-257: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:258-264: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:272-298: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:299-328: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:329-336: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:337-340: \item in \describe must have non-empty label
checkRd: (5) GAlignmentPairs-class.Rd:348-359: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:148-154: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:162-165: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:166-171: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:172-179: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:180-183: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:184-190: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:191-195: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:196-202: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:203-213: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:214-221: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:222-227: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:228-232: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:233-241: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:242-248: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:249-254: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:255-260: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:261-270: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:278-292: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:293-335: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:336-344: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:351-357: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:365-369: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:375-381: \item in \describe must have non-empty label
checkRd: (5) GAlignments-class.Rd:388-400: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:67-70: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:78-81: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:82-85: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:86-90: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:91-94: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:95-99: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:100-105: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:106-112: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:113-118: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:119-123: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:124-129: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:130-134: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:135-139: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:140-146: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:147-152: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:153-158: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:159-162: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:170-187: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:188-206: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:207-215: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:216-222: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:223-228: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:236-240: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:241-244: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:245-250: \item in \describe must have non-empty label
checkRd: (5) GAlignmentsList-class.Rd:256-262: \item in \describe must have non-empty label
checkRd: (5) GappedReads-class.Rd:40-44: \item in \describe must have non-empty label
checkRd: (5) OverlapEncodings-class.Rd:133-137: \item in \describe must have non-empty label
checkRd: (5) OverlapEncodings-class.Rd:138-155: \item in \describe must have non-empty label
checkRd: (5) OverlapEncodings-class.Rd:156-182: \item in \describe must have non-empty label
checkRd: (5) OverlapEncodings-class.Rd:183-185: \item in \describe must have non-empty label
checkRd: (5) OverlapEncodings-class.Rd:186-192: \item in \describe must have non-empty label
checkRd: (5) OverlapEncodings-class.Rd:200-204: \item in \describe must have non-empty label
checkRd: (5) OverlapEncodings-class.Rd:213-227: \item in \describe must have non-empty label
checkRd: (5) OverlapEncodings-class.Rd:228-235: \item in \describe must have non-empty label
checkRd: (5) OverlapEncodings-class.Rd:236-245: \item in \describe must have non-empty label
checkRd: (5) OverlapEncodings-class.Rd:246-261: \item in \describe must have non-empty label
checkRd: (5) summarizeOverlaps-methods.Rd:226-246: \item in \describe must have non-empty label
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘GenomicAlignments-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: junctions-methods
&gt; ### Title: Extract junctions from genomic alignments
&gt; ### Aliases: junctions-methods junctions junctions,GAlignments-method
&gt; ###   junctions,GAlignmentPairs-method junctions,GAlignmentsList-method
&gt; ###   NATURAL_INTRON_MOTIFS summarizeJunctions readTopHatJunctions
&gt; ###   readSTARJunctions
&gt; ### Keywords: methods manip
&gt; 
&gt; ### ** Examples
&gt; 
&gt; library(RNAseqData.HNRNPC.bam.chr14)
&gt; bamfile &lt;- RNAseqData.HNRNPC.bam.chr14_BAMFILES[1]
&gt; 
&gt; ## ---------------------------------------------------------------------
&gt; ## A. junctions()
&gt; ## ---------------------------------------------------------------------
&gt; 
&gt; gal &lt;- readGAlignments(bamfile)
&gt; table(njunc(gal))  # some alignments have 3 junctions!

     0      1      2      3 
607197 189110   4153     24 
&gt; juncs &lt;- junctions(gal)
&gt; juncs
GRangesList object of length 800484:
[[1]]
GRanges object with 0 ranges and 0 metadata columns:
   seqnames    ranges strand
      &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt;
  -------
  seqinfo: 93 sequences from an unspecified genome

[[2]]
GRanges object with 0 ranges and 0 metadata columns:
   seqnames    ranges strand
      &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt;
  -------
  seqinfo: 93 sequences from an unspecified genome

[[3]]
GRanges object with 0 ranges and 0 metadata columns:
   seqnames    ranges strand
      &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt;
  -------
  seqinfo: 93 sequences from an unspecified genome

...
&lt;800481 more elements&gt;
&gt; 
&gt; stopifnot(identical(unname(elementNROWS(juncs)), njunc(gal)))
&gt; 
&gt; galp &lt;- readGAlignmentPairs(bamfile)
Warning in .make_GAlignmentPairs_from_GAlignments(gal, strandMode = strandMode,  :
    376 alignments with ambiguous pairing were dumped.
    Use &#x27;getDumpedAlignments()&#x27; to retrieve them from the dump environment.
&gt; juncs &lt;- junctions(galp)
&gt; juncs
GRangesList object of length 400054:
[[1]]
GRanges object with 0 ranges and 0 metadata columns:
   seqnames    ranges strand
      &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt;
  -------
  seqinfo: 93 sequences from an unspecified genome

[[2]]
GRanges object with 0 ranges and 0 metadata columns:
   seqnames    ranges strand
      &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt;
  -------
  seqinfo: 93 sequences from an unspecified genome

[[3]]
GRanges object with 0 ranges and 0 metadata columns:
   seqnames    ranges strand
      &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt;
  -------
  seqinfo: 93 sequences from an unspecified genome

...
&lt;400051 more elements&gt;
&gt; 
&gt; stopifnot(identical(unname(elementNROWS(juncs)), njunc(galp)))
&gt; 
&gt; ## ---------------------------------------------------------------------
&gt; ## B. summarizeJunctions()
&gt; ## ---------------------------------------------------------------------
&gt; 
&gt; ## By default, only the &quot;score&quot;, &quot;plus_score&quot;, and &quot;minus_score&quot;
&gt; ## metadata columns are returned:
&gt; junc_summary &lt;- summarizeJunctions(gal)
&gt; junc_summary
GRanges object with 4635 ranges and 3 metadata columns:
         seqnames              ranges strand |     score plus_score minus_score
            &lt;Rle&gt;           &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt;  &lt;integer&gt;   &lt;integer&gt;
     [1]    chr14   19650127-19652243      * |         4          2           2
     [2]    chr14   19650127-19653624      * |         1          1           0
     [3]    chr14   19652355-19653624      * |         8          7           1
     [4]    chr14   19652355-19653657      * |         1          1           0
     [5]    chr14   19653773-19653892      * |         9          5           4
     ...      ...                 ...    ... .       ...        ...         ...
  [4631]    chr14 106912703-106922227      * |         1          0           1
  [4632]    chr14 106938165-106938301      * |        10          2           8
  [4633]    chr14 106938645-106944774      * |        24          7          17
  [4634]    chr14 106944969-106950170      * |         7          6           1
  [4635]    chr14 106950323-106960260      * |         1          1           0
  -------
  seqinfo: 93 sequences from an unspecified genome
&gt; 
&gt; ## The &quot;score&quot; metadata column reports the total number of alignments
&gt; ## crossing each junction, i.e., that have the junction encoded in their
&gt; ## CIGAR:
&gt; median(mcols(junc_summary)$score)
[1] 12
&gt; 
&gt; ## The &quot;plus_score&quot; and &quot;minus_score&quot; metadata columns report the
&gt; ## strand-specific number of alignments crossing each junction:
&gt; stopifnot(identical(mcols(junc_summary)$score,
+                     mcols(junc_summary)$plus_score +
+                     mcols(junc_summary)$minus_score))
&gt; 
&gt; ## If &#x27;with.revmap&#x27; is TRUE, the &quot;revmap&quot; metadata column is added to
&gt; ## the output. This metadata column is an IntegerList object represen-
&gt; ## ting the mapping from each element in the ouput (i.e. a junction) to
&gt; ## the corresponding elements in the input &#x27;x&#x27;. Here we&#x27;re going to use
&gt; ## this to compute a &#x27;score2&#x27; for each junction. We obtain this score
&gt; ## by summing the mapping qualities of the alignments crossing the
&gt; ## junction:
&gt; gal &lt;- readGAlignments(bamfile, param=ScanBamParam(what=&quot;mapq&quot;))
&gt; junc_summary &lt;- summarizeJunctions(gal, with.revmap=TRUE)
&gt; junc_score2 &lt;- sum(extractList(mcols(gal)$mapq,
+                                mcols(junc_summary)$revmap))
&gt; mcols(junc_summary)$score2 &lt;- junc_score2
&gt; 
&gt; ## If the name of the reference genome is specified thru the &#x27;genome&#x27;
&gt; ## argument (in which case the corresponding BSgenome data package needs
&gt; ## to be installed), then summarizeJunctions() returns the intron motif
&gt; ## and strand for each junction.
&gt; ## Since the reads in RNAseqData.HNRNPC.bam.chr14 were aligned to
&gt; ## the hg19 genome, the following requires that you have
&gt; ## BSgenome.Hsapiens.UCSC.hg19 installed:
&gt; junc_summary &lt;- summarizeJunctions(gal, with.revmap=TRUE, genome=&quot;hg19&quot;)
Error in .getInstalledPkgnameFromGenome(genome, masked = masked) : 
  Looks like you have more than one installed BSgenome data package
  that matches genome: hg19
  Please disambiguate by specifying the full name of the package you want
  to use (use &#x27;installed.genomes()&#x27; to get the list).
Calls: summarizeJunctions -&gt; &lt;Anonymous&gt; -&gt; .getInstalledPkgnameFromGenome
Execution halted
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘run_unitTests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘GenomicAlignmentsIntroduction.Rnw’ using ‘UTF-8’... OK
  ‘OverlapEncodings.Rnw’ using ‘UTF-8’... OK
  ‘WorkingWithAlignedNucleotides.Rnw’ using ‘UTF-8’... OK
  ‘summarizeOverlaps.Rnw’ using ‘UTF-8’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 1 WARNING, 5 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/GenomicAlignments.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">GenomicAlignments.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL GenomicAlignments
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library’
* installing *source* package ‘GenomicAlignments’ ...
** using staged installation
** libs
using C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/IRanges/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c IRanges_stubs.c -o IRanges_stubs.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/IRanges/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c R_init_GenomicAlignments.c -o R_init_GenomicAlignments.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/IRanges/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c S4Vectors_stubs.c -o S4Vectors_stubs.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/IRanges/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c cigar_utils.c -o cigar_utils.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/IRanges/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c coordinate_mapping_methods.c -o coordinate_mapping_methods.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/IRanges/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c encodeOverlaps_methods.c -o encodeOverlaps_methods.o
gcc -shared -L/usr/local/lib -o GenomicAlignments.so IRanges_stubs.o R_init_GenomicAlignments.o S4Vectors_stubs.o cigar_utils.o coordinate_mapping_methods.o encodeOverlaps_methods.o
installing to /home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/00LOCK-GenomicAlignments/00new/GenomicAlignments/libs
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (GenomicAlignments)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">GenomicAlignments.Rcheck/tests/run_unitTests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; require(&quot;GenomicAlignments&quot;) || stop(&quot;unable to load GenomicRanges package&quot;)
Loading required package: GenomicAlignments
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
Loading required package: GenomicRanges
Loading required package: SummarizedExperiment
Loading required package: MatrixGenerics
Loading required package: matrixStats

Attaching package: &#x27;MatrixGenerics&#x27;

The following objects are masked from &#x27;package:matrixStats&#x27;:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars

Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


Attaching package: &#x27;Biobase&#x27;

The following object is masked from &#x27;package:MatrixGenerics&#x27;:

    rowMedians

The following objects are masked from &#x27;package:matrixStats&#x27;:

    anyMissing, rowMedians

Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: Rsamtools
[1] TRUE
&gt; GenomicAlignments:::.test()


RUNIT TEST PROTOCOL -- Tue May  9 16:51:16 2023 
*********************************************** 
Number of test functions: 42 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
GenomicAlignments RUnit Tests - 42 test functions, 0 errors, 0 failures
Number of test functions: 42 
Number of errors: 0 
Number of failures: 0 
Warning message:
In .make_GAlignmentPairs_from_GAlignments(gal, strandMode = strandMode,  :
    4 alignments with ambiguous pairing were dumped.
    Use &#x27;getDumpedAlignments()&#x27; to retrieve them from the dump environment.
&gt; 
&gt; proc.time()
   user  system elapsed 
 34.097   1.359  35.041 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">GenomicAlignments.Rcheck/GenomicAlignments-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>GAlignmentPairs-class</TD><TD style="text-align: right;">1.730</TD><TD style="text-align: right;">0.560</TD><TD style="text-align: right;">2.291</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GAlignments-class</TD><TD style="text-align: right;">0.363</TD><TD style="text-align: right;">0.076</TD><TD style="text-align: right;">0.438</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GAlignmentsList-class</TD><TD style="text-align: right;">2.360</TD><TD style="text-align: right;">0.136</TD><TD style="text-align: right;">2.496</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GappedReads-class</TD><TD style="text-align: right;">0.110</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.112</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>OverlapEncodings-class</TD><TD style="text-align: right;">1.928</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">1.953</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cigar-utils</TD><TD style="text-align: right;">0.229</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.237</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>coordinate-mapping-methods</TD><TD style="text-align: right;">8.753</TD><TD style="text-align: right;">0.068</TD><TD style="text-align: right;">8.823</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>coverage-methods</TD><TD style="text-align: right;">3.334</TD><TD style="text-align: right;">0.060</TD><TD style="text-align: right;">3.395</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>encodeOverlaps-methods</TD><TD style="text-align: right;">0.037</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.037</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findCompatibleOverlaps-methods</TD><TD style="text-align: right;">1.234</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">1.255</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findMateAlignment</TD><TD style="text-align: right;">0.159</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.159</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findOverlaps-methods</TD><TD style="text-align: right;">0.526</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.531</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findSpliceOverlaps-methods</TD><TD style="text-align: right;">5.752</TD><TD style="text-align: right;">0.104</TD><TD style="text-align: right;">5.862</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>intra-range-methods</TD><TD style="text-align: right;">0.379</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.391</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
