<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for ReUseData on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><B>[<A href="..#R">R</A>]</B></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-31 05:44:42 -0000 (Wed, 31 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4219</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for ReUseData on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the ReUseData package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/ReUseData.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1665</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">ReUseData</A>&nbsp;1.1.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/ReUseData">landing page</A>)</SPAN><BR>Qian Liu<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-29&nbsp;10:19:22&nbsp;-0000&nbsp;(Mon,&nbsp;29&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/ReUseData</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">3a1df6c</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;15:42:19&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: ReUseData</TD></TR>
<TR><TD><B>Version</B>: 1.1.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:ReUseData.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings ReUseData_1.1.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-30 16:49:38 -0000 (Tue, 30 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-30 16:53:01 -0000 (Tue, 30 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 202.3 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: ReUseData.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:ReUseData.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings ReUseData_1.1.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/ReUseData.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘ReUseData/DESCRIPTION’ ... OK
* this is package ‘ReUseData’ version ‘1.1.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .BBSoptions
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘ReUseData’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                user system elapsed
dataUpdate    16.082  1.219  17.300
getCloudData  10.719  0.616  14.994
dataHub-class  8.726  0.551   9.283
getData        4.837  0.342   5.173
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘ReUseData_data.Rmd’ using ‘UTF-8’... OK
  ‘ReUseData_quickStart.Rmd’ using ‘UTF-8’... OK
  ‘ReUseData_recipe.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... ERROR
Error(s) in re-building vignettes:
  ...
--- re-building ‘ReUseData_data.Rmd’ using rmarkdown
Error: processing vignette &#x27;ReUseData_data.Rmd&#x27; failed with diagnostics:
there is no package called ‘BiocStyle’
--- failed re-building ‘ReUseData_data.Rmd’

--- re-building ‘ReUseData_quickStart.Rmd’ using rmarkdown
Error: processing vignette &#x27;ReUseData_quickStart.Rmd&#x27; failed with diagnostics:
there is no package called ‘BiocStyle’
--- failed re-building ‘ReUseData_quickStart.Rmd’

--- re-building ‘ReUseData_recipe.Rmd’ using rmarkdown
Error: processing vignette &#x27;ReUseData_recipe.Rmd&#x27; failed with diagnostics:
there is no package called ‘BiocStyle’
--- failed re-building ‘ReUseData_recipe.Rmd’

SUMMARY: processing the following files failed:
  ‘ReUseData_data.Rmd’ ‘ReUseData_quickStart.Rmd’
  ‘ReUseData_recipe.Rmd’

Error: Vignette re-building failed.
Execution halted

* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 1 NOTE
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/ReUseData.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">ReUseData.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD INSTALL ReUseData
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0/site-library’
* installing *source* package ‘ReUseData’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (ReUseData)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">ReUseData.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(ReUseData)
&gt; 
&gt; test_check(&quot;ReUseData&quot;)
Note: you need to assign a name for the recipe: rcpName &lt;- recipeLoad(&#x27;xx&#x27;)
Data recipe loaded!
Use inputs() to check required input parameters before evaluation.
Check here: https://rcwl.org/dataRecipes/echo_out.html
for user instructions (e.g., eligible input values, data source, etc.)

}[1;30mINFO[0m Final process status is success

Updating data record...
outfile.txt added

Updating data record...
2cb14b3f3e9a90_GRCh38.primary_assembly.genome.fa.1.bt2 added
2cb14b4acef023_GRCh38.primary_assembly.genome.fa.2.bt2 added
2cb14b44b0d5d9_GRCh38.primary_assembly.genome.fa.3.bt2 added
2cb14b383a314b_GRCh38.primary_assembly.genome.fa.4.bt2 added
2cb14b6d565ad9_GRCh38.primary_assembly.genome.fa.rev.1.bt2 added
2cb14b73030d88_GRCh38.primary_assembly.genome.fa.rev.2.bt2 added
2cb14b21f1330f_outfile.txt added
2cb14b1e4209b1_GRCh37_to_GRCh38.chain added
2cb14b78c457ea_GRCh37_to_NCBI34.chain added
2cb14b76c56ad2_GRCh37_to_NCBI35.chain added
2cb14b41ad4a81_GRCh37_to_NCBI36.chain added
2cb14b7e9bfc69_GRCh38_to_GRCh37.chain added
2cb14b4343010a_GRCh38_to_NCBI34.chain added
2cb14b3759e0b5_GRCh38_to_NCBI35.chain added
2cb14b3b08db53_GRCh38_to_NCBI36.chain added
2cb14b79b05f17_NCBI34_to_GRCh37.chain added
2cb14b7eb6ce6b_NCBI34_to_GRCh38.chain added
2cb14b6695a59c_NCBI35_to_GRCh37.chain added
2cb14b2b92bb4_NCBI35_to_GRCh38.chain added
2cb14b4d75fd8f_NCBI36_to_GRCh37.chain added
2cb14b5bc3416a_NCBI36_to_GRCh38.chain added
2cb14b42a66ec2_GRCm38_to_NCBIM36.chain added
2cb14b4de8e201_GRCm38_to_NCBIM37.chain added
2cb14b634a45c5_NCBIM36_to_GRCm38.chain added
2cb14b75406247_NCBIM37_to_GRCm38.chain added
2cb14b3b7033a6_1000G_omni2.5.b37.vcf.gz added
2cb14b4ebfa019_1000G_omni2.5.b37.vcf.gz.tbi added
2cb14b4b59ff2c_Mills_and_1000G_gold_standard.indels.b37.vcf.gz added
2cb14b4844378a_Mills_and_1000G_gold_standard.indels.b37.vcf.gz.tbi added
2cb14b254ad5f6_1000G_omni2.5.hg38.vcf.gz added
2cb14b3af86ec7_1000G_omni2.5.hg38.vcf.gz.tbi added
2cb14b782d21a_Mills_and_1000G_gold_standard.indels.hg38.vcf.gz added
2cb14b7019c619_Mills_and_1000G_gold_standard.indels.hg38.vcf.gz.tbi added
2cb14b7fa944a0_af-only-gnomad.raw.sites.vcf added
2cb14b3fbd0366_af-only-gnomad.raw.sites.vcf.idx added
2cb14b5d7020f2_Mutect2-exome-panel.vcf.idx added
2cb14b72ac5229_Mutect2-WGS-panel-b37.vcf added
2cb14b61ae3675_Mutect2-WGS-panel-b37.vcf.idx added
2cb14b7bb22aa4_small_exac_common_3.vcf added
2cb14b6b70aa13_small_exac_common_3.vcf.idx added
2cb14b5873a148_1000g_pon.hg38.vcf.gz added
2cb14b3d5f7525_1000g_pon.hg38.vcf.gz.tbi added
2cb14b6a0ca67c_af-only-gnomad.hg38.vcf.gz added
2cb14b1bb6a252_af-only-gnomad.hg38.vcf.gz.tbi added
2cb14b74b955db_small_exac_common_3.hg38.vcf.gz added
2cb14b251581cf_small_exac_common_3.hg38.vcf.gz.tbi added
2cb14b15670169_gencode.v41.annotation.gtf added
2cb14b73702446_gencode.v42.annotation.gtf added
2cb14bbab276c_gencode.vM30.annotation.gtf added
2cb14b18202d1e_gencode.vM31.annotation.gtf added
2cb14b40e621d6_gencode.v41.transcripts.fa added
2cb14b676e68d6_gencode.v41.transcripts.fa.fai added
2cb14b5ac69be0_gencode.v42.transcripts.fa added
2cb14becf03d7_gencode.v42.transcripts.fa.fai added
2cb14b4ab8ae9b_gencode.vM30.pc_transcripts.fa added
2cb14b5006fe28_gencode.vM30.pc_transcripts.fa.fai added
2cb14b4a3f377d_gencode.vM31.pc_transcripts.fa added
2cb14b19784eb5_gencode.vM31.pc_transcripts.fa.fai added
2cb14b1b60fd54_GRCh38.primary_assembly.genome.fa.1.ht2 added
2cb14b12836f07_GRCh38.primary_assembly.genome.fa.2.ht2 added
2cb14b3ec324ab_GRCh38.primary_assembly.genome.fa.3.ht2 added
2cb14b56596c1b_GRCh38.primary_assembly.genome.fa.4.ht2 added
2cb14b1a064121_GRCh38.primary_assembly.genome.fa.5.ht2 added
2cb14b2edceac4_GRCh38.primary_assembly.genome.fa.6.ht2 added
2cb14b5602b0bc_GRCh38.primary_assembly.genome.fa.7.ht2 added
2cb14b59c34487_GRCh38.primary_assembly.genome.fa.8.ht2 added
2cb14bc4d0bb6_GRCh38_full_analysis_set_plus_decoy_hla.fa.fai added
2cb14b48af02e5_GRCh38_full_analysis_set_plus_decoy_hla.fa.amb added
2cb14b3b717afd_GRCh38_full_analysis_set_plus_decoy_hla.fa.ann added
2cb14b7ff365a_GRCh38_full_analysis_set_plus_decoy_hla.fa.bwt added
2cb14b341facf8_GRCh38_full_analysis_set_plus_decoy_hla.fa.pac added
2cb14b13e51c45_GRCh38_full_analysis_set_plus_decoy_hla.fa.sa added
2cb14b455eab80_GRCh38_full_analysis_set_plus_decoy_hla.fa added
2cb14b1e2c5375_GRCh38.primary_assembly.genome.fa.fai added
2cb14b2f9bbe97_GRCh38.primary_assembly.genome.fa.amb added
2cb14b3a18015b_GRCh38.primary_assembly.genome.fa.ann added
2cb14b4341d544_GRCh38.primary_assembly.genome.fa.bwt added
2cb14b4502c000_GRCh38.primary_assembly.genome.fa.pac added
2cb14b2d8825a1_GRCh38.primary_assembly.genome.fa.sa added
2cb14b4eecfcb0_GRCh38.primary_assembly.genome.fa added
2cb14b5d22ed1e_hs37d5.fa.fai added
2cb14b6e6e4777_hs37d5.fa.amb added
2cb14b365b6587_hs37d5.fa.ann added
2cb14b37e988ff_hs37d5.fa.bwt added
2cb14b7d3d4b4e_hs37d5.fa.pac added
2cb14b1141422_hs37d5.fa.sa added
2cb14b7f08727_hs37d5.fa added
2cb14b477c82cb_complete_ref_lens.bin added
2cb14b1a8c62d7_ctable.bin added
2cb14b2351847b_ctg_offsets.bin added
2cb14b59fff1d2_duplicate_clusters.tsv added
2cb14b594f8782_info.json added
2cb14b79aaf096_mphf.bin added
2cb14b740632f4_pos.bin added
2cb14b82c7246_pre_indexing.log added
2cb14b4fada152_rank.bin added
2cb14b4dc9777b_ref_indexing.log added
2cb14b14797dfd_refAccumLengths.bin added
2cb14b185ca437_reflengths.bin added
2cb14b93af278_refseq.bin added
2cb14b1c78b457_seq.bin added
2cb14b4c7c5130_versionInfo.json added
2cb14b1d200ebd_salmon_index added
2cb14b61d75fd7_chrLength.txt added
2cb14b6aa8a4a5_chrName.txt added
2cb14b4cbbcd54_chrNameLength.txt added
2cb14b1bef6132_chrStart.txt added
2cb14b2dea79e9_exonGeTrInfo.tab added
2cb14b11be8d55_exonInfo.tab added
2cb14b497786d4_geneInfo.tab added
2cb14b7cd7769a_Genome added
2cb14b6ee17a73_genomeParameters.txt added
2cb14b37e5ce4b_Log.out added
2cb14b3332dc21_SA added
2cb14b26cb0372_SAindex added
2cb14b3523199a_sjdbInfo.txt added
2cb14b3446f043_sjdbList.fromGTF.out.tab added
2cb14b2ebb8a99_sjdbList.out.tab added
2cb14b7c9f9c65_transcriptInfo.tab added
2cb14b4ed3531b_GRCh38.GENCODE.v42_100 added
2cb14b520d0f14_knownGene_hg38.sql added
2cb14b569f8e38_knownGene_hg38.txt added
2cb14b2822da9d_refGene_hg38.sql added
2cb14b4bb7ffab_refGene_hg38.txt added
2cb14b4aa5c12c_knownGene_mm39.sql added
2cb14b304f4ce4_knownGene_mm39.txt added
2cb14b1b65a0fd_refGene_mm39.sql added
2cb14b186f38a7_refGene_mm39.txt added
trying URL &#x27;https://storage.googleapis.com/reusedata/echo/outfile.txt&#x27;
Content type &#x27;text/plain&#x27; length 30 bytes
==================================================
downloaded 30 bytes

trying URL &#x27;https://storage.googleapis.com/reusedata/echo/echo_out_Hello_World!_outfile.yml&#x27;
Content type &#x27;application/x-yaml&#x27; length 151 bytes
==================================================
downloaded 151 bytes

trying URL &#x27;https://storage.googleapis.com/reusedata/echo/echo_out_Hello_World!_outfile.cwl&#x27;
Content type &#x27;application/octet-stream&#x27; length 1152 bytes
==================================================
downloaded 1152 bytes

trying URL &#x27;https://storage.googleapis.com/reusedata/echo/echo_out_Hello_World!_outfile.md5&#x27;
Content type &#x27;application/octet-stream&#x27; length 45 bytes
==================================================
downloaded 45 bytes

trying URL &#x27;https://storage.googleapis.com/reusedata/echo/echo_out_Hello_World!_outfile.sh&#x27;
Content type &#x27;application/x-shellscript&#x27; length 68 bytes
==================================================
downloaded 68 bytes

Data is downloaded: 
/tmp/RtmphXM5jv/test_gcpData/outfile.txt
}[1;30mINFO[0m Final process status is success
NOTE: existing caches will be removed and regenerated!
Updating recipes...
STAR_index.R added
bowtie2_index.R added
echo_out.R added
ensembl_liftover.R added
gcp_broad_gatk_hg19.R added
gcp_broad_gatk_hg38.R added
gcp_gatk_mutect2_b37.R added
gcp_gatk_mutect2_hg38.R added
gencode_annotation.R added
gencode_genome_grch38.R added
gencode_transcripts.R added
hisat2_index.R added
reference_genome.R added
salmon_index.R added
ucsc_database.R added

Data recipe loaded!
Use inputs(ensembl_liftover) to check required input parameters before evaluation.
Check here: https://rcwl.org/dataRecipes/ensembl_liftover.html
for user instructions (e.g., eligible input values, data source, etc.)

[ FAIL 0 | WARN 0 | SKIP 0 | PASS 44 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
 32.164   2.433  36.213 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">ReUseData.Rcheck/ReUseData-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>dataHub-class</TD><TD style="text-align: right;">8.726</TD><TD style="text-align: right;">0.551</TD><TD style="text-align: right;">9.283</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dataSearch</TD><TD style="text-align: right;">1.394</TD><TD style="text-align: right;">0.059</TD><TD style="text-align: right;">1.454</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dataUpdate</TD><TD style="text-align: right;">16.082</TD><TD style="text-align: right;"> 1.219</TD><TD style="text-align: right;">17.300</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getCloudData</TD><TD style="text-align: right;">10.719</TD><TD style="text-align: right;"> 0.616</TD><TD style="text-align: right;">14.994</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getData</TD><TD style="text-align: right;">4.837</TD><TD style="text-align: right;">0.342</TD><TD style="text-align: right;">5.173</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>meta_data</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeHub-class</TD><TD style="text-align: right;">0.266</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.290</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeLoad</TD><TD style="text-align: right;">1.578</TD><TD style="text-align: right;">0.092</TD><TD style="text-align: right;">1.673</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeMake</TD><TD style="text-align: right;">4.557</TD><TD style="text-align: right;">0.255</TD><TD style="text-align: right;">4.808</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeSearch</TD><TD style="text-align: right;">0.745</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.748</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeUpdate</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
