<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for APAlyzer on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.17</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><B>[<A href="..#A">A</A>]</B></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-03-27 05:50:29 -0000 (Mon, 27 Mar 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-03-12&nbsp;r83975)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">6083</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for APAlyzer on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the APAlyzer package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/APAlyzer.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>81</B>/2195</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">APAlyzer</A>&nbsp;1.13.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/APAlyzer">landing page</A>)</SPAN><BR>Ruijia Wang<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-03-24&nbsp;01:40:07&nbsp;-0000&nbsp;(Fri,&nbsp;24&nbsp;Mar&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/APAlyzer</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">7ca6719</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:20:19&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: APAlyzer</TD></TR>
<TR><TD><B>Version</B>: 1.13.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:APAlyzer.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings APAlyzer_1.13.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-03-24 20:48:50 -0000 (Fri, 24 Mar 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-03-24 20:57:05 -0000 (Fri, 24 Mar 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 494.3 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: APAlyzer.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:APAlyzer.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings APAlyzer_1.13.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/APAlyzer.Rcheck’
* using R Under development (unstable) (2023-03-12 r83975)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘APAlyzer/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘APAlyzer’ version ‘1.13.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .BBSoptions
  .travis.yml
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘APAlyzer’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... NOTE
File
  LICENSE
is not mentioned in the DESCRIPTION file.
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... WARNING
&#x27;library&#x27; or &#x27;require&#x27; call not declared from: ‘HybridMTest’
&#x27;library&#x27; or &#x27;require&#x27; call to ‘HybridMTest’ in package code.
  Please use :: or requireNamespace() instead.
  See section &#x27;Suggested packages&#x27; in the &#x27;Writing R Extensions&#x27; manual.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
.GTF2LE: no visible binding for global variable ‘group_name’
.GTF2refUTRraw: no visible binding for global variable ‘GENEID’
.annotatePASRegion: no visible global function definition for
  ‘queryHits’
APABox: no visible binding for global variable ‘APAreg’
APABox: no visible binding for global variable ‘RED’
APAVolcano: no visible binding for global variable ‘APAreg’
APAVolcano: no visible binding for global variable ‘color’
PASEXP_IPA: no visible global function definition for ‘Print’
ThreeMostPairBam: no visible global function definition for ‘Print’
download_testbam: no visible global function definition for
  ‘download.file’
Undefined global functions or variables:
  APAreg GENEID Print RED color download.file group_name queryHits
Consider adding
  importFrom(&quot;utils&quot;, &quot;download.file&quot;)
to your NAMESPACE file.
* checking Rd files ... NOTE
prepare_Rd: download_testbam.Rd:7-8: Dropping empty section \arguments
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘APAlyzer-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: PASEXP_3UTR
&gt; ### Title: PASEXP_3UTR, calculate relative expression of aUTR and cUTR
&gt; ###   regions
&gt; ### Aliases: PASEXP_3UTR
&gt; 
&gt; ### ** Examples
&gt; 
&gt; ## count reads mapped to 3&#x27;UTR APA regions and
&gt; ## calculate relative expression of aUTR and cUTR regions
&gt; ## using forward sequencing
&gt;     library(&quot;TBX20BamSubset&quot;)
Loading required package: Rsamtools
Loading required package: GenomeInfoDb
Loading required package: BiocGenerics

Attaching package: ‘BiocGenerics’

The following objects are masked from ‘package:stats’:

    IQR, mad, sd, var, xtabs

The following objects are masked from ‘package:base’:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: ‘S4Vectors’

The following objects are masked from ‘package:base’:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomicRanges
Loading required package: Biostrings
Loading required package: XVector

Attaching package: ‘Biostrings’

The following object is masked from ‘package:base’:

    strsplit

&gt;     library(&quot;Rsamtools&quot;)
&gt;     library(&quot;GenomicAlignments&quot;)
Loading required package: SummarizedExperiment
Loading required package: MatrixGenerics
Loading required package: matrixStats

Attaching package: ‘MatrixGenerics’

The following objects are masked from ‘package:matrixStats’:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars

Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


Attaching package: ‘Biobase’

The following object is masked from ‘package:MatrixGenerics’:

    rowMedians

The following objects are masked from ‘package:matrixStats’:

    anyMissing, rowMedians

&gt; 	library(&quot;repmis&quot;)
&gt;     flsall = getBamFileList()
&gt; 	URL=&quot;https://github.com/RJWANGbioinfo/PAS_reference_RData/blob/master/&quot;
&gt; 	file=&quot;mm9_REF.RData&quot;
&gt; 	source_data(paste0(URL,file,&quot;?raw=True&quot;))
Downloading data from: https://github.com/RJWANGbioinfo/PAS_reference_RData/blob/master/mm9_REF.RData?raw=True 

Error in curl::curl_fetch_memory(url, handle = handle) : 
  SSL peer certificate or SSH remote key was not OK: [raw.githubusercontent.com] SSL certificate problem: certificate has expired
Calls: source_data ... request_fetch -&gt; request_fetch.write_memory -&gt; &lt;Anonymous&gt;
Execution halted
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’
 ERROR
Running the tests in ‘tests/runTests.R’ failed.
Last 13 lines of output:
  ── Error (&#x27;test_IPA_analysis.R:4:2&#x27;): IPA_analysis works properly ──────────────
  Error in `curl::curl_fetch_memory(url, handle = handle)`: SSL peer certificate or SSH remote key was not OK: [raw.githubusercontent.com] SSL certificate problem: certificate has expired
  Backtrace:
      ▆
   1. └─repmis::source_data(paste0(URL, file, &quot;?raw=True&quot;)) at test_IPA_analysis.R:4:8
   2.   └─repmis:::download_data_intern(url = url, sha1 = sha1, temp_file = temp_file)
   3.     └─httr::GET(url)
   4.       └─httr:::request_perform(req, hu$handle$handle)
   5.         ├─httr:::request_fetch(req$output, req$url, handle)
   6.         └─httr:::request_fetch.write_memory(req$output, req$url, handle)
   7.           └─curl::curl_fetch_memory(url, handle = handle)
  
  [ FAIL 2 | WARN 1 | SKIP 0 | PASS 4 ]
  Error: Test failures
  Execution halted
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘APAlyzer.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... ERROR
Error(s) in re-building vignettes:
  ...
--- re-building ‘APAlyzer.Rmd’ using rmarkdown
Quitting from lines 80-84 (APAlyzer.Rmd) 
Error: processing vignette &#x27;APAlyzer.Rmd&#x27; failed with diagnostics:
SSL peer certificate or SSH remote key was not OK: [raw.githubusercontent.com] SSL certificate problem: certificate has expired
--- failed re-building ‘APAlyzer.Rmd’

SUMMARY: processing the following file failed:
  ‘APAlyzer.Rmd’

Error: Vignette re-building failed.
Execution halted

* checking PDF version of manual ... OK
* DONE

Status: 3 ERRORs, 1 WARNING, 4 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/APAlyzer.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">APAlyzer.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL APAlyzer
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library’
* installing *source* package ‘APAlyzer’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (APAlyzer)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">APAlyzer.Rcheck/tests/runTests.Rout.fail</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; require(&quot;APAlyzer&quot;) || stop(&quot;unable to load Package:APAlyzer&quot;)
Loading required package: APAlyzer
[1] TRUE
&gt; require(&quot;GenomicRanges&quot;) || stop(&quot;unable to load Package:GenomicRanges&quot;)
Loading required package: GenomicRanges
Loading required package: stats4
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
[1] TRUE
&gt; require(&quot;GenomicFeatures&quot;) || stop(&quot;unable to load Package:GenomicFeatures&quot;)
Loading required package: GenomicFeatures
Loading required package: AnnotationDbi
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

[1] TRUE
&gt; require(&quot;GenomicAlignments&quot;) || stop(&quot;unable to load Package:GenomicAlignments&quot;)
Loading required package: GenomicAlignments
Loading required package: SummarizedExperiment
Loading required package: MatrixGenerics
Loading required package: matrixStats

Attaching package: &#x27;matrixStats&#x27;

The following objects are masked from &#x27;package:Biobase&#x27;:

    anyMissing, rowMedians


Attaching package: &#x27;MatrixGenerics&#x27;

The following objects are masked from &#x27;package:matrixStats&#x27;:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars

The following object is masked from &#x27;package:Biobase&#x27;:

    rowMedians

Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: Rsamtools
[1] TRUE
&gt; require(&quot;DESeq2&quot;) || stop(&quot;unable to load Package:DESeq2&quot;)
Loading required package: DESeq2
[1] TRUE
&gt; require(&quot;SummarizedExperiment&quot;) || stop(&quot;unable to load Package:SummarizedExperiment&quot;)
[1] TRUE
&gt; require(&quot;Rsubread&quot;) || stop(&quot;unable to load Package:Rsubread&quot;)
Loading required package: Rsubread
[1] TRUE
&gt; require(&quot;stats&quot;) || stop(&quot;unable to load Package:stats&quot;)
[1] TRUE
&gt; require(&quot;testthat&quot;) || stop(&quot;unable to load testthat&quot;)
Loading required package: testthat
[1] TRUE
&gt; test_check(&quot;APAlyzer&quot;)
[1] &quot;Reading and extracting (Strand = forward-reverse):  untreated3_chr4.bam&quot;
[1] &quot;Exporting: untreated3_chr4.3most.bam&quot;
[ FAIL 2 | WARN 1 | SKIP 0 | PASS 4 ]

══ Failed tests ════════════════════════════════════════════════════════════════
── Error (&#x27;test_APAlyzer.R:4:2&#x27;): APAlyzer works properly ──────────────────────
Error in `curl::curl_fetch_memory(url, handle = handle)`: SSL peer certificate or SSH remote key was not OK: [raw.githubusercontent.com] SSL certificate problem: certificate has expired
Backtrace:
    ▆
 1. └─repmis::source_data(paste0(URL, file, &quot;?raw=True&quot;)) at test_APAlyzer.R:4:8
 2.   └─repmis:::download_data_intern(url = url, sha1 = sha1, temp_file = temp_file)
 3.     └─httr::GET(url)
 4.       └─httr:::request_perform(req, hu$handle$handle)
 5.         ├─httr:::request_fetch(req$output, req$url, handle)
 6.         └─httr:::request_fetch.write_memory(req$output, req$url, handle)
 7.           └─curl::curl_fetch_memory(url, handle = handle)
── Error (&#x27;test_IPA_analysis.R:4:2&#x27;): IPA_analysis works properly ──────────────
Error in `curl::curl_fetch_memory(url, handle = handle)`: SSL peer certificate or SSH remote key was not OK: [raw.githubusercontent.com] SSL certificate problem: certificate has expired
Backtrace:
    ▆
 1. └─repmis::source_data(paste0(URL, file, &quot;?raw=True&quot;)) at test_IPA_analysis.R:4:8
 2.   └─repmis:::download_data_intern(url = url, sha1 = sha1, temp_file = temp_file)
 3.     └─httr::GET(url)
 4.       └─httr:::request_perform(req, hu$handle$handle)
 5.         ├─httr:::request_fetch(req$output, req$url, handle)
 6.         └─httr:::request_fetch.write_memory(req$output, req$url, handle)
 7.           └─curl::curl_fetch_memory(url, handle = handle)

[ FAIL 2 | WARN 1 | SKIP 0 | PASS 4 ]
Error: Test failures
Execution halted
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">APAlyzer.Rcheck/APAlyzer-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>APABox</TD><TD style="text-align: right;">0.486</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.528</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>APAVolcano</TD><TD style="text-align: right;">0.483</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.499</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>APAdiff</TD><TD style="text-align: right;">0.997</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">1.014</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GENEXP_CDS</TD><TD style="text-align: right;">15.788</TD><TD style="text-align: right;"> 3.304</TD><TD style="text-align: right;">18.763</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>PAS2GEF</TD><TD style="text-align: right;"> 0.051</TD><TD style="text-align: right;"> 0.170</TD><TD style="text-align: right;">70.640</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
