<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for ComplexHeatmap on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><B>[<A href="..#C">C</A>]</B></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-16 06:06:52 -0000 (Tue, 16 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4787</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for ComplexHeatmap on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the ComplexHeatmap package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/ComplexHeatmap.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>403</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">ComplexHeatmap</A>&nbsp;2.17.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/ComplexHeatmap">landing page</A>)</SPAN><BR>Zuguang Gu<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-13&nbsp;23:12:22&nbsp;-0000&nbsp;(Sat,&nbsp;13&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/ComplexHeatmap</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">db26314</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:38:43&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: ComplexHeatmap</TD></TR>
<TR><TD><B>Version</B>: 2.17.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:ComplexHeatmap.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings ComplexHeatmap_2.17.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-15 00:43:25 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-15 00:50:39 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 433.6 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: ComplexHeatmap.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:ComplexHeatmap.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings ComplexHeatmap_2.17.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/ComplexHeatmap.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘ComplexHeatmap/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘ComplexHeatmap’ version ‘2.17.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘ComplexHeatmap’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking startup messages can be suppressed ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                    user system elapsed
anno_horizon       5.749  0.040   5.789
AnnotationFunction 5.424  0.092   5.516
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘test-AnnotationFunction.R’
  Running ‘test-ColorMapping-class.R’
  Running ‘test-Heatmap-class.R’
  Running ‘test-Heatmap-cluster.R’
  Running ‘test-HeatmapAnnotation.R’
  Running ‘test-HeatmapList-class.R’
  Running ‘test-Legend.R’
  Running ‘test-SingleAnnotation.R’
  Running ‘test-annotation_axis.R’
  Running ‘test-dendrogram.R’
  Running ‘test-gridtext.R’
  Running ‘test-interactive.R’
  Running ‘test-multiple-page.R’
  Running ‘test-oncoPrint.R’
  Running ‘test-pheatmap.R’
  Running ‘test-textbox.R’
  Running ‘test-upset.R’
  Running ‘test-utils.R’
  Running ‘testthat-all.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘complex_heatmap.rmd’... OK
  ‘most_probably_asked_questions.Rmd’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: OK

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">ComplexHeatmap.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD INSTALL ComplexHeatmap
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0-bin/site-library’
* installing *source* package ‘ComplexHeatmap’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (ComplexHeatmap)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">ComplexHeatmap.Rcheck/tests/test-annotation_axis.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(ComplexHeatmap)
Loading required package: grid
========================================
ComplexHeatmap version 2.17.0
Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/
Github page: https://github.com/jokergoo/ComplexHeatmap
Documentation: http://jokergoo.github.io/ComplexHeatmap-reference

If you use it in published research, please cite either one:
- Gu, Z. Complex Heatmap Visualization. iMeta 2022.
- Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional 
    genomic data. Bioinformatics 2016.


The new InteractiveComplexHeatmap package can directly export static 
complex heatmaps into an interactive Shiny app with zero effort. Have a try!

This message can be suppressed by:
  suppressPackageStartupMessages(library(ComplexHeatmap))
========================================

&gt; 
&gt; 
&gt; gb = annotation_axis_grob(at = 1:5, labels = month.name[1:5], labels_rot = 0, 
+     side = &quot;left&quot;, facing = &quot;outside&quot;)
&gt; grid.newpage()
&gt; pushViewport(viewport(xscale = c(0, 4), yscale = c(0, 6), width = 0.6, height = 0.6))
&gt; grid.rect()
&gt; grid.text(&#x27;side = &quot;left&quot;, facing = &quot;outside&quot;&#x27;)
&gt; grid.draw(gb)
&gt; popViewport()
&gt; 
&gt; gb = annotation_axis_grob(at = 1:5, labels = month.name[1:5], labels_rot = 0, 
+     side = &quot;left&quot;, facing = &quot;inside&quot;)
&gt; grid.newpage()
&gt; pushViewport(viewport(xscale = c(0, 4), yscale = c(0, 6), width = 0.6, height = 0.6))
&gt; grid.rect()
&gt; grid.text(&#x27;side = &quot;left&quot;, facing = &quot;inside&quot;&#x27;)
&gt; grid.draw(gb)
&gt; popViewport()
&gt; 
&gt; gb = annotation_axis_grob(at = 1:5, labels = month.name[1:5], labels_rot = 0, 
+     side = &quot;right&quot;, facing = &quot;outside&quot;)
&gt; grid.newpage()
&gt; pushViewport(viewport(xscale = c(0, 4), yscale = c(0, 6), width = 0.6, height = 0.6))
&gt; grid.rect()
&gt; grid.text(&#x27;side = &quot;right&quot;, facing = &quot;outside&quot;&#x27;)
&gt; grid.draw(gb)
&gt; popViewport()
&gt; 
&gt; gb = annotation_axis_grob(at = 1:5, labels = month.name[1:5], labels_rot = 0, 
+     side = &quot;right&quot;, facing = &quot;inside&quot;)
&gt; grid.newpage()
&gt; pushViewport(viewport(xscale = c(0, 4), yscale = c(0, 6), width = 0.6, height = 0.6))
&gt; grid.rect()
&gt; grid.text(&#x27;side = &quot;right&quot;, facing = &quot;inside&quot;&#x27;)
&gt; grid.draw(gb)
&gt; popViewport()
&gt; 
&gt; gb = annotation_axis_grob(at = 1:3, labels = month.name[1:3], labels_rot = 0, 
+     side = &quot;top&quot;, facing = &quot;outside&quot;)
&gt; grid.newpage()
&gt; pushViewport(viewport(xscale = c(0, 4), yscale = c(0, 6), width = 0.6, height = 0.6))
&gt; grid.rect()
&gt; grid.text(&#x27;side = &quot;top&quot;, facing = &quot;outside&quot;&#x27;)
&gt; grid.draw(gb)
&gt; popViewport()
&gt; 
&gt; gb = annotation_axis_grob(at = 1:3, labels = month.name[1:3], labels_rot = 90, 
+     side = &quot;top&quot;, facing = &quot;outside&quot;)
&gt; grid.newpage()
&gt; pushViewport(viewport(xscale = c(0, 4), yscale = c(0, 6), width = 0.6, height = 0.6))
&gt; grid.rect()
&gt; grid.text(&#x27;side = &quot;top&quot;, facing = &quot;outside&quot;&#x27;)
&gt; grid.draw(gb)
&gt; popViewport()
&gt; 
&gt; gb = annotation_axis_grob(at = 1:3, labels = month.name[1:3], labels_rot = 45, 
+     side = &quot;top&quot;, facing = &quot;outside&quot;)
&gt; grid.newpage()
&gt; pushViewport(viewport(xscale = c(0, 4), yscale = c(0, 6), width = 0.6, height = 0.6))
&gt; grid.rect()
&gt; grid.text(&#x27;side = &quot;top&quot;, facing = &quot;outside&quot;&#x27;)
&gt; grid.draw(gb)
&gt; popViewport()
&gt; 
&gt; gb = annotation_axis_grob(at = 1:3, labels = month.name[1:3], labels_rot = 0, 
+     side = &quot;top&quot;, facing = &quot;inside&quot;)
&gt; grid.newpage()
&gt; pushViewport(viewport(xscale = c(0, 4), yscale = c(0, 6), width = 0.6, height = 0.6))
&gt; grid.rect()
&gt; grid.text(&#x27;side = &quot;top&quot;, facing = &quot;inside&quot;&#x27;)
&gt; grid.draw(gb)
&gt; popViewport()
&gt; 
&gt; gb = annotation_axis_grob(at = 1:3, labels = month.name[1:3], labels_rot = 0, 
+     side = &quot;bottom&quot;, facing = &quot;outside&quot;)
&gt; grid.newpage()
&gt; pushViewport(viewport(xscale = c(0, 4), yscale = c(0, 6), width = 0.6, height = 0.6))
&gt; grid.rect()
&gt; grid.text(&#x27;side = &quot;bottom&quot;, facing = &quot;outside&quot;&#x27;)
&gt; grid.draw(gb)
&gt; popViewport()
&gt; 
&gt; gb = annotation_axis_grob(at = 1:3, labels = month.name[1:3], labels_rot = 0, 
+     side = &quot;bottom&quot;, facing = &quot;inside&quot;)
&gt; grid.newpage()
&gt; pushViewport(viewport(xscale = c(0, 4), yscale = c(0, 6), width = 0.6, height = 0.6))
&gt; grid.rect()
&gt; grid.text(&#x27;side = &quot;bottom&quot;, facing = &quot;inside&quot;&#x27;)
&gt; grid.draw(gb)
&gt; popViewport()
&gt; 
&gt; grid.newpage()
&gt; pushViewport(viewport(xscale = c(0, 4), yscale = c(0, 6), width = 0.6, height = 0.6))
&gt; gb = annotation_axis_grob(labels_rot = 0, side = &quot;left&quot;, facing = &quot;outside&quot;)
&gt; grid.rect()
&gt; grid.text(&#x27;side = &quot;left&quot;, facing = &quot;outside&quot;&#x27;)
&gt; grid.draw(gb)
&gt; popViewport()
&gt; 
&gt; grid.newpage()
&gt; pushViewport(viewport(xscale = c(0, 4), yscale = c(0, 6), width = 0.6, height = 0.6))
&gt; gb = annotation_axis_grob(side = &quot;left&quot;, direction = &quot;reverse&quot;)
&gt; grid.rect()
&gt; grid.text(&#x27;side = &quot;left&quot;, direction = &quot;reverse&#x27;)
&gt; grid.draw(gb)
&gt; popViewport()
&gt; 
&gt; grid.newpage()
&gt; pushViewport(viewport(xscale = c(0, 4), yscale = c(0, 6), width = 0.6, height = 0.6))
&gt; gb = annotation_axis_grob(side = &quot;bottom&quot;, direction = &quot;reverse&quot;)
&gt; grid.rect()
&gt; grid.text(&#x27;side = &quot;bottom&quot;, direction = &quot;reverse&quot;&#x27;)
&gt; grid.draw(gb)
&gt; popViewport()
&gt; 
&gt; 
&gt; 
&gt; proc.time()
   user  system elapsed 
  2.728   0.131   2.840 
</PRE>
</DIV><P><SPAN class="filename">ComplexHeatmap.Rcheck/tests/test-AnnotationFunction.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(circlize)
========================================
circlize version 0.4.15
CRAN page: https://cran.r-project.org/package=circlize
Github page: https://github.com/jokergoo/circlize
Documentation: https://jokergoo.github.io/circlize_book/book/

If you use it in published research, please cite:
Gu, Z. circlize implements and enhances circular visualization
  in R. Bioinformatics 2014.

This message can be suppressed by:
  suppressPackageStartupMessages(library(circlize))
========================================

&gt; library(ComplexHeatmap)
Loading required package: grid
========================================
ComplexHeatmap version 2.17.0
Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/
Github page: https://github.com/jokergoo/ComplexHeatmap
Documentation: http://jokergoo.github.io/ComplexHeatmap-reference

If you use it in published research, please cite either one:
- Gu, Z. Complex Heatmap Visualization. iMeta 2022.
- Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional 
    genomic data. Bioinformatics 2016.


The new InteractiveComplexHeatmap package can directly export static 
complex heatmaps into an interactive Shiny app with zero effort. Have a try!

This message can be suppressed by:
  suppressPackageStartupMessages(library(ComplexHeatmap))
========================================

&gt; library(GetoptLong)
&gt; 
&gt; if(!exists(&quot;normalize_graphic_param_to_mat&quot;)) {
+ 	normalize_graphic_param_to_mat = ComplexHeatmap:::normalize_graphic_param_to_mat
+ }
&gt; 
&gt; if(!exists(&quot;height&quot;)) {
+ 	height = ComplexHeatmap:::height
+ }
&gt; 
&gt; if(!exists(&quot;width&quot;)) {
+ 	width = ComplexHeatmap:::width
+ }
&gt; 
&gt; normalize_graphic_param_to_mat(1, nc = 2, nr = 4, &quot;foo&quot;)
     [,1] [,2]
[1,]    1    1
[2,]    1    1
[3,]    1    1
[4,]    1    1
&gt; normalize_graphic_param_to_mat(1:2, nc = 2, nr = 4, &quot;foo&quot;)
     [,1] [,2]
[1,]    1    2
[2,]    1    2
[3,]    1    2
[4,]    1    2
&gt; normalize_graphic_param_to_mat(1:4, nc = 2, nr = 4, &quot;foo&quot;)
     [,1] [,2]
[1,]    1    1
[2,]    2    2
[3,]    3    3
[4,]    4    4
&gt; 
&gt; ### AnnotationFunction constructor #####
&gt; fun = function(index) {
+ 	x = runif(10)
+ 	pushViewport(viewport(xscale = c(0.5, 10.5), yscale = c(0, 1)))
+ 	grid.points(index, x[index])
+ 	popViewport()
+ }
&gt; anno = AnnotationFunction(fun = fun)
&gt; 
&gt; x = runif(10)
&gt; fun = function(index) {
+ 	pushViewport(viewport(xscale = c(0.5, 10.5), yscale = c(0, 1)))
+ 	grid.points(index, x[index])
+ 	popViewport()
+ }
&gt; anno = AnnotationFunction(fun = fun, var_import = &quot;x&quot;)
&gt; anno = AnnotationFunction(fun = fun, var_import = list(x))
&gt; 
&gt; 
&gt; x = runif(10)
&gt; cell_fun = function(i) {
+ 	pushViewport(viewport(yscale = c(0, 1)))
+ 	grid.points(unit(0.5, &quot;npc&quot;), x[i])
+ 	popViewport()
+ }
&gt; anno = AnnotationFunction(cell_fun = cell_fun, var_import = &quot;x&quot;)
&gt; ha = HeatmapAnnotation(foo = anno)
&gt; draw(ha, 1:10, test = T)
&gt; 
&gt; cell_fun = function(i) {
+ 	pushViewport(viewport(xscale = c(0, 1)))
+ 	grid.points(x[i], unit(0.5, &quot;npc&quot;))
+ 	popViewport()
+ }
&gt; anno = AnnotationFunction(cell_fun = cell_fun, var_import = &quot;x&quot;, which = &quot;row&quot;)
&gt; ha = rowAnnotation(foo = anno)
&gt; draw(ha, 1:10, test = T)
&gt; 
&gt; # devAskNewPage(ask = dev.interactive())
&gt; 
&gt; ########### testing anno_simple ############
&gt; anno = anno_simple(1:10)
&gt; draw(anno, test = &quot;as a simple vector&quot;)
&gt; draw(anno[1:5], test = &quot;subset of column annotation&quot;)
&gt; anno = anno_simple(1:10, which = &quot;row&quot;)
&gt; draw(anno, test = &quot;as row annotation&quot;)
&gt; draw(anno[1:5], test = &quot;subste of row annotation&quot;)
&gt; 
&gt; anno = anno_simple(1:10, col = structure(rand_color(10), names = 1:10))
&gt; draw(anno, test = &quot;self-define colors&quot;)
&gt; 
&gt; anno = anno_simple(1:10, border = TRUE)
&gt; draw(anno, test = &quot;border&quot;)
&gt; anno = anno_simple(1:10, gp = gpar(col = &quot;red&quot;))
&gt; draw(anno, test = &quot;gp for the grids&quot;)
&gt; 
&gt; anno = anno_simple(c(1:9, NA))
&gt; draw(anno, test = &quot;vector has NA values&quot;)
&gt; 
&gt; anno = anno_simple(cbind(1:10, 10:1))
&gt; draw(anno, test = &quot;a matrix&quot;)
&gt; draw(anno[1:5], test = &quot;subste of a matrix&quot;)
&gt; 
&gt; anno = anno_simple(1:10, pch = 1, pt_gp = gpar(col = &quot;red&quot;), pt_size = unit(seq(1, 10), &quot;mm&quot;))
&gt; draw(anno, test = &quot;with symbols + pt_gp + pt_size&quot;)
&gt; anno = anno_simple(1:10, pch = 1:10)
&gt; draw(anno, test = &quot;pch is a vector&quot;)
&gt; anno = anno_simple(1:10, pch = c(1:4, NA, 6:8, NA, 10, 11))
&gt; draw(anno, test = &quot;pch has NA values&quot;)
&gt; 
&gt; anno = anno_simple(cbind(1:10, 10:1), pch = 1, pt_gp = gpar(col = &quot;blue&quot;))
&gt; draw(anno, test = &quot;matrix with symbols&quot;)
&gt; anno = anno_simple(cbind(1:10, 10:1), pch = 1:2)
&gt; draw(anno, test = &quot;matrix, length of pch is number of annotations&quot;)
&gt; anno = anno_simple(cbind(1:10, 10:1), pch = 1:10)
&gt; draw(anno, test = &quot;matrix, length of pch is length of samples&quot;)
&gt; anno = anno_simple(cbind(1:10, 10:1), pch = matrix(1:20, nc = 2))
&gt; draw(anno, test = &quot;matrix, pch is a matrix&quot;)
&gt; pch = matrix(1:20, nc = 2)
&gt; pch[sample(length(pch), 10)] = NA
&gt; anno = anno_simple(cbind(1:10, 10:1), pch = pch)
&gt; draw(anno, test = &quot;matrix, pch is a matrix with NA values&quot;)
&gt; 
&gt; 
&gt; ####### test anno_empty ######
&gt; anno = anno_empty()
&gt; draw(anno, test = &quot;anno_empty&quot;)
&gt; anno = anno_empty(border = FALSE)
&gt; draw(anno, test = &quot;anno_empty without border&quot;)
&gt; 
&gt; if(0) {
+ ###### test anno_image #####
+ image1 = sample(dir(&quot;~/Downloads/IcoMoon-Free-master/PNG/64px&quot;, full.names = TRUE), 10)
+ anno = anno_image(image1)
+ draw(anno, test = &quot;png&quot;)
+ draw(anno[1:5], test = &quot;subset of png&quot;)
+ anno = anno_image(image1, which = &quot;row&quot;)
+ draw(anno, test = &quot;png on rows&quot;)
+ image2 = sample(dir(&quot;~/Downloads/IcoMoon-Free-master/SVG/&quot;, full.names = TRUE), 10)
+ anno = anno_image(image2)
+ draw(anno, test = &quot;svg&quot;)
+ image3 = sample(dir(&quot;~/Downloads/IcoMoon-Free-master/EPS/&quot;, full.names = TRUE), 10)
+ anno = anno_image(image3)
+ draw(anno, test = &quot;eps&quot;)
+ image4 = sample(dir(&quot;~/Downloads/IcoMoon-Free-master/PDF/&quot;, full.names = TRUE), 10)
+ anno = anno_image(image4)
+ draw(anno, test = &quot;pdf&quot;)
+ 
+ anno = anno_image(c(image1[1:3], image2[1:3], image3[1:3], image4[1:3]))
+ draw(anno, test = &quot;png+svg+eps+pdf&quot;)
+ 
+ anno = anno_image(image1, gp = gpar(fill = 1:10, col = &quot;black&quot;))
+ draw(anno, test = &quot;png + gp&quot;)
+ draw(anno[1:5], test = &quot;png + gp&quot;)
+ 
+ anno = anno_image(image1, space = unit(3, &quot;mm&quot;))
+ draw(anno, test = &quot;space&quot;)
+ 
+ image1[1] = &quot;&quot;
+ anno = anno_image(image1)
+ draw(anno, test = &quot;png&quot;)
+ }
&gt; 
&gt; ######## test anno_points #####
&gt; anno = anno_points(runif(10))
&gt; draw(anno, test = &quot;anno_points&quot;)
&gt; anno = anno_points(matrix(runif(20), nc = 2), pch = 1:2)
&gt; draw(anno, test = &quot;matrix&quot;)
&gt; anno = anno_points(c(1:5, 1:5))
&gt; draw(anno, test = &quot;anno_points&quot;)
&gt; anno = anno_points(cbind(c(1:5, 1:5), c(5:1, 5:1)), gp = gpar(col = 2:3))
&gt; draw(anno, test = &quot;matrix&quot;)
&gt; 
&gt; anno = anno_points(1:10, gp = gpar(col = rep(2:3, each = 5)), pch = rep(2:3, each = 5))
&gt; draw(anno, test = &quot;anno_points&quot;)
&gt; draw(anno, index = c(1, 3, 5, 7, 9, 2, 4, 6, 8, 10), test = &quot;anno_points&quot;)
&gt; 
&gt; anno = anno_points(c(1:5, NA, 7:10))
&gt; draw(anno, test = &quot;anno_points&quot;)
&gt; 
&gt; 
&gt; anno = anno_points(runif(10), axis_param = list(direction = &quot;reverse&quot;), ylim = c(0, 1))
&gt; draw(anno, test = &quot;anno_points&quot;)
&gt; 
&gt; anno = anno_points(runif(10), axis_param = list(direction = &quot;reverse&quot;), ylim = c(0, 1), which = &quot;row&quot;)
&gt; draw(anno, test = &quot;anno_points&quot;)
&gt; 
&gt; # pch as image
&gt; if(0) {
+ image1 = sample(dir(&quot;/desktop-home/guz/Downloads/IcoMoon-Free-master/PNG/64px&quot;, full.names = TRUE), 10)
+ x = runif(10)
+ anno1 = anno_points(x, pch = image1, pch_as_image = TRUE, size = unit(5, &quot;mm&quot;), height = unit(4, &quot;cm&quot;))
+ anno2 = anno_points(x, height = unit(4, &quot;cm&quot;))
+ draw(anno1, test = &quot;anno_points&quot;)
+ draw(anno2, test = &quot;anno_points&quot;)
+ }
&gt; 
&gt; ##### test anno_lines ###
&gt; anno = anno_lines(runif(10))
&gt; draw(anno, test = &quot;anno_lines&quot;)
&gt; anno = anno_lines(cbind(c(1:5, 1:5), c(5:1, 5:1)), gp = gpar(col = 2:3))
&gt; draw(anno, test = &quot;matrix&quot;)
&gt; anno = anno_lines(cbind(c(1:5, 1:5), c(5:1, 5:1)), gp = gpar(col = 2:3),
+ 	add_points = TRUE, pt_gp = gpar(col = 5:6), pch = c(1, 16))
&gt; draw(anno, test = &quot;matrix&quot;)
&gt; anno = anno_lines(sort(rnorm(10)), height = unit(2, &quot;cm&quot;), smooth = TRUE, add_points = TRUE)
&gt; draw(anno, test = &quot;anno_lines, smooth&quot;)
&gt; anno = anno_lines(cbind(sort(rnorm(10)), sort(rnorm(10), decreasing = TRUE)), 
+ 	height = unit(2, &quot;cm&quot;), smooth = TRUE, add_points = TRUE, gp = gpar(col = 2:3))
&gt; draw(anno, test = &quot;anno_lines, smooth, matrix&quot;)
&gt; 
&gt; anno = anno_lines(sort(rnorm(10)), width = unit(2, &quot;cm&quot;), smooth = TRUE, add_points = TRUE, which = &quot;row&quot;)
&gt; draw(anno, test = &quot;anno_lines, smooth, by row&quot;)
&gt; anno = anno_lines(cbind(sort(rnorm(10)), sort(rnorm(10), decreasing = TRUE)), 
+ 	width = unit(2, &quot;cm&quot;), smooth = TRUE, add_points = TRUE, gp = gpar(col = 2:3), which = &quot;row&quot;)
&gt; draw(anno, test = &quot;anno_lines, smooth, matrix, by row&quot;)
&gt; 
&gt; anno = anno_lines(c(1:5, NA, 7:10))
&gt; draw(anno, test = &quot;anno_lines&quot;)
&gt; 
&gt; anno = anno_lines(runif(10), axis_param = list(direction = &quot;reverse&quot;))
&gt; draw(anno, test = &quot;anno_lines&quot;)
&gt; 
&gt; ###### test anno_text #######
&gt; anno = anno_text(month.name)
&gt; draw(anno, test = &quot;month names&quot;)
&gt; anno = anno_text(month.name, gp = gpar(fontsize = 16))
&gt; draw(anno, test = &quot;month names with fontsize&quot;)
&gt; anno = anno_text(month.name, gp = gpar(fontsize = 1:12+4))
&gt; draw(anno, test = &quot;month names with changing fontsize&quot;)
&gt; anno = anno_text(month.name, which = &quot;row&quot;)
&gt; draw(anno, test = &quot;month names on rows&quot;)
&gt; anno = anno_text(month.name, location = 0, rot = 45, just = &quot;left&quot;, gp = gpar(col = 1:12))
&gt; draw(anno, test = &quot;with rotations&quot;)
&gt; anno = anno_text(month.name, location = 1, rot = 45, just = &quot;right&quot;, gp = gpar(fontsize = 1:12+4))
&gt; draw(anno, test = &quot;with rotations&quot;)
&gt; 
&gt; 
&gt; for(rot in seq(0, 360, by = 45)) {
+ 	anno = anno_text(month.name, which = &quot;row&quot;, location = 0, rot = rot, 
+ 		just = &quot;left&quot;)
+ 	draw(anno, test = paste0(&quot;rot =&quot;, rot))
+ }
&gt; 
&gt; 
&gt; ##### test anno_barplot #####
&gt; anno = anno_barplot(1:10)
&gt; draw(anno, test = &quot;a vector&quot;)
&gt; draw(anno[1:5], test = &quot;a vector, subset&quot;)
&gt; anno = anno_barplot(1:10, which = &quot;row&quot;)
&gt; draw(anno, test = &quot;a vector&quot;)
&gt; anno = anno_barplot(1:10, bar_width = 1)
&gt; draw(anno, test = &quot;bar_width&quot;)
&gt; anno = anno_barplot(1:10, gp = gpar(fill = 1:10))
&gt; draw(anno, test = &quot;fill colors&quot;)
&gt; 
&gt; anno = anno_barplot(matrix(nc = 2, c(1:10, 10:1)))
&gt; draw(anno, test = &quot;a matrix&quot;)
&gt; draw(anno[1:5], test = &quot;a matrix, subset&quot;)
&gt; anno = anno_barplot(matrix(nc = 2, c(1:10, 10:1)), which = &quot;row&quot;)
&gt; draw(anno, test = &quot;a matrix, on rows&quot;)
&gt; anno = anno_barplot(matrix(nc = 2, c(1:10, 10:1)), gp = gpar(fill = 2:3, col = 2:3))
&gt; draw(anno, test = &quot;a matrix with fill&quot;)
&gt; 
&gt; m = matrix(runif(4*10), nc = 4)
&gt; m = t(apply(m, 1, function(x) x/sum(x)))
&gt; anno = anno_barplot(m)
&gt; draw(anno, test = &quot;proportion matrix&quot;)
&gt; anno = anno_barplot(m, gp = gpar(fill = 2:5), bar_width = 1, height = unit(6, &quot;cm&quot;))
&gt; draw(anno, test = &quot;proportion matrix&quot;)
&gt; 
&gt; anno = anno_barplot(c(1:5, NA, 7:10))
&gt; draw(anno, test = &quot;a vector&quot;)
&gt; 
&gt; anno = anno_barplot(1:10, which = &quot;row&quot;, axis_param = list(direction = &quot;reverse&quot;))
&gt; draw(anno, test = &quot;a vector&quot;)
&gt; 
&gt; anno = anno_barplot(1:10, baseline = 5, which = &quot;row&quot;, axis_param = list(direction = &quot;reverse&quot;))
&gt; draw(anno, test = &quot;a vector&quot;)
&gt; 
&gt; anno = anno_barplot(matrix(nc = 2, c(1:10, 10:1)), which = &quot;row&quot;, axis_param = list(direction = &quot;reverse&quot;))
&gt; draw(anno, test = &quot;a vector&quot;)
&gt; 
&gt; 
&gt; anno = anno_barplot(matrix(nc = 2, c(1:10, 10:1)), beside = TRUE)
&gt; draw(anno, test = &quot;a matrix&quot;)
&gt; draw(anno[1:5], test = &quot;a matrix, subset&quot;)
&gt; anno = anno_barplot(matrix(nc = 2, c(1:10, 10:1)), beside = TRUE, which = &quot;row&quot;)
&gt; draw(anno, test = &quot;a matrix, on rows&quot;)
&gt; anno = anno_barplot(matrix(nc = 2, c(1:10, 10:1)), beside = TRUE, gp = gpar(fill = 2:3, col = 2:3))
&gt; draw(anno, test = &quot;a matrix with fill&quot;)
&gt; 
&gt; 
&gt; ##### test anno_boxplot #####
&gt; set.seed(123)
&gt; m = matrix(rnorm(100), 10)
&gt; anno = anno_boxplot(m, height = unit(4, &quot;cm&quot;))
&gt; draw(anno, test = &quot;anno_boxplot&quot;)
&gt; draw(anno[1:5], test = &quot;subset&quot;)
&gt; anno = anno_boxplot(m, height = unit(4, &quot;cm&quot;), gp = gpar(fill = 1:10))
&gt; draw(anno, test = &quot;anno_boxplot with gp&quot;)
&gt; anno = anno_boxplot(m, height = unit(4, &quot;cm&quot;), box_width = 0.9)
&gt; draw(anno, test = &quot;anno_boxplot with box_width&quot;)
&gt; 
&gt; m = matrix(rnorm(100), 10)
&gt; m[1, ] = NA
&gt; anno = anno_boxplot(m, height = unit(4, &quot;cm&quot;))
&gt; draw(anno, test = &quot;anno_boxplot&quot;)
&gt; 
&gt; 
&gt; ####### test anno_joyplot ####
&gt; m = matrix(rnorm(1000), nc = 10)
&gt; lt = apply(m, 2, function(x) data.frame(density(x)[c(&quot;x&quot;, &quot;y&quot;)]))
&gt; anno = anno_joyplot(lt, width = unit(4, &quot;cm&quot;), which = &quot;row&quot;)
&gt; draw(anno, test = &quot;joyplot&quot;)
&gt; anno = anno_joyplot(lt, width = unit(4, &quot;cm&quot;), which = &quot;row&quot;, gp = gpar(fill = 1:10))
&gt; draw(anno, test = &quot;joyplot + col&quot;)
&gt; anno = anno_joyplot(lt, width = unit(4, &quot;cm&quot;), which = &quot;row&quot;, scale = 1)
&gt; draw(anno, test = &quot;joyplot + scale&quot;)
&gt; 
&gt; m = matrix(rnorm(5000), nc = 50)
&gt; lt = apply(m, 2, function(x) data.frame(density(x)[c(&quot;x&quot;, &quot;y&quot;)]))
&gt; anno = anno_joyplot(lt, width = unit(4, &quot;cm&quot;), which = &quot;row&quot;, gp = gpar(fill = NA), scale = 4)
&gt; draw(anno, test = &quot;joyplot&quot;)
&gt; 
&gt; ######## test anno_horizon ######
&gt; lt = lapply(1:20, function(x) cumprod(1 + runif(1000, -x/100, x/100)) - 1)
&gt; anno = anno_horizon(lt, which = &quot;row&quot;)
&gt; draw(anno, test = &quot;horizon chart&quot;)
&gt; anno = anno_horizon(lt, which = &quot;row&quot;, gp = gpar(pos_fill = &quot;orange&quot;, neg_fill = &quot;darkgreen&quot;))
&gt; draw(anno, test = &quot;horizon chart, col&quot;)
&gt; anno = anno_horizon(lt, which = &quot;row&quot;, negative_from_top = TRUE)
&gt; draw(anno, test = &quot;horizon chart + negative_from_top&quot;)
&gt; anno = anno_horizon(lt, which = &quot;row&quot;, gap = unit(1, &quot;mm&quot;))
&gt; draw(anno, test = &quot;horizon chart + gap&quot;)
&gt; anno = anno_horizon(lt, which = &quot;row&quot;, gp = gpar(pos_fill = rep(c(&quot;orange&quot;, &quot;red&quot;), each = 10),
+ 	neg_fill = rep(c(&quot;darkgreen&quot;, &quot;blue&quot;), each = 10)))
&gt; draw(anno, test = &quot;horizon chart, col&quot;)
&gt; 
&gt; ####### test anno_histogram ####
&gt; m = matrix(rnorm(1000), nc = 10)
&gt; anno = anno_histogram(t(m), which = &quot;row&quot;)
&gt; draw(anno, test = &quot;row histogram&quot;)
&gt; draw(anno[1:5], test = &quot;subset row histogram&quot;)
&gt; anno = anno_histogram(t(m), which = &quot;row&quot;, gp = gpar(fill = 1:10))
&gt; draw(anno, test = &quot;row histogram with color&quot;)
&gt; anno = anno_histogram(t(m), which = &quot;row&quot;, n_breaks = 20)
&gt; draw(anno, test = &quot;row histogram with color&quot;)
&gt; m[1, ] = NA
&gt; anno = anno_histogram(t(m), which = &quot;row&quot;)
&gt; draw(anno, test = &quot;row histogram&quot;)
&gt; 
&gt; 
&gt; ####### test anno_density ######
&gt; anno = anno_density(t(m), which = &quot;row&quot;)
&gt; draw(anno, test = &quot;normal density&quot;)
&gt; draw(anno[1:5], test = &quot;normal density, subset&quot;)
&gt; anno = anno_density(t(m), which = &quot;row&quot;, type = &quot;violin&quot;)
&gt; draw(anno, test = &quot;violin&quot;)
&gt; anno = anno_density(t(m), which = &quot;row&quot;, type = &quot;heatmap&quot;)
&gt; draw(anno, test = &quot;heatmap&quot;)
&gt; anno = anno_density(t(m), which = &quot;row&quot;, type = &quot;heatmap&quot;, heatmap_colors = c(&quot;white&quot;, &quot;orange&quot;))
&gt; draw(anno, test = &quot;heatmap, colors&quot;)
&gt; 
&gt; anno = anno_density(t(m), which = &quot;row&quot;, xlim = c(-2, 2))
&gt; draw(anno, test = &quot;normal density&quot;)
&gt; anno = anno_density(t(m), which = &quot;row&quot;, type = &quot;violin&quot;, xlim = c(-2, 2))
&gt; draw(anno, test = &quot;violin&quot;)
&gt; anno = anno_density(t(m), which = &quot;row&quot;, type = &quot;heatmap&quot;, xlim = c(-2, 2))
&gt; draw(anno, test = &quot;heatmap&quot;)
&gt; 
&gt; ###### anno_mark ###
&gt; if(0) {
+ library(gridtext)
+ grid.text = function(text, x = 0.5, y = 0.5, gp = gpar(), rot = 0, default.units = &quot;npc&quot;, just = &quot;center&quot;) {
+ 	if(length(just) == 1) {
+ 		if(just == &quot;center&quot;) {
+ 			just = c(&quot;center&quot;, &quot;center&quot;)
+ 		} else if(just == &quot;bottom&quot;) {
+ 			just = c(&quot;center&quot;, &quot;bottom&quot;)
+ 		} else if (just == &quot;top&quot;) {
+ 			just = c(&quot;center&quot;, &quot;top&quot;)
+ 		} else if(just == &quot;left&quot;) {
+ 			just = c(&quot;left&quot;, &quot;center&quot;)
+ 		} else if(just == &quot;right&quot;) {
+ 			just = c(&quot;right&quot;, &quot;center&quot;)
+ 		}
+ 	}
+ 	just2 = c(0.5, 0.5)
+ 	if(is.character(just)) {
+ 		just2[1] = switch(just[1], &quot;center&quot; = 0.5, &quot;left&quot; = 0, &quot;right&quot; = 1)
+ 		just2[2] = switch(just[2], &quot;center&quot; = 0.5, &quot;bottom&quot; = 0, &quot;top&quot; = 1)
+ 	}
+ 	gb = richtext_grob(text, x = x, y = y, gp = gpar(fontsize = 10), box_gp = gpar(col = &quot;black&quot;),
+ 		default.units = default.units, hjust = just2[1], vjust = just2[2], rot = rot)
+ 	grid.draw(gb)
+ }
+ }
&gt; anno = anno_mark(at = c(1:4, 20, 60, 97:100), labels = month.name[1:10], which = &quot;row&quot;)
&gt; draw(anno, index = 1:100, test = &quot;anno_mark&quot;)
&gt; 
&gt; anno = anno_mark(at = c(1:4, 20, 60, 97:100), labels = month.name[1:10], labels_rot = 30, which = &quot;column&quot;)
&gt; draw(anno, index = 1:100, test = &quot;anno_mark&quot;)
&gt; 
&gt; m = matrix(1:1000, byrow = TRUE, nr = 100)
&gt; anno = anno_mark(at = c(1:4, 20, 60, 97:100), labels = month.name[1:10], which = &quot;row&quot;, labels_rot = 30)
&gt; Heatmap(m, cluster_rows = F, cluster_columns = F) + rowAnnotation(mark = anno)
&gt; Heatmap(m) + rowAnnotation(mark = anno)
&gt; 
&gt; ht_list = Heatmap(m, cluster_rows = F, cluster_columns = F) + rowAnnotation(mark = anno)
&gt; draw(ht_list, row_split = c(rep(&quot;a&quot;, 95), rep(&quot;b&quot;, 5)))
&gt; 
&gt; 
&gt; grid.newpage()
&gt; pushViewport(viewport(x = 0.45, w = 0.7, h = 0.95))
&gt; h = unit(0, &quot;mm&quot;)
&gt; for(rot in seq(0, 360, by = 30)[-13]) {
+ 	anno = anno_mark(at = c(1:4, 20, 60, 97:100), labels = strrep(letters[1:10], 4), labels_rot = rot, which = &quot;column&quot;, side = &quot;bottom&quot;)
+ 	h = h + height(anno)
+ 	pushViewport(viewport(y = h, height = height(anno), just = &quot;top&quot;))
+ 	grid.rect()
+ 	draw(anno, index = 1:100)
+ 	grid::grid.text(qq(&quot;labels_rot = @{rot}&quot;), unit(1, &quot;npc&quot;) + unit(2, &quot;mm&quot;), just = &quot;left&quot;)
+ 	popViewport()
+ }
&gt; 
&gt; 
&gt; grid.newpage()
&gt; pushViewport(viewport(w = 0.9, h = 0.9))
&gt; w = unit(0, &quot;mm&quot;)
&gt; for(rot in seq(0, 360, by = 30)) {
+ 	anno = anno_mark(at = c(1:4, 20, 60, 97:100), labels = strrep(letters[1:10], 4), labels_rot = rot, which = &quot;row&quot;, side = &quot;left&quot;)
+ 	w = w + width(anno)
+ 	pushViewport(viewport(x = w, width = width(anno), just = &quot;right&quot;))
+ 	grid.rect()
+ 	draw(anno, index = 1:100)
+ 	popViewport()
+ }
&gt; 
&gt; 
&gt; 
&gt; ### graphic parameters after reordering
&gt; index = c(1, 3, 5, 7, 9, 2, 4, 6, 8, 10)
&gt; anno = anno_simple(1:10, pch = 1:10, pt_gp = gpar(col = rep(c(1, 2), each = 5)),
+ 	pt_size = unit(1:10, &quot;mm&quot;))
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; anno = anno_simple(1:10, pch = 1:10, pt_gp = gpar(col = rep(c(1, 2), each = 5)),
+ 	pt_size = unit(1:10, &quot;mm&quot;), which = &quot;row&quot;)
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; 
&gt; 
&gt; anno = anno_points(1:10, pch = 1:10, gp = gpar(col = rep(c(1, 2), each = 5)),
+ 	size = unit(1:10, &quot;mm&quot;))
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; anno = anno_points(1:10, pch = 1:10, gp = gpar(col = rep(c(1, 2), each = 5)),
+ 	size = unit(1:10, &quot;mm&quot;), which = &quot;row&quot;)
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; 
&gt; 
&gt; anno = anno_lines(sort(runif(10)), pch = 1:10, pt_gp = gpar(col = rep(c(1, 2), each = 5)),
+ 	size = unit(1:10, &quot;mm&quot;), add_points = TRUE)
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; anno = anno_lines(sort(runif(10)), pch = 1:10, pt_gp = gpar(col = rep(c(1, 2), each = 5)),
+ 	size = unit(1:10, &quot;mm&quot;), add_points = TRUE, which = &quot;row&quot;)
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; 
&gt; 
&gt; anno = anno_barplot(1:10, gp = gpar(fill = rep(c(1, 2), each = 5)))
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; anno = anno_barplot(1:10, gp = gpar(fill = rep(c(1, 2), each = 5)), which = &quot;row&quot;)
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; 
&gt; anno = anno_barplot(cbind(1:10, 10:1), gp = gpar(fill = 1:2))
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; anno = anno_barplot(cbind(1:10, 10:1), gp = gpar(fill = 1:2), which = &quot;row&quot;)
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; 
&gt; 
&gt; m = matrix(rnorm(100), 10)
&gt; m = m[, order(apply(m, 2, median))]
&gt; anno = anno_boxplot(m, pch = 1:10, gp = gpar(fill = rep(c(1, 2), each = 5)),
+ 	size = unit(1:10, &quot;mm&quot;), height = unit(4, &quot;cm&quot;))
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; anno = anno_boxplot(t(m), pch = 1:10, gp = gpar(fill = rep(c(1, 2), each = 5)),
+ 	size = unit(1:10, &quot;mm&quot;), which = &quot;row&quot;, width = unit(4, &quot;cm&quot;))
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; 
&gt; anno = anno_histogram(m, gp = gpar(fill = rep(c(1, 2), each = 5)))
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; anno = anno_histogram(t(m), gp = gpar(fill = rep(c(1, 2), each = 5)), which = &quot;row&quot;)
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; 
&gt; anno = anno_density(m, gp = gpar(fill = rep(c(1, 2), each = 5)))
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; anno = anno_density(t(m), gp = gpar(fill = rep(c(1, 2), each = 5)), which = &quot;row&quot;)
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; 
&gt; 
&gt; anno = anno_density(m, type = &quot;violin&quot;, gp = gpar(fill = rep(c(1, 2), each = 5)))
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; anno = anno_density(t(m), type = &quot;violin&quot;, gp = gpar(fill = rep(c(1, 2), each = 5)), which = &quot;row&quot;)
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; 
&gt; 
&gt; anno = anno_text(month.name, gp = gpar(col = rep(c(1, 2), each = 5)))
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; anno = anno_text(month.name, gp = gpar(col = rep(c(1, 2), each = 5)), which= &quot;row&quot;)
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; 
&gt; lt = lapply(1:10, function(x) cumprod(1 + runif(1000, -x/100, x/100)) - 1)
&gt; anno = anno_horizon(lt, gp = gpar(pos_fill = rep(c(1, 2), each = 5), neg_fill = rep(c(3, 4), each = 5)), which = &quot;row&quot;)
&gt; draw(anno, index, test = &quot;a numeric vector&quot;)
&gt; 
&gt; m = matrix(rnorm(1000), nc = 10)
&gt; lt = apply(m, 2, function(x) data.frame(density(x)[c(&quot;x&quot;, &quot;y&quot;)]))
&gt; anno = anno_joyplot(lt, gp = gpar(fill = rep(c(1, 2), each = 5)), 
+ 	width = unit(4, &quot;cm&quot;), which = &quot;row&quot;)
&gt; draw(anno, index, test = &quot;joyplot&quot;)
&gt; 
&gt; 
&gt; anno = anno_block(gp = gpar(fill = 1:4))
&gt; draw(anno, index = 1:10, k = 1, n = 4, test = &quot;anno_block&quot;)
&gt; draw(anno, index = 1:10, k = 2, n = 4, test = &quot;anno_block&quot;)
&gt; 
&gt; anno = anno_block(gp = gpar(fill = 1:4), labels = letters[1:4], labels_gp = gpar(col = &quot;white&quot;))
&gt; draw(anno, index = 1:10, k = 2, n = 4, test = &quot;anno_block&quot;)
&gt; draw(anno, index = 1:10, k = 4, n = 4, test = &quot;anno_block&quot;)
&gt; # draw(anno, index = 1:10, k = 2, n = 2, test = &quot;anno_block&quot;)
&gt; 
&gt; anno = anno_block(gp = gpar(fill = 1:4), labels = letters[1:4], labels_gp = gpar(col = &quot;white&quot;), which = &quot;row&quot;)
&gt; draw(anno, index = 1:10, k = 2, n = 4, test = &quot;anno_block&quot;)
&gt; 
&gt; 
&gt; ### anno_zoom
&gt; fa = sort(sample(letters[1:3], 100, replace = TRUE, prob = c(1, 2, 3)))
&gt; panel_fun = function(index, nm) {
+ 	grid.rect()
+ 	grid.text(nm)
+ }
&gt; anno = anno_zoom(align_to = fa, which = &quot;row&quot;, panel_fun = panel_fun)
&gt; draw(anno, index = 1:100, test = &quot;anno_zoom&quot;)
&gt; 
&gt; anno = anno_zoom(align_to = list(a = which(fa == &quot;a&quot;)), which = &quot;row&quot;, panel_fun = panel_fun)
&gt; draw(anno, index = 1:100, test = &quot;anno_zoom&quot;)
&gt; 
&gt; 
&gt; panel_fun = function(index, nm) {
+ 	grid.rect(gp = gpar(fill = &quot;grey&quot;, col = NA))
+ 	grid.text(nm)
+ }
&gt; 
&gt; anno = anno_zoom(align_to = fa, which = &quot;row&quot;, panel_fun = panel_fun, link_gp = gpar(fill = &quot;grey&quot;, col = &quot;black&quot;), internal_line = FALSE)
&gt; draw(anno, index = 1:100, test = &quot;anno_zoom&quot;)
&gt; 
&gt; 
&gt; anno = anno_zoom(align_to = fa, which = &quot;row&quot;, panel_fun = panel_fun,
+ 	gap = unit(1, &quot;cm&quot;))
&gt; draw(anno, index = 1:100, test = &quot;anno_zoom, set gap&quot;)
&gt; 
&gt; anno = anno_zoom(align_to = fa, which = &quot;row&quot;, panel_fun = panel_fun,
+ 	size = 1:3)
&gt; draw(anno, index = 1:100, test = &quot;anno_zoom, size set as relative values&quot;)
&gt; 
&gt; anno = anno_zoom(align_to = fa, which = &quot;row&quot;, panel_fun = panel_fun,
+ 	size = 1:3, extend = unit(1, &quot;cm&quot;))
&gt; draw(anno, index = 1:100, test = &quot;anno_zoom, extend&quot;)
&gt; 
&gt; anno = anno_zoom(align_to = fa, which = &quot;row&quot;, panel_fun = panel_fun,
+ 	size = unit(1:3, &quot;cm&quot;))
&gt; draw(anno, index = 1:100, test = &quot;anno_zoom, size set as absolute values&quot;)
&gt; 
&gt; anno = anno_zoom(align_to = fa, which = &quot;row&quot;, panel_fun = panel_fun,
+ 	size = unit(c(2, 20, 40), &quot;cm&quot;))
&gt; draw(anno, index = 1:100, test = &quot;anno_zoom, big size&quot;)
&gt; 
&gt; anno = anno_zoom(align_to = fa, which = &quot;row&quot;, panel_fun = panel_fun,
+ 	size = 1:3, gap = unit(1, &quot;cm&quot;))
&gt; draw(anno, index = 1:100, test = &quot;anno_zoom, size set as relative values, gap&quot;)
&gt; 
&gt; anno = anno_zoom(align_to = fa, which = &quot;row&quot;, panel_fun = panel_fun,
+ 	size = unit(1:3, &quot;cm&quot;), gap = unit(1, &quot;cm&quot;))
&gt; draw(anno, index = 1:100, test = &quot;anno_zoom, size set as absolute values, gap&quot;)
&gt; 
&gt; 
&gt; anno = anno_zoom(align_to = fa, which = &quot;row&quot;, panel_fun = panel_fun,
+ 	size = unit(1:3, &quot;cm&quot;), side = &quot;left&quot;)
&gt; draw(anno, index = 1:100, test = &quot;anno_zoom, side&quot;)
&gt; 
&gt; 
&gt; anno = anno_zoom(align_to = fa, which = &quot;row&quot;, panel_fun = panel_fun,
+ 	size = unit(1:3, &quot;cm&quot;), link_gp = gpar(fill = 1:3))
&gt; draw(anno, index = 1:100, test = &quot;anno_zoom, link_gp&quot;)
&gt; 
&gt; anno = anno_zoom(align_to = fa, which = &quot;row&quot;, panel_fun = panel_fun,
+ 	size = unit(1:3, &quot;cm&quot;), link_gp = gpar(fill = 1:3),
+ 	link_width = unit(2, &quot;cm&quot;), width = unit(4, &quot;cm&quot;))
&gt; draw(anno, index = 1:100, test = &quot;anno_zoom, width&quot;)
&gt; 
&gt; anno = anno_zoom(align_to = list(a = 1:10, b = 30:45, c = 70:90), 
+ 	which = &quot;row&quot;, panel_fun = panel_fun, size = unit(1:3, &quot;cm&quot;))
&gt; draw(anno, index = 1:100, test = &quot;anno_zoom, a list of indices&quot;)
&gt; 
&gt; anno = anno_zoom(align_to = fa, which = &quot;column&quot;, panel_fun = panel_fun,
+ 	size = unit(1:3, &quot;cm&quot;))
&gt; draw(anno, index = 1:100, test = &quot;anno_zoom, column annotation&quot;)
&gt; 
&gt; 
&gt; m = matrix(rnorm(100*10), nrow = 100)
&gt; hc = hclust(dist(m))
&gt; fa2 = cutree(hc, k = 4)
&gt; anno = anno_zoom(align_to = fa2, which = &quot;row&quot;, panel_fun = panel_fun)
&gt; draw(anno, index = hc$order, test = &quot;anno_zoom, column annotation&quot;)
&gt; 
&gt; anno = anno_zoom(align_to = fa2, which = &quot;column&quot;, panel_fun = panel_fun)
&gt; draw(anno, index = hc$order, test = &quot;anno_zoom, column annotation&quot;)
&gt; 
&gt; 
&gt; anno = anno_zoom(align_to = fa2, which = &quot;row&quot;, panel_fun = panel_fun)
&gt; draw(Heatmap(m, cluster_rows = hc, right_annotation = rowAnnotation(foo = anno)))
&gt; draw(Heatmap(m, cluster_rows = hc, right_annotation = rowAnnotation(foo = anno), row_split = 2))
&gt; 
&gt; 
&gt; anno = anno_zoom(align_to = fa2, which = &quot;row&quot;, panel_fun = panel_fun, size = unit(1:4, &quot;cm&quot;))
&gt; draw(Heatmap(m, cluster_rows = hc, right_annotation = rowAnnotation(foo = anno)))
&gt; 
&gt; set.seed(123)
&gt; m = matrix(rnorm(100*10), nrow = 100)
&gt; subgroup = sample(letters[1:3], 100, replace = TRUE, prob = c(1, 5, 10))
&gt; rg = range(m)
&gt; panel_fun = function(index, nm) {
+ 	pushViewport(viewport(xscale = rg, yscale = c(0, 2)))
+ 	grid.rect()
+ 	grid.xaxis(gp = gpar(fontsize = 8))
+ 	grid.boxplot(m[index, ], pos = 1, direction = &quot;horizontal&quot;)
+ 	grid.text(paste(&quot;distribution of group&quot;, nm), mean(rg), y = 1.9, 
+ 		just = &quot;top&quot;, default.units = &quot;native&quot;, gp = gpar(fontsize = 10))
+ 	popViewport()
+ }
&gt; anno = anno_zoom(align_to = subgroup, which = &quot;row&quot;, panel_fun = panel_fun, 
+ 	size = unit(2, &quot;cm&quot;), gap = unit(1, &quot;cm&quot;), width = unit(4, &quot;cm&quot;))
&gt; draw(Heatmap(m, right_annotation = rowAnnotation(foo = anno), row_split = subgroup))
&gt; 
&gt; panel_fun2 = function(index, nm) {
+ 	pushViewport(viewport())
+ 	grid.rect()
+ 	n = floor(length(index)/4)
+ 	txt = paste(&quot;gene function&quot;, 1:n, collapse = &quot;\n&quot;)
+ 	grid.text(txt, 0.95, 0.5, default.units = &quot;npc&quot;, just = &quot;right&quot;, gp = gpar(fontsize = 8))
+ 	popViewport()
+ }
&gt; anno2 = anno_zoom(align_to = subgroup, which = &quot;row&quot;, panel_fun = panel_fun2, 
+ 	gap = unit(1, &quot;cm&quot;), width = unit(3, &quot;cm&quot;), side = &quot;left&quot;)
&gt; 
&gt; draw(Heatmap(m, right_annotation = rowAnnotation(subgroup = subgroup, foo = anno,
+ 	show_annotation_name = FALSE), 
+ 	left_annotation = rowAnnotation(bar = anno2, subgroup = subgroup, show_annotation_name = FALSE),
+ 	show_row_dend = FALSE,
+ 	row_split = subgroup))
&gt; 
&gt; draw(Heatmap(m, right_annotation = rowAnnotation(foo = anno), 
+ 	left_annotation = rowAnnotation(bar = anno2),
+ 	show_row_dend = FALSE,
+ 	row_split = subgroup))
&gt; 
&gt; set.seed(12345)
&gt; mat = matrix(rnorm(30*10), nr = 30)
&gt; row_split = c(rep(&quot;a&quot;, 10), rep(&quot;b&quot;, 5), rep(&quot;c&quot;, 2), rep(&quot;d&quot;, 3), 
+ 	          rep(&quot;e&quot;, 2), letters[10:17])
&gt; row_split = factor(row_split)
&gt; 
&gt; panel_fun = function(index, name) {
+ 	pushViewport(viewport())
+ 	grid.rect()
+ 	grid.text(name)
+ 	popViewport()
+ }
&gt; 
&gt; anno = anno_zoom(align_to = row_split, which = &quot;row&quot;, panel_fun = panel_fun, 
+ 	size = unit(0.5, &quot;cm&quot;), width = unit(4, &quot;cm&quot;))
&gt; 
&gt; # &gt; dev.size()
&gt; # [1] 3.938326 4.502203
&gt; dev.new(width = 3.938326, height = 4.502203)
dev.new(): using pdf(file=&quot;Rplots1.pdf&quot;)
&gt; draw(Heatmap(mat, right_annotation = rowAnnotation(foo = anno), 
+ 	row_split = row_split))
&gt; 
&gt; 
&gt; 
&gt; #### anno_customize ###
&gt; x = sort(sample(letters[1:3], 10, replace = TRUE))
&gt; graphics = list(
+ 	&quot;a&quot; = function(x, y, w, h) grid.points(x, y, pch = 16),
+ 	&quot;b&quot; = function(x, y, w, h) grid.rect(x, y, w*0.8, h*0.8, gp = gpar(fill = &quot;red&quot;)),
+ 	&quot;c&quot; = function(x, y, w, h) grid.segments(x - 0.5*w, y - 0.5*h, x + 0.5*w, y + 0.5*h, gp = gpar(lty = 2))
+ )
&gt; 
&gt; anno = anno_customize(x, graphics = graphics)
&gt; draw(anno, index = 1:10, test = &quot;&quot;)
&gt; 
&gt; anno = anno_customize(c(x, &quot;d&quot;), graphics = graphics)
Note: following levels in `x` have no graphics defined:
    d.
Set `verbose = FALSE` in `anno_customize()` to turn off this message.
&gt; 
&gt; ### anno_numeric ##
&gt; x = runif(10)
&gt; anno = anno_numeric(x)
&gt; draw(anno, 1:10, test = TRUE)
&gt; anno = anno_numeric(x, align_to = &quot;right&quot;)
&gt; draw(anno, 1:10, test = TRUE)
&gt; 
&gt; 
&gt; x = 10^(-runif(10, 1, 6))
&gt; anno = anno_numeric(x, x_convert = function(x) -log10(x), labels_format = function(x) sprintf(&quot;%.2e&quot;, x))
&gt; draw(anno, 1:10, test = TRUE)
&gt; 
&gt; x = runif(10, -1, 1)
&gt; anno = anno_numeric(x)
&gt; draw(anno, 1:10, test = TRUE)
&gt; anno = anno_numeric(x, labels_gp = gpar(col = c(&quot;green&quot;, &quot;red&quot;)))
&gt; draw(anno, 1:10, test = TRUE)
&gt; 
&gt; anno = anno_numeric(x, bg_gp = gpar(col = c(&quot;green&quot;, &quot;red&quot;)))
&gt; draw(anno, 1:10, test = TRUE)
&gt; 
&gt; 
&gt; x = runif(10, 0.5, 1.5)
&gt; anno = anno_numeric(x, align_to = 0)
&gt; draw(anno, 1:10, test = TRUE)
&gt; 
&gt; 
&gt; 
&gt; 
&gt; 
&gt; proc.time()
   user  system elapsed 
 24.036   0.279  24.298 
</PRE>
</DIV><P><SPAN class="filename">ComplexHeatmap.Rcheck/tests/test-ColorMapping-class.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(circlize)
========================================
circlize version 0.4.15
CRAN page: https://cran.r-project.org/package=circlize
Github page: https://github.com/jokergoo/circlize
Documentation: https://jokergoo.github.io/circlize_book/book/

If you use it in published research, please cite:
Gu, Z. circlize implements and enhances circular visualization
  in R. Bioinformatics 2014.

This message can be suppressed by:
  suppressPackageStartupMessages(library(circlize))
========================================

&gt; library(ComplexHeatmap)
Loading required package: grid
========================================
ComplexHeatmap version 2.17.0
Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/
Github page: https://github.com/jokergoo/ComplexHeatmap
Documentation: http://jokergoo.github.io/ComplexHeatmap-reference

If you use it in published research, please cite either one:
- Gu, Z. Complex Heatmap Visualization. iMeta 2022.
- Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional 
    genomic data. Bioinformatics 2016.


The new InteractiveComplexHeatmap package can directly export static 
complex heatmaps into an interactive Shiny app with zero effort. Have a try!

This message can be suppressed by:
  suppressPackageStartupMessages(library(ComplexHeatmap))
========================================

&gt; library(GetoptLong)
&gt; 
&gt; cm = ColorMapping(name = &quot;test&quot;,
+ 	colors = c(&quot;blue&quot;, &quot;white&quot;, &quot;red&quot;),
+ 	levels = c(&quot;a&quot;, &quot;b&quot;, &quot;c&quot;))
&gt; color_mapping_legend(cm)
&gt; 
&gt; cm = ColorMapping(name = &quot;test&quot;,
+ 	col_fun = colorRamp2(c(0, 0.5, 1), c(&quot;blue&quot;, &quot;white&quot;, &quot;red&quot;)))
&gt; color_mapping_legend(cm)
&gt; 
&gt; cm = ColorMapping(name = &quot;test&quot;,
+ 	colors = c(&quot;blue&quot;, &quot;white&quot;, &quot;red&quot;),
+ 	levels = c(1, 2, 3))
&gt; color_mapping_legend(cm)
&gt; 
&gt; ha = SingleAnnotation(value = rep(NA, 10), name = &quot;foo&quot;)
&gt; cm = ha@color_mapping
&gt; color_mapping_legend(cm)
&gt; 
&gt; 
&gt; proc.time()
   user  system elapsed 
  2.979   0.125   3.085 
</PRE>
</DIV><P><SPAN class="filename">ComplexHeatmap.Rcheck/tests/test-dendrogram.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(circlize)
========================================
circlize version 0.4.15
CRAN page: https://cran.r-project.org/package=circlize
Github page: https://github.com/jokergoo/circlize
Documentation: https://jokergoo.github.io/circlize_book/book/

If you use it in published research, please cite:
Gu, Z. circlize implements and enhances circular visualization
  in R. Bioinformatics 2014.

This message can be suppressed by:
  suppressPackageStartupMessages(library(circlize))
========================================

&gt; library(ComplexHeatmap)
Loading required package: grid
========================================
ComplexHeatmap version 2.17.0
Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/
Github page: https://github.com/jokergoo/ComplexHeatmap
Documentation: http://jokergoo.github.io/ComplexHeatmap-reference

If you use it in published research, please cite either one:
- Gu, Z. Complex Heatmap Visualization. iMeta 2022.
- Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional 
    genomic data. Bioinformatics 2016.


The new InteractiveComplexHeatmap package can directly export static 
complex heatmaps into an interactive Shiny app with zero effort. Have a try!

This message can be suppressed by:
  suppressPackageStartupMessages(library(ComplexHeatmap))
========================================

&gt; library(GetoptLong)
&gt; 
&gt; if(!exists(&quot;cut_dendrogram&quot;)) {
+ 	cut_dendrogram = ComplexHeatmap:::cut_dendrogram
+ }
&gt; 
&gt; library(dendextend)

---------------------
Welcome to dendextend version 1.17.1
Type citation(&#x27;dendextend&#x27;) for how to cite the package.

Type browseVignettes(package = &#x27;dendextend&#x27;) for the package vignette.
The github page is: https://github.com/talgalili/dendextend/

Suggestions and bug-reports can be submitted at: https://github.com/talgalili/dendextend/issues
You may ask questions at stackoverflow, use the r and dendextend tags: 
	 https://stackoverflow.com/questions/tagged/dendextend

	To suppress this message use:  suppressPackageStartupMessages(library(dendextend))
---------------------


Attaching package: &#x27;dendextend&#x27;

The following object is masked from &#x27;package:stats&#x27;:

    cutree

&gt; 
&gt; m = matrix(rnorm(100), 10)
&gt; dend1 = as.dendrogram(hclust(dist(m)))
&gt; dend1 = adjust_dend_by_x(dend1, sort(runif(10)))
&gt; 
&gt; m = matrix(rnorm(50), nr = 5)
&gt; dend2 = as.dendrogram(hclust(dist(m)))
&gt; 
&gt; dend3 = as.dendrogram(hclust(dist(m[1:2, ])))
&gt; 
&gt; 
&gt; dend_merge = merge_dendrogram(dend3, 
+ 	list(set(dend1, &quot;branches_col&quot;, &quot;red&quot;), 
+ 		 set(dend2, &quot;branches_col&quot;, &quot;blue&quot;))
+ )
&gt; 
&gt; grid.dendrogram(dend_merge, test = TRUE, facing = &quot;bottom&quot;)
&gt; grid.dendrogram(dend_merge, test = TRUE, facing = &quot;top&quot;)
&gt; grid.dendrogram(dend_merge, test = TRUE, facing = &quot;left&quot;)
&gt; grid.dendrogram(dend_merge, test = TRUE, facing = &quot;right&quot;)
&gt; 
&gt; grid.dendrogram(dend_merge, test = TRUE, facing = &quot;bottom&quot;, order = &quot;reverse&quot;)
&gt; grid.dendrogram(dend_merge, test = TRUE, facing = &quot;top&quot;, order = &quot;reverse&quot;)
&gt; grid.dendrogram(dend_merge, test = TRUE, facing = &quot;left&quot;, order = &quot;reverse&quot;)
&gt; grid.dendrogram(dend_merge, test = TRUE, facing = &quot;right&quot;, order = &quot;reverse&quot;)
&gt; 
&gt; 
&gt; m = matrix(rnorm(100), 10)
&gt; dend1 = as.dendrogram(hclust(dist(m)))
&gt; dend1 = adjust_dend_by_x(dend1, unit(1:10, &quot;cm&quot;))
&gt; grid.dendrogram(dend1, test = TRUE)
&gt; 
&gt; dl = cut_dendrogram(dend1, k = 3)
&gt; grid.dendrogram(dl$upper, test = TRUE)
&gt; 
&gt; 
&gt; m1 = matrix(rnorm(100), nr = 10)
&gt; m2 = matrix(rnorm(80), nr = 8)
&gt; m3 = matrix(rnorm(50), nr = 5)
&gt; dend1 = as.dendrogram(hclust(dist(m1)))
&gt; dend2 = as.dendrogram(hclust(dist(m2)))
&gt; dend3 = as.dendrogram(hclust(dist(m3)))
&gt; dend_p = as.dendrogram(hclust(dist(rbind(colMeans(m1), colMeans(m2), colMeans(m3)))))
&gt; dend_m = merge_dendrogram(dend_p, list(dend1, dend2, dend3))
&gt; grid.dendrogram(dend_m, test = T)
&gt; 
&gt; dend_m = merge_dendrogram(dend_p, list(dend1, dend2, dend3), only_parent = TRUE)
&gt; grid.dendrogram(dend_m, test = T)
&gt; 
&gt; require(dendextend)
&gt; dend1 = color_branches(dend1, k = 1, col = &quot;red&quot;)
&gt; dend2 = color_branches(dend2, k = 1, col = &quot;blue&quot;)
&gt; dend3 = color_branches(dend3, k = 1, col = &quot;green&quot;)
&gt; dend_p = color_branches(dend_p, k = 1, col = &quot;orange&quot;)
&gt; dend_m = merge_dendrogram(dend_p, list(dend1, dend2, dend3))
&gt; grid.dendrogram(dend_m, test = T)
&gt; 
&gt; 
&gt; m = matrix(rnorm(120), nc = 12)
&gt; colnames(m) = letters[1:12]
&gt; fa = rep(c(&quot;a&quot;, &quot;b&quot;, &quot;c&quot;), times = c(2, 4, 6))
&gt; dend = cluster_within_group(m, fa)
&gt; grid.dendrogram(dend, test = TRUE)
&gt; 
&gt; 
&gt; # stack overflow problem
&gt; m = matrix(1, nrow = 1000, ncol = 10)
&gt; m[1, 2] = 2
&gt; dend = as.dendrogram(hclust(dist(m)))
&gt; grid.dendrogram(dend, test = T)
&gt; 
&gt; # node attr
&gt; m = matrix(rnorm(100), 10)
&gt; dend = as.dendrogram(hclust(dist(m)))
&gt; require(dendextend)
&gt; dend1 = color_branches(dend, k = 2, col = 1:2)
&gt; grid.dendrogram(dend1, test = T)
&gt; dend1 = dend
&gt; dend1 = dendrapply(dend, function(d) {
+ 	attr(d, &quot;nodePar&quot;) = list(pch = sample(20, 1), cex = runif(1, min = 0.3, max = 1.3), col = rand_color(1))
+ 	d
+ })
&gt; grid.dendrogram(dend1, test = T)
&gt; 
&gt; Heatmap(m, cluster_rows = dend1, cluster_columns = dend1)
&gt; 
&gt; d1 = ComplexHeatmap:::dend_edit_node(dend, method = &quot;top-bottom&quot;, function(d, index) {
+ 	attr(d, &quot;depth&quot;) = length(index)
+ 	d
+ })
&gt; 
&gt; d2 = ComplexHeatmap:::dend_edit_node(dend, method = &quot;bottom-top&quot;, function(d, index) {
+ 	attr(d, &quot;depth&quot;) = length(index)
+ 	d
+ })
&gt; 
&gt; identical(d1, d2)
[1] TRUE
&gt; 
&gt; proc.time()
   user  system elapsed 
  9.235   0.293   9.509 
</PRE>
</DIV><P><SPAN class="filename">ComplexHeatmap.Rcheck/tests/test-gridtext.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(ComplexHeatmap)
Loading required package: grid
========================================
ComplexHeatmap version 2.17.0
Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/
Github page: https://github.com/jokergoo/ComplexHeatmap
Documentation: http://jokergoo.github.io/ComplexHeatmap-reference

If you use it in published research, please cite either one:
- Gu, Z. Complex Heatmap Visualization. iMeta 2022.
- Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional 
    genomic data. Bioinformatics 2016.


The new InteractiveComplexHeatmap package can directly export static 
complex heatmaps into an interactive Shiny app with zero effort. Have a try!

This message can be suppressed by:
  suppressPackageStartupMessages(library(ComplexHeatmap))
========================================

&gt; 
&gt; if(requireNamespace(&quot;gridtext&quot;)) {
+ ##### test anno_richtext ####
+ mat = matrix(rnorm(100), 10)
+ rownames(mat) = letters[1:10]
+ ht = Heatmap(mat, 
+ 	column_title = gt_render(&quot;Some &lt;span style=&#x27;color:blue&#x27;&gt;blue text **in bold.**&lt;/span&gt;&lt;br&gt;And *italics text.*&lt;br&gt;And some &lt;span style=&#x27;font-size:18pt; color:black&#x27;&gt;large&lt;/span&gt; text.&quot;, r = unit(2, &quot;pt&quot;), padding = unit(c(2, 2, 2, 2), &quot;pt&quot;)),
+ 	column_title_gp = gpar(box_fill = &quot;orange&quot;),
+ 	row_labels = gt_render(letters[1:10], padding = unit(c(2, 10, 2, 10), &quot;pt&quot;)),
+ 	row_names_gp = gpar(box_col = rep(2:3, times = 5), box_fill = ifelse(1:10%%2, &quot;yellow&quot;, &quot;white&quot;)),
+ 	row_km = 2, 
+ 	row_title = gt_render(c(&quot;title1&quot;, &quot;title2&quot;)), 
+ 	row_title_gp = gpar(box_fill = &quot;yellow&quot;),
+ 	heatmap_legend_param = list(
+ 		title = gt_render(&quot;&lt;span style=&#x27;color:orange&#x27;&gt;**Legend title**&lt;/span&gt;&quot;), 
+ 		title_gp = gpar(box_fill = &quot;grey&quot;),
+ 		at = c(-3, 0, 3), 
+ 		labels = gt_render(c(&quot;*negative* three&quot;, &quot;zero&quot;, &quot;*positive* three&quot;))
+ 	))
+ ht = rowAnnotation(
+ 	foo = anno_text(gt_render(sapply(LETTERS[1:10], strrep, 10), align_widths = TRUE), 
+ 	                gp = gpar(box_col = &quot;blue&quot;, box_lwd = 2), 
+ 	                just = &quot;right&quot;, 
+ 	                location = unit(1, &quot;npc&quot;)
+ 	)) + ht
+ draw(ht)
+ 
+ }
Loading required namespace: gridtext
&gt; 
&gt; proc.time()
   user  system elapsed 
  5.457   0.283   5.720 
</PRE>
</DIV><P><SPAN class="filename">ComplexHeatmap.Rcheck/tests/test-Heatmap-class.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(circlize)
========================================
circlize version 0.4.15
CRAN page: https://cran.r-project.org/package=circlize
Github page: https://github.com/jokergoo/circlize
Documentation: https://jokergoo.github.io/circlize_book/book/

If you use it in published research, please cite:
Gu, Z. circlize implements and enhances circular visualization
  in R. Bioinformatics 2014.

This message can be suppressed by:
  suppressPackageStartupMessages(library(circlize))
========================================

&gt; library(ComplexHeatmap)
Loading required package: grid
========================================
ComplexHeatmap version 2.17.0
Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/
Github page: https://github.com/jokergoo/ComplexHeatmap
Documentation: http://jokergoo.github.io/ComplexHeatmap-reference

If you use it in published research, please cite either one:
- Gu, Z. Complex Heatmap Visualization. iMeta 2022.
- Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional 
    genomic data. Bioinformatics 2016.


The new InteractiveComplexHeatmap package can directly export static 
complex heatmaps into an interactive Shiny app with zero effort. Have a try!

This message can be suppressed by:
  suppressPackageStartupMessages(library(ComplexHeatmap))
========================================

&gt; library(GetoptLong)
&gt; 
&gt; set.seed(123)
&gt; nr1 = 10; nr2 = 8; nr3 = 6
&gt; nc1 = 6; nc2 = 8; nc3 = 10
&gt; mat = cbind(rbind(matrix(rnorm(nr1*nc1, mean = 1,   sd = 0.5), nr = nr1),
+           matrix(rnorm(nr2*nc1, mean = 0,   sd = 0.5), nr = nr2),
+           matrix(rnorm(nr3*nc1, mean = 0,   sd = 0.5), nr = nr3)),
+     rbind(matrix(rnorm(nr1*nc2, mean = 0,   sd = 0.5), nr = nr1),
+           matrix(rnorm(nr2*nc2, mean = 1,   sd = 0.5), nr = nr2),
+           matrix(rnorm(nr3*nc2, mean = 0,   sd = 0.5), nr = nr3)),
+     rbind(matrix(rnorm(nr1*nc3, mean = 0.5, sd = 0.5), nr = nr1),
+           matrix(rnorm(nr2*nc3, mean = 0.5, sd = 0.5), nr = nr2),
+           matrix(rnorm(nr3*nc3, mean = 1,   sd = 0.5), nr = nr3))
+    )
&gt; 
&gt; rownames(mat) = paste0(&quot;row&quot;, seq_len(nrow(mat)))
&gt; colnames(mat) = paste0(&quot;column&quot;, seq_len(nrow(mat)))
&gt; 
&gt; ht = Heatmap(mat)
&gt; draw(ht, test = TRUE)
&gt; ht
&gt; 
&gt; 
&gt; ht = Heatmap(mat, col = colorRamp2(c(-3, 0, 3), c(&quot;green&quot;, &quot;white&quot;, &quot;red&quot;)))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, name = &quot;test&quot;)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, rect_gp = gpar(col = &quot;black&quot;))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, border = &quot;red&quot;)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ######## test title ##########
&gt; ht = Heatmap(mat, row_title = &quot;blablabla&quot;)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_title = &quot;blablabla&quot;, row_title_side = &quot;right&quot;)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_title = &quot;blablabla&quot;, row_title_gp = gpar(fontsize = 20, font = 2))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; # ht = Heatmap(mat, row_title = &quot;blablabla&quot;, row_title_rot = 45)
&gt; # draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_title = &quot;blablabla&quot;, row_title_rot = 0)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_title = &quot;blablabla&quot;, row_title_gp = gpar(fill = &quot;red&quot;, col = &quot;white&quot;))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_title = &quot;blablabla&quot;)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_title = &quot;blablabla&quot;, column_title_side = &quot;bottom&quot;)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_title = &quot;blablabla&quot;, column_title_gp = gpar(fontsize = 20, font = 2))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; # ht = Heatmap(mat, column_title = &quot;blablabla&quot;, column_title_rot = 45)
&gt; # draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_title = &quot;blablabla&quot;, column_title_rot = 90)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; 
&gt; ### test clustering ####
&gt; 
&gt; ht = Heatmap(mat, cluster_rows = FALSE)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, clustering_distance_rows = &quot;pearson&quot;)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, clustering_distance_rows = function(x) dist(x))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, clustering_distance_rows = function(x, y) 1 - cor(x, y))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, clustering_method_rows = &quot;single&quot;)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_dend_side = &quot;right&quot;)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_dend_width = unit(4, &quot;cm&quot;))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_dend_gp = gpar(lwd = 2, col = &quot;red&quot;))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; dend = as.dendrogram(hclust(dist(mat)))
&gt; ht = Heatmap(mat, cluster_rows = dend)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; library(dendextend)

---------------------
Welcome to dendextend version 1.17.1
Type citation(&#x27;dendextend&#x27;) for how to cite the package.

Type browseVignettes(package = &#x27;dendextend&#x27;) for the package vignette.
The github page is: https://github.com/talgalili/dendextend/

Suggestions and bug-reports can be submitted at: https://github.com/talgalili/dendextend/issues
You may ask questions at stackoverflow, use the r and dendextend tags: 
	 https://stackoverflow.com/questions/tagged/dendextend

	To suppress this message use:  suppressPackageStartupMessages(library(dendextend))
---------------------


Attaching package: &#x27;dendextend&#x27;

The following object is masked from &#x27;package:stats&#x27;:

    cutree

&gt; dend = color_branches(dend, k = 3)
&gt; ht = Heatmap(mat, cluster_rows = dend)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; 
&gt; ht = Heatmap(mat, cluster_columns = FALSE)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, clustering_distance_columns = &quot;pearson&quot;)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, clustering_distance_columns = function(x) dist(x))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, clustering_distance_columns = function(x, y) 1 - cor(x, y))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, clustering_method_columns = &quot;single&quot;)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_dend_side = &quot;bottom&quot;)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_dend_height = unit(4, &quot;cm&quot;))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_dend_gp = gpar(lwd = 2, col = &quot;red&quot;))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; dend = as.dendrogram(hclust(dist(t(mat))))
&gt; ht = Heatmap(mat, cluster_columns = dend)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; dend = color_branches(dend, k = 3)
&gt; ht = Heatmap(mat, cluster_columns = dend)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; 
&gt; ### test row/column order
&gt; od = c(seq(1, 24, by = 2), seq(2, 24, by = 2))
&gt; ht = Heatmap(mat, row_order = od)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_order = od, cluster_rows = TRUE)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_order = od)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_order = od, cluster_columns = TRUE)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; 
&gt; #### test row/column names #####
&gt; ht = Heatmap(unname(mat))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, show_row_names = FALSE)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_names_side = &quot;left&quot;)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; random_str2 = function(k) {
+ 	sapply(1:k, function(i) paste(sample(letters, sample(5:10, 1)), collapse = &quot;&quot;))
+ }
&gt; ht = Heatmap(mat, row_labels = random_str2(24))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_names_gp = gpar(fontsize = 20))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_names_gp = gpar(fontsize = 1:24/2 + 5))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_names_rot = 45)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_names_rot = 45, row_names_side = &quot;left&quot;)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, show_column_names = FALSE)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_names_side = &quot;top&quot;)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_labels = random_str2(24))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_names_gp = gpar(fontsize = 20))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_names_gp = gpar(fontsize = 1:24/2 + 5))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_names_rot = 45)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ### test annotations ####
&gt; anno = HeatmapAnnotation(
+ 	foo = 1:24,
+ 	df = data.frame(type = c(rep(&quot;A&quot;, 12), rep(&quot;B&quot;, 12))),
+ 	bar = anno_barplot(24:1))
&gt; ht = Heatmap(mat, top_annotation = anno)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, bottom_annotation = anno)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, top_annotation = anno, bottom_annotation = anno)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; 
&gt; ### test split ####
&gt; ht = Heatmap(mat, km = 3)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_km = 3)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, split = rep(c(&quot;A&quot;, &quot;B&quot;), times = c(6, 18)))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_split = rep(c(&quot;A&quot;, &quot;B&quot;), times = c(6, 18)))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_split = factor(rep(c(&quot;A&quot;, &quot;B&quot;), times = c(6, 18)), levels = c(&quot;B&quot;, &quot;A&quot;)))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_split = rep(c(&quot;A&quot;, &quot;B&quot;), 12), row_gap = unit(5, &quot;mm&quot;))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_split = data.frame(rep(c(&quot;A&quot;, &quot;B&quot;), 12), rep(c(&quot;C&quot;, &quot;D&quot;), each = 12)))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_split = data.frame(rep(c(&quot;A&quot;, &quot;B&quot;), 12), rep(c(&quot;C&quot;, &quot;D&quot;), each = 12)),
+ 	row_gap = unit(c(1, 2, 3), &quot;mm&quot;))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_km = 3, row_title = &quot;foo&quot;)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_km = 3, row_title = &quot;cluster%s&quot;)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_km = 3, row_title = &quot;cluster%s&quot;, row_title_rot = 0)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_km = 3, row_title = &quot;cluster%s&quot;, row_title_gp = gpar(fill = 2:4, col = &quot;white&quot;))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_km = 3, row_title = NULL)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_km = 3, row_names_gp = gpar(col = 2:4))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_split = rep(c(&quot;A&quot;, &quot;B&quot;), times = c(6, 18)), row_km = 3)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_split = rep(c(&quot;A&quot;, &quot;B&quot;), times = c(6, 18)), row_km = 3, row_title = &quot;cluster%s,group%s&quot;, row_title_rot = 0)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_split = 2)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_split = 2, row_title = &quot;foo&quot;)
&gt; ht = Heatmap(mat, row_split = 2, row_title = &quot;cluster%s&quot;)
&gt; 
&gt; 
&gt; dend = as.dendrogram(hclust(dist(mat)))
&gt; ht = Heatmap(mat, cluster_rows = dend, row_split = 2)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_split = 2, row_names_gp = gpar(col = 2:3))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; 
&gt; ### column split
&gt; ht = Heatmap(mat, column_km = 2)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_km = 2, column_gap = unit(1, &quot;cm&quot;))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_split = rep(c(&quot;A&quot;, &quot;B&quot;), times = c(6, 18)))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_split = data.frame(rep(c(&quot;A&quot;, &quot;B&quot;), 12), rep(c(&quot;C&quot;, &quot;D&quot;), each = 12)),
+ 	column_gap = unit(c(1, 2, 3), &quot;mm&quot;))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_km = 2, column_title = &quot;foo&quot;)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_km = 2, column_title = &quot;cluster%s&quot;)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_km = 2, column_title = &quot;cluster%s&quot;, column_title_rot = 90)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_km = 2, column_title = &quot;cluster%s&quot;, column_title_gp = gpar(fill = 2:3, col = &quot;white&quot;))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_km = 2, column_title = NULL)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_km = 2, column_names_gp = gpar(col = 2:3))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_split = factor(rep(c(&quot;A&quot;, &quot;B&quot;), times = c(6, 18)), levels = c(&quot;A&quot;, &quot;B&quot;)), column_km = 2)
&gt; draw(ht, test = TRUE)
&gt; ht = Heatmap(mat, column_split = factor(rep(c(&quot;A&quot;, &quot;B&quot;), times = c(6, 18)), levels = c(&quot;B&quot;, &quot;A&quot;)), column_km = 2)
&gt; 
&gt; 
&gt; ht = Heatmap(mat, column_split = rep(c(&quot;A&quot;, &quot;B&quot;), times = c(6, 18)), column_km = 2, 
+ 	column_title = &quot;cluster%s,group%s&quot;, column_title_rot = 90)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, column_split = 3)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; dend = as.dendrogram(hclust(dist(t(mat))))
&gt; ht = Heatmap(mat, cluster_columns = dend, column_split = 3)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, top_annotation = anno, bottom_annotation = anno, column_km = 2)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, top_annotation = anno, bottom_annotation = anno, column_split = 3)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ### combine row and column split
&gt; ht = Heatmap(mat, row_km = 3, column_km = 3)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_split = 3, column_split = 3)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_km = 3, column_split = 3)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_split = rep(c(&quot;A&quot;, &quot;B&quot;), 12), 
+ 	column_split = rep(c(&quot;C&quot;, &quot;D&quot;), 12))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, top_annotation = anno,
+ 	row_split = rep(c(&quot;A&quot;, &quot;B&quot;), 12), 
+ 	row_names_gp = gpar(col = 2:3), row_gap = unit(2, &quot;mm&quot;),
+ 	column_split = 3,
+ 	column_names_gp = gpar(col = 2:4), column_gap = unit(4, &quot;mm&quot;)
+ )
&gt; draw(ht, test = TRUE)
&gt; 
&gt; 
&gt; #### character matrix
&gt; mat3 = matrix(sample(letters[1:6], 100, replace = TRUE), 10, 10)
&gt; rownames(mat3) = {x = letters[1:10]; x[1] = &quot;aaaaaaaaaaaaaaaaaaaaaaa&quot;;x}
&gt; ht = Heatmap(mat3, rect_gp = gpar(col = &quot;white&quot;))
&gt; draw(ht, test = TRUE)
&gt; 
&gt; 
&gt; ### cell_fun
&gt; mat = matrix(1:9, 3, 3)
&gt; rownames(mat) = letters[1:3]
&gt; colnames(mat) = letters[1:3]
&gt; 
&gt; ht = Heatmap(mat, rect_gp = gpar(col = &quot;white&quot;), cell_fun = function(j, i, x, y, width, height, fill) grid.text(mat[i, j], x = x, y = y),
+ 	cluster_rows = FALSE, cluster_columns = FALSE, row_names_side = &quot;left&quot;, column_names_side = &quot;top&quot;,
+ 	column_names_rot = 0)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; 
&gt; ### test the size
&gt; ht = Heatmap(mat)
&gt; ht = prepare(ht)
&gt; ht@heatmap_param[c(&quot;width&quot;, &quot;height&quot;)]
$width
[1] 1npc

$height
[1] 1npc

&gt; ht@matrix_param[c(&quot;width&quot;, &quot;height&quot;)]
$width
[1] 3null

$height
[1] 3null

&gt; 
&gt; ht = Heatmap(mat, width = unit(10, &quot;cm&quot;), height = unit(10, &quot;cm&quot;))
&gt; ht = prepare(ht)
&gt; ht@heatmap_param[c(&quot;width&quot;, &quot;height&quot;)]
$width
[1] 114.853733333333mm

$height
[1] 114.853733333333mm

&gt; ht@matrix_param[c(&quot;width&quot;, &quot;height&quot;)]
$width
[1] 10cm

$height
[1] 10cm

&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, width = unit(10, &quot;cm&quot;))
&gt; ht = prepare(ht)
&gt; ht@heatmap_param[c(&quot;width&quot;, &quot;height&quot;)]
$width
[1] 114.853733333333mm

$height
[1] 1npc

&gt; ht@matrix_param[c(&quot;width&quot;, &quot;height&quot;)]
$width
[1] 10cm

$height
[1] 3null

&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, heatmap_width = unit(10, &quot;cm&quot;), heatmap_height = unit(10, &quot;cm&quot;))
&gt; ht = prepare(ht)
&gt; ht@heatmap_param[c(&quot;width&quot;, &quot;height&quot;)]
$width
[1] 10cm

$height
[1] 10cm

&gt; ht@matrix_param[c(&quot;width&quot;, &quot;height&quot;)]
$width
[1] 85.1462666666667mm

$height
[1] 85.1462666666667mm

&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, heatmap_width = unit(10, &quot;cm&quot;))
&gt; ht = prepare(ht)
&gt; ht@heatmap_param[c(&quot;width&quot;, &quot;height&quot;)]
$width
[1] 10cm

$height
[1] 1npc

&gt; ht@matrix_param[c(&quot;width&quot;, &quot;height&quot;)]
$width
[1] 85.1462666666667mm

$height
[1] 3null

&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, use_raster = TRUE)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_km = 2, use_raster = TRUE)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; ht = Heatmap(mat, row_km = 2, column_km = 2, use_raster = TRUE)
&gt; draw(ht, test = TRUE)
&gt; 
&gt; #### test global padding
&gt; ra = rowAnnotation(foo = 1:3)
&gt; ht = Heatmap(mat, show_column_names = FALSE) + ra
&gt; draw(ht)
&gt; 
&gt; ht = Heatmap(matrix(rnorm(100), 10), row_km = 2, row_title = &quot;&quot;)
&gt; draw(ht)
&gt; 
&gt; if(0) {
+ ht = Heatmap(matrix(rnorm(100), 10), heatmap_width = unit(5, &quot;mm&quot;))
+ draw(ht)
+ }
&gt; 
&gt; proc.time()
   user  system elapsed 
 28.317   0.643  28.944 
</PRE>
</DIV><P><SPAN class="filename">ComplexHeatmap.Rcheck/tests/test-Heatmap-cluster.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(circlize)
========================================
circlize version 0.4.15
CRAN page: https://cran.r-project.org/package=circlize
Github page: https://github.com/jokergoo/circlize
Documentation: https://jokergoo.github.io/circlize_book/book/

If you use it in published research, please cite:
Gu, Z. circlize implements and enhances circular visualization
  in R. Bioinformatics 2014.

This message can be suppressed by:
  suppressPackageStartupMessages(library(circlize))
========================================

&gt; library(ComplexHeatmap)
Loading required package: grid
========================================
ComplexHeatmap version 2.17.0
Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/
Github page: https://github.com/jokergoo/ComplexHeatmap
Documentation: http://jokergoo.github.io/ComplexHeatmap-reference

If you use it in published research, please cite either one:
- Gu, Z. Complex Heatmap Visualization. iMeta 2022.
- Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional 
    genomic data. Bioinformatics 2016.


The new InteractiveComplexHeatmap package can directly export static 
complex heatmaps into an interactive Shiny app with zero effort. Have a try!

This message can be suppressed by:
  suppressPackageStartupMessages(library(ComplexHeatmap))
========================================

&gt; library(GetoptLong)
&gt; 
&gt; # ht_opt(&quot;verbose&quot; = TRUE)
&gt; m = matrix(rnorm(50), nr = 10)
&gt; 
&gt; ht = Heatmap(m)
&gt; ht = make_row_cluster(ht)
&gt; 
&gt; ht = Heatmap(m, cluster_rows = FALSE)
&gt; ht = make_row_cluster(ht)
&gt; 
&gt; ht = Heatmap(m, row_km = 2)
&gt; ht = make_row_cluster(ht)
&gt; 
&gt; ht = Heatmap(m, row_split = sample(letters[1:2], 10, replace = TRUE))
&gt; ht = make_row_cluster(ht)
&gt; 
&gt; ht = Heatmap(m, cluster_rows = hclust(dist(m)))
&gt; ht = make_row_cluster(ht)
&gt; 
&gt; ht = Heatmap(m, cluster_rows = hclust(dist(m)), row_split = 2)
&gt; ht = make_row_cluster(ht)
&gt; 
&gt; # ht_opt(&quot;verbose&quot; = FALSE)
&gt; 
&gt; proc.time()
   user  system elapsed 
  3.084   0.153   3.217 
</PRE>
</DIV><P><SPAN class="filename">ComplexHeatmap.Rcheck/tests/test-HeatmapAnnotation.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(circlize)
========================================
circlize version 0.4.15
CRAN page: https://cran.r-project.org/package=circlize
Github page: https://github.com/jokergoo/circlize
Documentation: https://jokergoo.github.io/circlize_book/book/

If you use it in published research, please cite:
Gu, Z. circlize implements and enhances circular visualization
  in R. Bioinformatics 2014.

This message can be suppressed by:
  suppressPackageStartupMessages(library(circlize))
========================================

&gt; library(ComplexHeatmap)
Loading required package: grid
========================================
ComplexHeatmap version 2.17.0
Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/
Github page: https://github.com/jokergoo/ComplexHeatmap
Documentation: http://jokergoo.github.io/ComplexHeatmap-reference

If you use it in published research, please cite either one:
- Gu, Z. Complex Heatmap Visualization. iMeta 2022.
- Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional 
    genomic data. Bioinformatics 2016.


The new InteractiveComplexHeatmap package can directly export static 
complex heatmaps into an interactive Shiny app with zero effort. Have a try!

This message can be suppressed by:
  suppressPackageStartupMessages(library(ComplexHeatmap))
========================================

&gt; library(GetoptLong)
&gt; 
&gt; 
&gt; ha = HeatmapAnnotation(foo = 1:10)
&gt; ha
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_0 
  position: column 
  items: 10 
  width: 1npc 
  height: 5mm 
  this object is subsettable
  6.75733333333333mm extension on the right 

 name   annotation_type color_mapping height
  foo continuous vector        random    5mm
&gt; 
&gt; 
&gt; ha = HeatmapAnnotation(foo = cbind(1:10, 10:1))
&gt; ha
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_1 
  position: column 
  items: 10 
  width: 1npc 
  height: 10mm 
  this object is subsettable
  6.75733333333333mm extension on the right 

 name   annotation_type color_mapping height
  foo continuous matrix        random   10mm
&gt; draw(ha, test = &quot;matrix as column annotation&quot;)
&gt; 
&gt; ha = HeatmapAnnotation(foo = 1:10, bar = sample(c(&quot;a&quot;, &quot;b&quot;), 10, replace = TRUE),
+ 	pt = anno_points(1:10), annotation_name_side = &quot;left&quot;)
&gt; draw(ha, test = &quot;complex annotations&quot;)
&gt; 
&gt; ha = HeatmapAnnotation(foo = 1:10, bar = sample(c(&quot;a&quot;, &quot;b&quot;), 10, replace = TRUE),
+ 	pt = anno_points(1:10), annotation_name_side = &quot;left&quot;, height = unit(8, &quot;cm&quot;))
&gt; draw(ha, test = &quot;complex annotations&quot;)
&gt; 
&gt; 
&gt; ha = HeatmapAnnotation(foo = 1:10, bar = sample(c(&quot;a&quot;, &quot;b&quot;), 10, replace = TRUE))
&gt; 
&gt; ha = HeatmapAnnotation(foo = 1:10, 
+ 	bar = cbind(1:10, 10:1),
+ 	pt = anno_points(1:10),
+ 	gap = unit(2, &quot;mm&quot;))
&gt; draw(ha, test = &quot;complex annotations&quot;)
&gt; 
&gt; ha2 = re_size(ha, annotation_height = unit(1:3, &quot;cm&quot;))
&gt; draw(ha2, test = &quot;complex annotations&quot;)
&gt; ha2 = re_size(ha, annotation_height = 1, height = unit(6, &quot;cm&quot;))
&gt; draw(ha2, test = &quot;complex annotations&quot;)
&gt; ha2 = re_size(ha, annotation_height = 1:3, height = unit(6, &quot;cm&quot;))
&gt; draw(ha2, test = &quot;complex annotations&quot;)
&gt; ha2 = re_size(ha, annotation_height = unit(c(1, 2, 3), c(&quot;null&quot;, &quot;null&quot;, &quot;cm&quot;)), height = unit(6, &quot;cm&quot;))
&gt; draw(ha2, test = &quot;complex annotations&quot;)
&gt; ha2 = re_size(ha, annotation_height = unit(c(2, 2, 3), c(&quot;cm&quot;, &quot;null&quot;, &quot;cm&quot;)), height = unit(6, &quot;cm&quot;))
&gt; draw(ha2, test = &quot;complex annotations&quot;)
&gt; ha2 = re_size(ha, annotation_height = unit(c(2, 2, 3), c(&quot;cm&quot;, &quot;cm&quot;, &quot;cm&quot;)))
&gt; draw(ha2, test = &quot;complex annotations&quot;)
&gt; ha2 = re_size(ha[, 1:2], annotation_height = 1, height = unit(4, &quot;cm&quot;))
&gt; draw(ha2, test = &quot;complex annotations&quot;)
&gt; ha2 = re_size(ha[, 1:2], annotation_height = c(1, 4), height = unit(4, &quot;cm&quot;))
&gt; draw(ha2, test = &quot;complex annotations&quot;)
&gt; ha2 = re_size(ha[, 1:2], height = unit(6, &quot;cm&quot;))
&gt; draw(ha2, test = &quot;complex annotations&quot;)
&gt; 
&gt; ha2 = re_size(ha, height = unit(6, &quot;cm&quot;))
&gt; draw(ha2, test = &quot;complex annotations&quot;)
&gt; 
&gt; #### test anno_empty and self-defined anotation function
&gt; ha = HeatmapAnnotation(foo = anno_empty(), height = unit(4, &quot;cm&quot;))
&gt; draw(ha, 1:10, test = &quot;anno_empty&quot;)
&gt; ha = HeatmapAnnotation(foo = anno_empty(), bar = 1:10, height = unit(4, &quot;cm&quot;))
&gt; draw(ha, 1:10, test = &quot;anno_empty&quot;)
&gt; ha = HeatmapAnnotation(foo = anno_empty(), bar = 1:10, height = unit(4, &quot;cm&quot;))
&gt; draw(ha, 1:10, test = &quot;anno_empty&quot;)
&gt; 
&gt; ha = HeatmapAnnotation(foo = function(index) {grid.rect()}, bar = 1:10, height = unit(4, &quot;cm&quot;))
&gt; draw(ha, 1:10, test = &quot;self-defined function&quot;)
&gt; 
&gt; 
&gt; lt = lapply(1:10, function(x) cumprod(1 + runif(1000, -x/100, x/100)) - 1)
&gt; ha = HeatmapAnnotation(foo = 1:10, bar = sample(c(&quot;a&quot;, &quot;b&quot;), 10, replace = TRUE),
+ 	anno = anno_horizon(lt), which = &quot;row&quot;)
&gt; draw(ha, test = &quot;complex annotations on row&quot;)
&gt; 
&gt; ## test row annotation with no heatmap
&gt; rowAnnotation(foo = 1:10, bar = anno_points(10:1))
A HeatmapAnnotation object with 2 annotations
  name: heatmap_annotation_11 
  position: row 
  items: 10 
  width: 15.3514598035146mm 
  height: 1npc 
  this object is subsettable
  9.17784444444445mm extension on the bottom 

 name   annotation_type color_mapping width
  foo continuous vector        random   5mm
  bar     anno_points()                10mm
&gt; 
&gt; if(0) {
+ HeatmapAnnotation(1:10)
+ 
+ HeatmapAnnotation(data.frame(1:10))
+ }
&gt; 
&gt; 
&gt; ha = HeatmapAnnotation(summary = anno_summary(height = unit(4, &quot;cm&quot;)))
&gt; v = sample(letters[1:2], 50, replace = TRUE)
&gt; split = sample(letters[1:2], 50, replace = TRUE)
&gt; 
&gt; ht = Heatmap(v, top_annotation = ha, width = unit(1, &quot;cm&quot;), split = split)
&gt; draw(ht)
&gt; 
&gt; ha = HeatmapAnnotation(summary = anno_summary(gp = gpar(fill = 2:3), height = unit(4, &quot;cm&quot;)))
&gt; v = rnorm(50)
&gt; ht = Heatmap(v, top_annotation = ha, width = unit(1, &quot;cm&quot;), split = split)
&gt; draw(ht)
&gt; 
&gt; 
&gt; ### auto adjust
&gt; m = matrix(rnorm(100), 10)
&gt; ht_list = Heatmap(m, top_annotation = HeatmapAnnotation(foo = 1:10), column_dend_height = unit(4, &quot;cm&quot;)) +
+ 	Heatmap(m, top_annotation = HeatmapAnnotation(bar = anno_points(1:10)),
+ 		cluster_columns = FALSE)
&gt; draw(ht_list)
&gt; 
&gt; fun = function(index) {
+ 	grid.rect()
+ }
&gt; ha = HeatmapAnnotation(fun = fun, height = unit(4, &quot;cm&quot;))
&gt; draw(ha, 1:10, test = TRUE)
&gt; 
&gt; ha = rowAnnotation(fun = fun, width = unit(4, &quot;cm&quot;))
&gt; draw(ha, 1:10, test = TRUE)
&gt; 
&gt; 
&gt; ## test anno_mark
&gt; m = matrix(rnorm(1000), nrow = 100)
&gt; ha1 = rowAnnotation(foo = anno_mark(at = c(1:4, 20, 60, 97:100), labels = month.name[1:10]))
&gt; ht = Heatmap(m, name = &quot;mat&quot;, cluster_rows = FALSE, right_annotation = ha1)
&gt; draw(ht)
&gt; ht = Heatmap(m, name = &quot;mat&quot;, cluster_rows = FALSE) + ha1
&gt; draw(ht)
&gt; 
&gt; split = rep(&quot;a&quot;, 100); split[c(1:4, 20, 60, 98:100)] = &quot;b&quot;
&gt; ht = Heatmap(m, name = &quot;mat&quot;, cluster_rows = FALSE, right_annotation = ha1, row_split = split, gap = unit(1, &quot;cm&quot;))
&gt; draw(ht)
&gt; ht = Heatmap(m, name = &quot;mat&quot;, cluster_rows = FALSE, row_split = split, gap = unit(1, &quot;cm&quot;)) + ha1
&gt; draw(ht)
&gt; 
&gt; # ha has two annotations
&gt; ha2 = rowAnnotation(foo = anno_mark(at = c(1:4, 20, 60, 97:100), labels = month.name[1:10]), bar = 1:100)
&gt; ht = Heatmap(m, name = &quot;mat&quot;, cluster_rows = FALSE, right_annotation = ha2)
&gt; draw(ht)
&gt; ht = Heatmap(m, name = &quot;mat&quot;, cluster_rows = FALSE) + ha2
&gt; draw(ht)
&gt; 
&gt; ht = Heatmap(m, name = &quot;mat&quot;, cluster_rows = FALSE, right_annotation = ha2, row_split = split, gap = unit(1, &quot;cm&quot;))
&gt; draw(ht)
&gt; ht = Heatmap(m, name = &quot;mat&quot;, cluster_rows = FALSE, row_split = split, gap = unit(1, &quot;cm&quot;)) + ha2
&gt; draw(ht)
&gt; 
&gt; ## test anno_mark as column annotation
&gt; m = matrix(rnorm(1000), ncol = 100)
&gt; ha1 = columnAnnotation(foo = anno_mark(at = c(1:4, 20, 60, 97:100), labels = month.name[1:10]))
&gt; ht = Heatmap(m, name = &quot;mat&quot;, cluster_columns = FALSE, top_annotation = ha1)
&gt; draw(ht)
&gt; ht_list = ha1 %v% Heatmap(m, name = &quot;mat&quot;, cluster_columns = FALSE)
&gt; draw(ht_list)
&gt; 
&gt; split = rep(&quot;a&quot;, 100); split[c(1:4, 20, 60, 98:100)] = &quot;b&quot;
&gt; ht = Heatmap(m, name = &quot;mat&quot;, cluster_columns = FALSE, top_annotation = ha1, column_split = split, column_gap = unit(1, &quot;cm&quot;))
&gt; draw(ht)
&gt; ht_list = ha1 %v% Heatmap(m, name = &quot;mat&quot;, cluster_columns = FALSE, column_split = split, gap = unit(1, &quot;cm&quot;))
&gt; draw(ht_list)
&gt; 
&gt; # ha has two annotations
&gt; ha2 = HeatmapAnnotation(foo = anno_mark(at = c(1:4, 20, 60, 97:100), labels = month.name[1:10]), bar = 1:100)
&gt; ht = Heatmap(m, name = &quot;mat&quot;, cluster_columns = FALSE, top_annotation = ha2)
&gt; draw(ht)
&gt; ht_list = ha2 %v% Heatmap(m, name = &quot;mat&quot;, cluster_columns = FALSE)
&gt; draw(ht_list)
&gt; 
&gt; ht = Heatmap(m, name = &quot;mat&quot;, cluster_columns = FALSE, top_annotation = ha2, column_split = split, column_gap = unit(1, &quot;cm&quot;))
&gt; draw(ht)
&gt; ht_list = ha2 %v% Heatmap(m, name = &quot;mat&quot;, cluster_columns = FALSE, column_split = split, column_gap = unit(1, &quot;cm&quot;))
&gt; draw(ht_list)
&gt; 
&gt; 
&gt; ### when there are only simple annotations
&gt; col_fun = colorRamp2(c(0, 10), c(&quot;white&quot;, &quot;blue&quot;))
&gt; ha = HeatmapAnnotation(
+     foo = cbind(a = 1:10, b = 10:1), 
+     bar = sample(letters[1:3], 10, replace = TRUE),
+     col = list(foo = col_fun,
+                bar = c(&quot;a&quot; = &quot;red&quot;, &quot;b&quot; = &quot;green&quot;, &quot;c&quot; = &quot;blue&quot;)
+     ),
+     simple_anno_size = unit(1, &quot;cm&quot;)
+ )
&gt; draw(ha, test = TRUE)
&gt; 
&gt; set.seed(123)
&gt; mat1 = matrix(rnorm(80, 2), 8, 10)
&gt; mat1 = rbind(mat1, matrix(rnorm(40, -2), 4, 10))
&gt; rownames(mat1) = paste0(&quot;R&quot;, 1:12)
&gt; colnames(mat1) = paste0(&quot;C&quot;, 1:10)
&gt; 
&gt; mat2 = matrix(runif(60, max = 3, min = 1), 6, 10)
&gt; mat2 = rbind(mat2, matrix(runif(60, max = 2, min = 0), 6, 10))
&gt; rownames(mat2) = paste0(&quot;R&quot;, 1:12)
&gt; colnames(mat2) = paste0(&quot;C&quot;, 1:10)
&gt; 
&gt; ind = sample(12, 12)
&gt; mat1 = mat1[ind, ]
&gt; mat2 = mat2[ind, ]
&gt; 
&gt; ha1 = HeatmapAnnotation(foo1 = 1:10, 
+ 	                    annotation_height = unit(1, &quot;cm&quot;),
+ 	                    simple_anno_size_adjust = TRUE,
+                         annotation_name_side = &quot;left&quot;)
&gt; ha2 = HeatmapAnnotation(df = data.frame(foo1 = 1:10,
+                                         foo2 = 1:10,
+                                         foo4 = 1:10,
+                                         foo5 = 1:10))
&gt; ht1 = Heatmap(mat1, name = &quot;rnorm&quot;, top_annotation = ha1)
&gt; ht2 = Heatmap(mat2, name = &quot;runif&quot;, top_annotation = ha2)
&gt; 
&gt; draw(ht1 + ht2)
&gt; 
&gt; ##### test size of a single simple annotation
&gt; 
&gt; ha = HeatmapAnnotation(foo1 = 1:10, 
+ 	simple_anno_size = unit(1, &quot;cm&quot;)
+ )
&gt; ha = HeatmapAnnotation(foo1 = 1:10, 
+ 	annotation_height = unit(1, &quot;cm&quot;),
+ 	simple_anno_size_adjust = TRUE
+ )
&gt; ha = HeatmapAnnotation(foo1 = 1:10, 
+ 	height = unit(1, &quot;cm&quot;),
+ 	simple_anno_size_adjust = TRUE
+ )
&gt; 
&gt; 
&gt; ## annotation with the same names
&gt; 
&gt; set.seed(123)
&gt; m = matrix(rnorm(100), 10)
&gt; ha1 = HeatmapAnnotation(foo = sample(c(&quot;a&quot;, &quot;b&quot;), 10, replace = TRUE))
&gt; ha2 = HeatmapAnnotation(foo = sample(c(&quot;b&quot;, &quot;c&quot;), 10, replace = TRUE))
&gt; 
&gt; ht_list = Heatmap(m, top_annotation = ha1) + 
+ 	Heatmap(m, top_annotation = ha2)
&gt; draw(ht_list)
&gt; 
&gt; ha1 = HeatmapAnnotation(foo = sample(c(&quot;a&quot;, &quot;b&quot;), 10, replace = TRUE),
+ 	annotation_legend_param = list(
+ 		foo = list(title = &quot;letters&quot;, 
+ 			       at = c(&quot;a&quot;, &quot;b&quot;, &quot;c&quot;),
+ 			       labels = c(&quot;A&quot;, &quot;B&quot;, &quot;C&quot;)
+ 			  )
+ 	))
&gt; ha2 = HeatmapAnnotation(foo = sample(c(&quot;b&quot;, &quot;c&quot;), 10, replace = TRUE))
&gt; 
&gt; ht_list = Heatmap(m, top_annotation = ha1) + 
+ 	Heatmap(m, top_annotation = ha2)
&gt; draw(ht_list)
&gt; 
&gt; x = matrix(rnorm(6), ncol=3)
&gt; subtype_col = c(&quot;Basal&quot; = &quot;purple&quot;,&quot;Her2&quot; = &quot;black&quot;,&quot;Normal&quot; = &quot;blue&quot;)
&gt; h1 &lt;- HeatmapAnnotation(&quot;Subtype&quot; = c(&quot;Basal&quot;,&quot;Her2&quot;, &quot;Normal&quot;),
+                         col = list(&quot;Subtype&quot; = subtype_col))
&gt; h2 &lt;- HeatmapAnnotation(&quot;Subtype&quot; = c(&quot;Normal&quot;,&quot;Normal&quot;, &quot;Basal&quot;),
+                         col = list(&quot;Subtype&quot; = subtype_col))
&gt; 
&gt; ht_list = Heatmap(x,top_annotation = h1) + Heatmap(x,top_annotation = h2)
&gt; draw(ht_list)
&gt; 
&gt; 
&gt; ### test annotation_label
&gt; ha = HeatmapAnnotation(foo = 1:10, bar = letters[1:10],
+ 	annotation_label = c(&quot;anno1&quot;, &quot;anno2&quot;))
&gt; draw(ha, test = TRUE)
&gt; 
&gt; ha = HeatmapAnnotation(foo = 1:10, bar = letters[1:10],
+ 	annotation_label = list(foo = &quot;anno1&quot;))
&gt; draw(ha, test = TRUE)
&gt; 
&gt; 
&gt; ha = HeatmapAnnotation(foo = 1:10, bar = letters[1:10],
+ 	annotation_label = list(
+ 		foo = gt_render(&quot;foo&quot;, gp = gpar(box_fill = &quot;red&quot;))))
Loading required namespace: gridtext
&gt; draw(ha, test = TRUE)
&gt; 
&gt; ha = HeatmapAnnotation(foo = 1:10, bar = letters[1:10],
+ 	annotation_label = list(
+ 		foo = gt_render(&quot;foo&quot;, gp = gpar(box_fill = &quot;red&quot;)),
+ 		bar = gt_render(&quot;bar&quot;, gp = gpar(box_fill = &quot;blue&quot;))))
&gt; draw(ha, test = TRUE)
&gt; 
&gt; 
&gt; ### test whether arguments can be captured
&gt; HeatmapAnnotation(a = 1:10)
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_38 
  position: column 
  items: 10 
  width: 1npc 
  height: 5mm 
  this object is subsettable
  3.35373333333333mm extension on the right 

 name   annotation_type color_mapping height
    a continuous vector        random    5mm
&gt; rowAnnotation(a = 1:10)
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_39 
  position: row 
  items: 10 
  width: 5mm 
  height: 1npc 
  this object is subsettable
  3.35373333333333mm extension on the bottom 

 name   annotation_type color_mapping width
    a continuous vector        random   5mm
&gt; columnAnnotation(a = 1:10)
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_40 
  position: column 
  items: 10 
  width: 1npc 
  height: 5mm 
  this object is subsettable
  3.35373333333333mm extension on the right 

 name   annotation_type color_mapping height
    a continuous vector        random    5mm
&gt; do.call(HeatmapAnnotation, list(a = 1:10))
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_41 
  position: column 
  items: 10 
  width: 1npc 
  height: 5mm 
  this object is subsettable
  3.35373333333333mm extension on the right 

 name   annotation_type color_mapping height
    a continuous vector        random    5mm
&gt; do.call(rowAnnotation, list(a = 1:10))
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_42 
  position: row 
  items: 10 
  width: 5mm 
  height: 1npc 
  this object is subsettable
  3.35373333333333mm extension on the bottom 

 name   annotation_type color_mapping width
    a continuous vector        random   5mm
&gt; do.call(columnAnnotation, list(a = 1:10))
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_43 
  position: column 
  items: 10 
  width: 1npc 
  height: 5mm 
  this object is subsettable
  3.35373333333333mm extension on the right 

 name   annotation_type color_mapping height
    a continuous vector        random    5mm
&gt; do.call(&quot;HeatmapAnnotation&quot;, list(a = 1:10))
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_44 
  position: column 
  items: 10 
  width: 1npc 
  height: 5mm 
  this object is subsettable
  3.35373333333333mm extension on the right 

 name   annotation_type color_mapping height
    a continuous vector        random    5mm
&gt; do.call(&quot;rowAnnotation&quot;, list(a = 1:10))
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_45 
  position: row 
  items: 10 
  width: 5mm 
  height: 1npc 
  this object is subsettable
  3.35373333333333mm extension on the bottom 

 name   annotation_type color_mapping width
    a continuous vector        random   5mm
&gt; do.call(&quot;columnAnnotation&quot;, list(a = 1:10))
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_46 
  position: column 
  items: 10 
  width: 1npc 
  height: 5mm 
  this object is subsettable
  3.35373333333333mm extension on the right 

 name   annotation_type color_mapping height
    a continuous vector        random    5mm
&gt; 
&gt; f = function() HeatmapAnnotation(a = 1:10)
&gt; f()
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_47 
  position: column 
  items: 10 
  width: 1npc 
  height: 5mm 
  this object is subsettable
  3.35373333333333mm extension on the right 

 name   annotation_type color_mapping height
    a continuous vector        random    5mm
&gt; f = function() rowAnnotation(a = 1:10)
&gt; f()
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_48 
  position: row 
  items: 10 
  width: 5mm 
  height: 1npc 
  this object is subsettable
  3.35373333333333mm extension on the bottom 

 name   annotation_type color_mapping width
    a continuous vector        random   5mm
&gt; f = function() columnAnnotation(a = 1:10)
&gt; f()
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_49 
  position: column 
  items: 10 
  width: 1npc 
  height: 5mm 
  this object is subsettable
  3.35373333333333mm extension on the right 

 name   annotation_type color_mapping height
    a continuous vector        random    5mm
&gt; 
&gt; sapply(1, function(x) HeatmapAnnotation(a = 1:10))
[[1]]
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_50 
  position: column 
  items: 10 
  width: 1npc 
  height: 5mm 
  this object is subsettable
  3.35373333333333mm extension on the right 

 name   annotation_type color_mapping height
    a continuous vector        random    5mm

&gt; sapply(1, function(x) rowAnnotation(a = 1:10))
[[1]]
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_51 
  position: row 
  items: 10 
  width: 5mm 
  height: 1npc 
  this object is subsettable
  3.35373333333333mm extension on the bottom 

 name   annotation_type color_mapping width
    a continuous vector        random   5mm

&gt; sapply(1, function(x) columnAnnotation(a = 1:10))
[[1]]
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_52 
  position: column 
  items: 10 
  width: 1npc 
  height: 5mm 
  this object is subsettable
  3.35373333333333mm extension on the right 

 name   annotation_type color_mapping height
    a continuous vector        random    5mm

&gt; 
&gt; mapply(function(x, y) HeatmapAnnotation(a = 1:10), list(1), list(1))
[[1]]
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_53 
  position: column 
  items: 10 
  width: 1npc 
  height: 5mm 
  this object is subsettable
  3.35373333333333mm extension on the right 

 name   annotation_type color_mapping height
    a continuous vector        random    5mm

&gt; mapply(function(x, y) rowAnnotation(a = 1:10), list(1), list(1))
[[1]]
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_54 
  position: row 
  items: 10 
  width: 5mm 
  height: 1npc 
  this object is subsettable
  3.35373333333333mm extension on the bottom 

 name   annotation_type color_mapping width
    a continuous vector        random   5mm

&gt; mapply(function(x, y) columnAnnotation(a = 1:10), list(1), list(1))
[[1]]
A HeatmapAnnotation object with 1 annotation
  name: heatmap_annotation_55 
  position: column 
  items: 10 
  width: 1npc 
  height: 5mm 
  this object is subsettable
  3.35373333333333mm extension on the right 

 name   annotation_type color_mapping height
    a continuous vector        random    5mm

&gt; 
&gt; 
&gt; try({
+ 	HeatmapAnnotation(1:10)
+ 	HeatmapAnnotation(df = data.frame(a = 1:10), a = 1:10)
+ })
Error : The annotation should be specified as name-value pairs or via argument
`df` with a data frame.
&gt; 
&gt; proc.time()
   user  system elapsed 
 16.888   0.273  17.141 
</PRE>
</DIV><P><SPAN class="filename">ComplexHeatmap.Rcheck/tests/test-HeatmapList-class.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(circlize)
========================================
circlize version 0.4.15
CRAN page: https://cran.r-project.org/package=circlize
Github page: https://github.com/jokergoo/circlize
Documentation: https://jokergoo.github.io/circlize_book/book/

If you use it in published research, please cite:
Gu, Z. circlize implements and enhances circular visualization
  in R. Bioinformatics 2014.

This message can be suppressed by:
  suppressPackageStartupMessages(library(circlize))
========================================

&gt; library(ComplexHeatmap)
Loading required package: grid
========================================
ComplexHeatmap version 2.17.0
Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/
Github page: https://github.com/jokergoo/ComplexHeatmap
Documentation: http://jokergoo.github.io/ComplexHeatmap-reference

If you use it in published research, please cite either one:
- Gu, Z. Complex Heatmap Visualization. iMeta 2022.
- Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional 
    genomic data. Bioinformatics 2016.


The new InteractiveComplexHeatmap package can directly export static 
complex heatmaps into an interactive Shiny app with zero effort. Have a try!

This message can be suppressed by:
  suppressPackageStartupMessages(library(ComplexHeatmap))
========================================

&gt; library(GetoptLong)
&gt; 
&gt; set.seed(123)
&gt; nr1 = 10; nr2 = 8; nr3 = 6
&gt; nc1 = 6; nc2 = 8; nc3 = 10
&gt; mat1 = cbind(rbind(matrix(rnorm(nr1*nc1, mean = 1,   sd = 0.5), nr = nr1),
+           matrix(rnorm(nr2*nc1, mean = 0,   sd = 0.5), nr = nr2),
+           matrix(rnorm(nr3*nc1, mean = 0,   sd = 0.5), nr = nr3)),
+     rbind(matrix(rnorm(nr1*nc2, mean = 0,   sd = 0.5), nr = nr1),
+           matrix(rnorm(nr2*nc2, mean = 1,   sd = 0.5), nr = nr2),
+           matrix(rnorm(nr3*nc2, mean = 0,   sd = 0.5), nr = nr3)),
+     rbind(matrix(rnorm(nr1*nc3, mean = 0.5, sd = 0.5), nr = nr1),
+           matrix(rnorm(nr2*nc3, mean = 0.5, sd = 0.5), nr = nr2),
+           matrix(rnorm(nr3*nc3, mean = 1,   sd = 0.5), nr = nr3))
+    )
&gt; 
&gt; rownames(mat1) = paste0(&quot;row_1_&quot;, seq_len(nrow(mat1)))
&gt; colnames(mat1) = paste0(&quot;column_1_&quot;, seq_len(nrow(mat1)))
&gt; 
&gt; nr3 = 10; nr1 = 8; nr2 = 6
&gt; nc3 = 6; nc1 = 8; nc2 = 10
&gt; mat2 = cbind(rbind(matrix(rnorm(nr1*nc1, mean = 1,   sd = 0.5), nr = nr1),
+           matrix(rnorm(nr2*nc1, mean = 0,   sd = 0.5), nr = nr2),
+           matrix(rnorm(nr3*nc1, mean = 0,   sd = 0.5), nr = nr3)),
+     rbind(matrix(rnorm(nr1*nc2, mean = 0,   sd = 0.5), nr = nr1),
+           matrix(rnorm(nr2*nc2, mean = 1,   sd = 0.5), nr = nr2),
+           matrix(rnorm(nr3*nc2, mean = 0,   sd = 0.5), nr = nr3)),
+     rbind(matrix(rnorm(nr1*nc3, mean = 0.5, sd = 0.5), nr = nr1),
+           matrix(rnorm(nr2*nc3, mean = 0.5, sd = 0.5), nr = nr2),
+           matrix(rnorm(nr3*nc3, mean = 1,   sd = 0.5), nr = nr3))
+    )
&gt; 
&gt; rownames(mat2) = paste0(&quot;row_2_&quot;, seq_len(nrow(mat2)))
&gt; colnames(mat2) = paste0(&quot;column_2_&quot;, seq_len(nrow(mat2)))
&gt; 
&gt; 
&gt; ht_list = Heatmap(mat1) + Heatmap(mat2)
&gt; draw(ht_list)
&gt; 
&gt; ######### legend ############
&gt; draw(ht_list, heatmap_legend_side = &quot;bottom&quot;)
&gt; draw(ht_list, heatmap_legend_side = &quot;left&quot;)
&gt; draw(ht_list, heatmap_legend_side = &quot;top&quot;)
&gt; 
&gt; 
&gt; ########## width #############
&gt; ht_list = Heatmap(mat1, width = unit(6, &quot;cm&quot;)) + Heatmap(mat2)
&gt; draw(ht_list)
&gt; ht_list = Heatmap(mat1) + Heatmap(mat2, width = unit(8, &quot;cm&quot;))
&gt; draw(ht_list)
&gt; ht_list = Heatmap(mat1, width = unit(12, &quot;cm&quot;)) + Heatmap(mat2, width = unit(8, &quot;cm&quot;))
&gt; draw(ht_list)
&gt; 
&gt; ht_list = Heatmap(mat1, width = unit(6, &quot;cm&quot;)) + Heatmap(mat2)
&gt; draw(ht_list)
&gt; ht_list = Heatmap(mat1) + Heatmap(mat2, width = unit(6, &quot;cm&quot;))
&gt; draw(ht_list)
&gt; ht_list = Heatmap(mat1, width = unit(6, &quot;cm&quot;)) + Heatmap(mat2, width = unit(6, &quot;cm&quot;))
&gt; draw(ht_list)
&gt; ht_list = Heatmap(mat1, width = 4) + Heatmap(mat2)
&gt; draw(ht_list)
&gt; ht_list = Heatmap(mat1, width = 2) + Heatmap(mat2, width = 1)
&gt; draw(ht_list)
&gt; 
&gt; 
&gt; ########### height ###########
&gt; ht_list = Heatmap(mat1, height = unit(6, &quot;cm&quot;)) + Heatmap(mat2)
&gt; draw(ht_list)
&gt; ht_list = Heatmap(mat1, heatmap_height = unit(6, &quot;cm&quot;)) + Heatmap(mat2)
&gt; draw(ht_list)
&gt; ht_list = Heatmap(mat1, width = unit(6, &quot;cm&quot;), height = unit(6, &quot;cm&quot;)) + 
+ 	Heatmap(mat2, width = unit(6, &quot;cm&quot;), height = unit(6, &quot;cm&quot;))
&gt; draw(ht_list, column_title = &quot;foooooooooo&quot;, row_title = &quot;baaaaaaaaaaar&quot;)
&gt; 
&gt; ##### split #####
&gt; ht_list = Heatmap(mat1, name = &quot;m1&quot;, row_km = 2) + Heatmap(mat2, name = &quot;m2&quot;, row_km = 3)
&gt; draw(ht_list, main_heatmap = &quot;m1&quot;)
&gt; draw(ht_list, main_heatmap = &quot;m2&quot;)
&gt; 
&gt; ht_list = Heatmap(mat1, name = &quot;m1&quot;, row_km = 2, column_km = 3, width = unit(8, &quot;cm&quot;), height = unit(6, &quot;cm&quot;)) + 
+ 	Heatmap(mat2, name = &quot;m2&quot;, row_km = 3, column_km = 2, width = unit(8, &quot;cm&quot;), height = unit(10, &quot;cm&quot;))
&gt; draw(ht_list, main_heatmap = &quot;m1&quot;, column_title = &quot;foooooooooo&quot;, row_title = &quot;baaaaaaaaaaar&quot;)
&gt; draw(ht_list, main_heatmap = &quot;m2&quot;, column_title = &quot;foooooooooo&quot;, row_title = &quot;baaaaaaaaaaar&quot;)
&gt; 
&gt; ##### adjust column annotations #####
&gt; ha1 = HeatmapAnnotation(foo = 1:24, bar = anno_points(24:1, height = unit(4, &quot;cm&quot;)))
&gt; ha2 = HeatmapAnnotation(bar = anno_points(24:1), foo = 1:24)
&gt; ht_list = Heatmap(mat1, top_annotation = ha1) + Heatmap(mat2, top_annotation = ha2)
&gt; draw(ht_list)
&gt; ha2 = HeatmapAnnotation(foo = 1:24)
&gt; ht_list = Heatmap(mat1, top_annotation = ha1) + Heatmap(mat2, top_annotation = ha2)
&gt; draw(ht_list)
&gt; ht_list = Heatmap(mat1, top_annotation = ha1) + Heatmap(mat2)
&gt; draw(ht_list)
&gt; ht_list = Heatmap(mat1, bottom_annotation = ha1) + Heatmap(mat2)
&gt; draw(ht_list)
&gt; 
&gt; 
&gt; #### row annotations #####
&gt; ha = rowAnnotation(foo = 1:24, bar = anno_points(24:1), width = unit(6, &quot;cm&quot;))
&gt; ht_list = Heatmap(mat1) + ha
&gt; draw(ht_list)
&gt; ht_list = Heatmap(mat1, width = unit(6, &quot;cm&quot;)) + ha
&gt; draw(ht_list)
&gt; ht_list = Heatmap(mat1, width = unit(6, &quot;cm&quot;), row_km = 2) + ha
&gt; draw(ht_list)
&gt; 
&gt; ht_list = Heatmap(matrix(rnorm(100), 10), name = &quot;rnorm&quot;) +
+   rowAnnotation(foo = 1:10, bar = anno_points(10:1)) + 
+   Heatmap(matrix(runif(100), 10), name = &quot;runif&quot;)
&gt; summary(ht_list[1:5, ])
A horizontal heamtap list with 3 heatmap/annotations.
  rnorm: a matrix with 5 rows and 10 columns
  heatmap_annotation_4: a list of 2 annotations
    foo:   a simple annotation.
    bar:   a complex annotation.
  runif: a matrix with 5 rows and 10 columns
&gt; summary(ht_list[1:5, 1])
A horizontal heamtap list with 1 heatmap/annotations.
  rnorm: a matrix with 5 rows and 10 columns
&gt; summary(ht_list[1:5, &quot;rnorm&quot;])
A horizontal heamtap list with 1 heatmap/annotations.
  rnorm: a matrix with 5 rows and 10 columns
&gt; summary(ht_list[1:5, c(&quot;rnorm&quot;, &quot;foo&quot;)])
A horizontal heamtap list with 2 heatmap/annotations.
  rnorm: a matrix with 5 rows and 10 columns
  heatmap_annotation_4: a list of 1 annotations
    foo:   a simple annotation.
&gt; 
&gt; ht_list = Heatmap(matrix(rnorm(100), 10), name = &quot;rnorm&quot;) %v%
+   columnAnnotation(foo = 1:10, bar = anno_points(10:1)) %v%
+   Heatmap(matrix(runif(100), 10), name = &quot;runif&quot;)
&gt; summary(ht_list[, 1:5])
A vertical heamtap list with 3 heatmap/annotations.
  rnorm: a matrix with 10 rows and 5 columns
  heatmap_annotation_5: a list of 2 annotations
    foo:   a simple annotation.
    bar:   a complex annotation.
  runif: a matrix with 10 rows and 5 columns
&gt; summary(ht_list[1, 1:5])
A vertical heamtap list with 1 heatmap/annotations.
  rnorm: a matrix with 10 rows and 5 columns
&gt; summary(ht_list[&quot;rnorm&quot;, 1:5])
A vertical heamtap list with 1 heatmap/annotations.
  rnorm: a matrix with 10 rows and 5 columns
&gt; summary(ht_list[c(&quot;rnorm&quot;, &quot;foo&quot;), 1:5])
A vertical heamtap list with 2 heatmap/annotations.
  rnorm: a matrix with 10 rows and 5 columns
  heatmap_annotation_5: a list of 1 annotations
    foo:   a simple annotation.
&gt; 
&gt; 
&gt; 
&gt; 
&gt; proc.time()
   user  system elapsed 
 20.887   0.303  21.171 
</PRE>
</DIV><P><SPAN class="filename">ComplexHeatmap.Rcheck/tests/test-interactive.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; 
&gt; if(0) {
+ 
+ m = matrix(rnorm(100), 10)
+ rownames(m) = 1:10
+ colnames(m) = 1:10
+ 
+ ht = Heatmap(m)
+ ht = draw(ht)
+ selectArea(ht)
+ 
+ 
+ 
+ ht = Heatmap(m, row_km = 2, column_km = 2)
+ ht = draw(ht)
+ selectArea(ht)
+ 
+ 
+ ht = Heatmap(m, row_km = 2, column_km = 2) + Heatmap(m, row_km = 2, column_km = 2)
+ ht = draw(ht)
+ selectArea(ht)
+ 
+ pdf(&quot;~/test.pdf&quot;)
+ ht = Heatmap(m)
+ ht = draw(ht)
+ selectArea(ht, pos1 = unit(c(1, 1), &quot;cm&quot;), pos2 = unit(c(4, 4), &quot;cm&quot;), verbose = TRUE)
+ 
+ set.seed(123)
+ ht = Heatmap(m, row_km = 2, column_km = 2)
+ ht = draw(ht)
+ selectArea(ht, pos1 = unit(c(1, 1), &quot;cm&quot;), pos2 = unit(c(8, 8), &quot;cm&quot;), verbose = TRUE)
+ dev.off()
+ 
+ png(&quot;~/test-1.png&quot;)
+ ht = Heatmap(m)
+ ht = draw(ht)
+ selectArea(ht, pos1 = unit(c(1, 1), &quot;cm&quot;), pos2 = unit(c(4, 4), &quot;cm&quot;), verbose = TRUE)
+ dev.off()
+ 
+ png(&quot;~/test-2.png&quot;)
+ set.seed(123)
+ ht = Heatmap(m, row_km = 2, column_km = 2)
+ ht = draw(ht)
+ selectArea(ht, pos1 = unit(c(1, 1), &quot;cm&quot;), pos2 = unit(c(8, 8), &quot;cm&quot;), verbose = TRUE)
+ dev.off()
+ 
+ }
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.194   0.038   0.213 
</PRE>
</DIV><P><SPAN class="filename">ComplexHeatmap.Rcheck/tests/test-Legend.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(circlize)
========================================
circlize version 0.4.15
CRAN page: https://cran.r-project.org/package=circlize
Github page: https://github.com/jokergoo/circlize
Documentation: https://jokergoo.github.io/circlize_book/book/

If you use it in published research, please cite:
Gu, Z. circlize implements and enhances circular visualization
  in R. Bioinformatics 2014.

This message can be suppressed by:
  suppressPackageStartupMessages(library(circlize))
========================================

&gt; library(ComplexHeatmap)
Loading required package: grid
========================================
ComplexHeatmap version 2.17.0
Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/
Github page: https://github.com/jokergoo/ComplexHeatmap
Documentation: http://jokergoo.github.io/ComplexHeatmap-reference

If you use it in published research, please cite either one:
- Gu, Z. Complex Heatmap Visualization. iMeta 2022.
- Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional 
    genomic data. Bioinformatics 2016.


The new InteractiveComplexHeatmap package can directly export static 
complex heatmaps into an interactive Shiny app with zero effort. Have a try!

This message can be suppressed by:
  suppressPackageStartupMessages(library(ComplexHeatmap))
========================================

&gt; library(GetoptLong)
&gt; 
&gt; if(!exists(&quot;random_str&quot;)) {
+ 	random_str = ComplexHeatmap:::random_str
+ }
&gt; 
&gt; lgd = Legend(at = 1:6, legend_gp = gpar(fill = 1:6))
&gt; draw(lgd, test = &quot;default discrete legends style&quot;)
&gt; 
&gt; lgd = Legend(labels = 1:6, legend_gp = gpar(fill = 1:6))
&gt; draw(lgd, test = &quot;only specify labels with no at&quot;)
&gt; 
&gt; 
&gt; lgd = Legend(labels = month.name[1:6], title = &quot;foo&quot;, legend_gp = gpar(fill = 1:6))
&gt; draw(lgd, test = &quot;add labels and title&quot;)
&gt; 
&gt; lgd = Legend(labels = month.name[1:6], title = &quot;foo&quot;, legend_gp = gpar(fill = 1:6),
+ 	title_position = &quot;lefttop&quot;)
&gt; draw(lgd, test = &quot;title put in the lefttop&quot;)
&gt; 
&gt; lgd = Legend(labels = month.name[1:6], title = &quot;foo&quot;, legend_gp = gpar(fill = 1:6),
+ 	title_position = &quot;lefttop-rot&quot;)
&gt; draw(lgd, test = &quot;title put in the lefttop-rot&quot;)
&gt; 
&gt; lgd = Legend(labels = month.name[1:6], title = &quot;foo&quot;, legend_gp = gpar(fill = 1:6),
+ 	title_position = &quot;leftcenter-rot&quot;)
&gt; draw(lgd, test = &quot;title put in the leftcenter-rot&quot;)
&gt; 
&gt; lgd = Legend(labels = 1:6, title = &quot;fooooooo&quot;, legend_gp = gpar(fill = 1:6))
&gt; draw(lgd, test = &quot;title is longer than the legend body&quot;)
&gt; 
&gt; lgd = Legend(at = 1:6, legend_gp = gpar(fill = 1:6), grid_height = unit(1, &quot;cm&quot;), 
+ 	title = &quot;foo&quot;, grid_width = unit(5, &quot;mm&quot;))
&gt; draw(lgd, test = &quot;grid size&quot;)
&gt; 
&gt; lgd = Legend(labels = month.name[1:6], legend_gp = gpar(fill = 1:6), title = &quot;foo&quot;, 
+ 	labels_gp = gpar(col = &quot;red&quot;, fontsize = 14))
&gt; draw(lgd, test = &quot;labels_gp&quot;)
&gt; 
&gt; lgd = Legend(labels = month.name[1:6], legend_gp = gpar(fill = 1:6), title = &quot;foo&quot;, 
+ 	title_gp = gpar(col = &quot;red&quot;, fontsize = 14))
&gt; draw(lgd, test = &quot;title_gp&quot;)
&gt; 
&gt; lgd = Legend(labels = month.name[1:6], legend_gp = gpar(fill = 1:6), title = &quot;foo&quot;, 
+ 	border = &quot;red&quot;)
&gt; draw(lgd, test = &quot;legend border&quot;)
&gt; 
&gt; lgd = Legend(labels = month.name[1:10], legend_gp = gpar(fill = 1:10), title = &quot;foo&quot;, 
+ 	ncol = 3)
&gt; draw(lgd, test = &quot;in 3 columns&quot;)
&gt; 
&gt; lgd = Legend(labels = month.name[1:10], legend_gp = gpar(fill = 1:10), title = &quot;foo&quot;, 
+ 	ncol = 3, title_position = &quot;topcenter&quot;)
&gt; draw(lgd, test = &quot;in 3 columns, title in the center&quot;)
&gt; 
&gt; lgd = Legend(labels = month.name[1:10], legend_gp = gpar(fill = 1:10), title = &quot;foo&quot;, 
+ 	ncol = 3, by_row = TRUE)
&gt; draw(lgd, test = &quot;in 3 columns and by rows&quot;)
&gt; 
&gt; lgd = Legend(labels = month.name[1:10], legend_gp = gpar(fill = 1:10), title = &quot;foo&quot;, 
+ 	ncol = 3, gap = unit(1, &quot;cm&quot;))
&gt; draw(lgd, test = &quot;in 3 columns with gap between columns&quot;)
&gt; 
&gt; lgd = Legend(labels = month.name[1:10], legend_gp = gpar(fill = 1:10), title = &quot;foo&quot;, 
+ 	nrow = 3)
&gt; draw(lgd, test = &quot;in 3 rows&quot;)
&gt; 
&gt; lgd = Legend(labels = month.name[1:6], legend_gp = gpar(fill = 1:6), title = &quot;foooooo&quot;, 
+ 	nrow = 1, title_position = &quot;lefttop&quot;)
&gt; draw(lgd, test = &quot;1 row and title is on the left&quot;)
&gt; 
&gt; lgd = Legend(labels = month.name[1:6], legend_gp = gpar(fill = 1:6), title = &quot;foooooo&quot;, 
+ 	nrow = 1, title_position = &quot;lefttop-rot&quot;)
&gt; draw(lgd, test = &quot;1 row and title is on the left, 90 rotation&quot;)
&gt; 
&gt; lgd = Legend(labels = month.name[1:6], legend_gp = gpar(fill = 1:6), title = &quot;foooooo&quot;, 
+ 	nrow = 1, title_position = &quot;leftcenter&quot;)
&gt; draw(lgd, test = &quot;1 row and title is on the left, 90 rotation&quot;)
&gt; 
&gt; lgd = Legend(labels = month.name[1:6], title = &quot;foo&quot;, type = &quot;points&quot;, pch = 1:6, 
+ 	legend_gp = gpar(col = 1:6), background = &quot;red&quot;)
&gt; draw(lgd, test = &quot;points as legends&quot;)
&gt; 
&gt; lgd = Legend(labels = month.name[1:6], title = &quot;foo&quot;, type = &quot;points&quot;, pch = letters[1:6], 
+ 	legend_gp = gpar(col = 1:6), background = &quot;white&quot;)
&gt; draw(lgd, test = &quot;letters as legends&quot;)
&gt; 
&gt; lgd = Legend(labels = month.name[1:6], title = &quot;foo&quot;, type = &quot;lines&quot;, 
+ 	legend_gp = gpar(col = 1:6, lty = 1:6))
&gt; draw(lgd, test = &quot;lines as legends&quot;)
&gt; 
&gt; ###### vertical continous legend #######
&gt; col_fun = colorRamp2(c(0, 0.5, 1), c(&quot;blue&quot;, &quot;white&quot;, &quot;red&quot;))
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;)
&gt; draw(lgd, test = &quot;only col_fun&quot;)
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, at = c(0, 0.25, 0.5, 0.75, 1))
&gt; draw(lgd, test = &quot;with at&quot;)
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, at = rev(c(0, 0.25, 0.5, 0.75, 1)))
&gt; draw(lgd, test = &quot;with at&quot;)
&gt; 
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, at = c(0, 0.5, 1), labels = c(&quot;low&quot;, &quot;median&quot;, &quot;high&quot;))
&gt; draw(lgd, test = &quot;with labels&quot;)
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, legend_height = unit(6, &quot;cm&quot;))
&gt; draw(lgd, test = &quot;set legend_height&quot;)
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, labels_gp = gpar(col = &quot;red&quot;))
&gt; draw(lgd, test = &quot;set label color&quot;)
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, border = &quot;red&quot;)
&gt; draw(lgd, test = &quot;legend border&quot;)
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foooooooo&quot;, title_position = &quot;lefttop-rot&quot;)
&gt; draw(lgd, test = &quot;lefttop rot title&quot;)
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foooooooo&quot;, title_position = &quot;leftcenter-rot&quot;)
&gt; draw(lgd, test = &quot;leftcenter top title&quot;)
&gt; 
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, title_position = &quot;lefttop&quot;, direction = &quot;horizontal&quot;)
&gt; draw(lgd, test = &quot;lefttop title&quot;)
&gt; 
&gt; ###### horizontal continous legend #######
&gt; col_fun = colorRamp2(c(0, 0.5, 1), c(&quot;blue&quot;, &quot;white&quot;, &quot;red&quot;))
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, direction = &quot;horizontal&quot;)
&gt; draw(lgd, test = &quot;only col_fun&quot;)
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, at = c(0, 0.25, 0.5, 0.75, 1), direction = &quot;horizontal&quot;)
&gt; draw(lgd, test = &quot;with at&quot;)
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, at = rev(c(0, 0.25, 0.5, 0.75, 1)), direction = &quot;horizontal&quot;)
&gt; draw(lgd, test = &quot;with at&quot;)
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, at = c(0, 0.5, 1), labels = c(&quot;low&quot;, &quot;median&quot;, &quot;high&quot;),
+ 	direction = &quot;horizontal&quot;)
&gt; draw(lgd, test = &quot;with labels&quot;)
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, legend_width = unit(6, &quot;cm&quot;), direction = &quot;horizontal&quot;)
&gt; draw(lgd, test = &quot;set legend_width&quot;)
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, labels_gp = gpar(col = &quot;red&quot;), direction = &quot;horizontal&quot;)
&gt; draw(lgd, test = &quot;set label color&quot;)
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, border = &quot;red&quot;, direction = &quot;horizontal&quot;)
&gt; draw(lgd, test = &quot;legend border&quot;)
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foooooooo&quot;, direction = &quot;horizontal&quot;, 
+ 	title_position = &quot;topcenter&quot;)
&gt; draw(lgd, test = &quot;topcenter title&quot;)
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foooooooo&quot;, direction = &quot;horizontal&quot;, 
+ 	title_position = &quot;lefttop&quot;)
&gt; draw(lgd, test = &quot;lefttop title&quot;)
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foooooooo&quot;, direction = &quot;horizontal&quot;, 
+ 	title_position = &quot;leftcenter&quot;)
&gt; draw(lgd, test = &quot;leftcenter title&quot;)
&gt; 
&gt; 
&gt; ###### pack legend
&gt; lgd1 = Legend(at = 1:6, legend_gp = gpar(fill = 1:6), title = &quot;legend1&quot;)
&gt; lgd2 = Legend(col_fun = col_fun, title = &quot;legend2&quot;, at = c(0, 0.25, 0.5, 0.75, 1))
&gt; 
&gt; pd = packLegend(lgd1, lgd2)
&gt; draw(pd, test = &quot;two legends&quot;)
&gt; 
&gt; pd = packLegend(list = list(lgd1, lgd2))
&gt; draw(pd, test = &quot;two legends specified as a list&quot;)
&gt; 
&gt; pd = packLegend(lgd1, lgd2, direction = &quot;horizontal&quot;)
&gt; draw(pd, test = &quot;two legends packed horizontally&quot;)
&gt; 
&gt; lgd3 = Legend(at = 1:6, legend_gp = gpar(fill = 1:6), title = &quot;legend1&quot;)
&gt; lgd4 = Legend(col_fun = col_fun, title = &quot;legend2&quot;, at = c(0, 0.25, 0.5, 0.75, 1), direction = &quot;horizontal&quot;)
&gt; pd = packLegend(lgd3, lgd4)
&gt; draw(pd, test = &quot;two legends with different directions&quot;)
&gt; pd = packLegend(lgd3, lgd4, direction = &quot;horizontal&quot;)
&gt; draw(pd, test = &quot;two legends with different directions&quot;)
&gt; 
&gt; pd = packLegend(lgd1, lgd2, lgd1, lgd2)
&gt; draw(pd, test = &quot;many legends with same legends&quot;)
&gt; 
&gt; lgd3 = Legend(at = 1:6, legend_gp = gpar(fill = 1:6), title = &quot;legend1&quot;)
&gt; lgd4 = Legend(col_fun = col_fun, title = &quot;legend2&quot;, at = c(0, 0.25, 0.5, 0.75, 1))
&gt; pd = packLegend(lgd1, lgd2, lgd3, lgd4)
&gt; draw(pd, test = &quot;many legends with all different legends&quot;)
&gt; 
&gt; pd = packLegend(lgd1, lgd2, lgd1, lgd2, lgd1, lgd2)
&gt; draw(pd, test = &quot;many legends&quot;)
&gt; 
&gt; pd = packLegend(lgd1, lgd2, lgd1, lgd2, lgd1, lgd2, max_height = unit(1, &quot;npc&quot;))
&gt; draw(pd, test = &quot;many legends, max_height = unit(1, &#x27;npc&#x27;)&quot;)
&gt; ## reduce the height of the interactive window and rerun draw()
&gt; 
&gt; pd = packLegend(lgd1, lgd2, lgd1, lgd2, lgd1, lgd2, max_height = unit(10, &quot;cm&quot;))
&gt; draw(pd, test = &quot;many legends, max_height = unit(10, &#x27;cm&#x27;)&quot;)
&gt; 
&gt; pd = packLegend(lgd1, lgd2, lgd1, lgd2, lgd1, lgd2, max_height = unit(10, &quot;cm&quot;), gap = unit(1, &quot;cm&quot;))
&gt; draw(pd, test = &quot;many legends, max_height = unit(10, &#x27;cm&#x27;), with gap&quot;)
&gt; 
&gt; lgd_long = Legend(at = 1:50, legend_gp = gpar(fill = 1:50))
&gt; pd = packLegend(lgd1, lgd2, lgd1, lgd2, lgd1, lgd2, lgd_long, max_height = unit(10, &quot;cm&quot;))
&gt; draw(pd, test = &quot;many legends with a long one, max_height = unit(10, &#x27;cm&#x27;)&quot;)
&gt; 
&gt; lgd1 = Legend(at = 1:6, legend_gp = gpar(fill = 1:6), title = &quot;legend1&quot;,
+ 	nr = 1)
&gt; lgd2 = Legend(col_fun = col_fun, title = &quot;legend2&quot;, at = c(0, 0.25, 0.5, 0.75, 1),
+ 	direction = &quot;horizontal&quot;)
&gt; pd = packLegend(lgd1, lgd2, lgd1, lgd2, lgd1, lgd2, direction = &quot;horizontal&quot;)
&gt; draw(pd, test = &quot;many legends&quot;)
&gt; 
&gt; pd = packLegend(lgd1, lgd2, lgd1, lgd2, lgd1, lgd2, max_width = unit(1, &quot;npc&quot;), direction = &quot;horizontal&quot;)
&gt; draw(pd, test = &quot;many legends, max_width = unit(1, &#x27;npc&#x27;)&quot;)
&gt; ## reduce the height of the interactive window and rerun draw()
&gt; 
&gt; pd = packLegend(lgd1, lgd2, lgd1, lgd2, lgd1, lgd2, max_width = unit(10, &quot;cm&quot;), direction = &quot;horizontal&quot;)
&gt; draw(pd, test = &quot;many legends, max_width = unit(10, &#x27;cm&#x27;)&quot;)
&gt; 
&gt; 
&gt; ####### unequal interval breaks
&gt; col_fun = colorRamp2(c(0, 0.5, 1), c(&quot;blue&quot;, &quot;white&quot;, &quot;red&quot;))
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, at = c(0, 0.1, 0.15, 0.5, 0.9, 0.95, 1))
&gt; draw(lgd, test = &quot;unequal interval breaks&quot;)
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, at = c(0, 0.3, 1), legend_height = unit(4, &quot;cm&quot;))
&gt; draw(lgd, test = &quot;unequal interval breaks but not label position adjustment&quot;)
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, at = c(0, 0.1, 0.15, 0.5, 0.9, 0.95, 1),
+ 	direction = &quot;horizontal&quot;)
&gt; draw(lgd, test = &quot;unequal interval breaks&quot;)
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, at = c(0, 0.1, 0.15, 0.5, 0.9, 0.95, 1),
+ 	direction = &quot;horizontal&quot;, title_position = &quot;lefttop&quot;)
&gt; draw(lgd, test = &quot;unequal interval breaks&quot;)
&gt; 
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, at = c(0, 0.1, 0.15, 0.5, 0.9, 0.95, 1),
+ 	direction = &quot;horizontal&quot;, title_position = &quot;lefttop&quot;, labels_rot = 90)
&gt; draw(lgd, test = &quot;unequal interval breaks, label rot 90&quot;)
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, at = c(0, 0.1, 0.5, 0.75, 1),
+ 	labels = c(&quot;mininal&quot;, &quot;q10&quot;, &quot;median&quot;, &quot;q75&quot;, &quot;maximal&quot;),
+ 	direction = &quot;horizontal&quot;, title_position = &quot;lefttop&quot;)
&gt; draw(lgd, test = &quot;unequal interval breaks with labels&quot;)
&gt; 
&gt; 
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, at = c(0, 0.1, 0.5, 0.75, 1),
+ 	labels = c(&quot;mininal&quot;, &quot;q10&quot;, &quot;median&quot;, &quot;q75&quot;, &quot;maximal&quot;),
+ 	direction = &quot;horizontal&quot;)
&gt; draw(lgd, test = &quot;unequal interval breaks with labels&quot;)
&gt; 
&gt; 
&gt; col_fun = colorRamp2(c(0, 0.05, 0.1, 0.5, 1), c(&quot;green&quot;, &quot;white&quot;, &quot;red&quot;, &quot;black&quot;, &quot;blue&quot;))
&gt; lgd = Legend(col_fun = col_fun, title = &quot;foo&quot;, break_dist = 1:4)
&gt; draw(lgd, test = &quot;unequal interval breaks&quot;)
&gt; 
&gt; 
&gt; #### position of legends to heatmaps ##
&gt; if(0) {
+ m = matrix(rnorm(100), 10)
+ rownames(m) = random_str(10, len = 20)
+ colnames(m) = random_str(10, len = 20)
+ Heatmap(m)
+ }
&gt; 
&gt; 
&gt; 
&gt; proc.time()
   user  system elapsed 
  4.862   0.200   5.043 
</PRE>
</DIV><P><SPAN class="filename">ComplexHeatmap.Rcheck/tests/test-multiple-page.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(circlize)
========================================
circlize version 0.4.15
CRAN page: https://cran.r-project.org/package=circlize
Github page: https://github.com/jokergoo/circlize
Documentation: https://jokergoo.github.io/circlize_book/book/

If you use it in published research, please cite:
Gu, Z. circlize implements and enhances circular visualization
  in R. Bioinformatics 2014.

This message can be suppressed by:
  suppressPackageStartupMessages(library(circlize))
========================================

&gt; library(ComplexHeatmap)
Loading required package: grid
========================================
ComplexHeatmap version 2.17.0
Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/
Github page: https://github.com/jokergoo/ComplexHeatmap
Documentation: http://jokergoo.github.io/ComplexHeatmap-reference

If you use it in published research, please cite either one:
- Gu, Z. Complex Heatmap Visualization. iMeta 2022.
- Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional 
    genomic data. Bioinformatics 2016.


The new InteractiveComplexHeatmap package can directly export static 
complex heatmaps into an interactive Shiny app with zero effort. Have a try!

This message can be suppressed by:
  suppressPackageStartupMessages(library(ComplexHeatmap))
========================================

&gt; library(GetoptLong)
&gt; 
&gt; m = matrix(rnorm(100), 10)
&gt; 
&gt; postscript(&quot;test.ps&quot;)
&gt; lgd = Legend(labels = c(&quot;a&quot;, &quot;b&quot;, &quot;c&quot;))
&gt; draw(Heatmap(m), heatmap_legend_list = list(lgd))
&gt; dev.off()
null device 
          1 
&gt; 
&gt; check_pages = function() {
+ 	lines = readLines(&quot;test.ps&quot;)
+ 	print(lines[length(lines)-1])
+ 	invisible(file.remove(&quot;test.ps&quot;))
+ }
&gt; 
&gt; check_pages()
[1] &quot;%%Pages: 1&quot;
&gt; 
&gt; postscript(&quot;test.ps&quot;)
&gt; ha = HeatmapAnnotation(foo = 1:10, bar = anno_points(1:10))
&gt; Heatmap(m, top_annotation = ha)
&gt; dev.off()
null device 
          1 
&gt; 
&gt; check_pages()
[1] &quot;%%Pages: 1&quot;
&gt; 
&gt; proc.time()
   user  system elapsed 
  8.576   0.136   8.694 
</PRE>
</DIV><P><SPAN class="filename">ComplexHeatmap.Rcheck/tests/test-oncoPrint.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(circlize)
========================================
circlize version 0.4.15
CRAN page: https://cran.r-project.org/package=circlize
Github page: https://github.com/jokergoo/circlize
Documentation: https://jokergoo.github.io/circlize_book/book/

If you use it in published research, please cite:
Gu, Z. circlize implements and enhances circular visualization
  in R. Bioinformatics 2014.

This message can be suppressed by:
  suppressPackageStartupMessages(library(circlize))
========================================

&gt; library(ComplexHeatmap)
Loading required package: grid
========================================
ComplexHeatmap version 2.17.0
Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/
Github page: https://github.com/jokergoo/ComplexHeatmap
Documentation: http://jokergoo.github.io/ComplexHeatmap-reference

If you use it in published research, please cite either one:
- Gu, Z. Complex Heatmap Visualization. iMeta 2022.
- Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional 
    genomic data. Bioinformatics 2016.


The new InteractiveComplexHeatmap package can directly export static 
complex heatmaps into an interactive Shiny app with zero effort. Have a try!

This message can be suppressed by:
  suppressPackageStartupMessages(library(ComplexHeatmap))
========================================

&gt; library(GetoptLong)
&gt; 
&gt; mat = read.table(textConnection(
+ &quot;s1,s2,s3
+ g1,snv;indel,snv,indel
+ g2,,snv;indel,snv
+ g3,snv,,indel;snv&quot;), row.names = 1, header = TRUE, sep = &quot;,&quot;, stringsAsFactors = FALSE)
&gt; mat = as.matrix(mat)
&gt; 
&gt; get_type_fun = function(x) strsplit(x, &quot;;&quot;)[[1]]
&gt; 
&gt; alter_fun = list(
+     snv = function(x, y, w, h) grid.rect(x, y, w*0.9, h*0.9, 
+         gp = gpar(fill = col[&quot;snv&quot;], col = NA)),
+     indel = function(x, y, w, h) grid.rect(x, y, w*0.9, h*0.4, 
+         gp = gpar(fill = col[&quot;indel&quot;], col = NA))
+ )
&gt; 
&gt; col = c(snv = &quot;red&quot;, indel = &quot;blue&quot;)
&gt; ht = oncoPrint(mat, get_type = get_type_fun,
+     alter_fun = alter_fun, col = col)
All mutation types: snv, indel.
`alter_fun` is assumed vectorizable. If it does not generate correct
plot, please set `alter_fun_is_vectorized = FALSE` in `oncoPrint()`.
&gt; draw(ht)
&gt; 
&gt; ## turn off row names while turn on column names
&gt; ht = oncoPrint(mat, get_type = get_type_fun,
+     alter_fun = alter_fun, col = col, 
+     show_column_names = TRUE, show_row_names = FALSE, show_pct = FALSE)
All mutation types: snv, indel.
`alter_fun` is assumed vectorizable. If it does not generate correct
plot, please set `alter_fun_is_vectorized = FALSE` in `oncoPrint()`.
&gt; draw(ht)
&gt; 
&gt; ht = oncoPrint(mat, get_type = get_type_fun,
+     alter_fun = alter_fun, col = col, pct_side = &quot;right&quot;, 
+     row_names_side = &quot;left&quot;)
All mutation types: snv, indel.
`alter_fun` is assumed vectorizable. If it does not generate correct
plot, please set `alter_fun_is_vectorized = FALSE` in `oncoPrint()`.
&gt; draw(ht)
&gt; 
&gt; ht = oncoPrint(mat, get_type = get_type_fun,
+     alter_fun = alter_fun, col = col,
+     top_annotation = HeatmapAnnotation(column_barplot = anno_oncoprint_barplot())
+ )
All mutation types: snv, indel.
`alter_fun` is assumed vectorizable. If it does not generate correct
plot, please set `alter_fun_is_vectorized = FALSE` in `oncoPrint()`.
&gt; draw(ht)
&gt; 
&gt; ht = oncoPrint(mat, get_type = get_type_fun,
+     alter_fun = alter_fun, col = col,
+     top_annotation = HeatmapAnnotation(
+     	column_barplot = anno_oncoprint_barplot(),
+     	foo = 1:3,
+     	annotation_name_side = &quot;left&quot;)
+ )
All mutation types: snv, indel.
`alter_fun` is assumed vectorizable. If it does not generate correct
plot, please set `alter_fun_is_vectorized = FALSE` in `oncoPrint()`.
&gt; draw(ht)
&gt; 
&gt; ht = oncoPrint(mat, get_type = get_type_fun,
+     alter_fun = alter_fun, col = col,
+     top_annotation = HeatmapAnnotation(
+     	cbar = anno_oncoprint_barplot(),
+     	foo1 = 1:3,
+     	annotation_name_side = &quot;left&quot;),
+     left_annotation = rowAnnotation(foo2 = 1:3),
+     right_annotation = rowAnnotation(cbar = anno_oncoprint_barplot(), foo3 = 1:3),
+ )
All mutation types: snv, indel.
`alter_fun` is assumed vectorizable. If it does not generate correct
plot, please set `alter_fun_is_vectorized = FALSE` in `oncoPrint()`.
&gt; draw(ht)
&gt; 
&gt; 
&gt; ht = oncoPrint(mat, get_type = get_type_fun,
+     alter_fun = alter_fun, col = col,
+     top_annotation = HeatmapAnnotation(
+         cbar = anno_oncoprint_barplot(border = TRUE),
+         foo1 = 1:3,
+         annotation_name_side = &quot;left&quot;),
+     left_annotation = rowAnnotation(foo2 = 1:3),
+     right_annotation = rowAnnotation(
+         cbar = anno_oncoprint_barplot(border = TRUE), 
+         foo3 = 1:3),
+ )
All mutation types: snv, indel.
`alter_fun` is assumed vectorizable. If it does not generate correct
plot, please set `alter_fun_is_vectorized = FALSE` in `oncoPrint()`.
&gt; draw(ht)
&gt; 
&gt; ht = oncoPrint(mat, get_type = get_type_fun,
+     alter_fun = alter_fun, col = col,
+     right_annotation = rowAnnotation(rbar = anno_oncoprint_barplot(axis_param = list(side = &quot;bottom&quot;, labels_rot = 90)))
+ )
All mutation types: snv, indel.
`alter_fun` is assumed vectorizable. If it does not generate correct
plot, please set `alter_fun_is_vectorized = FALSE` in `oncoPrint()`.
&gt; draw(ht)
&gt; 
&gt; 
&gt; proc.time()
   user  system elapsed 
 10.659   0.199  10.838 
</PRE>
</DIV><P><SPAN class="filename">ComplexHeatmap.Rcheck/tests/test-pheatmap.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(ComplexHeatmap)
Loading required package: grid
========================================
ComplexHeatmap version 2.17.0
Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/
Github page: https://github.com/jokergoo/ComplexHeatmap
Documentation: http://jokergoo.github.io/ComplexHeatmap-reference

If you use it in published research, please cite either one:
- Gu, Z. Complex Heatmap Visualization. iMeta 2022.
- Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional 
    genomic data. Bioinformatics 2016.


The new InteractiveComplexHeatmap package can directly export static 
complex heatmaps into an interactive Shiny app with zero effort. Have a try!

This message can be suppressed by:
  suppressPackageStartupMessages(library(ComplexHeatmap))
========================================

&gt; 
&gt; if(requireNamespace(&quot;pheatmap&quot;)) {
+ 	mat = matrix(rnorm(100), 10)
+ 
+ 	compare_pheatmap(mat)
+ 
+ 	pheatmap(mat)
+ 	pheatmap(mat, col = rev(RColorBrewer::brewer.pal(n = 7, name = &quot;RdYlBu&quot;)))
+ 
+ 	test = matrix(rnorm(200), 20, 10)
+ 	test[1:10, seq(1, 10, 2)] = test[1:10, seq(1, 10, 2)] + 3
+ 	test[11:20, seq(2, 10, 2)] = test[11:20, seq(2, 10, 2)] + 2
+ 	test[15:20, seq(2, 10, 2)] = test[15:20, seq(2, 10, 2)] + 4
+ 	colnames(test) = paste(&quot;Test&quot;, 1:10, sep = &quot;&quot;)
+ 	rownames(test) = paste(&quot;Gene&quot;, 1:20, sep = &quot;&quot;)
+ 
+ 	# Draw heatmaps
+ 	compare_pheatmap(test)
+ 	compare_pheatmap(test, kmeans_k = 2)
+ 	compare_pheatmap(test, scale = &quot;row&quot;, clustering_distance_rows = &quot;correlation&quot;)
+ 	compare_pheatmap(test, color = colorRampPalette(c(&quot;navy&quot;, &quot;white&quot;, &quot;firebrick3&quot;))(50))
+ 	compare_pheatmap(test, cluster_row = FALSE)
+ 	compare_pheatmap(test, legend = FALSE)
+ 
+ 	# Show text within cells
+ 	compare_pheatmap(test, display_numbers = TRUE)
+ 	compare_pheatmap(test, display_numbers = TRUE, number_format = &quot;%.1e&quot;)
+ 	compare_pheatmap(test, display_numbers = matrix(ifelse(test &gt; 5, &quot;*&quot;, &quot;&quot;), nrow(test)))
+ 	compare_pheatmap(test, cluster_row = FALSE, legend_breaks = -1:4, legend_labels = c(&quot;0&quot;,
+ 		&quot;1e-4&quot;, &quot;1e-3&quot;, &quot;1e-2&quot;, &quot;1e-1&quot;, &quot;1&quot;))
+ 
+ 	# Fix cell sizes and save to file with correct size
+ 	compare_pheatmap(test, cellwidth = 15, cellheight = 12, main = &quot;Example heatmap&quot;)
+ 
+ 	# Generate annotations for rows and columns
+ 	annotation_col = data.frame(
+ 	    CellType = factor(rep(c(&quot;CT1&quot;, &quot;CT2&quot;), 5)), 
+ 	    Time = 1:5
+ 	)
+ 	rownames(annotation_col) = paste(&quot;Test&quot;, 1:10, sep = &quot;&quot;)
+ 
+ 	annotation_row = data.frame(
+ 	    GeneClass = factor(rep(c(&quot;Path1&quot;, &quot;Path2&quot;, &quot;Path3&quot;), c(10, 4, 6)))
+ 	)
+ 	rownames(annotation_row) = paste(&quot;Gene&quot;, 1:20, sep = &quot;&quot;)
+ 
+ 	# Display row and color annotations
+ 	compare_pheatmap(test, annotation_col = annotation_col)
+ 	compare_pheatmap(test, annotation_col = annotation_col, annotation_legend = FALSE)
+ 	compare_pheatmap(test, annotation_col = annotation_col, annotation_row = annotation_row)
+ 
+ 	# Change angle of text in the columns
+ 	compare_pheatmap(test, annotation_col = annotation_col, annotation_row = annotation_row, angle_col = &quot;45&quot;)
+ 	compare_pheatmap(test, annotation_col = annotation_col, angle_col = &quot;0&quot;)
+ 
+ 	# Specify colors
+ 	ann_colors = list(
+ 	    Time = c(&quot;white&quot;, &quot;firebrick&quot;),
+ 	    CellType = c(CT1 = &quot;#1B9E77&quot;, CT2 = &quot;#D95F02&quot;),
+ 	    GeneClass = c(Path1 = &quot;#7570B3&quot;, Path2 = &quot;#E7298A&quot;, Path3 = &quot;#66A61E&quot;)
+ 	)
+ 
+ 	compare_pheatmap(test, annotation_col = annotation_col, annotation_colors = ann_colors, main = &quot;Title&quot;)
+ 	compare_pheatmap(test, annotation_col = annotation_col, annotation_row = annotation_row, 
+ 	         annotation_colors = ann_colors)
+ 	compare_pheatmap(test, annotation_col = annotation_col, annotation_colors = ann_colors[2]) 
+ 
+ 	# Gaps in heatmaps
+ 	compare_pheatmap(test, annotation_col = annotation_col, cluster_rows = FALSE, gaps_row = c(10, 14))
+ 	compare_pheatmap(test, annotation_col = annotation_col, cluster_rows = FALSE, gaps_row = c(10, 14), 
+ 	         cutree_col = 2)
+ 
+ 	# Show custom strings as row/col names
+ 	labels_row = c(&quot;&quot;, &quot;&quot;, &quot;&quot;, &quot;&quot;, &quot;&quot;, &quot;&quot;, &quot;&quot;, &quot;&quot;, &quot;&quot;, &quot;&quot;, &quot;&quot;, &quot;&quot;, &quot;&quot;, &quot;&quot;, &quot;&quot;, 
+ 		&quot;&quot;, &quot;&quot;, &quot;Il10&quot;, &quot;Il15&quot;, &quot;Il1b&quot;)
+ 
+ 	compare_pheatmap(test, annotation_col = annotation_col, labels_row = labels_row)
+ 
+ 	# Specifying clustering from distance matrix
+ 	drows = dist(test, method = &quot;minkowski&quot;)
+ 	dcols = dist(t(test), method = &quot;minkowski&quot;)
+ 	compare_pheatmap(test, clustering_distance_rows = drows, clustering_distance_cols = dcols)
+ 
+ 	library(dendsort)
+ 
+ 	callback = function(hc, ...){dendsort(hc)}
+ 	compare_pheatmap(test, clustering_callback = callback)
+ }
Loading required namespace: pheatmap
Warning message:
argument `kmeans_k` is not suggested to use in pheatmap -&gt; Heatmap
translation because it changes the input matrix. You might check
`row_km` and `column_km` arguments in Heatmap(). 
&gt; 
&gt; 
&gt; set.seed(42)
&gt; nsamples &lt;- 10
&gt; 
&gt; mat &lt;- matrix(rpois(20*nsamples, 20), ncol=nsamples)
&gt; colnames(mat) &lt;- paste0(&quot;sample&quot;, seq_len(ncol(mat)))
&gt; rownames(mat) &lt;- paste0(&quot;gene&quot;, seq_len(nrow(mat)))
&gt; 
&gt; annot &lt;- data.frame(
+   labs = sample(c(&quot;A&quot;,&quot;B&quot;,&quot;C&quot;,&quot;D&quot;), size = ncol(mat), replace = TRUE),
+   row.names = colnames(mat)
+ )
&gt; ins &lt;- list(mat = mat, annotation_col = annot)
&gt; do.call(ComplexHeatmap::pheatmap, ins[1])
&gt; do.call(ComplexHeatmap::pheatmap, ins)
&gt; 
&gt; proc.time()
   user  system elapsed 
 35.205   0.577  35.770 
</PRE>
</DIV><P><SPAN class="filename">ComplexHeatmap.Rcheck/tests/test-SingleAnnotation.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(circlize)
========================================
circlize version 0.4.15
CRAN page: https://cran.r-project.org/package=circlize
Github page: https://github.com/jokergoo/circlize
Documentation: https://jokergoo.github.io/circlize_book/book/

If you use it in published research, please cite:
Gu, Z. circlize implements and enhances circular visualization
  in R. Bioinformatics 2014.

This message can be suppressed by:
  suppressPackageStartupMessages(library(circlize))
========================================

&gt; library(ComplexHeatmap)
Loading required package: grid
========================================
ComplexHeatmap version 2.17.0
Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/
Github page: https://github.com/jokergoo/ComplexHeatmap
Documentation: http://jokergoo.github.io/ComplexHeatmap-reference

If you use it in published research, please cite either one:
- Gu, Z. Complex Heatmap Visualization. iMeta 2022.
- Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional 
    genomic data. Bioinformatics 2016.


The new InteractiveComplexHeatmap package can directly export static 
complex heatmaps into an interactive Shiny app with zero effort. Have a try!

This message can be suppressed by:
  suppressPackageStartupMessages(library(ComplexHeatmap))
========================================

&gt; library(GetoptLong)
&gt; 
&gt; ha = SingleAnnotation(value = 1:10)
&gt; draw(ha, test = &quot;single column annotation&quot;)
&gt; ha = SingleAnnotation(value = 1:10, which = &quot;row&quot;)
&gt; draw(ha, test = &quot;single row annotation&quot;)
&gt; ha = SingleAnnotation(value = 1:10)
&gt; draw(ha, index = 6:10, test = &quot;single column annotation, subset&quot;)
&gt; draw(ha, index = 6:10, k = 1, n = 2, test = &quot;single column annotation, subset, k=1 n=2&quot;)
&gt; draw(ha, index = 6:10, k = 2, n = 2, test = &quot;single column annotation, subset, k=1 n=2&quot;)
&gt; 
&gt; x = 1:10
&gt; ha = SingleAnnotation(value = x)
&gt; draw(ha, test = &quot;single column annotation&quot;)
&gt; 
&gt; m = cbind(1:10, 10:1)
&gt; colnames(m) = c(&quot;a&quot;, &quot;b&quot;)
&gt; ha = SingleAnnotation(value = m)
&gt; draw(ha, test = &quot;matrix as column annotation&quot;)
&gt; 
&gt; ha = SingleAnnotation(value = 1:10, col = colorRamp2(c(1, 10), c(&quot;blue&quot;, &quot;red&quot;)))
&gt; draw(ha, test = &quot;color mapping function&quot;)
&gt; 
&gt; ha = SingleAnnotation(value = c(rep(c(&quot;a&quot;, &quot;b&quot;), 5)))
&gt; draw(ha, test = &quot;discrete annotation&quot;)
&gt; ha = SingleAnnotation(value = c(rep(c(&quot;a&quot;, &quot;b&quot;), 5)), col = c(&quot;a&quot; = &quot;red&quot;, &quot;b&quot; = &quot;blue&quot;))
&gt; draw(ha, test = &quot;discrete annotation with defined colors&quot;)
&gt; 
&gt; anno = anno_simple(1:10)
&gt; ha = SingleAnnotation(fun = anno)
&gt; draw(ha, test = &quot;AnnotationFunction as input&quot;)
&gt; 
&gt; anno = anno_barplot(matrix(nc = 2, c(1:10, 10:1)))
&gt; ha = SingleAnnotation(fun = anno)
&gt; draw(ha, test = &quot;anno_barplot as input&quot;)
&gt; draw(ha, index = 1:5, test = &quot;anno_barplot as input, 1:5&quot;)
&gt; draw(ha, index = 1:5, k = 1, n = 2, test = &quot;anno_barplot as input, 1:5, k = 1, n = 2&quot;)
&gt; draw(ha, index = 1:5, k = 2, n = 2, test = &quot;anno_barplot as input, 1:5, k = 2, n = 2&quot;)
&gt; 
&gt; lt = lapply(1:20, function(x) cumprod(1 + runif(1000, -x/100, x/100)) - 1)
&gt; anno = anno_horizon(lt, which = &quot;row&quot;)
&gt; ha = SingleAnnotation(fun = anno, which = &quot;row&quot;)
&gt; draw(ha, test = &quot;anno_horizon as input&quot;)
&gt; 
&gt; fun = local({
+ 	value = 1:10
+ 	function(index, k = 1, n = 1) {
+ 		pushViewport(viewport(xscale = c(0.5, length(index) + 0.5), yscale = range(value)))
+ 		grid.points(seq_along(index), value[index])
+ 		grid.rect()
+ 		if(k == 1) grid.yaxis()
+ 		popViewport()
+ 	}
+ })
&gt; ha = SingleAnnotation(fun = fun, height = unit(4, &quot;cm&quot;))
&gt; # ha[1:5]
&gt; draw(ha, index = c(1, 4, 2, 6), test = &quot;self-defined function&quot;)
&gt; draw(ha, index = c(1, 4, 2, 6), k = 1, n = 2, test = &quot;self-defined function, k = 1, n = 2&quot;)
&gt; draw(ha, index = c(1, 4, 2, 6), k = 2, n = 2, test = &quot;self-defined function, k = 2, n = 2&quot;)
&gt; 
&gt; 
&gt; # test gridtext
&gt; ha = SingleAnnotation(value = 1:10, label = gt_render(&quot;foo&quot;, r = unit(2, &quot;pt&quot;)), name_gp = gpar(box_fill = &quot;red&quot;))
Loading required namespace: gridtext
&gt; draw(ha, test = &quot;single column annotation&quot;)
&gt; 
&gt; 
&gt; 
&gt; proc.time()
   user  system elapsed 
  5.158   0.178   5.316 
</PRE>
</DIV><P><SPAN class="filename">ComplexHeatmap.Rcheck/tests/test-textbox.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; 
&gt; library(ComplexHeatmap)
Loading required package: grid
========================================
ComplexHeatmap version 2.17.0
Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/
Github page: https://github.com/jokergoo/ComplexHeatmap
Documentation: http://jokergoo.github.io/ComplexHeatmap-reference

If you use it in published research, please cite either one:
- Gu, Z. Complex Heatmap Visualization. iMeta 2022.
- Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional 
    genomic data. Bioinformatics 2016.


The new InteractiveComplexHeatmap package can directly export static 
complex heatmaps into an interactive Shiny app with zero effort. Have a try!

This message can be suppressed by:
  suppressPackageStartupMessages(library(ComplexHeatmap))
========================================

&gt; 
&gt; words = sapply(1:30, function(x) strrep(sample(letters, 1), sample(3:10, 1)))
&gt; grid.newpage()
&gt; grid.textbox(words, gp = gpar(fontsize = runif(30, min = 5, max = 30)))
&gt; 
&gt; sentenses = c(&quot;This is sentense 1&quot;, &quot;This is a long long long long long long long sentense.&quot;)
&gt; grid.newpage()
&gt; grid.textbox(sentenses)
&gt; grid.textbox(sentenses, word_wrap = TRUE)
&gt; grid.textbox(sentenses, word_wrap = TRUE, add_new_line = TRUE)
&gt; 
&gt; 
&gt; require(circlize)
Loading required package: circlize
========================================
circlize version 0.4.15
CRAN page: https://cran.r-project.org/package=circlize
Github page: https://github.com/jokergoo/circlize
Documentation: https://jokergoo.github.io/circlize_book/book/

If you use it in published research, please cite:
Gu, Z. circlize implements and enhances circular visualization
  in R. Bioinformatics 2014.

This message can be suppressed by:
  suppressPackageStartupMessages(library(circlize))
========================================

&gt; mat = matrix(rnorm(100*10), nrow = 100)
&gt; 
&gt; split = sample(letters[1:10], 100, replace = TRUE)
&gt; text = lapply(unique(split), function(x) {
+ 	data.frame(month.name, col = rand_color(12, friendly = TRUE), fontsize = runif(12, 6, 14))
+ })
&gt; names(text) = unique(split)
&gt; 
&gt; Heatmap(mat, cluster_rows = FALSE, row_split = split,
+     right_annotation = rowAnnotation(wc = anno_textbox(split, text))
+ )
&gt; 
&gt; proc.time()
   user  system elapsed 
  4.965   0.180   5.126 
</PRE>
</DIV><P><SPAN class="filename">ComplexHeatmap.Rcheck/tests/test-upset.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(circlize)
========================================
circlize version 0.4.15
CRAN page: https://cran.r-project.org/package=circlize
Github page: https://github.com/jokergoo/circlize
Documentation: https://jokergoo.github.io/circlize_book/book/

If you use it in published research, please cite:
Gu, Z. circlize implements and enhances circular visualization
  in R. Bioinformatics 2014.

This message can be suppressed by:
  suppressPackageStartupMessages(library(circlize))
========================================

&gt; library(ComplexHeatmap)
Loading required package: grid
========================================
ComplexHeatmap version 2.17.0
Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/
Github page: https://github.com/jokergoo/ComplexHeatmap
Documentation: http://jokergoo.github.io/ComplexHeatmap-reference

If you use it in published research, please cite either one:
- Gu, Z. Complex Heatmap Visualization. iMeta 2022.
- Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional 
    genomic data. Bioinformatics 2016.


The new InteractiveComplexHeatmap package can directly export static 
complex heatmaps into an interactive Shiny app with zero effort. Have a try!

This message can be suppressed by:
  suppressPackageStartupMessages(library(ComplexHeatmap))
========================================

&gt; library(GetoptLong)
&gt; 
&gt; set.seed(123)
&gt; lt = list(a = sample(letters, 10),
+ 	      b = sample(letters, 15),
+ 	      c = sample(letters, 20))
&gt; 
&gt; m = make_comb_mat(lt)
&gt; t(m)
A combination matrix with 3 sets and 6 combinations.
  ranges of combination set size: c(1, 8).
  mode for the combination size: distinct.
  sets are on columns

Combination sets are:
  a b c code size
  x x x  111    4
  x x    110    4
  x   x  101    2
    x x  011    6
    x    010    1
      x  001    8

Sets are:
  set size
    a   10
    b   15
    c   20
&gt; set_name(m)
[1] &quot;a&quot; &quot;b&quot; &quot;c&quot;
&gt; comb_name(m)
[1] &quot;111&quot; &quot;110&quot; &quot;101&quot; &quot;011&quot; &quot;010&quot; &quot;001&quot;
&gt; set_size(m)
 a  b  c 
10 15 20 
&gt; comb_size(m)
111 110 101 011 010 001 
  4   4   2   6   1   8 
&gt; lapply(comb_name(m), function(x) extract_comb(m, x))
[[1]]
[1] &quot;e&quot; &quot;j&quot; &quot;x&quot; &quot;y&quot;

[[2]]
[1] &quot;c&quot; &quot;k&quot; &quot;n&quot; &quot;s&quot;

[[3]]
[1] &quot;o&quot; &quot;r&quot;

[[4]]
[1] &quot;a&quot; &quot;g&quot; &quot;h&quot; &quot;i&quot; &quot;l&quot; &quot;u&quot;

[[5]]
[1] &quot;d&quot;

[[6]]
[1] &quot;b&quot; &quot;f&quot; &quot;m&quot; &quot;q&quot; &quot;t&quot; &quot;v&quot; &quot;w&quot; &quot;z&quot;

&gt; draw(UpSet(m))
&gt; draw(UpSet(m, comb_col = c(rep(2, 3), rep(3, 3), 1)))
&gt; draw(UpSet(t(m)))
&gt; 
&gt; set_name(t(m))
[1] &quot;a&quot; &quot;b&quot; &quot;c&quot;
&gt; comb_name(t(m))
[1] &quot;111&quot; &quot;110&quot; &quot;101&quot; &quot;011&quot; &quot;010&quot; &quot;001&quot;
&gt; set_size(t(m))
 a  b  c 
10 15 20 
&gt; comb_size(t(m))
111 110 101 011 010 001 
  4   4   2   6   1   8 
&gt; lapply(comb_name(t(m)), function(x) extract_comb(t(m), x))
[[1]]
[1] &quot;e&quot; &quot;j&quot; &quot;x&quot; &quot;y&quot;

[[2]]
[1] &quot;c&quot; &quot;k&quot; &quot;n&quot; &quot;s&quot;

[[3]]
[1] &quot;o&quot; &quot;r&quot;

[[4]]
[1] &quot;a&quot; &quot;g&quot; &quot;h&quot; &quot;i&quot; &quot;l&quot; &quot;u&quot;

[[5]]
[1] &quot;d&quot;

[[6]]
[1] &quot;b&quot; &quot;f&quot; &quot;m&quot; &quot;q&quot; &quot;t&quot; &quot;v&quot; &quot;w&quot; &quot;z&quot;

&gt; 
&gt; m = make_comb_mat(lt, mode = &quot;intersect&quot;)
&gt; lapply(comb_name(m), function(x) extract_comb(m, x))
[[1]]
[1] &quot;e&quot; &quot;j&quot; &quot;x&quot; &quot;y&quot;

[[2]]
[1] &quot;c&quot; &quot;e&quot; &quot;j&quot; &quot;k&quot; &quot;n&quot; &quot;s&quot; &quot;x&quot; &quot;y&quot;

[[3]]
[1] &quot;e&quot; &quot;j&quot; &quot;o&quot; &quot;r&quot; &quot;x&quot; &quot;y&quot;

[[4]]
 [1] &quot;a&quot; &quot;e&quot; &quot;g&quot; &quot;h&quot; &quot;i&quot; &quot;j&quot; &quot;l&quot; &quot;u&quot; &quot;x&quot; &quot;y&quot;

[[5]]
 [1] &quot;c&quot; &quot;e&quot; &quot;j&quot; &quot;k&quot; &quot;n&quot; &quot;o&quot; &quot;r&quot; &quot;s&quot; &quot;x&quot; &quot;y&quot;

[[6]]
 [1] &quot;a&quot; &quot;c&quot; &quot;d&quot; &quot;e&quot; &quot;g&quot; &quot;h&quot; &quot;i&quot; &quot;j&quot; &quot;k&quot; &quot;l&quot; &quot;n&quot; &quot;s&quot; &quot;u&quot; &quot;x&quot; &quot;y&quot;

[[7]]
 [1] &quot;a&quot; &quot;b&quot; &quot;e&quot; &quot;f&quot; &quot;g&quot; &quot;h&quot; &quot;i&quot; &quot;j&quot; &quot;l&quot; &quot;m&quot; &quot;o&quot; &quot;q&quot; &quot;r&quot; &quot;t&quot; &quot;u&quot; &quot;v&quot; &quot;w&quot; &quot;x&quot; &quot;y&quot;
[20] &quot;z&quot;

&gt; draw(UpSet(m))
&gt; 
&gt; m = make_comb_mat(lt, mode = &quot;union&quot;)
&gt; lapply(comb_name(m), function(x) extract_comb(m, x))
[[1]]
 [1] &quot;a&quot; &quot;b&quot; &quot;c&quot; &quot;d&quot; &quot;e&quot; &quot;f&quot; &quot;g&quot; &quot;h&quot; &quot;i&quot; &quot;j&quot; &quot;k&quot; &quot;l&quot; &quot;m&quot; &quot;n&quot; &quot;o&quot; &quot;q&quot; &quot;r&quot; &quot;s&quot; &quot;t&quot;
[20] &quot;u&quot; &quot;v&quot; &quot;w&quot; &quot;x&quot; &quot;y&quot; &quot;z&quot;

[[2]]
 [1] &quot;a&quot; &quot;c&quot; &quot;d&quot; &quot;e&quot; &quot;g&quot; &quot;h&quot; &quot;i&quot; &quot;j&quot; &quot;k&quot; &quot;l&quot; &quot;n&quot; &quot;o&quot; &quot;r&quot; &quot;s&quot; &quot;u&quot; &quot;x&quot; &quot;y&quot;

[[3]]
 [1] &quot;a&quot; &quot;b&quot; &quot;c&quot; &quot;e&quot; &quot;f&quot; &quot;g&quot; &quot;h&quot; &quot;i&quot; &quot;j&quot; &quot;k&quot; &quot;l&quot; &quot;m&quot; &quot;n&quot; &quot;o&quot; &quot;q&quot; &quot;r&quot; &quot;s&quot; &quot;t&quot; &quot;u&quot;
[20] &quot;v&quot; &quot;w&quot; &quot;x&quot; &quot;y&quot; &quot;z&quot;

[[4]]
 [1] &quot;a&quot; &quot;b&quot; &quot;c&quot; &quot;d&quot; &quot;e&quot; &quot;f&quot; &quot;g&quot; &quot;h&quot; &quot;i&quot; &quot;j&quot; &quot;k&quot; &quot;l&quot; &quot;m&quot; &quot;n&quot; &quot;o&quot; &quot;q&quot; &quot;r&quot; &quot;s&quot; &quot;t&quot;
[20] &quot;u&quot; &quot;v&quot; &quot;w&quot; &quot;x&quot; &quot;y&quot; &quot;z&quot;

[[5]]
 [1] &quot;c&quot; &quot;e&quot; &quot;j&quot; &quot;k&quot; &quot;n&quot; &quot;o&quot; &quot;r&quot; &quot;s&quot; &quot;x&quot; &quot;y&quot;

[[6]]
 [1] &quot;a&quot; &quot;c&quot; &quot;d&quot; &quot;e&quot; &quot;g&quot; &quot;h&quot; &quot;i&quot; &quot;j&quot; &quot;k&quot; &quot;l&quot; &quot;n&quot; &quot;s&quot; &quot;u&quot; &quot;x&quot; &quot;y&quot;

[[7]]
 [1] &quot;a&quot; &quot;b&quot; &quot;e&quot; &quot;f&quot; &quot;g&quot; &quot;h&quot; &quot;i&quot; &quot;j&quot; &quot;l&quot; &quot;m&quot; &quot;o&quot; &quot;q&quot; &quot;r&quot; &quot;t&quot; &quot;u&quot; &quot;v&quot; &quot;w&quot; &quot;x&quot; &quot;y&quot;
[20] &quot;z&quot;

&gt; draw(UpSet(m))
&gt; 
&gt; f = system.file(&quot;extdata&quot;, &quot;movies.csv&quot;, package = &quot;UpSetR&quot;)
&gt; if(file.exists(f)) {
+ 	movies &lt;- read.csv(system.file(&quot;extdata&quot;, &quot;movies.csv&quot;, package = &quot;UpSetR&quot;), header = T, sep = &quot;;&quot;)
+ 	m = make_comb_mat(movies, top_n_sets = 6)
+ 	t(m)
+ 	set_name(m)
+ 	comb_name(m)
+ 	set_size(m)
+ 	comb_size(m)
+ 	lapply(comb_name(m), function(x) extract_comb(m, x))
+ 
+ 	set_name(t(m))
+ 	comb_name(t(m))
+ 	set_size(t(m))
+ 	comb_size(t(m))
+ 	lapply(comb_name(t(m)), function(x) extract_comb(t(m), x))
+ 
+ 	draw(UpSet(m))
+ 	draw(UpSet(t(m)))
+ 
+ 	m = make_comb_mat(movies, top_n_sets = 6, mode = &quot;intersect&quot;)
+ 	m = make_comb_mat(movies, top_n_sets = 6, mode = &quot;union&quot;)
+ }
&gt; 
&gt; library(circlize)
&gt; library(GenomicRanges)
Loading required package: stats4
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors

Attaching package: &#x27;S4Vectors&#x27;

The following object is masked from &#x27;package:utils&#x27;:

    findMatches

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
&gt; lt = lapply(1:4, function(i) generateRandomBed())
&gt; lt = lapply(lt, function(df) GRanges(seqnames = df[, 1], ranges = IRanges(df[, 2], df[, 3])))
&gt; names(lt) = letters[1:4]
&gt; m = make_comb_mat(lt)
&gt; 
&gt; # if(0) {
&gt; # set.seed(123)
&gt; # lt = list(a = sample(letters, 10),
&gt; # 	      b = sample(letters, 15),
&gt; # 	      c = sample(letters, 20))
&gt; # v = gplots::venn(lt, show.plot = FALSE)
&gt; # rownames(v) = apply(v[, -1], 1, paste, collapse = &quot;&quot;)
&gt; # m = make_comb_mat(lt)
&gt; # cs = structure(comb_size(m), names = comb_name(m))
&gt; # }
&gt; 
&gt; if(file.exists(f)) {
+ 	movies &lt;- read.csv(f, header = T, sep = &quot;;&quot;)
+ 	genre = c(&quot;Action&quot;, &quot;Romance&quot;, &quot;Horror&quot;, &quot;Children&quot;, &quot;SciFi&quot;, &quot;Documentary&quot;)
+ 	rate = cut(movies$AvgRating, c(0, 1, 2, 3, 4, 5))
+ 	m_list = tapply(seq_len(nrow(movies)), rate, function(ind) {
+ 		make_comb_mat(movies[ind, genre, drop = FALSE])
+ 	})
+ 	m_list2 = normalize_comb_mat(m_list)
+ 
+ 	lapply(m_list2, set_name)
+ 	lapply(m_list2, set_size)
+ 	lapply(m_list2, comb_name)
+ 	lapply(m_list2, comb_size)
+ 
+ 	lapply(1:length(m_list), function(i) {
+ 		n1 = comb_name(m_list[[i]])
+ 		x1 = comb_size(m_list[[i]])
+ 		n2 = comb_name(m_list2[[i]])
+ 		x2 = comb_size(m_list2[[i]])
+ 		l = n2 %in% n1
+ 		x2[!l]
+ 	})
+ }
&gt; 
&gt; 
&gt; proc.time()
   user  system elapsed 
 17.293   0.264  17.539 
</PRE>
</DIV><P><SPAN class="filename">ComplexHeatmap.Rcheck/tests/test-utils.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(circlize)
========================================
circlize version 0.4.15
CRAN page: https://cran.r-project.org/package=circlize
Github page: https://github.com/jokergoo/circlize
Documentation: https://jokergoo.github.io/circlize_book/book/

If you use it in published research, please cite:
Gu, Z. circlize implements and enhances circular visualization
  in R. Bioinformatics 2014.

This message can be suppressed by:
  suppressPackageStartupMessages(library(circlize))
========================================

&gt; library(ComplexHeatmap)
Loading required package: grid
========================================
ComplexHeatmap version 2.17.0
Bioconductor page: http://bioconductor.org/packages/ComplexHeatmap/
Github page: https://github.com/jokergoo/ComplexHeatmap
Documentation: http://jokergoo.github.io/ComplexHeatmap-reference

If you use it in published research, please cite either one:
- Gu, Z. Complex Heatmap Visualization. iMeta 2022.
- Gu, Z. Complex heatmaps reveal patterns and correlations in multidimensional 
    genomic data. Bioinformatics 2016.


The new InteractiveComplexHeatmap package can directly export static 
complex heatmaps into an interactive Shiny app with zero effort. Have a try!

This message can be suppressed by:
  suppressPackageStartupMessages(library(ComplexHeatmap))
========================================

&gt; library(GetoptLong)
&gt; 
&gt; # things needed to be tested
&gt; # 1. the order
&gt; # 2. if the sum of sizes are larger than xlim
&gt; 
&gt; make_plot = function(pos1, pos2, range) {
+ 	oxpd = par(&quot;xpd&quot;)
+ 	par(xpd = NA)
+ 	plot(NULL, xlim = c(0, 4), ylim = range, ann = FALSE)
+ 	col = rand_color(nrow(pos1), transparency = 0.5)
+ 	rect(0.5, pos1[, 1], 1.5, pos1[, 2], col = col)
+ 	rect(2.5, pos2[, 1], 3.5, pos2[, 2], col = col)
+ 	segments(1.5, rowMeans(pos1), 2.5, rowMeans(pos2))
+ 	par(xpd = oxpd)
+ }
&gt; 
&gt; range = c(0, 10)
&gt; pos1 = rbind(c(1, 2), c(5, 7))
&gt; make_plot(pos1, smartAlign2(pos1, range = range), range)
&gt; 
&gt; range = c(0, 10)
&gt; pos1 = rbind(c(-0.5, 2), c(5, 7))
&gt; make_plot(pos1, smartAlign2(pos1, range = range), range)
&gt; 
&gt; pos1 = rbind(c(-1, 2), c(3, 4), c(5, 6), c(7, 11))
&gt; pos1 = pos1 + runif(length(pos1), max = 0.3, min = -0.3)
&gt; par(mfrow = c(3, 3))
&gt; for(i in 1:9) {
+ 	ind = sample(4, 4)
+ 	make_plot(pos1[ind, ], smartAlign2(pos1[ind, ], range = range), range)
+ }
&gt; par(mfrow = c(1, 1))
&gt; 
&gt; pos1 = rbind(c(3, 6), c(4, 7))
&gt; make_plot(pos1, smartAlign2(pos1, range = range), range)
&gt; 
&gt; pos1 = rbind(c(1, 8), c(3, 10))
&gt; make_plot(pos1, smartAlign2(pos1, range = range), range)
&gt; 
&gt; ########## new version of smartAlign2() ############
&gt; 
&gt; start = c(0.0400972528391016, 0.0491583597430212, 0.0424302664385027, 0.0547524243812509, 0.0820937279769642, 0.126861283282835, 0.178503822565168, 0.327742831447437, 0.570671411156898, 0.81775868755151)
&gt; end = c(0.0921142856224367, 0.107091640256979, 0.137858195099959, 0.159189883311057, 0.177521656638421, 0.20727333210178, 0.304669254357909, 0.463122553167947, 0.676924742689255, 0.929837466294643)
&gt; range = c(0, 1)
&gt; smartAlign2(start, end, range, plot = TRUE)
enter to continue
             [,1]       [,2]
 [1,] 0.002200888 0.05421792
 [2,] 0.054217921 0.11215120
 [3,] 0.112151202 0.20757913
 [4,] 0.207579130 0.31201659
 [5,] 0.312016589 0.40744452
 [6,] 0.407444518 0.48785657
 [7,] 0.487856567 0.61402200
 [8,] 0.614021999 0.74940172
 [9,] 0.749401720 0.85565505
[10,] 0.855655052 0.96773383
&gt; 
&gt; 
&gt; start &lt;- c(0.722121284290678, 0.701851666769472, 0.284795592003117, 0.335674695572052, 0.246977082249377, 0.767289857630785, 0.728198060058033, 0.299241440370817, -0.0149946764559372, 0.85294351791166, 0.126216621670218, 0.478169948493225)
&gt; end &lt;- c(0.766196472718668, 0.763101604258565, 0.34604552949221, 0.421334650222341, 0.344144413077725, 0.847196123677626, 0.813858014708322, 0.392347344675911, 0.108452620381171, 0.969486388630396, 0.249951602628847, 0.584914163656308)
&gt; od = order(start)
&gt; start = start[od]; end = end[od]
&gt; range = c(0, 1)
&gt; pos = smartAlign2(start, end, range)
&gt; n = nrow(pos)
&gt; pos[1:(n-1), 2] &gt; pos[2:n, 1]
 [1] TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE
&gt; 
&gt; 
&gt; if(0) {
+ 	go_id = random_GO(500)
+ 	mat = GO_similarity(go_id)
+ 	invisible(simplify(mat, order_by_size = FALSE))
+ }
&gt; 
&gt; proc.time()
   user  system elapsed 
  3.290   0.138   3.407 
</PRE>
</DIV><P><SPAN class="filename">ComplexHeatmap.Rcheck/tests/testthat-all.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; 
&gt; 
&gt; suppressWarnings(suppressPackageStartupMessages(library(ComplexHeatmap)))
&gt; library(testthat)
&gt; 
&gt; test_check(&quot;ComplexHeatmap&quot;)
[ FAIL 0 | WARN 0 | SKIP 0 | PASS 181 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
 25.343   0.586  37.454 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">ComplexHeatmap.Rcheck/ComplexHeatmap-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>AdditiveUnit-class</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>AdditiveUnit</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>AnnotationFunction-class</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>AnnotationFunction</TD><TD style="text-align: right;">5.424</TD><TD style="text-align: right;">0.092</TD><TD style="text-align: right;">5.516</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ColorMapping-class</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ColorMapping</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ComplexHeatmap-package</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Extract.AnnotationFunction</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Extract.Heatmap</TD><TD style="text-align: right;">0.69</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.69</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Extract.HeatmapAnnotation</TD><TD style="text-align: right;">0.057</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.056</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Extract.HeatmapList</TD><TD style="text-align: right;">0.191</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.192</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Extract.SingleAnnotation</TD><TD style="text-align: right;">0.022</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.022</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Extract.comb_mat</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Extract.gridtext</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Heatmap-class</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Heatmap</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Heatmap3D</TD><TD style="text-align: right;">0.204</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.212</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>HeatmapAnnotation-class</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>HeatmapAnnotation</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>HeatmapList-class</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>HeatmapList</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Legend</TD><TD style="text-align: right;">0.092</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.101</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Legends-class</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Legends</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SingleAnnotation-class</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SingleAnnotation</TD><TD style="text-align: right;">0.090</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.094</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>UpSet</TD><TD style="text-align: right;">0.657</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.662</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>add.AdditiveUnit</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>add_heatmap-Heatmap-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>add_heatmap-HeatmapAnnotation-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>add_heatmap-HeatmapList-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>add_heatmap-dispatch</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>adjust_dend_by_x</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.018</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>adjust_heatmap_list-HeatmapList-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>alter_graphic</TD><TD style="text-align: right;">0.212</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.211</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_barplot</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_block</TD><TD style="text-align: right;">1.387</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.387</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_boxplot</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_customize</TD><TD style="text-align: right;">0.837</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.841</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_density</TD><TD style="text-align: right;">0.745</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.757</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_empty</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_histogram</TD><TD style="text-align: right;">0.105</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.105</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_horizon</TD><TD style="text-align: right;">5.749</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">5.789</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_image</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_joyplot</TD><TD style="text-align: right;">0.536</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.549</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_lines</TD><TD style="text-align: right;">0.096</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.096</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_link</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_mark</TD><TD style="text-align: right;">0.581</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.585</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_numeric</TD><TD style="text-align: right;">0.226</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.230</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_oncoprint_barplot</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_points</TD><TD style="text-align: right;">0.026</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.027</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_simple</TD><TD style="text-align: right;">0.073</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.077</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_summary</TD><TD style="text-align: right;">0.412</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.415</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_text</TD><TD style="text-align: right;">0.094</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.095</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_textbox</TD><TD style="text-align: right;">0.821</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.825</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anno_zoom</TD><TD style="text-align: right;">0.449</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.454</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotation_axis_grob</TD><TD style="text-align: right;">0.077</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.081</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotation_legend_size-HeatmapList-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>attach_annotation-Heatmap-method</TD><TD style="text-align: right;">0.774</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.773</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bar3D</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bin_genome</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>c.ColorMapping</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>c.HeatmapAnnotation</TD><TD style="text-align: right;">0.060</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.059</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cluster_between_groups</TD><TD style="text-align: right;">0.035</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.035</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cluster_within_group</TD><TD style="text-align: right;">0.03</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.03</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>color_mapping_legend-ColorMapping-method</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>columnAnnotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>column_dend-Heatmap-method</TD><TD style="text-align: right;">0.390</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.389</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>column_dend-HeatmapList-method</TD><TD style="text-align: right;">1.357</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">1.361</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>column_dend-dispatch</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>column_order-Heatmap-method</TD><TD style="text-align: right;">0.389</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.391</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>column_order-HeatmapList-method</TD><TD style="text-align: right;">1.327</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.326</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>column_order-dispatch</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>comb_degree</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>comb_name</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>comb_size</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>compare_heatmap.2</TD><TD style="text-align: right;">1.360</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.361</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>compare_heatmap</TD><TD style="text-align: right;">0.984</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.984</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>compare_pheatmap</TD><TD style="text-align: right;">1.452</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">1.465</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>complement_size</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>component_height-Heatmap-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>component_height-HeatmapList-method</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>component_height-dispatch</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>component_width-Heatmap-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>component_width-HeatmapList-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>component_width-dispatch</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>copy_all-AnnotationFunction-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>copy_all-SingleAnnotation-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>copy_all-dispatch</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>decorate_annotation</TD><TD style="text-align: right;">0.320</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.323</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>decorate_column_dend</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>decorate_column_names</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>decorate_column_title</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>decorate_dend</TD><TD style="text-align: right;">0.201</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.201</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>decorate_dimnames</TD><TD style="text-align: right;">0.24</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.24</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>decorate_heatmap_body</TD><TD style="text-align: right;">0.172</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.172</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>decorate_row_dend</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>decorate_row_names</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>decorate_row_title</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>decorate_title</TD><TD style="text-align: right;">0.19</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.19</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>default_axis_param</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>default_get_type</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dend_heights</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dend_xy</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dendrogramGrob</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>densityHeatmap</TD><TD style="text-align: right;">1.490</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">1.502</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dim.Heatmap</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dist2</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>draw-AnnotationFunction-method</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>draw-Heatmap-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>draw-HeatmapAnnotation-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>draw-HeatmapList-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>draw-Legends-method</TD><TD style="text-align: right;">0.018</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.018</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>draw-SingleAnnotation-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>draw-dispatch</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>draw_annotation-Heatmap-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>draw_annotation_legend-HeatmapList-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>draw_dend-Heatmap-method</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>draw_dimnames-Heatmap-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>draw_heatmap_body-Heatmap-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>draw_heatmap_legend-HeatmapList-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>draw_heatmap_list-HeatmapList-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>draw_title-Heatmap-method</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>draw_title-HeatmapList-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>draw_title-dispatch</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>extract_comb</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>frequencyHeatmap</TD><TD style="text-align: right;">0.662</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.662</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>full_comb_code</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getXY_in_parent_vp</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_color_mapping_list-HeatmapAnnotation-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_legend_param_list-HeatmapAnnotation-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>grid.annotation_axis</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>grid.boxplot</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>grid.dendrogram</TD><TD style="text-align: right;">0.452</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.452</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>grid.draw.Legends</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>grid.textbox</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>gt_render</TD><TD style="text-align: right;">1.044</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">1.054</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>heatmap_legend_size-HeatmapList-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>height.AnnotationFunction</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>height.Heatmap</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>height.HeatmapAnnotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>height.HeatmapList</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>height.Legends</TD><TD style="text-align: right;">0.022</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.022</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>height.SingleAnnotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>heightAssign.AnnotationFunction</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>heightAssign.HeatmapAnnotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>heightAssign.SingleAnnotation</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>heightDetails.annotation_axis</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>heightDetails.legend</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>heightDetails.legend_body</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>heightDetails.packed_legends</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>heightDetails.textbox</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ht_global_opt</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ht_opt</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ht_size</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>is_abs_unit</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>length.HeatmapAnnotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>length.HeatmapList</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>list_components</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>list_to_matrix</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>make_column_cluster-Heatmap-method</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>make_comb_mat</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>make_layout-Heatmap-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>make_layout-HeatmapList-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>make_layout-dispatch</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>make_row_cluster-Heatmap-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>map_to_colors-ColorMapping-method</TD><TD style="text-align: right;">0.02</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.02</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>max_text_height</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>max_text_width</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>merge_dendrogram</TD><TD style="text-align: right;">0.109</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.109</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>names.HeatmapAnnotation</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>names.HeatmapList</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>namesAssign.HeatmapAnnotation</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ncol.Heatmap</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nobs.AnnotationFunction</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nobs.HeatmapAnnotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nobs.SingleAnnotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>normalize_comb_mat</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>normalize_genomic_signals_to_bins</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nrow.Heatmap</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>oncoPrint</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>order.comb_mat</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>packLegend</TD><TD style="text-align: right;">0.082</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.085</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pct_v_pct</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pheatmap</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pindex</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot.Heatmap</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot.HeatmapAnnotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot.HeatmapList</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>prepare-Heatmap-method</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>print.comb_mat</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>re_size-HeatmapAnnotation-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>restore_matrix</TD><TD style="text-align: right;">0.476</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.476</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rowAnnotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>row_anno_barplot</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>row_anno_boxplot</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>row_anno_density</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>row_anno_histogram</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>row_anno_points</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>row_anno_text</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>row_dend-Heatmap-method</TD><TD style="text-align: right;">0.392</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.392</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>row_dend-HeatmapList-method</TD><TD style="text-align: right;">1.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>row_dend-dispatch</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>row_order-Heatmap-method</TD><TD style="text-align: right;">0.387</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.391</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>row_order-HeatmapList-method</TD><TD style="text-align: right;">0.977</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.985</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>row_order-dispatch</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>set_component_height-Heatmap-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>set_component_width-Heatmap-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>set_name</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>set_nameAssign</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>set_size</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>show-AnnotationFunction-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>show-ColorMapping-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>show-Heatmap-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>show-HeatmapAnnotation-method</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>show-HeatmapList-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>show-SingleAnnotation-method</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>show-dispatch</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>size.AnnotationFunction</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>size.HeatmapAnnotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>size.SingleAnnotation</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sizeAssign.AnnotationFunction</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sizeAssign.HeatmapAnnotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sizeAssign.SingleAnnotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>smartAlign2</TD><TD style="text-align: right;">0.281</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.285</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>str.comb_mat</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>subset_gp</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>subset_matrix_by_row</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>subset_no</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>subset_vector</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>summary.Heatmap</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>summary.HeatmapList</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>t.comb_mat</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>test_alter_fun</TD><TD style="text-align: right;">0.072</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.072</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>textbox_grob</TD><TD style="text-align: right;">0.205</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.205</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>unify_mat_list</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>upset_left_annotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>upset_right_annotation</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>upset_top_annotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>width.AnnotationFunction</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>width.Heatmap</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>width.HeatmapAnnotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>width.HeatmapList</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>width.Legends</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>width.SingleAnnotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>widthAssign.AnnotationFunction</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>widthAssign.HeatmapAnnotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>widthAssign.SingleAnnotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>widthDetails.annotation_axis</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>widthDetails.legend</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>widthDetails.legend_body</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>widthDetails.packed_legends</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>widthDetails.textbox</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
