<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for QDNAseq on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><B>[<A href="../index.html#Q">Q</A>]</B></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-13 06:49:50 -0000 (Mon, 13 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4222</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for QDNAseq on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the QDNAseq package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/QDNAseq.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1549</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">QDNAseq</A>&nbsp;1.35.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/QDNAseq">landing page</A>)</SPAN><BR>Daoud Sie<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-11&nbsp;01:58:29&nbsp;-0000&nbsp;(Sat,&nbsp;11&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/QDNAseq</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">743de98</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:09:22&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: QDNAseq</TD></TR>
<TR><TD><B>Version</B>: 1.35.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:QDNAseq.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings QDNAseq_1.35.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-11 20:03:10 -0000 (Sat, 11 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-11 20:12:08 -0000 (Sat, 11 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 537.9 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: QDNAseq.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:QDNAseq.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings QDNAseq_1.35.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/QDNAseq.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘QDNAseq/DESCRIPTION’ ... OK
* this is package ‘QDNAseq’ version ‘1.35.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘QDNAseq’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                         user system elapsed
callBins               22.805  0.200  23.006
frequencyPlot          22.694  0.084  22.778
normalizeSegmentedBins  9.425  0.052   9.478
segmentBins             9.210  0.036   9.246
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘QDNAseq,copyneutral.R’
  Running ‘QDNAseq,gain-copyneutral-gain.R’
  Running ‘QDNAseq,parallel-reproducibility.R’
  Running ‘QDNAseq.R’
  Running ‘binReadCounts.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘QDNAseq.Rnw’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: OK

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">QDNAseq.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL QDNAseq
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘QDNAseq’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (QDNAseq)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">QDNAseq.Rcheck/tests/binReadCounts.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(QDNAseq)
&gt; 
&gt; if (requireNamespace(&quot;QDNAseq.hg19&quot;, quietly = TRUE)) {
+   bins &lt;- getBinAnnotations(500, genome = &quot;hg19&quot;)
+   print(bins)
+ 
+   bam &lt;- system.file(&quot;extdata&quot;, &quot;ex1.bam&quot;, package = &quot;Rsamtools&quot;)
+   print(bam)
+ 
+   counts &lt;- binReadCounts(bins, bamfiles = bam)
+   print(counts)
+ 
+   ## BUG: https://github.com/ccagc/QDNAseq/issues/89
+   counts2 &lt;- binReadCounts(bins, bamfiles = bam, chunkSize = 10e3)
+   print(counts2)
+ }
Loaded bin annotations for genome &#x27;hg19&#x27;, bin size 500 kbp, and experiment type &#x27;SR50&#x27; from annotation package QDNAseq.hg19 v1.29.0
QDNAseq bin annotations for Hsapiens, build hg19.
Created by Ilari Scheinin with QDNAseq 0.7.5, 2014-02-06 14:20:46.041402.
An object of class &#x27;AnnotatedDataFrame&#x27;
  rowNames: 1:1-500000 1:500001-1000000 ... Y:59000001-59373566 (6206
    total)
  varLabels: chromosome start ... use (9 total)
  varMetadata: labelDescription
[1] &quot;/home/biocbuild/bbs-3.17-bioc/R/library/Rsamtools/extdata/ex1.bam&quot;
    ex1 (1 of 1): extracting reads ... binning ...
QDNAseqReadCounts (storageMode: lockedEnvironment)
assayData: 6206 features, 1 samples 
  element names: counts 
protocolData: none
phenoData
  sampleNames: ex1
  varLabels: name total.reads used.reads expected.variance
  varMetadata: labelDescription
featureData
  featureNames: 1:1-500000 1:500001-1000000 ... Y:59000001-59373566
    (6206 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
    ex1 (1 of 1): extracting reads ...
binning chunk - seq1:1:10001
binning chunk - seq2:1:10001

QDNAseqReadCounts (storageMode: lockedEnvironment)
assayData: 6206 features, 1 samples 
  element names: counts 
protocolData: none
phenoData
  sampleNames: ex1
  varLabels: name total.reads used.reads expected.variance
  varMetadata: labelDescription
featureData
  featureNames: 1:1-500000 1:500001-1000000 ... Y:59000001-59373566
    (6206 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; 
&gt; 
&gt; proc.time()
   user  system elapsed 
  8.294   0.277   8.559 
</PRE>
</DIV><P><SPAN class="filename">QDNAseq.Rcheck/tests/QDNAseq,copyneutral.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(QDNAseq)
&gt; library(Biobase)
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

&gt; set.seed(0xBEEF)
&gt; 
&gt; data(LGG150)
&gt; data &lt;- LGG150
&gt; print(data)
QDNAseqReadCounts (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: counts 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads used.reads expected.variance
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; dataF &lt;- applyFilters(data, residual=TRUE, blacklist=TRUE)
38,819	total bins
38,819	of which in selected chromosomes
36,722	of which with reference sequence
33,347	final bins
&gt; print(dataF)
QDNAseqReadCounts (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: counts 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads used.reads expected.variance
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; dataC &lt;- correctBins(dataF)
Calculating correction for GC content and mappability
    Calculating fit for sample LGG150 (1 of 1) ...
Done.
&gt; 
&gt; ## Force all copy neutral data
&gt; cn &lt;- assayDataElement(dataC, &quot;copynumber&quot;)
&gt; cn[,1] &lt;- rnorm(nrow(cn), mean = 1.0, sd = 0.05)
&gt; assayDataElement(dataC, &quot;copynumber&quot;) &lt;- cn
&gt; 
&gt; print(dataC)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: copynumber 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; dataN &lt;- normalizeBins(dataC)
Applying median normalization ...
&gt; print(dataN)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: copynumber 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; fit &lt;- segmentBins(dataN)
Performing segmentation:
    Segmenting: LGG150 (1 of 1) ...
&gt; print(fit)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: copynumber, segmented 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; fitC &lt;- callBins(fit)
EM algorithm started ... 

[1] &quot;Total number of segments present in the data: 4&quot;
[1] &quot;Number of segments used for fitting the model: 4&quot;
39942648170980213.462.4744605114786712397.7112.9744605112251990397.793.5
Calling iteration1:

optim results
time: 0
minimum: 16682.0890247004
116682.0890247004-0.825303675797091-0.323169506423803-0.001754401929814960.2970863141060160.5078724084702190.9928769615298530.1100346090918040.0003692713871108020.0111811691191362-0.0001065348234756080.1102701734224670.145866377811007
39957698174401213.462.4744605114786712397.7112.9744605112251990397.793.5
Calling iteration2:

optim results
time: 0
minimum: 16682.0888522346
116682.0888522346-0.816384144994661-0.31848916294222-0.00167850977268490.292952991143050.5008064461942260.9819310567480430.1311557982956840.0002223141320088620.01118096583020170.0001872099918784220.1313921659017140.175722206269368
EM algorithm done ...

Computing posterior probabilities for all segments ...

Total time:0minutes

Adjusting segmented data for cellularity ... 

Cellularity sample1: 1

Adjusting normalized data for cellularity ... 

Cellularity sample1: 1

1
39984378396155213.664.1744605114786712397.7112.9744605112251990397.793.5
39984568562901213.665.4744605114786712397.7112.9744605112251990397.793.5
39984538562895213.665.4744605114786712397.7112.9744605112251990397.793.5
39984818829675213.667.4744605117824054397.7136744605112251990397.793.5
39989088830630213.667.4744605117824054397.7136744605112251990397.793.5
39989178830634213.667.4744605117824054397.7136744605112251990397.793.5
39989268830638213.667.4744605117824054397.7136744605112251990397.793.5
39989358830642213.667.4744605117824054397.7136744605112251990397.793.5
39989448830646213.667.4744605117824054397.7136744605112251990397.793.5
39989508830649213.667.4744605117824054397.7136744605112251990397.793.5
39989698897379213.667.9744605117824054397.7136744605112251990397.793.5
39993868897702213.667.9744605117824054397.7136744605112251990397.793.5
FINISHED!

Total time:0minutes

There were 24 warnings (use warnings() to see them)
&gt; print(fitC)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: calls, copynumber, probamp, probdloss, probgain, probloss, probnorm, segmented 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; 
&gt; ## Assert that everything is called copy-neutral
&gt; calls &lt;- assayDataElement(fitC, &quot;calls&quot;)
&gt; stopifnot(all(is.na(calls) | calls == 0))
&gt; 
&gt; formats &lt;- c(&quot;tsv&quot;, &quot;igv&quot;, &quot;bed&quot;, &quot;seg&quot;, &quot;vcf&quot;)
&gt; types &lt;- c(&quot;copynumber&quot;, &quot;segments&quot;, &quot;calls&quot;)
&gt; for (format in formats) {
+   for (type in types) {
+     fileext &lt;- sprintf(&quot;.%s.%s&quot;, type, format)
+     file &lt;- tempfile(pattern =  &quot;QDNAseq-%s&quot;, fileext = fileext)
+     message(sprintf(&quot;  - exportBins(&lt;%d samples&gt;, format=\&quot;%s\&quot;, type=\&quot;%s\&quot;)&quot;, ncol(fitC), format, type))
+     file &lt;- exportBins(fitC, format = format, type = type, file = file)
+     message(sprintf(&quot;    File(s) written: [n=%d] %s&quot;,
+             length(file), paste(sQuote(file), collapse = &quot;, &quot;)))
+     stopifnot(all(file_test(&quot;-f&quot;, file)))
+     if (format == &quot;seg&quot;) {
+       segs &lt;- read.table(file, sep = &quot;\t&quot;, header = TRUE)
+       print(segs)
+       stopifnot(nrow(segs) == 0L)
+     } else if (format == &quot;vcf&quot;) {
+       rows &lt;- readLines(file)
+       rows &lt;- grep(&quot;^#&quot;, rows, invert = TRUE, value = TRUE)
+       print(rows)
+       stopifnot(length(rows) == 0L)
+     }
+     file.remove(file)
+     stopifnot(!any(file_test(&quot;-f&quot;, file)))
+   }
+ }
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmprMtJNO/QDNAseq-%s66296fcfa4d5.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmprMtJNO/QDNAseq-%s662937401ae0.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmprMtJNO/QDNAseq-%s66296531f18a.calls.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmprMtJNO/QDNAseq-%s6629153aaee.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmprMtJNO/QDNAseq-%s66293df481ac.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmprMtJNO/QDNAseq-%s662978a4d54e.calls.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmprMtJNO/QDNAseq-LGG1506629430fe4b0.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmprMtJNO/QDNAseq-LGG150662978f46ae1.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmprMtJNO/QDNAseq-LGG15066297b478eb.calls.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmprMtJNO/QDNAseq-LGG15066293583a28a.copynumber.seg&#x27;
[1] SAMPLE_NAME     CHROMOSOME      START           STOP           
[5] DATAPOINTS      LOG2_RATIO_MEAN
&lt;0 rows&gt; (or 0-length row.names)
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmprMtJNO/QDNAseq-LGG15066296267f9a4.segments.seg&#x27;
[1] SAMPLE_NAME     CHROMOSOME      START           STOP           
[5] DATAPOINTS      LOG2_RATIO_MEAN
&lt;0 rows&gt; (or 0-length row.names)
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmprMtJNO/QDNAseq-LGG150662948f5aba6.calls.seg&#x27;
[1] SAMPLE_NAME     CHROMOSOME      START           STOP           
[5] DATAPOINTS      LOG2_RATIO_MEAN
&lt;0 rows&gt; (or 0-length row.names)
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmprMtJNO/QDNAseq-LGG1506629436e01c.copynumber.vcf&#x27;
character(0)
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmprMtJNO/QDNAseq-LGG15066297d2b8bec.segments.vcf&#x27;
character(0)
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmprMtJNO/QDNAseq-LGG150662910653205.calls.vcf&#x27;
character(0)
&gt; 
&gt; proc.time()
   user  system elapsed 
 24.237   0.377  24.601 
</PRE>
</DIV><P><SPAN class="filename">QDNAseq.Rcheck/tests/QDNAseq,gain-copyneutral-gain.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(QDNAseq)
&gt; library(Biobase)
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

&gt; library(utils)
&gt; set.seed(0xBEEF)
&gt; 
&gt; data(LGG150)
&gt; data &lt;- LGG150
&gt; print(data)
QDNAseqReadCounts (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: counts 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads used.reads expected.variance
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; dataF &lt;- applyFilters(data, residual=TRUE, blacklist=TRUE)
38,819	total bins
38,819	of which in selected chromosomes
36,722	of which with reference sequence
33,347	final bins
&gt; print(dataF)
QDNAseqReadCounts (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: counts 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads used.reads expected.variance
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; dataC &lt;- correctBins(dataF)
Calculating correction for GC content and mappability
    Calculating fit for sample LGG150 (1 of 1) ...
Done.
&gt; 
&gt; ## Force results to have segments gain, copy neutral, and gain.
&gt; cnAll &lt;- assayDataElement(dataC, &quot;copynumber&quot;)
&gt; cnAll[,1] &lt;- rnorm(nrow(cnAll), mean = 1.0, sd = 0.05)
&gt; chr7 &lt;- (chromosomes(dataC) == &quot;7&quot;)
&gt; cn &lt;- cnAll[chr7, , drop = FALSE]
&gt; n &lt;- nrow(cn)
&gt; idxs &lt;- seq(from=1/3*n - 0.1*n, to=1/3*n + 0.1*n)
&gt; cn[idxs,1] &lt;- rnorm(length(idxs), mean = 2.0, sd = 0.05)
&gt; idxs &lt;- seq(from=2/3*n - 0.1*n, to=2/3*n + 0.1*n)
&gt; cn[idxs,1] &lt;- rnorm(length(idxs), mean = 2.0, sd = 0.05)
&gt; cnAll[chr7, ] &lt;- cn
&gt; assayDataElement(dataC, &quot;copynumber&quot;) &lt;- cnAll
&gt; print(dataC)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: copynumber 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; dataN &lt;- normalizeBins(dataC)
Applying median normalization ...
&gt; print(dataN)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: copynumber 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; fit &lt;- segmentBins(dataN)
Performing segmentation:
    Segmenting: LGG150 (1 of 1) ...
&gt; print(fit)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: copynumber, segmented 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; fitC &lt;- callBins(fit)
EM algorithm started ... 

[1] &quot;Total number of segments present in the data: 8&quot;
[1] &quot;Number of segments used for fitting the model: 8&quot;
39945748214559213.462.7744605114786712397.7112.9744605112222393397.793.3
Calling iteration1:

optim results
time: 0
minimum: 16691.936015802
116691.936015802-0.76889026698241-0.310183146024425-0.01185150972096230.2885943760685340.4933553445096880.9879385470392230.0299884700901994-0.0005485736219481460.0111821966975442-4.08995447325668e-050.01000052582656670.0100005260086061
EM algorithm done ...

Computing posterior probabilities for all segments ...

Total time:0minutes

Adjusting segmented data for cellularity ... 

Cellularity sample1: 1

Adjusting normalized data for cellularity ... 

Cellularity sample1: 1

1
39987478439542213.664.4744605114786712397.7112.9744605112222393397.793.3
39987668606288213.665.7744605114786712397.7112.9744605112222393397.793.3
39987638606282213.665.7744605114786712397.7112.9744605112222393397.793.3
39987918874812213.667.8744605117824054397.7136744605112222393397.793.3
39992188875767213.667.8744605117824054397.7136744605112222393397.793.3
39992278875771213.667.8744605117824054397.7136744605112222393397.793.3
39992368875775213.667.8744605117824054397.7136744605112222393397.793.3
39992458875779213.667.8744605117824054397.7136744605112222393397.793.3
39992548875783213.667.8744605117824054397.7136744605112222393397.793.3
39992608875786213.667.8744605117824054397.7136744605112222393397.793.3
39992798942516213.668.3744605117824054397.7136744605112222393397.793.3
39996968942839213.768.3744605117824054397.7136744605112222393397.793.3
FINISHED!

Total time:0minutes

There were 32 warnings (use warnings() to see them)
&gt; print(fitC)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: calls, copynumber, probamp, probdloss, probgain, probloss, probnorm, segmented 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; ## Assert that everything is called copy-neutral
&gt; calls &lt;- assayDataElement(fitC, &quot;calls&quot;)
&gt; stopifnot(all(is.na(calls) | calls %in% c(0, 2)))
&gt; 
&gt; formats &lt;- c(&quot;tsv&quot;, &quot;igv&quot;, &quot;bed&quot;, &quot;seg&quot;, &quot;vcf&quot;)
&gt; types &lt;- c(&quot;copynumber&quot;, &quot;segments&quot;, &quot;calls&quot;)
&gt; for (format in formats) {
+   for (type in types) {
+     fileext &lt;- sprintf(&quot;.%s.%s&quot;, type, format)
+     file &lt;- tempfile(pattern =  &quot;QDNAseq-%s&quot;, fileext = fileext)
+     message(sprintf(&quot;  - exportBins(&lt;%d samples&gt;, format=\&quot;%s\&quot;, type=\&quot;%s\&quot;)&quot;, ncol(fitC), format, type))
+     file &lt;- exportBins(fitC, format = format, type = type, file = file)
+     message(sprintf(&quot;    File(s) written: [n=%d] %s&quot;,
+             length(file), paste(sQuote(file), collapse = &quot;, &quot;)))
+     stopifnot(all(file_test(&quot;-f&quot;, file)))
+ 
+     if (format == &quot;seg&quot;) {
+       segs &lt;- read.table(file, sep = &quot;\t&quot;, header = TRUE)
+       print(segs)
+       stopifnot(all(segs$CHROMOSOME == &quot;7&quot;), nrow(segs) == 2L)
+     } else if (format == &quot;vcf&quot;) {
+       segs &lt;- read.table(file, sep = &quot;\t&quot;, header = FALSE)
+       print(segs)
+       stopifnot(all(segs$V1 == &quot;7&quot;), nrow(segs) == 2L)
+     }
+ 
+     file.remove(file)
+     stopifnot(!any(file_test(&quot;-f&quot;, file)))
+   }
+ }
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpjBlX7H/QDNAseq-%s680a7ee5aa42.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpjBlX7H/QDNAseq-%s680a3e194db4.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpjBlX7H/QDNAseq-%s680a133c3dde.calls.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpjBlX7H/QDNAseq-%s680a3665ce64.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpjBlX7H/QDNAseq-%s680a7efaadca.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpjBlX7H/QDNAseq-%s680a3d6236c7.calls.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpjBlX7H/QDNAseq-LGG150680a703425ce.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpjBlX7H/QDNAseq-LGG150680a266fb9cd.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpjBlX7H/QDNAseq-LGG150680a338d8dd6.calls.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpjBlX7H/QDNAseq-LGG150680a7a8e0492.copynumber.seg&#x27;
                                                SAMPLE_NAME CHROMOSOME    START
1 /tmp/RtmpjBlX7H/QDNAseq-LGG150680a7a8e0492.copynumber.seg          7 37110001
2 /tmp/RtmpjBlX7H/QDNAseq-LGG150680a7a8e0492.copynumber.seg          7 90165001
       STOP DATAPOINTS LOG2_RATIO_MEAN
1  68955000       1487            0.99
2 122010000       2012            0.99
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpjBlX7H/QDNAseq-LGG150680a5f8232dd.segments.seg&#x27;
                                              SAMPLE_NAME CHROMOSOME    START
1 /tmp/RtmpjBlX7H/QDNAseq-LGG150680a5f8232dd.segments.seg          7 37110001
2 /tmp/RtmpjBlX7H/QDNAseq-LGG150680a5f8232dd.segments.seg          7 90165001
       STOP DATAPOINTS LOG2_RATIO_MEAN
1  68955000       1487            0.99
2 122010000       2012            0.99
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpjBlX7H/QDNAseq-LGG150680a5bf33a88.calls.seg&#x27;
                                           SAMPLE_NAME CHROMOSOME    START
1 /tmp/RtmpjBlX7H/QDNAseq-LGG150680a5bf33a88.calls.seg          7 37110001
2 /tmp/RtmpjBlX7H/QDNAseq-LGG150680a5bf33a88.calls.seg          7 90165001
       STOP DATAPOINTS LOG2_RATIO_MEAN
1  68955000       1487            0.99
2 122010000       2012            0.99
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpjBlX7H/QDNAseq-LGG150680a2c4bf3f7.copynumber.vcf&#x27;
  V1       V2 V3    V4    V5   V6   V7
1  7 37110001  . &lt;DIP&gt; &lt;DUP&gt; 1000 PASS
2  7 90165001  . &lt;DIP&gt; &lt;DUP&gt; 1000 PASS
                                                                      V8 V9 V10
1  SVTYPE=DUP;END=68955000;SVLEN=31845000;BINS=1487;SCORE=2;LOG2CNT=0.99 GT 0/1
2 SVTYPE=DUP;END=122010000;SVLEN=31845000;BINS=2012;SCORE=2;LOG2CNT=0.99 GT 0/1
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpjBlX7H/QDNAseq-LGG150680a143ff61d.segments.vcf&#x27;
  V1       V2 V3    V4    V5   V6   V7
1  7 37110001  . &lt;DIP&gt; &lt;DUP&gt; 1000 PASS
2  7 90165001  . &lt;DIP&gt; &lt;DUP&gt; 1000 PASS
                                                                      V8 V9 V10
1  SVTYPE=DUP;END=68955000;SVLEN=31845000;BINS=1487;SCORE=2;LOG2CNT=0.99 GT 0/1
2 SVTYPE=DUP;END=122010000;SVLEN=31845000;BINS=2012;SCORE=2;LOG2CNT=0.99 GT 0/1
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpjBlX7H/QDNAseq-LGG150680a24c9c643.calls.vcf&#x27;
  V1       V2 V3    V4    V5   V6   V7
1  7 37110001  . &lt;DIP&gt; &lt;DUP&gt; 1000 PASS
2  7 90165001  . &lt;DIP&gt; &lt;DUP&gt; 1000 PASS
                                                                      V8 V9 V10
1  SVTYPE=DUP;END=68955000;SVLEN=31845000;BINS=1487;SCORE=2;LOG2CNT=0.99 GT 0/1
2 SVTYPE=DUP;END=122010000;SVLEN=31845000;BINS=2012;SCORE=2;LOG2CNT=0.99 GT 0/1
&gt; 
&gt; proc.time()
   user  system elapsed 
 23.654   0.469  24.110 
</PRE>
</DIV><P><SPAN class="filename">QDNAseq.Rcheck/tests/QDNAseq,parallel-reproducibility.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; ######################################################################
&gt; # This scripts asserts that for each processing step of QDNAseq
&gt; # the output/results are reproducible (numerically equal).
&gt; ######################################################################
&gt; library(&quot;QDNAseq&quot;)
&gt; options(&quot;QDNAseq::verbose&quot;=FALSE)
&gt; 
&gt; # Load data
&gt; data(LGG150)
&gt; data &lt;- LGG150
&gt; 
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # TRUTH
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # Filter out &quot;bad&quot; bins
&gt; dataF &lt;- applyFilters(data, residual=TRUE, blacklist=TRUE)
&gt; 
&gt; # Correct read counts as a function of GC content and mappability
&gt; dataC &lt;- correctBins(dataF)
&gt; 
&gt; # Normalize binned read counts to have diploid normal copy number
&gt; dataN &lt;- normalizeBins(dataC)
&gt; 
&gt; # Segment copy numbers
&gt; set.seed(42)  ## segmentBins() relies on RNG via DNAcopy::segment()
&gt; fit &lt;- segmentBins(dataN)
&gt; 
&gt; # Call copy-number segments
&gt; fitC &lt;- callBins(fit)
There were 50 or more warnings (use warnings() to see the first 50)
&gt; 
&gt; 
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # REPRODUCIBILITY
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; strategies &lt;- c(&quot;sequential&quot;, &quot;multisession&quot;)
&gt; if (parallelly::supportsMulticore()) strategies &lt;- c(strategies, &quot;multicore&quot;)
&gt; 
&gt; oplan &lt;- future::plan(&quot;list&quot;)
&gt; for (strategy in strategies) {
+   message(sprintf(&quot;Reproducibility with plan(\&quot;%s\&quot;) ...&quot;, strategy))
+   
+   future::plan(strategy)
+   
+   dataFr &lt;- applyFilters(data, residual=TRUE, blacklist=TRUE)
+   stopifnot(all.equal(dataFr, dataF))
+   
+   dataCr &lt;- correctBins(dataF)
+   stopifnot(all.equal(dataCr, dataC))
+   
+   dataNr &lt;- normalizeBins(dataC)
+   stopifnot(all.equal(dataNr, dataN))
+   
+   set.seed(42)  ## segmentBins() relies on RNG via DNAcopy::segment()
+   fitr &lt;- segmentBins(dataNr)
+   stopifnot(all.equal(fitr, fit))
+   
+   fitCr &lt;- callBins(fitr)
+   stopifnot(all.equal(fitCr, fitC))
+   
+   message(sprintf(&quot;Reproducibility with plan(\&quot;%s\&quot;) ... done&quot;, strategy))
+ }
Reproducibility with plan(&quot;sequential&quot;) ...
Reproducibility with plan(&quot;sequential&quot;) ... done
Reproducibility with plan(&quot;multisession&quot;) ...
    Calculating fit for sample LGG150 (1 of 1) ...
    Segmenting: LGG150 (1 of 1) ...
Reproducibility with plan(&quot;multisession&quot;) ... done
Reproducibility with plan(&quot;multicore&quot;) ...
Reproducibility with plan(&quot;multicore&quot;) ... done
There were 50 or more warnings (use warnings() to see the first 50)
&gt; 
&gt; future::plan(oplan)
&gt; 
&gt; proc.time()
   user  system elapsed 
 82.868   1.913  96.487 
</PRE>
</DIV><P><SPAN class="filename">QDNAseq.Rcheck/tests/QDNAseq.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(QDNAseq)
&gt; library(Biobase) ## combine(), sampleNames()
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

&gt; library(utils)
&gt; 
&gt; # Load data
&gt; data(LGG150)
&gt; data &lt;- LGG150
&gt; print(data)
QDNAseqReadCounts (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: counts 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads used.reads expected.variance
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; stopifnot(inherits(data, &quot;QDNAseqReadCounts&quot;))
&gt; 
&gt; # Plot isobars of read counts
&gt; isobarPlot(data)
Plotting sample LGG150 median read counts
&gt; 
&gt; # Plot copy number profile
&gt; plot(data, ylim=c(-100, 200))
Plotting sample LGG150 (1 of 1) ...
&gt; highlightFilters(data, residual=TRUE, blacklist=TRUE)
Highlighted 3,375 bins.
&gt; 
&gt; # Filter out &quot;bad&quot; bins
&gt; dataF &lt;- applyFilters(data, residual=TRUE, blacklist=TRUE)
38,819	total bins
38,819	of which in selected chromosomes
36,722	of which with reference sequence
33,347	final bins
&gt; print(dataF)
QDNAseqReadCounts (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: counts 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads used.reads expected.variance
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; plot(dataF, ylim=c(-100, 200))
Plotting sample LGG150 (1 of 1) ...
&gt; stopifnot(inherits(dataF, &quot;QDNAseqReadCounts&quot;))
&gt; 
&gt; # Correct read counts as a function of GC content and mappability
&gt; dataC &lt;- correctBins(dataF)
Calculating correction for GC content and mappability
    Calculating fit for sample LGG150 (1 of 1) ...
Done.
&gt; print(dataC)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: copynumber 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; plot(dataC, ylim=c(-100, 200))
Plotting sample LGG150 (1 of 1) ...
&gt; stopifnot(inherits(dataC, &quot;QDNAseqCopyNumbers&quot;))
&gt; 
&gt; # Normalize binned read counts to have diploid normal copy number
&gt; dataN &lt;- normalizeBins(dataC)
Applying median normalization ...
&gt; print(dataN)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: copynumber 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; plot(dataN)
Plotting sample LGG150 (1 of 1) ...
&gt; stopifnot(inherits(dataN, &quot;QDNAseqCopyNumbers&quot;))
&gt; 
&gt; # Plot noise
&gt; noisePlot(dataF)
Calculating correction for GC content and mappability
    Calculating fit for sample LGG150 (1 of 1) ...
Done.
&gt; 
&gt; # Segment copy numbers
&gt; fit &lt;- segmentBins(dataN)
Performing segmentation:
    Segmenting: LGG150 (1 of 1) ...
&gt; print(fit)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: copynumber, segmented 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; plot(fit)
Plotting sample LGG150 (1 of 1) ...
&gt; stopifnot(inherits(fit, &quot;QDNAseqCopyNumbers&quot;))
&gt; 
&gt; # Call copy-number segments
&gt; fitC &lt;- callBins(fit)
EM algorithm started ... 

[1] &quot;Total number of segments present in the data: 10&quot;
[1] &quot;Number of segments used for fitting the model: 9&quot;
40085068177492214.162.4744605114786712397.7112.9744605112255294397.793.6
Calling iteration1:

optim results
time: 1
minimum: 16445.6418287481
116443.1634295677-0.939948404909675-0.5351158716347230.04999922851584940.1357341814220170.2320391157632851.492692224927890.46336036623653-0.0007447487965531270.0111894143724591-0.000962583906213420.3145629342085070.798554843389167
40100118180943214.262.5744605114786712397.7112.9744605112255294397.793.6
Calling iteration2:

optim results
time: 0
minimum: 16445.5781374328
116443.1331012469-0.932876677515094-0.5359433038810010.0499979048415910.1114729434735370.1905642555481931.861013902305770.4314700360573740.0001440687684084760.01118427659401560.0007568920162792620.2061088124299370.874022351578009
40100118180943214.262.5744605114786712397.7112.9744605112255294397.793.6
Calling iteration3:

optim results
time: 0
minimum: 16445.5975371627
116443.1273697069-0.924012853099611-0.5365177469718980.04999923288882760.108573194240180.1856071014916831.753751992204390.4694950401327692.93313204149142e-050.0111878614439676-0.0001759851682342870.07360543573369820.943675664015282
EM algorithm done ...

Computing posterior probabilities for all segments ...

Total time:0minutes

Adjusting segmented data for cellularity ... 

Cellularity sample1: 1

Adjusting normalized data for cellularity ... 

Cellularity sample1: 1

1
40126798402177214.464.2744605114786712397.7112.9744605112255294397.793.6
40126988568923214.465.4744605114786712397.7112.9744605112255294397.793.6
40126958568917214.465.4744605114786712397.7112.9744605112255294397.793.6
40127238835932214.467.5744605117824054397.7136744605112255294397.793.6
40131508836887214.467.5744605117824054397.7136744605112255294397.793.6
40131598836891214.467.5744605117824054397.7136744605112255294397.793.6
40131688836895214.467.5744605117824054397.7136744605112255294397.793.6
40131778836899214.467.5744605117824054397.7136744605112255294397.793.6
40131868836903214.467.5744605117824054397.7136744605112255294397.793.6
40131928836906214.467.5744605117824054397.7136744605112255294397.793.6
40132118903636214.468744605117824054397.7136744605112255294397.793.6
40136288903959214.468744605117824054397.7136744605112255294397.793.6
FINISHED!

Total time:0minutes

There were 50 or more warnings (use warnings() to see the first 50)
&gt; print(fitC)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: calls, copynumber, probamp, probdloss, probgain, probloss, probnorm, segmented 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; plot(fitC)
Plotting sample LGG150 (1 of 1) ...
&gt; 
&gt; 
&gt; # ---------------------------------------------------------------
&gt; # Exporting
&gt; # ---------------------------------------------------------------
&gt; message(&quot;* exportBins() ...&quot;)
* exportBins() ...
&gt; 
&gt; sets &lt;- list(data = data, dataC = dataC, fit = fit, fitC = fitC)
&gt; for (name in names(sets)) {
+   set &lt;- sets[[name]]
+   formats &lt;- c(&quot;tsv&quot;, &quot;igv&quot;, &quot;bed&quot;)
+   if (name == &quot;fitC&quot;) formats &lt;- c(formats, &quot;vcf&quot;, &quot;seg&quot;)
+   for (format in formats) {
+     types &lt;- c(&quot;copynumber&quot;)
+     if (name %in% c(&quot;fit&quot;, &quot;fitC&quot;)) types &lt;- c(types, &quot;segments&quot;)
+     if (name == &quot;fitC&quot;) types &lt;- c(types, &quot;calls&quot;)
+     for (type in types) {
+       fileext &lt;- sprintf(&quot;.%s.%s&quot;, type, format)
+       templates &lt;- c(&quot;QDNAseq-%s.&quot;, &quot;QDNAseq-%i.&quot;, &quot;QDNAseq-%03i.&quot;)
+       if (ncol(set) == 1L || !(format %in% c(&quot;bed&quot;, &quot;seg&quot;, &quot;vcf&quot;))) {
+         templates &lt;- c(&quot;QDNAseq.&quot;, templates)
+       }
+       for (template in templates) {
+         file &lt;- tempfile(pattern = template, fileext = fileext)
+         message(sprintf(&quot;  - exportBins(&lt;%d samples&gt;, format=\&quot;%s\&quot;, type=\&quot;%s\&quot;, file=\&quot;%s\&quot;)&quot;, ncol(set), format, type, template))
+         file &lt;- exportBins(set, file = file, format = format, type = type)
+         message(sprintf(&quot;    File(s) written: [n=%d] %s&quot;,
+                 length(file), paste(sQuote(file), collapse = &quot;, &quot;)))
+         stopifnot(all(file_test(&quot;-f&quot;, file)))
+         file.remove(file)
+         stopifnot(!any(file_test(&quot;-f&quot;, file)))
+       }
+     }
+   }
+ }
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e2ebc9204.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e1f05a42e.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e2d6d8449.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e40d4b1f1.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e50b3da.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e5cba687f.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e46a400e7.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844eae83f93.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e5b98116b.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150.844e1682bf8f.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e2a14de94.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e558c428a.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e3c40f377.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e3fc6a15b.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844eb7c6861.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e1e36906d.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e6c7bd705.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e72f98eae.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e444f4ed4.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e18cf4ab0.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e3fdc8e62.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150.844e715cd3ad.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e325829f9.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e58b1f86c.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e45425b5.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e2643b45c.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e687e9024.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e13443ead.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e1922691a.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e38ed0077.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e5d144042.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e47defb1e.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e57f2a4a6.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844ea81c48b.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e8b3ad0f.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e58435880.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e673c2d0a.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e4f57adf6.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e632b9813.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e42d43e76.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e65da6d85.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150.844ed4076a7.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e18608100.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e221b60fd.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e4d071802.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150.844e23dce961.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e4051f16a.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e3982ef07.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e16d6780f.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e4a1403e.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e525239b8.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e56b30671.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e75fe13eb.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e4aa63b1.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e2f64fedd.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e7a5239a0.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e2aee180d.calls.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e17e38f01.calls.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844ed96784e.calls.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e44108127.calls.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e50d08f78.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e6aaab890.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844ebef7c45.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e28c3341e.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e752c7d1b.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e14a32955.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e1068c9e.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e5c68aa26.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e63fad74b.calls.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e643224b1.calls.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e1f3ce89c.calls.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e49d544d1.calls.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e71729b58.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150.844e379d699c.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e6bf0a5ce.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e3e79b35b.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e5b7a52fd.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150.844e2c429738.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e77fca262.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e7250cb0c.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e30e3d776.calls.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150.844e4a4edc1a.calls.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e4903d17d.calls.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e26e1eb61.calls.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e4ef93fcb.copynumber.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150.844e7868d05a.copynumber.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e21342502.copynumber.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e79e757d8.copynumber.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e104c5f5b.segments.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150.844e2eca9d50.segments.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e3df7d8ff.segments.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e611ceed3.segments.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e197555e0.calls.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150.844e49e75545.calls.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e9e022f2.calls.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844eea1d2fb.calls.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e5e8a7e9a.copynumber.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150.844eae6af90.copynumber.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e6b0a7d21.copynumber.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e428555e5.copynumber.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e6f18d442.segments.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150.844ea4765bd.segments.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844ec5a9ab6.segments.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e608b6f9a.segments.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e41e4cf59.calls.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150.844e784b4084.calls.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e1f0522f5.calls.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e1d5f2256.calls.seg&#x27;
&gt; 
&gt; sets &lt;- list(data = data, dataC = dataC, fit = fit, fitC = fitC)
&gt; sets &lt;- lapply(sets, FUN = function(set) {
+   stopifnot(ncol(set) == 1L)
+   name &lt;- sampleNames(set)
+   setA &lt;- set
+   sampleNames(setA) &lt;- sprintf(&quot;%sa&quot;, name)
+   setB &lt;- set
+   sampleNames(setB) &lt;- sprintf(&quot;%sb&quot;, name)
+   combine(setA, setB)
+ })
&gt; 
&gt; for (name in names(sets)) {
+   set &lt;- sets[[name]]
+   stopifnot(ncol(set) == 2L)
+   formats &lt;- c(&quot;tsv&quot;, &quot;igv&quot;, &quot;bed&quot;)
+   if (name == &quot;fitC&quot;) formats &lt;- c(formats, &quot;vcf&quot;, &quot;seg&quot;)
+   for (format in formats) {
+     types &lt;- c(&quot;copynumber&quot;)
+     if (name %in% c(&quot;fit&quot;, &quot;fitC&quot;)) types &lt;- c(types, &quot;segments&quot;)
+     if (name == &quot;fitC&quot;) types &lt;- c(types, &quot;calls&quot;)
+     for (type in types) {
+       fileext &lt;- sprintf(&quot;.%s.%s&quot;, type, format)
+       templates &lt;- c(&quot;QDNAseq-%s.&quot;, &quot;QDNAseq-%i.&quot;, &quot;QDNAseq-%03i.&quot;)
+       if (ncol(set) == 1L || !(format %in% c(&quot;bed&quot;, &quot;seg&quot;, &quot;vcf&quot;))) {
+         templates &lt;- c(&quot;QDNAseq.&quot;, templates)
+       }
+       for (template in templates) {
+         file &lt;- tempfile(pattern = template, fileext = fileext)
+         message(sprintf(&quot;  - exportBins(&lt;%d samples&gt;, format=\&quot;%s\&quot;, type=\&quot;%s\&quot;, file=\&quot;%s\&quot;)&quot;, ncol(set), format, type, template))
+         file &lt;- exportBins(set, file = file, format = format, type = type)
+         message(sprintf(&quot;    File(s) written: [n=%d] %s&quot;,
+                 length(file), paste(sQuote(file), collapse = &quot;, &quot;)))
+         stopifnot(all(file_test(&quot;-f&quot;, file)))
+         file.remove(file)
+         stopifnot(!any(file_test(&quot;-f&quot;, file)))
+       }
+     }
+   }
+ }
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e248dd7bc.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e1701c558.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844efafed62.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e5571af32.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e6150a172.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e58b3bedf.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e7c539a94.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e3049e13e.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150a.844e511c8f39.copynumber.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150b.844e511c8f39.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e1d87bf96.copynumber.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-2.844e1d87bf96.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e2a313916.copynumber.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-002.844e2a313916.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e6168ee94.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e4c525ce6.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e68291216.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e4285dd68.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e65c7b2c6.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e3210675b.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e4c66005a.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e746985c1.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150a.844e109ae5f5.copynumber.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150b.844e109ae5f5.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e574cafea.copynumber.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-2.844e574cafea.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e5f7402e3.copynumber.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-002.844e5f7402e3.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e53203bda.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e4665842c.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e69bb68a0.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e5f7ad691.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e26f0f3c7.segments.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e2ba037fa.segments.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e57c61715.segments.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e45f616bc.segments.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e48ff5a50.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e7c53eed2.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e5cf7dc14.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e58af47b3.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e51c59e04.segments.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e3e487d87.segments.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e31630692.segments.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e4e193898.segments.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150a.844e6e925ec5.copynumber.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150b.844e6e925ec5.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e27f95cc.copynumber.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-2.844e27f95cc.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e6ba0f82e.copynumber.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-002.844e6ba0f82e.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150a.844e18c397db.segments.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150b.844e18c397db.segments.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e63e88460.segments.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-2.844e63e88460.segments.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e37f35514.segments.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-002.844e37f35514.segments.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844eeca9f1.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e266e61c8.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e1dbb07da.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e32fd114c.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e72d46222.segments.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e12248d9c.segments.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e4397f741.segments.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e4a21120d.segments.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e7198907f.calls.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e16b8331c.calls.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e10869639.calls.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e5b53f91f.calls.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e763309ad.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e37778a00.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e6f43119.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e4df920c2.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e7d6da0bd.segments.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e4ff38b6a.segments.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e4a4d0f94.segments.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e5a657cd1.segments.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq.844e28a2d31d.calls.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%s.844e1c12ad99.calls.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%i.844e18adfa58.calls.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpnzNV7l/QDNAseq-%03i.844e5a05d9af.calls.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150a.844e6a2be631.copynumber.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150b.844e6a2be631.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e740591d.copynumber.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-2.844e740591d.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e5c856f7b.copynumber.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-002.844e5c856f7b.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150a.844e55ccde60.segments.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150b.844e55ccde60.segments.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e2003f0f9.segments.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-2.844e2003f0f9.segments.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e406df3dc.segments.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-002.844e406df3dc.segments.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150a.844edc03374.calls.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150b.844edc03374.calls.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e20f09aea.calls.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-2.844e20f09aea.calls.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e66dc55a4.calls.bed&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-002.844e66dc55a4.calls.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;vcf&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150a.844e2b7b3b4f.copynumber.vcf&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150b.844e2b7b3b4f.copynumber.vcf&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;vcf&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e53edac37.copynumber.vcf&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-2.844e53edac37.copynumber.vcf&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;vcf&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e59b0b7c7.copynumber.vcf&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-002.844e59b0b7c7.copynumber.vcf&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;vcf&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150a.844e3d9fc8eb.segments.vcf&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150b.844e3d9fc8eb.segments.vcf&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;vcf&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e1785a378.segments.vcf&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-2.844e1785a378.segments.vcf&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;vcf&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e23d1c9d4.segments.vcf&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-002.844e23d1c9d4.segments.vcf&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;vcf&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150a.844e2f38596a.calls.vcf&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150b.844e2f38596a.calls.vcf&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;vcf&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e2e3dd694.calls.vcf&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-2.844e2e3dd694.calls.vcf&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;vcf&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e3458600d.calls.vcf&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-002.844e3458600d.calls.vcf&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;seg&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150a.844ea8c5289.copynumber.seg&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150b.844ea8c5289.copynumber.seg&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;seg&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e2470e041.copynumber.seg&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-2.844e2470e041.copynumber.seg&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;seg&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e6bcfea0e.copynumber.seg&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-002.844e6bcfea0e.copynumber.seg&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;seg&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150a.844e118083a3.segments.seg&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150b.844e118083a3.segments.seg&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;seg&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e726a0104.segments.seg&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-2.844e726a0104.segments.seg&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;seg&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e693d8acb.segments.seg&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-002.844e693d8acb.segments.seg&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;seg&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150a.844e61740f0d.calls.seg&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-LGG150b.844e61740f0d.calls.seg&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;seg&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-1.844e3cb71098.calls.seg&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-2.844e3cb71098.calls.seg&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;seg&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/RtmpnzNV7l/QDNAseq-001.844e43a3079c.calls.seg&#x27;, &#x27;/tmp/RtmpnzNV7l/QDNAseq-002.844e43a3079c.calls.seg&#x27;
&gt; 
&gt; message(&quot;* exportBins() ... done&quot;)
* exportBins() ... done
&gt; 
&gt; proc.time()
   user  system elapsed 
 79.031   1.268  80.293 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">QDNAseq.Rcheck/QDNAseq-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>addPhenodata</TD><TD style="text-align: right;">0.129</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.129</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>applyFilters</TD><TD style="text-align: right;">0.658</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.674</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>binReadCounts</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>callBins</TD><TD style="text-align: right;">22.805</TD><TD style="text-align: right;"> 0.200</TD><TD style="text-align: right;">23.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>compareToReference</TD><TD style="text-align: right;">2.740</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">2.769</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>correctBins</TD><TD style="text-align: right;">2.424</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">2.444</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>createBins</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>estimateCorrection</TD><TD style="text-align: right;">2.829</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">2.842</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>exportBins</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>frequencyPlot</TD><TD style="text-align: right;">22.694</TD><TD style="text-align: right;"> 0.084</TD><TD style="text-align: right;">22.778</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getBinAnnotations</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>highlightFilters</TD><TD style="text-align: right;">0.722</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">0.746</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>isobarPlot</TD><TD style="text-align: right;">1.179</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">1.206</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeCgh</TD><TD style="text-align: right;">2.600</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">2.616</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>noisePlot</TD><TD style="text-align: right;">2.847</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">2.859</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>normalizeBins</TD><TD style="text-align: right;">2.476</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">2.484</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>normalizeSegmentedBins</TD><TD style="text-align: right;">9.425</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">9.478</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot</TD><TD style="text-align: right;">3.156</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">3.180</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>poolRuns</TD><TD style="text-align: right;">0.146</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.146</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>segmentBins</TD><TD style="text-align: right;">9.210</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">9.246</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>smoothOutlierBins</TD><TD style="text-align: right;">2.542</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">2.554</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
