<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for ChIPQC on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><B>[<A href="../index.html#C">C</A>]</B></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-09 02:50:12 -0000 (Thu, 09 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4165</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for ChIPQC on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the ChIPQC package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/ChIPQC.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>309</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">ChIPQC</A>&nbsp;1.35.1</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/ChIPQC">landing page</A>)</SPAN><BR>Tom Carroll <tc.infomatics@gmail.com>, Rory Stark<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-08&nbsp;03:21:00&nbsp;-0000&nbsp;(Wed,&nbsp;08&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/ChIPQC</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">0aa2fcd</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-01-10&nbsp;13:07:36&nbsp;-0000&nbsp;(Tue,&nbsp;10&nbsp;Jan&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: ChIPQC</TD></TR>
<TR><TD><B>Version</B>: 1.35.1</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:ChIPQC.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings ChIPQC_1.35.1.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-08 18:28:37 -0000 (Wed, 08 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-08 18:37:54 -0000 (Wed, 08 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 556.9 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: ChIPQC.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 1</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:ChIPQC.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings ChIPQC_1.35.1.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/ChIPQC.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘ChIPQC/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘ChIPQC’ version ‘1.35.1’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘ChIPQC’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
Package in Depends field not imported from: ‘BiocParallel’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.
Unexported object imported by a &#x27;:::&#x27; call: ‘S4Vectors:::tabulate2’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... NOTE
Foreign function call to a different package:
  .Call(&quot;rle_sum_any&quot;, ..., PACKAGE = &quot;chipseq&quot;)
See chapter ‘System and foreign language interfaces’ in the ‘Writing R
Extensions’ manual.
* checking R code for possible problems ... NOTE
ChIPQC: no visible global function definition for ‘is’
ChIPQC: no visible global function definition for ‘multicoreWorkers’
ChIPQC: no visible global function definition for ‘snowWorkers’
GetGRanges: no visible global function definition for ‘seqlevels&lt;-’
findCovMaxPos: no visible global function definition for ‘seqlengths’
findCovMaxPos: no visible global function definition for ‘seqlengths&lt;-’
getAnnotation: no visible binding for global variable
  ‘TxDb.Hsapiens.UCSC.hg38.knownGene’
getAnnotation: no visible global function definition for ‘seqlengths’
makeCCplot: no visible binding for global variable ‘Shift_Size’
makeCCplot: no visible binding for global variable ‘CC_Score’
makeCoveragePlot: no visible binding for global variable ‘Depth’
makeCoveragePlot: no visible binding for global variable ‘log10_bp’
makeCoveragePlot: no visible binding for global variable ‘Sample’
makeFriblPlot: no visible binding for global variable ‘Sample’
makeFriblPlot: no visible binding for global variable ‘FRIBL’
makeFriblPlot: no visible binding for global variable ‘Reads’
makeFripPlot: no visible binding for global variable ‘Sample’
makeFripPlot: no visible binding for global variable ‘FRIP’
makeFripPlot: no visible binding for global variable ‘Reads’
makePeakProfilePlot: no visible binding for global variable ‘Distance’
makePeakProfilePlot: no visible binding for global variable ‘Signal’
makeRapPlot: no visible binding for global variable ‘Sample’
makeRapPlot: no visible binding for global variable ‘CountsInPeaks’
makeRegiPlot: no visible binding for global variable ‘Sample’
makeRegiPlot: no visible binding for global variable ‘GenomicIntervals’
makeRegiPlot: no visible binding for global variable ‘log2_Enrichment’
makeSSDPlot: no visible binding for global variable ‘Sample’
makeSSDPlot: no visible binding for global variable ‘SSD’
makeSSDPlot: no visible global function definition for ‘geom_point’
sampleQC: no visible global function definition for ‘seqlevels&lt;-’
plotCC,ChIPQCexperiment: no visible binding for global variable
  ‘Sample’
plotCC,list: no visible binding for global variable ‘Sample’
plotPeakProfile,ChIPQCexperiment: no visible binding for global
  variable ‘Sample’
plotPeakProfile,list: no visible binding for global variable ‘Sample’
Undefined global functions or variables:
  CC_Score CountsInPeaks Depth Distance FRIBL FRIP GenomicIntervals
  Reads SSD Sample Shift_Size Signal TxDb.Hsapiens.UCSC.hg38.knownGene
  geom_point is log10_bp log2_Enrichment multicoreWorkers seqlengths
  seqlengths&lt;- seqlevels&lt;- snowWorkers
Consider adding
  importFrom(&quot;methods&quot;, &quot;is&quot;)
  importFrom(&quot;stats&quot;, &quot;SSD&quot;)
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains &#x27;methods&#x27;).
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... WARNING
  Error loading dataset &#x27;exampleExp&#x27;:
   Error in .requirePackage(package) : 
    unable to find required package &#x27;ChIPQC&#x27;
  
  Error loading dataset &#x27;tamoxifen&#x27;:
   Error in .requirePackage(package) : 
    unable to find required package &#x27;ChIPQC&#x27;
  
  The dataset(s) may use package(s) not declared in the DESCRIPTION file.
* checking data for ASCII and uncompressed saves ... OK
* checking installed files from ‘inst/doc’ ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
               user system elapsed
ChIPQCreport 29.309  0.478  29.891
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘ChIPQC.Rnw’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 WARNING, 3 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/ChIPQC.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">ChIPQC.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL ChIPQC
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘ChIPQC’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (ChIPQC)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<HR>
<H3>Example timings</H3>
<P><SPAN class="filename">ChIPQC.Rcheck/ChIPQC-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>ChIPQC-data</TD><TD style="text-align: right;">2.734</TD><TD style="text-align: right;">0.065</TD><TD style="text-align: right;">2.799</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ChIPQC</TD><TD style="text-align: right;">0.491</TD><TD style="text-align: right;">0.035</TD><TD style="text-align: right;">0.527</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ChIPQCexperiment-class</TD><TD style="text-align: right;">0.499</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">0.543</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ChIPQCreport</TD><TD style="text-align: right;">29.309</TD><TD style="text-align: right;"> 0.478</TD><TD style="text-align: right;">29.891</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ChIPQCsample-class</TD><TD style="text-align: right;">1.363</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">1.387</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>FragmentLengthCrossCoverage-methods</TD><TD style="text-align: right;">0.071</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.071</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Normalisedaveragepeaksignal-methods</TD><TD style="text-align: right;">0.018</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.025</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>QCannotation-methods</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>QCcontrol-methods</TD><TD style="text-align: right;">0.225</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.225</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>QCdba-methods</TD><TD style="text-align: right;">0.122</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.122</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>QCmetadata-methods</TD><TD style="text-align: right;">0.111</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.125</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>QCmetrics-methods</TD><TD style="text-align: right;">0.520</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.521</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>QCsample-methods</TD><TD style="text-align: right;">0.065</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.065</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ReadLengthCrossCoverage-methods</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>RelativeCrossCoverage-methods</TD><TD style="text-align: right;">0.073</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.073</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>averagepeaksignal-methods</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.027</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>coveragehistogram-methods</TD><TD style="text-align: right;">0.027</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.027</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>crosscoverage-methods</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>duplicateRate-methods</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>duplicates-methods</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>flagtagcounts-methods</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fragmentlength-methods</TD><TD style="text-align: right;">0.1</TD><TD style="text-align: right;">0.0</TD><TD style="text-align: right;">0.1</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>frip-methods</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mapped-methods</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>peaks-methods</TD><TD style="text-align: right;">0.195</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.203</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotCC-methods</TD><TD style="text-align: right;">1.417</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">1.468</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotCorHeatmap-methods</TD><TD style="text-align: right;">0.544</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.552</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotCoverageHist-methods</TD><TD style="text-align: right;">0.657</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.657</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotFribl-methods</TD><TD style="text-align: right;">0.768</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.769</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotFrip-methods</TD><TD style="text-align: right;">0.722</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.726</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotPeakProfile-methods</TD><TD style="text-align: right;">1.758</TD><TD style="text-align: right;">0.057</TD><TD style="text-align: right;">1.816</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotPrincomp-methods</TD><TD style="text-align: right;">0.269</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.271</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotRap-methods</TD><TD style="text-align: right;">1.073</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.073</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotRegi-methods</TD><TD style="text-align: right;">1.372</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.372</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotSSD-methods</TD><TD style="text-align: right;">1.750</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">1.754</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readlength-methods</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>reads-methods</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>regi-methods</TD><TD style="text-align: right;">0.053</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.055</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ribl-methods</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rip-methods</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ssd-methods</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
