<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for PureCN on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><B>[<A href="../index.html#P">P</A>]</B></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-03 02:36:16 -0000 (Fri, 03 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4039</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for PureCN on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the PureCN package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/PureCN.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1540</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">PureCN</A>&nbsp;2.5.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/PureCN">landing page</A>)</SPAN><BR>Markus Riester<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-01&nbsp;03:13:00&nbsp;-0000&nbsp;(Wed,&nbsp;01&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/PureCN</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">7d7d59b</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:13:36&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: PureCN</TD></TR>
<TR><TD><B>Version</B>: 2.5.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:PureCN.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings PureCN_2.5.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-02 16:43:22 -0000 (Thu, 02 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-02 17:11:19 -0000 (Thu, 02 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 1677.0 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: PureCN.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:PureCN.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings PureCN_2.5.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/PureCN.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘PureCN/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘PureCN’ version ‘2.5.0’
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Package which this enhances but not available for checking: ‘genomicsdb’
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘PureCN’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                                user system elapsed
callAmplificationsInLowPurity 64.479  0.020  64.658
segmentationPSCBS             42.919  0.055  43.356
filterIntervals               30.180  0.040  30.247
runAbsoluteCN                 26.503  0.021  26.531
segmentationHclust            24.585  0.012  24.824
findFocal                     16.373  0.008  16.396
segmentationCBS               12.753  0.008  12.927
annotateTargets                9.751  0.251  10.031
processMultipleSamples         7.093  0.016   7.116
calculateTangentNormal         5.933  0.012   5.953
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘Quick.Rmd’ using ‘UTF-8’... OK
  ‘PureCN.Rnw’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 NOTE
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/PureCN.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">PureCN.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL PureCN
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘PureCN’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (PureCN)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">PureCN.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(PureCN)
Loading required package: DNAcopy
Loading required package: VariantAnnotation
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: MatrixGenerics
Loading required package: matrixStats

Attaching package: &#x27;MatrixGenerics&#x27;

The following objects are masked from &#x27;package:matrixStats&#x27;:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars

Loading required package: GenomeInfoDb
Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomicRanges
Loading required package: SummarizedExperiment
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


Attaching package: &#x27;Biobase&#x27;

The following object is masked from &#x27;package:MatrixGenerics&#x27;:

    rowMedians

The following objects are masked from &#x27;package:matrixStats&#x27;:

    anyMissing, rowMedians

Loading required package: Rsamtools
Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit


Attaching package: &#x27;VariantAnnotation&#x27;

The following object is masked from &#x27;package:base&#x27;:

    tabulate

&gt; 
&gt; test_check(&quot;PureCN&quot;)
WARN [2023-02-02 16:54:34] Attempted adding gene symbols to intervals. Heuristics have been used to pick symbols for overlapping genes.
WARN [2023-02-02 16:54:34] Chromosome naming style of txdb file (UCSC) was different from interval file (Ensembl).
WARN [2023-02-02 16:54:41] Attempted adding gene symbols to intervals. Heuristics have been used to pick symbols for overlapping genes.
FATAL [2023-02-02 16:54:45] tumor.coverage.file and interval.file do not align. 

FATAL [2023-02-02 16:54:45]  

FATAL [2023-02-02 16:54:45] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:54:45] parameters (PureCN 2.5.0). 

WARN [2023-02-02 16:54:45] Cannot find all contig lengths while exporting interval file.
INFO [2023-02-02 16:54:47] Processing seq1:1-21 (1/3)...
INFO [2023-02-02 16:54:47] Processing seq1:1227-1247 (2/3)...
INFO [2023-02-02 16:54:48] Processing seq2:594-614 (3/3)...
WARN [2023-02-02 16:54:49] Large potential mis-calibration of on- and off-target log2 ratios: 0.26
FATAL [2023-02-02 16:54:49] Need either f or purity and ploidy. 

FATAL [2023-02-02 16:54:49]  

FATAL [2023-02-02 16:54:49] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:54:49] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 16:54:49] f not in expected range. 

FATAL [2023-02-02 16:54:49]  

FATAL [2023-02-02 16:54:49] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:54:49] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 16:54:49] coverage not in expected range (&gt;=2) 

FATAL [2023-02-02 16:54:49]  

FATAL [2023-02-02 16:54:49] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:54:49] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 16:54:49] purity not in expected range. 

FATAL [2023-02-02 16:54:49]  

FATAL [2023-02-02 16:54:49] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:54:49] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 16:54:49] ploidy not in expected range. 

FATAL [2023-02-02 16:54:49]  

FATAL [2023-02-02 16:54:49] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:54:49] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 16:54:49] cell.fraction not in expected range. 

FATAL [2023-02-02 16:54:49]  

FATAL [2023-02-02 16:54:49] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:54:49] parameters (PureCN 2.5.0). 

INFO [2023-02-02 16:54:53] 576 on-target bins with low coverage in all samples.
WARN [2023-02-02 16:54:53] You are likely not using the correct baits file!
WARN [2023-02-02 16:54:53] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 16:54:53] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 16:54:53] Processing on-target regions...
INFO [2023-02-02 16:54:54] Removing 930 intervals with low coverage in normalDB.
INFO [2023-02-02 16:54:54] Removing 1 intervals with zero coverage in more than 3% of normalDB.
INFO [2023-02-02 16:54:56] Tumor/normal noise ratio: 19.041
WARN [2023-02-02 16:54:56] Extensive noise in tumor compared to normals.
INFO [2023-02-02 16:56:06] Tumor/normal noise ratio: 19.041
WARN [2023-02-02 16:56:06] Extensive noise in tumor compared to normals.
INFO [2023-02-02 16:56:10] Using BiocParallel for parallel optimization.
FATAL [2023-02-02 16:56:37] pvalue.cutoff not within expected range or format. 

FATAL [2023-02-02 16:56:37]  

FATAL [2023-02-02 16:56:37] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:56:37] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 16:56:37] pvalue.cutoff not within expected range or format. 

FATAL [2023-02-02 16:56:37]  

FATAL [2023-02-02 16:56:37] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:56:37] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 16:56:37] percentile.cutoff not in expected range (0 to 100). 

FATAL [2023-02-02 16:56:37]  

FATAL [2023-02-02 16:56:37] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:56:37] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 16:56:37] percentile.cutoff not in expected range (0 to 100). 

FATAL [2023-02-02 16:56:37]  

FATAL [2023-02-02 16:56:37] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:56:37] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 16:56:37] purity not within expected range or format. 

FATAL [2023-02-02 16:56:37]  

FATAL [2023-02-02 16:56:37] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:56:37] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 16:56:37] purity not within expected range or format. 

FATAL [2023-02-02 16:56:37]  

FATAL [2023-02-02 16:56:37] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:56:37] parameters (PureCN 2.5.0). 

INFO [2023-02-02 16:56:39] ------------------------------------------------------------
INFO [2023-02-02 16:56:39] PureCN 2.5.0
INFO [2023-02-02 16:56:39] ------------------------------------------------------------
INFO [2023-02-02 16:56:39] Loading coverage files...
INFO [2023-02-02 16:56:39] Mean target coverages: 112X (tumor) 99X (normal).
WARN [2023-02-02 16:56:40] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 16:56:40] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 16:56:40] Removing 228 intervals with missing log.ratio.
INFO [2023-02-02 16:56:40] Removing 22 small (&lt; 5bp) intervals.
INFO [2023-02-02 16:56:40] Removing 19 intervals with low total coverage in normal (&lt; 150.00 reads).
WARN [2023-02-02 16:56:40] No normalDB provided. Provide one for better results.
INFO [2023-02-02 16:56:40] Removing 1724 low count (&lt; 100 total reads) intervals.
INFO [2023-02-02 16:56:40] Removing 6 low coverage (&lt; 15.0000X) intervals.
INFO [2023-02-02 16:56:40] Using 8050 intervals (8050 on-target, 0 off-target).
INFO [2023-02-02 16:56:40] No off-target intervals. If this is hybrid-capture data, consider adding them.
INFO [2023-02-02 16:56:40] No interval.file provided. Cannot check for any GC-biases.
INFO [2023-02-02 16:56:40] Loading VCF...
INFO [2023-02-02 16:56:40] Found 127 variants in VCF file.
INFO [2023-02-02 16:56:40] 127 (100.0%) variants annotated as likely germline (DB INFO flag).
INFO [2023-02-02 16:56:40] LIB-02240e4 is tumor in VCF file.
INFO [2023-02-02 16:56:40] No homozygous variants in VCF, provide unfiltered VCF.
INFO [2023-02-02 16:56:40] Removing 1 non heterozygous (in matched normal) germline SNPs.
INFO [2023-02-02 16:56:40] Removing 0 low quality variants with non-offset BQ &lt; 25.
INFO [2023-02-02 16:56:40] Base quality scores range from 29 to 35 (offset by 1)
INFO [2023-02-02 16:56:41] Minimum number of supporting reads ranges from 2 to 5, depending on coverage and BQS.
INFO [2023-02-02 16:56:41] Removing 1 variants with AF &lt; 0.030 or AF &gt;= 1.000 or insufficient supporting reads or depth &lt; 15.
INFO [2023-02-02 16:56:41] Total size of targeted genomic region: 1.53Mb (2.33Mb with 50bp padding).
INFO [2023-02-02 16:56:41] 1.2% of targets contain variants.
INFO [2023-02-02 16:56:42] Removing 4 variants outside intervals.
INFO [2023-02-02 16:56:42] Found SOMATIC annotation in VCF.
INFO [2023-02-02 16:56:42] Setting somatic prior probabilities for somatic variants to 0.999000 or to 0.000100 otherwise.
INFO [2023-02-02 16:56:42] Found SOMATIC annotation in VCF. Setting mapping bias to 0.972.
INFO [2023-02-02 16:56:42] Excluding 0 novel or poor quality variants from segmentation.
INFO [2023-02-02 16:56:42] Sample sex: ?
INFO [2023-02-02 16:56:42] Segmenting data...
INFO [2023-02-02 16:56:42] Loading pre-computed boundaries for DNAcopy...
INFO [2023-02-02 16:56:42] Setting undo.SD parameter to 1.000000.
INFO [2023-02-02 16:56:43] Setting prune.hclust.h parameter to 0.200000.
INFO [2023-02-02 16:56:43] Found 59 segments with median size of 17.67Mb.
INFO [2023-02-02 16:56:44] Using 121 variants.
INFO [2023-02-02 16:56:44] Mean standard deviation of log-ratios: 0.34 (MAPD: 0.30)
INFO [2023-02-02 16:56:44] 2D-grid search of purity and ploidy...
INFO [2023-02-02 16:56:46] Local optima: 0.63/1.9, 0.5/2
INFO [2023-02-02 16:56:46] Testing local optimum 1/2 at purity 0.63 and total ploidy 1.90...
INFO [2023-02-02 16:56:47] Testing local optimum 2/2 at purity 0.50 and total ploidy 2.00...
INFO [2023-02-02 16:56:49] Skipping 1 solutions that converged to the same optima.
INFO [2023-02-02 16:56:49] Fitting variants with beta model for local optimum 1/2...
INFO [2023-02-02 16:56:49] Fitting variants for purity 0.65, tumor ploidy 1.75 and contamination 0.01.
INFO [2023-02-02 16:56:53] Optimized purity: 0.65
INFO [2023-02-02 16:56:53] Done.
INFO [2023-02-02 16:56:53] ------------------------------------------------------------
INFO [2023-02-02 16:56:54] Estimating callable regions.
FATAL [2023-02-02 16:56:56] exclude not a GRanges object. 

FATAL [2023-02-02 16:56:56]  

FATAL [2023-02-02 16:56:56] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:56:56] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 16:56:56] callable not a GRanges object. 

FATAL [2023-02-02 16:56:56]  

FATAL [2023-02-02 16:56:56] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:56:56] parameters (PureCN 2.5.0). 

WARN [2023-02-02 16:57:10] tumor.coverage.file and interval.file do not align.
INFO [2023-02-02 16:57:12] No Gene column in interval.file. You won&#x27;t get gene-level calls.
FATAL [2023-02-02 16:57:14] No gc_bias column in interval.file. 

FATAL [2023-02-02 16:57:14]  

FATAL [2023-02-02 16:57:14] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:57:14] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 16:57:14] Provided coverage is zero, most likely due to a corrupt BAM file. 

FATAL [2023-02-02 16:57:14]  

FATAL [2023-02-02 16:57:14] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:57:14] parameters (PureCN 2.5.0). 

INFO [2023-02-02 16:57:18] No reptiming column in interval.file.
INFO [2023-02-02 16:57:22] Reading /tmp/RtmpBApYiI/file3c1f3a79859a8d.rds...
INFO [2023-02-02 16:57:23] Reading /tmp/RtmpBApYiI/file3c1f3a79859a8d.rds...
INFO [2023-02-02 16:57:23] Reading /tmp/RtmpBApYiI/file3c1f3a79859a8d.rds...
INFO [2023-02-02 16:57:23] Reading /tmp/RtmpBApYiI/file3c1f3a79859a8d.rds...
INFO [2023-02-02 16:57:23] Reading /tmp/RtmpBApYiI/file3c1f3a79859a8d.rds...
INFO [2023-02-02 16:57:23] Reading /tmp/RtmpBApYiI/file3c1f3a79859a8d.rds...
FATAL [2023-02-02 16:57:23] Purity or Ploidy not numeric or in expected range. 

FATAL [2023-02-02 16:57:23]  

FATAL [2023-02-02 16:57:23] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:57:23] parameters (PureCN 2.5.0). 

INFO [2023-02-02 16:57:23] Reading /tmp/RtmpBApYiI/file3c1f3a79859a8d.rds...
INFO [2023-02-02 16:57:23] Reading /tmp/RtmpBApYiI/file3c1f3a79859a8d.rds...
FATAL [2023-02-02 16:57:24] &#x27;Failed&#x27; column in /tmp/RtmpBApYiI/file3c1f3a79859a8d.csv not 

FATAL [2023-02-02 16:57:24] logical(1). 

FATAL [2023-02-02 16:57:24]  

FATAL [2023-02-02 16:57:24] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:57:24] parameters (PureCN 2.5.0). 

INFO [2023-02-02 16:57:25] 576 on-target bins with low coverage in all samples.
WARN [2023-02-02 16:57:25] You are likely not using the correct baits file!
WARN [2023-02-02 16:57:25] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 16:57:25] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 16:57:25] Processing on-target regions...
INFO [2023-02-02 16:57:25] Removing 930 intervals with low coverage in normalDB.
INFO [2023-02-02 16:57:25] Removing 1 intervals with zero coverage in more than 3% of normalDB.
INFO [2023-02-02 16:57:30] 576 on-target bins with low coverage in all samples.
WARN [2023-02-02 16:57:31] You are likely not using the correct baits file!
WARN [2023-02-02 16:57:31] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 16:57:31] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 16:57:31] Processing on-target regions...
INFO [2023-02-02 16:57:31] Removing 930 intervals with low coverage in normalDB.
INFO [2023-02-02 16:57:31] Removing 1 intervals with zero coverage in more than 3% of normalDB.
INFO [2023-02-02 16:57:35] 576 on-target bins with low coverage in all samples.
WARN [2023-02-02 16:57:35] You are likely not using the correct baits file!
WARN [2023-02-02 16:57:35] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 16:57:35] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 16:57:35] Processing on-target regions...
INFO [2023-02-02 16:57:36] Removing 930 intervals with low coverage in normalDB.
INFO [2023-02-02 16:57:36] Removing 1 intervals with zero coverage in more than 3% of normalDB.
WARN [2023-02-02 16:57:37] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 16:57:37] Sample sex: NA
WARN [2023-02-02 16:57:38] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 16:57:38] Sample sex: NA
INFO [2023-02-02 16:57:39] 576 on-target bins with low coverage in all samples.
WARN [2023-02-02 16:57:39] You are likely not using the correct baits file!
WARN [2023-02-02 16:57:39] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 16:57:39] Allosome coverage missing, cannot determine sex.
FATAL [2023-02-02 16:57:39] Length of normal.coverage.files and sex different 

FATAL [2023-02-02 16:57:39]  

FATAL [2023-02-02 16:57:39] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:57:39] parameters (PureCN 2.5.0). 

WARN [2023-02-02 16:57:46] Target intervals were not sorted.
INFO [2023-02-02 16:57:46] 560 on-target bins with low coverage in all samples.
WARN [2023-02-02 16:57:46] You are likely not using the correct baits file!
WARN [2023-02-02 16:57:46] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 16:57:46] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 16:57:46] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 16:57:46] Processing on-target regions...
INFO [2023-02-02 16:57:47] Removing 978 intervals with low coverage in normalDB.
INFO [2023-02-02 16:57:47] Removing 11 intervals with zero coverage in more than 3% of normalDB.
FATAL [2023-02-02 16:57:50] tumor.coverage.file and normalDB do not align. 

FATAL [2023-02-02 16:57:50]  

FATAL [2023-02-02 16:57:50] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:57:50] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 16:57:50] At least 2 normal.coverage.files required. 

FATAL [2023-02-02 16:57:50]  

FATAL [2023-02-02 16:57:50] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:57:50] parameters (PureCN 2.5.0). 

INFO [2023-02-02 16:57:55] ------------------------------------------------------------
INFO [2023-02-02 16:57:55] PureCN 2.5.0
INFO [2023-02-02 16:57:55] ------------------------------------------------------------
INFO [2023-02-02 16:57:55] Loading coverage files...
INFO [2023-02-02 16:57:55] Mean target coverages: 112X (tumor) 99X (normal).
WARN [2023-02-02 16:57:56] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 16:57:56] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 16:57:56] Removing 228 intervals with missing log.ratio.
FATAL [2023-02-02 16:57:56] normalDB incompatible with this PureCN version. Please re-run 

FATAL [2023-02-02 16:57:56] NormalDB.R. 

FATAL [2023-02-02 16:57:56]  

FATAL [2023-02-02 16:57:56] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:57:56] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 16:57:56] normalDB incompatible with this PureCN version. Please re-run 

FATAL [2023-02-02 16:57:56] NormalDB.R. 

FATAL [2023-02-02 16:57:56]  

FATAL [2023-02-02 16:57:56] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:57:56] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 16:57:56] normal.coverage.files with _coverage.txt and _loess.txt suffix 

FATAL [2023-02-02 16:57:56] provided. Provide either only GC-normalized or raw coverage files! 

FATAL [2023-02-02 16:57:56]  

FATAL [2023-02-02 16:57:56] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:57:56] parameters (PureCN 2.5.0). 

INFO [2023-02-02 16:57:56] Removing 22 non heterozygous (in matched normal) germline SNPs.
INFO [2023-02-02 16:57:56] Removing 19 low quality variants with non-offset BQ &lt; 25.
INFO [2023-02-02 16:57:57] Base quality scores range from 24 to 35 (offset by 1)
INFO [2023-02-02 16:57:57] Minimum number of supporting reads ranges from 2 to 9, depending on coverage and BQS.
INFO [2023-02-02 16:57:57] Initial testing for significant sample cross-contamination: unlikely
INFO [2023-02-02 16:57:57] Removing 45 variants with AF &lt; 0.030 or AF &gt;= 1.000 or insufficient supporting reads or depth &lt; 15.
INFO [2023-02-02 16:57:58] Removing 22 non heterozygous (in matched normal) germline SNPs.
INFO [2023-02-02 16:57:58] Removing 19 low quality variants with non-offset BQ &lt; 25.
INFO [2023-02-02 16:57:58] Base quality scores range from 24 to 35 (offset by 1)
INFO [2023-02-02 16:57:58] Minimum number of supporting reads ranges from 2 to 9, depending on coverage and BQS.
INFO [2023-02-02 16:57:58] Initial testing for significant sample cross-contamination: unlikely
INFO [2023-02-02 16:57:59] Removing 45 variants with AF &lt; 0.030 or AF &gt;= 1.000 or insufficient supporting reads or depth &lt; 15.
INFO [2023-02-02 16:57:59] Removing 6 blacklisted variants.
INFO [2023-02-02 16:57:59] Removing 22 non heterozygous (in matched normal) germline SNPs.
INFO [2023-02-02 16:57:59] Removing 19 low quality variants with non-offset BQ &lt; 25.
INFO [2023-02-02 16:57:59] Base quality scores range from 24 to 35 (offset by 1)
INFO [2023-02-02 16:57:59] Minimum number of supporting reads ranges from 2 to 9, depending on coverage and BQS.
INFO [2023-02-02 16:58:00] Initial testing for significant sample cross-contamination: unlikely
INFO [2023-02-02 16:58:00] Removing 45 variants with AF &lt; 0.030 or AF &gt;= 1.000 or insufficient supporting reads or depth &lt; 15.
WARN [2023-02-02 16:58:00] MuTect stats file lacks contig and position columns.
INFO [2023-02-02 16:58:00] Removing 22 non heterozygous (in matched normal) germline SNPs.
INFO [2023-02-02 16:58:00] Removing 19 low quality variants with non-offset BQ &lt; 25.
INFO [2023-02-02 16:58:00] Base quality scores range from 24 to 35 (offset by 1)
INFO [2023-02-02 16:58:00] Minimum number of supporting reads ranges from 2 to 9, depending on coverage and BQS.
INFO [2023-02-02 16:58:01] Initial testing for significant sample cross-contamination: unlikely
INFO [2023-02-02 16:58:01] Removing 45 variants with AF &lt; 0.030 or AF &gt;= 1.000 or insufficient supporting reads or depth &lt; 15.
WARN [2023-02-02 16:58:01] MuTect stats file and VCF file do not align perfectly. Will remove 2325 unmatched variants.
WARN [2023-02-02 16:58:01] MuTect stats file lacks failure_reasons column. Keeping all variants listed in stats file.
INFO [2023-02-02 16:58:01] Removing 0 non heterozygous (in matched normal) germline SNPs.
INFO [2023-02-02 16:58:01] Removing 0 low quality variants with non-offset BQ &lt; 25.
INFO [2023-02-02 16:58:01] Base quality scores range from 31 to 33 (offset by 1)
INFO [2023-02-02 16:58:01] Minimum number of supporting reads ranges from 3 to 4, depending on coverage and BQS.
INFO [2023-02-02 16:58:02] Removing 0 variants with AF &lt; 0.030 or AF &gt;= 1.000 or insufficient supporting reads or depth &lt; 15.
INFO [2023-02-02 16:58:02] Removing 22 non heterozygous (in matched normal) germline SNPs.
INFO [2023-02-02 16:58:02] Base quality scores range from 7 to 35 (offset by 1)
INFO [2023-02-02 16:58:02] Minimum number of supporting reads ranges from 1 to 41, depending on coverage and BQS.
INFO [2023-02-02 16:58:03] Initial testing for significant sample cross-contamination: unlikely
INFO [2023-02-02 16:58:03] Removing 49 variants with AF &lt; 0.030 or AF &gt;= 1.000 or insufficient supporting reads or depth &lt; 15.
INFO [2023-02-02 16:58:03] Removing 22 non heterozygous (in matched normal) germline SNPs.
INFO [2023-02-02 16:58:03] Base quality scores range from 7 to 35 (offset by 1)
INFO [2023-02-02 16:58:03] Minimum number of supporting reads ranges from 1 to 41, depending on coverage and BQS.
INFO [2023-02-02 16:58:04] Initial testing for significant sample cross-contamination: unlikely
INFO [2023-02-02 16:58:04] Removing 49 variants with AF &lt; 0.030 or AF &gt;= 1.000 or insufficient supporting reads or depth &lt; 15.
INFO [2023-02-02 16:58:04] Removing 22 non heterozygous (in matched normal) germline SNPs.
FATAL [2023-02-02 16:58:04] No variants passed filter BQ. 

FATAL [2023-02-02 16:58:04]  

FATAL [2023-02-02 16:58:04] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:58:04] parameters (PureCN 2.5.0). 

INFO [2023-02-02 16:58:05] Found 11 variants in VCF file.
WARN [2023-02-02 16:58:05] vcf.file has no DB info field for membership in germline databases. Found and used valid population allele frequency &gt; 0.001000 instead.
INFO [2023-02-02 16:58:05] 1 (9.1%) variants annotated as likely germline (DB INFO flag).
INFO [2023-02-02 16:58:08] Found 11 variants in VCF file.
WARN [2023-02-02 16:58:08] vcf.file has no DB info field for membership in germline databases. Found and used valid population allele frequency &gt; 0.001000 instead.
INFO [2023-02-02 16:58:08] 1 (9.1%) variants annotated as likely germline (DB INFO flag).
INFO [2023-02-02 16:58:09] Found 11 variants in VCF file.
WARN [2023-02-02 16:58:09] vcf.file has no DB or POP_AF info field for membership in germline databases. Guessing it based on available dbSNP ID.
INFO [2023-02-02 16:58:10] 1 (9.1%) variants annotated as likely germline (DB INFO flag).
INFO [2023-02-02 16:58:10] Found 1000 variants in VCF file.
INFO [2023-02-02 16:58:10] Removing 2 triallelic sites.
WARN [2023-02-02 16:58:10] Having trouble guessing SOMATIC status...
WARN [2023-02-02 16:58:11] DP FORMAT field contains NAs. Removing 44 variants.
INFO [2023-02-02 16:58:11] 954 (100.0%) variants annotated as likely germline (DB INFO flag).
INFO [2023-02-02 16:58:13] Found 12 variants in VCF file.
INFO [2023-02-02 16:58:13] Removing 1 triallelic sites.
WARN [2023-02-02 16:58:13] DP FORMAT field contains NAs. Removing 1 variants.
WARN [2023-02-02 16:58:13] vcf.file has no DB info field for membership in germline databases. Found and used somatic status instead.
INFO [2023-02-02 16:58:13] 8 (80.0%) variants annotated as likely germline (DB INFO flag).
WARN [2023-02-02 16:58:14] BQ FORMAT field contains NAs. Removing 1 variants.
INFO [2023-02-02 16:58:14] Found 2331 variants in VCF file.
INFO [2023-02-02 16:58:14] 2282 (97.9%) variants annotated as likely germline (DB INFO flag).
INFO [2023-02-02 16:58:15] Found 2331 variants in VCF file.
INFO [2023-02-02 16:58:15] 2282 (97.9%) variants annotated as likely germline (DB INFO flag).
WARN [2023-02-02 16:58:16] Did not find base quality scores, will use global error rate of 0.0010 instead.
INFO [2023-02-02 16:58:16] Removing 21 low quality variants with non-offset BQ &lt; 25.
INFO [2023-02-02 16:58:16] Base quality scores range from 24 to 35 (offset by 1)
INFO [2023-02-02 16:58:16] Initial testing for significant sample cross-contamination: unlikely
INFO [2023-02-02 16:58:16] Removing 89 variants with AF &lt; 0.030 or AF &gt;= 0.970 or insufficient supporting reads or depth &lt; 15.
WARN [2023-02-02 16:58:17] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 16:58:17] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 16:58:17] Mean coverages: chrX: 108.58, chrY: 108.58, chr1-22: 104.87.
INFO [2023-02-02 16:58:18] Mean coverages: chrX: 108.58, chrY: 2.17, chr1-22: 104.87.
INFO [2023-02-02 16:58:18] Mean coverages: chrX: 108.58, chrY: 5.17, chr1-22: 104.87.
INFO [2023-02-02 16:58:18] No homozygous variants in VCF, provide unfiltered VCF.
INFO [2023-02-02 16:58:18] No germline variants in VCF.
FATAL [2023-02-02 16:58:18] No solution with id hello 

FATAL [2023-02-02 16:58:18]  

FATAL [2023-02-02 16:58:18] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:58:18] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 16:58:18] No solution with id 100 

FATAL [2023-02-02 16:58:18]  

FATAL [2023-02-02 16:58:18] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:58:18] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 16:58:20] all.data and w have different lengths. 

FATAL [2023-02-02 16:58:20]  

FATAL [2023-02-02 16:58:20] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:58:20] parameters (PureCN 2.5.0). 

INFO [2023-02-02 16:58:21] Splitting 5 large targets to an average width of 400.
WARN [2023-02-02 16:58:21] No mappability scores provided.
WARN [2023-02-02 16:58:21] No reptiming scores provided.
INFO [2023-02-02 16:58:21] Calculating GC-content...
INFO [2023-02-02 16:58:22] Splitting 5 large targets to an average width of 400.
WARN [2023-02-02 16:58:22] No mappability scores provided.
WARN [2023-02-02 16:58:22] No reptiming scores provided.
INFO [2023-02-02 16:58:22] Calculating GC-content...
INFO [2023-02-02 16:58:22] Splitting 5 large targets to an average width of 400.
INFO [2023-02-02 16:58:22] Removing 1 targets overlapping with exclude.
WARN [2023-02-02 16:58:22] No mappability scores provided.
WARN [2023-02-02 16:58:22] No reptiming scores provided.
INFO [2023-02-02 16:58:22] Calculating GC-content...
WARN [2023-02-02 16:58:23] Found small target regions (&lt; 100bp). Will resize them.
INFO [2023-02-02 16:58:23] Splitting 5 large targets to an average width of 400.
FATAL [2023-02-02 16:58:23] off.target.padding must be negative. 

FATAL [2023-02-02 16:58:23]  

FATAL [2023-02-02 16:58:23] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:58:23] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 16:58:23] Interval coordinates should start at 1, not at 0 

FATAL [2023-02-02 16:58:23]  

FATAL [2023-02-02 16:58:23] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:58:23] parameters (PureCN 2.5.0). 

WARN [2023-02-02 16:58:23] Found small target regions (&lt; 100bp). Will resize them.
INFO [2023-02-02 16:58:24] Splitting 5 large targets to an average width of 400.
FATAL [2023-02-02 16:58:24] No off-target regions after filtering for mappability and 

FATAL [2023-02-02 16:58:24] off.target.padding 

FATAL [2023-02-02 16:58:24]  

FATAL [2023-02-02 16:58:24] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:58:24] parameters (PureCN 2.5.0). 

WARN [2023-02-02 16:58:24] Found small target regions (&lt; 100bp). Will resize them.
INFO [2023-02-02 16:58:24] Splitting 5 large targets to an average width of 400.
WARN [2023-02-02 16:58:24] No mappability scores provided.
INFO [2023-02-02 16:58:24] Calculating GC-content...
WARN [2023-02-02 16:58:24] Found small target regions (&lt; 100bp). Will resize them.
INFO [2023-02-02 16:58:24] Averaging reptiming into bins of size 200...
INFO [2023-02-02 16:58:25] Splitting 5 large targets to an average width of 400.
WARN [2023-02-02 16:58:25] No mappability scores provided.
INFO [2023-02-02 16:58:25] Calculating GC-content...
WARN [2023-02-02 16:58:25] Found small target regions (&lt; 100bp). Will resize them.
INFO [2023-02-02 16:58:25] Splitting 6 large targets to an average width of 200.
WARN [2023-02-02 16:58:25] No mappability scores provided.
WARN [2023-02-02 16:58:25] No reptiming scores provided.
INFO [2023-02-02 16:58:25] Calculating GC-content...
WARN [2023-02-02 16:58:26] Found small target regions (&lt; 100bp). Will resize them.
INFO [2023-02-02 16:58:26] Splitting 5 large targets to an average width of 400.
WARN [2023-02-02 16:58:26] No mappability scores provided.
WARN [2023-02-02 16:58:26] No reptiming scores provided.
INFO [2023-02-02 16:58:26] Calculating GC-content...
INFO [2023-02-02 16:58:26] Splitting 5 large targets to an average width of 400.
INFO [2023-02-02 16:58:26] Tiling off-target regions to an average width of 200000.
WARN [2023-02-02 16:58:26] No mappability scores provided.
WARN [2023-02-02 16:58:26] No reptiming scores provided.
INFO [2023-02-02 16:58:26] Calculating GC-content...
WARN [2023-02-02 16:58:27] Intervals contain off-target regions. Will not change intervals.
WARN [2023-02-02 16:58:27] No mappability scores provided.
WARN [2023-02-02 16:58:27] No reptiming scores provided.
INFO [2023-02-02 16:58:27] Calculating GC-content...
INFO [2023-02-02 16:58:27] Splitting 5 large targets to an average width of 400.
WARN [2023-02-02 16:58:27] No reptiming scores provided.
INFO [2023-02-02 16:58:27] Calculating GC-content...
WARN [2023-02-02 16:58:27] Found small target regions (&lt; 100bp). Will resize them.
INFO [2023-02-02 16:58:28] Splitting 5 large targets to an average width of 400.
WARN [2023-02-02 16:58:28] No reptiming scores provided.
INFO [2023-02-02 16:58:28] Calculating GC-content...
WARN [2023-02-02 16:58:29] Found small target regions (&lt; 100bp). Will resize them.
INFO [2023-02-02 16:58:29] Splitting 5 large targets to an average width of 400.
WARN [2023-02-02 16:58:29] 1 intervals without mappability score (1 on-target).
INFO [2023-02-02 16:58:29] Removing 1 intervals with low mappability score (&lt;0.60).
WARN [2023-02-02 16:58:29] No reptiming scores provided.
INFO [2023-02-02 16:58:29] Calculating GC-content...
WARN [2023-02-02 16:58:30] Found small target regions (&lt; 100bp). Will resize them.
INFO [2023-02-02 16:58:30] Splitting 5 large targets to an average width of 400.
INFO [2023-02-02 16:58:30] Tiling off-target regions to an average width of 200000.
WARN [2023-02-02 16:58:30] No reptiming scores provided.
INFO [2023-02-02 16:58:30] Calculating GC-content...
INFO [2023-02-02 16:58:31] Splitting 5 large targets to an average width of 400.
WARN [2023-02-02 16:58:31] No mappability scores provided.
WARN [2023-02-02 16:58:31] No reptiming scores provided.
INFO [2023-02-02 16:58:31] Calculating GC-content...
WARN [2023-02-02 16:58:31] Chromosome naming style of interval file (Ensembl) was different from reference (UCSC).
INFO [2023-02-02 16:58:32] Splitting 5 large targets to an average width of 400.
WARN [2023-02-02 16:58:32] No mappability scores provided.
WARN [2023-02-02 16:58:32] No reptiming scores provided.
INFO [2023-02-02 16:58:32] Calculating GC-content...
FATAL [2023-02-02 16:58:32] Chromosome naming style of interval file unknown, should be UCSC. 

FATAL [2023-02-02 16:58:32]  

FATAL [2023-02-02 16:58:32] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:58:32] parameters (PureCN 2.5.0). 

WARN [2023-02-02 16:58:32] Chromosome naming style of interval file (Ensembl) was different from reference (UCSC).
WARN [2023-02-02 16:58:32] Chromosome naming style of mappability file (Ensembl) was different from reference (UCSC).
INFO [2023-02-02 16:58:32] Splitting 5 large targets to an average width of 400.
WARN [2023-02-02 16:58:32] No reptiming scores provided.
INFO [2023-02-02 16:58:32] Calculating GC-content...
WARN [2023-02-02 16:58:33] Found small target regions (&lt; 60bp). Will resize them.
INFO [2023-02-02 16:58:33] Splitting 5 large targets to an average width of 400.
WARN [2023-02-02 16:58:33] No mappability scores provided.
WARN [2023-02-02 16:58:33] No reptiming scores provided.
INFO [2023-02-02 16:58:33] Calculating GC-content...
WARN [2023-02-02 16:58:33] Found small target regions (&lt; 60bp). Will drop them.
INFO [2023-02-02 16:58:33] Splitting 4 large targets to an average width of 400.
WARN [2023-02-02 16:58:33] No mappability scores provided.
WARN [2023-02-02 16:58:33] No reptiming scores provided.
INFO [2023-02-02 16:58:33] Calculating GC-content...
WARN [2023-02-02 16:58:34] Found small target regions (&lt; 200bp). Will resize them.
INFO [2023-02-02 16:58:34] Splitting 5 large targets to an average width of 400.
WARN [2023-02-02 16:58:34] No mappability scores provided.
WARN [2023-02-02 16:58:34] No reptiming scores provided.
INFO [2023-02-02 16:58:34] Calculating GC-content...
INFO [2023-02-02 16:58:35] 576 on-target bins with low coverage in all samples.
WARN [2023-02-02 16:58:35] You are likely not using the correct baits file!
WARN [2023-02-02 16:58:35] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 16:58:35] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 16:58:35] Processing on-target regions...
INFO [2023-02-02 16:58:36] Removing 930 intervals with low coverage in normalDB.
INFO [2023-02-02 16:58:36] Removing 1 intervals with zero coverage in more than 3% of normalDB.
INFO [2023-02-02 16:58:40] Interval weights found, but currently not supported by copynumber. Will simply exclude intervals with low weight.
multipcf finished for chromosome arm 1p 
multipcf finished for chromosome arm 1q 
multipcf finished for chromosome arm 2p 
multipcf finished for chromosome arm 2q 
multipcf finished for chromosome arm 3p 
multipcf finished for chromosome arm 3q 
multipcf finished for chromosome arm 4p 
multipcf finished for chromosome arm 4q 
multipcf finished for chromosome arm 5p 
multipcf finished for chromosome arm 5q 
multipcf finished for chromosome arm 6p 
multipcf finished for chromosome arm 6q 
multipcf finished for chromosome arm 7p 
multipcf finished for chromosome arm 7q 
multipcf finished for chromosome arm 8p 
multipcf finished for chromosome arm 8q 
multipcf finished for chromosome arm 9p 
multipcf finished for chromosome arm 9q 
multipcf finished for chromosome arm 10p 
multipcf finished for chromosome arm 10q 
multipcf finished for chromosome arm 11p 
multipcf finished for chromosome arm 11q 
multipcf finished for chromosome arm 12p 
multipcf finished for chromosome arm 12q 
multipcf finished for chromosome arm 13q 
multipcf finished for chromosome arm 14q 
multipcf finished for chromosome arm 15q 
multipcf finished for chromosome arm 16p 
multipcf finished for chromosome arm 16q 
multipcf finished for chromosome arm 17p 
multipcf finished for chromosome arm 17q 
multipcf finished for chromosome arm 18p 
multipcf finished for chromosome arm 18q 
multipcf finished for chromosome arm 19p 
multipcf finished for chromosome arm 19q 
multipcf finished for chromosome arm 20p 
multipcf finished for chromosome arm 20q 
multipcf finished for chromosome arm 21q 
multipcf finished for chromosome arm 22q 
INFO [2023-02-02 16:58:44] Interval weights found, but currently not supported by copynumber. Will simply exclude intervals with low weight.
multipcf finished for chromosome arm 1p 
multipcf finished for chromosome arm 1q 
multipcf finished for chromosome arm 2p 
multipcf finished for chromosome arm 2q 
multipcf finished for chromosome arm 3p 
multipcf finished for chromosome arm 3q 
multipcf finished for chromosome arm 4p 
multipcf finished for chromosome arm 4q 
multipcf finished for chromosome arm 5p 
multipcf finished for chromosome arm 5q 
multipcf finished for chromosome arm 6p 
multipcf finished for chromosome arm 6q 
multipcf finished for chromosome arm 7p 
multipcf finished for chromosome arm 7q 
multipcf finished for chromosome arm 8p 
multipcf finished for chromosome arm 8q 
multipcf finished for chromosome arm 9p 
multipcf finished for chromosome arm 9q 
multipcf finished for chromosome arm 10p 
multipcf finished for chromosome arm 10q 
multipcf finished for chromosome arm 11p 
multipcf finished for chromosome arm 11q 
multipcf finished for chromosome arm 12p 
multipcf finished for chromosome arm 12q 
multipcf finished for chromosome arm 13q 
multipcf finished for chromosome arm 14q 
multipcf finished for chromosome arm 15q 
multipcf finished for chromosome arm 16p 
multipcf finished for chromosome arm 16q 
multipcf finished for chromosome arm 17p 
multipcf finished for chromosome arm 17q 
multipcf finished for chromosome arm 18p 
multipcf finished for chromosome arm 18q 
multipcf finished for chromosome arm 19p 
multipcf finished for chromosome arm 19q 
multipcf finished for chromosome arm 20p 
multipcf finished for chromosome arm 20q 
multipcf finished for chromosome arm 21q 
multipcf finished for chromosome arm 22q 
INFO [2023-02-02 16:58:45] ------------------------------------------------------------
INFO [2023-02-02 16:58:45] PureCN 2.5.0
INFO [2023-02-02 16:58:45] ------------------------------------------------------------
INFO [2023-02-02 16:58:45] Loading coverage files...
INFO [2023-02-02 16:58:46] seg.file and normal.coverage.file provided. Using both.
INFO [2023-02-02 16:58:46] Mean target coverages: 112X (tumor) 105X (normal).
WARN [2023-02-02 16:58:46] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 16:58:46] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 16:58:46] Removing 199 intervals with missing log.ratio.
INFO [2023-02-02 16:58:46] Using 9850 intervals (9850 on-target, 0 off-target).
INFO [2023-02-02 16:58:46] No off-target intervals. If this is hybrid-capture data, consider adding them.
INFO [2023-02-02 16:58:46] Loading VCF...
INFO [2023-02-02 16:58:47] Found 127 variants in VCF file.
INFO [2023-02-02 16:58:47] 127 (100.0%) variants annotated as likely germline (DB INFO flag).
INFO [2023-02-02 16:58:47] LIB-02240e4 is tumor in VCF file.
INFO [2023-02-02 16:58:47] No homozygous variants in VCF, provide unfiltered VCF.
INFO [2023-02-02 16:58:47] Removing 1 non heterozygous (in matched normal) germline SNPs.
INFO [2023-02-02 16:58:47] Removing 0 low quality variants with non-offset BQ &lt; 25.
INFO [2023-02-02 16:58:47] Base quality scores range from 29 to 35 (offset by 1)
INFO [2023-02-02 16:58:47] Minimum number of supporting reads ranges from 2 to 5, depending on coverage and BQS.
INFO [2023-02-02 16:58:48] Removing 1 variants with AF &lt; 0.030 or AF &gt;= 1.000 or insufficient supporting reads or depth &lt; 15.
INFO [2023-02-02 16:58:48] Total size of targeted genomic region: 1.68Mb (2.66Mb with 50bp padding).
INFO [2023-02-02 16:58:48] 1.0% of targets contain variants.
INFO [2023-02-02 16:58:48] Removing 0 variants outside intervals.
INFO [2023-02-02 16:58:49] Found SOMATIC annotation in VCF.
INFO [2023-02-02 16:58:49] Setting somatic prior probabilities for somatic variants to 0.999000 or to 0.000100 otherwise.
INFO [2023-02-02 16:58:49] Found SOMATIC annotation in VCF. Setting mapping bias to 0.976.
INFO [2023-02-02 16:58:49] Excluding 0 novel or poor quality variants from segmentation.
INFO [2023-02-02 16:58:49] Sample sex: ?
INFO [2023-02-02 16:58:49] Segmenting data...
INFO [2023-02-02 16:58:49] Loaded provided segmentation file file3c1f3a14e80d77.seg (format DNAcopy).
INFO [2023-02-02 16:58:49] Re-centering provided segment means (offset 0.0110).
INFO [2023-02-02 16:58:49] Setting prune.hclust.h parameter to 0.200000.
INFO [2023-02-02 16:58:49] Found 59 segments with median size of 30.15Mb.
INFO [2023-02-02 16:58:49] Using 125 variants.
INFO [2023-02-02 16:58:49] Mean standard deviation of log-ratios: 0.34 (MAPD: 0.29)
INFO [2023-02-02 16:58:49] 2D-grid search of purity and ploidy...
INFO [2023-02-02 16:58:51] Local optima: 0.6/1.9
INFO [2023-02-02 16:58:51] Testing local optimum 1/1 at purity 0.60 and total ploidy 1.90...
INFO [2023-02-02 16:58:53] Fitting variants with beta model for local optimum 1/1...
INFO [2023-02-02 16:58:53] Fitting variants for purity 0.60, tumor ploidy 1.76 and contamination 0.01.
INFO [2023-02-02 16:58:57] Fitting variants for purity 0.40, tumor ploidy 1.76 and contamination 0.01.
INFO [2023-02-02 16:59:02] Fitting variants for purity 0.45, tumor ploidy 1.76 and contamination 0.01.
INFO [2023-02-02 16:59:06] Fitting variants for purity 0.50, tumor ploidy 1.76 and contamination 0.01.
INFO [2023-02-02 16:59:11] Fitting variants for purity 0.55, tumor ploidy 1.76 and contamination 0.01.
INFO [2023-02-02 16:59:15] Fitting variants for purity 0.65, tumor ploidy 1.76 and contamination 0.01.
INFO [2023-02-02 16:59:19] Fitting variants for purity 0.70, tumor ploidy 1.76 and contamination 0.01.
INFO [2023-02-02 16:59:24] Optimized purity: 0.65
INFO [2023-02-02 16:59:24] Done.
INFO [2023-02-02 16:59:24] ------------------------------------------------------------
INFO [2023-02-02 16:59:25] Interval weights found, but currently not supported by copynumber. Will simply exclude intervals with low weight.
FATAL [2023-02-02 16:59:25] Cannot find centromeres for hg20. Provide them manually or select a 

FATAL [2023-02-02 16:59:25] supported genome. 

FATAL [2023-02-02 16:59:25]  

FATAL [2023-02-02 16:59:25] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:59:25] parameters (PureCN 2.5.0). 

INFO [2023-02-02 16:59:26] Found 20 variants in VCF file.
INFO [2023-02-02 16:59:26] 20 (100.0%) variants annotated as likely germline (DB INFO flag).
WARN [2023-02-02 16:59:27] Did not find base quality scores, will use global error rate of 0.0010 instead.
INFO [2023-02-02 16:59:28] Found 127 variants in VCF file.
INFO [2023-02-02 16:59:28] 127 (100.0%) variants annotated as likely germline (DB INFO flag).
WARN [2023-02-02 16:59:28] Did not find base quality scores, will use global error rate of 0.0010 instead.
INFO [2023-02-02 16:59:29] ------------------------------------------------------------
INFO [2023-02-02 16:59:29] PureCN 2.5.0
INFO [2023-02-02 16:59:29] ------------------------------------------------------------
INFO [2023-02-02 16:59:29] Loading coverage files...
INFO [2023-02-02 16:59:29] Mean target coverages: 112X (tumor) 99X (normal).
WARN [2023-02-02 16:59:30] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 16:59:30] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 16:59:30] Removing 228 intervals with missing log.ratio.
INFO [2023-02-02 16:59:30] Removing 22 small (&lt; 5bp) intervals.
INFO [2023-02-02 16:59:30] Removing 19 intervals with low total coverage in normal (&lt; 150.00 reads).
WARN [2023-02-02 16:59:30] No normalDB provided. Provide one for better results.
INFO [2023-02-02 16:59:30] Removing 1724 low count (&lt; 100 total reads) intervals.
INFO [2023-02-02 16:59:30] Removing 6 low coverage (&lt; 15.0000X) intervals.
INFO [2023-02-02 16:59:30] Using 8050 intervals (8050 on-target, 0 off-target).
INFO [2023-02-02 16:59:30] No off-target intervals. If this is hybrid-capture data, consider adding them.
INFO [2023-02-02 16:59:30] No interval.file provided. Cannot check for any GC-biases.
INFO [2023-02-02 16:59:30] Loading VCF...
INFO [2023-02-02 16:59:30] Found 127 variants in VCF file.
INFO [2023-02-02 16:59:30] 127 (100.0%) variants annotated as likely germline (DB INFO flag).
INFO [2023-02-02 16:59:30] LIB-02240e4 is tumor in VCF file.
INFO [2023-02-02 16:59:30] No homozygous variants in VCF, provide unfiltered VCF.
INFO [2023-02-02 16:59:30] Removing 1 non heterozygous (in matched normal) germline SNPs.
INFO [2023-02-02 16:59:30] Removing 0 low quality variants with non-offset BQ &lt; 25.
INFO [2023-02-02 16:59:30] Base quality scores range from 29 to 35 (offset by 1)
INFO [2023-02-02 16:59:30] Minimum number of supporting reads ranges from 2 to 5, depending on coverage and BQS.
INFO [2023-02-02 16:59:31] Removing 1 variants with AF &lt; 0.030 or AF &gt;= 1.000 or insufficient supporting reads or depth &lt; 15.
INFO [2023-02-02 16:59:31] Total size of targeted genomic region: 1.53Mb (2.33Mb with 50bp padding).
INFO [2023-02-02 16:59:31] 1.2% of targets contain variants.
INFO [2023-02-02 16:59:31] Removing 4 variants outside intervals.
INFO [2023-02-02 16:59:31] Found SOMATIC annotation in VCF.
INFO [2023-02-02 16:59:31] Setting somatic prior probabilities for somatic variants to 0.999000 or to 0.000100 otherwise.
INFO [2023-02-02 16:59:32] Found SOMATIC annotation in VCF. Setting mapping bias to 0.972.
INFO [2023-02-02 16:59:32] Excluding 0 novel or poor quality variants from segmentation.
INFO [2023-02-02 16:59:32] Sample sex: ?
INFO [2023-02-02 16:59:32] Segmenting data...
INFO [2023-02-02 16:59:32] Loading pre-computed boundaries for DNAcopy...
INFO [2023-02-02 16:59:32] Setting undo.SD parameter to 1.000000.
INFO [2023-02-02 16:59:33] Setting prune.hclust.h parameter to 0.200000.
INFO [2023-02-02 16:59:33] Found 59 segments with median size of 17.67Mb.
INFO [2023-02-02 16:59:33] Using 121 variants.
INFO [2023-02-02 16:59:33] Mean standard deviation of log-ratios: 0.34 (MAPD: 0.30)
INFO [2023-02-02 16:59:33] 2D-grid search of purity and ploidy...
INFO [2023-02-02 16:59:33] Local optima: 0.65/1.8, 0.52/2
INFO [2023-02-02 16:59:33] Testing local optimum 1/2 at purity 0.65 and total ploidy 1.80...
INFO [2023-02-02 16:59:35] Testing local optimum 2/2 at purity 0.52 and total ploidy 2.00...
INFO [2023-02-02 16:59:36] Skipping 1 solutions that converged to the same optima.
INFO [2023-02-02 16:59:36] Fitting variants with beta model for local optimum 1/2...
INFO [2023-02-02 16:59:36] Fitting variants for purity 0.65, tumor ploidy 1.75 and contamination 0.01.
INFO [2023-02-02 16:59:40] Optimized purity: 0.65
INFO [2023-02-02 16:59:40] Done.
INFO [2023-02-02 16:59:40] ------------------------------------------------------------
FATAL [2023-02-02 16:59:42] log.ratio NULL in .writeLogRatioFileGATK4 

FATAL [2023-02-02 16:59:42]  

FATAL [2023-02-02 16:59:42] This runtime error might be caused by invalid input data or parameters. 

FATAL [2023-02-02 16:59:42] Please report bug (PureCN 2.5.0). 

INFO [2023-02-02 16:59:43] Loaded provided segmentation file example_seg.txt (format DNAcopy).
INFO [2023-02-02 16:59:43] Re-centering provided segment means (offset -0.0033).
INFO [2023-02-02 16:59:43] Loaded provided segmentation file example_gatk4_modelfinal.seg.gz (format GATK4).
WARN [2023-02-02 16:59:43] Expecting numeric chromosome names in seg.file, assuming file is properly sorted.
INFO [2023-02-02 16:59:43] Re-centering provided segment means (offset -0.0037).
INFO [2023-02-02 16:59:44] 576 on-target bins with low coverage in all samples.
WARN [2023-02-02 16:59:44] You are likely not using the correct baits file!
WARN [2023-02-02 16:59:44] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 16:59:44] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 16:59:44] Processing on-target regions...
INFO [2023-02-02 16:59:44] Removing 930 intervals with low coverage in normalDB.
INFO [2023-02-02 16:59:44] Removing 1 intervals with zero coverage in more than 3% of normalDB.
INFO [2023-02-02 16:59:47] ------------------------------------------------------------
INFO [2023-02-02 16:59:47] PureCN 2.5.0
INFO [2023-02-02 16:59:47] ------------------------------------------------------------
INFO [2023-02-02 16:59:47] Using BiocParallel for parallel optimization.
INFO [2023-02-02 16:59:47] Loading coverage files...
INFO [2023-02-02 16:59:48] Mean target coverages: 112X (tumor) 99X (normal).
WARN [2023-02-02 16:59:48] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 16:59:48] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 16:59:48] Removing 228 intervals with missing log.ratio.
INFO [2023-02-02 16:59:48] Removing 705 intervals excluded in normalDB.
INFO [2023-02-02 16:59:48] normalDB provided. Setting minimum coverage for segmentation to 0.0015X.
INFO [2023-02-02 16:59:48] Removing 1066 low count (&lt; 100 total reads) intervals.
INFO [2023-02-02 16:59:48] Using 8050 intervals (8050 on-target, 0 off-target).
INFO [2023-02-02 16:59:48] No off-target intervals. If this is hybrid-capture data, consider adding them.
INFO [2023-02-02 16:59:48] No interval.file provided. Cannot check for any GC-biases.
INFO [2023-02-02 16:59:48] Sample sex: ?
INFO [2023-02-02 16:59:48] Segmenting data...
INFO [2023-02-02 16:59:48] Interval weights found, will use weighted CBS.
INFO [2023-02-02 16:59:48] Loading pre-computed boundaries for DNAcopy...
INFO [2023-02-02 16:59:48] Setting undo.SD parameter to 1.000000.
Setting multi-figure configuration
INFO [2023-02-02 16:59:50] Found 54 segments with median size of 23.50Mb.
INFO [2023-02-02 16:59:50] Mean standard deviation of log-ratios: 0.34 (MAPD: 0.30)
INFO [2023-02-02 16:59:50] 2D-grid search of purity and ploidy...
INFO [2023-02-02 16:59:50] Local optima: 0.65/1.8, 0.52/2
INFO [2023-02-02 16:59:53] Skipping 1 solutions that converged to the same optima.
INFO [2023-02-02 16:59:53] Done.
INFO [2023-02-02 16:59:53] ------------------------------------------------------------
INFO [2023-02-02 16:59:53] Reading /tmp/RtmpBApYiI/file3c1f3a567483b4.rds...
FATAL [2023-02-02 16:59:53] runAbsoluteCN was run without a VCF file. 

FATAL [2023-02-02 16:59:53]  

FATAL [2023-02-02 16:59:53] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:59:53] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 16:59:53] runAbsoluteCN was run without a VCF file. 

FATAL [2023-02-02 16:59:53]  

FATAL [2023-02-02 16:59:53] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:59:53] parameters (PureCN 2.5.0). 

INFO [2023-02-02 16:59:53] ------------------------------------------------------------
INFO [2023-02-02 16:59:53] PureCN 2.5.0
INFO [2023-02-02 16:59:53] ------------------------------------------------------------
INFO [2023-02-02 16:59:53] Loading coverage files...
INFO [2023-02-02 16:59:54] Mean target coverages: 112X (tumor) 99X (normal).
WARN [2023-02-02 16:59:54] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 16:59:54] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 16:59:54] Removing 228 intervals with missing log.ratio.
INFO [2023-02-02 16:59:54] Removing 705 intervals excluded in normalDB.
INFO [2023-02-02 16:59:54] normalDB provided. Setting minimum coverage for segmentation to 0.0015X.
INFO [2023-02-02 16:59:54] Removing 1066 low count (&lt; 100 total reads) intervals.
INFO [2023-02-02 16:59:54] Using 8050 intervals (8050 on-target, 0 off-target).
INFO [2023-02-02 16:59:54] No off-target intervals. If this is hybrid-capture data, consider adding them.
INFO [2023-02-02 16:59:54] No interval.file provided. Cannot check for any GC-biases.
INFO [2023-02-02 16:59:54] Sample sex: ?
INFO [2023-02-02 16:59:54] Segmenting data...
INFO [2023-02-02 16:59:54] Interval weights found, will use weighted PSCBS.
FATAL [2023-02-02 16:59:54] segmentationPSCBS requires VCF file. 

FATAL [2023-02-02 16:59:54]  

FATAL [2023-02-02 16:59:54] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 16:59:54] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:05] ------------------------------------------------------------
INFO [2023-02-02 17:00:05] PureCN 2.5.0
INFO [2023-02-02 17:00:05] ------------------------------------------------------------
INFO [2023-02-02 17:00:05] Loading coverage files...
FATAL [2023-02-02 17:00:06] Need a normal coverage file if log.ratio and seg.file are not provided. 

FATAL [2023-02-02 17:00:06]  

FATAL [2023-02-02 17:00:06] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:06] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:06] ------------------------------------------------------------
INFO [2023-02-02 17:00:06] PureCN 2.5.0
INFO [2023-02-02 17:00:06] ------------------------------------------------------------
FATAL [2023-02-02 17:00:06] min.ploidy or max.ploidy not within expected range. 

FATAL [2023-02-02 17:00:06]  

FATAL [2023-02-02 17:00:06] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:06] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:06] ------------------------------------------------------------
INFO [2023-02-02 17:00:06] PureCN 2.5.0
INFO [2023-02-02 17:00:06] ------------------------------------------------------------
FATAL [2023-02-02 17:00:06] min.ploidy or max.ploidy not within expected range. 

FATAL [2023-02-02 17:00:06]  

FATAL [2023-02-02 17:00:06] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:06] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:06] ------------------------------------------------------------
INFO [2023-02-02 17:00:06] PureCN 2.5.0
INFO [2023-02-02 17:00:06] ------------------------------------------------------------
INFO [2023-02-02 17:00:06] ------------------------------------------------------------
INFO [2023-02-02 17:00:06] PureCN 2.5.0
INFO [2023-02-02 17:00:06] ------------------------------------------------------------
INFO [2023-02-02 17:00:06] ------------------------------------------------------------
INFO [2023-02-02 17:00:06] PureCN 2.5.0
INFO [2023-02-02 17:00:06] ------------------------------------------------------------
FATAL [2023-02-02 17:00:06] test.num.copy not within expected range. 

FATAL [2023-02-02 17:00:06]  

FATAL [2023-02-02 17:00:06] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:06] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:06] ------------------------------------------------------------
INFO [2023-02-02 17:00:06] PureCN 2.5.0
INFO [2023-02-02 17:00:06] ------------------------------------------------------------
WARN [2023-02-02 17:00:06] test.num.copy outside recommended range.
FATAL [2023-02-02 17:00:06] max.non.clonal not within expected range or format. 

FATAL [2023-02-02 17:00:06]  

FATAL [2023-02-02 17:00:06] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:06] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:06] ------------------------------------------------------------
INFO [2023-02-02 17:00:06] PureCN 2.5.0
INFO [2023-02-02 17:00:06] ------------------------------------------------------------
WARN [2023-02-02 17:00:06] test.num.copy outside recommended range.
FATAL [2023-02-02 17:00:06] max.non.clonal not within expected range or format. 

FATAL [2023-02-02 17:00:06]  

FATAL [2023-02-02 17:00:06] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:06] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:06] ------------------------------------------------------------
INFO [2023-02-02 17:00:06] PureCN 2.5.0
INFO [2023-02-02 17:00:06] ------------------------------------------------------------
FATAL [2023-02-02 17:00:06] test.purity not within expected range. 

FATAL [2023-02-02 17:00:06]  

FATAL [2023-02-02 17:00:06] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:06] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:06] ------------------------------------------------------------
INFO [2023-02-02 17:00:06] PureCN 2.5.0
INFO [2023-02-02 17:00:06] ------------------------------------------------------------
INFO [2023-02-02 17:00:06] Loading coverage files...
FATAL [2023-02-02 17:00:07] Tumor and normal are identical. This won&#x27;t give any meaningful results 

FATAL [2023-02-02 17:00:07] and I&#x27;m stopping here. 

FATAL [2023-02-02 17:00:07]  

FATAL [2023-02-02 17:00:07] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:07] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:07] ------------------------------------------------------------
INFO [2023-02-02 17:00:07] PureCN 2.5.0
INFO [2023-02-02 17:00:07] ------------------------------------------------------------
INFO [2023-02-02 17:00:07] Loading coverage files...
FATAL [2023-02-02 17:00:07] Length of log.ratio different from tumor coverage. 

FATAL [2023-02-02 17:00:07]  

FATAL [2023-02-02 17:00:07] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:07] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:07] ------------------------------------------------------------
INFO [2023-02-02 17:00:07] PureCN 2.5.0
INFO [2023-02-02 17:00:07] ------------------------------------------------------------
FATAL [2023-02-02 17:00:07] prior.purity must have the same length as test.purity. 

FATAL [2023-02-02 17:00:07]  

FATAL [2023-02-02 17:00:07] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:07] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:07] ------------------------------------------------------------
INFO [2023-02-02 17:00:07] PureCN 2.5.0
INFO [2023-02-02 17:00:07] ------------------------------------------------------------
FATAL [2023-02-02 17:00:07] min.gof not within expected range or format. 

FATAL [2023-02-02 17:00:07]  

FATAL [2023-02-02 17:00:07] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:07] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:07] ------------------------------------------------------------
INFO [2023-02-02 17:00:07] PureCN 2.5.0
INFO [2023-02-02 17:00:07] ------------------------------------------------------------
FATAL [2023-02-02 17:00:07] prior.purity not within expected range or format. 

FATAL [2023-02-02 17:00:07]  

FATAL [2023-02-02 17:00:07] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:07] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:07] ------------------------------------------------------------
INFO [2023-02-02 17:00:07] PureCN 2.5.0
INFO [2023-02-02 17:00:07] ------------------------------------------------------------
FATAL [2023-02-02 17:00:07] prior.purity must add to 1. Sum is 1.5 

FATAL [2023-02-02 17:00:07]  

FATAL [2023-02-02 17:00:07] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:07] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:07] ------------------------------------------------------------
INFO [2023-02-02 17:00:07] PureCN 2.5.0
INFO [2023-02-02 17:00:07] ------------------------------------------------------------
FATAL [2023-02-02 17:00:07] max.homozygous.loss not within expected range or format. 

FATAL [2023-02-02 17:00:07]  

FATAL [2023-02-02 17:00:07] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:07] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:07] ------------------------------------------------------------
INFO [2023-02-02 17:00:07] PureCN 2.5.0
INFO [2023-02-02 17:00:07] ------------------------------------------------------------
FATAL [2023-02-02 17:00:07] prior.K not within expected range or format. 

FATAL [2023-02-02 17:00:07]  

FATAL [2023-02-02 17:00:07] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:07] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:07] ------------------------------------------------------------
INFO [2023-02-02 17:00:07] PureCN 2.5.0
INFO [2023-02-02 17:00:07] ------------------------------------------------------------
FATAL [2023-02-02 17:00:07] prior.contamination not within expected range or format. 

FATAL [2023-02-02 17:00:07]  

FATAL [2023-02-02 17:00:07] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:07] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:07] ------------------------------------------------------------
INFO [2023-02-02 17:00:07] PureCN 2.5.0
INFO [2023-02-02 17:00:07] ------------------------------------------------------------
FATAL [2023-02-02 17:00:07] Iterations not in the expected range from 10 to 250. 

FATAL [2023-02-02 17:00:07]  

FATAL [2023-02-02 17:00:07] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:07] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:07] ------------------------------------------------------------
INFO [2023-02-02 17:00:08] PureCN 2.5.0
INFO [2023-02-02 17:00:08] ------------------------------------------------------------
FATAL [2023-02-02 17:00:08] Iterations not in the expected range from 10 to 250. 

FATAL [2023-02-02 17:00:08]  

FATAL [2023-02-02 17:00:08] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:08] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:08] ------------------------------------------------------------
INFO [2023-02-02 17:00:08] PureCN 2.5.0
INFO [2023-02-02 17:00:08] ------------------------------------------------------------
INFO [2023-02-02 17:00:08] Loading coverage files...
FATAL [2023-02-02 17:00:08] Missing tumor.coverage.file requires seg.file or log.ratio and 

FATAL [2023-02-02 17:00:08] interval.file. 

FATAL [2023-02-02 17:00:08]  

FATAL [2023-02-02 17:00:08] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:08] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:08] ------------------------------------------------------------
INFO [2023-02-02 17:00:08] PureCN 2.5.0
INFO [2023-02-02 17:00:08] ------------------------------------------------------------
INFO [2023-02-02 17:00:08] ------------------------------------------------------------
INFO [2023-02-02 17:00:08] PureCN 2.5.0
INFO [2023-02-02 17:00:08] ------------------------------------------------------------
INFO [2023-02-02 17:00:08] Loading coverage files...
FATAL [2023-02-02 17:00:09] Interval files in normal and tumor different. 

FATAL [2023-02-02 17:00:09]  

FATAL [2023-02-02 17:00:09] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:09] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:09] ------------------------------------------------------------
INFO [2023-02-02 17:00:09] PureCN 2.5.0
INFO [2023-02-02 17:00:09] ------------------------------------------------------------
INFO [2023-02-02 17:00:09] Loading coverage files...
INFO [2023-02-02 17:00:10] Mean target coverages: 0X (tumor) 99X (normal).
WARN [2023-02-02 17:00:10] Large difference in coverage of tumor and normal.
FATAL [2023-02-02 17:00:10] No finite intervals. 

FATAL [2023-02-02 17:00:10]  

FATAL [2023-02-02 17:00:10] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:10] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:10] ------------------------------------------------------------
INFO [2023-02-02 17:00:10] PureCN 2.5.0
INFO [2023-02-02 17:00:10] ------------------------------------------------------------
INFO [2023-02-02 17:00:10] Loading coverage files...
INFO [2023-02-02 17:00:11] Mean target coverages: 112X (tumor) 99X (normal).
WARN [2023-02-02 17:00:11] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:00:11] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 17:00:11] Removing 228 intervals with missing log.ratio.
INFO [2023-02-02 17:00:11] Removing 22 small (&lt; 5bp) intervals.
INFO [2023-02-02 17:00:11] Removing 19 intervals with low total coverage in normal (&lt; 150.00 reads).
WARN [2023-02-02 17:00:11] No normalDB provided. Provide one for better results.
INFO [2023-02-02 17:00:11] Removing 1724 low count (&lt; 100 total reads) intervals.
INFO [2023-02-02 17:00:11] Removing 6 low coverage (&lt; 15.0000X) intervals.
INFO [2023-02-02 17:00:11] Using 8050 intervals (8050 on-target, 0 off-target).
INFO [2023-02-02 17:00:11] No off-target intervals. If this is hybrid-capture data, consider adding them.
INFO [2023-02-02 17:00:11] No interval.file provided. Cannot check for any GC-biases.
INFO [2023-02-02 17:00:11] Loading VCF...
INFO [2023-02-02 17:00:11] Found 127 variants in VCF file.
INFO [2023-02-02 17:00:12] 127 (100.0%) variants annotated as likely germline (DB INFO flag).
INFO [2023-02-02 17:00:12] LIB-02240e4 is tumor in VCF file.
INFO [2023-02-02 17:00:12] No homozygous variants in VCF, provide unfiltered VCF.
INFO [2023-02-02 17:00:12] Removing 1 non heterozygous (in matched normal) germline SNPs.
INFO [2023-02-02 17:00:12] Removing 0 low quality variants with non-offset BQ &lt; 25.
INFO [2023-02-02 17:00:12] Base quality scores range from 29 to 35 (offset by 1)
INFO [2023-02-02 17:00:12] Minimum number of supporting reads ranges from 2 to 5, depending on coverage and BQS.
INFO [2023-02-02 17:00:13] Removing 1 variants with AF &lt; 0.030 or AF &gt;= 1.000 or insufficient supporting reads or depth &lt; 15.
Error in FileForFormat(resource(con), fileFormat(con)) : 
  Format &#x27;txt&#x27; unsupported
In addition: Warning messages:
1: In for (i in seq_len(n)) { :
  closing unused connection 5 (/tmp/RtmpBApYiI/file3c1f3a8950b1f.tsv)
2: In for (i in seq_len(n)) { :
  closing unused connection 4 (/home/biocbuild/bbs-3.17-bioc/R/library/PureCN/extdata/example_gatk4_denoised_cr.tsv.gz)
FATAL [2023-02-02 17:00:13] Could not import snp.blacklist 

FATAL [2023-02-02 17:00:13] /home/biocbuild/bbs-3.17-bioc/R/library/PureCN/extdata/example_normal.txt.gz:Error 

FATAL [2023-02-02 17:00:13] in FileForFormat(resource(con), fileFormat(con)) : Format &#x27;txt&#x27; 

FATAL [2023-02-02 17:00:13] unsupported 

FATAL [2023-02-02 17:00:13]  

FATAL [2023-02-02 17:00:13] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:00:13] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:00:13] ------------------------------------------------------------
INFO [2023-02-02 17:00:13] PureCN 2.5.0
INFO [2023-02-02 17:00:13] ------------------------------------------------------------
INFO [2023-02-02 17:00:13] Loading coverage files...
INFO [2023-02-02 17:00:13] Mean target coverages: 112X (tumor) 99X (normal).
WARN [2023-02-02 17:00:14] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:00:14] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 17:00:14] Removing 228 intervals with missing log.ratio.
INFO [2023-02-02 17:00:14] Removing 22 small (&lt; 5bp) intervals.
INFO [2023-02-02 17:00:14] Removing 19 intervals with low total coverage in normal (&lt; 150.00 reads).
WARN [2023-02-02 17:00:14] No normalDB provided. Provide one for better results.
INFO [2023-02-02 17:00:14] Removing 233 low coverage (&lt; 15.0000X) intervals.
INFO [2023-02-02 17:00:14] Using 9547 intervals (9547 on-target, 0 off-target).
INFO [2023-02-02 17:00:14] No off-target intervals. If this is hybrid-capture data, consider adding them.
INFO [2023-02-02 17:00:14] No interval.file provided. Cannot check for any GC-biases.
INFO [2023-02-02 17:00:14] Loading VCF...
INFO [2023-02-02 17:00:14] Found 127 variants in VCF file.
INFO [2023-02-02 17:00:14] 127 (100.0%) variants annotated as likely germline (DB INFO flag).
INFO [2023-02-02 17:00:14] LIB-02240e4 is tumor in VCF file.
INFO [2023-02-02 17:00:15] No homozygous variants in VCF, provide unfiltered VCF.
INFO [2023-02-02 17:00:15] Removing 1 non heterozygous (in matched normal) germline SNPs.
INFO [2023-02-02 17:00:15] Removing 0 low quality variants with non-offset BQ &lt; 25.
INFO [2023-02-02 17:00:15] Base quality scores range from 29 to 35 (offset by 1)
INFO [2023-02-02 17:00:15] Minimum number of supporting reads ranges from 2 to 5, depending on coverage and BQS.
INFO [2023-02-02 17:00:15] Removing 1 variants with AF &lt; 0.030 or AF &gt;= 1.000 or insufficient supporting reads or depth &lt; 15.
INFO [2023-02-02 17:00:15] Total size of targeted genomic region: 1.63Mb (2.58Mb with 50bp padding).
INFO [2023-02-02 17:00:16] 1.0% of targets contain variants.
INFO [2023-02-02 17:00:16] Removing 2 variants outside intervals.
INFO [2023-02-02 17:00:16] Found SOMATIC annotation in VCF.
INFO [2023-02-02 17:00:16] Setting somatic prior probabilities for somatic variants to 0.999000 or to 0.000100 otherwise.
INFO [2023-02-02 17:00:16] Found SOMATIC annotation in VCF. Setting mapping bias to 0.973.
INFO [2023-02-02 17:00:16] Excluding 0 novel or poor quality variants from segmentation.
INFO [2023-02-02 17:00:16] Sample sex: ?
INFO [2023-02-02 17:00:16] Segmenting data...
INFO [2023-02-02 17:00:16] Loading pre-computed boundaries for DNAcopy...
INFO [2023-02-02 17:00:16] Setting undo.SD parameter to 1.000000.
INFO [2023-02-02 17:00:17] Setting prune.hclust.h parameter to 0.200000.
INFO [2023-02-02 17:00:17] Found 54 segments with median size of 24.88Mb.
INFO [2023-02-02 17:00:18] Using 123 variants.
INFO [2023-02-02 17:00:18] Mean standard deviation of log-ratios: 0.37 (MAPD: 0.32)
INFO [2023-02-02 17:00:18] 2D-grid search of purity and ploidy...
INFO [2023-02-02 17:00:19] Local optima: 0.62/1.9, 0.3/1.9, 0.32/2
INFO [2023-02-02 17:00:19] Testing local optimum 1/3 at purity 0.62 and total ploidy 1.90...
INFO [2023-02-02 17:00:21] Testing local optimum 2/3 at purity 0.30 and total ploidy 1.90...
INFO [2023-02-02 17:00:21] Recalibrating log-ratios...
INFO [2023-02-02 17:00:21] Testing local optimum 2/3 at purity 0.30 and total ploidy 1.90...
INFO [2023-02-02 17:00:22] Recalibrating log-ratios...
INFO [2023-02-02 17:00:22] Testing local optimum 2/3 at purity 0.30 and total ploidy 1.90...
INFO [2023-02-02 17:00:22] Recalibrating log-ratios...
INFO [2023-02-02 17:00:22] Testing local optimum 2/3 at purity 0.30 and total ploidy 1.90...
INFO [2023-02-02 17:00:23] Testing local optimum 3/3 at purity 0.32 and total ploidy 2.00...
INFO [2023-02-02 17:00:23] Recalibrating log-ratios...
INFO [2023-02-02 17:00:23] Testing local optimum 3/3 at purity 0.32 and total ploidy 2.00...
INFO [2023-02-02 17:00:24] Recalibrating log-ratios...
INFO [2023-02-02 17:00:24] Testing local optimum 3/3 at purity 0.32 and total ploidy 2.00...
INFO [2023-02-02 17:00:24] Recalibrating log-ratios...
INFO [2023-02-02 17:00:24] Testing local optimum 3/3 at purity 0.32 and total ploidy 2.00...
INFO [2023-02-02 17:00:25] Skipping 1 solutions that converged to the same optima.
INFO [2023-02-02 17:00:25] Skipping 1 solutions exceeding max.non.clonal (0.20): 0.3/1.9 (purity/tumor ploidy)
INFO [2023-02-02 17:00:25] Fitting variants with beta model for local optimum 1/3...
INFO [2023-02-02 17:00:25] Fitting variants for purity 0.65, tumor ploidy 1.74 and contamination 0.01.
INFO [2023-02-02 17:00:29] Optimized purity: 0.65
INFO [2023-02-02 17:00:29] Done.
INFO [2023-02-02 17:00:29] ------------------------------------------------------------
INFO [2023-02-02 17:00:29] ------------------------------------------------------------
INFO [2023-02-02 17:00:29] PureCN 2.5.0
INFO [2023-02-02 17:00:29] ------------------------------------------------------------
INFO [2023-02-02 17:00:29] Loading coverage files...
INFO [2023-02-02 17:00:30] Mean target coverages: 112X (tumor) 99X (normal).
WARN [2023-02-02 17:00:30] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:00:30] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 17:00:30] Removing 228 intervals with missing log.ratio.
INFO [2023-02-02 17:00:30] Removing 22 small (&lt; 5bp) intervals.
INFO [2023-02-02 17:00:30] Removing 19 intervals with low total coverage in normal (&lt; 150.00 reads).
WARN [2023-02-02 17:00:30] No normalDB provided. Provide one for better results.
INFO [2023-02-02 17:00:30] Removing 1724 low count (&lt; 100 total reads) intervals.
INFO [2023-02-02 17:00:30] Removing 6 low coverage (&lt; 15.0000X) intervals.
INFO [2023-02-02 17:00:30] Using 8050 intervals (8050 on-target, 0 off-target).
INFO [2023-02-02 17:00:30] No off-target intervals. If this is hybrid-capture data, consider adding them.
INFO [2023-02-02 17:00:30] No interval.file provided. Cannot check for any GC-biases.
INFO [2023-02-02 17:00:30] Loading VCF...
INFO [2023-02-02 17:00:30] Found 127 variants in VCF file.
WARN [2023-02-02 17:00:30] DP FORMAT field contains NAs. Removing 3 variants.
WARN [2023-02-02 17:00:30] DB INFO flag contains NAs
INFO [2023-02-02 17:00:31] 119 (96.0%) variants annotated as likely germline (DB INFO flag).
INFO [2023-02-02 17:00:31] LIB-02240e4 is tumor in VCF file.
INFO [2023-02-02 17:00:31] No homozygous variants in VCF, provide unfiltered VCF.
INFO [2023-02-02 17:00:31] Removing 0 low quality variants with non-offset BQ &lt; 25.
INFO [2023-02-02 17:00:31] Base quality scores range from 29 to 35 (offset by 1)
INFO [2023-02-02 17:00:31] Minimum number of supporting reads ranges from 2 to 5, depending on coverage and BQS.
INFO [2023-02-02 17:00:31] Removing 1 variants with AF &lt; 0.030 or AF &gt;= 0.970 or insufficient supporting reads or depth &lt; 15.
INFO [2023-02-02 17:00:31] Total size of targeted genomic region: 1.53Mb (2.33Mb with 50bp padding).
INFO [2023-02-02 17:00:32] 1.2% of targets contain variants.
INFO [2023-02-02 17:00:32] Setting somatic prior probabilities for dbSNP hits to 0.000500 or to 0.500000 otherwise.
INFO [2023-02-02 17:00:32] Excluding 5 novel or poor quality variants from segmentation.
INFO [2023-02-02 17:00:32] Sample sex: ?
INFO [2023-02-02 17:00:32] Segmenting data...
INFO [2023-02-02 17:00:32] Loading pre-computed boundaries for DNAcopy...
INFO [2023-02-02 17:00:32] Setting undo.SD parameter to 1.000000.
Setting multi-figure configuration
INFO [2023-02-02 17:00:34] Setting prune.hclust.h parameter to 0.200000.
INFO [2023-02-02 17:00:34] Found 59 segments with median size of 17.67Mb.
INFO [2023-02-02 17:00:34] Using 123 variants.
INFO [2023-02-02 17:00:34] Mean standard deviation of log-ratios: 0.34 (MAPD: 0.30)
INFO [2023-02-02 17:00:34] 2D-grid search of purity and ploidy...
INFO [2023-02-02 17:00:35] Local optima: 0.62/1.9, 0.3/1.9, 0.32/2
INFO [2023-02-02 17:00:35] Testing local optimum 1/3 at purity 0.62 and total ploidy 1.90...
INFO [2023-02-02 17:00:37] Testing local optimum 2/3 at purity 0.30 and total ploidy 1.90...
INFO [2023-02-02 17:00:37] Recalibrating log-ratios...
INFO [2023-02-02 17:00:37] Testing local optimum 2/3 at purity 0.30 and total ploidy 1.90...
INFO [2023-02-02 17:00:38] Recalibrating log-ratios...
INFO [2023-02-02 17:00:38] Testing local optimum 2/3 at purity 0.30 and total ploidy 1.90...
INFO [2023-02-02 17:00:38] Recalibrating log-ratios...
INFO [2023-02-02 17:00:38] Testing local optimum 2/3 at purity 0.30 and total ploidy 1.90...
INFO [2023-02-02 17:00:39] Testing local optimum 3/3 at purity 0.32 and total ploidy 2.00...
INFO [2023-02-02 17:00:39] Recalibrating log-ratios...
INFO [2023-02-02 17:00:39] Testing local optimum 3/3 at purity 0.32 and total ploidy 2.00...
INFO [2023-02-02 17:00:40] Recalibrating log-ratios...
INFO [2023-02-02 17:00:40] Testing local optimum 3/3 at purity 0.32 and total ploidy 2.00...
INFO [2023-02-02 17:00:40] Recalibrating log-ratios...
INFO [2023-02-02 17:00:40] Testing local optimum 3/3 at purity 0.32 and total ploidy 2.00...
INFO [2023-02-02 17:00:41] Skipping 2 solutions exceeding max.non.clonal (0.20): 0.3/1.9, 0.35/1.61 (purity/tumor ploidy)
INFO [2023-02-02 17:00:41] Fitting variants with beta model for local optimum 1/3...
INFO [2023-02-02 17:00:41] Fitting variants for purity 0.65, tumor ploidy 1.75 and contamination 0.01.
INFO [2023-02-02 17:00:45] Optimized purity: 0.65
INFO [2023-02-02 17:00:45] Done.
INFO [2023-02-02 17:00:45] ------------------------------------------------------------
INFO [2023-02-02 17:00:45] ------------------------------------------------------------
INFO [2023-02-02 17:00:45] PureCN 2.5.0
INFO [2023-02-02 17:00:45] ------------------------------------------------------------
INFO [2023-02-02 17:00:45] Loading coverage files...
INFO [2023-02-02 17:00:46] Mean target coverages: 112X (tumor) 99X (normal).
WARN [2023-02-02 17:00:46] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:00:46] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 17:00:46] No Gene column in interval.file. You won&#x27;t get gene-level calls.
INFO [2023-02-02 17:00:47] Removing 228 intervals with missing log.ratio.
INFO [2023-02-02 17:00:47] Removing 15 low/high GC targets.
INFO [2023-02-02 17:00:47] Removing 21 small (&lt; 5bp) intervals.
INFO [2023-02-02 17:00:47] Removing 19 intervals with low total coverage in normal (&lt; 150.00 reads).
WARN [2023-02-02 17:00:47] No normalDB provided. Provide one for better results.
INFO [2023-02-02 17:00:47] Removing 1718 low count (&lt; 100 total reads) intervals.
INFO [2023-02-02 17:00:47] Removing 6 low coverage (&lt; 15.0000X) intervals.
INFO [2023-02-02 17:00:47] Removing 36 low mappability intervals.
INFO [2023-02-02 17:00:47] Using 8006 intervals (8006 on-target, 0 off-target).
INFO [2023-02-02 17:00:47] No off-target intervals. If this is hybrid-capture data, consider adding them.
INFO [2023-02-02 17:00:47] AT/GC dropout: 1.01 (tumor), 1.03 (normal), 0.98 (coverage log-ratio). 
INFO [2023-02-02 17:00:47] Loading VCF...
INFO [2023-02-02 17:00:47] Found 127 variants in VCF file.
INFO [2023-02-02 17:00:48] 127 (100.0%) variants annotated as likely germline (DB INFO flag).
INFO [2023-02-02 17:00:48] LIB-02240e4 is tumor in VCF file.
INFO [2023-02-02 17:00:48] No homozygous variants in VCF, provide unfiltered VCF.
INFO [2023-02-02 17:00:48] Removing 1 non heterozygous (in matched normal) germline SNPs.
INFO [2023-02-02 17:00:48] Removing 0 low quality variants with non-offset BQ &lt; 25.
INFO [2023-02-02 17:00:48] Base quality scores range from 29 to 35 (offset by 1)
INFO [2023-02-02 17:00:48] Minimum number of supporting reads ranges from 2 to 5, depending on coverage and BQS.
INFO [2023-02-02 17:00:49] Removing 1 variants with AF &lt; 0.030 or AF &gt;= Inf or insufficient supporting reads or depth &lt; 15.
INFO [2023-02-02 17:00:49] Total size of targeted genomic region: 1.51Mb (2.31Mb with 50bp padding).
INFO [2023-02-02 17:00:49] 1.2% of targets contain variants.
INFO [2023-02-02 17:00:49] Removing 11 variants outside intervals.
INFO [2023-02-02 17:00:49] Found SOMATIC annotation in VCF.
INFO [2023-02-02 17:00:49] Setting somatic prior probabilities for somatic variants to 0.999000 or to 0.000100 otherwise.
INFO [2023-02-02 17:00:49] Found SOMATIC annotation in VCF. Setting mapping bias to 0.970.
INFO [2023-02-02 17:00:49] Excluding 0 novel or poor quality variants from segmentation.
INFO [2023-02-02 17:00:49] Sample sex: ?
INFO [2023-02-02 17:00:49] Segmenting data...
INFO [2023-02-02 17:00:50] MAPD of 114 allelic fractions: 0.14 (0.15 adjusted).
INFO [2023-02-02 17:00:50] Using unweighted PSCBS.
INFO [2023-02-02 17:00:50] Setting undo.SD parameter to 1.000000.
INFO [2023-02-02 17:01:17] Setting prune.hclust.h parameter to 0.200000.
INFO [2023-02-02 17:01:17] Found 72 segments with median size of 27.25Mb.
INFO [2023-02-02 17:01:17] Using 114 variants.
INFO [2023-02-02 17:01:17] Mean standard deviation of log-ratios: 0.34 (MAPD: 0.30)
INFO [2023-02-02 17:01:17] 2D-grid search of purity and ploidy...
INFO [2023-02-02 17:01:17] Local optima: 0.65/1.8, 0.38/2.2
INFO [2023-02-02 17:01:17] Testing local optimum 1/2 at purity 0.65 and total ploidy 1.80...
INFO [2023-02-02 17:01:18] Recalibrating log-ratios...
INFO [2023-02-02 17:01:18] Testing local optimum 1/2 at purity 0.65 and total ploidy 1.80...
INFO [2023-02-02 17:01:18] Recalibrating log-ratios...
INFO [2023-02-02 17:01:18] Testing local optimum 1/2 at purity 0.65 and total ploidy 1.80...
INFO [2023-02-02 17:01:19] Recalibrating log-ratios...
INFO [2023-02-02 17:01:19] Testing local optimum 1/2 at purity 0.65 and total ploidy 1.80...
INFO [2023-02-02 17:01:19] Testing local optimum 2/2 at purity 0.38 and total ploidy 2.20...
INFO [2023-02-02 17:01:21] Skipping 1 solutions exceeding max.non.clonal (0.20): 0.65/2.05 (purity/tumor ploidy)
INFO [2023-02-02 17:01:21] Fitting variants with beta model for local optimum 2/2...
INFO [2023-02-02 17:01:21] Fitting variants for purity 0.40, tumor ploidy 2.55 and contamination 0.01.
INFO [2023-02-02 17:01:25] Optimized purity: 0.40
INFO [2023-02-02 17:01:25] Done.
INFO [2023-02-02 17:01:25] ------------------------------------------------------------
FATAL [2023-02-02 17:01:25] This function requires gene-level calls. Please add a column &#x27;Gene&#x27; 

FATAL [2023-02-02 17:01:25] containing gene symbols to the interval.file. 

FATAL [2023-02-02 17:01:25]  

FATAL [2023-02-02 17:01:25] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:01:25] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:01:28] ------------------------------------------------------------
INFO [2023-02-02 17:01:28] PureCN 2.5.0
INFO [2023-02-02 17:01:28] ------------------------------------------------------------
INFO [2023-02-02 17:01:28] Loading coverage files...
INFO [2023-02-02 17:01:28] Mean target coverages: 112X (tumor) 99X (normal).
WARN [2023-02-02 17:01:29] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:01:29] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 17:01:29] Removing 228 intervals with missing log.ratio.
INFO [2023-02-02 17:01:29] Removing 22 small (&lt; 5bp) intervals.
INFO [2023-02-02 17:01:29] Removing 19 intervals with low total coverage in normal (&lt; 150.00 reads).
WARN [2023-02-02 17:01:29] No normalDB provided. Provide one for better results.
INFO [2023-02-02 17:01:29] Removing 1724 low count (&lt; 100 total reads) intervals.
INFO [2023-02-02 17:01:29] Removing 6 low coverage (&lt; 15.0000X) intervals.
INFO [2023-02-02 17:01:29] Using 8050 intervals (8050 on-target, 0 off-target).
INFO [2023-02-02 17:01:29] No off-target intervals. If this is hybrid-capture data, consider adding them.
INFO [2023-02-02 17:01:29] No interval.file provided. Cannot check for any GC-biases.
INFO [2023-02-02 17:01:29] Loading VCF...
INFO [2023-02-02 17:01:29] Found 127 variants in VCF file.
INFO [2023-02-02 17:01:29] 127 (100.0%) variants annotated as likely germline (DB INFO flag).
FATAL [2023-02-02 17:01:29] Different chromosome names in coverage and VCF. 

FATAL [2023-02-02 17:01:29]  

FATAL [2023-02-02 17:01:29] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:01:29] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:01:29] ------------------------------------------------------------
INFO [2023-02-02 17:01:29] PureCN 2.5.0
INFO [2023-02-02 17:01:29] ------------------------------------------------------------
INFO [2023-02-02 17:01:29] Loading coverage files...
INFO [2023-02-02 17:01:29] Mean target coverages: 112X (tumor) 99X (normal).
WARN [2023-02-02 17:01:29] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:01:29] Allosome coverage missing, cannot determine sex.
FATAL [2023-02-02 17:01:30] tumor.coverage.file and interval.file do not align. 

FATAL [2023-02-02 17:01:30]  

FATAL [2023-02-02 17:01:30] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:01:30] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:01:30] ------------------------------------------------------------
INFO [2023-02-02 17:01:30] PureCN 2.5.0
INFO [2023-02-02 17:01:30] ------------------------------------------------------------
INFO [2023-02-02 17:01:30] Loading coverage files...
INFO [2023-02-02 17:01:30] Mean target coverages: 112X (tumor) 99X (normal).
WARN [2023-02-02 17:01:30] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:01:30] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 17:01:32] Removing 228 intervals with missing log.ratio.
INFO [2023-02-02 17:01:32] Removing 15 low/high GC targets.
INFO [2023-02-02 17:01:32] Removing 21 small (&lt; 5bp) intervals.
INFO [2023-02-02 17:01:32] Removing 19 intervals with low total coverage in normal (&lt; 150.00 reads).
WARN [2023-02-02 17:01:32] No normalDB provided. Provide one for better results.
INFO [2023-02-02 17:01:32] Removing 1718 low count (&lt; 100 total reads) intervals.
INFO [2023-02-02 17:01:32] Removing 6 low coverage (&lt; 15.0000X) intervals.
INFO [2023-02-02 17:01:32] Removing 36 low mappability intervals.
INFO [2023-02-02 17:01:32] Using 8006 intervals (8006 on-target, 0 off-target).
INFO [2023-02-02 17:01:32] No off-target intervals. If this is hybrid-capture data, consider adding them.
INFO [2023-02-02 17:01:32] AT/GC dropout: 1.01 (tumor), 1.03 (normal), 0.98 (coverage log-ratio). 
INFO [2023-02-02 17:01:32] Loading VCF...
INFO [2023-02-02 17:01:32] Found 127 variants in VCF file.
INFO [2023-02-02 17:01:32] 127 (100.0%) variants annotated as likely germline (DB INFO flag).
INFO [2023-02-02 17:01:33] LIB-02240e4 is tumor in VCF file.
INFO [2023-02-02 17:01:33] No homozygous variants in VCF, provide unfiltered VCF.
INFO [2023-02-02 17:01:33] Removing 1 non heterozygous (in matched normal) germline SNPs.
INFO [2023-02-02 17:01:33] Removing 0 low quality variants with non-offset BQ &lt; 25.
INFO [2023-02-02 17:01:33] Base quality scores range from 29 to 35 (offset by 1)
INFO [2023-02-02 17:01:33] Minimum number of supporting reads ranges from 2 to 5, depending on coverage and BQS.
INFO [2023-02-02 17:01:33] Removing 1 variants with AF &lt; 0.030 or AF &gt;= 1.000 or insufficient supporting reads or depth &lt; 15.
INFO [2023-02-02 17:01:33] Total size of targeted genomic region: 1.51Mb (2.31Mb with 50bp padding).
INFO [2023-02-02 17:01:34] 1.2% of targets contain variants.
INFO [2023-02-02 17:01:34] Removing 11 variants outside intervals.
INFO [2023-02-02 17:01:34] Found SOMATIC annotation in VCF.
INFO [2023-02-02 17:01:34] Setting somatic prior probabilities for somatic variants to 0.999000 or to 0.000100 otherwise.
INFO [2023-02-02 17:01:34] Found SOMATIC annotation in VCF. Setting mapping bias to 0.970.
INFO [2023-02-02 17:01:34] Excluding 0 novel or poor quality variants from segmentation.
INFO [2023-02-02 17:01:34] Sample sex: ?
INFO [2023-02-02 17:01:34] Segmenting data...
INFO [2023-02-02 17:01:34] Loading pre-computed boundaries for DNAcopy...
INFO [2023-02-02 17:01:34] Setting undo.SD parameter to 1.000000.
INFO [2023-02-02 17:01:35] Setting prune.hclust.h parameter to 0.200000.
INFO [2023-02-02 17:01:36] Found 59 segments with median size of 17.67Mb.
INFO [2023-02-02 17:01:36] Using 114 variants.
INFO [2023-02-02 17:01:36] Mean standard deviation of log-ratios: 0.34 (MAPD: 0.30)
INFO [2023-02-02 17:01:36] 2D-grid search of purity and ploidy...
INFO [2023-02-02 17:01:37] Local optima: 0.63/1.9
INFO [2023-02-02 17:01:37] Testing local optimum 1/1 at purity 0.63 and total ploidy 1.90...
INFO [2023-02-02 17:01:38] Fitting variants with beta model for local optimum 1/1...
INFO [2023-02-02 17:01:38] Fitting variants for purity 0.65, tumor ploidy 1.75 and contamination 0.01.
INFO [2023-02-02 17:01:42] Optimized purity: 0.65
INFO [2023-02-02 17:01:42] Done.
INFO [2023-02-02 17:01:42] ------------------------------------------------------------
INFO [2023-02-02 17:01:42] ------------------------------------------------------------
INFO [2023-02-02 17:01:42] PureCN 2.5.0
INFO [2023-02-02 17:01:42] ------------------------------------------------------------
INFO [2023-02-02 17:01:42] Loading coverage files...
INFO [2023-02-02 17:01:42] Mean target coverages: 112X (tumor) 99X (normal).
WARN [2023-02-02 17:01:42] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:01:42] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 17:01:44] Removing 228 intervals with missing log.ratio.
INFO [2023-02-02 17:01:44] Removing 15 low/high GC targets.
INFO [2023-02-02 17:01:44] Removing 21 small (&lt; 5bp) intervals.
INFO [2023-02-02 17:01:44] Removing 19 intervals with low total coverage in normal (&lt; 150.00 reads).
WARN [2023-02-02 17:01:44] No normalDB provided. Provide one for better results.
INFO [2023-02-02 17:01:44] Removing 1718 low count (&lt; 100 total reads) intervals.
INFO [2023-02-02 17:01:44] Removing 6 low coverage (&lt; 15.0000X) intervals.
INFO [2023-02-02 17:01:44] Removing 36 low mappability intervals.
INFO [2023-02-02 17:01:44] Using 8006 intervals (8006 on-target, 0 off-target).
INFO [2023-02-02 17:01:44] No off-target intervals. If this is hybrid-capture data, consider adding them.
INFO [2023-02-02 17:01:44] AT/GC dropout: 1.01 (tumor), 1.03 (normal), 0.98 (coverage log-ratio). 
INFO [2023-02-02 17:01:44] Loading VCF...
INFO [2023-02-02 17:01:44] Found 127 variants in VCF file.
INFO [2023-02-02 17:01:44] 127 (100.0%) variants annotated as likely germline (DB INFO flag).
INFO [2023-02-02 17:01:44] LIB-02240e4 is tumor in VCF file.
INFO [2023-02-02 17:01:44] No homozygous variants in VCF, provide unfiltered VCF.
INFO [2023-02-02 17:01:45] Removing 1 non heterozygous (in matched normal) germline SNPs.
INFO [2023-02-02 17:01:45] Removing 0 low quality variants with non-offset BQ &lt; 25.
INFO [2023-02-02 17:01:45] Base quality scores range from 29 to 35 (offset by 1)
INFO [2023-02-02 17:01:45] Minimum number of supporting reads ranges from 2 to 5, depending on coverage and BQS.
INFO [2023-02-02 17:01:45] Removing 1 variants with AF &lt; 0.030 or AF &gt;= 1.000 or insufficient supporting reads or depth &lt; 15.
INFO [2023-02-02 17:01:45] Total size of targeted genomic region: 1.51Mb (2.31Mb with 50bp padding).
INFO [2023-02-02 17:01:45] 1.2% of targets contain variants.
INFO [2023-02-02 17:01:46] Removing 11 variants outside intervals.
INFO [2023-02-02 17:01:46] Found SOMATIC annotation in VCF.
INFO [2023-02-02 17:01:46] Setting somatic prior probabilities for somatic variants to 0.999000 or to 0.000100 otherwise.
INFO [2023-02-02 17:01:46] Found SOMATIC annotation in VCF. Setting mapping bias to 0.970.
INFO [2023-02-02 17:01:46] Excluding 0 novel or poor quality variants from segmentation.
INFO [2023-02-02 17:01:46] Sample sex: ?
INFO [2023-02-02 17:01:46] Segmenting data...
INFO [2023-02-02 17:01:46] Loading pre-computed boundaries for DNAcopy...
INFO [2023-02-02 17:01:46] Setting undo.SD parameter to 1.000000.
INFO [2023-02-02 17:01:47] Setting prune.hclust.h parameter to 0.200000.
INFO [2023-02-02 17:01:47] Found 59 segments with median size of 17.67Mb.
INFO [2023-02-02 17:01:47] Using 114 variants.
INFO [2023-02-02 17:01:47] Mean standard deviation of log-ratios: 0.34 (MAPD: 0.30)
INFO [2023-02-02 17:01:47] 2D-grid search of purity and ploidy...
INFO [2023-02-02 17:01:49] Local optima: 0.63/1.9
INFO [2023-02-02 17:01:49] Testing local optimum 1/1 at purity 0.63 and total ploidy 1.90...
INFO [2023-02-02 17:01:50] Fitting variants with beta model for local optimum 1/1...
INFO [2023-02-02 17:01:50] Fitting variants for purity 0.65, tumor ploidy 1.75 and contamination 0.01.
INFO [2023-02-02 17:01:54] Optimized purity: 0.65
INFO [2023-02-02 17:01:54] Done.
INFO [2023-02-02 17:01:54] ------------------------------------------------------------
FATAL [2023-02-02 17:01:54] chr1 not valid chromosome name(s). Valid names are: 

FATAL [2023-02-02 17:01:54] 1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22 

FATAL [2023-02-02 17:01:54]  

FATAL [2023-02-02 17:01:54] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:01:54] parameters (PureCN 2.5.0). 

INFO [2023-02-02 17:01:54] ------------------------------------------------------------
INFO [2023-02-02 17:01:54] PureCN 2.5.0
INFO [2023-02-02 17:01:54] ------------------------------------------------------------
INFO [2023-02-02 17:01:54] Loading coverage files...
WARN [2023-02-02 17:01:54] Provided sampleid (Sample2) does not match Sample1 found in segmentation.
WARN [2023-02-02 17:01:54] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:01:54] Allosome coverage missing, cannot determine sex.
INFO [2023-02-02 17:01:55] Removing 10 intervals with missing log.ratio.
INFO [2023-02-02 17:01:55] Using 10039 intervals (10039 on-target, 0 off-target).
INFO [2023-02-02 17:01:55] No off-target intervals. If this is hybrid-capture data, consider adding them.
INFO [2023-02-02 17:01:55] Loading VCF...
INFO [2023-02-02 17:01:55] Found 127 variants in VCF file.
INFO [2023-02-02 17:01:55] 127 (100.0%) variants annotated as likely germline (DB INFO flag).
INFO [2023-02-02 17:01:55] LIB-02240e4 is tumor in VCF file.
INFO [2023-02-02 17:01:55] No homozygous variants in VCF, provide unfiltered VCF.
INFO [2023-02-02 17:01:55] Removing 1 non heterozygous (in matched normal) germline SNPs.
INFO [2023-02-02 17:01:56] Removing 0 low quality variants with non-offset BQ &lt; 25.
INFO [2023-02-02 17:01:56] Base quality scores range from 29 to 35 (offset by 1)
INFO [2023-02-02 17:01:56] Minimum number of supporting reads ranges from 2 to 5, depending on coverage and BQS.
INFO [2023-02-02 17:01:56] Removing 1 variants with AF &lt; 0.030 or AF &gt;= 1.000 or insufficient supporting reads or depth &lt; 15.
INFO [2023-02-02 17:01:56] Total size of targeted genomic region: 1.71Mb (2.70Mb with 50bp padding).
INFO [2023-02-02 17:01:56] 1.0% of targets contain variants.
INFO [2023-02-02 17:01:56] Removing 0 variants outside intervals.
INFO [2023-02-02 17:01:56] Found SOMATIC annotation in VCF.
INFO [2023-02-02 17:01:56] Setting somatic prior probabilities for somatic variants to 0.999000 or to 0.000100 otherwise.
INFO [2023-02-02 17:01:57] Found SOMATIC annotation in VCF. Setting mapping bias to 0.976.
INFO [2023-02-02 17:01:57] Excluding 0 novel or poor quality variants from segmentation.
INFO [2023-02-02 17:01:57] Sample sex: ?
INFO [2023-02-02 17:01:57] Segmenting data...
INFO [2023-02-02 17:01:57] Loaded provided segmentation file example_seg.txt (format DNAcopy).
WARN [2023-02-02 17:01:57] Provided sampleid (Sample2) does not match Sample1 found in segmentation.
INFO [2023-02-02 17:01:57] Re-centering provided segment means (offset -0.0033).
INFO [2023-02-02 17:01:57] Loading pre-computed boundaries for DNAcopy...
INFO [2023-02-02 17:01:57] Setting undo.SD parameter to 0.000000.
Setting multi-figure configuration
INFO [2023-02-02 17:01:57] Setting prune.hclust.h parameter to 0.200000.
INFO [2023-02-02 17:01:57] Found 54 segments with median size of 24.88Mb.
INFO [2023-02-02 17:01:57] Using 125 variants.
INFO [2023-02-02 17:01:58] Mean standard deviation of log-ratios: 0.40 (MAPD: 0.00)
INFO [2023-02-02 17:01:58] 2D-grid search of purity and ploidy...
INFO [2023-02-02 17:01:59] Local optima: 0.6/1.9
INFO [2023-02-02 17:01:59] Testing local optimum 1/1 at purity 0.60 and total ploidy 1.90...
INFO [2023-02-02 17:02:00] Fitting variants with beta model for local optimum 1/1...
INFO [2023-02-02 17:02:00] Fitting variants for purity 0.65, tumor ploidy 1.74 and contamination 0.01.
INFO [2023-02-02 17:02:04] Optimized purity: 0.65
INFO [2023-02-02 17:02:04] Done.
INFO [2023-02-02 17:02:04] ------------------------------------------------------------
WARN [2023-02-02 17:02:05] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:02:05] Allosome coverage missing, cannot determine sex.
Setting multi-figure configuration
FATAL [2023-02-02 17:02:16] Segmentation file expected with colnames ID, chrom, loc.start, loc.end, 

FATAL [2023-02-02 17:02:16] num.mark, seg.mean 

FATAL [2023-02-02 17:02:16]  

FATAL [2023-02-02 17:02:16] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:02:16] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 17:02:16] seg.file contains multiple samples and sampleid missing. 

FATAL [2023-02-02 17:02:16]  

FATAL [2023-02-02 17:02:16] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:02:16] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 17:02:16] seg.file contains multiple samples and sampleid does not match any. 

FATAL [2023-02-02 17:02:16]  

FATAL [2023-02-02 17:02:16] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:02:16] parameters (PureCN 2.5.0). 

WARN [2023-02-02 17:02:17] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:02:17] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:02:28] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:02:28] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:02:28] Provided sampleid (example_tumor.txt.gz) does not match Sample1 found in segmentation.
WARN [2023-02-02 17:02:33] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:02:33] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:02:35] No normalDB provided. Provide one for better results.
WARN [2023-02-02 17:02:45] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:02:45] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:02:49] Provided sampleid (Sample.1) does not match Sample1 found in segmentation.
Setting multi-figure configuration
WARN [2023-02-02 17:02:59] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:02:59] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:03:00] No normalDB provided. Provide one for better results.
WARN [2023-02-02 17:03:00] Sampleid looks like a normal in VCF, not like a tumor.
WARN [2023-02-02 17:03:10] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:03:10] Allosome coverage missing, cannot determine sex.
FATAL [2023-02-02 17:03:10] normalDB not a valid normalDB object. Use createNormalDatabase to 

FATAL [2023-02-02 17:03:10] create one. 

FATAL [2023-02-02 17:03:10]  

FATAL [2023-02-02 17:03:10] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:03:10] parameters (PureCN 2.5.0). 

WARN [2023-02-02 17:03:11] You are likely not using the correct baits file!
WARN [2023-02-02 17:03:11] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:03:11] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:03:15] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:03:15] Allosome coverage missing, cannot determine sex.
FATAL [2023-02-02 17:03:15] normalDB appears to be empty. 

FATAL [2023-02-02 17:03:15]  

FATAL [2023-02-02 17:03:15] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:03:15] parameters (PureCN 2.5.0). 

WARN [2023-02-02 17:03:17] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:03:17] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:03:17] Intervals in coverage and interval.file have conflicting on/off-target annotation.
WARN [2023-02-02 17:03:32] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:03:32] Allosome coverage missing, cannot determine sex.
FATAL [2023-02-02 17:03:32] Seqlevels missing in provided segmentation: 6 

FATAL [2023-02-02 17:03:32]  

FATAL [2023-02-02 17:03:32] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:03:32] parameters (PureCN 2.5.0). 

sh: 1: gatk: not found
WARN [2023-02-02 17:03:39] Cannot find gatk binary in path.
WARN [2023-02-02 17:03:39] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:03:39] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:03:40] No normalDB provided. Provide one for better results.
FATAL [2023-02-02 17:03:40] segmentationHclust requires an input segmentation. 

FATAL [2023-02-02 17:03:40]  

FATAL [2023-02-02 17:03:40] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:03:40] parameters (PureCN 2.5.0). 

WARN [2023-02-02 17:03:40] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:03:40] Allosome coverage missing, cannot determine sex.
WARN [2023-02-02 17:03:41] No normalDB provided. Provide one for better results.
FATAL [2023-02-02 17:04:29] The normal.panel.vcf.file contains only a single sample. 

FATAL [2023-02-02 17:04:29]  

FATAL [2023-02-02 17:04:29] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:04:29] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 17:04:29] mapping.bias.file must be a file with *.rds suffix. 

FATAL [2023-02-02 17:04:29]  

FATAL [2023-02-02 17:04:29] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:04:29] parameters (PureCN 2.5.0). 

Failed with error:  &#x27;there is no package called &#x27;genomicsdb&#x27;&#x27;
FATAL [2023-02-02 17:04:31] min.normals (0) must be &gt;= 1. 

FATAL [2023-02-02 17:04:31]  

FATAL [2023-02-02 17:04:31] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:04:31] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 17:04:32] min.normals (10) cannot be larger than min.normals.assign.betafit (3). 

FATAL [2023-02-02 17:04:32]  

FATAL [2023-02-02 17:04:32] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:04:32] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 17:04:32] min.normals.assign.betafit (10) cannot be larger than 

FATAL [2023-02-02 17:04:32] min.normals.betafit (7). 

FATAL [2023-02-02 17:04:32]  

FATAL [2023-02-02 17:04:32] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:04:32] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 17:04:32] min.normals.betafit (20) cannot be larger than 

FATAL [2023-02-02 17:04:32] min.normals.position.specific.fit (10). 

FATAL [2023-02-02 17:04:32]  

FATAL [2023-02-02 17:04:32] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:04:32] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 17:04:33] min.betafit.rho not within expected range or format. 

FATAL [2023-02-02 17:04:33]  

FATAL [2023-02-02 17:04:33] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:04:33] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 17:04:33] max.betafit.rho not within expected range or format. 

FATAL [2023-02-02 17:04:33]  

FATAL [2023-02-02 17:04:33] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:04:33] parameters (PureCN 2.5.0). 

FATAL [2023-02-02 17:04:33] min.betafit.rho (0.9) cannot be larger than max.betafit.rho (0.2). 

FATAL [2023-02-02 17:04:33]  

FATAL [2023-02-02 17:04:33] This is most likely a user error due to invalid input data or 

FATAL [2023-02-02 17:04:33] parameters (PureCN 2.5.0). 

[ FAIL 0 | WARN 13 | SKIP 2 | PASS 397 ]

══ Skipped tests ═══════════════════════════════════════════════════════════════
• gatk binary &gt; 4.1.7.0 required (1)
• genomicsdb required (1)

[ FAIL 0 | WARN 13 | SKIP 2 | PASS 397 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
662.926   6.668 627.444 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">PureCN.Rcheck/PureCN-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>annotateTargets</TD><TD style="text-align: right;"> 9.751</TD><TD style="text-align: right;"> 0.251</TD><TD style="text-align: right;">10.031</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bootstrapResults</TD><TD style="text-align: right;">0.815</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.856</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>calculateBamCoverageByInterval</TD><TD style="text-align: right;">0.343</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.351</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>calculateLogRatio</TD><TD style="text-align: right;">0.873</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.887</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>calculateMappingBiasGatk4</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>calculateMappingBiasVcf</TD><TD style="text-align: right;">2.971</TD><TD style="text-align: right;">0.124</TD><TD style="text-align: right;">3.102</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>calculatePowerDetectSomatic</TD><TD style="text-align: right;">2.004</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">2.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>calculateTangentNormal</TD><TD style="text-align: right;">5.933</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">5.953</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>callAlterations</TD><TD style="text-align: right;">0.127</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.128</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>callAlterationsFromSegmentation</TD><TD style="text-align: right;">2.522</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">2.560</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>callAmplificationsInLowPurity</TD><TD style="text-align: right;">64.479</TD><TD style="text-align: right;"> 0.020</TD><TD style="text-align: right;">64.658</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>callCIN</TD><TD style="text-align: right;">0.272</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.276</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>callLOH</TD><TD style="text-align: right;">0.235</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.235</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>callMutationBurden</TD><TD style="text-align: right;">2.511</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">2.526</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>centromeres</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>correctCoverageBias</TD><TD style="text-align: right;">2.810</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">2.817</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>createCurationFile</TD><TD style="text-align: right;">0.330</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.338</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>createNormalDatabase</TD><TD style="text-align: right;">4.053</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">4.068</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filterIntervals</TD><TD style="text-align: right;">30.180</TD><TD style="text-align: right;"> 0.040</TD><TD style="text-align: right;">30.247</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filterVcfBasic</TD><TD style="text-align: right;">1.062</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.064</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filterVcfMuTect</TD><TD style="text-align: right;">1.006</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">1.011</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filterVcfMuTect2</TD><TD style="text-align: right;">1.064</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.064</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findFocal</TD><TD style="text-align: right;">16.373</TD><TD style="text-align: right;"> 0.008</TD><TD style="text-align: right;">16.396</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getSexFromCoverage</TD><TD style="text-align: right;">0.377</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.378</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getSexFromVcf</TD><TD style="text-align: right;">0.44</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.44</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotAbs</TD><TD style="text-align: right;">0.262</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.271</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>poolCoverage</TD><TD style="text-align: right;">0.936</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.940</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>predictSomatic</TD><TD style="text-align: right;">0.730</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.743</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>preprocessIntervals</TD><TD style="text-align: right;">1.109</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.113</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>processMultipleSamples</TD><TD style="text-align: right;">7.093</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">7.116</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readAllelicCountsFile</TD><TD style="text-align: right;">1.092</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.094</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readCoverageFile</TD><TD style="text-align: right;">0.414</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.419</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readCurationFile</TD><TD style="text-align: right;">0.220</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.225</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readIntervalFile</TD><TD style="text-align: right;">0.258</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.262</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readLogRatioFile</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readSegmentationFile</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>runAbsoluteCN</TD><TD style="text-align: right;">26.503</TD><TD style="text-align: right;"> 0.021</TD><TD style="text-align: right;">26.531</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>segmentationCBS</TD><TD style="text-align: right;">12.753</TD><TD style="text-align: right;"> 0.008</TD><TD style="text-align: right;">12.927</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>segmentationGATK4</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>segmentationHclust</TD><TD style="text-align: right;">24.585</TD><TD style="text-align: right;"> 0.012</TD><TD style="text-align: right;">24.824</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>segmentationPSCBS</TD><TD style="text-align: right;">42.919</TD><TD style="text-align: right;"> 0.055</TD><TD style="text-align: right;">43.356</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>setMappingBiasVcf</TD><TD style="text-align: right;">0.514</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.515</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>setPriorVcf</TD><TD style="text-align: right;">0.469</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.469</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
