<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for ReUseData on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><B>[<A href="..#R">R</A>]</B></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-16 06:07:02 -0000 (Tue, 16 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4787</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for ReUseData on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the ReUseData package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/ReUseData.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1665</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">ReUseData</A>&nbsp;1.1.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/ReUseData">landing page</A>)</SPAN><BR>Qian Liu<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-13&nbsp;23:12:22&nbsp;-0000&nbsp;(Sat,&nbsp;13&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/ReUseData</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">3a1df6c</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;15:42:19&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: ReUseData</TD></TR>
<TR><TD><B>Version</B>: 1.1.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:ReUseData.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings ReUseData_1.1.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-15 11:56:57 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-15 12:00:24 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 207.3 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: ReUseData.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:ReUseData.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings ReUseData_1.1.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/ReUseData.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘ReUseData/DESCRIPTION’ ... OK
* this is package ‘ReUseData’ version ‘1.1.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .BBSoptions
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘ReUseData’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                user system elapsed
dataUpdate    15.999  1.073  17.072
getCloudData  10.686  0.512  15.826
dataHub-class  9.039  0.948   9.988
getData        4.944  0.254   5.192
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘ReUseData_data.Rmd’ using ‘UTF-8’... OK
  ‘ReUseData_quickStart.Rmd’ using ‘UTF-8’... OK
  ‘ReUseData_recipe.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... ERROR
Error(s) in re-building vignettes:
  ...
--- re-building ‘ReUseData_data.Rmd’ using rmarkdown
Error: processing vignette &#x27;ReUseData_data.Rmd&#x27; failed with diagnostics:
there is no package called ‘BiocStyle’
--- failed re-building ‘ReUseData_data.Rmd’

--- re-building ‘ReUseData_quickStart.Rmd’ using rmarkdown
Error: processing vignette &#x27;ReUseData_quickStart.Rmd&#x27; failed with diagnostics:
there is no package called ‘BiocStyle’
--- failed re-building ‘ReUseData_quickStart.Rmd’

--- re-building ‘ReUseData_recipe.Rmd’ using rmarkdown
Error: processing vignette &#x27;ReUseData_recipe.Rmd&#x27; failed with diagnostics:
there is no package called ‘BiocStyle’
--- failed re-building ‘ReUseData_recipe.Rmd’

SUMMARY: processing the following files failed:
  ‘ReUseData_data.Rmd’ ‘ReUseData_quickStart.Rmd’
  ‘ReUseData_recipe.Rmd’

Error: Vignette re-building failed.
Execution halted

* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 1 NOTE
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/ReUseData.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">ReUseData.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD INSTALL ReUseData
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0-bin/site-library’
* installing *source* package ‘ReUseData’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (ReUseData)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">ReUseData.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(ReUseData)
&gt; 
&gt; test_check(&quot;ReUseData&quot;)
Note: you need to assign a name for the recipe: rcpName &lt;- recipeLoad(&#x27;xx&#x27;)
Data recipe loaded!
Use inputs() to check required input parameters before evaluation.
Check here: https://rcwl.org/dataRecipes/echo_out.html
for user instructions (e.g., eligible input values, data source, etc.)

}[1;30mINFO[0m Final process status is success

Updating data record...
outfile.txt added

Updating data record...
3c3ec06825096d_GRCh38.primary_assembly.genome.fa.1.bt2 added
3c3ec0457b0075_GRCh38.primary_assembly.genome.fa.2.bt2 added
3c3ec0631a932_GRCh38.primary_assembly.genome.fa.3.bt2 added
3c3ec036c3827c_GRCh38.primary_assembly.genome.fa.4.bt2 added
3c3ec07ea4bf97_GRCh38.primary_assembly.genome.fa.rev.1.bt2 added
3c3ec0377f511b_GRCh38.primary_assembly.genome.fa.rev.2.bt2 added
3c3ec02be8329c_outfile.txt added
3c3ec036f22e13_GRCh37_to_GRCh38.chain added
3c3ec045d1cb6d_GRCh37_to_NCBI34.chain added
3c3ec0d693bee_GRCh37_to_NCBI35.chain added
3c3ec01eb22b20_GRCh37_to_NCBI36.chain added
3c3ec02818420_GRCh38_to_GRCh37.chain added
3c3ec025a5daf1_GRCh38_to_NCBI34.chain added
3c3ec0604510d8_GRCh38_to_NCBI35.chain added
3c3ec040f6d6e3_GRCh38_to_NCBI36.chain added
3c3ec06cb30ce9_NCBI34_to_GRCh37.chain added
3c3ec059927a78_NCBI34_to_GRCh38.chain added
3c3ec021fc36e7_NCBI35_to_GRCh37.chain added
3c3ec07250d6a6_NCBI35_to_GRCh38.chain added
3c3ec032994dac_NCBI36_to_GRCh37.chain added
3c3ec010e31819_NCBI36_to_GRCh38.chain added
3c3ec0bb2dea3_GRCm38_to_NCBIM36.chain added
3c3ec042b13b5f_GRCm38_to_NCBIM37.chain added
3c3ec064566766_NCBIM36_to_GRCm38.chain added
3c3ec07a906302_NCBIM37_to_GRCm38.chain added
3c3ec07c51e049_1000G_omni2.5.b37.vcf.gz added
3c3ec0192d94a3_1000G_omni2.5.b37.vcf.gz.tbi added
3c3ec016bd7ccc_Mills_and_1000G_gold_standard.indels.b37.vcf.gz added
3c3ec017509247_Mills_and_1000G_gold_standard.indels.b37.vcf.gz.tbi added
3c3ec06aef6173_1000G_omni2.5.hg38.vcf.gz added
3c3ec079cacd32_1000G_omni2.5.hg38.vcf.gz.tbi added
3c3ec07f759bb4_Mills_and_1000G_gold_standard.indels.hg38.vcf.gz added
3c3ec0306a61e8_Mills_and_1000G_gold_standard.indels.hg38.vcf.gz.tbi added
3c3ec07ffc7664_af-only-gnomad.raw.sites.vcf added
3c3ec036391e30_af-only-gnomad.raw.sites.vcf.idx added
3c3ec02f0f217f_Mutect2-exome-panel.vcf.idx added
3c3ec0377bc780_Mutect2-WGS-panel-b37.vcf added
3c3ec0622150cd_Mutect2-WGS-panel-b37.vcf.idx added
3c3ec066014f93_small_exac_common_3.vcf added
3c3ec07d4d92ed_small_exac_common_3.vcf.idx added
3c3ec06f8a8cbb_1000g_pon.hg38.vcf.gz added
3c3ec04b37ab3_1000g_pon.hg38.vcf.gz.tbi added
3c3ec07fcf170d_af-only-gnomad.hg38.vcf.gz added
3c3ec0153067ad_af-only-gnomad.hg38.vcf.gz.tbi added
3c3ec064f88b8b_small_exac_common_3.hg38.vcf.gz added
3c3ec040c5edf0_small_exac_common_3.hg38.vcf.gz.tbi added
3c3ec01e37496_gencode.v41.annotation.gtf added
3c3ec03e8b0603_gencode.v42.annotation.gtf added
3c3ec062c224d7_gencode.vM30.annotation.gtf added
3c3ec074344b3c_gencode.vM31.annotation.gtf added
3c3ec0712453af_gencode.v41.transcripts.fa added
3c3ec073a53cf1_gencode.v41.transcripts.fa.fai added
3c3ec07fe729df_gencode.v42.transcripts.fa added
3c3ec033d58f0f_gencode.v42.transcripts.fa.fai added
3c3ec057fba457_gencode.vM30.pc_transcripts.fa added
3c3ec07a778ce2_gencode.vM30.pc_transcripts.fa.fai added
3c3ec030276f58_gencode.vM31.pc_transcripts.fa added
3c3ec0712938fa_gencode.vM31.pc_transcripts.fa.fai added
3c3ec0113509ae_GRCh38.primary_assembly.genome.fa.1.ht2 added
3c3ec04778019f_GRCh38.primary_assembly.genome.fa.2.ht2 added
3c3ec05c189a6e_GRCh38.primary_assembly.genome.fa.3.ht2 added
3c3ec0affd6e0_GRCh38.primary_assembly.genome.fa.4.ht2 added
3c3ec046ed9d53_GRCh38.primary_assembly.genome.fa.5.ht2 added
3c3ec0c82fc56_GRCh38.primary_assembly.genome.fa.6.ht2 added
3c3ec0afc4d44_GRCh38.primary_assembly.genome.fa.7.ht2 added
3c3ec07d26bb83_GRCh38.primary_assembly.genome.fa.8.ht2 added
3c3ec03b921dd6_GRCh38_full_analysis_set_plus_decoy_hla.fa.fai added
3c3ec0427814c4_GRCh38_full_analysis_set_plus_decoy_hla.fa.amb added
3c3ec05f480c50_GRCh38_full_analysis_set_plus_decoy_hla.fa.ann added
3c3ec021936d69_GRCh38_full_analysis_set_plus_decoy_hla.fa.bwt added
3c3ec03fc5a7b2_GRCh38_full_analysis_set_plus_decoy_hla.fa.pac added
3c3ec04ed2990c_GRCh38_full_analysis_set_plus_decoy_hla.fa.sa added
3c3ec02646e81c_GRCh38_full_analysis_set_plus_decoy_hla.fa added
3c3ec03f94bebf_GRCh38.primary_assembly.genome.fa.fai added
3c3ec0640300b9_GRCh38.primary_assembly.genome.fa.amb added
3c3ec0b3f73a7_GRCh38.primary_assembly.genome.fa.ann added
3c3ec05aacb0_GRCh38.primary_assembly.genome.fa.bwt added
3c3ec065e6754f_GRCh38.primary_assembly.genome.fa.pac added
3c3ec049ca79aa_GRCh38.primary_assembly.genome.fa.sa added
3c3ec0631cd187_GRCh38.primary_assembly.genome.fa added
3c3ec05a1ac08b_hs37d5.fa.fai added
3c3ec03aeecd5a_hs37d5.fa.amb added
3c3ec056c20e78_hs37d5.fa.ann added
3c3ec05a01ea6a_hs37d5.fa.bwt added
3c3ec06ec45c69_hs37d5.fa.pac added
3c3ec02ebdb2d0_hs37d5.fa.sa added
3c3ec05479774c_hs37d5.fa added
3c3ec01eebcbc1_complete_ref_lens.bin added
3c3ec01fe6ebca_ctable.bin added
3c3ec065ae80fa_ctg_offsets.bin added
3c3ec06663cd60_duplicate_clusters.tsv added
3c3ec07bff8638_info.json added
3c3ec070ae57da_mphf.bin added
3c3ec02d516ab3_pos.bin added
3c3ec0882828f_pre_indexing.log added
3c3ec07baaa51f_rank.bin added
3c3ec02a782636_ref_indexing.log added
3c3ec04414a065_refAccumLengths.bin added
3c3ec03e22b9e3_reflengths.bin added
3c3ec09c03287_refseq.bin added
3c3ec065a80dce_seq.bin added
3c3ec07de86195_versionInfo.json added
3c3ec05892cb93_salmon_index added
3c3ec0beef5ea_chrLength.txt added
3c3ec03d7d2055_chrName.txt added
3c3ec03c95cc4c_chrNameLength.txt added
3c3ec0172e6991_chrStart.txt added
3c3ec03dd7cd05_exonGeTrInfo.tab added
3c3ec0227c419b_exonInfo.tab added
3c3ec060f8e33b_geneInfo.tab added
3c3ec020f49e8c_Genome added
3c3ec07c970226_genomeParameters.txt added
3c3ec01be7b095_Log.out added
3c3ec077b6ad05_SA added
3c3ec05698ec90_SAindex added
3c3ec0aac0cfe_sjdbInfo.txt added
3c3ec026745fd5_sjdbList.fromGTF.out.tab added
3c3ec02b1263dd_sjdbList.out.tab added
3c3ec02997d8bf_transcriptInfo.tab added
3c3ec0465b4b9f_GRCh38.GENCODE.v42_100 added
3c3ec010c0e4d7_knownGene_hg38.sql added
3c3ec0ffba61f_knownGene_hg38.txt added
3c3ec0425ad1d8_refGene_hg38.sql added
3c3ec016f3cb2_refGene_hg38.txt added
3c3ec03d4d10d2_knownGene_mm39.sql added
3c3ec04add5467_knownGene_mm39.txt added
3c3ec07d19e1d1_refGene_mm39.sql added
3c3ec067c53709_refGene_mm39.txt added
trying URL &#x27;https://storage.googleapis.com/reusedata/echo/outfile.txt&#x27;
Content type &#x27;text/plain&#x27; length 30 bytes
==================================================
downloaded 30 bytes

trying URL &#x27;https://storage.googleapis.com/reusedata/echo/echo_out_Hello_World!_outfile.yml&#x27;
Content type &#x27;application/x-yaml&#x27; length 151 bytes
==================================================
downloaded 151 bytes

trying URL &#x27;https://storage.googleapis.com/reusedata/echo/echo_out_Hello_World!_outfile.cwl&#x27;
Content type &#x27;application/octet-stream&#x27; length 1152 bytes
==================================================
downloaded 1152 bytes

trying URL &#x27;https://storage.googleapis.com/reusedata/echo/echo_out_Hello_World!_outfile.md5&#x27;
Content type &#x27;application/octet-stream&#x27; length 45 bytes
==================================================
downloaded 45 bytes

trying URL &#x27;https://storage.googleapis.com/reusedata/echo/echo_out_Hello_World!_outfile.sh&#x27;
Content type &#x27;application/x-shellscript&#x27; length 68 bytes
==================================================
downloaded 68 bytes

Data is downloaded: 
/tmp/RtmpRjX59u/test_gcpData/outfile.txt
}[1;30mINFO[0m Final process status is success
NOTE: existing caches will be removed and regenerated!
Updating recipes...
STAR_index.R added
bowtie2_index.R added
echo_out.R added
ensembl_liftover.R added
gcp_broad_gatk_hg19.R added
gcp_broad_gatk_hg38.R added
gcp_gatk_mutect2_b37.R added
gcp_gatk_mutect2_hg38.R added
gencode_annotation.R added
gencode_genome_grch38.R added
gencode_transcripts.R added
hisat2_index.R added
reference_genome.R added
salmon_index.R added
ucsc_database.R added

Data recipe loaded!
Use inputs(ensembl_liftover) to check required input parameters before evaluation.
Check here: https://rcwl.org/dataRecipes/ensembl_liftover.html
for user instructions (e.g., eligible input values, data source, etc.)

[ FAIL 0 | WARN 0 | SKIP 0 | PASS 44 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
 32.146   2.299  36.750 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">ReUseData.Rcheck/ReUseData-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>dataHub-class</TD><TD style="text-align: right;">9.039</TD><TD style="text-align: right;">0.948</TD><TD style="text-align: right;">9.988</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dataSearch</TD><TD style="text-align: right;">1.332</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">1.367</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dataUpdate</TD><TD style="text-align: right;">15.999</TD><TD style="text-align: right;"> 1.073</TD><TD style="text-align: right;">17.072</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getCloudData</TD><TD style="text-align: right;">10.686</TD><TD style="text-align: right;"> 0.512</TD><TD style="text-align: right;">15.826</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getData</TD><TD style="text-align: right;">4.944</TD><TD style="text-align: right;">0.254</TD><TD style="text-align: right;">5.192</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>meta_data</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeHub-class</TD><TD style="text-align: right;">0.254</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.278</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeLoad</TD><TD style="text-align: right;">1.591</TD><TD style="text-align: right;">0.084</TD><TD style="text-align: right;">1.677</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeMake</TD><TD style="text-align: right;">4.738</TD><TD style="text-align: right;">0.235</TD><TD style="text-align: right;">4.969</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeSearch</TD><TD style="text-align: right;">0.732</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">0.784</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeUpdate</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
