<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for chimeraviz on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><B>[<A href="../index.html#C">C</A>]</B></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-23 01:33:49 -0000 (Thu, 23 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4245</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for chimeraviz on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the chimeraviz package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/chimeraviz.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>303</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">chimeraviz</A>&nbsp;1.25.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/chimeraviz">landing page</A>)</SPAN><BR>Stian Lågstad<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-21&nbsp;12:29:53&nbsp;-0000&nbsp;(Tue,&nbsp;21&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/chimeraviz</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">f00dfd8</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:14:55&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: chimeraviz</TD></TR>
<TR><TD><B>Version</B>: 1.25.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:chimeraviz.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings chimeraviz_1.25.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-22 02:14:34 -0000 (Wed, 22 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-22 02:30:51 -0000 (Wed, 22 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 977.4 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: chimeraviz.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:chimeraviz.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings chimeraviz_1.25.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/chimeraviz.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘chimeraviz/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘chimeraviz’ version ‘1.25.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Depends: includes the non-default packages:
  &#x27;Biostrings&#x27;, &#x27;GenomicRanges&#x27;, &#x27;IRanges&#x27;, &#x27;Gviz&#x27;, &#x27;S4Vectors&#x27;,
  &#x27;ensembldb&#x27;, &#x27;AnnotationFilter&#x27;, &#x27;data.table&#x27;
Adding so many packages to the search path is excessive and importing
selectively is preferable.
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘chimeraviz’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
Namespace in Imports field not imported from: &#x27;magick&#x27;
  All declared Imports should be used.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
plot_fusion_transcript_with_protein_domain: no visible binding for
  global variable &#x27;protein_domain_location&#x27;
Undefined global functions or variables:
  protein_domain_location
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                                             user system elapsed
plot_fusion                                39.302  1.344  46.702
plot_transcripts                           24.268  0.484  24.720
plot_fusion_transcript                     19.708  0.156  19.841
plot_fusion_transcripts_graph              12.401  0.096  12.491
plot_fusion_transcript_with_protein_domain 12.214  0.120  12.318
get_transcripts_ensembl_db                  8.068  0.328   8.397
select_transcript                           7.570  0.060   7.630
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘chimeraviz-vignette.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 3 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/chimeraviz.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">chimeraviz.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL chimeraviz
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘chimeraviz’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (chimeraviz)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">chimeraviz.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(chimeraviz)
Loading required package: Biostrings
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: XVector
Loading required package: GenomeInfoDb

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: GenomicRanges
Loading required package: Gviz
Loading required package: grid

Attaching package: &#x27;grid&#x27;

The following object is masked from &#x27;package:Biostrings&#x27;:

    pattern

Loading required package: ensembldb
Loading required package: GenomicFeatures
Loading required package: AnnotationDbi
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

Loading required package: AnnotationFilter

Attaching package: &#x27;AnnotationFilter&#x27;

The following object is masked from &#x27;package:Gviz&#x27;:

    feature

The following object is masked from &#x27;package:testthat&#x27;:

    not


Attaching package: &#x27;ensembldb&#x27;

The following object is masked from &#x27;package:stats&#x27;:

    filter

Loading required package: data.table

Attaching package: &#x27;data.table&#x27;

The following object is masked from &#x27;package:GenomicRanges&#x27;:

    shift

The following object is masked from &#x27;package:IRanges&#x27;:

    shift

The following objects are masked from &#x27;package:S4Vectors&#x27;:

    first, second



&gt; 
&gt; test_check(&quot;chimeraviz&quot;)
[ FAIL 0 | WARN 0 | SKIP 0 | PASS 224 ]
Warning message:
call dbDisconnect() when finished working with a connection 
&gt; 
&gt; proc.time()
   user  system elapsed 
216.584   5.926 228.553 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">chimeraviz.Rcheck/chimeraviz-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>add_fusion_reads_alignment</TD><TD style="text-align: right;">0.661</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.686</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>create_fusion_report</TD><TD style="text-align: right;">3.378</TD><TD style="text-align: right;">0.297</TD><TD style="text-align: right;">3.663</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>decide_transcript_category</TD><TD style="text-align: right;">0.272</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.281</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>down_shift</TD><TD style="text-align: right;">0.074</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.074</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>downstream_partner_gene</TD><TD style="text-align: right;">0.082</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.086</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fetch_reads_from_fastq</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fusion_spanning_reads_count</TD><TD style="text-align: right;">0.041</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fusion_split_reads_count</TD><TD style="text-align: right;">0.042</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.041</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fusion_to_data_frame</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_ensembl_ids</TD><TD style="text-align: right;">0.466</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.478</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_fusion_by_chromosome</TD><TD style="text-align: right;">0.041</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.041</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_fusion_by_gene_name</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_fusion_by_id</TD><TD style="text-align: right;">0.037</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_transcripts_ensembl_db</TD><TD style="text-align: right;">8.068</TD><TD style="text-align: right;">0.328</TD><TD style="text-align: right;">8.397</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_aeron</TD><TD style="text-align: right;">0.654</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.682</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_chimpipe</TD><TD style="text-align: right;">0.130</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.135</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_defuse</TD><TD style="text-align: right;">0.168</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.169</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_ericscript</TD><TD style="text-align: right;">0.119</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.131</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_fusioncatcher</TD><TD style="text-align: right;">0.118</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.122</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_fusionmap</TD><TD style="text-align: right;">0.116</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.117</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_infusion</TD><TD style="text-align: right;">0.114</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.116</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_jaffa</TD><TD style="text-align: right;">0.114</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.114</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_oncofuse</TD><TD style="text-align: right;">0.116</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.120</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_prada</TD><TD style="text-align: right;">0.120</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.125</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_soapfuse</TD><TD style="text-align: right;">0.124</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.124</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_squid</TD><TD style="text-align: right;">0.120</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.128</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_starfusion</TD><TD style="text-align: right;">0.12</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.12</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>partner_gene_ensembl_id</TD><TD style="text-align: right;">0.083</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.083</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>partner_gene_junction_sequence</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_circle</TD><TD style="text-align: right;">0.809</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.820</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_fusion</TD><TD style="text-align: right;">39.302</TD><TD style="text-align: right;"> 1.344</TD><TD style="text-align: right;">46.702</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_fusion_reads</TD><TD style="text-align: right;">2.543</TD><TD style="text-align: right;">0.088</TD><TD style="text-align: right;">2.623</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_fusion_transcript</TD><TD style="text-align: right;">19.708</TD><TD style="text-align: right;"> 0.156</TD><TD style="text-align: right;">19.841</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_fusion_transcript_with_protein_domain</TD><TD style="text-align: right;">12.214</TD><TD style="text-align: right;"> 0.120</TD><TD style="text-align: right;">12.318</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_fusion_transcripts_graph</TD><TD style="text-align: right;">12.401</TD><TD style="text-align: right;"> 0.096</TD><TD style="text-align: right;">12.491</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_transcripts</TD><TD style="text-align: right;">24.268</TD><TD style="text-align: right;"> 0.484</TD><TD style="text-align: right;">24.720</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>select_transcript</TD><TD style="text-align: right;">7.57</TD><TD style="text-align: right;">0.06</TD><TD style="text-align: right;">7.63</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>split_on_utr_and_add_feature</TD><TD style="text-align: right;">0.486</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.486</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>upstream_partner_gene</TD><TD style="text-align: right;">0.140</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.141</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>write_fusion_reference</TD><TD style="text-align: right;">0.053</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.053</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
