<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for OrganismDbi on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><B>[<A href="..#O">O</A>]</B></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-16 06:07:00 -0000 (Tue, 16 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4787</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for OrganismDbi on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the OrganismDbi package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/OrganismDbi.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1423</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">OrganismDbi</A>&nbsp;1.43.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/OrganismDbi">landing page</A>)</SPAN><BR>Bioconductor Package Maintainer<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-13&nbsp;23:12:22&nbsp;-0000&nbsp;(Sat,&nbsp;13&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/OrganismDbi</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">b2c5577</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:23:59&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: OrganismDbi</TD></TR>
<TR><TD><B>Version</B>: 1.43.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:OrganismDbi.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings OrganismDbi_1.43.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-15 09:50:41 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-15 10:06:14 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 933.2 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: OrganismDbi.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:OrganismDbi.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings OrganismDbi_1.43.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/OrganismDbi.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘OrganismDbi/DESCRIPTION’ ... OK
* this is package ‘OrganismDbi’ version ‘1.43.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘OrganismDbi’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... NOTE
Unexported object imported by a &#x27;:::&#x27; call: ‘BiocGenerics:::testPackage’
  See the note in ?`:::` about the use of this operator.
There are ::: calls to the package&#x27;s namespace in its code. A package
  almost never needs to use ::: for its own objects:
  ‘.biocAnnPackages’ ‘.extractPkgsAndCols’ ‘.gentlyExtractDbFiles’
  ‘.lookupDbNameFromKeytype’ ‘.mungeGraphData’ ‘.taxIdToOrgDb’
  ‘.taxIdToOrgDbName’ ‘.testGraphData’ ‘.testKeys’ ‘OrganismDb’
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... WARNING
checkRd: (5) OrganismDb.Rd:66-70: \item in \describe must have non-empty label
checkRd: (5) OrganismDb.Rd:71-94: \item in \describe must have non-empty label
checkRd: (5) OrganismDb.Rd:95-99: \item in \describe must have non-empty label
checkRd: (5) OrganismDb.Rd:100-106: \item in \describe must have non-empty label
checkRd: (5) OrganismDb.Rd:107-124: \item in \describe must have non-empty label
checkRd: (5) OrganismDb.Rd:125-148: \item in \describe must have non-empty label
checkRd: (5) OrganismDb.Rd:149-160: \item in \describe must have non-empty label
checkRd: (5) OrganismDb.Rd:161-164: \item in \describe must have non-empty label
checkRd: (5) OrganismDb.Rd:165-168: \item in \describe must have non-empty label
checkRd: (5) OrganismDb.Rd:169-175: \item in \describe must have non-empty label
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘OrganismDbi-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: makeOrganismDbFromBiomart
&gt; ### Title: Make a OrganismDb object from annotations available on a BioMart
&gt; ###   database
&gt; ### Aliases: makeOrganismDbFromBiomart
&gt; 
&gt; ### ** Examples
&gt; 
&gt; ## Discover which datasets are available in the &quot;ensembl&quot; BioMart
&gt; ## database:
&gt; library(biomaRt)
&gt; mart &lt;- useEnsembl(&quot;ensembl&quot;)
&gt; datasets &lt;- listDatasets(mart)
&gt; head(datasets)
                       dataset                           description
1 abrachyrhynchus_gene_ensembl Pink-footed goose genes (ASM259213v1)
2     acalliptera_gene_ensembl      Eastern happy genes (fAstCal1.2)
3   acarolinensis_gene_ensembl       Green anole genes (AnoCar2.0v2)
4    acchrysaetos_gene_ensembl       Golden eagle genes (bAquChr1.2)
5    acitrinellus_gene_ensembl        Midas cichlid genes (Midas_v5)
6    amelanoleuca_gene_ensembl       Giant panda genes (ASM200744v2)
      version
1 ASM259213v1
2  fAstCal1.2
3 AnoCar2.0v2
4  bAquChr1.2
5    Midas_v5
6 ASM200744v2
&gt; 
&gt; ## Retrieving an incomplete transcript dataset for Human from the
&gt; ## &quot;ensembl&quot; BioMart database:
&gt; transcript_ids &lt;- c(
+     &quot;ENST00000013894&quot;,
+     &quot;ENST00000268655&quot;,
+     &quot;ENST00000313243&quot;,
+     &quot;ENST00000435657&quot;,
+     &quot;ENST00000384428&quot;,
+     &quot;ENST00000478783&quot;
+ )
&gt; odb &lt;- makeOrganismDbFromBiomart(transcript_ids=transcript_ids)
Download and preprocess the &#x27;transcripts&#x27; data frame ... OK
Download and preprocess the &#x27;chrominfo&#x27; data frame ... OK
Download and preprocess the &#x27;splicings&#x27; data frame ... OK
Download and preprocess the &#x27;genes&#x27; data frame ... OK
Prepare the &#x27;metadata&#x27; data frame ... OK
Make the TxDb object ... OK
Warning: unable to access index for repository https://bioconductor.org/packages/3.18/data/annotation/src/contrib:
  cannot open URL &#x27;https://bioconductor.org/packages/3.18/data/annotation/src/contrib/PACKAGES&#x27;
Error in h(simpleError(msg, call)) : 
  error in evaluating the argument &#x27;x&#x27; in selecting a method for function &#x27;dbfile&#x27;: object &#x27;GO.db&#x27; not found
Calls: makeOrganismDbFromBiomart ... FUN -&gt; dbfile -&gt; get -&gt; get -&gt; .handleSimpleError -&gt; h
Execution halted
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘OrganismDbi_unit_tests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘OrganismDbi.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 1 WARNING, 1 NOTE
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/OrganismDbi.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">OrganismDbi.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD INSTALL OrganismDbi
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0-bin/site-library’
* installing *source* package ‘OrganismDbi’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (OrganismDbi)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">OrganismDbi.Rcheck/tests/OrganismDbi_unit_tests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; BiocGenerics:::testPackage(&quot;OrganismDbi&quot;)

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


Attaching package: &#x27;S4Vectors&#x27;

The following object is masked from &#x27;package:utils&#x27;:

    findMatches

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: Homo.sapiens
Loading required package: GO.db

Loading required package: org.Hs.eg.db

Loading required package: TxDb.Hsapiens.UCSC.hg19.knownGene
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
Loading required package: RSQLite
Loading required package: Rattus.norvegicus
Loading required package: org.Rn.eg.db

Loading required package: TxDb.Rnorvegicus.UCSC.rn5.refGene
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:1 mapping between keys and columns
&#x27;select()&#x27; returned 1:1 mapping between keys and columns
&#x27;select()&#x27; returned 1:1 mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:1 mapping between keys and columns
&#x27;select()&#x27; returned 1:1 mapping between keys and columns
&#x27;select()&#x27; returned 1:1 mapping between keys and columns
&#x27;select()&#x27; returned 1:1 mapping between keys and columns
&#x27;select()&#x27; returned 1:1 mapping between keys and columns
&#x27;select()&#x27; returned 1:1 mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned many:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned many:many mapping between keys and columns
&#x27;select()&#x27; returned 1:many mapping between keys and columns
&#x27;select()&#x27; returned 1:1 mapping between keys and columns
&#x27;select()&#x27; returned 1:1 mapping between keys and columns
&#x27;select()&#x27; returned 1:1 mapping between keys and columns
Loading required package: FDb.UCSC.tRNAs
Loading required package: BSgenome.Hsapiens.UCSC.hg19
Loading required package: BSgenome
Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: rtracklayer
  403 genes were dropped because they have exons located on both strands
  of the same reference sequence or on more than one reference sequence,
  so cannot be represented by a single genomic range.
  Use &#x27;single.strand.genes.only=FALSE&#x27; to get all the genes in a
  GRangesList object, or use suppressMessages() to suppress this message.


RUNIT TEST PROTOCOL -- Mon May 15 10:05:40 2023 
*********************************************** 
Number of test functions: 41 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
OrganismDbi RUnit Tests - 41 test functions, 0 errors, 0 failures
Number of test functions: 41 
Number of errors: 0 
Number of failures: 0 
Warning message:
call dbDisconnect() when finished working with a connection 
&gt; 
&gt; proc.time()
   user  system elapsed 
247.730   7.854 274.187 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">OrganismDbi.Rcheck/OrganismDbi-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>OrganismDb</TD><TD style="text-align: right;">9.368</TD><TD style="text-align: right;">0.472</TD><TD style="text-align: right;">9.844</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>coordinate-mapping-method</TD><TD style="text-align: right;">1.010</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">1.045</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
