<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for rCGH on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><B>[<A href="..#R">R</A>]</B></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-16 06:07:02 -0000 (Tue, 16 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4787</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for rCGH on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the rCGH package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/rCGH.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1619</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">rCGH</A>&nbsp;1.31.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/rCGH">landing page</A>)</SPAN><BR>Frederic Commo<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-13&nbsp;23:12:22&nbsp;-0000&nbsp;(Sat,&nbsp;13&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/rCGH</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">0cbed14</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:40:26&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: rCGH</TD></TR>
<TR><TD><B>Version</B>: 1.31.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:rCGH.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings rCGH_1.31.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-15 11:35:13 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-15 11:44:40 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 566.8 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: rCGH.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:rCGH.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings rCGH_1.31.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/rCGH.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘rCGH/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘rCGH’ version ‘1.31.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘rCGH’ can be installed ... OK
* checking installed package size ... NOTE
  installed size is  5.0Mb
  sub-directories of 1Mb or more:
    data      2.8Mb
    extdata   1.2Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... NOTE
Problems with news in ‘NEWS’:
  Cannot process chunk/lines:
    rCGH 1.1.8 has been released as rCGH-1.2.2
  Cannot process chunk/lines:
    Cleaner version from 1.1.4.
  Cannot process chunk/lines:
    Should not give any warning due to functions masked by imported packages
  Cannot process chunk/lines:
    Not submitted!
  Cannot process chunk/lines:
    New bioconductor devel versioning after release cycle.
  Cannot process chunk/lines:
    The first and corresponding bioconductor release version is 1.0.0.
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking startup messages can be suppressed ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking LazyData ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking installed files from ‘inst/doc’ ... OK
* checking files in ‘vignettes’ ... NOTE
The following directory looks like a leftover from &#x27;knitr&#x27;:
  ‘figure’
Please remove from your package.
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
               user system elapsed
byGeneTable  15.250  0.817  16.065
multiplot    11.901  0.505  12.404
plotProfile  10.706  0.176  10.880
rCGH-package  7.482  0.167   7.646
EMnormalize   7.341  0.167   7.507
plotLOH       7.097  0.164   7.256
recenter      6.930  0.275   7.203
plotDensity   6.891  0.156   7.043
view          6.216  0.263   6.477
segmentCGH    6.068  0.314   6.379
adjustSignal  5.321  0.028   5.347
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘rCGH.Rnw’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 3 NOTEs
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/rCGH.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">rCGH.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD INSTALL rCGH
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0-bin/site-library’
* installing *source* package ‘rCGH’ ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (rCGH)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">rCGH.Rcheck/tests/runTests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; BiocGenerics:::testPackage(&quot;rCGH&quot;)

	********************************************************
	Current version: 1.31.0
	This version may contain important changes.
	Use news(Version == &#x27;1.31.0&#x27;, package = &#x27;rCGH&#x27;).
	********************************************************

SNP probes will be used.
Reading information...
Reading values...
Adding presettings...
Genome build: hg19
Log2Ratios QCs:
	dLRs: 0.162
	MAD: 0.128

Scaling...
Signal filtering...
Modeling allelic Difference...
Computing LRR segmentation using UndoSD: 0.179
Merging segments shorter than 10Kb.
Number of segments: 25
Merging peaks closer than 0.1 ...
Gaussian mixture estimation:
n.peaks =  3

Group parameters:
Grp 1:
prop: 0.504,	mean: -0.061,	Sd: 0.149,	peak height: 1.344
Grp 2:
prop: 0.481,	mean: 0.861,	Sd: 0.149,	peak height: 1.284
Grp 3:
prop: 0.015,	mean: 2.04,	Sd: 0.149,	peak height: 0.041

Correction value:  -0.061
Use plotDensity() to visualize the LRR densities.
  403 genes were dropped because they have exons located on both strands
  of the same reference sequence or on more than one reference sequence,
  so cannot be represented by a single genomic range.
  Use &#x27;single.strand.genes.only=FALSE&#x27; to get all the genes in a
  GRangesList object, or use suppressMessages() to suppress this message.
Creating byGene table...
SNP probes will be used.
Reading information...
Reading values...
Adding presettings...
Genome build: hg19
SNP probes will be used.
Reading information...
Reading values...
Adding presettings...
Genome build: hg19


RUNIT TEST PROTOCOL -- Mon May 15 11:43:42 2023 
*********************************************** 
Number of test functions: 11 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
rCGH RUnit Tests - 11 test functions, 0 errors, 0 failures
Number of test functions: 11 
Number of errors: 0 
Number of failures: 0 
&gt; 
&gt; proc.time()
   user  system elapsed 
 34.014   1.041  35.028 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">rCGH.Rcheck/rCGH-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>AllAccessors</TD><TD style="text-align: right;">0.584</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">0.625</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EMnormalize</TD><TD style="text-align: right;">7.341</TD><TD style="text-align: right;">0.167</TD><TD style="text-align: right;">7.507</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>adjustSignal</TD><TD style="text-align: right;">5.321</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">5.347</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>byGeneTable</TD><TD style="text-align: right;">15.250</TD><TD style="text-align: right;"> 0.817</TD><TD style="text-align: right;">16.065</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>hg18</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>hg19</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>hg38</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>multiplot</TD><TD style="text-align: right;">11.901</TD><TD style="text-align: right;"> 0.505</TD><TD style="text-align: right;">12.404</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotDensity</TD><TD style="text-align: right;">6.891</TD><TD style="text-align: right;">0.156</TD><TD style="text-align: right;">7.043</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotLOH</TD><TD style="text-align: right;">7.097</TD><TD style="text-align: right;">0.164</TD><TD style="text-align: right;">7.256</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotProfile</TD><TD style="text-align: right;">10.706</TD><TD style="text-align: right;"> 0.176</TD><TD style="text-align: right;">10.880</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rCGH-Agilent-class</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rCGH-SNP6-class</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rCGH-class</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rCGH-cytoScan-class</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rCGH-generic-class</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rCGH-oncoScan-class</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rCGH-package</TD><TD style="text-align: right;">7.482</TD><TD style="text-align: right;">0.167</TD><TD style="text-align: right;">7.646</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readAffyCytoScan</TD><TD style="text-align: right;">0.429</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.445</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readAffyOncoScan</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readAffySNP6</TD><TD style="text-align: right;">0.361</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.374</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readAgilent</TD><TD style="text-align: right;">0.618</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">0.658</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readGeneric</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recenter</TD><TD style="text-align: right;">6.930</TD><TD style="text-align: right;">0.275</TD><TD style="text-align: right;">7.203</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>segmentCGH</TD><TD style="text-align: right;">6.068</TD><TD style="text-align: right;">0.314</TD><TD style="text-align: right;">6.379</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>setInfo</TD><TD style="text-align: right;">0.432</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">0.445</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>view</TD><TD style="text-align: right;">6.216</TD><TD style="text-align: right;">0.263</TD><TD style="text-align: right;">6.477</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
