<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for EWCE on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><B>[<A href="../index.html#E">E</A>]</B></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-23 01:33:58 -0000 (Thu, 23 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4245</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for EWCE on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the EWCE package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/EWCE.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard timeout">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>634</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">EWCE</A>&nbsp;1.7.1</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/EWCE">landing page</A>)</SPAN><BR>Alan Murphy<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-21&nbsp;12:29:53&nbsp;-0000&nbsp;(Tue,&nbsp;21&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/EWCE</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">22dfbab</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-01-19&nbsp;17:29:34&nbsp;-0000&nbsp;(Thu,&nbsp;19&nbsp;Jan&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph TIMEOUT">&nbsp;&nbsp;TIMEOUT&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: EWCE</TD></TR>
<TR><TD><B>Version</B>: 1.7.1</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:EWCE.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings EWCE_1.7.1.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-22 04:42:55 -0000 (Wed, 22 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-22 05:22:55 -0000 (Wed, 22 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 2400.8 seconds</TD></TR>
<TR><TD><B>RetCode</B>: None</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph TIMEOUT">&nbsp;&nbsp;TIMEOUT&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: EWCE.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:EWCE.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings EWCE_1.7.1.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/EWCE.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘EWCE/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘EWCE’ version ‘1.7.1’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘EWCE’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                                             user system elapsed
controlled_geneset_enrichment              39.537  1.031  44.629
merged_ewce                                32.612  1.609  41.693
generate_bootstrap_plots_for_transcriptome 19.733  0.984  23.083
filter_genes_without_1to1_homolog          19.042  0.551  21.107
filter_nonorthologs                        18.928  0.588  20.936
bootstrap_enrichment_test                  18.146  0.876  21.436
ewce_expression_data                       17.640  0.945  21.114
generate_bootstrap_plots                   17.145  1.124  20.264
check_ewce_genelist_inputs                 15.617  0.829  18.958
sct_normalize                              13.864  0.811  16.607
standardise_ctd                            13.586  0.620  18.843
filter_ctd_genes                           11.266  0.456  13.153
add_res_to_merging_list                     9.607  0.752  15.713
merge_ctd                                   9.403  0.618  11.756
fix_bad_mgi_symbols                         7.410  0.673  11.063
merge_sce                                   6.752  0.304   8.567
generate_celltype_data                      5.954  0.689   8.082
bin_columns_into_quantiles                  6.123  0.400   7.475
ctd_to_sce                                  6.277  0.196   8.008
bin_specificity_into_quantiles              5.727  0.356   7.558
drop_uninformative_genes                    5.636  0.422   8.578
merge_two_expfiles                          4.804  0.508   8.945
get_celltype_table                          4.520  0.464   6.517
plot_ctd                                    4.694  0.197   6.368
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘EWCE.Rmd’ using ‘UTF-8’... OK
  ‘extended.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ...</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">EWCE.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL EWCE
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘EWCE’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (EWCE)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">EWCE.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(EWCE)
Loading required package: RNOmni
&gt; 
&gt; test_check(&quot;EWCE&quot;)
1 core(s) assigned as workers (31 reserved).
Converting to sparse matrix.
+ Calculating normalized mean expression.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
+ Calculating normalized specificity.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
Loading required namespace: ggdendro
+ Saving results ==&gt;  /tmp/RtmptEOog7/ctd_testthat.rda
1 core(s) assigned as workers (31 reserved).
Converting to sparse matrix.
+ Calculating normalized mean expression.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
+ Calculating normalized specificity.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
+ Saving results ==&gt;  /tmp/RtmptEOog7/ctd_testthat.rda
snapshotDate(): 2023-02-14
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
1 core(s) assigned as workers (31 reserved).
Converting to sparse matrix.
Converting to sparse matrix.
1 core(s) assigned as workers (31 reserved).
Processing block [[1/5, 1/1]] ... OK
Processing block [[2/5, 1/1]] ... OK
Processing block [[3/5, 1/1]] ... OK
Processing block [[4/5, 1/1]] ... OK
Processing block [[5/5, 1/1]] ... OK
Processing block [[1/5, 1/1]] ... OK
Processing block [[2/5, 1/1]] ... OK
Processing block [[3/5, 1/1]] ... OK
Processing block [[4/5, 1/1]] ... OK
Processing block [[5/5, 1/1]] ... OK
Loading required namespace: sctransform
Converting to sparse matrix.
Calculating cell attributes from input UMI matrix: log_umi
Variance stabilizing transformation of count matrix of size 17757 by 3005
Model formula is y ~ log_umi
Get Negative Binomial regression parameters per gene
Using 2000 genes, 3005 cells
  |                                                                              |                                                                      |   0%  |                                                                              |==================                                                    |  25%  |                                                                              |===================================                                   |  50%  |                                                                              |====================================================                  |  75%  |                                                                              |======================================================================| 100%
Found 12 outliers - those will be ignored in fitting/regularization step

Second step: Get residuals using fitted parameters for 17757 genes
  |                                                                              |                                                                      |   0%  |                                                                              |==                                                                    |   3%  |                                                                              |====                                                                  |   6%  |                                                                              |======                                                                |   8%  |                                                                              |========                                                              |  11%  |                                                                              |==========                                                            |  14%  |                                                                              |============                                                          |  17%  |                                                                              |==============                                                        |  19%  |                                                                              |================                                                      |  22%  |                                                                              |==================                                                    |  25%  |                                                                              |===================                                                   |  28%  |                                                                              |=====================                                                 |  31%  |                                                                              |=======================                                               |  33%  |                                                                              |=========================                                             |  36%  |                                                                              |===========================                                           |  39%  |                                                                              |=============================                                         |  42%  |                                                                              |===============================                                       |  44%  |                                                                              |=================================                                     |  47%  |                                                                              |===================================                                   |  50%  |                                                                              |=====================================                                 |  53%  |                                                                              |=======================================                               |  56%  |                                                                              |=========================================                             |  58%  |                                                                              |===========================================                           |  61%  |                                                                              |=============================================                         |  64%  |                                                                              |===============================================                       |  67%  |                                                                              |=================================================                     |  69%  |                                                                              |===================================================                   |  72%  |                                                                              |====================================================                  |  75%  |                                                                              |======================================================                |  78%  |                                                                              |========================================================              |  81%  |                                                                              |==========================================================            |  83%  |                                                                              |============================================================          |  86%  |                                                                              |==============================================================        |  89%  |                                                                              |================================================================      |  92%  |                                                                              |==================================================================    |  94%  |                                                                              |====================================================================  |  97%  |                                                                              |======================================================================| 100%
Calculating gene attributes
Wall clock passed: Time difference of 1.119126 mins
Computing corrected UMI count matrix
  |                                                                              |                                                                      |   0%  |                                                                              |==                                                                    |   3%  |                                                                              |====                                                                  |   6%  |                                                                              |======                                                                |   8%  |                                                                              |========                                                              |  11%  |                                                                              |==========                                                            |  14%  |                                                                              |============                                                          |  17%  |                                                                              |==============                                                        |  19%  |                                                                              |================                                                      |  22%  |                                                                              |==================                                                    |  25%  |                                                                              |===================                                                   |  28%  |                                                                              |=====================                                                 |  31%  |                                                                              |=======================                                               |  33%  |                                                                              |=========================                                             |  36%  |                                                                              |===========================                                           |  39%  |                                                                              |=============================                                         |  42%  |                                                                              |===============================                                       |  44%  |                                                                              |=================================                                     |  47%  |                                                                              |===================================                                   |  50%  |                                                                              |=====================================                                 |  53%  |                                                                              |=======================================                               |  56%  |                                                                              |=========================================                             |  58%  |                                                                              |===========================================                           |  61%  |                                                                              |=============================================                         |  64%  |                                                                              |===============================================                       |  67%  |                                                                              |=================================================                     |  69%  |                                                                              |===================================================                   |  72%  |                                                                              |====================================================                  |  75%  |                                                                              |======================================================                |  78%  |                                                                              |========================================================              |  81%  |                                                                              |==========================================================            |  83%  |                                                                              |============================================================          |  86%  |                                                                              |==============================================================        |  89%  |                                                                              |================================================================      |  92%  |                                                                              |==================================================================    |  94%  |                                                                              |====================================================================  |  97%  |                                                                              |======================================================================| 100%
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
+ &lt;2 non-zero quantile bins detected in column. Assigning these values to default quantile  ( 20 )
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
1 core(s) assigned as workers (31 reserved).
Generating gene background for mouse x human ==&gt; human
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
21,207 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
17,355 genes extracted.
Extracting genes from ortholog_gene.
17,355 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 131 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 498 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   4,725 / 21,207 (22%)
Total genes remaining after convert_orthologs :
   16,482 / 21,207 (78%)
--

=========== REPORT SUMMARY ===========

16,482 / 21,207 (77.72%) target_species genes remain after ortholog conversion.
16,482 / 19,129 (86.16%) reference_species genes remain after ortholog conversion.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--

=========== REPORT SUMMARY ===========

19,129 / 19,129 (100%) target_species genes remain after ortholog conversion.
19,129 / 19,129 (100%) reference_species genes remain after ortholog conversion.
16,482 intersect background genes used.
Standardising CellTypeDataset
Converting to sparse matrix.
Converting to sparse matrix.
Checking gene list inputs.
Returning 16,482 unique genes from the user-supplied bg.
Standardising sct_data.
Converting gene list input to standardised human genes.
Running without gene size control.
17 hit genes remain after filtering.
Testing for enrichment in 7 cell types...
Sorting results by p-value.
Computing BH-corrected q-values.
2 significant cell type enrichment results @ q&lt;0.05 : 

              CellType annotLevel p fold_change sd_from_mean q
1            microglia          1 0    1.820518     2.409647 0
2 astrocytes_ependymal          1 0    1.357308     1.995351 0
Loading required namespace: cowplot
Generating gene background for mouse x human ==&gt; human
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
21,207 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
17,355 genes extracted.
Extracting genes from ortholog_gene.
17,355 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 131 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 498 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   4,725 / 21,207 (22%)
Total genes remaining after convert_orthologs :
   16,482 / 21,207 (78%)
--

=========== REPORT SUMMARY ===========

16,482 / 21,207 (77.72%) target_species genes remain after ortholog conversion.
16,482 / 19,129 (86.16%) reference_species genes remain after ortholog conversion.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--

=========== REPORT SUMMARY ===========

19,129 / 19,129 (100%) target_species genes remain after ortholog conversion.
19,129 / 19,129 (100%) reference_species genes remain after ortholog conversion.
16,482 intersect background genes used.
Standardising sct_data.
Converting to sparse matrix.
Converting to sparse matrix.
Aligning celltype names with standardise_ctd format.
Generating bootstrap plot: microglia
Generating bootstrap plot: astrocytes_ependymal
Warning: genelistSpecies not provided. Setting to &#x27;human&#x27; by default.
Preparing gene_df.
character format detected.
Converting to data.frame
Extracting genes from input_gene.
15,259 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
13,416 genes extracted.
Extracting genes from ortholog_gene.
13,416 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 46 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 56 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Returning gene_map as dictionary

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   2,016 / 15,259 (13%)
Total genes remaining after convert_orthologs :
   13,243 / 15,259 (87%)
Generating gene background for mouse x human ==&gt; human
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
21,207 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
17,355 genes extracted.
Extracting genes from ortholog_gene.
17,355 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 131 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 498 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   4,725 / 21,207 (22%)
Total genes remaining after convert_orthologs :
   16,482 / 21,207 (78%)
--

=========== REPORT SUMMARY ===========

16,482 / 21,207 (77.72%) target_species genes remain after ortholog conversion.
16,482 / 19,129 (86.16%) reference_species genes remain after ortholog conversion.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--

=========== REPORT SUMMARY ===========

19,129 / 19,129 (100%) target_species genes remain after ortholog conversion.
19,129 / 19,129 (100%) reference_species genes remain after ortholog conversion.
16,482 intersect background genes used.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
Returning 19,129 unique genes from entire human genome.
Using intersect between background gene lists: 16,482 genes.
Standardising sct_data.
Converting to sparse matrix.
Converting to sparse matrix.
Using 1st column of tt as gene column: HGNC.symbol
1 core(s) assigned as workers (31 reserved).
Standardising CellTypeDataset
Checking gene list inputs.
Returning 16,482 unique genes from the user-supplied bg.
Standardising sct_data.
Converting gene list input to standardised human genes.
Running without gene size control.
6 hit genes remain after filtering.
Testing for enrichment in 7 cell types...
Sorting results by p-value.
Computing BH-corrected q-values.
2 significant cell type enrichment results @ q&lt;0.05 : 

           CellType annotLevel p fold_change sd_from_mean q
1 endothelial_mural          1 0    1.884819     4.092025 0
2         microglia          1 0    2.492816     3.122445 0
1 core(s) assigned as workers (31 reserved).
Standardising CellTypeDataset
Checking gene list inputs.
Returning 16,482 unique genes from the user-supplied bg.
Standardising sct_data.
Converting gene list input to standardised human genes.
Running without gene size control.
5 hit genes remain after filtering.
Testing for enrichment in 7 cell types...
Sorting results by p-value.
Computing BH-corrected q-values.
2 significant cell type enrichment results @ q&lt;0.05 : 

      CellType annotLevel p fold_change sd_from_mean q
1 interneurons          1 0    1.837327     4.060324 0
2 pyramidal_SS          1 0    1.880474     2.861583 0
Warning: genelistSpecies not provided. Setting to &#x27;human&#x27; by default.
Aligning celltype names with standardise_ctd format.
Preparing gene_df.
character format detected.
Converting to data.frame
Extracting genes from input_gene.
15,259 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
13,416 genes extracted.
Extracting genes from ortholog_gene.
13,416 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 46 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 56 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Returning gene_map as dictionary

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   2,016 / 15,259 (13%)
Total genes remaining after convert_orthologs :
   13,243 / 15,259 (87%)
Generating gene background for mouse x human ==&gt; human
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
21,207 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
17,355 genes extracted.
Extracting genes from ortholog_gene.
17,355 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 131 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 498 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   4,725 / 21,207 (22%)
Total genes remaining after convert_orthologs :
   16,482 / 21,207 (78%)
--

=========== REPORT SUMMARY ===========

16,482 / 21,207 (77.72%) target_species genes remain after ortholog conversion.
16,482 / 19,129 (86.16%) reference_species genes remain after ortholog conversion.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--

=========== REPORT SUMMARY ===========

19,129 / 19,129 (100%) target_species genes remain after ortholog conversion.
19,129 / 19,129 (100%) reference_species genes remain after ortholog conversion.
16,482 intersect background genes used.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
Returning 19,129 unique genes from entire human genome.
Using intersect between background gene lists: 16,482 genes.
Standardising sct_data.
Converting to sparse matrix.
Converting to sparse matrix.
Using 1st column of tt as gene column: HGNC.symbol
Generating exp data for bootstrap genes.
Converting data for bootstrap tests to sparse matrices.
endothelial_mural : Saving bootstrap plot.
microglia : Saving bootstrap plot.
interneurons : Saving bootstrap plot.
oligodendrocytes : Saving bootstrap plot.
astrocytes_ependymal : Saving bootstrap plot.
pyramidal_CA1 : Saving bootstrap plot.
pyramidal_SS : Saving bootstrap plot.
Generating exp data for bootstrap genes.
Converting data for bootstrap tests to sparse matrices.
interneurons : Saving bootstrap plot.
pyramidal_SS : Saving bootstrap plot.
pyramidal_CA1 : Saving bootstrap plot.
oligodendrocytes : Saving bootstrap plot.
microglia : Saving bootstrap plot.
astrocytes_ependymal : Saving bootstrap plot.
endothelial_mural : Saving bootstrap plot.
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
Preparing gene_df.
character format detected.
Converting to data.frame
Extracting genes from input_gene.
22 genes extracted.
Converting human ==&gt; mouse orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Checking for genes without orthologs in mouse.
Extracting genes from input_gene.
17 genes extracted.
Extracting genes from ortholog_gene.
17 genes extracted.
Checking for genes without 1:1 orthologs.
Filtering gene_df with gene_map
Returning gene_map as dictionary

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   5 / 22 (23%)
Total genes remaining after convert_orthologs :
   17 / 22 (77%)
1 core(s) assigned as workers (31 reserved).
1 core(s) assigned as workers (31 reserved).
Generating gene background for mouse x human ==&gt; human
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
21,207 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
17,355 genes extracted.
Extracting genes from ortholog_gene.
17,355 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 131 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 498 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   4,725 / 21,207 (22%)
Total genes remaining after convert_orthologs :
   16,482 / 21,207 (78%)
--

=========== REPORT SUMMARY ===========

16,482 / 21,207 (77.72%) target_species genes remain after ortholog conversion.
16,482 / 19,129 (86.16%) reference_species genes remain after ortholog conversion.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--

=========== REPORT SUMMARY ===========

19,129 / 19,129 (100%) target_species genes remain after ortholog conversion.
19,129 / 19,129 (100%) reference_species genes remain after ortholog conversion.
16,482 intersect background genes used.
Standardising CellTypeDataset
Converting to sparse matrix.
Converting to sparse matrix.
Checking gene list inputs.
Returning 16,482 unique genes from the user-supplied bg.
Standardising sct_data.
Converting gene list input to standardised human genes.
Running with gene size control.
Retrieving all genes using: gprofiler
Retrieving all organisms available in gprofiler.
Using stored `gprofiler_orgs`.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: hsapiens
Gene table with 61,769 rows retrieved.
Returning all 61,769 genes from human.
61,769 human Ensembl IDs and 40,153 human Gene Symbols imported.
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
Controlled bootstrapping network generated.
17 hit genes remain after filtering.
Testing for enrichment in 7 cell types...
Sorting results by p-value.
Computing BH-corrected q-values.
1 significant cell type enrichment results @ q&lt;0.05 : 

   CellType annotLevel p fold_change sd_from_mean q
1 microglia          1 0     2.01616     3.758104 0
1 core(s) assigned as workers (31 reserved).
Returning 15,604 unique genes from the user-supplied bg.
Standardising CellTypeDataset
Converting to sparse matrix.
Converting to sparse matrix.
Checking gene list inputs.
Returning 15,604 unique genes from the user-supplied bg.
Standardising sct_data.
Converting gene list input to standardised human genes.
1 core(s) assigned as workers (31 reserved).
Returning 15,589 unique genes from the user-supplied bg.
Standardising CellTypeDataset
Converting to sparse matrix.
Converting to sparse matrix.
Checking gene list inputs.
Returning 15,589 unique genes from the user-supplied bg.
Standardising sct_data.
Converting gene list input to standardised human genes.
Preparing gene_df.
character format detected.
Converting to data.frame
Extracting genes from input_gene.
22 genes extracted.
Converting human ==&gt; rat orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Retrieving all organisms available in homologene.
Mapping species name: rat
Common name mapping found for rat
1 organism identified from search: 10116
Checking for genes without orthologs in rat.
Extracting genes from input_gene.
18 genes extracted.
Extracting genes from ortholog_gene.
18 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 2 genes that have multiple input_gene per ortholog_gene (many:1).
Filtering gene_df with gene_map
Returning gene_map as dictionary

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   7 / 22 (32%)
Total genes remaining after convert_orthologs :
   15 / 22 (68%)
1 core(s) assigned as workers (31 reserved).
Generating gene background for mouse x rat ==&gt; human
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
21,207 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
17,355 genes extracted.
Extracting genes from ortholog_gene.
17,355 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 131 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 498 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   4,725 / 21,207 (22%)
Total genes remaining after convert_orthologs :
   16,482 / 21,207 (78%)
--

=========== REPORT SUMMARY ===========

16,482 / 21,207 (77.72%) target_species genes remain after ortholog conversion.
16,482 / 19,129 (86.16%) reference_species genes remain after ortholog conversion.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: rat
Common name mapping found for rat
1 organism identified from search: 10116
Gene table with 20,616 rows retrieved.
Returning all 20,616 genes from rat.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
20,616 genes extracted.
Converting rat ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: rat
Common name mapping found for rat
1 organism identified from search: 10116
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
16,989 genes extracted.
Extracting genes from ortholog_gene.
16,989 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 122 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 607 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   4,813 / 20,616 (23%)
Total genes remaining after convert_orthologs :
   15,803 / 20,616 (77%)
--

=========== REPORT SUMMARY ===========

15,803 / 20,616 (76.65%) target_species genes remain after ortholog conversion.
15,803 / 19,129 (82.61%) reference_species genes remain after ortholog conversion.
15,450 intersect background genes used.
Standardising CellTypeDataset
Converting to sparse matrix.
Converting to sparse matrix.
Checking gene list inputs.
Returning 15,450 unique genes from the user-supplied bg.
Standardising sct_data.
Converting gene list input to standardised human genes.
Running without gene size control.
15 hit genes remain after filtering.
Testing for enrichment in 7 cell types...
Sorting results by p-value.
Computing BH-corrected q-values.
1 significant cell type enrichment results @ q&lt;0.05 : 

   CellType annotLevel p fold_change sd_from_mean q
1 microglia          1 0    2.188004     4.229286 0
Standardising CellTypeDataset
Found 5 matrix types across 2 CTD levels.
Processing level: 1
Converting to sparse matrix.
Processing level: 2
Converting to sparse matrix.
1 core(s) assigned as workers (31 reserved).
Generating gene background for monkey x mouse ==&gt; human
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: monkey
Common name mapping found for monkey
1 organism identified from search: 9544
Gene table with 16,843 rows retrieved.
Returning all 16,843 genes from monkey.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
16,843 genes extracted.
Converting monkey ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: monkey
Common name mapping found for monkey
1 organism identified from search: 9544
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
15,675 genes extracted.
Extracting genes from ortholog_gene.
15,675 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 203 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 221 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   1,730 / 16,816 (10%)
Total genes remaining after convert_orthologs :
   15,086 / 16,816 (90%)
--

=========== REPORT SUMMARY ===========

15,075 / 16,816 (89.65%) target_species genes remain after ortholog conversion.
15,075 / 19,129 (78.81%) reference_species genes remain after ortholog conversion.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
21,207 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
17,355 genes extracted.
Extracting genes from ortholog_gene.
17,355 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 131 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 498 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   4,725 / 21,207 (22%)
Total genes remaining after convert_orthologs :
   16,482 / 21,207 (78%)
--

=========== REPORT SUMMARY ===========

16,482 / 21,207 (77.72%) target_species genes remain after ortholog conversion.
16,482 / 19,129 (86.16%) reference_species genes remain after ortholog conversion.
13,873 intersect background genes used.
Standardising CellTypeDataset
Converting to sparse matrix.
Converting to sparse matrix.
Checking gene list inputs.
Returning 13,873 unique genes from the user-supplied bg.
Standardising sct_data.
Converting gene list input to standardised human genes.
Running without gene size control.
0 hit genes remain after filtering.
Testing for enrichment in 7 cell types...
Sorting results by p-value.
Computing BH-corrected q-values.
0 significant cell type enrichment results @ q&lt;0.05 : 

1 core(s) assigned as workers (31 reserved).
Generating gene background for godzilla x mouse ==&gt; human
1 core(s) assigned as workers (31 reserved).
Generating gene background for mouse x human ==&gt; human
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
21,207 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
17,355 genes extracted.
Extracting genes from ortholog_gene.
17,355 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 131 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 498 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   4,725 / 21,207 (22%)
Total genes remaining after convert_orthologs :
   16,482 / 21,207 (78%)
--

=========== REPORT SUMMARY ===========

16,482 / 21,207 (77.72%) target_species genes remain after ortholog conversion.
16,482 / 19,129 (86.16%) reference_species genes remain after ortholog conversion.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--

=========== REPORT SUMMARY ===========

19,129 / 19,129 (100%) target_species genes remain after ortholog conversion.
19,129 / 19,129 (100%) reference_species genes remain after ortholog conversion.
16,482 intersect background genes used.
Standardising CellTypeDataset
Converting to sparse matrix.
Converting to sparse matrix.
Checking gene list inputs.
Returning 16,482 unique genes from the user-supplied bg.
Standardising sct_data.
Converting gene list input to standardised human genes.
1 core(s) assigned as workers (31 reserved).
Returning 15,604 unique genes from the user-supplied bg.
Standardising CellTypeDataset
Converting to sparse matrix.
Converting to sparse matrix.
Checking gene list inputs.
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
Check 1000Check 3005
Warning: sctSpecies not provided. Setting to &#x27;mouse&#x27; by default.
1 core(s) assigned as workers (31 reserved).
Converting to sparse matrix.
Checking for non-expressed genes.
Checking for cells with no expressed genes.
DGE:: Limma...
248 / 1,000 genes dropped @ DGE adj_pval_thresh &lt; 1e-05
Time difference of 1.361462 secs
1 core(s) assigned as workers (31 reserved).
Converting to sparse matrix.
+ Calculating normalized mean expression.
Converting to sparse matrix.
Converting to sparse matrix.
+ Calculating normalized specificity.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
+ Saving results ==&gt;  /tmp/RtmptEOog7/ctd_kiCortexOnly.rda
1 core(s) assigned as workers (31 reserved).
Converting to sparse matrix.
+ Calculating normalized mean expression.
Converting to sparse matrix.
Converting to sparse matrix.
+ Calculating normalized specificity.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
+ Saving results ==&gt;  /tmp/RtmptEOog7/ctd_kiCortexOnly.rda
Loading precomputed example bootstrap results.
Aligning celltype names with standardise_ctd format.
7 signficiant enrichment results @ bonferroni &lt; 0.05
14.3% of hits are of the target cell type.
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
Standardising CellTypeDataset
Found 5 matrix types across 2 CTD levels.
Processing level: 1
Converting to sparse matrix.
Processing level: 2
Converting to sparse matrix.
Generating controlled bootstrap gene sets.
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
Warning: genelistSpecies not provided. Setting to &#x27;human&#x27; by default.
Preparing gene_df.
character format detected.
Converting to data.frame
Extracting genes from input_gene.
15,259 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
13,416 genes extracted.
Extracting genes from ortholog_gene.
13,416 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 46 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 56 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Returning gene_map as dictionary

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   2,016 / 15,259 (13%)
Total genes remaining after convert_orthologs :
   13,243 / 15,259 (87%)
Generating gene background for mouse x human ==&gt; human
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
21,207 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
17,355 genes extracted.
Extracting genes from ortholog_gene.
17,355 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 131 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 498 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   4,725 / 21,207 (22%)
Total genes remaining after convert_orthologs :
   16,482 / 21,207 (78%)
--

=========== REPORT SUMMARY ===========

16,482 / 21,207 (77.72%) target_species genes remain after ortholog conversion.
16,482 / 19,129 (86.16%) reference_species genes remain after ortholog conversion.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--

=========== REPORT SUMMARY ===========

19,129 / 19,129 (100%) target_species genes remain after ortholog conversion.
19,129 / 19,129 (100%) reference_species genes remain after ortholog conversion.
16,482 intersect background genes used.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
Returning 19,129 unique genes from entire human genome.
Using intersect between background gene lists: 16,482 genes.
Standardising sct_data.
Converting to sparse matrix.
Converting to sparse matrix.
Using 1st column of tt as gene column: HGNC.symbol
1 core(s) assigned as workers (31 reserved).
Standardising CellTypeDataset
Checking gene list inputs.
Returning 16,482 unique genes from the user-supplied bg.
Standardising sct_data.
Converting gene list input to standardised human genes.
Running without gene size control.
28 hit genes remain after filtering.
Testing for enrichment in 7 cell types...
Sorting results by p-value.
Computing BH-corrected q-values.
1 significant cell type enrichment results @ q&lt;0.05 : 

           CellType annotLevel p fold_change sd_from_mean q
1 endothelial_mural          1 0    1.628635     4.684982 0
1 core(s) assigned as workers (31 reserved).
Standardising CellTypeDataset
Checking gene list inputs.
Returning 16,482 unique genes from the user-supplied bg.
Standardising sct_data.
Converting gene list input to standardised human genes.
Running without gene size control.
26 hit genes remain after filtering.
Testing for enrichment in 7 cell types...
Sorting results by p-value.
Computing BH-corrected q-values.
3 significant cell type enrichment results @ q&lt;0.05 : 

       CellType annotLevel p fold_change sd_from_mean q
1  pyramidal_SS          1 0    1.341278     3.850814 0
2 pyramidal_CA1          1 0    1.214856     2.192119 0
3  interneurons          1 0    1.144435     1.745605 0
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
Warning: genelistSpecies not provided. Setting to &#x27;human&#x27; by default.
Preparing gene_df.
character format detected.
Converting to data.frame
Extracting genes from input_gene.
15,259 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
13,416 genes extracted.
Extracting genes from ortholog_gene.
13,416 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 46 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 56 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Returning gene_map as dictionary

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   2,016 / 15,259 (13%)
Total genes remaining after convert_orthologs :
   13,243 / 15,259 (87%)
Generating gene background for mouse x human ==&gt; human
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
21,207 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
17,355 genes extracted.
Extracting genes from ortholog_gene.
17,355 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 131 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 498 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   4,725 / 21,207 (22%)
Total genes remaining after convert_orthologs :
   16,482 / 21,207 (78%)
--

=========== REPORT SUMMARY ===========

16,482 / 21,207 (77.72%) target_species genes remain after ortholog conversion.
16,482 / 19,129 (86.16%) reference_species genes remain after ortholog conversion.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--

=========== REPORT SUMMARY ===========

19,129 / 19,129 (100%) target_species genes remain after ortholog conversion.
19,129 / 19,129 (100%) reference_species genes remain after ortholog conversion.
16,482 intersect background genes used.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
Returning 19,129 unique genes from entire human genome.
Using intersect between background gene lists: 16,482 genes.
Standardising sct_data.
Converting to sparse matrix.
Converting to sparse matrix.
Using 1st column of tt as gene column: HGNC.symbol
1 core(s) assigned as workers (31 reserved).
Standardising CellTypeDataset
Checking gene list inputs.
Returning 16,482 unique genes from the user-supplied bg.
Standardising sct_data.
Converting gene list input to standardised human genes.
Running without gene size control.
10 hit genes remain after filtering.
Testing for enrichment in 7 cell types...
Sorting results by p-value.
Computing BH-corrected q-values.
0 significant cell type enrichment results @ q&lt;0.05 : 

1 core(s) assigned as workers (31 reserved).
Standardising CellTypeDataset
Checking gene list inputs.
Returning 16,482 unique genes from the user-supplied bg.
Standardising sct_data.
Converting gene list input to standardised human genes.
Running without gene size control.
8 hit genes remain after filtering.
Testing for enrichment in 7 cell types...
Sorting results by p-value.
Computing BH-corrected q-values.
0 significant cell type enrichment results @ q&lt;0.05 : 

Warning: genelistSpecies not provided. Setting to &#x27;human&#x27; by default.
Preparing gene_df.
character format detected.
Converting to data.frame
Extracting genes from input_gene.
15,259 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
13,416 genes extracted.
Extracting genes from ortholog_gene.
13,416 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 46 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 56 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Returning gene_map as dictionary

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   2,016 / 15,259 (13%)
Total genes remaining after convert_orthologs :
   13,243 / 15,259 (87%)
Generating gene background for mouse x human ==&gt; human
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
21,207 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
17,355 genes extracted.
Extracting genes from ortholog_gene.
17,355 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 131 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 498 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   4,725 / 21,207 (22%)
Total genes remaining after convert_orthologs :
   16,482 / 21,207 (78%)
--

=========== REPORT SUMMARY ===========

16,482 / 21,207 (77.72%) target_species genes remain after ortholog conversion.
16,482 / 19,129 (86.16%) reference_species genes remain after ortholog conversion.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--

=========== REPORT SUMMARY ===========

19,129 / 19,129 (100%) target_species genes remain after ortholog conversion.
19,129 / 19,129 (100%) reference_species genes remain after ortholog conversion.
16,482 intersect background genes used.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
Returning 19,129 unique genes from entire human genome.
Using intersect between background gene lists: 16,482 genes.
Standardising sct_data.
Converting to sparse matrix.
Converting to sparse matrix.
Using 1st column of tt as gene column: HGNC.symbol
1 core(s) assigned as workers (31 reserved).
Standardising CellTypeDataset
Checking gene list inputs.
Returning 16,482 unique genes from the user-supplied bg.
Standardising sct_data.
Converting gene list input to standardised human genes.
Running without gene size control.
11 hit genes remain after filtering.
Testing for enrichment in 7 cell types...
Sorting results by p-value.
Computing BH-corrected q-values.
1 significant cell type enrichment results @ q&lt;0.05 : 

          CellType annotLevel p fold_change sd_from_mean q
1 oligodendrocytes          1 0    1.680834      3.07176 0
1 core(s) assigned as workers (31 reserved).
Standardising CellTypeDataset
Checking gene list inputs.
Returning 16,482 unique genes from the user-supplied bg.
Standardising sct_data.
Converting gene list input to standardised human genes.
Running without gene size control.
9 hit genes remain after filtering.
Testing for enrichment in 7 cell types...
Sorting results by p-value.
Computing BH-corrected q-values.
0 significant cell type enrichment results @ q&lt;0.05 : 

see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
Filtering CTD to 100 genes.
level: 1
Found 5 matrix types across 1 CTD levels.
   - mean_exp
   - median_exp
   - specificity
   - median_specificity
   - specificity_quantiles
level: 2
Found 5 matrix types across 1 CTD levels.
   - mean_exp
   - median_exp
   - specificity
   - median_specificity
   - specificity_quantiles
Standardising CellTypeDataset
Found 5 matrix types across 2 CTD levels.
Processing level: 1
Converting to sparse matrix.
Processing level: 2
Converting to sparse matrix.
Filtering CTD to 100 genes.
level: level_1
Found 5 matrix types across 1 CTD levels.
   - mean_exp
   - median_exp
   - specificity
   - median_specificity
   - specificity_quantiles
level: level_2
Found 5 matrix types across 1 CTD levels.
   - mean_exp
   - median_exp
   - specificity
   - median_specificity
   - specificity_quantiles
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
Check 1000Check 3005
1 core(s) assigned as workers (31 reserved).
Converting to sparse matrix.
Checking for non-expressed genes.
Checking for cells with no expressed genes.
DGE:: Limma...
248 / 1,000 genes dropped @ DGE adj_pval_thresh &lt; 1e-05
Time difference of 1.407872 secs
1 core(s) assigned as workers (31 reserved).
Converting to sparse matrix.
+ Calculating normalized mean expression.
Converting to sparse matrix.
Converting to sparse matrix.
+ Calculating normalized specificity.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
+ Saving results ==&gt;  /tmp/RtmptEOog7/ctd_kiCortexOnly.rda
+ Processing level 1 ...
Processing mean_exp
Processing specificity
Processing specificity_quantiles
+ Processing level 2 ...
Processing mean_exp
Processing specificity
Processing specificity_quantiles
No input_species provided. Setting to &#x27;mouse&#x27;
+ Processing level 1 ...
Processing mean_exp
Processing specificity
Processing specificity_quantiles
+ Processing level 2 ...
Processing mean_exp
Processing specificity
Processing specificity_quantiles
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
trying URL &#x27;http://www.informatics.jax.org/downloads/reports/MRK_List2.rpt&#x27;
Content type &#x27;unknown&#x27; length 82628227 bytes (78.8 MB)
==================================================
downloaded 78.8 MB

see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
1 rows do not have proper MGI symbols
Mar-01
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
1 rows do not have proper MGI symbols
Mar-01
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
1 rows do not have proper MGI symbols
Mar-01
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
10 of 10 are not proper HGNC symbols.
Possible corruption of gene names by excel: Mar-01
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
10 of 10 are not proper HGNC symbols.
Possible corruption of gene names by excel: Mar-01
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
10 of 10 are not proper HGNC symbols.
Possible corruption of gene names by excel: Mar-01
Maps last updated on: Thu Oct 24 12:31:05 2019
Maps last updated on: Thu Oct 24 12:31:05 2019
9 of 10 gene symbols corrected.
1 of 10 gene symbols cannot be mapped.
Converting to sparse matrix.
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
1 rows do not have proper MGI symbols
Tm4sf12
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
1 poorly annotated genes are replicates of existing genes. These are: 
Tspan12
Converting to sparse matrix.
1 rows should have been corrected by checking synonyms.
0 rows STILL do not have proper MGI symbols.

see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
0 rows do not have proper MGI symbols

see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
669 rows do not have proper MGI symbols
2310042E22Rik, BC005764, C130030K03Rik, Stmn1-rs1, Gm9846, E130309F12Rik, Fam211b, AI848285, Acpl2, 9630033F20Rik, Adrbk2, Syne1_loc2, Adc, Dlx1os, LOC106740, Pdxp, Atp6v0c-ps2, 2900056M20Rik, Epb4.1l1, A330050F15Rik
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
308 rows do not have proper MGI symbols
2310042E22Rik, BC005764, C130030K03Rik, Stmn1-rs1, Gm9846, E130309F12Rik, Fam211b, AI848285, Acpl2, 9630033F20Rik, Adrbk2, Syne1_loc2, Adc, Dlx1os, LOC106740, Pdxp, Atp6v0c-ps2, 2900056M20Rik, Epb4.1l1, A330050F15Rik
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
1 poorly annotated genes are replicates of existing genes. These are: 
Hjurp
Converting to sparse matrix.
185 rows should have been corrected by checking synonyms.
126 rows STILL do not have proper MGI symbols.
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
308 rows do not have proper MGI symbols
2310042E22Rik, BC005764, C130030K03Rik, Stmn1-rs1, Gm9846, E130309F12Rik, Fam211b, AI848285, Acpl2, 9630033F20Rik, Adrbk2, Syne1_loc2, Adc, Dlx1os, LOC106740, Pdxp, Atp6v0c-ps2, 2900056M20Rik, Epb4.1l1, A330050F15Rik
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
1 poorly annotated genes are replicates of existing genes. These are: 
Hjurp
Converting to sparse matrix.
185 rows should have been corrected by checking synonyms.
126 rows STILL do not have proper MGI symbols.
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
1 core(s) assigned as workers (31 reserved).
Preparing gene_df.
Dense matrix format detected.
Extracting genes from rownames.
100 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
91 genes extracted.
Extracting genes from ortholog_gene.
91 genes extracted.
Checking for genes without 1:1 orthologs.
Filtering gene_df with gene_map
Setting ortholog_gene to rownames.
Converting obj to sparseMatrix.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   9 / 100 (9%)
Total genes remaining after convert_orthologs :
   91 / 100 (91%)
Converting to sparse matrix.
Converting to DelayedArray.
+ Calculating normalized mean expression.
Converting to sparse matrix.
Converting to sparse matrix.
+ Calculating normalized specificity.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
Converting to sparse matrix.
+ Saving results ==&gt;  /tmp/RtmptEOog7/ctd_allKImouse.rda
+ Returning list of CTD file name, and the CTD itself.
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
Filtering by variance quantiles.
Converting to DelayedArray.
N remaining at each quantile filter:
 &gt;=0  &gt;=1  &gt;=2  &gt;=3  &gt;=4  &gt;=5  &gt;=6  &gt;=7  &gt;=8  &gt;=9 &gt;=10 
 300  155   99   77   60   54   46   41   40   38   36 
246 / 300 genes dropped @ DGE min_variance_quantile &gt;= 5
Converting to sparse matrix.
Calculating cell attributes from input UMI matrix: log_umi
Variance stabilizing transformation of count matrix of size 300 by 3005
Model formula is y ~ log_umi
Get Negative Binomial regression parameters per gene
Using 300 genes, 3005 cells
  |                                                                              |                                                                      |   0%  |                                                                              |======================================================================| 100%
Found 2 outliers - those will be ignored in fitting/regularization step

Second step: Get residuals using fitted parameters for 300 genes
  |                                                                              |                                                                      |   0%  |                                                                              |======================================================================| 100%
Calculating gene attributes
Wall clock passed: Time difference of 8.738413 secs
Computing corrected UMI count matrix
  |                                                                              |                                                                      |   0%  |                                                                              |======================================================================| 100%
Filtering by variance quantiles.
Converting to DelayedArray.
N remaining at each quantile filter:
&gt;=0 
300 
300 / 300 genes dropped @ DGE min_variance_quantile &gt;= 5
Filtering by variance quantiles.
Converting to DelayedArray.
N remaining at each quantile filter:
 &gt;=0  &gt;=1  &gt;=2  &gt;=3  &gt;=4  &gt;=5  &gt;=6  &gt;=7  &gt;=8  &gt;=9 &gt;=10 
 300  293  284  273  263  255  247  237  229  224  218 
45 / 300 genes dropped @ DGE min_variance_quantile &gt;= 5
Filtering by variance quantiles.
Converting to DelayedArray.
/ Reading and realizing block 1/1 ... OK
\ Processing it ... OK
N remaining at each quantile filter:
 &gt;=0  &gt;=1  &gt;=2  &gt;=3  &gt;=4  &gt;=5  &gt;=6  &gt;=7  &gt;=8  &gt;=9 &gt;=10 
 300  273  253  231  218  209  194  181  174  161  152 
91 / 300 genes dropped @ DGE min_variance_quantile &gt;= 5
Retrieving all organisms available in homologene.
Returning table with all species.
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
+ CTD ==&gt; SingleCellExperiment
Converting level:  level_1
Returning all possible matrix names.
Converting level:  level_2
Returning all possible matrix names.
The assay &#x27;mean_exp&#x27; will be used to determine the proportion of zeroes for each batch.
Converting CTD to merged SCEs
ctd1
+ CTD ==&gt; SingleCellExperiment
Converting level:  level_1
Returning all possible matrix names.
Converting level:  level_2
Returning all possible matrix names.
ctd2
+ CTD ==&gt; SingleCellExperiment
Converting level:  level_1
Returning all possible matrix names.
Converting level:  level_2
Returning all possible matrix names.
Merging SCE at level: 1
[1] &quot;ctd1&quot;
[1] &quot;15259 x 7&quot;
[1] &quot;ctd2&quot;
[1] &quot;15259 x 7&quot;
The assay &#x27;mean_exp&#x27; will be used to determine the proportion of zeroes for each batch.
Merging SCE at level: 2
[1] &quot;ctd1&quot;
[1] &quot;15259 x 48&quot;
[1] &quot;ctd2&quot;
[1] &quot;15259 x 48&quot;
The assay &#x27;mean_exp&#x27; will be used to determine the proportion of zeroes for each batch.
[1] &quot;level_1&quot;
[1] &quot;level_2&quot;
Saving SCE_merged ==&gt; /tmp/RtmptEOog7/merged/CTD_SCE_merged.union.rds
+ Must set `as_SCE=TRUE` in order to merge CTD.
+ Returning merged CTD.
1 core(s) assigned as workers (31 reserved).
Converting to sparse matrix.
+ Calculating normalized mean expression.
Converting to sparse matrix.
+ Calculating normalized specificity.
Converting to sparse matrix.
Converting to sparse matrix.
+ Saving results ==&gt;  /tmp/RtmptEOog7/ctd_file7896e1a165af5level_1.rda
+ Returning list of CTD file name, and the CTD itself.
1 core(s) assigned as workers (31 reserved).
Converting to sparse matrix.
+ Calculating normalized mean expression.
Converting to sparse matrix.
+ Calculating normalized specificity.
Converting to sparse matrix.
Converting to sparse matrix.
+ Saving results ==&gt;  /tmp/RtmptEOog7/ctd_file7896e2088cff8level_2.rda
+ Returning list of CTD file name, and the CTD itself.
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
0 non-overlapping gene(s) removed from exp1.
0 non-overlapping gene(s) removed from exp2.
19,972 intersecting genes remain.
Converting to data.frame
Converting to data.frame
Converting to sparse matrix.
see ?ewceData and browseVignettes(&#x27;ewceData&#x27;) for documentation
loading from cache
DGE:: Limma...
Loading required namespace: DESeq2
DGE:: DESeq2...
1 core(s) assigned as workers (31 reserved).
converting counts to integer mode
  Note: levels of factors in the design contain characters other than
  letters, numbers, &#x27;_&#x27; and &#x27;.&#x27;. It is recommended (but not required) to use
  only letters, numbers, and delimiters &#x27;_&#x27; or &#x27;.&#x27;, as these are safe characters
  for column names in R. [This is a message, not a warning or an error]
estimating size factors
  Note: levels of factors in the design contain characters other than
  letters, numbers, &#x27;_&#x27; and &#x27;.&#x27;. It is recommended (but not required) to use
  only letters, numbers, and delimiters &#x27;_&#x27; or &#x27;.&#x27;, as these are safe characters
  for column names in R. [This is a message, not a warning or an error]
estimating dispersions
gene-wise dispersion estimates
mean-dispersion relationship
-- note: fitType=&#x27;parametric&#x27;, but the dispersion trend was not well captured by the
   function: y = a/x + b, and a local regression fit was automatically substituted.
   specify fitType=&#x27;local&#x27; or &#x27;mean&#x27; to avoid this message next time.
  Note: levels of factors in the design contain characters other than
  letters, numbers, &#x27;_&#x27; and &#x27;.&#x27;. It is recommended (but not required) to use
  only letters, numbers, and delimiters &#x27;_&#x27; or &#x27;.&#x27;, as these are safe characters
  for column names in R. [This is a message, not a warning or an error]
final dispersion estimates
  Note: levels of factors in the design contain characters other than
  letters, numbers, &#x27;_&#x27; and &#x27;.&#x27;. It is recommended (but not required) to use
  only letters, numbers, and delimiters &#x27;_&#x27; or &#x27;.&#x27;, as these are safe characters
  for column names in R. [This is a message, not a warning or an error]
fitting model and testing
  Note: levels of factors in the design contain characters other than
  letters, numbers, &#x27;_&#x27; and &#x27;.&#x27;. It is recommended (but not required) to use
  only letters, numbers, and delimiters &#x27;_&#x27; or &#x27;.&#x27;, as these are safe characters
  for column names in R. [This is a message, not a warning or an error]
Loading required namespace: MAST
`fData` has no primerid.  I&#x27;ll make something up.
`cData` has no wellKey.  I&#x27;ll make something up.
[ FAIL 0 | WARN 40 | SKIP 0 | PASS 111 ]

[ FAIL 0 | WARN 40 | SKIP 0 | PASS 111 ]
&gt; 
&gt; proc.time()
    user   system  elapsed 
 730.296   32.727 1492.037 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">EWCE.Rcheck/EWCE-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>add_res_to_merging_list</TD><TD style="text-align: right;"> 9.607</TD><TD style="text-align: right;"> 0.752</TD><TD style="text-align: right;">15.713</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bin_columns_into_quantiles</TD><TD style="text-align: right;">6.123</TD><TD style="text-align: right;">0.400</TD><TD style="text-align: right;">7.475</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bin_specificity_into_quantiles</TD><TD style="text-align: right;">5.727</TD><TD style="text-align: right;">0.356</TD><TD style="text-align: right;">7.558</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bootstrap_enrichment_test</TD><TD style="text-align: right;">18.146</TD><TD style="text-align: right;"> 0.876</TD><TD style="text-align: right;">21.436</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>check_ewce_genelist_inputs</TD><TD style="text-align: right;">15.617</TD><TD style="text-align: right;"> 0.829</TD><TD style="text-align: right;">18.958</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>check_percent_hits</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>controlled_geneset_enrichment</TD><TD style="text-align: right;">39.537</TD><TD style="text-align: right;"> 1.031</TD><TD style="text-align: right;">44.629</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ctd_to_sce</TD><TD style="text-align: right;">6.277</TD><TD style="text-align: right;">0.196</TD><TD style="text-align: right;">8.008</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>drop_uninformative_genes</TD><TD style="text-align: right;">5.636</TD><TD style="text-align: right;">0.422</TD><TD style="text-align: right;">8.578</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ewce_expression_data</TD><TD style="text-align: right;">17.640</TD><TD style="text-align: right;"> 0.945</TD><TD style="text-align: right;">21.114</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ewce_plot</TD><TD style="text-align: right;">0.284</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.288</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>example_bootstrap_results</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>example_transcriptome_results</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filter_ctd_genes</TD><TD style="text-align: right;">11.266</TD><TD style="text-align: right;"> 0.456</TD><TD style="text-align: right;">13.153</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filter_genes_without_1to1_homolog</TD><TD style="text-align: right;">19.042</TD><TD style="text-align: right;"> 0.551</TD><TD style="text-align: right;">21.107</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filter_nonorthologs</TD><TD style="text-align: right;">18.928</TD><TD style="text-align: right;"> 0.588</TD><TD style="text-align: right;">20.936</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fix_bad_hgnc_symbols</TD><TD style="text-align: right;">1.321</TD><TD style="text-align: right;">0.132</TD><TD style="text-align: right;">3.931</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fix_bad_mgi_symbols</TD><TD style="text-align: right;"> 7.410</TD><TD style="text-align: right;"> 0.673</TD><TD style="text-align: right;">11.063</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fix_celltype_names</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>generate_bootstrap_plots</TD><TD style="text-align: right;">17.145</TD><TD style="text-align: right;"> 1.124</TD><TD style="text-align: right;">20.264</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>generate_bootstrap_plots_for_transcriptome</TD><TD style="text-align: right;">19.733</TD><TD style="text-align: right;"> 0.984</TD><TD style="text-align: right;">23.083</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>generate_celltype_data</TD><TD style="text-align: right;">5.954</TD><TD style="text-align: right;">0.689</TD><TD style="text-align: right;">8.082</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_celltype_table</TD><TD style="text-align: right;">4.520</TD><TD style="text-align: right;">0.464</TD><TD style="text-align: right;">6.517</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>list_species</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>load_rdata</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>merge_ctd</TD><TD style="text-align: right;"> 9.403</TD><TD style="text-align: right;"> 0.618</TD><TD style="text-align: right;">11.756</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>merge_sce</TD><TD style="text-align: right;">6.752</TD><TD style="text-align: right;">0.304</TD><TD style="text-align: right;">8.567</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>merge_two_expfiles</TD><TD style="text-align: right;">4.804</TD><TD style="text-align: right;">0.508</TD><TD style="text-align: right;">8.945</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>merged_ewce</TD><TD style="text-align: right;">32.612</TD><TD style="text-align: right;"> 1.609</TD><TD style="text-align: right;">41.693</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_ctd</TD><TD style="text-align: right;">4.694</TD><TD style="text-align: right;">0.197</TD><TD style="text-align: right;">6.368</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sct_normalize</TD><TD style="text-align: right;">13.864</TD><TD style="text-align: right;"> 0.811</TD><TD style="text-align: right;">16.607</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>standardise_ctd</TD><TD style="text-align: right;">13.586</TD><TD style="text-align: right;"> 0.620</TD><TD style="text-align: right;">18.843</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
