<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for crlmm on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><B>[<A href="..#C">C</A>]</B></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-10 10:04:28 -0000 (Wed, 10 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-03-12&nbsp;r83975)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">6211</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for crlmm on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the crlmm package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/crlmm.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>450</B>/2194</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">crlmm</A>&nbsp;1.59.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/crlmm">landing page</A>)</SPAN><BR>Benilton S Carvalho<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-08&nbsp;19:11:19&nbsp;-0000&nbsp;(Mon,&nbsp;08&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/crlmm</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">b4a59d2</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;13:48:29&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: crlmm</TD></TR>
<TR><TD><B>Version</B>: 1.59.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:crlmm.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings crlmm_1.59.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-09 13:08:26 -0000 (Tue, 09 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-09 13:26:53 -0000 (Tue, 09 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 1107.5 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: crlmm.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 1</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:crlmm.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings crlmm_1.59.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/crlmm.Rcheck’
* using R Under development (unstable) (2023-03-12 r83975)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘crlmm/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘crlmm’ version ‘1.59.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘crlmm’ can be installed ... OK
* used C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... NOTE
  installed size is  5.0Mb
  sub-directories of 1Mb or more:
    R      1.3Mb
    data   2.7Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... NOTE
Versioned &#x27;LinkingTo&#x27; value for ‘preprocessCore’ is only usable in R &gt;= 3.0.2
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking startup messages can be suppressed ... NOTE
No methods found in package ‘oligoClasses’ for request: ‘mean’ when loading ‘crlmm’

It looks like this package (or a package it requires) has a startup
message which cannot be suppressed: see ?packageStartupMessage.
* checking dependencies in R code ... NOTE
&#x27;library&#x27; or &#x27;require&#x27; call to ‘RUnit’ in package code.
  Please use :: or requireNamespace() instead.
  See section &#x27;Suggested packages&#x27; in the &#x27;Writing R Extensions&#x27; manual.
Namespace in Imports field not imported from: ‘splines’
  All declared Imports should be used.
Unexported object imported by a &#x27;:::&#x27; call: ‘Biobase:::assayDataEnvLock’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
.test: no visible global function definition for ‘defineTestSuite’
.test: no visible global function definition for ‘runTestSuite’
.test: no visible global function definition for ‘printTextProtocol’
calculateRBafCNSet : processByChromosome: no visible global function
  definition for ‘position’
genotypeInf: no visible binding for global variable ‘anno’
krlmm: no visible binding for global variable ‘VGLMparameters’
Undefined global functions or variables:
  VGLMparameters anno defineTestSuite position printTextProtocol
  runTestSuite
* checking Rd files ... WARNING
checkRd: (5) CNSet-methods.Rd:39-41: \item in \describe must have non-empty label
checkRd: (5) CNSet-methods.Rd:65-67: \item in \describe must have non-empty label
checkRd: (5) CNSet-methods.Rd:69-71: \item in \describe must have non-empty label
checkRd: (-1) constructInf.Rd:35: Escaped LaTeX specials: \_ \_
checkRd: (-1) genotype.Illumina.Rd:48: Escaped LaTeX specials: \_ \_
checkRd: (-1) preprocessInf.Rd:55: Escaped LaTeX specials: \_ \_
checkRd: (-1) readIdatFiles.Rd:39: Escaped LaTeX specials: \_ \_
checkRd: (-1) readIdatFiles.Rd:55: Escaped LaTeX specials: \_ \_
checkRd: (-1) readIdatFiles.Rd:56: Escaped LaTeX specials: \_
checkRd: (-1) snprma.Rd:38: Escaped LaTeX specials: \_ \_
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking for GNU extensions in Makefiles ... OK
* checking include directives in Makefiles ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
            user system elapsed
crlmm    123.673  2.156 125.836
genotype  97.892  4.319 102.225
snprma    22.748  0.996  23.744
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘crlmm_unit_tests.R’
  Running ‘doRUnit.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘AffyGW.Rnw’... OK
  ‘CopyNumberOverview.Rnw’... OK
  ‘IlluminaPreprocessCN.Rnw’... OK
  ‘Infrastructure.Rnw’... OK
  ‘genotyping.Rnw’... OK
  ‘gtypeDownstream.Rnw’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 WARNING, 6 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/crlmm.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">crlmm.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL crlmm
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library’
* installing *source* package ‘crlmm’ ...
** using staged installation
** libs
using C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/preprocessCore/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c gtypeCaller.c -o gtypeCaller.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/preprocessCore/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c init.c -o init.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/preprocessCore/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c trimmed.c -o trimmed.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/preprocessCore/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c utils.c -o utils.o
gcc -shared -L/usr/local/lib -o crlmm.so gtypeCaller.o init.o trimmed.o utils.o
installing to /home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/00LOCK-crlmm/00new/crlmm/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
No methods found in package ‘oligoClasses’ for request: ‘mean’ when loading ‘crlmm’
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
No methods found in package ‘oligoClasses’ for request: ‘mean’ when loading ‘crlmm’
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
No methods found in package ‘oligoClasses’ for request: ‘mean’ when loading ‘crlmm’
** testing if installed package keeps a record of temporary installation path
* DONE (crlmm)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">crlmm.Rcheck/tests/crlmm_unit_tests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; require(&quot;crlmm&quot;) || stop(&quot;unable to load crlmm package&quot;)
Loading required package: crlmm
Loading required package: oligoClasses
Welcome to oligoClasses version 1.63.0
Loading required package: preprocessCore
No methods found in package &#x27;oligoClasses&#x27; for request: &#x27;mean&#x27; when loading &#x27;crlmm&#x27;
Welcome to crlmm version 1.59.0
[1] TRUE
&gt; crlmm:::.test()
Loading required package: genomewidesnp6Crlmm
Annotation for genomewidesnp6Crlmm version 1.0.7 supports UCSC builds hg18 and hg19.
Loading required package: hapmapsnp6
/-------------------------------------------\
|         SAMPLE HAPMAP SNP 6.0             |
|-------------------------------------------|
| Data obtained from http://www.hapmap.org  |
| This package is meant to be used only for |
| demonstration of BioConductor packages.   |
| Access http://www.hapmap.org for details. |
|-------------------------------------------|
| The contents of this package are provided |
| in good faith and the maintainer does not |
| warrant their accuracy.                   |
\-------------------------------------------/
Loading annotations and mixture model parameters.
Processing 3 files.
  |                                                                              |                                                                      |   0%  |                                                                              |=======================                                               |  33%  |                                                                              |===============================================                       |  67%  |                                                                              |======================================================================| 100%
Loading annotations.
Determining gender.
Calling 906600 SNPs for recalibration... Done.
Estimating recalibration parameters.
Calling 906600 SNPs... Done.
Error in crlmm(cels[c(1, 1, 2)]) : sample identifiers are not unique
In addition: Warning message:
In crlmmGT(res[[&quot;A&quot;]], res[[&quot;B&quot;]], res[[&quot;SNR&quot;]], res[[&quot;mixtureParams&quot;]],  :
  Recalibration not possible. Possible cause: small sample size.


RUNIT TEST PROTOCOL -- Tue May  9 13:19:43 2023 
*********************************************** 
Number of test functions: 3 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
crlmm RUnit Tests - 3 test functions, 0 errors, 0 failures
Number of test functions: 3 
Number of errors: 0 
Number of failures: 0 
&gt; 
&gt; proc.time()
   user  system elapsed 
140.904   2.681 143.581 
</PRE>
</DIV><P><SPAN class="filename">crlmm.Rcheck/tests/doRUnit.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; ## from xmapcore package
&gt; if( require( &quot;RUnit&quot;, quietly=TRUE ) ) {
+ 	pkg &lt;- &quot;crlmm&quot;
+ 
+ 	if( Sys.getenv( &quot;RCMDCHECK&quot; ) == &quot;FALSE&quot; ) {
+ 		path &lt;- file.path( getwd(), &quot;..&quot;, &quot;inst&quot;, &quot;unitTests&quot; )
+ 	} else {
+ 		path &lt;- system.file( package=pkg, &quot;unitTests&quot; )
+ 	}
+ 
+ 	cat( &quot;\nRunning unit tests\n&quot; )
+ 	print( list( pkg=pkg, getwd=getwd(), pathToUnitTests=path ) )
+ 	library( package=pkg, character.only=TRUE )
+ 
+ 	##xmap.clear.cache()
+ 
+ 	## do not fail on warnings (needed for crlmm() test)
+ 	options(warn=0)
+ 
+ 	## Get the pattern (if there is one?)
+ 	patt &lt;- Sys.getenv( &quot;RUNITFILEPATTERN&quot; )
+ 	if( is.null( patt ) || nchar( patt ) == 0 ) {
+ 		testSuite &lt;- defineTestSuite(name=paste( pkg, &quot;unit testing&quot; ),
+ 					     dirs=path,
+ 					     testFileRegexp=paste( &quot;^test.+&quot;, patt, &quot;\\.[rR]$&quot;, sep=&quot;&quot; ))
+ 	} else {
+ 		##testSuite &lt;- defineTestSuite( name=paste( pkg, &quot;unit testing&quot; ), testFileRegexp=paste( &quot;^runit\\.&quot;, patt, &quot;\\.[rR]$&quot;, sep=&quot;&quot; ), dirs=path )
+ 		testSuite &lt;- defineTestSuite(name=paste( pkg, &quot;unit testing&quot; ),
+ 					     testFileRegexp=paste( &quot;^test.+&quot;, patt, &quot;\\.[rR]$&quot;, sep=&quot;&quot; ),
+ 					     dirs=path )
+ 	}
+ 	tests &lt;- runTestSuite( testSuite )
+ 
+ 	pathReport &lt;- file.path( path, &quot;report&quot; )
+ 
+ 	cat( &quot;------------------- UNIT TEST SUMMARY ---------------------\n\n&quot; )
+ 	printTextProtocol( tests, showDetails=FALSE )
+ 	printTextProtocol( tests, showDetails=FALSE, fileName=paste( pathReport, &quot;Summary.txt&quot;, sep=&quot;&quot; ) )
+ 	printTextProtocol( tests, showDetails=TRUE,  fileName=paste( pathReport, &quot;.txt&quot;, sep=&quot;&quot; ) )
+ 
+ 	printHTMLProtocol( tests, fileName=paste( pathReport, &quot;.html&quot;, sep=&quot;&quot; ) )
+ 
+ 	tmp &lt;- getErrors( tests )
+ 	if( tmp$nFail &gt; 0 | tmp$nErr &gt; 0 ){
+ 		stop( paste( &quot;\n\nunit testing failed (#test failures: &quot;, tmp$nFail, &quot;, #R errors: &quot;,  tmp$nErr, &quot;)\n\n&quot;, sep=&quot;&quot;))
+ 	}
+ } else {
+ 	warning( &quot;cannot run unit tests -- package RUnit is not available&quot; )
+ }

Running unit tests
$pkg
[1] &quot;crlmm&quot;

$getwd
[1] &quot;/home/biocbuild/bbs-3.17-bioc/meat/crlmm.Rcheck/tests&quot;

$pathToUnitTests
[1] &quot;/tmp/RtmpoCn9r8/RLIBS_3ac3ea7c326b7c/crlmm/unitTests&quot;

Loading required package: oligoClasses
Welcome to oligoClasses version 1.63.0
Loading required package: preprocessCore
No methods found in package &#x27;oligoClasses&#x27; for request: &#x27;mean&#x27; when loading &#x27;crlmm&#x27;
Welcome to crlmm version 1.59.0


Executing test function test_crlmm  ... Loading required package: genomewidesnp6Crlmm
Annotation for genomewidesnp6Crlmm version 1.0.7 supports UCSC builds hg18 and hg19.
Loading required package: hapmapsnp6
/-------------------------------------------\
|         SAMPLE HAPMAP SNP 6.0             |
|-------------------------------------------|
| Data obtained from http://www.hapmap.org  |
| This package is meant to be used only for |
| demonstration of BioConductor packages.   |
| Access http://www.hapmap.org for details. |
|-------------------------------------------|
| The contents of this package are provided |
| in good faith and the maintainer does not |
| warrant their accuracy.                   |
\-------------------------------------------/
Loading annotations and mixture model parameters.
Processing 3 files.
  |                                                                              |                                                                      |   0%  |                                                                              |=======================                                               |  33%  |                                                                              |===============================================                       |  67%  |                                                                              |======================================================================| 100%
Loading annotations.
Determining gender.
Calling 906600 SNPs for recalibration... Done.
Estimating recalibration parameters.
Calling 906600 SNPs... Done.
 done successfully.



Executing test function test_duplicates  ... Error in crlmm(cels[c(1, 1, 2)]) : sample identifiers are not unique
In addition: Warning messages:
1: In RNGkind(kind = testSuite$rngKind, normal.kind = testSuite$rngNormalKind) :
  RNGkind: Marsaglia-Multicarry has poor statistical properties
2: In RNGkind(kind = testSuite$rngKind, normal.kind = testSuite$rngNormalKind) :
  RNGkind: severe deviations from normality for Kinderman-Ramage + Marsaglia-Multicarry
3: In crlmmGT(res[[&quot;A&quot;]], res[[&quot;B&quot;]], res[[&quot;SNR&quot;]], res[[&quot;mixtureParams&quot;]],  :
  Recalibration not possible. Possible cause: small sample size.
 done successfully.



Executing test function test_dataExamples  ...  done successfully.

------------------- UNIT TEST SUMMARY ---------------------

RUNIT TEST PROTOCOL -- Tue May  9 13:22:02 2023 
*********************************************** 
Number of test functions: 3 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
crlmm unit testing - 3 test functions, 0 errors, 0 failures
Warning messages:
1: In RNGkind(kind = testSuite$rngKind, normal.kind = testSuite$rngNormalKind) :
  RNGkind: Marsaglia-Multicarry has poor statistical properties
2: In RNGkind(kind = testSuite$rngKind, normal.kind = testSuite$rngNormalKind) :
  RNGkind: severe deviations from normality for Kinderman-Ramage + Marsaglia-Multicarry
3: In RNGkind(kind = testSuite$rngKind, normal.kind = testSuite$rngNormalKind) :
  RNGkind: Marsaglia-Multicarry has poor statistical properties
4: In RNGkind(kind = testSuite$rngKind, normal.kind = testSuite$rngNormalKind) :
  RNGkind: severe deviations from normality for Kinderman-Ramage + Marsaglia-Multicarry
&gt; 
&gt; proc.time()
   user  system elapsed 
137.022   2.244 139.264 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">crlmm.Rcheck/crlmm-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>ListClassConstructors</TD><TD style="text-align: right;">1.958</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">1.982</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>PredictionRegion-class</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>batchStatisticAccessors</TD><TD style="text-align: right;">0.241</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.257</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>calculateRBaf</TD><TD style="text-align: right;">1.063</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">1.071</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>celfile-utils</TD><TD style="text-align: right;">1.817</TD><TD style="text-align: right;">0.284</TD><TD style="text-align: right;">2.122</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cnSetExample</TD><TD style="text-align: right;">1.210</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">1.261</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>constructInf</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>copynumberAccessors</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>crlmm</TD><TD style="text-align: right;">123.673</TD><TD style="text-align: right;">  2.156</TD><TD style="text-align: right;">125.836</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genotype.Illumina</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genotype</TD><TD style="text-align: right;"> 97.892</TD><TD style="text-align: right;">  4.319</TD><TD style="text-align: right;">102.225</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genotypeInf</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genotypes</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotSNPs</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>posteriorProbability</TD><TD style="text-align: right;">1.507</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">1.511</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>predictionRegion</TD><TD style="text-align: right;">0.338</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.346</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>preprocessInf</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readGenCallOutput</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readIdatFiles</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snprma</TD><TD style="text-align: right;">22.748</TD><TD style="text-align: right;"> 0.996</TD><TD style="text-align: right;">23.744</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>validCdfNames</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>xyplot</TD><TD style="text-align: right;">0.474</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">0.526</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
