<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for GenomicFeatures on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><B>[<A href="../index.html#G">G</A>]</B></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-09 02:50:23 -0000 (Thu, 09 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4165</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for GenomicFeatures on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the GenomicFeatures package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/GenomicFeatures.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard timeout">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>774</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">GenomicFeatures</A>&nbsp;1.51.4</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/GenomicFeatures">landing page</A>)</SPAN><BR>H. Pagès<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-08&nbsp;03:21:00&nbsp;-0000&nbsp;(Wed,&nbsp;08&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/GenomicFeatures</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">c864561</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-12-15&nbsp;02:06:06&nbsp;-0000&nbsp;(Thu,&nbsp;15&nbsp;Dec&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph TIMEOUT">&nbsp;&nbsp;TIMEOUT&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: GenomicFeatures</TD></TR>
<TR><TD><B>Version</B>: 1.51.4</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:GenomicFeatures.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings GenomicFeatures_1.51.4.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-08 20:09:22 -0000 (Wed, 08 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-08 20:49:22 -0000 (Wed, 08 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 2400.0 seconds</TD></TR>
<TR><TD><B>RetCode</B>: None</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph TIMEOUT">&nbsp;&nbsp;TIMEOUT&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: GenomicFeatures.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:GenomicFeatures.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings GenomicFeatures_1.51.4.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/GenomicFeatures.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘GenomicFeatures/DESCRIPTION’ ... OK
* this is package ‘GenomicFeatures’ version ‘1.51.4’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Depends: includes the non-default packages:
  &#x27;BiocGenerics&#x27;, &#x27;S4Vectors&#x27;, &#x27;IRanges&#x27;, &#x27;GenomeInfoDb&#x27;,
  &#x27;GenomicRanges&#x27;, &#x27;AnnotationDbi&#x27;
Adding so many packages to the search path is excessive and importing
selectively is preferable.
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘GenomicFeatures’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
&#x27;:::&#x27; call which should be &#x27;::&#x27;: ‘rtracklayer:::tableNames’
  See the note in ?`:::` about the use of this operator.
Unexported objects imported by &#x27;:::&#x27; calls:
  ‘AnnotationDbi:::.getMetaValue’ ‘AnnotationDbi:::.valid.colnames’
  ‘AnnotationDbi:::.valid.metadata.table’
  ‘AnnotationDbi:::.valid.table.colnames’ ‘AnnotationDbi:::dbEasyQuery’
  ‘AnnotationDbi:::dbQuery’ ‘AnnotationDbi:::smartKeys’
  ‘BiocGenerics:::testPackage’ ‘GenomeInfoDb:::check_tax_id’
  ‘GenomeInfoDb:::getSeqlevelsReplacementMode’
  ‘GenomeInfoDb:::lookup_organism_by_tax_id’
  ‘GenomeInfoDb:::lookup_tax_id_by_organism’
  ‘GenomeInfoDb:::make_circ_flags_from_circ_seqs’
  ‘GenomeInfoDb:::normarg_new2old’
  ‘GenomicRanges:::unsafe.transcriptLocs2refLocs’
  ‘GenomicRanges:::unsafe.transcriptWidths’
  ‘IRanges:::regroupBySupergroup’ ‘S4Vectors:::V_recycle’
  ‘S4Vectors:::anyMissingOrOutside’ ‘S4Vectors:::decodeRle’
  ‘S4Vectors:::extract_data_frame_rows’ ‘S4Vectors:::quick_togroup’
  ‘biomaRt:::martBM’ ‘biomaRt:::martDataset’ ‘biomaRt:::martHost’
  ‘rtracklayer:::resourceDescription’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... WARNING
checkRd: (5) FeatureDb-class.Rd:31-34: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:30-33: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:34-39: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:40-43: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:44-52: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:53-59: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:60-65: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:66-71: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:72-80: \item in \describe must have non-empty label
checkRd: (5) select-methods.Rd:21-25: \item in \describe must have non-empty label
checkRd: (5) select-methods.Rd:26-50: \item in \describe must have non-empty label
checkRd: (5) select-methods.Rd:51-55: \item in \describe must have non-empty label
checkRd: (5) select-methods.Rd:56-62: \item in \describe must have non-empty label
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                              user system  elapsed
makeFeatureDbFromUCSC      158.765 11.853 1182.154
coverageByTranscript       112.206  3.168  115.377
coordinate-mapping-methods  79.303  5.812   94.876
makeTxDbFromBiomart         70.909  3.501  390.904
exonicParts                 56.540  0.472   57.012
extractTranscriptSeqs       17.010  0.772   17.797
makeTxDbFromUCSC            12.125  1.021   59.198
transcriptLocs2refLocs      10.249  0.911   11.190
makeTxDbFromGFF             10.292  0.276   10.573
transcriptLengths            6.487  0.332    6.820
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘run_unitTests.R’</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">GenomicFeatures.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL GenomicFeatures
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘GenomicFeatures’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (GenomicFeatures)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">GenomicFeatures.Rcheck/tests/run_unitTests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; require(&quot;GenomicFeatures&quot;) || stop(&quot;unable to load GenomicFeatures package&quot;)
Loading required package: GenomicFeatures
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
Loading required package: GenomicRanges
Loading required package: AnnotationDbi
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

[1] TRUE
&gt; GenomicFeatures:::.test()
Loading required package: BSgenome
Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: rtracklayer
Download and preprocess the &#x27;transcripts&#x27; data frame ... OK
Download and preprocess the &#x27;chrominfo&#x27; data frame ... OK
Download and preprocess the &#x27;splicings&#x27; data frame ... OK
Download and preprocess the &#x27;genes&#x27; data frame ... OK
Prepare the &#x27;metadata&#x27; data frame ... OK
Make the TxDb object ... OK
Import genomic features from the file as a GRanges object ... OK
Prepare the &#x27;metadata&#x27; data frame ... OK
Make the TxDb object ... OK
Import genomic features from the file as a GRanges object ... OK
Prepare the &#x27;metadata&#x27; data frame ... OK
Make the TxDb object ... OK
Import genomic features from the file as a GRanges object ... OK
Prepare the &#x27;metadata&#x27; data frame ... OK
Make the TxDb object ... OK
Import genomic features from the file as a GRanges object ... </PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">GenomicFeatures.Rcheck/GenomicFeatures-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>FeatureDb-class</TD><TD style="text-align: right;">0.075</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.080</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TxDb-class</TD><TD style="text-align: right;">0.871</TD><TD style="text-align: right;">0.079</TD><TD style="text-align: right;">0.952</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>as-format-methods</TD><TD style="text-align: right;">2.761</TD><TD style="text-align: right;">0.039</TD><TD style="text-align: right;">2.801</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>coordinate-mapping-methods</TD><TD style="text-align: right;">79.303</TD><TD style="text-align: right;"> 5.812</TD><TD style="text-align: right;">94.876</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>coverageByTranscript</TD><TD style="text-align: right;">112.206</TD><TD style="text-align: right;">  3.168</TD><TD style="text-align: right;">115.377</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>exonicParts</TD><TD style="text-align: right;">56.540</TD><TD style="text-align: right;"> 0.472</TD><TD style="text-align: right;">57.012</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>extendExonsIntoIntrons</TD><TD style="text-align: right;">3.647</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">3.725</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>extractTranscriptSeqs</TD><TD style="text-align: right;">17.010</TD><TD style="text-align: right;"> 0.772</TD><TD style="text-align: right;">17.797</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>extractUpstreamSeqs</TD><TD style="text-align: right;">1.978</TD><TD style="text-align: right;">0.088</TD><TD style="text-align: right;">2.068</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>features</TD><TD style="text-align: right;">0.083</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.084</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getPromoterSeq-methods</TD><TD style="text-align: right;">0.787</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.810</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>id2name</TD><TD style="text-align: right;">0.269</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.277</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeFeatureDbFromUCSC</TD><TD style="text-align: right;"> 158.765</TD><TD style="text-align: right;">  11.853</TD><TD style="text-align: right;">1182.154</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeTxDb</TD><TD style="text-align: right;">0.150</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.155</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeTxDbFromBiomart</TD><TD style="text-align: right;"> 70.909</TD><TD style="text-align: right;">  3.501</TD><TD style="text-align: right;">390.904</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeTxDbFromEnsembl</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeTxDbFromGFF</TD><TD style="text-align: right;">10.292</TD><TD style="text-align: right;"> 0.276</TD><TD style="text-align: right;">10.573</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeTxDbFromGRanges</TD><TD style="text-align: right;">4.714</TD><TD style="text-align: right;">0.236</TD><TD style="text-align: right;">4.952</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeTxDbFromUCSC</TD><TD style="text-align: right;">12.125</TD><TD style="text-align: right;"> 1.021</TD><TD style="text-align: right;">59.198</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeTxDbPackage</TD><TD style="text-align: right;">0.338</TD><TD style="text-align: right;">0.018</TD><TD style="text-align: right;">3.667</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mapIdsToRanges</TD><TD style="text-align: right;">1.011</TD><TD style="text-align: right;">0.105</TD><TD style="text-align: right;">1.116</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mapRangesToIds</TD><TD style="text-align: right;">1.018</TD><TD style="text-align: right;">0.072</TD><TD style="text-align: right;">1.090</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>microRNAs</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nearest-methods</TD><TD style="text-align: right;">0.95</TD><TD style="text-align: right;">0.12</TD><TD style="text-align: right;">1.07</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>proteinToGenome</TD><TD style="text-align: right;">2.506</TD><TD style="text-align: right;">0.272</TD><TD style="text-align: right;">2.779</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>select-methods</TD><TD style="text-align: right;">0.275</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">0.321</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>transcriptLengths</TD><TD style="text-align: right;">6.487</TD><TD style="text-align: right;">0.332</TD><TD style="text-align: right;">6.820</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>transcriptLocs2refLocs</TD><TD style="text-align: right;">10.249</TD><TD style="text-align: right;"> 0.911</TD><TD style="text-align: right;">11.190</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>transcripts</TD><TD style="text-align: right;">4.187</TD><TD style="text-align: right;">0.264</TD><TD style="text-align: right;">4.452</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>transcriptsBy</TD><TD style="text-align: right;">1.309</TD><TD style="text-align: right;">0.056</TD><TD style="text-align: right;">1.364</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>transcriptsByOverlaps</TD><TD style="text-align: right;">0.280</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">0.304</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
