<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for QDNAseq on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><B>[<A href="..#Q">Q</A>]</B></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-10 10:04:38 -0000 (Wed, 10 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-03-12&nbsp;r83975)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">6211</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for QDNAseq on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the QDNAseq package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/QDNAseq.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1564</B>/2194</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">QDNAseq</A>&nbsp;1.37.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/QDNAseq">landing page</A>)</SPAN><BR>Daoud Sie<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-08&nbsp;19:11:19&nbsp;-0000&nbsp;(Mon,&nbsp;08&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/QDNAseq</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">9f21c56</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:30:57&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: QDNAseq</TD></TR>
<TR><TD><B>Version</B>: 1.37.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:QDNAseq.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings QDNAseq_1.37.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-10 01:16:36 -0000 (Wed, 10 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-10 01:25:27 -0000 (Wed, 10 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 530.7 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: QDNAseq.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:QDNAseq.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings QDNAseq_1.37.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/QDNAseq.Rcheck’
* using R Under development (unstable) (2023-03-12 r83975)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘QDNAseq/DESCRIPTION’ ... OK
* this is package ‘QDNAseq’ version ‘1.37.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘QDNAseq’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                         user system elapsed
callBins               22.137  0.236  22.374
frequencyPlot          22.063  0.087  22.152
normalizeSegmentedBins  9.306  0.039   9.347
segmentBins             9.080  0.032   9.112
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘QDNAseq,copyneutral.R’
  Running ‘QDNAseq,gain-copyneutral-gain.R’
  Running ‘QDNAseq,parallel-reproducibility.R’
  Running ‘QDNAseq.R’
  Running ‘binReadCounts.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘QDNAseq.Rnw’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: OK

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">QDNAseq.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL QDNAseq
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library’
* installing *source* package ‘QDNAseq’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (QDNAseq)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">QDNAseq.Rcheck/tests/binReadCounts.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(QDNAseq)
&gt; 
&gt; if (requireNamespace(&quot;QDNAseq.hg19&quot;, quietly = TRUE)) {
+   bins &lt;- getBinAnnotations(500, genome = &quot;hg19&quot;)
+   print(bins)
+ 
+   bam &lt;- system.file(&quot;extdata&quot;, &quot;ex1.bam&quot;, package = &quot;Rsamtools&quot;)
+   print(bam)
+ 
+   counts &lt;- binReadCounts(bins, bamfiles = bam)
+   print(counts)
+ 
+   ## BUG: https://github.com/ccagc/QDNAseq/issues/89
+   counts2 &lt;- binReadCounts(bins, bamfiles = bam, chunkSize = 10e3)
+   print(counts2)
+ }
Loaded bin annotations for genome &#x27;hg19&#x27;, bin size 500 kbp, and experiment type &#x27;SR50&#x27; from annotation package QDNAseq.hg19 v1.31.0
QDNAseq bin annotations for Hsapiens, build hg19.
Created by Ilari Scheinin with QDNAseq 0.7.5, 2014-02-06 14:20:46.041402.
An object of class &#x27;AnnotatedDataFrame&#x27;
  rowNames: 1:1-500000 1:500001-1000000 ... Y:59000001-59373566 (6206
    total)
  varLabels: chromosome start ... use (9 total)
  varMetadata: labelDescription
[1] &quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/Rsamtools/extdata/ex1.bam&quot;
    ex1 (1 of 1): extracting reads ... binning ...
QDNAseqReadCounts (storageMode: lockedEnvironment)
assayData: 6206 features, 1 samples 
  element names: counts 
protocolData: none
phenoData
  sampleNames: ex1
  varLabels: name total.reads used.reads expected.variance
  varMetadata: labelDescription
featureData
  featureNames: 1:1-500000 1:500001-1000000 ... Y:59000001-59373566
    (6206 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
    ex1 (1 of 1): extracting reads ...
binning chunk - seq1:1:10001
binning chunk - seq2:1:10001

QDNAseqReadCounts (storageMode: lockedEnvironment)
assayData: 6206 features, 1 samples 
  element names: counts 
protocolData: none
phenoData
  sampleNames: ex1
  varLabels: name total.reads used.reads expected.variance
  varMetadata: labelDescription
featureData
  featureNames: 1:1-500000 1:500001-1000000 ... Y:59000001-59373566
    (6206 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; 
&gt; 
&gt; proc.time()
   user  system elapsed 
  8.076   0.280   8.344 
</PRE>
</DIV><P><SPAN class="filename">QDNAseq.Rcheck/tests/QDNAseq,copyneutral.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(QDNAseq)
&gt; library(Biobase)
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

&gt; set.seed(0xBEEF)
&gt; 
&gt; data(LGG150)
&gt; data &lt;- LGG150
&gt; print(data)
QDNAseqReadCounts (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: counts 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads used.reads expected.variance
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; dataF &lt;- applyFilters(data, residual=TRUE, blacklist=TRUE)
38,819	total bins
38,819	of which in selected chromosomes
36,722	of which with reference sequence
33,347	final bins
&gt; print(dataF)
QDNAseqReadCounts (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: counts 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads used.reads expected.variance
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; dataC &lt;- correctBins(dataF)
Calculating correction for GC content and mappability
    Calculating fit for sample LGG150 (1 of 1) ...
Done.
&gt; 
&gt; ## Force all copy neutral data
&gt; cn &lt;- assayDataElement(dataC, &quot;copynumber&quot;)
&gt; cn[,1] &lt;- rnorm(nrow(cn), mean = 1.0, sd = 0.05)
&gt; assayDataElement(dataC, &quot;copynumber&quot;) &lt;- cn
&gt; 
&gt; print(dataC)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: copynumber 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; dataN &lt;- normalizeBins(dataC)
Applying median normalization ...
&gt; print(dataN)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: copynumber 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; fit &lt;- segmentBins(dataN)
Performing segmentation:
    Segmenting: LGG150 (1 of 1) ...
&gt; print(fit)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: copynumber, segmented 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; fitC &lt;- callBins(fit)
EM algorithm started ... 

[1] &quot;Total number of segments present in the data: 4&quot;
[1] &quot;Number of segments used for fitting the model: 4&quot;
39950258171793213.462.4632841514786712338112.963284151223663833893.4
Calling iteration1:

optim results
time: 0
minimum: 16682.0890247004
116682.0890247004-0.825303675797091-0.323169506423803-0.001754401929814960.2970863141060160.5078724084702190.9928769615298530.1100346090918040.0003692713871108020.0111811691191362-0.0001065348234756080.1102701734224670.145866377811007
39965308175214213.562.4632841514786712338112.963284151223663833893.4
Calling iteration2:

optim results
time: 0
minimum: 16682.0888522346
116682.0888522346-0.816384144994661-0.31848916294222-0.00167850977268490.292952991143050.5008064461942260.9819310567480430.1311557982956840.0002223141320088620.01118096583020170.0001872099918784220.1313921659017140.175722206269368
EM algorithm done ...

Computing posterior probabilities for all segments ...

Total time:0minutes

Adjusting segmented data for cellularity ... 

Cellularity sample1: 1

Adjusting normalized data for cellularity ... 

Cellularity sample1: 1

1
39991988396968213.664.1632841514786712338112.963284151223663833893.4
39992178563714213.665.4632841514786712338112.963284151223663833893.4
39992148563708213.665.4632841514786712338112.963284151223663833893.4
39992428830488213.667.463284151782405433813663284151223663833893.4
39996698831443213.767.463284151782405433813663284151223663833893.4
39996788831447213.767.463284151782405433813663284151223663833893.4
39996878831451213.767.463284151782405433813663284151223663833893.4
39996968831455213.767.463284151782405433813663284151223663833893.4
39997058831459213.767.463284151782405433813663284151223663833893.4
39997118831462213.767.463284151782405433813663284151223663833893.4
39997308898192213.767.963284151782405433813663284151223663833893.4
40001478898515213.767.963284151782405433813663284151223663833893.4
FINISHED!

Total time:0minutes

There were 24 warnings (use warnings() to see them)
&gt; print(fitC)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: calls, copynumber, probamp, probdloss, probgain, probloss, probnorm, segmented 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; 
&gt; ## Assert that everything is called copy-neutral
&gt; calls &lt;- assayDataElement(fitC, &quot;calls&quot;)
&gt; stopifnot(all(is.na(calls) | calls == 0))
&gt; 
&gt; formats &lt;- c(&quot;tsv&quot;, &quot;igv&quot;, &quot;bed&quot;, &quot;seg&quot;, &quot;vcf&quot;)
&gt; types &lt;- c(&quot;copynumber&quot;, &quot;segments&quot;, &quot;calls&quot;)
&gt; for (format in formats) {
+   for (type in types) {
+     fileext &lt;- sprintf(&quot;.%s.%s&quot;, type, format)
+     file &lt;- tempfile(pattern =  &quot;QDNAseq-%s&quot;, fileext = fileext)
+     message(sprintf(&quot;  - exportBins(&lt;%d samples&gt;, format=\&quot;%s\&quot;, type=\&quot;%s\&quot;)&quot;, ncol(fitC), format, type))
+     file &lt;- exportBins(fitC, format = format, type = type, file = file)
+     message(sprintf(&quot;    File(s) written: [n=%d] %s&quot;,
+             length(file), paste(sQuote(file), collapse = &quot;, &quot;)))
+     stopifnot(all(file_test(&quot;-f&quot;, file)))
+     if (format == &quot;seg&quot;) {
+       segs &lt;- read.table(file, sep = &quot;\t&quot;, header = TRUE)
+       print(segs)
+       stopifnot(nrow(segs) == 0L)
+     } else if (format == &quot;vcf&quot;) {
+       rows &lt;- readLines(file)
+       rows &lt;- grep(&quot;^#&quot;, rows, invert = TRUE, value = TRUE)
+       print(rows)
+       stopifnot(length(rows) == 0L)
+     }
+     file.remove(file)
+     stopifnot(!any(file_test(&quot;-f&quot;, file)))
+   }
+ }
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmptHFYul/QDNAseq-%s1966d8f90de53.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmptHFYul/QDNAseq-%s1966d8405e5d76.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmptHFYul/QDNAseq-%s1966d8198a08b1.calls.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmptHFYul/QDNAseq-%s1966d89bbc845.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmptHFYul/QDNAseq-%s1966d8785d7650.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmptHFYul/QDNAseq-%s1966d8133d5284.calls.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmptHFYul/QDNAseq-LGG1501966d849280f21.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmptHFYul/QDNAseq-LGG1501966d874babff.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmptHFYul/QDNAseq-LGG1501966d8d486bf2.calls.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmptHFYul/QDNAseq-LGG1501966d885510ab.copynumber.seg&#x27;
[1] SAMPLE_NAME     CHROMOSOME      START           STOP           
[5] DATAPOINTS      LOG2_RATIO_MEAN
&lt;0 rows&gt; (or 0-length row.names)
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmptHFYul/QDNAseq-LGG1501966d835575807.segments.seg&#x27;
[1] SAMPLE_NAME     CHROMOSOME      START           STOP           
[5] DATAPOINTS      LOG2_RATIO_MEAN
&lt;0 rows&gt; (or 0-length row.names)
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmptHFYul/QDNAseq-LGG1501966d8282bc379.calls.seg&#x27;
[1] SAMPLE_NAME     CHROMOSOME      START           STOP           
[5] DATAPOINTS      LOG2_RATIO_MEAN
&lt;0 rows&gt; (or 0-length row.names)
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmptHFYul/QDNAseq-LGG1501966d82be4503e.copynumber.vcf&#x27;
character(0)
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmptHFYul/QDNAseq-LGG1501966d847c23d87.segments.vcf&#x27;
character(0)
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmptHFYul/QDNAseq-LGG1501966d81e84de8.calls.vcf&#x27;
character(0)
&gt; 
&gt; proc.time()
   user  system elapsed 
 23.555   0.876  24.414 
</PRE>
</DIV><P><SPAN class="filename">QDNAseq.Rcheck/tests/QDNAseq,gain-copyneutral-gain.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(QDNAseq)
&gt; library(Biobase)
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

&gt; library(utils)
&gt; set.seed(0xBEEF)
&gt; 
&gt; data(LGG150)
&gt; data &lt;- LGG150
&gt; print(data)
QDNAseqReadCounts (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: counts 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads used.reads expected.variance
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; dataF &lt;- applyFilters(data, residual=TRUE, blacklist=TRUE)
38,819	total bins
38,819	of which in selected chromosomes
36,722	of which with reference sequence
33,347	final bins
&gt; print(dataF)
QDNAseqReadCounts (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: counts 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads used.reads expected.variance
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; dataC &lt;- correctBins(dataF)
Calculating correction for GC content and mappability
    Calculating fit for sample LGG150 (1 of 1) ...
Done.
&gt; 
&gt; ## Force results to have segments gain, copy neutral, and gain.
&gt; cnAll &lt;- assayDataElement(dataC, &quot;copynumber&quot;)
&gt; cnAll[,1] &lt;- rnorm(nrow(cnAll), mean = 1.0, sd = 0.05)
&gt; chr7 &lt;- (chromosomes(dataC) == &quot;7&quot;)
&gt; cn &lt;- cnAll[chr7, , drop = FALSE]
&gt; n &lt;- nrow(cn)
&gt; idxs &lt;- seq(from=1/3*n - 0.1*n, to=1/3*n + 0.1*n)
&gt; cn[idxs,1] &lt;- rnorm(length(idxs), mean = 2.0, sd = 0.05)
&gt; idxs &lt;- seq(from=2/3*n - 0.1*n, to=2/3*n + 0.1*n)
&gt; cn[idxs,1] &lt;- rnorm(length(idxs), mean = 2.0, sd = 0.05)
&gt; cnAll[chr7, ] &lt;- cn
&gt; assayDataElement(dataC, &quot;copynumber&quot;) &lt;- cnAll
&gt; print(dataC)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: copynumber 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; dataN &lt;- normalizeBins(dataC)
Applying median normalization ...
&gt; print(dataN)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: copynumber 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; fit &lt;- segmentBins(dataN)
Performing segmentation:
    Segmenting: LGG150 (1 of 1) ...
&gt; print(fit)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: copynumber, segmented 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; fitC &lt;- callBins(fit)
EM algorithm started ... 

[1] &quot;Total number of segments present in the data: 8&quot;
[1] &quot;Number of segments used for fitting the model: 8&quot;
39953358215372213.462.7632841514786712338112.963284151224731633893.5
Calling iteration1:

optim results
time: 0
minimum: 16691.936015802
116691.936015802-0.76889026698241-0.310183146024425-0.01185150972096230.2885943760685340.4933553445096880.9879385470392230.0299884700901994-0.0005485736219481460.0111821966975442-4.08995447325668e-050.01000052582656670.0100005260086061
EM algorithm done ...

Computing posterior probabilities for all segments ...

Total time:0minutes

Adjusting segmented data for cellularity ... 

Cellularity sample1: 1

Adjusting normalized data for cellularity ... 

Cellularity sample1: 1

1
39995088440355213.664.4632841514786712338112.963284151224731633893.5
39995278607101213.665.7632841514786712338112.963284151224731633893.5
39995248607095213.665.7632841514786712338112.963284151224731633893.5
39995528875625213.667.863284151782405433813663284151224731633893.5
39999798876580213.767.863284151782405433813663284151224731633893.5
39999888876584213.767.863284151782405433813663284151224731633893.5
39999978876588213.767.863284151782405433813663284151224731633893.5
40000068876592213.767.863284151782405433813663284151224731633893.5
40000158876596213.767.863284151782405433813663284151224731633893.5
40000218876599213.767.863284151782405433813663284151224731633893.5
40000408943329213.768.363284151782405433813663284151224731633893.5
40004578943652213.768.363284151782405433813663284151224731633893.5
FINISHED!

Total time:0minutes

There were 32 warnings (use warnings() to see them)
&gt; print(fitC)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: calls, copynumber, probamp, probdloss, probgain, probloss, probnorm, segmented 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; 
&gt; ## Assert that everything is called copy-neutral
&gt; calls &lt;- assayDataElement(fitC, &quot;calls&quot;)
&gt; stopifnot(all(is.na(calls) | calls %in% c(0, 2)))
&gt; 
&gt; formats &lt;- c(&quot;tsv&quot;, &quot;igv&quot;, &quot;bed&quot;, &quot;seg&quot;, &quot;vcf&quot;)
&gt; types &lt;- c(&quot;copynumber&quot;, &quot;segments&quot;, &quot;calls&quot;)
&gt; for (format in formats) {
+   for (type in types) {
+     fileext &lt;- sprintf(&quot;.%s.%s&quot;, type, format)
+     file &lt;- tempfile(pattern =  &quot;QDNAseq-%s&quot;, fileext = fileext)
+     message(sprintf(&quot;  - exportBins(&lt;%d samples&gt;, format=\&quot;%s\&quot;, type=\&quot;%s\&quot;)&quot;, ncol(fitC), format, type))
+     file &lt;- exportBins(fitC, format = format, type = type, file = file)
+     message(sprintf(&quot;    File(s) written: [n=%d] %s&quot;,
+             length(file), paste(sQuote(file), collapse = &quot;, &quot;)))
+     stopifnot(all(file_test(&quot;-f&quot;, file)))
+ 
+     if (format == &quot;seg&quot;) {
+       segs &lt;- read.table(file, sep = &quot;\t&quot;, header = TRUE)
+       print(segs)
+       stopifnot(all(segs$CHROMOSOME == &quot;7&quot;), nrow(segs) == 2L)
+     } else if (format == &quot;vcf&quot;) {
+       segs &lt;- read.table(file, sep = &quot;\t&quot;, header = FALSE)
+       print(segs)
+       stopifnot(all(segs$V1 == &quot;7&quot;), nrow(segs) == 2L)
+     }
+ 
+     file.remove(file)
+     stopifnot(!any(file_test(&quot;-f&quot;, file)))
+   }
+ }
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpgjKC8C/QDNAseq-%s19693f4b488516.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpgjKC8C/QDNAseq-%s19693f55a44684.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpgjKC8C/QDNAseq-%s19693f72d929d9.calls.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpgjKC8C/QDNAseq-%s19693f60f0b758.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpgjKC8C/QDNAseq-%s19693f5e682ad5.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpgjKC8C/QDNAseq-%s19693f1c8407e5.calls.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpgjKC8C/QDNAseq-LGG15019693f4384a937.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpgjKC8C/QDNAseq-LGG15019693fb35de0.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpgjKC8C/QDNAseq-LGG15019693f7d95fe89.calls.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpgjKC8C/QDNAseq-LGG15019693f65460434.copynumber.seg&#x27;
                                                  SAMPLE_NAME CHROMOSOME
1 /tmp/RtmpgjKC8C/QDNAseq-LGG15019693f65460434.copynumber.seg          7
2 /tmp/RtmpgjKC8C/QDNAseq-LGG15019693f65460434.copynumber.seg          7
     START      STOP DATAPOINTS LOG2_RATIO_MEAN
1 37110001  68955000       1487            0.99
2 90165001 122010000       2012            0.99
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpgjKC8C/QDNAseq-LGG15019693f612b985.segments.seg&#x27;
                                               SAMPLE_NAME CHROMOSOME    START
1 /tmp/RtmpgjKC8C/QDNAseq-LGG15019693f612b985.segments.seg          7 37110001
2 /tmp/RtmpgjKC8C/QDNAseq-LGG15019693f612b985.segments.seg          7 90165001
       STOP DATAPOINTS LOG2_RATIO_MEAN
1  68955000       1487            0.99
2 122010000       2012            0.99
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpgjKC8C/QDNAseq-LGG15019693f87d9e4f.calls.seg&#x27;
                                            SAMPLE_NAME CHROMOSOME    START
1 /tmp/RtmpgjKC8C/QDNAseq-LGG15019693f87d9e4f.calls.seg          7 37110001
2 /tmp/RtmpgjKC8C/QDNAseq-LGG15019693f87d9e4f.calls.seg          7 90165001
       STOP DATAPOINTS LOG2_RATIO_MEAN
1  68955000       1487            0.99
2 122010000       2012            0.99
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;copynumber&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpgjKC8C/QDNAseq-LGG15019693f4c580a21.copynumber.vcf&#x27;
  V1       V2 V3    V4    V5   V6   V7
1  7 37110001  . &lt;DIP&gt; &lt;DUP&gt; 1000 PASS
2  7 90165001  . &lt;DIP&gt; &lt;DUP&gt; 1000 PASS
                                                                      V8 V9 V10
1  SVTYPE=DUP;END=68955000;SVLEN=31845000;BINS=1487;SCORE=2;LOG2CNT=0.99 GT 0/1
2 SVTYPE=DUP;END=122010000;SVLEN=31845000;BINS=2012;SCORE=2;LOG2CNT=0.99 GT 0/1
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;segments&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpgjKC8C/QDNAseq-LGG15019693f1c254746.segments.vcf&#x27;
  V1       V2 V3    V4    V5   V6   V7
1  7 37110001  . &lt;DIP&gt; &lt;DUP&gt; 1000 PASS
2  7 90165001  . &lt;DIP&gt; &lt;DUP&gt; 1000 PASS
                                                                      V8 V9 V10
1  SVTYPE=DUP;END=68955000;SVLEN=31845000;BINS=1487;SCORE=2;LOG2CNT=0.99 GT 0/1
2 SVTYPE=DUP;END=122010000;SVLEN=31845000;BINS=2012;SCORE=2;LOG2CNT=0.99 GT 0/1
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;calls&quot;)
    File(s) written: [n=1] &#x27;/tmp/RtmpgjKC8C/QDNAseq-LGG15019693f4c6bb154.calls.vcf&#x27;
  V1       V2 V3    V4    V5   V6   V7
1  7 37110001  . &lt;DIP&gt; &lt;DUP&gt; 1000 PASS
2  7 90165001  . &lt;DIP&gt; &lt;DUP&gt; 1000 PASS
                                                                      V8 V9 V10
1  SVTYPE=DUP;END=68955000;SVLEN=31845000;BINS=1487;SCORE=2;LOG2CNT=0.99 GT 0/1
2 SVTYPE=DUP;END=122010000;SVLEN=31845000;BINS=2012;SCORE=2;LOG2CNT=0.99 GT 0/1
&gt; 
&gt; proc.time()
   user  system elapsed 
 22.889   0.436  23.309 
</PRE>
</DIV><P><SPAN class="filename">QDNAseq.Rcheck/tests/QDNAseq,parallel-reproducibility.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; ######################################################################
&gt; # This scripts asserts that for each processing step of QDNAseq
&gt; # the output/results are reproducible (numerically equal).
&gt; ######################################################################
&gt; library(&quot;QDNAseq&quot;)
&gt; options(&quot;QDNAseq::verbose&quot;=FALSE)
&gt; 
&gt; # Load data
&gt; data(LGG150)
&gt; data &lt;- LGG150
&gt; 
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # TRUTH
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # Filter out &quot;bad&quot; bins
&gt; dataF &lt;- applyFilters(data, residual=TRUE, blacklist=TRUE)
&gt; 
&gt; # Correct read counts as a function of GC content and mappability
&gt; dataC &lt;- correctBins(dataF)
&gt; 
&gt; # Normalize binned read counts to have diploid normal copy number
&gt; dataN &lt;- normalizeBins(dataC)
&gt; 
&gt; # Segment copy numbers
&gt; set.seed(42)  ## segmentBins() relies on RNG via DNAcopy::segment()
&gt; fit &lt;- segmentBins(dataN)
&gt; 
&gt; # Call copy-number segments
&gt; fitC &lt;- callBins(fit)
There were 50 or more warnings (use warnings() to see the first 50)
&gt; 
&gt; 
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # REPRODUCIBILITY
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; strategies &lt;- c(&quot;sequential&quot;, &quot;multisession&quot;)
&gt; if (parallelly::supportsMulticore()) strategies &lt;- c(strategies, &quot;multicore&quot;)
&gt; 
&gt; oplan &lt;- future::plan(&quot;list&quot;)
&gt; for (strategy in strategies) {
+   message(sprintf(&quot;Reproducibility with plan(\&quot;%s\&quot;) ...&quot;, strategy))
+   
+   future::plan(strategy)
+   
+   dataFr &lt;- applyFilters(data, residual=TRUE, blacklist=TRUE)
+   stopifnot(all.equal(dataFr, dataF))
+   
+   dataCr &lt;- correctBins(dataF)
+   stopifnot(all.equal(dataCr, dataC))
+   
+   dataNr &lt;- normalizeBins(dataC)
+   stopifnot(all.equal(dataNr, dataN))
+   
+   set.seed(42)  ## segmentBins() relies on RNG via DNAcopy::segment()
+   fitr &lt;- segmentBins(dataNr)
+   stopifnot(all.equal(fitr, fit))
+   
+   fitCr &lt;- callBins(fitr)
+   stopifnot(all.equal(fitCr, fitC))
+   
+   message(sprintf(&quot;Reproducibility with plan(\&quot;%s\&quot;) ... done&quot;, strategy))
+ }
Reproducibility with plan(&quot;sequential&quot;) ...
Reproducibility with plan(&quot;sequential&quot;) ... done
Reproducibility with plan(&quot;multisession&quot;) ...
    Calculating fit for sample LGG150 (1 of 1) ...
    Segmenting: LGG150 (1 of 1) ...
Reproducibility with plan(&quot;multisession&quot;) ... done
Reproducibility with plan(&quot;multicore&quot;) ...
Reproducibility with plan(&quot;multicore&quot;) ... done
There were 50 or more warnings (use warnings() to see the first 50)
&gt; 
&gt; future::plan(oplan)
&gt; 
&gt; proc.time()
   user  system elapsed 
 81.244   1.777  94.354 
</PRE>
</DIV><P><SPAN class="filename">QDNAseq.Rcheck/tests/QDNAseq.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(QDNAseq)
&gt; library(Biobase) ## combine(), sampleNames()
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

&gt; library(utils)
&gt; 
&gt; # Load data
&gt; data(LGG150)
&gt; data &lt;- LGG150
&gt; print(data)
QDNAseqReadCounts (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: counts 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads used.reads expected.variance
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; stopifnot(inherits(data, &quot;QDNAseqReadCounts&quot;))
&gt; 
&gt; # Plot isobars of read counts
&gt; isobarPlot(data)
Plotting sample LGG150 median read counts
&gt; 
&gt; # Plot copy number profile
&gt; plot(data, ylim=c(-100, 200))
Plotting sample LGG150 (1 of 1) ...
&gt; highlightFilters(data, residual=TRUE, blacklist=TRUE)
Highlighted 3,375 bins.
&gt; 
&gt; # Filter out &quot;bad&quot; bins
&gt; dataF &lt;- applyFilters(data, residual=TRUE, blacklist=TRUE)
38,819	total bins
38,819	of which in selected chromosomes
36,722	of which with reference sequence
33,347	final bins
&gt; print(dataF)
QDNAseqReadCounts (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: counts 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads used.reads expected.variance
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; plot(dataF, ylim=c(-100, 200))
Plotting sample LGG150 (1 of 1) ...
&gt; stopifnot(inherits(dataF, &quot;QDNAseqReadCounts&quot;))
&gt; 
&gt; # Correct read counts as a function of GC content and mappability
&gt; dataC &lt;- correctBins(dataF)
Calculating correction for GC content and mappability
    Calculating fit for sample LGG150 (1 of 1) ...
Done.
&gt; print(dataC)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: copynumber 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; plot(dataC, ylim=c(-100, 200))
Plotting sample LGG150 (1 of 1) ...
&gt; stopifnot(inherits(dataC, &quot;QDNAseqCopyNumbers&quot;))
&gt; 
&gt; # Normalize binned read counts to have diploid normal copy number
&gt; dataN &lt;- normalizeBins(dataC)
Applying median normalization ...
&gt; print(dataN)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: copynumber 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; plot(dataN)
Plotting sample LGG150 (1 of 1) ...
&gt; stopifnot(inherits(dataN, &quot;QDNAseqCopyNumbers&quot;))
&gt; 
&gt; # Plot noise
&gt; noisePlot(dataF)
Calculating correction for GC content and mappability
    Calculating fit for sample LGG150 (1 of 1) ...
Done.
&gt; 
&gt; # Segment copy numbers
&gt; fit &lt;- segmentBins(dataN)
Performing segmentation:
    Segmenting: LGG150 (1 of 1) ...
&gt; print(fit)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: copynumber, segmented 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; plot(fit)
Plotting sample LGG150 (1 of 1) ...
&gt; stopifnot(inherits(fit, &quot;QDNAseqCopyNumbers&quot;))
&gt; 
&gt; # Call copy-number segments
&gt; fitC &lt;- callBins(fit)
EM algorithm started ... 

[1] &quot;Total number of segments present in the data: 10&quot;
[1] &quot;Number of segments used for fitting the model: 9&quot;
40092668178304214.262.4632841514826474338113.2632841514825642338113.2
Calling iteration1:

optim results
time: 1
minimum: 16445.6418287481
116443.1634295677-0.939948404909663-0.535115871634710.04999922851584940.1357341814220170.2320391157632851.492692224927890.46336036623653-0.0007447487965531030.0111894143724591-0.000962583906213420.3145629342085070.798554843389167
40107718181755214.262.5632841514826474338113.2632841514825642338113.2
Calling iteration2:

optim results
time: 0
minimum: 16445.5781374328
116443.1331012469-0.932876677515082-0.5359433038809890.0499979048415910.1114729434735370.1905642555481931.861013902305770.4314700360573740.00014406876840850.01118427659401560.0007568920162792620.2061088124299370.874022351578009
40107718181755214.262.5632841514826474338113.2632841514825642338113.2
Calling iteration3:

optim results
time: 0
minimum: 16445.5975371627
116443.1273697069-0.9240128530996-0.5365177469718860.04999923288882760.108573194240180.1856071014916831.753751992204390.4694950401327692.93313204149379e-050.0111878614439676-0.0001759851682342870.07360543573369820.943675664015282
EM algorithm done ...

Computing posterior probabilities for all segments ...

Total time:0minutes

Adjusting segmented data for cellularity ... 

Cellularity sample1: 1

Adjusting normalized data for cellularity ... 

Cellularity sample1: 1

1
40134398402989214.464.2632841514826474338113.2632841514825642338113.2
40134588569735214.465.4632841514826474338113.2632841514825642338113.2
40134558569729214.465.4632841514826474338113.2632841514825642338113.2
40134838836744214.467.5632841517871768338136.4632841514825642338113.2
40139108837699214.467.5632841517871768338136.4632841514825642338113.2
40139198837703214.467.5632841517871768338136.4632841514825642338113.2
40139288837707214.467.5632841517871768338136.4632841514825642338113.2
40139378837711214.467.5632841517871768338136.4632841514825642338113.2
40139468837715214.467.5632841517871768338136.4632841514825642338113.2
40139528837718214.467.5632841517871768338136.4632841514825642338113.2
40139718904448214.468632841517871768338136.4632841514825642338113.2
40143888904771214.468632841517871768338136.4632841514825642338113.2
FINISHED!

Total time:0minutes

There were 50 or more warnings (use warnings() to see the first 50)
&gt; print(fitC)
QDNAseqCopyNumbers (storageMode: lockedEnvironment)
assayData: 38819 features, 1 samples 
  element names: calls, copynumber, probamp, probdloss, probgain, probloss, probnorm, segmented 
protocolData: none
phenoData
  sampleNames: LGG150
  varLabels: name reads ... loess.family (6 total)
  varMetadata: labelDescription
featureData
  featureNames: 7:1-15000 7:15001-30000 ... 10:135525001-135534747
    (38819 total)
  fvarLabels: chromosome start ... use (9 total)
  fvarMetadata: labelDescription
experimentData: use &#x27;experimentData(object)&#x27;
Annotation:  
&gt; plot(fitC)
Plotting sample LGG150 (1 of 1) ...
&gt; 
&gt; 
&gt; # ---------------------------------------------------------------
&gt; # Exporting
&gt; # ---------------------------------------------------------------
&gt; message(&quot;* exportBins() ...&quot;)
* exportBins() ...
&gt; 
&gt; sets &lt;- list(data = data, dataC = dataC, fit = fit, fitC = fitC)
&gt; for (name in names(sets)) {
+   set &lt;- sets[[name]]
+   formats &lt;- c(&quot;tsv&quot;, &quot;igv&quot;, &quot;bed&quot;)
+   if (name == &quot;fitC&quot;) formats &lt;- c(formats, &quot;vcf&quot;, &quot;seg&quot;)
+   for (format in formats) {
+     types &lt;- c(&quot;copynumber&quot;)
+     if (name %in% c(&quot;fit&quot;, &quot;fitC&quot;)) types &lt;- c(types, &quot;segments&quot;)
+     if (name == &quot;fitC&quot;) types &lt;- c(types, &quot;calls&quot;)
+     for (type in types) {
+       fileext &lt;- sprintf(&quot;.%s.%s&quot;, type, format)
+       templates &lt;- c(&quot;QDNAseq-%s.&quot;, &quot;QDNAseq-%i.&quot;, &quot;QDNAseq-%03i.&quot;)
+       if (ncol(set) == 1L || !(format %in% c(&quot;bed&quot;, &quot;seg&quot;, &quot;vcf&quot;))) {
+         templates &lt;- c(&quot;QDNAseq.&quot;, templates)
+       }
+       for (template in templates) {
+         file &lt;- tempfile(pattern = template, fileext = fileext)
+         message(sprintf(&quot;  - exportBins(&lt;%d samples&gt;, format=\&quot;%s\&quot;, type=\&quot;%s\&quot;, file=\&quot;%s\&quot;)&quot;, ncol(set), format, type, template))
+         file &lt;- exportBins(set, file = file, format = format, type = type)
+         message(sprintf(&quot;    File(s) written: [n=%d] %s&quot;,
+                 length(file), paste(sQuote(file), collapse = &quot;, &quot;)))
+         stopifnot(all(file_test(&quot;-f&quot;, file)))
+         file.remove(file)
+         stopifnot(!any(file_test(&quot;-f&quot;, file)))
+       }
+     }
+   }
+ }
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f035b6f477.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f013fea2b4.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f01de250ff.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f065114c11.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f069843b4f.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f037132840.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f02108ce1d.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f03db7c130.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f05d6d36bb.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150.1976f058af014c.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f05233a90b.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f0cd9bf97.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f0146123b7.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f022d95bcf.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f0625630a1.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f07a6c6522.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f06f87218e.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f021625c36.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f05047ea77.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f01f92622f.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f0aaa929b.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150.1976f027161eee.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f0121a9f2b.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f013848052.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f06b780242.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f01b607e3d.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f06c00c904.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f0630a9c5b.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f06c73b3f5.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f0799a5c44.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f05458504b.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f0222aa86d.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f0d98fef8.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f0723aa14b.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f073bf47e.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f0771d3a48.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f0294dc98b.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f02844c29b.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f034d4fb78.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f06bb0047.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f0f3c3e7.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150.1976f0708a483.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f01394bfde.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f01554e79e.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f029e20052.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150.1976f075eaf07f.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f0fc14cc1.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f0196921e0.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f0174d4cb5.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f060093738.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f038fb8410.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f021f7df50.copynumber.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f071f5626.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f04b16233b.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f0357c5fa2.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f072975868.segments.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f06676a178.calls.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f0217d28a7.calls.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f055a1f4c4.calls.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f052ea556e.calls.tsv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f01b1784eb.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f029fa450f.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f07514fddb.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f028b083e4.copynumber.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f01c34e65a.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f07c50f259.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f01fcdbe2c.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f04582afe6.segments.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f02495b4f5.calls.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f054a2b9a4.calls.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f04c3db02d.calls.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f0258978dc.calls.igv&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f05bab5e27.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150.1976f05fd2700b.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f03ade607b.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f058d5e79.copynumber.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f055bd608a.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150.1976f04a9fad3c.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f01ef6805a.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f06d0aad3f.segments.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f02aa8e474.calls.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150.1976f057f2046a.calls.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f0f028c8f.calls.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;bed&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f031c83a9a.calls.bed&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f0230827a5.copynumber.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150.1976f0447eec32.copynumber.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f0245f9302.copynumber.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f097ec91d.copynumber.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f065fc14d9.segments.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150.1976f07a0187c6.segments.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f05c691e8b.segments.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f011399c4.segments.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f023fbccd6.calls.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150.1976f0517e1c66.calls.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f029c41da8.calls.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;vcf&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f04030b330.calls.vcf&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f04dcf0ec0.copynumber.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150.1976f04991dbd4.copynumber.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f05b36316.copynumber.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f07264c3b5.copynumber.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f01e349578.segments.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150.1976f051f11343.segments.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f017ee3c91.segments.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f079dff39f.segments.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f031c3834e.calls.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150.1976f052cc9d0c.calls.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f07f6d5219.calls.seg&#x27;
  - exportBins(&lt;1 samples&gt;, format=&quot;seg&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f0780e3d8.calls.seg&#x27;
&gt; 
&gt; sets &lt;- list(data = data, dataC = dataC, fit = fit, fitC = fitC)
&gt; sets &lt;- lapply(sets, FUN = function(set) {
+   stopifnot(ncol(set) == 1L)
+   name &lt;- sampleNames(set)
+   setA &lt;- set
+   sampleNames(setA) &lt;- sprintf(&quot;%sa&quot;, name)
+   setB &lt;- set
+   sampleNames(setB) &lt;- sprintf(&quot;%sb&quot;, name)
+   combine(setA, setB)
+ })
&gt; 
&gt; for (name in names(sets)) {
+   set &lt;- sets[[name]]
+   stopifnot(ncol(set) == 2L)
+   formats &lt;- c(&quot;tsv&quot;, &quot;igv&quot;, &quot;bed&quot;)
+   if (name == &quot;fitC&quot;) formats &lt;- c(formats, &quot;vcf&quot;, &quot;seg&quot;)
+   for (format in formats) {
+     types &lt;- c(&quot;copynumber&quot;)
+     if (name %in% c(&quot;fit&quot;, &quot;fitC&quot;)) types &lt;- c(types, &quot;segments&quot;)
+     if (name == &quot;fitC&quot;) types &lt;- c(types, &quot;calls&quot;)
+     for (type in types) {
+       fileext &lt;- sprintf(&quot;.%s.%s&quot;, type, format)
+       templates &lt;- c(&quot;QDNAseq-%s.&quot;, &quot;QDNAseq-%i.&quot;, &quot;QDNAseq-%03i.&quot;)
+       if (ncol(set) == 1L || !(format %in% c(&quot;bed&quot;, &quot;seg&quot;, &quot;vcf&quot;))) {
+         templates &lt;- c(&quot;QDNAseq.&quot;, templates)
+       }
+       for (template in templates) {
+         file &lt;- tempfile(pattern = template, fileext = fileext)
+         message(sprintf(&quot;  - exportBins(&lt;%d samples&gt;, format=\&quot;%s\&quot;, type=\&quot;%s\&quot;, file=\&quot;%s\&quot;)&quot;, ncol(set), format, type, template))
+         file &lt;- exportBins(set, file = file, format = format, type = type)
+         message(sprintf(&quot;    File(s) written: [n=%d] %s&quot;,
+                 length(file), paste(sQuote(file), collapse = &quot;, &quot;)))
+         stopifnot(all(file_test(&quot;-f&quot;, file)))
+         file.remove(file)
+         stopifnot(!any(file_test(&quot;-f&quot;, file)))
+       }
+     }
+   }
+ }
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f01d6c4a48.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f01e63d273.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f0748b9117.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f048152ebc.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f07655d6dd.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f038e1da7.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f079dd6956.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f0195dfe82.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150a.1976f0480d09d9.copynumber.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150b.1976f0480d09d9.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f01e3cfc59.copynumber.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-2.1976f01e3cfc59.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f022dcc79f.copynumber.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-002.1976f022dcc79f.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f02e091eb2.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f0183e841f.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f07f45e62b.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f02f1cb876.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f03c3a50f5.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f050c40291.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f058e0d61f.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f07c6b0426.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150a.1976f01e931151.copynumber.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150b.1976f01e931151.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f02272b1f3.copynumber.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-2.1976f02272b1f3.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f021e673c.copynumber.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-002.1976f021e673c.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f010f7d506.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f040a7476c.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f0540f7a80.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f028e61198.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f03a873b0b.segments.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f05d2fdce.segments.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f07bb2aea4.segments.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f039f48d24.segments.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f0d53e1a7.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f0191ef8ed.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f058585f97.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f01df72be.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f0613427a9.segments.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f04eae3674.segments.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f056d9065.segments.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f05b119100.segments.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150a.1976f0680c34f6.copynumber.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150b.1976f0680c34f6.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f04d7a9a3e.copynumber.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-2.1976f04d7a9a3e.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f0794e8d59.copynumber.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-002.1976f0794e8d59.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150a.1976f0ae8fc96.segments.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150b.1976f0ae8fc96.segments.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f07b83b8f0.segments.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-2.1976f07b83b8f0.segments.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f0118d1178.segments.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-002.1976f0118d1178.segments.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f0a2ee2c1.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f02aa07167.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f04dc7626e.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f05af2e552.copynumber.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f03814786.segments.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f04a326694.segments.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f07985f6a4.segments.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f025f3f979.segments.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f04c50cdd0.calls.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f0a7dcbaa.calls.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f0669b40e5.calls.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;tsv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f020604850.calls.tsv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f03363dd42.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f021227bf1.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f02633461f.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f02f168be7.copynumber.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f05b170915.segments.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f0338727c6.segments.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f0483584d4.segments.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f0336f68ad.segments.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq.1976f035669a84.calls.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%s.1976f02969ac7d.calls.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%i.1976f021d9f21.calls.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;igv&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=1] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-%03i.1976f03ad42aea.calls.igv&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150a.1976f047b3d7d.copynumber.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150b.1976f047b3d7d.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f06a29d418.copynumber.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-2.1976f06a29d418.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f084ec528.copynumber.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-002.1976f084ec528.copynumber.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150a.1976f07dc9cad6.segments.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150b.1976f07dc9cad6.segments.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f07512d0ae.segments.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-2.1976f07512d0ae.segments.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f03d27e19.segments.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-002.1976f03d27e19.segments.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150a.1976f0f56dc4f.calls.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150b.1976f0f56dc4f.calls.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f07f41b36f.calls.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-2.1976f07f41b36f.calls.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;bed&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f02e72ef80.calls.bed&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-002.1976f02e72ef80.calls.bed&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;vcf&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150a.1976f05d1e3ebd.copynumber.vcf&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150b.1976f05d1e3ebd.copynumber.vcf&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;vcf&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f05a3498c1.copynumber.vcf&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-2.1976f05a3498c1.copynumber.vcf&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;vcf&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f031f43706.copynumber.vcf&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-002.1976f031f43706.copynumber.vcf&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;vcf&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150a.1976f02750a551.segments.vcf&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150b.1976f02750a551.segments.vcf&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;vcf&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f053ba8f65.segments.vcf&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-2.1976f053ba8f65.segments.vcf&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;vcf&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f057e8307f.segments.vcf&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-002.1976f057e8307f.segments.vcf&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;vcf&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150a.1976f073a17321.calls.vcf&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150b.1976f073a17321.calls.vcf&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;vcf&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f05e385b10.calls.vcf&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-2.1976f05e385b10.calls.vcf&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;vcf&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f03e837165.calls.vcf&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-002.1976f03e837165.calls.vcf&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;seg&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150a.1976f01401bb72.copynumber.seg&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150b.1976f01401bb72.copynumber.seg&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;seg&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f0119c3852.copynumber.seg&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-2.1976f0119c3852.copynumber.seg&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;seg&quot;, type=&quot;copynumber&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f05fa5ed56.copynumber.seg&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-002.1976f05fa5ed56.copynumber.seg&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;seg&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150a.1976f03a350191.segments.seg&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150b.1976f03a350191.segments.seg&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;seg&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f040b2c439.segments.seg&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-2.1976f040b2c439.segments.seg&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;seg&quot;, type=&quot;segments&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f03abcf66b.segments.seg&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-002.1976f03abcf66b.segments.seg&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;seg&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%s.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150a.1976f06dbc2957.calls.seg&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-LGG150b.1976f06dbc2957.calls.seg&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;seg&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-1.1976f08e8490d.calls.seg&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-2.1976f08e8490d.calls.seg&#x27;
  - exportBins(&lt;2 samples&gt;, format=&quot;seg&quot;, type=&quot;calls&quot;, file=&quot;QDNAseq-%03i.&quot;)
    File(s) written: [n=2] &#x27;/tmp/Rtmpm8rkn2/QDNAseq-001.1976f06e2c5f18.calls.seg&#x27;, &#x27;/tmp/Rtmpm8rkn2/QDNAseq-002.1976f06e2c5f18.calls.seg&#x27;
&gt; 
&gt; message(&quot;* exportBins() ... done&quot;)
* exportBins() ... done
&gt; 
&gt; proc.time()
   user  system elapsed 
 78.985   1.763  80.743 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">QDNAseq.Rcheck/QDNAseq-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>addPhenodata</TD><TD style="text-align: right;">0.122</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.126</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>applyFilters</TD><TD style="text-align: right;">0.745</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.749</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>binReadCounts</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>callBins</TD><TD style="text-align: right;">22.137</TD><TD style="text-align: right;"> 0.236</TD><TD style="text-align: right;">22.374</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>compareToReference</TD><TD style="text-align: right;">2.527</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">2.558</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>correctBins</TD><TD style="text-align: right;">2.339</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">2.358</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>createBins</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>estimateCorrection</TD><TD style="text-align: right;">2.292</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">2.304</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>exportBins</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>frequencyPlot</TD><TD style="text-align: right;">22.063</TD><TD style="text-align: right;"> 0.087</TD><TD style="text-align: right;">22.152</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getBinAnnotations</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>highlightFilters</TD><TD style="text-align: right;">0.712</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.747</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>isobarPlot</TD><TD style="text-align: right;">1.159</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">1.175</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeCgh</TD><TD style="text-align: right;">2.457</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">2.465</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>noisePlot</TD><TD style="text-align: right;">2.290</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">2.298</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>normalizeBins</TD><TD style="text-align: right;">2.349</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">2.361</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>normalizeSegmentedBins</TD><TD style="text-align: right;">9.306</TD><TD style="text-align: right;">0.039</TD><TD style="text-align: right;">9.347</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot</TD><TD style="text-align: right;">2.985</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">3.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>poolRuns</TD><TD style="text-align: right;">0.144</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.143</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>segmentBins</TD><TD style="text-align: right;">9.080</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">9.112</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>smoothOutlierBins</TD><TD style="text-align: right;">2.878</TD><TD style="text-align: right;">0.340</TD><TD style="text-align: right;">3.218</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
