<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for OUTRIDER on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><B>[<A href="../index.html#O">O</A>]</B></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-08 01:15:19 -0000 (Wed, 08 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4164</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for OUTRIDER on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the OUTRIDER package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/OUTRIDER.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1409</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">OUTRIDER</A>&nbsp;1.17.1</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/OUTRIDER">landing page</A>)</SPAN><BR>Christian Mertes<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-06&nbsp;00:12:45&nbsp;-0000&nbsp;(Mon,&nbsp;06&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/OUTRIDER</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">689b51d</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-01-05&nbsp;21:00:16&nbsp;-0000&nbsp;(Thu,&nbsp;05&nbsp;Jan&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: OUTRIDER</TD></TR>
<TR><TD><B>Version</B>: 1.17.1</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:OUTRIDER.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings OUTRIDER_1.17.1.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-07 10:33:14 -0000 (Tue, 07 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-07 10:44:25 -0000 (Tue, 07 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 670.8 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: OUTRIDER.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:OUTRIDER.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings OUTRIDER_1.17.1.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/OUTRIDER.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘OUTRIDER/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘OUTRIDER’ version ‘1.17.1’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘OUTRIDER’ can be installed ... OK
* used C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
padjOnSubset: no visible global function definition for ‘bpmapply’
Undefined global functions or variables:
  bpmapply
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... OK
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘OUTRIDER-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: OUTRIDER
&gt; ### Title: OUTRIDER - Finding expression outlier events
&gt; ### Aliases: OUTRIDER
&gt; 
&gt; ### ** Examples
&gt; 
&gt; ods &lt;- makeExampleOutriderDataSet()
&gt; implementation &lt;- &#x27;autoencoder&#x27;
&gt; ## Don&#x27;t show: 
&gt;     ods &lt;- ods[1:10,1:10]
&gt;     implementation &lt;- &#x27;pca&#x27;
&gt; ## End(Don&#x27;t show)
&gt; ods &lt;- OUTRIDER(ods, implementation=implementation)
Tue Feb  7 10:40:43 2023: SizeFactor estimation ...
Tue Feb  7 10:40:43 2023: Controlling for confounders ...
Using provided q with: 10
Tue Feb  7 10:40:43 2023: Using the pca implementation for controlling.
Tue Feb  7 10:40:43 2023: Used the pca implementation for controlling.
Tue Feb  7 10:40:43 2023: Fitting the data ...
Tue Feb  7 10:40:49 2023: P-value calculation ...
Tue Feb  7 10:40:50 2023: Zscore calculation ...
&gt; 
&gt; pValue(ods)[1:10,1:10]
            sample_1  sample_2  sample_3  sample_4  sample_5  sample_6
feature_1  0.9708112 0.9673898 1.0000000 1.0000000 0.9682199 0.9715971
feature_2  1.0000000 0.9645359 0.9956328 0.9691272 0.9565509 1.0000000
feature_3  0.9956214 1.0000000 0.9817906 0.9714409 0.9874758 1.0000000
feature_4  0.9172160 1.0000000 0.8531784 0.9520697 0.9681790 0.9964730
feature_5  0.9341059 0.9465044 0.8974455 0.9659065 1.0000000 1.0000000
feature_6  0.9589618 0.9899074 0.8672567 1.0000000 1.0000000 0.8466524
feature_7  0.9996621 0.9941735 0.9987931 0.9951721 0.9949311 0.9736960
feature_8  1.0000000 0.9589016 0.8852729 1.0000000 0.9101461 0.8960677
feature_9  0.9944175 0.9819508 0.9432241 1.0000000 0.9670230 1.0000000
feature_10 1.0000000 0.9714403 1.0000000 0.9861801 0.9942400 0.9858725
            sample_7  sample_8  sample_9 sample_10
feature_1  0.9654920 1.0000000 0.9479204 0.9599123
feature_2  1.0000000 0.9600618 0.9691272 0.9950101
feature_3  0.9944387 0.9835739 1.0000000 1.0000000
feature_4  0.9738876 0.9834010 0.9285130 0.9901875
feature_5  0.9985055 1.0000000 0.9314466 1.0000000
feature_6  0.9361394 0.9962630 0.9703483 0.9706845
feature_7  1.0000000 1.0000000 0.9939857 0.9878664
feature_8  0.9879652 0.9976097 0.9930989 0.9906877
feature_9  1.0000000 0.9898412 0.9810968 1.0000000
feature_10 0.9773058 0.9987815 1.0000000 1.0000000
&gt; res &lt;- results(ods, all=TRUE)
&gt; res
         geneID  sampleID    pValue padjust zScore  l2fc rawcounts normcounts
  1:  feature_1  sample_1 0.9708112       1  -0.26 -0.02        58     242.52
  2:  feature_2  sample_1 1.0000000       1   0.76  0.00      1033     315.69
  3:  feature_3  sample_1 0.9956214       1  -0.04  0.00       404     829.51
  4:  feature_4  sample_1 0.9172160       1  -0.67 -0.17         7      61.25
  5:  feature_5  sample_1 0.9341059       1   0.88  0.00       336      53.49
  6:  feature_6  sample_1 0.9589618       1   0.35 -0.04        39     184.39
  7:  feature_7  sample_1 0.9996621       1   0.27  0.00      1095    1416.78
  8:  feature_8  sample_1 1.0000000       1   0.56 -0.01       217      83.20
  9:  feature_9  sample_1 0.9944175       1   0.34  0.00       503     646.91
 10: feature_10  sample_1 1.0000000       1   0.54  0.00      1449    1078.11
 11:  feature_1  sample_2 0.9673898       1  -0.46 -0.03        49     241.77
 12:  feature_2  sample_2 0.9645359       1  -1.10 -0.03        42     308.64
 13:  feature_3  sample_2 1.0000000       1   0.67  0.00      1690     831.07
 14:  feature_4  sample_2 1.0000000       1   0.57 -0.01       235      69.70
 15:  feature_5  sample_2 0.9465044       1  -1.33 -0.12        11      49.18
 16:  feature_6  sample_2 0.9899074       1   0.56  0.00       398     188.64
 17:  feature_7  sample_2 0.9941735       1   0.99  0.00      6690    1417.86
 18:  feature_8  sample_2 0.9589016       1   0.09 -0.05        26      80.48
 19:  feature_9  sample_2 0.9819508       1   0.02 -0.01       152     643.96
 20: feature_10  sample_2 0.9714403       1   0.77  0.00     27702    1078.81
 21:  feature_1  sample_3 1.0000000       1   0.74  0.00       434     246.13
 22:  feature_2  sample_3 0.9956328       1   0.79  0.00      1592     315.79
 23:  feature_3  sample_3 0.9817906       1  -1.96 -0.01       130     825.21
 24:  feature_4  sample_3 0.8531784       1  -2.53 -0.42         2      46.66
 25:  feature_5  sample_3 0.8974455       1   0.91  0.00       497      53.54
 26:  feature_6  sample_3 0.8672567       1  -1.54 -0.32         3     141.84
 27:  feature_7  sample_3 0.9987931       1   0.16  0.00       980    1416.62
 28:  feature_8  sample_3 0.8852729       1  -2.08 -0.26         4      66.86
 29:  feature_9  sample_3 0.9432241       1  -2.80 -0.07        20     617.33
 30: feature_10  sample_3 1.0000000       1   0.57  0.00      1673    1078.21
 31:  feature_1  sample_4 1.0000000       1   0.83  0.00      1052     246.47
 32:  feature_2  sample_4 0.9691272       1  -0.74 -0.03        52     310.03
 33:  feature_3  sample_4 0.9714409       1   0.84  0.00      8658     831.47
 34:  feature_4  sample_4 0.9520697       1   0.59  0.00       541      69.87
 35:  feature_5  sample_4 0.9659065       1  -0.58 -0.08        17      50.67
 36:  feature_6  sample_4 1.0000000       1   0.58  0.00      2050     189.02
 37:  feature_7  sample_4 0.9951721       1  -0.39  0.00       623    1415.80
 38:  feature_8  sample_4 1.0000000       1   0.58  0.00       323      83.32
 39:  feature_9  sample_4 1.0000000       1   0.44  0.00      1830     647.85
 40: feature_10  sample_4 0.9861801       1  -0.71 -0.01       238    1074.34
 41:  feature_1  sample_5 0.9682199       1  -0.41 -0.03        51     241.96
 42:  feature_2  sample_5 0.9565509       1  -1.84 -0.05        30     305.80
 43:  feature_3  sample_5 0.9874758       1   0.80  0.00      4329     831.37
 44:  feature_4  sample_5 0.9681790       1   0.59  0.00       427      69.83
 45:  feature_5  sample_5 1.0000000       1   0.29 -0.03        41      52.37
 46:  feature_6  sample_5 1.0000000       1   0.58  0.00      2075     189.02
 47:  feature_7  sample_5 0.9949311       1  -0.43  0.00       605    1415.73
 48:  feature_8  sample_5 0.9101461       1   0.62  0.00      3174      83.55
 49:  feature_9  sample_5 0.9670230       1   0.47  0.00     22553     648.17
 50: feature_10  sample_5 0.9942400       1   0.17  0.00       580    1076.99
 51:  feature_1  sample_6 0.9715971       1   0.88  0.00      4121     246.64
 52:  feature_2  sample_6 1.0000000       1   0.78  0.00      1247     315.74
 53:  feature_3  sample_6 1.0000000       1   0.32  0.00       657     830.30
 54:  feature_4  sample_6 0.9964730       1   0.57 -0.01       278      69.75
 55:  feature_5  sample_6 1.0000000       1   0.45 -0.03        54      52.67
 56:  feature_6  sample_6 0.8466524       1  -2.16 -0.42         2     126.08
 57:  feature_7  sample_6 0.9736960       1   1.09  0.00     23938    1418.01
 58:  feature_8  sample_6 0.8960677       1  -1.66 -0.22         5      69.65
 59:  feature_9  sample_6 1.0000000       1   0.41  0.00      1030     647.57
 60: feature_10  sample_6 0.9858725       1  -0.75 -0.01       231    1074.20
 61:  feature_1  sample_7 0.9654920       1   0.89  0.00      5031     246.65
 62:  feature_2  sample_7 1.0000000       1   0.76  0.00      1041     315.69
 63:  feature_3  sample_7 0.9944387       1  -0.14  0.00       363     829.28
 64:  feature_4  sample_7 0.9738876       1   0.58  0.00       392      69.82
 65:  feature_5  sample_7 0.9985055       1   0.22 -0.04        37      52.24
 66:  feature_6  sample_7 0.9361394       1   0.04 -0.08        16     177.99
 67:  feature_7  sample_7 1.0000000       1   0.67  0.00      2051    1417.38
 68:  feature_8  sample_7 0.9879652       1   0.44 -0.02        80      82.55
 69:  feature_9  sample_7 1.0000000       1   0.44  0.00      1934     647.87
 70: feature_10  sample_7 0.9773058       1  -2.37 -0.01       112    1069.30
 71:  feature_1  sample_8 1.0000000       1   0.80  0.00       691     246.35
 72:  feature_2  sample_8 0.9600618       1   0.83  0.00      5113     315.93
 73:  feature_3  sample_8 0.9835739       1  -1.62 -0.01       148     825.98
 74:  feature_4  sample_8 0.9834010       1   0.34 -0.04        39      68.25
 75:  feature_5  sample_8 1.0000000       1   0.47 -0.03        56      52.71
 76:  feature_6  sample_8 0.9962630       1   0.57  0.00       939     188.91
 77:  feature_7  sample_8 1.0000000       1   0.56  0.00      1647    1417.21
 78:  feature_8  sample_8 0.9976097       1   0.51 -0.01       122      82.90
 79:  feature_9  sample_8 0.9898412       1   0.25  0.00       310     646.12
 80: feature_10  sample_8 0.9987815       1   0.44  0.00      1036    1077.81
 81:  feature_1  sample_9 0.9479204       1  -2.03 -0.06        22     235.98
 82:  feature_2  sample_9 0.9691272       1  -0.74 -0.03        52     310.03
 83:  feature_3  sample_9 1.0000000       1   0.54  0.00      1062     830.78
 84:  feature_4  sample_9 0.9285130       1  -0.43 -0.14         9      63.00
 85:  feature_5  sample_9 0.9314466       1  -2.05 -0.15         8      47.69
 86:  feature_6  sample_9 0.9703483       1   0.58  0.00     59866     189.11
 87:  feature_7  sample_9 0.9939857       1  -0.62  0.00       540    1415.45
 88:  feature_8  sample_9 0.9930989       1   0.48 -0.01       100      82.75
 89:  feature_9  sample_9 0.9810968       1  -0.01 -0.01       142     643.67
 90: feature_10  sample_9 1.0000000       1   0.67  0.00      2984    1078.49
 91:  feature_1 sample_10 0.9599123       1  -0.99 -0.04        35     239.85
 92:  feature_2 sample_10 0.9950101       1   0.50 -0.01       241     314.69
 93:  feature_3 sample_10 1.0000000       1   0.59  0.00      1238     830.89
 94:  feature_4 sample_10 0.9901875       1   0.39 -0.03        48      68.57
 95:  feature_5 sample_10 1.0000000       1   0.73 -0.01       120      53.21
 96:  feature_6 sample_10 0.9706845       1   0.45 -0.02        74     186.59
 97:  feature_7 sample_10 0.9878664       1  -2.29 -0.01       275    1412.93
 98:  feature_8 sample_10 0.9906877       1   0.46 -0.02        90      82.66
 99:  feature_9 sample_10 1.0000000       1   0.45  0.00      2206     647.91
100: feature_10 sample_10 1.0000000       1   0.66  0.00      2818    1078.47
         geneID  sampleID    pValue padjust zScore  l2fc rawcounts normcounts
     meanCorrected    theta aberrant AberrantBySample AberrantByGene padj_rank
  1:        243.43 120193.2    FALSE                0              0       5.5
  2:        312.80 124437.3    FALSE                0              0       5.5
  3:        829.59 144900.2    FALSE                0              0       5.5
  4:         65.67 102654.8    FALSE                0              0       5.5
  5:         51.78  64681.5    FALSE                0              0       5.5
  6:        176.16 153410.4    FALSE                0              0       5.5
  7:       1416.38 107763.6    FALSE                0              0       5.5
  8:         79.79 176265.5    FALSE                0              0       5.5
  9:        643.74 201805.4    FALSE                0              0       5.5
 10:       1076.47 271486.9    FALSE                0              0       5.5
 11:        243.43 120193.2    FALSE                0              0       5.5
 12:        312.80 124437.3    FALSE                0              0       5.5
 13:        829.59 144900.2    FALSE                0              0       5.5
 14:         65.67 102654.8    FALSE                0              0       5.5
 15:         51.78  64681.5    FALSE                0              0       5.5
 16:        176.16 153410.4    FALSE                0              0       5.5
 17:       1416.38 107763.6    FALSE                0              0       5.5
 18:         79.79 176265.5    FALSE                0              0       5.5
 19:        643.74 201805.4    FALSE                0              0       5.5
 20:       1076.47 271486.9    FALSE                0              0       5.5
 21:        243.43 120193.2    FALSE                0              0       5.5
 22:        312.80 124437.3    FALSE                0              0       5.5
 23:        829.59 144900.2    FALSE                0              0       5.5
 24:         65.67 102654.8    FALSE                0              0       5.5
 25:         51.78  64681.5    FALSE                0              0       5.5
 26:        176.16 153410.4    FALSE                0              0       5.5
 27:       1416.38 107763.6    FALSE                0              0       5.5
 28:         79.79 176265.5    FALSE                0              0       5.5
 29:        643.74 201805.4    FALSE                0              0       5.5
 30:       1076.47 271486.9    FALSE                0              0       5.5
 31:        243.43 120193.2    FALSE                0              0       5.5
 32:        312.80 124437.3    FALSE                0              0       5.5
 33:        829.59 144900.2    FALSE                0              0       5.5
 34:         65.67 102654.8    FALSE                0              0       5.5
 35:         51.78  64681.5    FALSE                0              0       5.5
 36:        176.16 153410.4    FALSE                0              0       5.5
 37:       1416.38 107763.6    FALSE                0              0       5.5
 38:         79.79 176265.5    FALSE                0              0       5.5
 39:        643.74 201805.4    FALSE                0              0       5.5
 40:       1076.47 271486.9    FALSE                0              0       5.5
 41:        243.43 120193.2    FALSE                0              0       5.5
 42:        312.80 124437.3    FALSE                0              0       5.5
 43:        829.59 144900.2    FALSE                0              0       5.5
 44:         65.67 102654.8    FALSE                0              0       5.5
 45:         51.78  64681.5    FALSE                0              0       5.5
 46:        176.16 153410.4    FALSE                0              0       5.5
 47:       1416.38 107763.6    FALSE                0              0       5.5
 48:         79.79 176265.5    FALSE                0              0       5.5
 49:        643.74 201805.4    FALSE                0              0       5.5
 50:       1076.47 271486.9    FALSE                0              0       5.5
 51:        243.43 120193.2    FALSE                0              0       5.5
 52:        312.80 124437.3    FALSE                0              0       5.5
 53:        829.59 144900.2    FALSE                0              0       5.5
 54:         65.67 102654.8    FALSE                0              0       5.5
 55:         51.78  64681.5    FALSE                0              0       5.5
 56:        176.16 153410.4    FALSE                0              0       5.5
 57:       1416.38 107763.6    FALSE                0              0       5.5
 58:         79.79 176265.5    FALSE                0              0       5.5
 59:        643.74 201805.4    FALSE                0              0       5.5
 60:       1076.47 271486.9    FALSE                0              0       5.5
 61:        243.43 120193.2    FALSE                0              0       5.5
 62:        312.80 124437.3    FALSE                0              0       5.5
 63:        829.59 144900.2    FALSE                0              0       5.5
 64:         65.67 102654.8    FALSE                0              0       5.5
 65:         51.78  64681.5    FALSE                0              0       5.5
 66:        176.16 153410.4    FALSE                0              0       5.5
 67:       1416.38 107763.6    FALSE                0              0       5.5
 68:         79.79 176265.5    FALSE                0              0       5.5
 69:        643.74 201805.4    FALSE                0              0       5.5
 70:       1076.47 271486.9    FALSE                0              0       5.5
 71:        243.43 120193.2    FALSE                0              0       5.5
 72:        312.80 124437.3    FALSE                0              0       5.5
 73:        829.59 144900.2    FALSE                0              0       5.5
 74:         65.67 102654.8    FALSE                0              0       5.5
 75:         51.78  64681.5    FALSE                0              0       5.5
 76:        176.16 153410.4    FALSE                0              0       5.5
 77:       1416.38 107763.6    FALSE                0              0       5.5
 78:         79.79 176265.5    FALSE                0              0       5.5
 79:        643.74 201805.4    FALSE                0              0       5.5
 80:       1076.47 271486.9    FALSE                0              0       5.5
 81:        243.43 120193.2    FALSE                0              0       5.5
 82:        312.80 124437.3    FALSE                0              0       5.5
 83:        829.59 144900.2    FALSE                0              0       5.5
 84:         65.67 102654.8    FALSE                0              0       5.5
 85:         51.78  64681.5    FALSE                0              0       5.5
 86:        176.16 153410.4    FALSE                0              0       5.5
 87:       1416.38 107763.6    FALSE                0              0       5.5
 88:         79.79 176265.5    FALSE                0              0       5.5
 89:        643.74 201805.4    FALSE                0              0       5.5
 90:       1076.47 271486.9    FALSE                0              0       5.5
 91:        243.43 120193.2    FALSE                0              0       5.5
 92:        312.80 124437.3    FALSE                0              0       5.5
 93:        829.59 144900.2    FALSE                0              0       5.5
 94:         65.67 102654.8    FALSE                0              0       5.5
 95:         51.78  64681.5    FALSE                0              0       5.5
 96:        176.16 153410.4    FALSE                0              0       5.5
 97:       1416.38 107763.6    FALSE                0              0       5.5
 98:         79.79 176265.5    FALSE                0              0       5.5
 99:        643.74 201805.4    FALSE                0              0       5.5
100:       1076.47 271486.9    FALSE                0              0       5.5
     meanCorrected    theta aberrant AberrantBySample AberrantByGene padj_rank
                FDR_set
  1: transcriptome-wide
  2: transcriptome-wide
  3: transcriptome-wide
  4: transcriptome-wide
  5: transcriptome-wide
  6: transcriptome-wide
  7: transcriptome-wide
  8: transcriptome-wide
  9: transcriptome-wide
 10: transcriptome-wide
 11: transcriptome-wide
 12: transcriptome-wide
 13: transcriptome-wide
 14: transcriptome-wide
 15: transcriptome-wide
 16: transcriptome-wide
 17: transcriptome-wide
 18: transcriptome-wide
 19: transcriptome-wide
 20: transcriptome-wide
 21: transcriptome-wide
 22: transcriptome-wide
 23: transcriptome-wide
 24: transcriptome-wide
 25: transcriptome-wide
 26: transcriptome-wide
 27: transcriptome-wide
 28: transcriptome-wide
 29: transcriptome-wide
 30: transcriptome-wide
 31: transcriptome-wide
 32: transcriptome-wide
 33: transcriptome-wide
 34: transcriptome-wide
 35: transcriptome-wide
 36: transcriptome-wide
 37: transcriptome-wide
 38: transcriptome-wide
 39: transcriptome-wide
 40: transcriptome-wide
 41: transcriptome-wide
 42: transcriptome-wide
 43: transcriptome-wide
 44: transcriptome-wide
 45: transcriptome-wide
 46: transcriptome-wide
 47: transcriptome-wide
 48: transcriptome-wide
 49: transcriptome-wide
 50: transcriptome-wide
 51: transcriptome-wide
 52: transcriptome-wide
 53: transcriptome-wide
 54: transcriptome-wide
 55: transcriptome-wide
 56: transcriptome-wide
 57: transcriptome-wide
 58: transcriptome-wide
 59: transcriptome-wide
 60: transcriptome-wide
 61: transcriptome-wide
 62: transcriptome-wide
 63: transcriptome-wide
 64: transcriptome-wide
 65: transcriptome-wide
 66: transcriptome-wide
 67: transcriptome-wide
 68: transcriptome-wide
 69: transcriptome-wide
 70: transcriptome-wide
 71: transcriptome-wide
 72: transcriptome-wide
 73: transcriptome-wide
 74: transcriptome-wide
 75: transcriptome-wide
 76: transcriptome-wide
 77: transcriptome-wide
 78: transcriptome-wide
 79: transcriptome-wide
 80: transcriptome-wide
 81: transcriptome-wide
 82: transcriptome-wide
 83: transcriptome-wide
 84: transcriptome-wide
 85: transcriptome-wide
 86: transcriptome-wide
 87: transcriptome-wide
 88: transcriptome-wide
 89: transcriptome-wide
 90: transcriptome-wide
 91: transcriptome-wide
 92: transcriptome-wide
 93: transcriptome-wide
 94: transcriptome-wide
 95: transcriptome-wide
 96: transcriptome-wide
 97: transcriptome-wide
 98: transcriptome-wide
 99: transcriptome-wide
100: transcriptome-wide
                FDR_set
&gt; 
&gt; plotAberrantPerSample(ods)
Warning: Transformation introduced infinite values in continuous y-axis
Warning: Removed 10 rows containing missing values (`geom_bar()`).
&gt; plotVolcano(ods, 1)
&gt; 
&gt; # example of restricting FDR correction to subsets of genes of interest 
&gt; genesOfInterest &lt;- list(&quot;sample_1&quot;=sample(rownames(ods), 3), 
+                          &quot;sample_2&quot;=sample(rownames(ods), 8), 
+                          &quot;sample_6&quot;=sample(rownames(ods), 5))
&gt; genesOfInterest
$sample_1
[1] &quot;feature_10&quot; &quot;feature_1&quot;  &quot;feature_6&quot; 

$sample_2
[1] &quot;feature_8&quot;  &quot;feature_10&quot; &quot;feature_4&quot;  &quot;feature_3&quot;  &quot;feature_6&quot; 
[6] &quot;feature_1&quot;  &quot;feature_2&quot;  &quot;feature_9&quot; 

$sample_6
[1] &quot;feature_5&quot;  &quot;feature_6&quot;  &quot;feature_8&quot;  &quot;feature_10&quot; &quot;feature_7&quot; 

&gt; ods &lt;- OUTRIDER(ods, subsets=list(&quot;exampleSubset&quot;=genesOfInterest))
Tue Feb  7 10:40:53 2023: SizeFactor estimation ...
Tue Feb  7 10:40:53 2023: Controlling for confounders ...
Using provided q with: 10
Tue Feb  7 10:40:54 2023: Using the autoencoder implementation for controlling.
[1] &quot;Tue Feb  7 10:40:56 2023: Initial PCA loss: 6.95599736916101&quot;
Killed
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  [1] &quot;Tue Feb  7 10:42:19 2023: 2 Final nb-AE loss: 4.14004814419489&quot;
  [1] &quot;Evaluation loss: 0.517781423170479 for q=3&quot;
  [1] &quot;Tue Feb  7 10:42:21 2023: Initial PCA loss: 4.4572558295302&quot;
  [1] &quot;Tue Feb  7 10:42:24 2023: Iteration: 1 loss: 4.07122946861174&quot;
  [1] &quot;Tue Feb  7 10:42:26 2023: Iteration: 2 loss: 4.05826513752563&quot;
  Time difference of 3.442074 secs
  [1] &quot;Tue Feb  7 10:42:26 2023: 2 Final nb-AE loss: 4.05826513752563&quot;
  [1] &quot;Evaluation loss: 0.454717572219274 for q=4&quot;
  [1] &quot;Tue Feb  7 10:42:27 2023: Initial PCA loss: 4.4282762368912&quot;
  [1] &quot;Tue Feb  7 10:42:29 2023: Iteration: 1 loss: 3.9973349377491&quot;
  [1] &quot;Tue Feb  7 10:42:30 2023: Iteration: 2 loss: 3.96432433728888&quot;
  Time difference of 2.703483 secs
  [1] &quot;Tue Feb  7 10:42:30 2023: 2 Final nb-AE loss: 3.96432433728888&quot;
  [1] &quot;Evaluation loss: 0.426020105666412 for q=5&quot;
  Killed
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘OUTRIDER.Rnw’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... ERROR
Error(s) in re-building vignettes:
  ...
--- re-building ‘OUTRIDER.Rnw’ using knitr
The magick package is required to crop &quot;figure/deVsOutlier-1.png&quot; but not available.
Killed

* checking PDF version of manual ... OK
* DONE

Status: 3 ERRORs, 2 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/OUTRIDER.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">OUTRIDER.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL OUTRIDER
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘OUTRIDER’ ...
** using staged installation
** libs
using C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
using C++11
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppArmadillo/include&#x27; -I/usr/local/include   -fopenmp -DARMA_DONT_USE_OPENMP -fPIC  -g -O2  -Wall -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppArmadillo/include&#x27; -I/usr/local/include   -fopenmp -DARMA_DONT_USE_OPENMP -fPIC  -g -O2  -Wall -c loss_n_gradient_functions.cpp -o loss_n_gradient_functions.o
g++ -std=gnu++11 -shared -L/home/biocbuild/bbs-3.17-bioc/R/lib -L/usr/local/lib -o OUTRIDER.so RcppExports.o loss_n_gradient_functions.o -fopenmp -llapack -L/home/biocbuild/bbs-3.17-bioc/R/lib -lRblas -lgfortran -lm -L/home/biocbuild/bbs-3.17-bioc/R/lib -lR
installing to /home/biocbuild/bbs-3.17-bioc/R/library/00LOCK-OUTRIDER/00new/OUTRIDER/libs
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (OUTRIDER)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">OUTRIDER.Rcheck/tests/testthat.Rout.fail</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(OUTRIDER)
Loading required package: BiocParallel
Loading required package: GenomicFeatures
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
Loading required package: GenomicRanges
Loading required package: AnnotationDbi
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

Loading required package: SummarizedExperiment
Loading required package: MatrixGenerics
Loading required package: matrixStats

Attaching package: &#x27;matrixStats&#x27;

The following objects are masked from &#x27;package:Biobase&#x27;:

    anyMissing, rowMedians


Attaching package: &#x27;MatrixGenerics&#x27;

The following objects are masked from &#x27;package:matrixStats&#x27;:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars

The following object is masked from &#x27;package:Biobase&#x27;:

    rowMedians

Loading required package: data.table

Attaching package: &#x27;data.table&#x27;

The following object is masked from &#x27;package:SummarizedExperiment&#x27;:

    shift

The following object is masked from &#x27;package:GenomicRanges&#x27;:

    shift

The following object is masked from &#x27;package:IRanges&#x27;:

    shift

The following objects are masked from &#x27;package:S4Vectors&#x27;:

    first, second

&gt; 
&gt; register(SerialParam())
&gt; 
&gt; test_check(&quot;OUTRIDER&quot;)
[1] &quot;Tue Feb  7 10:41:35 2023: Initial PCA loss: 6.23443546871152&quot;
[1] &quot;Tue Feb  7 10:41:38 2023: Iteration: 1 loss: 4.48133717042354&quot;
[1] &quot;Tue Feb  7 10:41:40 2023: Iteration: 2 loss: 4.42404695782621&quot;
[1] &quot;Tue Feb  7 10:41:41 2023: Iteration: 3 loss: 4.38477899522259&quot;
[1] &quot;Tue Feb  7 10:41:42 2023: Iteration: 4 loss: 4.36539405165039&quot;
[1] &quot;Tue Feb  7 10:41:43 2023: Iteration: 5 loss: 4.35942745103864&quot;
[1] &quot;Tue Feb  7 10:41:44 2023: Iteration: 6 loss: 4.35611040440345&quot;
Time difference of 8.072001 secs
[1] &quot;Tue Feb  7 10:41:44 2023: 6 Final nb-AE loss: 4.35611040440345&quot;
[1] &quot;Tue Feb  7 10:41:46 2023: Initial PCA loss: 6.23443546871152&quot;
[1] &quot;Tue Feb  7 10:41:48 2023: Iteration: 1 loss: 4.48133717042354&quot;
[1] &quot;Tue Feb  7 10:41:49 2023: Iteration: 2 loss: 4.42404695782621&quot;
[1] &quot;Tue Feb  7 10:41:50 2023: Iteration: 3 loss: 4.38477899522259&quot;
[1] &quot;Tue Feb  7 10:41:51 2023: Iteration: 4 loss: 4.36539405165039&quot;
[1] &quot;Tue Feb  7 10:41:52 2023: Iteration: 5 loss: 4.35942745103864&quot;
[1] &quot;Tue Feb  7 10:41:53 2023: Iteration: 6 loss: 4.35611040440345&quot;
Time difference of 6.006369 secs
[1] &quot;Tue Feb  7 10:41:53 2023: 6 Final nb-AE loss: 4.35611040440345&quot;
class: OutriderDataSet
class: RangedSummarizedExperiment 
dim: 100 50 
metadata(1): version
assays(1): counts
rownames(100): ENSG00000223972.4 ENSG00000227232.4 ...
  ENSG00000107404.13 ENSG00000162576.12
rowData names(0):
colnames(50): GTEX.UPK5.0426.SM.3GAEK GTEX.WFG7.2026.SM.3GIL7 ...
  GTEX.11UD2.2526.SM.5CVNU GTEX.13O61.1426.SM.5KM3D
colData names(1): sampleID
[1] &quot;Tue Feb  7 10:42:15 2023: Initial PCA loss: 4.4885467635678&quot;
[1] &quot;Tue Feb  7 10:42:18 2023: Iteration: 1 loss: 4.15953916852321&quot;
[1] &quot;Tue Feb  7 10:42:19 2023: Iteration: 2 loss: 4.14004814419489&quot;
Time difference of 3.183209 secs
[1] &quot;Tue Feb  7 10:42:19 2023: 2 Final nb-AE loss: 4.14004814419489&quot;
[1] &quot;Evaluation loss: 0.517781423170479 for q=3&quot;
[1] &quot;Tue Feb  7 10:42:21 2023: Initial PCA loss: 4.4572558295302&quot;
[1] &quot;Tue Feb  7 10:42:24 2023: Iteration: 1 loss: 4.07122946861174&quot;
[1] &quot;Tue Feb  7 10:42:26 2023: Iteration: 2 loss: 4.05826513752563&quot;
Time difference of 3.442074 secs
[1] &quot;Tue Feb  7 10:42:26 2023: 2 Final nb-AE loss: 4.05826513752563&quot;
[1] &quot;Evaluation loss: 0.454717572219274 for q=4&quot;
[1] &quot;Tue Feb  7 10:42:27 2023: Initial PCA loss: 4.4282762368912&quot;
[1] &quot;Tue Feb  7 10:42:29 2023: Iteration: 1 loss: 3.9973349377491&quot;
[1] &quot;Tue Feb  7 10:42:30 2023: Iteration: 2 loss: 3.96432433728888&quot;
Time difference of 2.703483 secs
[1] &quot;Tue Feb  7 10:42:30 2023: 2 Final nb-AE loss: 3.96432433728888&quot;
[1] &quot;Evaluation loss: 0.426020105666412 for q=5&quot;
Killed
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">OUTRIDER.Rcheck/OUTRIDER-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
