<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for breakpointR on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.17</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><B>[<A href="..#B">B</A>]</B></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-03-27 05:50:30 -0000 (Mon, 27 Mar 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-03-12&nbsp;r83975)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">6083</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for breakpointR on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the breakpointR package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/breakpointR.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>234</B>/2195</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">breakpointR</A>&nbsp;1.17.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/breakpointR">landing page</A>)</SPAN><BR>David Porubsky<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-03-24&nbsp;01:40:07&nbsp;-0000&nbsp;(Fri,&nbsp;24&nbsp;Mar&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/breakpointR</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">7e2ce62</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:18:28&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: breakpointR</TD></TR>
<TR><TD><B>Version</B>: 1.17.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:breakpointR.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings breakpointR_1.17.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-03-24 22:21:58 -0000 (Fri, 24 Mar 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-03-24 22:28:01 -0000 (Fri, 24 Mar 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 363.9 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: breakpointR.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:breakpointR.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings breakpointR_1.17.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/breakpointR.Rcheck’
* using R Under development (unstable) (2023-03-12 r83975)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘breakpointR/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘breakpointR’ version ‘1.17.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘breakpointR’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                     user system elapsed
genotyping         15.530  0.207  15.743
hotspotter          7.515  0.187   7.712
synchronizeReadDir  5.636  0.051   5.687
exportRegions       2.757  1.426   5.829
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘breakpointR.Rnw’... OK
 NONE
* checking re-building of vignette outputs ... ERROR
Error(s) in re-building vignettes:
--- re-building ‘breakpointR.Rnw’ using knitr
breakpointr            package:breakpointR             R Documentation

_M_a_i_n _f_u_n_c_t_i_o_n _f_o_r _t_h_e &#x27;_b_r_e_a_k_p_o_i_n_t_R&#x27; _p_a_c_k_a_g_e

_D_e_s_c_r_i_p_t_i_o_n:

     This function is an easy-to-use wrapper to find breakpoints with
     &#x27;runBreakpointr&#x27; in parallel, write the results to file, plot
     results and find hotspots.

_U_s_a_g_e:

     breakpointr(
       inputfolder,
       outputfolder,
       configfile = NULL,
       numCPU = 1,
       reuse.existing.files = FALSE,
       windowsize = 1e+06,
       binMethod = &quot;size&quot;,
       multi.sizes = NULL,
       pairedEndReads = FALSE,
       pair2frgm = FALSE,
       chromosomes = NULL,
       min.mapq = 10,
       filtAlt = FALSE,
       genoT = &quot;fisher&quot;,
       trim = 10,
       peakTh = 0.33,
       zlim = 3.291,
       background = 0.05,
       minReads = 10,
       maskRegions = NULL,
       callHotSpots = FALSE,
       conf = 0.99
     )
     
_A_r_g_u_m_e_n_t_s:

inputfolder: Folder with BAM files.

outputfolder: Folder to output the results. If it does not exist it
          will be created.

configfile: A file specifying the parameters of this function (without
          &#x27;inputfolder&#x27;, &#x27;outputfolder&#x27; and &#x27;configfile&#x27;). Having the
          parameters in a file can be handy if many samples with the
          same parameter settings are to be run. If a &#x27;configfile&#x27; is
          specified, it will take priority over the command line
          parameters.

  numCPU: The numbers of CPUs that are used. Should not be more than
          available on your machine.

reuse.existing.files: A logical indicating whether or not existing
          files in &#x27;outputfolder&#x27; should be reused.

windowsize: The window size used to calculate deltaWs, either number of
          reads or genomic size depending on &#x27;binMethod&#x27;.

binMethod: Method used to calculate optimal number of reads in the
          window (&quot;size&quot;, &quot;reads&quot;). By default &#x27;binMethod=&#x27;size&#x27;&#x27;.

multi.sizes: User defined multiplications of the original window size.

pairedEndReads: Set to &#x27;TRUE&#x27; if you have paired-end reads in your
          file.

pair2frgm: Set to &#x27;TRUE&#x27; if every paired-end read should be merged into
          a single fragment.

chromosomes: If only a subset of the chromosomes should be binned,
          specify them here.

min.mapq: Minimum mapping quality when importing from BAM files.

 filtAlt: Set to &#x27;TRUE&#x27; if you want to filter out alternative
          alignments defined in &#x27;XA&#x27; tag.

   genoT: A method (&#x27;fisher&#x27; or &#x27;binom&#x27;) to genotype regions defined by
          a set of breakpoints.

    trim: The amount of outliers in deltaWs removed to calculate the
          stdev (10 will remove top 10% and bottom 10% of deltaWs).

  peakTh: The treshold that the peak deltaWs must pass to be considered
          a breakpoint (e.g. 0.33 is 1/3 of max(deltaW)).

    zlim: The number of stdev that the deltaW must pass the peakTh
          (ensures only significantly higher peaks are considered).

background: The percent (e.g. 0.05 = 5%) of background reads allowed
          for WW or CC genotype calls.

minReads: The minimal number of reads between two breaks required for
          genotyping.

maskRegions: List of regions to be excluded from the analysis
          (tab-separated file: chromosomes start end).

callHotSpots: Search for regions of high abundance of breakpoints in
          single cells.

    conf: Desired confidence interval of localized breakpoints.

_V_a_l_u_e:

     &#x27;NULL&#x27;

_A_u_t_h_o_r(_s):

     David Porubsky, Aaron Taudt, Ashley Sanders

_E_x_a_m_p_l_e_s:

     ## Not run:
     
     ## The following call produces plots and genome browser files for all BAM files in &quot;my-data-folder&quot;
     breakpointr(inputfolder=&quot;my-data-folder&quot;, outputfolder=&quot;my-output-folder&quot;)
     ## End(Not run)
     

Killed

* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/breakpointR.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">breakpointR.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL breakpointR
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library’
* installing *source* package ‘breakpointR’ ...
** using staged installation
** R
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (breakpointR)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">breakpointR.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(breakpointR)
Loading required package: GenomicRanges
Loading required package: stats4
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
Loading required package: cowplot
Loading required package: breakpointRdata
&gt; 
&gt; test_check(&quot;breakpointR&quot;) 
[ FAIL 0 | WARN 0 | SKIP 0 | PASS 5 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
 21.014   1.149  22.204 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">breakpointR.Rcheck/breakpointR-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>breakSeekr</TD><TD style="text-align: right;">1.718</TD><TD style="text-align: right;">0.172</TD><TD style="text-align: right;">1.904</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>breakpointr</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>breakpointr2UCSC</TD><TD style="text-align: right;">1.072</TD><TD style="text-align: right;">0.217</TD><TD style="text-align: right;">1.289</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>confidenceInterval</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>confidenceInterval.binomial</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>deltaWCalculator</TD><TD style="text-align: right;">0.525</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.538</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>exportRegions</TD><TD style="text-align: right;">2.757</TD><TD style="text-align: right;">1.426</TD><TD style="text-align: right;">5.829</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genotype.binom</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genotype.fisher</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genotyping</TD><TD style="text-align: right;">15.530</TD><TD style="text-align: right;"> 0.207</TD><TD style="text-align: right;">15.743</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>hotspotter</TD><TD style="text-align: right;">7.515</TD><TD style="text-align: right;">0.187</TD><TD style="text-align: right;">7.712</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loadFromFiles</TD><TD style="text-align: right;">1.644</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">1.684</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotBreakpoints</TD><TD style="text-align: right;">2.642</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">2.710</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotBreakpointsPerChr</TD><TD style="text-align: right;">3.287</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">3.339</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotHeatmap</TD><TD style="text-align: right;">2.869</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">2.902</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ranges2UCSC</TD><TD style="text-align: right;">0.357</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.358</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readBamFileAsGRanges</TD><TD style="text-align: right;">0.215</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.236</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>removeReadPileupSpikes</TD><TD style="text-align: right;">2.488</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">2.552</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>runBreakpointr</TD><TD style="text-align: right;">3.115</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">3.127</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>summarizeBreaks</TD><TD style="text-align: right;">0.319</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.323</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>synchronizeReadDir</TD><TD style="text-align: right;">5.636</TD><TD style="text-align: right;">0.051</TD><TD style="text-align: right;">5.687</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
