<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for compcodeR on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.17</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><B>[<A href="..#C">C</A>]</B></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-27 02:34:31 -0000 (Mon, 27 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4259</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for compcodeR on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the compcodeR package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/compcodeR.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>388</B>/2169</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">compcodeR</A>&nbsp;1.35.1</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/compcodeR">landing page</A>)</SPAN><BR>Charlotte Soneson<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-23&nbsp;09:40:21&nbsp;-0000&nbsp;(Thu,&nbsp;23&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/compcodeR</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">98f38b6</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-09&nbsp;14:44:38&nbsp;-0000&nbsp;(Wed,&nbsp;09&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: compcodeR</TD></TR>
<TR><TD><B>Version</B>: 1.35.1</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:compcodeR.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings compcodeR_1.35.1.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-24 09:22:02 -0000 (Fri, 24 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-24 09:30:24 -0000 (Fri, 24 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 501.7 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: compcodeR.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:compcodeR.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings compcodeR_1.35.1.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/compcodeR.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘compcodeR/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘compcodeR’ version ‘1.35.1’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Package which this enhances but not available for checking: ‘rpanel’
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘compcodeR’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                           user system elapsed
NBPSeq.createRmd         41.057  4.611  49.117
DESeq2.createRmd         17.029  1.289  18.545
EBSeq.createRmd          10.544  4.583  20.814
DSS.createRmd             7.839  0.472   8.354
baySeq.createRmd          5.571  0.120   5.699
NOISeq.prenorm.createRmd  4.506  0.746   5.309
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  Tip labels:
    A1, A2, A3, B1, C1, C2, ...
  
  Rooted; includes branch lengths.
  
  length.matrix:
        A1    A2    A3   B1    C1    C2
  g1  2757  2757  2757 3086  3896  3896
  g2  3699  3699  3699 2513  2002  2002
  g3  1983  1983  1983 7404  3283  3283
  g4  3651  3651  3651 3419  3521  3521
  g5  2144  2144  2144 1441  6609  6609
  g6 18805 18805 18805 6302 20047 20047
  
  Killed
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘compcodeR.Rmd’ using ‘UTF-8’... OK
  ‘phylocompcodeR.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 1 NOTE
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/compcodeR.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">compcodeR.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL compcodeR
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘compcodeR’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (compcodeR)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">compcodeR.Rcheck/tests/testthat.Rout.fail</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; if (requireNamespace(&quot;testthat&quot;, quietly = TRUE)) {
+   
+   library(testthat)
+   library(compcodeR)
+   
+   test_check(&quot;compcodeR&quot;)
+   
+ }
Loading required package: sm
Package &#x27;sm&#x27;, version 2.2-5.7: type help(sm) for summary information
 [1] &quot;DESeq2.createRmd&quot;               &quot;DESeq2.length.createRmd&quot;       
 [3] &quot;DSS.createRmd&quot;                  &quot;EBSeq.createRmd&quot;               
 [5] &quot;NBPSeq.createRmd&quot;               &quot;NOISeq.prenorm.createRmd&quot;      
 [7] &quot;TCC.createRmd&quot;                  &quot;baySeq.createRmd&quot;              
 [9] &quot;edgeR.GLM.createRmd&quot;            &quot;edgeR.exact.createRmd&quot;         
[11] &quot;lengthNorm.limma.createRmd&quot;     &quot;lengthNorm.sva.limma.createRmd&quot;
[13] &quot;logcpm.limma.createRmd&quot;         &quot;phylolm.createRmd&quot;             
[15] &quot;sqrtcpm.limma.createRmd&quot;        &quot;ttest.createRmd&quot;               
[17] &quot;voom.limma.createRmd&quot;           &quot;voom.ttest.createRmd&quot;          
An object of class compData 
Dataset name: B_625_625 
Number of samples: 10 
Number of variables: 500 
Number of random outliers: 0 
Number of single outliers: 0 
Number of known truly differentially expressed genes: 50 
Differential expression analysis was performed by the method edgeR.3.41.2.exact.TMM.movingave.tagwise on Fri Feb 24 09:27:34 2023 . 

count.matrix:
   sample1 sample2 sample3 sample4 sample5 sample6
g1       4       8       5       2       8       8
g2     314     419     186     172     294     133
g3    1030    1583    1011    1033     747    1291
g4       0       1       1       0       2       2
g5      48     122     215      62     106      90
g6      27      16      18      44      29      74
+ 494 rows and 4 cols...

sample.annotations:
        condition depth.factor
sample1         1    1.1049378
sample2         1    1.2875127
sample3         1    1.1581115
sample4         1    1.3611698
sample5         1    1.0581603
sample6         2    0.7594447
+ 4 rows...

variable.annotations:
   truedispersions.S1 truedispersions.S2 truemeans.S1 truemeans.S2
g1         0.11610054         0.11610054     6.878049    16.363472
g2         0.43796296         0.43796296   303.073171   893.165617
g3         0.04321133         0.04321133  1223.414634  2760.990905
g4         0.55624371         0.55624371     0.902439     6.153083
g5         0.15968189         0.15968189   148.146341   316.915193
g6         0.36378999         0.36378999    40.439024   132.785087
   n.random.outliers.up.S1 n.random.outliers.up.S2 n.random.outliers.down.S1
g1                       0                       0                         0
g2                       0                       0                         0
g3                       0                       0                         0
g4                       0                       0                         0
g5                       0                       0                         0
g6                       0                       0                         0
   n.random.outliers.down.S2 n.single.outliers.up.S1 n.single.outliers.up.S2
g1                         0                       0                       0
g2                         0                       0                       0
g3                         0                       0                       0
g4                         0                       0                       0
g5                         0                       0                       0
g6                         0                       0                       0
   n.single.outliers.down.S1 n.single.outliers.down.S2  M.value    A.value
g1                         0                         0 1.158386  2.9291506
g2                         0                         0 1.349570  8.6365105
g3                         0                         0 1.050582 10.4859190
g4                         0                         0 1.432447  0.7393097
g5                         0                         0 1.185233  7.1150497
g6                         0                         0 2.176450  5.6916570
   truelog2foldchanges upregulation downregulation differential.expression
g1            1.250408            1              0                       1
g2            1.559262            1              0                       1
g3            1.174273            1              0                       1
g4            2.769408            1              0                       1
g5            1.097074            1              0                       1
g6            1.715273            1              0                       1
+ 494 rows...

Differential expression results:

         pvalue    adjpvalue    logFC     score dispersion.S1 dispersion.S2
g1 5.804805e-02 5.091934e-01 1.109582 0.9419519     0.2378950     0.2378950
g2 7.039843e-04 2.199951e-02 1.603157 0.9992960     0.2509797     0.2509797
g3 1.014563e-03 2.984008e-02 1.062861 0.9989854     0.1215078     0.1215078
g4 6.539309e-02 5.449424e-01 2.019660 0.9346069     0.5641107     0.5641107
g5 9.940688e-03 1.713912e-01 1.129158 0.9900593     0.2158532     0.2158532
g6 4.983594e-07 8.305991e-05 2.194748 0.9999995     0.1898635     0.1898635
+ 494 rows...

An object of class compData 
Dataset name: B_625_625 
Number of samples: 4 
Number of variables: 500 
Number of random outliers: 0 
Number of single outliers: 0 
Number of known truly differentially expressed genes: 50 
Differential expression analysis was performed by the method edgeR.3.41.2.exact.TMM.movingave.tagwise on Fri Feb 24 09:27:34 2023 . 

count.matrix:
   sample1 sample2 sample3 sample4
g1       4       8       5       2
g2     314     419     186     172
g3    1030    1583    1011    1033
g4       0       1       1       0
g5      48     122     215      62
g6      27      16      18      44
+ 494 rows...

sample.annotations:
        condition depth.factor
sample1         1    1.1049378
sample2         1    1.2875127
sample3         1    1.1581115
sample4         1    1.3611698
sample5         1    1.0581603
sample6         2    0.7594447
+ 4 rows...

variable.annotations:
   truedispersions.S1 truedispersions.S2 truemeans.S1 truemeans.S2
g1         0.11610054         0.11610054     6.878049    16.363472
g2         0.43796296         0.43796296   303.073171   893.165617
g3         0.04321133         0.04321133  1223.414634  2760.990905
g4         0.55624371         0.55624371     0.902439     6.153083
g5         0.15968189         0.15968189   148.146341   316.915193
g6         0.36378999         0.36378999    40.439024   132.785087
   n.random.outliers.up.S1 n.random.outliers.up.S2 n.random.outliers.down.S1
g1                       0                       0                         0
g2                       0                       0                         0
g3                       0                       0                         0
g4                       0                       0                         0
g5                       0                       0                         0
g6                       0                       0                         0
   n.random.outliers.down.S2 n.single.outliers.up.S1 n.single.outliers.up.S2
g1                         0                       0                       0
g2                         0                       0                       0
g3                         0                       0                       0
g4                         0                       0                       0
g5                         0                       0                       0
g6                         0                       0                       0
   n.single.outliers.down.S1 n.single.outliers.down.S2  M.value    A.value
g1                         0                         0 1.158386  2.9291506
g2                         0                         0 1.349570  8.6365105
g3                         0                         0 1.050582 10.4859190
g4                         0                         0 1.432447  0.7393097
g5                         0                         0 1.185233  7.1150497
g6                         0                         0 2.176450  5.6916570
   truelog2foldchanges upregulation downregulation differential.expression
g1            1.250408            1              0                       1
g2            1.559262            1              0                       1
g3            1.174273            1              0                       1
g4            2.769408            1              0                       1
g5            1.097074            1              0                       1
g6            1.715273            1              0                       1
+ 494 rows...

Differential expression results:

         pvalue    adjpvalue    logFC     score dispersion.S1 dispersion.S2
g1 5.804805e-02 5.091934e-01 1.109582 0.9419519     0.2378950     0.2378950
g2 7.039843e-04 2.199951e-02 1.603157 0.9992960     0.2509797     0.2509797
g3 1.014563e-03 2.984008e-02 1.062861 0.9989854     0.1215078     0.1215078
g4 6.539309e-02 5.449424e-01 2.019660 0.9346069     0.5641107     0.5641107
g5 9.940688e-03 1.713912e-01 1.129158 0.9900593     0.2158532     0.2158532
g6 4.983594e-07 8.305991e-05 2.194748 0.9999995     0.1898635     0.1898635
+ 494 rows...

An object of class phyloCompData 
Dataset name: B_625_625 
Number of samples: 10 
Number of variables: 498 
Number of random outliers: 0 
Number of single outliers: 0 
Number of known truly differentially expressed genes: 49 
Differential expression analysis was performed by the method length.3.55.4.limma.TMM.lengthNorm.RPKM.dataTrans.log2.no_trend on Fri Feb 24 09:28:06 2023 . 

count.matrix:
   sample1 sample2 sample3 sample4 sample5 sample6
g1       2       1       0       0       4       5
g2      18     418     321      25       5     287
g3       2       0       1       4       2      20
g4       8       6      69     126      30      97
g5    2495     148    1736     738    1531     570
g6      18      35      18       9      15     115
+ 492 rows and 4 cols...

sample.annotations:
        condition depth.factor
sample1         1    1.1087861
sample2         1    1.2699957
sample3         1    1.1005966
sample4         1    0.8652224
sample5         1    0.7795419
sample6         2    1.2785653
+ 4 rows...

variable.annotations:
   truedispersions.S1 truedispersions.S2 truemeans.S1 truemeans.S2
g1         0.63372227         0.63372227     7.736842     20.13148
g2         0.12249434         0.12249434    95.421053    231.21363
g3         0.72255054         0.72255054     3.087719      5.68084
g4         0.32345589         0.32345589    41.634146     96.64933
g5         0.03866619         0.03866619  2015.487805   3792.55744
g6         0.11797778         0.11797778    28.087719     84.52361
   n.random.outliers.up.S1 n.random.outliers.up.S2 n.random.outliers.down.S1
g1                       0                       0                         0
g2                       0                       0                         0
g3                       0                       0                         0
g4                       0                       0                         0
g5                       0                       0                         0
g6                       0                       0                         0
   n.random.outliers.down.S2 n.single.outliers.up.S1 n.single.outliers.up.S2
g1                         0                       0                       0
g2                         0                       0                       0
g3                         0                       0                       0
g4                         0                       0                       0
g5                         0                       0                       0
g6                         0                       0                       0
   n.single.outliers.down.S1 n.single.outliers.down.S2    M.value  A.value
g1                         0                         0  1.5018073 1.152706
g2                         0                         0  0.2078055 5.766156
g3                         0                         0  1.0483242 1.390350
g4                         0                         0 -1.3755162 4.062796
g5                         0                         0 -0.3685706 9.676270
g6                         0                         0  1.7362595 5.008725
   truelog2foldchanges upregulation downregulation differential.expression
g1           1.3796366            1              0                       1
g2           1.2768469            1              0                       1
g3           0.8795626            1              0                       1
g4           1.2149925            1              0                       1
g5           0.9120420            1              0                       1
g6           1.5894149            1              0                       1
   lengths.relmeans lengths.dispersions M.value.TPM A.value.TPM
g1             9917           4.4902541  -0.4669503   -8.754908
g2             9970           2.2026858   1.1693418   -6.592122
g3            10057           0.2040650   0.7688165  -11.695832
g4             9969           1.1343801   0.3068095   -7.838718
g5             9920           0.8830628   0.6697297   -2.400325
g6             9937           0.0484685   1.5876254   -8.210384
+ 492 rows...

Differential expression results:

         pvalue  adjpvalue      logFC     score
g1 0.8683815621 0.97180678 -0.3149727 0.1316184
g2 0.0368385542 0.41748040  1.3213195 0.9631614
g3 0.4388545775 0.92126849  0.9207942 0.5611454
g4 0.4357368637 0.92126849  0.4587872 0.5642631
g5 0.0055826869 0.13179123  0.8217073 0.9944173
g6 0.0003053375 0.03570528  1.7396031 0.9996947
+ 492 rows...

Without phylogenetic tree. 
length.matrix:
   sample1 sample2 sample3 sample4 sample5 sample6
g1     709       7    1472     475   10112    8029
g2    1533   53870   72873    4209    1085   22717
g3    3749   14799    7828    6446   14243    4638
g4     890    4717   14938   37345    8403    9837
g5   10753     940   13697    6162   13863    1721
g6    6679   15130    9003    9066    9614    9614

An object of class phyloCompData 
Dataset name: B_625_625 
Number of samples: 4 
Number of variables: 498 
Number of random outliers: 0 
Number of single outliers: 0 
Number of known truly differentially expressed genes: 49 
Differential expression analysis was performed by the method length.3.55.4.limma.TMM.lengthNorm.RPKM.dataTrans.log2.no_trend on Fri Feb 24 09:28:06 2023 . 

count.matrix:
   sample1 sample2 sample3 sample4
g1       2       1       0       0
g2      18     418     321      25
g3       2       0       1       4
g4       8       6      69     126
g5    2495     148    1736     738
g6      18      35      18       9
+ 492 rows...

sample.annotations:
        condition depth.factor
sample1         1    1.1087861
sample2         1    1.2699957
sample3         1    1.1005966
sample4         1    0.8652224
sample5         1    0.7795419
sample6         2    1.2785653
+ 4 rows...

variable.annotations:
   truedispersions.S1 truedispersions.S2 truemeans.S1 truemeans.S2
g1         0.63372227         0.63372227     7.736842     20.13148
g2         0.12249434         0.12249434    95.421053    231.21363
g3         0.72255054         0.72255054     3.087719      5.68084
g4         0.32345589         0.32345589    41.634146     96.64933
g5         0.03866619         0.03866619  2015.487805   3792.55744
g6         0.11797778         0.11797778    28.087719     84.52361
   n.random.outliers.up.S1 n.random.outliers.up.S2 n.random.outliers.down.S1
g1                       0                       0                         0
g2                       0                       0                         0
g3                       0                       0                         0
g4                       0                       0                         0
g5                       0                       0                         0
g6                       0                       0                         0
   n.random.outliers.down.S2 n.single.outliers.up.S1 n.single.outliers.up.S2
g1                         0                       0                       0
g2                         0                       0                       0
g3                         0                       0                       0
g4                         0                       0                       0
g5                         0                       0                       0
g6                         0                       0                       0
   n.single.outliers.down.S1 n.single.outliers.down.S2    M.value  A.value
g1                         0                         0  1.5018073 1.152706
g2                         0                         0  0.2078055 5.766156
g3                         0                         0  1.0483242 1.390350
g4                         0                         0 -1.3755162 4.062796
g5                         0                         0 -0.3685706 9.676270
g6                         0                         0  1.7362595 5.008725
   truelog2foldchanges upregulation downregulation differential.expression
g1           1.3796366            1              0                       1
g2           1.2768469            1              0                       1
g3           0.8795626            1              0                       1
g4           1.2149925            1              0                       1
g5           0.9120420            1              0                       1
g6           1.5894149            1              0                       1
   lengths.relmeans lengths.dispersions M.value.TPM A.value.TPM
g1             9917           4.4902541  -0.4669503   -8.754908
g2             9970           2.2026858   1.1693418   -6.592122
g3            10057           0.2040650   0.7688165  -11.695832
g4             9969           1.1343801   0.3068095   -7.838718
g5             9920           0.8830628   0.6697297   -2.400325
g6             9937           0.0484685   1.5876254   -8.210384
+ 492 rows...

Differential expression results:

         pvalue  adjpvalue      logFC     score
g1 0.8683815621 0.97180678 -0.3149727 0.1316184
g2 0.0368385542 0.41748040  1.3213195 0.9631614
g3 0.4388545775 0.92126849  0.9207942 0.5611454
g4 0.4357368637 0.92126849  0.4587872 0.5642631
g5 0.0055826869 0.13179123  0.8217073 0.9944173
g6 0.0003053375 0.03570528  1.7396031 0.9996947
+ 492 rows...

Without phylogenetic tree. 
length.matrix:
   sample1 sample2 sample3 sample4 sample5 sample6
g1     709       7    1472     475   10112    8029
g2    1533   53870   72873    4209    1085   22717
g3    3749   14799    7828    6446   14243    4638
g4     890    4717   14938   37345    8403    9837
g5   10753     940   13697    6162   13863    1721
g6    6679   15130    9003    9066    9614    9614

An object of class phyloCompData 
Dataset name: B_625_625 
Number of samples: 8 
Number of variables: 489 
Number of random outliers: 0 
Number of single outliers: 0 
Number of known truly differentially expressed genes: 49 
Differential expression analysis was performed by the method length.3.55.4.limma.TMM.lengthNorm.TPM.dataTrans.log2.no_trend on Fri Feb 24 09:28:30 2023 . 

count.matrix:
     A1   A2   A3  B1  C1  C2
g1    2    2    3   5  16   5
g2    2    4    4   1   3   4
g3    8   17   18  43 128  97
g4  112  144  180 147 693 519
g5   42   50   36  39 318 236
g6 2144 2813 2980 244 407 301
+ 483 rows and 2 cols...

sample.annotations:
   condition depth.factor id.species id.condition
A1         1    0.9345425          1            1
A2         1    1.1908725          1            1
A3         1    1.2883744          1            1
B1         1    1.3178424          2            1
C1         2    1.2366125          3            2
C2         2    0.9254193          3            2
+ 2 rows...

variable.annotations:
   truedispersions.S1 truedispersions.S2 truemeans.S1 truemeans.S2
g1         0.30951552         0.30951552     9.438596     16.76493
g2         0.10717866         0.10717866    11.087719     17.83999
g3         0.26784154         0.26784154    64.024390    130.13981
g4         0.14093922         0.14093922   380.097561   1569.50731
g5         0.06477025         0.06477025   246.073171    491.17016
g6         0.47668995         0.47668995   306.536585    461.43426
   n.random.outliers.up.S1 n.random.outliers.up.S2 n.random.outliers.down.S1
g1                       0                       0                         0
g2                       0                       0                         0
g3                       0                       0                         0
g4                       0                       0                         0
g5                       0                       0                         0
g6                       0                       0                         0
   n.random.outliers.down.S2 n.single.outliers.up.S1 n.single.outliers.up.S2
g1                         0                       0                       0
g2                         0                       0                       0
g3                         0                       0                       0
g4                         0                       0                       0
g5                         0                       0                       0
g6                         0                       0                       0
   n.single.outliers.down.S1 n.single.outliers.down.S2    M.value  A.value
g1                         0                         0  1.7048375 2.424683
g2                         0                         0  0.3415076 1.577042
g3                         0                         0  2.5311743 5.327476
g4                         0                         0  2.0962388 8.065006
g5                         0                         0  2.8226257 6.646218
g6                         0                         0 -2.3683284 9.160340
   truelog2foldchanges upregulation downregulation differential.expression
g1           0.8288024            1              0                       1
g2           0.6861520            1              0                       1
g3           1.0233688            1              0                       1
g4           2.0458701            1              0                       1
g5           0.9971356            1              0                       1
g6           0.5900659            1              0                       1
   prop.var.tree lengths.relmeans lengths.dispersions M.value.TPM A.value.TPM
g1             1         6468.643           0.5912307    1.100579   -9.389239
g2             1         2321.214           0.1018562    1.044485   -9.827207
g3             1         7789.857           1.5129387    1.226028   -6.795971
g4             1         3893.429           1.1472771    2.220931   -3.725295
g5             1         3626.786           1.2262940    1.524808   -4.967721
g6             1         9735.071           0.4032376   -1.871160   -4.437845
+ 483 rows...

Differential expression results:

         pvalue    adjpvalue     logFC     score
g1 1.358626e-02 0.0671079008  1.074670 0.9864137
g2 7.770022e-02 0.2122648431  1.018576 0.9222998
g3 1.423563e-01 0.3149875048  1.200119 0.8576437
g4 1.801480e-06 0.0004404618  2.195022 0.9999982
g5 2.877064e-04 0.0060718212  1.498899 0.9997123
g6 3.713227e-03 0.0275116347 -1.897069 0.9962868
+ 483 rows...

Phylogenetic tree:

Phylogenetic tree with 8 tips and 6 internal nodes.

Tip labels:
  A1, A2, A3, B1, C1, C2, ...

Rooted; includes branch lengths.

length.matrix:
      A1    A2    A3   B1    C1    C2
g1  2757  2757  2757 3086  3896  3896
g2  3699  3699  3699 2513  2002  2002
g3  1983  1983  1983 7404  3283  3283
g4  3651  3651  3651 3419  3521  3521
g5  2144  2144  2144 1441  6609  6609
g6 18805 18805 18805 6302 20047 20047

An object of class phyloCompData 
Dataset name: B_625_625 
Number of samples: 4 
Number of variables: 489 
Number of random outliers: 0 
Number of single outliers: 0 
Number of known truly differentially expressed genes: 49 
Differential expression analysis was performed by the method length.3.55.4.limma.TMM.lengthNorm.TPM.dataTrans.log2.no_trend on Fri Feb 24 09:28:30 2023 . 

count.matrix:
     A1   A2   A3  B1
g1    2    2    3   5
g2    2    4    4   1
g3    8   17   18  43
g4  112  144  180 147
g5   42   50   36  39
g6 2144 2813 2980 244
+ 483 rows...

sample.annotations:
   condition depth.factor id.species id.condition
A1         1    0.9345425          1            1
A2         1    1.1908725          1            1
A3         1    1.2883744          1            1
B1         1    1.3178424          2            1
C1         2    1.2366125          3            2
C2         2    0.9254193          3            2
+ 2 rows...

variable.annotations:
   truedispersions.S1 truedispersions.S2 truemeans.S1 truemeans.S2
g1         0.30951552         0.30951552     9.438596     16.76493
g2         0.10717866         0.10717866    11.087719     17.83999
g3         0.26784154         0.26784154    64.024390    130.13981
g4         0.14093922         0.14093922   380.097561   1569.50731
g5         0.06477025         0.06477025   246.073171    491.17016
g6         0.47668995         0.47668995   306.536585    461.43426
   n.random.outliers.up.S1 n.random.outliers.up.S2 n.random.outliers.down.S1
g1                       0                       0                         0
g2                       0                       0                         0
g3                       0                       0                         0
g4                       0                       0                         0
g5                       0                       0                         0
g6                       0                       0                         0
   n.random.outliers.down.S2 n.single.outliers.up.S1 n.single.outliers.up.S2
g1                         0                       0                       0
g2                         0                       0                       0
g3                         0                       0                       0
g4                         0                       0                       0
g5                         0                       0                       0
g6                         0                       0                       0
   n.single.outliers.down.S1 n.single.outliers.down.S2    M.value  A.value
g1                         0                         0  1.7048375 2.424683
g2                         0                         0  0.3415076 1.577042
g3                         0                         0  2.5311743 5.327476
g4                         0                         0  2.0962388 8.065006
g5                         0                         0  2.8226257 6.646218
g6                         0                         0 -2.3683284 9.160340
   truelog2foldchanges upregulation downregulation differential.expression
g1           0.8288024            1              0                       1
g2           0.6861520            1              0                       1
g3           1.0233688            1              0                       1
g4           2.0458701            1              0                       1
g5           0.9971356            1              0                       1
g6           0.5900659            1              0                       1
   prop.var.tree lengths.relmeans lengths.dispersions M.value.TPM A.value.TPM
g1             1         6468.643           0.5912307    1.100579   -9.389239
g2             1         2321.214           0.1018562    1.044485   -9.827207
g3             1         7789.857           1.5129387    1.226028   -6.795971
g4             1         3893.429           1.1472771    2.220931   -3.725295
g5             1         3626.786           1.2262940    1.524808   -4.967721
g6             1         9735.071           0.4032376   -1.871160   -4.437845
+ 483 rows...

Differential expression results:

         pvalue    adjpvalue     logFC     score
g1 1.358626e-02 0.0671079008  1.074670 0.9864137
g2 7.770022e-02 0.2122648431  1.018576 0.9222998
g3 1.423563e-01 0.3149875048  1.200119 0.8576437
g4 1.801480e-06 0.0004404618  2.195022 0.9999982
g5 2.877064e-04 0.0060718212  1.498899 0.9997123
g6 3.713227e-03 0.0275116347 -1.897069 0.9962868
+ 483 rows...

Phylogenetic tree:

Phylogenetic tree with 8 tips and 6 internal nodes.

Tip labels:
  A1, A2, A3, B1, C1, C2, ...

Rooted; includes branch lengths.

length.matrix:
      A1    A2    A3   B1    C1    C2
g1  2757  2757  2757 3086  3896  3896
g2  3699  3699  3699 2513  2002  2002
g3  1983  1983  1983 7404  3283  3283
g4  3651  3651  3651 3419  3521  3521
g5  2144  2144  2144 1441  6609  6609
g6 18805 18805 18805 6302 20047 20047

Killed
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">compcodeR.Rcheck/compcodeR-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>DESeq2.createRmd</TD><TD style="text-align: right;">17.029</TD><TD style="text-align: right;"> 1.289</TD><TD style="text-align: right;">18.545</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DESeq2.length.createRmd</TD><TD style="text-align: right;">1.525</TD><TD style="text-align: right;">0.027</TD><TD style="text-align: right;">1.554</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DSS.createRmd</TD><TD style="text-align: right;">7.839</TD><TD style="text-align: right;">0.472</TD><TD style="text-align: right;">8.354</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EBSeq.createRmd</TD><TD style="text-align: right;">10.544</TD><TD style="text-align: right;"> 4.583</TD><TD style="text-align: right;">20.814</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>NBPSeq.createRmd</TD><TD style="text-align: right;">41.057</TD><TD style="text-align: right;"> 4.611</TD><TD style="text-align: right;">49.117</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>NOISeq.prenorm.createRmd</TD><TD style="text-align: right;">4.506</TD><TD style="text-align: right;">0.746</TD><TD style="text-align: right;">5.309</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TCC.createRmd</TD><TD style="text-align: right;">2.719</TD><TD style="text-align: right;">0.177</TD><TD style="text-align: right;">3.145</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>baySeq.createRmd</TD><TD style="text-align: right;">5.571</TD><TD style="text-align: right;">0.120</TD><TD style="text-align: right;">5.699</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>checkDataObject</TD><TD style="text-align: right;">0.375</TD><TD style="text-align: right;">0.017</TD><TD style="text-align: right;">0.391</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>checkTableConsistency</TD><TD style="text-align: right;">1.317</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">1.364</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>check_compData</TD><TD style="text-align: right;">0.391</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.403</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>check_compData_results</TD><TD style="text-align: right;">0.753</TD><TD style="text-align: right;">0.021</TD><TD style="text-align: right;">0.773</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>check_phyloCompData</TD><TD style="text-align: right;">0.909</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">0.997</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>compData</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>convertListTocompData</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>convertListTophyloCompData</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>convertcompDataToList</TD><TD style="text-align: right;">4.597</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">4.661</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>convertphyloCompDataToList</TD><TD style="text-align: right;">0.307</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.319</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>edgeR.GLM.createRmd</TD><TD style="text-align: right;">1.216</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">1.240</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>edgeR.exact.createRmd</TD><TD style="text-align: right;">0.943</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">0.983</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>generateCodeHTMLs</TD><TD style="text-align: right;">0.833</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.867</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>generateSyntheticData</TD><TD style="text-align: right;">0.915</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">0.959</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>lengthNorm.limma.createRmd</TD><TD style="text-align: right;">0.630</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.666</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>lengthNorm.sva.limma.createRmd</TD><TD style="text-align: right;">1.912</TD><TD style="text-align: right;">0.096</TD><TD style="text-align: right;">2.031</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>listcreateRmd</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>logcpm.limma.createRmd</TD><TD style="text-align: right;">0.622</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.646</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nEffRatio</TD><TD style="text-align: right;">0.546</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.558</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>phyloCompData</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>phylolm.createRmd</TD><TD style="text-align: right;">0.731</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.735</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>runComparison</TD><TD style="text-align: right;">1.307</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">1.340</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>runComparisonGUI</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>runDiffExp</TD><TD style="text-align: right;">0.759</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.783</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>show-compData-method</TD><TD style="text-align: right;">4.734</TD><TD style="text-align: right;">0.092</TD><TD style="text-align: right;">4.826</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>show-phyloCompData-method</TD><TD style="text-align: right;">0.697</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.721</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sqrtcpm.limma.createRmd</TD><TD style="text-align: right;">0.628</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.641</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>summarizeSyntheticDataSet</TD><TD style="text-align: right;">0.408</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.416</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ttest.createRmd</TD><TD style="text-align: right;">0.638</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.643</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>voom.limma.createRmd</TD><TD style="text-align: right;">0.783</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.802</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>voom.ttest.createRmd</TD><TD style="text-align: right;">0.660</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.676</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
