<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for epialleleR on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><B>[<A href="../index.html#E">E</A>]</B></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-09 02:50:19 -0000 (Thu, 09 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4165</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for epialleleR on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the epialleleR package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/epialleleR.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>607</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">epialleleR</A>&nbsp;1.7.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/epialleleR">landing page</A>)</SPAN><BR>Oleksii Nikolaienko<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-08&nbsp;03:21:00&nbsp;-0000&nbsp;(Wed,&nbsp;08&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/epialleleR</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">11ca088</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:24:58&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: epialleleR</TD></TR>
<TR><TD><B>Version</B>: 1.7.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:epialleleR.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings epialleleR_1.7.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-08 19:40:14 -0000 (Wed, 08 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-08 19:46:42 -0000 (Wed, 08 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 387.6 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: epialleleR.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 1</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:epialleleR.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings epialleleR_1.7.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/epialleleR.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘epialleleR/DESCRIPTION’ ... OK
* this is package ‘epialleleR’ version ‘1.7.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘epialleleR’ can be installed ... OK
* used C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... NOTE
  installed size is 10.0Mb
  sub-directories of 1Mb or more:
    extdata   1.7Mb
    libs      8.1Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... NOTE
GNU make is a SystemRequirements.
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... WARNING
Note: information on .o files is not available
File ‘/home/biocbuild/bbs-3.17-bioc/R/library/epialleleR/libs/epialleleR.so’:
  Found ‘__sprintf_chk’, possibly from ‘sprintf’ (C)
  Found ‘abort’, possibly from ‘abort’ (C)
  Found ‘exit’, possibly from ‘exit’ (C)
  Found ‘stderr’, possibly from ‘stderr’ (C)
  Found ‘stdout’, possibly from ‘stdout’ (C)

Compiled code should not call entry points which might terminate R nor
write to stdout/stderr instead of to the console, nor use Fortran I/O
nor system RNGs nor [v]sprintf. The detected symbols are linked into
the code but might come from libraries and not actually be called.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                    user system elapsed
generateVcfReport 11.559  0.204  12.582
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘epialleleR.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 WARNING, 2 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/epialleleR.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">epialleleR.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL epialleleR
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘epialleleR’ ...
** using staged installation
** libs
using C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
using C++17
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/BH/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/BH/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c rcpp_cx_report.cpp -o rcpp_cx_report.o
rcpp_cx_report.cpp:182:1: warning: multi-line comment [-Wcomment]
  182 | //  #define spit_results {                                                       \
      | ^
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/BH/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c rcpp_extract_patterns.cpp -o rcpp_extract_patterns.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/BH/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c rcpp_fep.cpp -o rcpp_fep.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/BH/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c rcpp_get_base_freqs.cpp -o rcpp_get_base_freqs.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/BH/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c rcpp_get_xm_beta.cpp -o rcpp_get_xm_beta.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/BH/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c rcpp_match_target.cpp -o rcpp_match_target.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/BH/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c rcpp_read_bam.cpp -o rcpp_read_bam.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/BH/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c rcpp_threshold_reads.cpp -o rcpp_threshold_reads.o
g++ -std=gnu++17 -shared -L/home/biocbuild/bbs-3.17-bioc/R/lib -L/usr/local/lib -o epialleleR.so RcppExports.o rcpp_cx_report.o rcpp_extract_patterns.o rcpp_fep.o rcpp_get_base_freqs.o rcpp_get_xm_beta.o rcpp_match_target.o rcpp_read_bam.o rcpp_threshold_reads.o /home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/usrlib/libhts.a -lcurl -L/home/biocbuild/bbs-3.17-bioc/R/lib -lR
installing to /home/biocbuild/bbs-3.17-bioc/R/library/00LOCK-epialleleR/00new/epialleleR/libs
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (epialleleR)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">epialleleR.Rcheck/tests/runTests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; BiocGenerics:::testPackage(&quot;epialleleR&quot;)
Reading BED file [0.089s]
Reading BAM file [0.019s]
Extracting methylation patterns [0.058s]
1EE321356D67DEA7 C45AA3EC23289387 C99DC1A495039DE7 9CA57BB332312247 C1B2F64164F2B5AC D6D8A6CEE3E02B54 FFA36568BF9C4BA7 7D711760CE92D027 1E16614AEB3EA9E7 F9187A8B842FAD67 9F89EB005B13DFDF 4F548CE96DC1709D 403D02E1C58F4C1D 389A7F1E39043A5C 93B36DB89542952F 05CC5DDC846668CF CADA51AD93658B30 EFE504365E3FE93D 5FC1921CB9B410FD 2D4C54278C7C1E5F EC56ABBF0F4C30E5 0AA9634B9F10CD1D 1403B39B66F8ED47 444C9C81B472E2D9 B5223EC66A3A6D7B 4827640D510DEAE6 E2B8AE54BD72001B 2C4F768A8D26003B EC7E8FA0001ECC7B 6F195BFD2E40CF80 1A250541D33CAE05 00AE5E6E8E743CBB 18C0DC2CEE81C0D4 1DD27FCDA17241AD
Reading BED file [0.056s]
Reading BAM file [0.007s]
Extracting methylation patterns [0.071s]
Reading BAM file [0.007s]
Extracting methylation patterns [0.054s]
Reading BAM file [0.007s]
Extracting methylation patterns [0.050s]
Reading BED file [0.019s]
Reading BAM file [0.007s]
Computing ECDFs for within- and out-of-context per-read beta values [0.023s]
Reading BED file [0.018s]
Reading BAM file [0.007s]
Computing ECDFs for within- and out-of-context per-read beta values [0.015s]
Reading BED file [0.018s]
Reading BAM file [0.008s]
Preparing amplicon report [0.028s]
Reading BED file [0.018s]
Reading BAM file [0.007s]
Thresholding reads [0.000s]
Preparing amplicon report [0.028s]
Writing the report [0.001s]
Reading BED file [0.018s]
Reading BAM file [0.008s]
Thresholding reads [0.000s]
Preparing amplicon report [0.028s]
Reading BAM file [0.018s]
Thresholding reads [0.003s]
Preparing cytosine report [0.033s]
Reading BAM file [0.022s]
Thresholding reads [0.003s]
Preparing cytosine report [0.034s]
Writing the report [0.003s]
Reading BAM file [0.021s]
Thresholding reads [0.003s]
Preparing cytosine report [0.081s]
Reading BED file [0.016s]
Reading VCF file [1.024s]
Reading BAM file [0.007s]
Extracting base frequences [0.074s]
Reading VCF file [1.377s]
Reading BAM file [0.007s]
Extracting base frequencesAlready preprocessed VCF supplied as an input. Options &#x27;bed&#x27; and &#x27;zero.based.bed&#x27; will have no effect.
Reading BAM file [0.018s]
Thresholding reads [0.003s]
Extracting base frequences [0.183s]
Writing the report [0.011s]
Reading BED file [0.018s]
Reading VCF file [1.014s]
Reading BAM file [0.007s]
Extracting base frequences [0.075s]
Already preprocessed BAM supplied as an input. Options &#x27;min.mapq&#x27;, &#x27;min.baseq&#x27;, &#x27;skip.duplicates&#x27; and &#x27;nthreads&#x27; will have no effect.

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit



RUNIT TEST PROTOCOL -- Wed Feb  8 19:46:05 2023 
*********************************************** 
Number of test functions: 7 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
epialleleR RUnit Tests - 7 test functions, 0 errors, 0 failures
Number of test functions: 7 
Number of errors: 0 
Number of failures: 0 
&gt; 
&gt; proc.time()
   user  system elapsed 
 27.457   1.512  30.003 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">epialleleR.Rcheck/epialleleR-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>extractPatterns</TD><TD style="text-align: right;">1.399</TD><TD style="text-align: right;">0.140</TD><TD style="text-align: right;">1.542</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>generateBedEcdf</TD><TD style="text-align: right;">0.163</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.185</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>generateBedReport</TD><TD style="text-align: right;">0.267</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.261</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>generateCytosineReport</TD><TD style="text-align: right;">0.122</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.113</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>generateVcfReport</TD><TD style="text-align: right;">11.559</TD><TD style="text-align: right;"> 0.204</TD><TD style="text-align: right;">12.582</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>preprocessBam</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
