<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for TitanCNA on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><B>[<A href="../index.html#T">T</A>]</B></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-08 01:15:32 -0000 (Wed, 08 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4164</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for TitanCNA on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the TitanCNA package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/TitanCNA.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>2040</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">TitanCNA</A>&nbsp;1.37.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/TitanCNA">landing page</A>)</SPAN><BR>Gavin Ha<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-06&nbsp;00:12:45&nbsp;-0000&nbsp;(Mon,&nbsp;06&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/TitanCNA</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">5c82ab3</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:09:35&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: TitanCNA</TD></TR>
<TR><TD><B>Version</B>: 1.37.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:TitanCNA.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings TitanCNA_1.37.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-07 17:39:46 -0000 (Tue, 07 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-07 17:48:16 -0000 (Tue, 07 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 510.1 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: TitanCNA.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:TitanCNA.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings TitanCNA_1.37.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/TitanCNA.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘TitanCNA/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘TitanCNA’ version ‘1.37.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘TitanCNA’ can be installed ... WARNING
Found the following significant warnings:
  Warning: replacing previous import ‘GenomicRanges::shift’ by ‘data.table::shift’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘data.table::first’ by ‘dplyr::first’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘IRanges::desc’ by ‘dplyr::desc’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘IRanges::slice’ by ‘dplyr::slice’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘GenomeInfoDb::intersect’ by ‘dplyr::intersect’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘data.table::last’ by ‘dplyr::last’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘GenomicRanges::union’ by ‘dplyr::union’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘data.table::between’ by ‘dplyr::between’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘BiocGenerics::combine’ by ‘dplyr::combine’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘IRanges::collapse’ by ‘dplyr::collapse’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘GenomicRanges::setdiff’ by ‘dplyr::setdiff’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘dplyr::select’ by ‘VariantAnnotation::select’ when loading ‘TitanCNA’
See ‘/home/biocbuild/bbs-3.17-bioc/meat/TitanCNA.Rcheck/00install.out’ for details.
* used C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... NOTE
  installed size is  7.2Mb
  sub-directories of 1Mb or more:
    data      1.7Mb
    extdata   4.9Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
Killed
* checking Rd files ... NOTE
prepare_Rd: correctCN.Rd:51-53: Dropping empty section \details
* checking Rd metadata ... OK
* checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘list’
* checking for missing documentation entries ... WARNING
Undocumented code objects:
  ‘plotIdiogram.hg38’
All user-level objects in a package should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘TitanCNA-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: runEMclonalCN
&gt; ### Title: Function to run the Expectation Maximization Algorithm in
&gt; ###   &#x27;TitanCNA&#x27;.
&gt; ### Aliases: runEMclonalCN
&gt; ### Keywords: manip iteration htest models multivariate
&gt; 
&gt; ### ** Examples
&gt; 
&gt; message(&#x27;Running TITAN ...&#x27;)
Running TITAN ...
&gt; #### LOAD DATA ####
&gt; infile &lt;- system.file(&quot;extdata&quot;, &quot;test_alleleCounts_chr2.txt&quot;, 
+               package = &quot;TitanCNA&quot;)
&gt; data &lt;- loadAlleleCounts(infile)
titan: Loading data /home/biocbuild/bbs-3.17-bioc/R/library/TitanCNA/extdata/test_alleleCounts_chr2.txt
&gt; 
&gt; #### LOAD PARAMETERS ####
&gt; message(&#x27;titan: Loading default parameters&#x27;)
titan: Loading default parameters
&gt; numClusters &lt;- 2
&gt; params &lt;- loadDefaultParameters(copyNumber = 5, 
+               numberClonalClusters = numClusters, skew = 0.1)
&gt; 
&gt; #### READ COPY NUMBER FROM HMMCOPY FILE ####
&gt; message(&#x27;titan: Correcting GC content and mappability biases...&#x27;)
titan: Correcting GC content and mappability biases...
&gt; tumWig &lt;- system.file(&quot;extdata&quot;, &quot;test_tum_chr2.wig&quot;, package = &quot;TitanCNA&quot;)
&gt; normWig &lt;- system.file(&quot;extdata&quot;, &quot;test_norm_chr2.wig&quot;, package = &quot;TitanCNA&quot;)
&gt; gc &lt;- system.file(&quot;extdata&quot;, &quot;gc_chr2.wig&quot;, package = &quot;TitanCNA&quot;)
&gt; map &lt;- system.file(&quot;extdata&quot;, &quot;map_chr2.wig&quot;, package = &quot;TitanCNA&quot;)
&gt; cnData &lt;- correctReadDepth(tumWig, normWig, gc, map)
Reading GC and mappability files
Slurping: /home/biocbuild/bbs-3.17-bioc/R/library/TitanCNA/extdata/gc_chr2.wig
Parsing: fixedStep chrom=2 start=1 step=1000 span=1000
Sorting by decreasing chromosome size
Slurping: /home/biocbuild/bbs-3.17-bioc/R/library/TitanCNA/extdata/map_chr2.wig
Parsing: fixedStep chrom=2 start=1 step=1000 span=1000
Sorting by decreasing chromosome size
Loading tumour file:/home/biocbuild/bbs-3.17-bioc/R/library/TitanCNA/extdata/test_tum_chr2.wig
Slurping: /home/biocbuild/bbs-3.17-bioc/R/library/TitanCNA/extdata/test_tum_chr2.wig
Parsing: fixedStep chrom=2 start=1 step=1000 span=1000
Sorting by decreasing chromosome size
Loading normal file:/home/biocbuild/bbs-3.17-bioc/R/library/TitanCNA/extdata/test_norm_chr2.wig
Slurping: /home/biocbuild/bbs-3.17-bioc/R/library/TitanCNA/extdata/test_norm_chr2.wig
Parsing: fixedStep chrom=2 start=1 step=1000 span=1000
Sorting by decreasing chromosome size
Warning in .replace_seqlevels_style(x_seqlevels, value) :
  found more than one best sequence renaming map compatible with seqname style &quot;NCBI&quot; for this object, using the first one
Warning in .replace_seqlevels_style(x_seqlevels, value) :
  found more than one best sequence renaming map compatible with seqname style &quot;NCBI&quot; for this object, using the first one
Warning in .replace_seqlevels_style(x_seqlevels, value) :
  found more than one best sequence renaming map compatible with seqname style &quot;NCBI&quot; for this object, using the first one
Warning in .replace_seqlevels_style(x_seqlevels, value) :
  found more than one best sequence renaming map compatible with seqname style &quot;NCBI&quot; for this object, using the first one
Correcting Tumour
Applying filter on data...
Correcting for GC bias...
Correcting for mappability bias...
Warning in regularize.values(x, y, ties, missing(ties), na.rm = na.rm) :
  collapsing to unique &#x27;x&#x27; values
Correcting Normal
Applying filter on data...
Correcting for GC bias...
Correcting for mappability bias...
Warning in regularize.values(x, y, ties, missing(ties), na.rm = na.rm) :
  collapsing to unique &#x27;x&#x27; values
Normalizing Tumour by Normal
&gt; logR &lt;- getPositionOverlap(data$chr, data$posn, cnData)
&gt; data$logR &lt;- log(2^logR) #transform to natural log
&gt; 
&gt; #### FILTER DATA FOR DEPTH, MAPPABILITY, NA, etc ####
&gt; data &lt;- filterData(data, 1:24, minDepth = 10, maxDepth = 200, map = NULL)
Removed Chrs: 
&gt; 
&gt; #### EM (FWD-BACK) TO TRAIN PARAMETERS ####
&gt; #### Can use parallelization packages ####
&gt; K &lt;- length(params$genotypeParams$alphaKHyper)
&gt; params$genotypeParams$alphaKHyper &lt;- rep(500, K)
&gt; params$ploidyParams$phi_0 &lt;- 1.5
&gt; convergeParams &lt;- runEMclonalCN(data, params,
+                                 maxiter = 3, maxiterUpdate = 500, 
+                                 txnExpLen = 1e15, txnZstrength = 5e5, 
+                                 useOutlierState = FALSE, 
+                                 normalEstimateMethod = &quot;map&quot;, 
+                                 estimateS = TRUE, estimatePloidy = TRUE)
Killed
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘TitanCNA.Rnw’ using ‘UTF-8’... OK
 OK
* checking re-building of vignette outputs ... NOTE
Error(s) in re-building vignettes:
--- re-building ‘TitanCNA.Rnw’ using Sweave
Warning: replacing previous import ‘GenomicRanges::shift’ by ‘data.table::shift’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::first’ by ‘dplyr::first’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::desc’ by ‘dplyr::desc’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::slice’ by ‘dplyr::slice’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomeInfoDb::intersect’ by ‘dplyr::intersect’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::last’ by ‘dplyr::last’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomicRanges::union’ by ‘dplyr::union’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::between’ by ‘dplyr::between’ when loading ‘TitanCNA’
Warning: replacing previous import ‘BiocGenerics::combine’ by ‘dplyr::combine’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::collapse’ by ‘dplyr::collapse’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomicRanges::setdiff’ by ‘dplyr::setdiff’ when loading ‘TitanCNA’
Warning: replacing previous import ‘dplyr::select’ by ‘VariantAnnotation::select’ when loading ‘TitanCNA’
titan: Loading data /home/biocbuild/bbs-3.17-bioc/R/library/TitanCNA/extdata/test_alleleCounts_chr2.txt
Reading GC and mappability files
Slurping: /home/biocbuild/bbs-3.17-bioc/R/library/TitanCNA/extdata/gc_chr2.wig
Parsing: fixedStep chrom=2 start=1 step=1000 span=1000
Sorting by decreasing chromosome size
Slurping: /home/biocbuild/bbs-3.17-bioc/R/library/TitanCNA/extdata/map_chr2.wig
Parsing: fixedStep chrom=2 start=1 step=1000 span=1000
Sorting by decreasing chromosome size
Loading tumour file:/home/biocbuild/bbs-3.17-bioc/R/library/TitanCNA/extdata/test_tum_chr2.wig
Slurping: /home/biocbuild/bbs-3.17-bioc/R/library/TitanCNA/extdata/test_tum_chr2.wig
Parsing: fixedStep chrom=2 start=1 step=1000 span=1000
Sorting by decreasing chromosome size
Loading normal file:/home/biocbuild/bbs-3.17-bioc/R/library/TitanCNA/extdata/test_norm_chr2.wig
Slurping: /home/biocbuild/bbs-3.17-bioc/R/library/TitanCNA/extdata/test_norm_chr2.wig
Parsing: fixedStep chrom=2 start=1 step=1000 span=1000
Sorting by decreasing chromosome size
Warning in .replace_seqlevels_style(x_seqlevels, value) :
  found more than one best sequence renaming map compatible with seqname style &quot;NCBI&quot; for this object, using the first one
Warning in .replace_seqlevels_style(x_seqlevels, value) :
  found more than one best sequence renaming map compatible with seqname style &quot;NCBI&quot; for this object, using the first one
Warning in .replace_seqlevels_style(x_seqlevels, value) :
  found more than one best sequence renaming map compatible with seqname style &quot;NCBI&quot; for this object, using the first one
Warning in .replace_seqlevels_style(x_seqlevels, value) :
  found more than one best sequence renaming map compatible with seqname style &quot;NCBI&quot; for this object, using the first one
Correcting Tumour
Applying filter on data...
Correcting for GC bias...
Correcting for mappability bias...
Warning in regularize.values(x, y, ties, missing(ties), na.rm = na.rm) :
  collapsing to unique &#x27;x&#x27; values
Correcting Normal
Applying filter on data...
Correcting for GC bias...
Correcting for mappability bias...
Warning in regularize.values(x, y, ties, missing(ties), na.rm = na.rm) :
  collapsing to unique &#x27;x&#x27; values
Normalizing Tumour by Normal
Killed

* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 2 WARNINGs, 6 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/TitanCNA.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">TitanCNA.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL TitanCNA
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘TitanCNA’ ...
** using staged installation
** libs
using C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG   -I/usr/local/include    -fPIC  -g -O2  -Wall -c fwd_backC_clonalCN.c -o fwd_backC_clonalCN.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG   -I/usr/local/include    -fPIC  -g -O2  -Wall -c getPositionOverlapC.c -o getPositionOverlapC.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG   -I/usr/local/include    -fPIC  -g -O2  -Wall -c register.c -o register.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG   -I/usr/local/include    -fPIC  -g -O2  -Wall -c viterbiC_clonalCN.c -o viterbiC_clonalCN.o
gcc -shared -L/home/biocbuild/bbs-3.17-bioc/R/lib -L/usr/local/lib -o TitanCNA.so fwd_backC_clonalCN.o getPositionOverlapC.o register.o viterbiC_clonalCN.o -L/home/biocbuild/bbs-3.17-bioc/R/lib -lR
installing to /home/biocbuild/bbs-3.17-bioc/R/library/00LOCK-TitanCNA/00new/TitanCNA/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Warning: replacing previous import ‘GenomicRanges::shift’ by ‘data.table::shift’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::first’ by ‘dplyr::first’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::desc’ by ‘dplyr::desc’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::slice’ by ‘dplyr::slice’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomeInfoDb::intersect’ by ‘dplyr::intersect’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::last’ by ‘dplyr::last’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomicRanges::union’ by ‘dplyr::union’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::between’ by ‘dplyr::between’ when loading ‘TitanCNA’
Warning: replacing previous import ‘BiocGenerics::combine’ by ‘dplyr::combine’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::collapse’ by ‘dplyr::collapse’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomicRanges::setdiff’ by ‘dplyr::setdiff’ when loading ‘TitanCNA’
Warning: replacing previous import ‘dplyr::select’ by ‘VariantAnnotation::select’ when loading ‘TitanCNA’
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
Warning: replacing previous import ‘GenomicRanges::shift’ by ‘data.table::shift’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::first’ by ‘dplyr::first’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::desc’ by ‘dplyr::desc’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::slice’ by ‘dplyr::slice’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomeInfoDb::intersect’ by ‘dplyr::intersect’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::last’ by ‘dplyr::last’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomicRanges::union’ by ‘dplyr::union’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::between’ by ‘dplyr::between’ when loading ‘TitanCNA’
Warning: replacing previous import ‘BiocGenerics::combine’ by ‘dplyr::combine’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::collapse’ by ‘dplyr::collapse’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomicRanges::setdiff’ by ‘dplyr::setdiff’ when loading ‘TitanCNA’
Warning: replacing previous import ‘dplyr::select’ by ‘VariantAnnotation::select’ when loading ‘TitanCNA’
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
Warning: replacing previous import ‘GenomicRanges::shift’ by ‘data.table::shift’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::first’ by ‘dplyr::first’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::desc’ by ‘dplyr::desc’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::slice’ by ‘dplyr::slice’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomeInfoDb::intersect’ by ‘dplyr::intersect’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::last’ by ‘dplyr::last’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomicRanges::union’ by ‘dplyr::union’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::between’ by ‘dplyr::between’ when loading ‘TitanCNA’
Warning: replacing previous import ‘BiocGenerics::combine’ by ‘dplyr::combine’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::collapse’ by ‘dplyr::collapse’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomicRanges::setdiff’ by ‘dplyr::setdiff’ when loading ‘TitanCNA’
Warning: replacing previous import ‘dplyr::select’ by ‘VariantAnnotation::select’ when loading ‘TitanCNA’
** testing if installed package keeps a record of temporary installation path
* DONE (TitanCNA)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<HR>
<H3>Example timings</H3>
<P><SPAN class="filename">TitanCNA.Rcheck/TitanCNA-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>TitanCNA-output</TD><TD style="text-align: right;">1.240</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">1.262</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TitanCNA-package</TD><TD style="text-align: right;">37.275</TD><TD style="text-align: right;"> 0.219</TD><TD style="text-align: right;">37.827</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TitanCNA-plotting</TD><TD style="text-align: right;">8.071</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">8.129</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>computeSDbwIndex</TD><TD style="text-align: right;">6.393</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">6.403</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>correctCN</TD><TD style="text-align: right;">0.959</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.963</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>correctReadDepth</TD><TD style="text-align: right;">8.468</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">8.516</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filterData</TD><TD style="text-align: right;">10.253</TD><TD style="text-align: right;"> 0.012</TD><TD style="text-align: right;">10.271</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getPositionOverlap</TD><TD style="text-align: right;">10.085</TD><TD style="text-align: right;"> 0.008</TD><TD style="text-align: right;">10.098</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>haplotype</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loadAlleleCounts</TD><TD style="text-align: right;">1.165</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">1.197</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loadDefaultParameters</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
