<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for HTSeqGenie on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><B>[<A href="..#H">H</A>]</B></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-16 06:06:56 -0000 (Tue, 16 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4787</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for HTSeqGenie on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the HTSeqGenie package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/HTSeqGenie.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>938</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">HTSeqGenie</A>&nbsp;4.31.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/HTSeqGenie">landing page</A>)</SPAN><BR>Jens Reeder<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-13&nbsp;23:12:22&nbsp;-0000&nbsp;(Sat,&nbsp;13&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/HTSeqGenie</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">6787c4b</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:24:28&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: HTSeqGenie</TD></TR>
<TR><TD><B>Version</B>: 4.31.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:HTSeqGenie.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings HTSeqGenie_4.31.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-15 05:36:56 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-15 05:56:10 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 1154.6 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: HTSeqGenie.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:HTSeqGenie.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings HTSeqGenie_4.31.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/HTSeqGenie.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘HTSeqGenie/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘HTSeqGenie’ version ‘4.31.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .BBSoptions
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘HTSeqGenie’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... NOTE
Malformed Title field: should not end in a period.
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking startup messages can be suppressed ... OK
* checking dependencies in R code ... NOTE
Unexported objects imported by &#x27;:::&#x27; calls:
  ‘ShortRead:::.set_omp_threads’ ‘gmapR:::..gsnap’ ‘gmapR:::GeneGenome’
  ‘gmapR:::checkPackageInstalled’ ‘gmapR:::getDefaultGmapGenomePath’
  ‘gmapR:::getGeneRoi’ ‘gmapR:::subsetRegion’ ‘parallel:::mckill’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... NOTE
Foreign function call to a different package:
  .Call(ShortRead:::.set_omp_threads, ...)
See chapter ‘System and foreign language interfaces’ in the ‘Writing R
Extensions’ manual.
* checking R code for possible problems ... NOTE
.makePreprocessAlignPlots: no visible binding for global variable
  ‘data’
.plot.in.out.min.max.readlength: no visible global function definition
  for ‘median’
FastQStreamer.init: no visible binding for &#x27;&lt;&lt;-&#x27; assignment to
  ‘FastQStreamer.lfqs’
FastQStreamer.init: no visible binding for &#x27;&lt;&lt;-&#x27; assignment to
  ‘FastQStreamer.chunkid’
FastQStreamer.init: no visible binding for &#x27;&lt;&lt;-&#x27; assignment to
  ‘FastQStreamer.subsampling_filter’
FastQStreamer.init: no visible binding for &#x27;&lt;&lt;-&#x27; assignment to
  ‘FastQStreamer.max_nbchunks’
addHandler: no visible binding for &#x27;&lt;&lt;-&#x27; assignment to
  ‘logging.handlers’
addHandler: no visible binding for global variable ‘logging.handlers’
addHandler: no visible binding for &#x27;&lt;&lt;-&#x27; assignment to ‘logging.file’
buildAnyFastaGenome: no visible global function definition for ‘getSeq’
buildGenomicFeaturesFromTxDb: no visible global function definition for
  ‘gaps’
buildGenomicFeaturesFromTxDb: no visible global function definition for
  ‘ranges’
buildTP53FastaGenome: no visible global function definition for
  ‘getSeq’
buildTallyParam: no visible global function definition for ‘seqinfo’
computeCoverage: no visible global function definition for ‘resize’
computeCoverage: no visible global function definition for ‘coverage’
estimateCutoffs : &lt;anonymous&gt;: no visible binding for global variable
  ‘quantile’
getGenomeSegments: no visible global function definition for ‘seqinfo’
hashVariants: no visible global function definition for ‘ranges’
isAboveQualityThresh: no visible global function definition for ‘Views’
isAboveQualityThresh: no visible global function definition for
  ‘viewMeans’
listIterator.init: no visible binding for &#x27;&lt;&lt;-&#x27; assignment to
  ‘listIterator.n’
listIterator.init: no visible binding for &#x27;&lt;&lt;-&#x27; assignment to
  ‘listIterator.x’
logReset: no visible binding for &#x27;&lt;&lt;-&#x27; assignment to ‘logging.handlers’
logReset: no visible binding for &#x27;&lt;&lt;-&#x27; assignment to ‘logging.loglevel’
logReset: no visible binding for &#x27;&lt;&lt;-&#x27; assignment to ‘logging.file’
loglevel: no visible binding for global variable ‘logging.loglevel’
loglevel: no visible binding for global variable ‘logging.handlers’
processChunks : tracefun: no visible binding for global variable
  ‘sjobs’
processChunks : tracefun: no visible binding for global variable
  ‘chunkid’
setLevel: no visible binding for &#x27;&lt;&lt;-&#x27; assignment to ‘logging.loglevel’
statCountFeatures: no visible global function definition for ‘quantile’
truncateReads: no visible global function definition for ‘subseq’
vcfStat: no visible global function definition for ‘quantile’
writeToFile: no visible binding for global variable ‘logging.file’
Undefined global functions or variables:
  Views chunkid coverage data gaps getSeq logging.file logging.handlers
  logging.loglevel median quantile ranges resize seqinfo sjobs subseq
  viewMeans
Consider adding
  importFrom(&quot;stats&quot;, &quot;median&quot;, &quot;quantile&quot;)
  importFrom(&quot;utils&quot;, &quot;data&quot;)
to your NAMESPACE file.
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘doRUnit.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘HTSeqGenie.Rnw’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 5 NOTEs
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/HTSeqGenie.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">HTSeqGenie.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD INSTALL HTSeqGenie
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0-bin/site-library’
* installing *source* package ‘HTSeqGenie’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (HTSeqGenie)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">HTSeqGenie.Rcheck/tests/doRUnit.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;HTSeqGenie&quot;)
Loading required package: gmapR
Loading required package: GenomeInfoDb
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following object is masked from &#x27;package:utils&#x27;:

    findMatches

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomicRanges
Loading required package: Rsamtools
Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: ShortRead
Loading required package: BiocParallel
Loading required package: GenomicAlignments
Loading required package: SummarizedExperiment
Loading required package: MatrixGenerics
Loading required package: matrixStats

Attaching package: &#x27;MatrixGenerics&#x27;

The following objects are masked from &#x27;package:matrixStats&#x27;:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars

Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


Attaching package: &#x27;Biobase&#x27;

The following object is masked from &#x27;package:MatrixGenerics&#x27;:

    rowMedians

The following objects are masked from &#x27;package:matrixStats&#x27;:

    anyMissing, rowMedians

Loading required package: VariantAnnotation

Attaching package: &#x27;VariantAnnotation&#x27;

The following object is masked from &#x27;package:base&#x27;:

    tabulate

&gt; 
&gt; source(getPackageFile(&quot;unitTests/runTests.R&quot;))
Loading required package: GenomicFeatures
Loading required package: AnnotationDbi
did not source anything in dirname= ./R 


Executing test function test.alignReads  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:44:43.269655 INFO::preprocessReads.R/preprocessReads: starting...
2023-05-15 05:44:43.281464 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_1.fastq
2023-05-15 05:44:43.287579 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_2.fastq
2023-05-15 05:44:43.290749 DEBUG::tools.R/processChunks: starting...
2023-05-15 05:44:46.98935 DEBUG::tools.R/processChunks: waiting for chunkid=[  ] ...
2023-05-15 05:44:46.991366 DEBUG::tools.R/processChunks: starting chunkid= 1 ; see logfile= /tmp/Rtmpt80l8W/test.alignReads.2838c94cd0f171/chunks/chunk_000001/logs/progress.log
2023-05-15 05:44:49.887489 DEBUG::tools.R/processChunks: done with chunkid= 1 ; elapsed.time= 0.048 minutes
2023-05-15 05:44:49.889503 DEBUG::tools.R/processChunks: starting chunkid= 2 ; see logfile= /tmp/Rtmpt80l8W/test.alignReads.2838c94cd0f171/chunks/chunk_000002/logs/progress.log
2023-05-15 05:44:52.78387 DEBUG::tools.R/processChunks: done with chunkid= 2 ; elapsed.time= 0.048 minutes
2023-05-15 05:44:52.785945 DEBUG::tools.R/processChunks: starting chunkid= 3 ; see logfile= /tmp/Rtmpt80l8W/test.alignReads.2838c94cd0f171/chunks/chunk_000003/logs/progress.log
2023-05-15 05:44:55.704869 DEBUG::tools.R/processChunks: done with chunkid= 3 ; elapsed.time= 0.049 minutes
2023-05-15 05:44:55.748259 DEBUG::tools.R/processChunks: done
2023-05-15 05:44:55.753381 INFO::detectAdapterContam.R/mergeDetectAdapterContam: merging file= adapter_contaminated_1 ...
2023-05-15 05:44:55.756462 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.alignReads.2838c94cd0f171/results/test_pe.adapter_contaminated_1.RData
2023-05-15 05:44:55.760297 INFO::detectAdapterContam.R/mergeDetectAdapterContam: merging file= adapter_contaminated_2 ...
2023-05-15 05:44:55.76272 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.alignReads.2838c94cd0f171/results/test_pe.adapter_contaminated_2.RData
2023-05-15 05:44:55.775546 INFO::preprocessReads.R/mergeSummaryPreprocess: total_reads=26 highqual_reads=25 adapter_contam=1 read_length=75 rRNA_contam_reads=0 processed_reads=25 input_min_read_length=75 input_max_read_length=75 processed_min_read_length=75 processed_max_read_length=75
2023-05-15 05:44:55.778681 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.alignReads.2838c94cd0f171/results/test_pe.summary_preprocess.tab
2023-05-15 05:44:55.783429 INFO::preprocessReads.R/mergePreprocessedReads: merging file= /tmp/Rtmpt80l8W/test.alignReads.2838c94cd0f171/bams/processed.aligner_input_1.fastq ...
2023-05-15 05:44:55.789746 INFO::preprocessReads.R/mergePreprocessedReads: merging file= /tmp/Rtmpt80l8W/test.alignReads.2838c94cd0f171/bams/processed.aligner_input_2.fastq ...
2023-05-15 05:44:55.794725 INFO::preprocessReads.R/buildShortReadReports: generating report_dir= /tmp/Rtmpt80l8W/test.alignReads.2838c94cd0f171/reports/shortReadReport_1 ...
2023-05-15 05:44:57.746526 INFO::preprocessReads.R/buildShortReadReports: generating report_dir= /tmp/Rtmpt80l8W/test.alignReads.2838c94cd0f171/reports/shortReadReport_2 ...
2023-05-15 05:45:00.843553 INFO::preprocessReads.R/preprocessReads: done
2023-05-15 05:45:00.899337 INFO::alignReads.R/alignReads: starting alignment...
2023-05-15 05:45:00.907233 DEBUG::tools.R/processChunks: starting...
2023-05-15 05:45:04.314602 DEBUG::tools.R/processChunks: waiting for chunkid=[  ] ...
2023-05-15 05:45:04.316726 DEBUG::tools.R/processChunks: starting chunkid= 1 ; see logfile= /tmp/Rtmpt80l8W/test.alignReads.2838c94cd0f171/chunks/chunk_000001/logs/progress.log
2023-05-15 05:45:07.43999 DEBUG::tools.R/processChunks: done with chunkid= 1 ; elapsed.time= 0.052 minutes
2023-05-15 05:45:07.442586 DEBUG::tools.R/processChunks: starting chunkid= 2 ; see logfile= /tmp/Rtmpt80l8W/test.alignReads.2838c94cd0f171/chunks/chunk_000002/logs/progress.log
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPEsInManyTxsOfMultiTxGene:1:1:1:16#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPE1MicrobialPE2MultimappingIntergenic:1:1:1:19#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPEsInOneTxOfMultiTxGene:1:1:1:15#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPE1UniqueUtrPE2MultimappingIntergenic:1:1:1:18#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
2023-05-15 05:45:10.263182 DEBUG::tools.R/processChunks: done with chunkid= 2 ; elapsed.time= 0.047 minutes
2023-05-15 05:45:10.265707 DEBUG::tools.R/processChunks: starting chunkid= 3 ; see logfile= /tmp/Rtmpt80l8W/test.alignReads.2838c94cd0f171/chunks/chunk_000003/logs/progress.log
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPE1GeneFusionMidExonDiffChrPE2MultimappingIntergenic:1:1:1:25#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
2023-05-15 05:45:13.072689 DEBUG::tools.R/processChunks: done with chunkid= 3 ; elapsed.time= 0.047 minutes
2023-05-15 05:45:13.076329 DEBUG::tools.R/processChunks: done
2023-05-15 05:45:13.078645 INFO::alignReads.R/mergeBAMsAcrossDirs: starting...
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualAdapterContamIn3PrimeEnd:1:1:1:7#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPEsInManyTxsOfMultiTxGene:1:1:1:16#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPE1MicrobialPE2MultimappingIntergenic:1:1:1:19#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPE1GeneFusionMidExonDiffChrPE2MultimappingIntergenic:1:1:1:25#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPE1UniqueUtrPE2MultimappingIntergenic:1:1:1:18#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPEsInOneTxOfMultiTxGene:1:1:1:15#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
2023-05-15 05:45:13.421092 INFO::alignReads.R/mergeBAMsAcrossDirs: done
2023-05-15 05:45:13.434201 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.alignReads.2838c94cd0f171/results/test_pe.summary_alignment.tab
2023-05-15 05:45:13.446368 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.alignReads.2838c94cd0f171/results/test_pe.summary_analyzed_bamstats.tab
2023-05-15 05:45:13.449083 INFO::reportPipelineQA.R/writePreprocessAlignReport: creating joint preprocess-alignment report...
2023-05-15 05:45:13.766778 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.alignReads.2838c94cd0f171/results/test_pe.summary_target_lengths.tab
2023-05-15 05:45:13.830209 INFO::reportPipelineQA.R/writePreprocessAlignReport: done
2023-05-15 05:45:13.83161 INFO::alignReads.R/alignReads: done
 done successfully.



Executing test function test.alignReads.sparsechunks  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:45:14.140013 INFO::preprocessReads.R/preprocessReads: starting...
2023-05-15 05:45:14.144033 INFO::io.R/FastQStreamer.init: counting number of reads in file= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_1.fastq
2023-05-15 05:45:14.229191 INFO::io.R/FastQStreamer.init: subsampling_filter set (subsampled reads= 2 , totnbreads= 26 )
2023-05-15 05:45:14.235852 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_1.fastq
2023-05-15 05:45:14.240137 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_2.fastq
2023-05-15 05:45:14.242824 DEBUG::tools.R/processChunks: starting...
2023-05-15 05:45:18.016831 DEBUG::tools.R/processChunks: waiting for chunkid=[  ] ...
2023-05-15 05:45:18.018903 DEBUG::tools.R/processChunks: starting chunkid= 1 ; see logfile= /tmp/Rtmpt80l8W/test.alignReads.sparsechunks.2838c9314f24bc/chunks/chunk_000001/logs/progress.log
2023-05-15 05:45:20.878566 DEBUG::tools.R/processChunks: done with chunkid= 1 ; elapsed.time= 0.048 minutes
2023-05-15 05:45:20.880876 DEBUG::tools.R/processChunks: starting chunkid= 2 ; see logfile= /tmp/Rtmpt80l8W/test.alignReads.sparsechunks.2838c9314f24bc/chunks/chunk_000002/logs/progress.log
2023-05-15 05:45:23.740961 DEBUG::tools.R/processChunks: done with chunkid= 2 ; elapsed.time= 0.048 minutes
2023-05-15 05:45:23.742984 DEBUG::tools.R/processChunks: starting chunkid= 3 ; see logfile= /tmp/Rtmpt80l8W/test.alignReads.sparsechunks.2838c9314f24bc/chunks/chunk_000003/logs/progress.log
2023-05-15 05:45:26.701001 DEBUG::tools.R/processChunks: done with chunkid= 3 ; elapsed.time= 0.049 minutes
2023-05-15 05:45:26.702986 DEBUG::tools.R/processChunks: starting chunkid= 4 ; see logfile= /tmp/Rtmpt80l8W/test.alignReads.sparsechunks.2838c9314f24bc/chunks/chunk_000004/logs/progress.log
2023-05-15 05:45:29.540973 DEBUG::tools.R/processChunks: done with chunkid= 4 ; elapsed.time= 0.047 minutes
2023-05-15 05:45:29.582069 DEBUG::tools.R/processChunks: done
2023-05-15 05:45:29.58657 INFO::detectAdapterContam.R/mergeDetectAdapterContam: merging file= adapter_contaminated_1 ...
2023-05-15 05:45:29.589782 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.alignReads.sparsechunks.2838c9314f24bc/results/test_pe.adapter_contaminated_1.RData
2023-05-15 05:45:29.59386 INFO::detectAdapterContam.R/mergeDetectAdapterContam: merging file= adapter_contaminated_2 ...
2023-05-15 05:45:29.596372 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.alignReads.sparsechunks.2838c9314f24bc/results/test_pe.adapter_contaminated_2.RData
2023-05-15 05:45:29.610268 INFO::preprocessReads.R/mergeSummaryPreprocess: total_reads=2 highqual_reads=1 adapter_contam=0 read_length=75 rRNA_contam_reads=0 processed_reads=1 input_min_read_length=75 input_max_read_length=75 processed_min_read_length=75 processed_max_read_length=75
2023-05-15 05:45:29.613389 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.alignReads.sparsechunks.2838c9314f24bc/results/test_pe.summary_preprocess.tab
2023-05-15 05:45:29.618304 INFO::preprocessReads.R/mergePreprocessedReads: merging file= /tmp/Rtmpt80l8W/test.alignReads.sparsechunks.2838c9314f24bc/bams/processed.aligner_input_1.fastq ...
2023-05-15 05:45:29.624582 INFO::preprocessReads.R/mergePreprocessedReads: merging file= /tmp/Rtmpt80l8W/test.alignReads.sparsechunks.2838c9314f24bc/bams/processed.aligner_input_2.fastq ...
2023-05-15 05:45:29.629248 INFO::preprocessReads.R/buildShortReadReports: generating report_dir= /tmp/Rtmpt80l8W/test.alignReads.sparsechunks.2838c9314f24bc/reports/shortReadReport_1 ...
2023-05-15 05:45:31.483813 INFO::preprocessReads.R/buildShortReadReports: generating report_dir= /tmp/Rtmpt80l8W/test.alignReads.sparsechunks.2838c9314f24bc/reports/shortReadReport_2 ...
2023-05-15 05:45:34.804878 INFO::preprocessReads.R/preprocessReads: done
 done successfully.



Executing test function test.alignReadsOneSingleEnd  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:45:35.017393 INFO::alignReads.R/alignReadsChunk: running gsnap...
2023-05-15 05:45:35.02327 INFO::alignReads.R/alignReadsChunk: calling gsnap (from gmapR) with args= -D /home/biocbuild/.local/share/gmap -t 1 -d TP53_demo_3.2.2 --quality-protocol=illumina  -A sam --read-group-id=test_se -m 0 --split-output /tmp/Rtmpt80l8W/test.alignReadsOneSingleEnd.2838c95e1d84b3/bams/test.alignReads /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_1.fastq 2&gt;&amp;1
2023-05-15 05:45:35.239519 INFO::alignReads.R/createSummaryAlignment: counting unique bam reads...
2023-05-15 05:45:35.363652 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.alignReadsOneSingleEnd.2838c95e1d84b3/results/test.alignReads.summary_alignment.tab
2023-05-15 05:45:35.416909 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.alignReadsOneSingleEnd.2838c95e1d84b3/results/test.alignReads.summary_analyzed_bamstats.tab
2023-05-15 05:45:35.418955 INFO::alignReads.R/alignReadsChunk: done
 done successfully.



Executing test function test.annotateVariants  ... Timing stopped at: 0.002 0.001 0.002
Error in DEACTIVATED(&quot;Skipped annotateVariants() test&quot;) : 
  Skipped annotateVariants() test
In addition: There were 11 warnings (use warnings() to see them)
 done successfully.



Executing test function test.callVariantsVariantTools.genotype  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:45:35.832052 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.genotype.2838c9130ba5c0/results/test_pe.coverage.RData
2023-05-15 05:45:35.834542 INFO::coverage.R/saveCoverage: saving file= /tmp/Rtmpt80l8W/test.genotype.2838c9130ba5c0/results/test_pe.coverage.bw
2023-05-15 05:45:36.009938 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.genotype.2838c9130ba5c0/results/test_pe.summary_coverage.tab
2023-05-15 05:45:36.012222 INFO::analyzeVariants.R/wrap.callVariants: Tallying variants...
2023-05-15 05:45:47.348171 INFO::analyzeVariants.R/wrap.callVariants: calling variants...
2023-05-15 05:45:47.454668 INFO::analyzeVariants.R/wrap.postFilerVariants: post filtering variants...
2023-05-15 05:45:47.476135 INFO::analyzeVariants.R/wrap.callVariants: Saving GRanges of raw and filtered variants...
2023-05-15 05:45:47.47807 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.genotype.2838c9130ba5c0/results/test_pe.raw_variants.RData
2023-05-15 05:45:47.48142 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.genotype.2838c9130ba5c0/results/test_pe.filtered_variants.RData
2023-05-15 05:45:47.483296 INFO::analyzeVariants.R/wrap.callVariants: ...done
2023-05-15 05:45:47.484909 INFO::analyzeVariants.R/writeVCF: writing vcf file...
2023-05-15 05:45:48.004159 INFO::analyzeVariants.R/writeVCF: ...done
2023-05-15 05:45:48.005618 INFO::analyzeVariants.R/.callGenotypes: calling genotypes...
2023-05-15 05:46:51.979798 INFO::analyzeVariants.R/writeVCF: writing vcf file...
2023-05-15 05:46:52.560016 INFO::analyzeVariants.R/writeVCF: ...done
2023-05-15 05:46:52.561314 INFO::analyzeVariants.R/.callGenotypes: done
[W::bcf_hdr_check_sanity] PL should be declared as Number=G
 done successfully.



Executing test function test.wrap.callVariants  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:46:53.132743 INFO::analyzeVariants.R/wrap.callVariants: Tallying variants...
2023-05-15 05:47:04.033736 INFO::analyzeVariants.R/wrap.callVariants: calling variants...
2023-05-15 05:47:04.128625 INFO::analyzeVariants.R/wrap.postFilerVariants: post filtering variants...
2023-05-15 05:47:04.149651 INFO::analyzeVariants.R/wrap.callVariants: Saving GRanges of raw and filtered variants...
2023-05-15 05:47:04.151366 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.wrap.callVariants.2838c927d0a06a/results/test_pe.raw_variants.RData
2023-05-15 05:47:04.154543 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.wrap.callVariants.2838c927d0a06a/results/test_pe.filtered_variants.RData
2023-05-15 05:47:04.156249 INFO::analyzeVariants.R/wrap.callVariants: ...done
 done successfully.



Executing test function test.wrap.callVariants.filters  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:47:04.325469 INFO::analyzeVariants.R/wrap.callVariants: Tallying variants...
2023-05-15 05:47:15.132434 INFO::analyzeVariants.R/wrap.callVariants: calling variants...
2023-05-15 05:47:15.202609 INFO::analyzeVariants.R/wrap.postFilerVariants: post filtering variants...
2023-05-15 05:47:15.22332 INFO::analyzeVariants.R/wrap.callVariants: Saving GRanges of raw and filtered variants...
2023-05-15 05:47:15.22501 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.wrap.callVariants.filters.2838c95da09434/results/test_pe.raw_variants.RData
2023-05-15 05:47:15.228106 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.wrap.callVariants.filters.2838c95da09434/results/test_pe.filtered_variants.RData
2023-05-15 05:47:15.229739 INFO::analyzeVariants.R/wrap.callVariants: ...done
2023-05-15 05:47:15.231251 INFO::analyzeVariants.R/wrap.callVariants: Tallying variants...
2023-05-15 05:47:26.061962 INFO::analyzeVariants.R/wrap.callVariants: calling variants...
2023-05-15 05:47:26.111809 INFO::analyzeVariants.R/wrap.postFilerVariants: post filtering variants...
2023-05-15 05:47:26.133959 INFO::analyzeVariants.R/wrap.callVariants: Saving GRanges of raw and filtered variants...
2023-05-15 05:47:26.135778 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.wrap.callVariants.filters.2838c95da09434/results/test_pe.raw_variants.RData
2023-05-15 05:47:26.139192 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.wrap.callVariants.filters.2838c95da09434/results/test_pe.filtered_variants.RData
2023-05-15 05:47:26.141972 INFO::analyzeVariants.R/wrap.callVariants: ...done
 done successfully.



Executing test function test.wrap.callVariants.which  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:47:26.33623 INFO::analyzeVariants.R/wrap.callVariants: Tallying variants...
2023-05-15 05:47:26.338319 INFO::analyzeVariants.R/buildTallyParam: restricting variant calls using &#x27;which&#x27;
2023-05-15 05:47:28.934413 INFO::analyzeVariants.R/wrap.callVariants: calling variants...
2023-05-15 05:47:29.032507 INFO::analyzeVariants.R/wrap.postFilerVariants: post filtering variants...
2023-05-15 05:47:29.054312 INFO::analyzeVariants.R/wrap.callVariants: Saving GRanges of raw and filtered variants...
2023-05-15 05:47:29.056092 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.wrap.callVariants.which.2838c9262d90b9/results/test_pe.raw_variants.RData
2023-05-15 05:47:29.058339 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.wrap.callVariants.which.2838c9262d90b9/results/test_pe.filtered_variants.RData
2023-05-15 05:47:29.060093 INFO::analyzeVariants.R/wrap.callVariants: ...done
 done successfully.



Executing test function test.writeVCF.NULL  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:47:29.240127 INFO::analyzeVariants.R/writeVCF: writing vcf file...
2023-05-15 05:47:29.24187 INFO::analyzeVariants.R/writeVCF: ...done
 done successfully.



Executing test function test.writeVCF.vcfStat  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:47:29.4824 INFO::analyzeVariants.R/writeVCF: writing vcf file...
2023-05-15 05:47:29.781687 INFO::analyzeVariants.R/writeVCF: ...done
 done successfully.



Executing test function test.isFirstFragment  ...  done successfully.



Executing test function test.buildCountsGRangesList  ...   403 genes were dropped because they have exons located on both strands
  of the same reference sequence or on more than one reference sequence,
  so cannot be represented by a single genomic range.
  Use &#x27;single.strand.genes.only=FALSE&#x27; to get all the genes in a
  GRangesList object, or use suppressMessages() to suppress this message.
 done successfully.



Executing test function test.generateSingleGeneDERs  ...  done successfully.



Executing test function test.computeCoverage  ...  done successfully.



Executing test function test.isSparse  ...  done successfully.



Executing test function test.mergeCoverage  ... 2023-05-15 05:48:41.611933 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/pgymntwb/merged/results/bla.coverage.RData
2023-05-15 05:48:41.615522 INFO::coverage.R/saveCoverage: saving file= /tmp/Rtmpt80l8W/pgymntwb/merged/results/bla.coverage.bw
2023-05-15 05:48:41.733643 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/pgymntwb/merged/results/bla.summary_coverage.tab
 done successfully.



Executing test function test.mergeCoverage.sparse  ... 2023-05-15 05:48:43.066813 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/ukcdtqib/merged/results/bla.coverage.RData
2023-05-15 05:48:43.069375 INFO::coverage.R/saveCoverage: saving file= /tmp/Rtmpt80l8W/ukcdtqib/merged/results/bla.coverage.bw
2023-05-15 05:48:43.085337 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/ukcdtqib/merged/results/bla.summary_coverage.tab
 done successfully.



Executing test function test.checkConfig.analyzeVariants  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
 done successfully.



Executing test function test.findTemplate  ...  done successfully.



Executing test function test.checkConfig  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
 done successfully.



Executing test function test.checkConfig.alignReads  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
 done successfully.



Executing test function test.getConfig  ...  done successfully.



Executing test function test.loadConfig  ...  done successfully.



Executing test function test.parseDCF  ...  done successfully.



Executing test function test.updateConfig  ...  done successfully.



Executing test function test.getAdapterSeqs  ...  done successfully.



Executing test function test.isAdapter  ...  done successfully.



Executing test function test.isAdapter3.primeEnd  ...  done successfully.



Executing test function test.detectRRNA  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:48:44.880477 INFO::detectRRNA.R/detectRRNA: searching for rRNA contamination...
2023-05-15 05:48:44.882198 INFO::io.R/writeFastQFiles: writing filename= /tmp/Rtmpt80l8W/test.detectRRNA.2838c9556754fa/bams/rRNA_contam/input1.fastq
2023-05-15 05:48:44.887244 INFO::alignReads.R/alignReadsChunk: calling gsnap (from gmapR) with args= -t 0 -D /home/biocbuild/.local/share/gmap -n 1 -d TP53_demo_3.2.2 -A sam -B 2 --pairmax-rna=200000 --split-output /tmp/Rtmpt80l8W/test.detectRRNA.2838c9556754fa/bams/rRNA_contam/test_se /tmp/Rtmpt80l8W/test.detectRRNA.2838c9556754fa/bams/rRNA_contam/input1.fastq 2&gt;&amp;1
2023-05-15 05:48:45.039757 INFO::detectRRNA.R/detectRRNA: contaminated fraction= 1
2023-05-15 05:48:45.041215 INFO::detectRRNA.R/detectRRNA: done
 done successfully.



Executing test function test.detectRRNA.paired_end  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:48:45.268279 INFO::detectRRNA.R/detectRRNA: searching for rRNA contamination...
2023-05-15 05:48:45.269803 INFO::io.R/writeFastQFiles: writing filename= /tmp/Rtmpt80l8W/test.detectRRNA.paired_end.2838c9458415df/bams/rRNA_contam/input1.fastq
2023-05-15 05:48:45.272185 INFO::io.R/writeFastQFiles: writing filename= /tmp/Rtmpt80l8W/test.detectRRNA.paired_end.2838c9458415df/bams/rRNA_contam/input2.fastq
2023-05-15 05:48:45.275942 INFO::alignReads.R/alignReadsChunk: calling gsnap (from gmapR) with args= -t 0 -D /home/biocbuild/.local/share/gmap -n 1 -d TP53_demo_3.2.2 -A sam -B 2 --pairmax-rna=200000 --split-output /tmp/Rtmpt80l8W/test.detectRRNA.paired_end.2838c9458415df/bams/rRNA_contam/test_pe /tmp/Rtmpt80l8W/test.detectRRNA.paired_end.2838c9458415df/bams/rRNA_contam/input1.fastq -a paired /tmp/Rtmpt80l8W/test.detectRRNA.paired_end.2838c9458415df/bams/rRNA_contam/input2.fastq 2&gt;&amp;1
2023-05-15 05:48:45.581846 INFO::detectRRNA.R/detectRRNA: contaminated fraction= 1
2023-05-15 05:48:45.583361 INFO::detectRRNA.R/detectRRNA: done
 done successfully.



Executing test function test.getRRNAIds  ... 2023-05-15 05:48:45.612943 INFO::alignReads.R/alignReadsChunk: calling gsnap (from gmapR) with args= -t 0 -D /home/biocbuild/.local/share/gmap -n 1 -d TP53_demo_3.2.2 -A sam -B 2 --pairmax-rna=200000 --split-output /tmp/Rtmpt80l8W/test_get_rRNA_idscyphsonu/test_pe /tmp/Rtmpt80l8W/test_get_rRNA_idscyphsonu/1.fastq -a paired /tmp/Rtmpt80l8W/test_get_rRNA_idscyphsonu/2.fastq 2&gt;&amp;1
 done successfully.



Executing test function test.getRRNAIds_random  ... 2023-05-15 05:48:45.948811 INFO::alignReads.R/alignReadsChunk: calling gsnap (from gmapR) with args= -t 0 -D /home/biocbuild/.local/share/gmap -n 1 -d TP53_demo_3.2.2 -A sam -B 2 --pairmax-rna=200000 --split-output /tmp/Rtmpt80l8W/test_get_rRNAIds_randombnvsyxel/test_pe /tmp/Rtmpt80l8W/test_get_rRNAIds_randombnvsyxel/1.fastq 2&gt;&amp;1
 done successfully.



Executing test function test.filterByLength  ... 2023-05-15 05:48:46.175472 INFO::filterQuality.R/filterQuality: filterByLength...
2023-05-15 05:48:46.177608 INFO::filterQuality.R/filterByLength: fraction of filtered short reads= 0.5
2023-05-15 05:48:46.178967 INFO::filterQuality.R/filterByLength: done
2023-05-15 05:48:46.248872 INFO::filterQuality.R/filterQuality: filterByLength...
2023-05-15 05:48:46.250233 INFO::filterQuality.R/filterByLength: fraction of filtered short reads= 1
2023-05-15 05:48:46.2515 INFO::filterQuality.R/filterByLength: done
 done successfully.



Executing test function test.isAboveQualityThresh  ...  done successfully.



Executing test function test.trimTailsByQuality  ... 2023-05-15 05:48:46.327433 INFO::preprocessReads.R/preprocessReadsChunk: Starting trimTailsByQuality ...
2023-05-15 05:48:46.34078 INFO::preprocessReads.R/preprocessReadsChunk: done
2023-05-15 05:48:46.342464 INFO::preprocessReads.R/preprocessReadsChunk: Starting trimTailsByQuality ...
2023-05-15 05:48:46.349289 INFO::preprocessReads.R/preprocessReadsChunk: done
2023-05-15 05:48:46.350923 INFO::preprocessReads.R/preprocessReadsChunk: Starting trimTailsByQuality ...
2023-05-15 05:48:46.357802 INFO::preprocessReads.R/preprocessReadsChunk: done
2023-05-15 05:48:46.359414 INFO::preprocessReads.R/preprocessReadsChunk: Starting trimTailsByQuality ...
2023-05-15 05:48:46.366224 INFO::preprocessReads.R/preprocessReadsChunk: done
 done successfully.



Executing test function test.callVariantsGATK  ... Timing stopped at: 0 0 0.001
Error in DEACTIVATED(&quot;callVariantsGATK() tests need gatk.path option set&quot;) : 
  callVariantsGATK() tests need gatk.path option set
In addition: There were 50 or more warnings (use warnings() to see the first 50)
 done successfully.



Executing test function test.callVariantsGATK.withFiltering  ... Timing stopped at: 0 0 0
Error in DEACTIVATED(&quot;callVariantsGATK() tests need gatk.path option set&quot;) : 
  callVariantsGATK() tests need gatk.path option set
 done successfully.



Executing test function test.checkGATKJar  ... Timing stopped at: 0 0 0.001
Error in DEACTIVATED(&quot;checkGATKJar() test needs gatk.path option set&quot;) : 
  checkGATKJar() test needs gatk.path option set
 done successfully.



Executing test function test.excludeVariantsByRegion  ...  done successfully.



Executing test function test.gatk  ... Timing stopped at: 0 0 0
Error in DEACTIVATED(&quot;gatk() tests need gatk.path option set&quot;) : 
  gatk() tests need gatk.path option set
 done successfully.



Executing test function test.realignIndels  ... Timing stopped at: 0 0 0.001
Error in DEACTIVATED(&quot;test.realignIndels() tests need gatk.path option set&quot;) : 
  test.realignIndels() tests need gatk.path option set
 done successfully.



Executing test function test.realignIndelsGATK  ... Timing stopped at: 0.001 0 0
Error in DEACTIVATED(&quot;realignIndelsGATK() tests need gatk.path option set&quot;) : 
  realignIndelsGATK() tests need gatk.path option set
 done successfully.



Executing test function test.realignIndelsGATK.parallel  ... Timing stopped at: 0 0 0
Error in DEACTIVATED(&quot;realignIndelsGATK() tests need gatk.path option set&quot;) : 
  realignIndelsGATK() tests need gatk.path option set
 done successfully.



Executing test function test_zipUp  ...  done successfully.



Executing test function test.FastQStreamer.getReads.pefq  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:48:46.809539 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_1.fastq
2023-05-15 05:48:46.813117 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_2.fastq
 done successfully.



Executing test function test.FastQStreamer.getReads.pefq.subsample  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:48:47.087422 INFO::io.R/FastQStreamer.init: counting number of reads in file= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_1.fastq
2023-05-15 05:48:47.150635 INFO::io.R/FastQStreamer.init: subsampling_filter set (subsampled reads= 17 , totnbreads= 26 )
2023-05-15 05:48:47.154659 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_1.fastq
2023-05-15 05:48:47.158185 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_2.fastq
 done successfully.



Executing test function test.FastQStreamer.getReads.segz  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:48:47.50953 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/reads.fastq.gz
 done successfully.



Executing test function test.FastQStreamer.getReads.truncated  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:48:47.727515 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/truncated.fastq.gz
 done successfully.



Executing test function test.FastQStreamer.subsampler.isdeterministic  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:48:47.998531 INFO::io.R/FastQStreamer.init: counting number of reads in file= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_1.fastq
2023-05-15 05:48:48.022486 INFO::io.R/FastQStreamer.init: subsampling_filter set (subsampled reads= 17 , totnbreads= 26 )
2023-05-15 05:48:48.026126 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_1.fastq
2023-05-15 05:48:48.029568 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_2.fastq
checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:48:48.368606 INFO::io.R/FastQStreamer.init: counting number of reads in file= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_1.fastq
2023-05-15 05:48:48.392673 INFO::io.R/FastQStreamer.init: subsampling_filter set (subsampled reads= 17 , totnbreads= 26 )
2023-05-15 05:48:48.39651 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_1.fastq
2023-05-15 05:48:48.400118 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_2.fastq
 done successfully.



Executing test function test.createTmpDir  ...  done successfully.



Executing test function test.detectQualityInFASTQFile  ...  done successfully.



Executing test function test.getObjectFilename  ...  done successfully.



Executing test function test.safeUnlink  ...  done successfully.



Executing test function test.writeAudit  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
 done successfully.



Executing test function test.mergeLanes  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:48:49.452227 INFO::preprocessReads.R/preprocessReads: starting...
2023-05-15 05:48:49.459884 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_1.fastq
2023-05-15 05:48:49.464945 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_2.fastq
2023-05-15 05:48:49.468151 DEBUG::tools.R/processChunks: starting...
2023-05-15 05:48:53.453406 DEBUG::tools.R/processChunks: waiting for chunkid=[  ] ...
2023-05-15 05:48:53.455674 DEBUG::tools.R/processChunks: starting chunkid= 1 ; see logfile= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/chunks/chunk_000001/logs/progress.log
2023-05-15 05:48:56.254282 DEBUG::tools.R/processChunks: done with chunkid= 1 ; elapsed.time= 0.047 minutes
2023-05-15 05:48:56.2565 DEBUG::tools.R/processChunks: starting chunkid= 2 ; see logfile= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/chunks/chunk_000002/logs/progress.log
2023-05-15 05:48:59.052752 DEBUG::tools.R/processChunks: done with chunkid= 2 ; elapsed.time= 0.047 minutes
2023-05-15 05:48:59.054708 DEBUG::tools.R/processChunks: starting chunkid= 3 ; see logfile= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/chunks/chunk_000003/logs/progress.log
2023-05-15 05:49:01.900046 DEBUG::tools.R/processChunks: done with chunkid= 3 ; elapsed.time= 0.047 minutes
2023-05-15 05:49:01.970327 DEBUG::tools.R/processChunks: done
2023-05-15 05:49:01.974447 INFO::detectAdapterContam.R/mergeDetectAdapterContam: merging file= adapter_contaminated_1 ...
2023-05-15 05:49:01.976894 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.adapter_contaminated_1.RData
2023-05-15 05:49:01.980038 INFO::detectAdapterContam.R/mergeDetectAdapterContam: merging file= adapter_contaminated_2 ...
2023-05-15 05:49:01.982514 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.adapter_contaminated_2.RData
2023-05-15 05:49:01.994583 INFO::preprocessReads.R/mergeSummaryPreprocess: total_reads=26 highqual_reads=25 adapter_contam=1 read_length=75 rRNA_contam_reads=0 processed_reads=25 input_min_read_length=75 input_max_read_length=75 processed_min_read_length=75 processed_max_read_length=75
2023-05-15 05:49:01.99749 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.summary_preprocess.tab
2023-05-15 05:49:02.001593 INFO::preprocessReads.R/mergePreprocessedReads: merging file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/bams/processed.aligner_input_1.fastq ...
2023-05-15 05:49:02.008092 INFO::preprocessReads.R/mergePreprocessedReads: merging file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/bams/processed.aligner_input_2.fastq ...
2023-05-15 05:49:02.013245 INFO::preprocessReads.R/buildShortReadReports: generating report_dir= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/reports/shortReadReport_1 ...
2023-05-15 05:49:03.972041 INFO::preprocessReads.R/buildShortReadReports: generating report_dir= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/reports/shortReadReport_2 ...
2023-05-15 05:49:05.865925 INFO::preprocessReads.R/preprocessReads: done
2023-05-15 05:49:05.95014 INFO::alignReads.R/alignReads: starting alignment...
2023-05-15 05:49:05.955699 DEBUG::tools.R/processChunks: starting...
2023-05-15 05:49:10.321745 DEBUG::tools.R/processChunks: waiting for chunkid=[  ] ...
2023-05-15 05:49:10.324258 DEBUG::tools.R/processChunks: starting chunkid= 1 ; see logfile= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/chunks/chunk_000001/logs/progress.log
2023-05-15 05:49:13.05755 DEBUG::tools.R/processChunks: done with chunkid= 1 ; elapsed.time= 0.046 minutes
2023-05-15 05:49:13.060198 DEBUG::tools.R/processChunks: starting chunkid= 2 ; see logfile= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/chunks/chunk_000002/logs/progress.log
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPEsInManyTxsOfMultiTxGene:1:1:1:16#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPE1MicrobialPE2MultimappingIntergenic:1:1:1:19#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPEsInOneTxOfMultiTxGene:1:1:1:15#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPE1UniqueUtrPE2MultimappingIntergenic:1:1:1:18#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
2023-05-15 05:49:15.788738 DEBUG::tools.R/processChunks: done with chunkid= 2 ; elapsed.time= 0.045 minutes
2023-05-15 05:49:15.79125 DEBUG::tools.R/processChunks: starting chunkid= 3 ; see logfile= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/chunks/chunk_000003/logs/progress.log
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPE1GeneFusionMidExonDiffChrPE2MultimappingIntergenic:1:1:1:25#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
2023-05-15 05:49:18.51973 DEBUG::tools.R/processChunks: done with chunkid= 3 ; elapsed.time= 0.045 minutes
2023-05-15 05:49:18.523169 DEBUG::tools.R/processChunks: done
2023-05-15 05:49:18.525392 INFO::alignReads.R/mergeBAMsAcrossDirs: starting...
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualAdapterContamIn3PrimeEnd:1:1:1:7#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPEsInManyTxsOfMultiTxGene:1:1:1:16#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPE1MicrobialPE2MultimappingIntergenic:1:1:1:19#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPE1GeneFusionMidExonDiffChrPE2MultimappingIntergenic:1:1:1:25#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPE1UniqueUtrPE2MultimappingIntergenic:1:1:1:18#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPEsInOneTxOfMultiTxGene:1:1:1:15#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
2023-05-15 05:49:18.820986 INFO::alignReads.R/mergeBAMsAcrossDirs: done
2023-05-15 05:49:18.832051 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.summary_alignment.tab
2023-05-15 05:49:18.842779 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.summary_analyzed_bamstats.tab
2023-05-15 05:49:18.845607 INFO::reportPipelineQA.R/writePreprocessAlignReport: creating joint preprocess-alignment report...
2023-05-15 05:49:19.142148 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.summary_target_lengths.tab
2023-05-15 05:49:19.204142 INFO::reportPipelineQA.R/writePreprocessAlignReport: done
2023-05-15 05:49:19.205606 INFO::alignReads.R/alignReads: done
2023-05-15 05:49:19.28938 INFO::countGenomicFeatures.R/countGenomicFeatures: starting...
2023-05-15 05:49:19.316114 DEBUG::tools.R/processChunks: starting...
2023-05-15 05:49:23.21694 DEBUG::tools.R/processChunks: waiting for chunkid=[  ] ...
2023-05-15 05:49:23.219174 DEBUG::tools.R/processChunks: starting chunkid= 1 ; see logfile= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/chunks/chunk_000001/logs/progress.log
2023-05-15 05:49:25.82057 DEBUG::tools.R/processChunks: done with chunkid= 1 ; elapsed.time= 0.043 minutes
2023-05-15 05:49:25.823013 DEBUG::tools.R/processChunks: starting chunkid= 2 ; see logfile= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/chunks/chunk_000002/logs/progress.log
2023-05-15 05:49:28.423996 DEBUG::tools.R/processChunks: done with chunkid= 2 ; elapsed.time= 0.043 minutes
2023-05-15 05:49:28.426241 DEBUG::tools.R/processChunks: starting chunkid= 3 ; see logfile= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/chunks/chunk_000003/logs/progress.log
2023-05-15 05:49:31.022529 DEBUG::tools.R/processChunks: done with chunkid= 3 ; elapsed.time= 0.043 minutes
2023-05-15 05:49:31.025927 DEBUG::tools.R/processChunks: done
2023-05-15 05:49:31.028279 INFO::countGenomicFeatures.R/mergeCounts: starting...
2023-05-15 05:49:31.04706 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.counts_exon.tab
2023-05-15 05:49:31.06275 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.counts_exon_disjoint.tab
2023-05-15 05:49:31.072677 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.counts_gene.tab
2023-05-15 05:49:31.080456 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.counts_gene_coding.tab
2023-05-15 05:49:31.088156 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.counts_gene_exonic.tab
2023-05-15 05:49:31.095992 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.counts_intergenic.tab
2023-05-15 05:49:31.108907 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.counts_intron.tab
2023-05-15 05:49:31.112753 INFO::countGenomicFeatures.R/mergeCounts: done
2023-05-15 05:49:31.120869 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.summary_counts.tab
2023-05-15 05:49:31.123218 INFO::reportPipelineQA.R/writeGenomicFeaturesReport: creating report of genomic features counts...
2023-05-15 05:49:31.481157 INFO::reportPipelineQA.R/writeGenomicFeaturesReport: done
2023-05-15 05:49:31.482477 INFO::countGenomicFeatures.R/countGenomicFeatures: done...
2023-05-15 05:49:31.56797 INFO::coverage.R/calculateCoverage: starting...
2023-05-15 05:49:31.573363 DEBUG::tools.R/processChunks: starting...
2023-05-15 05:49:35.80777 DEBUG::tools.R/processChunks: waiting for chunkid=[  ] ...
2023-05-15 05:49:35.810817 DEBUG::tools.R/processChunks: starting chunkid= 1 ; see logfile= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/chunks/chunk_000001/logs/progress.log
2023-05-15 05:49:38.024392 DEBUG::tools.R/processChunks: done with chunkid= 1 ; elapsed.time= 0.037 minutes
2023-05-15 05:49:38.026755 DEBUG::tools.R/processChunks: starting chunkid= 2 ; see logfile= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/chunks/chunk_000002/logs/progress.log
2023-05-15 05:49:40.255404 DEBUG::tools.R/processChunks: done with chunkid= 2 ; elapsed.time= 0.037 minutes
2023-05-15 05:49:40.257787 DEBUG::tools.R/processChunks: starting chunkid= 3 ; see logfile= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/chunks/chunk_000003/logs/progress.log
2023-05-15 05:49:42.468739 DEBUG::tools.R/processChunks: done with chunkid= 3 ; elapsed.time= 0.037 minutes
2023-05-15 05:49:42.472455 DEBUG::tools.R/processChunks: done
2023-05-15 05:49:44.524921 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.coverage.RData
2023-05-15 05:49:44.527451 INFO::coverage.R/saveCoverage: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.coverage.bw
2023-05-15 05:49:44.547874 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.summary_coverage.tab
2023-05-15 05:49:44.550062 INFO::coverage.R/calculateCoverage: done
2023-05-15 05:49:44.552602 INFO::analyzeVariants/analyzeVariants: starting ...
2023-05-15 05:49:44.639471 INFO::analyzeVariants.R/wrap.callVariants: Tallying variants...
2023-05-15 05:49:48.405688 INFO::analyzeVariants.R/wrap.callVariants: calling variants...
2023-05-15 05:49:48.508014 INFO::analyzeVariants.R/wrap.postFilerVariants: post filtering variants...
2023-05-15 05:49:48.530852 INFO::analyzeVariants.R/wrap.callVariants: Saving GRanges of raw and filtered variants...
2023-05-15 05:49:48.532691 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.raw_variants.RData
2023-05-15 05:49:48.535031 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.filtered_variants.RData
2023-05-15 05:49:48.536901 INFO::analyzeVariants.R/wrap.callVariants: ...done
2023-05-15 05:49:48.538344 INFO::analyzeVariants.R/writeVCF: writing vcf file...
2023-05-15 05:49:48.840021 INFO::analyzeVariants.R/writeVCF: ...done
2023-05-15 05:49:48.970305 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/results/test_pe.summary_variants.tab
2023-05-15 05:49:48.972977 INFO::analyzeVariants/analyzeVariants: done
2023-05-15 05:49:48.977417 INFO::Pipeline run successful.
2023-05-15 05:49:49.179317 INFO::mergeLanes.R/doMergeLanes: starting...
2023-05-15 05:49:49.185851 INFO::detectAdapterContam.R/mergeDetectAdapterContam: merging file= adapter_contaminated_1 ...
2023-05-15 05:49:49.189136 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.adapter_contaminated_1.RData
2023-05-15 05:49:49.193472 INFO::detectAdapterContam.R/mergeDetectAdapterContam: merging file= adapter_contaminated_2 ...
2023-05-15 05:49:49.196449 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.adapter_contaminated_2.RData
2023-05-15 05:49:49.212467 INFO::preprocessReads.R/mergeSummaryPreprocess: total_reads=26 highqual_reads=25 adapter_contam=1 read_length=75 rRNA_contam_reads=0 processed_reads=25 input_min_read_length=75 input_max_read_length=75 processed_min_read_length=75 processed_max_read_length=75
2023-05-15 05:49:49.215878 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.summary_preprocess.tab
2023-05-15 05:49:49.219023 INFO::alignReads.R/mergeBAMsAcrossDirs: starting...
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualAdapterContamIn3PrimeEnd:1:1:1:7#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPEsInManyTxsOfMultiTxGene:1:1:1:16#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPE1MicrobialPE2MultimappingIntergenic:1:1:1:19#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPE1GeneFusionMidExonDiffChrPE2MultimappingIntergenic:1:1:1:25#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPE1UniqueUtrPE2MultimappingIntergenic:1:1:1:18#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;highqualPEsInOneTxOfMultiTxGene:1:1:1:15#0&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
2023-05-15 05:49:49.56596 INFO::alignReads.R/mergeBAMsAcrossDirs: done
2023-05-15 05:49:49.576252 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.summary_alignment.tab
2023-05-15 05:49:49.588053 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.summary_analyzed_bamstats.tab
2023-05-15 05:49:49.591119 INFO::reportPipelineQA.R/writePreprocessAlignReport: creating joint preprocess-alignment report...
2023-05-15 05:49:49.890298 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.summary_target_lengths.tab
2023-05-15 05:49:49.956175 INFO::reportPipelineQA.R/writePreprocessAlignReport: done
2023-05-15 05:49:50.005678 INFO::countGenomicFeatures.R/mergeCounts: starting...
2023-05-15 05:49:50.035059 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.counts_exon.tab
2023-05-15 05:49:50.052362 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.counts_exon_disjoint.tab
2023-05-15 05:49:50.062966 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.counts_gene.tab
2023-05-15 05:49:50.071843 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.counts_gene_coding.tab
2023-05-15 05:49:50.080932 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.counts_gene_exonic.tab
2023-05-15 05:49:50.090394 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.counts_intergenic.tab
2023-05-15 05:49:50.107773 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.counts_intron.tab
2023-05-15 05:49:50.112043 INFO::countGenomicFeatures.R/mergeCounts: done
2023-05-15 05:49:50.122035 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.summary_counts.tab
2023-05-15 05:49:50.124948 INFO::reportPipelineQA.R/writeGenomicFeaturesReport: creating report of genomic features counts...
2023-05-15 05:49:50.41407 INFO::reportPipelineQA.R/writeGenomicFeaturesReport: done
2023-05-15 05:49:52.871394 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.coverage.RData
2023-05-15 05:49:52.874802 INFO::coverage.R/saveCoverage: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.coverage.bw
2023-05-15 05:49:52.890821 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.summary_coverage.tab
2023-05-15 05:49:52.934623 INFO::analyzeVariants/analyzeVariants: starting ...
2023-05-15 05:49:53.027849 INFO::analyzeVariants.R/wrap.callVariants: Tallying variants...
2023-05-15 05:49:56.8629 INFO::analyzeVariants.R/wrap.callVariants: calling variants...
2023-05-15 05:49:56.961707 INFO::analyzeVariants.R/wrap.postFilerVariants: post filtering variants...
2023-05-15 05:49:56.99065 INFO::analyzeVariants.R/wrap.callVariants: Saving GRanges of raw and filtered variants...
2023-05-15 05:49:56.992523 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.raw_variants.RData
2023-05-15 05:49:56.994846 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.filtered_variants.RData
2023-05-15 05:49:56.996622 INFO::analyzeVariants.R/wrap.callVariants: ...done
2023-05-15 05:49:56.998101 INFO::analyzeVariants.R/writeVCF: writing vcf file...
2023-05-15 05:49:57.297202 INFO::analyzeVariants.R/writeVCF: ...done
2023-05-15 05:49:57.425185 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.mergeLanes.2838c92a9fa6c3/merged/results/merged.summary_variants.tab
2023-05-15 05:49:57.427902 INFO::analyzeVariants/analyzeVariants: done
2023-05-15 05:49:57.431617 INFO::mergeLanes.R/doMergeLanes: merge lanes successful.
 done successfully.



Executing test function test.markDuplicates  ... Timing stopped at: 0 0 0
Error in DEACTIVATED(&quot;Skipped markDuplicates() test&quot;) : 
  Skipped markDuplicates() test
In addition: There were 28 warnings (use warnings() to see them)
 done successfully.



Executing test function test.markDuplicates_w_outfile  ... Timing stopped at: 0 0 0.001
Error in DEACTIVATED(&quot;Skipped markDuplicates() test&quot;) : 
  Skipped markDuplicates() test
 done successfully.



Executing test function test.preprocessReads  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:49:57.792825 INFO::preprocessReads.R/preprocessReads: starting...
2023-05-15 05:49:57.817022 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_1.fastq
2023-05-15 05:49:57.840331 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_2.fastq
2023-05-15 05:49:57.843547 DEBUG::tools.R/processChunks: starting...
2023-05-15 05:50:01.506949 DEBUG::tools.R/processChunks: waiting for chunkid=[  ] ...
2023-05-15 05:50:01.509181 DEBUG::tools.R/processChunks: starting chunkid= 1 ; see logfile= /tmp/Rtmpt80l8W/test.preprocessReads.2838c950a1a08f/chunks/chunk_000001/logs/progress.log
2023-05-15 05:50:04.332192 DEBUG::tools.R/processChunks: done with chunkid= 1 ; elapsed.time= 0.047 minutes
2023-05-15 05:50:04.400368 DEBUG::tools.R/processChunks: done
2023-05-15 05:50:04.403588 INFO::detectAdapterContam.R/mergeDetectAdapterContam: merging file= adapter_contaminated_1 ...
2023-05-15 05:50:04.405874 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.preprocessReads.2838c950a1a08f/results/test_pe.adapter_contaminated_1.RData
2023-05-15 05:50:04.408308 INFO::detectAdapterContam.R/mergeDetectAdapterContam: merging file= adapter_contaminated_2 ...
2023-05-15 05:50:04.410527 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.preprocessReads.2838c950a1a08f/results/test_pe.adapter_contaminated_2.RData
2023-05-15 05:50:04.419338 INFO::preprocessReads.R/mergeSummaryPreprocess: total_reads=26 highqual_reads=25 adapter_contam=1 read_length=75 rRNA_contam_reads=0 processed_reads=25 input_min_read_length=75 input_max_read_length=75 processed_min_read_length=75 processed_max_read_length=75
2023-05-15 05:50:04.422027 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.preprocessReads.2838c950a1a08f/results/test_pe.summary_preprocess.tab
2023-05-15 05:50:04.425148 INFO::preprocessReads.R/mergePreprocessedReads: merging file= /tmp/Rtmpt80l8W/test.preprocessReads.2838c950a1a08f/bams/processed.aligner_input_1.fastq ...
2023-05-15 05:50:04.430922 INFO::preprocessReads.R/mergePreprocessedReads: merging file= /tmp/Rtmpt80l8W/test.preprocessReads.2838c950a1a08f/bams/processed.aligner_input_2.fastq ...
2023-05-15 05:50:04.436692 INFO::preprocessReads.R/buildShortReadReports: generating report_dir= /tmp/Rtmpt80l8W/test.preprocessReads.2838c950a1a08f/reports/shortReadReport_1 ...
2023-05-15 05:50:06.245786 INFO::preprocessReads.R/buildShortReadReports: generating report_dir= /tmp/Rtmpt80l8W/test.preprocessReads.2838c950a1a08f/reports/shortReadReport_2 ...
2023-05-15 05:50:07.701897 INFO::preprocessReads.R/preprocessReads: done
 done successfully.



Executing test function test.preprocessReads.minichunks  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:50:08.039229 INFO::preprocessReads.R/preprocessReads: starting...
2023-05-15 05:50:08.047615 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_1.fastq
2023-05-15 05:50:08.052956 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_2.fastq
2023-05-15 05:50:08.056329 DEBUG::tools.R/processChunks: starting...
2023-05-15 05:50:11.696076 DEBUG::tools.R/processChunks: waiting for chunkid=[  ] ...
2023-05-15 05:50:11.698336 DEBUG::tools.R/processChunks: starting chunkid= 1 ; see logfile= /tmp/Rtmpt80l8W/test.preprocessReads.minichunks.2838c931a1a11/chunks/chunk_000001/logs/progress.log
2023-05-15 05:50:14.494411 DEBUG::tools.R/processChunks: done with chunkid= 1 ; elapsed.time= 0.047 minutes
2023-05-15 05:50:14.496488 DEBUG::tools.R/processChunks: starting chunkid= 2 ; see logfile= /tmp/Rtmpt80l8W/test.preprocessReads.minichunks.2838c931a1a11/chunks/chunk_000002/logs/progress.log
2023-05-15 05:50:17.346244 DEBUG::tools.R/processChunks: done with chunkid= 2 ; elapsed.time= 0.047 minutes
2023-05-15 05:50:17.348391 DEBUG::tools.R/processChunks: starting chunkid= 3 ; see logfile= /tmp/Rtmpt80l8W/test.preprocessReads.minichunks.2838c931a1a11/chunks/chunk_000003/logs/progress.log
2023-05-15 05:50:20.172631 DEBUG::tools.R/processChunks: done with chunkid= 3 ; elapsed.time= 0.047 minutes
2023-05-15 05:50:20.240386 DEBUG::tools.R/processChunks: done
2023-05-15 05:50:20.244282 INFO::detectAdapterContam.R/mergeDetectAdapterContam: merging file= adapter_contaminated_1 ...
2023-05-15 05:50:20.24677 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.preprocessReads.minichunks.2838c931a1a11/results/test_pe.adapter_contaminated_1.RData
2023-05-15 05:50:20.249819 INFO::detectAdapterContam.R/mergeDetectAdapterContam: merging file= adapter_contaminated_2 ...
2023-05-15 05:50:20.252166 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.preprocessReads.minichunks.2838c931a1a11/results/test_pe.adapter_contaminated_2.RData
2023-05-15 05:50:20.263497 INFO::preprocessReads.R/mergeSummaryPreprocess: total_reads=26 highqual_reads=25 adapter_contam=1 read_length=75 rRNA_contam_reads=0 processed_reads=25 input_min_read_length=75 input_max_read_length=75 processed_min_read_length=75 processed_max_read_length=75
2023-05-15 05:50:20.266252 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.preprocessReads.minichunks.2838c931a1a11/results/test_pe.summary_preprocess.tab
2023-05-15 05:50:20.269989 INFO::preprocessReads.R/mergePreprocessedReads: merging file= /tmp/Rtmpt80l8W/test.preprocessReads.minichunks.2838c931a1a11/bams/processed.aligner_input_1.fastq ...
2023-05-15 05:50:20.275924 INFO::preprocessReads.R/mergePreprocessedReads: merging file= /tmp/Rtmpt80l8W/test.preprocessReads.minichunks.2838c931a1a11/bams/processed.aligner_input_2.fastq ...
2023-05-15 05:50:20.280964 INFO::preprocessReads.R/buildShortReadReports: generating report_dir= /tmp/Rtmpt80l8W/test.preprocessReads.minichunks.2838c931a1a11/reports/shortReadReport_1 ...
2023-05-15 05:50:22.085461 INFO::preprocessReads.R/buildShortReadReports: generating report_dir= /tmp/Rtmpt80l8W/test.preprocessReads.minichunks.2838c931a1a11/reports/shortReadReport_2 ...
2023-05-15 05:50:23.54371 INFO::preprocessReads.R/preprocessReads: done
 done successfully.



Executing test function test.preprocessReads_single_end  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:50:23.887309 INFO::preprocessReads.R/preprocessReads: starting...
2023-05-15 05:50:23.912035 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/test-data/unit_tests_1.fastq
2023-05-15 05:50:23.915601 DEBUG::tools.R/processChunks: starting...
2023-05-15 05:50:27.579564 DEBUG::tools.R/processChunks: waiting for chunkid=[  ] ...
2023-05-15 05:50:27.581738 DEBUG::tools.R/processChunks: starting chunkid= 1 ; see logfile= /tmp/Rtmpt80l8W/test.preprocessReads_single_end.2838c96330f454/chunks/chunk_000001/logs/progress.log
2023-05-15 05:50:30.08157 DEBUG::tools.R/processChunks: done with chunkid= 1 ; elapsed.time= 0.042 minutes
2023-05-15 05:50:30.148506 DEBUG::tools.R/processChunks: done
2023-05-15 05:50:30.152114 INFO::detectAdapterContam.R/mergeDetectAdapterContam: merging file= adapter_contaminated_1 ...
2023-05-15 05:50:30.154714 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.preprocessReads_single_end.2838c96330f454/results/test_se.adapter_contaminated_1.RData
2023-05-15 05:50:30.163957 INFO::preprocessReads.R/mergeSummaryPreprocess: total_reads=26 highqual_reads=25 adapter_contam=1 read_length=75 rRNA_contam_reads=0 processed_reads=25 input_min_read_length=75 input_max_read_length=75 processed_min_read_length=75 processed_max_read_length=75
2023-05-15 05:50:30.166516 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.preprocessReads_single_end.2838c96330f454/results/test_se.summary_preprocess.tab
2023-05-15 05:50:30.169263 INFO::preprocessReads.R/mergePreprocessedReads: merging file= /tmp/Rtmpt80l8W/test.preprocessReads_single_end.2838c96330f454/bams/processed.aligner_input_1.fastq ...
2023-05-15 05:50:30.174239 INFO::preprocessReads.R/buildShortReadReports: generating report_dir= /tmp/Rtmpt80l8W/test.preprocessReads_single_end.2838c96330f454/reports/shortReadReport_1 ...
2023-05-15 05:50:32.08771 INFO::preprocessReads.R/preprocessReads: done
 done successfully.



Executing test function test.readRNASeqEnds  ...  done successfully.



Executing test function test.readRNASeqEnds.dupmark  ...  done successfully.



Executing test function test.how_many  ...  done successfully.



Executing test function test.plotDF  ...  done successfully.



Executing test function test.relativeBarPlot  ...  done successfully.



Executing test function test.runPipeline  ... checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:50:32.88796 INFO::preprocessReads.R/preprocessReads: starting...
2023-05-15 05:50:32.91528 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/extdata/H1993_TP53_subset2500_1.fastq.gz
2023-05-15 05:50:32.941561 INFO::io.R/FastQStreamer.init: initialised FastQ streamer for filename= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/extdata/H1993_TP53_subset2500_2.fastq.gz
2023-05-15 05:50:32.944938 DEBUG::tools.R/processChunks: starting...
2023-05-15 05:50:37.409729 DEBUG::tools.R/processChunks: waiting for chunkid=[  ] ...
2023-05-15 05:50:37.411902 DEBUG::tools.R/processChunks: starting chunkid= 1 ; see logfile= test/chunks/chunk_000001/logs/progress.log
2023-05-15 05:50:40.52912 DEBUG::tools.R/processChunks: done with chunkid= 1 ; elapsed.time= 0.052 minutes
2023-05-15 05:50:40.598788 DEBUG::tools.R/processChunks: done
2023-05-15 05:50:40.601926 INFO::detectAdapterContam.R/mergeDetectAdapterContam: merging file= adapter_contaminated_1 ...
2023-05-15 05:50:40.604061 INFO::io.R/saveWithID: saving file= test/results/test.adapter_contaminated_1.RData
2023-05-15 05:50:40.606386 INFO::detectAdapterContam.R/mergeDetectAdapterContam: merging file= adapter_contaminated_2 ...
2023-05-15 05:50:40.608388 INFO::io.R/saveWithID: saving file= test/results/test.adapter_contaminated_2.RData
2023-05-15 05:50:40.616855 INFO::preprocessReads.R/mergeSummaryPreprocess: total_reads=2500 highqual_reads=2500 adapter_contam=0 read_length=100 rRNA_contam_reads=0 processed_reads=2500 input_min_read_length=100 input_max_read_length=100 processed_min_read_length=71 processed_max_read_length=100
2023-05-15 05:50:40.619376 INFO::io.R/saveWithID: saving file= test/results/test.summary_preprocess.tab
2023-05-15 05:50:40.622361 INFO::preprocessReads.R/mergePreprocessedReads: merging file= test/bams/processed.aligner_input_1.fastq ...
2023-05-15 05:50:40.628919 INFO::preprocessReads.R/mergePreprocessedReads: merging file= test/bams/processed.aligner_input_2.fastq ...
2023-05-15 05:50:40.634955 INFO::preprocessReads.R/buildShortReadReports: generating report_dir= test/reports/shortReadReport_1 ...
2023-05-15 05:50:42.726654 INFO::preprocessReads.R/buildShortReadReports: generating report_dir= test/reports/shortReadReport_2 ...
2023-05-15 05:50:44.460278 INFO::preprocessReads.R/preprocessReads: done
2023-05-15 05:50:44.554896 INFO::alignReads.R/alignReads: starting alignment...
2023-05-15 05:50:44.560779 DEBUG::tools.R/processChunks: starting...
2023-05-15 05:50:48.687675 DEBUG::tools.R/processChunks: waiting for chunkid=[  ] ...
2023-05-15 05:50:48.690025 DEBUG::tools.R/processChunks: starting chunkid= 1 ; see logfile= test/chunks/chunk_000001/logs/progress.log
[bam_translate] PG tag &quot;A&quot; on read &quot;HISEQ1:109:D093LACXX:4:2205:16499:6611&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;HISEQ1:109:D093LACXX:4:1206:17367:135118&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;HISEQ1:109:D093LACXX:4:1304:1838:53573&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;A&quot; on read &quot;HISEQ1:109:D093LACXX:4:2303:2956:116587&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;M&quot; on read &quot;HISEQ1:109:D093LACXX:4:2205:16499:6611&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;T&quot; on read &quot;HISEQ1:109:D093LACXX:4:1206:20028:108227&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;T&quot; on read &quot;HISEQ1:109:D093LACXX:4:1207:19518:143960&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
[bam_translate] PG tag &quot;T&quot; on read &quot;HISEQ1:109:D093LACXX:4:2207:4065:173048&quot; encountered with no corresponding entry in header, tag lost. Unknown tags are only reported once per input file for each tag ID.
2023-05-15 05:50:54.404701 DEBUG::tools.R/processChunks: done with chunkid= 1 ; elapsed.time= 0.095 minutes
2023-05-15 05:50:54.408156 DEBUG::tools.R/processChunks: done
2023-05-15 05:50:54.410138 INFO::alignReads.R/mergeBAMsAcrossDirs: starting...
2023-05-15 05:50:54.454119 INFO::alignReads.R/mergeBAMsAcrossDirs: done
2023-05-15 05:50:54.462905 INFO::io.R/saveWithID: saving file= test/results/test.summary_alignment.tab
2023-05-15 05:50:54.470788 INFO::io.R/saveWithID: saving file= test/results/test.summary_analyzed_bamstats.tab
2023-05-15 05:50:54.473773 INFO::reportPipelineQA.R/writePreprocessAlignReport: creating joint preprocess-alignment report...
2023-05-15 05:50:54.673736 INFO::io.R/saveWithID: saving file= test/results/test.summary_target_lengths.tab
2023-05-15 05:50:54.734933 INFO::reportPipelineQA.R/writePreprocessAlignReport: done
2023-05-15 05:50:54.736372 INFO::alignReads.R/alignReads: done
2023-05-15 05:50:54.823866 INFO::countGenomicFeatures.R/countGenomicFeatures: starting...
2023-05-15 05:50:54.84787 DEBUG::tools.R/processChunks: starting...
2023-05-15 05:50:58.778696 DEBUG::tools.R/processChunks: waiting for chunkid=[  ] ...
2023-05-15 05:50:58.781308 DEBUG::tools.R/processChunks: starting chunkid= 1 ; see logfile= test/chunks/chunk_000001/logs/progress.log
2023-05-15 05:51:01.484381 DEBUG::tools.R/processChunks: done with chunkid= 1 ; elapsed.time= 0.045 minutes
2023-05-15 05:51:01.487939 DEBUG::tools.R/processChunks: done
2023-05-15 05:51:01.490286 INFO::countGenomicFeatures.R/mergeCounts: starting...
2023-05-15 05:51:01.503569 INFO::io.R/saveWithID: saving file= test/results/test.counts_exon.tab
2023-05-15 05:51:01.5145 INFO::io.R/saveWithID: saving file= test/results/test.counts_exon_disjoint.tab
2023-05-15 05:51:01.52247 INFO::io.R/saveWithID: saving file= test/results/test.counts_gene.tab
2023-05-15 05:51:01.528213 INFO::io.R/saveWithID: saving file= test/results/test.counts_gene_coding.tab
2023-05-15 05:51:01.534048 INFO::io.R/saveWithID: saving file= test/results/test.counts_gene_exonic.tab
2023-05-15 05:51:01.539612 INFO::io.R/saveWithID: saving file= test/results/test.counts_intergenic.tab
2023-05-15 05:51:01.548141 INFO::io.R/saveWithID: saving file= test/results/test.counts_intron.tab
2023-05-15 05:51:01.552598 INFO::countGenomicFeatures.R/mergeCounts: done
2023-05-15 05:51:01.557873 INFO::io.R/saveWithID: saving file= test/results/test.summary_counts.tab
2023-05-15 05:51:01.560386 INFO::reportPipelineQA.R/writeGenomicFeaturesReport: creating report of genomic features counts...
2023-05-15 05:51:01.843376 INFO::reportPipelineQA.R/writeGenomicFeaturesReport: done
2023-05-15 05:51:01.844712 INFO::countGenomicFeatures.R/countGenomicFeatures: done...
2023-05-15 05:51:01.938556 INFO::coverage.R/calculateCoverage: starting...
2023-05-15 05:51:01.944247 DEBUG::tools.R/processChunks: starting...
2023-05-15 05:51:05.712286 DEBUG::tools.R/processChunks: waiting for chunkid=[  ] ...
2023-05-15 05:51:05.714645 DEBUG::tools.R/processChunks: starting chunkid= 1 ; see logfile= test/chunks/chunk_000001/logs/progress.log
2023-05-15 05:51:08.130948 DEBUG::tools.R/processChunks: done with chunkid= 1 ; elapsed.time= 0.04 minutes
2023-05-15 05:51:08.134346 DEBUG::tools.R/processChunks: done
2023-05-15 05:51:10.167037 INFO::io.R/saveWithID: saving file= test/results/test.coverage.RData
2023-05-15 05:51:10.16936 INFO::coverage.R/saveCoverage: saving file= test/results/test.coverage.bw
2023-05-15 05:51:10.287019 INFO::io.R/saveWithID: saving file= test/results/test.summary_coverage.tab
2023-05-15 05:51:10.28898 INFO::coverage.R/calculateCoverage: done
2023-05-15 05:51:10.290767 INFO::analyzeVariants/analyzeVariants: starting ...
2023-05-15 05:51:10.386389 INFO::analyzeVariants.R/wrap.callVariants: Tallying variants...
2023-05-15 05:51:22.854392 INFO::analyzeVariants.R/wrap.callVariants: calling variants...
2023-05-15 05:51:22.950628 INFO::analyzeVariants.R/wrap.postFilerVariants: post filtering variants...
2023-05-15 05:51:22.972017 INFO::analyzeVariants.R/wrap.callVariants: Saving GRanges of raw and filtered variants...
2023-05-15 05:51:22.9738 INFO::io.R/saveWithID: saving file= test/results/test.raw_variants.RData
2023-05-15 05:51:22.977622 INFO::io.R/saveWithID: saving file= test/results/test.filtered_variants.RData
2023-05-15 05:51:22.979483 INFO::analyzeVariants.R/wrap.callVariants: ...done
2023-05-15 05:51:22.980853 INFO::analyzeVariants.R/writeVCF: writing vcf file...
2023-05-15 05:51:23.269512 INFO::analyzeVariants.R/writeVCF: ...done
2023-05-15 05:51:23.395689 INFO::io.R/saveWithID: saving file= test/results/test.summary_variants.tab
2023-05-15 05:51:23.398244 INFO::analyzeVariants/analyzeVariants: done
2023-05-15 05:51:23.405022 INFO::Pipeline run successful.
 done successfully.



Executing test function test.calculateTargetLength  ... checkConfig.R/checkConfig.template: loading template config= //tmp/Rtmpt80l8W/tp53-config.txt 
checkConfig.R/checkConfig.template: loading template config= /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/config/default-config.txt 
2023-05-15 05:51:23.727953 INFO::io.R/saveWithID: saving file= /tmp/Rtmpt80l8W/test.calcTargetLengths.2838c99eff219/results/test_pe.summary_target_lengths.tab
 done successfully.



Executing test function test.sclapply  ...  done successfully.



Executing test function test.tryKeepTraceback  ...  done successfully.



Executing test function test.truncateReads  ...  done successfully.



Executing test function test.truncateReads.trim5  ...  done successfully.

RUNIT TEST PROTOCOL -- Mon May 15 05:51:38 2023 
*********************************************** 
Number of test functions: 61 
Number of deactivated test functions: 10 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
HTSeqGenie unit testing - 61 test functions, 0 errors, 0 failures



Details 
*************************** 
Test Suite: HTSeqGenie unit testing 
Test function regexp: ^test.+ 
Test file regexp: ^runit.+\.[rR]$ 
Involved directory: 
/home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests 
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.alignReads.R 
test.alignReads: (2 checks) ... OK (32.44 seconds)
test.alignReads.sparsechunks: (2 checks) ... OK (20.96 seconds)
test.alignReadsOneSingleEnd: (2 checks) ... OK (0.61 seconds)
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.analyzeVariants.R 
test.annotateVariants : DEACTIVATED, Skipped annotateVariants() test
test.callVariantsVariantTools.genotype: (2 checks) ... OK (77.52 seconds)
test.wrap.callVariants: (4 checks) ... OK (11.21 seconds)
test.wrap.callVariants.filters: (2 checks) ... OK (21.98 seconds)
test.wrap.callVariants.which: (1 checks) ... OK (2.92 seconds)
test.writeVCF.NULL: (1 checks) ... OK (0.18 seconds)
test.writeVCF.vcfStat: (4 checks) ... OK (0.67 seconds)
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.bamUtils.R 
test.isFirstFragment: (5 checks) ... OK (0 seconds)
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.buildGenomicFeatures.R 
test.buildCountsGRangesList: (3 checks) ... OK (37.54 seconds)
test.generateSingleGeneDERs: (4 checks) ... OK (31.45 seconds)
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.calculateCoverage.R 
test.computeCoverage: (9 checks) ... OK (0.97 seconds)
test.isSparse: (5 checks) ... OK (0.13 seconds)
test.mergeCoverage: (1 checks) ... OK (1.73 seconds)
test.mergeCoverage.sparse: (2 checks) ... OK (1.38 seconds)
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.checkConfig.R 
test.checkConfig.analyzeVariants: (5 checks) ... OK (0.22 seconds)
test.findTemplate: (5 checks) ... OK (0.01 seconds)
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.config.R 
test.checkConfig: (5 checks) ... OK (0.44 seconds)
test.checkConfig.alignReads: (5 checks) ... OK (0.52 seconds)
test.getConfig: (14 checks) ... OK (0 seconds)
test.loadConfig: (2 checks) ... OK (0 seconds)
test.parseDCF: (7 checks) ... OK (0 seconds)
test.updateConfig: (1 checks) ... OK (0 seconds)
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.detectAdapterContam.R 
test.getAdapterSeqs: (5 checks) ... OK (0.05 seconds)
test.isAdapter: (5 checks) ... OK (0.16 seconds)
test.isAdapter3.primeEnd: (1 checks) ... OK (0.12 seconds)
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.detectRRNA.R 
test.detectRRNA: (1 checks) ... OK (0.38 seconds)
test.detectRRNA.paired_end: (1 checks) ... OK (0.54 seconds)
test.getRRNAIds: (1 checks) ... OK (0.33 seconds)
test.getRRNAIds_random: (1 checks) ... OK (0.25 seconds)
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.filterQuality.R 
test.filterByLength: (2 checks) ... OK (0.08 seconds)
test.isAboveQualityThresh: (3 checks) ... OK (0.06 seconds)
test.trimTailsByQuality: (4 checks) ... OK (0.05 seconds)
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.gatk.R 
test.callVariantsGATK : DEACTIVATED, callVariantsGATK() tests need gatk.path option set
test.callVariantsGATK.withFiltering : DEACTIVATED, callVariantsGATK() tests need gatk.path option set
test.checkGATKJar : DEACTIVATED, checkGATKJar() test needs gatk.path option set
test.excludeVariantsByRegion: (3 checks) ... OK (0.22 seconds)
test.gatk : DEACTIVATED, gatk() tests need gatk.path option set
test.realignIndels : DEACTIVATED, test.realignIndels() tests need gatk.path option set
test.realignIndelsGATK : DEACTIVATED, realignIndelsGATK() tests need gatk.path option set
test.realignIndelsGATK.parallel : DEACTIVATED, realignIndelsGATK() tests need gatk.path option set
test_zipUp: (3 checks) ... OK (0 seconds)
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.io.R 
test.FastQStreamer.getReads.pefq: (1 checks) ... OK (0.28 seconds)
test.FastQStreamer.getReads.pefq.subsample: (1 checks) ... OK (0.41 seconds)
test.FastQStreamer.getReads.segz: (1 checks) ... OK (0.23 seconds)
test.FastQStreamer.getReads.truncated: (1 checks) ... OK (0.27 seconds)
test.FastQStreamer.subsampler.isdeterministic: (3 checks) ... OK (0.76 seconds)
test.createTmpDir: (4 checks) ... OK (0 seconds)
test.detectQualityInFASTQFile: (3 checks) ... OK (0.18 seconds)
test.getObjectFilename: (4 checks) ... OK (0.01 seconds)
test.safeUnlink: (2 checks) ... OK (0.01 seconds)
test.writeAudit: (0 checks) ... OK (0.26 seconds)
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.mergeLanes.R 
test.mergeLanes: (6 checks) ... OK (68.44 seconds)
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.picard.R 
test.markDuplicates : DEACTIVATED, Skipped markDuplicates() test
test.markDuplicates_w_outfile : DEACTIVATED, Skipped markDuplicates() test
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.preprocessReads.R 
test.preprocessReads: (5 checks) ... OK (10.24 seconds)
test.preprocessReads.minichunks: (5 checks) ... OK (15.84 seconds)
test.preprocessReads_single_end: (5 checks) ... OK (8.54 seconds)
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.readRNASeqEnds.R 
test.readRNASeqEnds: (6 checks) ... OK (0.11 seconds)
test.readRNASeqEnds.dupmark: (2 checks) ... OK (0.12 seconds)
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.reportPipelineQA.R 
test.how_many: (3 checks) ... OK (0 seconds)
test.plotDF: (3 checks) ... OK (0.15 seconds)
test.relativeBarPlot: (2 checks) ... OK (0.06 seconds)
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.runPipeline.R 
test.runPipeline: (1 checks) ... OK (50.85 seconds)
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.targetLengths.R 
test.calculateTargetLength: (4 checks) ... OK (0.32 seconds)
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.tools.R 
test.sclapply: (10 checks) ... OK (14.77 seconds)
test.tryKeepTraceback: (3 checks) ... OK (0.01 seconds)
--------------------------- 
Test file: /home/biocbuild/R/R-4.3.0-bin/site-library/HTSeqGenie/unitTests/runit.trimReads.R 
test.truncateReads: (7 checks) ... OK (0.08 seconds)
test.truncateReads.trim5: (15 checks) ... OK (0.23 seconds) 

runTests.R: OK !
There were 50 or more warnings (use warnings() to see the first 50)
&gt; 
&gt; proc.time()
   user  system elapsed 
315.201  82.920 439.137 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">HTSeqGenie.Rcheck/HTSeqGenie-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>HTSeqGenie</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>buildGenomicFeaturesFromTxDb</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>runPipeline</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
