<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for ISoLDE on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><B>[<A href="../index.html#I">I</A>]</B></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-23 01:34:06 -0000 (Thu, 23 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4245</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for ISoLDE on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the ISoLDE package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/ISoLDE.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>996</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">ISoLDE</A>&nbsp;1.27.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/ISoLDE">landing page</A>)</SPAN><BR>Christelle Reynès<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-21&nbsp;12:29:53&nbsp;-0000&nbsp;(Tue,&nbsp;21&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/ISoLDE</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">00ce572</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:13:17&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: ISoLDE</TD></TR>
<TR><TD><B>Version</B>: 1.27.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:ISoLDE.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings ISoLDE_1.27.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-22 07:07:20 -0000 (Wed, 22 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-22 07:08:42 -0000 (Wed, 22 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 82.1 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: ISoLDE.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 1</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:ISoLDE.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings ISoLDE_1.27.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/ISoLDE.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘ISoLDE/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘ISoLDE’ version ‘1.27.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘ISoLDE’ can be installed ... OK
* used C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... OK
* checking package directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... OK
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... WARNING
Files in the &#x27;vignettes&#x27; directory but no files in &#x27;inst/doc&#x27;:
  ‘ISoLDE_result_on_example.pdf’ ‘vignette.Rnw’ ‘workflow.pdf’
Files named as vignettes but with no recognized vignette engine:
   ‘vignettes/vignette.Rnw’
(Is a VignetteBuilder field missing?)
* checking examples ... OK
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testsIsolde.R’
 OK
* checking PDF version of manual ... OK
* DONE

Status: 1 WARNING, 1 NOTE
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/ISoLDE.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">ISoLDE.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL ISoLDE
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘ISoLDE’ ...
** using staged installation
** libs
using C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG   -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c Isolde.c -o Isolde.o
gcc -shared -L/home/biocbuild/bbs-3.17-bioc/R/lib -L/usr/local/lib -o ISoLDE.so Isolde.o -fopenmp -L/home/biocbuild/bbs-3.17-bioc/R/lib -lR
installing to /home/biocbuild/bbs-3.17-bioc/R/library/00LOCK-ISoLDE/00new/ISoLDE/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (ISoLDE)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">ISoLDE.Rcheck/tests/testsIsolde.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(ISoLDE)
&gt; 
&gt; targetfile &lt;- system.file(&quot;extdata&quot;, &quot;target_file.txt&quot;, package = &quot;ISoLDE&quot;)
&gt; rawfile &lt;- system.file(&quot;extdata&quot;, &quot;rawASRcounts_file.txt&quot;, package = &quot;ISoLDE&quot;)
&gt; normfile &lt;- system.file(&quot;extdata&quot;, &quot;normASRcounts_file.txt&quot;, package = &quot;ISoLDE&quot;)
&gt; 
&gt; rawASRcounts &lt;- readRawInput(raw_file = rawfile, del = &quot;tab&quot;, rownames = TRUE, colnames = TRUE)
Reading raw input file...
Done
&gt; normASRcounts &lt;- readNormInput(norm_file = normfile, del = &quot;tab&quot;, rownames = TRUE, colnames = TRUE, dec = &quot;.&quot;)
Reading normalized input file...
Done
&gt; target &lt;- readTarget(target_file = targetfile, asr_counts = rawASRcounts, del = &quot;tab&quot;)
Reading target file...
Done
&gt; 
&gt; res_filterT &lt;- filterT(rawASRcounts = rawASRcounts, normASRcounts = normASRcounts, target = target, bias=&quot;parental&quot;)
Filtering your data...
Filtering threshold is 3
Done
&gt; filteredASRcounts &lt;- res_filterT$filteredASRcounts
&gt; resiso&lt;-isolde_test(bias = &quot;parental&quot;, asr_counts = filteredASRcounts, target = target, graph = TRUE, text = FALSE, ext = &quot;png&quot;, nboot = 3000,pcore=1)
Warning: pcore is a percentage between 0 and 100, be sure you want to use 1% of your available cores.
ISoLDE is searching for allele-specific gene expression due to parental biases...
Done
According to the satisfying number of replicates the full method can be applied. See the vignette for more details
Testing...
This step can last for a few minutes. Please be patient :)
Note: the full method uses a bootstrap step which means some results might change from one test to another.
Open MP enabled 
Have Cores : 32 
UseCores : 1 
nu0= 0.100000 
Open MP enabled 
Have Cores : 32 
UseCores : 1 
lambda0 : 1.600000 
Open MP enabled 
Have Cores : 32 
UseCores : 1 
nu0= 0.100000 
Open MP enabled 
Have Cores : 32 
UseCores : 1 
lambda0 : 1.600000 
Open MP enabled 
Have Cores : 32 
UseCores : 1 
nu0= 0.100000 
Open MP enabled 
Have Cores : 32 
UseCores : 1 
lambda0 : 1.600000 
Done
45 allele specific expressed (ASE) genes found and written in file ISoLDE_result_ASE_02-22-2023_07-08-35.tsv.
6093 biallelic (BA) genes found and written in file ISoLDE_result_BA_02-22-2023_07-08-35.tsv.
39 Undetermined (UN) genes found and written in file ISoLDE_result_UN_02-22-2023_07-08-35.tsv.
Among these undetermined (UN) genes, 18 genes are coherent.
These genes correspond to the ones with the flag column set on &quot;FLAG_consistency&quot; of &quot;FLAG_significance&quot; in the undetermined genes file. See the vignette for more information on flags.
0 genes have been filtered due to few reads.
&gt; res_filterT &lt;- filterT(rawASRcounts = rawASRcounts, normASRcounts = normASRcounts, target = target, bias=&quot;strain&quot;)
Filtering your data...
Filtering threshold is 3
Done
&gt; filteredASRcounts &lt;- res_filterT$filteredASRcounts
&gt; resiso&lt;-isolde_test(bias = &quot;strain&quot;, asr_counts = filteredASRcounts, target = target, graph = TRUE, text = FALSE, ext = &quot;png&quot;, nboot = 3000, method = &quot;threshold&quot;)
ISoLDE is searching for allele-specific gene expression due to strain biases...
The predefined threshold method is about to proceed, but we strongly recommand the bootstrap method because ISoLDE detected more than 2 replicates in each cross.
Testing...
Done
308 allele specific expressed (ASE) genes found and written in file ISoLDE_result_ASE_02-22-2023_07-08-37.tsv.
4622 biallelic (BA) genes found and written in file ISoLDE_result_BA_02-22-2023_07-08-37.tsv.
1330 Undetermined (UN) genes found and written in file ISoLDE_result_UN_02-22-2023_07-08-37.tsv.
Among these undetermined (UN) genes, 913 genes are coherent.
These genes correspond to the ones with the flag column set on &quot;FLAG_consistency&quot; of &quot;FLAG_significance&quot; in the undetermined genes file. See the vignette for more information on flags.
0 genes have been filtered due to few reads.
&gt; 
&gt; proc.time()
   user  system elapsed 
 54.297   0.977  55.259 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">ISoLDE.Rcheck/ISoLDE-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>filterT</TD><TD style="text-align: right;">0.495</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.503</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filteredASRcounts</TD><TD style="text-align: right;">0.018</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.018</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>isolde_test</TD><TD style="text-align: right;">1.931</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">1.979</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>normASRcounts</TD><TD style="text-align: right;">0.492</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.496</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rawASRcounts</TD><TD style="text-align: right;">0.135</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.135</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readNormInput</TD><TD style="text-align: right;">0.472</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.477</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readRawInput</TD><TD style="text-align: right;">0.104</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.104</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readTarget</TD><TD style="text-align: right;">0.081</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.081</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
