<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for oligoClasses on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.17</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><B>[<A href="..#O">O</A>]</B></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-03-01 07:38:46 -0000 (Wed, 01 Mar 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4266</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for oligoClasses on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the oligoClasses package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/oligoClasses.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1370</B>/2171</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">oligoClasses</A>&nbsp;1.61.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/oligoClasses">landing page</A>)</SPAN><BR>Benilton Carvalho <beniltoncarvalho@gmail.com> and Robert Scharpf<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-27&nbsp;07:53:22&nbsp;-0000&nbsp;(Mon,&nbsp;27&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/oligoClasses</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">65d0f5c</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:03:35&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: oligoClasses</TD></TR>
<TR><TD><B>Version</B>: 1.61.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:oligoClasses.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings oligoClasses_1.61.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-28 10:09:32 -0000 (Tue, 28 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-28 10:13:55 -0000 (Tue, 28 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 263.3 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: oligoClasses.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 3</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:oligoClasses.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings oligoClasses_1.61.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/oligoClasses.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘oligoClasses/DESCRIPTION’ ... OK
* this is package ‘oligoClasses’ version ‘1.61.0’
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Packages which this enhances but not available for checking:
  &#x27;doMPI&#x27;, &#x27;doRedis&#x27;
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘oligoClasses’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
Namespace in Imports field not imported from: ‘RSQLite’
  All declared Imports should be used.
Unexported object imported by a &#x27;:::&#x27; call: ‘Biobase:::assayDataEnvLock’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
getSequenceLengths: no visible binding for global variable ‘seqlengths’
chromosome,gSetList: no visible global function definition for
  ‘chromosomeList’
coerce,CNSet-CopyNumberSet: no visible global function definition for
  ‘totalCopynumber’
geometry,FeatureSet: no visible global function definition for ‘getPD’
Undefined global functions or variables:
  chromosomeList getPD seqlengths totalCopynumber
* checking Rd files ... WARNING
checkRd: (5) BeadStudioSet-class.Rd:52: \item in \describe must have non-empty label
checkRd: (5) BeadStudioSet-class.Rd:53-54: \item in \describe must have non-empty label
checkRd: (5) BeadStudioSet-class.Rd:55-57: \item in \describe must have non-empty label
checkRd: (5) BeadStudioSet-class.Rd:58: \item in \describe must have non-empty label
checkRd: (5) BeadStudioSet-class.Rd:59-60: \item in \describe must have non-empty label
checkRd: (5) BeadStudioSet-class.Rd:63: \item in \describe must have non-empty label
checkRd: (5) BeadStudioSet-class.Rd:64-65: \item in \describe must have non-empty label
checkRd: (5) BeadStudioSet-class.Rd:66-67: \item in \describe must have non-empty label
checkRd: (5) BeadStudioSet-class.Rd:68: \item in \describe must have non-empty label
checkRd: (5) BeadStudioSetList-class.Rd:44-58: \item in \describe must have non-empty label
checkRd: (5) BeadStudioSetList-class.Rd:64-69: \item in \describe must have non-empty label
checkRd: (5) BeadStudioSetList-class.Rd:71-78: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:85-86: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:88-89: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:91-93: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:96-98: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:100-101: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:103-105: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:107: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:109: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:113: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:115-116: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:118: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:120-121: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:124-126: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:131-132: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:135-136: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:138: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:140-142: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:144-145: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:147-148: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:150-151: \item in \describe must have non-empty label
checkRd: (5) CNSet-class.Rd:153: \item in \describe must have non-empty label
checkRd: (5) GRanges-methods.Rd:28-38: \item in \describe must have non-empty label
checkRd: (5) GRanges-methods.Rd:49-60: \item in \describe must have non-empty label
checkRd: (5) GRanges-methods.Rd:62-67: \item in \describe must have non-empty label
checkRd: (5) GRanges-methods.Rd:69-75: \item in \describe must have non-empty label
checkRd: (5) GRanges-methods.Rd:77-85: \item in \describe must have non-empty label
checkRd: (5) GenomeAnnotatedDataFrame-class.Rd:35-42: \item in \describe must have non-empty label
checkRd: (5) GenomeAnnotatedDataFrame-class.Rd:44-54: \item in \describe must have non-empty label
checkRd: (5) GenomeAnnotatedDataFrame-class.Rd:56-61: \item in \describe must have non-empty label
checkRd: (5) GenomeAnnotatedDataFrame-class.Rd:68-73: \item in \describe must have non-empty label
checkRd: (5) GenomeAnnotatedDataFrame-class.Rd:75-81: \item in \describe must have non-empty label
checkRd: (5) GenomeAnnotatedDataFrame-class.Rd:84-89: \item in \describe must have non-empty label
checkRd: (5) GenomeAnnotatedDataFrame-class.Rd:91-100: \item in \describe must have non-empty label
checkRd: (5) SnpSet2-class.Rd:55-60: \item in \describe must have non-empty label
checkRd: (5) SnpSet2-class.Rd:62-67: \item in \describe must have non-empty label
checkRd: (5) SnpSet2-class.Rd:69-73: \item in \describe must have non-empty label
checkRd: (5) chromosome-methods.Rd:44-45: \item in \describe must have non-empty label
checkRd: (5) gSet-class.Rd:43-44: \item in \describe must have non-empty label
checkRd: (5) gSet-class.Rd:45-46: \item in \describe must have non-empty label
checkRd: (5) gSet-class.Rd:47-49: \item in \describe must have non-empty label
checkRd: (5) gSet-class.Rd:50-52: \item in \describe must have non-empty label
checkRd: (5) gSet-class.Rd:53-59: \item in \describe must have non-empty label
checkRd: (5) gSet-class.Rd:61-66: \item in \describe must have non-empty label
checkRd: (5) gSet-class.Rd:69-70: \item in \describe must have non-empty label
checkRd: (5) gSet-class.Rd:72-77: \item in \describe must have non-empty label
checkRd: (5) gSet-class.Rd:79-80: \item in \describe must have non-empty label
checkRd: (5) gSet-class.Rd:81-86: \item in \describe must have non-empty label
checkRd: (5) gSetList-class.Rd:51-56: \item in \describe must have non-empty label
checkRd: (5) gSetList-class.Rd:58-64: \item in \describe must have non-empty label
checkRd: (5) gSetList-class.Rd:66-71: \item in \describe must have non-empty label
checkRd: (5) gSetList-class.Rd:74-79: \item in \describe must have non-empty label
checkRd: (5) gSetList-class.Rd:89-99: \item in \describe must have non-empty label
checkRd: (5) oligoSnpSet-methods.Rd:28-39: \item in \describe must have non-empty label
checkRd: (5) oligoSnpSet-methods.Rd:41-46: \item in \describe must have non-empty label
checkRd: (5) oligoSnpSet-methods.Rd:48-54: \item in \describe must have non-empty label
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... WARNING
Undocumented S4 methods:
  generic &#x27;[&#x27; and siglist &#x27;CNSet,ANY,ANY,ANY&#x27;
  generic &#x27;[&#x27; and siglist &#x27;gSetList,ANY,ANY,ANY&#x27;
All user-level objects in a package (including S4 classes and methods)
should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in Makefiles ... OK
* checking for GNU extensions in Makefiles ... OK
* checking include directives in Makefiles ... OK
* checking files in ‘vignettes’ ... WARNING
Files in the &#x27;vignettes&#x27; directory but no files in &#x27;inst/doc&#x27;:
  ‘scriptsForExampleData/CreateExampleData.R’
Package has no Sweave vignette sources and no VignetteBuilder field.
* checking examples ... OK
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘doRUnit.R’
 OK
* checking PDF version of manual ... OK
* DONE

Status: 3 WARNINGs, 3 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/oligoClasses.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">oligoClasses.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL oligoClasses
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘oligoClasses’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (oligoClasses)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">oligoClasses.Rcheck/tests/doRUnit.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; ## from xmapcore package
&gt; if( require( &quot;RUnit&quot;, quietly=TRUE ) ) {
+         ## loading Biobase below b/c we&#x27;re simply Import:ing it, rather than Depend:ing on it
+         ## then functions from there are not visible downstream
+         library(Biobase)
+ 	pkg &lt;- &quot;oligoClasses&quot;
+ 
+ 	if( Sys.getenv( &quot;RCMDCHECK&quot; ) == &quot;FALSE&quot; ) {
+ 		path &lt;- file.path( getwd(), &quot;..&quot;, &quot;inst&quot;, &quot;unitTests&quot; )
+ 	} else {
+ 		path &lt;- system.file( package=pkg, &quot;unitTests&quot; )
+ 	}
+ 
+ 	cat( &quot;\nRunning unit tests\n&quot; )
+ 	print( list( pkg=pkg, getwd=getwd(), pathToUnitTests=path ) )
+ 	library( package=pkg, character.only=TRUE )
+ 
+ 	##xmap.clear.cache()
+ 
+ 	##Fail on warnings
+ 	##options( warn=2 )
+ 	options(warn=0)
+ 
+ 	## Get the pattern (if there is one?)
+ 	patt &lt;- Sys.getenv( &quot;RUNITFILEPATTERN&quot; )
+ 	if( is.null( patt ) || nchar( patt ) == 0 ) {
+ 		testSuite &lt;- defineTestSuite(name=paste( pkg, &quot;unit testing&quot; ),
+ 					     dirs=path,
+ 					     testFileRegexp=paste( &quot;^test.+&quot;, patt, &quot;\\.[rR]$&quot;, sep=&quot;&quot; ))
+ 	} else {
+ 		##testSuite &lt;- defineTestSuite( name=paste( pkg, &quot;unit testing&quot; ), testFileRegexp=paste( &quot;^runit\\.&quot;, patt, &quot;\\.[rR]$&quot;, sep=&quot;&quot; ), dirs=path )
+ 		testSuite &lt;- defineTestSuite(name=paste( pkg, &quot;unit testing&quot; ),
+ 					     testFileRegexp=paste( &quot;^test.+&quot;, patt, &quot;\\.[rR]$&quot;, sep=&quot;&quot; ),
+ 					     dirs=path )
+ 	}
+ 	tests &lt;- runTestSuite( testSuite )
+ 
+ 	pathReport &lt;- file.path( path, &quot;report&quot; )
+ 
+ 	cat( &quot;------------------- UNIT TEST SUMMARY ---------------------\n\n&quot; )
+ 	printTextProtocol( tests, showDetails=FALSE )
+ 	printTextProtocol( tests, showDetails=FALSE, fileName=paste( pathReport, &quot;Summary.txt&quot;, sep=&quot;&quot; ) )
+ 	printTextProtocol( tests, showDetails=TRUE,  fileName=paste( pathReport, &quot;.txt&quot;, sep=&quot;&quot; ) )
+ 
+ 	printHTMLProtocol( tests, fileName=paste( pathReport, &quot;.html&quot;, sep=&quot;&quot; ) )
+ 
+ 	tmp &lt;- getErrors( tests )
+ 	if( tmp$nFail &gt; 0 | tmp$nErr &gt; 0 ){
+ 		stop( paste( &quot;\n\nunit testing failed (#test failures: &quot;, tmp$nFail, &quot;, #R errors: &quot;,  tmp$nErr, &quot;)\n\n&quot;, sep=&quot;&quot;))
+ 	}
+ } else {
+ 	warning( &quot;cannot run unit tests -- package RUnit is not available&quot; )
+ }
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


Running unit tests
$pkg
[1] &quot;oligoClasses&quot;

$getwd
[1] &quot;/home/biocbuild/bbs-3.17-bioc/meat/oligoClasses.Rcheck/tests&quot;

$pathToUnitTests
[1] &quot;/home/biocbuild/bbs-3.17-bioc/R/library/oligoClasses/unitTests&quot;

Welcome to oligoClasses version 1.61.0


Executing test function test_annotation  ... Annotation for genomewidesnp6Crlmm version 1.0.7 supports UCSC builds hg18 and hg19.
Build  requested, but only build hg18 is available.
 done successfully.



Executing test function test_BafLrrSetList  ...  done successfully.



Executing test function test_BeadStudioSet  ...  done successfully.



Executing test function test_CNSet_construction  ...  done successfully.



Executing test function test_CopyNumberSet_construction  ... Loading required package: pd.mapping50k.hind240
Loading required package: Biostrings
Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: XVector
Loading required package: GenomeInfoDb

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: RSQLite
Loading required package: oligo
================================================================================
Welcome to oligo version 1.63.0
================================================================================
Loading required package: DBI
Loading required package: pd.mapping50k.xba240
 done successfully.



Executing test function test_GenomeAnnotatedDataFrameWithFF  ... Loading required package: ff
Loading required package: bit

Attaching package: &#x27;bit&#x27;

The following object is masked from &#x27;package:base&#x27;:

    xor

Attaching package ff
- getOption(&quot;fftempdir&quot;)==&quot;/tmp/Rtmperf8KF/ff&quot;

- getOption(&quot;ffextension&quot;)==&quot;ff&quot;

- getOption(&quot;ffdrop&quot;)==TRUE

- getOption(&quot;fffinonexit&quot;)==TRUE

- getOption(&quot;ffpagesize&quot;)==65536

- getOption(&quot;ffcaching&quot;)==&quot;mmnoflush&quot;  -- consider &quot;ffeachflush&quot; if your system stalls on large writes

- getOption(&quot;ffbatchbytes&quot;)==16777216 -- consider a different value for tuning your system

- getOption(&quot;ffmaxbytes&quot;)==536870912 -- consider a different value for tuning your system

================================================================================
Large dataset support for &#x27;oligo/crlmm&#x27;: Enabled
    - Probesets: 20,000
    - Samples..: 100
    - Path.....: /home/biocbuild/bbs-3.17-bioc/meat/oligoClasses.Rcheck/tests
================================================================================

Attaching package: &#x27;ff&#x27;

The following objects are masked from &#x27;package:Biostrings&#x27;:

    mismatch, pattern

The following objects are masked from &#x27;package:utils&#x27;:

    write.csv, write.csv2

The following objects are masked from &#x27;package:base&#x27;:

    is.factor, is.ordered

 done successfully.



Executing test function test_GenomeAnnotatedDataFrame_construction  ...  done successfully.



Executing test function test_dataExamples  ...  done successfully.



Executing test function test_oligoSnpSet_construction  ...  done successfully.



Executing test function test_conversions  ...  done successfully.



Executing test function test_makeFeatureRanges  ...  done successfully.



Executing test function test_oligoSnpSet  ...  done successfully.

------------------- UNIT TEST SUMMARY ---------------------

RUNIT TEST PROTOCOL -- Tue Feb 28 10:13:48 2023 
*********************************************** 
Number of test functions: 12 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
oligoClasses unit testing - 12 test functions, 0 errors, 0 failures
Warning messages:
1: In RNGkind(kind = testSuite$rngKind, normal.kind = testSuite$rngNormalKind) :
  RNGkind: Marsaglia-Multicarry has poor statistical properties
2: In RNGkind(kind = testSuite$rngKind, normal.kind = testSuite$rngNormalKind) :
  RNGkind: severe deviations from normality for Kinderman-Ramage + Marsaglia-Multicarry
3: In RNGkind(kind = testSuite$rngKind, normal.kind = testSuite$rngNormalKind) :
  RNGkind: Marsaglia-Multicarry has poor statistical properties
4: In RNGkind(kind = testSuite$rngKind, normal.kind = testSuite$rngNormalKind) :
  RNGkind: severe deviations from normality for Kinderman-Ramage + Marsaglia-Multicarry
5: In RNGkind(kind = testSuite$rngKind, normal.kind = testSuite$rngNormalKind) :
  RNGkind: Marsaglia-Multicarry has poor statistical properties
6: In RNGkind(kind = testSuite$rngKind, normal.kind = testSuite$rngNormalKind) :
  RNGkind: severe deviations from normality for Kinderman-Ramage + Marsaglia-Multicarry
7: In RNGkind(kind = testSuite$rngKind, normal.kind = testSuite$rngNormalKind) :
  RNGkind: Marsaglia-Multicarry has poor statistical properties
8: In RNGkind(kind = testSuite$rngKind, normal.kind = testSuite$rngNormalKind) :
  RNGkind: severe deviations from normality for Kinderman-Ramage + Marsaglia-Multicarry
&gt; 
&gt; proc.time()
   user  system elapsed 
 22.142   0.954  23.939 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">oligoClasses.Rcheck/oligoClasses-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>AlleleSet-class</TD><TD style="text-align: right;">0.150</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.157</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>AssayData-methods</TD><TD style="text-align: right;">1.931</TD><TD style="text-align: right;">0.088</TD><TD style="text-align: right;">2.036</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>AssayDataList</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>BeadStudioSet-class</TD><TD style="text-align: right;">0.054</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.054</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>CNSet-class</TD><TD style="text-align: right;">0.041</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.042</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>CopyNumberSet-class</TD><TD style="text-align: right;">0.027</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.030</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>CopyNumberSet-methods</TD><TD style="text-align: right;">0.133</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">0.187</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>FeatureSetExtensions-class</TD><TD style="text-align: right;">0.156</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.160</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GRanges-methods</TD><TD style="text-align: right;">0.406</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.406</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GenomeAnnotatedDataFrameFrom-methods</TD><TD style="text-align: right;">0.806</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">0.846</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SnpSet-methods</TD><TD style="text-align: right;">0.027</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.026</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SnpSet2-class</TD><TD style="text-align: right;">0.037</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.037</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SnpSuperSet-class</TD><TD style="text-align: right;">0.055</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.059</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>affyPlatforms</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>batch</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>celfileDate</TD><TD style="text-align: right;">0.022</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.026</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>celfileName</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>checkExists</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>checkOrder</TD><TD style="text-align: right;">0.205</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.205</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>chromosome-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>chromosome2integer</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>clusterOpts</TD><TD style="text-align: right;">0.834</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.843</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>data-efsExample</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>data-scqsExample</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>data-sfsExample</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>data-sqsExample</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>db</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ff_matrix</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ff_or_matrix-class</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fileConnections</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>flags</TD><TD style="text-align: right;">0.033</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.034</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>gSet-class</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>gSetList-class</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genomeBuild</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>geometry-methods</TD><TD style="text-align: right;">0.748</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">0.792</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getBar</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getSequenceLengths</TD><TD style="text-align: right;">0.114</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.123</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>i2p_p2i</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>integerMatrix</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>is.ffmatrix</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>isPackageLoaded</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>kind</TD><TD style="text-align: right;">0.290</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.294</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>largeObjects</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ldOpts</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>library2</TD><TD style="text-align: right;">0.030</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>list.celfiles</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>locusLevelData</TD><TD style="text-align: right;">0.065</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.065</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeFeatureGRanges</TD><TD style="text-align: right;">0.325</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.336</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>oligoSetExample</TD><TD style="text-align: right;">0.155</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.167</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pdPkgFromBioC</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>requireAnnotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>splitVec</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
