<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for aroma.light on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><B>[<A href="../index.html#A">A</A>]</B></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-13 06:49:13 -0000 (Mon, 13 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4222</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for aroma.light on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the aroma.light package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/aroma.light.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>73</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">aroma.light</A>&nbsp;3.29.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/aroma.light">landing page</A>)</SPAN><BR>Henrik Bengtsson<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-11&nbsp;01:58:29&nbsp;-0000&nbsp;(Sat,&nbsp;11&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/aroma.light</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">f38e878</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;14:41:49&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: aroma.light</TD></TR>
<TR><TD><B>Version</B>: 3.29.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:aroma.light.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings aroma.light_3.29.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-11 11:00:55 -0000 (Sat, 11 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-11 11:02:34 -0000 (Sat, 11 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 99.2 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: aroma.light.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:aroma.light.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings aroma.light_3.29.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/aroma.light.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘aroma.light/DESCRIPTION’ ... OK
* this is package ‘aroma.light’ version ‘3.29.0’
* package encoding: latin1
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  inst/rsp/.rspPlugins
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘aroma.light’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... NOTE
checkRd: (-1) averageQuantile.Rd:49: Escaped LaTeX specials: \&amp;
checkRd: (-1) normalizeQuantileRank.Rd:97: Escaped LaTeX specials: \&amp;
checkRd: (-1) normalizeQuantileRank.matrix.Rd:83: Escaped LaTeX specials: \&amp;
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                    user system elapsed
normalizeCurveFit 11.549  0.136  11.686
normalizeAffine   11.326  0.035  11.363
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘backtransformAffine.matrix.R’
  Running ‘backtransformPrincipalCurve.matrix.R’
  Running ‘callNaiveGenotypes.R’
  Running ‘distanceBetweenLines.R’
  Running ‘findPeaksAndValleys.R’
  Running ‘fitPrincipalCurve.matrix.R’
  Running ‘fitXYCurve.matrix.R’
  Running ‘iwpca.matrix.R’
  Running ‘likelihood.smooth.spline.R’
  Running ‘medianPolish.matrix.R’
  Running ‘normalizeAffine.matrix.R’
  Running ‘normalizeAverage.list.R’
  Running ‘normalizeAverage.matrix.R’
  Running ‘normalizeCurveFit.matrix.R’
  Running ‘normalizeDifferencesToAverage.R’
  Running ‘normalizeFragmentLength-ex1.R’
  Running ‘normalizeFragmentLength-ex2.R’
  Running ‘normalizeQuantileRank.list.R’
  Running ‘normalizeQuantileRank.matrix.R’
  Running ‘normalizeQuantileSpline.matrix.R’
  Running ‘normalizeTumorBoost,flavors.R’
  Running ‘normalizeTumorBoost.R’
  Running ‘robustSmoothSpline.R’
  Running ‘rowAverages.matrix.R’
  Running ‘sampleCorrelations.matrix.R’
  Running ‘sampleTuples.R’
  Running ‘wpca.matrix.R’
  Running ‘wpca2.matrix.R’
 OK
* checking PDF version of manual ... OK
* DONE

Status: 2 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/aroma.light.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">aroma.light.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL aroma.light
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘aroma.light’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (aroma.light)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">aroma.light.Rcheck/tests/backtransformAffine.matrix.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; X &lt;- matrix(1:8, nrow=4, ncol=2)
&gt; X[2,2] &lt;- NA_integer_
&gt; 
&gt; print(X)
     [,1] [,2]
[1,]    1    5
[2,]    2   NA
[3,]    3    7
[4,]    4    8
&gt; 
&gt; # Returns a 4x2 matrix
&gt; print(backtransformAffine(X, a=c(1,5)))
     [,1] [,2]
[1,]    0    0
[2,]    1   NA
[3,]    2    2
[4,]    3    3
&gt; 
&gt; # Returns a 4x2 matrix
&gt; print(backtransformAffine(X, b=c(1,1/2)))
     [,1] [,2]
[1,]    1   10
[2,]    2   NA
[3,]    3   14
[4,]    4   16
&gt; 
&gt; # Returns a 4x2 matrix
&gt; print(backtransformAffine(X, a=matrix(1:4,ncol=1)))
     [,1] [,2]
[1,]    0    4
[2,]    0   NA
[3,]    0    4
[4,]    0    4
&gt; 
&gt; # Returns a 4x2 matrix
&gt; print(backtransformAffine(X, a=matrix(1:3,ncol=1)))
     [,1] [,2]
[1,]    0    4
[2,]    0   NA
[3,]    0    4
[4,]    3    7
&gt; 
&gt; # Returns a 4x2 matrix
&gt; print(backtransformAffine(X, a=matrix(1:2,ncol=1), b=c(1,2)))
     [,1] [,2]
[1,]    0    2
[2,]    0   NA
[3,]    2    3
[4,]    2    3
&gt; 
&gt; # Returns a 4x1 matrix
&gt; print(backtransformAffine(X, b=c(1,1/2), project=TRUE))
     [,1]
[1,]  2.8
[2,]  1.6
[3,]  5.2
[4,]  6.4
&gt; 
&gt; # If the columns of X are identical, and a identity
&gt; # backtransformation is applied and projected, the
&gt; # same matrix is returned.
&gt; X &lt;- matrix(1:4, nrow=4, ncol=3)
&gt; Y &lt;- backtransformAffine(X, b=c(1,1,1), project=TRUE)
&gt; print(X)
     [,1] [,2] [,3]
[1,]    1    1    1
[2,]    2    2    2
[3,]    3    3    3
[4,]    4    4    4
&gt; print(Y)
     [,1]
[1,]    1
[2,]    2
[3,]    3
[4,]    4
&gt; stopifnot(sum(X[,1]-Y) &lt;= .Machine$double.eps)
&gt; 
&gt; 
&gt; # If the columns of X are identical, and a identity
&gt; # backtransformation is applied and projected, the
&gt; # same matrix is returned.
&gt; X &lt;- matrix(1:4, nrow=4, ncol=3)
&gt; X[,2] &lt;- X[,2]*2; X[,3] &lt;- X[,3]*3
&gt; print(X)
     [,1] [,2] [,3]
[1,]    1    2    3
[2,]    2    4    6
[3,]    3    6    9
[4,]    4    8   12
&gt; Y &lt;- backtransformAffine(X, b=c(1,2,3))
&gt; print(Y)
     [,1] [,2] [,3]
[1,]    1    1    1
[2,]    2    2    2
[3,]    3    3    3
[4,]    4    4    4
&gt; Y &lt;- backtransformAffine(X, b=c(1,2,3), project=TRUE)
&gt; print(Y)
     [,1]
[1,]    1
[2,]    2
[3,]    3
[4,]    4
&gt; stopifnot(sum(X[,1]-Y) &lt;= .Machine$double.eps)
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.286   0.056   0.325 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/backtransformPrincipalCurve.matrix.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; # Consider the case where K=4 measurements have been done
&gt; # for the same underlying signals &#x27;x&#x27;.  The different measurements
&gt; # have different systematic variation
&gt; #
&gt; #   y_k = f(x_k) + eps_k; k = 1,...,K.
&gt; #
&gt; # In this example, we assume non-linear measurement functions
&gt; #
&gt; #   f(x) = a + b*x + x^c + eps(b*x)
&gt; #
&gt; # where &#x27;a&#x27; is an offset, &#x27;b&#x27; a scale factor, and &#x27;c&#x27; an exponential.
&gt; # We also assume heteroscedastic zero-mean noise with standard
&gt; # deviation proportional to the rescaled underlying signal &#x27;x&#x27;.
&gt; #
&gt; # Furthermore, we assume that measurements k=2 and k=3 undergo the
&gt; # same transformation, which may illustrate that the come from
&gt; # the same batch. However, when *fitting* the model below we
&gt; # will assume they are independent.
&gt; 
&gt; # Transforms
&gt; a &lt;- c(2, 15, 15,   3)
&gt; b &lt;- c(2,  3,  3,   4)
&gt; c &lt;- c(1,  2,  2, 1/2)
&gt; K &lt;- length(a)
&gt; 
&gt; # The true signal
&gt; N &lt;- 1000
&gt; x &lt;- rexp(N)
&gt; 
&gt; # The noise
&gt; bX &lt;- outer(b,x)
&gt; E &lt;- apply(bX, MARGIN=2, FUN=function(x) rnorm(K, mean=0, sd=0.1*x))
&gt; 
&gt; # The transformed signals with noise
&gt; Xc &lt;- t(sapply(c, FUN=function(c) x^c))
&gt; Y &lt;- a + bX + Xc + E
&gt; Y &lt;- t(Y)
&gt; 
&gt; 
&gt; 
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # Fit principal curve
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # Fit principal curve through Y = (y_1, y_2, ..., y_K)
&gt; fit &lt;- fitPrincipalCurve(Y)
&gt; 
&gt; # Flip direction of &#x27;lambda&#x27;?
&gt; rho &lt;- cor(fit$lambda, Y[,1], use=&quot;complete.obs&quot;)
&gt; flip &lt;- (rho &lt; 0)
&gt; if (flip) {
+   fit$lambda &lt;- max(fit$lambda, na.rm=TRUE)-fit$lambda
+ }
&gt; 
&gt; L &lt;- ncol(fit$s)
&gt; 
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # Backtransform data according to model fit
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # Backtransform toward the principal curve (the &quot;common scale&quot;)
&gt; YN1 &lt;- backtransformPrincipalCurve(Y, fit=fit)
&gt; stopifnot(ncol(YN1) == K)
&gt; 
&gt; 
&gt; # Backtransform toward the first dimension
&gt; YN2 &lt;- backtransformPrincipalCurve(Y, fit=fit, targetDimension=1)
&gt; stopifnot(ncol(YN2) == K)
&gt; 
&gt; 
&gt; # Backtransform toward the last (fitted) dimension
&gt; YN3 &lt;- backtransformPrincipalCurve(Y, fit=fit, targetDimension=L)
&gt; stopifnot(ncol(YN3) == K)
&gt; 
&gt; 
&gt; # Backtransform toward the third dimension (dimension by dimension)
&gt; # Note, this assumes that K == L.
&gt; YN4 &lt;- Y
&gt; for (cc in 1:L) {
+   YN4[,cc] &lt;- backtransformPrincipalCurve(Y, fit=fit,
+                                   targetDimension=1, dimensions=cc)
+ }
&gt; stopifnot(identical(YN4, YN2))
&gt; 
&gt; 
&gt; # Backtransform a subset toward the first dimension
&gt; # Note, this assumes that K == L.
&gt; YN5 &lt;- backtransformPrincipalCurve(Y, fit=fit,
+                                targetDimension=1, dimensions=2:3)
&gt; stopifnot(identical(YN5, YN2[,2:3]))
&gt; stopifnot(ncol(YN5) == 2)
&gt; 
&gt; 
&gt; # Extract signals from measurement #2 and backtransform according
&gt; # its model fit.  Signals are standardized to target dimension 1.
&gt; y6 &lt;- Y[,2,drop=FALSE]
&gt; yN6 &lt;- backtransformPrincipalCurve(y6, fit=fit, dimensions=2,
+                                                targetDimension=1)
&gt; stopifnot(identical(yN6, YN2[,2,drop=FALSE]))
&gt; stopifnot(ncol(yN6) == 1)
&gt; 
&gt; 
&gt; # Extract signals from measurement #2 and backtransform according
&gt; # the the model fit of measurement #3 (because we believe these
&gt; # two have undergone very similar transformations.
&gt; # Signals are standardized to target dimension 1.
&gt; y7 &lt;- Y[,2,drop=FALSE]
&gt; yN7 &lt;- backtransformPrincipalCurve(y7, fit=fit, dimensions=3,
+                                                targetDimension=1)
&gt; stopifnot(ncol(yN7) == 1)
&gt; 
&gt; rho &lt;- cor(yN7, yN6)
&gt; print(rho)
          [,1]
[1,] 0.9999692
&gt; stopifnot(rho &gt; 0.999)
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.856   0.037   0.877 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/callNaiveGenotypes.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; layout(matrix(1:3, ncol=1))
&gt; par(mar=c(2,4,4,1)+0.1)
&gt; 
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # A bimodal distribution
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; xAA &lt;- rnorm(n=10000, mean=0, sd=0.1)
&gt; xBB &lt;- rnorm(n=10000, mean=1, sd=0.1)
&gt; x &lt;- c(xAA,xBB)
&gt; fit &lt;- findPeaksAndValleys(x)
&gt; print(fit)
    type            x      density
1   peak -0.005335622 1.6834295626
2 valley  0.493738593 0.0003266488
3   peak  0.992812807 1.6862211175
&gt; calls &lt;- callNaiveGenotypes(x, cn=rep(1,length(x)), verbose=-20)
Calling genotypes from allele B fractions (BAFs)...
 Fitting naive genotype model...
  Fitting naive genotype model from normal allele B fractions (BAFs)...
   Flavor: density
   Censoring BAFs...
    Before:
         Min.   1st Qu.    Median      Mean   3rd Qu.      Max. 
    -0.427674  0.001282  0.498209  0.500544  0.998904  1.410812 
    [1] 20000
    After:
        Min.  1st Qu.   Median     Mean  3rd Qu.     Max. 
        -Inf 0.001282 0.498209          0.998904      Inf 
    [1] 16888
   Censoring BAFs...done
   Copy number level #1 (C=1) of 1...
    Identified extreme points in density of BAF:
        type          x     density
    1   peak 0.01489627 1.632033649
    2 valley 0.49826924 0.004078009
    3   peak 0.97821404 1.639213105
    Local minimas (&quot;valleys&quot;) in BAF:
        type         x     density
    2 valley 0.4982692 0.004078009
   Copy number level #1 (C=1) of 1...done
  Fitting naive genotype model from normal allele B fractions (BAFs)...done
  [[1]]
  [[1]]$flavor
  [1] &quot;density&quot;
  
  [[1]]$cn
  [1] 1
  
  [[1]]$nbrOfGenotypeGroups
  [1] 2
  
  [[1]]$tau
  [1] 0.4982692
  
  [[1]]$n
  [1] 16888
  
  [[1]]$fit
      type          x     density
  1   peak 0.01489627 1.632033649
  2 valley 0.49826924 0.004078009
  3   peak 0.97821404 1.639213105
  
  [[1]]$fitValleys
      type         x     density
  2 valley 0.4982692 0.004078009
  
  
  attr(,&quot;class&quot;)
  [1] &quot;NaiveGenotypeModelFit&quot; &quot;list&quot;                 
 Fitting naive genotype model...done
 Copy number level #1 (C=1) of 1...
  Model fit:
  $flavor
  [1] &quot;density&quot;
  
  $cn
  [1] 1
  
  $nbrOfGenotypeGroups
  [1] 2
  
  $tau
  [1] 0.4982692
  
  $n
  [1] 16888
  
  $fit
      type          x     density
  1   peak 0.01489627 1.632033649
  2 valley 0.49826924 0.004078009
  3   peak 0.97821404 1.639213105
  
  $fitValleys
      type         x     density
  2 valley 0.4982692 0.004078009
  
  Genotype threshholds [1]: 0.498269243117731
  TCN=1 =&gt; BAF in {0,1}.
  Call regions: A = (-Inf,0.498], B = (0.498,+Inf)
 Copy number level #1 (C=1) of 1...done
Calling genotypes from allele B fractions (BAFs)...done
&gt; xc &lt;- split(x, calls)
&gt; print(table(calls))
calls
    0     1 
10000 10000 
&gt; xx &lt;- c(list(x),xc)
&gt; plotDensity(xx, adjust=1.5, lwd=2, col=seq_along(xx), main=&quot;(AA,BB)&quot;)
&gt; abline(v=fit$x)
&gt; 
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # A trimodal distribution with missing values
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; xAB &lt;- rnorm(n=10000, mean=1/2, sd=0.1)
&gt; x &lt;- c(xAA,xAB,xBB)
&gt; x[sample(length(x), size=0.05*length(x))] &lt;- NA_real_
&gt; x[sample(length(x), size=0.01*length(x))] &lt;- -Inf
&gt; x[sample(length(x), size=0.01*length(x))] &lt;- +Inf
&gt; fit &lt;- findPeaksAndValleys(x)
&gt; print(fit)
    type           x   density
1   peak -0.00254362 1.1644696
2 valley  0.24763984 0.1949578
3   peak  0.49365357 1.1873199
4 valley  0.74800675 0.1822808
5   peak  0.99402048 1.1706770
&gt; calls &lt;- callNaiveGenotypes(x)
&gt; xc &lt;- split(x, calls)
&gt; print(table(calls))
calls
   0  0.5    1 
9530 9346 9645 
&gt; xx &lt;- c(list(x),xc)
&gt; plotDensity(xx, adjust=1.5, lwd=2, col=seq_along(xx), main=&quot;(AA,AB,BB)&quot;)
&gt; abline(v=fit$x)
&gt; 
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # A trimodal distribution with clear separation
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; xAA &lt;- rnorm(n=10000, mean=0, sd=0.02)
&gt; xAB &lt;- rnorm(n=10000, mean=1/2, sd=0.02)
&gt; xBB &lt;- rnorm(n=10000, mean=1, sd=0.02)
&gt; x &lt;- c(xAA,xAB,xBB)
&gt; fit &lt;- findPeaksAndValleys(x)
&gt; print(fit)
    type            x      density
1   peak -0.001822875 2.605009e+00
2 valley  0.246960281 3.365926e-05
3   peak  0.498538753 2.613372e+00
4 valley  0.747321909 3.249835e-05
5   peak  0.996105065 2.600892e+00
&gt; calls &lt;- callNaiveGenotypes(x)
&gt; xc &lt;- split(x, calls)
&gt; print(table(calls))
calls
    0   0.5     1 
10000 10000 10000 
&gt; xx &lt;- c(list(x),xc)
&gt; plotDensity(xx, adjust=1.5, lwd=2, col=seq_along(xx), main=&quot;(AA&#x27;,AB&#x27;,BB&#x27;)&quot;)
&gt; abline(v=fit$x)
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.632   0.080   0.696 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/distanceBetweenLines.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; for (zzz in 0) {
+ 
+ # This example requires plot3d() in R.basic [http://www.braju.com/R/]
+ if (!require(pkgName &lt;- &quot;R.basic&quot;, character.only=TRUE)) break
+ 
+ layout(matrix(1:4, nrow=2, ncol=2, byrow=TRUE))
+ 
+ ############################################################
+ # Lines in two-dimensions
+ ############################################################
+ x &lt;- list(a=c(1,0), b=c(1,2))
+ y &lt;- list(a=c(0,2), b=c(1,1))
+ fit &lt;- distanceBetweenLines(ax=x$a, bx=x$b, ay=y$a, by=y$b)
+ 
+ xlim &lt;- ylim &lt;- c(-1,8)
+ plot(NA, xlab=&quot;&quot;, ylab=&quot;&quot;, xlim=ylim, ylim=ylim)
+ 
+ # Highlight the offset coordinates for both lines
+ points(t(x$a), pch=&quot;+&quot;, col=&quot;red&quot;)
+ text(t(x$a), label=expression(a[x]), adj=c(-1,0.5))
+ points(t(y$a), pch=&quot;+&quot;, col=&quot;blue&quot;)
+ text(t(y$a), label=expression(a[y]), adj=c(-1,0.5))
+ 
+ v &lt;- c(-1,1)*10
+ xv &lt;- list(x=x$a[1]+x$b[1]*v, y=x$a[2]+x$b[2]*v)
+ yv &lt;- list(x=y$a[1]+y$b[1]*v, y=y$a[2]+y$b[2]*v)
+ 
+ lines(xv, col=&quot;red&quot;)
+ lines(yv, col=&quot;blue&quot;)
+ 
+ points(t(fit$xs), cex=2.0, col=&quot;red&quot;)
+ text(t(fit$xs), label=expression(x(s)), adj=c(+2,0.5))
+ points(t(fit$yt), cex=1.5, col=&quot;blue&quot;)
+ text(t(fit$yt), label=expression(y(t)), adj=c(-1,0.5))
+ print(fit)
+ 
+ 
+ ############################################################
+ # Lines in three-dimensions
+ ############################################################
+ x &lt;- list(a=c(0,0,0), b=c(1,1,1))  # The &#x27;diagonal&#x27;
+ y &lt;- list(a=c(2,1,2), b=c(2,1,3))  # A &#x27;fitted&#x27; line
+ fit &lt;- distanceBetweenLines(ax=x$a, bx=x$b, ay=y$a, by=y$b)
+ 
+ xlim &lt;- ylim &lt;- zlim &lt;- c(-1,3)
+ dummy &lt;- t(c(1,1,1))*100
+ 
+ # Coordinates for the lines in 3d
+ v &lt;- seq(-10,10, by=1)
+ xv &lt;- list(x=x$a[1]+x$b[1]*v, y=x$a[2]+x$b[2]*v, z=x$a[3]+x$b[3]*v)
+ yv &lt;- list(x=y$a[1]+y$b[1]*v, y=y$a[2]+y$b[2]*v, z=y$a[3]+y$b[3]*v)
+ 
+ for (theta in seq(30,140,length.out=3)) {
+   plot3d(dummy, theta=theta, phi=30, xlab=&quot;&quot;, ylab=&quot;&quot;, zlab=&quot;&quot;,
+                              xlim=ylim, ylim=ylim, zlim=zlim)
+ 
+   # Highlight the offset coordinates for both lines
+   points3d(t(x$a), pch=&quot;+&quot;, col=&quot;red&quot;)
+   text3d(t(x$a), label=expression(a[x]), adj=c(-1,0.5))
+   points3d(t(y$a), pch=&quot;+&quot;, col=&quot;blue&quot;)
+   text3d(t(y$a), label=expression(a[y]), adj=c(-1,0.5))
+ 
+   # Draw the lines
+   lines3d(xv, col=&quot;red&quot;)
+   lines3d(yv, col=&quot;blue&quot;)
+ 
+   # Draw the two points that are closest to each other
+   points3d(t(fit$xs), cex=2.0, col=&quot;red&quot;)
+   text3d(t(fit$xs), label=expression(x(s)), adj=c(+2,0.5))
+   points3d(t(fit$yt), cex=1.5, col=&quot;blue&quot;)
+   text3d(t(fit$yt), label=expression(y(t)), adj=c(-1,0.5))
+ 
+   # Draw the distance between the two points
+   lines3d(rbind(fit$xs,fit$yt), col=&quot;purple&quot;, lwd=2)
+ }
+ 
+ print(fit)
+ 
+ } # for (zzz in 0)
Loading required package: R.basic
Warning message:
In library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :
  there is no package called &#x27;R.basic&#x27;
&gt; rm(zzz)
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.444   0.046   0.473 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/findPeaksAndValleys.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; layout(matrix(1:3, ncol=1))
&gt; par(mar=c(2,4,4,1)+0.1)
&gt; 
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # A unimodal distribution
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; x1 &lt;- rnorm(n=10000, mean=0, sd=1)
&gt; x &lt;- x1
&gt; fit &lt;- findPeaksAndValleys(x)
&gt; print(fit)
    type           x      density
1   peak -0.03118874 0.4003120310
2 valley  3.49040055 0.0008355327
3   peak  3.70944678 0.0010590083
&gt; plot(density(x), lwd=2, main=&quot;x1&quot;)
&gt; abline(v=fit$x)
&gt; 
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # A trimodal distribution
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; x2 &lt;- rnorm(n=10000, mean=4, sd=1)
&gt; x3 &lt;- rnorm(n=10000, mean=8, sd=1)
&gt; x &lt;- c(x1,x2,x3)
&gt; fit &lt;- findPeaksAndValleys(x)
&gt; print(fit)
    type          x    density
1   peak -0.0178739 0.12595393
2 valley  1.9445306 0.04307758
3   peak  3.9782953 0.12461450
4 valley  5.9763799 0.04559176
5   peak  7.9744645 0.12438143
&gt; plot(density(x), lwd=2, main=&quot;c(x1,x2,x3)&quot;)
&gt; abline(v=fit$x)
&gt; 
&gt; 
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # A trimodal distribution with clear separation
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; x1b &lt;- rnorm(n=10000, mean=0, sd=0.1)
&gt; x2b &lt;- rnorm(n=10000, mean=4, sd=0.1)
&gt; x3b &lt;- rnorm(n=10000, mean=8, sd=0.1)
&gt; x &lt;- c(x1b,x2b,x3b)
&gt; 
&gt; # Illustrating explicit usage of density()
&gt; d &lt;- density(x)
&gt; fit &lt;- findPeaksAndValleys(d, tol=0)
&gt; print(fit)
    type           x      density
1   peak -0.02884018 3.424584e-01
2 valley  1.96712411 1.254893e-06
3   peak  3.98455037 3.432327e-01
4 valley  5.98051466 1.172145e-06
5   peak  7.97647894 3.427120e-01
&gt; plot(d, lwd=2, main=&quot;c(x1b,x2b,x3b)&quot;)
&gt; abline(v=fit$x)
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.413   0.025   0.423 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/fitPrincipalCurve.matrix.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; # Simulate data from the model y &lt;- a + bx + x^c + eps(bx)
&gt; J &lt;- 1000
&gt; x &lt;- rexp(J)
&gt; a &lt;- c(2,15,3)
&gt; b &lt;- c(2,3,4)
&gt; c &lt;- c(1,2,1/2)
&gt; bx &lt;- outer(b,x)
&gt; xc &lt;- t(sapply(c, FUN=function(c) x^c))
&gt; eps &lt;- apply(bx, MARGIN=2, FUN=function(x) rnorm(length(b), mean=0, sd=0.1*x))
&gt; y &lt;- a + bx + xc + eps
&gt; y &lt;- t(y)
&gt; 
&gt; # Fit principal curve through (y_1, y_2, y_3)
&gt; fit &lt;- fitPrincipalCurve(y, verbose=TRUE)
Fitting principal curve...
 Data size: 1000x3
 Identifying missing values...
 Identifying missing values...done
 Data size after removing non-finite data points: 1000x3
 Calling principal_curve()...
Starting curve---distance^2: 2495386
Iteration 1---distance^2: 366.9718
Iteration 2---distance^2: 366.1937
Iteration 3---distance^2: 366.2027
  Converged: TRUE
  Number of iterations: 3
  Processing time/iteration: 0.1s (0.0s/iteration)
 Calling principal_curve()...done
Fitting principal curve...done
&gt; 
&gt; # Flip direction of &#x27;lambda&#x27;?
&gt; rho &lt;- cor(fit$lambda, y[,1], use=&quot;complete.obs&quot;)
&gt; flip &lt;- (rho &lt; 0)
&gt; if (flip) {
+   fit$lambda &lt;- max(fit$lambda, na.rm=TRUE)-fit$lambda
+ }
&gt; 
&gt; 
&gt; # Backtransform (y_1, y_2, y_3) to be proportional to each other
&gt; yN &lt;- backtransformPrincipalCurve(y, fit=fit)
&gt; 
&gt; # Same backtransformation dimension by dimension
&gt; yN2 &lt;- y
&gt; for (cc in 1:ncol(y)) {
+   yN2[,cc] &lt;- backtransformPrincipalCurve(y, fit=fit, dimensions=cc)
+ }
&gt; stopifnot(identical(yN2, yN))
&gt; 
&gt; 
&gt; xlim &lt;- c(0, 1.04*max(x))
&gt; ylim &lt;- range(c(y,yN), na.rm=TRUE)
&gt; 
&gt; 
&gt; # Pairwise signals vs x before and after transform
&gt; layout(matrix(1:4, nrow=2, byrow=TRUE))
&gt; par(mar=c(4,4,3,2)+0.1)
&gt; for (cc in 1:3) {
+   ylab &lt;- substitute(y[c], env=list(c=cc))
+   plot(NA, xlim=xlim, ylim=ylim, xlab=&quot;x&quot;, ylab=ylab)
+   abline(h=a[cc], lty=3)
+   mtext(side=4, at=a[cc], sprintf(&quot;a=%g&quot;, a[cc]),
+         cex=0.8, las=2, line=0, adj=1.1, padj=-0.2)
+   points(x, y[,cc])
+   points(x, yN[,cc], col=&quot;tomato&quot;)
+   legend(&quot;topleft&quot;, col=c(&quot;black&quot;, &quot;tomato&quot;), pch=19,
+                     c(&quot;orignal&quot;, &quot;transformed&quot;), bty=&quot;n&quot;)
+ }
&gt; title(main=&quot;Pairwise signals vs x before and after transform&quot;, outer=TRUE, line=-2)
&gt; 
&gt; 
&gt; # Pairwise signals before and after transform
&gt; layout(matrix(1:4, nrow=2, byrow=TRUE))
&gt; par(mar=c(4,4,3,2)+0.1)
&gt; for (rr in 3:2) {
+   ylab &lt;- substitute(y[c], env=list(c=rr))
+   for (cc in 1:2) {
+     if (cc == rr) {
+       plot.new()
+       next
+     }
+     xlab &lt;- substitute(y[c], env=list(c=cc))
+     plot(NA, xlim=ylim, ylim=ylim, xlab=xlab, ylab=ylab)
+     abline(a=0, b=1, lty=2)
+     points(y[,c(cc,rr)])
+     points(yN[,c(cc,rr)], col=&quot;tomato&quot;)
+     legend(&quot;topleft&quot;, col=c(&quot;black&quot;, &quot;tomato&quot;), pch=19,
+                       c(&quot;orignal&quot;, &quot;transformed&quot;), bty=&quot;n&quot;)
+   }
+ }
&gt; title(main=&quot;Pairwise signals before and after transform&quot;, outer=TRUE, line=-2)
&gt; 
&gt; proc.time()
   user  system elapsed 
  1.089   0.062   1.136 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/fitXYCurve.matrix.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; # Simulate data from the model y &lt;- a + bx + x^c + eps(bx)
&gt; x &lt;- rexp(1000)
&gt; a &lt;- c(2,15)
&gt; b &lt;- c(2,1)
&gt; c &lt;- c(1,2)
&gt; bx &lt;- outer(b,x)
&gt; xc &lt;- t(sapply(c, FUN=function(c) x^c))
&gt; eps &lt;- apply(bx, MARGIN=2, FUN=function(x) rnorm(length(x), mean=0, sd=0.1*x))
&gt; Y &lt;- a + bx + xc + eps
&gt; Y &lt;- t(Y)
&gt; 
&gt; lim &lt;- c(0,70)
&gt; plot(Y, xlim=lim, ylim=lim)
&gt; 
&gt; # Fit principal curve through a subset of (y_1, y_2)
&gt; subset &lt;- sample(nrow(Y), size=0.3*nrow(Y))
&gt; fit &lt;- fitXYCurve(Y[subset,], bandwidth=0.2)
&gt; 
&gt; lines(fit, col=&quot;red&quot;, lwd=2)
&gt; 
&gt; # Backtransform (y_1, y_2) keeping y_1 unchanged
&gt; YN &lt;- backtransformXYCurve(Y, fit=fit)
&gt; points(YN, col=&quot;blue&quot;)
&gt; abline(a=0, b=1, col=&quot;red&quot;, lwd=2)
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.420   0.055   0.458 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/iwpca.matrix.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; for (zzz in 0) {
+ 
+ # This example requires plot3d() in R.basic [http://www.braju.com/R/]
+ if (!require(pkgName &lt;- &quot;R.basic&quot;, character.only=TRUE)) break
+ 
+ # Simulate data from the model y &lt;- a + bx + eps(bx)
+ x &lt;- rexp(1000)
+ a &lt;- c(2,15,3)
+ b &lt;- c(2,3,4)
+ bx &lt;- outer(b,x)
+ eps &lt;- apply(bx, MARGIN=2, FUN=function(x) rnorm(length(x), mean=0, sd=0.1*x))
+ y &lt;- a + bx + eps
+ y &lt;- t(y)
+ 
+ # Add some outliers by permuting the dimensions for 1/10 of the observations
+ idx &lt;- sample(1:nrow(y), size=1/10*nrow(y))
+ y[idx,] &lt;- y[idx,c(2,3,1)]
+ 
+ # Plot the data with fitted lines at four different view points
+ opar &lt;- par(mar=c(1,1,1,1)+0.1)
+ N &lt;- 4
+ layout(matrix(1:N, nrow=2, byrow=TRUE))
+ theta &lt;- seq(0,270,length.out=N)
+ phi &lt;- rep(20, length.out=N)
+ xlim &lt;- ylim &lt;- zlim &lt;- c(0,45)
+ persp &lt;- list()
+ for (kk in seq_along(theta)) {
+   # Plot the data
+   persp[[kk]] &lt;- plot3d(y, theta=theta[kk], phi=phi[kk], xlim=xlim, ylim=ylim, zlim=zlim)
+ }
+ 
+ # Weights on the observations
+ # Example a: Equal weights
+ w &lt;- NULL
+ # Example b: More weight on the outliers (uncomment to test)
+ w &lt;- rep(1, length(x)); w[idx] &lt;- 0.8
+ 
+ # ...and show all iterations too with different colors.
+ maxIter &lt;- c(seq(1,20,length.out=10),Inf)
+ col &lt;- topo.colors(length(maxIter))
+ # Show the fitted value for every iteration
+ for (ii in seq_along(maxIter)) {
+   # Fit a line using IWPCA through data
+   fit &lt;- iwpca(y, w=w, maxIter=maxIter[ii], swapDirections=TRUE)
+ 
+   ymid &lt;- fit$xMean
+   d0 &lt;- apply(y, MARGIN=2, FUN=min) - ymid
+   d1 &lt;- apply(y, MARGIN=2, FUN=max) - ymid
+   b &lt;- fit$vt[1,]
+   y0 &lt;- -b * max(abs(d0))
+   y1 &lt;-  b * max(abs(d1))
+   yline &lt;- matrix(c(y0,y1), nrow=length(b), ncol=2)
+   yline &lt;- yline + ymid
+ 
+   for (kk in seq_along(theta)) {
+     # Set pane to draw in
+     par(mfg=c((kk-1) %/% 2, (kk-1) %% 2) + 1)
+     # Set the viewpoint of the pane
+     options(persp.matrix=persp[[kk]])
+ 
+     # Get the first principal component
+     points3d(t(ymid), col=col[ii])
+     lines3d(t(yline), col=col[ii])
+ 
+     # Highlight the last one
+     if (ii == length(maxIter))
+       lines3d(t(yline), col=&quot;red&quot;, lwd=3)
+   }
+ }
+ 
+ par(opar)
+ 
+ } # for (zzz in 0)
Loading required package: R.basic
Warning message:
In library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :
  there is no package called &#x27;R.basic&#x27;
&gt; rm(zzz)
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.409   0.041   0.433 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/likelihood.smooth.spline.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; # Define f(x)
&gt; f &lt;- expression(0.1*x^4 + 1*x^3 + 2*x^2 + x + 10*sin(2*x))
&gt; 
&gt; # Simulate data from this function in the range [a,b]
&gt; a &lt;- -2; b &lt;- 5
&gt; x &lt;- seq(a, b, length.out=3000)
&gt; y &lt;- eval(f)
&gt; 
&gt; # Add some noise to the data
&gt; y &lt;- y + rnorm(length(y), 0, 10)
&gt; 
&gt; # Plot the function and its second derivative
&gt; plot(x,y, type=&quot;l&quot;, lwd=4)
&gt; 
&gt; # Fit a cubic smoothing spline and plot it
&gt; g &lt;- smooth.spline(x,y, df=16)
&gt; lines(g, col=&quot;yellow&quot;, lwd=2, lty=2)
&gt; 
&gt; # Calculating the (log) likelihood of the fitted spline
&gt; l &lt;- likelihood(g)
&gt; 
&gt; cat(&quot;Log likelihood with unique x values:\n&quot;)
Log likelihood with unique x values:
&gt; print(l)
Likelihood of smoothing spline: -282042 
 Log base: 2.718282 
 Weighted residuals sum of square: 282042.1 
 Penalty: -0.1221863 
 Smoothing parameter lambda: 0.0009257147 
 Roughness score: 131.9913 
&gt; 
&gt; # Note that this is not the same as the log likelihood of the
&gt; # data on the fitted spline iff the x values are non-unique
&gt; x[1:5] &lt;- x[1]  # Non-unique x values
&gt; g &lt;- smooth.spline(x,y, df=16)
&gt; l &lt;- likelihood(g)
&gt; 
&gt; cat(&quot;\nLog likelihood of the *spline* data set:\n&quot;)

Log likelihood of the *spline* data set:
&gt; print(l)
Likelihood of smoothing spline: -281997.6 
 Log base: 2.718282 
 Weighted residuals sum of square: 281997.7 
 Penalty: -0.1223189 
 Smoothing parameter lambda: 0.0009261969 
 Roughness score: 132.0657 
&gt; 
&gt; # In cases with non unique x values one has to proceed as
&gt; # below if one want to get the log likelihood for the original
&gt; # data.
&gt; l &lt;- likelihood(g, x=x, y=y)
&gt; cat(&quot;\nLog likelihood of the *original* data set:\n&quot;)

Log likelihood of the *original* data set:
&gt; print(l)
Likelihood of smoothing spline: -282037 
 Log base: 2.718282 
 Weighted residuals sum of square: 282037.1 
 Penalty: -0.1223191 
 Smoothing parameter lambda: 0.0009261969 
 Roughness score: 132.066 
&gt; 
&gt; 
&gt; 
&gt; 
&gt; 
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.458   0.037   0.478 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/medianPolish.matrix.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; # Deaths from sport parachuting;  from ABC of EDA, p.224:
&gt; deaths &lt;- matrix(c(14,15,14, 7,4,7, 8,2,10, 15,9,10, 0,2,0), ncol=3, byrow=TRUE)
&gt; rownames(deaths) &lt;- c(&quot;1-24&quot;, &quot;25-74&quot;, &quot;75-199&quot;, &quot;200++&quot;, &quot;NA&quot;)
&gt; colnames(deaths) &lt;- 1973:1975
&gt; 
&gt; print(deaths)
       1973 1974 1975
1-24     14   15   14
25-74     7    4    7
75-199    8    2   10
200++    15    9   10
NA        0    2    0
&gt; 
&gt; mp &lt;- medianPolish(deaths)
&gt; mp1 &lt;- medpolish(deaths, trace=FALSE)
&gt; print(mp)

Median Polish Results (Dataset: &quot;deaths&quot;)

Overall: 8

Row Effects:
  1-24  25-74 75-199  200++     NA 
     6     -1      0      2     -8 

Column Effects:
1973 1974 1975 
   0   -1    0 

Residuals:
       1973 1974 1975
1-24      0    2    0
25-74     0   -2    0
75-199    0   -5    2
200++     5    0    0
NA        0    3    0

&gt; 
&gt; ff &lt;- c(&quot;overall&quot;, &quot;row&quot;, &quot;col&quot;, &quot;residuals&quot;)
&gt; stopifnot(all.equal(mp[ff], mp1[ff]))
&gt; 
&gt; # Validate decomposition:
&gt; stopifnot(all.equal(deaths, mp$overall+outer(mp$row,mp$col,&quot;+&quot;)+mp$resid))
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.291   0.060   0.334 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/normalizeAffine.matrix.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; pathname &lt;- system.file(&quot;data-ex&quot;, &quot;PMT-RGData.dat&quot;, package=&quot;aroma.light&quot;)
&gt; rg &lt;- read.table(pathname, header=TRUE, sep=&quot;\t&quot;)
&gt; nbrOfScans &lt;- max(rg$slide)
&gt; 
&gt; rg &lt;- as.list(rg)
&gt; for (field in c(&quot;R&quot;, &quot;G&quot;))
+   rg[[field]] &lt;- matrix(as.double(rg[[field]]), ncol=nbrOfScans)
&gt; rg$slide &lt;- rg$spot &lt;- NULL
&gt; rg &lt;- as.matrix(as.data.frame(rg))
&gt; colnames(rg) &lt;- rep(c(&quot;R&quot;, &quot;G&quot;), each=nbrOfScans)
&gt; 
&gt; rgC &lt;- rg
&gt; 
&gt; layout(matrix(c(1,2,0,3,4,0,5,6,7), ncol=3, byrow=TRUE))
&gt; 
&gt; for (channel in c(&quot;R&quot;, &quot;G&quot;)) {
+   sidx &lt;- which(colnames(rg) == channel)
+   channelColor &lt;- switch(channel, R=&quot;red&quot;, G=&quot;green&quot;)
+ 
+   # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   # The raw data
+   # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   plotMvsAPairs(rg, channel=channel)
+   title(main=paste(&quot;Observed&quot;, channel))
+   box(col=channelColor)
+ 
+   # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   # The calibrated data
+   # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   rgC[,sidx] &lt;- calibrateMultiscan(rg[,sidx], average=NULL)
+ 
+   plotMvsAPairs(rgC, channel=channel)
+   title(main=paste(&quot;Calibrated&quot;, channel))
+   box(col=channelColor)
+ } # for (channel ...)
There were 50 or more warnings (use warnings() to see the first 50)
&gt; 
&gt; 
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # The average calibrated data
&gt; #
&gt; # Note how the red signals are weaker than the green. The reason
&gt; # for this can be that the scale factor in the green channel is
&gt; # greater than in the red channel, but it can also be that there
&gt; # is a remaining relative difference in bias between the green
&gt; # and the red channel, a bias that precedes the scanning.
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; rgCA &lt;- matrix(NA_real_, nrow=nrow(rg), ncol=2)
&gt; colnames(rgCA) &lt;- c(&quot;R&quot;, &quot;G&quot;)
&gt; for (channel in c(&quot;R&quot;, &quot;G&quot;)) {
+   sidx &lt;- which(colnames(rg) == channel)
+   rgCA[,channel] &lt;- calibrateMultiscan(rg[,sidx])
+ }
&gt; 
&gt; plotMvsA(rgCA)
&gt; title(main=&quot;Average calibrated&quot;)
&gt; 
&gt; 
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # The affine normalized average calibrated data
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # Create a matrix where the columns represent the channels
&gt; # to be normalized.
&gt; rgCAN &lt;- rgCA
&gt; # Affine normalization of channels
&gt; rgCAN &lt;- normalizeAffine(rgCAN)
&gt; 
&gt; plotMvsA(rgCAN)
&gt; title(main=&quot;Affine normalized A.C.&quot;)
&gt; 
&gt; 
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # It is always ok to rescale the affine normalized data if its
&gt; # done on (R,G); not on (A,M)! However, this is only needed for
&gt; # esthetic purposes.
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; rgCAN &lt;- rgCAN * 2^5
&gt; plotMvsA(rgCAN)
&gt; title(main=&quot;Rescaled normalized&quot;)
&gt; 
&gt; 
&gt; 
&gt; proc.time()
   user  system elapsed 
  2.618   0.104   2.705 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/normalizeAverage.list.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; # Simulate ten samples of different lengths
&gt; N &lt;- 10000
&gt; X &lt;- list()
&gt; for (kk in 1:8) {
+   rfcn &lt;- list(rnorm, rgamma)[[sample(2, size=1)]]
+   size &lt;- runif(1, min=0.3, max=1)
+   a &lt;- rgamma(1, shape=20, rate=10)
+   b &lt;- rgamma(1, shape=10, rate=10)
+   values &lt;- rfcn(size*N, a, b)
+ 
+   # &quot;Censor&quot; values
+   values[values &lt; 0 | values &gt; 8] &lt;- NA_real_
+ 
+   X[[kk]] &lt;- values
+ }
&gt; 
&gt; # Add 20% missing values
&gt; X &lt;- lapply(X, FUN=function(x) {
+   x[sample(length(x), size=0.20*length(x))] &lt;- NA_real_
+   x
+ })
&gt; 
&gt; # Normalize quantiles
&gt; Xn &lt;- normalizeAverage(X, na.rm=TRUE, targetAvg=median(unlist(X), na.rm=TRUE))
&gt; 
&gt; # Plot the data
&gt; layout(matrix(1:2, ncol=1))
&gt; xlim &lt;- range(X, Xn, na.rm=TRUE)
&gt; plotDensity(X, lwd=2, xlim=xlim, main=&quot;The original distributions&quot;)
&gt; plotDensity(Xn, lwd=2, xlim=xlim, main=&quot;The normalized distributions&quot;)
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.445   0.053   0.482 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/normalizeAverage.matrix.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; # Simulate three samples with on average 20% missing values
&gt; N &lt;- 10000
&gt; X &lt;- cbind(rnorm(N, mean=3, sd=1),
+            rnorm(N, mean=4, sd=2),
+            rgamma(N, shape=2, rate=1))
&gt; X[sample(3*N, size=0.20*3*N)] &lt;- NA_real_
&gt; 
&gt; # Normalize quantiles
&gt; Xn &lt;- normalizeAverage(X, na.rm=TRUE, targetAvg=median(X, na.rm=TRUE))
&gt; 
&gt; # Plot the data
&gt; layout(matrix(1:2, ncol=1))
&gt; xlim &lt;- range(X, Xn, na.rm=TRUE)
&gt; plotDensity(X, lwd=2, xlim=xlim, main=&quot;The three original distributions&quot;)
&gt; plotDensity(Xn, lwd=2, xlim=xlim, main=&quot;The three normalized distributions&quot;)
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.377   0.044   0.405 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/normalizeCurveFit.matrix.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; pathname &lt;- system.file(&quot;data-ex&quot;, &quot;PMT-RGData.dat&quot;, package=&quot;aroma.light&quot;)
&gt; rg &lt;- read.table(pathname, header=TRUE, sep=&quot;\t&quot;)
&gt; nbrOfScans &lt;- max(rg$slide)
&gt; 
&gt; rg &lt;- as.list(rg)
&gt; for (field in c(&quot;R&quot;, &quot;G&quot;))
+   rg[[field]] &lt;- matrix(as.double(rg[[field]]), ncol=nbrOfScans)
&gt; rg$slide &lt;- rg$spot &lt;- NULL
&gt; rg &lt;- as.matrix(as.data.frame(rg))
&gt; colnames(rg) &lt;- rep(c(&quot;R&quot;, &quot;G&quot;), each=nbrOfScans)
&gt; 
&gt; layout(matrix(c(1,2,0,3,4,0,5,6,7), ncol=3, byrow=TRUE))
&gt; 
&gt; rgC &lt;- rg
&gt; for (channel in c(&quot;R&quot;, &quot;G&quot;)) {
+   sidx &lt;- which(colnames(rg) == channel)
+   channelColor &lt;- switch(channel, R=&quot;red&quot;, G=&quot;green&quot;)
+ 
+   # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   # The raw data
+   # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   plotMvsAPairs(rg[,sidx])
+   title(main=paste(&quot;Observed&quot;, channel))
+   box(col=channelColor)
+ 
+   # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   # The calibrated data
+   # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   rgC[,sidx] &lt;- calibrateMultiscan(rg[,sidx], average=NULL)
+ 
+   plotMvsAPairs(rgC[,sidx])
+   title(main=paste(&quot;Calibrated&quot;, channel))
+   box(col=channelColor)
+ } # for (channel ...)
&gt; 
&gt; 
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # The average calibrated data
&gt; #
&gt; # Note how the red signals are weaker than the green. The reason
&gt; # for this can be that the scale factor in the green channel is
&gt; # greater than in the red channel, but it can also be that there
&gt; # is a remaining relative difference in bias between the green
&gt; # and the red channel, a bias that precedes the scanning.
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; rgCA &lt;- rg
&gt; for (channel in c(&quot;R&quot;, &quot;G&quot;)) {
+   sidx &lt;- which(colnames(rg) == channel)
+   rgCA[,sidx] &lt;- calibrateMultiscan(rg[,sidx])
+ }
&gt; 
&gt; rgCAavg &lt;- matrix(NA_real_, nrow=nrow(rgCA), ncol=2)
&gt; colnames(rgCAavg) &lt;- c(&quot;R&quot;, &quot;G&quot;)
&gt; for (channel in c(&quot;R&quot;, &quot;G&quot;)) {
+   sidx &lt;- which(colnames(rg) == channel)
+   rgCAavg[,channel] &lt;- apply(rgCA[,sidx], MARGIN=1, FUN=median, na.rm=TRUE)
+ }
&gt; 
&gt; # Add some &quot;fake&quot; outliers
&gt; outliers &lt;- 1:600
&gt; rgCAavg[outliers,&quot;G&quot;] &lt;- 50000
&gt; 
&gt; plotMvsA(rgCAavg)
&gt; title(main=&quot;Average calibrated (AC)&quot;)
&gt; 
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # Normalize data
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # Weight-down outliers when normalizing
&gt; weights &lt;- rep(1, nrow(rgCAavg))
&gt; weights[outliers] &lt;- 0.001
&gt; 
&gt; # Affine normalization of channels
&gt; rgCANa &lt;- normalizeAffine(rgCAavg, weights=weights)
&gt; # It is always ok to rescale the affine normalized data if its
&gt; # done on (R,G); not on (A,M)! However, this is only needed for
&gt; # esthetic purposes.
&gt; rgCANa &lt;- rgCANa *2^1.4
&gt; plotMvsA(rgCANa)
&gt; title(main=&quot;Normalized AC&quot;)
&gt; 
&gt; # Curve-fit (lowess) normalization
&gt; rgCANlw &lt;- normalizeLowess(rgCAavg, weights=weights)
Warning message:
In normalizeCurveFit.matrix(X, method = &quot;lowess&quot;, ...) :
  Weights were rounded to {0,1} since &#x27;lowess&#x27; normalization supports only zero-one weights.
&gt; plotMvsA(rgCANlw, col=&quot;orange&quot;, add=TRUE)
&gt; 
&gt; # Curve-fit (loess) normalization
&gt; rgCANl &lt;- normalizeLoess(rgCAavg, weights=weights)
&gt; plotMvsA(rgCANl, col=&quot;red&quot;, add=TRUE)
&gt; 
&gt; # Curve-fit (robust spline) normalization
&gt; rgCANrs &lt;- normalizeRobustSpline(rgCAavg, weights=weights)
&gt; plotMvsA(rgCANrs, col=&quot;blue&quot;, add=TRUE)
&gt; 
&gt; legend(x=0,y=16, legend=c(&quot;affine&quot;, &quot;lowess&quot;, &quot;loess&quot;, &quot;r. spline&quot;), pch=19,
+        col=c(&quot;black&quot;, &quot;orange&quot;, &quot;red&quot;, &quot;blue&quot;), ncol=2, x.intersp=0.3, bty=&quot;n&quot;)
&gt; 
&gt; 
&gt; plotMvsMPairs(cbind(rgCANa, rgCANlw), col=&quot;orange&quot;, xlab=expression(M[affine]))
&gt; title(main=&quot;Normalized AC&quot;)
&gt; plotMvsMPairs(cbind(rgCANa, rgCANl), col=&quot;red&quot;, add=TRUE)
&gt; plotMvsMPairs(cbind(rgCANa, rgCANrs), col=&quot;blue&quot;, add=TRUE)
&gt; abline(a=0, b=1, lty=2)
&gt; legend(x=-6,y=6, legend=c(&quot;lowess&quot;, &quot;loess&quot;, &quot;r. spline&quot;), pch=19,
+        col=c(&quot;orange&quot;, &quot;red&quot;, &quot;blue&quot;), ncol=2, x.intersp=0.3, bty=&quot;n&quot;)
&gt; 
&gt; 
&gt; proc.time()
   user  system elapsed 
 11.848   0.099  11.931 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/normalizeDifferencesToAverage.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; # Simulate three shifted tracks of different lengths with same profiles
&gt; ns &lt;- c(A=2, B=1, C=0.25)*1000
&gt; xx &lt;- lapply(ns, FUN=function(n) { seq(from=1, to=max(ns), length.out=n) })
&gt; zz &lt;- mapply(seq_along(ns), ns, FUN=function(z,n) rep(z,n))
&gt; 
&gt; yy &lt;- list(
+   A = rnorm(ns[&quot;A&quot;], mean=0, sd=0.5),
+   B = rnorm(ns[&quot;B&quot;], mean=5, sd=0.4),
+   C = rnorm(ns[&quot;C&quot;], mean=-5, sd=1.1)
+ )
&gt; yy &lt;- lapply(yy, FUN=function(y) {
+   n &lt;- length(y)
+   y[1:(n/2)] &lt;- y[1:(n/2)] + 2
+   y[1:(n/4)] &lt;- y[1:(n/4)] - 4
+   y
+ })
&gt; 
&gt; # Shift all tracks toward the first track
&gt; yyN &lt;- normalizeDifferencesToAverage(yy, baseline=1)
&gt; 
&gt; # The baseline channel is not changed
&gt; stopifnot(identical(yy[[1]], yyN[[1]]))
&gt; 
&gt; # Get the estimated parameters
&gt; fit &lt;- attr(yyN, &quot;fit&quot;)
&gt; 
&gt; # Plot the tracks
&gt; layout(matrix(1:2, ncol=1))
&gt; x &lt;- unlist(xx)
&gt; col &lt;- unlist(zz)
&gt; y &lt;- unlist(yy)
&gt; yN &lt;- unlist(yyN)
&gt; plot(x, y, col=col, ylim=c(-10,10))
&gt; plot(x, yN, col=col, ylim=c(-10,10))
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.434   0.062   0.479 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/normalizeFragmentLength-ex1.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # Example 1: Single-enzyme fragment-length normalization of 6 arrays
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # Number samples
&gt; I &lt;- 9
&gt; 
&gt; # Number of loci
&gt; J &lt;- 1000
&gt; 
&gt; # Fragment lengths
&gt; fl &lt;- seq(from=100, to=1000, length.out=J)
&gt; 
&gt; # Simulate data points with unknown fragment lengths
&gt; hasUnknownFL &lt;- seq(from=1, to=J, by=50)
&gt; fl[hasUnknownFL] &lt;- NA_real_
&gt; 
&gt; # Simulate data
&gt; y &lt;- matrix(0, nrow=J, ncol=I)
&gt; maxY &lt;- 12
&gt; for (kk in 1:I) {
+   k &lt;- runif(n=1, min=3, max=5)
+   mu &lt;- function(fl) {
+     mu &lt;- rep(maxY, length(fl))
+     ok &lt;- !is.na(fl)
+     mu[ok] &lt;- mu[ok] - fl[ok]^{1/k}
+     mu
+   }
+   eps &lt;- rnorm(J, mean=0, sd=1)
+   y[,kk] &lt;- mu(fl) + eps
+ }
&gt; 
&gt; # Normalize data (to a zero baseline)
&gt; yN &lt;- apply(y, MARGIN=2, FUN=function(y) {
+   normalizeFragmentLength(y, fragmentLengths=fl, onMissing=&quot;median&quot;)
+ })
&gt; 
&gt; # The correction factors
&gt; rho &lt;- y-yN
&gt; print(summary(rho))
       V1              V2              V3              V4       
 Min.   :6.697   Min.   :5.847   Min.   :5.190   Min.   :7.460  
 1st Qu.:7.195   1st Qu.:6.293   1st Qu.:5.764   1st Qu.:7.754  
 Median :7.609   Median :6.808   Median :6.381   Median :8.063  
 Mean   :7.691   Mean   :6.943   Mean   :6.492   Mean   :8.120  
 3rd Qu.:8.184   3rd Qu.:7.548   3rd Qu.:7.193   3rd Qu.:8.461  
 Max.   :8.876   Max.   :8.500   Max.   :8.151   Max.   :9.002  
       V5              V6              V7              V8       
 Min.   :7.427   Min.   :3.022   Min.   :6.420   Min.   :5.350  
 1st Qu.:7.755   1st Qu.:3.914   1st Qu.:6.780   1st Qu.:5.875  
 Median :8.063   Median :4.811   Median :7.161   Median :6.440  
 Mean   :8.146   Mean   :4.934   Mean   :7.268   Mean   :6.578  
 3rd Qu.:8.516   3rd Qu.:5.885   3rd Qu.:7.714   3rd Qu.:7.234  
 Max.   :9.123   Max.   :7.368   Max.   :8.510   Max.   :8.291  
       V9       
 Min.   :7.993  
 1st Qu.:8.193  
 Median :8.458  
 Mean   :8.511  
 3rd Qu.:8.798  
 Max.   :9.243  
&gt; # The correction for units with unknown fragment lengths
&gt; # equals the median correction factor of all other units
&gt; print(summary(rho[hasUnknownFL,]))
       V1              V2              V3              V4       
 Min.   :7.609   Min.   :6.808   Min.   :6.381   Min.   :8.063  
 1st Qu.:7.609   1st Qu.:6.808   1st Qu.:6.381   1st Qu.:8.063  
 Median :7.609   Median :6.808   Median :6.381   Median :8.063  
 Mean   :7.609   Mean   :6.808   Mean   :6.381   Mean   :8.063  
 3rd Qu.:7.609   3rd Qu.:6.808   3rd Qu.:6.381   3rd Qu.:8.063  
 Max.   :7.609   Max.   :6.808   Max.   :6.381   Max.   :8.063  
       V5              V6              V7              V8             V9       
 Min.   :8.063   Min.   :4.811   Min.   :7.161   Min.   :6.44   Min.   :8.458  
 1st Qu.:8.063   1st Qu.:4.811   1st Qu.:7.161   1st Qu.:6.44   1st Qu.:8.458  
 Median :8.063   Median :4.811   Median :7.161   Median :6.44   Median :8.458  
 Mean   :8.063   Mean   :4.811   Mean   :7.161   Mean   :6.44   Mean   :8.458  
 3rd Qu.:8.063   3rd Qu.:4.811   3rd Qu.:7.161   3rd Qu.:6.44   3rd Qu.:8.458  
 Max.   :8.063   Max.   :4.811   Max.   :7.161   Max.   :6.44   Max.   :8.458  
&gt; 
&gt; # Plot raw data
&gt; layout(matrix(1:9, ncol=3))
&gt; xlim &lt;- c(0,max(fl, na.rm=TRUE))
&gt; ylim &lt;- c(0,max(y, na.rm=TRUE))
&gt; xlab &lt;- &quot;Fragment length&quot;
&gt; ylab &lt;- expression(log2(theta))
&gt; for (kk in 1:I) {
+   plot(fl, y[,kk], xlim=xlim, ylim=ylim, xlab=xlab, ylab=ylab)
+   ok &lt;- (is.finite(fl) &amp; is.finite(y[,kk]))
+   lines(lowess(fl[ok], y[ok,kk]), col=&quot;red&quot;, lwd=2)
+ }
&gt; 
&gt; # Plot normalized data
&gt; layout(matrix(1:9, ncol=3))
&gt; ylim &lt;- c(-1,1)*max(y, na.rm=TRUE)/2
&gt; for (kk in 1:I) {
+   plot(fl, yN[,kk], xlim=xlim, ylim=ylim, xlab=xlab, ylab=ylab)
+   ok &lt;- (is.finite(fl) &amp; is.finite(y[,kk]))
+   lines(lowess(fl[ok], yN[ok,kk]), col=&quot;blue&quot;, lwd=2)
+ }
&gt; 
&gt; proc.time()
   user  system elapsed 
  1.016   0.085   1.085 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/normalizeFragmentLength-ex2.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; # Example 2: Two-enzyme fragment-length normalization of 6 arrays
&gt; # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
&gt; set.seed(0xbeef)
&gt; 
&gt; # Number samples
&gt; I &lt;- 5
&gt; 
&gt; # Number of loci
&gt; J &lt;- 3000
&gt; 
&gt; # Fragment lengths (two enzymes)
&gt; fl &lt;- matrix(0, nrow=J, ncol=2)
&gt; fl[,1] &lt;- seq(from=100, to=1000, length.out=J)
&gt; fl[,2] &lt;- seq(from=1000, to=100, length.out=J)
&gt; 
&gt; # Let 1/2 of the units be on both enzymes
&gt; fl[seq(from=1, to=J, by=4),1] &lt;- NA_real_
&gt; fl[seq(from=2, to=J, by=4),2] &lt;- NA_real_
&gt; 
&gt; # Let some have unknown fragment lengths
&gt; hasUnknownFL &lt;- seq(from=1, to=J, by=15)
&gt; fl[hasUnknownFL,] &lt;- NA_real_
&gt; 
&gt; # Sty/Nsp mixing proportions:
&gt; rho &lt;- rep(1, I)
&gt; rho[1] &lt;- 1/3;  # Less Sty in 1st sample
&gt; rho[3] &lt;- 3/2;  # More Sty in 3rd sample
&gt; 
&gt; 
&gt; # Simulate data
&gt; z &lt;- array(0, dim=c(J,2,I))
&gt; maxLog2Theta &lt;- 12
&gt; for (ii in 1:I) {
+   # Common effect for both enzymes
+   mu &lt;- function(fl) {
+     k &lt;- runif(n=1, min=3, max=5)
+     mu &lt;- rep(maxLog2Theta, length(fl))
+     ok &lt;- is.finite(fl)
+     mu[ok] &lt;- mu[ok] - fl[ok]^{1/k}
+     mu
+   }
+ 
+   # Calculate the effect for each data point
+   for (ee in 1:2) {
+     z[,ee,ii] &lt;- mu(fl[,ee])
+   }
+ 
+   # Update the Sty/Nsp mixing proportions
+   ee &lt;- 2
+   z[,ee,ii] &lt;- rho[ii]*z[,ee,ii]
+ 
+   # Add random errors
+   for (ee in 1:2) {
+     eps &lt;- rnorm(J, mean=0, sd=1/sqrt(2))
+     z[,ee,ii] &lt;- z[,ee,ii] + eps
+   }
+ }
&gt; 
&gt; 
&gt; hasFl &lt;- is.finite(fl)
&gt; 
&gt; unitSets &lt;- list(
+   nsp  = which( hasFl[,1] &amp; !hasFl[,2]),
+   sty  = which(!hasFl[,1] &amp;  hasFl[,2]),
+   both = which( hasFl[,1] &amp;  hasFl[,2]),
+   none = which(!hasFl[,1] &amp; !hasFl[,2])
+ )
&gt; 
&gt; # The observed data is a mix of two enzymes
&gt; theta &lt;- matrix(NA_real_, nrow=J, ncol=I)
&gt; 
&gt; # Single-enzyme units
&gt; for (ee in 1:2) {
+   uu &lt;- unitSets[[ee]]
+   theta[uu,] &lt;- 2^z[uu,ee,]
+ }
&gt; 
&gt; # Both-enzyme units (sum on intensity scale)
&gt; uu &lt;- unitSets$both
&gt; theta[uu,] &lt;- (2^z[uu,1,]+2^z[uu,2,])/2
&gt; 
&gt; # Missing units (sample from the others)
&gt; uu &lt;- unitSets$none
&gt; theta[uu,] &lt;- apply(theta, MARGIN=2, sample, size=length(uu))
&gt; 
&gt; # Calculate target array
&gt; thetaT &lt;- rowMeans(theta, na.rm=TRUE)
&gt; targetFcns &lt;- list()
&gt; for (ee in 1:2) {
+   uu &lt;- unitSets[[ee]]
+   fit &lt;- lowess(fl[uu,ee], log2(thetaT[uu]))
+   class(fit) &lt;- &quot;lowess&quot;
+   targetFcns[[ee]] &lt;- function(fl, ...) {
+     predict(fit, newdata=fl)
+   }
+ }
&gt; 
&gt; 
&gt; # Fit model only to a subset of the data
&gt; subsetToFit &lt;- setdiff(1:J, seq(from=1, to=J, by=10))
&gt; 
&gt; # Normalize data (to a target baseline)
&gt; thetaN &lt;- matrix(NA_real_, nrow=J, ncol=I)
&gt; fits &lt;- vector(&quot;list&quot;, I)
&gt; for (ii in 1:I) {
+   lthetaNi &lt;- normalizeFragmentLength(log2(theta[,ii]), targetFcns=targetFcns,
+                      fragmentLengths=fl, onMissing=&quot;median&quot;,
+                      subsetToFit=subsetToFit, .returnFit=TRUE)
+   fits[[ii]] &lt;- attr(lthetaNi, &quot;modelFit&quot;)
+   thetaN[,ii] &lt;- 2^lthetaNi
+ }
&gt; 
&gt; 
&gt; # Plot raw data
&gt; xlim &lt;- c(0, max(fl, na.rm=TRUE))
&gt; ylim &lt;- c(0, max(log2(theta), na.rm=TRUE))
&gt; Mlim &lt;- c(-1,1)*4
&gt; xlab &lt;- &quot;Fragment length&quot;
&gt; ylab &lt;- expression(log2(theta))
&gt; Mlab &lt;- expression(M==log[2](theta/theta[R]))
&gt; 
&gt; layout(matrix(1:(3*I), ncol=I, byrow=TRUE))
&gt; for (ii in 1:I) {
+   plot(NA, xlim=xlim, ylim=ylim, xlab=xlab, ylab=ylab, main=&quot;raw&quot;)
+ 
+   # Single-enzyme units
+   for (ee in 1:2) {
+     # The raw data
+     uu &lt;- unitSets[[ee]]
+     points(fl[uu,ee], log2(theta[uu,ii]), col=ee+1)
+   }
+ 
+   # Both-enzyme units (use fragment-length for enzyme #1)
+   uu &lt;- unitSets$both
+   points(fl[uu,1], log2(theta[uu,ii]), col=3+1)
+ 
+   for (ee in 1:2) {
+     # The true effects
+     uu &lt;- unitSets[[ee]]
+     lines(lowess(fl[uu,ee], log2(theta[uu,ii])), col=&quot;black&quot;, lwd=4, lty=3)
+ 
+     # The estimated effects
+     fit &lt;- fits[[ii]][[ee]]$fit
+     lines(fit, col=&quot;orange&quot;, lwd=3)
+ 
+     muT &lt;- targetFcns[[ee]](fl[uu,ee])
+     lines(fl[uu,ee], muT, col=&quot;cyan&quot;, lwd=1)
+   }
+ }
&gt; 
&gt; # Calculate log-ratios
&gt; thetaR &lt;- rowMeans(thetaN, na.rm=TRUE)
&gt; M &lt;- log2(thetaN/thetaR)
&gt; 
&gt; # Plot normalized data
&gt; for (ii in 1:I) {
+   plot(NA, xlim=xlim, ylim=Mlim, xlab=xlab, ylab=Mlab, main=&quot;normalized&quot;)
+   # Single-enzyme units
+   for (ee in 1:2) {
+     # The normalized data
+     uu &lt;- unitSets[[ee]]
+     points(fl[uu,ee], M[uu,ii], col=ee+1)
+   }
+   # Both-enzyme units (use fragment-length for enzyme #1)
+   uu &lt;- unitSets$both
+   points(fl[uu,1], M[uu,ii], col=3+1)
+ }
&gt; 
&gt; ylim &lt;- c(0,1.5)
&gt; for (ii in 1:I) {
+   data &lt;- list()
+   for (ee in 1:2) {
+     # The normalized data
+     uu &lt;- unitSets[[ee]]
+     data[[ee]] &lt;- M[uu,ii]
+   }
+   uu &lt;- unitSets$both
+   if (length(uu) &gt; 0)
+     data[[3]] &lt;- M[uu,ii]
+ 
+   uu &lt;- unitSets$none
+   if (length(uu) &gt; 0)
+     data[[4]] &lt;- M[uu,ii]
+ 
+   cols &lt;- seq_along(data)+1
+   plotDensity(data, col=cols, xlim=Mlim, xlab=Mlab, main=&quot;normalized&quot;)
+ 
+   abline(v=0, lty=2)
+ }
&gt; 
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.947   0.070   1.001 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/normalizeQuantileRank.list.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; # Simulate ten samples of different lengths
&gt; N &lt;- 10000
&gt; X &lt;- list()
&gt; for (kk in 1:8) {
+   rfcn &lt;- list(rnorm, rgamma)[[sample(2, size=1)]]
+   size &lt;- runif(1, min=0.3, max=1)
+   a &lt;- rgamma(1, shape=20, rate=10)
+   b &lt;- rgamma(1, shape=10, rate=10)
+   values &lt;- rfcn(size*N, a, b)
+ 
+   # &quot;Censor&quot; values
+   values[values &lt; 0 | values &gt; 8] &lt;- NA_real_
+ 
+   X[[kk]] &lt;- values
+ }
&gt; 
&gt; # Add 20% missing values
&gt; X &lt;- lapply(X, FUN=function(x) {
+   x[sample(length(x), size=0.20*length(x))] &lt;- NA_real_
+   x
+ })
&gt; 
&gt; # Normalize quantiles
&gt; Xn &lt;- normalizeQuantile(X)
&gt; 
&gt; # Plot the data
&gt; layout(matrix(1:2, ncol=1))
&gt; xlim &lt;- range(X, na.rm=TRUE)
&gt; plotDensity(X, lwd=2, xlim=xlim, main=&quot;The original distributions&quot;)
&gt; plotDensity(Xn, lwd=2, xlim=xlim, main=&quot;The normalized distributions&quot;)
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.407   0.113   0.505 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/normalizeQuantileRank.matrix.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; # Simulate three samples with on average 20% missing values
&gt; N &lt;- 10000
&gt; X &lt;- cbind(rnorm(N, mean=3, sd=1),
+            rnorm(N, mean=4, sd=2),
+            rgamma(N, shape=2, rate=1))
&gt; X[sample(3*N, size=0.20*3*N)] &lt;- NA_real_
&gt; 
&gt; # Normalize quantiles
&gt; Xn &lt;- normalizeQuantile(X)
&gt; 
&gt; # Plot the data
&gt; layout(matrix(1:2, ncol=1))
&gt; xlim &lt;- range(X, Xn, na.rm=TRUE)
&gt; plotDensity(X, lwd=2, xlim=xlim, main=&quot;The three original distributions&quot;)
&gt; plotDensity(Xn, lwd=2, xlim=xlim, main=&quot;The three normalized distributions&quot;)
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.363   0.073   0.419 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/normalizeQuantileSpline.matrix.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; # Simulate three samples with on average 20% missing values
&gt; N &lt;- 10000
&gt; X &lt;- cbind(rnorm(N, mean=3, sd=1),
+            rnorm(N, mean=4, sd=2),
+            rgamma(N, shape=2, rate=1))
&gt; X[sample(3*N, size=0.20*3*N)] &lt;- NA_real_
&gt; 
&gt; # Plot the data
&gt; layout(matrix(c(1,0,2:5), ncol=2, byrow=TRUE))
&gt; xlim &lt;- range(X, na.rm=TRUE)
&gt; plotDensity(X, lwd=2, xlim=xlim, main=&quot;The three original distributions&quot;)
&gt; 
&gt; Xn &lt;- normalizeQuantile(X)
&gt; plotDensity(Xn, lwd=2, xlim=xlim, main=&quot;The three normalized distributions&quot;)
&gt; plotXYCurve(X, Xn, xlim=xlim, main=&quot;The three normalized distributions&quot;)
&gt; 
&gt; Xn2 &lt;- normalizeQuantileSpline(X, xTarget=Xn[,1], spar=0.99)
&gt; plotDensity(Xn2, lwd=2, xlim=xlim, main=&quot;The three normalized distributions&quot;)
&gt; plotXYCurve(X, Xn2, xlim=xlim, main=&quot;The three normalized distributions&quot;)
&gt; 
&gt; proc.time()
   user  system elapsed 
  1.060   0.080   1.124 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/normalizeTumorBoost,flavors.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; library(&quot;R.utils&quot;)
Loading required package: R.oo
Loading required package: R.methodsS3
R.methodsS3 v1.8.2 (2022-06-13 22:00:14 UTC) successfully loaded. See ?R.methodsS3 for help.
R.oo v1.25.0 (2022-06-12 02:20:02 UTC) successfully loaded. See ?R.oo for help.

Attaching package: &#x27;R.oo&#x27;

The following object is masked from &#x27;package:R.methodsS3&#x27;:

    throw

The following objects are masked from &#x27;package:methods&#x27;:

    getClasses, getMethods

The following objects are masked from &#x27;package:base&#x27;:

    attach, detach, load, save

R.utils v2.12.2 (2022-11-11 22:00:03 UTC) successfully loaded. See ?R.utils for help.

Attaching package: &#x27;R.utils&#x27;

The following object is masked from &#x27;package:utils&#x27;:

    timestamp

The following objects are masked from &#x27;package:base&#x27;:

    cat, commandArgs, getOption, isOpen, nullfile, parse, warnings

&gt; 
&gt; # Load data
&gt; pathname &lt;- system.file(&quot;data-ex/TumorBoost,fracB,exampleData.Rbin&quot;, package=&quot;aroma.light&quot;)
&gt; data &lt;- loadObject(pathname)
&gt; 
&gt; # Drop loci with missing values
&gt; data &lt;- na.omit(data)
&gt; 
&gt; attachLocally(data)
&gt; pos &lt;- position/1e6
&gt; 
&gt; # Call naive genotypes
&gt; muN &lt;- callNaiveGenotypes(betaN)
&gt; 
&gt; # Genotype classes
&gt; isAA &lt;- (muN == 0)
&gt; isAB &lt;- (muN == 1/2)
&gt; isBB &lt;- (muN == 1)
&gt; 
&gt; # Sanity checks
&gt; stopifnot(all(muN[isAA] == 0))
&gt; stopifnot(all(muN[isAB] == 1/2))
&gt; stopifnot(all(muN[isBB] == 1))
&gt; 
&gt; # TumorBoost normalization with different flavors
&gt; betaTNs &lt;- list()
&gt; for (flavor in c(&quot;v1&quot;, &quot;v2&quot;, &quot;v3&quot;, &quot;v4&quot;)) {
+   betaTN &lt;- normalizeTumorBoost(betaT=betaT, betaN=betaN, preserveScale=FALSE, flavor=flavor)
+ 
+   # Assert that no non-finite values are introduced
+   stopifnot(all(is.finite(betaTN)))
+ 
+   # Assert that nothing is flipped
+   stopifnot(all(betaTN[isAA] &lt; 1/2))
+   stopifnot(all(betaTN[isBB] &gt; 1/2))
+ 
+   betaTNs[[flavor]] &lt;- betaTN
+ }
&gt; 
&gt; # Plot
&gt; layout(matrix(1:4, ncol=1))
&gt; par(mar=c(2.5,4,0.5,1)+0.1)
&gt; ylim &lt;- c(-0.05, 1.05)
&gt; col &lt;- rep(&quot;#999999&quot;, length(muN))
&gt; col[muN == 1/2] &lt;- &quot;#000000&quot;
&gt; for (flavor in names(betaTNs)) {
+   betaTN &lt;- betaTNs[[flavor]]
+   ylab &lt;- sprintf(&quot;betaTN[%s]&quot;, flavor)
+   plot(pos, betaTN, col=col, ylim=ylim, ylab=ylab)
+ }
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.672   0.046   0.701 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/normalizeTumorBoost.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; library(&quot;R.utils&quot;)
Loading required package: R.oo
Loading required package: R.methodsS3
R.methodsS3 v1.8.2 (2022-06-13 22:00:14 UTC) successfully loaded. See ?R.methodsS3 for help.
R.oo v1.25.0 (2022-06-12 02:20:02 UTC) successfully loaded. See ?R.oo for help.

Attaching package: &#x27;R.oo&#x27;

The following object is masked from &#x27;package:R.methodsS3&#x27;:

    throw

The following objects are masked from &#x27;package:methods&#x27;:

    getClasses, getMethods

The following objects are masked from &#x27;package:base&#x27;:

    attach, detach, load, save

R.utils v2.12.2 (2022-11-11 22:00:03 UTC) successfully loaded. See ?R.utils for help.

Attaching package: &#x27;R.utils&#x27;

The following object is masked from &#x27;package:utils&#x27;:

    timestamp

The following objects are masked from &#x27;package:base&#x27;:

    cat, commandArgs, getOption, isOpen, nullfile, parse, warnings

&gt; 
&gt; # Load data
&gt; pathname &lt;- system.file(&quot;data-ex/TumorBoost,fracB,exampleData.Rbin&quot;, package=&quot;aroma.light&quot;)
&gt; data &lt;- loadObject(pathname)
&gt; attachLocally(data)
&gt; pos &lt;- position/1e6
&gt; muN &lt;- genotypeN
&gt; 
&gt; layout(matrix(1:4, ncol=1))
&gt; par(mar=c(2.5,4,0.5,1)+0.1)
&gt; ylim &lt;- c(-0.05, 1.05)
&gt; col &lt;- rep(&quot;#999999&quot;, length(muN))
&gt; col[muN == 1/2] &lt;- &quot;#000000&quot;
&gt; 
&gt; # Allele B fractions for the normal sample
&gt; plot(pos, betaN, col=col, ylim=ylim)
&gt; 
&gt; # Allele B fractions for the tumor sample
&gt; plot(pos, betaT, col=col, ylim=ylim)
&gt; 
&gt; # TumorBoost w/ naive genotype calls
&gt; betaTN &lt;- normalizeTumorBoost(betaT=betaT, betaN=betaN, preserveScale=FALSE)
&gt; plot(pos, betaTN, col=col, ylim=ylim)
&gt; 
&gt; # TumorBoost w/ external multi-sample genotype calls
&gt; betaTNx &lt;- normalizeTumorBoost(betaT=betaT, betaN=betaN, muN=muN, preserveScale=FALSE)
&gt; plot(pos, betaTNx, col=col, ylim=ylim)
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.565   0.065   0.616 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/robustSmoothSpline.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; data(cars)
&gt; attach(cars)
&gt; plot(speed, dist, main = &quot;data(cars)  &amp;  robust smoothing splines&quot;)
&gt; 
&gt; # Fit a smoothing spline using L_2 norm
&gt; cars.spl &lt;- smooth.spline(speed, dist)
&gt; lines(cars.spl, col = &quot;blue&quot;)
&gt; 
&gt; # Fit a smoothing spline using L_1 norm
&gt; cars.rspl &lt;- robustSmoothSpline(speed, dist)
&gt; lines(cars.rspl, col = &quot;red&quot;)
&gt; 
&gt; # Fit a smoothing spline using L_2 norm with 10 degrees of freedom
&gt; lines(smooth.spline(speed, dist, df=10), lty=2, col = &quot;blue&quot;)
&gt; 
&gt; # Fit a smoothing spline using L_1 norm with 10 degrees of freedom
&gt; lines(robustSmoothSpline(speed, dist, df=10), lty=2, col = &quot;red&quot;)
&gt; 
&gt; # Fit a smoothing spline using Tukey&#x27;s biweight norm
&gt; cars.rspl &lt;- robustSmoothSpline(speed, dist, method = &quot;symmetric&quot;)
&gt; lines(cars.rspl, col = &quot;purple&quot;)
&gt; 
&gt; legend(5,120, c(
+       paste(&quot;smooth.spline [C.V.] =&gt; df =&quot;,round(cars.spl$df,1)),
+       paste(&quot;robustSmoothSpline L1 [C.V.] =&gt; df =&quot;,round(cars.rspl$df,1)),
+       paste(&quot;robustSmoothSpline symmetric [C.V.] =&gt; df =&quot;,round(cars.rspl$df,1)),
+       &quot;standard with s( * , df = 10)&quot;, &quot;robust with s( * , df = 10)&quot;
+     ),
+     col = c(&quot;blue&quot;,&quot;red&quot;,&quot;purple&quot;,&quot;blue&quot;,&quot;red&quot;), lty = c(1,1,1,2,2),
+     bg=&#x27;bisque&#x27;)
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.440   0.044   0.467 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/rowAverages.matrix.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; X &lt;- matrix(1:30, nrow=5L, ncol=6L)
&gt; mu &lt;- rowMeans(X)
&gt; sd &lt;- apply(X, MARGIN=1L, FUN=sd)
&gt; 
&gt; y &lt;- rowAverages(X)
&gt; stopifnot(all(y == mu))
&gt; stopifnot(all(attr(y,&quot;deviance&quot;) == sd))
&gt; stopifnot(all(attr(y,&quot;df&quot;) == ncol(X)))
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.274   0.062   0.320 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/sampleCorrelations.matrix.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; # Simulate 20000 genes with 10 observations each
&gt; X &lt;- matrix(rnorm(n=20000), ncol=10)
&gt; 
&gt; # Calculate the correlation for 5000 random gene pairs
&gt; cor &lt;- sampleCorrelations(X, npairs=5000)
&gt; print(summary(cor))
     Min.   1st Qu.    Median      Mean   3rd Qu.      Max. 
-0.908889 -0.240635  0.008322  0.006143  0.258380  0.886216 
&gt; 
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.654   0.055   0.694 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/sampleTuples.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; pairs &lt;- sampleTuples(1:10, size=5, length=2)
&gt; print(pairs)
     [,1] [,2]
[1,]    3    8
[2,]    2    3
[3,]    8    4
[4,]    6    9
[5,]    9    7
&gt; 
&gt; triples &lt;- sampleTuples(1:10, size=5, length=3)
&gt; print(triples)
     [,1] [,2] [,3]
[1,]    1    6    9
[2,]    8    4   10
[3,]    6    9    8
[4,]   10    1    7
[5,]    7   10    4
&gt; 
&gt; # Allow tuples with repeated elements
&gt; quadruples &lt;- sampleTuples(1:3, size=5, length=4, replace=TRUE)
&gt; print(quadruples)
     [,1] [,2] [,3] [,4]
[1,]    2    1    1    1
[2,]    2    2    3    3
[3,]    2    2    2    3
[4,]    1    1    3    3
[5,]    1    1    1    1
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.285   0.059   0.327 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/wpca.matrix.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; for (zzz in 0) {
+ 
+ # This example requires plot3d() in R.basic [http://www.braju.com/R/]
+ if (!require(pkgName &lt;- &quot;R.basic&quot;, character.only=TRUE)) break
+ 
+ # -------------------------------------------------------------
+ # A first example
+ # -------------------------------------------------------------
+ # Simulate data from the model y &lt;- a + bx + eps(bx)
+ x &lt;- rexp(1000)
+ a &lt;- c(2,15,3)
+ b &lt;- c(2,3,15)
+ bx &lt;- outer(b,x)
+ eps &lt;- apply(bx, MARGIN=2, FUN=function(x) rnorm(length(x), mean=0, sd=0.1*x))
+ y &lt;- a + bx + eps
+ y &lt;- t(y)
+ 
+ # Add some outliers by permuting the dimensions for 1/3 of the observations
+ idx &lt;- sample(1:nrow(y), size=1/3*nrow(y))
+ y[idx,] &lt;- y[idx,c(2,3,1)]
+ 
+ # Down-weight the outliers W times to demonstrate how weights are used
+ W &lt;- 10
+ 
+ # Plot the data with fitted lines at four different view points
+ N &lt;- 4
+ theta &lt;- seq(0,180,length.out=N)
+ phi &lt;- rep(30, length.out=N)
+ 
+ # Use a different color for each set of weights
+ col &lt;- topo.colors(W)
+ 
+ opar &lt;- par(mar=c(1,1,1,1)+0.1)
+ layout(matrix(1:N, nrow=2, byrow=TRUE))
+ for (kk in seq(theta)) {
+   # Plot the data
+   plot3d(y, theta=theta[kk], phi=phi[kk])
+ 
+   # First, same weights for all observations
+   w &lt;- rep(1, length=nrow(y))
+ 
+   for (ww in 1:W) {
+     # Fit a line using IWPCA through data
+     fit &lt;- wpca(y, w=w, swapDirections=TRUE)
+ 
+     # Get the first principal component
+     ymid &lt;- fit$xMean
+     d0 &lt;- apply(y, MARGIN=2, FUN=min) - ymid
+     d1 &lt;- apply(y, MARGIN=2, FUN=max) - ymid
+     b &lt;- fit$vt[1,]
+     y0 &lt;- -b * max(abs(d0))
+     y1 &lt;-  b * max(abs(d1))
+     yline &lt;- matrix(c(y0,y1), nrow=length(b), ncol=2)
+     yline &lt;- yline + ymid
+ 
+     points3d(t(ymid), col=col)
+     lines3d(t(yline), col=col)
+ 
+     # Down-weight outliers only, because here we know which they are.
+     w[idx] &lt;- w[idx]/2
+   }
+ 
+   # Highlight the last one
+   lines3d(t(yline), col=&quot;red&quot;, lwd=3)
+ }
+ 
+ par(opar)
+ 
+ } # for (zzz in 0)
Loading required package: R.basic
Warning message:
In library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :
  there is no package called &#x27;R.basic&#x27;
&gt; rm(zzz)
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.371   0.061   0.416 
</PRE>
</DIV><P><SPAN class="filename">aroma.light.Rcheck/tests/wpca2.matrix.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;aroma.light&quot;)
aroma.light v3.29.0 (2023-02-11) successfully loaded. See ?aroma.light for help.
&gt; 
&gt; # -------------------------------------------------------------
&gt; # A second example
&gt; # -------------------------------------------------------------
&gt; # Data
&gt; x &lt;- c(1,2,3,4,5)
&gt; y &lt;- c(2,4,3,3,6)
&gt; 
&gt; opar &lt;- par(bty=&quot;L&quot;)
&gt; opalette &lt;- palette(c(&quot;blue&quot;, &quot;red&quot;, &quot;black&quot;))
&gt; xlim &lt;- ylim &lt;- c(0,6)
&gt; 
&gt; # Plot the data and the center mass
&gt; plot(x,y, pch=16, cex=1.5, xlim=xlim, ylim=ylim)
&gt; points(mean(x), mean(y), cex=2, lwd=2, col=&quot;blue&quot;)
&gt; 
&gt; 
&gt; # Linear regression y ~ x
&gt; fit &lt;- lm(y ~ x)
&gt; abline(fit, lty=1, col=1)
&gt; 
&gt; # Linear regression y ~ x through without intercept
&gt; fit &lt;- lm(y ~ x - 1)
&gt; abline(fit, lty=2, col=1)
&gt; 
&gt; 
&gt; # Linear regression x ~ y
&gt; fit &lt;- lm(x ~ y)
&gt; c &lt;- coefficients(fit)
&gt; b &lt;- 1/c[2]
&gt; a &lt;- -b*c[1]
&gt; abline(a=a, b=b, lty=1, col=2)
&gt; 
&gt; # Linear regression x ~ y through without intercept
&gt; fit &lt;- lm(x ~ y - 1)
&gt; b &lt;- 1/coefficients(fit)
&gt; abline(a=0, b=b, lty=2, col=2)
&gt; 
&gt; 
&gt; # Orthogonal linear &quot;regression&quot;
&gt; fit &lt;- wpca(cbind(x,y))
&gt; 
&gt; b &lt;- fit$vt[1,2]/fit$vt[1,1]
&gt; a &lt;- fit$xMean[2]-b*fit$xMean[1]
&gt; abline(a=a, b=b, lwd=2, col=3)
&gt; 
&gt; # Orthogonal linear &quot;regression&quot; without intercept
&gt; fit &lt;- wpca(cbind(x,y), center=FALSE)
&gt; b &lt;- fit$vt[1,2]/fit$vt[1,1]
&gt; a &lt;- fit$xMean[2]-b*fit$xMean[1]
&gt; abline(a=a, b=b, lty=2, lwd=2, col=3)
&gt; 
&gt; legend(xlim[1],ylim[2], legend=c(&quot;lm(y~x)&quot;, &quot;lm(y~x-1)&quot;, &quot;lm(x~y)&quot;,
+           &quot;lm(x~y-1)&quot;, &quot;pca&quot;, &quot;pca w/o intercept&quot;), lty=rep(1:2,3),
+                      lwd=rep(c(1,1,2),each=2), col=rep(1:3,each=2))
&gt; 
&gt; palette(opalette)
&gt; par(opar)
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.395   0.056   0.435 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">aroma.light.Rcheck/aroma.light-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>backtransformAffine</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>backtransformPrincipalCurve</TD><TD style="text-align: right;">0.573</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">0.596</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>calibrateMultiscan</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>callNaiveGenotypes</TD><TD style="text-align: right;">0.321</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.337</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>distanceBetweenLines</TD><TD style="text-align: right;">0.121</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.124</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findPeaksAndValleys</TD><TD style="text-align: right;">0.051</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.055</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fitPrincipalCurve</TD><TD style="text-align: right;">0.581</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.589</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fitXYCurve</TD><TD style="text-align: right;">0.207</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.207</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>iwpca</TD><TD style="text-align: right;">0.078</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.078</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>likelihood.smooth.spline</TD><TD style="text-align: right;">0.148</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.149</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>medianPolish</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>normalizeAffine</TD><TD style="text-align: right;">11.326</TD><TD style="text-align: right;"> 0.035</TD><TD style="text-align: right;">11.363</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>normalizeCurveFit</TD><TD style="text-align: right;">11.549</TD><TD style="text-align: right;"> 0.136</TD><TD style="text-align: right;">11.686</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>normalizeDifferencesToAverage</TD><TD style="text-align: right;">0.255</TD><TD style="text-align: right;">0.026</TD><TD style="text-align: right;">0.281</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>normalizeFragmentLength</TD><TD style="text-align: right;">1.621</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">1.670</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>normalizeQuantileRank</TD><TD style="text-align: right;">0.802</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.806</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>normalizeQuantileRank.matrix</TD><TD style="text-align: right;">0.056</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.056</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>normalizeQuantileSpline</TD><TD style="text-align: right;">1.143</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">1.163</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>normalizeTumorBoost</TD><TD style="text-align: right;">0.286</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.298</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>robustSmoothSpline</TD><TD style="text-align: right;">0.393</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.401</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sampleCorrelations</TD><TD style="text-align: right;">0.371</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.371</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sampleTuples</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>wpca</TD><TD style="text-align: right;">0.079</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.080</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
