<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for Rsamtools on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><B>[<A href="../index.html#R">R</A>]</B></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-08 01:15:26 -0000 (Wed, 08 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4164</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for Rsamtools on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the Rsamtools package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/Rsamtools.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1719</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">Rsamtools</A>&nbsp;2.15.1</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/Rsamtools">landing page</A>)</SPAN><BR>Bioconductor Package Maintainer<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-06&nbsp;00:12:45&nbsp;-0000&nbsp;(Mon,&nbsp;06&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/Rsamtools</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">1f65f29</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-12-30&nbsp;12:59:29&nbsp;-0000&nbsp;(Fri,&nbsp;30&nbsp;Dec&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: Rsamtools</TD></TR>
<TR><TD><B>Version</B>: 2.15.1</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:Rsamtools.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings Rsamtools_2.15.1.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-07 14:02:05 -0000 (Tue, 07 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-07 14:06:13 -0000 (Tue, 07 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 248.6 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: Rsamtools.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:Rsamtools.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings Rsamtools_2.15.1.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/Rsamtools.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘Rsamtools/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘Rsamtools’ version ‘2.15.1’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘Rsamtools’ can be installed ... WARNING
Found the following significant warnings:
  /home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include/samtools-1.7-compat.h:14:41: warning: ignoring return value of ‘int64_t bgzf_seek(BGZF*, int64_t, int)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
  bamfile.c:168:20: warning: ignoring return value of ‘bgzf_seek’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
  /home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include/samtools-1.7-compat.h:14:32: warning: ignoring return value of ‘bgzf_seek’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
  tabixfile.c:190:5: warning: ‘bgzf_is_bgzf’ is deprecated: Use bgzf_compression() or hts_detect_format() instead [-Wdeprecated-declarations]
See ‘/home/biocbuild/bbs-3.17-bioc/meat/Rsamtools.Rcheck/00install.out’ for details.
* used C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* used C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... NOTE
  installed size is 10.5Mb
  sub-directories of 1Mb or more:
    extdata   2.7Mb
    libs      6.0Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
Unexported objects imported by &#x27;:::&#x27; calls:
  ‘S4Vectors:::explodeIntBits’ ‘S4Vectors:::implodeIntBits’
  ‘S4Vectors:::makePowersOfTwo’ ‘S4Vectors:::quick_unlist’
  ‘S4Vectors:::selectSome’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... NOTE
GNU make is a SystemRequirements.
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... WARNING
Note: information on .o files is not available
File ‘/home/biocbuild/bbs-3.17-bioc/R/library/Rsamtools/libs/Rsamtools.so’:
  Found ‘__sprintf_chk’, possibly from ‘sprintf’ (C)
  Found ‘abort’, possibly from ‘abort’ (C)
  Found ‘exit’, possibly from ‘exit’ (C)
  Found ‘stderr’, possibly from ‘stderr’ (C)
  Found ‘stdout’, possibly from ‘stdout’ (C)

Compiled code should not call entry points which might terminate R nor
write to stdout/stderr instead of to the console, nor use Fortran I/O
nor system RNGs nor [v]sprintf. The detected symbols are linked into
the code but might come from libraries and not actually be called.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
         user system elapsed
pileup 44.579  0.131  44.791
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘Rsamtools_unit_tests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘Rsamtools-Overview.Rnw’ using ‘UTF-8’... failed to complete the test
 ERROR
Errors in running code in vignettes:
when running code in ‘Rsamtools-Overview.Rnw’
  ...
integer-Rle of length 80424 with 13290 runs
  Lengths:  8  8  5  2  1  3  7  7 ... 11  8  7 17  4  4  9
  Values :  6  5  4  3  4  3  5  3 ...  5  4  5  4  3  2  1


&gt; m &lt;- matrix(unlist(lapply(cvg, lapply, as.vector)), 
+     ncol = length(cvg))
Killed

... incomplete output.  Crash?

* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 2 WARNINGs, 3 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/Rsamtools.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">Rsamtools.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL Rsamtools
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘Rsamtools’ ...
** using staged installation
** libs
using C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
using C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c Biostrings_stubs.c -o Biostrings_stubs.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c COMPAT_bcf_hdr_read.c -o COMPAT_bcf_hdr_read.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c IRanges_stubs.c -o IRanges_stubs.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall  -c PileupBuffer.cpp -o PileupBuffer.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall  -c PosCacheColl.cpp -o PosCacheColl.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c R_init_Rsamtools.c -o R_init_Rsamtools.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall  -c ResultManager.cpp -o ResultManager.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c S4Vectors_stubs.c -o S4Vectors_stubs.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c XVector_stubs.c -o XVector_stubs.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c as_bam.c -o as_bam.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c bam_data.c -o bam_data.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall  -c bam_mate_iter.cpp -o bam_mate_iter.o
In file included from Template.h:9,
                 from BamIterator.h:10,
                 from BamRangeIterator.h:7,
                 from bam_mate_iter.cpp:2:
BamRangeIterator.h: In member function ‘virtual void BamRangeIterator::finalize_inprogress(bamFile)’:
/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include/samtools-1.7-compat.h:14:41: warning: ignoring return value of ‘int64_t bgzf_seek(BGZF*, int64_t, int)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
   14 | #define bam_seek(fp, pos, dir) bgzf_seek(fp, pos, dir)
      |                                ~~~~~~~~~^~~~~~~~~~~~~~
BamRangeIterator.h:138:16: note: in expansion of macro ‘bam_seek’
  138 |         (void) bam_seek(bfile, pos, SEEK_SET);
      |                ^~~~~~~~
BamIterator.h: In constructor ‘BamIterator::BamIterator(bamFile, const bam_index_t*)’:
/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include/samtools-1.7-compat.h:14:41: warning: ignoring return value of ‘int64_t bgzf_seek(BGZF*, int64_t, int)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
   14 | #define bam_seek(fp, pos, dir) bgzf_seek(fp, pos, dir)
      |                                ~~~~~~~~~^~~~~~~~~~~~~~
BamIterator.h:87:16: note: in expansion of macro ‘bam_seek’
   87 |         (void) bam_seek(bfile, 0, 0);
      |                ^~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c bam_plbuf.c -o bam_plbuf.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c bam_sort.c -o bam_sort.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c bambuffer.c -o bambuffer.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c bamfile.c -o bamfile.o
bamfile.c: In function ‘bamfile_isincomplete’:
bamfile.c:168:20: warning: ignoring return value of ‘bgzf_seek’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
  168 |             (void) bgzf_seek(bfile-&gt;file-&gt;x.bam, offset, SEEK_SET);
      |                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
bamfile.c: In function ‘bamfile_open’:
bamfile.c:29:17: warning: ‘cfile’ may be used uninitialized in this function [-Wmaybe-uninitialized]
   29 |         index = hts_idx_load2(file, indexname);
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
bamfile.c:73:17: note: ‘cfile’ was declared here
   73 |     const char *cfile;
      |                 ^~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c bcffile.c -o bcffile.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c bedidx.c -o bedidx.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c encode.c -o encode.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c fafile.c -o fafile.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c idxstats.c -o idxstats.o
In file included from bamfile.h:5,
                 from idxstats.c:1:
idxstats.c: In function ‘idxstats_bamfile’:
/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include/samtools-1.7-compat.h:14:32: warning: ignoring return value of ‘bgzf_seek’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
   14 | #define bam_seek(fp, pos, dir) bgzf_seek(fp, pos, dir)
      |                                ^~~~~~~~~~~~~~~~~~~~~~~
idxstats.c:20:12: note: in expansion of macro ‘bam_seek’
   20 |     (void) bam_seek(fp, 0, 0);
      |            ^~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c io_sam.c -o io_sam.o
In file included from io_sam.c:2:
io_sam.c: In function ‘_scan_bam_all’:
/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include/samtools-1.7-compat.h:14:32: warning: ignoring return value of ‘bgzf_seek’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
   14 | #define bam_seek(fp, pos, dir) bgzf_seek(fp, pos, dir)
      |                                ^~~~~~~~~~~~~~~~~~~~~~~
io_sam.c:307:12: note: in expansion of macro ‘bam_seek’
  307 |     (void) bam_seek(bfile-&gt;file-&gt;x.bam, bfile-&gt;pos0, SEEK_SET);
      |            ^~~~~~~~
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall  -c pbuffer_wrapper.cpp -o pbuffer_wrapper.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall  -c pileup.cpp -o pileup.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c pileupbam.c -o pileupbam.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c sam_opts.c -o sam_opts.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c sam_utils.c -o sam_utils.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c samtools_patch.c -o samtools_patch.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c scan_bam_data.c -o scan_bam_data.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c tabixfile.c -o tabixfile.o
tabixfile.c: In function ‘index_tabix’:
tabixfile.c:190:5: warning: ‘bgzf_is_bgzf’ is deprecated: Use bgzf_compression() or hts_detect_format() instead [-Wdeprecated-declarations]
  190 |     if (bgzf_is_bgzf(fn) != 1)
      |     ^~
In file included from tabixfile.c:3:
/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include/htslib/bgzf.h:270:9: note: declared here
  270 |     int bgzf_is_bgzf(const char *fn) HTS_DEPRECATED(&quot;Use bgzf_compression() or hts_detect_format() instead&quot;);
      |         ^~~~~~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c tagfilter.c -o tagfilter.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c utilities.c -o utilities.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c zip_compression.c -o zip_compression.o
g++ -std=gnu++14 -shared -L/home/biocbuild/bbs-3.17-bioc/R/lib -L/usr/local/lib -o Rsamtools.so Biostrings_stubs.o COMPAT_bcf_hdr_read.o IRanges_stubs.o PileupBuffer.o PosCacheColl.o R_init_Rsamtools.o ResultManager.o S4Vectors_stubs.o XVector_stubs.o as_bam.o bam_data.o bam_mate_iter.o bam_plbuf.o bam_sort.o bambuffer.o bamfile.o bcffile.o bedidx.o encode.o fafile.o idxstats.o io_sam.o pbuffer_wrapper.o pileup.o pileupbam.o sam_opts.o sam_utils.o samtools_patch.o scan_bam_data.o tabixfile.o tagfilter.o utilities.o zip_compression.o /home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/usrlib/libhts.a -lcurl -L/home/biocbuild/bbs-3.17-bioc/R/lib -lR
installing to /home/biocbuild/bbs-3.17-bioc/R/library/00LOCK-Rsamtools/00new/Rsamtools/libs
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (Rsamtools)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">Rsamtools.Rcheck/tests/Rsamtools_unit_tests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; BiocGenerics:::testPackage(&#x27;Rsamtools&#x27;)

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min


Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname


Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

[E::idx_find_and_load] Could not retrieve index file for &#x27;&#x27;
[E::hts_idx_load3] Could not load local index file &#x27;&#x27;
[E::idx_find_and_load] Could not retrieve index file for &#x27;/tmp/Rtmp7EbdvJ/file3d139d6cb0ebda/index&#x27;
[E::COMPAT_bcf_hdr_read] Input is not detected as bcf or vcf format
[E::idx_find_and_load] Could not retrieve index file for &#x27;/tmp/Rtmp7EbdvJ/file3d139d42832cec&#x27;
[E::hts_idx_load3] Could not load local index file &#x27;/tmp/Rtmp7EbdvJ/file3d139d42832cec&#x27;
[E::idx_find_and_load] Could not retrieve index file for &#x27;/tmp/Rtmp7EbdvJ/file3d139d17600687&#x27;
[E::hts_idx_load3] Could not load local index file &#x27;/tmp/Rtmp7EbdvJ/file3d139d17600687&#x27;
[E::hts_idx_push] Chromosome blocks not continuous
[E::sam_index] Read &#x27;B7_589:7:76:306:561&#x27; with ref_name=&#x27;seq2&#x27;, ref_length=1568, flags=83, pos=987 cannot be indexed
[E::hts_open_format] Failed to open file &quot;http://httpbin.org/status/504&quot; : Connection timed out


RUNIT TEST PROTOCOL -- Tue Feb  7 14:04:59 2023 
*********************************************** 
Number of test functions: 180 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
Rsamtools RUnit Tests - 180 test functions, 0 errors, 0 failures
Number of test functions: 180 
Number of errors: 0 
Number of failures: 0 
Warning messages:
1: &#x27;applyPileups&#x27; is deprecated.
Use &#x27;pileup&#x27; instead.
See help(&quot;Deprecated&quot;) 
2: &#x27;applyPileups&#x27; is deprecated.
Use &#x27;pileup&#x27; instead.
See help(&quot;Deprecated&quot;) 
3: In read.table(conn, colClasses = colClasses, col.names = names(colClasses),  :
  not all columns named in &#x27;colClasses&#x27; exist
4: In read.table(conn, colClasses = colClasses, col.names = names(colClasses),  :
  not all columns named in &#x27;colClasses&#x27; exist
5: In read.table(conn, colClasses = colClasses, col.names = names(colClasses),  :
  not all columns named in &#x27;colClasses&#x27; exist
6: In read.table(conn, colClasses = colClasses, col.names = names(colClasses),  :
  not all columns named in &#x27;colClasses&#x27; exist
&gt; 
&gt; proc.time()
   user  system elapsed 
 18.848   1.111  23.200 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">Rsamtools.Rcheck/Rsamtools-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>ApplyPileupsParam-class</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.021</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>BamFile-class</TD><TD style="text-align: right;">0.483</TD><TD style="text-align: right;">0.035</TD><TD style="text-align: right;">0.549</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>BamViews-class</TD><TD style="text-align: right;">0.050</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.055</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>BcfFile-class</TD><TD style="text-align: right;">0.328</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.335</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>FaFile-class</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.068</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>PileupFiles-class</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Rsamtools-package</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ScanBamParam-class</TD><TD style="text-align: right;">0.585</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.622</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ScanBcfParam-class</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TabixFile-class</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>applyPileups</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>headerTabix</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>indexTabix</TD><TD style="text-align: right;">0.033</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.034</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pileup</TD><TD style="text-align: right;">44.579</TD><TD style="text-align: right;"> 0.131</TD><TD style="text-align: right;">44.791</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>quickBamFlagSummary</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readPileup</TD><TD style="text-align: right;">0.088</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.089</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>scanBam</TD><TD style="text-align: right;">0.616</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.699</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>scanBcf</TD><TD style="text-align: right;">0.424</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.442</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>scanFa</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>scanTabix</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.062</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>seqnamesTabix</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>testPairedEndBam</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>zip</TD><TD style="text-align: right;">0.026</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
