<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for SNPRelate on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.17</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><B>[<A href="..#S">S</A>]</B></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-03-27 05:50:44 -0000 (Mon, 27 Mar 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-03-12&nbsp;r83975)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">6083</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for SNPRelate on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the SNPRelate package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/SNPRelate.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1928</B>/2195</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">SNPRelate</A>&nbsp;1.33.2</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/SNPRelate">landing page</A>)</SPAN><BR>Xiuwen Zheng<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-03-24&nbsp;01:40:07&nbsp;-0000&nbsp;(Fri,&nbsp;24&nbsp;Mar&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/SNPRelate</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">317c54e</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-03-21&nbsp;23:47:31&nbsp;-0000&nbsp;(Tue,&nbsp;21&nbsp;Mar&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: SNPRelate</TD></TR>
<TR><TD><B>Version</B>: 1.33.2</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:SNPRelate.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings SNPRelate_1.33.2.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-03-25 12:57:56 -0000 (Sat, 25 Mar 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-03-25 13:02:24 -0000 (Sat, 25 Mar 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 268.3 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: SNPRelate.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:SNPRelate.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings SNPRelate_1.33.2.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/SNPRelate.Rcheck’
* using R Under development (unstable) (2023-03-12 r83975)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘SNPRelate/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘SNPRelate’ version ‘1.33.2’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘SNPRelate’ can be installed ... OK
* used C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* used C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... NOTE
  installed size is  5.4Mb
  sub-directories of 1Mb or more:
    libs   3.6Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking startup messages can be suppressed ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking LazyData ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... OK
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                user system elapsed
snpgdsMergeGRM 8.111  0.068   8.180
snpgdsGRM      5.536  0.052   5.588
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘SNPRelate.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 2 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/SNPRelate.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">SNPRelate.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL SNPRelate
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library’
* installing *source* package ‘SNPRelate’ ...
** using staged installation
** libs
using C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
using C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG -DUSING_R -DUSE_FC_LEN_T -I. -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/gdsfmt/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c ConvToGDS.cpp -o ConvToGDS.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG -DUSING_R -DUSE_FC_LEN_T -I. -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/gdsfmt/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c R_SNPRelate.c -o R_SNPRelate.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG -DUSING_R -DUSE_FC_LEN_T -I. -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/gdsfmt/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c SNPRelate.cpp -o SNPRelate.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG -DUSING_R -DUSE_FC_LEN_T -I. -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/gdsfmt/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c ThreadPool.cpp -o ThreadPool.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG -DUSING_R -DUSE_FC_LEN_T -I. -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/gdsfmt/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c dGenGWAS.cpp -o dGenGWAS.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG -DUSING_R -DUSE_FC_LEN_T -I. -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/gdsfmt/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c dVect.cpp -o dVect.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG -DUSING_R -DUSE_FC_LEN_T -I. -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/gdsfmt/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c genBeta.cpp -o genBeta.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG -DUSING_R -DUSE_FC_LEN_T -I. -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/gdsfmt/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c genEIGMIX.cpp -o genEIGMIX.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG -DUSING_R -DUSE_FC_LEN_T -I. -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/gdsfmt/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c genFst.cpp -o genFst.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG -DUSING_R -DUSE_FC_LEN_T -I. -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/gdsfmt/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c genHWE.cpp -o genHWE.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG -DUSING_R -DUSE_FC_LEN_T -I. -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/gdsfmt/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c genIBD.cpp -o genIBD.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG -DUSING_R -DUSE_FC_LEN_T -I. -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/gdsfmt/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c genIBS.cpp -o genIBS.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG -DUSING_R -DUSE_FC_LEN_T -I. -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/gdsfmt/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c genKING.cpp -o genKING.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG -DUSING_R -DUSE_FC_LEN_T -I. -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/gdsfmt/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c genLD.cpp -o genLD.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG -DUSING_R -DUSE_FC_LEN_T -I. -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/gdsfmt/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c genPCA.cpp -o genPCA.o
g++ -std=gnu++17 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG -DUSING_R -DUSE_FC_LEN_T -I. -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/gdsfmt/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c genSlideWin.cpp -o genSlideWin.o
g++ -std=gnu++17 -shared -L/usr/local/lib -o SNPRelate.so ConvToGDS.o R_SNPRelate.o SNPRelate.o ThreadPool.o dGenGWAS.o dVect.o genBeta.o genEIGMIX.o genFst.o genHWE.o genIBD.o genIBS.o genKING.o genLD.o genPCA.o genSlideWin.o -lpthread -llapack -L/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/lib -lRblas -lgfortran -lm
installing to /home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/00LOCK-SNPRelate/00new/SNPRelate/libs
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (SNPRelate)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">SNPRelate.Rcheck/tests/runTests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; BiocGenerics:::testPackage(&quot;SNPRelate&quot;)
SNPRelate
Genetic Relationship Matrix (GRM, GCTA):
Excluding 8,088 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 1,000
    using 1 thread
GRM Calculation:    the sum of all selected genotypes (0,1,2) = 282597
CPU capabilities:
Sat Mar 25 12:59:28 2023    (internal increment: 14768)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 3s
Saving to the GDS file:
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 12:59:31 2023    Done.
Genetic Relationship Matrix (GRM, GCTA):
Excluding 7,088 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 2,000
    using 1 thread
GRM Calculation:    the sum of all selected genotypes (0,1,2) = 559412
CPU capabilities:
Sat Mar 25 12:59:31 2023    (internal increment: 14768)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 3s
Saving to the GDS file:
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 12:59:34 2023    Done.
Genetic Relationship Matrix (GRM, GCTA):
Excluding 5,288 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 3,800
    using 1 thread
GRM Calculation:    the sum of all selected genotypes (0,1,2) = 1066957
CPU capabilities:
Sat Mar 25 12:59:34 2023    (internal increment: 14768)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 2s
Saving to the GDS file:
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 12:59:36 2023    Done.
GRM merging:
    open &#x27;tmp1.gds&#x27; (1,000 variants)
    open &#x27;tmp2.gds&#x27; (2,000 variants)
    open &#x27;tmp3.gds&#x27; (3,800 variants)
Weight: 0.147059, 0.294118, 0.558824
Output: tmp.gds
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Genetic Relationship Matrix (GRM, GCTA):
Excluding 2,288 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 6,800
    using 1 thread
GRM Calculation:    the sum of all selected genotypes (0,1,2) = 1908966
CPU capabilities:
Sat Mar 25 12:59:37 2023    (internal increment: 14768)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 2s
Sat Mar 25 12:59:39 2023    Done.
Genetic Relationship Matrix (GRM, IndivBeta):
Excluding 8,088 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 1,000
    using 1 thread
GRM Calculation:    the sum of all selected genotypes (0,1,2) = 282597
CPU capabilities:
Sat Mar 25 12:59:39 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 1s
Saving to the GDS file:
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 12:59:40 2023    Done.
Genetic Relationship Matrix (GRM, IndivBeta):
Excluding 7,088 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 2,000
    using 1 thread
GRM Calculation:    the sum of all selected genotypes (0,1,2) = 559412
CPU capabilities:
Sat Mar 25 12:59:40 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 1s
Saving to the GDS file:
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 12:59:41 2023    Done.
Genetic Relationship Matrix (GRM, IndivBeta):
Excluding 5,288 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 3,800
    using 1 thread
GRM Calculation:    the sum of all selected genotypes (0,1,2) = 1066957
CPU capabilities:
Sat Mar 25 12:59:41 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Saving to the GDS file:
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 12:59:41 2023    Done.
GRM merging:
    open &#x27;tmp1.gds&#x27; (1,000 variants)
    open &#x27;tmp2.gds&#x27; (2,000 variants)
    open &#x27;tmp3.gds&#x27; (3,800 variants)
Weight: 0.147059, 0.294118, 0.558824
Output: tmp.gds
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Writing ...
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Genetic Relationship Matrix (GRM, IndivBeta):
Excluding 2,288 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 6,800
    using 1 thread
GRM Calculation:    the sum of all selected genotypes (0,1,2) = 1908966
CPU capabilities:
Sat Mar 25 12:59:42 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 12:59:42 2023    Done.
Linkage Disequilibrium (LD) estimation on genotypes:
    # of samples: 279
    # of SNPs: 1,000
    using 1 thread
    method: covariance
LD matrix:    the sum of all selected genotypes (0,1,2) = 283058
Linkage Disequilibrium (LD) estimation on genotypes:
    # of samples: 279
    # of SNPs: 1,000
    using 1 thread
    method: correlation
LD matrix:    the sum of all selected genotypes (0,1,2) = 283058
FUNCTION: SNPGDSFileClass
FUNCTION: SNPRelate-package
Start file conversion from PLINK BED to SNP GDS ...
    BED file: &#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/plinkhapmap.bed.gz&#x27;
        SNP-major mode (Sample X SNP), 45.7K
    FAM file: &#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/plinkhapmap.fam.gz&#x27;
    BIM file: &#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/plinkhapmap.bim.gz&#x27;
Sat Mar 25 13:00:35 2023     (store sample id, snp id, position, and chromosome)
    start writing: 60 samples, 5000 SNPs ...
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:00:35 2023 	Done.
Optimize the access efficiency ...
Clean up the fragments of GDS file:
    open the file &#x27;HapMap.gds&#x27; (98.1K)
    # of fragments: 38
    save to &#x27;HapMap.gds.tmp&#x27;
    rename &#x27;HapMap.gds.tmp&#x27; (97.8K, reduced: 240B)
    # of fragments: 18
Principal Component Analysis (PCA) on genotypes:
Excluding 203 SNPs on non-autosomes
Excluding 28 SNPs (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 60
    # of SNPs: 4,769
    using 1 thread
    # of principal components: 32
PCA:    the sum of all selected genotypes (0,1,2) = 124273
CPU capabilities:
Sat Mar 25 13:00:35 2023    (internal increment: 68676)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 1s
Sat Mar 25 13:00:36 2023    Begin (eigenvalues and eigenvectors)
Sat Mar 25 13:00:36 2023    Done.
IBD analysis (PLINK method of moment) on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
PLINK IBD:    the sum of all selected genotypes (0,1,2) = 2446510
Sat Mar 25 13:00:36 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:00:36 2023    Done.
Identity-By-State (IBS) analysis on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
IBS:    the sum of all selected genotypes (0,1,2) = 2446510
Sat Mar 25 13:00:36 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 1s
Sat Mar 25 13:00:37 2023    Done.
Linkage Disequilibrium (LD) estimation on genotypes:
    # of samples: 279
    # of SNPs: 200
    using 1 thread
    method: composite
LD matrix:    the sum of all selected genotypes (0,1,2) = 55417
FUNCTION: hapmap_geno
FUNCTION: snpgdsAdmixPlot
Eigen-analysis on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
Eigen-analysis:    the sum of all selected genotypes (0,1,2) = 2446510
CPU capabilities:
Sat Mar 25 13:00:37 2023    (internal increment: 14768)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 3s
Sat Mar 25 13:00:40 2023    Begin (eigenvalues and eigenvectors)
Sat Mar 25 13:00:40 2023    Done.
FUNCTION: snpgdsAdmixProp
Eigen-analysis on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
Eigen-analysis:    the sum of all selected genotypes (0,1,2) = 2446510
CPU capabilities:
Sat Mar 25 13:00:40 2023    (internal increment: 14768)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 3s
Sat Mar 25 13:00:43 2023    Begin (eigenvalues and eigenvectors)
Sat Mar 25 13:00:43 2023    Done.
FUNCTION: snpgdsAlleleSwitch
Strand-switching at 50 SNP locus/loci.
Unable to determine switching at 10 SNP locus/loci.
FUNCTION: snpgdsApartSelection
Sat Mar 25 13:00:43 2023	Chromosome 1, # of SNPs: 367
Sat Mar 25 13:00:43 2023	Chromosome 2, # of SNPs: 367
Sat Mar 25 13:00:43 2023	Chromosome 3, # of SNPs: 317
Sat Mar 25 13:00:43 2023	Chromosome 4, # of SNPs: 295
Sat Mar 25 13:00:43 2023	Chromosome 5, # of SNPs: 295
Sat Mar 25 13:00:43 2023	Chromosome 6, # of SNPs: 283
Sat Mar 25 13:00:43 2023	Chromosome 7, # of SNPs: 245
Sat Mar 25 13:00:43 2023	Chromosome 8, # of SNPs: 234
Sat Mar 25 13:00:43 2023	Chromosome 9, # of SNPs: 202
Sat Mar 25 13:00:43 2023	Chromosome 10, # of SNPs: 224
Sat Mar 25 13:00:43 2023	Chromosome 11, # of SNPs: 223
Sat Mar 25 13:00:43 2023	Chromosome 12, # of SNPs: 208
Sat Mar 25 13:00:43 2023	Chromosome 13, # of SNPs: 172
Sat Mar 25 13:00:43 2023	Chromosome 14, # of SNPs: 147
Sat Mar 25 13:00:43 2023	Chromosome 15, # of SNPs: 121
Sat Mar 25 13:00:43 2023	Chromosome 16, # of SNPs: 129
Sat Mar 25 13:00:43 2023	Chromosome 17, # of SNPs: 116
Sat Mar 25 13:00:43 2023	Chromosome 18, # of SNPs: 129
Sat Mar 25 13:00:43 2023	Chromosome 19, # of SNPs: 73
Sat Mar 25 13:00:43 2023	Chromosome 20, # of SNPs: 106
Sat Mar 25 13:00:43 2023	Chromosome 21, # of SNPs: 62
Sat Mar 25 13:00:43 2023	Chromosome 22, # of SNPs: 51
Sat Mar 25 13:00:43 2023	Chromosome 23, # of SNPs: 204
Total # of SNPs selected:4570
FUNCTION: snpgdsBED2GDS
Start file conversion from PLINK BED to SNP GDS ...
    BED file: &#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/plinkhapmap.bed.gz&#x27;
        SNP-major mode (Sample X SNP), 45.7K
    FAM file: &#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/plinkhapmap.fam.gz&#x27;
    BIM file: &#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/plinkhapmap.bim.gz&#x27;
Sat Mar 25 13:00:43 2023     (store sample id, snp id, position, and chromosome)
    start writing: 60 samples, 5000 SNPs ...
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:00:43 2023 	Done.
Optimize the access efficiency ...
Clean up the fragments of GDS file:
    open the file &#x27;HapMap.gds&#x27; (98.1K)
    # of fragments: 38
    save to &#x27;HapMap.gds.tmp&#x27;
    rename &#x27;HapMap.gds.tmp&#x27; (97.8K, reduced: 240B)
    # of fragments: 18
FUNCTION: snpgdsClose
FUNCTION: snpgdsCombineGeno
Create a GDS genotype file:
The new dataset consists of 10 samples and 3000 SNPs
    write sample.id
    write snp.id
    write snp.rs.id
    write snp.position
    write snp.chromosome
    write snp.allele
SNP genotypes are stored in SNP-major mode (Sample X SNP).
Create a GDS genotype file:
The new dataset consists of 20 samples and 3000 SNPs
    write sample.id
    write snp.id
    write snp.rs.id
    write snp.position
    write snp.chromosome
    write snp.allele
SNP genotypes are stored in SNP-major mode (Sample X SNP).
Merge SNP GDS files:
    open &#x27;t1.gds&#x27; ...
        10 samples, 3000 SNPs
    open &#x27;t2.gds&#x27; ...
        20 samples, 3000 SNPs
Concatenating samples (mapping to the first GDS file) ...
    reference: 3000 SNPs (100.0%)
    file 2: 0 allele flips, 0 ambiguous locus/loci
        [no flip]: 3000
    create &#x27;test.gds&#x27;: 30 samples, 3000 SNPs
    FileFormat = SNP_ARRAY
    writing genotypes ...
Clean up the fragments of GDS file:
    open the file &#x27;test.gds&#x27; (46.2K)
    # of fragments: 32
    save to &#x27;test.gds.tmp&#x27;
    rename &#x27;test.gds.tmp&#x27; (46.0K, reduced: 204B)
    # of fragments: 15
Done.
Create a GDS genotype file:
The new dataset consists of 279 samples and 100 SNPs
    write sample.id
    write snp.id
    write snp.rs.id
    write snp.position
    write snp.chromosome
    write snp.allele
SNP genotypes are stored in SNP-major mode (Sample X SNP).
Create a GDS genotype file:
The new dataset consists of 279 samples and 200 SNPs
    write sample.id
    write snp.id
    write snp.rs.id
    write snp.position
    write snp.chromosome
    write snp.allele
SNP genotypes are stored in SNP-major mode (Sample X SNP).
Merge SNP GDS files:
    open &#x27;t1.gds&#x27; ...
        279 samples, 100 SNPs
    open &#x27;t2.gds&#x27; ...
        279 samples, 200 SNPs
Concatenating SNPs ...
    create &#x27;test.gds&#x27;: 279 samples, 300 SNPs
    FileFormat = SNP_ARRAY
    writing genotypes ...
Clean up the fragments of GDS file:
    open the file &#x27;test.gds&#x27; (19.1K)
    # of fragments: 32
    save to &#x27;test.gds.tmp&#x27;
    rename &#x27;test.gds.tmp&#x27; (18.9K, reduced: 204B)
    # of fragments: 15
Done.
FUNCTION: snpgdsCreateGeno
Principal Component Analysis (PCA) on genotypes:
Excluding 42 SNPs on non-autosomes
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 958
    using 1 thread
    # of principal components: 32
PCA:    the sum of all selected genotypes (0,1,2) = 264760
CPU capabilities:
Sat Mar 25 13:00:44 2023    (internal increment: 14768)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 2s
Sat Mar 25 13:00:46 2023    Begin (eigenvalues and eigenvectors)
Sat Mar 25 13:00:46 2023    Done.
FUNCTION: snpgdsCreateGenoSet
SNP pruning based on LD:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
    sliding window: 500,000 basepairs, Inf SNPs
    |LD| threshold: 0.2
    method: composite
Chromosome 1: 76.12%, 545/716
Chromosome 2: 72.78%, 540/742
Chromosome 3: 74.71%, 455/609
Chromosome 4: 73.49%, 413/562
Chromosome 5: 76.86%, 435/566
Chromosome 6: 75.75%, 428/565
Chromosome 7: 75.42%, 356/472
Chromosome 8: 71.11%, 347/488
Chromosome 9: 77.88%, 324/416
Chromosome 10: 74.12%, 358/483
Chromosome 11: 77.85%, 348/447
Chromosome 12: 76.81%, 328/427
Chromosome 13: 76.16%, 262/344
Chromosome 14: 76.60%, 216/282
Chromosome 15: 76.34%, 200/262
Chromosome 16: 72.66%, 202/278
Chromosome 17: 73.91%, 153/207
Chromosome 18: 73.68%, 196/266
Chromosome 19: 85.00%, 102/120
Chromosome 20: 71.62%, 164/229
Chromosome 21: 76.98%, 97/126
Chromosome 22: 75.86%, 88/116
6,557 markers are selected in total.
Create a GDS genotype file:
The new dataset consists of 279 samples and 6557 SNPs
    write sample.id
    write snp.id
    write snp.rs.id
    write snp.position
    write snp.chromosome
    write snp.allele
SNP genotypes are stored in SNP-major mode (Sample X SNP).
FUNCTION: snpgdsCutTree
Individual dissimilarity analysis on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
Dissimilarity:    the sum of all selected genotypes (0,1,2) = 2446510
Dissimilarity:	Sat Mar 25 13:00:47 2023	0%
Dissimilarity:	Sat Mar 25 13:00:49 2023	100%
Determine groups by permutation (Z threshold: 15, outlier threshold: 5):
Create 3 groups.
Create 4 groups.
FUNCTION: snpgdsDiss
Individual dissimilarity analysis on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
Dissimilarity:    the sum of all selected genotypes (0,1,2) = 2446510
Dissimilarity:	Sat Mar 25 13:00:50 2023	0%
Dissimilarity:	Sat Mar 25 13:00:52 2023	100%
Determine groups by permutation (Z threshold: 15, outlier threshold: 5):
Create 3 groups.
FUNCTION: snpgdsDrawTree
Individual dissimilarity analysis on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
Dissimilarity:    the sum of all selected genotypes (0,1,2) = 2446510
Dissimilarity:	Sat Mar 25 13:00:53 2023	0%
Dissimilarity:	Sat Mar 25 13:00:55 2023	100%
Determine groups by permutation (Z threshold: 15, outlier threshold: 5):
Create 3 groups.
FUNCTION: snpgdsEIGMIX
Eigen-analysis on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
Eigen-analysis:    the sum of all selected genotypes (0,1,2) = 2446510
CPU capabilities:
Sat Mar 25 13:00:55 2023    (internal increment: 14768)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 3s
Sat Mar 25 13:00:58 2023    Begin (eigenvalues and eigenvectors)
Sat Mar 25 13:00:58 2023    Done.
FUNCTION: snpgdsErrMsg
FUNCTION: snpgdsExampleFileName
FUNCTION: snpgdsFst
Fst estimation on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
Method: Weir &amp; Cockerham, 1984
# of Populations: 4
    CEU (92), HCB (47), JPT (47), YRI (93)
Fst estimation on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
Method: Weir &amp; Hill, 2002
# of Populations: 4
    CEU (92), HCB (47), JPT (47), YRI (93)
FUNCTION: snpgdsGDS2BED
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: 0.95)
Converting from GDS to PLINK binary PED:
Working space: 279 samples, 8722 SNPs
Output a BIM file.
Output a BED file ...
		Sat Mar 25 13:00:58 2023	0%
		Sat Mar 25 13:00:58 2023	100%
Done.
FUNCTION: snpgdsGDS2Eigen
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: 0.95)
Converting from GDS to EIGENSOFT:
	save to *.snp: 8722 snps
	save to *.ind: 279 samples
	Output: 	Sat Mar 25 13:00:58 2023	0%
	Output: 	Sat Mar 25 13:00:58 2023	100%
Done.
FUNCTION: snpgdsGDS2PED
Converting from GDS to PLINK PED:
	Output a MAP file DONE.
	Output a PED file ...
		Output: 	Sat Mar 25 13:00:58 2023	0%
		Output: 	Sat Mar 25 13:00:59 2023	100%
FUNCTION: snpgdsGEN2GDS
running snpgdsGEN2GDS ...
FUNCTION: snpgdsGRM
Genetic Relationship Matrix (GRM, GCTA):
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
GRM Calculation:    the sum of all selected genotypes (0,1,2) = 2446510
CPU capabilities:
Sat Mar 25 13:00:59 2023    (internal increment: 14768)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 3s
Sat Mar 25 13:01:02 2023    Done.
Genetic Relationship Matrix (GRM, GCTA):
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
GRM Calculation:    the sum of all selected genotypes (0,1,2) = 2446510
CPU capabilities:
Sat Mar 25 13:01:02 2023    (internal increment: 14768)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 3s
Saving to the GDS file:
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:05 2023    Done.
FUNCTION: snpgdsGetGeno
Genotype matrix: 1000 SNPs X 279 samples
Genotype matrix: 279 samples X 1000 SNPs
FUNCTION: snpgdsHCluster
Individual dissimilarity analysis on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
Dissimilarity:    the sum of all selected genotypes (0,1,2) = 2446510
Dissimilarity:	Sat Mar 25 13:01:05 2023	0%
Dissimilarity:	Sat Mar 25 13:01:07 2023	100%
Determine groups by permutation (Z threshold: 15, outlier threshold: 5):
Create 3 groups.
FUNCTION: snpgdsHWE
Keeping 716 SNPs according to chromosome 1
Excluding 160 SNPs (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
FUNCTION: snpgdsIBDKING
IBD analysis (KING method of moment) on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1,217 SNPs (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 92
    # of SNPs: 7,506
    using 1 thread
No family is specified, and all individuals are treated as singletons.
Relationship inference in the presence of population stratification.
KING IBD:    the sum of all selected genotypes (0,1,2) = 702139
CPU capabilities:
Sat Mar 25 13:01:08 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:08 2023    Done.
IBD analysis (KING method of moment) on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1,217 SNPs (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 92
    # of SNPs: 7,506
    using 1 thread
No family is specified, and all individuals are treated as singletons.
Relationship inference in the presence of population stratification.
KING IBD:    the sum of all selected genotypes (0,1,2) = 702139
CPU capabilities:
Sat Mar 25 13:01:08 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:08 2023    Done.
IBD analysis (KING method of moment) on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1,217 SNPs (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 92
    # of SNPs: 7,506
    using 1 thread
# of families: 20, and within- and between-family relationship are estimated differently.
Relationship inference in the presence of population stratification.
KING IBD:    the sum of all selected genotypes (0,1,2) = 702139
CPU capabilities:
Sat Mar 25 13:01:09 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 1s
Sat Mar 25 13:01:10 2023    Done.
IBD analysis (KING method of moment) on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1,217 SNPs (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 92
    # of SNPs: 7,506
    using 1 thread
Relationship inference in a homogeneous population.
KING IBD:    the sum of all selected genotypes (0,1,2) = 702139
Sat Mar 25 13:01:10 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:10 2023    Done.
IBD analysis (KING method of moment) on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1,217 SNPs (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 92
    # of SNPs: 7,506
    using 1 thread
Relationship inference in a homogeneous population.
KING IBD:    the sum of all selected genotypes (0,1,2) = 702139
Sat Mar 25 13:01:10 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:10 2023    Done.
FUNCTION: snpgdsIBDMLE
SNP pruning based on LD:
Excluding 365 SNPs on non-autosomes
Excluding 1,581 SNPs (monomorphic: TRUE, MAF: 0.05, missing rate: 0.05)
    # of samples: 30
    # of SNPs: 7,142
    using 1 thread
    sliding window: 500,000 basepairs, Inf SNPs
    |LD| threshold: 0.2
    method: composite
Chromosome 1: 54.75%, 392/716
Chromosome 2: 54.31%, 403/742
Chromosome 3: 55.99%, 341/609
Chromosome 4: 56.58%, 318/562
Chromosome 5: 56.36%, 319/566
Chromosome 6: 52.74%, 298/565
Chromosome 7: 56.14%, 265/472
Chromosome 8: 51.84%, 253/488
Chromosome 9: 54.81%, 228/416
Chromosome 10: 49.90%, 241/483
Chromosome 11: 54.81%, 245/447
Chromosome 12: 54.57%, 233/427
Chromosome 13: 53.49%, 184/344
Chromosome 14: 56.03%, 158/282
Chromosome 15: 54.58%, 143/262
Chromosome 16: 54.68%, 152/278
Chromosome 17: 55.56%, 115/207
Chromosome 18: 55.64%, 148/266
Chromosome 19: 66.67%, 80/120
Chromosome 20: 53.28%, 122/229
Chromosome 21: 50.79%, 64/126
Chromosome 22: 51.72%, 60/116
4,762 markers are selected in total.
Identity-By-Descent analysis (MLE) on genotypes:
Excluding 8,838 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 30
    # of SNPs: 250
    using 1 thread
MLE IBD:    the sum of all selected genotypes (0,1,2) = 7859
MLE IBD:	Sat Mar 25 13:01:10 2023	0%
MLE IBD:	Sat Mar 25 13:01:11 2023	100%
Identity-By-Descent analysis (MLE) on genotypes:
Excluding 8,838 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 25
    # of SNPs: 250
    using 1 thread
Specifying allele frequencies, mean: 0.525, sd: 0.288
MLE IBD:    the sum of all selected genotypes (0,1,2) = 6545
MLE IBD:	Sat Mar 25 13:01:11 2023	0%
MLE IBD:	Sat Mar 25 13:01:11 2023	100%
FUNCTION: snpgdsIBDMLELogLik
SNP pruning based on LD:
Excluding 365 SNPs on non-autosomes
Excluding 1,581 SNPs (monomorphic: TRUE, MAF: 0.05, missing rate: 0.05)
    # of samples: 30
    # of SNPs: 7,142
    using 1 thread
    sliding window: 500,000 basepairs, Inf SNPs
    |LD| threshold: 0.2
    method: composite
Chromosome 1: 54.75%, 392/716
Chromosome 2: 54.31%, 403/742
Chromosome 3: 55.99%, 341/609
Chromosome 4: 56.58%, 318/562
Chromosome 5: 56.36%, 319/566
Chromosome 6: 52.74%, 298/565
Chromosome 7: 56.14%, 265/472
Chromosome 8: 51.84%, 253/488
Chromosome 9: 54.81%, 228/416
Chromosome 10: 49.90%, 241/483
Chromosome 11: 54.81%, 245/447
Chromosome 12: 54.57%, 233/427
Chromosome 13: 53.49%, 184/344
Chromosome 14: 56.03%, 158/282
Chromosome 15: 54.58%, 143/262
Chromosome 16: 54.68%, 152/278
Chromosome 17: 55.56%, 115/207
Chromosome 18: 55.64%, 148/266
Chromosome 19: 66.67%, 80/120
Chromosome 20: 53.28%, 122/229
Chromosome 21: 50.79%, 64/126
Chromosome 22: 51.72%, 60/116
4,762 markers are selected in total.
Identity-By-Descent analysis (MLE) on genotypes:
Excluding 8,838 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 30
    # of SNPs: 250
    using 1 thread
MLE IBD:    the sum of all selected genotypes (0,1,2) = 7859
MLE IBD:	Sat Mar 25 13:01:11 2023	0%
MLE IBD:	Sat Mar 25 13:01:12 2023	100%
Identity-By-Descent analysis (MLE) on genotypes:
Excluding 8,838 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 25
    # of SNPs: 250
    using 1 thread
Specifying allele frequencies, mean: 0.525, sd: 0.288
MLE IBD:    the sum of all selected genotypes (0,1,2) = 6545
MLE IBD:	Sat Mar 25 13:01:12 2023	0%
MLE IBD:	Sat Mar 25 13:01:12 2023	100%
FUNCTION: snpgdsIBDMoM
IBD analysis (PLINK method of moment) on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1,217 SNPs (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 92
    # of SNPs: 7,506
    using 1 thread
PLINK IBD:    the sum of all selected genotypes (0,1,2) = 702139
Sat Mar 25 13:01:12 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:12 2023    Done.
IBD analysis (PLINK method of moment) on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 563 SNPs (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 93
    # of SNPs: 8,160
    using 1 thread
PLINK IBD:    the sum of all selected genotypes (0,1,2) = 755648
Sat Mar 25 13:01:12 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:12 2023    Done.
IBD analysis (PLINK method of moment) on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 563 SNPs (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 93
    # of SNPs: 8,160
    using 1 thread
Specifying allele frequencies, mean: 0.500, sd: 0.315
*** A correction factor based on allele count is not used, since the allele frequencies are specified.
PLINK IBD:    the sum of all selected genotypes (0,1,2) = 755648
Sat Mar 25 13:01:12 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:12 2023    Done.
IBD analysis (PLINK method of moment) on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 563 SNPs (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 25
    # of SNPs: 8,160
    using 1 thread
Specifying allele frequencies, mean: 0.500, sd: 0.315
*** A correction factor based on allele count is not used, since the allele frequencies are specified.
PLINK IBD:    the sum of all selected genotypes (0,1,2) = 203285
Sat Mar 25 13:01:12 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:12 2023    Done.
FUNCTION: snpgdsIBDSelection
IBD analysis (PLINK method of moment) on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 563 SNPs (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 93
    # of SNPs: 8,160
    using 1 thread
PLINK IBD:    the sum of all selected genotypes (0,1,2) = 755648
Sat Mar 25 13:01:12 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:12 2023    Done.
FUNCTION: snpgdsIBS
Identity-By-State (IBS) analysis on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
IBS:    the sum of all selected genotypes (0,1,2) = 2446510
Sat Mar 25 13:01:12 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 1s
Sat Mar 25 13:01:13 2023    Done.
FUNCTION: snpgdsIBSNum
Identity-By-State (IBS) analysis on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
IBS:    the sum of all selected genotypes (0,1,2) = 2446510
Sat Mar 25 13:01:13 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:13 2023    Done.
FUNCTION: snpgdsIndInb
Estimating individual inbreeding coefficients:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
FUNCTION: snpgdsIndInbCoef
FUNCTION: snpgdsIndivBeta
Individual Inbreeding and Relatedness (beta estimator):
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
Individual Beta:    the sum of all selected genotypes (0,1,2) = 2446510
CPU capabilities:
Sat Mar 25 13:01:13 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 1s
Sat Mar 25 13:01:14 2023    Done.
FUNCTION: snpgdsLDMat
Linkage Disequilibrium (LD) estimation on genotypes:
    # of samples: 279
    # of SNPs: 203
    using 1 thread
    method: composite
LD matrix:    the sum of all selected genotypes (0,1,2) = 56582
Linkage Disequilibrium (LD) estimation on genotypes:
    # of samples: 279
    # of SNPs: 203
    using 1 thread
    sliding window size: 203
    method: composite
LD matrix:    the sum of all selected genotypes (0,1,2) = 56582
FUNCTION: snpgdsLDpair
FUNCTION: snpgdsLDpruning
SNP pruning based on LD:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
    sliding window: 500,000 basepairs, Inf SNPs
    |LD| threshold: 0.2
    method: composite
Chromosome 1: 76.12%, 545/716
Chromosome 2: 72.78%, 540/742
Chromosome 3: 74.71%, 455/609
Chromosome 4: 73.49%, 413/562
Chromosome 5: 76.86%, 435/566
Chromosome 6: 75.75%, 428/565
Chromosome 7: 75.42%, 356/472
Chromosome 8: 71.11%, 347/488
Chromosome 9: 77.88%, 324/416
Chromosome 10: 74.12%, 358/483
Chromosome 11: 77.85%, 348/447
Chromosome 12: 76.81%, 328/427
Chromosome 13: 76.16%, 262/344
Chromosome 14: 76.60%, 216/282
Chromosome 15: 76.34%, 200/262
Chromosome 16: 72.66%, 202/278
Chromosome 17: 73.91%, 153/207
Chromosome 18: 73.68%, 196/266
Chromosome 19: 85.00%, 102/120
Chromosome 20: 71.62%, 164/229
Chromosome 21: 76.98%, 97/126
Chromosome 22: 75.86%, 88/116
6,557 markers are selected in total.
List of 22
 $ chr1 : int [1:545] 1 2 4 5 7 10 12 14 15 16 ...
 $ chr2 : int [1:540] 717 718 719 720 721 723 724 725 726 727 ...
 $ chr3 : int [1:455] 1459 1460 1461 1464 1466 1468 1469 1471 1472 1473 ...
 $ chr4 : int [1:413] 2068 2069 2070 2071 2072 2074 2075 2076 2077 2078 ...
 $ chr5 : int [1:435] 2630 2631 2633 2635 2636 2637 2638 2640 2642 2643 ...
 $ chr6 : int [1:428] 3196 3197 3198 3200 3201 3204 3205 3206 3207 3208 ...
 $ chr7 : int [1:356] 3761 3762 3763 3766 3767 3768 3770 3771 3772 3773 ...
 $ chr8 : int [1:347] 4233 4234 4235 4236 4237 4238 4239 4240 4241 4242 ...
 $ chr9 : int [1:324] 4721 4722 4724 4727 4728 4730 4731 4732 4733 4735 ...
 $ chr10: int [1:358] 5138 5139 5140 5143 5144 5145 5146 5147 5148 5149 ...
 $ chr11: int [1:348] 5620 5621 5623 5624 5625 5626 5628 5629 5630 5631 ...
 $ chr12: int [1:328] 6067 6068 6069 6070 6073 6074 6075 6077 6078 6079 ...
 $ chr13: int [1:262] 6494 6497 6498 6499 6500 6501 6503 6505 6507 6509 ...
 $ chr14: int [1:216] 6840 6841 6842 6843 6844 6845 6846 6847 6848 6850 ...
 $ chr15: int [1:200] 7120 7121 7122 7124 7125 7126 7127 7128 7129 7130 ...
 $ chr16: int [1:202] 7382 7383 7384 7385 7387 7388 7389 7391 7392 7394 ...
 $ chr17: int [1:153] 7660 7661 7662 7663 7664 7665 7666 7667 7668 7669 ...
 $ chr18: int [1:196] 7867 7868 7869 7870 7871 7872 7873 7874 7875 7877 ...
 $ chr19: int [1:102] 8133 8135 8136 8137 8138 8139 8140 8141 8142 8144 ...
 $ chr20: int [1:164] 8253 8254 8257 8258 8259 8260 8261 8262 8265 8266 ...
 $ chr21: int [1:97] 8482 8484 8485 8486 8487 8488 8489 8490 8491 8492 ...
 $ chr22: int [1:88] 8608 8609 8610 8612 8613 8614 8615 8617 8618 8619 ...
FUNCTION: snpgdsMergeGRM
Genetic Relationship Matrix (GRM, GCTA):
Excluding 2,288 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 6,800
    using 1 thread
GRM Calculation:    the sum of all selected genotypes (0,1,2) = 1908966
CPU capabilities:
Sat Mar 25 13:01:14 2023    (internal increment: 14768)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 3s
Sat Mar 25 13:01:17 2023    Done.
Genetic Relationship Matrix (GRM, GCTA):
Excluding 5,688 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 3,400
    using 1 thread
GRM Calculation:    the sum of all selected genotypes (0,1,2) = 951558
CPU capabilities:
Sat Mar 25 13:01:17 2023    (internal increment: 14768)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 3s
Saving to the GDS file:
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:20 2023    Done.
Genetic Relationship Matrix (GRM, GCTA):
Excluding 5,688 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 3,400
    using 1 thread
GRM Calculation:    the sum of all selected genotypes (0,1,2) = 957408
CPU capabilities:
Sat Mar 25 13:01:20 2023    (internal increment: 14768)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 3s
Saving to the GDS file:
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:23 2023    Done.
GRM merging:
    open &#x27;tmp1.gds&#x27; (3,400 variants)
    open &#x27;tmp2.gds&#x27; (3,400 variants)
Weight: 0.5, 0.5
Output: tmp.gds
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
GRM merging:
    open &#x27;tmp1.gds&#x27; (3,400 variants)
    open &#x27;tmp2.gds&#x27; (3,400 variants)
Weight: 0.5, 0.5
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
FUNCTION: snpgdsOpen
FUNCTION: snpgdsOption
FUNCTION: snpgdsPCA
Principal Component Analysis (PCA) on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
    # of principal components: 32
PCA:    the sum of all selected genotypes (0,1,2) = 2446510
CPU capabilities:
Sat Mar 25 13:01:23 2023    (internal increment: 14768)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 3s
Sat Mar 25 13:01:26 2023    Begin (eigenvalues and eigenvectors)
Sat Mar 25 13:01:26 2023    Done.
FUNCTION: snpgdsPCACorr
Principal Component Analysis (PCA) on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
    # of principal components: 32
PCA:    the sum of all selected genotypes (0,1,2) = 2446510
CPU capabilities:
Sat Mar 25 13:01:26 2023    (internal increment: 14768)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 3s
Sat Mar 25 13:01:29 2023    Begin (eigenvalues and eigenvectors)
Sat Mar 25 13:01:29 2023    Done.
SNP Correlation:
    # of samples: 279
    # of SNPs: 9,088
    using 1 thread
Correlation:    the sum of all selected genotypes (0,1,2) = 2553065
Sat Mar 25 13:01:29 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:29 2023    Done.
SNP Correlation:
    # of samples: 279
    # of SNPs: 9,088
    using 1 thread
Creating &#x27;test.gds&#x27; ...
Correlation:    the sum of all selected genotypes (0,1,2) = 2553065
Sat Mar 25 13:01:29 2023
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:29 2023    Done.
FUNCTION: snpgdsPCASNPLoading
Principal Component Analysis (PCA) on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
    # of principal components: 8
PCA:    the sum of all selected genotypes (0,1,2) = 2446510
CPU capabilities:
Sat Mar 25 13:01:29 2023    (internal increment: 14768)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 3s
Sat Mar 25 13:01:32 2023    Begin (eigenvalues and eigenvectors)
Sat Mar 25 13:01:32 2023    Done.
SNP Loading:
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
    using the top 8 eigenvectors
SNP Loading:    the sum of all selected genotypes (0,1,2) = 2446510
Sat Mar 25 13:01:32 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:32 2023    Done.
FUNCTION: snpgdsPCASampLoading
Principal Component Analysis (PCA) on genotypes:
Excluding 365 SNPs on non-autosomes
Excluding 1 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
    # of principal components: 8
PCA:    the sum of all selected genotypes (0,1,2) = 2446510
CPU capabilities:
Sat Mar 25 13:01:32 2023    (internal increment: 14768)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 3s
Sat Mar 25 13:01:35 2023    Begin (eigenvalues and eigenvectors)
Sat Mar 25 13:01:35 2023    Done.
SNP Loading:
    # of samples: 279
    # of SNPs: 8,722
    using 1 thread
    using the top 8 eigenvectors
SNP Loading:    the sum of all selected genotypes (0,1,2) = 2446510
Sat Mar 25 13:01:35 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:35 2023    Done.
Sample Loading:
    # of samples: 100
    # of SNPs: 8,722
    using 1 thread
    using the top 8 eigenvectors
Sample Loading:    the sum of all selected genotypes (0,1,2) = 878146
Sat Mar 25 13:01:35 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:35 2023    Done.
FUNCTION: snpgdsPED2GDS
Converting from GDS to PLINK PED:
	Output a MAP file DONE.
	Output a PED file ...
		Output: 	Sat Mar 25 13:01:35 2023	0%
		Output: 	Sat Mar 25 13:01:36 2023	100%
PLINK PED/MAP to GDS Format:
Import 9088 variants from &#x27;tmp.map&#x27;
Chromosome:
  1  10  11  12  13  14  15  16  17  18  19   2  20  21  22   3   4   5   6   7 
716 483 447 427 344 282 262 278 207 266 120 742 229 126 116 609 562 566 565 472 
  8   9   X 
488 416 365 
Reading &#x27;tmp.ped&#x27;
Output: &#x27;test.gds&#x27;
Import 279 samples
Transpose the genotypic matrix ...
Done.
Optimize the access efficiency ...
Clean up the fragments of GDS file:
    open the file &#x27;test.gds&#x27; (1.3M)
    # of fragments: 50
    save to &#x27;test.gds.tmp&#x27;
    rename &#x27;test.gds.tmp&#x27; (711.4K, reduced: 618.7K)
    # of fragments: 26
FUNCTION: snpgdsPairIBD
SNP pruning based on LD:
Excluding 365 SNPs on non-autosomes
Excluding 1,646 SNPs (monomorphic: TRUE, MAF: 0.05, missing rate: 0.05)
    # of samples: 93
    # of SNPs: 7,077
    using 1 thread
    sliding window: 500,000 basepairs, Inf SNPs
    |LD| threshold: 0.2
    method: composite
Chromosome 1: 62.29%, 446/716
Chromosome 2: 62.67%, 465/742
Chromosome 3: 59.93%, 365/609
Chromosome 4: 64.23%, 361/562
Chromosome 5: 62.37%, 353/566
Chromosome 6: 59.82%, 338/565
Chromosome 7: 63.14%, 298/472
Chromosome 8: 57.58%, 281/488
Chromosome 9: 62.98%, 262/416
Chromosome 10: 60.46%, 292/483
Chromosome 11: 63.09%, 282/447
Chromosome 12: 62.76%, 268/427
Chromosome 13: 63.08%, 217/344
Chromosome 14: 63.83%, 180/282
Chromosome 15: 63.74%, 167/262
Chromosome 16: 62.23%, 173/278
Chromosome 17: 65.70%, 136/207
Chromosome 18: 59.40%, 158/266
Chromosome 19: 68.33%, 82/120
Chromosome 20: 66.38%, 152/229
Chromosome 21: 61.11%, 77/126
Chromosome 22: 57.76%, 67/116
5,420 markers are selected in total.
Identity-By-Descent analysis (MLE) on genotypes:
Excluding 8,838 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 25
    # of SNPs: 250
    using 1 thread
Specifying allele frequencies, mean: 0.486, sd: 0.284
MLE IBD:    the sum of all selected genotypes (0,1,2) = 6112
MLE IBD:	Sat Mar 25 13:01:37 2023	0%
MLE IBD:	Sat Mar 25 13:01:37 2023	100%
IBD analysis (PLINK method of moment) on genotypes:
Excluding 8,838 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 25
    # of SNPs: 250
    using 1 thread
Specifying allele frequencies, mean: 0.486, sd: 0.284
*** A correction factor based on allele count is not used, since the allele frequencies are specified.
PLINK IBD:    the sum of all selected genotypes (0,1,2) = 6112
Sat Mar 25 13:01:37 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:37 2023    Done.
Identity-By-Descent analysis (MLE) on genotypes:
Excluding 8,838 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 25
    # of SNPs: 250
    using 1 thread
Specifying allele frequencies, mean: 0.486, sd: 0.284
MLE IBD:    the sum of all selected genotypes (0,1,2) = 6112
MLE IBD:	Sat Mar 25 13:01:37 2023	0%
MLE IBD:	Sat Mar 25 13:01:38 2023	100%
Genotype matrix: 250 SNPs X 25 samples
[1] -370.7482
[1] -402.2141
[1] -383.7897
[1] -377.9084
[1] -381.3139
[1] -397.5581
[1] -378.3344
[1] -370.703
[1] -376.103
[1] -377.7911
[1] -375.5425
[1] -373.13
[1] -383.6992
[1] -393.5194
[1] -371.9843
[1] -369.6468
[1] -374.5139
[1] -377.841
[1] -387.5622
[1] -377.1646
[1] -377.4659
[1] -375.2204
[1] -372.0639
[1] -379.816
FUNCTION: snpgdsPairIBDMLELogLik
SNP pruning based on LD:
Excluding 365 SNPs on non-autosomes
Excluding 1,646 SNPs (monomorphic: TRUE, MAF: 0.05, missing rate: 0.05)
    # of samples: 93
    # of SNPs: 7,077
    using 1 thread
    sliding window: 500,000 basepairs, Inf SNPs
    |LD| threshold: 0.2
    method: composite
Chromosome 1: 62.29%, 446/716
Chromosome 2: 62.67%, 465/742
Chromosome 3: 59.93%, 365/609
Chromosome 4: 64.23%, 361/562
Chromosome 5: 62.37%, 353/566
Chromosome 6: 59.82%, 338/565
Chromosome 7: 63.14%, 298/472
Chromosome 8: 57.58%, 281/488
Chromosome 9: 62.98%, 262/416
Chromosome 10: 60.46%, 292/483
Chromosome 11: 63.09%, 282/447
Chromosome 12: 62.76%, 268/427
Chromosome 13: 63.08%, 217/344
Chromosome 14: 63.83%, 180/282
Chromosome 15: 63.74%, 167/262
Chromosome 16: 62.23%, 173/278
Chromosome 17: 65.70%, 136/207
Chromosome 18: 59.40%, 158/266
Chromosome 19: 68.33%, 82/120
Chromosome 20: 66.38%, 152/229
Chromosome 21: 61.11%, 77/126
Chromosome 22: 57.76%, 67/116
5,420 markers are selected in total.
Identity-By-Descent analysis (MLE) on genotypes:
Excluding 8,838 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 25
    # of SNPs: 250
    using 1 thread
Specifying allele frequencies, mean: 0.486, sd: 0.284
MLE IBD:    the sum of all selected genotypes (0,1,2) = 6112
MLE IBD:	Sat Mar 25 13:01:38 2023	0%
MLE IBD:	Sat Mar 25 13:01:39 2023	100%
IBD analysis (PLINK method of moment) on genotypes:
Excluding 8,838 SNPs (non-autosomes or non-selection)
Excluding 0 SNP (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 25
    # of SNPs: 250
    using 1 thread
Specifying allele frequencies, mean: 0.486, sd: 0.284
*** A correction factor based on allele count is not used, since the allele frequencies are specified.
PLINK IBD:    the sum of all selected genotypes (0,1,2) = 6112
Sat Mar 25 13:01:39 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:39 2023    Done.
Genotype matrix: 250 SNPs X 25 samples
[1] -370.7482
[1] -402.2141
[1] -383.7897
[1] -377.9084
[1] -381.3139
[1] -397.5581
[1] -378.3344
[1] -370.703
[1] -376.103
[1] -377.7911
[1] -375.5425
[1] -373.13
[1] -383.6992
[1] -393.5194
[1] -371.9843
[1] -369.6468
[1] -374.5139
[1] -377.841
[1] -387.5622
[1] -377.1646
[1] -377.4659
[1] -375.2204
[1] -372.0639
[1] -379.816
FUNCTION: snpgdsPairScore
Excluding 365 SNPs on non-autosomes
Pair Score Calculation:
    # of samples: 120
    # of SNPs: 8,723
Method: IBS
Genotype Score:    the sum of all selected genotypes (0,1,2) = 1050236
List of 3
 $ sample.id: chr [1:120] &quot;NA19139&quot; &quot;NA19160&quot; &quot;NA07034&quot; &quot;NA12814&quot; ...
 $ snp.id   : int [1:8723] 1 2 3 4 5 6 7 8 9 10 ...
 $ score    :&#x27;data.frame&#x27;:	60 obs. of  5 variables:
  ..$ Avg    : num [1:60] 1.72 1.73 1.71 1.72 1.73 ...
  ..$ SD     : num [1:60] 0.452 0.443 0.457 0.45 0.443 ...
  ..$ Num    : int [1:60] 8684 8627 8669 8637 8682 8634 8654 8678 8680 8679 ...
  ..$ Sample1: chr [1:60] &quot;NA19139&quot; &quot;NA10847&quot; &quot;NA18515&quot; &quot;NA19129&quot; ...
  ..$ Sample2: chr [1:60] &quot;NA19138&quot; &quot;NA12146&quot; &quot;NA18516&quot; &quot;NA19128&quot; ...
Pair Score Calculation:
    # of samples: 120
    # of SNPs: 8,723
Method: IBS
Genotype Score:    the sum of all selected genotypes (0,1,2) = 1050236
List of 3
 $ sample.id: chr [1:120] &quot;NA19139&quot; &quot;NA19160&quot; &quot;NA07034&quot; &quot;NA12814&quot; ...
 $ snp.id   : int [1:8723] 1 2 3 4 5 6 7 8 9 10 ...
 $ score    :&#x27;data.frame&#x27;:	60 obs. of  5 variables:
  ..$ Avg    : num [1:60] 0.999 1 1 1 1 ...
  ..$ SD     : num [1:60] 0.024 0 0.0186 0.0215 0.0215 ...
  ..$ Num    : int [1:60] 8684 8627 8669 8637 8682 8634 8654 8678 8680 8679 ...
  ..$ Sample1: chr [1:60] &quot;NA19139&quot; &quot;NA10847&quot; &quot;NA18515&quot; &quot;NA19129&quot; ...
  ..$ Sample2: chr [1:60] &quot;NA19138&quot; &quot;NA12146&quot; &quot;NA18516&quot; &quot;NA19128&quot; ...
Pair Score Calculation:
    # of samples: 120
    # of SNPs: 8,723
Method: IBS
Genotype Score:    the sum of all selected genotypes (0,1,2) = 1050236
List of 3
 $ sample.id: chr [1:120] &quot;NA19139&quot; &quot;NA19160&quot; &quot;NA07034&quot; &quot;NA12814&quot; ...
 $ snp.id   : int [1:8723] 1 2 3 4 5 6 7 8 9 10 ...
 $ score    : num [1:3, 1:8723] 1.75 0.437 60 1.583 0.497 ...
  ..- attr(*, &quot;dimnames&quot;)=List of 2
  .. ..$ : chr [1:3] &quot;Avg&quot; &quot;SD&quot; &quot;Num&quot;
  .. ..$ : NULL
Pair Score Calculation:
    # of samples: 120
    # of SNPs: 8,723
Method: IBS
Genotype Score:    the sum of all selected genotypes (0,1,2) = 1050236
List of 3
 $ sample.id: chr [1:120] &quot;NA19139&quot; &quot;NA19160&quot; &quot;NA07034&quot; &quot;NA12814&quot; ...
 $ snp.id   : int [1:8723] 1 2 3 4 5 6 7 8 9 10 ...
 $ score    : int [1:60, 1:8723] 1 1 2 2 2 2 2 1 2 2 ...
Pair Score Calculation:
    # of samples: 120
    # of SNPs: 8,723
Method: IBS
Output: /home/biocbuild/bbs-3.17-bioc/meat/SNPRelate.Rcheck/tests/tmp.gds
Genotype Score:    the sum of all selected genotypes (0,1,2) = 1050236
FUNCTION: snpgdsSNPList
FUNCTION: snpgdsSNPListClass
FUNCTION: snpgdsSNPListIntersect
FUNCTION: snpgdsSNPRateFreq
FUNCTION: snpgdsSampMissRate
FUNCTION: snpgdsSelectSNP
Excluding 365 SNPs on non-autosomes
Excluding 1,221 SNPs (monomorphic: TRUE, MAF: 0.05, missing rate: 0.95)
FUNCTION: snpgdsSlidingWindow
Sliding Window Analysis:
Excluding 8 SNPs (monomorphic: TRUE, MAF: NaN, missing rate: NaN)
    # of samples: 279
    # of SNPs: 9,080
    using 1 thread
    window size: 500000, shift: 100000 (basepair)
Chromosome Set: 1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23
Sat Mar 25 13:01:39 2023, Chromosome 1 (716 SNPs), 2448 windows
Sat Mar 25 13:01:39 2023, Chromosome 2 (742 SNPs), 2416 windows
Sat Mar 25 13:01:40 2023, Chromosome 3 (609 SNPs), 1985 windows
Sat Mar 25 13:01:40 2023, Chromosome 4 (562 SNPs), 1894 windows
Sat Mar 25 13:01:40 2023, Chromosome 5 (566 SNPs), 1797 windows
Sat Mar 25 13:01:40 2023, Chromosome 6 (565 SNPs), 1694 windows
Sat Mar 25 13:01:40 2023, Chromosome 7 (472 SNPs), 1573 windows
Sat Mar 25 13:01:40 2023, Chromosome 8 (488 SNPs), 1445 windows
Sat Mar 25 13:01:40 2023, Chromosome 9 (416 SNPs), 1393 windows
Sat Mar 25 13:01:40 2023, Chromosome 10 (483 SNPs), 1343 windows
Sat Mar 25 13:01:40 2023, Chromosome 11 (447 SNPs), 1338 windows
Sat Mar 25 13:01:40 2023, Chromosome 12 (427 SNPs), 1316 windows
Sat Mar 25 13:01:40 2023, Chromosome 13 (344 SNPs), 948 windows
Sat Mar 25 13:01:40 2023, Chromosome 14 (281 SNPs), 847 windows
Sat Mar 25 13:01:40 2023, Chromosome 15 (262 SNPs), 774 windows
Sat Mar 25 13:01:40 2023, Chromosome 16 (278 SNPs), 873 windows
Sat Mar 25 13:01:40 2023, Chromosome 17 (207 SNPs), 773 windows
Sat Mar 25 13:01:40 2023, Chromosome 18 (266 SNPs), 753 windows
Sat Mar 25 13:01:40 2023, Chromosome 19 (120 SNPs), 627 windows
Sat Mar 25 13:01:40 2023, Chromosome 20 (229 SNPs), 602 windows
Sat Mar 25 13:01:40 2023, Chromosome 21 (126 SNPs), 311 windows
Sat Mar 25 13:01:40 2023, Chromosome 22 (116 SNPs), 312 windows
Sat Mar 25 13:01:40 2023, Chromosome 23 (358 SNPs), 1507 windows
Sat Mar 25 13:01:40 2023 	Done.
FUNCTION: snpgdsSummary
The file name: /home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/hapmap_geno.gds 
The total number of samples: 279 
The total number of SNPs: 9088 
SNP genotypes are stored in SNP-major mode (Sample X SNP).
FUNCTION: snpgdsTranspose
The file name: /home/biocbuild/bbs-3.17-bioc/meat/SNPRelate.Rcheck/tests/test.gds 
The total number of samples: 279 
The total number of SNPs: 9088 
SNP genotypes are stored in SNP-major mode (Sample X SNP).
SNP genotypes: 279 samples, 9088 SNPs
Genotype matrix is being transposed ...
Clean up the fragments of GDS file:
    open the file &#x27;test.gds&#x27; (1.3M)
    # of fragments: 28
    save to &#x27;test.gds.tmp&#x27;
    rename &#x27;test.gds.tmp&#x27; (709.6K, reduced: 619.1K)
    # of fragments: 26
The file name: /home/biocbuild/bbs-3.17-bioc/meat/SNPRelate.Rcheck/tests/test.gds 
The total number of samples: 279 
The total number of SNPs: 9088 
SNP genotypes are stored in individual-major mode (SNP X Sample).
FUNCTION: snpgdsVCF2GDS
##fileformat=VCFv4.1
##fileDate=20090805
##source=myImputationProgramV3.1
##reference=file:///seq/references/1000GenomesPilot-NCBI36.fasta
##contig=&lt;ID=20,length=62435964,assembly=B36,md5=f126cdf8a6e0c7f379d618ff66beb2da,species=&quot;Homo sapiens&quot;,taxonomy=x&gt;
##phasing=partial
##INFO=&lt;ID=NS,Number=1,Type=Integer,Description=&quot;Number of Samples With Data&quot;&gt;
##INFO=&lt;ID=DP,Number=1,Type=Integer,Description=&quot;Total Depth&quot;&gt;
##INFO=&lt;ID=AF,Number=A,Type=Float,Description=&quot;Allele Frequency&quot;&gt;
##INFO=&lt;ID=AA,Number=1,Type=String,Description=&quot;Ancestral Allele&quot;&gt;
##INFO=&lt;ID=DB,Number=0,Type=Flag,Description=&quot;dbSNP membership, build 129&quot;&gt;
##INFO=&lt;ID=H2,Number=0,Type=Flag,Description=&quot;HapMap2 membership&quot;&gt;
##FILTER=&lt;ID=q10,Description=&quot;Quality below 10&quot;&gt;
##FILTER=&lt;ID=s50,Description=&quot;Less than 50% of samples have data&quot;&gt;
##FORMAT=&lt;ID=GT,Number=1,Type=String,Description=&quot;Genotype&quot;&gt;
##FORMAT=&lt;ID=GQ,Number=1,Type=Integer,Description=&quot;Genotype Quality&quot;&gt;
##FORMAT=&lt;ID=DP,Number=1,Type=Integer,Description=&quot;Read Depth&quot;&gt;
##FORMAT=&lt;ID=HQ,Number=2,Type=Integer,Description=&quot;Haplotype Quality&quot;&gt;
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA00001	NA00002	NA00003
20	14370	rs6054257	G	A	29	PASS	NS=3;DP=14;AF=0.5;DB;H2	GT:GQ:DP:HQ	0|0:48:1:51,51	1|0:48:8:51,51	1/1:43:5:.,.
20	17330	.	T	A	3	q10	NS=3;DP=11;AF=0.017	GT:GQ:DP:HQ	0|0:49:3:58,50	0|1:3:5:65,3	0/0:41:3
20	1110696	rs6040355	A	G,T	67	PASS	NS=2;DP=10;AF=0.333,0.667;AA=T;DB	GT:GQ:DP:HQ	1|2:21:6:23,27	2|1:2:0:18,2	2/2:35:4
20	1230237	.	T	.	47	PASS	NS=3;DP=13;AA=T	GT:GQ:DP:HQ	0|0:54:7:56,60	0|0:48:4:51,51	0/0:61:2
20	1234567	microsat1	GTC	G,GTCT	50	PASS	NS=3;DP=9;AA=G	GT:GQ:DP	0/1:35:4	0/2:17:2	1/1:40:3
Start file conversion from VCF to SNP GDS ...
Method: extracting biallelic SNPs
Number of samples: 3
Parsing &quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/sequence.vcf&quot; ...
	import 2 variants.
+ genotype   { Bit2 3x2, 2B } *
Optimize the access efficiency ...
Clean up the fragments of GDS file:
    open the file &#x27;test1.gds&#x27; (2.9K)
    # of fragments: 46
    save to &#x27;test1.gds.tmp&#x27;
    rename &#x27;test1.gds.tmp&#x27; (2.6K, reduced: 312B)
    # of fragments: 20
The file name: /home/biocbuild/bbs-3.17-bioc/meat/SNPRelate.Rcheck/tests/test1.gds 
The total number of samples: 3 
The total number of SNPs: 2 
SNP genotypes are stored in SNP-major mode (Sample X SNP).
Start file conversion from VCF to SNP GDS ...
Method: extracting biallelic SNPs
Number of samples: 3
Parsing &quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/sequence.vcf&quot; ...
	import 2 variants.
+ genotype   { Bit2 3x2, 2B } *
SNP genotypes: 3 samples, 2 SNPs
Genotype matrix is being transposed ...
Optimize the access efficiency ...
Clean up the fragments of GDS file:
    open the file &#x27;test2.gds&#x27; (3.0K)
    # of fragments: 48
    save to &#x27;test2.gds.tmp&#x27;
    rename &#x27;test2.gds.tmp&#x27; (2.6K, reduced: 417B)
    # of fragments: 20
The file name: /home/biocbuild/bbs-3.17-bioc/meat/SNPRelate.Rcheck/tests/test2.gds 
The total number of samples: 3 
The total number of SNPs: 2 
SNP genotypes are stored in individual-major mode (SNP X Sample).
Start file conversion from VCF to SNP GDS ...
Method: dosage (0,1,2) of reference allele for all variant sites
Number of samples: 3
Parsing &quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/sequence.vcf&quot; ...
	import 5 variants.
+ genotype   { Bit2 3x5, 4B } *
SNP genotypes: 3 samples, 5 SNPs
Genotype matrix is being transposed ...
Optimize the access efficiency ...
Clean up the fragments of GDS file:
    open the file &#x27;test3.gds&#x27; (3.1K)
    # of fragments: 48
    save to &#x27;test3.gds.tmp&#x27;
    rename &#x27;test3.gds.tmp&#x27; (2.7K, reduced: 419B)
    # of fragments: 20
Some of &#x27;snp.allele&#x27; are not standard (e.g., A/G,T).
The file name: /home/biocbuild/bbs-3.17-bioc/meat/SNPRelate.Rcheck/tests/test3.gds 
The total number of samples: 3 
The total number of SNPs: 5 
SNP genotypes are stored in individual-major mode (SNP X Sample).
The number of valid samples: 3 
The number of biallelic unique SNPs: 2 
Start file conversion from VCF to SNP GDS ...
Method: dosage (0,1,2) of reference allele for all variant sites
Number of samples: 3
Parsing &quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/sequence.vcf&quot; ...
	import 5 variants.
+ genotype   { Bit2 3x5, 4B } *
Optimize the access efficiency ...
Clean up the fragments of GDS file:
    open the file &#x27;test4.gds&#x27; (3.0K)
    # of fragments: 46
    save to &#x27;test4.gds.tmp&#x27;
    rename &#x27;test4.gds.tmp&#x27; (2.7K, reduced: 312B)
    # of fragments: 20
Some of &#x27;snp.allele&#x27; are not standard (e.g., A/G,T).
The file name: /home/biocbuild/bbs-3.17-bioc/meat/SNPRelate.Rcheck/tests/test4.gds 
The total number of samples: 3 
The total number of SNPs: 5 
SNP genotypes are stored in SNP-major mode (Sample X SNP).
The number of valid samples: 3 
The number of biallelic unique SNPs: 2 
Start file conversion from VCF to SNP GDS ...
Method: dosage (0,1,2) of reference allele for all variant sites
Number of samples: 3
Parsing &quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/sequence.vcf&quot; ...
	import 5 variants.
+ genotype   { Bit2 3x5, 4B } *
Optimize the access efficiency ...
Clean up the fragments of GDS file:
    open the file &#x27;test5.gds&#x27; (3.0K)
    # of fragments: 46
    save to &#x27;test5.gds.tmp&#x27;
    rename &#x27;test5.gds.tmp&#x27; (2.7K, reduced: 312B)
    # of fragments: 20
Some of &#x27;snp.allele&#x27; are not standard (e.g., T/A,G).
The file name: /home/biocbuild/bbs-3.17-bioc/meat/SNPRelate.Rcheck/tests/test5.gds 
The total number of samples: 3 
The total number of SNPs: 5 
SNP genotypes are stored in SNP-major mode (Sample X SNP).
The number of valid samples: 3 
The number of biallelic unique SNPs: 2 
FUNCTION: snpgdsVCF2GDS_R
##fileformat=VCFv4.1
##fileDate=20090805
##source=myImputationProgramV3.1
##reference=file:///seq/references/1000GenomesPilot-NCBI36.fasta
##contig=&lt;ID=20,length=62435964,assembly=B36,md5=f126cdf8a6e0c7f379d618ff66beb2da,species=&quot;Homo sapiens&quot;,taxonomy=x&gt;
##phasing=partial
##INFO=&lt;ID=NS,Number=1,Type=Integer,Description=&quot;Number of Samples With Data&quot;&gt;
##INFO=&lt;ID=DP,Number=1,Type=Integer,Description=&quot;Total Depth&quot;&gt;
##INFO=&lt;ID=AF,Number=A,Type=Float,Description=&quot;Allele Frequency&quot;&gt;
##INFO=&lt;ID=AA,Number=1,Type=String,Description=&quot;Ancestral Allele&quot;&gt;
##INFO=&lt;ID=DB,Number=0,Type=Flag,Description=&quot;dbSNP membership, build 129&quot;&gt;
##INFO=&lt;ID=H2,Number=0,Type=Flag,Description=&quot;HapMap2 membership&quot;&gt;
##FILTER=&lt;ID=q10,Description=&quot;Quality below 10&quot;&gt;
##FILTER=&lt;ID=s50,Description=&quot;Less than 50% of samples have data&quot;&gt;
##FORMAT=&lt;ID=GT,Number=1,Type=String,Description=&quot;Genotype&quot;&gt;
##FORMAT=&lt;ID=GQ,Number=1,Type=Integer,Description=&quot;Genotype Quality&quot;&gt;
##FORMAT=&lt;ID=DP,Number=1,Type=Integer,Description=&quot;Read Depth&quot;&gt;
##FORMAT=&lt;ID=HQ,Number=2,Type=Integer,Description=&quot;Haplotype Quality&quot;&gt;
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA00001	NA00002	NA00003
20	14370	rs6054257	G	A	29	PASS	NS=3;DP=14;AF=0.5;DB;H2	GT:GQ:DP:HQ	0|0:48:1:51,51	1|0:48:8:51,51	1/1:43:5:.,.
20	17330	.	T	A	3	q10	NS=3;DP=11;AF=0.017	GT:GQ:DP:HQ	0|0:49:3:58,50	0|1:3:5:65,3	0/0:41:3
20	1110696	rs6040355	A	G,T	67	PASS	NS=2;DP=10;AF=0.333,0.667;AA=T;DB	GT:GQ:DP:HQ	1|2:21:6:23,27	2|1:2:0:18,2	2/2:35:4
20	1230237	.	T	.	47	PASS	NS=3;DP=13;AA=T	GT:GQ:DP:HQ	0|0:54:7:56,60	0|0:48:4:51,51	0/0:61:2
20	1234567	microsat1	GTC	G,GTCT	50	PASS	NS=3;DP=9;AA=G	GT:GQ:DP	0/1:35:4	0/2:17:2	1/1:40:3
Start snpgdsVCF2GDS ...
	Extracting bi-allelic and polymorhpic SNPs.
	Scanning ...
	file: /home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/sequence.vcf
	content: 5 rows x 12 columns
Sat Mar 25 13:01:41 2023 	store sample id, snp id, position, and chromosome.
	start writing: 3 samples, 2 SNPs ...
	file: /home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/sequence.vcf
[1] 1
Sat Mar 25 13:01:41 2023 	Done.
The file name: /home/biocbuild/bbs-3.17-bioc/meat/SNPRelate.Rcheck/tests/test1.gds 
The total number of samples: 3 
The total number of SNPs: 2 
SNP genotypes are stored in SNP-major mode (Sample X SNP).
Start snpgdsVCF2GDS ...
	Extracting bi-allelic and polymorhpic SNPs.
	Scanning ...
	file: /home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/sequence.vcf
	content: 5 rows x 12 columns
Sat Mar 25 13:01:41 2023 	store sample id, snp id, position, and chromosome.
	start writing: 3 samples, 2 SNPs ...
	file: /home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/sequence.vcf
[1] 1
Sat Mar 25 13:01:41 2023 	Done.
The file name: /home/biocbuild/bbs-3.17-bioc/meat/SNPRelate.Rcheck/tests/test2.gds 
The total number of samples: 3 
The total number of SNPs: 2 
SNP genotypes are stored in SNP-major mode (Sample X SNP).
Start snpgdsVCF2GDS ...
	Storing dosage of the reference allele for all variant sites, including bi-allelic SNPs, multi-allelic SNPs, indels and structural variants.
	Scanning ...
	file: /home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/sequence.vcf
	content: 5 rows x 12 columns
Sat Mar 25 13:01:41 2023 	store sample id, snp id, position, and chromosome.
	start writing: 3 samples, 5 SNPs ...
	file: /home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/sequence.vcf
Sat Mar 25 13:01:41 2023 	Done.
Some of &#x27;snp.allele&#x27; are not standard (e.g., A/G,T).
The file name: /home/biocbuild/bbs-3.17-bioc/meat/SNPRelate.Rcheck/tests/test3.gds 
The total number of samples: 3 
The total number of SNPs: 5 
SNP genotypes are stored in SNP-major mode (Sample X SNP).
The number of valid samples: 3 
The number of biallelic unique SNPs: 2 
Start snpgdsVCF2GDS ...
	Storing dosage of the reference allele for all variant sites, including bi-allelic SNPs, multi-allelic SNPs, indels and structural variants.
	Scanning ...
	file: /home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/sequence.vcf
	content: 5 rows x 12 columns
Sat Mar 25 13:01:41 2023 	store sample id, snp id, position, and chromosome.
	start writing: 3 samples, 5 SNPs ...
	file: /home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/SNPRelate/extdata/sequence.vcf
Sat Mar 25 13:01:41 2023 	Done.
Some of &#x27;snp.allele&#x27; are not standard (e.g., A/G,T).
The file name: /home/biocbuild/bbs-3.17-bioc/meat/SNPRelate.Rcheck/tests/test4.gds 
The total number of samples: 3 
The total number of SNPs: 5 
SNP genotypes are stored in SNP-major mode (Sample X SNP).
The number of valid samples: 3 
The number of biallelic unique SNPs: 2 
SNP Correlation:
    # of samples: 90
    # of SNPs: 9,088
    using 1 thread
Correlation:    the sum of all selected genotypes (0,1,2) = 824424
Sat Mar 25 13:01:44 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:44 2023    Done.
SNP Correlation:
    # of samples: 90
    # of SNPs: 9,088
    using 1 thread
Creating &#x27;test.gds&#x27; ...
Correlation:    the sum of all selected genotypes (0,1,2) = 824424
Sat Mar 25 13:01:44 2023
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:44 2023    Done.
SNP Loading:
    # of samples: 90
    # of SNPs: 8,695
    using 1 thread
    using the top 8 eigenvectors
SNP Loading:    the sum of all selected genotypes (0,1,2) = 787449
Sat Mar 25 13:01:44 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:44 2023    Done.
Sample Loading:
    # of samples: 100
    # of SNPs: 8,695
    using 1 thread
    using the top 8 eigenvectors
Sample Loading:    the sum of all selected genotypes (0,1,2) = 875255
Sat Mar 25 13:01:44 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:44 2023    Done.
SNP Correlation:
    # of samples: 90
    # of SNPs: 9,088
    using 2 threads
Correlation:    the sum of all selected genotypes (0,1,2) = 824424
Sat Mar 25 13:01:45 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:45 2023    Done.
SNP Correlation:
    # of samples: 90
    # of SNPs: 9,088
    using 2 threads
Creating &#x27;test.gds&#x27; ...
Correlation:    the sum of all selected genotypes (0,1,2) = 824424
Sat Mar 25 13:01:45 2023
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:45 2023    Done.
SNP Loading:
    # of samples: 90
    # of SNPs: 8,695
    using 1 thread
    using the top 8 eigenvectors
SNP Loading:    the sum of all selected genotypes (0,1,2) = 787449
Sat Mar 25 13:01:45 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:45 2023    Done.
Sample Loading:
    # of samples: 100
    # of SNPs: 8,695
    using 1 thread
    using the top 8 eigenvectors
Sample Loading:    the sum of all selected genotypes (0,1,2) = 875255
Sat Mar 25 13:01:45 2023    (internal increment: 65536)
[..................................................]  0%, ETC: ---        [==================================================] 100%, completed, 0s
Sat Mar 25 13:01:45 2023    Done.


RUNIT TEST PROTOCOL -- Sat Mar 25 13:01:45 2023 
*********************************************** 
Number of test functions: 13 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
SNPRelate RUnit Tests - 13 test functions, 0 errors, 0 failures
Number of test functions: 13 
Number of errors: 0 
Number of failures: 0 
&gt; 
&gt; proc.time()
   user  system elapsed 
137.602   1.512 138.204 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">SNPRelate.Rcheck/SNPRelate-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>SNPGDSFileClass-class</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.041</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SNPRelate-package</TD><TD style="text-align: right;">2.267</TD><TD style="text-align: right;">0.072</TD><TD style="text-align: right;">2.341</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsAdmixPlot</TD><TD style="text-align: right;">2.578</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">2.583</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsAdmixProp</TD><TD style="text-align: right;">2.394</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">2.395</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsAlleleSwitch</TD><TD style="text-align: right;">0.099</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.107</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsApartSelection</TD><TD style="text-align: right;">0.123</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.131</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsBED2GDS</TD><TD style="text-align: right;">0.123</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.128</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsClose</TD><TD style="text-align: right;">0.03</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.03</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsCombineGeno</TD><TD style="text-align: right;">0.128</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.132</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsCreateGeno</TD><TD style="text-align: right;">2.397</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">2.404</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsCreateGenoSet</TD><TD style="text-align: right;">0.236</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.236</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsCutTree</TD><TD style="text-align: right;">3.164</TD><TD style="text-align: right;">0.104</TD><TD style="text-align: right;">3.268</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsDiss</TD><TD style="text-align: right;">2.748</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">2.761</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsDrawTree</TD><TD style="text-align: right;">2.548</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">2.564</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsEIGMIX</TD><TD style="text-align: right;">2.540</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">2.564</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsErrMsg</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsExampleFileName</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsFst</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsGDS2BED</TD><TD style="text-align: right;">0.051</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.059</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsGDS2Eigen</TD><TD style="text-align: right;">0.124</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">0.172</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsGDS2PED</TD><TD style="text-align: right;">0.354</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.382</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsGEN2GDS</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsGRM</TD><TD style="text-align: right;">5.536</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">5.588</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsGetGeno</TD><TD style="text-align: right;">0.084</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.083</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsHCluster</TD><TD style="text-align: right;">2.794</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">2.813</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsHWE</TD><TD style="text-align: right;">0.017</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsIBDKING</TD><TD style="text-align: right;">2.188</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">2.211</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsIBDMLE</TD><TD style="text-align: right;">0.728</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.739</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsIBDMLELogLik</TD><TD style="text-align: right;">0.917</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.929</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsIBDMoM</TD><TD style="text-align: right;">0.378</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.386</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsIBDSelection</TD><TD style="text-align: right;">0.126</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.134</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsIBS</TD><TD style="text-align: right;">0.447</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.459</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsIBSNum</TD><TD style="text-align: right;">0.506</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.509</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsIndInb</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.035</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsIndInbCoef</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsIndivBeta</TD><TD style="text-align: right;">0.340</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.344</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsLDMat</TD><TD style="text-align: right;">0.290</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.306</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsLDpair</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsLDpruning</TD><TD style="text-align: right;">0.059</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.065</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsMergeGRM</TD><TD style="text-align: right;">8.111</TD><TD style="text-align: right;">0.068</TD><TD style="text-align: right;">8.180</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsOpen</TD><TD style="text-align: right;">0.026</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.026</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsOption</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsPCA</TD><TD style="text-align: right;">2.697</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">2.713</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsPCACorr</TD><TD style="text-align: right;">2.998</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">3.026</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsPCASNPLoading</TD><TD style="text-align: right;">2.763</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">2.771</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsPCASampLoading</TD><TD style="text-align: right;">2.444</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">2.463</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsPED2GDS</TD><TD style="text-align: right;">1.209</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">1.259</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsPairIBD</TD><TD style="text-align: right;">1.468</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">1.485</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsPairIBDMLELogLik</TD><TD style="text-align: right;">0.596</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.599</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsPairScore</TD><TD style="text-align: right;">0.219</TD><TD style="text-align: right;">0.096</TD><TD style="text-align: right;">0.316</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsSNPList</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsSNPListIntersect</TD><TD style="text-align: right;">0.053</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.054</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsSNPRateFreq</TD><TD style="text-align: right;">0.165</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.165</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsSampMissRate</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsSelectSNP</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsSlidingWindow</TD><TD style="text-align: right;">0.993</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">1.005</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsSummary</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsTranspose</TD><TD style="text-align: right;">0.125</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.132</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsVCF2GDS</TD><TD style="text-align: right;">0.203</TD><TD style="text-align: right;">0.060</TD><TD style="text-align: right;">0.265</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpgdsVCF2GDS_R</TD><TD style="text-align: right;">0.075</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.079</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
