<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for crisprDesign on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><B>[<A href="..#C">C</A>]</B></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-31 05:44:32 -0000 (Wed, 31 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4219</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for crisprDesign on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the crisprDesign package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/crisprDesign.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>443</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">crisprDesign</A>&nbsp;1.3.1</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/crisprDesign">landing page</A>)</SPAN><BR>Jean-Philippe Fortin<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-29&nbsp;10:19:22&nbsp;-0000&nbsp;(Mon,&nbsp;29&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/crisprDesign</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">7438ab6</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-26&nbsp;15:25:39&nbsp;-0000&nbsp;(Wed,&nbsp;26&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: crisprDesign</TD></TR>
<TR><TD><B>Version</B>: 1.3.1</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:crisprDesign.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings crisprDesign_1.3.1.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-30 04:31:59 -0000 (Tue, 30 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-30 04:45:09 -0000 (Tue, 30 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 790.0 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: crisprDesign.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:crisprDesign.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings crisprDesign_1.3.1.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/crisprDesign.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘crisprDesign/DESCRIPTION’ ... OK
* this is package ‘crisprDesign’ version ‘1.3.1’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .BBSoptions
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘crisprDesign’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking LazyData ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                   user system elapsed
addSNPAnnotation 17.568  0.224  17.805
addEditedAlleles 16.778  0.239  17.022
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘intro.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 NOTE
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/crisprDesign.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">crisprDesign.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD INSTALL crisprDesign
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0/site-library’
* installing *source* package ‘crisprDesign’ ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (crisprDesign)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">crisprDesign.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(crisprDesign)
Loading required package: crisprBase
&gt; 
&gt; test_check(&quot;crisprDesign&quot;)
[runCrisprBowtie] Using BSgenome.Hsapiens.UCSC.hg38 
[runCrisprBowtie] Searching for SpCas9 protospacers 
# reads processed: 7
# reads with at least one alignment: 6 (85.71%)
# reads that failed to align: 1 (14.29%)
Reported 6 alignments
[runCrisprBowtie] Using BSgenome.Hsapiens.UCSC.hg38 
[runCrisprBowtie] Searching for SpCas9 protospacers 
# reads processed: 7
# reads with at least one alignment: 6 (85.71%)
# reads that failed to align: 1 (14.29%)
Reported 6 alignments
[runCrisprBowtie] Using BSgenome.Hsapiens.UCSC.hg38 
[runCrisprBowtie] Searching for SpCas9 protospacers 
# reads processed: 6
# reads with at least one alignment: 6 (100.00%)
# reads that failed to align: 0 (0.00%)
Reported 6 alignments
[runCrisprBowtie] Using BSgenome.Hsapiens.UCSC.hg38 
[runCrisprBowtie] Searching for SpCas9 protospacers 
# reads processed: 6
# reads with at least one alignment: 6 (100.00%)
# reads that failed to align: 0 (0.00%)
Reported 6 alignments
[runCrisprBowtie] Using BSgenome.Hsapiens.UCSC.hg38 
[runCrisprBowtie] Searching for SpCas9 protospacers 
# reads processed: 6
# reads with at least one alignment: 6 (100.00%)
# reads that failed to align: 0 (0.00%)
Reported 6 alignments
[runCrisprBowtie] Using BSgenome.Hsapiens.UCSC.hg38 
[runCrisprBowtie] Searching for SpCas9 protospacers 
# reads processed: 6
# reads with at least one alignment: 6 (100.00%)
# reads that failed to align: 0 (0.00%)
Reported 6 alignments
[runCrisprBowtie] Using BSgenome.Hsapiens.UCSC.hg38 
[runCrisprBowtie] Searching for SpCas9 protospacers 
# reads processed: 6
# reads with at least one alignment: 6 (100.00%)
# reads that failed to align: 0 (0.00%)
Reported 6 alignments
[runCrisprBowtie] Using BSgenome.Hsapiens.UCSC.hg38 
[runCrisprBowtie] Searching for SpCas9 protospacers 
# reads processed: 6
# reads with at least one alignment: 6 (100.00%)
# reads that failed to align: 0 (0.00%)
Reported 6 alignments
[runCrisprBowtie] Using BSgenome.Hsapiens.UCSC.hg38 
[runCrisprBowtie] Searching for SpCas9 protospacers 
# reads processed: 6
# reads with at least one alignment: 6 (100.00%)
# reads that failed to align: 0 (0.00%)
Reported 6 alignments
[runCrisprBowtie] Using BSgenome.Hsapiens.UCSC.hg38 
[runCrisprBowtie] Searching for SpCas9 protospacers 
# reads processed: 6
# reads with at least one alignment: 6 (100.00%)
# reads that failed to align: 0 (0.00%)
Reported 6 alignments
[runCrisprBowtie] Using BSgenome.Hsapiens.UCSC.hg38 
[runCrisprBowtie] Searching for SpCas9 protospacers 
# reads processed: 6
# reads with at least one alignment: 6 (100.00%)
# reads that failed to align: 0 (0.00%)
Reported 6 alignments
[runCrisprBowtie] Using BSgenome.Hsapiens.UCSC.hg38 
[runCrisprBowtie] Searching for SpCas9 protospacers 
# reads processed: 6
# reads with at least one alignment: 6 (100.00%)
# reads that failed to align: 0 (0.00%)
Reported 6 alignments
[ FAIL 0 | WARN 0 | SKIP 78 | PASS 1018 ]

══ Skipped tests ═══════════════════════════════════════════════════════════════
• empty test (69)
• long run time (8)
• no del for guideSetExample (1)

[ FAIL 0 | WARN 0 | SKIP 78 | PASS 1018 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
238.342   5.018 244.461 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">crisprDesign.Rcheck/crisprDesign-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>GuideSet-class</TD><TD style="text-align: right;">0.193</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.198</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GuideSet2DataFrames</TD><TD style="text-align: right;">2.369</TD><TD style="text-align: right;">0.100</TD><TD style="text-align: right;">2.471</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>PairedGuideSet-class</TD><TD style="text-align: right;">0.536</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">0.581</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TxDb2GRangesList</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addCompositeScores</TD><TD style="text-align: right;">1.483</TD><TD style="text-align: right;">0.080</TD><TD style="text-align: right;">1.563</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addDistanceToTss</TD><TD style="text-align: right;">0.054</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.057</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addEditedAlleles</TD><TD style="text-align: right;">16.778</TD><TD style="text-align: right;"> 0.239</TD><TD style="text-align: right;">17.022</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addExonTable</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addGeneAnnotation</TD><TD style="text-align: right;">2.077</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">2.226</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addIsoformAnnotation</TD><TD style="text-align: right;">0.149</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.152</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addNtcs</TD><TD style="text-align: right;">0.461</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.464</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addOffTargetScores</TD><TD style="text-align: right;">0.338</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.346</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addOnTargetScores</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addOpsBarcodes</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addPamScores</TD><TD style="text-align: right;">0.631</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.648</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addRepeats</TD><TD style="text-align: right;">0.319</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.328</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addRestrictionEnzymes</TD><TD style="text-align: right;">1.051</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">1.062</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addSNPAnnotation</TD><TD style="text-align: right;">17.568</TD><TD style="text-align: right;"> 0.224</TD><TD style="text-align: right;">17.805</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addSequenceFeatures</TD><TD style="text-align: right;">0.607</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.643</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addSpacerAlignments</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addTssAnnotation</TD><TD style="text-align: right;">0.235</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.263</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addTxTable</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>completeSpacers</TD><TD style="text-align: right;">0.512</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.533</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>convertToMinMaxGRanges</TD><TD style="text-align: right;">0.078</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.086</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>convertToProtospacerGRanges</TD><TD style="text-align: right;">0.292</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.313</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>designOpsLibrary</TD><TD style="text-align: right;">0.152</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.155</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findSpacerPairs</TD><TD style="text-align: right;">2.876</TD><TD style="text-align: right;">0.140</TD><TD style="text-align: right;">3.017</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findSpacers</TD><TD style="text-align: right;">1.949</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">1.991</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getBarcodeDistanceMatrix</TD><TD style="text-align: right;">1.156</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">1.184</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getConsensusIsoform</TD><TD style="text-align: right;">0.092</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.092</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getMrnaSequences</TD><TD style="text-align: right;">0.402</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.413</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getPreMrnaSequences</TD><TD style="text-align: right;">0.366</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.374</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getTssObjectFromTxObject</TD><TD style="text-align: right;">0.077</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.077</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getTxDb</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getTxInfoDataFrame</TD><TD style="text-align: right;">0.900</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">0.932</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>preparePfamTable</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>queryTss</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>queryTxObject</TD><TD style="text-align: right;">0.034</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rankSpacers</TD><TD style="text-align: right;">2.412</TD><TD style="text-align: right;">0.060</TD><TD style="text-align: right;">2.473</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>removeRepeats</TD><TD style="text-align: right;">0.327</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.343</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>updateOpsLibrary</TD><TD style="text-align: right;">0.269</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.284</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>validateOpsLibrary</TD><TD style="text-align: right;">0.177</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.182</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
