<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for GenomicFeatures on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><B>[<A href="../index.html#G">G</A>]</B></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-08 01:15:05 -0000 (Wed, 08 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4164</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for GenomicFeatures on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the GenomicFeatures package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/GenomicFeatures.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>774</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">GenomicFeatures</A>&nbsp;1.51.4</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/GenomicFeatures">landing page</A>)</SPAN><BR>H. Pagès<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-06&nbsp;00:12:45&nbsp;-0000&nbsp;(Mon,&nbsp;06&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/GenomicFeatures</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">c864561</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-12-15&nbsp;02:06:06&nbsp;-0000&nbsp;(Thu,&nbsp;15&nbsp;Dec&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: GenomicFeatures</TD></TR>
<TR><TD><B>Version</B>: 1.51.4</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:GenomicFeatures.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings GenomicFeatures_1.51.4.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-07 02:27:33 -0000 (Tue, 07 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-07 02:35:47 -0000 (Tue, 07 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 493.9 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: GenomicFeatures.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:GenomicFeatures.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings GenomicFeatures_1.51.4.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/GenomicFeatures.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘GenomicFeatures/DESCRIPTION’ ... OK
* this is package ‘GenomicFeatures’ version ‘1.51.4’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Depends: includes the non-default packages:
  &#x27;BiocGenerics&#x27;, &#x27;S4Vectors&#x27;, &#x27;IRanges&#x27;, &#x27;GenomeInfoDb&#x27;,
  &#x27;GenomicRanges&#x27;, &#x27;AnnotationDbi&#x27;
Adding so many packages to the search path is excessive and importing
selectively is preferable.
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘GenomicFeatures’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
&#x27;:::&#x27; call which should be &#x27;::&#x27;: ‘rtracklayer:::tableNames’
  See the note in ?`:::` about the use of this operator.
Unexported objects imported by &#x27;:::&#x27; calls:
  ‘AnnotationDbi:::.getMetaValue’ ‘AnnotationDbi:::.valid.colnames’
  ‘AnnotationDbi:::.valid.metadata.table’
  ‘AnnotationDbi:::.valid.table.colnames’ ‘AnnotationDbi:::dbEasyQuery’
  ‘AnnotationDbi:::dbQuery’ ‘AnnotationDbi:::smartKeys’
  ‘BiocGenerics:::testPackage’ ‘GenomeInfoDb:::check_tax_id’
  ‘GenomeInfoDb:::getSeqlevelsReplacementMode’
  ‘GenomeInfoDb:::lookup_organism_by_tax_id’
  ‘GenomeInfoDb:::lookup_tax_id_by_organism’
  ‘GenomeInfoDb:::make_circ_flags_from_circ_seqs’
  ‘GenomeInfoDb:::normarg_new2old’
  ‘GenomicRanges:::unsafe.transcriptLocs2refLocs’
  ‘GenomicRanges:::unsafe.transcriptWidths’
  ‘IRanges:::regroupBySupergroup’ ‘S4Vectors:::V_recycle’
  ‘S4Vectors:::anyMissingOrOutside’ ‘S4Vectors:::decodeRle’
  ‘S4Vectors:::extract_data_frame_rows’ ‘S4Vectors:::quick_togroup’
  ‘biomaRt:::martBM’ ‘biomaRt:::martDataset’ ‘biomaRt:::martHost’
  ‘rtracklayer:::resourceDescription’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... WARNING
checkRd: (5) FeatureDb-class.Rd:31-34: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:30-33: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:34-39: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:40-43: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:44-52: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:53-59: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:60-65: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:66-71: \item in \describe must have non-empty label
checkRd: (5) TxDb-class.Rd:72-80: \item in \describe must have non-empty label
checkRd: (5) select-methods.Rd:21-25: \item in \describe must have non-empty label
checkRd: (5) select-methods.Rd:26-50: \item in \describe must have non-empty label
checkRd: (5) select-methods.Rd:51-55: \item in \describe must have non-empty label
checkRd: (5) select-methods.Rd:56-62: \item in \describe must have non-empty label
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘GenomicFeatures-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: mapToTranscripts
&gt; ### Title: Map range coordinates between transcripts and genome space
&gt; ### Aliases: coordinate-mapping mapToTranscripts
&gt; ###   mapToTranscripts,GenomicRanges,GenomicRanges-method
&gt; ###   mapToTranscripts,GenomicRanges,GRangesList-method
&gt; ###   mapToTranscripts,ANY,TxDb-method pmapToTranscripts
&gt; ###   pmapToTranscripts,GenomicRanges,GenomicRanges-method
&gt; ###   pmapToTranscripts,GenomicRanges,GRangesList-method
&gt; ###   pmapToTranscripts,GRangesList,GRangesList-method mapFromTranscripts
&gt; ###   mapFromTranscripts,GenomicRanges,GenomicRanges-method
&gt; ###   mapFromTranscripts,GenomicRanges,GRangesList-method
&gt; ###   pmapFromTranscripts
&gt; ###   pmapFromTranscripts,IntegerRanges,GenomicRanges-method
&gt; ###   pmapFromTranscripts,IntegerRanges,GRangesList-method
&gt; ###   pmapFromTranscripts,GenomicRanges,GenomicRanges-method
&gt; ###   pmapFromTranscripts,GenomicRanges,GRangesList-method
&gt; ### Keywords: methods utilities
&gt; 
&gt; ### ** Examples
&gt; 
&gt; ## ---------------------------------------------------------------------
&gt; ## A. Basic Use: Conversion between CDS and Exon coordinates and the
&gt; ##    genome
&gt; ## ---------------------------------------------------------------------
&gt; 
&gt; ## Gene &quot;Dgkb&quot; has ENTREZID &quot;217480&quot;:
&gt; library(org.Mm.eg.db)

&gt; Dgkb_geneid &lt;- get(&quot;Dgkb&quot;, org.Mm.egSYMBOL2EG)
&gt; 
&gt; ## The gene is on the positive strand, chromosome 12:
&gt; library(TxDb.Mmusculus.UCSC.mm10.knownGene)
&gt; txdb &lt;- TxDb.Mmusculus.UCSC.mm10.knownGene
&gt; tx_by_gene &lt;- transcriptsBy(txdb, by=&quot;gene&quot;)
&gt; Dgkb_transcripts &lt;- tx_by_gene[[Dgkb_geneid]]
&gt; Dgkb_transcripts  # all 7 Dgkb transcripts are on chr12, positive strand
GRanges object with 7 ranges and 2 metadata columns:
      seqnames            ranges strand |     tx_id              tx_name
         &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt;          &lt;character&gt;
  [1]    chr12 37817726-37982010      + |     94103 ENSMUST00000222337.1
  [2]    chr12 37880174-38136840      + |     94104 ENSMUST00000221176.1
  [3]    chr12 37880337-38632119      + |     94105 ENSMUST00000220990.1
  [4]    chr12 37880547-38580923      + |     94106 ENSMUST00000221540.1
  [5]    chr12 37880705-38634239      + |     94107 ENSMUST00000040500.8
  [6]    chr12 37880716-38175084      + |     94108 ENSMUST00000221098.1
  [7]    chr12 38019180-38174661      + |     94111 ENSMUST00000220606.1
  -------
  seqinfo: 66 sequences (1 circular) from mm10 genome
&gt; 
&gt; ## To map coordinates from local CDS or exon space to genome 
&gt; ## space use mapFromTranscripts().
&gt; 
&gt; ## When mapping CDS coordinates to genome space the &#x27;transcripts&#x27;
&gt; ## argument is the collection of CDS regions by transcript.
&gt; coord &lt;- GRanges(&quot;chr12&quot;, IRanges(4, width=1))
&gt; ## Get the names of the transcripts in the gene:
&gt; Dgkb_tx_names &lt;- mcols(Dgkb_transcripts)$tx_name
&gt; Dgkb_tx_names
[1] &quot;ENSMUST00000222337.1&quot; &quot;ENSMUST00000221176.1&quot; &quot;ENSMUST00000220990.1&quot;
[4] &quot;ENSMUST00000221540.1&quot; &quot;ENSMUST00000040500.8&quot; &quot;ENSMUST00000221098.1&quot;
[7] &quot;ENSMUST00000220606.1&quot;
&gt; ## Use these names to isolate the region of interest:
&gt; cds_by_tx &lt;- cdsBy(txdb, &quot;tx&quot;, use.names=TRUE)
&gt; Dgkb_cds_by_tx &lt;- cds_by_tx[intersect(Dgkb_tx_names, names(cds_by_tx))]
&gt; ## Dgkb CDS grouped by transcript (no-CDS transcripts omitted):
&gt; Dgkb_cds_by_tx
GRangesList object of length 4:
$ENSMUST00000222337.1
GRanges object with 1 range and 3 metadata columns:
      seqnames            ranges strand |    cds_id    cds_name exon_rank
         &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt; &lt;character&gt; &lt;integer&gt;
  [1]    chr12 37981941-37982010      + |    166108        &lt;NA&gt;         2
  -------
  seqinfo: 66 sequences (1 circular) from mm10 genome

$ENSMUST00000221176.1
GRanges object with 9 ranges and 3 metadata columns:
      seqnames            ranges strand |    cds_id    cds_name exon_rank
         &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt; &lt;character&gt; &lt;integer&gt;
  [1]    chr12 37981941-37982010      + |    166108        &lt;NA&gt;         2
  [2]    chr12 38084167-38084243      + |    166109        &lt;NA&gt;         3
  [3]    chr12 38087573-38087593      + |    166110        &lt;NA&gt;         4
  [4]    chr12 38100364-38100517      + |    166111        &lt;NA&gt;         5
  [5]    chr12 38114533-38114673      + |    166112        &lt;NA&gt;         6
  [6]    chr12 38120606-38120655      + |    166113        &lt;NA&gt;         7
  [7]    chr12 38124169-38124243      + |    166114        &lt;NA&gt;         8
  [8]    chr12 38127264-38127383      + |    166115        &lt;NA&gt;         9
  [9]    chr12 38136541-38136681      + |    166117        &lt;NA&gt;        10
  -------
  seqinfo: 66 sequences (1 circular) from mm10 genome

...
&lt;2 more elements&gt;
&gt; lengths(Dgkb_cds_by_tx)  # nb of CDS per transcript
ENSMUST00000222337.1 ENSMUST00000221176.1 ENSMUST00000220990.1 
                   1                    9                   24 
ENSMUST00000040500.8 
                  24 
&gt; ## A requirement for mapping from transcript space to genome space
&gt; ## is that seqnames in &#x27;x&#x27; match the names in &#x27;transcripts&#x27;.
&gt; names(Dgkb_cds_by_tx) &lt;- rep(seqnames(coord), length(Dgkb_cds_by_tx))
&gt; ## There are 6 results, one for each transcript.
&gt; mapFromTranscripts(coord, Dgkb_cds_by_tx)
GRanges object with 4 ranges and 2 metadata columns:
      seqnames    ranges strand |     xHits transcriptsHits
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt;       &lt;integer&gt;
  [1]    chr12  37981944      + |         1               1
  [2]    chr12  37981944      + |         1               2
  [3]    chr12  37981944      + |         1               3
  [4]    chr12  37981944      + |         1               4
  -------
  seqinfo: 66 sequences from an unspecified genome; no seqlengths
&gt; 
&gt; ## To map exon coordinates to genome space the &#x27;transcripts&#x27;
&gt; ## argument is the collection of exon regions by transcript.
&gt; coord &lt;- GRanges(&quot;chr12&quot;, IRanges(100, width=1))
&gt; ex_by_tx &lt;- exonsBy(txdb, &quot;tx&quot;, use.names=TRUE)
&gt; Dgkb_ex_by_tx &lt;- ex_by_tx[Dgkb_tx_names]
&gt; names(Dgkb_ex_by_tx) &lt;- rep(seqnames(coord), length(Dgkb_ex_by_tx))
&gt; ## Again the output has 6 results, one for each transcript.
&gt; mapFromTranscripts(coord, Dgkb_ex_by_tx)
GRanges object with 7 ranges and 2 metadata columns:
      seqnames    ranges strand |     xHits transcriptsHits
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt;       &lt;integer&gt;
  [1]    chr12  37817825      + |         1               1
  [2]    chr12  37880273      + |         1               2
  [3]    chr12  37981772      + |         1               3
  [4]    chr12  37880646      + |         1               4
  [5]    chr12  37880804      + |         1               5
  [6]    chr12  37880815      + |         1               6
  [7]    chr12  38019279      + |         1               7
  -------
  seqinfo: 66 sequences from an unspecified genome; no seqlengths
&gt; 
&gt; ## To go the reverse direction and map from genome space to
&gt; ## local CDS or exon space, use mapToTranscripts().
&gt; 
&gt; ## Genomic position 37981944 maps to CDS position 4:
&gt; coord &lt;- GRanges(&quot;chr12&quot;, IRanges(37981944, width=1))
&gt; mapToTranscripts(coord, Dgkb_cds_by_tx)
GRanges object with 4 ranges and 2 metadata columns:
      seqnames    ranges strand |     xHits transcriptsHits
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt;       &lt;integer&gt;
  [1]    chr12         4      + |         1               1
  [2]    chr12         4      + |         1               2
  [3]    chr12         4      + |         1               3
  [4]    chr12         4      + |         1               4
  -------
  seqinfo: 1 sequence from an unspecified genome
&gt; 
&gt; ## Genomic position 37880273 maps to exon position 100:
&gt; coord &lt;- GRanges(&quot;chr12&quot;, IRanges(37880273, width=1))
&gt; mapToTranscripts(coord, Dgkb_ex_by_tx)
GRanges object with 1 range and 2 metadata columns:
      seqnames    ranges strand |     xHits transcriptsHits
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt;       &lt;integer&gt;
  [1]    chr12       100      + |         1               2
  -------
  seqinfo: 1 sequence from an unspecified genome
&gt; 
&gt; 
&gt; ## The following examples use more than 2GB of memory, which is more
&gt; ## than what 32-bit Windows can handle:
&gt; is_32bit_windows &lt;- .Platform$OS.type == &quot;windows&quot; &amp;&amp;
+                     .Platform$r_arch == &quot;i386&quot;
&gt; if (!is_32bit_windows) {
+ ## ---------------------------------------------------------------------
+ ## B. Map sequence locations in exons to the genome
+ ## ---------------------------------------------------------------------
+ 
+ ## NAGNAG alternative splicing plays an essential role in biological
+ ## processes and represents a highly adaptable system for
+ ## posttranslational regulation of gene function. The majority of
+ ## NAGNAG studies largely focus on messenger RNA. A study by Sun,
+ ## Lin, and Yan (http://www.hindawi.com/journals/bmri/2014/736798/)
+ ## demonstrated that NAGNAG splicing is also operative in large
+ ## intergenic noncoding RNA (lincRNA). One finding of interest was
+ ## that linc-POLR3G-10 exhibited two NAGNAG acceptors located in two
+ ## distinct transcripts: TCONS_00010012 and TCONS_00010010.
+ 
+ ## Extract the exon coordinates of TCONS_00010012 and TCONS_00010010:
+ lincrna &lt;- c(&quot;TCONS_00010012&quot;, &quot;TCONS_00010010&quot;)
+ library(TxDb.Hsapiens.UCSC.hg19.lincRNAsTranscripts)
+ txdb &lt;- TxDb.Hsapiens.UCSC.hg19.lincRNAsTranscripts
+ exons &lt;- exonsBy(txdb, by=&quot;tx&quot;, use.names=TRUE)[lincrna]
+ exons
+ 
+ ## The two NAGNAG acceptors were identified in the upstream region of
+ ## the fourth and fifth exons located in TCONS_00010012.
+ ## Extract the sequences for transcript TCONS_00010012:
+ library(BSgenome.Hsapiens.UCSC.hg19)
+ genome &lt;- BSgenome.Hsapiens.UCSC.hg19
+ exons_seq &lt;- getSeq(genome, exons[[1]])
+ 
+ ## TCONS_00010012 has 4 exons:
+ exons_seq
+ 
+ ## The most common triplet among the lincRNA sequences was CAG. Identify
+ ## the location of this pattern in all exons.
+ cag_loc &lt;- vmatchPattern(&quot;CAG&quot;, exons_seq)
+ 
+ ## Convert the first occurance of CAG in each exon back to genome
+ ## coordinates.
+ first_loc &lt;- do.call(c, sapply(cag_loc, &quot;[&quot;, 1, simplify=TRUE))
+ pmapFromTranscripts(first_loc, exons[[1]])
+ 
+ ## ---------------------------------------------------------------------
+ ## C. Map dbSNP variants to CDS or cDNA coordinates
+ ## ---------------------------------------------------------------------
+ 
+ ## The GIPR gene encodes a G-protein coupled receptor for gastric
+ ## inhibitory polypeptide (GIP). Originally GIP was identified to
+ ## inhibited gastric acid secretion and gastrin release but was later
+ ## demonstrated to stimulate insulin release in the presence of elevated
+ ## glucose.
+ 
+ ## In this example 5 SNPs located in the GIPR gene are mapped to cDNA
+ ## coordinates. A list of SNPs in GIPR can be downloaded from dbSNP or
+ ## NCBI.
+ rsids &lt;- c(&quot;rs4803846&quot;, &quot;rs139322374&quot;, &quot;rs7250736&quot;, &quot;rs7250754&quot;,
+            &quot;rs9749185&quot;)
+ 
+ ## Extract genomic coordinates with a SNPlocs package.
+ library(SNPlocs.Hsapiens.dbSNP144.GRCh38)
+ snps &lt;- snpsById(SNPlocs.Hsapiens.dbSNP144.GRCh38, rsids)
+ 
+ ## Gene regions of GIPR can be extracted from a TxDb package of
+ ## compatible build. The TxDb package uses Entrez gene identifiers
+ ## and GIPR is a gene symbol. Let&#x27;s first lookup its Entrez gene ID.
+ library(org.Hs.eg.db)
+ GIPR_geneid &lt;- get(&quot;GIPR&quot;, org.Hs.egSYMBOL2EG)
+ 
+ ## The transcriptsBy() extractor returns a range for each transcript that
+ ## includes the UTR and exon regions (i.e., cDNA).
+ library(TxDb.Hsapiens.UCSC.hg38.knownGene)
+ txdb &lt;- TxDb.Hsapiens.UCSC.hg38.knownGene
+ tx_by_gene &lt;- transcriptsBy(txdb, &quot;gene&quot;)
+ GIPR_transcripts &lt;- tx_by_gene[GIPR_geneid]
+ GIPR_transcripts  # all 8 GIPR transcripts are on chr19, positive strand
+ 
+ ## Before mapping, the chromosome names (seqlevels) in the two
+ ## objects must be harmonized. The style is NCBI for &#x27;snps&#x27; and
+ ## UCSC for &#x27;GIPR_transcripts&#x27;.
+ seqlevelsStyle(snps)
+ seqlevelsStyle(GIPR_transcripts)
+ 
+ ## Modify the style (and genome) in &#x27;snps&#x27; to match &#x27;GIPR_transcripts&#x27;.
+ seqlevelsStyle(snps) &lt;- seqlevelsStyle(GIPR_transcripts)
+ 
+ ## The &#x27;GIPR_transcripts&#x27; object is a GRangesList of length 1. This single
+ ## list element contains the cDNA range for 8 different transcripts. To
+ ## map to each transcript individually &#x27;GIPR_transcripts&#x27; must be unlisted
+ ## before mapping.
+ 
+ ## Map all 5 SNPS to all 8 transcripts:
+ mapToTranscripts(snps, unlist(GIPR_transcripts))
+ 
+ ## Map the first SNP to transcript &quot;ENST00000590918.5&quot; and the second to
+ ## &quot;ENST00000263281.7&quot;.
+ pmapToTranscripts(snps[1:2], unlist(GIPR_transcripts)[1:2])
+ 
+ ## The cdsBy() extractor returns coding regions by gene or by transcript.
+ ## Extract the coding regions for transcript &quot;ENST00000263281.7&quot;.
+ cds &lt;- cdsBy(txdb, &quot;tx&quot;, use.names=TRUE)[&quot;ENST00000263281.7&quot;]
+ cds
+ 
+ ## The &#x27;cds&#x27; object is a GRangesList of length 1 containing all CDS ranges
+ ## for the single transcript &quot;ENST00000263281.7&quot;.
+ 
+ ## To map to the concatenated group of ranges leave &#x27;cds&#x27; as a GRangesList.
+ mapToTranscripts(snps, cds)
+ 
+ ## Only the second SNP could be mapped. Unlisting the &#x27;cds&#x27; object maps
+ ## the SNPs to the individual cds ranges (vs the concatenated range).
+ mapToTranscripts(snps[2], unlist(cds))
+ 
+ ## The location is the same because the SNP hit the first CDS range. If
+ ## the transcript were on the &quot;-&quot; strand the difference in concatenated
+ ## vs non-concatenated position would be more obvious.
+ 
+ ## Change strand:
+ strand(cds) &lt;- strand(snps) &lt;- &quot;-&quot;
+ mapToTranscripts(snps[2], unlist(cds))
+ }
Loading required package: BSgenome
Loading required package: Biostrings
Loading required package: XVector

Attaching package: ‘Biostrings’

The following object is masked from ‘package:base’:

    strsplit

Loading required package: rtracklayer
Killed
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘run_unitTests.R’
 ERROR
Running the tests in ‘tests/run_unitTests.R’ failed.
Last 13 lines of output:
  4: In valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
    GRanges object contains 4 out-of-bound ranges located on sequences 1,
    2, 3, and 4. Note that ranges located on a sequence whose length is
    unknown (NA) or on a circular sequence are not considered out-of-bound
    (use seqlengths() and isCircular() to get the lengths and circularity
    flags of the underlying sequences). You can use trim() to trim these
    ranges. See ?`trim,GenomicRanges-method` for more information.
  Import genomic features from the file as a GRanges object ... OK
  Prepare the &#x27;metadata&#x27; data frame ... OK
  Make the TxDb object ... OK
  Import genomic features from the file as a GRanges object ... OK
  Prepare the &#x27;metadata&#x27; data frame ... OK
  Make the TxDb object ... OK
  Import genomic features from the file as a GRanges object ... OK
  Prepare the &#x27;metadata&#x27; data frame ... Killed
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘GenomicFeatures.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 2 ERRORs, 1 WARNING, 2 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/GenomicFeatures.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">GenomicFeatures.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL GenomicFeatures
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘GenomicFeatures’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (GenomicFeatures)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">GenomicFeatures.Rcheck/tests/run_unitTests.Rout.fail</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; require(&quot;GenomicFeatures&quot;) || stop(&quot;unable to load GenomicFeatures package&quot;)
Loading required package: GenomicFeatures
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
Loading required package: GenomicRanges
Loading required package: AnnotationDbi
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

[1] TRUE
&gt; GenomicFeatures:::.test()
Loading required package: BSgenome
Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: rtracklayer
Timing stopped at: 3.231 0.036 24.19
Error in curl::curl_fetch_memory(url, handle = handle) : 
  Timeout was reached: [www.ensembl.org:443] Operation timed out after 10000 milliseconds with 0 bytes received
In addition: Warning messages:
1: In valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
  GRanges object contains 3 out-of-bound ranges located on sequences a,
  b, and c. Note that ranges located on a sequence whose length is
  unknown (NA) or on a circular sequence are not considered out-of-bound
  (use seqlengths() and isCircular() to get the lengths and circularity
  flags of the underlying sequences). You can use trim() to trim these
  ranges. See ?`trim,GenomicRanges-method` for more information.
2: In valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
  GRanges object contains 1 out-of-bound range located on sequence c.
  Note that ranges located on a sequence whose length is unknown (NA) or
  on a circular sequence are not considered out-of-bound (use
  seqlengths() and isCircular() to get the lengths and circularity flags
  of the underlying sequences). You can use trim() to trim these ranges.
  See ?`trim,GenomicRanges-method` for more information.
3: In valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
  GRanges object contains 1 out-of-bound range located on sequence c.
  Note that ranges located on a sequence whose length is unknown (NA) or
  on a circular sequence are not considered out-of-bound (use
  seqlengths() and isCircular() to get the lengths and circularity flags
  of the underlying sequences). You can use trim() to trim these ranges.
  See ?`trim,GenomicRanges-method` for more information.
4: In valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
  GRanges object contains 4 out-of-bound ranges located on sequences 1,
  2, 3, and 4. Note that ranges located on a sequence whose length is
  unknown (NA) or on a circular sequence are not considered out-of-bound
  (use seqlengths() and isCircular() to get the lengths and circularity
  flags of the underlying sequences). You can use trim() to trim these
  ranges. See ?`trim,GenomicRanges-method` for more information.
Import genomic features from the file as a GRanges object ... OK
Prepare the &#x27;metadata&#x27; data frame ... OK
Make the TxDb object ... OK
Import genomic features from the file as a GRanges object ... OK
Prepare the &#x27;metadata&#x27; data frame ... OK
Make the TxDb object ... OK
Import genomic features from the file as a GRanges object ... OK
Prepare the &#x27;metadata&#x27; data frame ... Killed
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">GenomicFeatures.Rcheck/GenomicFeatures-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>FeatureDb-class</TD><TD style="text-align: right;">0.069</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.070</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TxDb-class</TD><TD style="text-align: right;">0.809</TD><TD style="text-align: right;">0.059</TD><TD style="text-align: right;">0.876</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>as-format-methods</TD><TD style="text-align: right;">2.505</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">2.528</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
