<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for Biostrings on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.17</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><B>[<A href="..#B">B</A>]</B></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-27 02:34:30 -0000 (Mon, 27 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4259</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for Biostrings on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the Biostrings package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/Biostrings.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>200</B>/2169</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">Biostrings</A>&nbsp;2.67.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/Biostrings">landing page</A>)</SPAN><BR>H. Pagès<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-23&nbsp;09:40:21&nbsp;-0000&nbsp;(Thu,&nbsp;23&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/Biostrings</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">c94e8fb</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;14:36:29&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: Biostrings</TD></TR>
<TR><TD><B>Version</B>: 2.67.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:Biostrings.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings Biostrings_2.67.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-24 07:57:12 -0000 (Fri, 24 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-24 08:07:49 -0000 (Fri, 24 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 636.9 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: Biostrings.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:Biostrings.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings Biostrings_2.67.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/Biostrings.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘Biostrings/DESCRIPTION’ ... OK
* this is package ‘Biostrings’ version ‘2.67.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘Biostrings’ can be installed ... OK
* used C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... NOTE
  installed size is 14.6Mb
  sub-directories of 1Mb or more:
    R         1.7Mb
    extdata  11.1Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... NOTE
Package listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘methods’
A package should be listed in only one of these fields.
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
Unexported objects imported by &#x27;:::&#x27; calls:
  ‘BiocGenerics:::testPackage’ ‘IRanges:::.showAtomicList’
  ‘IRanges:::from_Vector_to_CompressedList’
  ‘IRanges:::new_CompressedList_from_list’ ‘IRanges:::new_Views’
  ‘IRanges:::regroupBySupergroup’ ‘IRanges:::show_IntegerRangesList’
  ‘IRanges:::unlist_as_integer’ ‘S4Vectors:::anyMissingOrOutside’
  ‘XVector:::close_filexp’
  ‘XVector:::extract_character_from_XRaw_by_positions’
  ‘XVector:::extract_character_from_XRaw_by_ranges’
  ‘XVector:::new_XVectorList_from_list_of_XVector’
  ‘XVector:::open_output_file’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... WARNING
checkRd: (5) AAString-class.Rd:74-78: \item in \describe must have non-empty label
checkRd: (5) AAString-class.Rd:86-92: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:87-90: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:91-96: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:97-99: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:100-103: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:104-107: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:108-111: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:112-116: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:117-121: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:122-125: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:126-129: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:130-133: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:134-137: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:138-141: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:147-151: \item in \describe must have non-empty label
checkRd: (5) AlignedXStringSet-class.Rd:152-156: \item in \describe must have non-empty label
checkRd: (5) DNAString-class.Rd:57-61: \item in \describe must have non-empty label
checkRd: (5) DNAString-class.Rd:69-75: \item in \describe must have non-empty label
checkRd: (5) InDel-class.Rd:32-35: \item in \describe must have non-empty label
checkRd: (5) InDel-class.Rd:36-39: \item in \describe must have non-empty label
checkRd: (5) MIndex-class.Rd:49-52: \item in \describe must have non-empty label
checkRd: (5) MIndex-class.Rd:53-56: \item in \describe must have non-empty label
checkRd: (5) MIndex-class.Rd:57-61: \item in \describe must have non-empty label
checkRd: (5) MIndex-class.Rd:62-66: \item in \describe must have non-empty label
checkRd: (5) MIndex-class.Rd:67-71: \item in \describe must have non-empty label
checkRd: (5) MIndex-class.Rd:79-83: \item in \describe must have non-empty label
checkRd: (5) MIndex-class.Rd:91-99: \item in \describe must have non-empty label
checkRd: (5) MIndex-class.Rd:110-115: \item in \describe must have non-empty label
checkRd: (5) MIndex-class.Rd:116-119: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:113-116: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:117-122: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:123-140: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:141-145: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:146-152: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:160-164: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:165-168: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:169-172: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:180-185: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:193-197: \item in \describe must have non-empty label
checkRd: (5) MaskedXString-class.Rd:198-203: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:157-161: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:162-167: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:168-178: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:179-189: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:190-200: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:201-216: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:217-220: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:221-224: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:225-228: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:229-232: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:233-236: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:237-240: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:241-244: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:245-249: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:250-253: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:261-267: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:268-274: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:275-282: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:290-298: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:299-304: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:305-312: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:313-320: \item in \describe must have non-empty label
checkRd: (5) MultipleAlignment-class.Rd:321-337: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:241-244: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:245-249: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:250-253: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:254-257: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:258-261: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:262-269: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:270-273: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:282-285: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:294-297: \item in \describe must have non-empty label
checkRd: (5) PDict-class.Rd:298-301: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:154-160: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:161-165: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:166-169: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:177-180: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:181-187: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:188-193: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:202-210: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:211-214: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:215-218: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:219-222: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:230-234: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:235-238: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:239-243: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:244-250: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:258-262: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:263-268: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:269-274: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:275-279: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:280-283: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:284-289: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:295-299: \item in \describe must have non-empty label
checkRd: (5) PairwiseAlignments-class.Rd:300-304: \item in \describe must have non-empty label
checkRd: (5) QualityScaledXStringSet-class.Rd:111-114: \item in \describe must have non-empty label
checkRd: (5) QualityScaledXStringSet-class.Rd:115-119: \item in \describe must have non-empty label
checkRd: (5) QualityScaledXStringSet-class.Rd:120-123: \item in \describe must have non-empty label
checkRd: (5) QualityScaledXStringSet-class.Rd:124-129: \item in \describe must have non-empty label
checkRd: (5) QualityScaledXStringSet-class.Rd:130-133: \item in \describe must have non-empty label
checkRd: (5) QualityScaledXStringSet-class.Rd:143-146: \item in \describe must have non-empty label
checkRd: (5) RNAString-class.Rd:55-59: \item in \describe must have non-empty label
checkRd: (5) RNAString-class.Rd:67-73: \item in \describe must have non-empty label
checkRd: (5) XString-class.Rd:94-98: \item in \describe must have non-empty label
checkRd: (5) XString-class.Rd:106-111: \item in \describe must have non-empty label
checkRd: (5) XString-class.Rd:112-115: \item in \describe must have non-empty label
checkRd: (5) XString-class.Rd:123-126: \item in \describe must have non-empty label
checkRd: (5) XString-class.Rd:127-130: \item in \describe must have non-empty label
checkRd: (5) XString-class.Rd:138-148: \item in \describe must have non-empty label
checkRd: (5) XString-class.Rd:156-169: \item in \describe must have non-empty label
checkRd: (5) XString-class.Rd:170-173: \item in \describe must have non-empty label
checkRd: (5) XStringPartialMatches-class.Rd:32-35: \item in \describe must have non-empty label
checkRd: (5) XStringPartialMatches-class.Rd:36-39: \item in \describe must have non-empty label
checkRd: (5) XStringPartialMatches-class.Rd:49-54: \item in \describe must have non-empty label
checkRd: (-1) XStringQuality-class.Rd:86: Escaped LaTeX specials: \#
checkRd: (5) XStringQuality-class.Rd:106-109: \item in \describe must have non-empty label
checkRd: (5) XStringQuality-class.Rd:111-115: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:116-118: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:119-121: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:122-124: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:135-138: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:139-145: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:146-155: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:156-162: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:163-166: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:176-193: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:194-204: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:205-215: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:216-221: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:231-234: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:235-238: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:239-242: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:251-254: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:255-258: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:259-262: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:263-266: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:275-280: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:281-286: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:287-290: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:291-298: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:299-302: \item in \describe must have non-empty label
checkRd: (5) XStringSet-class.Rd:303-311: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:55-65: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:66-72: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:80-85: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:86-90: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:91-94: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:95-98: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:106-109: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:110-113: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:122-126: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:127-131: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:141-143: \item in \describe must have non-empty label
checkRd: (5) XStringSet-comparison.Rd:144-146: \item in \describe must have non-empty label
checkRd: (5) XStringViews-class.Rd:73-77: \item in \describe must have non-empty label
checkRd: (5) XStringViews-class.Rd:87-93: \item in \describe must have non-empty label
checkRd: (5) XStringViews-class.Rd:103-117: \item in \describe must have non-empty label
checkRd: (5) XStringViews-class.Rd:118-121: \item in \describe must have non-empty label
checkRd: (5) XStringViews-class.Rd:122-131: \item in \describe must have non-empty label
checkRd: (5) XStringViews-class.Rd:136-143: \item in \describe must have non-empty label
checkRd: (5) XStringViews-class.Rd:144-147: \item in \describe must have non-empty label
checkRd: (5) injectHardMask.Rd:58-61: \item in \describe must have non-empty label
checkRd: (5) injectHardMask.Rd:62-70: \item in \describe must have non-empty label
checkRd: (5) injectHardMask.Rd:71-74: \item in \describe must have non-empty label
checkRd: (-1) matchLRPatterns.Rd:65: Escaped LaTeX specials: \_ \_
checkRd: (-1) matchProbePair.Rd:53: Escaped LaTeX specials: \_ \_
checkRd: (-1) matchProbePair.Rd:54: Escaped LaTeX specials: \_ \_
checkRd: (-1) matchProbePair.Rd:55: Escaped LaTeX specials: \_ \_
checkRd: (-1) matchProbePair.Rd:56: Escaped LaTeX specials: \_ \_
checkRd: (-1) phiX174Phage.Rd:18: Escaped LaTeX specials: \_
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... WARNING
Undocumented code objects:
  ‘strsplit’ ‘twoWayAlphabetFrequency’
Undocumented S4 methods:
  generic &#x27;match&#x27; and siglist &#x27;Vector,XStringSet&#x27;
  generic &#x27;match&#x27; and siglist &#x27;XStringSet,Vector&#x27;
  generic &#x27;match&#x27; and siglist &#x27;XStringSet,vector&#x27;
  generic &#x27;match&#x27; and siglist &#x27;vector,XStringSet&#x27;
  generic &#x27;parallel_slot_names&#x27; and siglist &#x27;ByPos_MIndex&#x27;
  generic &#x27;parallel_slot_names&#x27; and siglist &#x27;MIndex&#x27;
  generic &#x27;pcompare&#x27; and siglist &#x27;Vector,XStringSet&#x27;
  generic &#x27;pcompare&#x27; and siglist &#x27;XStringSet,Vector&#x27;
  generic &#x27;pcompare&#x27; and siglist &#x27;XStringSet,vector&#x27;
  generic &#x27;pcompare&#x27; and siglist &#x27;vector,XStringSet&#x27;
  generic &#x27;relistToClass&#x27; and siglist &#x27;XString&#x27;
  generic &#x27;strsplit&#x27; and siglist &#x27;XStringSet&#x27;
  generic &#x27;twoWayAlphabetFrequency&#x27; and siglist &#x27;XString,XString&#x27;
  generic &#x27;twoWayAlphabetFrequency&#x27; and siglist &#x27;XString,XStringSet&#x27;
  generic &#x27;twoWayAlphabetFrequency&#x27; and siglist &#x27;XStringSet,XString&#x27;
  generic &#x27;twoWayAlphabetFrequency&#x27; and siglist &#x27;XStringSet,XStringSet&#x27;
  generic &#x27;unstrsplit&#x27; and siglist &#x27;XStringSet&#x27;
  generic &#x27;unstrsplit&#x27; and siglist &#x27;XStringSetList&#x27;
All user-level objects in a package (including S4 classes and methods)
should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘Biostrings-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: matchPDict
&gt; ### Title: Matching a dictionary of patterns against a reference
&gt; ### Aliases: matchPDict-exact matchPDict matchPDict,XString-method
&gt; ###   matchPDict,XStringSet-method matchPDict,XStringViews-method
&gt; ###   matchPDict,MaskedXString-method countPDict countPDict,XString-method
&gt; ###   countPDict,XStringSet-method countPDict,XStringViews-method
&gt; ###   countPDict,MaskedXString-method whichPDict whichPDict,XString-method
&gt; ###   whichPDict,XStringSet-method whichPDict,XStringViews-method
&gt; ###   whichPDict,MaskedXString-method vmatchPDict vmatchPDict,ANY-method
&gt; ###   vmatchPDict,XString-method vmatchPDict,MaskedXString-method
&gt; ###   vcountPDict vcountPDict,XString-method vcountPDict,XStringSet-method
&gt; ###   vcountPDict,XStringViews-method vcountPDict,MaskedXString-method
&gt; ###   vwhichPDict vwhichPDict,XString-method vwhichPDict,XStringSet-method
&gt; ###   vwhichPDict,XStringViews-method vwhichPDict,MaskedXString-method
&gt; ###   extractAllMatches
&gt; ### Keywords: methods
&gt; 
&gt; ### ** Examples
&gt; 
&gt; ## ---------------------------------------------------------------------
&gt; ## A. A SIMPLE EXAMPLE OF EXACT MATCHING
&gt; ## ---------------------------------------------------------------------
&gt; 
&gt; ## Creating the pattern dictionary:
&gt; library(drosophila2probe)
Loading required package: AnnotationDbi
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

&gt; dict0 &lt;- DNAStringSet(drosophila2probe)
&gt; dict0                                # The original dictionary.
DNAStringSet object of length 265400:
         width seq
     [1]    25 CCTGAATCCTGGCAATGTCATCATC
     [2]    25 ATCCTGGCAATGTCATCATCAATGG
     [3]    25 ATCAGTTGTCAACGGCTAATACGCG
     [4]    25 ATCAATGGCGATTGCCGCGTCTGCA
     [5]    25 CCGCGTCTGCAATGTGAGGGCCTAA
     ...   ... ...
[265396]    25 TACTACTTGAGCCACAACCATCTGA
[265397]    25 AGGGACTAAAGAGGCCCCATGCTCT
[265398]    25 CATGCTCTGTCTGGTGTCAGCGCTA
[265399]    25 GTCAGCGCTACATGGTCCAGGACAA
[265400]    25 CCAGGACAAGTATGGACTTCCCCAC
&gt; length(dict0)                        # Hundreds of thousands of patterns.
[1] 265400
&gt; pdict0 &lt;- PDict(dict0)               # Store the original dictionary in
&gt;                                      # a PDict object (preprocessing).
&gt; 
&gt; ## Using the pattern dictionary on chromosome 3R:
&gt; library(BSgenome.Dmelanogaster.UCSC.dm3)
Loading required package: BSgenome
Loading required package: GenomicRanges
Loading required package: rtracklayer
&gt; chr3R &lt;- Dmelanogaster$chr3R         # Load chromosome 3R
&gt; chr3R
27905053-letter DNAString object
seq: GAATTCTCTCTTGTTGTAGTCTCTTGACAAAATGCA...TATGTCGCAGCGTGACTGTTCGCATTCTAGGAATTC
&gt; mi0 &lt;- matchPDict(pdict0, chr3R)     # Search...
&gt; 
&gt; ## Looking at the matches:
&gt; start_index &lt;- startIndex(mi0)       # Get the start index.
&gt; length(start_index)                  # Same as the original dictionary.
[1] 265400
&gt; start_index[[8220]]                  # Starts of the 8220th pattern.
[1]  6788511  9806158 23620037
&gt; end_index &lt;- endIndex(mi0)           # Get the end index.
&gt; end_index[[8220]]                    # Ends of the 8220th pattern.
[1]  6788535  9806182 23620061
&gt; nmatch_per_pat &lt;- elementNROWS(mi0)  # Get the number of matches per pattern.
&gt; nmatch_per_pat[[8220]]
[1] 3
&gt; mi0[[8220]]                          # Get the matches for the 8220th pattern.
IRanges object with 3 ranges and 0 metadata columns:
          start       end     width
      &lt;integer&gt; &lt;integer&gt; &lt;integer&gt;
  [1]   6788511   6788535        25
  [2]   9806158   9806182        25
  [3]  23620037  23620061        25
&gt; start(mi0[[8220]])                   # Equivalent to startIndex(mi0)[[8220]].
[1]  6788511  9806158 23620037
&gt; sum(nmatch_per_pat)                  # Total number of matches.
[1] 32993
&gt; table(nmatch_per_pat)
nmatch_per_pat
     0      1      2      3      4      5      6      7      8      9     10 
233623  31208    341     42     92     44     21      3      8      4     13 
    12 
     1 
&gt; i0 &lt;- which(nmatch_per_pat == max(nmatch_per_pat))
&gt; pdict0[[i0]]                         # The pattern with most occurrences.
25-letter DNAString object
seq: AAGGAAGTTGCACGCTGCACTGTCG
&gt; mi0[[i0]]                            # Its matches as an IRanges object.
IRanges object with 12 ranges and 0 metadata columns:
           start       end     width
       &lt;integer&gt; &lt;integer&gt; &lt;integer&gt;
   [1]  27806218  27806242        25
   [2]  27806784  27806808        25
   [3]  27807348  27807372        25
   [4]  27807630  27807654        25
   [5]  27807912  27807936        25
   ...       ...       ...       ...
   [8]  27809042  27809066        25
   [9]  27809325  27809349        25
  [10]  27809607  27809631        25
  [11]  27809889  27809913        25
  [12]  27810997  27811021        25
&gt; Views(chr3R, mi0[[i0]])              # And as an XStringViews object.
Views on a 27905053-letter DNAString subject
subject: GAATTCTCTCTTGTTGTAGTCTCTTGACAAAATG...TGTCGCAGCGTGACTGTTCGCATTCTAGGAATTC
views:
          start      end width
   [1] 27806218 27806242    25 [AAGGAAGTTGCACGCTGCACTGTCG]
   [2] 27806784 27806808    25 [AAGGAAGTTGCACGCTGCACTGTCG]
   [3] 27807348 27807372    25 [AAGGAAGTTGCACGCTGCACTGTCG]
   [4] 27807630 27807654    25 [AAGGAAGTTGCACGCTGCACTGTCG]
   [5] 27807912 27807936    25 [AAGGAAGTTGCACGCTGCACTGTCG]
   ...      ...      ...   ... ...
   [8] 27809042 27809066    25 [AAGGAAGTTGCACGCTGCACTGTCG]
   [9] 27809325 27809349    25 [AAGGAAGTTGCACGCTGCACTGTCG]
  [10] 27809607 27809631    25 [AAGGAAGTTGCACGCTGCACTGTCG]
  [11] 27809889 27809913    25 [AAGGAAGTTGCACGCTGCACTGTCG]
  [12] 27810997 27811021    25 [AAGGAAGTTGCACGCTGCACTGTCG]
&gt; 
&gt; ## Get the coverage of the original subject:
&gt; cov3R &lt;- as.integer(coverage(mi0, width=length(chr3R)))
&gt; max(cov3R)
[1] 10
&gt; mean(cov3R)
[1] 0.02955827
&gt; sum(cov3R != 0) / length(cov3R)      # Only 2.44% of chr3R is covered.
[1] 0.02441407
&gt; if (interactive()) {
+   plotCoverage &lt;- function(cx, start, end)
+   {
+     plot.new()
+     plot.window(c(start, end), c(0, 20))
+     axis(1)
+     axis(2)
+     axis(4)
+     lines(start:end, cx[start:end], type=&quot;l&quot;)
+   }
+   plotCoverage(cov3R, 27600000, 27900000)
+ }
&gt; 
&gt; ## ---------------------------------------------------------------------
&gt; ## B. NAMING THE PATTERNS
&gt; ## ---------------------------------------------------------------------
&gt; 
&gt; ## The names of the original patterns, if any, are propagated to the
&gt; ## PDict and MIndex objects:
&gt; names(dict0) &lt;- mkAllStrings(letters, 4)[seq_len(length(dict0))]
&gt; dict0
DNAStringSet object of length 265400:
         width seq                                          names               
     [1]    25 CCTGAATCCTGGCAATGTCATCATC                    aaaa
     [2]    25 ATCCTGGCAATGTCATCATCAATGG                    aaab
     [3]    25 ATCAGTTGTCAACGGCTAATACGCG                    aaac
     [4]    25 ATCAATGGCGATTGCCGCGTCTGCA                    aaad
     [5]    25 CCGCGTCTGCAATGTGAGGGCCTAA                    aaae
     ...   ... ...
[265396]    25 TACTACTTGAGCCACAACCATCTGA                    pcpn
[265397]    25 AGGGACTAAAGAGGCCCCATGCTCT                    pcpo
[265398]    25 CATGCTCTGTCTGGTGTCAGCGCTA                    pcpp
[265399]    25 GTCAGCGCTACATGGTCCAGGACAA                    pcpq
[265400]    25 CCAGGACAAGTATGGACTTCCCCAC                    pcpr
&gt; dict0[[&quot;abcd&quot;]]
25-letter DNAString object
seq: CCTCTCAGAGAAACCCACACAAAAA
&gt; pdict0n &lt;- PDict(dict0)
&gt; names(pdict0n)[1:30]
 [1] &quot;aaaa&quot; &quot;aaab&quot; &quot;aaac&quot; &quot;aaad&quot; &quot;aaae&quot; &quot;aaaf&quot; &quot;aaag&quot; &quot;aaah&quot; &quot;aaai&quot; &quot;aaaj&quot;
[11] &quot;aaak&quot; &quot;aaal&quot; &quot;aaam&quot; &quot;aaan&quot; &quot;aaao&quot; &quot;aaap&quot; &quot;aaaq&quot; &quot;aaar&quot; &quot;aaas&quot; &quot;aaat&quot;
[21] &quot;aaau&quot; &quot;aaav&quot; &quot;aaaw&quot; &quot;aaax&quot; &quot;aaay&quot; &quot;aaaz&quot; &quot;aaba&quot; &quot;aabb&quot; &quot;aabc&quot; &quot;aabd&quot;
&gt; pdict0n[[&quot;abcd&quot;]]
25-letter DNAString object
seq: CCTCTCAGAGAAACCCACACAAAAA
&gt; mi0n &lt;- matchPDict(pdict0n, chr3R)
&gt; names(mi0n)[1:30]
 [1] &quot;aaaa&quot; &quot;aaab&quot; &quot;aaac&quot; &quot;aaad&quot; &quot;aaae&quot; &quot;aaaf&quot; &quot;aaag&quot; &quot;aaah&quot; &quot;aaai&quot; &quot;aaaj&quot;
[11] &quot;aaak&quot; &quot;aaal&quot; &quot;aaam&quot; &quot;aaan&quot; &quot;aaao&quot; &quot;aaap&quot; &quot;aaaq&quot; &quot;aaar&quot; &quot;aaas&quot; &quot;aaat&quot;
[21] &quot;aaau&quot; &quot;aaav&quot; &quot;aaaw&quot; &quot;aaax&quot; &quot;aaay&quot; &quot;aaaz&quot; &quot;aaba&quot; &quot;aabb&quot; &quot;aabc&quot; &quot;aabd&quot;
&gt; mi0n[[&quot;abcd&quot;]]
IRanges object with 0 ranges and 0 metadata columns:
       start       end     width
   &lt;integer&gt; &lt;integer&gt; &lt;integer&gt;
&gt; 
&gt; ## This is particularly useful when unlisting an MIndex object:
&gt; unlist(mi0)[1:10]
IRanges object with 10 ranges and 0 metadata columns:
           start       end     width
       &lt;integer&gt; &lt;integer&gt; &lt;integer&gt;
   [1]  16069898  16069922        25
   [2]  16069961  16069985        25
   [3]  16070066  16070090        25
   [4]  16070081  16070105        25
   [5]  16070100  16070124        25
   [6]  16070112  16070136        25
   [7]  16070136  16070160        25
   [8]  16070159  16070183        25
   [9]  16070174  16070198        25
  [10]  16070192  16070216        25
&gt; unlist(mi0n)[1:10]  # keep track of where the matches are coming from
IRanges object with 10 ranges and 0 metadata columns:
           start       end     width
       &lt;integer&gt; &lt;integer&gt; &lt;integer&gt;
  aadu  16069898  16069922        25
  aadv  16069961  16069985        25
  aadw  16070066  16070090        25
  aadx  16070081  16070105        25
  aady  16070100  16070124        25
  aadz  16070112  16070136        25
  aaea  16070136  16070160        25
  aaeb  16070159  16070183        25
  aaec  16070174  16070198        25
  aaed  16070192  16070216        25
&gt; 
&gt; ## ---------------------------------------------------------------------
&gt; ## C. PERFORMANCE
&gt; ## ---------------------------------------------------------------------
&gt; 
&gt; ## If getting the number of matches is what matters only (without
&gt; ## regarding their positions), then countPDict() will be faster,
&gt; ## especially when there is a high number of matches:
&gt; 
&gt; nmatch_per_pat0 &lt;- countPDict(pdict0, chr3R)
&gt; stopifnot(identical(nmatch_per_pat0, nmatch_per_pat))
&gt; 
&gt; if (interactive()) {
+   ## What&#x27;s the impact of the dictionary width on performance?
+   ## Below is some code that can be used to figure out (will take a long
+   ## time to run). For different widths of the original dictionary, we
+   ## look at:
+   ##   o pptime: preprocessing time (in sec.) i.e. time needed for
+   ##             building the PDict object from the truncated input
+   ##             sequences;
+   ##   o nnodes: nb of nodes in the resulting Aho-Corasick tree;
+   ##   o nupatt: nb of unique truncated input sequences;
+   ##   o matchtime: time (in sec.) needed to find all the matches;
+   ##   o totalcount: total number of matches.
+   getPDictStats &lt;- function(dict, subject)
+   {
+     ans_width &lt;- width(dict[1])
+     ans_pptime &lt;- system.time(pdict &lt;- PDict(dict))[[&quot;elapsed&quot;]]
+     pptb &lt;- pdict@threeparts@pptb
+     ans_nnodes &lt;- nnodes(pptb)
+     ans_nupatt &lt;- sum(!duplicated(pdict))
+     ans_matchtime &lt;- system.time(
+                        mi0 &lt;- matchPDict(pdict, subject)
+                      )[[&quot;elapsed&quot;]]
+     ans_totalcount &lt;- sum(elementNROWS(mi0))
+     list(
+       width=ans_width,
+       pptime=ans_pptime,
+       nnodes=ans_nnodes,
+       nupatt=ans_nupatt,
+       matchtime=ans_matchtime,
+       totalcount=ans_totalcount
+     )
+   }
+   stats &lt;- lapply(8:25,
+                function(width)
+                    getPDictStats(DNAStringSet(dict0, end=width), chr3R))
+   stats &lt;- data.frame(do.call(rbind, stats))
+   stats
+ }
&gt; 
&gt; ## ---------------------------------------------------------------------
&gt; ## D. USING A NON-PREPROCESSED DICTIONARY
&gt; ## ---------------------------------------------------------------------
&gt; 
&gt; dict3 &lt;- DNAStringSet(mkAllStrings(DNA_BASES, 3))  # all trinucleotides
&gt; dict3
DNAStringSet object of length 64:
     width seq
 [1]     3 AAA
 [2]     3 AAC
 [3]     3 AAG
 [4]     3 AAT
 [5]     3 ACA
 ...   ... ...
[60]     3 TGT
[61]     3 TTA
[62]     3 TTC
[63]     3 TTG
[64]     3 TTT
&gt; pdict3 &lt;- PDict(dict3)
&gt; 
&gt; ## The 3 following calls are equivalent (from faster to slower):
&gt; res3a &lt;- countPDict(pdict3, chr3R)
&gt; res3b &lt;- countPDict(dict3, chr3R)
&gt; res3c &lt;- sapply(dict3,
+              function(pattern) countPattern(pattern, chr3R))
&gt; stopifnot(identical(res3a, res3b))
&gt; stopifnot(identical(res3a, res3c))
&gt; 
&gt; ## One reason for using a non-preprocessed dictionary is to get rid of
&gt; ## all the constraints associated with preprocessing, e.g., when
&gt; ## preprocessing with PDict(), the input dictionary must be DNA and a
&gt; ## Trusted Band must be defined (explicitly or implicitly).
&gt; ## See &#x27;?PDict&#x27; for more information about these constraints.
&gt; ## In particular, using a non-preprocessed dictionary can be
&gt; ## useful for the kind of inexact matching that can&#x27;t be achieved
&gt; ## with a PDict object (if performance is not an issue).
&gt; ## See &#x27;?`matchPDict-inexact`&#x27; for more information about inexact
&gt; ## matching.
&gt; 
&gt; dictD &lt;- xscat(dict3, &quot;N&quot;, reverseComplement(dict3))
&gt; 
&gt; ## The 2 following calls are equivalent (from faster to slower):
&gt; resDa &lt;- matchPDict(dictD, chr3R, fixed=FALSE)
&gt; resDb &lt;- sapply(dictD,
+                 function(pattern)
+                   matchPattern(pattern, chr3R, fixed=FALSE))
&gt; stopifnot(all(sapply(seq_len(length(dictD)),
+                      function(i)
+                        identical(resDa[[i]], as(resDb[[i]], &quot;IRanges&quot;)))))
&gt; 
&gt; ## A non-preprocessed dictionary can be of any base class i.e. BString,
&gt; ## RNAString, and AAString, in addition to DNAString:
&gt; matchPDict(AAStringSet(c(&quot;DARC&quot;, &quot;EGH&quot;)), AAString(&quot;KMFPRNDEGHSTTWTEE&quot;))
MIndex object of length 2
[[1]]
IRanges object with 0 ranges and 0 metadata columns:
       start       end     width
   &lt;integer&gt; &lt;integer&gt; &lt;integer&gt;

[[2]]
IRanges object with 1 range and 0 metadata columns:
          start       end     width
      &lt;integer&gt; &lt;integer&gt; &lt;integer&gt;
  [1]         8        10         3

&gt; 
&gt; ## ---------------------------------------------------------------------
&gt; ## E. vcountPDict()
&gt; ## ---------------------------------------------------------------------
&gt; 
&gt; ## Load Fly upstream sequences (i.e. the sequences 2000 bases upstream of
&gt; ## annotated transcription starts):
&gt; dm3_upstream_filepath &lt;- system.file(&quot;extdata&quot;,
+                                      &quot;dm3_upstream2000.fa.gz&quot;,
+                                      package=&quot;Biostrings&quot;)
&gt; dm3_upstream &lt;- readDNAStringSet(dm3_upstream_filepath)
&gt; dm3_upstream
DNAStringSet object of length 26454:
        width seq                                           names               
    [1]  2000 GTTGGTGGCCCACCAGTGCCA...CAAGTTTACCGGTTGCACGGT NM_078863_up_2000...
    [2]  2000 TTATTTATGTAGGCGCCCGTT...ATACGGAAAGTCATCCTCGAT NM_001201794_up_2...
    [3]  2000 TTATTTATGTAGGCGCCCGTT...ATACGGAAAGTCATCCTCGAT NM_001201795_up_2...
    [4]  2000 TTATTTATGTAGGCGCCCGTT...ATACGGAAAGTCATCCTCGAT NM_001201796_up_2...
    [5]  2000 TTATTTATGTAGGCGCCCGTT...ATACGGAAAGTCATCCTCGAT NM_001201797_up_2...
    ...   ... ...
[26450]  2000 ATTTACAAGACTAATAAAGAT...AAAATTAAATTTCAATAAAAC NM_001111010_up_2...
[26451]  2000 GATATACGAAGGACGACCTGC...TTGTTTGAGTTGTTATATATT NM_001015258_up_2...
[26452]  2000 GATATACGAAGGACGACCTGC...TTGTTTGAGTTGTTATATATT NM_001110997_up_2...
[26453]  2000 GATATACGAAGGACGACCTGC...TTGTTTGAGTTGTTATATATT NM_001276245_up_2...
[26454]  2000 CGTATGTATTAGTTAACTCTG...TCAAAGTGTAAGAACAAATTG NM_001015497_up_2...
&gt; 
&gt; subject &lt;- dm3_upstream[1:100]
&gt; mat1 &lt;- vcountPDict(pdict0, subject)
&gt; dim(mat1)  # length(pdict0) x length(subject)
[1] 265400    100
&gt; nhit_per_probe &lt;- rowSums(mat1)
&gt; table(nhit_per_probe)
nhit_per_probe
     0      1      2      3      8      9 
265217     82     59     14     14     14 
&gt; 
&gt; ## Without vcountPDict(), &#x27;mat1&#x27; could have been computed with:
&gt; mat2 &lt;- sapply(unname(subject), function(x) countPDict(pdict0, x))
&gt; stopifnot(identical(mat1, mat2))
&gt; ## but using vcountPDict() is faster (10x or more, depending of the
&gt; ## average length of the sequences in &#x27;subject&#x27;).
&gt; 
&gt; if (interactive()) {
+   ## This will fail (with message &quot;allocMatrix: too many elements
+   ## specified&quot;) because, on most platforms, vectors and matrices in R
+   ## are limited to 2^31 elements:
+   subject &lt;- dm3_upstream
+   vcountPDict(pdict0, subject)
+   length(pdict0) * length(dm3_upstream)
+   1 * length(pdict0) * length(dm3_upstream)  # &gt; 2^31
+   ## But this will work:
+   nhit_per_seq &lt;- vcountPDict(pdict0, subject, collapse=2)
+   sum(nhit_per_seq &gt;= 1)  # nb of subject sequences with at least 1 hit
+   table(nhit_per_seq)  # max is 74
+   which.max(nhit_per_seq)  # 1133
+   sum(countPDict(pdict0, subject[[1133]]))  # 74
+ }
&gt; 
&gt; ## ---------------------------------------------------------------------
&gt; ## F. RELATIONSHIP BETWEEN vcountPDict(), countPDict() AND
&gt; ## vcountPattern()
&gt; ## ---------------------------------------------------------------------
&gt; subject &lt;- dm3_upstream
&gt; 
&gt; ## The 4 following calls are equivalent (from faster to slower):
&gt; mat3a &lt;- vcountPDict(pdict3, subject)
&gt; mat3b &lt;- vcountPDict(dict3, subject)
&gt; mat3c &lt;- sapply(dict3,
+                 function(pattern) vcountPattern(pattern, subject))
&gt; mat3d &lt;- sapply(unname(subject),
+                 function(x) countPDict(pdict3, x))
&gt; stopifnot(identical(mat3a, mat3b))
&gt; stopifnot(identical(mat3a, t(mat3c)))
&gt; stopifnot(identical(mat3a, mat3d))
&gt; 
&gt; ## The 3 following calls are equivalent (from faster to slower):
&gt; nhitpp3a &lt;- vcountPDict(pdict3, subject, collapse=1)  # rowSums(mat3a)
&gt; nhitpp3b &lt;- vcountPDict(dict3, subject, collapse=1)
&gt; nhitpp3c &lt;- sapply(dict3,
+                    function(pattern) sum(vcountPattern(pattern, subject)))
Killed
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘run_unitTests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘Biostrings2Classes.Rnw’ using ‘UTF-8’... OK
  ‘BiostringsQuickOverview.Rnw’ using ‘UTF-8’... OK
  ‘MultipleAlignments.Rnw’ using ‘UTF-8’... OK
  ‘PairwiseAlignments.Rnw’ using ‘UTF-8’... OK
  ‘matchprobes.Rnw’ using ‘UTF-8’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 2 WARNINGs, 4 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/Biostrings.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">Biostrings.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL Biostrings
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘Biostrings’ ...
** using staged installation
** libs
using C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c BAB_class.c -o BAB_class.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c BitMatrix.c -o BitMatrix.o
BitMatrix.c:299:13: warning: ‘BitMatrix_print’ defined but not used [-Wunused-function]
  299 | static void BitMatrix_print(BitMatrix *bitmat)
      |             ^~~~~~~~~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c IRanges_stubs.c -o IRanges_stubs.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c MIndex_class.c -o MIndex_class.o
MIndex_class.c: In function ‘SparseMIndex_endIndex’:
MIndex_class.c:184:27: warning: unused variable ‘poffsets_order’ [-Wunused-variable]
  184 |         IntAE *poffsets, *poffsets_order;
      |                           ^~~~~~~~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c PreprocessedTB_class.c -o PreprocessedTB_class.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c R_init_Biostrings.c -o R_init_Biostrings.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c RoSeqs_utils.c -o RoSeqs_utils.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c S4Vectors_stubs.c -o S4Vectors_stubs.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c SparseList_utils.c -o SparseList_utils.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c XStringSetList_class.c -o XStringSetList_class.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c XStringSet_class.c -o XStringSet_class.o
XStringSet_class.c: In function ‘new_XStringSet_from_CHARACTER’:
XStringSet_class.c:124:17: warning: ‘lkup_len’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  124 |                 _copy_CHARSXP_to_Chars_holder(&amp;ans_elt_holder, x_elt,
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  125 |                                 INTEGER(start)[i], lkup0, lkup_len);
      |                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c XString_class.c -o XString_class.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c XVector_stubs.c -o XVector_stubs.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c align_needwunsQS.c -o align_needwunsQS.o
align_needwunsQS.c: In function ‘align_needwunsQS’:
align_needwunsQS.c:155:29: warning: ‘sc’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  155 |         INTEGER(ans_elt)[0] = score;
      |         ~~~~~~~~~~~~~~~~~~~~^~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c align_pairwiseAlignment.c -o align_pairwiseAlignment.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c align_utils.c -o align_utils.o
align_utils.c: In function ‘PairwiseAlignmentsSingleSubject_align_aligned’:
align_utils.c:250:42: warning: ‘indelWidthSubject’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  250 |                                 jPattern += indelWidthSubject;
      |                                 ~~~~~~~~~^~~~~~~~~~~~~~~~~~~~
align_utils.c:231:63: warning: ‘indelStartSubject’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  231 |                         if ((numberOfIndelSubject == 0) || (j &lt; indelStartSubject)) {
      |                                                            ~~~^~~~~~~~~~~~~~~~~~~~
align_utils.c:239:72: warning: ‘indelWidthPattern’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  239 |                                                 mappedStringPtr[index] = gapCodeValue;
      |                                                 ~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~
align_utils.c:232:78: warning: ‘indelStartPattern’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  232 |                                 if ((numberOfIndelPattern == 0) || (jPattern &lt; indelStartPattern)) {
      |                                                                    ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c find_palindromes.c -o find_palindromes.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c gtestsim.c -o gtestsim.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c inject_code.c -o inject_code.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c letter_frequency.c -o letter_frequency.o
letter_frequency.c: In function ‘XStringSet_two_way_letter_frequency’:
letter_frequency.c:957:48: warning: unused variable ‘x_pos’ [-Wunused-variable]
  957 |   int x_width, y_width, x_length, *ans_mat, i, x_pos;
      |                                                ^~~~~
letter_frequency.c:956:13: warning: unused variable ‘ans_dimnames’ [-Wunused-variable]
  956 |   SEXP ans, ans_dimnames;
      |             ^~~~~~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c lowlevel_matching.c -o lowlevel_matching.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c match_PWM.c -o match_PWM.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c match_pattern.c -o match_pattern.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c match_pattern_boyermoore.c -o match_pattern_boyermoore.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c match_pattern_indels.c -o match_pattern_indels.o
match_pattern_indels.c:7:13: warning: ‘test_match_pattern_indels’ defined but not used [-Wunused-function]
    7 | static void test_match_pattern_indels(const char *p, const char *s,
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c match_pattern_shiftor.c -o match_pattern_shiftor.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c match_pdict.c -o match_pdict.o
match_pdict.c: In function ‘vmatch_PDict3Parts_XStringSet’:
match_pdict.c:419:33: warning: ‘ans_col’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  419 |                         ans_col += tb_length;
      |                         ~~~~~~~~^~~~~~~~~~~~
match_pdict.c:392:65: note: ‘ans_col’ was declared here
  392 |         int tb_length, S_length, collapse0, i, j, match_count, *ans_col;
      |                                                                 ^~~~~~~
match_pdict.c: In function ‘vmatch_XStringSet_XStringSet’:
match_pdict.c:470:41: warning: ‘ans_elt’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  470 |                                 ans_elt += P_length;
      |                                 ~~~~~~~~^~~~~~~~~~~
match_pdict.c:441:64: note: ‘ans_elt’ was declared here
  441 |         int P_length, S_length, collapse0, i, j, match_count, *ans_elt;
      |                                                                ^~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c match_pdict_ACtree2.c -o match_pdict_ACtree2.o
match_pdict_ACtree2.c: In function ‘split_and_move_pointers’:
match_pdict_ACtree2.c:1031:17: warning: variable ‘node0’ set but not used [-Wunused-but-set-variable]
 1031 |         ACnode *node0, *node1, *node2;
      |                 ^~~~~
match_pdict_ACtree2.c: In function ‘merge_pointers’:
match_pdict_ACtree2.c:1076:17: warning: variable ‘node0’ set but not used [-Wunused-but-set-variable]
 1076 |         ACnode *node0, *node1, *node2;
      |                 ^~~~~
At top level:
match_pdict_ACtree2.c:602:21: warning: ‘a_nice_max_nodeextbuf_nelt’ defined but not used [-Wunused-function]
  602 | static unsigned int a_nice_max_nodeextbuf_nelt(int nnodes)
      |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~
match_pdict_ACtree2.c:139:13: warning: ‘debug_node_counting_functions’ defined but not used [-Wunused-function]
  139 | static void debug_node_counting_functions(int maxdepth)
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c match_pdict_Twobit.c -o match_pdict_Twobit.o
match_pdict_Twobit.c: In function ‘build_Twobit’:
match_pdict_Twobit.c:75:27: warning: ‘twobit_sign2pos’ may be used uninitialized in this function [-Wmaybe-uninitialized]
   75 |         PROTECT(ans_elt = new_XInteger_from_tag(&quot;XInteger&quot;, twobit_sign2pos));
      |                           ^~~~~~~~~~~~~~~~~~~~~
match_pdict_Twobit.c:110:19: note: ‘twobit_sign2pos’ was declared here
  110 |         SEXP ans, twobit_sign2pos;
      |                   ^~~~~~~~~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c match_pdict_utils.c -o match_pdict_utils.o
match_pdict_utils.c: In function ‘match_ppheadtail0’:
match_pdict_utils.c:653:56: warning: unused variable ‘ncol’ [-Wunused-variable]
  653 |         int nelt, min_safe_tb_end, max_safe_tb_end, j, ncol;
      |                                                        ^~~~
match_pdict_utils.c: In function ‘match_ppheadtail’:
match_pdict_utils.c:713:13: warning: unused variable ‘nelt’ [-Wunused-variable]
  713 |         int nelt, nkey0, nkey1, nkey2, i, key;
      |             ^~~~
match_pdict_utils.c: In function ‘_match_pdict_all_flanks’:
match_pdict_utils.c:820:51: warning: unused variable ‘subtotal_NFC’ [-Wunused-variable]
  820 |         static unsigned long int total_NFC = 0UL, subtotal_NFC = 0UL;
      |                                                   ^~~~~~~~~~~~
match_pdict_utils.c:820:34: warning: unused variable ‘total_NFC’ [-Wunused-variable]
  820 |         static unsigned long int total_NFC = 0UL, subtotal_NFC = 0UL;
      |                                  ^~~~~~~~~
match_pdict_utils.c:819:40: warning: unused variable ‘NFC’ [-Wunused-variable]
  819 |         unsigned long int ndup, nloci, NFC; // NFC = Number of Flank Comparisons
      |                                        ^~~
match_pdict_utils.c:819:33: warning: unused variable ‘nloci’ [-Wunused-variable]
  819 |         unsigned long int ndup, nloci, NFC; // NFC = Number of Flank Comparisons
      |                                 ^~~~~
match_pdict_utils.c:819:27: warning: unused variable ‘ndup’ [-Wunused-variable]
  819 |         unsigned long int ndup, nloci, NFC; // NFC = Number of Flank Comparisons
      |                           ^~~~
At top level:
match_pdict_utils.c:820:34: warning: ‘total_NFC’ defined but not used [-Wunused-variable]
  820 |         static unsigned long int total_NFC = 0UL, subtotal_NFC = 0UL;
      |                                  ^~~~~~~~~
match_pdict_utils.c:820:51: warning: ‘subtotal_NFC’ defined but not used [-Wunused-variable]
  820 |         static unsigned long int total_NFC = 0UL, subtotal_NFC = 0UL;
      |                                                   ^~~~~~~~~~~~
match_pdict_utils.c:261:13: warning: ‘match_headtail_by_loc’ defined but not used [-Wunused-function]
  261 | static void match_headtail_by_loc(const HeadTail *headtail,
      |             ^~~~~~~~~~~~~~~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c match_reporting.c -o match_reporting.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c matchprobes.c -o matchprobes.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c pmatchPattern.c -o pmatchPattern.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c read_fasta_files.c -o read_fasta_files.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c read_fastq_files.c -o read_fastq_files.o
read_fastq_files.c: In function ‘parse_FASTQ_file’:
read_fastq_files.c:392:28: warning: ‘dont_load’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  392 |                         if (dont_load || loader-&gt;new_empty_seq_hook == NULL)
      |                            ^
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c replaceAt.c -o replaceAt.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c replace_letter_at.c -o replace_letter_at.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c strutils.c -o strutils.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c translate.c -o translate.o
translate.c: In function ‘DNAStringSet_translate’:
translate.c:110:36: warning: ‘if_ambig0’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  110 |                                 if (if_ambig == TRANSLATE_ERROR) {
      |                                    ^
translate.c:136:36: note: ‘if_ambig0’ was declared here
  136 |         int ncodes, if_non_ambig0, if_ambig0, ans_length, i, errcode;
      |                                    ^~~~~~~~~
translate.c:106:36: warning: ‘if_non_ambig0’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  106 |                                 if (if_non_ambig == TRANSLATE_TO_X)
      |                                    ^
translate.c:136:21: note: ‘if_non_ambig0’ was declared here
  136 |         int ncodes, if_non_ambig0, if_ambig0, ans_length, i, errcode;
      |                     ^~~~~~~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c unstrsplit_methods.c -o unstrsplit_methods.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c utils.c -o utils.o
utils.c: In function ‘_get_twobit_signature’:
utils.c:157:16: warning: ‘twobit_sign’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  157 |         return twobit_sign;
      |                ^~~~~~~~~~~
utils.c: In function ‘_get_twobit_signature_at’:
utils.c:164:19: warning: ‘twobit_sign’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  164 |         int i, j, twobit_sign;
      |                   ^~~~~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c xscat.c -o xscat.o
In file included from /home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include/S4Vectors_defines.h:18,
                 from /home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include/IRanges_defines.h:18,
                 from ../inst/include/Biostrings_defines.h:18,
                 from Biostrings.h:1,
                 from xscat.c:1:
xscat.c: In function ‘XString_xscat’:
/home/biocbuild/bbs-3.17-bioc/R/include/Rdefines.h:91:33: warning: ‘ans_length’ may be used uninitialized [-Wmaybe-uninitialized]
   91 | #define NEW_RAW(n)              Rf_allocVector(RAWSXP,n)
      |                                 ^~~~~~~~~~~~~~
xscat.c:18:20: note: ‘ans_length’ was declared here
   18 |         int nargs, ans_length, tag_offset, j;
      |                    ^~~~~~~~~~
xscat.c:52:23: warning: ‘ans_classname’ may be used uninitialized in this function [-Wmaybe-uninitialized]
   52 |         PROTECT(ans = new_XRaw_from_tag(ans_classname, ans_tag));
      |                       ^~~~~~~~~~~~~~~~~
In file included from /home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include/S4Vectors_defines.h:18,
                 from /home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include/IRanges_defines.h:18,
                 from ../inst/include/Biostrings_defines.h:18,
                 from Biostrings.h:1,
                 from xscat.c:1:
xscat.c: In function ‘XStringSet_xscat’:
/home/biocbuild/bbs-3.17-bioc/R/include/Rdefines.h:85:33: warning: ‘ans_length’ may be used uninitialized [-Wmaybe-uninitialized]
   85 | #define NEW_INTEGER(n)          Rf_allocVector(INTSXP,n)
      |                                 ^~~~~~~~~~~~~~
xscat.c:66:39: note: ‘ans_length’ was declared here
   66 |         int nargs, *arg_lengths, *ii, ans_length, i, j, *width;
      |                                       ^~~~~~~~~~
xscat.c:108:23: warning: ‘ans_element_type’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  108 |         PROTECT(ans = _alloc_XStringSet(ans_element_type, ans_width));
      |                       ^~~~~~~~~~~~~~~~~
gcc -shared -L/home/biocbuild/bbs-3.17-bioc/R/lib -L/usr/local/lib -o Biostrings.so BAB_class.o BitMatrix.o IRanges_stubs.o MIndex_class.o PreprocessedTB_class.o R_init_Biostrings.o RoSeqs_utils.o S4Vectors_stubs.o SparseList_utils.o XStringSetList_class.o XStringSet_class.o XString_class.o XVector_stubs.o align_needwunsQS.o align_pairwiseAlignment.o align_utils.o find_palindromes.o gtestsim.o inject_code.o letter_frequency.o lowlevel_matching.o match_PWM.o match_pattern.o match_pattern_boyermoore.o match_pattern_indels.o match_pattern_shiftor.o match_pdict.o match_pdict_ACtree2.o match_pdict_Twobit.o match_pdict_utils.o match_reporting.o matchprobes.o pmatchPattern.o read_fasta_files.o read_fastq_files.o replaceAt.o replace_letter_at.o strutils.o translate.o unstrsplit_methods.o utils.o xscat.o -L/home/biocbuild/bbs-3.17-bioc/R/lib -lR
installing to /home/biocbuild/bbs-3.17-bioc/R/library/00LOCK-Biostrings/00new/Biostrings/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
in method for ‘twoWayAlphabetFrequencyByQuality’ with signature ‘&quot;QualityScaledXStringSet&quot;’: no definition for class “QualityScaledXStringSet”
Creating a new generic function for ‘strsplit’ in package ‘Biostrings’
Creating a generic function for ‘ls’ from package ‘base’ in package ‘Biostrings’
Creating a new generic function for ‘pattern’ in package ‘Biostrings’
Creating a new generic function for ‘offset’ in package ‘Biostrings’
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (Biostrings)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">Biostrings.Rcheck/tests/run_unitTests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; require(&quot;Biostrings&quot;) || stop(&quot;unable to load Biostrings package&quot;)
Loading required package: Biostrings
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: XVector
Loading required package: GenomeInfoDb

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

[1] TRUE
&gt; Biostrings:::.test()


RUNIT TEST PROTOCOL -- Fri Feb 24 08:04:46 2023 
*********************************************** 
Number of test functions: 41 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
Biostrings RUnit Tests - 41 test functions, 0 errors, 0 failures
Number of test functions: 41 
Number of errors: 0 
Number of failures: 0 
Warning messages:
1: In XStringSet(&quot;DNA&quot;, x, start = start, end = end, width = width,  :
  metadata columns on input DNAStringSet object were dropped
2: In XStringSet(&quot;DNA&quot;, x, start = start, end = end, width = width,  :
  metadata columns on input DNAStringSet object were dropped
3: In XStringSet(&quot;DNA&quot;, x, start = start, end = end, width = width,  :
  metadata columns on input DNAStringSet object were dropped
&gt; 
&gt; proc.time()
   user  system elapsed 
 11.925   0.569  12.585 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">Biostrings.Rcheck/Biostrings-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>AAString-class</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>AMINO_ACID_CODE</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>AlignedXStringSet-class</TD><TD style="text-align: right;">0.069</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.077</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DNAString-class</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GENETIC_CODE</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>HNF4alpha</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.030</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>IUPAC_CODE_MAP</TD><TD style="text-align: right;">0.162</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.178</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>MIndex-class</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>MaskedXString-class</TD><TD style="text-align: right;">0.209</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">0.266</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>MultipleAlignment-class</TD><TD style="text-align: right;">1.507</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">1.529</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>PDict-class</TD><TD style="text-align: right;">4.499</TD><TD style="text-align: right;">0.108</TD><TD style="text-align: right;">4.612</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>PairwiseAlignments-class</TD><TD style="text-align: right;">0.734</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.734</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>PairwiseAlignments-io</TD><TD style="text-align: right;">2.825</TD><TD style="text-align: right;">0.208</TD><TD style="text-align: right;">3.033</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>QualityScaledXStringSet-class</TD><TD style="text-align: right;">0.183</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.192</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>RNAString-class</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>XString-class</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>XStringQuality-class</TD><TD style="text-align: right;">0.183</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.183</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>XStringSet-class</TD><TD style="text-align: right;">12.198</TD><TD style="text-align: right;"> 0.516</TD><TD style="text-align: right;">12.781</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>XStringSet-comparison</TD><TD style="text-align: right;">3.995</TD><TD style="text-align: right;">0.097</TD><TD style="text-align: right;">4.093</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>XStringSet-io</TD><TD style="text-align: right;">9.087</TD><TD style="text-align: right;">0.279</TD><TD style="text-align: right;">9.376</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>XStringSetList-class</TD><TD style="text-align: right;">0.303</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.307</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>XStringViews-class</TD><TD style="text-align: right;">0.144</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.148</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>align-utils</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>chartr</TD><TD style="text-align: right;">0.683</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.721</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>detail</TD><TD style="text-align: right;">0.361</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">0.436</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dinucleotideFrequencyTest</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findPalindromes</TD><TD style="text-align: right;">44.646</TD><TD style="text-align: right;"> 0.048</TD><TD style="text-align: right;">44.716</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getSeq</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">0.062</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>gregexpr2</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>injectHardMask</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>letter</TD><TD style="text-align: right;">0.026</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.027</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>letterFrequency</TD><TD style="text-align: right;">0.909</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.947</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>longestConsecutive</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>lowlevel-matching</TD><TD style="text-align: right;">0.590</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.614</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>maskMotif</TD><TD style="text-align: right;">1.533</TD><TD style="text-align: right;">0.056</TD><TD style="text-align: right;">1.594</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>match-utils</TD><TD style="text-align: right;">0.027</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.027</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>matchLRPatterns</TD><TD style="text-align: right;">0.557</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.594</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
