<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for GWASTools on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><B>[<A href="../index.html#G">G</A>]</B></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-13 06:49:33 -0000 (Mon, 13 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4222</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for GWASTools on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the GWASTools package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/GWASTools.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>872</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">GWASTools</A>&nbsp;1.45.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/GWASTools">landing page</A>)</SPAN><BR>Stephanie M. Gogarten<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-11&nbsp;01:58:29&nbsp;-0000&nbsp;(Sat,&nbsp;11&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/GWASTools</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">0127e8c</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:06:22&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: GWASTools</TD></TR>
<TR><TD><B>Version</B>: 1.45.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:GWASTools.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings GWASTools_1.45.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-11 16:05:26 -0000 (Sat, 11 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-11 16:11:55 -0000 (Sat, 11 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 388.6 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: GWASTools.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 1</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:GWASTools.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings GWASTools_1.45.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/GWASTools.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘GWASTools/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘GWASTools’ version ‘1.45.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘GWASTools’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
Unexported object imported by a &#x27;:::&#x27; call: ‘S4Vectors:::make_zero_col_DataFrame’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... WARNING
checkRd: (5) GdsGenotypeReader-class.Rd:36-67: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:76-78: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:79-81: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:82-86: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:87-95: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:96-100: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:101-105: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:106-110: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:111-115: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:116-129: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:130-141: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:142-150: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:151-154: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:155-158: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:159-162: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:163-166: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:42-76: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:93-95: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:96-98: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:99-103: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:104-112: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:113-117: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:118-122: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:123-128: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:129-132: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:133-138: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:139-142: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:143-148: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:149-152: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:153-158: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:159-162: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:163-168: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:169-172: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:173-180: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:181-184: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:185-188: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:189-192: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:193-196: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:197-200: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:24-35: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:43-67: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:68-71: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:72-75: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:76-78: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:79-83: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:84-87: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:95-97: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:98-100: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:101-103: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:60-74: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:82-84: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:85-87: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:88-92: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:93-100: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:101-105: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:106-110: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:111-115: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:116-120: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:121-125: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:126-129: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:130-143: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:144-155: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:156-161: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:162-165: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:166-169: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:170-175: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:176-179: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:180-183: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:184-190: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:191-194: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:195-197: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:198-201: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:202-204: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:205-208: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:209-211: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:212-214: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:215-218: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:219-222: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:223-226: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:227-230: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:231-234: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:42-50: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:56-57: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:58-59: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:60-61: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:62-63: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:64-65: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:66-69: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:70-73: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:74-77: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:78-80: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:81-83: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:84-87: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:88-91: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:64-78: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:93-95: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:96-98: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:99-103: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:104-111: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:112-116: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:117-121: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:122-126: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:127-130: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:131-136: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:137-142: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:143-148: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:149-155: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:156-161: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:162-167: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:168-171: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:172-175: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:176-181: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:182-185: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:186-189: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:190-196: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:197-200: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:201-204: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:205-208: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:209-211: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:212-214: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:215-218: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:219-222: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:223-226: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:227-230: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:231-234: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:30-46: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:57-59: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:60-62: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:63-67: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:68-76: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:77-81: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:82-86: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:87-98: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:99-109: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:110-113: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:114-117: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:118-121: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:122-125: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:126-129: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:33-57: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:67-69: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:70-72: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:73-77: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:78-86: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:87-91: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:92-96: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:97-108: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:109-113: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:114-117: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:118-121: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:122-125: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:126-129: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:130-133: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:42-73: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:91-93: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:94-96: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:97-101: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:102-110: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:111-115: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:116-120: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:121-126: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:127-130: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:131-136: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:137-140: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:141-146: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:147-150: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:151-156: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:157-160: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:161-166: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:167-170: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:171-178: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:179-182: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:183-186: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:187-190: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:191-194: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:195-198: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:28-35: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:43-66: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:67-70: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:71-74: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:75-79: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:80-85: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:86-90: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:91-95: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:103-105: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:106-108: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:109-111: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:30-50: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:59-63: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:64-68: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:69-72: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:73-80: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:81-84: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:85-88: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:89-92: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:93-96: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:100-103: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:104-108: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:109-113: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:114-117: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:31-56: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:65-67: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:68-70: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:71-73: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:74-80: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:81-87: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:88-91: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:92-100: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:101-104: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:105-108: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:109-112: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:113-116: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:117-120: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:121-125: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:126-131: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:36-58: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:67-71: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:72-80: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:81-85: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:86-90: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:91-95: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:96-103: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:104-107: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:108-111: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:112-115: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:116-119: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:123-126: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:127-131: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:132-136: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:137-140: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:141-144: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:145-148: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:149-152: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:153-156: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:157-160: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:38-66: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:75-77: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:78-80: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:81-83: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:84-90: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:91-101: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:102-108: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:109-114: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:115-120: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:121-129: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:130-133: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:134-137: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:138-141: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:142-145: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:146-149: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:150-155: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:156-161: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:162-165: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:166-169: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:170-173: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:174-177: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:178-181: \item in \describe must have non-empty label
checkRd: (-1) chromIntensityPlot.Rd:38: Escaped LaTeX specials: \&amp;
checkRd: (-1) meanIntensityByScanChrom.Rd:3: Escaped LaTeX specials: \&amp;
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking LazyData ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
          user system elapsed
vcfWrite 15.39  0.428  15.819
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘test.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘Affymetrix.Rnw’ using ‘UTF-8’... OK
  ‘DataCleaning.Rnw’ using ‘UTF-8’... OK
  ‘Formats.Rnw’ using ‘UTF-8’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 WARNING, 1 NOTE
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/GWASTools.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">GWASTools.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL GWASTools
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘GWASTools’ ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (GWASTools)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">GWASTools.Rcheck/tests/test.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(GWASdata)
Loading required package: GWASTools
Loading required package: Biobase
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

&gt; library(ncdf4)
&gt; library(gdsfmt)
&gt; BiocGenerics:::testPackage(&quot;GWASTools&quot;, pattern=&quot;.*_test\\.R$&quot;)
block 1 of 1
adding variables: quality, X, Y
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe19a992d5&#x27; (67.1K)
    # of fragments: 35
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe19a992d5.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe19a992d5.tmp&#x27; (66.9K, reduced: 216B)
    # of fragments: 17
adding variables: quality, X, Y
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe7b9fbe97&#x27; (67.1K)
    # of fragments: 35
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe7b9fbe97.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe7b9fbe97.tmp&#x27; (66.9K, reduced: 216B)
    # of fragments: 17
Analyzing: X280 
Analyzing: X281 
Analyzing: X280 
Analyzing: X281 
Analyzing: X280 
Analyzing: X281 
Sat Feb 11 16:08:46 2023 Chromosome 1
Sat Feb 11 16:08:46 2023 Chromosome 2
Sat Feb 11 16:08:46 2023 Chromosome 3
Sat Feb 11 16:08:46 2023 Chromosome 4
Sat Feb 11 16:08:46 2023 Chromosome 5
Sat Feb 11 16:08:46 2023 Chromosome 6
Sat Feb 11 16:08:46 2023 Chromosome 7
Sat Feb 11 16:08:46 2023 Chromosome 8
Sat Feb 11 16:08:46 2023 Chromosome 9
Sat Feb 11 16:08:46 2023 Chromosome 10
Sat Feb 11 16:08:46 2023 Chromosome 11
Sat Feb 11 16:08:46 2023 Chromosome 12
Sat Feb 11 16:08:46 2023 Chromosome 13
Sat Feb 11 16:08:46 2023 Chromosome 14
Sat Feb 11 16:08:46 2023 Chromosome 15
Sat Feb 11 16:08:46 2023 Chromosome 16
Sat Feb 11 16:08:46 2023 Chromosome 17
Sat Feb 11 16:08:46 2023 Chromosome 18
Sat Feb 11 16:08:46 2023 Chromosome 19
Sat Feb 11 16:08:46 2023 Chromosome 20
Sat Feb 11 16:08:46 2023 Chromosome 21
Sat Feb 11 16:08:46 2023 Chromosome 22
Sat Feb 11 16:08:46 2023 Chromosome 23
Sat Feb 11 16:08:46 2023 Chromosome 24
Sat Feb 11 16:08:46 2023 Chromosome 25
Sat Feb 11 16:08:46 2023 Chromosome 26
Sat Feb 11 16:08:46 2023 Total # of SNPs selected: 7
Sat Feb 11 16:08:46 2023 Chromosome 1
Sat Feb 11 16:08:46 2023 Chromosome 2
Sat Feb 11 16:08:46 2023 Chromosome 3
Sat Feb 11 16:08:46 2023 Chromosome 4
Sat Feb 11 16:08:46 2023 Chromosome 5
Sat Feb 11 16:08:46 2023 Chromosome 6
Sat Feb 11 16:08:46 2023 Chromosome 7
Sat Feb 11 16:08:46 2023 Chromosome 8
Sat Feb 11 16:08:46 2023 Chromosome 9
Sat Feb 11 16:08:46 2023 Chromosome 10
Sat Feb 11 16:08:46 2023 Chromosome 11
Sat Feb 11 16:08:46 2023 Chromosome 12
Sat Feb 11 16:08:46 2023 Chromosome 13
Sat Feb 11 16:08:46 2023 Chromosome 14
Sat Feb 11 16:08:46 2023 Chromosome 15
Sat Feb 11 16:08:46 2023 Chromosome 16
Sat Feb 11 16:08:46 2023 Chromosome 17
Sat Feb 11 16:08:46 2023 Chromosome 18
Sat Feb 11 16:08:46 2023 Chromosome 19
Sat Feb 11 16:08:46 2023 Chromosome 20
Sat Feb 11 16:08:46 2023 Chromosome 21
Sat Feb 11 16:08:46 2023 Chromosome 22
Sat Feb 11 16:08:46 2023 Chromosome 23
Sat Feb 11 16:08:46 2023 Chromosome 24
Sat Feb 11 16:08:46 2023 Chromosome 25
Sat Feb 11 16:08:46 2023 Chromosome 26
Sat Feb 11 16:08:46 2023 Total # of SNPs selected: 9
Loading required package: VariantAnnotation
Loading required package: MatrixGenerics
Loading required package: matrixStats

Attaching package: &#x27;matrixStats&#x27;

The following objects are masked from &#x27;package:Biobase&#x27;:

    anyMissing, rowMedians


Attaching package: &#x27;MatrixGenerics&#x27;

The following objects are masked from &#x27;package:matrixStats&#x27;:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars

The following object is masked from &#x27;package:Biobase&#x27;:

    rowMedians

Loading required package: GenomeInfoDb
Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomicRanges
Loading required package: SummarizedExperiment
Loading required package: Rsamtools
Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit


Attaching package: &#x27;VariantAnnotation&#x27;

The following object is masked from &#x27;package:base&#x27;:

    tabulate

Loading required package: SNPRelate
SNPRelate
Reading in Phenotype and Covariate Data...
Running analysis with 40 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.6589 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
[1] &quot;hi&quot;
   LR.Stat    LR.pval 
8.55400428 0.03584773 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
1.6733633 0.6428719 
[1] &quot;hi&quot;
    LR.Stat     LR.pval 
11.30391026  0.01019107 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
3.5479450 0.3145968 
[1] &quot;hi&quot;
   LR.Stat    LR.pval 
8.85703394 0.03125334 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
4.5360542 0.2090968 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
5.2442899 0.1547585 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
2.6576472 0.4474727 
[1] &quot;hi&quot;
    LR.Stat     LR.pval 
15.67110354  0.00132434 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
1.7737546 0.6206628 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
2.7135101 0.4379363 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
2.4907942 0.4769571 
[1] &quot;hi&quot;
    LR.Stat     LR.pval 
10.06391337  0.01803059 
[1] &quot;hi&quot;
 LR.Stat  LR.pval 
4.834600 0.184317 
[1] &quot;hi&quot;
   LR.Stat    LR.pval 
6.85119755 0.07679492 
[1] &quot;hi&quot;
     LR.Stat      LR.pval 
12.410321031  0.006101951 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
5.9806637 0.1125548 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
4.5674714 0.2063504 
[1] &quot;hi&quot;
    LR.Stat     LR.pval 
10.29919824  0.01618679 
[1] &quot;hi&quot;
     LR.Stat      LR.pval 
12.161441314  0.006850104 
[1] &quot;hi&quot;
   LR.Stat    LR.pval 
8.85117856 0.03133641 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
4.6736748 0.1973122 
Block 1 of 1 Completed - 1.809 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
[1] &quot;hi&quot;
     LR.Stat      LR.pval 
1.567110e+01 7.536688e-05 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
1.6624991 0.1972662 
[1] &quot;hi&quot;
   LR.Stat    LR.pval 
8.38393977 0.00378551 
Block 1 of 1 Completed - 1.237 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.5904 secs
Reading in Phenotype and Covariate Data...
Running analysis with 500 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.7824 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.8496 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.3268 secs
Reading in Phenotype and Covariate Data...
Running analysis with 23 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.1941 secs
Reading in Phenotype and Covariate Data...
Running analysis with 23 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.1917 secs
Reading in Phenotype and Covariate Data...
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.1931 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 4 Completed - 0.06229 secs
Block 2 of 4 Completed - 0.06402 secs
Block 3 of 4 Completed - 0.06421 secs
Block 4 of 4 Completed - 0.002446 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.2985 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.2726 secs
Reading in Phenotype and Covariate Data...
Running analysis with 40 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.5294 secs
Reading in Phenotype and Covariate Data...
Running analysis with 40 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.3116 secs
Reading in Phenotype and Covariate Data...
Running analysis with 40 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.3753 secs
Loading required package: zoo

Attaching package: &#x27;zoo&#x27;

The following objects are masked from &#x27;package:Rsamtools&#x27;:

    index, index&lt;-

The following objects are masked from &#x27;package:base&#x27;:

    as.Date, as.Date.numeric

Reading in Phenotype and Covariate Data...
Running analysis with 40 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.4131 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.2256 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.0951 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.0993 secs
Sat Feb 11 16:09:13 2023	Load genotype dataset ...
Sat Feb 11 16:09:13 2023	1/2	batch1
Sat Feb 11 16:09:14 2023	Load genotype dataset ...
Sat Feb 11 16:09:14 2023	1/4	batch1
Sat Feb 11 16:09:14 2023	2/4	batch2
Sat Feb 11 16:09:14 2023	3/4	batch3
Sat Feb 11 16:09:14 2023	4/4	batch4
Sat Feb 11 16:09:14 2023	Load genotype dataset ...
Sat Feb 11 16:09:14 2023	1/4	batch1
Sat Feb 11 16:09:14 2023	2/4	batch2
Sat Feb 11 16:09:14 2023	3/4	batch3
Sat Feb 11 16:09:14 2023	4/4	batch4
Sat Feb 11 16:09:14 2023	Load genotype dataset ...
Sat Feb 11 16:09:14 2023	1/4	batch1
Sat Feb 11 16:09:14 2023	2/4	batch2
Sat Feb 11 16:09:14 2023	3/4	batch3
Sat Feb 11 16:09:14 2023	4/4	batch4
Sat Feb 11 16:09:14 2023	Load genotype dataset ...
Sat Feb 11 16:09:14 2023	1/4	batch1
Sat Feb 11 16:09:14 2023	2/4	batch2
Sat Feb 11 16:09:14 2023	3/4	batch3
Sat Feb 11 16:09:14 2023	4/4	batch4
Sat Feb 11 16:09:14 2023	Load genotype dataset ...
Sat Feb 11 16:09:14 2023	1/4	batch1
Sat Feb 11 16:09:14 2023	2/4	batch2
Sat Feb 11 16:09:14 2023	3/4	batch3
Sat Feb 11 16:09:14 2023	4/4	batch4
Sat Feb 11 16:09:15 2023	Load genotype dataset ...
Sat Feb 11 16:09:15 2023	1/4	batch1
Sat Feb 11 16:09:15 2023	2/4	batch2
Sat Feb 11 16:09:15 2023	3/4	batch3
Sat Feb 11 16:09:15 2023	4/4	batch4
Sat Feb 11 16:09:15 2023	Load genotype dataset ...
Sat Feb 11 16:09:15 2023	1/4	batch1
Sat Feb 11 16:09:15 2023	2/4	batch2
Sat Feb 11 16:09:15 2023	3/4	batch3
Sat Feb 11 16:09:15 2023	4/4	batch4
Sat Feb 11 16:09:15 2023	Load genotype dataset ...
Sat Feb 11 16:09:15 2023	1/4	batch1
Sat Feb 11 16:09:15 2023	2/4	batch2
Sat Feb 11 16:09:15 2023	3/4	batch3
Sat Feb 11 16:09:15 2023	4/4	batch4
Sat Feb 11 16:09:15 2023	Load genotype dataset ...
Sat Feb 11 16:09:15 2023	1/2	batch1
Sat Feb 11 16:09:16 2023	Load genotype dataset ...
Sat Feb 11 16:09:16 2023	1/4	batch1
Sat Feb 11 16:09:16 2023	2/4	batch2
Sat Feb 11 16:09:16 2023	3/4	batch3
Sat Feb 11 16:09:16 2023	4/4	batch4
Sat Feb 11 16:09:17 2023	Load genotype dataset ...
Sat Feb 11 16:09:17 2023	1/2	batch1
Sat Feb 11 16:09:17 2023	Load genotype dataset ...
Sat Feb 11 16:09:17 2023	1/2	batch1
Sat Feb 11 16:09:17 2023	Load genotype dataset ...
Sat Feb 11 16:09:17 2023	1/2	batch1
Sat Feb 11 16:09:17 2023	Load genotype dataset ...
Sat Feb 11 16:09:17 2023	1/2	batch1
Sat Feb 11 16:09:18 2023	Load genotype dataset ...
Sat Feb 11 16:09:18 2023	1/2	batch1
Sat Feb 11 16:09:18 2023	Load genotype dataset ...
Sat Feb 11 16:09:18 2023	1/2	batch1
Sat Feb 11 16:09:18 2023	Load genotype dataset ...
Sat Feb 11 16:09:18 2023	1/2	batch1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 100
Block 2 of 100
Block 3 of 100
Block 4 of 100
Block 5 of 100
Block 6 of 100
Block 7 of 100
Block 8 of 100
Block 9 of 100
Block 10 of 100
Block 11 of 100
Block 12 of 100
Block 13 of 100
Block 14 of 100
Block 15 of 100
Block 16 of 100
Block 17 of 100
Block 18 of 100
Block 19 of 100
Block 20 of 100
Block 21 of 100
Block 22 of 100
Block 23 of 100
Block 24 of 100
Block 25 of 100
Block 26 of 100
Block 27 of 100
Block 28 of 100
Block 29 of 100
Block 30 of 100
Block 31 of 100
Block 32 of 100
Block 33 of 100
Block 34 of 100
Block 35 of 100
Block 36 of 100
Block 37 of 100
Block 38 of 100
Block 39 of 100
Block 40 of 100
Block 41 of 100
Block 42 of 100
Block 43 of 100
Block 44 of 100
Block 45 of 100
Block 46 of 100
Block 47 of 100
Block 48 of 100
Block 49 of 100
Block 50 of 100
Block 51 of 100
Block 52 of 100
Block 53 of 100
Block 54 of 100
Block 55 of 100
Block 56 of 100
Block 57 of 100
Block 58 of 100
Block 59 of 100
Block 60 of 100
Block 61 of 100
Block 62 of 100
Block 63 of 100
Block 64 of 100
Block 65 of 100
Block 66 of 100
Block 67 of 100
Block 68 of 100
Block 69 of 100
Block 70 of 100
Block 71 of 100
Block 72 of 100
Block 73 of 100
Block 74 of 100
Block 75 of 100
Block 76 of 100
Block 77 of 100
Block 78 of 100
Block 79 of 100
Block 80 of 100
Block 81 of 100
Block 82 of 100
Block 83 of 100
Block 84 of 100
Block 85 of 100
Block 86 of 100
Block 87 of 100
Block 88 of 100
Block 89 of 100
Block 90 of 100
Block 91 of 100
Block 92 of 100
Block 93 of 100
Block 94 of 100
Block 95 of 100
Block 96 of 100
Block 97 of 100
Block 98 of 100
Block 99 of 100
Block 100 of 100
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 100
Block 2 of 100
Block 3 of 100
Block 4 of 100
Block 5 of 100
Block 6 of 100
Block 7 of 100
Block 8 of 100
Block 9 of 100
Block 10 of 100
Block 11 of 100
Block 12 of 100
Block 13 of 100
Block 14 of 100
Block 15 of 100
Block 16 of 100
Block 17 of 100
Block 18 of 100
Block 19 of 100
Block 20 of 100
Block 21 of 100
Block 22 of 100
Block 23 of 100
Block 24 of 100
Block 25 of 100
Block 26 of 100
Block 27 of 100
Block 28 of 100
Block 29 of 100
Block 30 of 100
Block 31 of 100
Block 32 of 100
Block 33 of 100
Block 34 of 100
Block 35 of 100
Block 36 of 100
Block 37 of 100
Block 38 of 100
Block 39 of 100
Block 40 of 100
Block 41 of 100
Block 42 of 100
Block 43 of 100
Block 44 of 100
Block 45 of 100
Block 46 of 100
Block 47 of 100
Block 48 of 100
Block 49 of 100
Block 50 of 100
Block 51 of 100
Block 52 of 100
Block 53 of 100
Block 54 of 100
Block 55 of 100
Block 56 of 100
Block 57 of 100
Block 58 of 100
Block 59 of 100
Block 60 of 100
Block 61 of 100
Block 62 of 100
Block 63 of 100
Block 64 of 100
Block 65 of 100
Block 66 of 100
Block 67 of 100
Block 68 of 100
Block 69 of 100
Block 70 of 100
Block 71 of 100
Block 72 of 100
Block 73 of 100
Block 74 of 100
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 100
Block 2 of 100
Block 3 of 100
Block 4 of 100
Block 5 of 100
Block 6 of 100
Block 7 of 100
Block 8 of 100
Block 9 of 100
Block 10 of 100
Block 11 of 100
Block 12 of 100
Block 13 of 100
Block 14 of 100
Block 15 of 100
Block 16 of 100
Block 17 of 100
Block 18 of 100
Block 19 of 100
Block 20 of 100
Block 21 of 100
Block 22 of 100
Block 23 of 100
Block 24 of 100
Block 25 of 100
Block 26 of 100
Block 27 of 100
Block 28 of 100
Block 29 of 100
Block 30 of 100
Block 31 of 100
Block 32 of 100
Block 33 of 100
Block 34 of 100
Block 35 of 100
Block 36 of 100
Block 37 of 100
Block 38 of 100
Block 39 of 100
Block 40 of 100
Block 41 of 100
Block 42 of 100
Block 43 of 100
Block 44 of 100
Block 45 of 100
Block 46 of 100
Block 47 of 100
Block 48 of 100
Block 49 of 100
Block 50 of 100
Block 51 of 100
Block 52 of 100
Block 53 of 100
Block 54 of 100
Block 55 of 100
Block 56 of 100
Block 57 of 100
Block 58 of 100
Block 59 of 100
Block 60 of 100
Block 61 of 100
Block 62 of 100
Block 63 of 100
Block 64 of 100
Block 65 of 100
Block 66 of 100
Block 67 of 100
Block 68 of 100
Block 69 of 100
Block 70 of 100
Block 71 of 100
Block 72 of 100
Block 73 of 100
Block 74 of 100
Block 75 of 100
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 100
Block 2 of 100
Block 3 of 100
Block 4 of 100
Block 5 of 100
Block 6 of 100
Block 7 of 100
Block 8 of 100
Block 9 of 100
Block 10 of 100
Block 11 of 100
Block 12 of 100
Block 13 of 100
Block 14 of 100
Block 15 of 100
Block 16 of 100
Block 17 of 100
Block 18 of 100
Block 19 of 100
Block 20 of 100
Block 21 of 100
Block 22 of 100
Block 23 of 100
Block 24 of 100
Block 25 of 100
Block 26 of 100
Block 27 of 100
Block 28 of 100
Block 29 of 100
Block 30 of 100
Block 31 of 100
Block 32 of 100
Block 33 of 100
Block 34 of 100
Block 35 of 100
Block 36 of 100
Block 37 of 100
Block 38 of 100
Block 39 of 100
Block 40 of 100
Block 41 of 100
Block 42 of 100
Block 43 of 100
Block 44 of 100
Block 45 of 100
Block 46 of 100
Block 47 of 100
Block 48 of 100
Block 49 of 100
Block 50 of 100
Block 51 of 100
Block 52 of 100
Block 53 of 100
Block 54 of 100
Block 55 of 100
Block 56 of 100
Block 57 of 100
Block 58 of 100
Block 59 of 100
Block 60 of 100
Block 61 of 100
Block 62 of 100
Block 63 of 100
Block 64 of 100
Block 65 of 100
Block 66 of 100
Block 67 of 100
Block 68 of 100
Block 69 of 100
Block 70 of 100
Block 71 of 100
Block 72 of 100
Block 73 of 100
Block 74 of 100
Block 75 of 100
Block 76 of 100
Block 77 of 100
Block 78 of 100
Block 79 of 100
Block 80 of 100
Block 81 of 100
Block 82 of 100
Block 83 of 100
Block 84 of 100
Block 85 of 100
Block 86 of 100
Block 87 of 100
Block 88 of 100
Block 89 of 100
Block 90 of 100
Block 91 of 100
Block 92 of 100
Block 93 of 100
Block 94 of 100
Block 95 of 100
Block 96 of 100
Block 97 of 100
Block 98 of 100
Block 99 of 100
Block 100 of 100
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 100
Block 2 of 100
Block 3 of 100
Block 4 of 100
Block 5 of 100
Block 6 of 100
Block 7 of 100
Block 8 of 100
Block 9 of 100
Block 10 of 100
Block 11 of 100
Block 12 of 100
Block 13 of 100
Block 14 of 100
Block 15 of 100
Block 16 of 100
Block 17 of 100
Block 18 of 100
Block 19 of 100
Block 20 of 100
Block 21 of 100
Block 22 of 100
Block 23 of 100
Block 24 of 100
Block 25 of 100
Block 26 of 100
Block 27 of 100
Block 28 of 100
Block 29 of 100
Block 30 of 100
Block 31 of 100
Block 32 of 100
Block 33 of 100
Block 34 of 100
Block 35 of 100
Block 36 of 100
Block 37 of 100
Block 38 of 100
Block 39 of 100
Block 40 of 100
Block 41 of 100
Block 42 of 100
Block 43 of 100
Block 44 of 100
Block 45 of 100
Block 46 of 100
Block 47 of 100
Block 48 of 100
Block 49 of 100
Block 50 of 100
Block 51 of 100
Block 52 of 100
Block 53 of 100
Block 54 of 100
Block 55 of 100
Block 56 of 100
Block 57 of 100
Block 58 of 100
Block 59 of 100
Block 60 of 100
Block 61 of 100
Block 62 of 100
Block 63 of 100
Block 64 of 100
Block 65 of 100
Block 66 of 100
Block 67 of 100
Block 68 of 100
Block 69 of 100
Block 70 of 100
Block 71 of 100
Block 72 of 100
Block 73 of 100
Block 74 of 100
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 100
Block 2 of 100
Block 3 of 100
Block 4 of 100
Block 5 of 100
Block 6 of 100
Block 7 of 100
Block 8 of 100
Block 9 of 100
Block 10 of 100
Block 11 of 100
Block 12 of 100
Block 13 of 100
Block 14 of 100
Block 15 of 100
Block 16 of 100
Block 17 of 100
Block 18 of 100
Block 19 of 100
Block 20 of 100
Block 21 of 100
Block 22 of 100
Block 23 of 100
Block 24 of 100
Block 25 of 100
Block 26 of 100
Block 27 of 100
Block 28 of 100
Block 29 of 100
Block 30 of 100
Block 31 of 100
Block 32 of 100
Block 33 of 100
Block 34 of 100
Block 35 of 100
Block 36 of 100
Block 37 of 100
Block 38 of 100
Block 39 of 100
Block 40 of 100
Block 41 of 100
Block 42 of 100
Block 43 of 100
Block 44 of 100
Block 45 of 100
Block 46 of 100
Block 47 of 100
Block 48 of 100
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 4
Block 2 of 4
Block 3 of 4
Block 4 of 4
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 4
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 4
Block 2 of 4
Block 3 of 4
Block 4 of 4
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 4
Block 2 of 4
Block 3 of 4
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 4
Block 2 of 4
Block 3 of 4
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 33
Block 2 of 33
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 13
Block 2 of 13
Block 3 of 13
Block 4 of 13
Block 5 of 13
Block 6 of 13
Block 7 of 13
Block 8 of 13
Block 9 of 13
Block 10 of 13
Block 11 of 13
Block 12 of 13
Block 13 of 13
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 13
Block 2 of 13
Block 3 of 13
Block 4 of 13
Block 5 of 13
Block 6 of 13
Block 7 of 13
Block 8 of 13
Block 9 of 13
Block 10 of 13
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 13
Block 2 of 13
Block 3 of 13
Block 4 of 13
Block 5 of 13
Block 6 of 13
Block 7 of 13
Block 8 of 13
Block 9 of 13
Block 10 of 13
Block 11 of 13
Block 12 of 13
Block 13 of 13
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 13
Block 2 of 13
Block 3 of 13
Block 4 of 13
Block 5 of 13
Block 6 of 13
Block 7 of 13
Block 8 of 13
Block 9 of 13
Block 10 of 13
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 6
Block 2 of 6
Block 3 of 6
Block 4 of 6
Block 5 of 6
Block 6 of 6
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 6
Block 2 of 6
Block 3 of 6
Block 4 of 6
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 6
Block 2 of 6
Block 3 of 6
Block 4 of 6
Block 5 of 6
Block 6 of 6
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 6
Block 2 of 6
Block 3 of 6
Block 4 of 6
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 13
Block 2 of 13
Block 3 of 13
Block 4 of 13
Block 5 of 13
Block 6 of 13
Block 7 of 13
Block 8 of 13
Block 9 of 13
Block 10 of 13
Block 11 of 13
Block 12 of 13
Block 13 of 13
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 13
Block 2 of 13
Block 3 of 13
Block 4 of 13
Block 5 of 13
Block 6 of 13
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 13
Block 2 of 13
Block 3 of 13
Block 4 of 13
Block 5 of 13
Block 6 of 13
Block 7 of 13
Block 8 of 13
Block 9 of 13
Block 10 of 13
Block 11 of 13
Block 12 of 13
Block 13 of 13
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 13
Block 2 of 13
Block 3 of 13
Block 4 of 13
Block 5 of 13
Block 6 of 13
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 6
Block 2 of 6
Block 3 of 6
Block 4 of 6
Block 5 of 6
Block 6 of 6
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 6
Block 2 of 6
Block 3 of 6
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 6
Block 2 of 6
Block 3 of 6
Block 4 of 6
Block 5 of 6
Block 6 of 6
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 6
Block 2 of 6
Block 3 of 6
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 500
Block 2 of 500
Block 3 of 500
Block 4 of 500
Block 5 of 500
Block 6 of 500
Block 7 of 500
Block 8 of 500
Block 9 of 500
Block 10 of 500
Block 11 of 500
Block 12 of 500
Block 13 of 500
Block 14 of 500
Block 15 of 500
Block 16 of 500
Block 17 of 500
Block 17 of 500
Block 18 of 500
Block 19 of 500
Block 20 of 500
Block 21 of 500
Block 22 of 500
Block 23 of 500
Block 24 of 500
Block 25 of 500
Block 26 of 500
Block 27 of 500
Block 28 of 500
Block 29 of 500
Block 30 of 500
Block 31 of 500
Block 32 of 500
Block 33 of 500
Block 34 of 500
Block 35 of 500
Block 36 of 500
Block 37 of 500
Block 38 of 500
Block 39 of 500
Block 40 of 500
Block 41 of 500
Block 42 of 500
Block 43 of 500
Block 44 of 500
Block 45 of 500
Block 46 of 500
Block 47 of 500
Block 48 of 500
Block 49 of 500
Block 50 of 500
Block 51 of 500
Block 52 of 500
Block 53 of 500
Block 53 of 500
Block 54 of 500
Block 55 of 500
Block 56 of 500
Block 57 of 500
Block 58 of 500
Block 59 of 500
Block 60 of 500
Block 61 of 500
Block 62 of 500
Block 63 of 500
Block 64 of 500
Block 65 of 500
Block 66 of 500
Block 67 of 500
Block 68 of 500
Block 69 of 500
Block 70 of 500
Block 71 of 500
Block 72 of 500
Block 73 of 500
Block 74 of 500
Block 75 of 500
Block 76 of 500
Block 77 of 500
Block 78 of 500
Block 79 of 500
Block 80 of 500
Block 81 of 500
Block 82 of 500
Block 83 of 500
Block 84 of 500
Block 85 of 500
Block 86 of 500
Block 87 of 500
Block 88 of 500
Block 89 of 500
Block 90 of 500
Block 91 of 500
Block 92 of 500
Block 93 of 500
Block 94 of 500
Block 95 of 500
Block 96 of 500
Block 97 of 500
Block 98 of 500
Block 99 of 500
Block 100 of 500
Block 101 of 500
Block 102 of 500
Block 103 of 500
Block 104 of 500
Block 105 of 500
Block 106 of 500
Block 107 of 500
Block 108 of 500
Block 109 of 500
Block 110 of 500
Block 111 of 500
Block 112 of 500
Block 113 of 500
Block 114 of 500
Block 115 of 500
Block 116 of 500
Block 117 of 500
Block 118 of 500
Block 119 of 500
Block 120 of 500
Block 121 of 500
Block 122 of 500
Block 123 of 500
Block 124 of 500
Block 125 of 500
Block 126 of 500
Block 127 of 500
Block 128 of 500
Block 129 of 500
Block 130 of 500
Block 131 of 500
Block 132 of 500
Block 133 of 500
Block 134 of 500
Block 135 of 500
Block 136 of 500
Block 137 of 500
Block 138 of 500
Block 139 of 500
Block 140 of 500
Block 141 of 500
Block 142 of 500
Block 143 of 500
Block 144 of 500
Block 145 of 500
Block 146 of 500
Block 147 of 500
Block 148 of 500
Block 149 of 500
Block 150 of 500
Block 151 of 500
Block 152 of 500
Block 153 of 500
Block 154 of 500
Block 155 of 500
Block 156 of 500
Block 157 of 500
Block 158 of 500
Block 159 of 500
Block 160 of 500
Block 161 of 500
Block 162 of 500
Block 163 of 500
Block 164 of 500
Block 165 of 500
Block 166 of 500
Block 167 of 500
Block 168 of 500
Block 169 of 500
Block 170 of 500
Block 171 of 500
Block 172 of 500
Block 173 of 500
Block 174 of 500
Block 175 of 500
Block 176 of 500
Block 177 of 500
Block 178 of 500
Block 179 of 500
Block 180 of 500
Block 181 of 500
Block 182 of 500
Block 183 of 500
Block 184 of 500
Block 185 of 500
Block 185 of 500
Block 186 of 500
Block 187 of 500
Block 188 of 500
Block 189 of 500
Block 190 of 500
Block 191 of 500
Block 192 of 500
Block 193 of 500
Block 194 of 500
Block 195 of 500
Block 196 of 500
Block 197 of 500
Block 198 of 500
Block 199 of 500
Block 200 of 500
Block 201 of 500
Block 202 of 500
Block 203 of 500
Block 204 of 500
Block 205 of 500
Block 206 of 500
Block 207 of 500
Block 208 of 500
Block 209 of 500
Block 210 of 500
Block 211 of 500
Block 212 of 500
Block 213 of 500
Block 214 of 500
Block 215 of 500
Block 216 of 500
Block 217 of 500
Block 218 of 500
Block 219 of 500
Block 220 of 500
Block 221 of 500
Block 222 of 500
Block 223 of 500
Block 224 of 500
Block 225 of 500
Block 226 of 500
Block 227 of 500
Block 228 of 500
Block 229 of 500
Block 230 of 500
Block 231 of 500
Block 232 of 500
Block 233 of 500
Block 234 of 500
Block 235 of 500
Block 236 of 500
Block 237 of 500
Block 238 of 500
Block 239 of 500
Block 240 of 500
Block 241 of 500
Block 242 of 500
Block 243 of 500
Block 244 of 500
Block 245 of 500
Block 246 of 500
Block 247 of 500
Block 248 of 500
Block 249 of 500
Block 250 of 500
Block 251 of 500
Block 252 of 500
Block 253 of 500
Block 254 of 500
Block 255 of 500
Block 256 of 500
Block 257 of 500
Block 258 of 500
Block 259 of 500
Block 260 of 500
Block 261 of 500
Block 262 of 500
Block 263 of 500
Block 264 of 500
Block 265 of 500
Block 266 of 500
Block 267 of 500
Block 268 of 500
Block 268 of 500
Block 269 of 500
Block 270 of 500
Block 271 of 500
Block 272 of 500
Block 273 of 500
Block 274 of 500
Block 275 of 500
Block 276 of 500
Block 277 of 500
Block 278 of 500
Block 279 of 500
Block 280 of 500
Block 281 of 500
Block 282 of 500
Block 283 of 500
Block 284 of 500
Block 285 of 500
Block 286 of 500
Block 287 of 500
Block 288 of 500
Block 289 of 500
Block 290 of 500
Block 291 of 500
Block 292 of 500
Block 293 of 500
Block 294 of 500
Block 295 of 500
Block 296 of 500
Block 297 of 500
Block 298 of 500
Block 299 of 500
Block 300 of 500
Block 301 of 500
Block 302 of 500
Block 303 of 500
Block 304 of 500
Block 305 of 500
Block 306 of 500
Block 307 of 500
Block 308 of 500
Block 309 of 500
Block 310 of 500
Block 311 of 500
Block 312 of 500
Block 313 of 500
Block 314 of 500
Block 315 of 500
Block 316 of 500
Block 317 of 500
Block 318 of 500
Block 319 of 500
Block 320 of 500
Block 321 of 500
Block 322 of 500
Block 323 of 500
Block 324 of 500
Block 325 of 500
Block 326 of 500
Block 327 of 500
Block 328 of 500
Block 329 of 500
Block 330 of 500
Block 331 of 500
Block 332 of 500
Block 333 of 500
Block 334 of 500
Block 335 of 500
Block 336 of 500
Block 337 of 500
Block 338 of 500
Block 339 of 500
Block 340 of 500
Block 341 of 500
Block 342 of 500
Block 343 of 500
Block 344 of 500
Block 345 of 500
Block 346 of 500
Block 347 of 500
Block 348 of 500
Block 349 of 500
Block 350 of 500
Block 351 of 500
Block 352 of 500
Block 353 of 500
Block 354 of 500
Block 355 of 500
Block 355 of 500
Block 356 of 500
Block 357 of 500
Block 358 of 500
Block 359 of 500
Block 360 of 500
Block 361 of 500
Block 362 of 500
Block 363 of 500
Block 364 of 500
Block 365 of 500
Block 366 of 500
Block 367 of 500
Block 368 of 500
Block 369 of 500
Block 370 of 500
Block 371 of 500
Block 372 of 500
Block 373 of 500
Block 374 of 500
Block 375 of 500
Block 376 of 500
Block 377 of 500
Block 378 of 500
Block 379 of 500
Block 380 of 500
Block 381 of 500
Block 382 of 500
Block 383 of 500
Block 384 of 500
Block 385 of 500
Block 386 of 500
Block 387 of 500
Block 388 of 500
Block 389 of 500
Block 390 of 500
Block 391 of 500
Block 392 of 500
Block 393 of 500
Block 394 of 500
Block 395 of 500
Block 396 of 500
Block 397 of 500
Block 398 of 500
Block 399 of 500
Block 400 of 500
Block 401 of 500
Block 402 of 500
Block 403 of 500
Block 404 of 500
Block 405 of 500
Block 406 of 500
Block 407 of 500
Block 408 of 500
Block 409 of 500
Block 410 of 500
Block 411 of 500
Block 412 of 500
Block 413 of 500
Block 414 of 500
Block 415 of 500
Block 416 of 500
Block 417 of 500
Block 418 of 500
Block 419 of 500
Block 420 of 500
Block 421 of 500
Block 422 of 500
Block 423 of 500
Block 424 of 500
Block 425 of 500
Block 426 of 500
Block 427 of 500
Block 428 of 500
Block 429 of 500
Block 430 of 500
Block 431 of 500
Block 432 of 500
Block 433 of 500
Block 434 of 500
Block 435 of 500
Block 436 of 500
Block 437 of 500
Block 438 of 500
Block 439 of 500
Block 440 of 500
Block 441 of 500
Block 442 of 500
Block 443 of 500
Block 444 of 500
Block 445 of 500
Block 446 of 500
Block 447 of 500
Block 448 of 500
Block 449 of 500
Block 450 of 500
Block 451 of 500
Block 452 of 500
Block 453 of 500
Block 454 of 500
Block 455 of 500
Block 456 of 500
Block 457 of 500
Block 458 of 500
Block 459 of 500
Block 460 of 500
Block 461 of 500
Block 462 of 500
Block 463 of 500
Block 464 of 500
Block 465 of 500
Block 466 of 500
Block 467 of 500
Block 468 of 500
Block 469 of 500
Block 470 of 500
Block 471 of 500
Block 472 of 500
Block 473 of 500
Block 474 of 500
Block 475 of 500
Block 476 of 500
Block 477 of 500
Block 478 of 500
Block 479 of 500
Block 480 of 500
Block 481 of 500
Block 482 of 500
Block 483 of 500
Block 484 of 500
Block 485 of 500
Block 486 of 500
Block 487 of 500
Block 488 of 500
Block 489 of 500
Block 490 of 500
Block 491 of 500
Block 492 of 500
Block 493 of 500
Block 494 of 500
Block 495 of 500
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 500
Block 2 of 500
Block 3 of 500
Block 4 of 500
Block 5 of 500
Block 6 of 500
Block 7 of 500
Block 8 of 500
Block 9 of 500
Block 10 of 500
Block 11 of 500
Block 12 of 500
Block 13 of 500
Block 14 of 500
Block 15 of 500
Block 16 of 500
Block 17 of 500
Block 17 of 500
Block 18 of 500
Block 19 of 500
Block 20 of 500
Block 21 of 500
Block 22 of 500
Block 23 of 500
Block 24 of 500
Block 25 of 500
Block 26 of 500
Block 27 of 500
Block 28 of 500
Block 29 of 500
Block 30 of 500
Block 31 of 500
Block 32 of 500
Block 33 of 500
Block 34 of 500
Block 35 of 500
Block 36 of 500
Block 37 of 500
Block 38 of 500
Block 39 of 500
Block 40 of 500
Block 41 of 500
Block 42 of 500
Block 43 of 500
Block 44 of 500
Block 45 of 500
Block 46 of 500
Block 47 of 500
Block 48 of 500
Block 49 of 500
Block 50 of 500
Block 51 of 500
Block 52 of 500
Block 53 of 500
Block 53 of 500
Block 54 of 500
Block 55 of 500
Block 56 of 500
Block 57 of 500
Block 58 of 500
Block 59 of 500
Block 60 of 500
Block 61 of 500
Block 62 of 500
Block 63 of 500
Block 64 of 500
Block 65 of 500
Block 66 of 500
Block 67 of 500
Block 68 of 500
Block 69 of 500
Block 70 of 500
Block 71 of 500
Block 72 of 500
Block 73 of 500
Block 74 of 500
Block 75 of 500
Block 76 of 500
Block 77 of 500
Block 78 of 500
Block 79 of 500
Block 80 of 500
Block 81 of 500
Block 82 of 500
Block 83 of 500
Block 84 of 500
Block 85 of 500
Block 86 of 500
Block 87 of 500
Block 88 of 500
Block 89 of 500
Block 90 of 500
Block 91 of 500
Block 92 of 500
Block 93 of 500
Block 94 of 500
Block 95 of 500
Block 96 of 500
Block 97 of 500
Block 98 of 500
Block 99 of 500
Block 100 of 500
Block 101 of 500
Block 102 of 500
Block 103 of 500
Block 104 of 500
Block 105 of 500
Block 106 of 500
Block 107 of 500
Block 108 of 500
Block 109 of 500
Block 110 of 500
Block 111 of 500
Block 112 of 500
Block 113 of 500
Block 114 of 500
Block 115 of 500
Block 116 of 500
Block 117 of 500
Block 118 of 500
Block 119 of 500
Block 120 of 500
Block 121 of 500
Block 122 of 500
Block 123 of 500
Block 124 of 500
Block 125 of 500
Block 126 of 500
Block 127 of 500
Block 128 of 500
Block 129 of 500
Block 130 of 500
Block 131 of 500
Block 132 of 500
Block 133 of 500
Block 134 of 500
Block 135 of 500
Block 136 of 500
Block 137 of 500
Block 138 of 500
Block 139 of 500
Block 140 of 500
Block 141 of 500
Block 142 of 500
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 500
Block 2 of 500
Block 3 of 500
Block 4 of 500
Block 5 of 500
Block 6 of 500
Block 7 of 500
Block 8 of 500
Block 9 of 500
Block 10 of 500
Block 11 of 500
Block 12 of 500
Block 13 of 500
Block 14 of 500
Block 15 of 500
Block 16 of 500
Block 17 of 500
Block 17 of 500
Block 18 of 500
Block 19 of 500
Block 20 of 500
Block 21 of 500
Block 22 of 500
Block 23 of 500
Block 24 of 500
Block 25 of 500
Block 26 of 500
Block 27 of 500
Block 28 of 500
Block 29 of 500
Block 30 of 500
Block 31 of 500
Block 32 of 500
Block 33 of 500
Block 34 of 500
Block 35 of 500
Block 36 of 500
Block 37 of 500
Block 38 of 500
Block 39 of 500
Block 40 of 500
Block 41 of 500
Block 42 of 500
Block 43 of 500
Block 44 of 500
Block 45 of 500
Block 46 of 500
Block 47 of 500
Block 48 of 500
Block 49 of 500
Block 50 of 500
Block 51 of 500
Block 52 of 500
Block 53 of 500
Block 53 of 500
Block 54 of 500
Block 55 of 500
Block 56 of 500
Block 57 of 500
Block 58 of 500
Block 59 of 500
Block 60 of 500
Block 61 of 500
Block 62 of 500
Block 63 of 500
Block 64 of 500
Block 65 of 500
Block 66 of 500
Block 67 of 500
Block 68 of 500
Block 69 of 500
Block 70 of 500
Block 71 of 500
Block 72 of 500
Block 73 of 500
Block 74 of 500
Block 75 of 500
Block 76 of 500
Block 77 of 500
Block 78 of 500
Block 79 of 500
Block 80 of 500
Block 81 of 500
Block 82 of 500
Block 83 of 500
Block 84 of 500
Block 85 of 500
Block 86 of 500
Block 87 of 500
Block 88 of 500
Block 89 of 500
Block 90 of 500
Block 91 of 500
Block 92 of 500
Block 93 of 500
Block 94 of 500
Block 95 of 500
Block 96 of 500
Block 97 of 500
Block 98 of 500
Block 99 of 500
Block 100 of 500
Block 101 of 500
Block 102 of 500
Block 103 of 500
Block 104 of 500
Block 105 of 500
Block 106 of 500
Block 107 of 500
Block 108 of 500
Block 109 of 500
Block 110 of 500
Block 111 of 500
Block 112 of 500
Block 113 of 500
Block 114 of 500
Block 115 of 500
Block 116 of 500
Block 117 of 500
Block 118 of 500
Block 119 of 500
Block 120 of 500
Block 121 of 500
Block 122 of 500
Block 123 of 500
Block 124 of 500
Block 125 of 500
Block 126 of 500
Block 127 of 500
Block 128 of 500
Block 129 of 500
Block 130 of 500
Block 131 of 500
Block 132 of 500
Block 133 of 500
Block 134 of 500
Block 135 of 500
Block 136 of 500
Block 137 of 500
Block 138 of 500
Block 139 of 500
Block 140 of 500
Block 141 of 500
Block 142 of 500
Block 143 of 500
Block 144 of 500
Block 145 of 500
Block 146 of 500
Block 147 of 500
Block 148 of 500
Block 149 of 500
Block 150 of 500
Block 151 of 500
Block 152 of 500
Block 153 of 500
Block 154 of 500
Block 155 of 500
Block 156 of 500
Block 157 of 500
Block 158 of 500
Block 159 of 500
Block 160 of 500
Block 161 of 500
Block 162 of 500
Block 163 of 500
Block 164 of 500
Block 165 of 500
Block 166 of 500
Block 167 of 500
Block 168 of 500
Block 169 of 500
Block 170 of 500
Block 171 of 500
Block 172 of 500
Block 173 of 500
Block 174 of 500
Block 175 of 500
Block 176 of 500
Block 177 of 500
Block 178 of 500
Block 179 of 500
Block 180 of 500
Block 181 of 500
Block 182 of 500
Block 183 of 500
Block 184 of 500
Block 185 of 500
Block 185 of 500
Block 186 of 500
Block 187 of 500
Block 188 of 500
Block 189 of 500
Block 190 of 500
Block 191 of 500
Block 192 of 500
Block 193 of 500
Block 194 of 500
Block 195 of 500
Block 196 of 500
Block 197 of 500
Block 198 of 500
Block 199 of 500
Block 200 of 500
Block 201 of 500
Block 202 of 500
Block 203 of 500
Block 204 of 500
Block 205 of 500
Block 206 of 500
Block 207 of 500
Block 208 of 500
Block 209 of 500
Block 210 of 500
Block 211 of 500
Block 212 of 500
Block 213 of 500
Block 214 of 500
Block 215 of 500
Block 216 of 500
Block 217 of 500
Block 218 of 500
Block 219 of 500
Block 220 of 500
Block 221 of 500
Block 222 of 500
Block 223 of 500
Block 224 of 500
Block 225 of 500
Block 226 of 500
Block 227 of 500
Block 228 of 500
Block 229 of 500
Block 230 of 500
Block 231 of 500
Block 232 of 500
Block 233 of 500
Block 234 of 500
Block 235 of 500
Block 236 of 500
Block 237 of 500
Block 238 of 500
Block 239 of 500
Block 240 of 500
Block 241 of 500
Block 242 of 500
Block 243 of 500
Block 244 of 500
Block 245 of 500
Block 246 of 500
Block 247 of 500
Block 248 of 500
Block 249 of 500
Block 250 of 500
Block 251 of 500
Block 252 of 500
Block 253 of 500
Block 254 of 500
Block 255 of 500
Block 256 of 500
Block 257 of 500
Block 258 of 500
Block 259 of 500
Block 260 of 500
Block 261 of 500
Block 262 of 500
Block 263 of 500
Block 264 of 500
Block 265 of 500
Block 266 of 500
Block 267 of 500
Block 268 of 500
Block 268 of 500
Block 269 of 500
Block 270 of 500
Block 271 of 500
Block 272 of 500
Block 273 of 500
Block 274 of 500
Block 275 of 500
Block 276 of 500
Block 277 of 500
Block 278 of 500
Block 279 of 500
Block 280 of 500
Block 281 of 500
Block 282 of 500
Block 283 of 500
Block 284 of 500
Block 285 of 500
Block 286 of 500
Block 287 of 500
Block 288 of 500
Block 289 of 500
Block 290 of 500
Block 291 of 500
Block 292 of 500
Block 293 of 500
Block 294 of 500
Block 295 of 500
Block 296 of 500
Block 297 of 500
Block 298 of 500
Block 299 of 500
Block 300 of 500
Block 301 of 500
Block 302 of 500
Block 303 of 500
Block 304 of 500
Block 305 of 500
Block 306 of 500
Block 307 of 500
Block 308 of 500
Block 309 of 500
Block 310 of 500
Block 311 of 500
Block 312 of 500
Block 313 of 500
Block 314 of 500
Block 315 of 500
Block 316 of 500
Block 317 of 500
Block 318 of 500
Block 319 of 500
Block 320 of 500
Block 321 of 500
Block 322 of 500
Block 323 of 500
Block 324 of 500
Block 325 of 500
Block 326 of 500
Block 327 of 500
Block 328 of 500
Block 329 of 500
Block 330 of 500
Block 331 of 500
Block 332 of 500
Block 333 of 500
Block 334 of 500
Block 335 of 500
Block 336 of 500
Block 337 of 500
Block 338 of 500
Block 339 of 500
Block 340 of 500
Block 341 of 500
Block 342 of 500
Block 343 of 500
Block 344 of 500
Block 345 of 500
Block 346 of 500
Block 347 of 500
Block 348 of 500
Block 349 of 500
Block 350 of 500
Block 351 of 500
Block 352 of 500
Block 353 of 500
Block 354 of 500
Block 355 of 500
Block 355 of 500
Block 356 of 500
Block 357 of 500
Block 358 of 500
Block 359 of 500
Block 360 of 500
Block 361 of 500
Block 362 of 500
Block 363 of 500
Block 364 of 500
Block 365 of 500
Block 366 of 500
Block 367 of 500
Block 368 of 500
Block 369 of 500
Block 370 of 500
Block 371 of 500
Block 372 of 500
Block 373 of 500
Block 374 of 500
Block 375 of 500
Block 376 of 500
Block 377 of 500
Block 378 of 500
Block 379 of 500
Block 380 of 500
Block 381 of 500
Block 382 of 500
Block 383 of 500
Block 384 of 500
Block 385 of 500
Block 386 of 500
Block 387 of 500
Block 388 of 500
Block 389 of 500
Block 390 of 500
Block 391 of 500
Block 392 of 500
Block 393 of 500
Block 394 of 500
Block 395 of 500
Block 396 of 500
Block 397 of 500
Block 398 of 500
Block 399 of 500
Block 400 of 500
Block 401 of 500
Block 402 of 500
Block 403 of 500
Block 404 of 500
Block 405 of 500
Block 406 of 500
Block 407 of 500
Block 408 of 500
Block 409 of 500
Block 410 of 500
Block 411 of 500
Block 412 of 500
Block 413 of 500
Block 414 of 500
Block 415 of 500
Block 416 of 500
Block 417 of 500
Block 418 of 500
Block 419 of 500
Block 420 of 500
Block 421 of 500
Block 422 of 500
Block 423 of 500
Block 424 of 500
Block 425 of 500
Block 426 of 500
Block 427 of 500
Block 428 of 500
Block 429 of 500
Block 430 of 500
Block 431 of 500
Block 432 of 500
Block 433 of 500
Block 434 of 500
Block 435 of 500
Block 436 of 500
Block 437 of 500
Block 438 of 500
Block 439 of 500
Block 440 of 500
Block 441 of 500
Block 442 of 500
Block 443 of 500
Block 444 of 500
Block 445 of 500
Block 446 of 500
Block 447 of 500
Block 448 of 500
Block 449 of 500
Block 450 of 500
Block 451 of 500
Block 452 of 500
Block 453 of 500
Block 454 of 500
Block 455 of 500
Block 456 of 500
Block 457 of 500
Block 458 of 500
Block 459 of 500
Block 460 of 500
Block 461 of 500
Block 462 of 500
Block 463 of 500
Block 464 of 500
Block 465 of 500
Block 466 of 500
Block 467 of 500
Block 468 of 500
Block 469 of 500
Block 470 of 500
Block 471 of 500
Block 472 of 500
Block 473 of 500
Block 474 of 500
Block 475 of 500
Block 476 of 500
Block 477 of 500
Block 478 of 500
Block 479 of 500
Block 480 of 500
Block 481 of 500
Block 482 of 500
Block 483 of 500
Block 484 of 500
Block 485 of 500
Block 486 of 500
Block 487 of 500
Block 488 of 500
Block 489 of 500
Block 490 of 500
Block 491 of 500
Block 492 of 500
Block 493 of 500
Block 494 of 500
Block 495 of 500
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 500
Block 2 of 500
Block 3 of 500
Block 4 of 500
Block 5 of 500
Block 6 of 500
Block 7 of 500
Block 8 of 500
Block 9 of 500
Block 10 of 500
Block 11 of 500
Block 12 of 500
Block 13 of 500
Block 14 of 500
Block 15 of 500
Block 16 of 500
Block 17 of 500
Block 17 of 500
Block 18 of 500
Block 19 of 500
Block 20 of 500
Block 21 of 500
Block 22 of 500
Block 23 of 500
Block 24 of 500
Block 25 of 500
Block 26 of 500
Block 27 of 500
Block 28 of 500
Block 29 of 500
Block 30 of 500
Block 31 of 500
Block 32 of 500
Block 33 of 500
Block 34 of 500
Block 35 of 500
Block 36 of 500
Block 37 of 500
Block 38 of 500
Block 39 of 500
Block 40 of 500
Block 41 of 500
Block 42 of 500
Block 43 of 500
Block 44 of 500
Block 45 of 500
Block 46 of 500
Block 47 of 500
Block 48 of 500
Block 49 of 500
Block 50 of 500
Block 51 of 500
Block 52 of 500
Block 53 of 500
Block 53 of 500
Block 54 of 500
Block 55 of 500
Block 56 of 500
Block 57 of 500
Block 58 of 500
Block 59 of 500
Block 60 of 500
Block 61 of 500
Block 62 of 500
Block 63 of 500
Block 64 of 500
Block 65 of 500
Block 66 of 500
Block 67 of 500
Block 68 of 500
Block 69 of 500
Block 70 of 500
Block 71 of 500
Block 72 of 500
Block 73 of 500
Block 74 of 500
Block 75 of 500
Block 76 of 500
Block 77 of 500
Block 78 of 500
Block 79 of 500
Block 80 of 500
Block 81 of 500
Block 82 of 500
Block 83 of 500
Block 84 of 500
Block 85 of 500
Block 86 of 500
Block 87 of 500
Block 88 of 500
Block 89 of 500
Block 90 of 500
Block 91 of 500
Block 92 of 500
Block 93 of 500
Block 94 of 500
Block 95 of 500
Block 96 of 500
Block 97 of 500
Block 98 of 500
Block 99 of 500
Block 100 of 500
Block 101 of 500
Block 102 of 500
Block 103 of 500
Block 104 of 500
Block 105 of 500
Block 106 of 500
Block 107 of 500
Block 108 of 500
Block 109 of 500
Block 110 of 500
Block 111 of 500
Block 112 of 500
Block 113 of 500
Block 114 of 500
Block 115 of 500
Block 116 of 500
Block 117 of 500
Block 118 of 500
Block 119 of 500
Block 120 of 500
Block 121 of 500
Block 122 of 500
Block 123 of 500
Block 124 of 500
Block 125 of 500
Block 126 of 500
Block 127 of 500
Block 128 of 500
Block 129 of 500
Block 130 of 500
Block 131 of 500
Block 132 of 500
Block 133 of 500
Block 134 of 500
Block 135 of 500
Block 136 of 500
Block 137 of 500
Block 138 of 500
Block 139 of 500
Block 140 of 500
Block 141 of 500
Block 142 of 500
Block 143 of 500
Block 144 of 500
Block 145 of 500
Block 146 of 500
Block 147 of 500
Block 148 of 500
Block 149 of 500
Block 150 of 500
Block 151 of 500
Block 152 of 500
Block 153 of 500
Block 154 of 500
Block 155 of 500
Block 156 of 500
Block 157 of 500
Block 158 of 500
Block 159 of 500
Block 160 of 500
Block 161 of 500
Block 162 of 500
Block 163 of 500
Block 164 of 500
Block 165 of 500
Block 166 of 500
Block 167 of 500
Block 168 of 500
Block 169 of 500
Block 170 of 500
Block 171 of 500
Block 172 of 500
Block 173 of 500
Block 174 of 500
Block 175 of 500
Block 176 of 500
Block 177 of 500
Block 178 of 500
Block 179 of 500
Block 180 of 500
Block 181 of 500
Block 182 of 500
Block 183 of 500
Block 184 of 500
Block 185 of 500
Block 185 of 500
Block 186 of 500
Block 187 of 500
Block 188 of 500
Block 189 of 500
Block 190 of 500
Block 191 of 500
Block 192 of 500
Block 193 of 500
Block 194 of 500
Block 195 of 500
Block 196 of 500
Block 197 of 500
Block 198 of 500
Block 199 of 500
Block 200 of 500
Block 201 of 500
Block 202 of 500
Block 203 of 500
Block 204 of 500
Block 205 of 500
Block 206 of 500
Block 207 of 500
Block 208 of 500
Block 209 of 500
Block 210 of 500
Block 211 of 500
Block 212 of 500
Block 213 of 500
Block 214 of 500
Block 215 of 500
Block 216 of 500
Block 217 of 500
Block 218 of 500
Block 219 of 500
Block 220 of 500
Block 221 of 500
Block 222 of 500
Block 223 of 500
Block 224 of 500
Block 225 of 500
Block 226 of 500
Block 227 of 500
Block 228 of 500
Block 229 of 500
Block 230 of 500
Block 231 of 500
Block 232 of 500
Block 233 of 500
Block 234 of 500
Block 235 of 500
Block 236 of 500
Block 237 of 500
Block 238 of 500
Block 239 of 500
Block 240 of 500
Block 241 of 500
Block 242 of 500
Block 243 of 500
Block 244 of 500
Block 245 of 500
Block 246 of 500
Block 247 of 500
Block 248 of 500
Block 249 of 500
Block 250 of 500
Block 251 of 500
Block 252 of 500
Block 253 of 500
Block 254 of 500
Block 255 of 500
Block 256 of 500
Block 257 of 500
Block 258 of 500
Block 259 of 500
Block 260 of 500
Block 261 of 500
Block 262 of 500
Block 263 of 500
Block 264 of 500
Block 265 of 500
Block 266 of 500
Block 267 of 500
Block 268 of 500
Block 268 of 500
Block 269 of 500
Block 270 of 500
Block 271 of 500
Block 272 of 500
Block 273 of 500
Block 274 of 500
Block 275 of 500
Block 276 of 500
Block 277 of 500
Block 278 of 500
Block 279 of 500
Block 280 of 500
Block 281 of 500
Block 282 of 500
Block 283 of 500
Block 284 of 500
Block 285 of 500
Block 286 of 500
Block 287 of 500
Block 288 of 500
Block 289 of 500
Block 290 of 500
Block 291 of 500
Block 292 of 500
Block 293 of 500
Block 294 of 500
Block 295 of 500
Block 296 of 500
Block 297 of 500
Block 298 of 500
Block 299 of 500
Block 300 of 500
Block 301 of 500
Block 302 of 500
Block 303 of 500
Block 304 of 500
Block 305 of 500
Block 306 of 500
Block 307 of 500
Block 308 of 500
Block 309 of 500
Block 310 of 500
Block 311 of 500
Block 312 of 500
Block 313 of 500
Block 314 of 500
Block 315 of 500
Block 316 of 500
Block 317 of 500
Block 318 of 500
Block 319 of 500
Block 320 of 500
Block 321 of 500
Block 322 of 500
Block 323 of 500
Block 324 of 500
Block 325 of 500
Block 326 of 500
Block 327 of 500
Block 328 of 500
Block 329 of 500
Block 330 of 500
Block 331 of 500
Block 332 of 500
Block 333 of 500
Block 334 of 500
Block 335 of 500
Block 336 of 500
Block 337 of 500
Block 338 of 500
Block 339 of 500
Block 340 of 500
Block 341 of 500
Block 342 of 500
Block 343 of 500
Block 344 of 500
Block 345 of 500
Block 346 of 500
Block 347 of 500
Block 348 of 500
Block 349 of 500
Block 350 of 500
Block 351 of 500
Block 352 of 500
Block 353 of 500
Block 354 of 500
Block 355 of 500
Block 355 of 500
Block 356 of 500
Block 357 of 500
Block 358 of 500
Block 359 of 500
Block 360 of 500
Block 361 of 500
Block 362 of 500
Block 363 of 500
Block 364 of 500
Block 365 of 500
Block 366 of 500
Block 367 of 500
Block 368 of 500
Block 369 of 500
Block 370 of 500
Block 371 of 500
Block 372 of 500
Block 373 of 500
Block 374 of 500
Block 375 of 500
Block 376 of 500
Block 377 of 500
Block 378 of 500
Block 379 of 500
Block 380 of 500
Block 381 of 500
Block 382 of 500
Block 383 of 500
Block 384 of 500
Block 385 of 500
Block 386 of 500
Block 387 of 500
Block 388 of 500
Block 389 of 500
Block 390 of 500
Block 391 of 500
Block 392 of 500
Block 393 of 500
Block 394 of 500
Block 395 of 500
Block 396 of 500
Block 397 of 500
Block 398 of 500
Block 399 of 500
Block 400 of 500
Block 401 of 500
Block 402 of 500
Block 403 of 500
Block 404 of 500
Block 405 of 500
Block 406 of 500
Block 407 of 500
Block 408 of 500
Block 409 of 500
Block 410 of 500
Block 411 of 500
Block 412 of 500
Block 413 of 500
Block 414 of 500
Block 415 of 500
Block 416 of 500
Block 417 of 500
Block 418 of 500
Block 419 of 500
Block 420 of 500
Block 421 of 500
Block 422 of 500
Block 423 of 500
Block 424 of 500
Block 425 of 500
Block 426 of 500
Block 427 of 500
Block 428 of 500
Block 429 of 500
Block 430 of 500
Block 431 of 500
Block 432 of 500
Block 433 of 500
Block 434 of 500
Block 435 of 500
Block 436 of 500
Block 437 of 500
Block 438 of 500
Block 439 of 500
Block 440 of 500
Block 441 of 500
Block 442 of 500
Block 443 of 500
Block 444 of 500
Block 445 of 500
Block 446 of 500
Block 447 of 500
Block 448 of 500
Block 449 of 500
Block 450 of 500
Block 451 of 500
Block 452 of 500
Block 453 of 500
Block 454 of 500
Block 455 of 500
Block 456 of 500
Block 457 of 500
Block 458 of 500
Block 459 of 500
Block 460 of 500
Block 461 of 500
Block 462 of 500
Block 463 of 500
Block 464 of 500
Block 465 of 500
Block 466 of 500
Block 467 of 500
Block 468 of 500
Block 469 of 500
Block 470 of 500
Block 471 of 500
Block 472 of 500
Block 473 of 500
Block 474 of 500
Block 475 of 500
Block 476 of 500
Block 477 of 500
Block 478 of 500
Block 479 of 500
Block 480 of 500
Block 481 of 500
Block 482 of 500
Block 483 of 500
Block 484 of 500
Block 485 of 500
Block 486 of 500
Block 487 of 500
Block 488 of 500
Block 489 of 500
Block 490 of 500
Block 491 of 500
Block 492 of 500
Block 493 of 500
Block 494 of 500
Block 495 of 500
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 500
Block 2 of 500
Block 3 of 500
Block 4 of 500
Block 5 of 500
Block 6 of 500
Block 7 of 500
Block 8 of 500
Block 9 of 500
Block 10 of 500
Block 11 of 500
Block 12 of 500
Block 13 of 500
Block 14 of 500
Block 15 of 500
Block 16 of 500
Block 17 of 500
Block 17 of 500
Block 18 of 500
Block 19 of 500
Block 20 of 500
Block 21 of 500
Block 22 of 500
Block 23 of 500
Block 24 of 500
Block 25 of 500
Block 26 of 500
Block 27 of 500
Block 28 of 500
Block 29 of 500
Block 30 of 500
Block 31 of 500
Block 32 of 500
Block 33 of 500
Block 34 of 500
Block 35 of 500
Block 36 of 500
Block 37 of 500
Block 38 of 500
Block 39 of 500
Block 40 of 500
Block 41 of 500
Block 42 of 500
Block 43 of 500
Block 44 of 500
Block 45 of 500
Block 46 of 500
Block 47 of 500
Block 48 of 500
Block 49 of 500
Block 50 of 500
Block 51 of 500
Block 52 of 500
Block 53 of 500
Block 53 of 500
Block 54 of 500
Block 55 of 500
Block 56 of 500
Block 57 of 500
Block 58 of 500
Block 59 of 500
Block 60 of 500
Block 61 of 500
Block 62 of 500
Block 63 of 500
Block 64 of 500
Block 65 of 500
Block 66 of 500
Block 67 of 500
Block 68 of 500
Block 69 of 500
Block 70 of 500
Block 71 of 500
Block 72 of 500
Block 73 of 500
Block 74 of 500
Block 75 of 500
Block 76 of 500
Block 77 of 500
Block 78 of 500
Block 79 of 500
Block 80 of 500
Block 81 of 500
Block 82 of 500
Block 83 of 500
Block 84 of 500
Block 85 of 500
Block 86 of 500
Block 87 of 500
Block 88 of 500
Block 89 of 500
Block 90 of 500
Block 91 of 500
Block 92 of 500
Block 93 of 500
Block 94 of 500
Block 95 of 500
Block 96 of 500
Block 97 of 500
Block 98 of 500
Block 99 of 500
Block 100 of 500
Block 101 of 500
Block 102 of 500
Block 103 of 500
Block 104 of 500
Block 105 of 500
Block 106 of 500
Block 107 of 500
Block 108 of 500
Block 109 of 500
Block 110 of 500
Block 111 of 500
Block 112 of 500
Block 113 of 500
Block 114 of 500
Block 115 of 500
Block 116 of 500
Block 117 of 500
Block 118 of 500
Block 119 of 500
Block 120 of 500
Block 121 of 500
Block 122 of 500
Block 123 of 500
Block 124 of 500
Block 125 of 500
Block 126 of 500
Block 127 of 500
Block 128 of 500
Block 129 of 500
Block 130 of 500
Block 131 of 500
Block 132 of 500
Block 133 of 500
Block 134 of 500
Block 135 of 500
Block 136 of 500
Block 137 of 500
Block 138 of 500
Block 139 of 500
Block 140 of 500
Block 141 of 500
Block 142 of 500
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 500
Block 2 of 500
Block 3 of 500
Block 4 of 500
Block 5 of 500
Block 6 of 500
Block 7 of 500
Block 8 of 500
Block 9 of 500
Block 10 of 500
Block 11 of 500
Block 12 of 500
Block 13 of 500
Block 14 of 500
Block 15 of 500
Block 16 of 500
Block 17 of 500
Block 17 of 500
Block 18 of 500
Block 19 of 500
Block 20 of 500
Block 21 of 500
Block 22 of 500
Block 23 of 500
Block 24 of 500
Block 25 of 500
Block 26 of 500
Block 27 of 500
Block 28 of 500
Block 29 of 500
Block 30 of 500
Block 31 of 500
Block 32 of 500
Block 33 of 500
Block 34 of 500
Block 35 of 500
Block 36 of 500
Block 37 of 500
Block 38 of 500
Block 39 of 500
Block 40 of 500
Block 41 of 500
Block 42 of 500
Block 43 of 500
Block 44 of 500
Block 45 of 500
Block 46 of 500
Block 47 of 500
Block 48 of 500
Block 49 of 500
Block 50 of 500
Block 51 of 500
Block 52 of 500
Block 53 of 500
Block 53 of 500
Block 54 of 500
Block 55 of 500
Block 56 of 500
Block 57 of 500
Block 58 of 500
Block 59 of 500
Block 60 of 500
Block 61 of 500
Block 62 of 500
Block 63 of 500
Block 64 of 500
Block 65 of 500
Block 66 of 500
Block 67 of 500
Block 68 of 500
Block 69 of 500
Block 70 of 500
Block 71 of 500
Block 72 of 500
Block 73 of 500
Block 74 of 500
Block 75 of 500
Block 76 of 500
Block 77 of 500
Block 78 of 500
Block 79 of 500
Block 80 of 500
Block 81 of 500
Block 82 of 500
Block 83 of 500
Block 84 of 500
Block 85 of 500
Block 86 of 500
Block 87 of 500
Block 88 of 500
Block 89 of 500
Block 90 of 500
Block 91 of 500
Block 92 of 500
Block 93 of 500
Block 94 of 500
Block 95 of 500
Block 96 of 500
Block 97 of 500
Block 98 of 500
Block 99 of 500
Block 100 of 500
Block 101 of 500
Block 102 of 500
Block 103 of 500
Block 104 of 500
Block 105 of 500
Block 106 of 500
Block 107 of 500
Block 108 of 500
Block 109 of 500
Block 110 of 500
Block 111 of 500
Block 112 of 500
Block 113 of 500
Block 114 of 500
Block 115 of 500
Block 116 of 500
Block 117 of 500
Block 118 of 500
Block 119 of 500
Block 120 of 500
Block 121 of 500
Block 122 of 500
Block 123 of 500
Block 124 of 500
Block 125 of 500
Block 126 of 500
Block 127 of 500
Block 128 of 500
Block 129 of 500
Block 130 of 500
Block 131 of 500
Block 132 of 500
Block 133 of 500
Block 134 of 500
Block 135 of 500
Block 136 of 500
Block 137 of 500
Block 138 of 500
Block 139 of 500
Block 140 of 500
Block 141 of 500
Block 142 of 500
Block 143 of 500
Block 144 of 500
Block 145 of 500
Block 146 of 500
Block 147 of 500
Block 148 of 500
Block 149 of 500
Block 150 of 500
Block 151 of 500
Block 152 of 500
Block 153 of 500
Block 154 of 500
Block 155 of 500
Block 156 of 500
Block 157 of 500
Block 158 of 500
Block 159 of 500
Block 160 of 500
Block 161 of 500
Block 162 of 500
Block 163 of 500
Block 164 of 500
Block 165 of 500
Block 166 of 500
Block 167 of 500
Block 168 of 500
Block 169 of 500
Block 170 of 500
Block 171 of 500
Block 172 of 500
Block 173 of 500
Block 174 of 500
Block 175 of 500
Block 176 of 500
Block 177 of 500
Block 178 of 500
Block 179 of 500
Block 180 of 500
Block 181 of 500
Block 182 of 500
Block 183 of 500
Block 184 of 500
Block 185 of 500
Block 185 of 500
Block 186 of 500
Block 187 of 500
Block 188 of 500
Block 189 of 500
Block 190 of 500
Block 191 of 500
Block 192 of 500
Block 193 of 500
Block 194 of 500
Block 195 of 500
Block 196 of 500
Block 197 of 500
Block 198 of 500
Block 199 of 500
Block 200 of 500
Block 201 of 500
Block 202 of 500
Block 203 of 500
Block 204 of 500
Block 205 of 500
Block 206 of 500
Block 207 of 500
Block 208 of 500
Block 209 of 500
Block 210 of 500
Block 211 of 500
Block 212 of 500
Block 213 of 500
Block 214 of 500
Block 215 of 500
Block 216 of 500
Block 217 of 500
Block 218 of 500
Block 219 of 500
Block 220 of 500
Block 221 of 500
Block 222 of 500
Block 223 of 500
Block 224 of 500
Block 225 of 500
Block 226 of 500
Block 227 of 500
Block 228 of 500
Block 229 of 500
Block 230 of 500
Block 231 of 500
Block 232 of 500
Block 233 of 500
Block 234 of 500
Block 235 of 500
Block 236 of 500
Block 237 of 500
Block 238 of 500
Block 239 of 500
Block 240 of 500
Block 241 of 500
Block 242 of 500
Block 243 of 500
Block 244 of 500
Block 245 of 500
Block 246 of 500
Block 247 of 500
Block 248 of 500
Block 249 of 500
Block 250 of 500
Block 251 of 500
Block 252 of 500
Block 253 of 500
Block 254 of 500
Block 255 of 500
Block 256 of 500
Block 257 of 500
Block 258 of 500
Block 259 of 500
Block 260 of 500
Block 261 of 500
Block 262 of 500
Block 263 of 500
Block 264 of 500
Block 265 of 500
Block 266 of 500
Block 267 of 500
Block 268 of 500
Block 268 of 500
Block 269 of 500
Block 270 of 500
Block 271 of 500
Block 272 of 500
Block 273 of 500
Block 274 of 500
Block 275 of 500
Block 276 of 500
Block 277 of 500
Block 278 of 500
Block 279 of 500
Block 280 of 500
Block 281 of 500
Block 282 of 500
Block 283 of 500
Block 284 of 500
Block 285 of 500
Block 286 of 500
Block 287 of 500
Block 288 of 500
Block 289 of 500
Block 290 of 500
Block 291 of 500
Block 292 of 500
Block 293 of 500
Block 294 of 500
Block 295 of 500
Block 296 of 500
Block 297 of 500
Block 298 of 500
Block 299 of 500
Block 300 of 500
Block 301 of 500
Block 302 of 500
Block 303 of 500
Block 304 of 500
Block 305 of 500
Block 306 of 500
Block 307 of 500
Block 308 of 500
Block 309 of 500
Block 310 of 500
Block 311 of 500
Block 312 of 500
Block 313 of 500
Block 314 of 500
Block 315 of 500
Block 316 of 500
Block 317 of 500
Block 318 of 500
Block 319 of 500
Block 320 of 500
Block 321 of 500
Block 322 of 500
Block 323 of 500
Block 324 of 500
Block 325 of 500
Block 326 of 500
Block 327 of 500
Block 328 of 500
Block 329 of 500
Block 330 of 500
Block 331 of 500
Block 332 of 500
Block 333 of 500
Block 334 of 500
Block 335 of 500
Block 336 of 500
Block 337 of 500
Block 338 of 500
Block 339 of 500
Block 340 of 500
Block 341 of 500
Block 342 of 500
Block 343 of 500
Block 344 of 500
Block 345 of 500
Block 346 of 500
Block 347 of 500
Block 348 of 500
Block 349 of 500
Block 350 of 500
Block 351 of 500
Block 352 of 500
Block 353 of 500
Block 354 of 500
Block 355 of 500
Block 355 of 500
Block 356 of 500
Block 357 of 500
Block 358 of 500
Block 359 of 500
Block 360 of 500
Block 361 of 500
Block 362 of 500
Block 363 of 500
Block 364 of 500
Block 365 of 500
Block 366 of 500
Block 367 of 500
Block 368 of 500
Block 369 of 500
Block 370 of 500
Block 371 of 500
Block 372 of 500
Block 373 of 500
Block 374 of 500
Block 375 of 500
Block 376 of 500
Block 377 of 500
Block 378 of 500
Block 379 of 500
Block 380 of 500
Block 381 of 500
Block 382 of 500
Block 383 of 500
Block 384 of 500
Block 385 of 500
Block 386 of 500
Block 387 of 500
Block 388 of 500
Block 389 of 500
Block 390 of 500
Block 391 of 500
Block 392 of 500
Block 393 of 500
Block 394 of 500
Block 395 of 500
Block 396 of 500
Block 397 of 500
Block 398 of 500
Block 399 of 500
Block 400 of 500
Block 401 of 500
Block 402 of 500
Block 403 of 500
Block 404 of 500
Block 405 of 500
Block 406 of 500
Block 407 of 500
Block 408 of 500
Block 409 of 500
Block 410 of 500
Block 411 of 500
Block 412 of 500
Block 413 of 500
Block 414 of 500
Block 415 of 500
Block 416 of 500
Block 417 of 500
Block 418 of 500
Block 419 of 500
Block 420 of 500
Block 421 of 500
Block 422 of 500
Block 423 of 500
Block 424 of 500
Block 425 of 500
Block 426 of 500
Block 427 of 500
Block 428 of 500
Block 429 of 500
Block 430 of 500
Block 431 of 500
Block 432 of 500
Block 433 of 500
Block 434 of 500
Block 435 of 500
Block 436 of 500
Block 437 of 500
Block 438 of 500
Block 439 of 500
Block 440 of 500
Block 441 of 500
Block 442 of 500
Block 443 of 500
Block 444 of 500
Block 445 of 500
Block 446 of 500
Block 447 of 500
Block 448 of 500
Block 449 of 500
Block 450 of 500
Block 451 of 500
Block 452 of 500
Block 453 of 500
Block 454 of 500
Block 455 of 500
Block 456 of 500
Block 457 of 500
Block 458 of 500
Block 459 of 500
Block 460 of 500
Block 461 of 500
Block 462 of 500
Block 463 of 500
Block 464 of 500
Block 465 of 500
Block 466 of 500
Block 467 of 500
Block 468 of 500
Block 469 of 500
Block 470 of 500
Block 471 of 500
Block 472 of 500
Block 473 of 500
Block 474 of 500
Block 475 of 500
Block 476 of 500
Block 477 of 500
Block 478 of 500
Block 479 of 500
Block 480 of 500
Block 481 of 500
Block 482 of 500
Block 483 of 500
Block 484 of 500
Block 485 of 500
Block 486 of 500
Block 487 of 500
Block 488 of 500
Block 489 of 500
Block 490 of 500
Block 491 of 500
Block 492 of 500
Block 493 of 500
Block 494 of 500
Block 495 of 500
adding variables: quality, X, Y
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafea12367e&#x27; (67.1K)
    # of fragments: 35
    save to &#x27;/tmp/Rtmpthm57M/file2ceafea12367e.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafea12367e.tmp&#x27; (66.9K, reduced: 216B)
    # of fragments: 17
Sat Feb 11 16:09:49 2023	begin convertNcdfGds ...

Sat Feb 11 16:09:49 2023	Creating GDS file ...

Sat Feb 11 16:09:50 2023	Adding sample data...

Sat Feb 11 16:09:50 2023	end convertNcdfGds.

Sat Feb 11 16:09:50 2023	begin checkNcdfGds ...

The numbers of samples are equal

The numbers of snps are equal.

Sat Feb 11 16:09:50 2023	Checking in the sample order ...

OK!!!

Sat Feb 11 16:09:50 2023	begin checkNcdfGds ...

The numbers of samples are equal

The numbers of snps are equal.

Sat Feb 11 16:09:50 2023	Checking in the sample order ...

The 1th sample error!
Sat Feb 11 16:09:50 2023	begin convertGdsNcdf ...

Sat Feb 11 16:09:50 2023		Creating NetCDF file ...

Sat Feb 11 16:09:50 2023		Adding sample data ...

Sat Feb 11 16:09:50 2023	end convertGdsNcdf.

Sat Feb 11 16:09:50 2023	begin convertGdsNcdf ...

Sat Feb 11 16:09:50 2023		Creating NetCDF file ...

Sat Feb 11 16:09:50 2023		Adding sample data ...

Sat Feb 11 16:09:50 2023	end convertGdsNcdf.

SNP genotypes: 20 samples, 260 SNPs
Genotype matrix is being transposed ...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe16eb4de0&#x27; (4.3K)
    # of fragments: 13
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe16eb4de0.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe16eb4de0.tmp&#x27; (3.0K, reduced: 1.4K)
    # of fragments: 11
Sat Feb 11 16:09:50 2023	begin convertGdsNcdf ...

Sat Feb 11 16:09:50 2023		Creating NetCDF file ...

Sat Feb 11 16:09:50 2023		Adding sample data ...

Sat Feb 11 16:09:50 2023	end convertGdsNcdf.

Sat Feb 11 16:09:50 2023	begin convertNcdfGds ...

Sat Feb 11 16:09:50 2023	Creating GDS file ...

Sat Feb 11 16:09:50 2023	Adding sample data...

Sat Feb 11 16:09:50 2023	end convertNcdfGds.

Sat Feb 11 16:09:50 2023	begin convertNcdfGds ...

Sat Feb 11 16:09:50 2023	Creating GDS file ...

Sat Feb 11 16:09:50 2023	Adding sample data...

Sat Feb 11 16:09:50 2023	end convertNcdfGds.

Sat Feb 11 16:09:50 2023	begin convertNcdfGds ...

Sat Feb 11 16:09:50 2023	Creating GDS file ...

Sat Feb 11 16:09:50 2023	Adding sample data...

Sat Feb 11 16:09:50 2023	end convertNcdfGds.

Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe4504093a&#x27; (88.4K)
    # of fragments: 31
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe4504093a.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe4504093a.tmp&#x27; (88.3K, reduced: 192B)
    # of fragments: 15
adding variables: genotype
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe1ee9a78e&#x27; (25.3K)
    # of fragments: 26
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe1ee9a78e.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe1ee9a78e.tmp&#x27; (25.1K, reduced: 156B)
    # of fragments: 13
adding variables: genotype
adding variables: genotype
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe7617cbbd&#x27; (28.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe7617cbbd.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe7617cbbd.tmp&#x27; (28.3K, reduced: 180B)
    # of fragments: 15
adding variables: genotype
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe7c1bbff0&#x27; (27.1K)
    # of fragments: 26
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe7c1bbff0.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe7c1bbff0.tmp&#x27; (27.0K, reduced: 156B)
    # of fragments: 13
adding variables: genotype
adding variables: quality, X, Y
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe4f871b39&#x27; (67.1K)
    # of fragments: 35
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe4f871b39.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe4f871b39.tmp&#x27; (66.9K, reduced: 216B)
    # of fragments: 17
adding variables: quality, X, Y
adding variables: genotype
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe57593ee9&#x27; (28.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe57593ee9.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe57593ee9.tmp&#x27; (28.3K, reduced: 180B)
    # of fragments: 15
working on genotype
genotype - sample 10 of 10
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe376a4612&#x27; (2.6K)
    # of fragments: 31
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe376a4612.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe376a4612.tmp&#x27; (2.4K, reduced: 192B)
    # of fragments: 15
working on genotype
genotype - sample 10 of 10
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe6e448cc9&#x27; (2.6K)
    # of fragments: 31
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe6e448cc9.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe6e448cc9.tmp&#x27; (2.4K, reduced: 192B)
    # of fragments: 15

Comparing identical datasets

Matching variants on position, alleles

Calculating squared correlation of allelic dosages at 20 overlapping variants in 10 duplicate sample pairs

Detecting that the two datasets are counting different A alleles at 0 variants

Getting genotypes:

Calculating correlation by SNP

Block 1 of 2
Block 2 of 2

Calculating correlation by sample

Block 1 of 2
Block 2 of 2

Finished!


Comparing no matching samples


Comparing &gt;1 dup pair

Matching variants on position, alleles

Calculating squared correlation of allelic dosages at 20 overlapping variants in 9 duplicate sample pairs

Detecting that the two datasets are counting different A alleles at 0 variants

Getting genotypes:

Calculating correlation by SNP

Block 1 of 1

Calculating correlation by sample

Block 1 of 1

Finished!


Comparing different datasets

Matching variants on position, alleles

Calculating squared correlation of allelic dosages at 19 overlapping variants in 9 duplicate sample pairs

Detecting that the two datasets are counting different A alleles at 1 variants

Getting genotypes:

Calculating correlation by SNP

Block 1 of 1

Calculating correlation by sample

Block 1 of 1

Finished!

	selecting genotypes from genoData1
	selecting genotypes from genoData2
subject 1 out of 3, 2 replications
subject 2 out of 3, 2 replications
subject 3 out of 3, 2 replications
subject 1 out of 3, 2 replications
subject 2 out of 3, 2 replications
subject 3 out of 3, 2 replications
subject 1 out of 3, 2 replications
subject 2 out of 3, 2 replications
subject 3 out of 3, 2 replications
Calculating allele freqency in genoData1
subject 1 out of 3, 2 replications
subject 2 out of 3, 2 replications
subject 3 out of 3, 2 replications
subject 1 out of 3, 2 replications
subject 2 out of 3, 2 replications
subject 3 out of 3, 3 replications
subject 1 out of 3, 2 replications
subject 2 out of 3, 2 replications
subject 3 out of 3, 2 replications
subject 1 out of 3, 2 replications
subject 2 out of 3, 2 replications
subject 3 out of 3, 3 replications
subject 1 out of 3, 2 replications
subject 2 out of 3, 2 replications
subject 3 out of 3, 3 replications
subject 1 out of 2, 2 replications
subject 2 out of 2, 2 replications
Calculating allele freqency in genoData1
subject 1 out of 4
subject 2 out of 4
subject 3 out of 4
subject 4 out of 4
subject 1 out of 2, 3 replications
subject 2 out of 2, 2 replications
calculating dosage correlation by SNP, in blocks of 5,000 SNPs
Block 1 of 1
subject 1 out of 2, 2 replications
subject 2 out of 2, 2 replications
calculating dosage correlation by SNP, in blocks of 5,000 SNPs
Block 1 of 1
subject 1 out of 2, 3 replications
subject 2 out of 2, 2 replications
calculating dosage correlation by SNP, in blocks of 5,000 SNPs
Block 1 of 1
subject 1 out of 2, 2 replications
subject 2 out of 2, 2 replications
subject 1 out of 2, 3 replications
subject 2 out of 2, 2 replications
subject 1 out of 2, 3 replications
subject 2 out of 2, 2 replications
Beginning Calculations...
Block 1 of 1 Completed - 0.000761 secs
Beginning Calculations...
Block 1 of 1 Completed - 0.0006166 secs
Beginning Calculations...
Block 1 of 1 Completed - 0.0007675 secs
Beginning Calculations...
Block 1 of 1 Completed - 0.002095 secs
Beginning Calculations...
Block 1 of 1 Completed - 0.000591 secs
Beginning Calculations...
Block 1 of 1 Completed - 0.001238 secs
working on genotype
genotype - sample 10 of 10
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe6dde8985&#x27; (2.3K)
    # of fragments: 27
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe6dde8985.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe6dde8985.tmp&#x27; (2.1K, reduced: 168B)
    # of fragments: 13
working on genotype
genotype - looping over samples
All variables match.
working on genotype
genotype - sample 10 of 10
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe379dd35&#x27; (2.3K)
    # of fragments: 27
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe379dd35.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe379dd35.tmp&#x27; (2.2K, reduced: 168B)
    # of fragments: 13
working on genotype
genotype - looping over samples
All variables match.
working on genotype
working on genotype
working on genotype
genotype - sample 10 of 10
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe6c51a7cb&#x27; (2.3K)
    # of fragments: 27
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe6c51a7cb.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe6c51a7cb.tmp&#x27; (2.2K, reduced: 168B)
    # of fragments: 13
working on genotype
genotype - looping over samples
All variables match.
working on genotype
genotype - looping over samples
working on genotype
genotype - looping over samples
All variables match.
working on genotype
genotype - looping over samples
working on genotype
genotype - sample 10 of 10
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafeb3b4f59&#x27; (2.3K)
    # of fragments: 27
    save to &#x27;/tmp/Rtmpthm57M/file2ceafeb3b4f59.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafeb3b4f59.tmp&#x27; (2.2K, reduced: 168B)
    # of fragments: 13
working on genotype
genotype - looping over samples
working on genotype
genotype - sample 10 of 10
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe34372d65&#x27; (2.3K)
    # of fragments: 27
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe34372d65.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe34372d65.tmp&#x27; (2.2K, reduced: 168B)
    # of fragments: 13
working on genotype
genotype - looping over samples
All variables match.
working on genotype
genotype - sample 10 of 10
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe7124b342&#x27; (3.2K)
    # of fragments: 27
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe7124b342.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe7124b342.tmp&#x27; (3.0K, reduced: 168B)
    # of fragments: 13
working on genotype
genotype - looping over samples
All variables match.
working on genotype
genotype - sample 10 of 20
genotype - sample 20 of 20
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe16256634&#x27; (2.5K)
    # of fragments: 27
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe16256634.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe16256634.tmp&#x27; (2.3K, reduced: 168B)
    # of fragments: 13
working on genotype
genotype - looping over samples
All variables match.
working on genotype
genotype - sample 10 of 10
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe12412624&#x27; (2.3K)
    # of fragments: 27
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe12412624.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe12412624.tmp&#x27; (2.2K, reduced: 168B)
    # of fragments: 13
working on genotype
genotype - looping over samples
All variables match.
working on genotype
working on genotype
genotype - block 1 of 1
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe5e3e5315&#x27; (2.3K)
    # of fragments: 27
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe5e3e5315.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe5e3e5315.tmp&#x27; (2.2K, reduced: 168B)
    # of fragments: 13
working on genotype
genotype - looping over snps
All variables match.
working on genotype
genotype - block 1 of 13
genotype - block 2 of 13
genotype - block 3 of 13
genotype - block 4 of 13
genotype - block 5 of 13
genotype - block 6 of 13
genotype - block 7 of 13
genotype - block 8 of 13
genotype - block 9 of 13
genotype - block 10 of 13
genotype - block 11 of 13
genotype - block 12 of 13
genotype - block 13 of 13
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe554a3f1&#x27; (2.3K)
    # of fragments: 27
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe554a3f1.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe554a3f1.tmp&#x27; (2.2K, reduced: 168B)
    # of fragments: 13
working on genotype
genotype - looping over snps
All variables match.
working on genotype
genotype - block 1 of 2
genotype - block 2 of 2
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe343982c4&#x27; (2.3K)
    # of fragments: 27
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe343982c4.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe343982c4.tmp&#x27; (2.2K, reduced: 168B)
    # of fragments: 13
working on genotype
genotype - looping over snps
All variables match.
working on genotype
genotype - block 1 of 260
genotype - block 2 of 260
genotype - block 3 of 260
genotype - block 4 of 260
genotype - block 5 of 260
genotype - block 6 of 260
genotype - block 7 of 260
genotype - block 8 of 260
genotype - block 9 of 260
genotype - block 10 of 260
genotype - block 11 of 260
genotype - block 12 of 260
genotype - block 13 of 260
genotype - block 14 of 260
genotype - block 15 of 260
genotype - block 16 of 260
genotype - block 17 of 260
genotype - block 18 of 260
genotype - block 19 of 260
genotype - block 20 of 260
genotype - block 21 of 260
genotype - block 22 of 260
genotype - block 23 of 260
genotype - block 24 of 260
genotype - block 25 of 260
genotype - block 26 of 260
genotype - block 27 of 260
genotype - block 28 of 260
genotype - block 29 of 260
genotype - block 30 of 260
genotype - block 31 of 260
genotype - block 32 of 260
genotype - block 33 of 260
genotype - block 34 of 260
genotype - block 35 of 260
genotype - block 36 of 260
genotype - block 37 of 260
genotype - block 38 of 260
genotype - block 39 of 260
genotype - block 40 of 260
genotype - block 41 of 260
genotype - block 42 of 260
genotype - block 43 of 260
genotype - block 44 of 260
genotype - block 45 of 260
genotype - block 46 of 260
genotype - block 47 of 260
genotype - block 48 of 260
genotype - block 49 of 260
genotype - block 50 of 260
genotype - block 51 of 260
genotype - block 52 of 260
genotype - block 53 of 260
genotype - block 54 of 260
genotype - block 55 of 260
genotype - block 56 of 260
genotype - block 57 of 260
genotype - block 58 of 260
genotype - block 59 of 260
genotype - block 60 of 260
genotype - block 61 of 260
genotype - block 62 of 260
genotype - block 63 of 260
genotype - block 64 of 260
genotype - block 65 of 260
genotype - block 66 of 260
genotype - block 67 of 260
genotype - block 68 of 260
genotype - block 69 of 260
genotype - block 70 of 260
genotype - block 71 of 260
genotype - block 72 of 260
genotype - block 73 of 260
genotype - block 74 of 260
genotype - block 75 of 260
genotype - block 76 of 260
genotype - block 77 of 260
genotype - block 78 of 260
genotype - block 79 of 260
genotype - block 80 of 260
genotype - block 81 of 260
genotype - block 82 of 260
genotype - block 83 of 260
genotype - block 84 of 260
genotype - block 85 of 260
genotype - block 86 of 260
genotype - block 87 of 260
genotype - block 88 of 260
genotype - block 89 of 260
genotype - block 90 of 260
genotype - block 91 of 260
genotype - block 92 of 260
genotype - block 93 of 260
genotype - block 94 of 260
genotype - block 95 of 260
genotype - block 96 of 260
genotype - block 97 of 260
genotype - block 98 of 260
genotype - block 99 of 260
genotype - block 100 of 260
genotype - block 101 of 260
genotype - block 102 of 260
genotype - block 103 of 260
genotype - block 104 of 260
genotype - block 105 of 260
genotype - block 106 of 260
genotype - block 107 of 260
genotype - block 108 of 260
genotype - block 109 of 260
genotype - block 110 of 260
genotype - block 111 of 260
genotype - block 112 of 260
genotype - block 113 of 260
genotype - block 114 of 260
genotype - block 115 of 260
genotype - block 116 of 260
genotype - block 117 of 260
genotype - block 118 of 260
genotype - block 119 of 260
genotype - block 120 of 260
genotype - block 121 of 260
genotype - block 122 of 260
genotype - block 123 of 260
genotype - block 124 of 260
genotype - block 125 of 260
genotype - block 126 of 260
genotype - block 127 of 260
genotype - block 128 of 260
genotype - block 129 of 260
genotype - block 130 of 260
genotype - block 131 of 260
genotype - block 132 of 260
genotype - block 133 of 260
genotype - block 134 of 260
genotype - block 135 of 260
genotype - block 136 of 260
genotype - block 137 of 260
genotype - block 138 of 260
genotype - block 139 of 260
genotype - block 140 of 260
genotype - block 141 of 260
genotype - block 142 of 260
genotype - block 143 of 260
genotype - block 144 of 260
genotype - block 145 of 260
genotype - block 146 of 260
genotype - block 147 of 260
genotype - block 148 of 260
genotype - block 149 of 260
genotype - block 150 of 260
genotype - block 151 of 260
genotype - block 152 of 260
genotype - block 153 of 260
genotype - block 154 of 260
genotype - block 155 of 260
genotype - block 156 of 260
genotype - block 157 of 260
genotype - block 158 of 260
genotype - block 159 of 260
genotype - block 160 of 260
genotype - block 161 of 260
genotype - block 162 of 260
genotype - block 163 of 260
genotype - block 164 of 260
genotype - block 165 of 260
genotype - block 166 of 260
genotype - block 167 of 260
genotype - block 168 of 260
genotype - block 169 of 260
genotype - block 170 of 260
genotype - block 171 of 260
genotype - block 172 of 260
genotype - block 173 of 260
genotype - block 174 of 260
genotype - block 175 of 260
genotype - block 176 of 260
genotype - block 177 of 260
genotype - block 178 of 260
genotype - block 179 of 260
genotype - block 180 of 260
genotype - block 181 of 260
genotype - block 182 of 260
genotype - block 183 of 260
genotype - block 184 of 260
genotype - block 185 of 260
genotype - block 186 of 260
genotype - block 187 of 260
genotype - block 188 of 260
genotype - block 189 of 260
genotype - block 190 of 260
genotype - block 191 of 260
genotype - block 192 of 260
genotype - block 193 of 260
genotype - block 194 of 260
genotype - block 195 of 260
genotype - block 196 of 260
genotype - block 197 of 260
genotype - block 198 of 260
genotype - block 199 of 260
genotype - block 200 of 260
genotype - block 201 of 260
genotype - block 202 of 260
genotype - block 203 of 260
genotype - block 204 of 260
genotype - block 205 of 260
genotype - block 206 of 260
genotype - block 207 of 260
genotype - block 208 of 260
genotype - block 209 of 260
genotype - block 210 of 260
genotype - block 211 of 260
genotype - block 212 of 260
genotype - block 213 of 260
genotype - block 214 of 260
genotype - block 215 of 260
genotype - block 216 of 260
genotype - block 217 of 260
genotype - block 218 of 260
genotype - block 219 of 260
genotype - block 220 of 260
genotype - block 221 of 260
genotype - block 222 of 260
genotype - block 223 of 260
genotype - block 224 of 260
genotype - block 225 of 260
genotype - block 226 of 260
genotype - block 227 of 260
genotype - block 228 of 260
genotype - block 229 of 260
genotype - block 230 of 260
genotype - block 231 of 260
genotype - block 232 of 260
genotype - block 233 of 260
genotype - block 234 of 260
genotype - block 235 of 260
genotype - block 236 of 260
genotype - block 237 of 260
genotype - block 238 of 260
genotype - block 239 of 260
genotype - block 240 of 260
genotype - block 241 of 260
genotype - block 242 of 260
genotype - block 243 of 260
genotype - block 244 of 260
genotype - block 245 of 260
genotype - block 246 of 260
genotype - block 247 of 260
genotype - block 248 of 260
genotype - block 249 of 260
genotype - block 250 of 260
genotype - block 251 of 260
genotype - block 252 of 260
genotype - block 253 of 260
genotype - block 254 of 260
genotype - block 255 of 260
genotype - block 256 of 260
genotype - block 257 of 260
genotype - block 258 of 260
genotype - block 259 of 260
genotype - block 260 of 260
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe22b6320b&#x27; (2.3K)
    # of fragments: 27
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe22b6320b.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe22b6320b.tmp&#x27; (2.2K, reduced: 168B)
    # of fragments: 13
working on genotype
genotype - looping over snps
All variables match.
working on genotype
genotype - block 1 of 1
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe3cbeea03&#x27; (3.2K)
    # of fragments: 27
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe3cbeea03.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe3cbeea03.tmp&#x27; (3.0K, reduced: 168B)
    # of fragments: 13
working on genotype
genotype - looping over snps
All variables match.
working on genotype
genotype - block 1 of 1
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe227e0f8d&#x27; (2.5K)
    # of fragments: 27
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe227e0f8d.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe227e0f8d.tmp&#x27; (2.3K, reduced: 168B)
    # of fragments: 13
working on genotype
genotype - looping over snps
All variables match.
working on genotype
genotype - block 1 of 11
genotype - block 2 of 11
genotype - block 3 of 11
genotype - block 4 of 11
genotype - block 5 of 11
genotype - block 6 of 11
genotype - block 7 of 11
genotype - block 8 of 11
genotype - block 9 of 11
genotype - block 10 of 11
genotype - block 11 of 11
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe105c9912&#x27; (2.4K)
    # of fragments: 27
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe105c9912.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe105c9912.tmp&#x27; (2.2K, reduced: 168B)
    # of fragments: 13
working on genotype
genotype - looping over snps
All variables match.
working on genotype
genotype - block 1 of 1
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafef9bacb0&#x27; (3.1K)
    # of fragments: 27
    save to &#x27;/tmp/Rtmpthm57M/file2ceafef9bacb0.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafef9bacb0.tmp&#x27; (3.0K, reduced: 168B)
    # of fragments: 13
working on genotype
genotype - looping over snps
All variables match.
working on genotype
genotype - block 1 of 1
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe7b27256&#x27; (2.5K)
    # of fragments: 27
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe7b27256.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe7b27256.tmp&#x27; (2.3K, reduced: 168B)
    # of fragments: 13
working on genotype
genotype - looping over snps
All variables match.
working on genotype
genotype - sample 10 of 10
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe112fb612&#x27; (2.4K)
    # of fragments: 27
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe112fb612.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe112fb612.tmp&#x27; (2.2K, reduced: 168B)
    # of fragments: 13
working on genotype
genotype - looping over samples
All variables match.
working on genotype1
genotype1 - sample 10 of 10
working on genotype2
genotype2 - sample 10 of 10
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe65d913a3&#x27; (2.6K)
    # of fragments: 32
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe65d913a3.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe65d913a3.tmp&#x27; (2.4K, reduced: 204B)
    # of fragments: 15
working on genotype1
genotype1 - looping over samples
working on genotype2
genotype2 - looping over samples
All variables match.
working on genotype1
genotype1 - sample 10 of 10
working on genotype2
genotype2 - sample 10 of 10
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafeb636d6b&#x27; (4.0K)
    # of fragments: 32
    save to &#x27;/tmp/Rtmpthm57M/file2ceafeb636d6b.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafeb636d6b.tmp&#x27; (3.8K, reduced: 204B)
    # of fragments: 15
working on genotype1
genotype1 - looping over samples
working on genotype2
genotype2 - looping over samples
All variables match.
working on genotype1
genotype1 - sample 10 of 20
genotype1 - sample 20 of 20
working on genotype2
genotype2 - sample 10 of 20
genotype2 - sample 20 of 20
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe4d4dffe2&#x27; (2.9K)
    # of fragments: 32
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe4d4dffe2.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe4d4dffe2.tmp&#x27; (2.7K, reduced: 204B)
    # of fragments: 15
working on genotype1
genotype1 - looping over samples
working on genotype2
genotype2 - looping over samples
All variables match.
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27; (2.5K, reduced: 723B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27; (2.5K, reduced: 723B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27; (2.5K, reduced: 723B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27; (2.5K, reduced: 723B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27; (2.5K, reduced: 723B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27; (2.5K, reduced: 723B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27; (2.5K, reduced: 727B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27; (2.5K, reduced: 727B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27; (2.5K, reduced: 727B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27; (2.5K, reduced: 727B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27; (2.5K, reduced: 727B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe6097aa63.tmp&#x27; (2.5K, reduced: 727B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe1000c31b&#x27; (2.6K)
    # of fragments: 31
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe1000c31b.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe1000c31b.tmp&#x27; (2.4K, reduced: 204B)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe1000c31b&#x27; (2.6K)
    # of fragments: 31
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe1000c31b.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe1000c31b.tmp&#x27; (2.4K, reduced: 204B)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe1000c31b&#x27; (2.6K)
    # of fragments: 31
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe1000c31b.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe1000c31b.tmp&#x27; (2.4K, reduced: 204B)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe1000c31b&#x27; (2.6K)
    # of fragments: 31
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe1000c31b.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe1000c31b.tmp&#x27; (2.4K, reduced: 204B)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe1000c31b&#x27; (2.6K)
    # of fragments: 31
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe1000c31b.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe1000c31b.tmp&#x27; (2.4K, reduced: 204B)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe1000c31b&#x27; (2.6K)
    # of fragments: 31
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe1000c31b.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe1000c31b.tmp&#x27; (2.4K, reduced: 204B)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27; (2.5K, reduced: 723B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27; (2.5K, reduced: 723B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27; (2.5K, reduced: 723B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27; (2.5K, reduced: 723B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27; (2.5K, reduced: 723B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27; (2.5K, reduced: 723B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27; (2.5K, reduced: 723B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27; (2.5K, reduced: 723B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27; (2.5K, reduced: 723B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27; (2.5K, reduced: 723B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27; (2.5K, reduced: 723B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978&#x27; (3.2K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe57b8a978.tmp&#x27; (2.5K, reduced: 723B)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a&#x27; (3.6K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a.tmp&#x27; (2.5K, reduced: 1.0K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a&#x27; (3.6K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a.tmp&#x27; (2.5K, reduced: 1.0K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a&#x27; (3.6K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a.tmp&#x27; (2.5K, reduced: 1.0K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a&#x27; (3.6K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a.tmp&#x27; (2.5K, reduced: 1.0K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a&#x27; (3.6K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a.tmp&#x27; (2.5K, reduced: 1.0K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a&#x27; (3.6K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a.tmp&#x27; (2.5K, reduced: 1.0K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 100
Block 2 of 100
Block 3 of 100
Block 4 of 100
Block 5 of 100
Block 6 of 100
Block 7 of 100
Block 8 of 100
Block 9 of 100
Block 10 of 100
Block 11 of 100
Block 12 of 100
Block 13 of 100
Block 14 of 100
Block 15 of 100
Block 16 of 100
Block 17 of 100
Block 18 of 100
Block 19 of 100
Block 20 of 100
Block 21 of 100
Block 22 of 100
Block 23 of 100
Block 24 of 100
Block 25 of 100
Block 26 of 100
Block 27 of 100
Block 28 of 100
Block 29 of 100
Block 30 of 100
Block 31 of 100
Block 32 of 100
Block 33 of 100
Block 34 of 100
Block 35 of 100
Block 36 of 100
Block 37 of 100
Block 38 of 100
Block 39 of 100
Block 40 of 100
Block 41 of 100
Block 42 of 100
Block 43 of 100
Block 44 of 100
Block 45 of 100
Block 46 of 100
Block 47 of 100
Block 48 of 100
Block 49 of 100
Block 50 of 100
Block 51 of 100
Block 52 of 100
Block 53 of 100
Block 54 of 100
Block 55 of 100
Block 56 of 100
Block 57 of 100
Block 58 of 100
Block 59 of 100
Block 60 of 100
Block 61 of 100
Block 62 of 100
Block 63 of 100
Block 64 of 100
Block 65 of 100
Block 66 of 100
Block 67 of 100
Block 68 of 100
Block 69 of 100
Block 70 of 100
Block 71 of 100
Block 72 of 100
Block 73 of 100
Block 74 of 100
Block 75 of 100
Block 76 of 100
Block 77 of 100
Block 78 of 100
Block 79 of 100
Block 80 of 100
Block 81 of 100
Block 82 of 100
Block 83 of 100
Block 84 of 100
Block 85 of 100
Block 86 of 100
Block 87 of 100
Block 88 of 100
Block 89 of 100
Block 90 of 100
Block 91 of 100
Block 92 of 100
Block 93 of 100
Block 94 of 100
Block 95 of 100
Block 96 of 100
Block 97 of 100
Block 98 of 100
Block 99 of 100
Block 100 of 100
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a&#x27; (3.6K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a.tmp&#x27; (2.5K, reduced: 1.0K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 100
Block 2 of 100
Block 3 of 100
Block 4 of 100
Block 5 of 100
Block 6 of 100
Block 7 of 100
Block 8 of 100
Block 9 of 100
Block 10 of 100
Block 11 of 100
Block 12 of 100
Block 13 of 100
Block 14 of 100
Block 15 of 100
Block 16 of 100
Block 17 of 100
Block 18 of 100
Block 19 of 100
Block 20 of 100
Block 21 of 100
Block 22 of 100
Block 23 of 100
Block 24 of 100
Block 25 of 100
Block 26 of 100
Block 27 of 100
Block 28 of 100
Block 29 of 100
Block 30 of 100
Block 31 of 100
Block 32 of 100
Block 33 of 100
Block 34 of 100
Block 35 of 100
Block 36 of 100
Block 37 of 100
Block 38 of 100
Block 39 of 100
Block 40 of 100
Block 41 of 100
Block 42 of 100
Block 43 of 100
Block 44 of 100
Block 45 of 100
Block 46 of 100
Block 47 of 100
Block 48 of 100
Block 49 of 100
Block 50 of 100
Block 51 of 100
Block 52 of 100
Block 53 of 100
Block 54 of 100
Block 55 of 100
Block 56 of 100
Block 57 of 100
Block 58 of 100
Block 59 of 100
Block 60 of 100
Block 61 of 100
Block 62 of 100
Block 63 of 100
Block 64 of 100
Block 65 of 100
Block 66 of 100
Block 67 of 100
Block 68 of 100
Block 69 of 100
Block 70 of 100
Block 71 of 100
Block 72 of 100
Block 73 of 100
Block 74 of 100
Block 75 of 100
Block 76 of 100
Block 77 of 100
Block 78 of 100
Block 79 of 100
Block 80 of 100
Block 81 of 100
Block 82 of 100
Block 83 of 100
Block 84 of 100
Block 85 of 100
Block 86 of 100
Block 87 of 100
Block 88 of 100
Block 89 of 100
Block 90 of 100
Block 91 of 100
Block 92 of 100
Block 93 of 100
Block 94 of 100
Block 95 of 100
Block 96 of 100
Block 97 of 100
Block 98 of 100
Block 99 of 100
Block 100 of 100
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a&#x27; (3.6K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe2b410a2a.tmp&#x27; (2.5K, reduced: 1.0K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading sample file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe587c9a40&#x27; (34.7K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe587c9a40.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe587c9a40.tmp&#x27; (4.9K, reduced: 29.8K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading sample file...
Reading genotype file...
Block 1 of 4
Block 2 of 4
Block 3 of 4
Block 4 of 4
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe587c9a40&#x27; (34.7K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe587c9a40.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe587c9a40.tmp&#x27; (4.9K, reduced: 29.8K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading sample file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe587c9a40&#x27; (34.7K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe587c9a40.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe587c9a40.tmp&#x27; (4.9K, reduced: 29.8K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading sample file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe587c9a40&#x27; (34.7K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe587c9a40.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe587c9a40.tmp&#x27; (4.9K, reduced: 29.8K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading sample file...
Reading genotype file...
Block 1 of 4
Block 2 of 4
Block 3 of 4
Block 4 of 4
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe587c9a40&#x27; (34.7K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe587c9a40.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe587c9a40.tmp&#x27; (4.9K, reduced: 29.8K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading sample file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe587c9a40&#x27; (34.7K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe587c9a40.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe587c9a40.tmp&#x27; (4.9K, reduced: 29.8K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading sample file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe7f0cbe9c&#x27; (4.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe7f0cbe9c.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe7f0cbe9c.tmp&#x27; (3.5K, reduced: 1010B)
    # of fragments: 14
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading sample file...
Reading genotype file...
Block 1 of 4
Block 2 of 4
Block 3 of 4
Block 4 of 4
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe7f0cbe9c&#x27; (4.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe7f0cbe9c.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe7f0cbe9c.tmp&#x27; (3.5K, reduced: 1010B)
    # of fragments: 14
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 4
Block 2 of 4
Block 3 of 4
Block 4 of 4
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading sample file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe7f0cbe9c&#x27; (4.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe7f0cbe9c.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe7f0cbe9c.tmp&#x27; (3.5K, reduced: 1010B)
    # of fragments: 14
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading sample file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe7f0cbe9c&#x27; (4.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe7f0cbe9c.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe7f0cbe9c.tmp&#x27; (3.5K, reduced: 1.0K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading sample file...
Reading genotype file...
Block 1 of 4
Block 2 of 4
Block 3 of 4
Block 4 of 4
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe7f0cbe9c&#x27; (4.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe7f0cbe9c.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe7f0cbe9c.tmp&#x27; (3.5K, reduced: 1.0K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 4
Block 2 of 4
Block 3 of 4
Block 4 of 4
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading sample file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe7f0cbe9c&#x27; (4.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe7f0cbe9c.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe7f0cbe9c.tmp&#x27; (3.5K, reduced: 1.0K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading sample file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe54a7467f&#x27; (34.7K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe54a7467f.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe54a7467f.tmp&#x27; (4.9K, reduced: 29.8K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading sample file...
Reading genotype file...
Block 1 of 33
Block 2 of 33
Block 3 of 33
Block 4 of 33
Block 5 of 33
Block 6 of 33
Block 7 of 33
Block 8 of 33
Block 9 of 33
Block 10 of 33
Block 11 of 33
Block 12 of 33
Block 13 of 33
Block 14 of 33
Block 15 of 33
Block 16 of 33
Block 17 of 33
Block 18 of 33
Block 19 of 33
Block 20 of 33
Block 21 of 33
Block 22 of 33
Block 23 of 33
Block 24 of 33
Block 25 of 33
Block 26 of 33
Block 27 of 33
Block 28 of 33
Block 29 of 33
Block 30 of 33
Block 31 of 33
Block 32 of 33
Block 33 of 33
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe54a7467f&#x27; (34.7K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe54a7467f.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe54a7467f.tmp&#x27; (4.9K, reduced: 29.8K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading sample file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading sample file...
Reading genotype file...
Block 1 of 4
Block 2 of 4
Block 3 of 4
Block 4 of 4
Writing annotation...
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading sample file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading sample file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe641ede53&#x27; (34.7K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe641ede53.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe641ede53.tmp&#x27; (4.9K, reduced: 29.8K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe31d63728&#x27; (29.3K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe31d63728.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe31d63728.tmp&#x27; (4.2K, reduced: 25.1K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 4
Block 2 of 4
Block 3 of 4
Block 4 of 4
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe31d63728&#x27; (29.3K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe31d63728.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe31d63728.tmp&#x27; (4.2K, reduced: 25.1K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe31d63728&#x27; (29.3K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe31d63728.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe31d63728.tmp&#x27; (4.2K, reduced: 25.1K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 33
Block 2 of 33
Block 3 of 33
Block 4 of 33
Block 5 of 33
Block 6 of 33
Block 7 of 33
Block 8 of 33
Block 9 of 33
Block 10 of 33
Block 11 of 33
Block 12 of 33
Block 13 of 33
Block 14 of 33
Block 15 of 33
Block 16 of 33
Block 17 of 33
Block 18 of 33
Block 19 of 33
Block 20 of 33
Block 21 of 33
Block 22 of 33
Block 23 of 33
Block 24 of 33
Block 25 of 33
Block 26 of 33
Block 27 of 33
Block 28 of 33
Block 29 of 33
Block 30 of 33
Block 31 of 33
Block 32 of 33
Block 33 of 33
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe31d63728&#x27; (29.3K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe31d63728.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe31d63728.tmp&#x27; (4.2K, reduced: 25.1K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae&#x27; (350.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27; (87.7K, reduced: 262.8K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae&#x27; (350.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27; (86.3K, reduced: 264.2K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae&#x27; (350.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27; (87.7K, reduced: 262.8K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae&#x27; (350.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27; (86.3K, reduced: 264.2K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae&#x27; (350.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27; (87.7K, reduced: 262.8K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae&#x27; (350.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27; (86.3K, reduced: 264.2K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae&#x27; (350.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27; (91.1K, reduced: 259.3K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae&#x27; (350.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27; (90.6K, reduced: 259.9K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae&#x27; (350.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27; (91.1K, reduced: 259.3K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae&#x27; (350.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27; (90.6K, reduced: 259.9K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae&#x27; (350.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27; (91.1K, reduced: 259.3K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae&#x27; (350.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafebaee5ae.tmp&#x27; (90.6K, reduced: 259.9K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe2a5cb136&#x27; (24.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe2a5cb136.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe2a5cb136.tmp&#x27; (10.9K, reduced: 13.6K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe2a5cb136&#x27; (24.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe2a5cb136.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe2a5cb136.tmp&#x27; (10.9K, reduced: 13.6K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe2a5cb136&#x27; (24.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe2a5cb136.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe2a5cb136.tmp&#x27; (10.9K, reduced: 13.6K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe2a5cb136&#x27; (24.6K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe2a5cb136.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe2a5cb136.tmp&#x27; (10.2K, reduced: 14.3K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe2a5cb136&#x27; (24.6K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe2a5cb136.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe2a5cb136.tmp&#x27; (10.2K, reduced: 14.3K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe2a5cb136&#x27; (24.6K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe2a5cb136.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe2a5cb136.tmp&#x27; (10.2K, reduced: 14.3K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe2641909f&#x27; (350.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe2641909f.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe2641909f.tmp&#x27; (87.7K, reduced: 262.8K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe2641909f&#x27; (350.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe2641909f.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe2641909f.tmp&#x27; (86.3K, reduced: 264.2K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 500
Block 2 of 500
Block 3 of 500
Block 4 of 500
Block 5 of 500
Block 6 of 500
Block 7 of 500
Block 8 of 500
Block 9 of 500
Block 10 of 500
Block 11 of 500
Block 12 of 500
Block 13 of 500
Block 14 of 500
Block 15 of 500
Block 16 of 500
Block 17 of 500
Block 18 of 500
Block 19 of 500
Block 20 of 500
Block 21 of 500
Block 22 of 500
Block 23 of 500
Block 24 of 500
Block 25 of 500
Block 26 of 500
Block 27 of 500
Block 28 of 500
Block 29 of 500
Block 30 of 500
Block 31 of 500
Block 32 of 500
Block 33 of 500
Block 34 of 500
Block 35 of 500
Block 36 of 500
Block 37 of 500
Block 38 of 500
Block 39 of 500
Block 40 of 500
Block 41 of 500
Block 42 of 500
Block 43 of 500
Block 44 of 500
Block 45 of 500
Block 46 of 500
Block 47 of 500
Block 48 of 500
Block 49 of 500
Block 50 of 500
Block 51 of 500
Block 52 of 500
Block 53 of 500
Block 54 of 500
Block 55 of 500
Block 56 of 500
Block 57 of 500
Block 58 of 500
Block 59 of 500
Block 60 of 500
Block 61 of 500
Block 62 of 500
Block 63 of 500
Block 64 of 500
Block 65 of 500
Block 66 of 500
Block 67 of 500
Block 68 of 500
Block 69 of 500
Block 70 of 500
Block 71 of 500
Block 72 of 500
Block 73 of 500
Block 74 of 500
Block 75 of 500
Block 76 of 500
Block 77 of 500
Block 78 of 500
Block 79 of 500
Block 80 of 500
Block 81 of 500
Block 82 of 500
Block 83 of 500
Block 84 of 500
Block 85 of 500
Block 86 of 500
Block 87 of 500
Block 88 of 500
Block 89 of 500
Block 90 of 500
Block 91 of 500
Block 92 of 500
Block 93 of 500
Block 94 of 500
Block 95 of 500
Block 96 of 500
Block 97 of 500
Block 98 of 500
Block 99 of 500
Block 100 of 500
Block 101 of 500
Block 102 of 500
Block 103 of 500
Block 104 of 500
Block 105 of 500
Block 106 of 500
Block 107 of 500
Block 108 of 500
Block 109 of 500
Block 110 of 500
Block 111 of 500
Block 112 of 500
Block 113 of 500
Block 114 of 500
Block 115 of 500
Block 116 of 500
Block 117 of 500
Block 118 of 500
Block 119 of 500
Block 120 of 500
Block 121 of 500
Block 122 of 500
Block 123 of 500
Block 124 of 500
Block 125 of 500
Block 126 of 500
Block 127 of 500
Block 128 of 500
Block 129 of 500
Block 130 of 500
Block 131 of 500
Block 132 of 500
Block 133 of 500
Block 134 of 500
Block 135 of 500
Block 136 of 500
Block 137 of 500
Block 138 of 500
Block 139 of 500
Block 140 of 500
Block 141 of 500
Block 142 of 500
Block 143 of 500
Block 144 of 500
Block 145 of 500
Block 146 of 500
Block 147 of 500
Block 148 of 500
Block 149 of 500
Block 150 of 500
Block 151 of 500
Block 152 of 500
Block 153 of 500
Block 154 of 500
Block 155 of 500
Block 156 of 500
Block 157 of 500
Block 158 of 500
Block 159 of 500
Block 160 of 500
Block 161 of 500
Block 162 of 500
Block 163 of 500
Block 164 of 500
Block 165 of 500
Block 166 of 500
Block 167 of 500
Block 168 of 500
Block 169 of 500
Block 170 of 500
Block 171 of 500
Block 172 of 500
Block 173 of 500
Block 174 of 500
Block 175 of 500
Block 176 of 500
Block 177 of 500
Block 178 of 500
Block 179 of 500
Block 180 of 500
Block 181 of 500
Block 182 of 500
Block 183 of 500
Block 184 of 500
Block 185 of 500
Block 186 of 500
Block 187 of 500
Block 188 of 500
Block 189 of 500
Block 190 of 500
Block 191 of 500
Block 192 of 500
Block 193 of 500
Block 194 of 500
Block 195 of 500
Block 196 of 500
Block 197 of 500
Block 198 of 500
Block 199 of 500
Block 200 of 500
Block 201 of 500
Block 202 of 500
Block 203 of 500
Block 204 of 500
Block 205 of 500
Block 206 of 500
Block 207 of 500
Block 208 of 500
Block 209 of 500
Block 210 of 500
Block 211 of 500
Block 212 of 500
Block 213 of 500
Block 214 of 500
Block 215 of 500
Block 216 of 500
Block 217 of 500
Block 218 of 500
Block 219 of 500
Block 220 of 500
Block 221 of 500
Block 222 of 500
Block 223 of 500
Block 224 of 500
Block 225 of 500
Block 226 of 500
Block 227 of 500
Block 228 of 500
Block 229 of 500
Block 230 of 500
Block 231 of 500
Block 232 of 500
Block 233 of 500
Block 234 of 500
Block 235 of 500
Block 236 of 500
Block 237 of 500
Block 238 of 500
Block 239 of 500
Block 240 of 500
Block 241 of 500
Block 242 of 500
Block 243 of 500
Block 244 of 500
Block 245 of 500
Block 246 of 500
Block 247 of 500
Block 248 of 500
Block 249 of 500
Block 250 of 500
Block 251 of 500
Block 252 of 500
Block 253 of 500
Block 254 of 500
Block 255 of 500
Block 256 of 500
Block 257 of 500
Block 258 of 500
Block 259 of 500
Block 260 of 500
Block 261 of 500
Block 262 of 500
Block 263 of 500
Block 264 of 500
Block 265 of 500
Block 266 of 500
Block 267 of 500
Block 268 of 500
Block 269 of 500
Block 270 of 500
Block 271 of 500
Block 272 of 500
Block 273 of 500
Block 274 of 500
Block 275 of 500
Block 276 of 500
Block 277 of 500
Block 278 of 500
Block 279 of 500
Block 280 of 500
Block 281 of 500
Block 282 of 500
Block 283 of 500
Block 284 of 500
Block 285 of 500
Block 286 of 500
Block 287 of 500
Block 288 of 500
Block 289 of 500
Block 290 of 500
Block 291 of 500
Block 292 of 500
Block 293 of 500
Block 294 of 500
Block 295 of 500
Block 296 of 500
Block 297 of 500
Block 298 of 500
Block 299 of 500
Block 300 of 500
Block 301 of 500
Block 302 of 500
Block 303 of 500
Block 304 of 500
Block 305 of 500
Block 306 of 500
Block 307 of 500
Block 308 of 500
Block 309 of 500
Block 310 of 500
Block 311 of 500
Block 312 of 500
Block 313 of 500
Block 314 of 500
Block 315 of 500
Block 316 of 500
Block 317 of 500
Block 318 of 500
Block 319 of 500
Block 320 of 500
Block 321 of 500
Block 322 of 500
Block 323 of 500
Block 324 of 500
Block 325 of 500
Block 326 of 500
Block 327 of 500
Block 328 of 500
Block 329 of 500
Block 330 of 500
Block 331 of 500
Block 332 of 500
Block 333 of 500
Block 334 of 500
Block 335 of 500
Block 336 of 500
Block 337 of 500
Block 338 of 500
Block 339 of 500
Block 340 of 500
Block 341 of 500
Block 342 of 500
Block 343 of 500
Block 344 of 500
Block 345 of 500
Block 346 of 500
Block 347 of 500
Block 348 of 500
Block 349 of 500
Block 350 of 500
Block 351 of 500
Block 352 of 500
Block 353 of 500
Block 354 of 500
Block 355 of 500
Block 356 of 500
Block 357 of 500
Block 358 of 500
Block 359 of 500
Block 360 of 500
Block 361 of 500
Block 362 of 500
Block 363 of 500
Block 364 of 500
Block 365 of 500
Block 366 of 500
Block 367 of 500
Block 368 of 500
Block 369 of 500
Block 370 of 500
Block 371 of 500
Block 372 of 500
Block 373 of 500
Block 374 of 500
Block 375 of 500
Block 376 of 500
Block 377 of 500
Block 378 of 500
Block 379 of 500
Block 380 of 500
Block 381 of 500
Block 382 of 500
Block 383 of 500
Block 384 of 500
Block 385 of 500
Block 386 of 500
Block 387 of 500
Block 388 of 500
Block 389 of 500
Block 390 of 500
Block 391 of 500
Block 392 of 500
Block 393 of 500
Block 394 of 500
Block 395 of 500
Block 396 of 500
Block 397 of 500
Block 398 of 500
Block 399 of 500
Block 400 of 500
Block 401 of 500
Block 402 of 500
Block 403 of 500
Block 404 of 500
Block 405 of 500
Block 406 of 500
Block 407 of 500
Block 408 of 500
Block 409 of 500
Block 410 of 500
Block 411 of 500
Block 412 of 500
Block 413 of 500
Block 414 of 500
Block 415 of 500
Block 416 of 500
Block 417 of 500
Block 418 of 500
Block 419 of 500
Block 420 of 500
Block 421 of 500
Block 422 of 500
Block 423 of 500
Block 424 of 500
Block 425 of 500
Block 426 of 500
Block 427 of 500
Block 428 of 500
Block 429 of 500
Block 430 of 500
Block 431 of 500
Block 432 of 500
Block 433 of 500
Block 434 of 500
Block 435 of 500
Block 436 of 500
Block 437 of 500
Block 438 of 500
Block 439 of 500
Block 440 of 500
Block 441 of 500
Block 442 of 500
Block 443 of 500
Block 444 of 500
Block 445 of 500
Block 446 of 500
Block 447 of 500
Block 448 of 500
Block 449 of 500
Block 450 of 500
Block 451 of 500
Block 452 of 500
Block 453 of 500
Block 454 of 500
Block 455 of 500
Block 456 of 500
Block 457 of 500
Block 458 of 500
Block 459 of 500
Block 460 of 500
Block 461 of 500
Block 462 of 500
Block 463 of 500
Block 464 of 500
Block 465 of 500
Block 466 of 500
Block 467 of 500
Block 468 of 500
Block 469 of 500
Block 470 of 500
Block 471 of 500
Block 472 of 500
Block 473 of 500
Block 474 of 500
Block 475 of 500
Block 476 of 500
Block 477 of 500
Block 478 of 500
Block 479 of 500
Block 480 of 500
Block 481 of 500
Block 482 of 500
Block 483 of 500
Block 484 of 500
Block 485 of 500
Block 486 of 500
Block 487 of 500
Block 488 of 500
Block 489 of 500
Block 490 of 500
Block 491 of 500
Block 492 of 500
Block 493 of 500
Block 494 of 500
Block 495 of 500
Block 496 of 500
Block 497 of 500
Block 498 of 500
Block 499 of 500
Block 500 of 500
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe2641909f&#x27; (350.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe2641909f.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe2641909f.tmp&#x27; (87.7K, reduced: 262.8K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 500
Block 2 of 500
Block 3 of 500
Block 4 of 500
Block 5 of 500
Block 6 of 500
Block 7 of 500
Block 8 of 500
Block 9 of 500
Block 10 of 500
Block 11 of 500
Block 12 of 500
Block 13 of 500
Block 14 of 500
Block 15 of 500
Block 16 of 500
Block 17 of 500
Block 18 of 500
Block 19 of 500
Block 20 of 500
Block 21 of 500
Block 22 of 500
Block 23 of 500
Block 24 of 500
Block 25 of 500
Block 26 of 500
Block 27 of 500
Block 28 of 500
Block 29 of 500
Block 30 of 500
Block 31 of 500
Block 32 of 500
Block 33 of 500
Block 34 of 500
Block 35 of 500
Block 36 of 500
Block 37 of 500
Block 38 of 500
Block 39 of 500
Block 40 of 500
Block 41 of 500
Block 42 of 500
Block 43 of 500
Block 44 of 500
Block 45 of 500
Block 46 of 500
Block 47 of 500
Block 48 of 500
Block 49 of 500
Block 50 of 500
Block 51 of 500
Block 52 of 500
Block 53 of 500
Block 54 of 500
Block 55 of 500
Block 56 of 500
Block 57 of 500
Block 58 of 500
Block 59 of 500
Block 60 of 500
Block 61 of 500
Block 62 of 500
Block 63 of 500
Block 64 of 500
Block 65 of 500
Block 66 of 500
Block 67 of 500
Block 68 of 500
Block 69 of 500
Block 70 of 500
Block 71 of 500
Block 72 of 500
Block 73 of 500
Block 74 of 500
Block 75 of 500
Block 76 of 500
Block 77 of 500
Block 78 of 500
Block 79 of 500
Block 80 of 500
Block 81 of 500
Block 82 of 500
Block 83 of 500
Block 84 of 500
Block 85 of 500
Block 86 of 500
Block 87 of 500
Block 88 of 500
Block 89 of 500
Block 90 of 500
Block 91 of 500
Block 92 of 500
Block 93 of 500
Block 94 of 500
Block 95 of 500
Block 96 of 500
Block 97 of 500
Block 98 of 500
Block 99 of 500
Block 100 of 500
Block 101 of 500
Block 102 of 500
Block 103 of 500
Block 104 of 500
Block 105 of 500
Block 106 of 500
Block 107 of 500
Block 108 of 500
Block 109 of 500
Block 110 of 500
Block 111 of 500
Block 112 of 500
Block 113 of 500
Block 114 of 500
Block 115 of 500
Block 116 of 500
Block 117 of 500
Block 118 of 500
Block 119 of 500
Block 120 of 500
Block 121 of 500
Block 122 of 500
Block 123 of 500
Block 124 of 500
Block 125 of 500
Block 126 of 500
Block 127 of 500
Block 128 of 500
Block 129 of 500
Block 130 of 500
Block 131 of 500
Block 132 of 500
Block 133 of 500
Block 134 of 500
Block 135 of 500
Block 136 of 500
Block 137 of 500
Block 138 of 500
Block 139 of 500
Block 140 of 500
Block 141 of 500
Block 142 of 500
Block 143 of 500
Block 144 of 500
Block 145 of 500
Block 146 of 500
Block 147 of 500
Block 148 of 500
Block 149 of 500
Block 150 of 500
Block 151 of 500
Block 152 of 500
Block 153 of 500
Block 154 of 500
Block 155 of 500
Block 156 of 500
Block 157 of 500
Block 158 of 500
Block 159 of 500
Block 160 of 500
Block 161 of 500
Block 162 of 500
Block 163 of 500
Block 164 of 500
Block 165 of 500
Block 166 of 500
Block 167 of 500
Block 168 of 500
Block 169 of 500
Block 170 of 500
Block 171 of 500
Block 172 of 500
Block 173 of 500
Block 174 of 500
Block 175 of 500
Block 176 of 500
Block 177 of 500
Block 178 of 500
Block 179 of 500
Block 180 of 500
Block 181 of 500
Block 182 of 500
Block 183 of 500
Block 184 of 500
Block 185 of 500
Block 186 of 500
Block 187 of 500
Block 188 of 500
Block 189 of 500
Block 190 of 500
Block 191 of 500
Block 192 of 500
Block 193 of 500
Block 194 of 500
Block 195 of 500
Block 196 of 500
Block 197 of 500
Block 198 of 500
Block 199 of 500
Block 200 of 500
Block 201 of 500
Block 202 of 500
Block 203 of 500
Block 204 of 500
Block 205 of 500
Block 206 of 500
Block 207 of 500
Block 208 of 500
Block 209 of 500
Block 210 of 500
Block 211 of 500
Block 212 of 500
Block 213 of 500
Block 214 of 500
Block 215 of 500
Block 216 of 500
Block 217 of 500
Block 218 of 500
Block 219 of 500
Block 220 of 500
Block 221 of 500
Block 222 of 500
Block 223 of 500
Block 224 of 500
Block 225 of 500
Block 226 of 500
Block 227 of 500
Block 228 of 500
Block 229 of 500
Block 230 of 500
Block 231 of 500
Block 232 of 500
Block 233 of 500
Block 234 of 500
Block 235 of 500
Block 236 of 500
Block 237 of 500
Block 238 of 500
Block 239 of 500
Block 240 of 500
Block 241 of 500
Block 242 of 500
Block 243 of 500
Block 244 of 500
Block 245 of 500
Block 246 of 500
Block 247 of 500
Block 248 of 500
Block 249 of 500
Block 250 of 500
Block 251 of 500
Block 252 of 500
Block 253 of 500
Block 254 of 500
Block 255 of 500
Block 256 of 500
Block 257 of 500
Block 258 of 500
Block 259 of 500
Block 260 of 500
Block 261 of 500
Block 262 of 500
Block 263 of 500
Block 264 of 500
Block 265 of 500
Block 266 of 500
Block 267 of 500
Block 268 of 500
Block 269 of 500
Block 270 of 500
Block 271 of 500
Block 272 of 500
Block 273 of 500
Block 274 of 500
Block 275 of 500
Block 276 of 500
Block 277 of 500
Block 278 of 500
Block 279 of 500
Block 280 of 500
Block 281 of 500
Block 282 of 500
Block 283 of 500
Block 284 of 500
Block 285 of 500
Block 286 of 500
Block 287 of 500
Block 288 of 500
Block 289 of 500
Block 290 of 500
Block 291 of 500
Block 292 of 500
Block 293 of 500
Block 294 of 500
Block 295 of 500
Block 296 of 500
Block 297 of 500
Block 298 of 500
Block 299 of 500
Block 300 of 500
Block 301 of 500
Block 302 of 500
Block 303 of 500
Block 304 of 500
Block 305 of 500
Block 306 of 500
Block 307 of 500
Block 308 of 500
Block 309 of 500
Block 310 of 500
Block 311 of 500
Block 312 of 500
Block 313 of 500
Block 314 of 500
Block 315 of 500
Block 316 of 500
Block 317 of 500
Block 318 of 500
Block 319 of 500
Block 320 of 500
Block 321 of 500
Block 322 of 500
Block 323 of 500
Block 324 of 500
Block 325 of 500
Block 326 of 500
Block 327 of 500
Block 328 of 500
Block 329 of 500
Block 330 of 500
Block 331 of 500
Block 332 of 500
Block 333 of 500
Block 334 of 500
Block 335 of 500
Block 336 of 500
Block 337 of 500
Block 338 of 500
Block 339 of 500
Block 340 of 500
Block 341 of 500
Block 342 of 500
Block 343 of 500
Block 344 of 500
Block 345 of 500
Block 346 of 500
Block 347 of 500
Block 348 of 500
Block 349 of 500
Block 350 of 500
Block 351 of 500
Block 352 of 500
Block 353 of 500
Block 354 of 500
Block 355 of 500
Block 356 of 500
Block 357 of 500
Block 358 of 500
Block 359 of 500
Block 360 of 500
Block 361 of 500
Block 362 of 500
Block 363 of 500
Block 364 of 500
Block 365 of 500
Block 366 of 500
Block 367 of 500
Block 368 of 500
Block 369 of 500
Block 370 of 500
Block 371 of 500
Block 372 of 500
Block 373 of 500
Block 374 of 500
Block 375 of 500
Block 376 of 500
Block 377 of 500
Block 378 of 500
Block 379 of 500
Block 380 of 500
Block 381 of 500
Block 382 of 500
Block 383 of 500
Block 384 of 500
Block 385 of 500
Block 386 of 500
Block 387 of 500
Block 388 of 500
Block 389 of 500
Block 390 of 500
Block 391 of 500
Block 392 of 500
Block 393 of 500
Block 394 of 500
Block 395 of 500
Block 396 of 500
Block 397 of 500
Block 398 of 500
Block 399 of 500
Block 400 of 500
Block 401 of 500
Block 402 of 500
Block 403 of 500
Block 404 of 500
Block 405 of 500
Block 406 of 500
Block 407 of 500
Block 408 of 500
Block 409 of 500
Block 410 of 500
Block 411 of 500
Block 412 of 500
Block 413 of 500
Block 414 of 500
Block 415 of 500
Block 416 of 500
Block 417 of 500
Block 418 of 500
Block 419 of 500
Block 420 of 500
Block 421 of 500
Block 422 of 500
Block 423 of 500
Block 424 of 500
Block 425 of 500
Block 426 of 500
Block 427 of 500
Block 428 of 500
Block 429 of 500
Block 430 of 500
Block 431 of 500
Block 432 of 500
Block 433 of 500
Block 434 of 500
Block 435 of 500
Block 436 of 500
Block 437 of 500
Block 438 of 500
Block 439 of 500
Block 440 of 500
Block 441 of 500
Block 442 of 500
Block 443 of 500
Block 444 of 500
Block 445 of 500
Block 446 of 500
Block 447 of 500
Block 448 of 500
Block 449 of 500
Block 450 of 500
Block 451 of 500
Block 452 of 500
Block 453 of 500
Block 454 of 500
Block 455 of 500
Block 456 of 500
Block 457 of 500
Block 458 of 500
Block 459 of 500
Block 460 of 500
Block 461 of 500
Block 462 of 500
Block 463 of 500
Block 464 of 500
Block 465 of 500
Block 466 of 500
Block 467 of 500
Block 468 of 500
Block 469 of 500
Block 470 of 500
Block 471 of 500
Block 472 of 500
Block 473 of 500
Block 474 of 500
Block 475 of 500
Block 476 of 500
Block 477 of 500
Block 478 of 500
Block 479 of 500
Block 480 of 500
Block 481 of 500
Block 482 of 500
Block 483 of 500
Block 484 of 500
Block 485 of 500
Block 486 of 500
Block 487 of 500
Block 488 of 500
Block 489 of 500
Block 490 of 500
Block 491 of 500
Block 492 of 500
Block 493 of 500
Block 494 of 500
Block 495 of 500
Block 496 of 500
Block 497 of 500
Block 498 of 500
Block 499 of 500
Block 500 of 500
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe2641909f&#x27; (350.5K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe2641909f.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe2641909f.tmp&#x27; (86.3K, reduced: 264.2K)
    # of fragments: 14
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 1
Writing annotation...
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 1
Writing annotation...
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Writing annotation...
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Determining number of SNPs and samples...
Including all SNPs.
scan.df not given. Assigning scanIDs automatically.
Reading SNP files...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Writing annotation...
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818&#x27; (338.0K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818.tmp&#x27; (85.1K, reduced: 252.9K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818&#x27; (338.0K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818.tmp&#x27; (84.0K, reduced: 254.0K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 3
Block 1 of 3
Block 2 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818&#x27; (338.0K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818.tmp&#x27; (85.1K, reduced: 252.9K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 3
Block 1 of 3
Block 2 of 3
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818&#x27; (338.0K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818.tmp&#x27; (84.0K, reduced: 254.0K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 2
Block 1 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818&#x27; (338.0K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818.tmp&#x27; (85.1K, reduced: 252.9K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 2
Block 1 of 2
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818&#x27; (338.0K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818.tmp&#x27; (84.0K, reduced: 254.0K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 500
Block 2 of 500
Block 3 of 500
Block 4 of 500
Block 5 of 500
Block 6 of 500
Block 7 of 500
Block 8 of 500
Block 9 of 500
Block 10 of 500
Block 11 of 500
Block 12 of 500
Block 13 of 500
Block 14 of 500
Block 15 of 500
Block 16 of 500
Block 17 of 500
Block 18 of 500
Block 19 of 500
Block 20 of 500
Block 21 of 500
Block 22 of 500
Block 23 of 500
Block 24 of 500
Block 25 of 500
Block 26 of 500
Block 27 of 500
Block 28 of 500
Block 29 of 500
Block 30 of 500
Block 31 of 500
Block 32 of 500
Block 33 of 500
Block 34 of 500
Block 35 of 500
Block 36 of 500
Block 37 of 500
Block 38 of 500
Block 39 of 500
Block 40 of 500
Block 41 of 500
Block 42 of 500
Block 43 of 500
Block 44 of 500
Block 45 of 500
Block 46 of 500
Block 47 of 500
Block 48 of 500
Block 49 of 500
Block 50 of 500
Block 51 of 500
Block 52 of 500
Block 53 of 500
Block 54 of 500
Block 55 of 500
Block 56 of 500
Block 57 of 500
Block 58 of 500
Block 58 of 500
Block 59 of 500
Block 60 of 500
Block 61 of 500
Block 62 of 500
Block 63 of 500
Block 64 of 500
Block 65 of 500
Block 66 of 500
Block 67 of 500
Block 68 of 500
Block 69 of 500
Block 70 of 500
Block 71 of 500
Block 72 of 500
Block 73 of 500
Block 74 of 500
Block 75 of 500
Block 76 of 500
Block 77 of 500
Block 78 of 500
Block 79 of 500
Block 80 of 500
Block 81 of 500
Block 82 of 500
Block 83 of 500
Block 84 of 500
Block 85 of 500
Block 86 of 500
Block 87 of 500
Block 88 of 500
Block 89 of 500
Block 90 of 500
Block 91 of 500
Block 92 of 500
Block 93 of 500
Block 94 of 500
Block 95 of 500
Block 96 of 500
Block 97 of 500
Block 98 of 500
Block 99 of 500
Block 100 of 500
Block 101 of 500
Block 102 of 500
Block 103 of 500
Block 104 of 500
Block 105 of 500
Block 106 of 500
Block 107 of 500
Block 108 of 500
Block 109 of 500
Block 110 of 500
Block 111 of 500
Block 112 of 500
Block 113 of 500
Block 114 of 500
Block 115 of 500
Block 116 of 500
Block 117 of 500
Block 118 of 500
Block 119 of 500
Block 120 of 500
Block 121 of 500
Block 122 of 500
Block 123 of 500
Block 124 of 500
Block 125 of 500
Block 126 of 500
Block 127 of 500
Block 128 of 500
Block 129 of 500
Block 130 of 500
Block 131 of 500
Block 132 of 500
Block 133 of 500
Block 134 of 500
Block 135 of 500
Block 136 of 500
Block 137 of 500
Block 138 of 500
Block 139 of 500
Block 140 of 500
Block 141 of 500
Block 142 of 500
Block 143 of 500
Block 144 of 500
Block 145 of 500
Block 146 of 500
Block 147 of 500
Block 148 of 500
Block 149 of 500
Block 150 of 500
Block 151 of 500
Block 152 of 500
Block 153 of 500
Block 154 of 500
Block 155 of 500
Block 156 of 500
Block 157 of 500
Block 158 of 500
Block 159 of 500
Block 160 of 500
Block 161 of 500
Block 162 of 500
Block 163 of 500
Block 164 of 500
Block 165 of 500
Block 166 of 500
Block 167 of 500
Block 168 of 500
Block 169 of 500
Block 170 of 500
Block 171 of 500
Block 172 of 500
Block 173 of 500
Block 174 of 500
Block 175 of 500
Block 176 of 500
Block 177 of 500
Block 178 of 500
Block 179 of 500
Block 180 of 500
Block 181 of 500
Block 182 of 500
Block 183 of 500
Block 184 of 500
Block 185 of 500
Block 186 of 500
Block 187 of 500
Block 188 of 500
Block 189 of 500
Block 190 of 500
Block 191 of 500
Block 192 of 500
Block 193 of 500
Block 194 of 500
Block 195 of 500
Block 196 of 500
Block 197 of 500
Block 198 of 500
Block 199 of 500
Block 200 of 500
Block 201 of 500
Block 202 of 500
Block 203 of 500
Block 204 of 500
Block 205 of 500
Block 206 of 500
Block 207 of 500
Block 208 of 500
Block 209 of 500
Block 210 of 500
Block 211 of 500
Block 212 of 500
Block 213 of 500
Block 214 of 500
Block 215 of 500
Block 216 of 500
Block 217 of 500
Block 218 of 500
Block 219 of 500
Block 220 of 500
Block 221 of 500
Block 222 of 500
Block 223 of 500
Block 224 of 500
Block 225 of 500
Block 226 of 500
Block 227 of 500
Block 228 of 500
Block 229 of 500
Block 230 of 500
Block 231 of 500
Block 232 of 500
Block 233 of 500
Block 234 of 500
Block 235 of 500
Block 236 of 500
Block 237 of 500
Block 238 of 500
Block 239 of 500
Block 240 of 500
Block 241 of 500
Block 242 of 500
Block 243 of 500
Block 244 of 500
Block 245 of 500
Block 246 of 500
Block 247 of 500
Block 247 of 500
Block 248 of 500
Block 249 of 500
Block 250 of 500
Block 251 of 500
Block 252 of 500
Block 253 of 500
Block 254 of 500
Block 255 of 500
Block 256 of 500
Block 257 of 500
Block 258 of 500
Block 259 of 500
Block 260 of 500
Block 261 of 500
Block 262 of 500
Block 263 of 500
Block 264 of 500
Block 265 of 500
Block 266 of 500
Block 267 of 500
Block 268 of 500
Block 268 of 500
Block 269 of 500
Block 270 of 500
Block 271 of 500
Block 272 of 500
Block 273 of 500
Block 274 of 500
Block 275 of 500
Block 276 of 500
Block 277 of 500
Block 278 of 500
Block 279 of 500
Block 280 of 500
Block 281 of 500
Block 282 of 500
Block 283 of 500
Block 284 of 500
Block 285 of 500
Block 286 of 500
Block 287 of 500
Block 288 of 500
Block 289 of 500
Block 290 of 500
Block 291 of 500
Block 292 of 500
Block 293 of 500
Block 294 of 500
Block 295 of 500
Block 296 of 500
Block 297 of 500
Block 298 of 500
Block 298 of 500
Block 299 of 500
Block 300 of 500
Block 301 of 500
Block 302 of 500
Block 303 of 500
Block 304 of 500
Block 305 of 500
Block 306 of 500
Block 307 of 500
Block 308 of 500
Block 309 of 500
Block 310 of 500
Block 311 of 500
Block 312 of 500
Block 313 of 500
Block 314 of 500
Block 315 of 500
Block 316 of 500
Block 317 of 500
Block 318 of 500
Block 319 of 500
Block 320 of 500
Block 321 of 500
Block 322 of 500
Block 323 of 500
Block 324 of 500
Block 325 of 500
Block 326 of 500
Block 327 of 500
Block 328 of 500
Block 329 of 500
Block 330 of 500
Block 331 of 500
Block 332 of 500
Block 333 of 500
Block 334 of 500
Block 335 of 500
Block 336 of 500
Block 337 of 500
Block 338 of 500
Block 339 of 500
Block 340 of 500
Block 341 of 500
Block 342 of 500
Block 343 of 500
Block 344 of 500
Block 345 of 500
Block 346 of 500
Block 347 of 500
Block 348 of 500
Block 349 of 500
Block 350 of 500
Block 351 of 500
Block 352 of 500
Block 353 of 500
Block 354 of 500
Block 355 of 500
Block 356 of 500
Block 357 of 500
Block 358 of 500
Block 359 of 500
Block 360 of 500
Block 361 of 500
Block 362 of 500
Block 363 of 500
Block 364 of 500
Block 365 of 500
Block 366 of 500
Block 367 of 500
Block 368 of 500
Block 369 of 500
Block 370 of 500
Block 371 of 500
Block 372 of 500
Block 373 of 500
Block 374 of 500
Block 375 of 500
Block 376 of 500
Block 377 of 500
Block 378 of 500
Block 379 of 500
Block 380 of 500
Block 380 of 500
Block 381 of 500
Block 382 of 500
Block 383 of 500
Block 384 of 500
Block 385 of 500
Block 386 of 500
Block 387 of 500
Block 388 of 500
Block 389 of 500
Block 390 of 500
Block 391 of 500
Block 392 of 500
Block 393 of 500
Block 394 of 500
Block 395 of 500
Block 396 of 500
Block 397 of 500
Block 398 of 500
Block 399 of 500
Block 400 of 500
Block 401 of 500
Block 402 of 500
Block 403 of 500
Block 404 of 500
Block 405 of 500
Block 406 of 500
Block 407 of 500
Block 408 of 500
Block 409 of 500
Block 410 of 500
Block 411 of 500
Block 412 of 500
Block 413 of 500
Block 414 of 500
Block 415 of 500
Block 416 of 500
Block 417 of 500
Block 418 of 500
Block 419 of 500
Block 420 of 500
Block 421 of 500
Block 422 of 500
Block 423 of 500
Block 424 of 500
Block 425 of 500
Block 426 of 500
Block 427 of 500
Block 428 of 500
Block 429 of 500
Block 430 of 500
Block 431 of 500
Block 432 of 500
Block 433 of 500
Block 434 of 500
Block 435 of 500
Block 436 of 500
Block 437 of 500
Block 438 of 500
Block 439 of 500
Block 440 of 500
Block 441 of 500
Block 442 of 500
Block 443 of 500
Block 444 of 500
Block 445 of 500
Block 446 of 500
Block 447 of 500
Block 448 of 500
Block 449 of 500
Block 450 of 500
Block 451 of 500
Block 452 of 500
Block 453 of 500
Block 454 of 500
Block 455 of 500
Block 456 of 500
Block 457 of 500
Block 458 of 500
Block 459 of 500
Block 460 of 500
Block 461 of 500
Block 462 of 500
Block 463 of 500
Block 464 of 500
Block 465 of 500
Block 466 of 500
Block 467 of 500
Block 468 of 500
Block 469 of 500
Block 470 of 500
Block 471 of 500
Block 472 of 500
Block 473 of 500
Block 474 of 500
Block 475 of 500
Block 476 of 500
Block 477 of 500
Block 478 of 500
Block 479 of 500
Block 480 of 500
Block 481 of 500
Block 482 of 500
Block 483 of 500
Block 484 of 500
Block 485 of 500
Block 486 of 500
Block 487 of 500
Block 488 of 500
Block 489 of 500
Block 490 of 500
Block 491 of 500
Block 492 of 500
Block 493 of 500
Block 494 of 500
Block 495 of 500
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818&#x27; (338.0K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818.tmp&#x27; (85.1K, reduced: 252.9K)
    # of fragments: 14
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 500
Block 2 of 500
Block 3 of 500
Block 4 of 500
Block 5 of 500
Block 6 of 500
Block 7 of 500
Block 8 of 500
Block 9 of 500
Block 10 of 500
Block 11 of 500
Block 12 of 500
Block 13 of 500
Block 14 of 500
Block 15 of 500
Block 16 of 500
Block 17 of 500
Block 18 of 500
Block 19 of 500
Block 20 of 500
Block 21 of 500
Block 22 of 500
Block 23 of 500
Block 24 of 500
Block 25 of 500
Block 26 of 500
Block 27 of 500
Block 28 of 500
Block 29 of 500
Block 30 of 500
Block 31 of 500
Block 32 of 500
Block 33 of 500
Block 34 of 500
Block 35 of 500
Block 36 of 500
Block 37 of 500
Block 38 of 500
Block 39 of 500
Block 40 of 500
Block 41 of 500
Block 42 of 500
Block 43 of 500
Block 44 of 500
Block 45 of 500
Block 46 of 500
Block 47 of 500
Block 48 of 500
Block 49 of 500
Block 50 of 500
Block 51 of 500
Block 52 of 500
Block 53 of 500
Block 54 of 500
Block 55 of 500
Block 56 of 500
Block 57 of 500
Block 58 of 500
Block 58 of 500
Block 59 of 500
Block 60 of 500
Block 61 of 500
Block 62 of 500
Block 63 of 500
Block 64 of 500
Block 65 of 500
Block 66 of 500
Block 67 of 500
Block 68 of 500
Block 69 of 500
Block 70 of 500
Block 71 of 500
Block 72 of 500
Block 73 of 500
Block 74 of 500
Block 75 of 500
Block 76 of 500
Block 77 of 500
Block 78 of 500
Block 79 of 500
Block 80 of 500
Block 81 of 500
Block 82 of 500
Block 83 of 500
Block 84 of 500
Block 85 of 500
Block 86 of 500
Block 87 of 500
Block 88 of 500
Block 89 of 500
Block 90 of 500
Block 91 of 500
Block 92 of 500
Block 93 of 500
Block 94 of 500
Block 95 of 500
Block 96 of 500
Block 97 of 500
Block 98 of 500
Block 99 of 500
Block 100 of 500
Block 101 of 500
Block 102 of 500
Block 103 of 500
Block 104 of 500
Block 105 of 500
Block 106 of 500
Block 107 of 500
Block 108 of 500
Block 109 of 500
Block 110 of 500
Block 111 of 500
Block 112 of 500
Block 113 of 500
Block 114 of 500
Block 115 of 500
Block 116 of 500
Block 117 of 500
Block 118 of 500
Block 119 of 500
Block 120 of 500
Block 121 of 500
Block 122 of 500
Block 123 of 500
Block 124 of 500
Block 125 of 500
Block 126 of 500
Block 127 of 500
Block 128 of 500
Block 129 of 500
Block 130 of 500
Block 131 of 500
Block 132 of 500
Block 133 of 500
Block 134 of 500
Block 135 of 500
Block 136 of 500
Block 137 of 500
Block 138 of 500
Block 139 of 500
Block 140 of 500
Block 141 of 500
Block 142 of 500
Block 143 of 500
Block 144 of 500
Block 145 of 500
Block 146 of 500
Block 147 of 500
Block 148 of 500
Block 149 of 500
Block 150 of 500
Block 151 of 500
Block 152 of 500
Block 153 of 500
Block 154 of 500
Block 155 of 500
Block 156 of 500
Block 157 of 500
Block 158 of 500
Block 159 of 500
Block 160 of 500
Block 161 of 500
Block 162 of 500
Block 163 of 500
Block 164 of 500
Block 165 of 500
Block 166 of 500
Block 167 of 500
Block 168 of 500
Block 169 of 500
Block 170 of 500
Block 171 of 500
Block 172 of 500
Block 173 of 500
Block 174 of 500
Block 175 of 500
Block 176 of 500
Block 177 of 500
Block 178 of 500
Block 179 of 500
Block 180 of 500
Block 181 of 500
Block 182 of 500
Block 183 of 500
Block 184 of 500
Block 185 of 500
Block 186 of 500
Block 187 of 500
Block 188 of 500
Block 189 of 500
Block 190 of 500
Block 191 of 500
Block 192 of 500
Block 193 of 500
Block 194 of 500
Block 195 of 500
Block 196 of 500
Block 197 of 500
Block 198 of 500
Block 199 of 500
Block 200 of 500
Block 201 of 500
Block 202 of 500
Block 203 of 500
Block 204 of 500
Block 205 of 500
Block 206 of 500
Block 207 of 500
Block 208 of 500
Block 209 of 500
Block 210 of 500
Block 211 of 500
Block 212 of 500
Block 213 of 500
Block 214 of 500
Block 215 of 500
Block 216 of 500
Block 217 of 500
Block 218 of 500
Block 219 of 500
Block 220 of 500
Block 221 of 500
Block 222 of 500
Block 223 of 500
Block 224 of 500
Block 225 of 500
Block 226 of 500
Block 227 of 500
Block 228 of 500
Block 229 of 500
Block 230 of 500
Block 231 of 500
Block 232 of 500
Block 233 of 500
Block 234 of 500
Block 235 of 500
Block 236 of 500
Block 237 of 500
Block 238 of 500
Block 239 of 500
Block 240 of 500
Block 241 of 500
Block 242 of 500
Block 243 of 500
Block 244 of 500
Block 245 of 500
Block 246 of 500
Block 247 of 500
Block 247 of 500
Block 248 of 500
Block 249 of 500
Block 250 of 500
Block 251 of 500
Block 252 of 500
Block 253 of 500
Block 254 of 500
Block 255 of 500
Block 256 of 500
Block 257 of 500
Block 258 of 500
Block 259 of 500
Block 260 of 500
Block 261 of 500
Block 262 of 500
Block 263 of 500
Block 264 of 500
Block 265 of 500
Block 266 of 500
Block 267 of 500
Block 268 of 500
Block 268 of 500
Block 269 of 500
Block 270 of 500
Block 271 of 500
Block 272 of 500
Block 273 of 500
Block 274 of 500
Block 275 of 500
Block 276 of 500
Block 277 of 500
Block 278 of 500
Block 279 of 500
Block 280 of 500
Block 281 of 500
Block 282 of 500
Block 283 of 500
Block 284 of 500
Block 285 of 500
Block 286 of 500
Block 287 of 500
Block 288 of 500
Block 289 of 500
Block 290 of 500
Block 291 of 500
Block 292 of 500
Block 293 of 500
Block 294 of 500
Block 295 of 500
Block 296 of 500
Block 297 of 500
Block 298 of 500
Block 298 of 500
Block 299 of 500
Block 300 of 500
Block 301 of 500
Block 302 of 500
Block 303 of 500
Block 304 of 500
Block 305 of 500
Block 306 of 500
Block 307 of 500
Block 308 of 500
Block 309 of 500
Block 310 of 500
Block 311 of 500
Block 312 of 500
Block 313 of 500
Block 314 of 500
Block 315 of 500
Block 316 of 500
Block 317 of 500
Block 318 of 500
Block 319 of 500
Block 320 of 500
Block 321 of 500
Block 322 of 500
Block 323 of 500
Block 324 of 500
Block 325 of 500
Block 326 of 500
Block 327 of 500
Block 328 of 500
Block 329 of 500
Block 330 of 500
Block 331 of 500
Block 332 of 500
Block 333 of 500
Block 334 of 500
Block 335 of 500
Block 336 of 500
Block 337 of 500
Block 338 of 500
Block 339 of 500
Block 340 of 500
Block 341 of 500
Block 342 of 500
Block 343 of 500
Block 344 of 500
Block 345 of 500
Block 346 of 500
Block 347 of 500
Block 348 of 500
Block 349 of 500
Block 350 of 500
Block 351 of 500
Block 352 of 500
Block 353 of 500
Block 354 of 500
Block 355 of 500
Block 356 of 500
Block 357 of 500
Block 358 of 500
Block 359 of 500
Block 360 of 500
Block 361 of 500
Block 362 of 500
Block 363 of 500
Block 364 of 500
Block 365 of 500
Block 366 of 500
Block 367 of 500
Block 368 of 500
Block 369 of 500
Block 370 of 500
Block 371 of 500
Block 372 of 500
Block 373 of 500
Block 374 of 500
Block 375 of 500
Block 376 of 500
Block 377 of 500
Block 378 of 500
Block 379 of 500
Block 380 of 500
Block 380 of 500
Block 381 of 500
Block 382 of 500
Block 383 of 500
Block 384 of 500
Block 385 of 500
Block 386 of 500
Block 387 of 500
Block 388 of 500
Block 389 of 500
Block 390 of 500
Block 391 of 500
Block 392 of 500
Block 393 of 500
Block 394 of 500
Block 395 of 500
Block 396 of 500
Block 397 of 500
Block 398 of 500
Block 399 of 500
Block 400 of 500
Block 401 of 500
Block 402 of 500
Block 403 of 500
Block 404 of 500
Block 405 of 500
Block 406 of 500
Block 407 of 500
Block 408 of 500
Block 409 of 500
Block 410 of 500
Block 411 of 500
Block 412 of 500
Block 413 of 500
Block 414 of 500
Block 415 of 500
Block 416 of 500
Block 417 of 500
Block 418 of 500
Block 419 of 500
Block 420 of 500
Block 421 of 500
Block 422 of 500
Block 423 of 500
Block 424 of 500
Block 425 of 500
Block 426 of 500
Block 427 of 500
Block 428 of 500
Block 429 of 500
Block 430 of 500
Block 431 of 500
Block 432 of 500
Block 433 of 500
Block 434 of 500
Block 435 of 500
Block 436 of 500
Block 437 of 500
Block 438 of 500
Block 439 of 500
Block 440 of 500
Block 441 of 500
Block 442 of 500
Block 443 of 500
Block 444 of 500
Block 445 of 500
Block 446 of 500
Block 447 of 500
Block 448 of 500
Block 449 of 500
Block 450 of 500
Block 451 of 500
Block 452 of 500
Block 453 of 500
Block 454 of 500
Block 455 of 500
Block 456 of 500
Block 457 of 500
Block 458 of 500
Block 459 of 500
Block 460 of 500
Block 461 of 500
Block 462 of 500
Block 463 of 500
Block 464 of 500
Block 465 of 500
Block 466 of 500
Block 467 of 500
Block 468 of 500
Block 469 of 500
Block 470 of 500
Block 471 of 500
Block 472 of 500
Block 473 of 500
Block 474 of 500
Block 475 of 500
Block 476 of 500
Block 477 of 500
Block 478 of 500
Block 479 of 500
Block 480 of 500
Block 481 of 500
Block 482 of 500
Block 483 of 500
Block 484 of 500
Block 485 of 500
Block 486 of 500
Block 487 of 500
Block 488 of 500
Block 489 of 500
Block 490 of 500
Block 491 of 500
Block 492 of 500
Block 493 of 500
Block 494 of 500
Block 495 of 500
Writing annotation...
Compressing...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818&#x27; (338.0K)
    # of fragments: 30
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe3851c818.tmp&#x27; (84.0K, reduced: 254.0K)
    # of fragments: 14
All row numbers refer to row in the full pedigree (not just within family). 
 Correct current problems and rerun pedigreeCheck.
There may be additional problems not investigated because of the current problems.

All row numbers refer to row in the full pedigree (not just within family). 
 Correct current problems and rerun pedigreeCheck.
There may be additional problems not investigated because of the current problems.

All row numbers refer to row in the full pedigree (not just within family). 
 Correct current problems and rerun pedigreeCheck.
There may be additional problems not investigated because of the current problems.

All row numbers refer to row in the full pedigree (not just within family). 
 Correct current problems and rerun pedigreeCheck.
There may be additional problems not investigated because of the current problems.

All row numbers refer to row in the full pedigree (not just within family). 
 Correct current problems and rerun pedigreeCheck.
There may be additional problems not investigated because of the current problems.

All row numbers refer to row in the full pedigree (not just within family). 
 Correct current problems and rerun pedigreeCheck.
There may be additional problems not investigated because of the current problems.

All row numbers refer to row in the full pedigree (not just within family). 
 Correct current problems and rerun pedigreeCheck.
There may be additional problems not investigated because of the current problems.

All row numbers refer to row in the full pedigree (not just within family). 
 Correct current problems and rerun pedigreeCheck.
There may be additional problems not investigated because of the current problems.

All row numbers refer to row in the full pedigree (not just within family). 
 Correct current problems and rerun pedigreeCheck.
There may be additional problems not investigated because of the current problems.

All row numbers refer to row in the full pedigree (not just within family). 
 Correct current problems and rerun pedigreeCheck.
There may be additional problems not investigated because of the current problems.

All row numbers refer to row in the full pedigree (not just within family). 
 Correct current problems and rerun pedigreeCheck.
There may be additional problems not investigated because of the current problems.

All row numbers refer to row in the full pedigree (not just within family). 
 Correct current problems and rerun pedigreeCheck.
There may be additional problems not investigated because of the current problems.

Checking SNPs against map file
Checking sample data and genotypes in each line of ped file
Checking SNPs against map file
Checking sample data and genotypes in each line of ped file
Checking SNPs against map file
Checking sample data and genotypes in each line of ped file
Checking SNPs against map file
Checking sample data and genotypes in each line of ped file
writing block 1 of 1: scans 1-20
x being coerced from class: matrix to data.table
Checking SNPs against map file
Checking sample data and genotypes in each line of ped file
Checking SNPs against map file
Checking sample data and genotypes in each line of ped file
writing block 1 of 1: scans 1-20
x being coerced from class: matrix to data.table
Checking SNPs against map file
Checking sample data and genotypes in each line of ped file
writing block 1 of 1: scans 1-3
x being coerced from class: matrix to data.table
writing block 1 of 1: scans 1-3
x being coerced from class: matrix to data.table
block number= 1 vector index= 1
block number= 1 vector index= 1
block number= 1 vector index= 1
block number= 2 vector index= 51
block number= 3 vector index= 101
block number= 4 vector index= 151
block number= 5 vector index= 201
block number= 6 vector index= 251
block number= 1 vector index= 1
block number= 2 vector index= 2
block number= 3 vector index= 3
block number= 4 vector index= 4
block number= 5 vector index= 5
block number= 6 vector index= 6
block number= 7 vector index= 7
block number= 8 vector index= 8
block number= 9 vector index= 9
block number= 10 vector index= 10
block number= 11 vector index= 11
block number= 12 vector index= 12
block number= 13 vector index= 13
block number= 14 vector index= 14
block number= 15 vector index= 15
block number= 16 vector index= 16
block number= 17 vector index= 17
block number= 18 vector index= 18
block number= 19 vector index= 19
block number= 20 vector index= 20
block number= 21 vector index= 21
block number= 22 vector index= 22
block number= 23 vector index= 23
block number= 24 vector index= 24
block number= 25 vector index= 25
block number= 26 vector index= 26
block number= 27 vector index= 27
block number= 28 vector index= 28
block number= 29 vector index= 29
block number= 30 vector index= 30
block number= 31 vector index= 31
block number= 32 vector index= 32
block number= 33 vector index= 33
block number= 34 vector index= 34
block number= 35 vector index= 35
block number= 36 vector index= 36
block number= 37 vector index= 37
block number= 38 vector index= 38
block number= 39 vector index= 39
block number= 40 vector index= 40
block number= 41 vector index= 41
block number= 42 vector index= 42
block number= 43 vector index= 43
block number= 44 vector index= 44
block number= 45 vector index= 45
block number= 46 vector index= 46
block number= 47 vector index= 47
block number= 48 vector index= 48
block number= 49 vector index= 49
block number= 50 vector index= 50
block number= 51 vector index= 51
block number= 52 vector index= 52
block number= 53 vector index= 53
block number= 54 vector index= 54
block number= 55 vector index= 55
block number= 56 vector index= 56
block number= 57 vector index= 57
block number= 58 vector index= 58
block number= 59 vector index= 59
block number= 60 vector index= 60
block number= 61 vector index= 61
block number= 62 vector index= 62
block number= 63 vector index= 63
block number= 64 vector index= 64
block number= 65 vector index= 65
block number= 66 vector index= 66
block number= 67 vector index= 67
block number= 68 vector index= 68
block number= 69 vector index= 69
block number= 70 vector index= 70
block number= 71 vector index= 71
block number= 72 vector index= 72
block number= 73 vector index= 73
block number= 74 vector index= 74
block number= 75 vector index= 75
block number= 76 vector index= 76
block number= 77 vector index= 77
block number= 78 vector index= 78
block number= 79 vector index= 79
block number= 80 vector index= 80
block number= 81 vector index= 81
block number= 82 vector index= 82
block number= 83 vector index= 83
block number= 84 vector index= 84
block number= 85 vector index= 85
block number= 86 vector index= 86
block number= 87 vector index= 87
block number= 88 vector index= 88
block number= 89 vector index= 89
block number= 90 vector index= 90
block number= 91 vector index= 91
block number= 92 vector index= 92
block number= 93 vector index= 93
block number= 94 vector index= 94
block number= 95 vector index= 95
block number= 96 vector index= 96
block number= 97 vector index= 97
block number= 98 vector index= 98
block number= 99 vector index= 99
block number= 100 vector index= 100
block number= 101 vector index= 101
block number= 102 vector index= 102
block number= 103 vector index= 103
block number= 104 vector index= 104
block number= 105 vector index= 105
block number= 106 vector index= 106
block number= 107 vector index= 107
block number= 108 vector index= 108
block number= 109 vector index= 109
block number= 110 vector index= 110
block number= 111 vector index= 111
block number= 112 vector index= 112
block number= 113 vector index= 113
block number= 114 vector index= 114
block number= 115 vector index= 115
block number= 116 vector index= 116
block number= 117 vector index= 117
block number= 118 vector index= 118
block number= 119 vector index= 119
block number= 120 vector index= 120
block number= 121 vector index= 121
block number= 122 vector index= 122
block number= 123 vector index= 123
block number= 124 vector index= 124
block number= 125 vector index= 125
block number= 126 vector index= 126
block number= 127 vector index= 127
block number= 128 vector index= 128
block number= 129 vector index= 129
block number= 130 vector index= 130
block number= 131 vector index= 131
block number= 132 vector index= 132
block number= 133 vector index= 133
block number= 134 vector index= 134
block number= 135 vector index= 135
block number= 136 vector index= 136
block number= 137 vector index= 137
block number= 138 vector index= 138
block number= 139 vector index= 139
block number= 140 vector index= 140
block number= 141 vector index= 141
block number= 142 vector index= 142
block number= 143 vector index= 143
block number= 144 vector index= 144
block number= 145 vector index= 145
block number= 146 vector index= 146
block number= 147 vector index= 147
block number= 148 vector index= 148
block number= 149 vector index= 149
block number= 150 vector index= 150
block number= 151 vector index= 151
block number= 152 vector index= 152
block number= 153 vector index= 153
block number= 154 vector index= 154
block number= 155 vector index= 155
block number= 156 vector index= 156
block number= 157 vector index= 157
block number= 158 vector index= 158
block number= 159 vector index= 159
block number= 160 vector index= 160
block number= 161 vector index= 161
block number= 162 vector index= 162
block number= 163 vector index= 163
block number= 164 vector index= 164
block number= 165 vector index= 165
block number= 166 vector index= 166
block number= 167 vector index= 167
block number= 168 vector index= 168
block number= 169 vector index= 169
block number= 170 vector index= 170
block number= 171 vector index= 171
block number= 172 vector index= 172
block number= 173 vector index= 173
block number= 174 vector index= 174
block number= 175 vector index= 175
block number= 176 vector index= 176
block number= 177 vector index= 177
block number= 178 vector index= 178
block number= 179 vector index= 179
block number= 180 vector index= 180
block number= 181 vector index= 181
block number= 182 vector index= 182
block number= 183 vector index= 183
block number= 184 vector index= 184
block number= 185 vector index= 185
block number= 186 vector index= 186
block number= 187 vector index= 187
block number= 188 vector index= 188
block number= 189 vector index= 189
block number= 190 vector index= 190
block number= 191 vector index= 191
block number= 192 vector index= 192
block number= 193 vector index= 193
block number= 194 vector index= 194
block number= 195 vector index= 195
block number= 196 vector index= 196
block number= 197 vector index= 197
block number= 198 vector index= 198
block number= 199 vector index= 199
block number= 200 vector index= 200
block number= 201 vector index= 201
block number= 202 vector index= 202
block number= 203 vector index= 203
block number= 204 vector index= 204
block number= 205 vector index= 205
block number= 206 vector index= 206
block number= 207 vector index= 207
block number= 208 vector index= 208
block number= 209 vector index= 209
block number= 210 vector index= 210
block number= 211 vector index= 211
block number= 212 vector index= 212
block number= 213 vector index= 213
block number= 214 vector index= 214
block number= 215 vector index= 215
block number= 216 vector index= 216
block number= 217 vector index= 217
block number= 218 vector index= 218
block number= 219 vector index= 219
block number= 220 vector index= 220
block number= 221 vector index= 221
block number= 222 vector index= 222
block number= 223 vector index= 223
block number= 224 vector index= 224
block number= 225 vector index= 225
block number= 226 vector index= 226
block number= 227 vector index= 227
block number= 228 vector index= 228
block number= 229 vector index= 229
block number= 230 vector index= 230
block number= 231 vector index= 231
block number= 232 vector index= 232
block number= 233 vector index= 233
block number= 234 vector index= 234
block number= 235 vector index= 235
block number= 236 vector index= 236
block number= 237 vector index= 237
block number= 238 vector index= 238
block number= 239 vector index= 239
block number= 240 vector index= 240
block number= 241 vector index= 241
block number= 242 vector index= 242
block number= 243 vector index= 243
block number= 244 vector index= 244
block number= 245 vector index= 245
block number= 246 vector index= 246
block number= 247 vector index= 247
block number= 248 vector index= 248
block number= 249 vector index= 249
block number= 250 vector index= 250
block number= 251 vector index= 251
block number= 252 vector index= 252
block number= 253 vector index= 253
block number= 254 vector index= 254
block number= 255 vector index= 255
block number= 256 vector index= 256
block number= 257 vector index= 257
block number= 258 vector index= 258
block number= 259 vector index= 259
block number= 260 vector index= 260
sample 10 of 20
sample 20 of 20
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe26c1dd5b&#x27; (3.4K)
    # of fragments: 22
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe26c1dd5b.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe26c1dd5b.tmp&#x27; (3.2K, reduced: 132B)
    # of fragments: 11
working on genotype
genotype - looping over samples
All variables match.
sample 10 of 10
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe7a0167e8&#x27; (2.6K)
    # of fragments: 22
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe7a0167e8.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe7a0167e8.tmp&#x27; (2.5K, reduced: 132B)
    # of fragments: 11
working on genotype
genotype - looping over samples
All variables match.
sample 10 of 20
sample 20 of 20
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe77483b08&#x27; (3.4K)
    # of fragments: 22
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe77483b08.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe77483b08.tmp&#x27; (3.2K, reduced: 132B)
    # of fragments: 11
sample 10 of 10
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe676905eb&#x27; (2.6K)
    # of fragments: 22
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe676905eb.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe676905eb.tmp&#x27; (2.5K, reduced: 132B)
    # of fragments: 11
Loading required namespace: snpStats
Start file conversion from VCF to SNP GDS ...
Method: extracting biallelic SNPs
Number of samples: 5
Parsing &quot;/home/biocbuild/bbs-3.17-bioc/R/library/VariantAnnotation/extdata/chr22.vcf.gz&quot; ...
	import 9969 variants.
+ genotype   { Bit2 5x9969, 12.2K } *
Optimize the access efficiency ...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe6158c8f9&#x27; (73.0K)
    # of fragments: 46
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe6158c8f9.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe6158c8f9.tmp&#x27; (72.7K, reduced: 312B)
    # of fragments: 20
Block 1 of 10... 1000 SNPs
x being coerced from class: matrix to data.table
Block 2 of 10... 1000 SNPs
x being coerced from class: matrix to data.table
Block 3 of 10... 1000 SNPs
x being coerced from class: matrix to data.table
Block 4 of 10... 1000 SNPs
x being coerced from class: matrix to data.table
Block 5 of 10... 1000 SNPs
x being coerced from class: matrix to data.table
Block 6 of 10... 1000 SNPs
x being coerced from class: matrix to data.table
Block 7 of 10... 1000 SNPs
x being coerced from class: matrix to data.table
Block 8 of 10... 1000 SNPs
x being coerced from class: matrix to data.table
Block 9 of 10... 1000 SNPs
x being coerced from class: matrix to data.table
Block 10 of 10... 969 SNPs
x being coerced from class: matrix to data.table
Block 1 of 1... 3 SNPs
x being coerced from class: matrix to data.table
Block 1 of 1... 3 SNPs
x being coerced from class: matrix to data.table
Block 1 of 1... 3 SNPs
x being coerced from class: matrix to data.table
Block 1 of 1... 3 SNPs
x being coerced from class: matrix to data.table
Block 1 of 1... 3 SNPs
x being coerced from class: matrix to data.table
Block 1 of 1... 3 SNPs
x being coerced from class: matrix to data.table
Block 1 of 1... 3 SNPs
x being coerced from class: matrix to data.table
Block 1 of 1... 3 SNPs
x being coerced from class: matrix to data.table
Block 1 of 3... 2 SNPs
x being coerced from class: matrix to data.table
Block 2 of 3... 4 SNPs
x being coerced from class: matrix to data.table
Block 3 of 3... 2 SNPs
x being coerced from class: matrix to data.table
Block 1 of 3... 0 SNPs
Block 2 of 3... 1 SNPs
x being coerced from class: matrix to data.table
Block 3 of 3... 2 SNPs
x being coerced from class: matrix to data.table
Start file conversion from VCF to SNP GDS ...
Method: extracting biallelic SNPs
Number of samples: 3
Parsing &quot;/home/biocbuild/bbs-3.17-bioc/R/library/SNPRelate/extdata/sequence.vcf&quot; ...
	import 2 variants.
+ genotype   { Bit2 3x2, 2B } *
Optimize the access efficiency ...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe1d446f58&#x27; (2.9K)
    # of fragments: 46
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe1d446f58.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe1d446f58.tmp&#x27; (2.6K, reduced: 312B)
    # of fragments: 20
Block 1 of 1... 2 SNPs
x being coerced from class: matrix to data.table
Start file conversion from VCF to SNP GDS ...
Method: extracting biallelic SNPs
Number of samples: 3
Parsing &quot;/tmp/Rtmpthm57M/file2ceafe64916932&quot; ...
	import 2 variants.
+ genotype   { Bit2 3x2, 2B } *
Optimize the access efficiency ...
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/Rtmpthm57M/file2ceafe668d936&#x27; (2.9K)
    # of fragments: 46
    save to &#x27;/tmp/Rtmpthm57M/file2ceafe668d936.tmp&#x27;
    rename &#x27;/tmp/Rtmpthm57M/file2ceafe668d936.tmp&#x27; (2.6K, reduced: 312B)
    # of fragments: 20
Block 1 of 1... 5 SNPs
x being coerced from class: matrix to data.table
Checked 5 SNPs
Block 1 of 1... 5 SNPs
x being coerced from class: matrix to data.table
Block 1 of 1... 5 SNPs
x being coerced from class: matrix to data.table
Block 1 of 1... 5 SNPs
x being coerced from class: matrix to data.table
Checked 5 SNPs
Block 1 of 1... 3 SNPs
x being coerced from class: matrix to data.table
Excluding 1 genoData samples from check
Excluding 2 genoData SNPs from check
Note, sample order in VCF differs from genoData
Checked 3 SNPs


RUNIT TEST PROTOCOL -- Sat Feb 11 16:10:35 2023 
*********************************************** 
Number of test functions: 211 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
GWASTools RUnit Tests - 211 test functions, 0 errors, 0 failures
Number of test functions: 211 
Number of errors: 0 
Number of failures: 0 
There were 50 or more warnings (use warnings() to see the first 50)
&gt; 
&gt; proc.time()
   user  system elapsed 
118.533   5.128 123.773 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">GWASTools.Rcheck/GWASTools-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>BAFfromClusterMeans</TD><TD style="text-align: right;">1.486</TD><TD style="text-align: right;">0.035</TD><TD style="text-align: right;">1.529</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>BAFfromGenotypes</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GdsGenotypeReader-class</TD><TD style="text-align: right;">0.077</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.077</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GdsIntensityReader-class</TD><TD style="text-align: right;">0.031</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GdsReader-class</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GenotypeData-class</TD><TD style="text-align: right;">0.229</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.242</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GenotypeIterator-class</TD><TD style="text-align: right;">0.037</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>HLA</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>IntensityData-class</TD><TD style="text-align: right;">0.033</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.034</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>MatrixGenotypeReader-class</TD><TD style="text-align: right;">0.02</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.02</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>NcdfGenotypeReader-class</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.050</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>NcdfIntensityReader-class</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>NcdfReader-class</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ScanAnnotationDataFrame-class</TD><TD style="text-align: right;">0.076</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.081</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ScanAnnotationSQLite-class</TD><TD style="text-align: right;">0.209</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.213</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SnpAnnotationDataFrame-class</TD><TD style="text-align: right;">0.089</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.092</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SnpAnnotationSQLite-class</TD><TD style="text-align: right;">0.187</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.188</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>alleleFrequency</TD><TD style="text-align: right;">0.140</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.141</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>allequal</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anomDetectBAF</TD><TD style="text-align: right;">1.118</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.120</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anomDetectLOH</TD><TD style="text-align: right;">0.749</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.754</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anomIdentifyLowQuality</TD><TD style="text-align: right;">0.759</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">0.800</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>anomSegStats</TD><TD style="text-align: right;">0.411</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.416</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>apartSnpSelection</TD><TD style="text-align: right;">0.073</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.075</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>assocCoxPH</TD><TD style="text-align: right;">0.602</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.601</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>assocRegression</TD><TD style="text-align: right;">0.676</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.691</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>batchTest</TD><TD style="text-align: right;">2.132</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">2.172</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>centromeres</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>chromIntensityPlot</TD><TD style="text-align: right;">0.118</TD><TD style="text-align: right;">0.017</TD><TD style="text-align: right;">0.133</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>convertNcdfGds</TD><TD style="text-align: right;">0.327</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.348</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>createDataFile</TD><TD style="text-align: right;">0.430</TD><TD style="text-align: right;">0.059</TD><TD style="text-align: right;">0.524</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>duplicateDiscordance</TD><TD style="text-align: right;">0.429</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.433</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>duplicateDiscordanceAcrossDatasets</TD><TD style="text-align: right;">0.185</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.186</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>duplicateDiscordanceProbability</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>exactHWE</TD><TD style="text-align: right;">0.124</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.124</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findBAFvariance</TD><TD style="text-align: right;">0.306</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.314</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>gdsSubset</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">0.017</TD><TD style="text-align: right;">0.042</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genoClusterPlot</TD><TD style="text-align: right;">0.674</TD><TD style="text-align: right;">0.051</TD><TD style="text-align: right;">0.760</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genotypeToCharacter</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getobj</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>hetByScanChrom</TD><TD style="text-align: right;">0.109</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.113</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>hetBySnpSex</TD><TD style="text-align: right;">0.11</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.11</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ibdPlot</TD><TD style="text-align: right;">0.230</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.238</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>imputedDosageFile</TD><TD style="text-align: right;">1.165</TD><TD style="text-align: right;">0.257</TD><TD style="text-align: right;">1.431</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>intensityOutliersPlot</TD><TD style="text-align: right;">0.589</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.605</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>manhattanPlot</TD><TD style="text-align: right;">0.050</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.056</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>meanIntensityByScanChrom</TD><TD style="text-align: right;">0.508</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.521</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mendelErr</TD><TD style="text-align: right;">0.902</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.912</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mendelList</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>missingGenotypeByScanChrom</TD><TD style="text-align: right;">0.114</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.121</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>missingGenotypeBySnpSex</TD><TD style="text-align: right;">0.089</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.088</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pasteSorted</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pcaSnpFilters</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pedigreeCheck</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.039</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pedigreeDeleteDuplicates</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pedigreeMaxUnrelated</TD><TD style="text-align: right;">0.070</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.071</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pedigreePairwiseRelatedness</TD><TD style="text-align: right;">0.034</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.033</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plinkUtils</TD><TD style="text-align: right;">2.391</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">2.419</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pseudoautoIntensityPlot</TD><TD style="text-align: right;">0.069</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.078</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pseudoautosomal</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qqPlot</TD><TD style="text-align: right;">0.213</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.214</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qualityScoreByScan</TD><TD style="text-align: right;">0.220</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.219</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qualityScoreBySnp</TD><TD style="text-align: right;">0.031</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.035</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readWriteFirst</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>relationsMeanVar</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>saveas</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>setMissingGenotypes</TD><TD style="text-align: right;">0.078</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.096</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>simulateGenotypeMatrix</TD><TD style="text-align: right;">0.681</TD><TD style="text-align: right;">0.099</TD><TD style="text-align: right;">0.784</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpCorrelationPlot</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.055</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpStats</TD><TD style="text-align: right;">0.548</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.560</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>vcfWrite</TD><TD style="text-align: right;">15.390</TD><TD style="text-align: right;"> 0.428</TD><TD style="text-align: right;">15.819</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
