<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for MAGAR on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><B>[<A href="../index.html#M">M</A>]</B></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-23 01:34:08 -0000 (Thu, 23 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4245</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for MAGAR on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the MAGAR package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/MAGAR.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1067</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">MAGAR</A>&nbsp;1.7.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/MAGAR">landing page</A>)</SPAN><BR>Michael Scherer<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-21&nbsp;12:29:53&nbsp;-0000&nbsp;(Tue,&nbsp;21&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/MAGAR</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">264f11c</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:24:03&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: MAGAR</TD></TR>
<TR><TD><B>Version</B>: 1.7.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:MAGAR.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings MAGAR_1.7.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-22 07:26:05 -0000 (Wed, 22 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-22 07:36:20 -0000 (Wed, 22 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 615.0 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: MAGAR.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:MAGAR.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings MAGAR_1.7.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/MAGAR.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘MAGAR/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘MAGAR’ version ‘1.7.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .travis.yml
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘MAGAR’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
getOverlapUniverse: warning in overlapQTLs(meth.qtl.res = meth.qtl.res,
  type = type): partial argument match of &#x27;meth.qtl.res&#x27; to
  &#x27;meth.qtl.result.list&#x27;
doGenoImport: no visible global function definition for
  ‘makeGRangesFromDataFrame’
doGenoImport: no visible global function definition for
  ‘seqlevelsStyle&lt;-’
doGenoImport: no visible global function definition for ‘findOverlaps’
doGenoImport: no visible global function definition for ‘ggplot’
doGenoImport: no visible global function definition for ‘aes’
doGenoImport: no visible binding for global variable ‘PC1’
doGenoImport: no visible binding for global variable ‘PC2’
doGenoImport: no visible global function definition for ‘geom_point’
doGenoImport: no visible global function definition for ‘xlab’
doGenoImport: no visible global function definition for ‘ylab’
doGenoImport: no visible global function definition for ‘theme_bw’
doGenoImport: no visible global function definition for ‘theme’
doGenoImport: no visible global function definition for ‘element_blank’
doGenoImport: no visible global function definition for ‘element_text’
doGenoImport: no visible global function definition for ‘element_line’
doGenoImport: no visible global function definition for ‘ggsave’
doGenoImportIDAT: no visible global function definition for
  ‘featureData’
doGenoImportIDAT: no visible global function definition for
  ‘featureNames’
doGenoImportIDAT: no visible global function definition for
  ‘chromosome’
doGenoImportIDAT: no visible global function definition for ‘calls’
doGenoImportIDAT: no visible global function definition for
  ‘makeGRangesFromDataFrame’
doGenoImportIDAT: no visible global function definition for ‘GRanges’
doGenoImportIDAT: no visible global function definition for ‘Rle’
doGenoImportIDAT: no visible global function definition for ‘IRanges’
doGenoImportIDAT: no visible global function definition for
  ‘findOverlaps’
doGenoImportIDAT: no visible global function definition for ‘queryHits’
doGenoImportIDAT: no visible global function definition for
  ‘subjectHits’
doGenoImportImputed: no visible global function definition for ‘ggplot’
doGenoImportImputed: no visible global function definition for ‘aes’
doGenoImportImputed: no visible binding for global variable ‘PC1’
doGenoImportImputed: no visible binding for global variable ‘PC2’
doGenoImportImputed: no visible global function definition for
  ‘geom_point’
doGenoImportImputed: no visible global function definition for ‘xlab’
doGenoImportImputed: no visible global function definition for ‘ylab’
doGenoImportImputed: no visible global function definition for
  ‘theme_bw’
doGenoImportImputed: no visible global function definition for ‘theme’
doGenoImportImputed: no visible global function definition for
  ‘element_blank’
doGenoImportImputed: no visible global function definition for
  ‘element_text’
doGenoImportImputed: no visible global function definition for
  ‘element_line’
doGenoImportImputed: no visible global function definition for ‘ggsave’
doImport: no visible binding for global variable ‘anno’
doMethImport: no visible global function definition for ‘GRanges’
doMethImport: no visible global function definition for ‘Rle’
doMethImport: no visible global function definition for ‘IRanges’
doMethImport: no visible global function definition for
  ‘makeGRangesFromDataFrame’
doMethImport: no visible global function definition for ‘findOverlaps’
doMethImport: no visible global function definition for ‘queryHits’
doMethQTLChromosome: no visible global function definition for ‘ggplot’
doMethQTLChromosome: no visible global function definition for ‘aes’
doMethQTLChromosome: no visible binding for global variable ‘Size’
doMethQTLChromosome: no visible binding for global variable ‘..count..’
doMethQTLChromosome: no visible global function definition for
  ‘geom_histogram’
doMethQTLChromosome: no visible global function definition for
  ‘geom_vline’
doMethQTLChromosome: no visible global function definition for
  ‘theme_bw’
doMethQTLChromosome: no visible global function definition for ‘theme’
doMethQTLChromosome: no visible global function definition for
  ‘element_blank’
doMethQTLChromosome: no visible global function definition for
  ‘element_text’
doMethQTLChromosome: no visible global function definition for
  ‘element_line’
doMethQTLChromosome: no visible global function definition for ‘ggsave’
doMethQTLChromosome: no visible global function definition for
  ‘mclapply’
getOverlapUniverse: no visible global function definition for
  ‘makeGRangesFromDataFrame’
qtlAnnotationEnrichment: no visible global function definition for
  ‘findOverlaps’
qtlCorrelateCorBlockStat: no visible global function definition for
  ‘ggplot’
qtlCorrelateCorBlockStat: no visible global function definition for
  ‘aes_string’
qtlCorrelateCorBlockStat: no visible global function definition for
  ‘geom_point’
qtlCorrelateCorBlockStat: no visible global function definition for
  ‘geom_smooth’
qtlCorrelateCorBlockStat: no visible global function definition for
  ‘ggtitle’
qtlCorrelateCorBlockStat: no visible global function definition for
  ‘ylab’
qtlDistanceScatterplot: no visible global function definition for
  ‘ggplot’
qtlDistanceScatterplot: no visible global function definition for ‘aes’
qtlDistanceScatterplot: no visible binding for global variable
  ‘Distance’
qtlDistanceScatterplot: no visible binding for global variable
  ‘P.value’
qtlDistanceScatterplot: no visible global function definition for
  ‘geom_point’
qtlDistanceScatterplot: no visible global function definition for
  ‘ggtitle’
qtlDistanceScatterplot: no visible global function definition for
  ‘xlab’
qtlDistanceScatterplot: no visible global function definition for
  ‘ylab’
qtlDistanceScatterplot: no visible binding for global variable ‘Beta’
qtlDistanceScatterplot: no visible global function definition for
  ‘labs’
qtlDistanceScatterplot: no visible global function definition for
  ‘scale_color_gradient2’
qtlDistanceScatterplot: no visible global function definition for
  ‘scale_color_continuous’
qtlDistanceScatterplot: no visible global function definition for
  ‘annotate’
qtlDistanceScatterplot: no visible global function definition for
  ‘ggsave’
qtlLOLAEnrichment: no visible global function definition for
  ‘loadRegionDB’
qtlLOLAEnrichment: no visible global function definition for ‘runLOLA’
qtlPlotAnnotationEnrichment: no visible global function definition for
  ‘ggplot’
qtlPlotAnnotationEnrichment: no visible global function definition for
  ‘aes’
qtlPlotAnnotationEnrichment: no visible binding for global variable
  ‘Type’
qtlPlotAnnotationEnrichment: no visible binding for global variable
  ‘Annotation’
qtlPlotAnnotationEnrichment: no visible binding for global variable
  ‘OddsRatio’
qtlPlotAnnotationEnrichment: no visible global function definition for
  ‘geom_tile’
qtlPlotAnnotationEnrichment: no visible global function definition for
  ‘scale_fill_gradient2’
qtlPlotBaseSubstitution: no visible global function definition for
  ‘ggplot’
qtlPlotBaseSubstitution: no visible global function definition for
  ‘aes’
qtlPlotBaseSubstitution: no visible binding for global variable
  ‘Substitution’
qtlPlotBaseSubstitution: no visible binding for global variable
  ‘OddsRatio’
qtlPlotBaseSubstitution: no visible global function definition for
  ‘geom_tile’
qtlPlotBaseSubstitution: no visible global function definition for
  ‘scale_fill_gradient2’
qtlPlotClusterSize: no visible global function definition for ‘ggplot’
qtlPlotClusterSize: no visible global function definition for ‘aes’
qtlPlotClusterSize: no visible binding for global variable ‘Size’
qtlPlotClusterSize: no visible binding for global variable ‘..count..’
qtlPlotClusterSize: no visible global function definition for
  ‘geom_histogram’
qtlPlotSNPCorrelationBlock: no visible global function definition for
  ‘ggplot’
qtlPlotSNPCorrelationBlock: no visible global function definition for
  ‘aes’
qtlPlotSNPCorrelationBlock: no visible binding for global variable
  ‘SNP’
qtlPlotSNPCorrelationBlock: no visible binding for global variable
  ‘CpG’
qtlPlotSNPCorrelationBlock: no visible global function definition for
  ‘geom_point’
qtlPlotSNPCorrelationBlock: no visible binding for global variable
  ‘Representative’
qtlPlotSNPCorrelationBlock: no visible global function definition for
  ‘geom_smooth’
qtlPlotSNPCorrelationBlock: no visible global function definition for
  ‘facet_grid’
qtlPlotSNPCorrelationBlock: no visible global function definition for
  ‘theme’
qtlPlotSNPCorrelationBlock: no visible global function definition for
  ‘scale_color_manual’
qtlPlotSNPCpGInteraction: no visible global function definition for
  ‘ggplot’
qtlPlotSNPCpGInteraction: no visible global function definition for
  ‘aes’
qtlPlotSNPCpGInteraction: no visible binding for global variable ‘SNP’
qtlPlotSNPCpGInteraction: no visible binding for global variable ‘CpG’
qtlPlotSNPCpGInteraction: no visible global function definition for
  ‘geom_boxplot’
qtlPlotSNPCpGInteraction: no visible global function definition for
  ‘theme_bw’
qtlPlotSNPCpGInteraction: no visible global function definition for
  ‘ylab’
qtlPlotSNPCpGInteraction: no visible global function definition for
  ‘xlab’
qtlPlotSNPCpGInteraction: no visible global function definition for
  ‘ggsave’
qtlPlotSNPCpGInteraction: no visible binding for global variable
  ‘SNPDosage’
qtlPlotSNPCpGInteraction: no visible global function definition for
  ‘geom_point’
qtlPlotSNPCpGInteraction: no visible global function definition for
  ‘geom_smooth’
Undefined global functions or variables:
  ..count.. Annotation Beta CpG Distance GRanges IRanges OddsRatio
  P.value PC1 PC2 Representative Rle SNP SNPDosage Size Substitution
  Type aes aes_string anno annotate calls chromosome element_blank
  element_line element_text facet_grid featureData featureNames
  findOverlaps geom_boxplot geom_histogram geom_point geom_smooth
  geom_tile geom_vline ggplot ggsave ggtitle labs loadRegionDB
  makeGRangesFromDataFrame mclapply queryHits runLOLA
  scale_color_continuous scale_color_gradient2 scale_color_manual
  scale_fill_gradient2 seqlevelsStyle&lt;- subjectHits theme theme_bw xlab
  ylab
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                           user system elapsed
doMethQTL                13.340  0.556  13.906
doMethQTLChromosome      11.349  0.140  11.496
computeCorrelationBlocks  5.574  0.160   5.750
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘MAGAR.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 2 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/MAGAR.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">MAGAR.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL MAGAR
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘MAGAR’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
No methods found in package ‘oligoClasses’ for request: ‘mean’ when loading ‘crlmm’
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
No methods found in package ‘oligoClasses’ for request: ‘mean’ when loading ‘crlmm’
** testing if installed package can be loaded from final location
No methods found in package ‘oligoClasses’ for request: ‘mean’ when loading ‘crlmm’
** testing if installed package keeps a record of temporary installation path
* DONE (MAGAR)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">MAGAR.Rcheck/tests/runTests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; BiocGenerics:::testPackage(&quot;MAGAR&quot;)

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min


Attaching package: &#x27;MatrixGenerics&#x27;

The following objects are masked from &#x27;package:matrixStats&#x27;:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars


Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:Matrix&#x27;:

    expand, unname

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname


Attaching package: &#x27;DelayedArray&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    apply, rowsum, scale, sweep


Attaching package: &#x27;HDF5Array&#x27;

The following object is masked from &#x27;package:rhdf5&#x27;:

    h5ls


Attaching package: &#x27;bit&#x27;

The following object is masked from &#x27;package:base&#x27;:

    xor

Attaching package ff
- getOption(&quot;fftempdir&quot;)==&quot;/tmp/RtmpX26gjC/ff&quot;

- getOption(&quot;ffextension&quot;)==&quot;ff&quot;

- getOption(&quot;ffdrop&quot;)==TRUE

- getOption(&quot;fffinonexit&quot;)==TRUE

- getOption(&quot;ffpagesize&quot;)==65536

- getOption(&quot;ffcaching&quot;)==&quot;mmnoflush&quot;  -- consider &quot;ffeachflush&quot; if your system stalls on large writes

- getOption(&quot;ffbatchbytes&quot;)==16777216 -- consider a different value for tuning your system

- getOption(&quot;ffmaxbytes&quot;)==536870912 -- consider a different value for tuning your system


Attaching package: &#x27;ff&#x27;

The following object is masked from &#x27;package:DelayedArray&#x27;:

    maxlength

The following objects are masked from &#x27;package:utils&#x27;:

    write.csv, write.csv2

The following objects are masked from &#x27;package:base&#x27;:

    is.factor, is.ordered

Spam version 2.9-1 (2022-08-07) is loaded.
Type &#x27;help( Spam)&#x27; or &#x27;demo( spam)&#x27; for a short introduction 
and overview of this package.
Help for individual functions is also obtained by adding the
suffix &#x27;.spam&#x27; to the function name, e.g. &#x27;help( chol.spam)&#x27;.

Attaching package: &#x27;spam&#x27;

The following object is masked from &#x27;package:Matrix&#x27;:

    det

The following object is masked from &#x27;package:stats4&#x27;:

    mle

The following objects are masked from &#x27;package:base&#x27;:

    backsolve, forwardsolve


Try help(fields) to get started.

Attaching package: &#x27;gplots&#x27;

The following object is masked from &#x27;package:IRanges&#x27;:

    space

The following object is masked from &#x27;package:S4Vectors&#x27;:

    space

The following object is masked from &#x27;package:stats&#x27;:

    lowess


Attaching package: &#x27;grid&#x27;

The following object is masked from &#x27;package:ff&#x27;:

    pattern


Attaching package: &#x27;gridExtra&#x27;

The following object is masked from &#x27;package:BiocGenerics&#x27;:

    combine


Attaching package: &#x27;limma&#x27;

The following object is masked from &#x27;package:BiocGenerics&#x27;:

    plotMA

Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


Attaching package: &#x27;Biobase&#x27;

The following object is masked from &#x27;package:MatrixGenerics&#x27;:

    rowMedians

The following objects are masked from &#x27;package:matrixStats&#x27;:

    anyMissing, rowMedians


Attaching package: &#x27;scales&#x27;

The following object is masked from &#x27;package:viridis&#x27;:

    viridis_pal


Attaching package: &#x27;AnnotationDbi&#x27;

The following object is masked from &#x27;package:MASS&#x27;:

    select



Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:grid&#x27;:

    pattern

The following objects are masked from &#x27;package:ff&#x27;:

    mismatch, pattern

The following object is masked from &#x27;package:base&#x27;:

    strsplit

locfit 1.5-9.7 	 2023-01-02
Setting options(&#x27;download.file.method.GEOquery&#x27;=&#x27;auto&#x27;)
Setting options(&#x27;GEOquery.inmemory.gpl&#x27;=FALSE)

Attaching package: &#x27;plyr&#x27;

The following object is masked from &#x27;package:XVector&#x27;:

    compact

The following object is masked from &#x27;package:IRanges&#x27;:

    desc

The following object is masked from &#x27;package:S4Vectors&#x27;:

    rename

The following object is masked from &#x27;package:matrixStats&#x27;:

    count

Welcome to oligoClasses version 1.61.0

Attaching package: &#x27;oligoClasses&#x27;

The following object is masked from &#x27;package:minfi&#x27;:

    getM

No methods found in package &#x27;oligoClasses&#x27; for request: &#x27;mean&#x27; when loading &#x27;crlmm&#x27;
Welcome to crlmm version 1.57.0

Attaching package: &#x27;MAGAR&#x27;

The following object is masked from &#x27;package:foreach&#x27;:

    getResult

2023-02-22 07:34:47     1.2  STATUS STARTED Unit Testing
2023-02-22 07:34:47     1.2  STATUS     STARTED Testing constructors
Object of class MethQTLInput
	 Contains 3 samples
	 Methylation data for 5 CpGs
	 Genotyping data for 4 SNPs
	 Genome assembly: hg19 
Object of class MethQTLResult
	 Contains 10 methQTL
	 Contains 0 correlation blocks
	 methQTL called using classical.linear 
	 representative CpGs computed with row.medians 
2023-02-22 07:34:47     1.2  STATUS     COMPLETED Testing constructors
2023-02-22 07:34:47     1.2  STATUS     STARTED Testing options
2023-02-22 07:34:47     1.2  STATUS     COMPLETED Testing options
2023-02-22 07:34:47     1.2  STATUS     STARTED Testing cor blocks
2023-02-22 07:34:47     1.2  STATUS         STARTED Compute correlation blocks
2023-02-22 07:34:47     1.2  STATUS             STARTED Compute correlation matrix
2023-02-22 07:34:47     1.2  STATUS             COMPLETED Compute correlation matrix
2023-02-22 07:34:47     1.2  STATUS             STARTED Compute pairwise distances
2023-02-22 07:34:48     1.2  STATUS             COMPLETED Compute pairwise distances
2023-02-22 07:34:50     1.2  STATUS             STARTED Weight distances
2023-02-22 07:34:50     1.2  STATUS             COMPLETED Weight distances
2023-02-22 07:34:51     1.2  STATUS             STARTED Compute graph
2023-02-22 07:34:51     1.2  STATUS             COMPLETED Compute graph
2023-02-22 07:34:51     1.2  STATUS             STARTED Compute clustering
2023-02-22 07:34:52     1.2  STATUS             COMPLETED Compute clustering
2023-02-22 07:34:52     1.2  STATUS         COMPLETED Compute correlation blocks
2023-02-22 07:34:52     1.2  STATUS     COMPLETED Testing cor blocks
2023-02-22 07:34:52     1.2  STATUS     STARTED Test methQTL calling
2023-02-22 07:34:52     1.2    INFO         Loading default option setting
2023-02-22 07:34:53     1.2  STATUS         STARTED Imputation procedure knn 
2023-02-22 07:34:53     1.2  STATUS         COMPLETED Imputation procedure knn 
2023-02-22 07:34:53     1.2  STATUS         STARTED Computing methQTLs
2023-02-22 07:34:53     1.2  STATUS             STARTED Computing methQTL for chromosome chr18
2023-02-22 07:34:53     1.2  STATUS                 STARTED Compute methQTL per correlation block
2023-02-22 07:34:53     1.2  STATUS                     STARTED Setting up Multicore
2023-02-22 07:34:53     1.2    INFO                         Using 1 cores
2023-02-22 07:34:53     1.2  STATUS                     COMPLETED Setting up Multicore
2023-02-22 07:34:53     1.2  STATUS                 COMPLETED Compute methQTL per correlation block
2023-02-22 07:34:53     1.2  STATUS             COMPLETED Computing methQTL for chromosome chr18
2023-02-22 07:34:55     1.2  STATUS         COMPLETED Computing methQTLs
2023-02-22 07:34:55     1.2  STATUS     COMPLETED Test methQTL calling
2023-02-22 07:34:55     1.2  STATUS COMPLETED Unit Testing

Object of class MethQTLInput
	 Contains 3 samples
	 Methylation data for 5 CpGs
	 Genotyping data for 4 SNPs
	 Genome assembly: hg19 
Object of class MethQTLResult
	 Contains 10 methQTL
	 Contains 0 correlation blocks
	 methQTL called using classical.linear 
	 representative CpGs computed with row.medians 
2023-02-22 07:34:55     1.2    INFO Loading default option setting
2023-02-22 07:34:55     1.2  STATUS STARTED Imputation procedure knn 
2023-02-22 07:34:55     1.2  STATUS COMPLETED Imputation procedure knn 

2023-02-22 07:34:55     1.2  STATUS STARTED Computing methQTLs
2023-02-22 07:34:55     1.2  STATUS     STARTED Computing methQTL for chromosome chr18
2023-02-22 07:34:55     1.2  STATUS         STARTED Compute methQTL per correlation block
2023-02-22 07:34:55     1.2  STATUS             STARTED Setting up Multicore
2023-02-22 07:34:55     1.2    INFO                 Using 1 cores
2023-02-22 07:34:55     1.2  STATUS             COMPLETED Setting up Multicore
2023-02-22 07:34:55     1.3  STATUS         COMPLETED Compute methQTL per correlation block
2023-02-22 07:34:55     1.3  STATUS     COMPLETED Computing methQTL for chromosome chr18
2023-02-22 07:34:57     1.3  STATUS COMPLETED Computing methQTLs

2023-02-22 07:34:57     1.3  STATUS STARTED Compute correlation blocks
2023-02-22 07:34:57     1.3  STATUS     STARTED Compute correlation matrix
2023-02-22 07:34:57     1.3  STATUS     COMPLETED Compute correlation matrix
2023-02-22 07:34:57     1.3  STATUS     STARTED Compute pairwise distances
2023-02-22 07:34:58     1.3  STATUS     COMPLETED Compute pairwise distances
2023-02-22 07:35:00     1.3  STATUS     STARTED Weight distances
2023-02-22 07:35:00     1.3  STATUS     COMPLETED Weight distances
2023-02-22 07:35:01     1.3  STATUS     STARTED Compute graph
2023-02-22 07:35:01     1.3  STATUS     COMPLETED Compute graph
2023-02-22 07:35:01     1.3  STATUS     STARTED Compute clustering
2023-02-22 07:35:02     1.3  STATUS     COMPLETED Compute clustering
2023-02-22 07:35:02     1.3  STATUS COMPLETED Compute correlation blocks



RUNIT TEST PROTOCOL -- Wed Feb 22 07:35:02 2023 
*********************************************** 
Number of test functions: 4 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
MAGAR RUnit Tests - 4 test functions, 0 errors, 0 failures
Number of test functions: 4 
Number of errors: 0 
Number of failures: 0 
&gt; 
&gt; proc.time()
   user  system elapsed 
 40.152   2.492  42.648 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">MAGAR.Rcheck/MAGAR-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>computeCorrelationBlocks</TD><TD style="text-align: right;">5.574</TD><TD style="text-align: right;">0.160</TD><TD style="text-align: right;">5.750</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>doImport</TD><TD style="text-align: right;">0.078</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.082</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>doMethQTL</TD><TD style="text-align: right;">13.340</TD><TD style="text-align: right;"> 0.556</TD><TD style="text-align: right;">13.906</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>doMethQTLChromosome</TD><TD style="text-align: right;">11.349</TD><TD style="text-align: right;"> 0.140</TD><TD style="text-align: right;">11.496</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filterPval</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.030</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getAnno</TD><TD style="text-align: right;">0.104</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.113</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getCorrelationBlocks</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getGeno</TD><TD style="text-align: right;">0.131</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.137</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getMethData</TD><TD style="text-align: right;">0.104</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.111</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getOverlapUniverse</TD><TD style="text-align: right;">0.155</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.159</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getOverlappingQTL</TD><TD style="text-align: right;">0.022</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.026</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getPheno</TD><TD style="text-align: right;">0.072</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.078</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getResult</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getResultGWASMap</TD><TD style="text-align: right;">0.103</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.103</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getSamples</TD><TD style="text-align: right;">0.077</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.081</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getSpecificQTL</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.022</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>imputeMeth</TD><TD style="text-align: right;">0.202</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.210</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>joinMethQTLResult</TD><TD style="text-align: right;">0.073</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.085</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loadMethQTLInput</TD><TD style="text-align: right;">0.075</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.076</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loadMethQTLResult</TD><TD style="text-align: right;">0.021</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.021</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>overlapInputs</TD><TD style="text-align: right;">0.077</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.077</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>overlapQTLs</TD><TD style="text-align: right;">0.018</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.022</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qtlAnnotationEnrichment</TD><TD style="text-align: right;">1.117</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">1.159</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qtlBaseSubstitutionEnrichment</TD><TD style="text-align: right;">0.073</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.077</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qtlDistanceScatterplot</TD><TD style="text-align: right;">0.657</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.673</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qtlGetOption</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qtlJSON2options</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qtlManhattanPlot</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.042</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qtlOptions2JSON</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qtlPlotBaseSubstitution</TD><TD style="text-align: right;">0.083</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.084</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qtlPlotClusterSize</TD><TD style="text-align: right;">0.296</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.304</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qtlPlotSNPCpGInteraction</TD><TD style="text-align: right;">0.333</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.332</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qtlSetOption</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qtlTFBSMotifEnrichment</TD><TD style="text-align: right;">0.025</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.025</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qtlUpSetPlotCorBlocks</TD><TD style="text-align: right;">1.767</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">1.811</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qtlUpSetPlotTagCpGs</TD><TD style="text-align: right;">0.681</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.689</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qtlUpsetPlot</TD><TD style="text-align: right;">0.612</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.612</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qtlVennPlot</TD><TD style="text-align: right;">0.458</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">0.479</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>saveMethQTLInput</TD><TD style="text-align: right;">0.248</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.268</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>saveMethQTLResult</TD><TD style="text-align: right;">0.113</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.117</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
