<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for VanillaICE on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><B>[<A href="..#V">V</A>]</B></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-16 06:07:06 -0000 (Tue, 16 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4787</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for VanillaICE on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the VanillaICE package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/VanillaICE.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>2144</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">VanillaICE</A>&nbsp;1.63.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/VanillaICE">landing page</A>)</SPAN><BR>Robert Scharpf<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-13&nbsp;23:12:22&nbsp;-0000&nbsp;(Sat,&nbsp;13&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/VanillaICE</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">9e822b0</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;13:45:56&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: VanillaICE</TD></TR>
<TR><TD><B>Version</B>: 1.63.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:VanillaICE.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings VanillaICE_1.63.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-15 17:07:22 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-15 17:14:29 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 426.6 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: VanillaICE.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 1</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:VanillaICE.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings VanillaICE_1.63.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/VanillaICE.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘VanillaICE/DESCRIPTION’ ... OK
* this is package ‘VanillaICE’ version ‘1.63.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Packages which this enhances but not available for checking:
  &#x27;doMPI&#x27;, &#x27;doRedis&#x27;
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘VanillaICE’ can be installed ... OK
* used C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking startup messages can be suppressed ... NOTE
No methods found in package ‘oligoClasses’ for request: ‘mean’ when loading ‘crlmm’

It looks like this package (or a package it requires) has a startup
message which cannot be suppressed: see ?packageStartupMessage.
* checking dependencies in R code ... NOTE
Namespaces in Imports field not imported from:
  ‘BSgenome.Hsapiens.UCSC.hg18’ ‘MatrixGenerics’
  All declared Imports should be used.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... WARNING
checkRd: (5) VanillaICE.Rd:0-7: Must have a \description
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... OK
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking include directives in Makefiles ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                  user system elapsed
ArrayViews-class 5.365  0.432   5.801
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘doRUnit.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘VanillaICE.Rnw’ using ‘UTF-8’... OK
  ‘crlmmDownstream.Rnw’ using ‘UTF-8’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 WARNING, 4 NOTEs
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/VanillaICE.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">VanillaICE.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD INSTALL VanillaICE
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0-bin/site-library’
* installing *source* package ‘VanillaICE’ ...
** using staged installation
** libs
using C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
gcc -I&quot;/home/biocbuild/R/R-4.3.0-bin/lib/R/include&quot; -DNDEBUG   -I/usr/local/include    -fPIC  -g -O2  -c rinit.c -o rinit.o
gcc -I&quot;/home/biocbuild/R/R-4.3.0-bin/lib/R/include&quot; -DNDEBUG   -I/usr/local/include    -fPIC  -g -O2  -c viterbi.c -o viterbi.o
gcc -shared -L/home/biocbuild/R/R-4.3.0-bin/lib/R/lib -L/usr/local/lib -o VanillaICE.so rinit.o viterbi.o -lm -L/home/biocbuild/R/R-4.3.0-bin/lib/R/lib -lR
installing to /home/biocbuild/R/R-4.3.0-bin/site-library/00LOCK-VanillaICE/00new/VanillaICE/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
No methods found in package ‘oligoClasses’ for request: ‘mean’ when loading ‘crlmm’
Creating a new generic function for ‘distance’ in package ‘VanillaICE’
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
No methods found in package ‘oligoClasses’ for request: ‘mean’ when loading ‘crlmm’
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
No methods found in package ‘oligoClasses’ for request: ‘mean’ when loading ‘crlmm’
** testing if installed package keeps a record of temporary installation path
* DONE (VanillaICE)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">VanillaICE.Rcheck/tests/doRUnit.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; ## from xmapcore package
&gt; if( require( &quot;RUnit&quot;, quietly=TRUE ) ) {
+ 	pkg &lt;- &quot;VanillaICE&quot;
+ 
+ 	if( Sys.getenv( &quot;RCMDCHECK&quot; ) == &quot;FALSE&quot; ) {
+ 		path &lt;- file.path( getwd(), &quot;..&quot;, &quot;inst&quot;, &quot;unitTests&quot; )
+ 	} else {
+ 		path &lt;- system.file( package=pkg, &quot;unitTests&quot; )
+ 	}
+ 
+ 	cat( &quot;\nRunning unit tests\n&quot; )
+ 	print( list( pkg=pkg, getwd=getwd(), pathToUnitTests=path ) )
+ 	library( package=pkg, character.only=TRUE )
+ 
+ 	##xmap.clear.cache()
+ 
+ 	##Fail on warnings
+ 	options( warn=2 )
+ 
+ 	## Get the pattern (if there is one?)
+ 	patt &lt;- Sys.getenv( &quot;RUNITFILEPATTERN&quot; )
+ 	if( is.null( patt ) || nchar( patt ) == 0 ) {
+             testSuite &lt;- defineTestSuite(name=paste( pkg, &quot;unit testing&quot; ),
+                                          rngKind=&quot;Mersenne-Twister&quot;,
+                                          dirs=path,
+                                          testFileRegexp=paste( &quot;^test.+&quot;, patt, &quot;\\.[rR]$&quot;, sep=&quot;&quot; ))
+ 	} else {
+ 		##testSuite &lt;- defineTestSuite( name=paste( pkg, &quot;unit testing&quot; ), testFileRegexp=paste( &quot;^runit\\.&quot;, patt, &quot;\\.[rR]$&quot;, sep=&quot;&quot; ), dirs=path )
+             testSuite &lt;- defineTestSuite(name=paste( pkg, &quot;unit testing&quot; ),
+                                          rngKind=&quot;Mersenne-Twister&quot;,
+                                          testFileRegexp=paste( &quot;^test.+&quot;, patt, &quot;\\.[rR]$&quot;, sep=&quot;&quot; ),
+                                          dirs=path )
+ 	}
+ 	tests &lt;- runTestSuite( testSuite )
+ 
+ 	pathReport &lt;- file.path( path, &quot;report&quot; )
+ 
+ 	cat( &quot;------------------- UNIT TEST SUMMARY ---------------------\n\n&quot; )
+ 	printTextProtocol( tests, showDetails=FALSE )
+ 	printTextProtocol( tests, showDetails=FALSE, fileName=paste( pathReport, &quot;Summary.txt&quot;, sep=&quot;&quot; ) )
+ 	printTextProtocol( tests, showDetails=TRUE,  fileName=paste( pathReport, &quot;.txt&quot;, sep=&quot;&quot; ) )
+ 
+ 	printHTMLProtocol( tests, fileName=paste( pathReport, &quot;.html&quot;, sep=&quot;&quot; ) )
+ 
+ 	tmp &lt;- getErrors( tests )
+ 	if( tmp$nFail &gt; 0 | tmp$nErr &gt; 0 ){
+ 		stop( paste( &quot;\n\nunit testing failed (#test failures: &quot;, tmp$nFail, &quot;, #R errors: &quot;,  tmp$nErr, &quot;)\n\n&quot;, sep=&quot;&quot;))
+ 	}
+ } else {
+ 	warning( &quot;cannot run unit tests -- package RUnit is not available&quot; )
+ }

Running unit tests
$pkg
[1] &quot;VanillaICE&quot;

$getwd
[1] &quot;/home/biocbuild/bbs-3.18-bioc/meat/VanillaICE.Rcheck/tests&quot;

$pathToUnitTests
[1] &quot;/tmp/RtmpyhCyuH/RLIBS_979fb14f4c845/VanillaICE/unitTests&quot;

Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: GenomicRanges
Loading required package: stats4
Loading required package: S4Vectors

Attaching package: &#x27;S4Vectors&#x27;

The following object is masked from &#x27;package:utils&#x27;:

    findMatches

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
Loading required package: SummarizedExperiment
Loading required package: MatrixGenerics
Loading required package: matrixStats

Attaching package: &#x27;MatrixGenerics&#x27;

The following objects are masked from &#x27;package:matrixStats&#x27;:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars

Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


Attaching package: &#x27;Biobase&#x27;

The following object is masked from &#x27;package:MatrixGenerics&#x27;:

    rowMedians

The following objects are masked from &#x27;package:matrixStats&#x27;:

    anyMissing, rowMedians

No methods found in package &#x27;oligoClasses&#x27; for request: &#x27;mean&#x27; when loading &#x27;crlmm&#x27;
Welcome to VanillaICE version 1.63.0
Loading required package: BSgenome.Hsapiens.UCSC.hg18
Loading required package: BSgenome
Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:VanillaICE&#x27;:

    deletion

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: rtracklayer
Loading required package: data.table

Attaching package: &#x27;data.table&#x27;

The following object is masked from &#x27;package:SummarizedExperiment&#x27;:

    shift

The following object is masked from &#x27;package:GenomicRanges&#x27;:

    shift

The following object is masked from &#x27;package:IRanges&#x27;:

    shift

The following objects are masked from &#x27;package:S4Vectors&#x27;:

    first, second

Loading required package: foreach
Loading required package: oligoClasses
Welcome to oligoClasses version 1.63.0


Executing test function test_ArrayViews  ...  done successfully.



Executing test function test_columnSubset  ... Writing parsed files to /tmp/RtmpnWtXOY
 done successfully.



Executing test function test_FilterParam  ...  done successfully.



Executing test function test_EmissionParam  ...  done successfully.



Executing test function test_SnpArrayExperiment  ... Error in (function (classes, fdef, mtable)  : 
  unable to find an inherited method for function &#x27;SnpDataFrame&#x27; for signature &#x27;&quot;numeric&quot;&#x27;
Error in VanillaICE:::SnpDataFrame(DataFrame(x = 3)) : 
  isSnp must be specified
Error in .local(object, isSnp, ...) : isSnp must be specified
Error in validObject(.Object) : 
  invalid class &quot;RangedSummarizedExperiment&quot; object: 
    &#x27;x@assays&#x27; is not parallel to &#x27;x&#x27;
Error in SnpArrayExperiment(x, y, rowRanges = rowranges) : 
   isSnp must be the same length as rowRanges
 done successfully.



Executing test function test_SnpArrayExperiment2  ...  done successfully.



Executing test function test_SnpGRanges  ...  done successfully.



Executing test function test_acf2  ...  done successfully.



Executing test function test_hmm_cnset  ... Loading required package: crlmm
Loading required package: preprocessCore
Welcome to crlmm version 1.59.0

Attaching package: &#x27;crlmm&#x27;

The following object is masked from &#x27;package:VanillaICE&#x27;:

    genotypes

 done successfully.



Executing test function test_state4  ...  done successfully.



Executing test function test_rowMAD  ...  done successfully.



Executing test function test_scaleBy  ...  done successfully.



Executing test function test_duplicatedMapLocs  ...  done successfully.



Executing test function test_sweepMode.R  ...  done successfully.



Executing test function test_Viterbi  ...  done successfully.



Executing test function test_baf_emission  ...  done successfully.



Executing test function test_cn_NAs  ...  done successfully.



Executing test function test_emission_for_list  ...  done successfully.



Executing test function test_emission_update  ...  done successfully.



Executing test function test_multiple_chromosomes  ...  done successfully.



Executing test function test_null_assignment  ...  done successfully.



Executing test function test_oligoset_comparison  ...  done successfully.



Executing test function test_summarized_exp  ...  done successfully.



Executing test function test_updating  ...  done successfully.

------------------- UNIT TEST SUMMARY ---------------------

RUNIT TEST PROTOCOL -- Mon May 15 17:12:56 2023 
*********************************************** 
Number of test functions: 24 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
VanillaICE unit testing - 24 test functions, 0 errors, 0 failures
&gt; 
&gt; proc.time()
   user  system elapsed 
 28.808   1.034  29.830 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">VanillaICE.Rcheck/VanillaICE-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>ArrayViews-class</TD><TD style="text-align: right;">5.365</TD><TD style="text-align: right;">0.432</TD><TD style="text-align: right;">5.801</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>CopyNumScanParams</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EmissionParam-methods</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>FilterParam-class</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>HMM</TD><TD style="text-align: right;">0.318</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">0.350</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>HMMList-class</TD><TD style="text-align: right;">0.951</TD><TD style="text-align: right;">0.120</TD><TD style="text-align: right;">1.071</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>HmmParam</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>IO</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>IdiogramParams-class</TD><TD style="text-align: right;">0.147</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.162</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SnpArrayExperiment-class</TD><TD style="text-align: right;">0.096</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.104</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SnpExperiment</TD><TD style="text-align: right;">0.138</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.155</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SnpGRanges</TD><TD style="text-align: right;">0.074</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.074</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TransitionParam</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cnvFilter</TD><TD style="text-align: right;">1.967</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">2.010</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dropDuplicatedMapLocs</TD><TD style="text-align: right;">0.145</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.158</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>emissionParam</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getExampleSnpExperiment</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getHmmParams</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>hmm2</TD><TD style="text-align: right;">1.356</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">1.372</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>isHeterozygous</TD><TD style="text-align: right;">0.271</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.290</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>parseSourceFile</TD><TD style="text-align: right;">0.438</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.474</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotting</TD><TD style="text-align: right;">1.055</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">1.083</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>robust-statistics</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpArrayAssays</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sourcePaths</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sweepMode</TD><TD style="text-align: right;">0.261</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.261</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>threshold</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>viewports</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
