<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for MungeSumstats on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><B>[<A href="..#M">M</A>]</B></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-10 10:04:36 -0000 (Wed, 10 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-03-12&nbsp;r83975)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">6211</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for MungeSumstats on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the MungeSumstats package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/MungeSumstats.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1320</B>/2194</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">MungeSumstats</A>&nbsp;1.9.6</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/MungeSumstats">landing page</A>)</SPAN><BR>Alan Murphy<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-08&nbsp;19:11:19&nbsp;-0000&nbsp;(Mon,&nbsp;08&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/MungeSumstats</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">3c91c85</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-05-05&nbsp;09:31:26&nbsp;-0000&nbsp;(Fri,&nbsp;05&nbsp;May&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: MungeSumstats</TD></TR>
<TR><TD><B>Version</B>: 1.9.6</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:MungeSumstats.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings MungeSumstats_1.9.6.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-09 22:36:22 -0000 (Tue, 09 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-09 23:01:10 -0000 (Tue, 09 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 1488.1 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: MungeSumstats.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:MungeSumstats.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings MungeSumstats_1.9.6.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/MungeSumstats.Rcheck’
* using R Under development (unstable) (2023-03-12 r83975)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘MungeSumstats/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘MungeSumstats’ version ‘1.9.6’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘MungeSumstats’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking startup messages can be suppressed ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking R/sysdata.rda ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                    user system elapsed
get_genome_builds 83.114  6.485  91.163
format_sumstats   52.159  7.192  60.829
liftover           1.231  0.128   6.707
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘MungeSumstats.Rmd’ using ‘UTF-8’... OK
  ‘OpenGWAS.Rmd’ using ‘UTF-8’... OK
  ‘docker.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: OK

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">MungeSumstats.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL MungeSumstats
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library’
* installing *source* package ‘MungeSumstats’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (MungeSumstats)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">MungeSumstats.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(MungeSumstats)
&gt; 
&gt; test_check(&quot;MungeSumstats&quot;)
Collecting metadata from Open GWAS.
Filtering metadata by substring criteria.
Found 3 GWAS datasets matching search criteria across:
   - 3 trait(s)
   - 1 population(s)
   - 2 category(ies)
   - 2 subcategory(ies)
   - 2 publication(s)
   - 2 consortia(ium)
   - 1 genome build(s)
Collecting metadata from Open GWAS.
Filtering metadata by substring criteria.
Filtering metadata by sample/case/control/SNP size criteria.
Excluding sample/case/control size with NAs.
Found 3 GWAS datasets matching search criteria across:
   - 3 trait(s)
   - 1 population(s)
   - 2 category(ies)
   - 2 subcategory(ies)
   - 2 publication(s)
   - 2 consortia(ium)
   - 1 genome build(s)
Collecting metadata from Open GWAS.
Filtering metadata by substring criteria.
Found 45 GWAS datasets matching search criteria across:
   - 42 trait(s)
   - 3 population(s)
   - 2 category(ies)
   - 2 subcategory(ies)
   - 7 publication(s)
   - 5 consortia(ium)
   - 1 genome build(s)
Downloading VCF ==&gt; /tmp/RtmpiIp3aX/ieu-a-298.vcf.gz
Downloading with download.file.
trying URL &#x27;https://gwas.mrcieu.ac.uk/files/ieu-a-298/ieu-a-298.vcf.gz&#x27;
Content type &#x27;application/gzip&#x27; length 234480 bytes (228 KB)
==================================================
downloaded 228 KB

Downloading VCF index ==&gt; https://gwas.mrcieu.ac.uk/files/ieu-a-298/ieu-a-298.vcf.gz.tbi
Downloading with download.file.
trying URL &#x27;https://gwas.mrcieu.ac.uk/files/ieu-a-298/ieu-a-298.vcf.gz.tbi&#x27;
Content type &#x27;application/gzip&#x27; length 37803 bytes (36 KB)
==================================================
downloaded 36 KB

Processing 1 datasets from Open GWAS.

========== Processing dataset : a-fake-id ==========

Downloading VCF ==&gt; /tmp/RtmpiIp3aX/a-fake-id.vcf.gz
Downloading with download.file.
trying URL &#x27;https://gwas.mrcieu.ac.uk/files/a-fake-id/a-fake-id.vcf.gz&#x27;
Processing 1 datasets from Open GWAS.

========== Processing dataset : ieu-a-298 ==========

Using previously downloaded VCF.
Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/ieu-a-298/ieu-a-298.tsv.gz
Loading required namespace: GenomicFiles
Using local VCF.
File already tabix-indexed.
Finding empty VCF columns based on first 10,000 rows.
1 sample detected: ieu-a-298
Constructing ScanVcfParam object.
VCF contains: 10,684 variant(s) x 1 sample(s)
Reading VCF file: single-threaded
Converting VCF to data.table.
Expanding VCF first, so number of rows may increase.
Checking for empty columns.
Unlisting 4 columns.
Time difference of 0.8 secs
VCF data.table contains: 10,684 rows x 11 columns.
Time difference of 1.6 secs
Renaming ID as SNP.
VCF file has -log10 P-values; these will be converted to unadjusted p-values in the &#x27;P&#x27; column.
No INFO (SI) column detected.
Standardising column headers.
First line of summary statistics file: 
SNP	chr	BP	end	REF	ALT	FILTER	ES	SE	LP	SS	P	
Summary statistics report:
   - 10,684 rows
   - 10,684 unique variants
   - 553 genome-wide significant variants (P&lt;5e-8)
   - 22 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Ensuring that the N column is all integers.
The sumstats N column is not all integers, this could effect downstream analysis. These will be converted to integers.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
N already exists within sumstats_dt.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/ieu-a-298/ieu-a-298.tsv.gz
Summary statistics report:
   - 10,684 rows (100% of original 10,684 rows)
   - 10,684 unique variants
   - 553 genome-wide significant variants (P&lt;5e-8)
   - 22 chromosomes
Done munging in 0.097 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR      BP A1 A2     END FILTER   BETA     SE      LP     N
1: rs76805690   1 2256288  C  A 2256288   PASS 0.0389 0.0175 1.58536 74046
2: rs75379543   1 2261983  C  A 2261983   PASS 0.0427 0.0167 1.96738 74046
3: rs75273719   1 2263666  G  A 2263666   PASS 0.0502 0.0171 2.47353 74046
4:   rs903904   1 2263888  C  T 2263888   PASS 0.0413 0.0169 1.82769 74046
             P
1: 0.025980051
2: 0.010780031
3: 0.003361012
4: 0.014869967
Returning path to saved data.

ieu-a-298 : Done in 0.097 minutes.

Done with all processing in 0.1 minutes.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd2d795900.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd6f82826c
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A0	A1	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd2d795900.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fdb939eb1.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd6f82826c
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fdb939eb1.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd203ca29.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd78deb18e
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A2	A1	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for correct direction of A1 (reference) and A2 (alternative allele).
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 93 SNPs using BSgenome::snpsById...
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

BSgenome::snpsById done in 51 seconds.
There are 47 SNPs where A1 doesn&#x27;t match the reference genome.
These will be flipped with their effect columns.
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicate SNPs from SNP ID.
Checking for SNPs with duplicated base-pair positions.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Checking for bi-allelic SNPs.
Warning: When method is an integer, must be &gt;0.
67 SNPs (72%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd203ca29.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.866 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  G  A 0.63060 -0.017 0.003 2.359e-10
3: rs34305371   1 72733610  G  A 0.91231 -0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd6e6d4105.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd78deb18e
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Ensuring all SNPs are on the reference genome.
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 93 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 27 seconds.
Checking for correct direction of A1 (reference) and A2 (alternative allele).
There are 46 SNPs where A1 doesn&#x27;t match the reference genome.
These will be flipped with their effect columns.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicate SNPs from SNP ID.
Checking for SNPs with duplicated base-pair positions.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Checking for bi-allelic SNPs.
Warning: When method is an integer, must be &gt;0.
67 SNPs (72%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd6e6d4105.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.476 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  G  A 0.63060 -0.017 0.003 2.359e-10
3: rs34305371   1 72733610  G  A 0.91231 -0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd6ab6e358.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd3dae6b15
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicate SNPs from SNP ID.
Checking for SNPs with duplicated base-pair positions.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Checking for bi-allelic SNPs.
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 93 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 26 seconds.
1 SNPs are non-biallelic. These will be removed.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/snp_bi_allelic.tsv.gz
Warning: When method is an integer, must be &gt;0.
46 SNPs (50%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd6ab6e358.tsv.gz
Summary statistics report:
   - 92 rows (98.9% of original 93 rows)
   - 92 unique variants
   - 69 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.462 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd30fc347d.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd3dae6b15
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicate SNPs from SNP ID.
Checking for SNPs with duplicated base-pair positions.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Checking for bi-allelic SNPs.
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 93 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 22 seconds.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd30fc347d.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.398 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Sorting coordinates with &#x27;data.table&#x27;.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd2b52986b.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Found 1 Indels. These will be removed from the sumstats. 
WARNING If you want to keep Indels, set the drop_indel param to FALSE &amp; rerun MungeSumstats::format_sumstats()
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/indel.tsv.gz


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd6866a803.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd378c1971
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Ensuring parameters comply with LDSC format.
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
1 SNP IDs are not correctly formatted. These will be corrected from the reference genome.
Loading SNPlocs data.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Ensuring all SNPs are on the reference genome.
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 93 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 24 seconds.
Checking for correct direction of A1 (reference) and A2 (alternative allele).
There are 46 SNPs where A1 doesn&#x27;t match the reference genome.
These will be flipped with their effect columns.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicate SNPs from SNP ID.
Checking for SNPs with duplicated base-pair positions.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Checking for bi-allelic SNPs.
Computing Z-score from P using formula: `sign(BETA)*sqrt(stats::qchisq(P,1,lower=FALSE)`
Assigning N=1001 for all SNPs.
67 SNPs (72%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd6866a803.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.428 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P IMPUTATION_SNP
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08             NA
2: rs11210860   1 43982527  G  A 0.63060 -0.017 0.003 2.359e-10             NA
3: rs34305371   1 72733610  G  A 0.91231 -0.035 0.005 3.762e-14             NA
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08             NA
   flipped         Z IMPUTATION_z_score_p    N
1:      NA  5.630777                 TRUE 1001
2:    TRUE -6.335939                 TRUE 1001
3:    TRUE -7.568968                 TRUE 1001
4:      NA -5.630488                 TRUE 1001
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd6659357b.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd2a701bb1
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	N_CON	N_CAS	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Computing effective sample size using the LDSC method:
 Neff = (N_CAS+N_CON) * (N_CAS/(N_CAS+N_CON)) / mean((N_CAS/(N_CAS+N_CON))[(N_CAS+N_CON)==max(N_CAS+N_CON)]))
Computing sample size using the sum method:
 N = N_CAS + N_CON
Computing effective sample size using the GIANT method:
 Neff = 2 / (1/N_CAS + 1/N_CON)
Computing effective sample size using the METAL method:
 Neff = 4 / (1/N_CAS + 1/N_CON)
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd6659357b.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P N_CON N_CAS
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08   100   120
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10   100   120
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14   100   120
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08   100   120
   Neff_ldsc   N Neff_giant Neff_metal
1:       220 220        109        218
2:       220 220        109        218
3:       220 220        109        218
4:       220 220        109        218
Returning path to saved data.
Using local VCF.
bgzip-compressing VCF file.
Finding empty VCF columns based on first 10,000 rows.
Dropping 1 duplicate column(s).
1 sample detected: EBI-a-GCST005647
Constructing ScanVcfParam object.
VCF contains: 39,630,630 variant(s) x 1 sample(s)
Reading VCF file: single-threaded
Converting VCF to data.table.
Expanding VCF first, so number of rows may increase.
Dropping 1 duplicate column(s).
Checking for empty columns.
Unlisting 3 columns.
Dropped 314 duplicate rows.
Time difference of 0.1 secs
VCF data.table contains: 101 rows x 11 columns.
Time difference of 0.6 secs
Renaming ID as SNP.
VCF file has -log10 P-values; these will be converted to unadjusted p-values in the &#x27;P&#x27; column.
No INFO (SI) column detected.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd6e056f28.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd17421870
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
SNP	chr	BP	end	REF	ALT	FILTER	AF	ES	LP	SE	P	N	
Summary statistics report:
   - 101 rows
   - 101 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
N already exists within sumstats_dt.
2 SNPs (2%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd6e056f28.tsv.gz
Summary statistics report:
   - 101 rows (100% of original 101 rows)
   - 101 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
           SNP CHR    BP A1 A2   END FILTER    FRQ    BETA       LP     SE
1:  rs58108140   1 10583  G  A 10583   PASS 0.1589  0.0312 0.369267 0.0393
2:    rs806731   1 30923  G  T 30923   PASS 0.7843 -0.0114 0.126854 0.0353
3: rs116400033   1 51479  T  A 51479   PASS 0.1829  0.0711 1.262410 0.0370
4: rs146477069   1 54421  A  G 54421   PASS 0.0352 -0.0240 0.112102 0.0830
            P      N
1: 0.42730011 293723
2: 0.74669974 293723
3: 0.05464998 293723
4: 0.77249913 293723
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd42461884.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Standardising column headers.
First line of summary statistics file: 
SNP	chr	BP	end	REF	ALT	FILTER	AF	ES	LP	SE	P	N	Beta	
Summary statistics report:
   - 101 rows
   - 101 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
N already exists within sumstats_dt.
2 SNPs (2%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd42461884.tsv.gz
Summary statistics report:
   - 101 rows (100% of original 101 rows)
   - 101 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
           SNP CHR    BP A1 A2   END FILTER    FRQ      ES       LP     SE
1:  rs58108140   1 10583  G  A 10583   PASS 0.1589  0.0312 0.369267 0.0393
2:    rs806731   1 30923  G  T 30923   PASS 0.7843 -0.0114 0.126854 0.0353
3: rs116400033   1 51479  T  A 51479   PASS 0.1829  0.0711 1.262410 0.0370
4: rs146477069   1 54421  A  G 54421   PASS 0.0352 -0.0240 0.112102 0.0830
            P      N    BETA
1: 0.42730011 293723  0.0312
2: 0.74669974 293723 -0.0114
3: 0.05464998 293723  0.0711
4: 0.77249913 293723 -0.0240
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd29760c8f.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd17421870
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
SNP	chr	BP	end	REF	ALT	FILTER	AF	ES	LP	P	N	
Summary statistics report:
   - 101 rows
   - 101 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
The sumstats SE column is not present...Deriving SE from Beta and P
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
N already exists within sumstats_dt.
2 SNPs (2%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd29760c8f.tsv.gz
Summary statistics report:
   - 101 rows (100% of original 101 rows)
   - 101 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
           SNP CHR    BP A1 A2   END FILTER    FRQ    BETA       LP          P
1:  rs58108140   1 10583  G  A 10583   PASS 0.1589  0.0312 0.369267 0.42730011
2:    rs806731   1 30923  G  T 30923   PASS 0.7843 -0.0114 0.126854 0.74669974
3: rs116400033   1 51479  T  A 51479   PASS 0.1829  0.0711 1.262410 0.05464998
4: rs146477069   1 54421  A  G 54421   PASS 0.0352 -0.0240 0.112102 0.77249913
        N         SE IMPUTATION_SE
1: 293723 0.03930361          TRUE
2: 293723 0.03529477          TRUE
3: 293723 0.03699948          TRUE
4: 293723 0.08301411          TRUE
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd1ee06286.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd17421870
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
SNP	CHR	BP	A1	A2	FRQ	Z	SE	P	N	
Summary statistics report:
   - 25 rows
   - 25 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
The sumstats BETA column is not present...Deriving BETA from Z and SE
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
N already exists within sumstats_dt.
13 SNPs (52%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd1ee06286.tsv.gz
Summary statistics report:
   - 25 rows (100% of original 25 rows)
   - 25 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
           SNP CHR     BP A1 A2       FRQ      Z          SE      P      N
1:  rs12184267   1 715265  C  T 0.9591931 -0.916 0.007518884 0.3598 225955
2:  rs12184277   1 715367  A  G 0.9589313 -0.656 0.007491601 0.5116 226215
3:  rs12184279   1 717485  C  A 0.9594241 -1.050 0.007534860 0.2938 226224
4: rs116801199   1 720381  G  T 0.9578380 -0.300 0.007391344 0.7644 226626
           BETA IMPUTATION_BETA
1: -0.006887298            TRUE
2: -0.004914490            TRUE
3: -0.007911603            TRUE
4: -0.002217403            TRUE
Returning path to saved data.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Sorting coordinates with &#x27;data.table&#x27;.
Filtering SNPs based on INFO score.
46 SNPs are below the INFO threshold of 0.9 and will be removed.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/info_filter.tsv.gz
INFO_filter==0. Skipping INFO score filtering step.
Filtering SNPs based on INFO score.
All rows have INFO&gt;=0.9
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Sorting coordinates with &#x27;data.table&#x27;.
3 p-values are &gt;1 which LDSC/MAGMA may not be able to handle. These will be converted to 1.
5 p-values are &lt;0 which LDSC/MAGMA may not be able to handle. These will be converted to 0.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Sorting coordinates with &#x27;data.table&#x27;.
8 p-values are &lt;=5e-324 which LDSC/MAGMA may not be able to handle. These will be converted to 0.
Reading header.
Tabular format detected.
Reading header.
Tabular format detected.
Reading header.
Tabular format detected.
Reading header.
VCF format detected.This will be converted to a standardised table format.
Importing tabular file: /home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/MungeSumstats/extdata/eduAttainOkbay.txt
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Computing Z-score from P using formula: `sign(BETA)*sqrt(stats::qchisq(P,1,lower=FALSE)`
Standardising column headers.
First line of summary statistics file: 
SNP	CHR	BP	A1	A2	FRQ	BETA	SE	P	Z	newZ	
Computing Z-score from BETA ans SE using formula: `BETA/SE`


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd477dbfda.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd51cfd7e6
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	EAF	Beta	SE	Pval	CHR_BP_A2_A1	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Summary statistics file does not have obvious CHR/BP columns. Checking to see if they are joined in another column.
Column CHR_BP_A2_A1 has been separated into the columns CHR, BP, A2, A1
If this is the incorrect format for the column, update the column name to the correct format e.g.`CHR:BP:A2:A1` and format_sumstats().
Standardising column headers.
First line of summary statistics file: 
SNP	FRQ	BETA	SE	P	CHR	BP	A2	A1	
Checking for incorrect base-pair positions
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd477dbfda.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd4c86f836.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd51cfd7e6
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd4c86f836.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd1b9266e7.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd6c069d6
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	EAF	Beta	SE	Pval	CHR_BP_A2_A1	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Summary statistics file does not have obvious CHR/BP columns. Checking to see if they are joined in another column.
Column CHR_BP_A2_A1 has been separated into the columns CHR, BP, A2, A1
If this is the incorrect format for the column, update the column name to the correct format e.g.`CHR:BP:A2:A1` and format_sumstats().
Standardising column headers.
First line of summary statistics file: 
SNP	FRQ	BETA	SE	P	CHR	BP	A2	A1	
Checking for incorrect base-pair positions
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd1b9266e7.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd5f56fe1b.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd6c069d6
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd5f56fe1b.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd22cb20b.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fdd3b3c3f
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	EAF	Beta	SE	Pval	alleles	allele	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Warning: Multiple columns in the sumstats file seem to relate to alleles A1&gt;A2.
The column ALLELES will be kept whereas the column(s) ALLELE will be removed.
If this is not the correct column to keep, please remove all incorrect columns from those listed here before 
running `format_sumstats()`.
Column ALLELES has been separated into the columns A1, A2
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd22cb20b.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd1a3e3c2f.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fdd3b3c3f
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd1a3e3c2f.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd700ae565.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd7edfe667
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	A1	A2	EAF	Beta	SE	Pval	CHR_BP	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Summary statistics file does not have obvious CHR/BP columns. Checking to see if they are joined in another column.
Column CHR_BP has been separated into the columns CHR, BP
Standardising column headers.
First line of summary statistics file: 
SNP	A1	A2	FRQ	BETA	SE	P	CHR	BP	
Checking for incorrect base-pair positions
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd700ae565.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.002 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd5d89e25b.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd7edfe667
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd5d89e25b.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd8ba55d1.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd45cabce
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	A1	A2	EAF	Beta	SE	Pval	CHR_BP	CHR_BP_2	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Summary statistics file does not have obvious CHR/BP columns. Checking to see if they are joined in another column.
Warning: Multiple columns in the sumstats file seem to relate to Chromosome:Base Pair position.
The column CHR_BP_2 will be kept whereas the column(s) CHR_BP will be removed.
If this is not the correct column to keep, please remove all incorrect columns from those listed here before 
running `format_sumstats()`.
Column CHR_BP_2 has been separated into the columns CHR, BP
Standardising column headers.
First line of summary statistics file: 
SNP	A1	A2	FRQ	BETA	SE	P	CHR	BP	
Checking for incorrect base-pair positions
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd8ba55d1.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fdc65cf1c.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd45cabce
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fdc65cf1c.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.002 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd6fcfee4a.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd3b4b2487
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd6fcfee4a.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd11c465f0.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd1532f1ac
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd11c465f0.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.
Setting sorted=FALSE (required when formatted=FALSE).


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd34b0de0c.tsv.gz
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Assigning N=1000 for all SNPs.
N already exists within sumstats_dt.
[1] &quot;Testing: compute_n=&#x27;ldsc&#x27;&quot;
Computing effective sample size using the LDSC method:
 Neff = (N_CAS+N_CON) * (N_CAS/(N_CAS+N_CON)) / mean((N_CAS/(N_CAS+N_CON))[(N_CAS+N_CON)==max(N_CAS+N_CON)]))
[1] &quot;Testing: compute_n=&#x27;giant&#x27;&quot;
Computing effective sample size using the GIANT method:
 Neff = 2 / (1/N_CAS + 1/N_CON)
[1] &quot;Testing: compute_n=&#x27;metal&#x27;&quot;
Computing effective sample size using the METAL method:
 Neff = 4 / (1/N_CAS + 1/N_CON)
[1] &quot;Testing: compute_n=&#x27;sum&#x27;&quot;
Computing sample size using the sum method:
 N = N_CAS + N_CON


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd5917a76.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd4b9c0164
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd5917a76.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd258d8bab.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Saving output messages to:
/tmp/RtmpiIp3aX/file12d3fd258d8bab_log_msg.txt
Any runtime errors will be saved to:
/tmp/RtmpiIp3aX/file12d3fd258d8bab_log_output.txt
Messages will not be printed to terminal.
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd4bd7f1f5.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd3f476c9b
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd4bd7f1f5.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd47041b0c.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd14e9f300
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 186 rows
   - 93 unique variants
   - 140 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
93 sumstat rows are duplicated. These duplicates will be removed.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd47041b0c.tsv.gz
Summary statistics report:
   - 93 rows (50% of original 186 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd7801d23c.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd14e9f300
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd7801d23c.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd76e12b66.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd14e9f300
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 94 rows
   - 94 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicate SNPs from SNP ID.
Checking for SNPs with duplicated base-pair positions.
1 base-pair positions are duplicated in the sumstats file. These duplicates will be removed.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Checking for bi-allelic SNPs.
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 93 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 22 seconds.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd76e12b66.tsv.gz
Summary statistics report:
   - 93 rows (98.9% of original 94 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.395 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd1d213579.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd6842266a
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Filtering effect columns, ensuring none equal 0.
5 SNPs have effect values = 0 and will be removed
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
44 SNPs (50%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd1d213579.tsv.gz
Summary statistics report:
   - 88 rows (94.6% of original 93 rows)
   - 88 unique variants
   - 65 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd304d7f0a.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd5231b25f
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	FRQ	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs based on FRQ.
38 SNPs are below the FRQ threshold of 0.9 and will be removed.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/frq_filter.tsv.gz
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
55 SNPs (100%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd304d7f0a.tsv.gz
Summary statistics report:
   - 55 rows (59.1% of original 93 rows)
   - 55 unique variants
   - 41 genome-wide significant variants (P&lt;5e-8)
   - 16 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     EAF   BETA    SE         P      FRQ
1: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10 1.863269
2: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14 1.169733
3:  rs1008078   1 91189731  T  C 0.37310 -0.016 0.003 6.005e-10 1.401423
4: rs61787263   1 98618714  T  C 0.76120  0.016 0.003 5.391e-08 1.873332
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd6736e8cf.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd5231b25f
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	FRQ	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs based on FRQ.
38 SNPs are below the FRQ threshold of 0.9 and will be removed.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/frq_filter.tsv.gz
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
55 SNPs (100%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=FALSE, the FRQ column will be renamed MAJOR_ALLELE_FRQ to differentiate the values from 
minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd6736e8cf.tsv.gz
Summary statistics report:
   - 55 rows (59.1% of original 93 rows)
   - 55 unique variants
   - 41 genome-wide significant variants (P&lt;5e-8)
   - 16 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     EAF   BETA    SE         P
1: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
2: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
3:  rs1008078   1 91189731  T  C 0.37310 -0.016 0.003 6.005e-10
4: rs61787263   1 98618714  T  C 0.76120  0.016 0.003 5.391e-08
   MAJOR_ALLELE_FRQ
1:         1.863269
2:         1.169733
3:         1.401423
4:         1.873332
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd6940fb53.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd280e7d58
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
SNP	CHR	BP	A1	A2	FRQ	BETA	SE	P	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd6940fb53.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd23b8c905.tsv.gz
Standardising column headers.
First line of summary statistics file: 
SNP	CHR	BP	A1	A2	Uniq.a1a2	EAF	BETA	P	
Summary statistics report:
   - 2 rows
   - 2 unique variants
   - 1 genome-wide significant variants (P&lt;5e-8)
   - 2 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Ensuring all SNPs are on the reference genome.
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 2 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 20 seconds.
Found 1 Indels. These won&#x27;t be checked against the reference genome as it does not contain Indels.
WARNING If your sumstat doesn&#x27;t contain Indels, set the indel param to FALSE &amp; rerun MungeSumstats::format_sumstats()
Checking for correct direction of A1 (reference) and A2 (alternative allele).
There are 1 SNPs where A1 doesn&#x27;t match the reference genome.
These will be flipped with their effect columns.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicate SNPs from SNP ID.
Found 1 Indels. These won&#x27;t be checked for duplicates based on RS ID as there can be multiples.
WARNING If your sumstat doesn&#x27;t contain Indels, set the indel param to FALSE &amp; rerun MungeSumstats::format_sumstats()
Checking for SNPs with duplicated base-pair positions.
Found 1 Indels. These won&#x27;t be checked for duplicates based on base-pair position as there can be multiples.
WARNING If your sumstat doesn&#x27;t contain Indels, set the indel param to FALSE &amp; rerun MungeSumstats::format_sumstats()
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
SE is not present but can be imputed with BETA &amp; P. Set impute_se=TRUE and rerun to do this.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Checking for bi-allelic SNPs.
Warning: When method is an integer, must be &gt;0.
1 SNPs (50%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd23b8c905.tsv.gz
Summary statistics report:
   - 2 rows (100% of original 2 rows)
   - 2 unique variants
   - 1 genome-wide significant variants (P&lt;5e-8)
   - 2 chromosomes
Done munging in 0.362 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR        BP A1 A2      UNIQ.A1A2       FRQ         BETA
1: rs12987662   2 100821548  C  A             aa 0.6213000 -0.027000000
2: rs34589910   4   6364621  C CG 4:6364621_C_CG 0.0945334 -0.006257323
              P
1: 2.693000e-24
2: 4.883341e-01
Returning data directly.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd33ac4596.tsv.gz
Standardising column headers.
First line of summary statistics file: 
SNP	CHR	BP	A1	A2	Uniq.a1a2	EAF	BETA	P	
Summary statistics report:
   - 3 rows
   - 3 unique variants
   - 1 genome-wide significant variants (P&lt;5e-8)
   - 3 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
2 SNP IDs appear to be made up of chr:bp, these will be replaced by their SNP ID from the reference genome
Loading SNPlocs data.
Found  Indels. These won&#x27;t be checked against the reference genome as it does not contain Indels.
WARNING If your sumstat doesn&#x27;t contain Indels, set the indel param to FALSE &amp; rerun MungeSumstats::format_sumstats()
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Ensuring all SNPs are on the reference genome.
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 2 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 21 seconds.
Checking for correct direction of A1 (reference) and A2 (alternative allele).
There are 2 SNPs where A1 doesn&#x27;t match the reference genome.
These will be flipped with their effect columns.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicate SNPs from SNP ID.
Checking for SNPs with duplicated base-pair positions.
INFO column not available. Skipping INFO score filtering step.
SE is not present but can be imputed with BETA &amp; P. Set impute_se=TRUE and rerun to do this.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Checking for bi-allelic SNPs.
Warning: When method is an integer, must be &gt;0.
2 SNPs (100%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd33ac4596.tsv.gz
Summary statistics report:
   - 2 rows (66.7% of original 3 rows)
   - 2 unique variants
   - 1 genome-wide significant variants (P&lt;5e-8)
   - 2 chromosomes
Done munging in 0.387 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR        BP A1 A2 UNIQ.A1A2    FRQ    BETA         P
1: rs12987662   2 100821548  C  A        aa 0.6213 -0.0270 2.693e-24
2:  rs9320913   6  98584733  C  A        bb 0.5433 -0.0123 2.100e-07
Returning data directly.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd692d4dd3.tsv
Converting full summary stats file to tabix format for fast querying...
Reading header.
Ensuring file is bgzipped.
Tabix-indexing file.
Removing temporary .tsv file.
Reading header.
Reading entire file.
Sorting coordinates with &#x27;GenomicRanges&#x27;.
Converting summary statistics to GenomicRanges.
Sorting coordinates with &#x27;data.table&#x27;.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd3c6c73a4.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd4246742f
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	INFO	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
Filtering SNPs based on INFO score.
38 SNPs are below the INFO threshold of 0.9 and will be removed.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/info_filter.tsv.gz
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
28 SNPs (50.9%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd3c6c73a4.tsv.gz
Summary statistics report:
   - 55 rows (59.1% of original 93 rows)
   - 55 unique variants
   - 41 genome-wide significant variants (P&lt;5e-8)
   - 16 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P     INFO
1: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10 1.863269
2: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14 1.169733
3:  rs1008078   1 91189731  T  C 0.37310 -0.016 0.003 6.005e-10 1.401423
4: rs61787263   1 98618714  T  C 0.76120  0.016 0.003 5.391e-08 1.873332
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd2421b21b.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd5756435a
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd2421b21b.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd246c4c34.tsv.gz
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd246c4c34.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Sorting coordinates with &#x27;data.table&#x27;.
Performing data liftover from hg19 to hg38.
Converting summary statistics to GenomicRanges.
Downloading chain file from Ensembl.
trying URL &#x27;ftp://ftp.ensembl.org/pub/assembly_mapping/homo_sapiens/GRCh37_to_GRCh38.chain.gz&#x27;
Content type &#x27;unknown&#x27; length 285250 bytes (278 KB)
==================================================
/tmp/RtmpiIp3aX/GRCh37_to_GRCh38.chain.gz
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Performing data liftover from hg19 to hg38.
Converting summary statistics to GenomicRanges.
Using existing chain file from ensembl.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd41d57256.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd3fb5bc51
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Ensuring all SNPs are on the reference genome.
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 93 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 24 seconds.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Performing data liftover from hg19 to hg38.
Converting summary statistics to GenomicRanges.
Using existing chain file from ensembl.
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd41d57256.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.446 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8430543  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43516856  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72267927  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72296486  T  C 0.23690 -0.017 0.003 1.797e-08
   IMPUTATION_gen_build
1:                 TRUE
2:                 TRUE
3:                 TRUE
4:                 TRUE
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd52610ad4.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd41d57256.tsv.gz
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
SNP	CHR	BP	A1	A2	FRQ	BETA	SE	P	IMPUTATION_gen_build	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Ensuring all SNPs are on the reference genome.
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 93 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 63 seconds.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Performing data liftover from hg38 to hg19.
Converting summary statistics to GenomicRanges.
Using existing chain file from ensembl.
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd52610ad4.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 1.089 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
   IMPUTATION_GEN_BUILD IMPUTATION_gen_build
1:                 TRUE                 TRUE
2:                 TRUE                 TRUE
3:                 TRUE                 TRUE
4:                 TRUE                 TRUE
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd717d266d.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd3fb5bc51
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Ensuring all SNPs are on the reference genome.
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 93 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 26 seconds.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd717d266d.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.46 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.
[1] &quot;/tmp/RtmpiIp3aX/data/file1/file12d3fd3584742b.tsv.gz&quot;
[1] &quot;/tmp/RtmpiIp3aX/data/file2/file12d3fd346d30c0.tsv.gz&quot;
[1] &quot;/tmp/RtmpiIp3aX/data/file3/file12d3fd3a8de566.tsv.gz&quot;
[1] &quot;/tmp/RtmpiIp3aX/data/file4/file12d3fd74b1cbd1.tsv.gz&quot;
[1] &quot;/tmp/RtmpiIp3aX/data/file5/file12d3fd7422ed11.tsv.gz&quot;
[1] &quot;/tmp/RtmpiIp3aX/data/file6/file12d3fd1997e642.tsv.gz&quot;
[1] &quot;/tmp/RtmpiIp3aX/data/file7/file12d3fd1d0c714a.tsv.gz&quot;
[1] &quot;/tmp/RtmpiIp3aX/data/file8/file12d3fd707a0308.tsv.gz&quot;
[1] &quot;/tmp/RtmpiIp3aX/data/file9/file12d3fd63b7b2d9.tsv.gz&quot;
[1] &quot;/tmp/RtmpiIp3aX/data/file10/file12d3fd5ee1e3a1.tsv.gz&quot;
10 file(s) found.
Parsing info from 10 log file(s).


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd6a43f315.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd2f178d2f
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 92 unique variants
   - 69 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
WARNING: 1 rows in sumstats file are missing data and will be removed.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
46 SNPs (50%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd6a43f315.tsv.gz
Summary statistics report:
   - 92 rows (98.9% of original 93 rows)
   - 92 unique variants
   - 69 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2    FRQ   BETA    SE         P
1: rs10061788   5 87934707  A  G 0.2164  0.021 0.004 2.464e-09
2:  rs1007883  16 51163406  T  C 0.3713 -0.015 0.003 5.326e-08
3:  rs1008078   1 91189731  T  C 0.3731 -0.016 0.003 6.005e-10
4:  rs1043209  14 23373986  A  G 0.6026  0.018 0.003 1.816e-11
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd390368f0.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd2f178d2f
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd390368f0.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2    FRQ   BETA    SE         P
1: rs10061788   5 87934707  A  G 0.2164  0.021 0.004 2.464e-09
2:  rs1007883  16 51163406  T  C 0.3713 -0.015 0.003 5.326e-08
3:  rs1008078   1 91189731  T  C 0.3731 -0.016 0.003 6.005e-10
4:  rs1043209  14 23373986  A  G 0.6026  0.018 0.003 1.816e-11
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd3c6c45d3.tsv.gz
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 21 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Loading SNPlocs data.
There is no Chromosome or Base Pair Position column found within the data. It must be inferred from other column information.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 1 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 1 seconds.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd3c6c45d3.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.05 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2    FRQ   BETA    SE         P
1: rs10061788   5 87934707  A  G 0.2164  0.021 0.004 2.464e-09
2:  rs1007883  16 51163406  T  C 0.3713 -0.015 0.003 5.326e-08
3:  rs1008078   1 91189731  T  C 0.3731 -0.016 0.003 6.005e-10
4:  rs1043209  14 23373986  A  G 0.6026  0.018 0.003 1.816e-11
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd563aa564.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd75deae26
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
1 SNPs found with multiple RSIDs on one row, the first will be taken. If you would rather remove these SNPs set
`remove_multi_rs_snp=TRUE`.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd563aa564.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
   convert_multi_rs_SNP
1:                   NA
2:                   NA
3:                   NA
4:                   NA
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd48e73246.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd75deae26
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd48e73246.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd1d3d61f.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd3ade6a74
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 92 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/snp_multi_rs_one_row.tsv.gz
1 SNPs found with multiple RSIDs on one row, these will be removed. If you would rather take the first RS ID set
`remove_multi_rs_snp`=FALSE
Checking SNP RSIDs.
1 SNP IDs are not correctly formatted. These will be corrected from the reference genome.
Loading SNPlocs data.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/snp_not_found_from_chr_bp.tsv.gz
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Ensuring all SNPs are on the reference genome.
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 90 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 22 seconds.
1 SNPs are not on the reference genome. These will be corrected from the reference genome.
Loading SNPlocs data.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/snp_not_found_from_chr_bp_2.tsv.gz
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 90 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 23 seconds.
Checking for correct direction of A1 (reference) and A2 (alternative allele).
There are 43 SNPs where A1 doesn&#x27;t match the reference genome.
These will be flipped with their effect columns.
Checking for missing data.
WARNING: 1 rows in sumstats file are missing data and will be removed.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/missing_data.tsv.gz
Checking for duplicate columns.
Checking for duplicate SNPs from SNP ID.
1 RSIDs are duplicated in the sumstats file. These duplicates will be removed
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/dup_snp_id.tsv.gz
Checking for SNPs with duplicated base-pair positions.
1 base-pair positions are duplicated in the sumstats file. These duplicates will be removed.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/dup_base_pair_position.tsv.gz
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
1 SNPs have SE values &lt;= 0 and will be removed
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/se_neg.tsv.gz
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Checking for strand ambiguous SNPs.
8 SNPs are strand-ambiguous alleles including 4 A/T and 4 C/G ambiguous SNPs. These will be removed
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/snp_strand_ambiguous.tsv.gz
Checking for bi-allelic SNPs.
Warning: When method is an integer, must be &gt;0.
54 SNPs (68.4%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd1d3d61f.tsv.gz
Summary statistics report:
   - 79 rows (84.9% of original 93 rows)
   - 79 unique variants
   - 57 genome-wide significant variants (P&lt;5e-8)
   - 18 chromosomes
Done munging in 0.816 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P IMPUTATION_SNP
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08             NA
2: rs34305371   1 72733610  G  A 0.91231 -0.035 0.005 3.762e-14             NA
3:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08             NA
4:  rs1008078   1 91189731  C  T 0.62690  0.016 0.003 6.005e-10             NA
   flipped
1:      NA
2:    TRUE
3:      NA
4:    TRUE
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd68b08e84.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd225bf208
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
chromosome	rs_id	markername	position_hg18	Effect_allele	Other_allele	EAF_HapMapCEU	N_SMK	Effect_SMK	StdErr_SMK	P_value_SMK	N_NONSMK	Effect_NonSMK	StdErr_NonSMK	P_value_NonSMK	
Summary statistics report:
   - 5 rows
   - 5 unique variants
   - 1 chromosomes
Checking for multi-GWAS.
WARNING: Multiple traits found in sumstats file only one of which can be analysed: 
SMK, NONSMK
Standardising column headers.
First line of summary statistics file: 
CHR	SNP	MARKERNAME	POSITION_HG18	A2	A1	EAF_HAPMAPCEU	N	EFFECT	STDERR	P_VALUE	N_NONSMK	EFFECT_NONSMK	STDERR_NONSMK	P_VALUE_NONSMK	
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
1 SNP IDs are not correctly formatted and will be removed.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Summary statistics file does not have obvious CHR/BP columns. Checking to see if they are joined in another column.
Column MARKERNAME has been separated into the columns CHR, BP
Standardising column headers.
First line of summary statistics file: 
CHR	SNP	POSITION_HG18	A2	A1	EAF_HAPMAPCEU	N	BETA	SE	P	N_NONSMK	EFFECT_NONSMK	STDERR_NONSMK	P_VALUE_NONSMK	BP	
Checking for incorrect base-pair positions
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Ensuring that the N column is all integers.
The sumstats N column is not all integers, this could effect downstream analysis. These will be converted to integers.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Making X/Y/MT CHR uppercase.
N already exists within sumstats_dt.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd68b08e84.tsv.gz
Summary statistics report:
   - 4 rows (80% of original 5 rows)
   - 4 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Done munging in 0.002 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
         SNP  CHR        BP A1 A2 POSITION_HG18 EAF_HAPMAPCEU     N    BETA
1: rs1000050 chr1 161003087  C  T     161003087        0.9000 36257  0.0001
2: rs1000073 chr1 155522020  G  A     155522020        0.3136 36335  0.0046
3: rs1000075 chr1  94939420  C  T      94939420        0.3583 38959 -0.0013
4: rs1000085 chr1  66630503  G  C      66630503        0.1667 38761  0.0053
       SE      P N_NONSMK EFFECT_NONSMK STDERR_NONSMK P_VALUE_NONSMK
1: 0.0109 0.9931   127514        0.0058        0.0059         0.3307
2: 0.0083 0.5812   126780        0.0038        0.0045         0.3979
3: 0.0082 0.8687   147567       -0.0043        0.0044         0.3259
4: 0.0095 0.5746   147259       -0.0034        0.0052         0.5157
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd123d9fc8.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd223bf75e
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	N	N_fixed	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Ensuring that the N column is all integers.
The sumstats N column is not all integers, this could effect downstream analysis. These will be converted to integers.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
N already exists within sumstats_dt.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd123d9fc8.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P N N_FIXED
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08 5       5
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10 1       1
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14 1       1
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08 7       7
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd5928d455.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd7412d309
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	N	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
The sumstats N column is not all integers, this could effect downstream analysis.These will NOT be converted to integers.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
1 SNPs have N values 5 standard deviations above the mean and will be removed
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/n_large.tsv.gz
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
N already exists within sumstats_dt.
47 SNPs (51.1%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd5928d455.tsv.gz
Summary statistics report:
   - 92 rows (98.9% of original 93 rows)
   - 92 unique variants
   - 69 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P N
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08 3
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10 5
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14 3
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08 3
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd59badbb9.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd7412d309
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	N	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
The sumstats N column is not all integers, this could effect downstream analysis.These will NOT be converted to integers.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
1 SNPs have N values 5 standard deviations above the mean and will be removed
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/n_large.tsv.gz
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
N already exists within sumstats_dt.
47 SNPs (51.1%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd59badbb9.tsv.gz
Summary statistics report:
   - 92 rows (98.9% of original 93 rows)
   - 92 unique variants
   - 69 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P N
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08 3
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10 5
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14 3
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08 3
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd172c6a51.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd7412d309
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	N	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
The sumstats N column is not all integers, this could effect downstream analysis.These will NOT be converted to integers.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
1 SNPs have N values 5 standard deviations above the mean and will be removed
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/n_large.tsv.gz
Removing rows where is.na(N)
0 SNPs have N values that are NA and will be removed.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/n_null.tsv.gz
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
N already exists within sumstats_dt.
47 SNPs (51.1%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd172c6a51.tsv.gz
Summary statistics report:
   - 92 rows (98.9% of original 93 rows)
   - 92 unique variants
   - 69 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P N
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08 3
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10 5
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14 3
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08 3
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd7344e38.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd29a25ad5
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking for incorrect base-pair positions
WARNING: No A2 column found in the data, multi-allelic can&#x27;t not be accurately chosen (as any
of the choices could be valid). bi_allelic_filter has been forced to TRUE.
Loading SNPlocs data.
There is no A1 or A2 allele information column found within the data. It must be inferred from other column information.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 93 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 24 seconds.
Deriving both A1 and A2 from reference genome
WARNING: Inferring the alternative allele (A2) from the reference genome. In some instances, there are more than one
alternative allele. Arbitrarily, only the first will be kept. See column `alt_alleles` in your returned sumstats file
for all alternative alleles.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/alleles_not_found_from_snp.tsv.gz
Checking for missing data.
Checking for duplicate columns.
Checking for duplicate SNPs from SNP ID.
Checking for SNPs with duplicated base-pair positions.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Checking for bi-allelic SNPs.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd7344e38.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.415 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P alt_alleles
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08           C
2: rs11210860   1 43982527  G  A 0.36940  0.017 0.003 2.359e-10           A
3: rs34305371   1 72733610  G  A 0.08769  0.035 0.005 3.762e-14           A
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08           C
   IMPUTATION_A1 IMPUTATION_A2
1:          TRUE          TRUE
2:          TRUE          TRUE
3:          TRUE          TRUE
4:          TRUE          TRUE
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd39af0a89.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd29a25ad5
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A2 is uppercase
Checking for incorrect base-pair positions
Loading SNPlocs data.
There is no A1 or A2 allele information column found within the data. It must be inferred from other column information.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 93 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 24 seconds.
One of A1/A2 are missing, allele flipping will be tested
Deriving A1 from reference genome
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/alleles_not_found_from_snp.tsv.gz
Checking for correct direction of A1 (reference) and A2 (alternative allele).
There are 46 SNPs where A1 doesn&#x27;t match the reference genome.
These will be flipped with their effect columns.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd39af0a89.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.426 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P IMPUTATION_A1
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08          TRUE
2: rs11210860   1 43982527  G  G 0.36940 -0.017 0.003 2.359e-10          TRUE
3: rs34305371   1 72733610  G  G 0.08769 -0.035 0.005 3.762e-14          TRUE
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08          TRUE
   flipped
1:      NA
2:    TRUE
3:    TRUE
4:      NA
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd7d278964.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd29a25ad5
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking for incorrect base-pair positions
WARNING: No A2 column found in the data, multi-allelic can&#x27;t not be accurately chosen (as any
of the choices could be valid). bi_allelic_filter has been forced to TRUE.
Loading SNPlocs data.
There is no A1 or A2 allele information column found within the data. It must be inferred from other column information.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 93 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 25 seconds.
One of A1/A2 are missing, allele flipping will be tested
Deriving A2 from reference genome
WARNING: Inferring the alternative allele (A2) from the reference genome. In some instances, there are more than one
alternative allele. Arbitrarily, only the first will be kept. See column `alt_alleles` in your returned sumstats file
for all alternative alleles.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/alleles_not_found_from_snp.tsv.gz
Checking for correct direction of A1 (reference) and A2 (alternative allele).
Checking for missing data.
Checking for duplicate columns.
Checking for duplicate SNPs from SNP ID.
Checking for SNPs with duplicated base-pair positions.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Checking for bi-allelic SNPs.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd7d278964.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.446 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P alt_alleles
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08           C
2: rs11210860   1 43982527  A  A 0.36940  0.017 0.003 2.359e-10           A
3: rs34305371   1 72733610  A  A 0.08769  0.035 0.005 3.762e-14           A
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08           C
   IMPUTATION_A2
1:          TRUE
2:          TRUE
3:          TRUE
4:          TRUE
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd51c374af.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd29a25ad5
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for correct direction of A1 (reference) and A2 (alternative allele).
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 93 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 26 seconds.
There are 46 SNPs where A1 doesn&#x27;t match the reference genome.
These will be flipped with their effect columns.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd51c374af.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.463 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  G  A 0.36940 -0.017 0.003 2.359e-10
3: rs34305371   1 72733610  G  A 0.08769 -0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd13e9b002.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd5a0cc8bd
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Summary statistics file does not have obvious CHR/BP columns. Checking to see if they are joined in another column.
Standardising column headers.
First line of summary statistics file: 
SNP	BP	A1	A2	FRQ	BETA	SE	P	
Loading SNPlocs data.
There is no Chromosome or Base Pair Position column found within the data. It must be inferred from other column information.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 93 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 24 seconds.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd13e9b002.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.432 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd34b05991.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd738c5db7
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Summary statistics file does not have obvious CHR/BP columns. Checking to see if they are joined in another column.
Standardising column headers.
First line of summary statistics file: 
SNP	A1	A2	FRQ	BETA	SE	P	
Loading SNPlocs data.
There is no Chromosome or Base Pair Position column found within the data. It must be inferred from other column information.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 93 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 25 seconds.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/chr_bp_not_found_from_snp.tsv.gz
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd34b05991.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.448 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd4cfcf49d.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd22802b08
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
1 SNP IDs are not correctly formatted. These will be corrected from the reference genome.
Loading SNPlocs data.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd4cfcf49d.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.004 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd1242bffe.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd22802b08
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd1242bffe.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd6e6f9936.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fdd1b987c
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
1 SNP IDs appear to be made up of chr:bp, these will be replaced by their SNP ID from the reference genome
Loading SNPlocs data.
1 SNP IDs are not correctly formatted and will be removed.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Summary statistics file does not have obvious CHR/BP columns. Checking to see if they are joined in another column.
Standardising column headers.
First line of summary statistics file: 
SNP	A1	A2	FRQ	BETA	SE	P	
Loading SNPlocs data.
There is no Chromosome or Base Pair Position column found within the data. It must be inferred from other column information.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 92 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 26 seconds.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
46 SNPs (50%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd6e6f9936.tsv.gz
Summary statistics report:
   - 92 rows (98.9% of original 93 rows)
   - 92 unique variants
   - 69 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.469 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd47dfe86e.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd1ac78514
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
1 SNP IDs are not correctly formatted. These will be corrected from the reference genome.
Loading SNPlocs data.
1 SNP IDs appear to be made up of chr:bp, these will be replaced by their SNP ID from the reference genome
Loading SNPlocs data.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd47dfe86e.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.007 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd38b86e31.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd227a04ed
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd4f7a80f8.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fdd1b987c
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd4f7a80f8.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd42b1de01.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd72a9ff28
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Loading SNPlocs data.
There is no SNP column found within the data. It must be inferred from other column information.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd42b1de01.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.084 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd614c5c27.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd4561b7f0
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Ensuring all SNPs are on the reference genome.
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 93 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 23 seconds.
1 SNPs are not on the reference genome. These will be corrected from the reference genome.
Loading SNPlocs data.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/snp_not_found_from_chr_bp.tsv.gz
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 93 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 24 seconds.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd614c5c27.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.842 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd6ffcb045.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd4561b7f0
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Ensuring all SNPs are on the reference genome.
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 93 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 23 seconds.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd6ffcb045.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.41 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.
Inferring genome build of 1 sumstats file(s).
ss1
Inferring genome build.
Reading in only the first 50 rows of sumstats.
Importing tabular file: /home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/MungeSumstats/extdata/eduAttainOkbay.txt
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 50 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 23 seconds.
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 50 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 34 seconds.
Inferred genome build: GRCH37
Time difference of 1.010856 mins
GRCH37: 1 file(s)


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd4f37cdc3.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd25d22375
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 23 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
1 SNPs have been removed as their BP column is not in the range of 1 to the length of the chromosome
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/bad_bp.tsv.gz
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
2 SNPs are on chromosomes X, Y, MT and will be removed
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/chr_excl.tsv.gz
Warning: When method is an integer, must be &gt;0.
45 SNPs (50%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd4f37cdc3.tsv.gz
Summary statistics report:
   - 90 rows (96.8% of original 93 rows)
   - 90 unique variants
   - 67 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd13c6639c.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd25d22375
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd13c6639c.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.
Reading header.
Reading entire file.
Reading header.
Reading header.
Reading header.
Reading header.
Reading header.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd4eb2bcdf
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
SNP	CHR	BP	A1	A2	FRQ	BETA	SE	P	
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd6a598905
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
SNP	CHR	BP	A1	A2	FRQ	BETA	SE	P	


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd607a008c.vcf.bgz
Sorting coordinates with &#x27;data.table&#x27;.
Converting summary statistics to GenomicRanges.
Converting summary statistics to VRanges.
Writing in VCF format ==&gt;  /tmp/RtmpiIp3aX/file12d3fd607a008c.vcf.bgz
Using local VCF.
Finding empty VCF columns based on first 10,000 rows.
1 sample detected: GWAS
Constructing ScanVcfParam object.
Reading VCF file: single-threaded
Converting VCF to data.table.
Expanding VCF first, so number of rows may increase.
Checking for empty columns.
Time difference of 0.1 secs
VCF data.table contains: 93 rows x 11 columns.
Time difference of 0.5 secs
No INFO (SI) column detected.
Standardising column headers.
First line of summary statistics file: 
ID	chr	BP	end	REF	ALT	SNP	FRQ	BETA	SE	P	
Using local VCF.
bgzip-compressing VCF file.
Finding empty VCF columns based on first 10,000 rows.
Dropping 1 duplicate column(s).
1 sample detected: EBI-a-GCST005647
Constructing ScanVcfParam object.
VCF contains: 39,630,630 variant(s) x 1 sample(s)
Reading VCF file: single-threaded
Converting VCF to data.table.
Expanding VCF first, so number of rows may increase.
Dropping 1 duplicate column(s).
Checking for empty columns.
Unlisting 3 columns.
Dropped 314 duplicate rows.
Time difference of 0.1 secs
VCF data.table contains: 101 rows x 11 columns.
Time difference of 0.6 secs
Renaming ID as SNP.
VCF file has -log10 P-values; these will be converted to unadjusted p-values in the &#x27;P&#x27; column.
No INFO (SI) column detected.
Standardising column headers.
First line of summary statistics file: 
SNP	chr	BP	end	REF	ALT	FILTER	AF	ES	LP	SE	P	


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd3a26ff82.vcf.bgz
Sorting coordinates with &#x27;data.table&#x27;.
Converting summary statistics to GenomicRanges.
Converting summary statistics to VRanges.
Writing in VCF format ==&gt;  /tmp/RtmpiIp3aX/file12d3fd3a26ff82.vcf.bgz
Using local VCF.
Finding empty VCF columns based on first 10,000 rows.
1 sample detected: GWAS
Constructing ScanVcfParam object.
Reading VCF file: single-threaded
Converting VCF to data.table.
Expanding VCF first, so number of rows may increase.
Checking for empty columns.
Time difference of 0.1 secs
VCF data.table contains: 101 rows x 13 columns.
Time difference of 0.5 secs
VCF file has -log10 P-values; these will be converted to unadjusted p-values in the &#x27;P&#x27; column.
No INFO (SI) column detected.
Standardising column headers.
First line of summary statistics file: 
ID	chr	BP	end	REF	SNP	END	FILTER	FRQ	BETA	LP	SE	P	


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd5607443e.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Standardising column headers.
First line of summary statistics file: 
SNP	P	FRQ	BETA	CHR	BP	
Summary statistics report:
   - 5 rows
   - 5 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
5 SNP IDs contain other information in the same column. These will be separated.
Checking for merged allele column.
Column SNP_INFO has been separated into the columns A1, A2
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
SE is not present but can be imputed with BETA &amp; P. Set impute_se=TRUE and rerun to do this.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
3 SNPs (60%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd5607443e.tsv.gz
Summary statistics report:
   - 5 rows (100% of original 5 rows)
   - 5 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
           SNP CHR    BP A1 A2           P       FRQ      BETA
1: rs140052487   1 54353  C  A 0.037219838 0.3000548 0.8797957
2: rs558796213   1 54564  G  T 0.004382482 0.5848666 0.7068747
3: rs561234294   1 54591  A  G 0.070968402 0.3334671 0.7319726
4:   rs2462492   1 54676  C  T 0.065769040 0.6220120 0.9316344
Returning data directly.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

******::NOTE::******
 - Log results will be saved to `tempdir()` by default.
 - This means all log data from the run will be  deleted upon ending the R session.
 - To keep it, change `log_folder` to an actual directory  (e.g. log_folder=&#x27;./&#x27;).
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd5aceaeca.tsv.gz
Log data to be saved to ==&gt;  /tmp/RtmpiIp3aX
Standardising column headers.
First line of summary statistics file: 
SNP	P	FRQ	BETA	CHR	BP	A1	A2	
Summary statistics report:
   - 5 rows
   - 5 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
SE is not present but can be imputed with BETA &amp; P. Set impute_se=TRUE and rerun to do this.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
3 SNPs (60%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd5aceaeca.tsv.gz
Summary statistics report:
   - 5 rows (100% of original 5 rows)
   - 5 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
           SNP CHR    BP A1 A2           P       FRQ      BETA
1: rs140052487   1 54353  C  A 0.037219838 0.3000548 0.8797957
2: rs558796213   1 54564  G  T 0.004382482 0.5848666 0.7068747
3: rs561234294   1 54591  A  G 0.070968402 0.3334671 0.7319726
4:   rs2462492   1 54676  C  T 0.065769040 0.6220120 0.9316344
Returning data directly.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd48baef24.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd7f1d80dc
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd38b79f6a.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd54fd3c6a
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd38b79f6a.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd3df9b1a5.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd54fd3c6a
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd3df9b1a5.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd68b3e3e0.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fdd317f68
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd68b3e3e0.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd3fd81b0a.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd238d096a
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd3fd81b0a.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd67750ffb.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd6577d6fa
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
5 SNPs have SE values &lt;= 0 and will be removed
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
44 SNPs (50%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd67750ffb.tsv.gz
Summary statistics report:
   - 88 rows (94.6% of original 93 rows)
   - 88 unique variants
   - 65 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	Support	
Returning unmapped column names without making them uppercase.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	Support	
Returning unmapped column names without making them uppercase.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd41b91453.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd30d7700e
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 85 rows
   - 85 unique variants
   - 63 genome-wide significant variants (P&lt;5e-8)
   - 19 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Checking for strand ambiguous SNPs.
Warning: When method is an integer, must be &gt;0.
43 SNPs (50.6%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd41b91453.tsv.gz
Summary statistics report:
   - 85 rows (100% of original 85 rows)
   - 85 unique variants
   - 63 genome-wide significant variants (P&lt;5e-8)
   - 19 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd7fb2c5f8.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd30d7700e
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Checking for strand ambiguous SNPs.
8 SNPs are strand-ambiguous alleles including 4 A/T and 4 C/G ambiguous SNPs. These will be removed
Warning: When method is an integer, must be &gt;0.
43 SNPs (50.6%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd7fb2c5f8.tsv.gz
Summary statistics report:
   - 85 rows (91.4% of original 93 rows)
   - 85 unique variants
   - 63 genome-wide significant variants (P&lt;5e-8)
   - 19 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd14e06644.tsv.gz


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd66363fd2.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fdce44561
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd66363fd2.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     EAF   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning data directly.
Converting summary statistics to GenomicRanges.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd4aee8f1e.tsv.gz


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd12939007.tsv.gz


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd5e4b375.tsv.gz


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd364d57eb.tsv.gz


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd43875b38.tsv.gz


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd30165784.tsv.gz


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd658380b.tsv.gz


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd54f526f4.tsv.gz


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd146efb16.tsv.gz


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd6bd00f05.tsv.gz


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd3c6a36ef.tsv.gz


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd2c1694c9.tsv.gz
Using local VCF.
bgzip-compressing VCF file.
Finding empty VCF columns based on first 10,000 rows.
Dropping 1 duplicate column(s).
1 sample detected: EBI-a-GCST005647
Constructing ScanVcfParam object.
VCF contains: 39,630,630 variant(s) x 1 sample(s)
Reading VCF file: single-threaded
Converting VCF to data.table.
Expanding VCF first, so number of rows may increase.
Dropping 1 duplicate column(s).
Checking for empty columns.
Unlisting 3 columns.
Dropped 314 duplicate rows.
Time difference of 0.1 secs
VCF data.table contains: 101 rows x 11 columns.
Time difference of 0.6 secs
Renaming ID as SNP.
VCF file has -log10 P-values; these will be converted to unadjusted p-values in the &#x27;P&#x27; column.
No INFO (SI) column detected.
Standardising column headers.
First line of summary statistics file: 
SNP	chr	BP	end	REF	ALT	FILTER	AF	ES	LP	SE	P	
Summary statistics report:
   - 101 rows
   - 101 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
2 SNPs (2%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd2c1694c9.tsv.gz
Summary statistics report:
   - 101 rows (100% of original 101 rows)
   - 101 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Done munging in 0.025 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
           SNP CHR    BP A1 A2   END FILTER    FRQ    BETA       LP     SE
1:  rs58108140   1 10583  G  A 10583   PASS 0.1589  0.0312 0.369267 0.0393
2:    rs806731   1 30923  G  T 30923   PASS 0.7843 -0.0114 0.126854 0.0353
3: rs116400033   1 51479  T  A 51479   PASS 0.1829  0.0711 1.262410 0.0370
4: rs146477069   1 54421  A  G 54421   PASS 0.0352 -0.0240 0.112102 0.0830
            P
1: 0.42730011
2: 0.74669974
3: 0.05464998
4: 0.77249913
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd351809b2.tsv.gz
Using local VCF.
bgzip-compressing VCF file.
Finding empty VCF columns based on first 10,000 rows.
Dropping 1 duplicate column(s).
1 sample detected: EBI-a-GCST005647
Constructing ScanVcfParam object.
VCF contains: 39,630,630 variant(s) x 1 sample(s)
Reading VCF file: single-threaded
Converting VCF to data.table.
Expanding VCF first, so number of rows may increase.
Dropping 1 duplicate column(s).
Checking for empty columns.
Unlisting 3 columns.
Dropped 314 duplicate rows.
Time difference of 0.1 secs
VCF data.table contains: 101 rows x 11 columns.
Time difference of 0.6 secs
Renaming ID as SNP.
VCF file has -log10 P-values; these will be converted to unadjusted p-values in the &#x27;P&#x27; column.
No INFO (SI) column detected.
Standardising column headers.
First line of summary statistics file: 
SNP	chr	BP	end	REF	ALT	FILTER	AF	ES	LP	SE	P	
Summary statistics report:
   - 101 rows
   - 101 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for correct direction of A1 (reference) and A2 (alternative allele).
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 101 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 22 seconds.
There are 1 SNPs where A1 doesn&#x27;t match the reference genome.
These will be flipped with their effect columns.
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicate SNPs from SNP ID.
Found 10 Indels. These won&#x27;t be checked for duplicates based on RS ID as there can be multiples.
WARNING If your sumstat doesn&#x27;t contain Indels, set the indel param to FALSE &amp; rerun MungeSumstats::format_sumstats()
Checking for SNPs with duplicated base-pair positions.
Found 10 Indels. These won&#x27;t be checked for duplicates based on base-pair position as there can be multiples.
WARNING If your sumstat doesn&#x27;t contain Indels, set the indel param to FALSE &amp; rerun MungeSumstats::format_sumstats()
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Checking for bi-allelic SNPs.
Warning: When method is an integer, must be &gt;0.
2 SNPs (2%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd351809b2.tsv.gz
Summary statistics report:
   - 101 rows (100% of original 101 rows)
   - 101 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Done munging in 0.416 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
           SNP CHR    BP A1 A2   END FILTER    FRQ    BETA       LP     SE
1:  rs58108140   1 10583  G  A 10583   PASS 0.1589  0.0312 0.369267 0.0393
2:    rs806731   1 30923  G  T 30923   PASS 0.7843 -0.0114 0.126854 0.0353
3: rs116400033   1 51479  T  A 51479   PASS 0.1829  0.0711 1.262410 0.0370
4: rs146477069   1 54421  A  G 54421   PASS 0.0352 -0.0240 0.112102 0.0830
            P
1: 0.42730011
2: 0.74669974
3: 0.05464998
4: 0.77249913
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd76d11e05.tsv.gz
Using local VCF.
bgzip-compressing VCF file.
Finding empty VCF columns based on first 10,000 rows.
Dropping 1 duplicate column(s).
1 sample detected: EBI-a-GCST005647
Constructing ScanVcfParam object.
VCF contains: 39,630,630 variant(s) x 1 sample(s)
Reading VCF file: single-threaded
Converting VCF to data.table.
Expanding VCF first, so number of rows may increase.
Dropping 1 duplicate column(s).
Checking for empty columns.
Unlisting 3 columns.
Dropped 314 duplicate rows.
Time difference of 0.1 secs
VCF data.table contains: 101 rows x 11 columns.
Time difference of 0.8 secs
Renaming ID as SNP.
VCF file has -log10 P-values; these will be converted to unadjusted p-values in the &#x27;P&#x27; column.
No INFO (SI) column detected.
Standardising column headers.
First line of summary statistics file: 
SNP	chr	BP	end	REF	ALT	FILTER	AF	ES	LP	SE	P	
Summary statistics report:
   - 101 rows
   - 101 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
2 SNPs (2%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd76d11e05.tsv.gz
Summary statistics report:
   - 101 rows (100% of original 101 rows)
   - 101 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Done munging in 0.029 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
           SNP CHR    BP A1 A2   END FILTER    FRQ    BETA       LP     SE
1:  rs58108140   1 10583  G  A 10583   PASS 0.1589  0.0312 0.369267 0.0393
2:    rs806731   1 30923  G  T 30923   PASS 0.7843 -0.0114 0.126854 0.0353
3: rs116400033   1 51479  T  A 51479   PASS 0.1829  0.0711 1.262410 0.0370
4: rs146477069   1 54421  A  G 54421   PASS 0.0352 -0.0240 0.112102 0.0830
            P
1: 0.42730011
2: 0.74669974
3: 0.05464998
4: 0.77249913
Returning data directly.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd7c4beaf.tsv.gz
Using local VCF.
bgzip-compressing VCF file.
Finding empty VCF columns based on first 10,000 rows.
Dropping 1 duplicate column(s).
1 sample detected: EBI-a-GCST005647
Constructing ScanVcfParam object.
VCF contains: 39,630,630 variant(s) x 1 sample(s)
Reading VCF file: single-threaded
Converting VCF to data.table.
Expanding VCF first, so number of rows may increase.
Dropping 1 duplicate column(s).
Checking for empty columns.
Unlisting 3 columns.
Dropped 314 duplicate rows.
Time difference of 0.1 secs
VCF data.table contains: 101 rows x 11 columns.
Time difference of 0.6 secs
Renaming ID as SNP.
VCF file has -log10 P-values; these will be converted to unadjusted p-values in the &#x27;P&#x27; column.
No INFO (SI) column detected.
Standardising column headers.
First line of summary statistics file: 
SNP	chr	BP	end	REF	ALT	FILTER	AF	ES	LP	SE	P	
Summary statistics report:
   - 101 rows
   - 101 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
2 SNPs (2%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd7c4beaf.tsv.gz
Summary statistics report:
   - 101 rows (100% of original 101 rows)
   - 101 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Done munging in 0.025 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
           SNP CHR    BP A1 A2   END FILTER    FRQ    BETA       LP     SE
1:  rs58108140   1 10583  G  A 10583   PASS 0.1589  0.0312 0.369267 0.0393
2:    rs806731   1 30923  G  T 30923   PASS 0.7843 -0.0114 0.126854 0.0353
3: rs116400033   1 51479  T  A 51479   PASS 0.1829  0.0711 1.262410 0.0370
4: rs146477069   1 54421  A  G 54421   PASS 0.0352 -0.0240 0.112102 0.0830
            P
1: 0.42730011
2: 0.74669974
3: 0.05464998
4: 0.77249913
Returning data directly.
Converting summary statistics to GenomicRanges.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd71f19013.tsv.gz
Using local VCF.
bgzip-compressing VCF file.
Finding empty VCF columns based on first 10,000 rows.
Dropping 1 duplicate column(s).
1 sample detected: EBI-a-GCST005647
Constructing ScanVcfParam object.
VCF contains: 39,630,630 variant(s) x 1 sample(s)
Reading VCF file: single-threaded
Converting VCF to data.table.
Expanding VCF first, so number of rows may increase.
Dropping 1 duplicate column(s).
Checking for empty columns.
Unlisting 3 columns.
Dropped 314 duplicate rows.
Time difference of 0.1 secs
VCF data.table contains: 101 rows x 11 columns.
Time difference of 0.6 secs
Renaming ID as SNP.
VCF file has -log10 P-values; these will be converted to unadjusted p-values in the &#x27;P&#x27; column.
No INFO (SI) column detected.
Standardising column headers.
First line of summary statistics file: 
SNP	chr	BP	end	REF	ALT	FILTER	AF	ES	LP	SE	P	
Summary statistics report:
   - 101 rows
   - 101 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
2 SNPs (2%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd71f19013.tsv.gz
Summary statistics report:
   - 101 rows (100% of original 101 rows)
   - 101 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Done munging in 0.025 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
           SNP CHR    BP A1 A2   END FILTER    FRQ    BETA       LP     SE
1:  rs58108140   1 10583  G  A 10583   PASS 0.1589  0.0312 0.369267 0.0393
2:    rs806731   1 30923  G  T 30923   PASS 0.7843 -0.0114 0.126854 0.0353
3: rs116400033   1 51479  T  A 51479   PASS 0.1829  0.0711 1.262410 0.0370
4: rs146477069   1 54421  A  G 54421   PASS 0.0352 -0.0240 0.112102 0.0830
            P
1: 0.42730011
2: 0.74669974
3: 0.05464998
4: 0.77249913
Returning data directly.
Converting summary statistics to GenomicRanges.
Converting summary statistics to VRanges.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd368295f.tsv.gz
Using local VCF.
bgzip-compressing VCF file.
Finding empty VCF columns based on first 10,000 rows.
Dropping 1 duplicate column(s).
1 sample detected: EBI-a-GCST005647
Constructing ScanVcfParam object.
VCF contains: 39,630,630 variant(s) x 1 sample(s)
Reading VCF file: single-threaded
Converting VCF to data.table.
Expanding VCF first, so number of rows may increase.
Dropping 1 duplicate column(s).
Checking for empty columns.
Unlisting 3 columns.
Dropped 314 duplicate rows.
Time difference of 0.1 secs
VCF data.table contains: 101 rows x 11 columns.
Time difference of 0.6 secs
Renaming ID as SNP.
VCF file has -log10 P-values; these will be converted to unadjusted p-values in the &#x27;P&#x27; column.
No INFO (SI) column detected.
Standardising column headers.
First line of summary statistics file: 
SNP	chr	BP	end	REF	ALT	FILTER	AF	ES	LP	SE	P	
Ensuring parameters comply with LDSC format.

Setting `compute_z=TRUE` to comply with LDSC format.
Summary statistics report:
   - 101 rows
   - 101 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Checking for correct direction of A1 (reference) and A2 (alternative allele).
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 101 SNPs using BSgenome::snpsById...
BSgenome::snpsById done in 27 seconds.
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Computing Z-score from P using formula: `sign(BETA)*sqrt(stats::qchisq(P,1,lower=FALSE)`
Assigning N=1001 for all SNPs.
2 SNPs (2%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd368295f.tsv.gz
Summary statistics report:
   - 101 rows (100% of original 101 rows)
   - 101 unique variants
   - 0 genome-wide significant variants (P&lt;5e-8)
   - 1 chromosomes
Done munging in 0.489 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
           SNP CHR    BP A1 A2   END FILTER    FRQ    BETA       LP     SE
1:  rs58108140   1 10583  G  A 10583   PASS 0.1589  0.0312 0.369267 0.0393
2:    rs806731   1 30923  G  T 30923   PASS 0.7843 -0.0114 0.126854 0.0353
3: rs116400033   1 51479  T  A 51479   PASS 0.1829  0.0711 1.262410 0.0370
4: rs146477069   1 54421  A  G 54421   PASS 0.0352 -0.0240 0.112102 0.0830
            P          Z    N
1: 0.42730011  0.7938202 1001
2: 0.74669974 -0.3229941 1001
3: 0.05464998  1.9216487 1001
4: 0.77249913 -0.2891075 1001
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd785ea2cb.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd308a6
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for incorrect base-pair positions
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Sorting coordinates with &#x27;data.table&#x27;.
.tsv
=== write tests ===
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd36506092.tsv

=== read tests ===
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd36506092.tsv
Checking for empty columns.
.tsv.gz
=== write tests ===
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd3be5fe03.tsv.gz

=== read tests ===
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd3be5fe03.tsv.gz
Checking for empty columns.
.tsv.bgz
=== write tests ===
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd2878a7a2.tsv.bgz

=== read tests ===
Importing tabular bgz file: /tmp/RtmpiIp3aX/file12d3fd2878a7a2.tsv.bgz
Checking for empty columns.
.tsv.gz
=== write tests ===
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd32e69e1d.tsv
Writing uncompressed instead of gzipped to enable tabix indexing.
Converting full summary stats file to tabix format for fast querying...
Reading header.
Ensuring file is bgzipped.
Tabix-indexing file.
Removing temporary .tsv file.

=== read tests ===
Importing tabular bgz file: /tmp/RtmpiIp3aX/file12d3fd32e69e1d.tsv.bgz
Checking for empty columns.
.tsv.bgz
=== write tests ===
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd7fec7f9f.tsv
Writing uncompressed instead of gzipped to enable tabix indexing.
Converting full summary stats file to tabix format for fast querying...
Reading header.
Ensuring file is bgzipped.
Tabix-indexing file.
Removing temporary .tsv file.

=== read tests ===
Importing tabular bgz file: /tmp/RtmpiIp3aX/file12d3fd7fec7f9f.tsv.bgz
Checking for empty columns.
.csv
=== write tests ===
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd7dc9d12f.csv

=== read tests ===
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd7dc9d12f.csv
Checking for empty columns.
.csv.gz
=== write tests ===
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd786423.csv.gz

=== read tests ===
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd786423.csv.gz
Checking for empty columns.
.vcf
=== write tests ===


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

save_path suggests VCF output but write_vcf=FALSE. Switching output to tabular format (.tsv.gz).
Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd161cdc32.tsv.gz
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd161cdc32.tsv.gz

=== read tests ===
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd161cdc32.tsv.gz
Checking for empty columns.
.vcf.gz
=== write tests ===


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

save_path suggests VCF output but write_vcf=FALSE. Switching output to tabular format (.tsv.gz).
Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd3d2e07ef.tsv.gz
Writing in tabular format ==&gt; /tmp/RtmpiIp3aX/file12d3fd3d2e07ef.tsv.gz

=== read tests ===
Importing tabular file: /tmp/RtmpiIp3aX/file12d3fd3d2e07ef.tsv.gz
Checking for empty columns.
.vcf
=== write tests ===
Sorting coordinates with &#x27;data.table&#x27;.
Converting summary statistics to GenomicRanges.
Converting summary statistics to VRanges.
Writing in VCF format ==&gt;  /tmp/RtmpiIp3aX/file12d3fd202f4b1a.vcf

=== read tests ===
Using local VCF.
bgzip-compressing VCF file.
Finding empty VCF columns based on first 10,000 rows.
1 sample detected: GWAS
Constructing ScanVcfParam object.
Reading VCF file: single-threaded
Converting VCF to data.table.
Expanding VCF first, so number of rows may increase.
Checking for empty columns.
Time difference of 0.1 secs
VCF data.table contains: 93 rows x 11 columns.
Time difference of 0.5 secs
No INFO (SI) column detected.
.vcf.gz
=== write tests ===
Sorting coordinates with &#x27;data.table&#x27;.
Converting summary statistics to GenomicRanges.
Converting summary statistics to VRanges.
Writing in VCF format ==&gt;  /tmp/RtmpiIp3aX/file12d3fd73817127.vcf.gz

=== read tests ===
Using local VCF.
Finding empty VCF columns based on first 10,000 rows.
1 sample detected: GWAS
Constructing ScanVcfParam object.
Reading VCF file: single-threaded
Converting VCF to data.table.
Expanding VCF first, so number of rows may increase.
Checking for empty columns.
Time difference of 0.1 secs
VCF data.table contains: 93 rows x 11 columns.
Time difference of 0.5 secs
No INFO (SI) column detected.
.vcf
=== write tests ===
Sorting coordinates with &#x27;data.table&#x27;.
Converting summary statistics to GenomicRanges.
Converting summary statistics to VRanges.
Writing in VCF format ==&gt;  /tmp/RtmpiIp3aX/file12d3fd5119cc8.vcf
.vcf
=== write tests ===


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpiIp3aX/file12d3fd421ccea2.vcf.bgz
Sorting coordinates with &#x27;data.table&#x27;.
Converting summary statistics to GenomicRanges.
Converting summary statistics to VRanges.
Writing in VCF format ==&gt;  /tmp/RtmpiIp3aX/file12d3fd421ccea2.vcf.bgz

=== read tests ===
Using local VCF.
File already tabix-indexed.
Finding empty VCF columns based on first 10,000 rows.
1 sample detected: GWAS
Constructing ScanVcfParam object.
Reading VCF file: single-threaded
Converting VCF to data.table.
Expanding VCF first, so number of rows may increase.
Checking for empty columns.
Time difference of 0.1 secs
VCF data.table contains: 93 rows x 11 columns.
Time difference of 0.6 secs
No INFO (SI) column detected.
.vcf.bgz
=== write tests ===
Sorting coordinates with &#x27;data.table&#x27;.
Converting summary statistics to GenomicRanges.
Converting summary statistics to VRanges.
Writing in VCF format ==&gt;  /tmp/RtmpiIp3aX/file12d3fd2d31edd2.vcf.bgz

=== read tests ===
Using local VCF.
File already tabix-indexed.
Finding empty VCF columns based on first 10,000 rows.
1 sample detected: GWAS
Constructing ScanVcfParam object.
Reading VCF file: single-threaded
Converting VCF to data.table.
Expanding VCF first, so number of rows may increase.
Checking for empty columns.
Time difference of 0.1 secs
VCF data.table contains: 93 rows x 11 columns.
Time difference of 0.5 secs
No INFO (SI) column detected.
[ FAIL 0 | WARN 3 | SKIP 0 | PASS 192 ]

[ FAIL 0 | WARN 3 | SKIP 0 | PASS 192 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
774.665  71.510 891.139 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">MungeSumstats.Rcheck/MungeSumstats-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>compute_nsize</TD><TD style="text-align: right;">0.066</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.067</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>download_vcf</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>find_sumstats</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>format_sumstats</TD><TD style="text-align: right;">52.159</TD><TD style="text-align: right;"> 7.192</TD><TD style="text-align: right;">60.829</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>formatted_example</TD><TD style="text-align: right;">0.035</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_genome_builds</TD><TD style="text-align: right;">83.114</TD><TD style="text-align: right;"> 6.485</TD><TD style="text-align: right;">91.163</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_sumstats</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>index_tabular</TD><TD style="text-align: right;">0.056</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.061</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>index_vcf</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>liftover</TD><TD style="text-align: right;">1.231</TD><TD style="text-align: right;">0.128</TD><TD style="text-align: right;">6.707</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>list_sumstats</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>load_snp_loc_data</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>parse_logs</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>read_header</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>read_sumstats</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>read_vcf</TD><TD style="text-align: right;">2.148</TD><TD style="text-align: right;">0.240</TD><TD style="text-align: right;">2.390</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>standardise_header</TD><TD style="text-align: right;">0.025</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.025</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>vcf2df</TD><TD style="text-align: right;">0.694</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">0.739</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>write_sumstats</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
