<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for orthogene on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><B>[<A href="../index.html#O">O</A>]</B></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-10 01:25:52 -0000 (Fri, 10 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4219</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for orthogene on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the orthogene package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/orthogene.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1405</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">orthogene</A>&nbsp;1.5.1</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/orthogene">landing page</A>)</SPAN><BR>Brian Schilder<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-09&nbsp;03:02:02&nbsp;-0000&nbsp;(Thu,&nbsp;09&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/orthogene</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">ef65a54</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-12&nbsp;14:01:14&nbsp;-0000&nbsp;(Sat,&nbsp;12&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: orthogene</TD></TR>
<TR><TD><B>Version</B>: 1.5.1</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:orthogene.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings orthogene_1.5.1.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-09 17:34:33 -0000 (Thu, 09 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-09 17:50:08 -0000 (Thu, 09 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 934.4 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: orthogene.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:orthogene.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings orthogene_1.5.1.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/orthogene.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘orthogene/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘orthogene’ version ‘1.5.1’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘orthogene’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking R/sysdata.rda ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                         user system elapsed
plot_orthotree         20.973  0.727  78.281
prepare_tree           10.121  0.331  65.039
map_orthologs           1.047  7.070 102.001
all_genes               5.857  0.292 125.997
convert_orthologs       5.138  0.115  77.229
aggregate_mapped_genes  3.145  0.107   5.707
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  Retrieving all organisms available in babelgene.
  Mapping species name: zebrafish
  Common name mapping found for zebrafish
  1 organism identified from search: 7955
  Preparing babelgene::orthologs_df.
  Gene table with 30,886 rows retrieved.
  Returning all 30,886 genes from zebrafish.
  Testing for gene overlap with: fly
  Retrieving all genes using: babelgene.
  Retrieving all organisms available in babelgene.
  Mapping species name: fly
  Common name mapping found for fly
  1 organism identified from search: 7227
  Preparing babelgene::orthologs_df.
  Killed
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘docker.Rmd’ using ‘UTF-8’... OK
  ‘infer_species.Rmd’ using ‘UTF-8’... OK
  ‘orthogene.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/orthogene.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">orthogene.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL orthogene
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘orthogene’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (orthogene)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">orthogene.Rcheck/tests/testthat.Rout.fail</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(orthogene)
&gt; 
&gt; test_check(&quot;orthogene&quot;)
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Mapping many:many rows.
1110012L19Rik : converting 1 row(s) --&gt; 2 row(s).
2610034B18Rik : converting 1 row(s) --&gt; 2 row(s).
AA415398 : converting 1 row(s) --&gt; 2 row(s).
Ankhd1 : converting 1 row(s) --&gt; 2 row(s).
Anxa8 : converting 1 row(s) --&gt; 2 row(s).
Apitd1 : converting 1 row(s) --&gt; 2 row(s).
Arhgap8 : converting 1 row(s) --&gt; 2 row(s).
Asb3 : converting 1 row(s) --&gt; 2 row(s).
C4a : converting 1 row(s) --&gt; 2 row(s).
C4b : converting 1 row(s) --&gt; 2 row(s).
Cbs : converting 1 row(s) --&gt; 2 row(s).
Ccz1 : converting 1 row(s) --&gt; 2 row(s).
Ckmt1 : converting 1 row(s) --&gt; 2 row(s).
Coro7 : converting 1 row(s) --&gt; 2 row(s).
Cryaa : converting 1 row(s) --&gt; 2 row(s).
D10Jhu81e : converting 1 row(s) --&gt; 2 row(s).
F8a : converting 1 row(s) --&gt; 3 row(s).
Fam21 : converting 1 row(s) --&gt; 2 row(s).
Fcgr4 : converting 1 row(s) --&gt; 2 row(s).
Gpr89 : converting 1 row(s) --&gt; 2 row(s).
Gstt2 : converting 1 row(s) --&gt; 2 row(s).
H3f3a : converting 1 row(s) --&gt; 2 row(s).
H3f3b : converting 1 row(s) --&gt; 2 row(s).
Hspa1a : converting 1 row(s) --&gt; 2 row(s).
Icosl : converting 1 row(s) --&gt; 2 row(s).
Klhl23 : converting 1 row(s) --&gt; 2 row(s).
Mrpl23 : converting 1 row(s) --&gt; 2 row(s).
Nbl1 : converting 1 row(s) --&gt; 2 row(s).
Nomo1 : converting 1 row(s) --&gt; 3 row(s).
Pmf1 : converting 1 row(s) --&gt; 2 row(s).
Pom121 : converting 1 row(s) --&gt; 2 row(s).
Pramef8 : converting 1 row(s) --&gt; 2 row(s).
Prodh : converting 1 row(s) --&gt; 2 row(s).
Ranbp2 : converting 1 row(s) --&gt; 7 row(s).
Serf1 : converting 1 row(s) --&gt; 2 row(s).
Sgk3 : converting 1 row(s) --&gt; 2 row(s).
Slx1b : converting 1 row(s) --&gt; 2 row(s).
Smn1 : converting 1 row(s) --&gt; 2 row(s).
Spin2d : converting 1 row(s) --&gt; 2 row(s).
Aggregating rows using: monocle3
Converting obj to sparseMatrix.
Matrix aggregated:
  - Input: 15,259 x 7 
  - Output: 13,316 x 7
Retrieving all organisms available in gprofiler.
Using stored `gprofiler_orgs`.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: mmusculus
482 / 482 (100%) genes mapped.
Aggregating rows using: monocle3
Converting obj to sparseMatrix.
Matrix aggregated:
  - Input: 482 x 7 
  - Output: 92 x 7
Loading required namespace: DelayedArray
Converting to DelayedArray.
Mapping many:many rows.
Converting obj to sparseMatrix.
1110012L19Rik : converting 1 row(s) --&gt; 2 row(s).
2610034B18Rik : converting 1 row(s) --&gt; 2 row(s).
AA415398 : converting 1 row(s) --&gt; 2 row(s).
Ankhd1 : converting 1 row(s) --&gt; 2 row(s).
Anxa8 : converting 1 row(s) --&gt; 2 row(s).
Apitd1 : converting 1 row(s) --&gt; 2 row(s).
Arhgap8 : converting 1 row(s) --&gt; 2 row(s).
Asb3 : converting 1 row(s) --&gt; 2 row(s).
C4a : converting 1 row(s) --&gt; 2 row(s).
C4b : converting 1 row(s) --&gt; 2 row(s).
Cbs : converting 1 row(s) --&gt; 2 row(s).
Ccz1 : converting 1 row(s) --&gt; 2 row(s).
Ckmt1 : converting 1 row(s) --&gt; 2 row(s).
Coro7 : converting 1 row(s) --&gt; 2 row(s).
Cryaa : converting 1 row(s) --&gt; 2 row(s).
D10Jhu81e : converting 1 row(s) --&gt; 2 row(s).
F8a : converting 1 row(s) --&gt; 3 row(s).
Fam21 : converting 1 row(s) --&gt; 2 row(s).
Fcgr4 : converting 1 row(s) --&gt; 2 row(s).
Gpr89 : converting 1 row(s) --&gt; 2 row(s).
Gstt2 : converting 1 row(s) --&gt; 2 row(s).
H3f3a : converting 1 row(s) --&gt; 2 row(s).
H3f3b : converting 1 row(s) --&gt; 2 row(s).
Hspa1a : converting 1 row(s) --&gt; 2 row(s).
Icosl : converting 1 row(s) --&gt; 2 row(s).
Klhl23 : converting 1 row(s) --&gt; 2 row(s).
Mrpl23 : converting 1 row(s) --&gt; 2 row(s).
Nbl1 : converting 1 row(s) --&gt; 2 row(s).
Nomo1 : converting 1 row(s) --&gt; 3 row(s).
Pmf1 : converting 1 row(s) --&gt; 2 row(s).
Pom121 : converting 1 row(s) --&gt; 2 row(s).
Pramef8 : converting 1 row(s) --&gt; 2 row(s).
Prodh : converting 1 row(s) --&gt; 2 row(s).
Ranbp2 : converting 1 row(s) --&gt; 7 row(s).
Serf1 : converting 1 row(s) --&gt; 2 row(s).
Sgk3 : converting 1 row(s) --&gt; 2 row(s).
Slx1b : converting 1 row(s) --&gt; 2 row(s).
Smn1 : converting 1 row(s) --&gt; 2 row(s).
Spin2d : converting 1 row(s) --&gt; 2 row(s).
Aggregating rows using: monocle3
Converting obj to sparseMatrix.
Matrix aggregated:
  - Input: 15,259 x 7 
  - Output: 13,316 x 7
Converting to DelayedArray.
Converting mouse ==&gt; human orthologs using: gprofiler
Retrieving all organisms available in gprofiler.
Using stored `gprofiler_orgs`.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: mmusculus
Retrieving all organisms available in gprofiler.
Using stored `gprofiler_orgs`.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: hsapiens
Mapping many:many rows.
Converting obj to sparseMatrix.
1700030C10Rik : converting 1 row(s) --&gt; 2 row(s).
4933427D14Rik : converting 1 row(s) --&gt; 2 row(s).
A530058N18Rik : converting 1 row(s) --&gt; 2 row(s).
Adat3 : converting 1 row(s) --&gt; 2 row(s).
Ahcy : converting 1 row(s) --&gt; 2 row(s).
Aldoa : converting 1 row(s) --&gt; 2 row(s).
Ap4b1 : converting 1 row(s) --&gt; 2 row(s).
Arfip1 : converting 1 row(s) --&gt; 2 row(s).
Arhgap26 : converting 1 row(s) --&gt; 2 row(s).
Arxes1 : converting 1 row(s) --&gt; 2 row(s).
Arxes2 : converting 1 row(s) --&gt; 2 row(s).
Asnsd1 : converting 1 row(s) --&gt; 2 row(s).
Atn1 : converting 1 row(s) --&gt; 2 row(s).
Atp5o : converting 1 row(s) --&gt; 2 row(s).
Bcl2l2 : converting 1 row(s) --&gt; 2 row(s).
Bfar : converting 1 row(s) --&gt; 2 row(s).
Btf3l4 : converting 1 row(s) --&gt; 2 row(s).
Calm1 : converting 1 row(s) --&gt; 3 row(s).
Calm2 : converting 1 row(s) --&gt; 3 row(s).
Calm3 : converting 1 row(s) --&gt; 3 row(s).
Cdk2ap1 : converting 1 row(s) --&gt; 2 row(s).
Cldn25 : converting 1 row(s) --&gt; 2 row(s).
Cox17 : converting 1 row(s) --&gt; 2 row(s).
Cplx2 : converting 1 row(s) --&gt; 2 row(s).
Ctnnd1 : converting 1 row(s) --&gt; 2 row(s).
Cycs : converting 1 row(s) --&gt; 2 row(s).
Ddit3 : converting 1 row(s) --&gt; 2 row(s).
Dohh : converting 1 row(s) --&gt; 2 row(s).
Dpep2 : converting 1 row(s) --&gt; 2 row(s).
Dpm1 : converting 1 row(s) --&gt; 2 row(s).
Eno1 : converting 1 row(s) --&gt; 2 row(s).
Erdr1 : converting 1 row(s) --&gt; 2 row(s).
Erh : converting 1 row(s) --&gt; 2 row(s).
Evi2b : converting 1 row(s) --&gt; 2 row(s).
Fam220a : converting 1 row(s) --&gt; 2 row(s).
Fam78a : converting 1 row(s) --&gt; 2 row(s).
Ftl1 : converting 1 row(s) --&gt; 2 row(s).
Gcat : converting 1 row(s) --&gt; 2 row(s).
Gm12238 : converting 1 row(s) --&gt; 2 row(s).
Gm14295 : converting 1 row(s) --&gt; 3 row(s).
Gm14327 : converting 1 row(s) --&gt; 2 row(s).
Gm5089 : converting 1 row(s) --&gt; 2 row(s).
Gm5468 : converting 1 row(s) --&gt; 2 row(s).
Gm6498 : converting 1 row(s) --&gt; 2 row(s).
Gm6793 : converting 1 row(s) --&gt; 2 row(s).
Gm7903 : converting 1 row(s) --&gt; 2 row(s).
Hist1h2ad : converting 1 row(s) --&gt; 9 row(s).
Hist1h2an : converting 1 row(s) --&gt; 9 row(s).
Hmgn2 : converting 1 row(s) --&gt; 2 row(s).
Hspa1b : converting 1 row(s) --&gt; 2 row(s).
Kpna2 : converting 1 row(s) --&gt; 2 row(s).
Lilrb4 : converting 1 row(s) --&gt; 2 row(s).
Lrrc8a : converting 1 row(s) --&gt; 2 row(s).
Map2k7 : converting 1 row(s) --&gt; 2 row(s).
Mfap1a : converting 1 row(s) --&gt; 2 row(s).
Mfap1b : converting 1 row(s) --&gt; 2 row(s).
Myef2 : converting 1 row(s) --&gt; 2 row(s).
Myl12a : converting 1 row(s) --&gt; 2 row(s).
Myl12b : converting 1 row(s) --&gt; 2 row(s).
Ndor1 : converting 1 row(s) --&gt; 3 row(s).
Nnt : converting 1 row(s) --&gt; 2 row(s).
Nrg1 : converting 1 row(s) --&gt; 2 row(s).
Nudt10 : converting 1 row(s) --&gt; 2 row(s).
Nudt11 : converting 1 row(s) --&gt; 2 row(s).
Nudt8 : converting 1 row(s) --&gt; 2 row(s).
Nupl1 : converting 1 row(s) --&gt; 2 row(s).
Nutf2 : converting 1 row(s) --&gt; 2 row(s).
Pagr1a : converting 1 row(s) --&gt; 2 row(s).
Pam16 : converting 1 row(s) --&gt; 2 row(s).
Pde2a : converting 1 row(s) --&gt; 2 row(s).
Plp2 : converting 1 row(s) --&gt; 2 row(s).
Pms2 : converting 1 row(s) --&gt; 2 row(s).
Ppih : converting 1 row(s) --&gt; 2 row(s).
Psme2 : converting 1 row(s) --&gt; 2 row(s).
Psme2b : converting 1 row(s) --&gt; 2 row(s).
Ptp4a1 : converting 1 row(s) --&gt; 2 row(s).
Raver1 : converting 1 row(s) --&gt; 2 row(s).
Rnasek : converting 1 row(s) --&gt; 2 row(s).
Rnaset2a : converting 1 row(s) --&gt; 4 row(s).
Rnf26 : converting 1 row(s) --&gt; 2 row(s).
Rpl10 : converting 1 row(s) --&gt; 2 row(s).
Rpl34-ps1 : converting 1 row(s) --&gt; 2 row(s).
Rpl35 : converting 1 row(s) --&gt; 2 row(s).
Rpl36a : converting 1 row(s) --&gt; 2 row(s).
Rps27 : converting 1 row(s) --&gt; 2 row(s).
Rsc1a1 : converting 1 row(s) --&gt; 2 row(s).
Sap18 : converting 1 row(s) --&gt; 2 row(s).
Sft2d1 : converting 1 row(s) --&gt; 2 row(s).
Slc25a37 : converting 1 row(s) --&gt; 2 row(s).
Smok3b : converting 1 row(s) --&gt; 2 row(s).
Snhg4 : converting 1 row(s) --&gt; 2 row(s).
Snora75 : converting 1 row(s) --&gt; 4 row(s).
Snord22 : converting 1 row(s) --&gt; 2 row(s).
Snurf : converting 1 row(s) --&gt; 2 row(s).
Spag6 : converting 1 row(s) --&gt; 2 row(s).
St6galnac2 : converting 1 row(s) --&gt; 2 row(s).
Styx : converting 1 row(s) --&gt; 2 row(s).
Synj2bp : converting 1 row(s) --&gt; 2 row(s).
T2 : converting 1 row(s) --&gt; 2 row(s).
Taf9 : converting 1 row(s) --&gt; 2 row(s).
Tgtp1 : converting 1 row(s) --&gt; 2 row(s).
Timm10b : converting 1 row(s) --&gt; 2 row(s).
Tmsb10 : converting 1 row(s) --&gt; 2 row(s).
Tmsb15l : converting 1 row(s) --&gt; 2 row(s).
Tuba3a : converting 1 row(s) --&gt; 2 row(s).
Txnl4a : converting 1 row(s) --&gt; 2 row(s).
U05342 : converting 1 row(s) --&gt; 2 row(s).
U2af1l4 : converting 1 row(s) --&gt; 2 row(s).
Uba52 : converting 1 row(s) --&gt; 2 row(s).
Yy2 : converting 1 row(s) --&gt; 2 row(s).
Zbtb9 : converting 1 row(s) --&gt; 2 row(s).
Zc3h11a : converting 1 row(s) --&gt; 2 row(s).
Aggregating rows using: stats
Converting obj to sparseMatrix.
Matrix aggregated:
  - Input: 15,259 x 7 
  - Output: 12,726 x 7
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: zebrafish
Common name mapping found for zebrafish
1 organism identified from search: 7955
Gene table with 20,897 rows retrieved.
Returning all 20,897 genes from zebrafish.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: fly
Common name mapping found for fly
1 organism identified from search: 7227
Gene table with 8,438 rows retrieved.
Returning all 8,438 genes from fly.
Retrieving all genes using: babelgene.
Retrieving all organisms available in babelgene.
Mapping species name: dmelanogaster
1 organism identified from search: 7227
Preparing babelgene::orthologs_df.
Gene table with 20,237 rows retrieved.
20237
Retrieving all genes using: gprofiler
Retrieving all organisms available in gprofiler.
Using stored `gprofiler_orgs`.
Mapping species name: dmelanogaster
1 organism identified from search: dmelanogaster
Gene table with 4,493 rows retrieved.
4493
WARNING: In order to set gene_output=&#x27;rownames&#x27; while drop_nonorths=FALSE, must convert gene_df into a sparse matrix. Setting as_sparse=TRUE.
WARNING: In order to set gene_output=&#x27;rownames&#x27; while non121_strategy=&#x27;kbs&#x27;, must convert gene_df into a sparse matrix. Setting as_sparse=TRUE.
Converting to DelayedArray.
Preparing gene_df.
sparseMatrix format detected.
Extracting genes from rownames.
15,259 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
13,416 genes extracted.
Extracting genes from ortholog_gene.
13,416 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 46 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 56 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Setting ortholog_gene to rownames.
Converting obj to sparseMatrix.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   2,016 / 15,259 (13%)
Total genes remaining after convert_orthologs :
   13,243 / 15,259 (87%)
Preparing gene_df.
Dense matrix format detected.
Extracting genes from rownames.
15,259 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
13,416 genes extracted.
Extracting genes from ortholog_gene.
13,416 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 46 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 56 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Setting ortholog_gene to rownames.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   2,016 / 15,259 (13%)
Total genes remaining after convert_orthologs :
   13,243 / 15,259 (87%)
Preparing gene_df.
Dense matrix format detected.
Extracting genes from rownames.
15,259 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
13,416 genes extracted.
Extracting genes from ortholog_gene.
13,416 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 46 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 56 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
WARNING: Will convert gene_df from dense matrix to data.frame when gene_output=&#x27;columns&#x27;.
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   2,016 / 15,259 (13%)
Total genes remaining after convert_orthologs :
   13,243 / 15,259 (87%)
Preparing gene_df.
data.frame format detected.
Extracting genes from rownames.
15,259 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
13,416 genes extracted.
Extracting genes from ortholog_gene.
13,416 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 46 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 56 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   2,016 / 15,259 (13%)
Total genes remaining after convert_orthologs :
   13,243 / 15,259 (87%)
Preparing gene_df.
data.frame format detected.
Preparing gene_df.
data.frame format detected.
Extracting genes from rownames.
15,259 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Standardising gene names first.
Retrieving all organisms available in gprofiler.
Using stored `gprofiler_orgs`.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: mmusculus
Extracting genes from name.
14,193 genes extracted.
14,193 / 15,259 (93.01%) genes mapped.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without 1:1 orthologs.
Filtering gene_df with gene_map
Sorting rownames alphanumerically.
Adding input_gene col to gene_df.
Adding input_gene_standard col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   2,702 / 15,259 (18%)
Total genes remaining after convert_orthologs :
   12,557 / 15,259 (82%)
Preparing gene_df.
data.frame format detected.
+ orthologs previously converted.
Detected that gene_df was previously converted to orthologs.
 Skipping map_orthologs step.
Checking for genes without orthologs in human.
Extracting genes from input_gene.
12,557 genes extracted.
Extracting genes from ortholog_gene.
12,557 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 30 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   52 / 12,557 (0.41%)
Total genes remaining after convert_orthologs :
   12,505 / 12,557 (100%)
Preparing gene_df.
data.frame format detected.
Extracting genes from rownames.
15,259 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
13,416 genes extracted.
Extracting genes from ortholog_gene.
13,416 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 46 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 56 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Returning gene_map as dictionary

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   2,016 / 15,259 (13%)
Total genes remaining after convert_orthologs :
   13,243 / 15,259 (87%)
Preparing gene_df.
character format detected.
Converting to data.frame
Extracting genes from input_gene.
100 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
91 genes extracted.
Extracting genes from ortholog_gene.
91 genes extracted.
Checking for genes without 1:1 orthologs.
Filtering gene_df with gene_map
Returning gene_map as dictionary

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   9 / 100 (9%)
Total genes remaining after convert_orthologs :
   91 / 100 (91%)
Preparing gene_df.
character format detected.
Converting to data.frame
Extracting genes from input_gene.
100 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
91 genes extracted.
Extracting genes from ortholog_gene.
91 genes extracted.
Checking for genes without 1:1 orthologs.
Filtering gene_df with gene_map
Setting ortholog_gene to rownames.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   9 / 100 (9%)
Total genes remaining after convert_orthologs :
   91 / 100 (91%)
Preparing gene_df.
character format detected.
Converting to data.frame
Extracting genes from input_gene.
100 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
91 genes extracted.
Extracting genes from ortholog_gene.
91 genes extracted.
Checking for genes without 1:1 orthologs.
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   9 / 100 (9%)
Total genes remaining after convert_orthologs :
   91 / 100 (91%)
Preparing gene_df.
sparseMatrix format detected.
Extracting genes from rownames.
15,259 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
13,416 genes extracted.
Extracting genes from ortholog_gene.
13,416 genes extracted.
Checking for genes without 1:1 orthologs.
Filtering gene_df with gene_map
Mapping many:many rows.
1110012L19Rik : converting 1 row(s) --&gt; 2 row(s).
2610034B18Rik : converting 1 row(s) --&gt; 2 row(s).
AA415398 : converting 1 row(s) --&gt; 2 row(s).
Ankhd1 : converting 1 row(s) --&gt; 2 row(s).
Anxa8 : converting 1 row(s) --&gt; 2 row(s).
Apitd1 : converting 1 row(s) --&gt; 2 row(s).
Arhgap8 : converting 1 row(s) --&gt; 2 row(s).
Asb3 : converting 1 row(s) --&gt; 2 row(s).
C4a : converting 1 row(s) --&gt; 2 row(s).
C4b : converting 1 row(s) --&gt; 2 row(s).
Cbs : converting 1 row(s) --&gt; 2 row(s).
Ccz1 : converting 1 row(s) --&gt; 2 row(s).
Ckmt1 : converting 1 row(s) --&gt; 2 row(s).
Coro7 : converting 1 row(s) --&gt; 2 row(s).
Cryaa : converting 1 row(s) --&gt; 2 row(s).
D10Jhu81e : converting 1 row(s) --&gt; 2 row(s).
F8a : converting 1 row(s) --&gt; 3 row(s).
Fam21 : converting 1 row(s) --&gt; 2 row(s).
Fcgr4 : converting 1 row(s) --&gt; 2 row(s).
Gpr89 : converting 1 row(s) --&gt; 2 row(s).
Gstt2 : converting 1 row(s) --&gt; 2 row(s).
H3f3a : converting 1 row(s) --&gt; 2 row(s).
H3f3b : converting 1 row(s) --&gt; 2 row(s).
Hspa1a : converting 1 row(s) --&gt; 2 row(s).
Icosl : converting 1 row(s) --&gt; 2 row(s).
Klhl23 : converting 1 row(s) --&gt; 2 row(s).
Mrpl23 : converting 1 row(s) --&gt; 2 row(s).
Nbl1 : converting 1 row(s) --&gt; 2 row(s).
Nomo1 : converting 1 row(s) --&gt; 3 row(s).
Pmf1 : converting 1 row(s) --&gt; 2 row(s).
Pom121 : converting 1 row(s) --&gt; 2 row(s).
Pramef8 : converting 1 row(s) --&gt; 2 row(s).
Prodh : converting 1 row(s) --&gt; 2 row(s).
Ranbp2 : converting 1 row(s) --&gt; 7 row(s).
Serf1 : converting 1 row(s) --&gt; 2 row(s).
Sgk3 : converting 1 row(s) --&gt; 2 row(s).
Slx1b : converting 1 row(s) --&gt; 2 row(s).
Smn1 : converting 1 row(s) --&gt; 2 row(s).
Spin2d : converting 1 row(s) --&gt; 2 row(s).
Aggregating rows using: monocle3
Converting obj to sparseMatrix.
Matrix aggregated:
  - Input: 13,370 x 7 
  - Output: 13,316 x 7

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   1,943 / 15,259 (13%)
Total genes remaining after convert_orthologs :
   13,316 / 15,259 (87%)
Preparing gene_df.
sparseMatrix format detected.
Extracting genes from rownames.
15,259 genes extracted.
Converting mouse ==&gt; human orthologs using: babelgene
Retrieving all organisms available in babelgene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: Mus musculus
Retrieving all organisms available in babelgene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: Homo sapiens
Retrieving all genes using: babelgene.
Retrieving all organisms available in babelgene.
Mapping species name: Mus musculus
1 organism identified from search: 10090
Preparing babelgene::orthologs_df.
Gene table with 29,651 rows retrieved.
Checking for genes without orthologs in human.
Extracting genes from input_gene.
29,651 genes extracted.
Dropping 37 NAs of all kinds from input_gene.
Extracting genes from ortholog_gene.
29,614 genes extracted.
Dropping 146 NAs of all kinds from ortholog_gene.
Checking for genes without 1:1 orthologs.
Dropping 9,398 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 10,409 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Setting ortholog_gene to rownames.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   4,034 / 15,259 (26%)
Total genes remaining after convert_orthologs :
   11,225 / 15,259 (74%)
Preparing gene_df.
sparseMatrix format detected.
Extracting genes from rownames.
15,259 genes extracted.
Converting mouse ==&gt; mouse orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Checking for genes without orthologs in mouse.
Extracting genes from input_gene.
15,335 genes extracted.
Extracting genes from ortholog_gene.
15,335 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 1,342 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 807 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Setting ortholog_gene to rownames.
Converting obj to sparseMatrix.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   1,545 / 15,259 (10%)
Total genes remaining after convert_orthologs :
   13,714 / 15,259 (90%)
Preparing gene_df.
DelayedArray format detected.
Extracting genes from rownames.
15,259 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
13,416 genes extracted.
Extracting genes from ortholog_gene.
13,416 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 46 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 56 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Setting ortholog_gene to rownames.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   2,016 / 15,259 (13%)
Total genes remaining after convert_orthologs :
   13,243 / 15,259 (87%)
Generating gene background for mouse x rat ==&gt; human
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
21,207 genes extracted.
Converting mouse ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
17,355 genes extracted.
Extracting genes from ortholog_gene.
17,355 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 131 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 498 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   4,725 / 21,207 (22%)
Total genes remaining after convert_orthologs :
   16,482 / 21,207 (78%)
--

=========== REPORT SUMMARY ===========

16,482 / 21,207 (77.72%) target_species genes remain after ortholog conversion.
16,482 / 19,129 (86.16%) reference_species genes remain after ortholog conversion.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: rat
Common name mapping found for rat
1 organism identified from search: 10116
Gene table with 20,616 rows retrieved.
Returning all 20,616 genes from rat.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
20,616 genes extracted.
Converting rat ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: rat
Common name mapping found for rat
1 organism identified from search: 10116
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
16,989 genes extracted.
Extracting genes from ortholog_gene.
16,989 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 122 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 607 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   4,813 / 20,616 (23%)
Total genes remaining after convert_orthologs :
   15,803 / 20,616 (77%)
--

=========== REPORT SUMMARY ===========

15,803 / 20,616 (76.65%) target_species genes remain after ortholog conversion.
15,803 / 19,129 (82.61%) reference_species genes remain after ortholog conversion.
15,450 intersect background genes used.
Generating gene background for mouse x rat ==&gt; rat
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: rat
Common name mapping found for rat
1 organism identified from search: 10116
Gene table with 20,616 rows retrieved.
Returning all 20,616 genes from rat.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
21,207 genes extracted.
Converting mouse ==&gt; rat orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Retrieving all organisms available in homologene.
Mapping species name: rat
Common name mapping found for rat
1 organism identified from search: 10116
Checking for genes without orthologs in rat.
Extracting genes from input_gene.
20,872 genes extracted.
Extracting genes from ortholog_gene.
20,872 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 2,075 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 1,975 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   4,186 / 21,207 (20%)
Total genes remaining after convert_orthologs :
   17,021 / 21,207 (80%)
--

=========== REPORT SUMMARY ===========

17,021 / 21,207 (80.26%) target_species genes remain after ortholog conversion.
17,021 / 20,616 (82.56%) reference_species genes remain after ortholog conversion.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: rat
Common name mapping found for rat
1 organism identified from search: 10116
Gene table with 20,616 rows retrieved.
Returning all 20,616 genes from rat.
--

=========== REPORT SUMMARY ===========

20,616 / 20,616 (100%) target_species genes remain after ortholog conversion.
20,616 / 20,616 (100%) reference_species genes remain after ortholog conversion.
17,021 intersect background genes used.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: rat
Common name mapping found for rat
1 organism identified from search: 10116
Gene table with 20,616 rows retrieved.
Returning all 20,616 genes from rat.
Returning 20,616 unique genes from entire rat genome.
Generating gene background for human x rat ==&gt; mouse
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
19,129 genes extracted.
Converting human ==&gt; mouse orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Checking for genes without orthologs in mouse.
Extracting genes from input_gene.
17,355 genes extracted.
Extracting genes from ortholog_gene.
17,355 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 498 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 131 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   2,647 / 19,129 (14%)
Total genes remaining after convert_orthologs :
   16,482 / 19,129 (86%)
--

=========== REPORT SUMMARY ===========

16,482 / 19,129 (86.16%) target_species genes remain after ortholog conversion.
16,482 / 21,207 (77.72%) reference_species genes remain after ortholog conversion.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: rat
Common name mapping found for rat
1 organism identified from search: 10116
Gene table with 20,616 rows retrieved.
Returning all 20,616 genes from rat.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
20,616 genes extracted.
Converting rat ==&gt; mouse orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: rat
Common name mapping found for rat
1 organism identified from search: 10116
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Checking for genes without orthologs in mouse.
Extracting genes from input_gene.
20,872 genes extracted.
Extracting genes from ortholog_gene.
20,872 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 1,975 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 2,075 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   3,595 / 20,616 (17%)
Total genes remaining after convert_orthologs :
   17,021 / 20,616 (83%)
--

=========== REPORT SUMMARY ===========

17,021 / 20,616 (82.56%) target_species genes remain after ortholog conversion.
17,021 / 21,207 (80.26%) reference_species genes remain after ortholog conversion.
15,450 intersect background genes used.
Generating gene background for monkey x chimp ==&gt; human
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: monkey
Common name mapping found for monkey
1 organism identified from search: 9544
Gene table with 16,843 rows retrieved.
Returning all 16,843 genes from monkey.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
16,843 genes extracted.
Converting monkey ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: monkey
Common name mapping found for monkey
1 organism identified from search: 9544
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
15,675 genes extracted.
Extracting genes from ortholog_gene.
15,675 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 203 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 221 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   1,730 / 16,816 (10%)
Total genes remaining after convert_orthologs :
   15,086 / 16,816 (90%)
--

=========== REPORT SUMMARY ===========

15,075 / 16,816 (89.65%) target_species genes remain after ortholog conversion.
15,075 / 19,129 (78.81%) reference_species genes remain after ortholog conversion.
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: chimp
Common name mapping found for chimp
1 organism identified from search: 9598
Gene table with 18,730 rows retrieved.
Returning all 18,730 genes from chimp.
--
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
--
Preparing gene_df.
data.frame format detected.
Extracting genes from Gene.Symbol.
18,730 genes extracted.
Converting chimp ==&gt; human orthologs using: homologene
Retrieving all organisms available in homologene.
Mapping species name: chimp
Common name mapping found for chimp
1 organism identified from search: 9598
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Checking for genes without orthologs in human.
Extracting genes from input_gene.
17,737 genes extracted.
Extracting genes from ortholog_gene.
17,737 genes extracted.
Checking for genes without 1:1 orthologs.
Dropping 281 genes that have multiple input_gene per ortholog_gene (many:1).
Dropping 175 genes that have multiple ortholog_gene per input_gene (1:many).
Filtering gene_df with gene_map
Adding input_gene col to gene_df.
Adding ortholog_gene col to gene_df.

=========== REPORT SUMMARY ===========

Total genes dropped after convert_orthologs :
   1,511 / 18,703 (8.1%)
Total genes remaining after convert_orthologs :
   17,192 / 18,703 (92%)
--

=========== REPORT SUMMARY ===========

17,183 / 18,703 (91.87%) target_species genes remain after ortholog conversion.
17,183 / 19,129 (89.83%) reference_species genes remain after ortholog conversion.
14,187 intersect background genes used.
===== mouse tests =====
Preparing gene_df.
sparseMatrix format detected.
Extracting genes from rownames.
200 genes extracted.
Testing for gene overlap with: human
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
Testing for gene overlap with: monkey
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: monkey
Common name mapping found for monkey
1 organism identified from search: 9544
Gene table with 16,843 rows retrieved.
Returning all 16,843 genes from monkey.
Testing for gene overlap with: rat
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: rat
Common name mapping found for rat
1 organism identified from search: 10116
Gene table with 20,616 rows retrieved.
Returning all 20,616 genes from rat.
Testing for gene overlap with: mouse
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
Testing for gene overlap with: zebrafish
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: zebrafish
Common name mapping found for zebrafish
1 organism identified from search: 7955
Gene table with 20,897 rows retrieved.
Returning all 20,897 genes from zebrafish.
Testing for gene overlap with: fly
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: fly
Common name mapping found for fly
1 organism identified from search: 7227
Gene table with 8,438 rows retrieved.
Returning all 8,438 genes from fly.
Top match:
  - species: mouse 
  - percent_match: 96%
===== mouse tests2 =====
Preparing gene_df.
character format detected.
Converting to data.frame
Extracting genes from input_gene.
200 genes extracted.
Testing for gene overlap with: human
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
Testing for gene overlap with: monkey
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: monkey
Common name mapping found for monkey
1 organism identified from search: 9544
Gene table with 16,843 rows retrieved.
Returning all 16,843 genes from monkey.
Testing for gene overlap with: rat
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: rat
Common name mapping found for rat
1 organism identified from search: 10116
Gene table with 20,616 rows retrieved.
Returning all 20,616 genes from rat.
Testing for gene overlap with: mouse
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
Testing for gene overlap with: zebrafish
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: zebrafish
Common name mapping found for zebrafish
1 organism identified from search: 7955
Gene table with 20,897 rows retrieved.
Returning all 20,897 genes from zebrafish.
Testing for gene overlap with: fly
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: fly
Common name mapping found for fly
1 organism identified from search: 7227
Gene table with 8,438 rows retrieved.
Returning all 8,438 genes from fly.
Top match:
  - species: mouse 
  - percent_match: 96%
===== human tests =====
Preparing gene_df.
character format detected.
Converting to data.frame
Extracting genes from input_gene.
100 genes extracted.
Testing for gene overlap with: human
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Gene table with 19,129 rows retrieved.
Returning all 19,129 genes from human.
Testing for gene overlap with: monkey
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: monkey
Common name mapping found for monkey
1 organism identified from search: 9544
Gene table with 16,843 rows retrieved.
Returning all 16,843 genes from monkey.
Testing for gene overlap with: rat
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: rat
Common name mapping found for rat
1 organism identified from search: 10116
Gene table with 20,616 rows retrieved.
Returning all 20,616 genes from rat.
Testing for gene overlap with: mouse
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Gene table with 21,207 rows retrieved.
Returning all 21,207 genes from mouse.
Testing for gene overlap with: zebrafish
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: zebrafish
Common name mapping found for zebrafish
1 organism identified from search: 7955
Gene table with 20,897 rows retrieved.
Returning all 20,897 genes from zebrafish.
Testing for gene overlap with: fly
Retrieving all genes using: homologene.
Retrieving all organisms available in homologene.
Mapping species name: fly
Common name mapping found for fly
1 organism identified from search: 7227
Gene table with 8,438 rows retrieved.
Returning all 8,438 genes from fly.
Top match:
  - species: human 
  - percent_match: 98%
===== mouse tests =====
Preparing gene_df.
sparseMatrix format detected.
Extracting genes from rownames.
200 genes extracted.
Testing for gene overlap with: human
Retrieving all genes using: babelgene.
Retrieving all organisms available in babelgene.
Mapping species name: human
Common name mapping found for human
1 organism identified from search: 9606
Preparing babelgene::orthologs_df.
Gene table with 20,492 rows retrieved.
Returning all 20,492 genes from human.
Testing for gene overlap with: monkey
Retrieving all genes using: babelgene.
Retrieving all organisms available in babelgene.
Mapping species name: monkey
Common name mapping found for monkey
1 organism identified from search: 9544
Preparing babelgene::orthologs_df.
Gene table with 20,402 rows retrieved.
Returning all 20,402 genes from monkey.
Testing for gene overlap with: rat
Retrieving all genes using: babelgene.
Retrieving all organisms available in babelgene.
Mapping species name: rat
Common name mapping found for rat
1 organism identified from search: 10116
Preparing babelgene::orthologs_df.
Gene table with 21,427 rows retrieved.
Returning all 21,427 genes from rat.
Testing for gene overlap with: mouse
Retrieving all genes using: babelgene.
Retrieving all organisms available in babelgene.
Mapping species name: mouse
Common name mapping found for mouse
1 organism identified from search: 10090
Preparing babelgene::orthologs_df.
Gene table with 29,651 rows retrieved.
Returning all 29,651 genes from mouse.
Testing for gene overlap with: zebrafish
Retrieving all genes using: babelgene.
Retrieving all organisms available in babelgene.
Mapping species name: zebrafish
Common name mapping found for zebrafish
1 organism identified from search: 7955
Preparing babelgene::orthologs_df.
Gene table with 30,886 rows retrieved.
Returning all 30,886 genes from zebrafish.
Testing for gene overlap with: fly
Retrieving all genes using: babelgene.
Retrieving all organisms available in babelgene.
Mapping species name: fly
Common name mapping found for fly
1 organism identified from search: 7227
Preparing babelgene::orthologs_df.
Killed
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">orthogene.Rcheck/orthogene-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>aggregate_mapped_genes</TD><TD style="text-align: right;">3.145</TD><TD style="text-align: right;">0.107</TD><TD style="text-align: right;">5.707</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>all_genes</TD><TD style="text-align: right;">  5.857</TD><TD style="text-align: right;">  0.292</TD><TD style="text-align: right;">125.997</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>convert_orthologs</TD><TD style="text-align: right;"> 5.138</TD><TD style="text-align: right;"> 0.115</TD><TD style="text-align: right;">77.229</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>create_background</TD><TD style="text-align: right;">2.975</TD><TD style="text-align: right;">0.148</TD><TD style="text-align: right;">3.125</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>infer_species</TD><TD style="text-align: right;">1.354</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">1.398</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>map_genes</TD><TD style="text-align: right;">0.129</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.736</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>map_orthologs</TD><TD style="text-align: right;">  1.047</TD><TD style="text-align: right;">  7.070</TD><TD style="text-align: right;">102.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>map_species</TD><TD style="text-align: right;">0.522</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">1.158</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_orthotree</TD><TD style="text-align: right;">20.973</TD><TD style="text-align: right;"> 0.727</TD><TD style="text-align: right;">78.281</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>prepare_tree</TD><TD style="text-align: right;">10.121</TD><TD style="text-align: right;"> 0.331</TD><TD style="text-align: right;">65.039</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>report_orthologs</TD><TD style="text-align: right;">1.086</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">1.140</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
