<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for MethylSeekR on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><B>[<A href="../index.html#M">M</A>]</B></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-08 01:15:14 -0000 (Wed, 08 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4164</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for MethylSeekR on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the MethylSeekR package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/MethylSeekR.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1165</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">MethylSeekR</A>&nbsp;1.39.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/MethylSeekR">landing page</A>)</SPAN><BR>Lukas Burger<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-06&nbsp;00:12:45&nbsp;-0000&nbsp;(Mon,&nbsp;06&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/MethylSeekR</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">98c8238</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:07:49&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: MethylSeekR</TD></TR>
<TR><TD><B>Version</B>: 1.39.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:MethylSeekR.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings MethylSeekR_1.39.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-07 07:58:12 -0000 (Tue, 07 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-07 08:06:13 -0000 (Tue, 07 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 480.7 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: MethylSeekR.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:MethylSeekR.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings MethylSeekR_1.39.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/MethylSeekR.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘MethylSeekR/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘MethylSeekR’ version ‘1.39.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘MethylSeekR’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... NOTE
Malformed Description field: should contain one or more complete sentences.
Package listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘parallel’
A package should be listed in only one of these fields.
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
Namespace in Imports field not imported from: ‘geneplotter’
  All declared Imports should be used.
Package in Depends field not imported from: ‘rtracklayer’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
PMDviterbiSegmentation : &lt;anonymous&gt;: no visible global function
  definition for ‘values’
PMDviterbiSegmentation : &lt;anonymous&gt;: no visible global function
  definition for ‘predict’
PMDviterbiSegmentation : &lt;anonymous&gt;: no visible global function
  definition for ‘Rle’
PMDviterbiSegmentation : &lt;anonymous&gt;: no visible global function
  definition for ‘runValue’
calculateAlphaDistr : &lt;anonymous&gt;: no visible global function
  definition for ‘Rle’
calculateFDRs: no visible global function definition for ‘values’
calculateFDRs: no visible global function definition for ‘queryHits’
calculateFDRs: no visible global function definition for ‘values&lt;-’
calculateFDRs : &lt;anonymous&gt;: no visible global function definition for
  ‘Rle’
calculateFDRs : &lt;anonymous&gt;: no visible global function definition for
  ‘values’
calculateFDRs : &lt;anonymous&gt;: no visible global function definition for
  ‘runValue’
createGRangesObjectPMDSegmentation : &lt;anonymous&gt;: no visible global
  function definition for ‘Rle’
createGRangesObjectPMDSegmentation : &lt;anonymous&gt;: no visible global
  function definition for ‘nrun’
createGRangesObjectPMDSegmentation : &lt;anonymous&gt;: no visible global
  function definition for ‘runValue’
plotAlphaDistributionOneChr: no visible global function definition for
  ‘values’
plotFinalSegmentation: no visible global function definition for
  ‘values’
plotFinalSegmentation: no visible global function definition for ‘Rle’
plotPMDSegmentation: no visible global function definition for ‘values’
readMethylome: no visible global function definition for ‘values’
removeSNPs: no visible global function definition for ‘queryHits’
savePMDSegments: no visible global function definition for ‘values’
saveUMRLMRSegments: no visible global function definition for ‘values’
segmentUMRsLMRs: no visible global function definition for ‘values’
segmentUMRsLMRs : &lt;anonymous&gt;: no visible global function definition
  for ‘Rle’
segmentUMRsLMRs : &lt;anonymous&gt;: no visible global function definition
  for ‘values’
segmentUMRsLMRs : &lt;anonymous&gt;: no visible global function definition
  for ‘runValue’
segmentUMRsLMRs: no visible global function definition for ‘queryHits’
segmentUMRsLMRs: no visible global function definition for
  ‘subjectHits’
segmentUMRsLMRs: no visible global function definition for ‘Rle’
segmentUMRsLMRs: no visible global function definition for ‘values&lt;-’
segmentUMRsLMRs: no visible global function definition for ‘DataFrame’
segmentUMRsLMRs: no visible global function definition for
  ‘colorRampPalette’
trainPMDHMM: no visible global function definition for ‘values’
Undefined global functions or variables:
  DataFrame Rle colorRampPalette nrun predict queryHits runValue
  subjectHits values values&lt;-
Consider adding
  importFrom(&quot;grDevices&quot;, &quot;colorRampPalette&quot;)
  importFrom(&quot;stats&quot;, &quot;predict&quot;)
to your NAMESPACE file.
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘MethylSeekR-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: MethylSeekR-package
&gt; ### Title: Segmentation of Bis-seq methylation data
&gt; ### Aliases: MethylSeekR-package MethylSeekR
&gt; ### Keywords: package
&gt; 
&gt; ### ** Examples
&gt; 
&gt; 
&gt; library(MethylSeekR)
&gt; 
&gt; # get chromosome lengths
&gt; library(&quot;BSgenome.Hsapiens.UCSC.hg18&quot;)
Loading required package: BSgenome
Loading required package: Biostrings
Loading required package: XVector

Attaching package: ‘Biostrings’

The following object is masked from ‘package:base’:

    strsplit

&gt; sLengths=seqlengths(Hsapiens)
&gt; 
&gt; # read methylation data
&gt; methFname &lt;- system.file(&quot;extdata&quot;, &quot;Lister2009_imr90_hg18_chr22.tab&quot;,
+ package=&quot;MethylSeekR&quot;)
&gt; meth.gr &lt;- readMethylome(FileName=methFname, seqLengths=sLengths)
reading methylome data
Read 200000 records
&gt; 
&gt; #read SNP data
&gt; snpFname &lt;- system.file(&quot;extdata&quot;, &quot;SNVs_hg18_chr22.tab&quot;,
+ package=&quot;MethylSeekR&quot;)
&gt; snps.gr &lt;- readSNPTable(FileName=snpFname, seqLengths=sLengths)
reading SNP table
Read 11750 records
&gt; 
&gt; # remove SNPs
&gt; meth.gr &lt;- removeSNPs(meth.gr, snps.gr)
removing SNPs
11604 (6.2%) CpGs removed
&gt; 
&gt; #calculate alpha distribution for one chromosome
&gt; plotAlphaDistributionOneChr(m=meth.gr, chr.sel=&quot;chr22&quot;, num.cores=1)
determining alpha distribution for chromosome: chr22
&gt; 
&gt; #segment PMDs
&gt; PMDsegments.gr &lt;- segmentPMDs(m=meth.gr, chr.sel=&quot;chr22&quot;, num.cores=1,
+ seqLengths=sLengths)
training PMD-HMM on chromosome chr22
performing viterbi segmentation
creating GRanges object
&gt; 
&gt; #plot PMD segmentation examples
&gt; plotPMDSegmentation(m=meth.gr, segs=PMDsegments.gr, numRegions=1)
&gt; 
&gt; #save PMD segments
&gt; savePMDSegments(PMDs=PMDsegments.gr, GRangesFilename=&quot;PMDs.gr.rds&quot;,
+  TableFilename=&quot;PMDs.tab&quot;)
&gt; 
&gt; #load CpG islands
&gt; library(rtracklayer)
&gt; session &lt;- browserSession()
&gt; genome(session) &lt;- &quot;hg18&quot;
&gt; query &lt;- ucscTableQuery(session, table = &quot;cpgIslandExt&quot;)
Killed
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘MethylSeekR.Rnw’... failed to complete the test
 ERROR
Errors in running code in vignettes:
when running code in ‘MethylSeekR.Rnw’
  ...
&gt; plotAlphaDistributionOneChr(m = meth.gr, chr.sel = &quot;chr22&quot;, 
+     num.cores = 1)
determining alpha distribution for chromosome: chr22

&gt; PMDsegments.gr &lt;- segmentPMDs(m = meth.gr, chr.sel = &quot;chr22&quot;, 
+     seqLengths = sLengths, num.cores = 1)
training PMD-HMM on chromosome chr22
Killed

... incomplete output.  Crash?

* checking re-building of vignette outputs ... NOTE
Error(s) in re-building vignettes:
--- re-building ‘MethylSeekR.Rnw’ using Sweave
Loading required package: rtracklayer
Loading required package: GenomicRanges
Loading required package: stats4
Loading required package: BiocGenerics

Attaching package: ‘BiocGenerics’

The following objects are masked from ‘package:stats’:

    IQR, mad, sd, var, xtabs

The following objects are masked from ‘package:base’:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors

Attaching package: ‘S4Vectors’

The following objects are masked from ‘package:base’:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
Loading required package: parallel
Loading required package: mhsmm
Loading required package: mvtnorm
Loading required package: BSgenome
Loading required package: Biostrings
Loading required package: XVector

Attaching package: ‘Biostrings’

The following object is masked from ‘package:base’:

    strsplit

reading methylome data
Read 200000 records
reading SNP table
Read 11750 records
removing SNPs
11604 (6.2%) CpGs removed
determining alpha distribution for chromosome: chr22
training PMD-HMM on chromosome chr22
Killed

* checking PDF version of manual ... OK
* DONE

Status: 2 ERRORs, 4 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/MethylSeekR.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">MethylSeekR.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL MethylSeekR
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘MethylSeekR’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (MethylSeekR)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<HR>
<H3>Example timings</H3>
<P><SPAN class="filename">MethylSeekR.Rcheck/MethylSeekR-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
