<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for phyloseq on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><B>[<A href="../index.html#P">P</A>]</B></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-03 02:36:15 -0000 (Fri, 03 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4039</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for phyloseq on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the phyloseq package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/phyloseq.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1470</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">phyloseq</A>&nbsp;1.43.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/phyloseq">landing page</A>)</SPAN><BR>Paul J. McMurdie<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-01&nbsp;03:13:00&nbsp;-0000&nbsp;(Wed,&nbsp;01&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/phyloseq</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">371958d</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:06:40&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: phyloseq</TD></TR>
<TR><TD><B>Version</B>: 1.43.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:phyloseq.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings phyloseq_1.43.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-02 15:55:38 -0000 (Thu, 02 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-02 16:05:07 -0000 (Thu, 02 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 569.8 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: phyloseq.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 1</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:phyloseq.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings phyloseq_1.43.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/phyloseq.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘phyloseq/DESCRIPTION’ ... OK
* this is package ‘phyloseq’ version ‘1.43.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .travis.yml
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘phyloseq’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
DPCoA: no visible global function definition for ‘as.dist’
chunkReOrder: no visible global function definition for ‘tail’
chunkReOrder: no visible global function definition for ‘head’
export_env_file: no visible global function definition for
  ‘write.table’
export_mothur_dist: no visible global function definition for ‘as.dist’
export_mothur_dist: no visible global function definition for
  ‘write.table’
fastUniFrac: no visible global function definition for ‘combn’
fastUniFrac: no visible global function definition for ‘as.dist’
import_RDP_otu: no visible global function definition for ‘read.table’
import_env_file: no visible global function definition for ‘read.table’
import_mothur_constaxonomy: no visible global function definition for
  ‘read.table’
import_mothur_dist: no visible global function definition for ‘as.dist’
import_mothur_groups: no visible global function definition for
  ‘read.table’
import_mothur_shared: no visible global function definition for
  ‘read.table’
import_qiime_otu_tax: no visible global function definition for ‘:=’
import_qiime_otu_tax: no visible binding for global variable ‘Consensus
  Lineage’
import_qiime_otu_tax: no visible binding for global variable ‘#OTU ID’
import_qiime_sample_data: no visible global function definition for
  ‘read.table’
import_uparse: no visible global function definition for ‘:=’
import_uparse: no visible binding for global variable ‘count’
import_uparse: no visible binding for global variable ‘queryString’
import_uparse: no visible binding for global variable ‘queryID’
import_uparse: no visible binding for global variable ‘Classification’
import_uparse: no visible global function definition for
  ‘dcast.data.table’
import_uparse: no visible binding for global variable ‘OTULabel’
import_usearch_uc: no visible global function definition for ‘:=’
import_usearch_uc: no visible binding for global variable ‘read’
microbio_me_qiime: no visible global function definition for
  ‘download.file’
microbio_me_qiime: no visible global function definition for ‘unzip’
microbio_me_qiime: no visible global function definition for ‘untar’
nodeplotboot : &lt;anonymous&gt;: no visible global function definition for
  ‘complete.cases’
nodeplotboot : &lt;anonymous&gt;: no visible binding for global variable ‘x’
nodeplotboot : &lt;anonymous&gt;: no visible binding for global variable ‘y’
nodeplotdefault : &lt;anonymous&gt;: no visible binding for global variable
  ‘x’
nodeplotdefault : &lt;anonymous&gt;: no visible binding for global variable
  ‘y’
nodeplotdefault : &lt;anonymous&gt;: no visible binding for global variable
  ‘label’
ordinate: no visible global function definition for ‘as.formula’
plot_clusgap: no visible binding for global variable ‘k’
plot_clusgap: no visible binding for global variable ‘gap’
plot_clusgap: no visible binding for global variable ‘SE.sim’
plot_heatmap: no visible global function definition for
  ‘capture.output’
plot_heatmap: no visible binding for global variable ‘Sample’
plot_heatmap: no visible binding for global variable ‘OTU’
plot_heatmap: no visible binding for global variable ‘Abundance’
plot_net : vertex_layout: no visible binding for global variable ‘x’
plot_net : link_layout: no visible binding for global variable ‘x’
plot_net : link_layout: no visible binding for global variable ‘y’
plot_net: no visible binding for global variable ‘x’
plot_net: no visible binding for global variable ‘y’
plot_net: no visible binding for global variable ‘xend’
plot_net: no visible binding for global variable ‘yend’
plot_network: no visible binding for global variable ‘x’
plot_network: no visible binding for global variable ‘y’
plot_richness: no visible binding for global variable ‘value’
plot_richness: no visible binding for global variable ‘se’
plot_scree: no visible binding for global variable ‘axis’
plot_scree: no visible binding for global variable ‘eigenvalue’
plot_tree: no visible binding for global variable ‘xleft’
plot_tree: no visible binding for global variable ‘xright’
plot_tree: no visible binding for global variable ‘y’
plot_tree: no visible binding for global variable ‘x’
plot_tree: no visible binding for global variable ‘vmin’
plot_tree: no visible binding for global variable ‘vmax’
plot_tree: no visible binding for global variable ‘OTU’
plot_tree: no visible binding for global variable ‘label’
plot_tree: no visible binding for global variable ‘Abundance’
plot_tree: no visible binding for global variable ‘Sample’
plot_tree: no visible global function definition for ‘:=’
plot_tree: no visible binding for global variable ‘h.adj.index’
plot_tree: no visible binding for global variable ‘xdodge’
plot_tree: no visible binding for global variable ‘xfartiplab’
plot_tree: no visible binding for global variable ‘.SD’
rp.joint.fill: no visible global function definition for ‘relevel’
tip_glom: no visible global function definition for ‘as.dist’
tip_glom: no visible global function definition for ‘cutree’
tip_glom: no visible global function definition for ‘as.hclust’
tree_layout: no visible global function definition for ‘:=’
tree_layout: no visible binding for global variable ‘OTU’
tree_layout: no visible binding for global variable ‘V2’
tree_layout: no visible binding for global variable ‘xleft’
tree_layout: no visible binding for global variable ‘V1’
tree_layout: no visible binding for global variable ‘xright’
tree_layout: no visible binding for global variable ‘y’
tree_layout: no visible binding for global variable ‘x’
tree_layout: no visible binding for global variable ‘label’
tree_layout: no visible global function definition for ‘J’
tree_layout: no visible binding for global variable ‘vmin’
tree_layout: no visible binding for global variable ‘vmax’
JSD,matrix: no visible global function definition for ‘combn’
JSD,matrix: no visible binding for global variable ‘i’
JSD,matrix: no visible global function definition for ‘as.dist’
capscale.phyloseq,phyloseq-formula-character: no visible global
  function definition for ‘as.formula’
capscale.phyloseq,phyloseq-formula-dist: no visible global function
  definition for ‘as.formula’
cca.phyloseq,phyloseq-formula: no visible global function definition
  for ‘as.formula’
distance,phyloseq-character: no visible global function definition for
  ‘as.dist’
merge_phyloseq_pair,sample_data-sample_data : &lt;anonymous&gt;: no visible
  binding for global variable ‘X0’
merge_samples,sample_data: no visible global function definition for
  ‘aggregate’
plot_phyloseq,phyloseq: no visible binding for global variable
  ‘esophagus’
Undefined global functions or variables:
  #OTU ID .SD := Abundance Classification Consensus Lineage J OTU
  OTULabel SE.sim Sample V1 V2 X0 aggregate as.dist as.formula
  as.hclust axis capture.output combn complete.cases count cutree
  dcast.data.table download.file eigenvalue esophagus gap h.adj.index
  head i k label queryID queryString read read.table relevel se tail
  untar unzip value vmax vmin write.table x xdodge xend xfartiplab
  xleft xright y yend
Consider adding
  importFrom(&quot;graphics&quot;, &quot;axis&quot;)
  importFrom(&quot;stats&quot;, &quot;aggregate&quot;, &quot;as.dist&quot;, &quot;as.formula&quot;, &quot;as.hclust&quot;,
             &quot;complete.cases&quot;, &quot;cutree&quot;, &quot;relevel&quot;)
  importFrom(&quot;utils&quot;, &quot;capture.output&quot;, &quot;combn&quot;, &quot;download.file&quot;, &quot;head&quot;,
             &quot;read.table&quot;, &quot;tail&quot;, &quot;untar&quot;, &quot;unzip&quot;, &quot;write.table&quot;)
to your NAMESPACE file.
* checking Rd files ... WARNING
checkRd: (7) import_qiime.Rd:120: Invalid URL: &quot;http://www.qiime.org/&quot;
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking installed files from ‘inst/doc’ ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
               user system elapsed
plot_richness 5.511  0.020   6.805
plot_net      4.840  0.024   5.589
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat-phyloseq.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘phyloseq-FAQ.Rmd’ using ‘UTF-8’... OK
  ‘phyloseq-analysis.Rmd’... OK
  ‘phyloseq-basics.Rmd’... OK
  ‘phyloseq-mixture-models.Rmd’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 WARNING, 2 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/phyloseq.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">phyloseq.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL phyloseq
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘phyloseq’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (phyloseq)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">phyloseq.Rcheck/tests/testthat-phyloseq.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(&quot;testthat&quot;)
&gt; packageVersion(&quot;phyloseq&quot;)
[1] &#x27;1.43.0&#x27;
&gt; # As suggested for opt-out option on testing by users, recommended by CRAN
&gt; # http://adv-r.had.co.nz/Testing.html
&gt; # Previously, best practice was to put all test files in inst/tests and ensure that R CMD check ran them by putting the following code in tests/test-all.R:  
&gt; # library(testthat)
&gt; # library(yourpackage)
&gt; # test_package(&quot;yourpackage&quot;)
&gt; # Now, recommended practice is to put your tests in tests/testthat, and ensure R CMD check runs them by putting the following code in tests/test-all.R:
&gt; # library(testthat)
&gt; # test_check(&quot;yourpackage&quot;)
&gt; # The advantage of this new structure is that the user has control over whether or not tests are installed using the –install-tests parameter to R CMD install, or INSTALL_opts = c(“–install-tests”) argument to install.packages(). I’m not sure why you wouldn’t want to install the tests, but now you have the flexibility as requested by CRAN maintainers.
&gt; test_check(&quot;phyloseq&quot;)
Loading required package: phyloseq
Found biom-format file, now parsing it... 
Done parsing biom... 
Importing Sample Metdadata from mapping file...
Merging the imported objects... 
Successfully merged, phyloseq-class created. 
 Returning... 
Found biom-format file, now parsing it... 
Done parsing biom... 
Importing Sample Metdadata from mapping file...
Merging the imported objects... 
Successfully merged, phyloseq-class created. 
 Returning... 
Reading `ucfile` into memory and parsing into table 
Initially read 100 entries. 
... Now removing unassigned OTUs (* or NA)... 
Removed 7 entries that had no OTU assignment. 
A total of 93 will be assigned to the OTU table.
Processing otu/tax file...
Reading file into memory prior to parsing...
Detecting first header line...
Header is on line 2  
Converting input file to a table...
Defining OTU table... 
Parsing taxonomy table...
Processing phylogenetic tree...
 /home/biocbuild/bbs-3.17-bioc/R/library/phyloseq/extdata/GP_tree_rand_short.newick.gz ...
Processing map file...
Processing otu/tax file...
Reading file into memory prior to parsing...
Detecting first header line...
Header is on line 2  
Converting input file to a table...
Defining OTU table... 
Parsing taxonomy table...
Processing phylogenetic tree...
 /home/biocbuild/bbs-3.17-bioc/R/library/phyloseq/extdata/GP_tree_rand_short.newick.gz ...
Processing Reference Sequences...
[ FAIL 0 | WARN 49 | SKIP 0 | PASS 604 ]

[ FAIL 0 | WARN 49 | SKIP 0 | PASS 604 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
 82.906   0.622 105.357 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">phyloseq.Rcheck/phyloseq-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>DPCoA</TD><TD style="text-align: right;">3.072</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">3.759</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>JSD</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>UniFrac-methods</TD><TD style="text-align: right;">0.112</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.219</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>access</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>assign-otu_table</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>assign-phy_tree</TD><TD style="text-align: right;">0.039</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.070</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>assign-sample_data</TD><TD style="text-align: right;">0.10</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.13</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>assign-sample_names</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>assign-tax_table</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>assign-taxa_are_rows</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>assign-taxa_names</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>build_tax_table</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>capscale-phyloseq-methods</TD><TD style="text-align: right;">0.790</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">1.128</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cca-rda-phyloseq-methods</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>chunkReOrder</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>data-GlobalPatterns</TD><TD style="text-align: right;">1.670</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">1.970</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>data-enterotype</TD><TD style="text-align: right;">1.051</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">1.265</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>data-esophagus</TD><TD style="text-align: right;">0.627</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.803</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>data-soilrep</TD><TD style="text-align: right;">1.665</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">2.392</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>distance</TD><TD style="text-align: right;">0.193</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.309</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>distanceMethodList</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>envHash2otu_table</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>estimate_richness</TD><TD style="text-align: right;">0.041</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.059</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>export_env_file</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>export_mothur_dist</TD><TD style="text-align: right;">0.039</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.054</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>extract-methods</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filter_taxa</TD><TD style="text-align: right;">1.914</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">2.086</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filterfun_sample</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.035</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>gapstat_ord</TD><TD style="text-align: right;">1.669</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">1.843</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genefilter_sample-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.component.classes</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_sample-methods</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_taxa-methods</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_taxa_unique</TD><TD style="text-align: right;">0.207</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.242</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_variable</TD><TD style="text-align: right;">0.164</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.170</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getslots.phyloseq</TD><TD style="text-align: right;">0.169</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.172</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_RDP_otu</TD><TD style="text-align: right;">0.551</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.594</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_biom</TD><TD style="text-align: right;">0.228</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.320</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_env_file</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_mothur</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_mothur_dist</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_pyrotagger_tab</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_qiime</TD><TD style="text-align: right;">0.918</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.127</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_qiime_otu_tax</TD><TD style="text-align: right;">0.809</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">0.984</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_qiime_sample_data</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_uparse</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_usearch_uc</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.033</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>index_reorder</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>intersect_taxa</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>make_network</TD><TD style="text-align: right;">2.002</TD><TD style="text-align: right;">0.071</TD><TD style="text-align: right;">2.246</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>merge_phyloseq</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>merge_phyloseq_pair-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>merge_samples-methods</TD><TD style="text-align: right;">0.685</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.936</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>merge_taxa-methods</TD><TD style="text-align: right;">0.062</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.083</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>microbio_me_qiime</TD><TD style="text-align: right;">0.615</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.685</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mt-methods</TD><TD style="text-align: right;">1.657</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.722</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nodeplotblank</TD><TD style="text-align: right;">0.352</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.428</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nodeplotboot</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nodeplotdefault</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nsamples-methods</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ntaxa-methods</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ordinate</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>otu_table-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>parseTaxonomy-functions</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>phy_tree-methods</TD><TD style="text-align: right;">0.159</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.171</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>phyloseq</TD><TD style="text-align: right;">0.022</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>phyloseq_to_deseq2</TD><TD style="text-align: right;">4.185</TD><TD style="text-align: right;">0.021</TD><TD style="text-align: right;">4.554</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>phyloseq_to_metagenomeSeq</TD><TD style="text-align: right;">2.075</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">2.380</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_bar</TD><TD style="text-align: right;">2.110</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">2.545</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_clusgap</TD><TD style="text-align: right;">3.784</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">4.824</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_heatmap</TD><TD style="text-align: right;">3.367</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">3.913</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_net</TD><TD style="text-align: right;">4.840</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">5.589</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_network</TD><TD style="text-align: right;">1.595</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">1.687</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_ordination</TD><TD style="text-align: right;">0.668</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.692</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_phyloseq-methods</TD><TD style="text-align: right;">0.264</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.396</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_richness</TD><TD style="text-align: right;">5.511</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">6.805</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_scree</TD><TD style="text-align: right;">1.817</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">3.065</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_tree</TD><TD style="text-align: right;">0.754</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.181</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>prune_samples-methods</TD><TD style="text-align: right;">0.421</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.637</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>prune_taxa-methods</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>psmelt</TD><TD style="text-align: right;">0.754</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.950</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rank_names</TD><TD style="text-align: right;">0.021</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rarefy_even_depth</TD><TD style="text-align: right;">0.088</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.104</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>read_tree</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>read_tree_greengenes</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.021</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>reconcile_categories</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>refseq-methods</TD><TD style="text-align: right;">0.155</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.175</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rm_outlierf</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sample_data-methods</TD><TD style="text-align: right;">0.054</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.122</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sample_names-methods</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sample_sums</TD><TD style="text-align: right;">0.025</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sample_variables</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>show-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>splat.phyloseq.objects</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>subset_ord_plot</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>subset_samples-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>subset_taxa-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>tax_glom</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>tax_table-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>taxa_names-methods</TD><TD style="text-align: right;">0.025</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.025</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>taxa_sums</TD><TD style="text-align: right;">0.031</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.031</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>threshrank</TD><TD style="text-align: right;">2.426</TD><TD style="text-align: right;">0.610</TD><TD style="text-align: right;">3.235</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>threshrankfun</TD><TD style="text-align: right;">0.051</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.055</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>tip_glom</TD><TD style="text-align: right;">0.796</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.867</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>topf</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>topk</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>topp</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>transformcounts</TD><TD style="text-align: right;">0.099</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.100</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>transpose-methods</TD><TD style="text-align: right;">0.479</TD><TD style="text-align: right;">0.394</TD><TD style="text-align: right;">1.543</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>tree_layout</TD><TD style="text-align: right;">0.602</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.619</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
