<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for pRoloc on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><B>[<A href="../index.html#P">P</A>]</B></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-13 06:49:49 -0000 (Mon, 13 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4222</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for pRoloc on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the pRoloc package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/pRoloc.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1521</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">pRoloc</A>&nbsp;1.39.1</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/pRoloc">landing page</A>)</SPAN><BR>Laurent Gatto<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-11&nbsp;01:58:29&nbsp;-0000&nbsp;(Sat,&nbsp;11&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/pRoloc</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">c3f181e</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-01-31&nbsp;13:25:34&nbsp;-0000&nbsp;(Tue,&nbsp;31&nbsp;Jan&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: pRoloc</TD></TR>
<TR><TD><B>Version</B>: 1.39.1</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:pRoloc.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings pRoloc_1.39.1.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-11 19:53:09 -0000 (Sat, 11 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-11 20:11:12 -0000 (Sat, 11 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 1082.7 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: pRoloc.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:pRoloc.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings pRoloc_1.39.1.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/pRoloc.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘pRoloc/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘pRoloc’ version ‘1.39.1’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘pRoloc’ can be installed ... OK
* used C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
&#x27;:::&#x27; calls which should be &#x27;::&#x27;:
  ‘MLInterfaces:::.macroF1’ ‘MLInterfaces:::.precision’
  ‘MLInterfaces:::.recall’ ‘MLInterfaces:::es2df’
  See the note in ?`:::` about the use of this operator.
Unexported object imported by a &#x27;:::&#x27; call: ‘caret:::predict.plsda’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
Found the following possibly unsafe calls:
File ‘pRoloc/R/annotation.R’:
  unlockBinding(&quot;params&quot;, .pRolocEnv)
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... OK
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                         user system elapsed
perTurboClassification 15.700  0.084  15.784
SpatProtVis-class      10.002  0.216  10.219
plot2D                  9.189  0.269   7.840
svmClassification       8.540  0.128   8.668
nnetClassification      7.885  0.056   7.942
rfClassification        7.458  0.052   7.511
ClustDistList-class     6.270  0.327  13.746
ksvmClassification      6.339  0.136   6.474
ClustDist-class         6.012  0.220  39.338
clustDist               5.657  0.152  11.905
addGoAnnotations        5.162  0.109  11.659
AnnotationParams-class  0.409  0.008   8.006
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘v01-pRoloc-tutorial.Rmd’ using ‘UTF-8’... OK
  ‘v02-pRoloc-ml.Rmd’ using ‘UTF-8’... OK
  ‘v03-pRoloc-bayesian.Rmd’ using ‘UTF-8’... OK
  ‘v04-pRoloc-goannotations.Rmd’ using ‘UTF-8’... OK
  ‘v05-pRoloc-transfer-learning.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... ERROR
Error(s) in re-building vignettes:
  ...
--- re-building ‘v01-pRoloc-tutorial.Rmd’ using rmarkdown
--- finished re-building ‘v01-pRoloc-tutorial.Rmd’

--- re-building ‘v02-pRoloc-ml.Rmd’ using rmarkdown
--- finished re-building ‘v02-pRoloc-ml.Rmd’

--- re-building ‘v03-pRoloc-bayesian.Rmd’ using rmarkdown
--- finished re-building ‘v03-pRoloc-bayesian.Rmd’

--- re-building ‘v04-pRoloc-goannotations.Rmd’ using rmarkdown
--- finished re-building ‘v04-pRoloc-goannotations.Rmd’

--- re-building ‘v05-pRoloc-transfer-learning.Rmd’ using rmarkdown
Quitting from lines 143-147 (v05-pRoloc-transfer-learning.Rmd) 
Error: processing vignette &#x27;v05-pRoloc-transfer-learning.Rmd&#x27; failed with diagnostics:
Timeout was reached: [www.ensembl.org:80] Operation timed out after 10000 milliseconds with 327022 bytes received
--- failed re-building ‘v05-pRoloc-transfer-learning.Rmd’

SUMMARY: processing the following file failed:
  ‘v05-pRoloc-transfer-learning.Rmd’

Error: Vignette re-building failed.
Execution halted

* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 3 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/pRoloc.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">pRoloc.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL pRoloc
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘pRoloc’ ...
** using staged installation
** libs
using C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppArmadillo/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall  -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppArmadillo/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall  -c dmvtCpp.cpp -o dmvtCpp.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppArmadillo/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall  -c pRoloc.cpp -o pRoloc.o
g++ -std=gnu++14 -shared -L/home/biocbuild/bbs-3.17-bioc/R/lib -L/usr/local/lib -o pRoloc.so RcppExports.o dmvtCpp.o pRoloc.o -llapack -L/home/biocbuild/bbs-3.17-bioc/R/lib -lRblas -lgfortran -lm -L/home/biocbuild/bbs-3.17-bioc/R/lib -lR
installing to /home/biocbuild/bbs-3.17-bioc/R/library/00LOCK-pRoloc/00new/pRoloc/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (pRoloc)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">pRoloc.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; 
&gt; # The setting of R_TESTS exists to work around an R bug. See
&gt; # https://github.com/hadley/testthat/issues/144
&gt; # We should remove it when the issue is resolved.
&gt; Sys.setenv(&quot;R_TESTS&quot; = &quot;&quot;)
&gt; 
&gt; library(&quot;testthat&quot;)
&gt; library(&quot;pRoloc&quot;)
Loading required package: MSnbase
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

Loading required package: mzR
Loading required package: Rcpp
Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: ProtGenerics

Attaching package: &#x27;ProtGenerics&#x27;

The following object is masked from &#x27;package:stats&#x27;:

    smooth


This is MSnbase version 2.25.1 
  Visit https://lgatto.github.io/MSnbase/ to get started.


Attaching package: &#x27;MSnbase&#x27;

The following object is masked from &#x27;package:base&#x27;:

    trimws

Loading required package: MLInterfaces
Loading required package: annotate
Loading required package: AnnotationDbi
Loading required package: IRanges
Loading required package: XML

Attaching package: &#x27;annotate&#x27;

The following object is masked from &#x27;package:mzR&#x27;:

    nChrom

Loading required package: cluster
Loading required package: BiocParallel

This is pRoloc version 1.39.1 
  Visit https://lgatto.github.io/pRoloc/ to get started.

&gt; library(&quot;pRolocdata&quot;)

This is pRolocdata version 1.37.0.
Use &#x27;pRolocdata()&#x27; to list available data sets.
&gt; 
&gt; test_check(&quot;pRoloc&quot;)
Object of class &quot;SpatProtVis&quot;
 Data: dunkley2006 
 Visualisation methods: PCA, MDS, kpca
Done.
Connecting to Biomart...
Object of class &quot;AnnotationParams&quot;
 Using the &#x27;ENSEMBL_MART_ENSEMBL&#x27; BioMart database
 Using the &#x27;hsapiens_gene_ensembl&#x27; dataset
 Using &#x27;uniprotswissprot&#x27; as filter
 Created on Sat Feb 11 20:03:17 2023
GO Term Evidence Code
 Experimental Evidence Codes
  EXP: Inferred from Experiment
   IDA: Inferred from Direct Assay
   IPI: Inferred from Physical Interaction
   IMP: Inferred from Mutant Phenotype
   IGI: Inferred from Genetic Interaction
   IEP: Inferred from Expression Pattern
 Computational Analysis Evidence Codes
  ISS: Inferred from Sequence or Structural Similarity
   ISO: Inferred from Sequence Orthology
   ISA: Inferred from Sequence Alignment
   ISM: Inferred from Sequence Model
   IGC: Inferred from Genomic Context
   IBA: Inferred from Biological aspect of Ancestor
   IBD: Inferred from Biological aspect of Descendant
   IKR: Inferred from Key Residues
   IRD: Inferred from Rapid Divergence
   RCA: inferred from Reviewed Computational Analysis
 Author Statement Evidence Codes
   TAS: Traceable Author Statement
   NAS: Non-traceable Author Statement
 Curator Statement Evidence Codes
   IC: Inferred by Curator
   ND: No biological Data available
 Automatically-assigned Evidence Codes
   IEA: Inferred from Electronic Annotation
 Obsolete Evidence Codes
   NR: Not Recorded
Connecting to Biomart...
organelleMarkers
ORG1 ORG2 ORG3 ORG4 ORG5 
   3    2    3    1    1 
organelleMarkers
ORG1 ORG2 ORG3 ORG4 ORG5 
   5    2    1    1    1 
organelleMarkers
   ORG1 unknown 
      3       7 
organelleMarkers
   ORG1 unknown 
      2       8 
[1] &quot;markers&quot;
[1] &quot;xx&quot;
[1] &quot;markers&quot;
[1] &quot;xx&quot;
[1] &quot;markers&quot;
[1] &quot;xx&quot;
[1] &quot;markers&quot;
# weights:  269
initial  value 666.839513 
iter  10 value 207.573433
iter  20 value 34.513121
iter  30 value 1.438035
iter  40 value 0.496168
iter  50 value 0.419018
iter  60 value 0.386372
iter  70 value 0.360129
iter  80 value 0.324298
iter  90 value 0.294123
iter 100 value 0.252996
final  value 0.252996 
stopped after 100 iterations
[1] &quot;xx&quot;
# weights:  269
initial  value 666.839513 
iter  10 value 207.573433
iter  20 value 34.513121
iter  30 value 1.438035
iter  40 value 0.496168
iter  50 value 0.419018
iter  60 value 0.386372
iter  70 value 0.360129
iter  80 value 0.324298
iter  90 value 0.294123
iter 100 value 0.252996
final  value 0.252996 
stopped after 100 iterations
[1] &quot;markers&quot;
[1] &quot;xx&quot;
[1] &quot;pd.markers&quot;
Common markers:  0 
Unique x markers:  49 
Unique y markers:  34 
Common unkowns:  11 
Unique x unknowns:  40 
Unique y unknowns:  66 
               my
mx              ER lumen ER membrane Golgi Mitochondrion  PM Plastid Ribosome
  ER lumen            14           0     0             0   0       0        0
  ER membrane          0          45     0             0   0       0        0
  Golgi                0           0    28             0   0       0        0
  Mitochondrion        0           0     0            55   0       0        0
  PM                   0           0     0             0  46       0        0
  Plastid              0           0     0             0   0      20        0
  Ribosome             0           0     0             0   0       0       19
  TGN                  0           0     0             0   0       0        0
  unknown              0           0     0             0   0       0        0
  vacuole              0           0     0             0   0       0        0
               my
mx              TGN unknown vacuole
  ER lumen        0       0       0
  ER membrane     0       0       0
  Golgi           0       0       0
  Mitochondrion   0       0       0
  PM              0       0       0
  Plastid         0       0       0
  Ribosome        0       0       0
  TGN            13       0       0
  unknown         0     428       0
  vacuole         0       0      21
[ FAIL 0 | WARN 7 | SKIP 0 | PASS 146 ]

[ FAIL 0 | WARN 7 | SKIP 0 | PASS 146 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
145.990   3.364 167.788 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">pRoloc.Rcheck/pRoloc-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>AnnotationParams-class</TD><TD style="text-align: right;">0.409</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">8.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ClustDist-class</TD><TD style="text-align: right;"> 6.012</TD><TD style="text-align: right;"> 0.220</TD><TD style="text-align: right;">39.338</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ClustDistList-class</TD><TD style="text-align: right;"> 6.270</TD><TD style="text-align: right;"> 0.327</TD><TD style="text-align: right;">13.746</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GenRegRes-class</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>QSep-class</TD><TD style="text-align: right;">0.545</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.554</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SpatProtVis-class</TD><TD style="text-align: right;">10.002</TD><TD style="text-align: right;"> 0.216</TD><TD style="text-align: right;">10.219</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addGoAnnotations</TD><TD style="text-align: right;"> 5.162</TD><TD style="text-align: right;"> 0.109</TD><TD style="text-align: right;">11.659</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addMarkers</TD><TD style="text-align: right;">0.239</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.248</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>checkFeatureNamesOverlap</TD><TD style="text-align: right;">0.465</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.476</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>checkFvarOverlap</TD><TD style="text-align: right;">0.039</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.039</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>chi2-methods</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>classWeights</TD><TD style="text-align: right;">0.144</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.144</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>clustDist</TD><TD style="text-align: right;"> 5.657</TD><TD style="text-align: right;"> 0.152</TD><TD style="text-align: right;">11.905</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>empPvalues</TD><TD style="text-align: right;">0.108</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.112</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fDataToUnknown</TD><TD style="text-align: right;">0.059</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.063</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filterBinMSnSet</TD><TD style="text-align: right;">0.210</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.214</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filterZeroCols</TD><TD style="text-align: right;">0.181</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.189</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getGOFromFeatures</TD><TD style="text-align: right;">0.285</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">1.200</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getMarkerClasses</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getMarkers</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.051</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getPredictions</TD><TD style="text-align: right;">0.339</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.344</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getStockcol</TD><TD style="text-align: right;">0.113</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.113</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>goIdToTerm</TD><TD style="text-align: right;">2.576</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">2.583</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>highlightOnPlot</TD><TD style="text-align: right;">0.410</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.415</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>knnClassification</TD><TD style="text-align: right;">1.633</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">1.697</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>knntlClassification</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ksvmClassification</TD><TD style="text-align: right;">6.339</TD><TD style="text-align: right;">0.136</TD><TD style="text-align: right;">6.474</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeGoSet</TD><TD style="text-align: right;">0.656</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">1.411</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>markerMSnSet</TD><TD style="text-align: right;">0.359</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.363</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>markers</TD><TD style="text-align: right;">0.073</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.072</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>minMarkers</TD><TD style="text-align: right;">0.055</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.055</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mixing_posterior_check</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>move2Ds</TD><TD style="text-align: right;">2.720</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">2.736</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mrkConsProfiles</TD><TD style="text-align: right;">0.252</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.260</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mrkHClust</TD><TD style="text-align: right;">0.092</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.096</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nbClassification</TD><TD style="text-align: right;">3.383</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">3.431</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nicheMeans2D</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nndist-methods</TD><TD style="text-align: right;">0.251</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.251</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nnetClassification</TD><TD style="text-align: right;">7.885</TD><TD style="text-align: right;">0.056</TD><TD style="text-align: right;">7.942</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>orgQuants</TD><TD style="text-align: right;">0.253</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.254</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pRolocmarkers</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>perTurboClassification</TD><TD style="text-align: right;">15.700</TD><TD style="text-align: right;"> 0.084</TD><TD style="text-align: right;">15.784</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>phenoDisco</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot2D</TD><TD style="text-align: right;">9.189</TD><TD style="text-align: right;">0.269</TD><TD style="text-align: right;">7.840</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot2Ds</TD><TD style="text-align: right;">0.476</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.489</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotConsProfiles</TD><TD style="text-align: right;">0.368</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.373</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotDist</TD><TD style="text-align: right;">0.240</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.244</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plsdaClassification</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rfClassification</TD><TD style="text-align: right;">7.458</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">7.511</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sampleMSnSet</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.080</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>showGOEvidenceCodes</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>spatial2D</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>svmClassification</TD><TD style="text-align: right;">8.540</TD><TD style="text-align: right;">0.128</TD><TD style="text-align: right;">8.668</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>testMSnSet</TD><TD style="text-align: right;">0.352</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.360</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>testMarkers</TD><TD style="text-align: right;">0.034</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.035</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>thetas</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>zerosInBinMSnSet</TD><TD style="text-align: right;">3.828</TD><TD style="text-align: right;">0.232</TD><TD style="text-align: right;">4.062</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
