<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for SAIGEgds on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><B>[<A href="../index.html#S">S</A>]</B></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-23 01:34:25 -0000 (Thu, 23 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4245</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for SAIGEgds on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the SAIGEgds package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/SAIGEgds.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1748</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">SAIGEgds</A>&nbsp;1.13.2</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/SAIGEgds">landing page</A>)</SPAN><BR>Xiuwen Zheng<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-21&nbsp;12:29:53&nbsp;-0000&nbsp;(Tue,&nbsp;21&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/SAIGEgds</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">05a3b25</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-02-17&nbsp;06:19:37&nbsp;-0000&nbsp;(Fri,&nbsp;17&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: SAIGEgds</TD></TR>
<TR><TD><B>Version</B>: 1.13.2</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:SAIGEgds.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings SAIGEgds_1.13.2.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-22 11:57:44 -0000 (Wed, 22 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-22 12:01:12 -0000 (Wed, 22 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 208.3 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: SAIGEgds.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:SAIGEgds.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings SAIGEgds_1.13.2.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/SAIGEgds.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘SAIGEgds/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘SAIGEgds’ version ‘1.13.2’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘SAIGEgds’ can be installed ... OK
* used C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... NOTE
  installed size is  6.9Mb
  sub-directories of 1Mb or more:
    libs   5.2Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
&#x27;:::&#x27; call which should be &#x27;::&#x27;: ‘SPAtest:::ScoreTest_wSaddleApprox_NULL_Model’
  See the note in ?`:::` about the use of this operator.
Unexported objects imported by &#x27;:::&#x27; calls:
  ‘SPAtest:::ScoreTest_wSaddleApprox_Get_X1’ ‘SeqArray:::.IsForking’
  ‘SeqArray:::.NumParallel’ ‘SeqArray:::.file_split’
  ‘SeqArray:::.seqGet2bGeno’ ‘SeqArray:::.seqProgForward’
  ‘SeqArray:::.seqProgress’ ‘SeqArray:::process_index’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... NOTE
GNU make is a SystemRequirements.
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                         user system elapsed
seqGLMM_GxG_spa        10.734  0.008  10.744
seqAssocGLMM_spaACAT_O  8.317  0.136   8.454
seqAssocGLMM_spaBurden  7.308  0.244   7.551
seqAssocGLMM_spaACAT_V  7.343  0.056   7.400
seqAssocGLMM_SPA        6.979  0.008   6.987
seqFitNullGLMM_SPA      6.216  0.008   6.224
SAIGEgds-package        5.995  0.016   6.031
glmmHeritability        5.872  0.008   5.880
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘SAIGEgds.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 4 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/SAIGEgds.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">SAIGEgds.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL SAIGEgds
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘SAIGEgds’ ...
** using staged installation
** libs
using C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
using C++11
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppArmadillo/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c SPATest.cpp -o SPATest.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppArmadillo/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c saige_fitnull.cpp -o saige_fitnull.o
saige_fitnull.cpp: In function ‘SEXPREC* saige_get_sparse(SEXP)’:
saige_fitnull.cpp:256:17: warning: comparison of integer expressions of different signedness: ‘const size_t’ {aka ‘const long unsigned int’} and ‘R_len_t’ {aka ‘int’} [-Wsign-compare]
  256 |         if (num &gt; Rf_length(geno))
      |             ~~~~^~~~~~~~~~~~~~~~~
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppArmadillo/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c saige_main.cpp -o saige_main.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppArmadillo/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c vectorization.cpp -o vectorization.o
g++ -std=gnu++11 -shared -L/home/biocbuild/bbs-3.17-bioc/R/lib -L/usr/local/lib -o SAIGEgds.so SPATest.o saige_fitnull.o saige_main.o vectorization.o -lm -llapack -L/home/biocbuild/bbs-3.17-bioc/R/lib -lRblas -lgfortran -lm -lgcc -L/home/biocbuild/bbs-3.17-bioc/R/lib -lR
installing to /home/biocbuild/bbs-3.17-bioc/R/library/00LOCK-SAIGEgds/00new/SAIGEgds/libs
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (SAIGEgds)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">SAIGEgds.Rcheck/tests/runTests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; BiocGenerics:::testPackage(&quot;SAIGEgds&quot;)
Chromosome 1, # of units: 100
Chromosome 2, # of units: 3
# of units in total: 103
SAIGE ACAT-O analysis (combining burden &amp; ACAT-V):
    # of samples: 1,000
    # of variants in total: 10,000
    # of units: 103
    avg. # of variants per unit: 193.2136
    min # of variants in a unit: 1
    max # of variants in a unit: 200
    sd  # of variants in a unit: 34.06529
    p-value threshold for SPA adjustment: 0.05
    variance ratio for approximation: 0.9410507
    variant weights: beta(1,1), beta(1,25)
    mac threshold for burden test: 10
    # of processes: 1
Calculating minor allele frequencies (MAF):
[..................................................]  0%, ETC: ---    [==================================================] 100%, completed, 0s
    MAF: avg (0.08934), min (0.00200), max (0.50000), sd (0.12705)
Calculating p-values:
[..................................................]  0%, ETC: ---    [==================================================] 100%, completed, 2s
Done.
SAIGE ACAT-V analysis:
    # of samples: 1,000
    # of variants in total: 10,000
    # of units: 103
    avg. # of variants per unit: 193.2136
    min # of variants in a unit: 1
    max # of variants in a unit: 200
    sd  # of variants in a unit: 34.06529
    p-value threshold for SPA adjustment: 0.05
    variance ratio for approximation: 0.9410507
    variant weights: beta(1,1), beta(1,25)
    mac threshold for burden test: 10
    # of processes: 1
Calculating minor allele frequencies (MAF):
[..................................................]  0%, ETC: ---    [==================================================] 100%, completed, 0s
    MAF: avg (0.08934), min (0.00200), max (0.50000), sd (0.12705)
Calculating p-values:
[..................................................]  0%, ETC: ---    [==================================================] 100%, completed, 2s
Done.
SAIGE burden analysis:
    # of samples: 1,000
    # of variants in total: 10,000
    # of units: 103
    avg. # of variants per unit: 193.2136
    min # of variants in a unit: 1
    max # of variants in a unit: 200
    sd  # of variants in a unit: 34.06529
    p-value threshold for SPA adjustment: 0.05
    variance ratio for approximation: 0.9410507
    variant weights: beta(1,1), beta(1,25)
    # of processes: 1
Calculating minor allele frequencies (MAF):
[..................................................]  0%, ETC: ---    [==================================================] 100%, completed, 0s
    MAF: avg (0.08934), min (0.00200), max (0.50000), sd (0.12705)
Calculating p-values:
[..................................................]  0%, ETC: ---    [==================================================] 100%, completed, 2s
Done.
SAIGE association analysis:
Wed Feb 22 12:00:29 2023
Filtering variants:
[..................................................]  0%, ETC: ---    [==================================================] 100%, completed, 0s
# of selected variants: 9,976
Fit the null model: y ~ x1 + x2 + var(GRM)
    # of samples: 1,000
    # of variants: 9,976
    using 1 thread
Transform on the design matrix with QR decomposition:
    new formula: y ~ x_0 + x_1 + x_2 - 1
Start loading SNP genotypes:
[..................................................]  0%, ETC: ---    [==================================================] 100%, completed, 0s
    using 6.6M (sparse matrix)
Binary outcome: y
       y Number Proportion
       0    902      0.902
       1     98      0.098
Initial fixed-effect coefficients:
         x_0        x_1        x_2
    2.520514 -0.7666948 -0.4557928
Initial variance component estimates, tau:
    Sigma_E: 1, Sigma_G: 0.5
Iteration 1:
    tau: (1, 0.4994116)
    fixed coeff: (2.520514, -0.7666948, -0.4557928)
Iteration 2:
    tau: (1, 0.3287896)
    fixed coeff: (2.521231, -0.776603, -0.4592503)
Iteration 3:
    tau: (1, 0.2817812)
    fixed coeff: (2.525954, -0.7738757, -0.4579659)
Iteration 4:
    tau: (1, 0.3211452)
    fixed coeff: (2.525719, -0.7730823, -0.4577413)
Iteration 5:
    tau: (1, 0.3361534)
    fixed coeff: (2.527166, -0.7739766, -0.4579633)
Final tau: (1, 0.3322063)
    fixed coeff: (2.527666, -0.774237, -0.4580237)
Calculate the average ratio of variances:
Wed Feb 22 12:00:35 2023
     1, maf: 0.0775, mac: 155,	ratio: 0.9387 (var1: 0.0736, var2: 0.0785)
     2, maf: 0.0355, mac: 71,	ratio: 0.9362 (var1: 0.0671, var2: 0.0716)
     3, maf: 0.0730, mac: 146,	ratio: 0.9323 (var1: 0.0799, var2: 0.0857)
     4, maf: 0.0160, mac: 32,	ratio: 0.9481 (var1: 0.0653, var2: 0.0689)
     5, maf: 0.0585, mac: 117,	ratio: 0.9382 (var1: 0.0743, var2: 0.0791)
     6, maf: 0.0155, mac: 31,	ratio: 0.9388 (var1: 0.0754, var2: 0.0803)
     7, maf: 0.3075, mac: 615,	ratio: 0.9400 (var1: 0.0521, var2: 0.0554)
     8, maf: 0.0715, mac: 143,	ratio: 0.9416 (var1: 0.0714, var2: 0.0759)
     9, maf: 0.0115, mac: 23,	ratio: 0.9375 (var1: 0.0844, var2: 0.09)
    10, maf: 0.0470, mac: 94,	ratio: 0.9466 (var1: 0.0719, var2: 0.0759)
    11, maf: 0.0310, mac: 62,	ratio: 0.9335 (var1: 0.073, var2: 0.0782)
    12, maf: 0.1340, mac: 268,	ratio: 0.9381 (var1: 0.0633, var2: 0.0675)
    13, maf: 0.0855, mac: 171,	ratio: 0.9412 (var1: 0.0746, var2: 0.0792)
    14, maf: 0.1610, mac: 322,	ratio: 0.9379 (var1: 0.0647, var2: 0.069)
    15, maf: 0.0340, mac: 68,	ratio: 0.9471 (var1: 0.0685, var2: 0.0724)
    16, maf: 0.0215, mac: 43,	ratio: 0.9353 (var1: 0.0805, var2: 0.086)
    17, maf: 0.0600, mac: 120,	ratio: 0.9489 (var1: 0.0707, var2: 0.0745)
    18, maf: 0.0285, mac: 57,	ratio: 0.9455 (var1: 0.0732, var2: 0.0774)
    19, maf: 0.0145, mac: 29,	ratio: 0.9436 (var1: 0.0705, var2: 0.0747)
    20, maf: 0.0130, mac: 26,	ratio: 0.9454 (var1: 0.0688, var2: 0.0728)
    21, maf: 0.1350, mac: 270,	ratio: 0.9422 (var1: 0.0624, var2: 0.0663)
    22, maf: 0.0315, mac: 63,	ratio: 0.9342 (var1: 0.0699, var2: 0.0748)
    23, maf: 0.0475, mac: 95,	ratio: 0.9348 (var1: 0.0737, var2: 0.0788)
    24, maf: 0.2475, mac: 495,	ratio: 0.9360 (var1: 0.06, var2: 0.0641)
    25, maf: 0.0135, mac: 27,	ratio: 0.9523 (var1: 0.0622, var2: 0.0654)
    26, maf: 0.4915, mac: 983,	ratio: 0.9433 (var1: 0.0362, var2: 0.0383)
    27, maf: 0.0525, mac: 105,	ratio: 0.9366 (var1: 0.0734, var2: 0.0783)
    28, maf: 0.0195, mac: 39,	ratio: 0.9421 (var1: 0.0741, var2: 0.0787)
    29, maf: 0.0270, mac: 54,	ratio: 0.9502 (var1: 0.0718, var2: 0.0756)
    30, maf: 0.0380, mac: 76,	ratio: 0.9453 (var1: 0.0733, var2: 0.0776)
    ratio avg. is 0.9410507, sd: 0.005353982
Wed Feb 22 12:00:36 2023
Done.
SAIGE association analysis:
Wed Feb 22 12:00:36 2023
Filtering variants:
[..................................................]  0%, ETC: ---    [==================================================] 100%, completed, 0s
# of selected variants: 9,976
Fit the null model: yy ~ x1 + x2 + var(GRM)
    # of samples: 1,000
    # of variants: 9,976
    using 1 thread
Transform on the design matrix with QR decomposition:
    new formula: y ~ x_0 + x_1 + x_2 - 1
Start loading SNP genotypes:
[..................................................]  0%, ETC: ---    [==================================================] 100%, completed, 0s
    using 6.6M (sparse matrix)
Quantitative outcome: yy
        mean        sd    min    max
    4.987525 0.9817471 1.6383 7.6701
Inverse normal transformation on residuals with standard deviation: 0.981695
Initial fixed-effect coefficients:
              x_0          x_1         x_2
    -8.300445e-17 -0.003167949 0.001143224
Initial variance component estimates, tau:
    Sigma_E: 0.481718, Sigma_G: 0.481718
Iteration 1:
    tau: (0.5780754, 0.4791097)
    fixed coeff: (-8.300445e-17, -0.003167949, 0.001143224)
Iteration 2:
    tau: (0.7700948, 0.1571539)
    fixed coeff: (7.969752e-07, -0.01596609, -0.004398386)
Iteration 3:
    tau: (0.8649753, 0.0670912)
    fixed coeff: (3.558187e-07, -0.01030783, 3.137729e-05)
Iteration 4:
    tau: (0.9159832, 0.02884355)
    fixed coeff: (2.850052e-07, -0.006806145, 0.00088392)
Iteration 5:
    tau: (0.942882, 0)
    fixed coeff: (1.393627e-06, -0.004853796, 0.001081237)
Final tau: (0.9701727, 0)
    fixed coeff: (-7.874859e-17, -0.003167949, 0.001143224)
Calculate the average ratio of variances:
Wed Feb 22 12:00:44 2023
     1, maf: 0.0775, mac: 155,	ratio: 1.0307 (var1: 1.02, var2: 0.985)
     2, maf: 0.0355, mac: 71,	ratio: 1.0307 (var1: 0.955, var2: 0.927)
     3, maf: 0.0730, mac: 146,	ratio: 1.0307 (var1: 1, var2: 0.975)
     4, maf: 0.0160, mac: 32,	ratio: 1.0307 (var1: 0.994, var2: 0.964)
     5, maf: 0.0585, mac: 117,	ratio: 1.0307 (var1: 1.01, var2: 0.984)
     6, maf: 0.0155, mac: 31,	ratio: 1.0307 (var1: 0.998, var2: 0.968)
     7, maf: 0.3075, mac: 615,	ratio: 1.0307 (var1: 0.707, var2: 0.686)
     8, maf: 0.0715, mac: 143,	ratio: 1.0307 (var1: 0.969, var2: 0.94)
     9, maf: 0.0115, mac: 23,	ratio: 1.0307 (var1: 1, var2: 0.974)
    10, maf: 0.0470, mac: 94,	ratio: 1.0307 (var1: 0.999, var2: 0.969)
    11, maf: 0.0310, mac: 62,	ratio: 1.0307 (var1: 0.997, var2: 0.967)
    12, maf: 0.1340, mac: 268,	ratio: 1.0307 (var1: 0.882, var2: 0.855)
    13, maf: 0.0855, mac: 171,	ratio: 1.0307 (var1: 0.975, var2: 0.946)
    14, maf: 0.1610, mac: 322,	ratio: 1.0307 (var1: 0.878, var2: 0.851)
    15, maf: 0.0340, mac: 68,	ratio: 1.0307 (var1: 0.99, var2: 0.96)
    16, maf: 0.0215, mac: 43,	ratio: 1.0307 (var1: 0.986, var2: 0.957)
    17, maf: 0.0600, mac: 120,	ratio: 1.0307 (var1: 0.987, var2: 0.958)
    18, maf: 0.0285, mac: 57,	ratio: 1.0307 (var1: 0.972, var2: 0.943)
    19, maf: 0.0145, mac: 29,	ratio: 1.0307 (var1: 1, var2: 0.97)
    20, maf: 0.0130, mac: 26,	ratio: 1.0307 (var1: 1, var2: 0.973)
    21, maf: 0.1350, mac: 270,	ratio: 1.0307 (var1: 0.866, var2: 0.84)
    22, maf: 0.0315, mac: 63,	ratio: 1.0307 (var1: 1.03, var2: 0.998)
    23, maf: 0.0475, mac: 95,	ratio: 1.0307 (var1: 0.954, var2: 0.925)
    24, maf: 0.2475, mac: 495,	ratio: 1.0307 (var1: 0.791, var2: 0.767)
    25, maf: 0.0135, mac: 27,	ratio: 1.0307 (var1: 0.999, var2: 0.969)
    26, maf: 0.4915, mac: 983,	ratio: 1.0307 (var1: 0.491, var2: 0.476)
    27, maf: 0.0525, mac: 105,	ratio: 1.0307 (var1: 0.98, var2: 0.951)
    28, maf: 0.0195, mac: 39,	ratio: 1.0307 (var1: 0.99, var2: 0.961)
    29, maf: 0.0270, mac: 54,	ratio: 1.0307 (var1: 0.972, var2: 0.943)
    30, maf: 0.0380, mac: 76,	ratio: 1.0307 (var1: 1.01, var2: 0.976)
    ratio avg. is 1.030744, sd: 9.45658e-16
Wed Feb 22 12:00:44 2023
Done.
SAIGE association analysis:
    # of samples: 1,000
    # of variants: 10,000
    MAF threshold: NaN
    MAC threshold: 4
    missing threshold for variants: 0.1
    p-value threshold for SPA adjustment: 0.05
    variance ratio for approximation: 0.9410507
    # of processes: 1
[..................................................]  0%, ETC: ---    [==================================================] 100%, completed, 0s
# of variants after filtering by MAF, MAC and missing thresholds: 10,000
Done.
SAIGE association analysis:
    # of samples: 1,000
    # of variants: 10,000
    MAF threshold: NaN
    MAC threshold: 4
    missing threshold for variants: 0.1
    p-value threshold for SPA adjustment: 0.05
    variance ratio for approximation: 1.030744
    # of processes: 1
[..................................................]  0%, ETC: ---    [==================================================] 100%, completed, 1s
# of variants after filtering by MAF, MAC and missing thresholds: 10,000
Done.


RUNIT TEST PROTOCOL -- Wed Feb 22 12:00:45 2023 
*********************************************** 
Number of test functions: 4 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
SAIGEgds RUnit Tests - 4 test functions, 0 errors, 0 failures
Number of test functions: 4 
Number of errors: 0 
Number of failures: 0 
&gt; 
&gt; proc.time()
   user  system elapsed 
 25.866   0.560  26.416 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">SAIGEgds.Rcheck/SAIGEgds-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>SAIGEgds-package</TD><TD style="text-align: right;">5.995</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">6.031</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>glmmHeritability</TD><TD style="text-align: right;">5.872</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">5.880</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pACAT</TD><TD style="text-align: right;">0.096</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.096</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>seqAssocGLMM_SPA</TD><TD style="text-align: right;">6.979</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">6.987</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>seqAssocGLMM_spaACAT_O</TD><TD style="text-align: right;">8.317</TD><TD style="text-align: right;">0.136</TD><TD style="text-align: right;">8.454</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>seqAssocGLMM_spaACAT_V</TD><TD style="text-align: right;">7.343</TD><TD style="text-align: right;">0.056</TD><TD style="text-align: right;">7.400</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>seqAssocGLMM_spaBurden</TD><TD style="text-align: right;">7.308</TD><TD style="text-align: right;">0.244</TD><TD style="text-align: right;">7.551</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>seqFitNullGLMM_SPA</TD><TD style="text-align: right;">6.216</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">6.224</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>seqGLMM_GxG_spa</TD><TD style="text-align: right;">10.734</TD><TD style="text-align: right;"> 0.008</TD><TD style="text-align: right;">10.744</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>seqSAIGE_LoadPval</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
