<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for sitadela on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.17</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><B>[<A href="..#S">S</A>]</B></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-27 02:34:43 -0000 (Mon, 27 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4259</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for sitadela on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the sitadela package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/sitadela.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1885</B>/2169</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">sitadela</A>&nbsp;1.7.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/sitadela">landing page</A>)</SPAN><BR>Panagiotis Moulos<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-23&nbsp;09:40:21&nbsp;-0000&nbsp;(Thu,&nbsp;23&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/sitadela</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">23f5097</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:24:14&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: sitadela</TD></TR>
<TR><TD><B>Version</B>: 1.7.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:sitadela.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings sitadela_1.7.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-24 18:43:40 -0000 (Fri, 24 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-24 18:53:38 -0000 (Fri, 24 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 597.9 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: sitadela.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:sitadela.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings sitadela_1.7.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/sitadela.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘sitadela/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘sitadela’ version ‘1.7.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘sitadela’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                     user system elapsed
addCustomAnnotation 5.956  0.228   6.186
testFuns            4.393  0.176   9.634
getAnnotation       1.316  0.128   8.731
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’
 ERROR
Running the tests in ‘tests/runTests.R’ failed.
Last 13 lines of output:
  1 Test Suite : 
  sitadela RUnit Tests - 8 test functions, 0 errors, 1 failure
  FAILURE in test_sitadela_ensembl_often: Error in checkTrue(is.null(f1)) : Test not TRUE
  
  
  Test files with failing tests
  
     test_sitadela.R 
       test_sitadela_ensembl_often 
  
  
  Error in BiocGenerics:::testPackage(&quot;sitadela&quot;) : 
    unit tests failed for package sitadela
  In addition: There were 17 warnings (use warnings() to see them)
  Execution halted
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘sitadela.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/sitadela.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">sitadela.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL sitadela
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘sitadela’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
sitadela database found at /home/biocbuild/.local/share/R/sitadela directory
** testing if installed package can be loaded from final location
sitadela database found at /home/biocbuild/.local/share/R/sitadela directory
** testing if installed package keeps a record of temporary installation path
* DONE (sitadela)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">sitadela.Rcheck/tests/runTests.Rout.fail</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; BiocGenerics:::testPackage(&quot;sitadela&quot;)
sitadela database found at /home/biocbuild/.local/share/R/sitadela directory
Scheduling 1 tests
==================================================


Running test 1 of 1 scheduled
Testing level gene from mm10 from latest version
Using Ensembl host https://apr2022.archive.ensembl.org
Test 1 failed with error:
Timeout was reached: [apr2022.archive.ensembl.org:443] Operation timed out after 10000 milliseconds with 350915 bytes received
Testing finished!

Summary
==================================================

Succesful tests: 0 out of 1
Failed tests   : 1 out of 1
 
Check the output for failure details
Timing stopped at: 0.692 0.056 13.96
Error in checkTrue(is.null(f1)) : Test not TRUE

Scheduling 1 tests
==================================================


Running test 1 of 1 scheduled
Testing level gene from danrer11 from latest version
Using Ensembl host https://apr2022.archive.ensembl.org
Test 1 successful!
Created 32020 features
Sample data:
                   chromosome start   end            gene_id gc_content strand
ENSDARG00000099104       chr1  6408 12027 ENSDARG00000099104      43.42      -
ENSDARG00000102407       chr1 11822 16373 ENSDARG00000102407      54.39      +
ENSDARG00000102097       chr1 18716 23389 ENSDARG00000102097      52.70      +
ENSDARG00000099319       chr1 25585 27255 ENSDARG00000099319      37.64      +
ENSDARG00000099640       chr1 27690 34330 ENSDARG00000099640      53.02      +
ENSDARG00000104071       chr1 36552 39191 ENSDARG00000104071      50.64      +
                    gene_name        biotype
ENSDARG00000099104      rpl24 protein_coding
ENSDARG00000102407      cep97 protein_coding
ENSDARG00000102097     nfkbiz protein_coding
ENSDARG00000099319 CU651657.1 protein_coding
ENSDARG00000099640        eed protein_coding
ENSDARG00000104071    hikeshi protein_coding
Testing finished!

Summary
==================================================

Succesful tests: 1 out of 1
Failed tests   : 0 out of 1
 
Scheduling 1 tests
==================================================


Running test 1 of 1 scheduled
Testing level transcript from danrer11 from latest version
Using Ensembl host https://apr2022.archive.ensembl.org
Test 1 successful!
Created 59253 features
Sample data:
                   chromosome start   end      transcript_id            gene_id
ENSDART00000164359       chr1  6408 12027 ENSDART00000164359 ENSDARG00000099104
ENSDART00000157701       chr1  6642  9919 ENSDART00000157701 ENSDARG00000099104
ENSDART00000158290       chr1  6642  7335 ENSDART00000158290 ENSDARG00000099104
ENSDART00000167898       chr1  6644 11725 ENSDART00000167898 ENSDARG00000099104
ENSDART00000166393       chr1 11822 16373 ENSDART00000166393 ENSDARG00000102407
ENSDART00000161842       chr1 11992 14058 ENSDART00000161842 ENSDARG00000102407
                   strand gene_name        biotype
ENSDART00000164359      -     rpl24 protein_coding
ENSDART00000157701      -     rpl24 protein_coding
ENSDART00000158290      -     rpl24 protein_coding
ENSDART00000167898      -     rpl24 protein_coding
ENSDART00000166393      +     cep97 protein_coding
ENSDART00000161842      +     cep97 protein_coding
Testing finished!

Summary
==================================================

Succesful tests: 1 out of 1
Failed tests   : 0 out of 1
 
Scheduling 2 tests
==================================================

Running test 1 of 2 scheduled
Opening sitadela SQLite database /tmp/RtmpdyitFS/file1cbc7211e02962
  Importing GTF /home/biocbuild/bbs-3.17-bioc/R/library/sitadela/dummy.gtf.gz as GTF to make id map
  Making id map
  Importing GTF /home/biocbuild/bbs-3.17-bioc/R/library/sitadela/dummy.gtf.gz as TxDb
Import genomic features from the file as a GRanges object ... OK
Prepare the &#x27;metadata&#x27; data frame ... OK
Make the TxDb object ... OK
Retrieving gene annotation for dummy from dummy_db version 1 from /home/biocbuild/bbs-3.17-bioc/R/library/sitadela/dummy.gtf.gz
Retrieving transcript annotation for dummy from dummy_db version 1
Retrieving summarized transcript annotation for dummy from dummy_db version 1
Retrieving 3&#x27; UTR annotation for dummy from dummy_db version 1
Retrieving summarized 3&#x27; UTR annotation per gene for dummy from dummy_db version 1
Retrieving summarized 3&#x27; UTR annotation per transcript for dummy from dummy_db version 1
Retrieving exon annotation for dummy from dummy_db version 1
Retrieving summarized exon annotation for dummy from dummy_db version 1
Retrieving extended exon annotation for dummy from dummy_db version 1
Retrieving summarized transcript exon annotation for dummy from dummy_db version 1
Test 1 successful!

Running test 2 of 2 scheduled
Test 2 successful!
Deleting temporary database /tmp/RtmpdyitFS/file1cbc7211e02962
Scheduling 2 tests
==================================================

Running test 1 of 2 scheduled

********************************************************
This is sitadela 1.7.0 genomic region annotation builder
********************************************************
sitadela database found at /tmp/RtmpdyitFS directory

========================================================
2023-02-24 18:49:33 - Try 1
========================================================

Opening sitadela SQLite database /tmp/RtmpdyitFS/file1cbc7238f6fc3a
Retrieving genome information for dm6 from ensembl
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Retrieving gene annotation for dm6 from ensembl version 102
Using Ensembl host https://nov2020.archive.ensembl.org
Retrieving transcript annotation for dm6 from ensembl version 102
Using Ensembl host https://nov2020.archive.ensembl.org
Merging transcripts for dm6 from ensembl version 102
Retrieving 3&#x27; UTR annotation for dm6 from ensembl version 102
Using Ensembl host https://nov2020.archive.ensembl.org
Merging gene 3&#x27; UTRs for dm6 from ensembl version 102
Merging transcript 3&#x27; UTRs for dm6 from ensembl version 102
Retrieving exon annotation for dm6 from ensembl version 102
Using Ensembl host https://nov2020.archive.ensembl.org
Retrieving extended exon annotation for dm6 from ensembl version 102
Using Ensembl host https://nov2020.archive.ensembl.org
Merging exons for dm6 from ensembl version 102
Merging exons for dm6 from ensembl version 102

-------------------------------------------------------
Building process complete!
-------------------------------------------------------

Test 1 successful!

Running test 2 of 2 scheduled
Test 2 successful!
Deleting temporary database /tmp/RtmpdyitFS/file1cbc7238f6fc3a
Loading required package: RMySQL
Loading required package: DBI
Scheduling 1 tests
==================================================

==========&gt; Now testing hg19
Connecting to UCSC database hg19...

---------------------------------------------------
Running test 1 of 1 scheduled
Source: refseq
Type  : transcript
Query is: SELECT refFlat.chrom AS `chromosome`, refFlat.txStart AS `start`, refFlat.txEnd AS `end`, refFlat.name AS `transcript_id`, refFlat.strand AS `strand`, refFlat.geneName AS `gene_name`, ensemblSource.source AS `biotype` FROM `refFlat` INNER JOIN `knownToRefSeq` ON refFlat.name=knownToRefSeq.value INNER JOIN `knownCanonical` ON knownToRefSeq.name=knownCanonical.transcript  INNER JOIN `knownToEnsembl` ON knownCanonical.transcript=knownToEnsembl.name INNER JOIN `ensemblSource` ON knownToEnsembl.value=ensemblSource.name ORDER BY `chromosome`,`start`
Test 1 successful!
Fetched 31914 features
Sample data:
  chromosome start   end transcript_id strand gene_name
1       chr1 11873 14409     NR_046018      +   DDX11L1
2       chr1 14361 29370     NR_024540      -    WASH7P
3       chr1 14361 29370     NR_024540      -    WASH7P
4       chr1 34610 36081     NR_026820      -   FAM138F
5       chr1 34610 36081     NR_026820      -   FAM138F
6       chr1 34610 36081     NR_026822      -   FAM138C
                             biotype
1 transcribed_unprocessed_pseudogene
2             unprocessed_pseudogene
3             unprocessed_pseudogene
4                            lincRNA
5                            lincRNA
6                            lincRNA
---------------------------------------------------

Disconnecting from UCSC database...
Testing finished!

Summary
==================================================

Succesful tests: 1 out of 1
Failed tests   : 0 out of 1
 
Scheduling 1 tests
==================================================

==========&gt; Now testing mm10
Connecting to UCSC database mm10...

---------------------------------------------------
Running test 1 of 1 scheduled
Source: refseq
Type  : gene
Query is: SELECT  refFlat.chrom AS `chromosome`, refFlat.txStart AS `start`, refFlat.txEnd AS `end`, refFlat.name AS `gene_id`, 0 AS `gc_content`, refFlat.strand AS `strand`, refFlat.geneName AS `gene_name`, &#x27;NA&#x27; AS `biotype` FROM `refFlat` INNER JOIN `knownToRefSeq` ON refFlat.name=knownToRefSeq.value INNER JOIN `knownCanonical` ON knownToRefSeq.name=knownCanonical.transcript  GROUP BY refFlat.name ORDER BY `chromosome`,`start`
Test 1 successful!
Fetched 23491 features
Sample data:
  chromosome   start     end      gene_id gc_content strand gene_name biotype
1       chr1 3214481 3671498 NM_001011874          0      -      Xkr4      NA
2       chr1 4119865 4360303 NM_001370921          0      -       Rp1      NA
3       chr1 4490927 4497354    NM_011441          0      -     Sox17      NA
4       chr1 4773199 4785726 NM_001177658          0      -    Mrpl15      NA
5       chr1 4807822 4846735    NM_008866          0      +    Lypla1      NA
6       chr1 4857693 4897909    NM_011541          0      +     Tcea1      NA
---------------------------------------------------

Disconnecting from UCSC database...
Testing finished!

Summary
==================================================

Succesful tests: 1 out of 1
Failed tests   : 0 out of 1
 
Scheduling 1 tests
==================================================

==========&gt; Now testing hg19
Connecting to UCSC database hg19...

---------------------------------------------------
Running test 1 of 1 scheduled
Source: ucsc
Type  : transcript
Query is: SELECT knownGene.chrom AS `chromosome`, knownGene.txStart AS `start`, knownGene.txEnd AS `end`, knownGene.name AS `transcript_id`, knownGene.strand AS `strand`, `geneName` AS `gene_name`, `source` AS `biotype` FROM `knownGene` INNER JOIN `knownToRefSeq` ON knownGene.name=knownToRefSeq.name INNER JOIN `knownToEnsembl` ON knownGene.name=knownToEnsembl.name INNER JOIN `ensemblSource` ON knownToEnsembl.value=ensemblSource.name INNER JOIN `refFlat` ON knownToRefSeq.value=refFlat.name GROUP BY `transcript_id` ORDER BY `chromosome`,`start`
Test 1 successful!
Fetched 71001 features
Sample data:
  chromosome start   end transcript_id strand gene_name
1       chr1 11873 14409    uc001aaa.3      +   DDX11L1
2       chr1 11873 14409    uc010nxr.1      +   DDX11L1
3       chr1 11873 14409    uc010nxq.1      +   DDX11L1
4       chr1 14361 16765    uc009vis.3      -    WASH7P
5       chr1 14361 19759    uc009vit.3      -    WASH7P
6       chr1 14361 19759    uc009viu.3      -    WASH7P
                             biotype
1               processed_transcript
2               processed_transcript
3 transcribed_unprocessed_pseudogene
4             unprocessed_pseudogene
5             unprocessed_pseudogene
6             unprocessed_pseudogene
---------------------------------------------------

Disconnecting from UCSC database...
Testing finished!

Summary
==================================================

Succesful tests: 1 out of 1
Failed tests   : 0 out of 1
 
Scheduling 1 tests
==================================================

==========&gt; Now testing dm3
Connecting to UCSC database dm3...

---------------------------------------------------
Running test 1 of 1 scheduled
Source: ucsc
Type  : gene
Query is: SELECT flyBaseCanonical.chrom AS `chromosome`, `chromStart` AS `start`, `chromEnd` AS `end`, `transcript` AS `gene_id`, 0 AS `gc_content`, flyBaseGene.strand AS `strand`, `geneName` AS `gene_name`, `source` AS `biotype` FROM `flyBaseCanonical` INNER JOIN `flyBaseGene` ON flyBaseCanonical.transcript=flyBaseGene.name INNER JOIN `flyBaseToRefSeq` ON flyBaseCanonical.transcript=flyBaseToRefSeq.name INNER JOIN `refFlat` ON flyBaseToRefSeq.value=refFlat.name INNER JOIN `ensemblToGeneName` ON ensemblToGeneName.value=refFlat.geneName INNER JOIN `ensemblSource` ON ensemblToGeneName.name=ensemblSource.name GROUP BY `gene_id` ORDER BY `chromosome`,`start`
Test 1 successful!
Fetched 10792 features
Sample data:
  chromosome start   end    gene_id gc_content strand gene_name        biotype
1      chr2L  9835 18583  CG2671-RC          0      -    l(2)gl protein_coding
2      chr2L 21918 25151  CG2657-RB          0      -     Ir21a protein_coding
3      chr2L 25401 59242 CG31973-RB          0      -      Cda5 protein_coding
4      chr2L 67043 71390 CG11371-RB          0      +       dbr protein_coding
5      chr2L 72387 76203 CG11372-RA          0      +  galectin protein_coding
6      chr2L 76445 77639 CG11374-RB          0      +   CG11374 protein_coding
---------------------------------------------------

Disconnecting from UCSC database...
Testing finished!

Summary
==================================================

Succesful tests: 1 out of 1
Failed tests   : 0 out of 1
 
Scheduling 1 tests
==================================================

==========&gt; Now testing equCab3
Connecting to UCSC database equCab3...

---------------------------------------------------
Running test 1 of 1 scheduled
Source: refseq
Type  : gene
Query is: SELECT `chromosome`,`start`,`end`,`gene_id`, `gc_content`,`strand`,`gene_name`,`biotype` FROM (SELECT MAX(`txEnd` - `txStart`) AS `width`, refFlat.chrom AS `chromosome`, `txStart` AS `start`, `txEnd` AS `end`, refFlat.name AS `gene_id`, 0 AS `gc_content`, refFlat.strand AS `strand`, refFlat.geneName AS `gene_name`, ensemblSource.source AS `biotype` FROM `refFlat` INNER JOIN `ensemblToGeneName` ON refFlat.geneName=ensemblToGeneName.value INNER JOIN `ensemblSource` ON ensemblToGeneName.name=ensemblSource.name  GROUP BY `gene_name` ORDER BY `chromosome`,`start`) AS tmp
Test 1 successful!
Fetched 1007 features
Sample data:
  chromosome    start      end      gene_id gc_content strand gene_name
1       chr1   131450   143029 NM_001111303          0      -    CYP2E1
2       chr1 11323857 11426445 NM_001163863          0      +     FGFR2
3       chr1 15653377 15670230 NM_001163949          0      -     PNLIP
4       chr1 20910562 20943976 NM_001308957          0      -      SMC3
5       chr1 27056246 27100149 NM_001309513          0      -    PDCD11
6       chr1 27571464 27577548 NM_001082523          0      +   CYP17A1
         biotype
1 protein_coding
2 protein_coding
3 protein_coding
4 protein_coding
5 protein_coding
6 protein_coding
---------------------------------------------------

Disconnecting from UCSC database...
Testing finished!

Summary
==================================================

Succesful tests: 1 out of 1
Failed tests   : 0 out of 1
 
Scheduling 1 tests
==================================================

==========&gt; Now testing danRer11
Connecting to UCSC database danRer11...

---------------------------------------------------
Running test 1 of 1 scheduled
Source: refseq
Type  : transcript
Query is: SELECT refFlat.chrom AS `chromosome`, refFlat.txStart AS `start`, refFlat.txEnd AS `end`, refFlat.name AS `transcript_id`, refFlat.strand AS `strand`, refFlat.geneName AS `gene_name`, ensemblSource.source AS `biotype` FROM `refFlat` INNER JOIN `ensemblToGeneName` ON refFlat.geneName=ensemblToGeneName.value INNER JOIN `ensemblSource` ON ensemblToGeneName.name=ensemblSource.name  GROUP BY `transcript_id` ORDER BY `chromosome`,`start`
Test 1 successful!
Fetched 15068 features
Sample data:
  chromosome start   end transcript_id strand gene_name        biotype
1       chr1  6641 11878     NM_173235      -     rpl24 protein_coding
2       chr1 11966 16373     NM_198371      +     cep97 protein_coding
3       chr1 27687 34330  NM_001017766      +       eed protein_coding
4       chr1 36733 39191  NM_001017577      +   hikeshi protein_coding
5       chr1 39324 44525     NM_199632      -   tmem39a protein_coding
6       chr1 44838 49831     NM_200393      -    ildr1a protein_coding
---------------------------------------------------

Disconnecting from UCSC database...
Testing finished!

Summary
==================================================

Succesful tests: 1 out of 1
Failed tests   : 0 out of 1
 
Scheduling 1 tests
==================================================

==========&gt; Now testing hg19
Connecting to UCSC database hg19...

---------------------------------------------------
Running test 1 of 1 scheduled
Source: refseq
Type  : gene
Query is: SELECT refFlat.chrom AS `chromosome`, refFlat.txStart AS `start`, refFlat.txEnd AS `end`, CONCAT(refFlat.name,&#x27;.&#x27;,hgFixed.gbCdnaInfo.version) AS `gene_id`, 0 AS `gc_content`, refFlat.strand AS `strand`, refFlat.geneName AS `gene_name`, ensemblSource.source AS `biotype` FROM `refFlat` INNER JOIN `knownToRefSeq` ON refFlat.name=knownToRefSeq.value INNER JOIN `knownCanonical` ON knownToRefSeq.name=knownCanonical.transcript INNER JOIN hgFixed.gbCdnaInfo ON refFlat.name=hgFixed.gbCdnaInfo.acc INNER JOIN `knownToEnsembl` ON knownCanonical.transcript=knownToEnsembl.name INNER JOIN `ensemblSource` ON knownToEnsembl.value=ensemblSource.name GROUP BY refFlat.name ORDER BY `chromosome`,`start`
Test 1 successful!
Fetched 22116 features
Sample data:
  chromosome  start    end        gene_id gc_content strand gene_name
1       chr1  11873  14409    NR_046018.2          0      +   DDX11L1
2       chr1  14361  29370    NR_024540.1          0      -    WASH7P
3       chr1  34610  36081    NR_026820.1          0      -   FAM138F
4       chr1  34610  36081    NR_026822.1          0      -   FAM138C
5       chr1  69090  70008 NM_001005484.1          0      +     OR4F5
6       chr1 134772 140566    NR_039983.2          0      - LOC729737
                             biotype
1 transcribed_unprocessed_pseudogene
2             unprocessed_pseudogene
3                            lincRNA
4                            lincRNA
5                     protein_coding
6                          antisense
---------------------------------------------------

Disconnecting from UCSC database...
Testing finished!

Summary
==================================================

Succesful tests: 1 out of 1
Failed tests   : 0 out of 1
 
Scheduling 1 tests
==================================================

==========&gt; Now testing mm10
Connecting to UCSC database mm10...

---------------------------------------------------
Running test 1 of 1 scheduled
Source: refseq
Type  : gene
Query is: SELECT  refFlat.chrom AS `chromosome`, refFlat.txStart AS `start`, refFlat.txEnd AS `end`, CONCAT(refFlat.name,&#x27;.&#x27;,hgFixed.gbCdnaInfo.version) AS `gene_id`, 0 AS `gc_content`, refFlat.strand AS `strand`, refFlat.geneName AS `gene_name`, &#x27;NA&#x27; AS `biotype` FROM `refFlat` INNER JOIN `knownToRefSeq` ON refFlat.name=knownToRefSeq.value INNER JOIN `knownCanonical` ON knownToRefSeq.name=knownCanonical.transcript INNER JOIN hgFixed.gbCdnaInfo ON refFlat.name=hgFixed.gbCdnaInfo.acc GROUP BY refFlat.name ORDER BY `chromosome`,`start`
Test 1 successful!
Fetched 23491 features
Sample data:
  chromosome   start     end        gene_id gc_content strand gene_name biotype
1       chr1 3214481 3671498 NM_001011874.1          0      -      Xkr4      NA
2       chr1 4119865 4360303 NM_001370921.1          0      -       Rp1      NA
3       chr1 4490927 4497354    NM_011441.5          0      -     Sox17      NA
4       chr1 4773199 4785726 NM_001177658.1          0      -    Mrpl15      NA
5       chr1 4807822 4846735    NM_008866.3          0      +    Lypla1      NA
6       chr1 4857693 4897909    NM_011541.4          0      +     Tcea1      NA
---------------------------------------------------

Disconnecting from UCSC database...
Testing finished!

Summary
==================================================

Succesful tests: 1 out of 1
Failed tests   : 0 out of 1
 
Scheduling 1 tests
==================================================

==========&gt; Now testing danRer11
Connecting to UCSC database danRer11...

---------------------------------------------------
Running test 1 of 1 scheduled
Source: refseq
Type  : transcript
Query is: SELECT refFlat.chrom AS `chromosome`, refFlat.txStart AS `start`, refFlat.txEnd AS `end`, CONCAT(refFlat.name,&#x27;.&#x27;,hgFixed.gbCdnaInfo.version) AS `transcript_id`, refFlat.strand AS `strand`, refFlat.geneName AS `gene_name`, ensemblSource.source AS `biotype` FROM `refFlat` INNER JOIN `ensemblToGeneName` ON refFlat.geneName=ensemblToGeneName.value INNER JOIN `ensemblSource` ON ensemblToGeneName.name=ensemblSource.name INNER JOIN hgFixed.gbCdnaInfo ON refFlat.name=hgFixed.gbCdnaInfo.acc GROUP BY `transcript_id` ORDER BY `chromosome`,`start`
Test 1 successful!
Fetched 15068 features
Sample data:
  chromosome start   end  transcript_id strand gene_name        biotype
1       chr1  6641 11878    NM_173235.3      -     rpl24 protein_coding
2       chr1 11966 16373    NM_198371.2      +     cep97 protein_coding
3       chr1 27687 34330 NM_001017766.2      +       eed protein_coding
4       chr1 36733 39191 NM_001017577.1      +   hikeshi protein_coding
5       chr1 39324 44525    NM_199632.1      -   tmem39a protein_coding
6       chr1 44838 49831    NM_200393.1      -    ildr1a protein_coding
---------------------------------------------------

Disconnecting from UCSC database...
Testing finished!

Summary
==================================================

Succesful tests: 1 out of 1
Failed tests   : 0 out of 1
 


RUNIT TEST PROTOCOL -- Fri Feb 24 18:50:49 2023 
*********************************************** 
Number of test functions: 8 
Number of errors: 0 
Number of failures: 1 

 
1 Test Suite : 
sitadela RUnit Tests - 8 test functions, 0 errors, 1 failure
FAILURE in test_sitadela_ensembl_often: Error in checkTrue(is.null(f1)) : Test not TRUE


Test files with failing tests

   test_sitadela.R 
     test_sitadela_ensembl_often 


Error in BiocGenerics:::testPackage(&quot;sitadela&quot;) : 
  unit tests failed for package sitadela
In addition: There were 17 warnings (use warnings() to see them)
Execution halted
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">sitadela.Rcheck/sitadela-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>addAnnotation</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addCustomAnnotation</TD><TD style="text-align: right;">5.956</TD><TD style="text-align: right;">0.228</TD><TD style="text-align: right;">6.186</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getAnnotation</TD><TD style="text-align: right;">1.316</TD><TD style="text-align: right;">0.128</TD><TD style="text-align: right;">8.731</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getInstalledAnnotations</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getSeqInfo</TD><TD style="text-align: right;">0.228</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">0.678</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getUserAnnotations</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getsetDbPath</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>importCustomAnnotation</TD><TD style="text-align: right;">1.126</TD><TD style="text-align: right;">0.132</TD><TD style="text-align: right;">1.258</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loadAnnotation</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>removeAnnotation</TD><TD style="text-align: right;">4.173</TD><TD style="text-align: right;">0.200</TD><TD style="text-align: right;">4.372</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>testFuns</TD><TD style="text-align: right;">4.393</TD><TD style="text-align: right;">0.176</TD><TD style="text-align: right;">9.634</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
