<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for sangeranalyseR on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><B>[<A href="../index.html#S">S</A>]</B></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-09 02:50:48 -0000 (Thu, 09 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4165</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for sangeranalyseR on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the sangeranalyseR package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/sangeranalyseR.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1751</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">sangeranalyseR</A>&nbsp;1.9.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/sangeranalyseR">landing page</A>)</SPAN><BR>Kuan-Hao Chao<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-08&nbsp;03:21:00&nbsp;-0000&nbsp;(Wed,&nbsp;08&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/sangeranalyseR</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">3895514</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:22:09&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: sangeranalyseR</TD></TR>
<TR><TD><B>Version</B>: 1.9.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:sangeranalyseR.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings sangeranalyseR_1.9.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-08 23:57:29 -0000 (Wed, 08 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-09 00:05:12 -0000 (Thu, 09 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 462.8 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: sangeranalyseR.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 7</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:sangeranalyseR.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings sangeranalyseR_1.9.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/sangeranalyseR.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘sangeranalyseR/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘sangeranalyseR’ version ‘1.9.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Depends: includes the non-default packages:
  &#x27;stringr&#x27;, &#x27;ape&#x27;, &#x27;Biostrings&#x27;, &#x27;DECIPHER&#x27;, &#x27;parallel&#x27;, &#x27;reshape2&#x27;,
  &#x27;phangorn&#x27;, &#x27;sangerseqR&#x27;, &#x27;gridExtra&#x27;, &#x27;shiny&#x27;, &#x27;shinydashboard&#x27;,
  &#x27;shinyjs&#x27;, &#x27;data.table&#x27;, &#x27;plotly&#x27;, &#x27;DT&#x27;, &#x27;zeallot&#x27;, &#x27;excelR&#x27;,
  &#x27;shinycssloaders&#x27;, &#x27;ggdendro&#x27;, &#x27;shinyWidgets&#x27;, &#x27;openxlsx&#x27;, &#x27;tools&#x27;,
  &#x27;rmarkdown&#x27;, &#x27;knitr&#x27;, &#x27;seqinr&#x27;, &#x27;BiocStyle&#x27;, &#x27;logger&#x27;
Adding so many packages to the search path is excessive and importing
selectively is preferable.
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .travis.yml
  docs/build/html/.buildinfo
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
* checking for portable file names ... WARNING
Found the following file with a non-portable file name:
  docs/wallpaperflare.com_wallpaper (1).jpg
These are not fully portable file names.
See section ‘Package structure’ in the ‘Writing R Extensions’ manual.
Found the following non-portable file paths:
  sangeranalyseR/docs/build/html/_downloads/081987116457968e1b81ae1616525c7d/Sanger_contigs_unalignment.fa
  sangeranalyseR/docs/build/html/_downloads/13b875bc704f73b8bd67dc07d86fc474/Sanger_contigs_alignment.fa
  sangeranalyseR/docs/build/html/_downloads/144d09562ae5b2c7dc079e4bc2b33fb7/Sanger_contigs_alignment.fa
  sangeranalyseR/docs/build/html/_downloads/165af37e8893cfdfa4dba91c44cd791b/Sanger_contigs_unalignment.fa
  sangeranalyseR/docs/build/html/_downloads/345ac400befc4f176c75732770ec3fcb/Achl_RBNII384-13_contig.fa
  sangeranalyseR/docs/build/html/_downloads/438c8bd47d93d92c65b35b613ef14739/Sanger_all_trimmed_reads.fa
  sangeranalyseR/docs/build/html/_downloads/6211ea1e7e3ccc1fcc9b74b78224eab6/Achl_ACHLO006-09_reads_unalignment.fa
  sangeranalyseR/docs/build/html/_downloads/67bcad64cca804ec2e14cc42d016c08f/Achl_ACHLO006-09_reads_alignment.fa
  sangeranalyseR/docs/build/html/_downloads/81a7338101f110748cb72056d9975225/Sanger_all_trimmed_reads.fa
  sangeranalyseR/docs/build/html/_downloads/95ef9378c1407481d69c3bf882a580bb/Achl_RBNII384-13_reads_unalignment.fa
  sangeranalyseR/docs/build/html/_downloads/a664bed9cc5d32924dc2ce60f8c60554/Sanger_all_trimmed_reads.fa
  sangeranalyseR/docs/build/html/_downloads/b608796fffaaa7335a7341cffc9a9db7/Achl_ACHLO006-09_contig.fa
  sangeranalyseR/docs/build/html/_downloads/b6b18aa3e34c6ae0be86c3940972dd38/Sanger_contigs_alignment.fa
  sangeranalyseR/docs/build/html/_downloads/eeb554b575ee5b29ef733dafbf876c36/Achl_RBNII384-13_reads_alignment.fa
  sangeranalyseR/docs/build/html/_downloads/fef71e7b2e1e49c01df301435e972047/Sanger_contigs_unalignment.fa

Tarballs are only required to store paths of up to 100 bytes and cannot
store those of more than 256 bytes, with restrictions including to 100
bytes for the final component.
See section ‘Package structure’ in the ‘Writing R Extensions’ manual.
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘sangeranalyseR’ can be installed ... OK
* checking installed package size ... NOTE
  installed size is  8.5Mb
  sub-directories of 1Mb or more:
    data      2.2Mb
    extdata   5.5Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... NOTE
File
  LICENSE
is not mentioned in the DESCRIPTION file.
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... WARNING
Found the following files with non-ASCII characters:
  ClassSangerAlignment.R
  ClassSangerContig.R
  ClassSangerRead.R
Portable packages must use only ASCII characters in their R code,
except perhaps in comments.
Use \uxxxx escapes for other characters.
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
Packages in Depends field not imported from:
  ‘BiocStyle’ ‘knitr’ ‘phangorn’ ‘reshape2’ ‘zeallot’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.
package &#x27;methods&#x27; is used but not declared
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
QualityBasePlotly: no visible global function definition for ‘%&gt;%’
SangerAlignment: no visible global function definition for ‘new’
SangerAlignmentServer: no visible global function definition for ‘%&gt;%’
SangerAlignmentServer: no visible binding for &#x27;&lt;&lt;-&#x27; assignment to
  ‘NEW_SANGER_ALIGNED_CONSENSUS_READ_SET’
SangerAlignmentServer: no visible global function definition for
  ‘colorRamp’
SangerContig: no visible global function definition for ‘new’
SangerContigServer: no visible global function definition for ‘%&gt;%’
SangerContigServer: no visible binding for &#x27;&lt;&lt;-&#x27; assignment to
  ‘NEW_SANGER_CONTIG’
SangerContigServer: no visible global function definition for
  ‘colorRamp’
SangerRead: no visible global function definition for ‘new’
SetCharStyleList: no visible global function definition for ‘%&gt;%’
alignContigs: no visible global function definition for ‘as.phylo’
alignContigs: no visible global function definition for ‘rtree’
checkAb1FastaCsv: no visible global function definition for ‘read.csv’
checkTargetFastaName: no visible global function definition for
  ‘isEmpty’
chromatogram_overwrite: no visible global function definition for ‘rgb’
chromatogram_overwrite: no visible global function definition for ‘par’
chromatogram_overwrite: no visible global function definition for
  ‘quantile’
chromatogram_overwrite: no visible global function definition for ‘IQR’
chromatogram_overwrite: no visible global function definition for ‘pdf’
chromatogram_overwrite: no visible global function definition for
  ‘rect’
chromatogram_overwrite: no visible global function definition for
  ‘lines’
chromatogram_overwrite: no visible global function definition for
  ‘mtext’
chromatogram_overwrite: no visible global function definition for
  ‘axis’
chromatogram_overwrite: no visible global function definition for
  ‘dev.off’
primarySeqDisplay: no visible global function definition for ‘rgb’
primarySeqTrimmedDisplay: no visible global function definition for
  ‘rgb’
secondSeqTrimmedDisplay: no visible global function definition for
  ‘rgb’
secondarySeqDisplay: no visible global function definition for ‘rgb’
initialize,ChromatogramParam: no visible global function definition for
  ‘callNextMethod’
initialize,ObjectResults: no visible global function definition for
  ‘callNextMethod’
initialize,QualityReport: no visible global function definition for
  ‘callNextMethod’
initialize,SangerAlignment: no visible global function definition for
  ‘str_split’
initialize,SangerAlignment : &lt;anonymous&gt;: no visible global function
  definition for ‘new’
initialize,SangerAlignment: no visible global function definition for
  ‘read.csv’
initialize,SangerAlignment: no visible global function definition for
  ‘new’
initialize,SangerAlignment: no visible global function definition for
  ‘callNextMethod’
initialize,SangerContig : &lt;anonymous&gt;: no visible global function
  definition for ‘new’
initialize,SangerContig: no visible global function definition for
  ‘read.csv’
initialize,SangerContig: no visible global function definition for
  ‘new’
initialize,SangerContig: no visible global function definition for
  ‘callNextMethod’
initialize,SangerRead: no visible global function definition for ‘new’
initialize,SangerRead: no visible global function definition for
  ‘AAString’
initialize,SangerRead: no visible global function definition for
  ‘callNextMethod’
launchAppSA,SangerAlignment: no visible global function definition for
  ‘shinyOptions’
launchAppSA,SangerAlignment: no visible global function definition for
  ‘shinyApp’
launchAppSC,SangerContig: no visible global function definition for
  ‘shinyOptions’
launchAppSC,SangerContig: no visible global function definition for
  ‘shinyApp’
Undefined global functions or variables:
  %&gt;% AAString IQR as.phylo axis callNextMethod colorRamp dev.off
  isEmpty lines mtext new par pdf quantile read.csv rect rgb rtree
  shinyApp shinyOptions str_split
Consider adding
  importFrom(&quot;grDevices&quot;, &quot;colorRamp&quot;, &quot;dev.off&quot;, &quot;pdf&quot;, &quot;rgb&quot;)
  importFrom(&quot;graphics&quot;, &quot;axis&quot;, &quot;lines&quot;, &quot;mtext&quot;, &quot;par&quot;, &quot;rect&quot;)
  importFrom(&quot;methods&quot;, &quot;callNextMethod&quot;, &quot;new&quot;)
  importFrom(&quot;stats&quot;, &quot;IQR&quot;, &quot;quantile&quot;)
  importFrom(&quot;utils&quot;, &quot;read.csv&quot;)
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains &#x27;methods&#x27;).
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... WARNING
Undocumented code objects:
  ‘chromatogram_overwrite’
All user-level objects in a package should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
* checking for code/documentation mismatches ... WARNING
S4 class codoc mismatches from documentation object &#x27;ObjectResults-class&#x27;:
Slots for class &#x27;ObjectResults&#x27;
  Code: creationResult errorMessages errorTypes printLevel
        readResultTable warningMessages warningTypes
  Docs: printLevel

* checking Rd \usage sections ... WARNING
Undocumented arguments in documentation object &#x27;SangerAlignment&#x27;
  ‘printLevel’ ‘processMethod’
Documented arguments not in \usage in documentation object &#x27;SangerAlignment&#x27;:
  ‘minFractionCallSA’ ‘maxFractionLostSA’

Undocumented arguments in documentation object &#x27;SangerContig&#x27;
  ‘printLevel’ ‘processMethod’

Undocumented arguments in documentation object &#x27;SangerRead&#x27;
  ‘printLevel’

Functions with \usage entries need to have the appropriate \alias
entries, and all their arguments documented.
The \usage entries must correspond to syntactically valid R code.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... WARNING
  Failed with error:  &#x27;package &#x27;phangorn&#x27; could not be loaded&#x27;
  Error loading dataset &#x27;qualityReportData&#x27;:
   Error in .requirePackage(package) : 
    unable to find required package &#x27;sangeranalyseR&#x27;
  
  Failed with error:  &#x27;package &#x27;phangorn&#x27; could not be loaded&#x27;
  Error loading dataset &#x27;sangerAlignmentData&#x27;:
   Error in .requirePackage(package) : 
    unable to find required package &#x27;sangeranalyseR&#x27;
  
  Failed with error:  &#x27;package &#x27;phangorn&#x27; could not be loaded&#x27;
  Error loading dataset &#x27;sangerContigData&#x27;:
   Error in .requirePackage(package) : 
    unable to find required package &#x27;sangeranalyseR&#x27;
  
  Failed with error:  &#x27;package &#x27;phangorn&#x27; could not be loaded&#x27;
  Error loading dataset &#x27;sangerReadFData&#x27;:
   Error in .requirePackage(package) : 
    unable to find required package &#x27;sangeranalyseR&#x27;
  
  The dataset(s) may use package(s) not declared in the DESCRIPTION file.
* checking data for ASCII and uncompressed saves ... WARNING
  
  Note: significantly better compression could be obtained
        by using R CMD build --resave-data
                            old_size new_size compress
  sangerAlignmentData.RData    1.5Mb    697Kb       xz
  sangerContigData.RData       451Kb    203Kb       xz
  sangerReadFData.RData        269Kb    118Kb       xz
* checking line endings in Makefiles ... OK
* checking for GNU extensions in Makefiles ... OK
* checking include directives in Makefiles ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                        user system elapsed
SangerAlignment-class 69.398 14.078  79.180
SangerAlignment       14.332  3.283  20.822
SangerContig-class    13.952  3.027  16.485
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘sangeranalyseR.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 7 WARNINGs, 6 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/sangeranalyseR.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">sangeranalyseR.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL sangeranalyseR
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘sangeranalyseR’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Note: wrong number of arguments to &#x27;==&#x27; 
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (sangeranalyseR)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">sangeranalyseR.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(sangeranalyseR)
Loading required package: stringr
Loading required package: ape
Loading required package: Biostrings
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: XVector
Loading required package: GenomeInfoDb

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:ape&#x27;:

    complement

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: DECIPHER
Loading required package: RSQLite
Loading required package: parallel
Loading required package: reshape2
Loading required package: phangorn
Loading required package: sangerseqR
Loading required package: gridExtra

Attaching package: &#x27;gridExtra&#x27;

The following object is masked from &#x27;package:BiocGenerics&#x27;:

    combine

Loading required package: shiny
Loading required package: shinydashboard

Attaching package: &#x27;shinydashboard&#x27;

The following object is masked from &#x27;package:graphics&#x27;:

    box

Loading required package: shinyjs

Attaching package: &#x27;shinyjs&#x27;

The following object is masked from &#x27;package:shiny&#x27;:

    runExample

The following object is masked from &#x27;package:RSQLite&#x27;:

    show

The following object is masked from &#x27;package:Biostrings&#x27;:

    show

The following object is masked from &#x27;package:GenomeInfoDb&#x27;:

    show

The following object is masked from &#x27;package:XVector&#x27;:

    show

The following object is masked from &#x27;package:IRanges&#x27;:

    show

The following object is masked from &#x27;package:S4Vectors&#x27;:

    show

The following object is masked from &#x27;package:stats4&#x27;:

    show

The following objects are masked from &#x27;package:methods&#x27;:

    removeClass, show

Loading required package: data.table

Attaching package: &#x27;data.table&#x27;

The following objects are masked from &#x27;package:reshape2&#x27;:

    dcast, melt

The following object is masked from &#x27;package:IRanges&#x27;:

    shift

The following objects are masked from &#x27;package:S4Vectors&#x27;:

    first, second

Loading required package: plotly
Loading required package: ggplot2

Attaching package: &#x27;plotly&#x27;

The following object is masked from &#x27;package:ggplot2&#x27;:

    last_plot

The following object is masked from &#x27;package:phangorn&#x27;:

    add_boxplot

The following object is masked from &#x27;package:XVector&#x27;:

    slice

The following object is masked from &#x27;package:IRanges&#x27;:

    slice

The following object is masked from &#x27;package:S4Vectors&#x27;:

    rename

The following object is masked from &#x27;package:stats&#x27;:

    filter

The following object is masked from &#x27;package:graphics&#x27;:

    layout

Loading required package: DT

Attaching package: &#x27;DT&#x27;

The following objects are masked from &#x27;package:shiny&#x27;:

    dataTableOutput, renderDataTable

Loading required package: zeallot
Loading required package: excelR
Loading required package: shinycssloaders
Loading required package: ggdendro
Loading required package: shinyWidgets

Attaching package: &#x27;shinyWidgets&#x27;

The following object is masked from &#x27;package:shinyjs&#x27;:

    alert

Loading required package: openxlsx
Loading required package: tools
Loading required package: rmarkdown
Loading required package: knitr
Loading required package: seqinr

Attaching package: &#x27;seqinr&#x27;

The following object is masked from &#x27;package:shiny&#x27;:

    a

The following object is masked from &#x27;package:sangerseqR&#x27;:

    read.abif

The following object is masked from &#x27;package:Biostrings&#x27;:

    translate

The following objects are masked from &#x27;package:ape&#x27;:

    as.alignment, consensus

Loading required package: BiocStyle

Attaching package: &#x27;BiocStyle&#x27;

The following objects are masked from &#x27;package:rmarkdown&#x27;:

    html_document, md_document, pdf_document

The following object is masked from &#x27;package:shiny&#x27;:

    markdown

Loading required package: logger
Welcome to sangeranalyseR
&gt; 
&gt; 
&gt; test_check(&quot;sangeranalyseR&quot;)
INFO [2023-02-09 00:03:20] #################################################
INFO [2023-02-09 00:03:20] #### Start creating SangerAlignment instance ####
INFO [2023-02-09 00:03:20] #################################################
INFO [2023-02-09 00:03:20]   &gt;&gt; You are using Regular Expression Method to group AB1 files!
INFO [2023-02-09 00:03:20] ========================================================
INFO [2023-02-09 00:03:20] ================ Creating &#x27;SangerContig&#x27; ===============
INFO [2023-02-09 00:03:20] ========================================================
INFO [2023-02-09 00:03:20]   &gt;&gt; Contig Name: &#x27;Achl_ACHLO006-09&#x27;
SUCCESS [2023-02-09 00:03:22] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:22] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:22] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:22]    &gt;&gt; &#x27;Achl_ACHLO006-09_1_F.ab1&#x27; is created (Forward Read; ABIF).
SUCCESS [2023-02-09 00:03:23] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:23] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:23] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:23]    &gt;&gt; &#x27;Achl_ACHLO006-09_2_R.ab1&#x27; is created (Reverse Read; ABIF).
INFO [2023-02-09 00:03:23]    &gt;&gt; The number of reads detected: 2
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.08 secs
SUCCESS [2023-02-09 00:03:25] ==========================================================
SUCCESS [2023-02-09 00:03:25] ======== &#x27;SangerContig&#x27; S4 instance is created !! ========
SUCCESS [2023-02-09 00:03:25] ==========================================================
INFO [2023-02-09 00:03:25]    &gt;&gt; 2 read(s) created from ABIF file.
INFO [2023-02-09 00:03:25]      &gt;&gt; 1 reads assigned to &#x27;forward reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:25]      &gt;&gt; 1 reads assigned to &#x27;reverse reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:26] ========================================================
INFO [2023-02-09 00:03:26] ================ Creating &#x27;SangerContig&#x27; ===============
INFO [2023-02-09 00:03:26] ========================================================
INFO [2023-02-09 00:03:26]   &gt;&gt; Contig Name: &#x27;Achl_ACHLO007-09&#x27;
SUCCESS [2023-02-09 00:03:26] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:26] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:26] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:26]    &gt;&gt; &#x27;Achl_ACHLO007-09_1_F.ab1&#x27; is created (Forward Read; ABIF).
SUCCESS [2023-02-09 00:03:27] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:27] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:27] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:27]    &gt;&gt; &#x27;Achl_ACHLO007-09_2_R.ab1&#x27; is created (Reverse Read; ABIF).
INFO [2023-02-09 00:03:27]    &gt;&gt; The number of reads detected: 2
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.04 secs
SUCCESS [2023-02-09 00:03:29] ==========================================================
SUCCESS [2023-02-09 00:03:29] ======== &#x27;SangerContig&#x27; S4 instance is created !! ========
SUCCESS [2023-02-09 00:03:29] ==========================================================
INFO [2023-02-09 00:03:29]    &gt;&gt; 2 read(s) created from ABIF file.
INFO [2023-02-09 00:03:29]      &gt;&gt; 1 reads assigned to &#x27;forward reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:29]      &gt;&gt; 1 reads assigned to &#x27;reverse reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:29] ========================================================
INFO [2023-02-09 00:03:29] ================ Creating &#x27;SangerContig&#x27; ===============
INFO [2023-02-09 00:03:29] ========================================================
INFO [2023-02-09 00:03:29]   &gt;&gt; Contig Name: &#x27;Achl_ACHLO040-09&#x27;
SUCCESS [2023-02-09 00:03:30] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:30] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:30] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:30]    &gt;&gt; &#x27;Achl_ACHLO040-09_1_F.ab1&#x27; is created (Forward Read; ABIF).
SUCCESS [2023-02-09 00:03:31] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:31] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:31] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:31]    &gt;&gt; &#x27;Achl_ACHLO040-09_2_R.ab1&#x27; is created (Reverse Read; ABIF).
INFO [2023-02-09 00:03:31]    &gt;&gt; The number of reads detected: 2
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.05 secs
SUCCESS [2023-02-09 00:03:33] ==========================================================
SUCCESS [2023-02-09 00:03:33] ======== &#x27;SangerContig&#x27; S4 instance is created !! ========
SUCCESS [2023-02-09 00:03:33] ==========================================================
INFO [2023-02-09 00:03:33]    &gt;&gt; 2 read(s) created from ABIF file.
INFO [2023-02-09 00:03:33]      &gt;&gt; 1 reads assigned to &#x27;forward reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:33]      &gt;&gt; 1 reads assigned to &#x27;reverse reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:33] ========================================================
INFO [2023-02-09 00:03:33] ================ Creating &#x27;SangerContig&#x27; ===============
INFO [2023-02-09 00:03:33] ========================================================
INFO [2023-02-09 00:03:33]   &gt;&gt; Contig Name: &#x27;Achl_ACHLO041-09&#x27;
SUCCESS [2023-02-09 00:03:34] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:34] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:34] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:34]    &gt;&gt; &#x27;Achl_ACHLO041-09_1_F.ab1&#x27; is created (Forward Read; ABIF).
SUCCESS [2023-02-09 00:03:35] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:35] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:35] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:35]    &gt;&gt; &#x27;Achl_ACHLO041-09_2_R.ab1&#x27; is created (Reverse Read; ABIF).
INFO [2023-02-09 00:03:35]    &gt;&gt; The number of reads detected: 2
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.04 secs
SUCCESS [2023-02-09 00:03:37] ==========================================================
SUCCESS [2023-02-09 00:03:37] ======== &#x27;SangerContig&#x27; S4 instance is created !! ========
SUCCESS [2023-02-09 00:03:37] ==========================================================
INFO [2023-02-09 00:03:37]    &gt;&gt; 2 read(s) created from ABIF file.
INFO [2023-02-09 00:03:37]      &gt;&gt; 1 reads assigned to &#x27;forward reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:37]      &gt;&gt; 1 reads assigned to &#x27;reverse reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:37] Aligning consensus reads ... 
SUCCESS [2023-02-09 00:03:38] #############################################################
SUCCESS [2023-02-09 00:03:38] ######## &#x27;SangerAlignment&#x27; S4 instance is created !! ########
SUCCESS [2023-02-09 00:03:38] #############################################################
INFO [2023-02-09 00:03:38]   &gt;&gt; 4 contigs detected from &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:38]       &gt;&gt; Contig &#x27;Achl_ACHLO006-09&#x27;:
INFO [2023-02-09 00:03:38]           &gt;&gt; 1 forward reads.
INFO [2023-02-09 00:03:38]           &gt;&gt; 1 reverse reads.
INFO [2023-02-09 00:03:38]       &gt;&gt; Contig &#x27;Achl_ACHLO007-09&#x27;:
INFO [2023-02-09 00:03:38]           &gt;&gt; 1 forward reads.
INFO [2023-02-09 00:03:38]           &gt;&gt; 1 reverse reads.
INFO [2023-02-09 00:03:38]       &gt;&gt; Contig &#x27;Achl_ACHLO040-09&#x27;:
INFO [2023-02-09 00:03:38]           &gt;&gt; 1 forward reads.
INFO [2023-02-09 00:03:38]           &gt;&gt; 1 reverse reads.
INFO [2023-02-09 00:03:38]       &gt;&gt; Contig &#x27;Achl_ACHLO041-09&#x27;:
INFO [2023-02-09 00:03:38]           &gt;&gt; 1 forward reads.
INFO [2023-02-09 00:03:38]           &gt;&gt; 1 reverse reads.
INFO [2023-02-09 00:03:38]   &gt;&gt; 8 reads created from ABIF file.
INFO [2023-02-09 00:03:38]   &gt;&gt; Reads are trimmed by &#x27;M1 - Mott’s trimming algorithm&#x27;.
DEBUG [2023-02-09 00:03:38]    &gt;&gt; For more information, please run &#x27;object&#x27;.
DEBUG [2023-02-09 00:03:38]    &gt;&gt; Run &#x27;object@objectResults@readResultTable&#x27; to check the results of each Sanger reads
INFO [2023-02-09 00:03:38] #################################################
INFO [2023-02-09 00:03:38] #### Start creating SangerAlignment instance ####
INFO [2023-02-09 00:03:38] #################################################
INFO [2023-02-09 00:03:38] **** You are using Regular Expression Method to group reads in FASTA file (No CSV file)!
INFO [2023-02-09 00:03:38] ========================================================
INFO [2023-02-09 00:03:38] ================ Creating &#x27;SangerContig&#x27; ===============
INFO [2023-02-09 00:03:38] ========================================================
INFO [2023-02-09 00:03:38]   &gt;&gt; Contig Name: &#x27;Achl_ACHLO006-09&#x27;
SUCCESS [2023-02-09 00:03:39] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:39] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:39] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:39]    &gt;&gt; &#x27;Achl_ACHLO006-09_1_F&#x27; is created (Forward Read; FASTA).
SUCCESS [2023-02-09 00:03:39] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:39] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:39] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:39]    &gt;&gt; &#x27;Achl_ACHLO006-09_2_R&#x27; is created (Reverse Read; FASTA).
INFO [2023-02-09 00:03:39]    &gt;&gt; The number of reads detected: 2
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.04 secs
SUCCESS [2023-02-09 00:03:41] ==========================================================
SUCCESS [2023-02-09 00:03:41] ======== &#x27;SangerContig&#x27; S4 instance is created !! ========
SUCCESS [2023-02-09 00:03:41] ==========================================================
INFO [2023-02-09 00:03:41]    &gt;&gt; 2 read(s) created from FASTA file.
INFO [2023-02-09 00:03:41]      &gt;&gt; 1 reads assigned to &#x27;forward reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:41]      &gt;&gt; 1 reads assigned to &#x27;reverse reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:41] ========================================================
INFO [2023-02-09 00:03:41] ================ Creating &#x27;SangerContig&#x27; ===============
INFO [2023-02-09 00:03:41] ========================================================
INFO [2023-02-09 00:03:41]   &gt;&gt; Contig Name: &#x27;Achl_ACHLO007-09&#x27;
SUCCESS [2023-02-09 00:03:42] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:42] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:42] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:42]    &gt;&gt; &#x27;Achl_ACHLO007-09_1_F&#x27; is created (Forward Read; FASTA).
SUCCESS [2023-02-09 00:03:43] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:43] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:43] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:43]    &gt;&gt; &#x27;Achl_ACHLO007-09_2_R&#x27; is created (Reverse Read; FASTA).
INFO [2023-02-09 00:03:43]    &gt;&gt; The number of reads detected: 2
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.04 secs
SUCCESS [2023-02-09 00:03:45] ==========================================================
SUCCESS [2023-02-09 00:03:45] ======== &#x27;SangerContig&#x27; S4 instance is created !! ========
SUCCESS [2023-02-09 00:03:45] ==========================================================
INFO [2023-02-09 00:03:45]    &gt;&gt; 2 read(s) created from FASTA file.
INFO [2023-02-09 00:03:45]      &gt;&gt; 1 reads assigned to &#x27;forward reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:45]      &gt;&gt; 1 reads assigned to &#x27;reverse reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:45] ========================================================
INFO [2023-02-09 00:03:45] ================ Creating &#x27;SangerContig&#x27; ===============
INFO [2023-02-09 00:03:45] ========================================================
INFO [2023-02-09 00:03:45]   &gt;&gt; Contig Name: &#x27;Achl_ACHLO040-09&#x27;
SUCCESS [2023-02-09 00:03:45] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:45] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:45] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:45]    &gt;&gt; &#x27;Achl_ACHLO040-09_1_F&#x27; is created (Forward Read; FASTA).
SUCCESS [2023-02-09 00:03:46] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:46] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:46] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:46]    &gt;&gt; &#x27;Achl_ACHLO040-09_2_R&#x27; is created (Reverse Read; FASTA).
INFO [2023-02-09 00:03:46]    &gt;&gt; The number of reads detected: 2
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.05 secs
SUCCESS [2023-02-09 00:03:48] ==========================================================
SUCCESS [2023-02-09 00:03:48] ======== &#x27;SangerContig&#x27; S4 instance is created !! ========
SUCCESS [2023-02-09 00:03:48] ==========================================================
INFO [2023-02-09 00:03:48]    &gt;&gt; 2 read(s) created from FASTA file.
INFO [2023-02-09 00:03:48]      &gt;&gt; 1 reads assigned to &#x27;forward reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:48]      &gt;&gt; 1 reads assigned to &#x27;reverse reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:48] ========================================================
INFO [2023-02-09 00:03:48] ================ Creating &#x27;SangerContig&#x27; ===============
INFO [2023-02-09 00:03:48] ========================================================
INFO [2023-02-09 00:03:48]   &gt;&gt; Contig Name: &#x27;Achl_ACHLO041-09&#x27;
SUCCESS [2023-02-09 00:03:48] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:48] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:48] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:48]    &gt;&gt; &#x27;Achl_ACHLO041-09_1_F&#x27; is created (Forward Read; FASTA).
SUCCESS [2023-02-09 00:03:49] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:49] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:49] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:49]    &gt;&gt; &#x27;Achl_ACHLO041-09_2_R&#x27; is created (Reverse Read; FASTA).
INFO [2023-02-09 00:03:49]    &gt;&gt; The number of reads detected: 2
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.04 secs
SUCCESS [2023-02-09 00:03:51] ==========================================================
SUCCESS [2023-02-09 00:03:51] ======== &#x27;SangerContig&#x27; S4 instance is created !! ========
SUCCESS [2023-02-09 00:03:51] ==========================================================
INFO [2023-02-09 00:03:51]    &gt;&gt; 2 read(s) created from FASTA file.
INFO [2023-02-09 00:03:51]      &gt;&gt; 1 reads assigned to &#x27;forward reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:51]      &gt;&gt; 1 reads assigned to &#x27;reverse reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:51] Aligning consensus reads ... 
SUCCESS [2023-02-09 00:03:52] #############################################################
SUCCESS [2023-02-09 00:03:52] ######## &#x27;SangerAlignment&#x27; S4 instance is created !! ########
SUCCESS [2023-02-09 00:03:52] #############################################################
INFO [2023-02-09 00:03:52]   &gt;&gt; 4 contigs detected from &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:52]       &gt;&gt; Contig &#x27;Achl_ACHLO006-09&#x27;:
INFO [2023-02-09 00:03:52]           &gt;&gt; 1 forward reads.
INFO [2023-02-09 00:03:52]           &gt;&gt; 1 reverse reads.
INFO [2023-02-09 00:03:52]       &gt;&gt; Contig &#x27;Achl_ACHLO007-09&#x27;:
INFO [2023-02-09 00:03:52]           &gt;&gt; 1 forward reads.
INFO [2023-02-09 00:03:52]           &gt;&gt; 1 reverse reads.
INFO [2023-02-09 00:03:52]       &gt;&gt; Contig &#x27;Achl_ACHLO040-09&#x27;:
INFO [2023-02-09 00:03:52]           &gt;&gt; 1 forward reads.
INFO [2023-02-09 00:03:52]           &gt;&gt; 1 reverse reads.
INFO [2023-02-09 00:03:52]       &gt;&gt; Contig &#x27;Achl_ACHLO041-09&#x27;:
INFO [2023-02-09 00:03:52]           &gt;&gt; 1 forward reads.
INFO [2023-02-09 00:03:52]           &gt;&gt; 1 reverse reads.
INFO [2023-02-09 00:03:52]   &gt;&gt; 8 reads created from FASTA file.
DEBUG [2023-02-09 00:03:52]    &gt;&gt; For more information, please run &#x27;object&#x27;.
DEBUG [2023-02-09 00:03:52]    &gt;&gt; Run &#x27;object@objectResults@readResultTable&#x27; to check the results of each Sanger reads
INFO [2023-02-09 00:03:52] ========================================================
INFO [2023-02-09 00:03:52] ================ Creating &#x27;SangerContig&#x27; ===============
INFO [2023-02-09 00:03:52] ========================================================
INFO [2023-02-09 00:03:52]   &gt;&gt; Contig Name: &#x27;Achl_ACHLO006-09&#x27;
INFO [2023-02-09 00:03:52]   &gt;&gt; You are using Regular Expression Method to group AB1 files!
INFO [2023-02-09 00:03:52] &gt;&gt; Your contig name is Achl_ACHLO006-09
SUCCESS [2023-02-09 00:03:53] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:53] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:53] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:53]    &gt;&gt; &#x27;Achl_ACHLO006-09_1_F.ab1&#x27; is created (Forward Read; ABIF).
SUCCESS [2023-02-09 00:03:54] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:54] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:54] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:54]    &gt;&gt; &#x27;Achl_ACHLO006-09_2_R.ab1&#x27; is created (Reverse Read; ABIF).
INFO [2023-02-09 00:03:54]    &gt;&gt; The number of reads detected: 2
INFO [2023-02-09 00:03:54] Correcting frameshifts in reads using amino acidreference sequence
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.04 secs
SUCCESS [2023-02-09 00:03:56] ==========================================================
SUCCESS [2023-02-09 00:03:56] ======== &#x27;SangerContig&#x27; S4 instance is created !! ========
SUCCESS [2023-02-09 00:03:56] ==========================================================
INFO [2023-02-09 00:03:56]    &gt;&gt; 2 read(s) created from ABIF file.
INFO [2023-02-09 00:03:56]      &gt;&gt; 1 reads assigned to &#x27;forward reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:56]      &gt;&gt; 1 reads assigned to &#x27;reverse reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:56]    &gt;&gt; Trimmed by &#x27;M1 - Mott’s trimming algorithm&#x27;.
DEBUG [2023-02-09 00:03:56]    &gt;&gt; For more information, please run &#x27;object&#x27;
DEBUG [2023-02-09 00:03:56]    &gt;&gt; Run &#x27;object@objectResults@readResultTable&#x27; to check the results of each Sanger reads
INFO [2023-02-09 00:03:56] ========================================================
INFO [2023-02-09 00:03:56] ================ Creating &#x27;SangerContig&#x27; ===============
INFO [2023-02-09 00:03:56] ========================================================
INFO [2023-02-09 00:03:56]   &gt;&gt; Contig Name: &#x27;Achl_RBNII384-13&#x27;
INFO [2023-02-09 00:03:56]   &gt;&gt; You are using Regular Expression Method to group AB1 files!
INFO [2023-02-09 00:03:56] &gt;&gt; Your contig name is Achl_RBNII384-13
SUCCESS [2023-02-09 00:03:56] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:56] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:56] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:56]    &gt;&gt; &#x27;Achl_RBNII384-13_1_F.ab1&#x27; is created (Forward Read; ABIF).
SUCCESS [2023-02-09 00:03:57] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:57] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:03:57] --------------------------------------------------------
SUCCESS [2023-02-09 00:03:57]    &gt;&gt; &#x27;Achl_RBNII384-13_2_R.ab1&#x27; is created (Reverse Read; ABIF).
INFO [2023-02-09 00:03:57]    &gt;&gt; The number of reads detected: 2
INFO [2023-02-09 00:03:57] Correcting frameshifts in reads using amino acidreference sequence
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.04 secs
SUCCESS [2023-02-09 00:03:59] ==========================================================
SUCCESS [2023-02-09 00:03:59] ======== &#x27;SangerContig&#x27; S4 instance is created !! ========
SUCCESS [2023-02-09 00:03:59] ==========================================================
INFO [2023-02-09 00:03:59]    &gt;&gt; 2 read(s) created from ABIF file.
INFO [2023-02-09 00:03:59]      &gt;&gt; 1 reads assigned to &#x27;forward reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:59]      &gt;&gt; 1 reads assigned to &#x27;reverse reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:03:59]    &gt;&gt; Trimmed by &#x27;M1 - Mott’s trimming algorithm&#x27;.
DEBUG [2023-02-09 00:03:59]    &gt;&gt; For more information, please run &#x27;object&#x27;
DEBUG [2023-02-09 00:03:59]    &gt;&gt; Run &#x27;object@objectResults@readResultTable&#x27; to check the results of each Sanger reads
INFO [2023-02-09 00:03:59] ========================================================
INFO [2023-02-09 00:03:59] ================ Creating &#x27;SangerContig&#x27; ===============
INFO [2023-02-09 00:03:59] ========================================================
INFO [2023-02-09 00:03:59]   &gt;&gt; Contig Name: &#x27;Achl_ACHLO006-09&#x27;
INFO [2023-02-09 00:03:59]   &gt;&gt; You are using Regular Expression Method to group reads in FASTA file (No CSV file)!
INFO [2023-02-09 00:03:59] &gt;&gt; Your contig name is Achl_ACHLO006-09
SUCCESS [2023-02-09 00:04:00] --------------------------------------------------------
SUCCESS [2023-02-09 00:04:00] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:04:00] --------------------------------------------------------
SUCCESS [2023-02-09 00:04:00]    &gt;&gt; &#x27;Achl_ACHLO006-09_1_F&#x27; is created (Forward Read; FASTA).
SUCCESS [2023-02-09 00:04:00] --------------------------------------------------------
SUCCESS [2023-02-09 00:04:00] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:04:00] --------------------------------------------------------
SUCCESS [2023-02-09 00:04:00]    &gt;&gt; &#x27;Achl_ACHLO006-09_2_R&#x27; is created (Reverse Read; FASTA).
INFO [2023-02-09 00:04:00]    &gt;&gt; The number of reads detected: 2
INFO [2023-02-09 00:04:00] Correcting frameshifts in reads using amino acidreference sequence
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.05 secs
SUCCESS [2023-02-09 00:04:02] ==========================================================
SUCCESS [2023-02-09 00:04:02] ======== &#x27;SangerContig&#x27; S4 instance is created !! ========
SUCCESS [2023-02-09 00:04:02] ==========================================================
INFO [2023-02-09 00:04:02]    &gt;&gt; 2 read(s) created from FASTA file.
INFO [2023-02-09 00:04:02]      &gt;&gt; 1 reads assigned to &#x27;forward reads&#x27; according to &#x27;regular expression&#x27;.
INFO [2023-02-09 00:04:02]      &gt;&gt; 1 reads assigned to &#x27;reverse reads&#x27; according to &#x27;regular expression&#x27;.
DEBUG [2023-02-09 00:04:02]    &gt;&gt; For more information, please run &#x27;object&#x27;
DEBUG [2023-02-09 00:04:02]    &gt;&gt; Run &#x27;object@objectResults@readResultTable&#x27; to check the results of each Sanger reads
INFO [2023-02-09 00:04:02] ========================================================
INFO [2023-02-09 00:04:02] ================ Creating &#x27;SangerContig&#x27; ===============
INFO [2023-02-09 00:04:02] ========================================================
INFO [2023-02-09 00:04:02]   &gt;&gt; Contig Name: &#x27;Achl_RBNII384-13&#x27;
INFO [2023-02-09 00:04:02]   &gt;&gt; You are using CSV Name Conversion Method to group reads in FASTA file (with Csv file)!
INFO [2023-02-09 00:04:02] &gt;&gt; Your contig name is Achl_RBNII384-13
SUCCESS [2023-02-09 00:04:03] --------------------------------------------------------
SUCCESS [2023-02-09 00:04:03] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:04:03] --------------------------------------------------------
SUCCESS [2023-02-09 00:04:03]    &gt;&gt; &#x27;Achl_RBNII384-13_1_F&#x27; is created (Forward Read; FASTA).
SUCCESS [2023-02-09 00:04:03] --------------------------------------------------------
SUCCESS [2023-02-09 00:04:03] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:04:03] --------------------------------------------------------
SUCCESS [2023-02-09 00:04:03]    &gt;&gt; &#x27;Achl_RBNII384-13_2_R&#x27; is created (Reverse Read; FASTA).
INFO [2023-02-09 00:04:03]    &gt;&gt; The number of reads detected: 2
INFO [2023-02-09 00:04:03] Correcting frameshifts in reads using amino acidreference sequence
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.04 secs
SUCCESS [2023-02-09 00:04:05] ==========================================================
SUCCESS [2023-02-09 00:04:05] ======== &#x27;SangerContig&#x27; S4 instance is created !! ========
SUCCESS [2023-02-09 00:04:05] ==========================================================
INFO [2023-02-09 00:04:05]    &gt;&gt; 2 read(s) created from FASTA file.
INFO [2023-02-09 00:04:05]      &gt;&gt; 1 reads assigned to &#x27;forward reads&#x27; according to &#x27;csv file&#x27;.
INFO [2023-02-09 00:04:05]      &gt;&gt; 1 reads assigned to &#x27;reverse reads&#x27; according to &#x27;csv file&#x27;.
DEBUG [2023-02-09 00:04:05]    &gt;&gt; For more information, please run &#x27;object&#x27;
DEBUG [2023-02-09 00:04:05]    &gt;&gt; Run &#x27;object@objectResults@readResultTable&#x27; to check the results of each Sanger reads
INFO [2023-02-09 00:04:05] ------------------------------------------------
INFO [2023-02-09 00:04:05] -------- Creating &#x27;SangerRead&#x27; instance --------
INFO [2023-02-09 00:04:05] ------------------------------------------------
INFO [2023-02-09 00:04:05] &gt;&gt; Forward Read: Creating abif &amp; sangerseq ...
INFO [2023-02-09 00:04:05]     &gt;&gt; Creating Forward Read raw abif ...
INFO [2023-02-09 00:04:06]     &gt;&gt; Creating Forward Read raw sangerseq ...
INFO [2023-02-09 00:04:06]           * Making basecall !!
INFO [2023-02-09 00:04:06]           * Updating slots in &#x27;SangerRead&#x27; instance !!
SUCCESS [2023-02-09 00:04:06] --------------------------------------------------------
SUCCESS [2023-02-09 00:04:06] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:04:06] --------------------------------------------------------
SUCCESS [2023-02-09 00:04:06]    &gt;&gt; &#x27;Achl_ACHLO006-09_1_F.ab1&#x27; is created (Forward Read; ABIF).
INFO [2023-02-09 00:04:06]    &gt;&gt; Read is trimmed by &#x27;M1 - Mott’s trimming algorithm&#x27;.
DEBUG [2023-02-09 00:04:06]    &gt;&gt; For more information, please run &#x27;object&#x27;.
DEBUG [2023-02-09 00:04:06]    &gt;&gt; Run &#x27;object@objectResults@readResultTable&#x27; to check the result of the Sanger read
INFO [2023-02-09 00:04:06] ------------------------------------------------
INFO [2023-02-09 00:04:06] -------- Creating &#x27;SangerRead&#x27; instance --------
INFO [2023-02-09 00:04:06] ------------------------------------------------
INFO [2023-02-09 00:04:06] &gt;&gt; Reverse Read: Creating abif &amp; sangerseq ...
INFO [2023-02-09 00:04:06]     &gt;&gt; Creating Reverse Read raw abif ...
INFO [2023-02-09 00:04:06]     &gt;&gt; Creating Reverse Read raw sangerseq ...
INFO [2023-02-09 00:04:06]           * Making basecall !!
INFO [2023-02-09 00:04:07]           * Updating slots in &#x27;SangerRead&#x27; instance !!
SUCCESS [2023-02-09 00:04:07] --------------------------------------------------------
SUCCESS [2023-02-09 00:04:07] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:04:07] --------------------------------------------------------
SUCCESS [2023-02-09 00:04:07]    &gt;&gt; &#x27;Achl_ACHLO006-09_2_R.ab1&#x27; is created (Reverse Read; ABIF).
INFO [2023-02-09 00:04:07]    &gt;&gt; Read is trimmed by &#x27;M1 - Mott’s trimming algorithm&#x27;.
DEBUG [2023-02-09 00:04:07]    &gt;&gt; For more information, please run &#x27;object&#x27;.
DEBUG [2023-02-09 00:04:07]    &gt;&gt; Run &#x27;object@objectResults@readResultTable&#x27; to check the result of the Sanger read
INFO [2023-02-09 00:04:07] ------------------------------------------------
INFO [2023-02-09 00:04:07] -------- Creating &#x27;SangerRead&#x27; instance --------
INFO [2023-02-09 00:04:07] ------------------------------------------------
INFO [2023-02-09 00:04:07] Forward Read: Creating SangerRead from FASTA ...
SUCCESS [2023-02-09 00:04:07] --------------------------------------------------------
SUCCESS [2023-02-09 00:04:07] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:04:07] --------------------------------------------------------
SUCCESS [2023-02-09 00:04:07]    &gt;&gt; &#x27;Achl_ACHLO006-09_1_F&#x27; is created (Forward Read; FASTA).
INFO [2023-02-09 00:04:07]    &gt;&gt; Read is trimmed by &#x27;M1 - Mott’s trimming algorithm&#x27;.
DEBUG [2023-02-09 00:04:07]    &gt;&gt; For more information, please run &#x27;object&#x27;.
DEBUG [2023-02-09 00:04:07]    &gt;&gt; Run &#x27;object@objectResults@readResultTable&#x27; to check the result of the Sanger read
INFO [2023-02-09 00:04:07] ------------------------------------------------
INFO [2023-02-09 00:04:07] -------- Creating &#x27;SangerRead&#x27; instance --------
INFO [2023-02-09 00:04:07] ------------------------------------------------
INFO [2023-02-09 00:04:07] Reverse Read: Creating SangerRead from FASTA ...
SUCCESS [2023-02-09 00:04:08] --------------------------------------------------------
SUCCESS [2023-02-09 00:04:08] -------- &#x27;SangerRead&#x27; S4 instance is created !! --------
SUCCESS [2023-02-09 00:04:08] --------------------------------------------------------
SUCCESS [2023-02-09 00:04:08]    &gt;&gt; &#x27;Achl_ACHLO006-09_2_R&#x27; is created (Reverse Read; FASTA).
INFO [2023-02-09 00:04:08]    &gt;&gt; Read is trimmed by &#x27;M1 - Mott’s trimming algorithm&#x27;.
DEBUG [2023-02-09 00:04:08]    &gt;&gt; For more information, please run &#x27;object&#x27;.
DEBUG [2023-02-09 00:04:08]    &gt;&gt; Run &#x27;object@objectResults@readResultTable&#x27; to check the result of the Sanger read
ERROR [2023-02-09 00:04:08] FILE_NOT_EXIST_ERROR
&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/sangeranalyseR/extdata//Allolobophora_chlorotica/ACHLO/TEST_Achl_ACHLO006-09_2_R.ab1&#x27; file does not exist.
DEBUG [2023-02-09 00:04:08]    &gt;&gt; For more information, please run &#x27;object&#x27;.
DEBUG [2023-02-09 00:04:08]    &gt;&gt; Run &#x27;object@objectResults@readResultTable&#x27; to check the result of the Sanger read
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.05 secs
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.04 secs
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.04 secs
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.04 secs
INFO [2023-02-09 00:04:18] Aligning consensus reads ... 
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.04 secs
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.04 secs
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.04 secs
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.04 secs
INFO [2023-02-09 00:04:30] Aligning consensus reads ... 
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.04 secs
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.04 secs
INFO [2023-02-09 00:04:36]           * Making basecall !!
INFO [2023-02-09 00:04:36]           * Updating slots in &#x27;SangerRead&#x27; instance !!
INFO [2023-02-09 00:04:37]           * Making basecall !!
INFO [2023-02-09 00:04:37]           * Updating slots in &#x27;SangerRead&#x27; instance !!
INFO [2023-02-09 00:04:37]           * Making basecall !!
INFO [2023-02-09 00:04:38]           * Updating slots in &#x27;SangerRead&#x27; instance !!
INFO [2023-02-09 00:04:38]           * Making basecall !!
INFO [2023-02-09 00:04:38]           * Updating slots in &#x27;SangerRead&#x27; instance !!
INFO [2023-02-09 00:04:39]           * Making basecall !!
INFO [2023-02-09 00:04:39]           * Updating slots in &#x27;SangerRead&#x27; instance !!
INFO [2023-02-09 00:04:44] &gt;&gt;&gt; outputDir : /tmp/RtmpCRZIDx
INFO [2023-02-09 00:04:44] Start writing &#x27;/home/biocbuild/bbs-3.17-bioc/R/library/sangeranalyseR/extdata//Allolobophora_chlorotica/ACHLO/Achl_ACHLO006-09_1_F.ab1&#x27; to FASTA format ...
INFO [2023-02-09 00:04:44] &gt;&gt; &#x27;/tmp/RtmpCRZIDx/Achl_ACHLO006-09_1_F.fa&#x27; is written
INFO [2023-02-09 00:04:44] &gt;&gt;&gt; outputDir : /tmp/RtmpCRZIDx
INFO [2023-02-09 00:04:44] Start writing &#x27;/home/biocbuild/bbs-3.17-bioc/R/library/sangeranalyseR/extdata//fasta/SangerRead/Achl_ACHLO006-09_1_F.fa&#x27; to FASTA format ...
INFO [2023-02-09 00:04:44] &gt;&gt; &#x27;/tmp/RtmpCRZIDx/Achl_ACHLO006-09_1_F.fa&#x27; is written
INFO [2023-02-09 00:04:44] &gt;&gt;&gt; outputDir : /tmp/RtmpCRZIDx
INFO [2023-02-09 00:04:44] Start writing &#x27;/home/biocbuild/bbs-3.17-bioc/R/library/sangeranalyseR/extdata//fasta/SangerRead/Achl_ACHLO006-09_2_R.fa&#x27; to FASTA format ...
INFO [2023-02-09 00:04:44] &gt;&gt; &#x27;/tmp/RtmpCRZIDx/Achl_ACHLO006-09_2_R.fa&#x27; is written
INFO [2023-02-09 00:04:44] Aligning consensus reads ... 
Assessing frameshifts in nucleotide sequences:
================================================================================

Time difference of 0.09 secs
INFO [2023-02-09 00:04:49]           * Making basecall !!
INFO [2023-02-09 00:04:49]           * Updating slots in &#x27;SangerRead&#x27; instance !!
INFO [2023-02-09 00:04:49]           * Making basecall !!
INFO [2023-02-09 00:04:50]           * Updating slots in &#x27;SangerRead&#x27; instance !!
[ FAIL 0 | WARN 0 | SKIP 0 | PASS 899 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
 90.002  10.898  99.246 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">sangeranalyseR.Rcheck/sangeranalyseR-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>ChromatogramParam-class</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>MakeBaseCalls-methods</TD><TD style="text-align: right;">1.377</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">1.384</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ObjectResults-class</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>QualityReport-class-qualityBasePlot</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>QualityReport-class-updateQualityParam</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>QualityReport-class</TD><TD style="text-align: right;">0.775</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.786</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerAlignment-class-generateReportSA</TD><TD style="text-align: right;">0.069</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.069</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerAlignment-class-launchAppSA</TD><TD style="text-align: right;">0.089</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.090</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerAlignment-class-updateQualityParam</TD><TD style="text-align: right;">0.066</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.070</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerAlignment-class-writeFastaSA</TD><TD style="text-align: right;">0.127</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.131</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerAlignment-class</TD><TD style="text-align: right;">69.398</TD><TD style="text-align: right;">14.078</TD><TD style="text-align: right;">79.180</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerAlignment</TD><TD style="text-align: right;">14.332</TD><TD style="text-align: right;"> 3.283</TD><TD style="text-align: right;">20.822</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerContig-class-generateReportSC</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerContig-class-launchAppSC</TD><TD style="text-align: right;">0.053</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.057</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerContig-class-readTable</TD><TD style="text-align: right;">0.139</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.139</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerContig-class-updateQualityParam</TD><TD style="text-align: right;">0.037</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.037</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerContig-class-writeFastaSC</TD><TD style="text-align: right;">0.098</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.098</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerContig-class</TD><TD style="text-align: right;">13.952</TD><TD style="text-align: right;"> 3.027</TD><TD style="text-align: right;">16.485</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerContig</TD><TD style="text-align: right;">0.067</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.085</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerRead-class-MakeBaseCalls</TD><TD style="text-align: right;">0.665</TD><TD style="text-align: right;">0.068</TD><TD style="text-align: right;">0.739</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerRead-class-generateReportSR</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerRead-class-qualityBasePlot</TD><TD style="text-align: right;">0.027</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerRead-class-readTable</TD><TD style="text-align: right;">0.128</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.141</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerRead-class-updateQualityParam</TD><TD style="text-align: right;">0.964</TD><TD style="text-align: right;">0.185</TD><TD style="text-align: right;">1.170</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerRead-class-writeFastaSR</TD><TD style="text-align: right;">0.05</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.05</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerRead-class</TD><TD style="text-align: right;">2.837</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">3.048</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SangerRead</TD><TD style="text-align: right;">0.710</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.715</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>generateReport</TD><TD style="text-align: right;">0.128</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.136</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>generateReportSA-methods</TD><TD style="text-align: right;">0.069</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.069</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>generateReportSC-methods</TD><TD style="text-align: right;">0.037</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>generateReportSR-methods</TD><TD style="text-align: right;">0.030</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.031</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>launchApp</TD><TD style="text-align: right;">0.105</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.105</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>launchAppSA-methods</TD><TD style="text-align: right;">0.066</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.070</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>launchAppSC-methods</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.037</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qualityBasePlot-methods</TD><TD style="text-align: right;">0.525</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.552</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readTable-methods</TD><TD style="text-align: right;">0.136</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.135</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>updateQualityParam-methods</TD><TD style="text-align: right;">0.133</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.138</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>writeFasta</TD><TD style="text-align: right;">0.132</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.135</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>writeFastaSA-methods</TD><TD style="text-align: right;">0.098</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.105</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>writeFastaSC-methods</TD><TD style="text-align: right;">0.081</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.089</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>writeFastaSR-methods</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.051</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
