<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for GWASTools on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><B>[<A href="../index.html#G">G</A>]</B></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-01-29 16:33:52 -0000 (Sun, 29 Jan 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4021</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for GWASTools on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the GWASTools package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/GWASTools.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>871</B>/2162</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">GWASTools</A>&nbsp;1.45.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/GWASTools">landing page</A>)</SPAN><BR>Stephanie M. Gogarten<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-01-27&nbsp;13:09:27&nbsp;-0000&nbsp;(Fri,&nbsp;27&nbsp;Jan&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/GWASTools</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">0127e8c</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:06:22&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: GWASTools</TD></TR>
<TR><TD><B>Version</B>: 1.45.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:GWASTools.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings GWASTools_1.45.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-01-29 00:38:55 -0000 (Sun, 29 Jan 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-01-29 00:44:53 -0000 (Sun, 29 Jan 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 357.9 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: GWASTools.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:GWASTools.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings GWASTools_1.45.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/GWASTools.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘GWASTools/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘GWASTools’ version ‘1.45.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘GWASTools’ can be installed ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... WARNING
Invalid citation information in ‘inst/CITATION’:
  Warning message:
  In utils::packageDescription(basename(dir), dirname(dir)) :
    no package &#x27;GWASTools&#x27; was found
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... WARNING
Error in library(GWASTools, lib.loc = &quot;/home/biocbuild/bbs-3.17-bioc/R/library&quot;) : 
  there is no package called ‘GWASTools’
Execution halted

It looks like this package has a loading problem when not on .libPaths:
see the messages for details.
* checking dependencies in R code ... NOTE
Unexported object imported by a &#x27;:::&#x27; call: ‘S4Vectors:::make_zero_col_DataFrame’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... WARNING
checkRd: (5) GdsGenotypeReader-class.Rd:36-67: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:76-78: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:79-81: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:82-86: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:87-95: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:96-100: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:101-105: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:106-110: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:111-115: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:116-129: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:130-141: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:142-150: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:151-154: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:155-158: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:159-162: \item in \describe must have non-empty label
checkRd: (5) GdsGenotypeReader-class.Rd:163-166: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:42-76: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:93-95: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:96-98: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:99-103: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:104-112: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:113-117: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:118-122: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:123-128: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:129-132: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:133-138: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:139-142: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:143-148: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:149-152: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:153-158: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:159-162: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:163-168: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:169-172: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:173-180: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:181-184: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:185-188: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:189-192: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:193-196: \item in \describe must have non-empty label
checkRd: (5) GdsIntensityReader-class.Rd:197-200: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:24-35: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:43-67: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:68-71: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:72-75: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:76-78: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:79-83: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:84-87: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:95-97: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:98-100: \item in \describe must have non-empty label
checkRd: (5) GdsReader-class.Rd:101-103: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:60-74: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:82-84: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:85-87: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:88-92: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:93-100: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:101-105: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:106-110: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:111-115: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:116-120: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:121-125: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:126-129: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:130-143: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:144-155: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:156-161: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:162-165: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:166-169: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:170-175: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:176-179: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:180-183: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:184-190: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:191-194: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:195-197: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:198-201: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:202-204: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:205-208: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:209-211: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:212-214: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:215-218: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:219-222: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:223-226: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:227-230: \item in \describe must have non-empty label
checkRd: (5) GenotypeData-class.Rd:231-234: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:42-50: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:56-57: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:58-59: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:60-61: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:62-63: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:64-65: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:66-69: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:70-73: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:74-77: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:78-80: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:81-83: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:84-87: \item in \describe must have non-empty label
checkRd: (5) GenotypeIterator-class.Rd:88-91: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:64-78: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:93-95: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:96-98: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:99-103: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:104-111: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:112-116: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:117-121: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:122-126: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:127-130: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:131-136: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:137-142: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:143-148: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:149-155: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:156-161: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:162-167: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:168-171: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:172-175: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:176-181: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:182-185: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:186-189: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:190-196: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:197-200: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:201-204: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:205-208: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:209-211: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:212-214: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:215-218: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:219-222: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:223-226: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:227-230: \item in \describe must have non-empty label
checkRd: (5) IntensityData-class.Rd:231-234: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:30-46: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:57-59: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:60-62: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:63-67: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:68-76: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:77-81: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:82-86: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:87-98: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:99-109: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:110-113: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:114-117: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:118-121: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:122-125: \item in \describe must have non-empty label
checkRd: (5) MatrixGenotypeReader-class.Rd:126-129: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:33-57: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:67-69: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:70-72: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:73-77: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:78-86: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:87-91: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:92-96: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:97-108: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:109-113: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:114-117: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:118-121: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:122-125: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:126-129: \item in \describe must have non-empty label
checkRd: (5) NcdfGenotypeReader-class.Rd:130-133: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:42-73: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:91-93: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:94-96: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:97-101: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:102-110: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:111-115: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:116-120: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:121-126: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:127-130: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:131-136: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:137-140: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:141-146: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:147-150: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:151-156: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:157-160: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:161-166: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:167-170: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:171-178: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:179-182: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:183-186: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:187-190: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:191-194: \item in \describe must have non-empty label
checkRd: (5) NcdfIntensityReader-class.Rd:195-198: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:28-35: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:43-66: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:67-70: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:71-74: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:75-79: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:80-85: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:86-90: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:91-95: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:103-105: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:106-108: \item in \describe must have non-empty label
checkRd: (5) NcdfReader-class.Rd:109-111: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:30-50: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:59-63: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:64-68: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:69-72: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:73-80: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:81-84: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:85-88: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:89-92: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:93-96: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:100-103: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:104-108: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:109-113: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationDataFrame-class.Rd:114-117: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:31-56: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:65-67: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:68-70: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:71-73: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:74-80: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:81-87: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:88-91: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:92-100: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:101-104: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:105-108: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:109-112: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:113-116: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:117-120: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:121-125: \item in \describe must have non-empty label
checkRd: (5) ScanAnnotationSQLite-class.Rd:126-131: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:36-58: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:67-71: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:72-80: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:81-85: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:86-90: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:91-95: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:96-103: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:104-107: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:108-111: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:112-115: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:116-119: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:123-126: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:127-131: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:132-136: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:137-140: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:141-144: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:145-148: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:149-152: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:153-156: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationDataFrame-class.Rd:157-160: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:38-66: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:75-77: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:78-80: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:81-83: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:84-90: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:91-101: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:102-108: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:109-114: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:115-120: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:121-129: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:130-133: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:134-137: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:138-141: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:142-145: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:146-149: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:150-155: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:156-161: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:162-165: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:166-169: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:170-173: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:174-177: \item in \describe must have non-empty label
checkRd: (5) SnpAnnotationSQLite-class.Rd:178-181: \item in \describe must have non-empty label
checkRd: (-1) chromIntensityPlot.Rd:38: Escaped LaTeX specials: \&amp;
checkRd: (-1) meanIntensityByScanChrom.Rd:3: Escaped LaTeX specials: \&amp;
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking LazyData ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... SKIPPED
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘test.R’
 ERROR
Running the tests in ‘tests/test.R’ failed.
Last 13 lines of output:
  Reading genotype file...
  Block 1 of 1
  Determining number of SNPs and samples...
  Reading SNP files...
  Reading genotype file...
  Block 1 of 1
  Determining number of SNPs and samples...
  Reading SNP files...
  Reading genotype file...
  Block 1 of 1
  Determining number of SNPs and samples...
  Reading SNP files...
  Reading genotype file...
  Block 1 of 1
  Killed
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘Affymetrix.Rnw’ using ‘UTF-8’... OK
  ‘DataCleaning.Rnw’ using ‘UTF-8’... OK
  ‘Formats.Rnw’ using ‘UTF-8’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 3 WARNINGs, 1 NOTE
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/GWASTools.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">GWASTools.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL GWASTools
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/site-library’
* installing *source* package ‘GWASTools’ ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (GWASTools)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">GWASTools.Rcheck/tests/test.Rout.fail</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(GWASdata)
Loading required package: GWASTools
Loading required package: Biobase
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

&gt; library(ncdf4)
&gt; library(gdsfmt)
&gt; BiocGenerics:::testPackage(&quot;GWASTools&quot;, pattern=&quot;.*_test\\.R$&quot;)
block 1 of 1
adding variables: quality, X, Y
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/RtmpPSFDx5/file2ede30785efff9&#x27; (67.1K)
    # of fragments: 35
    save to &#x27;/tmp/RtmpPSFDx5/file2ede30785efff9.tmp&#x27;
    rename &#x27;/tmp/RtmpPSFDx5/file2ede30785efff9.tmp&#x27; (66.9K, reduced: 216B)
    # of fragments: 17
adding variables: quality, X, Y
Clean up the fragments of GDS file:
    open the file &#x27;/tmp/RtmpPSFDx5/file2ede30697b0dea&#x27; (67.1K)
    # of fragments: 35
    save to &#x27;/tmp/RtmpPSFDx5/file2ede30697b0dea.tmp&#x27;
    rename &#x27;/tmp/RtmpPSFDx5/file2ede30697b0dea.tmp&#x27; (66.9K, reduced: 216B)
    # of fragments: 17
Analyzing: X280 
Analyzing: X281 
Analyzing: X280 
Analyzing: X281 
Analyzing: X280 
Analyzing: X281 
Sun Jan 29 00:42:15 2023 Chromosome 1
Sun Jan 29 00:42:15 2023 Chromosome 2
Sun Jan 29 00:42:15 2023 Chromosome 3
Sun Jan 29 00:42:15 2023 Chromosome 4
Sun Jan 29 00:42:15 2023 Chromosome 5
Sun Jan 29 00:42:15 2023 Chromosome 6
Sun Jan 29 00:42:15 2023 Chromosome 7
Sun Jan 29 00:42:15 2023 Chromosome 8
Sun Jan 29 00:42:15 2023 Chromosome 9
Sun Jan 29 00:42:15 2023 Chromosome 10
Sun Jan 29 00:42:15 2023 Chromosome 11
Sun Jan 29 00:42:15 2023 Chromosome 12
Sun Jan 29 00:42:15 2023 Chromosome 13
Sun Jan 29 00:42:15 2023 Chromosome 14
Sun Jan 29 00:42:15 2023 Chromosome 15
Sun Jan 29 00:42:15 2023 Chromosome 16
Sun Jan 29 00:42:15 2023 Chromosome 17
Sun Jan 29 00:42:15 2023 Chromosome 18
Sun Jan 29 00:42:15 2023 Chromosome 19
Sun Jan 29 00:42:15 2023 Chromosome 20
Sun Jan 29 00:42:15 2023 Chromosome 21
Sun Jan 29 00:42:15 2023 Chromosome 22
Sun Jan 29 00:42:15 2023 Chromosome 23
Sun Jan 29 00:42:15 2023 Chromosome 24
Sun Jan 29 00:42:15 2023 Chromosome 25
Sun Jan 29 00:42:15 2023 Chromosome 26
Sun Jan 29 00:42:15 2023 Total # of SNPs selected: 7
Sun Jan 29 00:42:15 2023 Chromosome 1
Sun Jan 29 00:42:15 2023 Chromosome 2
Sun Jan 29 00:42:15 2023 Chromosome 3
Sun Jan 29 00:42:15 2023 Chromosome 4
Sun Jan 29 00:42:15 2023 Chromosome 5
Sun Jan 29 00:42:15 2023 Chromosome 6
Sun Jan 29 00:42:15 2023 Chromosome 7
Sun Jan 29 00:42:15 2023 Chromosome 8
Sun Jan 29 00:42:15 2023 Chromosome 9
Sun Jan 29 00:42:15 2023 Chromosome 10
Sun Jan 29 00:42:15 2023 Chromosome 11
Sun Jan 29 00:42:15 2023 Chromosome 12
Sun Jan 29 00:42:15 2023 Chromosome 13
Sun Jan 29 00:42:15 2023 Chromosome 14
Sun Jan 29 00:42:15 2023 Chromosome 15
Sun Jan 29 00:42:15 2023 Chromosome 16
Sun Jan 29 00:42:15 2023 Chromosome 17
Sun Jan 29 00:42:15 2023 Chromosome 18
Sun Jan 29 00:42:15 2023 Chromosome 19
Sun Jan 29 00:42:15 2023 Chromosome 20
Sun Jan 29 00:42:15 2023 Chromosome 21
Sun Jan 29 00:42:15 2023 Chromosome 22
Sun Jan 29 00:42:15 2023 Chromosome 23
Sun Jan 29 00:42:15 2023 Chromosome 24
Sun Jan 29 00:42:15 2023 Chromosome 25
Sun Jan 29 00:42:15 2023 Chromosome 26
Sun Jan 29 00:42:15 2023 Total # of SNPs selected: 9
Loading required package: VariantAnnotation
Loading required package: MatrixGenerics
Loading required package: matrixStats

Attaching package: &#x27;matrixStats&#x27;

The following objects are masked from &#x27;package:Biobase&#x27;:

    anyMissing, rowMedians


Attaching package: &#x27;MatrixGenerics&#x27;

The following objects are masked from &#x27;package:matrixStats&#x27;:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars

The following object is masked from &#x27;package:Biobase&#x27;:

    rowMedians

Loading required package: GenomeInfoDb
Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomicRanges
Loading required package: SummarizedExperiment
Loading required package: Rsamtools
Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit


Attaching package: &#x27;VariantAnnotation&#x27;

The following object is masked from &#x27;package:base&#x27;:

    tabulate

Loading required package: SNPRelate
SNPRelate
Reading in Phenotype and Covariate Data...
Running analysis with 40 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.6905 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
[1] &quot;hi&quot;
   LR.Stat    LR.pval 
8.55400428 0.03584773 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
1.6733633 0.6428719 
[1] &quot;hi&quot;
    LR.Stat     LR.pval 
11.30391026  0.01019107 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
3.5479450 0.3145968 
[1] &quot;hi&quot;
   LR.Stat    LR.pval 
8.85703394 0.03125334 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
4.5360542 0.2090968 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
5.2442899 0.1547585 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
2.6576472 0.4474727 
[1] &quot;hi&quot;
    LR.Stat     LR.pval 
15.67110354  0.00132434 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
1.7737546 0.6206628 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
2.7135101 0.4379363 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
2.4907942 0.4769571 
[1] &quot;hi&quot;
    LR.Stat     LR.pval 
10.06391337  0.01803059 
[1] &quot;hi&quot;
 LR.Stat  LR.pval 
4.834600 0.184317 
[1] &quot;hi&quot;
   LR.Stat    LR.pval 
6.85119755 0.07679492 
[1] &quot;hi&quot;
     LR.Stat      LR.pval 
12.410321031  0.006101951 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
5.9806637 0.1125548 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
4.5674714 0.2063504 
[1] &quot;hi&quot;
    LR.Stat     LR.pval 
10.29919824  0.01618679 
[1] &quot;hi&quot;
     LR.Stat      LR.pval 
12.161441314  0.006850104 
[1] &quot;hi&quot;
   LR.Stat    LR.pval 
8.85117856 0.03133641 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
4.6736748 0.1973122 
Block 1 of 1 Completed - 2.049 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
[1] &quot;hi&quot;
     LR.Stat      LR.pval 
1.567110e+01 7.536688e-05 
[1] &quot;hi&quot;
  LR.Stat   LR.pval 
1.6624991 0.1972662 
[1] &quot;hi&quot;
   LR.Stat    LR.pval 
8.38393977 0.00378551 
Block 1 of 1 Completed - 1.359 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.583 secs
Reading in Phenotype and Covariate Data...
Running analysis with 500 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.778 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.8407 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.3305 secs
Reading in Phenotype and Covariate Data...
Running analysis with 23 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.1824 secs
Reading in Phenotype and Covariate Data...
Running analysis with 23 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.1928 secs
Reading in Phenotype and Covariate Data...
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.1945 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 4 Completed - 0.06144 secs
Block 2 of 4 Completed - 0.06473 secs
Block 3 of 4 Completed - 0.06605 secs
Block 4 of 4 Completed - 0.002516 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.2945 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.2718 secs
Reading in Phenotype and Covariate Data...
Running analysis with 40 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.5332 secs
Reading in Phenotype and Covariate Data...
Running analysis with 40 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.2946 secs
Reading in Phenotype and Covariate Data...
Running analysis with 40 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.359 secs
Loading required package: zoo

Attaching package: &#x27;zoo&#x27;

The following objects are masked from &#x27;package:Rsamtools&#x27;:

    index, index&lt;-

The following objects are masked from &#x27;package:base&#x27;:

    as.Date, as.Date.numeric

Reading in Phenotype and Covariate Data...
Running analysis with 40 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.4157 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.2163 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.09253 secs
Reading in Phenotype and Covariate Data...
Running analysis with 50 Samples
Beginning Calculations...
Block 1 of 1 Completed - 0.09464 secs
Sun Jan 29 00:42:45 2023	Load genotype dataset ...
Sun Jan 29 00:42:45 2023	1/2	batch1
Sun Jan 29 00:42:45 2023	Load genotype dataset ...
Sun Jan 29 00:42:45 2023	1/4	batch1
Sun Jan 29 00:42:45 2023	2/4	batch2
Sun Jan 29 00:42:45 2023	3/4	batch3
Sun Jan 29 00:42:45 2023	4/4	batch4
Sun Jan 29 00:42:45 2023	Load genotype dataset ...
Sun Jan 29 00:42:46 2023	1/4	batch1
Sun Jan 29 00:42:46 2023	2/4	batch2
Sun Jan 29 00:42:46 2023	3/4	batch3
Sun Jan 29 00:42:46 2023	4/4	batch4
Sun Jan 29 00:42:46 2023	Load genotype dataset ...
Sun Jan 29 00:42:46 2023	1/4	batch1
Sun Jan 29 00:42:46 2023	2/4	batch2
Sun Jan 29 00:42:46 2023	3/4	batch3
Sun Jan 29 00:42:46 2023	4/4	batch4
Sun Jan 29 00:42:46 2023	Load genotype dataset ...
Sun Jan 29 00:42:46 2023	1/4	batch1
Sun Jan 29 00:42:46 2023	2/4	batch2
Sun Jan 29 00:42:46 2023	3/4	batch3
Sun Jan 29 00:42:46 2023	4/4	batch4
Sun Jan 29 00:42:46 2023	Load genotype dataset ...
Sun Jan 29 00:42:46 2023	1/4	batch1
Sun Jan 29 00:42:46 2023	2/4	batch2
Sun Jan 29 00:42:46 2023	3/4	batch3
Sun Jan 29 00:42:46 2023	4/4	batch4
Sun Jan 29 00:42:46 2023	Load genotype dataset ...
Sun Jan 29 00:42:46 2023	1/4	batch1
Sun Jan 29 00:42:46 2023	2/4	batch2
Sun Jan 29 00:42:46 2023	3/4	batch3
Sun Jan 29 00:42:46 2023	4/4	batch4
Sun Jan 29 00:42:46 2023	Load genotype dataset ...
Sun Jan 29 00:42:46 2023	1/4	batch1
Sun Jan 29 00:42:46 2023	2/4	batch2
Sun Jan 29 00:42:46 2023	3/4	batch3
Sun Jan 29 00:42:46 2023	4/4	batch4
Sun Jan 29 00:42:46 2023	Load genotype dataset ...
Sun Jan 29 00:42:46 2023	1/4	batch1
Sun Jan 29 00:42:46 2023	2/4	batch2
Sun Jan 29 00:42:46 2023	3/4	batch3
Sun Jan 29 00:42:46 2023	4/4	batch4
Sun Jan 29 00:42:47 2023	Load genotype dataset ...
Sun Jan 29 00:42:47 2023	1/2	batch1
Sun Jan 29 00:42:47 2023	Load genotype dataset ...
Sun Jan 29 00:42:47 2023	1/4	batch1
Sun Jan 29 00:42:48 2023	2/4	batch2
Sun Jan 29 00:42:48 2023	3/4	batch3
Sun Jan 29 00:42:48 2023	4/4	batch4
Sun Jan 29 00:42:48 2023	Load genotype dataset ...
Sun Jan 29 00:42:48 2023	1/2	batch1
Sun Jan 29 00:42:48 2023	Load genotype dataset ...
Sun Jan 29 00:42:49 2023	1/2	batch1
Sun Jan 29 00:42:49 2023	Load genotype dataset ...
Sun Jan 29 00:42:49 2023	1/2	batch1
Sun Jan 29 00:42:49 2023	Load genotype dataset ...
Sun Jan 29 00:42:49 2023	1/2	batch1
Sun Jan 29 00:42:49 2023	Load genotype dataset ...
Sun Jan 29 00:42:49 2023	1/2	batch1
Sun Jan 29 00:42:49 2023	Load genotype dataset ...
Sun Jan 29 00:42:49 2023	1/2	batch1
Sun Jan 29 00:42:49 2023	Load genotype dataset ...
Sun Jan 29 00:42:49 2023	1/2	batch1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Block 3 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 3
Block 2 of 3
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 100
Block 2 of 100
Block 3 of 100
Block 4 of 100
Block 5 of 100
Block 6 of 100
Block 7 of 100
Block 8 of 100
Block 9 of 100
Block 10 of 100
Block 11 of 100
Block 12 of 100
Block 13 of 100
Block 14 of 100
Block 15 of 100
Block 16 of 100
Block 17 of 100
Block 18 of 100
Block 19 of 100
Block 20 of 100
Block 21 of 100
Block 22 of 100
Block 23 of 100
Block 24 of 100
Block 25 of 100
Block 26 of 100
Block 27 of 100
Block 28 of 100
Block 29 of 100
Block 30 of 100
Block 31 of 100
Block 32 of 100
Block 33 of 100
Block 34 of 100
Block 35 of 100
Block 36 of 100
Block 37 of 100
Block 38 of 100
Block 39 of 100
Block 40 of 100
Block 41 of 100
Block 42 of 100
Block 43 of 100
Block 44 of 100
Block 45 of 100
Block 46 of 100
Block 47 of 100
Block 48 of 100
Block 49 of 100
Block 50 of 100
Block 51 of 100
Block 52 of 100
Block 53 of 100
Block 54 of 100
Block 55 of 100
Block 56 of 100
Block 57 of 100
Block 58 of 100
Block 59 of 100
Block 60 of 100
Block 61 of 100
Block 62 of 100
Block 63 of 100
Block 64 of 100
Block 65 of 100
Block 66 of 100
Block 67 of 100
Block 68 of 100
Block 69 of 100
Block 70 of 100
Block 71 of 100
Block 72 of 100
Block 73 of 100
Block 74 of 100
Block 75 of 100
Block 76 of 100
Block 77 of 100
Block 78 of 100
Block 79 of 100
Block 80 of 100
Block 81 of 100
Block 82 of 100
Block 83 of 100
Block 84 of 100
Block 85 of 100
Block 86 of 100
Block 87 of 100
Block 88 of 100
Block 89 of 100
Block 90 of 100
Block 91 of 100
Block 92 of 100
Block 93 of 100
Block 94 of 100
Block 95 of 100
Block 96 of 100
Block 97 of 100
Block 98 of 100
Block 99 of 100
Block 100 of 100
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 100
Block 2 of 100
Block 3 of 100
Block 4 of 100
Block 5 of 100
Block 6 of 100
Block 7 of 100
Block 8 of 100
Block 9 of 100
Block 10 of 100
Block 11 of 100
Block 12 of 100
Block 13 of 100
Block 14 of 100
Block 15 of 100
Block 16 of 100
Block 17 of 100
Block 18 of 100
Block 19 of 100
Block 20 of 100
Block 21 of 100
Block 22 of 100
Block 23 of 100
Block 24 of 100
Block 25 of 100
Block 26 of 100
Block 27 of 100
Block 28 of 100
Block 29 of 100
Block 30 of 100
Block 31 of 100
Block 32 of 100
Block 33 of 100
Block 34 of 100
Block 35 of 100
Block 36 of 100
Block 37 of 100
Block 38 of 100
Block 39 of 100
Block 40 of 100
Block 41 of 100
Block 42 of 100
Block 43 of 100
Block 44 of 100
Block 45 of 100
Block 46 of 100
Block 47 of 100
Block 48 of 100
Block 49 of 100
Block 50 of 100
Block 51 of 100
Block 52 of 100
Block 53 of 100
Block 54 of 100
Block 55 of 100
Block 56 of 100
Block 57 of 100
Block 58 of 100
Block 59 of 100
Block 60 of 100
Block 61 of 100
Block 62 of 100
Block 63 of 100
Block 64 of 100
Block 65 of 100
Block 66 of 100
Block 67 of 100
Block 68 of 100
Block 69 of 100
Block 70 of 100
Block 71 of 100
Block 72 of 100
Block 73 of 100
Block 74 of 100
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 100
Block 2 of 100
Block 3 of 100
Block 4 of 100
Block 5 of 100
Block 6 of 100
Block 7 of 100
Block 8 of 100
Block 9 of 100
Block 10 of 100
Block 11 of 100
Block 12 of 100
Block 13 of 100
Block 14 of 100
Block 15 of 100
Block 16 of 100
Block 17 of 100
Block 18 of 100
Block 19 of 100
Block 20 of 100
Block 21 of 100
Block 22 of 100
Block 23 of 100
Block 24 of 100
Block 25 of 100
Block 26 of 100
Block 27 of 100
Block 28 of 100
Block 29 of 100
Block 30 of 100
Block 31 of 100
Block 32 of 100
Block 33 of 100
Block 34 of 100
Block 35 of 100
Block 36 of 100
Block 37 of 100
Block 38 of 100
Block 39 of 100
Block 40 of 100
Block 41 of 100
Block 42 of 100
Block 43 of 100
Block 44 of 100
Block 45 of 100
Block 46 of 100
Block 47 of 100
Block 48 of 100
Block 49 of 100
Block 50 of 100
Block 51 of 100
Block 52 of 100
Block 53 of 100
Block 54 of 100
Block 55 of 100
Block 56 of 100
Block 57 of 100
Block 58 of 100
Block 59 of 100
Block 60 of 100
Block 61 of 100
Block 62 of 100
Block 63 of 100
Block 64 of 100
Block 65 of 100
Block 66 of 100
Block 67 of 100
Block 68 of 100
Block 69 of 100
Block 70 of 100
Block 71 of 100
Block 72 of 100
Block 73 of 100
Block 74 of 100
Block 75 of 100
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 100
Block 2 of 100
Block 3 of 100
Block 4 of 100
Block 5 of 100
Block 6 of 100
Block 7 of 100
Block 8 of 100
Block 9 of 100
Block 10 of 100
Block 11 of 100
Block 12 of 100
Block 13 of 100
Block 14 of 100
Block 15 of 100
Block 16 of 100
Block 17 of 100
Block 18 of 100
Block 19 of 100
Block 20 of 100
Block 21 of 100
Block 22 of 100
Block 23 of 100
Block 24 of 100
Block 25 of 100
Block 26 of 100
Block 27 of 100
Block 28 of 100
Block 29 of 100
Block 30 of 100
Block 31 of 100
Block 32 of 100
Block 33 of 100
Block 34 of 100
Block 35 of 100
Block 36 of 100
Block 37 of 100
Block 38 of 100
Block 39 of 100
Block 40 of 100
Block 41 of 100
Block 42 of 100
Block 43 of 100
Block 44 of 100
Block 45 of 100
Block 46 of 100
Block 47 of 100
Block 48 of 100
Block 49 of 100
Block 50 of 100
Block 51 of 100
Block 52 of 100
Block 53 of 100
Block 54 of 100
Block 55 of 100
Block 56 of 100
Block 57 of 100
Block 58 of 100
Block 59 of 100
Block 60 of 100
Block 61 of 100
Block 62 of 100
Block 63 of 100
Block 64 of 100
Block 65 of 100
Block 66 of 100
Block 67 of 100
Block 68 of 100
Block 69 of 100
Block 70 of 100
Block 71 of 100
Block 72 of 100
Block 73 of 100
Block 74 of 100
Block 75 of 100
Block 76 of 100
Block 77 of 100
Block 78 of 100
Block 79 of 100
Block 80 of 100
Block 81 of 100
Block 82 of 100
Block 83 of 100
Block 84 of 100
Block 85 of 100
Block 86 of 100
Block 87 of 100
Block 88 of 100
Block 89 of 100
Block 90 of 100
Block 91 of 100
Block 92 of 100
Block 93 of 100
Block 94 of 100
Block 95 of 100
Block 96 of 100
Block 97 of 100
Block 98 of 100
Block 99 of 100
Block 100 of 100
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 100
Block 2 of 100
Block 3 of 100
Block 4 of 100
Block 5 of 100
Block 6 of 100
Block 7 of 100
Block 8 of 100
Block 9 of 100
Block 10 of 100
Block 11 of 100
Block 12 of 100
Block 13 of 100
Block 14 of 100
Block 15 of 100
Block 16 of 100
Block 17 of 100
Block 18 of 100
Block 19 of 100
Block 20 of 100
Block 21 of 100
Block 22 of 100
Block 23 of 100
Block 24 of 100
Block 25 of 100
Block 26 of 100
Block 27 of 100
Block 28 of 100
Block 29 of 100
Block 30 of 100
Block 31 of 100
Block 32 of 100
Block 33 of 100
Block 34 of 100
Block 35 of 100
Block 36 of 100
Block 37 of 100
Block 38 of 100
Block 39 of 100
Block 40 of 100
Block 41 of 100
Block 42 of 100
Block 43 of 100
Block 44 of 100
Block 45 of 100
Block 46 of 100
Block 47 of 100
Block 48 of 100
Block 49 of 100
Block 50 of 100
Block 51 of 100
Block 52 of 100
Block 53 of 100
Block 54 of 100
Block 55 of 100
Block 56 of 100
Block 57 of 100
Block 58 of 100
Block 59 of 100
Block 60 of 100
Block 61 of 100
Block 62 of 100
Block 63 of 100
Block 64 of 100
Block 65 of 100
Block 66 of 100
Block 67 of 100
Block 68 of 100
Block 69 of 100
Block 70 of 100
Block 71 of 100
Block 72 of 100
Block 73 of 100
Block 74 of 100
Determining number of SNPs and samples...
Reading SNP file...
Reading genotype file...
Block 1 of 100
Block 2 of 100
Block 3 of 100
Block 4 of 100
Block 5 of 100
Block 6 of 100
Block 7 of 100
Block 8 of 100
Block 9 of 100
Block 10 of 100
Block 11 of 100
Block 12 of 100
Block 13 of 100
Block 14 of 100
Block 15 of 100
Block 16 of 100
Block 17 of 100
Block 18 of 100
Block 19 of 100
Block 20 of 100
Block 21 of 100
Block 22 of 100
Block 23 of 100
Block 24 of 100
Block 25 of 100
Block 26 of 100
Block 27 of 100
Block 28 of 100
Block 29 of 100
Block 30 of 100
Block 31 of 100
Block 32 of 100
Block 33 of 100
Block 34 of 100
Block 35 of 100
Block 36 of 100
Block 37 of 100
Block 38 of 100
Block 39 of 100
Block 40 of 100
Block 41 of 100
Block 42 of 100
Block 43 of 100
Block 44 of 100
Block 45 of 100
Block 46 of 100
Block 47 of 100
Block 48 of 100
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 4
Block 2 of 4
Block 3 of 4
Block 4 of 4
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 4
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 4
Block 2 of 4
Block 3 of 4
Block 4 of 4
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 4
Block 2 of 4
Block 3 of 4
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 2
Block 2 of 2
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 4
Block 2 of 4
Block 3 of 4
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 2
Determining number of SNPs and samples...
Reading sample file...
Reading genotype file...
Block 1 of 33
Block 2 of 33
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 13
Block 2 of 13
Block 3 of 13
Block 4 of 13
Block 5 of 13
Block 6 of 13
Block 7 of 13
Block 8 of 13
Block 9 of 13
Block 10 of 13
Block 11 of 13
Block 12 of 13
Block 13 of 13
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 13
Block 2 of 13
Block 3 of 13
Block 4 of 13
Block 5 of 13
Block 6 of 13
Block 7 of 13
Block 8 of 13
Block 9 of 13
Block 10 of 13
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 13
Block 2 of 13
Block 3 of 13
Block 4 of 13
Block 5 of 13
Block 6 of 13
Block 7 of 13
Block 8 of 13
Block 9 of 13
Block 10 of 13
Block 11 of 13
Block 12 of 13
Block 13 of 13
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 13
Block 2 of 13
Block 3 of 13
Block 4 of 13
Block 5 of 13
Block 6 of 13
Block 7 of 13
Block 8 of 13
Block 9 of 13
Block 10 of 13
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 6
Block 2 of 6
Block 3 of 6
Block 4 of 6
Block 5 of 6
Block 6 of 6
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 6
Block 2 of 6
Block 3 of 6
Block 4 of 6
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 6
Block 2 of 6
Block 3 of 6
Block 4 of 6
Block 5 of 6
Block 6 of 6
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 6
Block 2 of 6
Block 3 of 6
Block 4 of 6
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 13
Block 2 of 13
Block 3 of 13
Block 4 of 13
Block 5 of 13
Block 6 of 13
Block 7 of 13
Block 8 of 13
Block 9 of 13
Block 10 of 13
Block 11 of 13
Block 12 of 13
Block 13 of 13
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 13
Block 2 of 13
Block 3 of 13
Block 4 of 13
Block 5 of 13
Block 6 of 13
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 13
Block 2 of 13
Block 3 of 13
Block 4 of 13
Block 5 of 13
Block 6 of 13
Block 7 of 13
Block 8 of 13
Block 9 of 13
Block 10 of 13
Block 11 of 13
Block 12 of 13
Block 13 of 13
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 13
Block 2 of 13
Block 3 of 13
Block 4 of 13
Block 5 of 13
Block 6 of 13
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 6
Block 2 of 6
Block 3 of 6
Block 4 of 6
Block 5 of 6
Block 6 of 6
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 6
Block 2 of 6
Block 3 of 6
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 6
Block 2 of 6
Block 3 of 6
Block 4 of 6
Block 5 of 6
Block 6 of 6
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 6
Block 2 of 6
Block 3 of 6
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Determining number of SNPs and samples...
Reading SNP files...
Reading genotype file...
Block 1 of 1
Killed
</PRE>
</DIV><HR>
<H3>Example timings</H3>
</BODY>
</HTML>
