<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for SpliceWiz on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><B>[<A href="../index.html#S">S</A>]</B></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-08 01:15:30 -0000 (Wed, 08 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4164</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for SpliceWiz on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the SpliceWiz package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/SpliceWiz.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1932</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">SpliceWiz</A>&nbsp;1.1.5</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/SpliceWiz">landing page</A>)</SPAN><BR>Alex Chit Hei Wong<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-06&nbsp;00:12:45&nbsp;-0000&nbsp;(Mon,&nbsp;06&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/SpliceWiz</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">79fa481</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-12-19&nbsp;23:37:31&nbsp;-0000&nbsp;(Mon,&nbsp;19&nbsp;Dec&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: SpliceWiz</TD></TR>
<TR><TD><B>Version</B>: 1.1.5</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:SpliceWiz.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings SpliceWiz_1.1.5.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-07 16:39:31 -0000 (Tue, 07 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-07 16:47:57 -0000 (Tue, 07 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 505.9 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: SpliceWiz.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:SpliceWiz.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings SpliceWiz_1.1.5.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/SpliceWiz.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘SpliceWiz/DESCRIPTION’ ... OK
* this is package ‘SpliceWiz’ version ‘1.1.5’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘SpliceWiz’ can be installed ... OK
* used C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... NOTE
  installed size is 12.1Mb
  sub-directories of 1Mb or more:
    R      2.0Mb
    libs   9.0Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
Killed
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... NOTE
Killed
The \usage entries for S3 methods should use the \method markup and not
their full name.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in shell scripts ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... OK
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘SpliceWiz-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: ASE-GLM-edgeR
&gt; ### Title: Using Generalised linear models to analyse differential ASEs
&gt; ###   using edgeR
&gt; ### Aliases: ASE-GLM-edgeR fitASE_edgeR fitASE_edgeR_custom testASE_edgeR
&gt; ###   addPSI_edgeR
&gt; 
&gt; ### ** Examples
&gt; 
&gt; # Load the NxtSE object and set up the annotations
&gt; # - see ?makeSE on example code of generating this NxtSE object
&gt; se &lt;- SpliceWiz_example_NxtSE()
&gt; 
&gt; colData(se)$treatment &lt;- rep(c(&quot;A&quot;, &quot;B&quot;), each = 3)
&gt; colData(se)$replicate &lt;- rep(c(&quot;P&quot;,&quot;Q&quot;,&quot;R&quot;), 2)
&gt; require(&quot;edgeR&quot;)
Loading required package: edgeR
Loading required package: limma
&gt; 
&gt; fit &lt;- fitASE_edgeR(
+     se, 
+     strModelFormula = &quot;~0 + replicate + treatment&quot;, 
+     strASEFormula = &quot;~0 + replicate + treatment + treatment:ASE&quot;
+ )
Feb 07 16:45:23 Fitting edgeR contrast for included / excluded counts separately
Feb 07 16:45:24 Fitting edgeR contrast for included / excluded counts together
&gt; 
&gt; # Get coefficient terms of Included / Excluded counts isolated model
&gt; colnames(fit$model_IncExc)
[1] &quot;replicateP&quot; &quot;replicateQ&quot; &quot;replicateR&quot; &quot;treatmentB&quot;
&gt; # [1] &quot;replicateP&quot; &quot;replicateQ&quot; &quot;replicateR&quot; &quot;treatmentB&quot;
&gt; 
&gt; # Get coefficient terms of PSI model
&gt; colnames(fit$model_ASE)
[1] &quot;replicateP&quot;             &quot;replicateQ&quot;             &quot;replicateR&quot;            
[4] &quot;treatmentB&quot;             &quot;treatmentA:ASEIncluded&quot; &quot;treatmentB:ASEIncluded&quot;
&gt; # [1] &quot;replicateP&quot; &quot;replicateQ&quot; &quot;replicateR&quot; &quot;treatmentB&quot;            
&gt; # [5] &quot;treatmentA:ASEIncluded&quot; &quot;treatmentB:ASEIncluded&quot;
&gt; 
&gt; # Contrast between treatment &quot;B&quot; against treatment &quot;A&quot;
&gt; res &lt;- testASE_edgeR(se, fit,
+     contrast_IncExc = c(0,0,0,1),
+     contrast_ASE = c(0,0,0,0,-1,1)
+ )
&gt; 
&gt; ### # Add mean PSI values to results:
&gt; res_withPSI &lt;- addPSI_edgeR(res, se, &quot;treatment&quot;, c(&quot;B&quot;, &quot;A&quot;))
&gt; 
&gt; 
&gt; ### Using custom model matrices to model counts
&gt; #   - the equivalent analysis can be performed as follows:
&gt; 
&gt; # Sample annotations for isoform count expressions
&gt; colData &lt;- as.data.frame(colData(se))
&gt; 
&gt; # Sample annotations for isoform count PSI analysis
&gt; colData_ASE &lt;- rbind(colData, colData)
&gt; colData_ASE$ASE &lt;- rep(c(&quot;Included&quot;, &quot;Excluded&quot;), each = nrow(colData))
&gt; rownames(colData_ASE) &lt;- c(
+     paste0(rownames(colData), &quot;.Included&quot;),
+     paste0(rownames(colData), &quot;.Excluded&quot;)
+ )
&gt; 
&gt; model_IncExc &lt;- model.matrix(
+     ~0 + replicate + treatment,
+     data = colData
+ )
&gt; 
&gt; model_ASE &lt;- model.matrix(
+     ~0 + replicate + treatment + treatment:ASE,
+     data = colData_ASE
+ )
&gt; 
&gt; fit &lt;- fitASE_edgeR_custom(se, model_IncExc, model_ASE)
Feb 07 16:45:28 Fitting edgeR contrast for included / excluded counts separately
Feb 07 16:45:28 Fitting edgeR contrast for included / excluded counts together
&gt; 
&gt; res_customModel &lt;- testASE_edgeR(se, fit,
+     contrast_IncExc = c(0,0,0,1),
+     contrast_ASE = c(0,0,0,0,-1,1)
+ )
&gt; 
&gt; # Check this produces identical results:
&gt; identical(res_customModel, res)
[1] TRUE
&gt; 
&gt; ### Time series examples using edgeR and splines 
&gt; # - similar to section 4.8 in the edgeR vignette
&gt; 
&gt; colData(se)$timepoint &lt;- rep(c(1,2,3), each = 2)
&gt; colData(se)$batch &lt;- rep(c(&quot;1&quot;, &quot;2&quot;), 3)
&gt; 
&gt; # First, we set up a polynomial spline with 2 degrees of freedom:
&gt; Time &lt;- poly(colData(se)$timepoint, df = 2)
&gt; 
&gt; # Next, we define the batch factor:
&gt; Batch &lt;- factor(colData(se)$batch)
&gt; 
&gt; # Finally, we construct the same factors for ASE analysis. Note that
&gt; #   each factor must be repeated twice
&gt; 
&gt; Time_ASE &lt;- rbind(Time, Time)
&gt; Batch_ASE &lt;- c(Batch, Batch)
&gt; ASE &lt;- factor(
+     rep(c(&quot;Included&quot;, &quot;Excluded&quot;), each = nrow(colData(se)))
+ )
&gt; 
&gt; # Now, we set up the model matrices for isoform and PSI count modelling
&gt; model_IncExc &lt;- model.matrix(~0 + Batch + Time)
&gt; model_ASE &lt;- model.matrix(~0 + Batch_ASE + Time_ASE + Time_ASE:ASE)
&gt; 
&gt; fit &lt;- fitASE_edgeR_custom(se, model_IncExc, model_ASE)
Feb 07 16:45:28 Fitting edgeR contrast for included / excluded counts separately
Killed
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  ||                           Annotations : transcripts.gtf (GTF)              ||
  ||                                                                            ||
  \\============================================================================//
  
  //=============== Running (07-Feb-2023 16:46:02, pid=4178521) ================\\
  ||                                                                            ||
  || Check the input reads.                                                     ||
  || The input file contains base space reads.                                  ||
  || Initialise the memory objects.                                             ||
  || Estimate the mean read length.                                             ||
  || Create the output BAM file.                                                ||
  || Check the index.                                                           ||
  Unable top open index &#x27;./reference_index&#x27;. Please make sure that the correct prefix is specified and you have the permission to read these files. For example, if there are files &#x27;/opt/my_index.reads&#x27;, &#x27;/opt/my_index.files&#x27; and etc, the index prefix should be specified as &#x27;/opt/my_index&#x27; without any suffix. 
  
  Killed
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘SW_Cookbook.Rmd’ using ‘UTF-8’... OK
  ‘SW_QuickStart.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... ERROR
Error(s) in re-building vignettes:
  ...
--- re-building ‘SW_Cookbook.Rmd’ using rmarkdown
--- finished re-building ‘SW_Cookbook.Rmd’

--- re-building ‘SW_QuickStart.Rmd’ using rmarkdown
Killed
SUMMARY: processing the following file failed:
  ‘SW_QuickStart.Rmd’

Error: Vignette re-building failed.
Execution halted

* checking PDF version of manual ... OK
* DONE

Status: 3 ERRORs, 4 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/SpliceWiz.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">SpliceWiz.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL SpliceWiz
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘SpliceWiz’ ...
** using staged installation
** libs
using C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
using C++11
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/ompBAM/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppProgress/include&#x27; -I/usr/local/include   -fopenmp -DSPLICEWIZ -fPIC  -g -O2  -Wall -c BAM2blocks.cpp -o BAM2blocks.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/ompBAM/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppProgress/include&#x27; -I/usr/local/include   -fopenmp -DSPLICEWIZ -fPIC  -g -O2  -Wall -c FastaReader.cpp -o FastaReader.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/ompBAM/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppProgress/include&#x27; -I/usr/local/include   -fopenmp -DSPLICEWIZ -fPIC  -g -O2  -Wall -c FragmentBlocks.cpp -o FragmentBlocks.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/ompBAM/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppProgress/include&#x27; -I/usr/local/include   -fopenmp -DSPLICEWIZ -fPIC  -g -O2  -Wall -c GZTools.cpp -o GZTools.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/ompBAM/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppProgress/include&#x27; -I/usr/local/include   -fopenmp -DSPLICEWIZ -fPIC  -g -O2  -Wall -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/ompBAM/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppProgress/include&#x27; -I/usr/local/include   -fopenmp -DSPLICEWIZ -fPIC  -g -O2  -Wall -c ReadBlockProcessor.cpp -o ReadBlockProcessor.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/ompBAM/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppProgress/include&#x27; -I/usr/local/include   -fopenmp -DSPLICEWIZ -fPIC  -g -O2  -Wall -c ReadBlockProcessor_CoverageBlocks.cpp -o ReadBlockProcessor_CoverageBlocks.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/ompBAM/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppProgress/include&#x27; -I/usr/local/include   -fopenmp -DSPLICEWIZ -fPIC  -g -O2  -Wall -c ReadBlockProcessor_FragmentsMap.cpp -o ReadBlockProcessor_FragmentsMap.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/ompBAM/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppProgress/include&#x27; -I/usr/local/include   -fopenmp -DSPLICEWIZ -fPIC  -g -O2  -Wall -c ReadBlockProcessor_TandemJunctions.cpp -o ReadBlockProcessor_TandemJunctions.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/ompBAM/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppProgress/include&#x27; -I/usr/local/include   -fopenmp -DSPLICEWIZ -fPIC  -g -O2  -Wall -c covTools.cpp -o covTools.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/ompBAM/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppProgress/include&#x27; -I/usr/local/include   -fopenmp -DSPLICEWIZ -fPIC  -g -O2  -Wall -c main.cpp -o main.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/ompBAM/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppProgress/include&#x27; -I/usr/local/include   -fopenmp -DSPLICEWIZ -fPIC  -g -O2  -Wall -c swEngine.cpp -o swEngine.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/ompBAM/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/RcppProgress/include&#x27; -I/usr/local/include   -fopenmp -DSPLICEWIZ -fPIC  -g -O2  -Wall -c synthReadGenerator.cpp -o synthReadGenerator.o
g++ -std=gnu++11 -shared -L/home/biocbuild/bbs-3.17-bioc/R/lib -L/usr/local/lib -o SpliceWiz.so BAM2blocks.o FastaReader.o FragmentBlocks.o GZTools.o RcppExports.o ReadBlockProcessor.o ReadBlockProcessor_CoverageBlocks.o ReadBlockProcessor_FragmentsMap.o ReadBlockProcessor_TandemJunctions.o covTools.o main.o swEngine.o synthReadGenerator.o -fopenmp -DSPLICEWIZ -L/home/biocbuild/bbs-3.17-bioc/R/lib -lR
installing to /home/biocbuild/bbs-3.17-bioc/R/library/00LOCK-SpliceWiz/00new/SpliceWiz/libs
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (SpliceWiz)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">SpliceWiz.Rcheck/tests/testthat.Rout.fail</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(SpliceWiz)
Loading required package: NxtIRFdata
SpliceWiz package loaded with 2 threads
Use setSWthreads() to set the number of SpliceWiz threads
&gt; 
&gt; test_check(&quot;SpliceWiz&quot;)
Loading required package: Rsubread
Feb 07 16:45:53 Converting FASTA to local TwoBitFile...done
Feb 07 16:45:55 Connecting to genome TwoBitFile...done
Feb 07 16:45:55 Making local copy of GTF file...done
Feb 07 16:45:55 Extracting temp genome FASTA from TwoBit file
Feb 07 16:46:02 Generating synthetic reads, saving to /tmp/Rtmpb6n8cv/refWithMapExcl/Mappability/Reads.fa
0%   10   20   30   40   50   60   70   80   90   100%
[----|----|----|----|----|----|----|----|----|----|
**************************************************|
10145 synthetic reads generated
Feb 07 16:46:02 Cleaning temp genome / gene annotation files


        ==========     _____ _    _ ____  _____  ______          _____  
        =====         / ____| |  | |  _ \|  __ \|  ____|   /\   |  __ \ 
          =====      | (___ | |  | | |_) | |__) | |__     /  \  | |  | |
            ====      \___ \| |  | |  _ &lt;|  _  /|  __|   / /\ \ | |  | |
              ====    ____) | |__| | |_) | | \ \| |____ / ____ \| |__| |
        ==========   |_____/ \____/|____/|_|  \_\______/_/    \_\_____/
       Rsubread 2.13.2

//================================= setting ==================================\\
||                                                                            ||
|| Function      : Read alignment + Junction detection (RNA-Seq)              ||
|| Input file    : Reads.fa                                                   ||
|| Output file   : AlignedReads.bam (BAM)                                     ||
|| Index name    : reference_index                                            ||
||                                                                            ||
||                    ------------------------------------                    ||
||                                                                            ||
||                               Threads : 1                                  ||
||                          Phred offset : 33                                 ||
||                             Min votes : 1 / 14                             ||
||                        Max mismatches : 3                                  ||
||                      Max indel length : 5                                  ||
||            Report multi-mapping reads : yes                                ||
|| Max alignments per multi-mapping read : 1                                  ||
||                           Annotations : transcripts.gtf (GTF)              ||
||                                                                            ||
\\============================================================================//

//=============== Running (07-Feb-2023 16:46:02, pid=4178521) ================\\
||                                                                            ||
|| Check the input reads.                                                     ||
|| The input file contains base space reads.                                  ||
|| Initialise the memory objects.                                             ||
|| Estimate the mean read length.                                             ||
|| Create the output BAM file.                                                ||
|| Check the index.                                                           ||
Unable top open index &#x27;./reference_index&#x27;. Please make sure that the correct prefix is specified and you have the permission to read these files. For example, if there are files &#x27;/opt/my_index.reads&#x27;, &#x27;/opt/my_index.files&#x27; and etc, the index prefix should be specified as &#x27;/opt/my_index&#x27; without any suffix. 

Killed
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">SpliceWiz.Rcheck/SpliceWiz-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
