<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for CoGAPS on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><B>[<A href="../index.html#C">C</A>]</B></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-01-29 16:33:41 -0000 (Sun, 29 Jan 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4021</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for CoGAPS on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the CoGAPS package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/CoGAPS.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>374</B>/2162</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">CoGAPS</A>&nbsp;3.19.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/CoGAPS">landing page</A>)</SPAN><BR>Elana J. Fertig<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-01-27&nbsp;13:09:27&nbsp;-0000&nbsp;(Fri,&nbsp;27&nbsp;Jan&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/CoGAPS</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">bef6c4c</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:05:12&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: CoGAPS</TD></TR>
<TR><TD><B>Version</B>: 3.19.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:CoGAPS.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings CoGAPS_3.19.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-01-28 20:18:05 -0000 (Sat, 28 Jan 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-01-28 20:25:34 -0000 (Sat, 28 Jan 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 448.8 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: CoGAPS.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 4</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:CoGAPS.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings CoGAPS_3.19.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/CoGAPS.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘CoGAPS/DESCRIPTION’ ... OK
* this is package ‘CoGAPS’ version ‘3.19.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘CoGAPS’ can be installed ... WARNING
Found the following significant warnings:
  GapsRunner.cpp:430:5: warning: enumeration value ‘GAPS_ALL_PHASES’ not handled in switch [-Wswitch]
See ‘/home/biocbuild/bbs-3.17-bioc/meat/CoGAPS.Rcheck/00install.out’ for details.
* used C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... WARNING
Invalid citation information in ‘inst/CITATION’:
  Warning message:
  In utils::packageDescription(basename(dir), dirname(dir)) :
    no package &#x27;CoGAPS&#x27; was found
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... WARNING
Error in library(CoGAPS, lib.loc = &quot;/home/biocbuild/bbs-3.17-bioc/R/library&quot;) : 
  there is no package called ‘CoGAPS’
Execution halted

It looks like this package has a loading problem when not on .libPaths:
see the messages for details.
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
plot.CogapsResult: no visible global function definition for ‘axis’
plot.CogapsResult: no visible global function definition for ‘text’
plot.CogapsResult: no visible global function definition for ‘par’
Undefined global functions or variables:
  axis par text
Consider adding
  importFrom(&quot;graphics&quot;, &quot;axis&quot;, &quot;par&quot;, &quot;text&quot;)
to your NAMESPACE file.
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... WARNING
Codoc mismatches from documentation object &#x27;plotPatternMarkers&#x27;:
plotPatternMarkers
  Code: function(object, data, patternMarkers, patternPalette,
                 sampleNames, samplePalette = NULL, heatmapCol =
                 bluered, colDendrogram = TRUE, scale = &quot;row&quot;, ...)
  Docs: function(object, data, patternPalette, sampleNames,
                 samplePalette = NULL, heatmapCol = bluered,
                 colDenogram = TRUE, scale = &quot;row&quot;, ...)
  Argument names in code not in docs:
    patternMarkers colDendrogram
  Argument names in docs not in code:
    colDenogram
  Mismatches in argument names (first 3):
    Position: 3 Code: patternMarkers Docs: patternPalette
    Position: 4 Code: patternPalette Docs: sampleNames
    Position: 5 Code: sampleNames Docs: samplePalette

* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in shell scripts ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... OK
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... SKIPPED
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘CoGAPS.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 4 WARNINGs, 1 NOTE
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/CoGAPS.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">CoGAPS.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL CoGAPS
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/site-library’
* installing *source* package ‘CoGAPS’ ...
** using staged installation
checking whether the C++ compiler works... yes
checking for C++ compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C++ compiler... yes
checking whether g++ -std=gnu++14 accepts -g... yes
checking how to run the C++ preprocessor... g++ -std=gnu++14 -E
checking whether we are using the GNU C++ compiler... (cached) yes
checking whether g++ -std=gnu++14 accepts -g... (cached) yes
./configure: line 2752: AX_COMPILER_VENDOR: command not found
./configure: line 2753: AX_COMPILER_VERSION: command not found
./configure: line 2764: AX_OPENMP: command not found
building on  compiler version 
Using AVX instructions if available
configure: creating ./config.status
config.status: creating src/Makevars
** libs
using C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c Cogaps.cpp -o Cogaps.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c GapsParameters.cpp -o GapsParameters.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c GapsResult.cpp -o GapsResult.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c GapsRunner.cpp -o GapsRunner.o
In file included from GapsRunner.cpp:8:
gibbs_sampler/AsynchronousGibbsSampler.h:105: warning: ignoring ‘#pragma omp parallel’ [-Wunknown-pragmas]
  105 |         #pragma omp parallel for num_threads(nThreads)
      | 
In file included from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/mpl/aux_/na_assert.hpp:23,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/mpl/arg.hpp:25,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/mpl/placeholders.hpp:24,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/iterator/iterator_categories.hpp:16,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/iterator/iterator_facade.hpp:13,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/range/iterator_range_core.hpp:27,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/algorithm/string/replace.hpp:16,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/date_time/date_facet.hpp:17,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/date_time/gregorian/gregorian_io.hpp:16,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/date_time/gregorian/gregorian.hpp:31,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/date_time/posix_time/time_formatters.hpp:12,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/date_time/posix_time/posix_time.hpp:24,
                 from GapsRunner.cpp:27:
/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/mpl/assert.hpp:194:21: warning: unnecessary parentheses in declaration of ‘assert_arg’ [-Wparentheses]
  194 | failed ************ (Pred::************
      |                     ^~~~~~~~~~~~~~~~~~~
  195 |       assert_arg( void (*)(Pred), typename assert_arg_pred&lt;Pred&gt;::type )
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  196 |     );
      |     ~                
/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/mpl/assert.hpp:194:21: note: remove parentheses
  194 | failed ************ (Pred::************
      |                     ^~~~~~~~~~~~~~~~~~~
      |                     -
  195 |       assert_arg( void (*)(Pred), typename assert_arg_pred&lt;Pred&gt;::type )
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  196 |     );
      |     ~                
      |     -
/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/mpl/assert.hpp:199:21: warning: unnecessary parentheses in declaration of ‘assert_not_arg’ [-Wparentheses]
  199 | failed ************ (boost::mpl::not_&lt;Pred&gt;::************
      |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  200 |       assert_not_arg( void (*)(Pred), typename assert_arg_pred_not&lt;Pred&gt;::type )
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  201 |     );
      |     ~                
/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/mpl/assert.hpp:199:21: note: remove parentheses
  199 | failed ************ (boost::mpl::not_&lt;Pred&gt;::************
      |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                     -
  200 |       assert_not_arg( void (*)(Pred), typename assert_arg_pred_not&lt;Pred&gt;::type )
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  201 |     );
      |     ~                
      |     -
GapsRunner.cpp: In instantiation of ‘GapsResult runCoGAPSAlgorithm(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with Sampler = AsynchronousGibbsSampler&lt;SparseNormalModel&gt;; DataType = Matrix]’:
GapsRunner.cpp:70:73:   required from ‘GapsResult chooseSampler(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataModel = SparseNormalModel; DataType = Matrix]’
GapsRunner.cpp:85:48:   required from ‘GapsResult chooseDataModel(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataType = Matrix]’
GapsRunner.cpp:104:27:   required from ‘GapsResult run_helper(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataType = Matrix]’
GapsRunner.cpp:113:22:   required from here
GapsRunner.cpp:430:5: warning: enumeration value ‘GAPS_ALL_PHASES’ not handled in switch [-Wswitch]
  430 |     switch (phase)
      |     ^~~~~~
GapsRunner.cpp: In instantiation of ‘GapsResult runCoGAPSAlgorithm(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with Sampler = SingleThreadedGibbsSampler&lt;SparseNormalModel&gt;; DataType = Matrix]’:
GapsRunner.cpp:74:71:   required from ‘GapsResult chooseSampler(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataModel = SparseNormalModel; DataType = Matrix]’
GapsRunner.cpp:85:48:   required from ‘GapsResult chooseDataModel(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataType = Matrix]’
GapsRunner.cpp:104:27:   required from ‘GapsResult run_helper(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataType = Matrix]’
GapsRunner.cpp:113:22:   required from here
GapsRunner.cpp:430:5: warning: enumeration value ‘GAPS_ALL_PHASES’ not handled in switch [-Wswitch]
GapsRunner.cpp: In instantiation of ‘GapsResult runCoGAPSAlgorithm(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with Sampler = AsynchronousGibbsSampler&lt;DenseNormalModel&gt;; DataType = Matrix]’:
GapsRunner.cpp:70:73:   required from ‘GapsResult chooseSampler(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataModel = DenseNormalModel; DataType = Matrix]’
GapsRunner.cpp:88:43:   required from ‘GapsResult chooseDataModel(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataType = Matrix]’
GapsRunner.cpp:104:27:   required from ‘GapsResult run_helper(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataType = Matrix]’
GapsRunner.cpp:113:22:   required from here
GapsRunner.cpp:430:5: warning: enumeration value ‘GAPS_ALL_PHASES’ not handled in switch [-Wswitch]
GapsRunner.cpp: In instantiation of ‘GapsResult runCoGAPSAlgorithm(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with Sampler = SingleThreadedGibbsSampler&lt;DenseNormalModel&gt;; DataType = Matrix]’:
GapsRunner.cpp:74:71:   required from ‘GapsResult chooseSampler(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataModel = DenseNormalModel; DataType = Matrix]’
GapsRunner.cpp:88:43:   required from ‘GapsResult chooseDataModel(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataType = Matrix]’
GapsRunner.cpp:104:27:   required from ‘GapsResult run_helper(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataType = Matrix]’
GapsRunner.cpp:113:22:   required from here
GapsRunner.cpp:430:5: warning: enumeration value ‘GAPS_ALL_PHASES’ not handled in switch [-Wswitch]
GapsRunner.cpp: In instantiation of ‘GapsResult runCoGAPSAlgorithm(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with Sampler = AsynchronousGibbsSampler&lt;SparseNormalModel&gt;; DataType = std::__cxx11::basic_string&lt;char&gt;]’:
GapsRunner.cpp:70:73:   required from ‘GapsResult chooseSampler(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataModel = SparseNormalModel; DataType = std::__cxx11::basic_string&lt;char&gt;]’
GapsRunner.cpp:85:48:   required from ‘GapsResult chooseDataModel(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataType = std::__cxx11::basic_string&lt;char&gt;]’
GapsRunner.cpp:104:27:   required from ‘GapsResult run_helper(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataType = std::__cxx11::basic_string&lt;char&gt;]’
GapsRunner.cpp:119:22:   required from here
GapsRunner.cpp:430:5: warning: enumeration value ‘GAPS_ALL_PHASES’ not handled in switch [-Wswitch]
GapsRunner.cpp: In instantiation of ‘GapsResult runCoGAPSAlgorithm(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with Sampler = SingleThreadedGibbsSampler&lt;SparseNormalModel&gt;; DataType = std::__cxx11::basic_string&lt;char&gt;]’:
GapsRunner.cpp:74:71:   required from ‘GapsResult chooseSampler(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataModel = SparseNormalModel; DataType = std::__cxx11::basic_string&lt;char&gt;]’
GapsRunner.cpp:85:48:   required from ‘GapsResult chooseDataModel(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataType = std::__cxx11::basic_string&lt;char&gt;]’
GapsRunner.cpp:104:27:   required from ‘GapsResult run_helper(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataType = std::__cxx11::basic_string&lt;char&gt;]’
GapsRunner.cpp:119:22:   required from here
GapsRunner.cpp:430:5: warning: enumeration value ‘GAPS_ALL_PHASES’ not handled in switch [-Wswitch]
GapsRunner.cpp: In instantiation of ‘GapsResult runCoGAPSAlgorithm(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with Sampler = AsynchronousGibbsSampler&lt;DenseNormalModel&gt;; DataType = std::__cxx11::basic_string&lt;char&gt;]’:
GapsRunner.cpp:70:73:   required from ‘GapsResult chooseSampler(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataModel = DenseNormalModel; DataType = std::__cxx11::basic_string&lt;char&gt;]’
GapsRunner.cpp:88:43:   required from ‘GapsResult chooseDataModel(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataType = std::__cxx11::basic_string&lt;char&gt;]’
GapsRunner.cpp:104:27:   required from ‘GapsResult run_helper(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataType = std::__cxx11::basic_string&lt;char&gt;]’
GapsRunner.cpp:119:22:   required from here
GapsRunner.cpp:430:5: warning: enumeration value ‘GAPS_ALL_PHASES’ not handled in switch [-Wswitch]
GapsRunner.cpp: In instantiation of ‘GapsResult runCoGAPSAlgorithm(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with Sampler = SingleThreadedGibbsSampler&lt;DenseNormalModel&gt;; DataType = std::__cxx11::basic_string&lt;char&gt;]’:
GapsRunner.cpp:74:71:   required from ‘GapsResult chooseSampler(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataModel = DenseNormalModel; DataType = std::__cxx11::basic_string&lt;char&gt;]’
GapsRunner.cpp:88:43:   required from ‘GapsResult chooseDataModel(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataType = std::__cxx11::basic_string&lt;char&gt;]’
GapsRunner.cpp:104:27:   required from ‘GapsResult run_helper(const DataType&amp;, GapsParameters&amp;, const DataType&amp;, GapsRandomState*) [with DataType = std::__cxx11::basic_string&lt;char&gt;]’
GapsRunner.cpp:119:22:   required from here
GapsRunner.cpp:430:5: warning: enumeration value ‘GAPS_ALL_PHASES’ not handled in switch [-Wswitch]
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c GapsStatistics.cpp -o GapsStatistics.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c test-runner.cpp -o test-runner.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c atomic/Atom.cpp -o atomic/Atom.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c atomic/ConcurrentAtom.cpp -o atomic/ConcurrentAtom.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c atomic/AtomicDomain.cpp -o atomic/AtomicDomain.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c atomic/ConcurrentAtomicDomain.cpp -o atomic/ConcurrentAtomicDomain.o
atomic/ConcurrentAtomicDomain.cpp:64: warning: ignoring ‘#pragma omp critical’ [-Wunknown-pragmas]
   64 |     #pragma omp critical(AtomicInsertOrErase)
      | 
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c atomic/ProposalQueue.cpp -o atomic/ProposalQueue.o
atomic/ProposalQueue.cpp:101: warning: ignoring ‘#pragma omp atomic’ [-Wunknown-pragmas]
  101 |     #pragma omp atomic
      | 
atomic/ProposalQueue.cpp:107: warning: ignoring ‘#pragma omp atomic’ [-Wunknown-pragmas]
  107 |     #pragma omp atomic
      | 
atomic/ProposalQueue.cpp:113: warning: ignoring ‘#pragma omp atomic’ [-Wunknown-pragmas]
  113 |     #pragma omp atomic
      | 
atomic/ProposalQueue.cpp:119: warning: ignoring ‘#pragma omp atomic’ [-Wunknown-pragmas]
  119 |     #pragma omp atomic
      | 
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c data_structures/HashSets.cpp -o data_structures/HashSets.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c data_structures/HybridMatrix.cpp -o data_structures/HybridMatrix.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c data_structures/HybridVector.cpp -o data_structures/HybridVector.o
data_structures/HybridVector.cpp:60: warning: ignoring ‘#pragma omp atomic’ [-Wunknown-pragmas]
   60 |         #pragma omp atomic
      | 
data_structures/HybridVector.cpp:65: warning: ignoring ‘#pragma omp atomic’ [-Wunknown-pragmas]
   65 |     #pragma omp atomic
      | 
data_structures/HybridVector.cpp:77: warning: ignoring ‘#pragma omp atomic’ [-Wunknown-pragmas]
   77 |         #pragma omp atomic
      | 
data_structures/HybridVector.cpp:82: warning: ignoring ‘#pragma omp atomic’ [-Wunknown-pragmas]
   82 |     #pragma omp atomic
      | 
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c data_structures/Matrix.cpp -o data_structures/Matrix.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c data_structures/SparseIterator.cpp -o data_structures/SparseIterator.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c data_structures/SparseMatrix.cpp -o data_structures/SparseMatrix.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c data_structures/SparseVector.cpp -o data_structures/SparseVector.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c data_structures/Vector.cpp -o data_structures/Vector.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c file_parser/CharacterDelimitedParser.cpp -o file_parser/CharacterDelimitedParser.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c file_parser/FileParser.cpp -o file_parser/FileParser.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c file_parser/MatrixElement.cpp -o file_parser/MatrixElement.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c file_parser/MtxParser.cpp -o file_parser/MtxParser.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c gibbs_sampler/AlphaParameters.cpp -o gibbs_sampler/AlphaParameters.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c gibbs_sampler/DenseNormalModel.cpp -o gibbs_sampler/DenseNormalModel.o
gibbs_sampler/DenseNormalModel.cpp:26: warning: ignoring ‘#pragma omp parallel’ [-Wunknown-pragmas]
   26 |     #pragma omp parallel for num_threads(nThreads)
      | 
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c gibbs_sampler/SparseNormalModel.cpp -o gibbs_sampler/SparseNormalModel.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c math/Math.cpp -o math/Math.o
In file included from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/mpl/aux_/na_assert.hpp:23,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/mpl/arg.hpp:25,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/mpl/placeholders.hpp:24,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/iterator/iterator_categories.hpp:16,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/iterator/iterator_facade.hpp:13,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/range/iterator_range_core.hpp:27,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/lexical_cast.hpp:30,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/math/tools/convert_from_string.hpp:15,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/math/constants/constants.hpp:14,
                 from /home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/math/distributions/exponential.hpp:10,
                 from math/Math.cpp:4:
/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/mpl/assert.hpp:194:21: warning: unnecessary parentheses in declaration of ‘assert_arg’ [-Wparentheses]
  194 | failed ************ (Pred::************
      |                     ^~~~~~~~~~~~~~~~~~~
  195 |       assert_arg( void (*)(Pred), typename assert_arg_pred&lt;Pred&gt;::type )
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  196 |     );
      |     ~                
/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/mpl/assert.hpp:194:21: note: remove parentheses
  194 | failed ************ (Pred::************
      |                     ^~~~~~~~~~~~~~~~~~~
      |                     -
  195 |       assert_arg( void (*)(Pred), typename assert_arg_pred&lt;Pred&gt;::type )
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  196 |     );
      |     ~                
      |     -
/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/mpl/assert.hpp:199:21: warning: unnecessary parentheses in declaration of ‘assert_not_arg’ [-Wparentheses]
  199 | failed ************ (boost::mpl::not_&lt;Pred&gt;::************
      |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  200 |       assert_not_arg( void (*)(Pred), typename assert_arg_pred_not&lt;Pred&gt;::type )
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  201 |     );
      |     ~                
/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include/boost/mpl/assert.hpp:199:21: note: remove parentheses
  199 | failed ************ (boost::mpl::not_&lt;Pred&gt;::************
      |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                     -
  200 |       assert_not_arg( void (*)(Pred), typename assert_arg_pred_not&lt;Pred&gt;::type )
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  201 |     );
      |     ~                
      |     -
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c math/MatrixMath.cpp -o math/MatrixMath.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c math/Random.cpp -o math/Random.o
g++ -std=gnu++14  -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=0 -DGAPS_DISABLE_CHECKPOINTS -D__GAPS_R_BUILD__ -Iinclude -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/BH/include&#x27; -I/usr/local/include   -march=native  -fPIC  -g -O2  -Wall  -c math/VectorMath.cpp -o math/VectorMath.o
g++ -std=gnu++14 -shared -L/home/biocbuild/bbs-3.17-bioc/R/lib -L/usr/local/lib -o CoGAPS.so Cogaps.o GapsParameters.o GapsResult.o GapsRunner.o GapsStatistics.o RcppExports.o test-runner.o atomic/Atom.o atomic/ConcurrentAtom.o atomic/AtomicDomain.o atomic/ConcurrentAtomicDomain.o atomic/ProposalQueue.o data_structures/HashSets.o data_structures/HybridMatrix.o data_structures/HybridVector.o data_structures/Matrix.o data_structures/SparseIterator.o data_structures/SparseMatrix.o data_structures/SparseVector.o data_structures/Vector.o file_parser/CharacterDelimitedParser.o file_parser/FileParser.o file_parser/MatrixElement.o file_parser/MtxParser.o gibbs_sampler/AlphaParameters.o gibbs_sampler/DenseNormalModel.o gibbs_sampler/SparseNormalModel.o math/Math.o math/MatrixMath.o math/Random.o math/VectorMath.o -L/home/biocbuild/bbs-3.17-bioc/R/lib -lR
installing to /home/biocbuild/bbs-3.17-bioc/R/site-library/00LOCK-CoGAPS/00new/CoGAPS/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (CoGAPS)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">CoGAPS.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(CoGAPS)
&gt; 
&gt; test_check(&quot;CoGAPS&quot;)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.mtx (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.mtx (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running genome-wide CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.mtx (1363 genes and 9 samples)
    worker 1 is starting!
    worker 2 is starting!
    worker 3 is starting!
    worker 4 is starting!
    worker 1 is finished! Time: 00:00:00
    worker 2 is finished! Time: 00:00:00
    worker 3 is finished! Time: 00:00:00
    worker 4 is finished! Time: 00:00:00
    worker 1 is starting!
    worker 2 is starting!
    worker 1 is finished! Time: 00:00:00
    worker 3 is starting!
    worker 4 is starting!
    worker 2 is finished! Time: 00:00:00
    worker 3 is finished! Time: 00:00:00
    worker 4 is finished! Time: 00:00:00

This is CoGAPS version 3.19.0 
Running genome-wide CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.mtx (1363 genes and 9 samples)
    worker 1 is starting!
    worker 2 is starting!
    worker 3 is starting!
    worker 4 is starting!
    worker 1 is finished! Time: 00:00:00
    worker 3 is finished! Time: 00:00:00
    worker 2 is finished! Time: 00:00:00
    worker 4 is finished! Time: 00:00:00
    worker 1 is starting!
    worker 2 is starting!
    worker 3 is starting!
    worker 4 is starting!
    worker 1 is finished! Time: 00:00:00
    worker 4 is finished! Time: 00:00:00
    worker 2 is finished! Time: 00:00:00
    worker 3 is finished! Time: 00:00:00

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.mtx (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.mtx (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.mtx (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.mtx (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.mtx (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.mtx (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on GIST.matrix (1363 genes and 9 samples)
    worker 1 is starting!
    worker 1 is finished! Time: 00:00:00

This is CoGAPS version 3.19.0 
Running genome-wide CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.mtx (1363 genes and 9 samples)
    worker 1 is starting!
    worker 3 is starting!
    worker 2 is starting!
    worker 4 is starting!
    worker 3 is finished! Time: 00:00:00
    worker 1 is finished! Time: 00:00:00
    worker 2 is finished! Time: 00:00:00
    worker 4 is finished! Time: 00:00:00
    worker 1 is starting!
    worker 4 is starting!
    worker 2 is starting!
    worker 3 is starting!
    worker 4 is finished! Time: 00:00:00
    worker 3 is finished! Time: 00:00:00
    worker 1 is finished! Time: 00:00:00
    worker 2 is finished! Time: 00:00:00

This is CoGAPS version 3.19.0 
Running single-cell CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.mtx (9 genes and 1363 samples)
    worker 1 is starting!
    worker 2 is starting!
    worker 3 is starting!
    worker 4 is starting!
    worker 2 is finished! Time: 00:00:00
    worker 1 is finished! Time: 00:00:00
    worker 3 is finished! Time: 00:00:00
    worker 4 is finished! Time: 00:00:00
    worker 2 is starting!
    worker 1 is starting!
    worker 3 is starting!
    worker 2 is finished! Time: 00:00:00
    worker 4 is starting!
    worker 4 is finished! Time: 00:00:00
    worker 1 is finished! Time: 00:00:00
    worker 3 is finished! Time: 00:00:00

This is CoGAPS version 3.19.0 
Running genome-wide CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.mtx (1363 genes and 9 samples)
    worker 1 is starting!
    worker 2 is starting!
    worker 3 is starting!
    worker 4 is starting!
    worker 1 is finished! Time: 00:00:00
    worker 2 is finished! Time: 00:00:00
    worker 4 is finished! Time: 00:00:00
    worker 3 is finished! Time: 00:00:00
    worker 1 is starting!
    worker 2 is starting!
    worker 3 is starting!
    worker 1 is finished! Time: 00:00:00
    worker 4 is starting!
    worker 2 is finished! Time: 00:00:00
    worker 3 is finished! Time: 00:00:00
    worker 4 is finished! Time: 00:00:00

This is CoGAPS version 3.19.0 
Running single-cell CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.mtx (9 genes and 1363 samples)
    worker 2 is starting!
    worker 1 is starting!
    worker 3 is starting!
    worker 4 is starting!
    worker 2 is finished! Time: 00:00:00
    worker 1 is finished! Time: 00:00:00
    worker 4 is finished! Time: 00:00:00
    worker 3 is finished! Time: 00:00:00
    worker 1 is starting!
    worker 2 is starting!
    worker 3 is starting!
    worker 4 is starting!
    worker 4 is finished! Time: 00:00:00
    worker 1 is finished! Time: 00:00:00
    worker 3 is finished! Time: 00:00:00
    worker 2 is finished! Time: 00:00:00

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on testDataFrame (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on testMatrix (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.csv (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.tsv (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.mtx (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.gct (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on testDataFrame (9 genes and 1363 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on testMatrix (9 genes and 1363 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.csv (9 genes and 1363 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.tsv (9 genes and 1363 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.mtx (9 genes and 1363 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.gct (9 genes and 1363 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on testDataFrame (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on testDataFrame (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on testDataFrame (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on testDataFrame (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running genome-wide CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.tsv (1363 genes and 9 samples)
    worker 2 is starting!
    worker 1 is starting!
    worker 3 is starting!
    worker 4 is starting!
    worker 2 is finished! Time: 00:00:00
    worker 3 is finished! Time: 00:00:00
    worker 1 is finished! Time: 00:00:00
    worker 4 is finished! Time: 00:00:00
    worker 1 is starting!
    worker 1 is finished! Time: 00:00:00
    worker 2 is starting!
    worker 3 is starting!
    worker 4 is starting!
    worker 2 is finished! Time: 00:00:00
    worker 3 is finished! Time: 00:00:00
    worker 4 is finished! Time: 00:00:00

This is CoGAPS version 3.19.0 
Running genome-wide CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.tsv (1363 genes and 9 samples)
    worker 2 is starting!
    worker 1 is starting!
    worker 3 is starting!
    worker 4 is starting!
    worker 2 is finished! Time: 00:00:00
    worker 1 is finished! Time: 00:00:00
    worker 3 is finished! Time: 00:00:00
    worker 4 is finished! Time: 00:00:00
    worker 1 is starting!
    worker 2 is starting!
    worker 1 is finished! Time: 00:00:00
    worker 3 is starting!
    worker 4 is starting!
    worker 2 is finished! Time: 00:00:00
    worker 4 is finished! Time: 00:00:00
    worker 3 is finished! Time: 00:00:00

This is CoGAPS version 3.19.0 
Running single-cell CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.csv (9 genes and 1363 samples)
    worker 2 is starting!
    worker 1 is starting!
    worker 3 is starting!
    worker 4 is starting!
    worker 2 is finished! Time: 00:00:00
    worker 1 is finished! Time: 00:00:00
    worker 3 is finished! Time: 00:00:00
    worker 4 is finished! Time: 00:00:00
    worker 2 is starting!
    worker 1 is starting!
    worker 3 is starting!
    worker 2 is finished! Time: 00:00:00
    worker 4 is starting!
    worker 3 is finished! Time: 00:00:00
    worker 1 is finished! Time: 00:00:00
    worker 4 is finished! Time: 00:00:00

This is CoGAPS version 3.19.0 
Running single-cell CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.mtx (9 genes and 1363 samples)
    worker 1 is starting!
    worker 2 is starting!
    worker 3 is starting!
    worker 1 is finished! Time: 00:00:00
    worker 4 is starting!
    worker 2 is finished! Time: 00:00:00
    worker 3 is finished! Time: 00:00:00
    worker 4 is finished! Time: 00:00:00
    worker 1 is starting!
    worker 2 is starting!
    worker 1 is finished! Time: 00:00:00
    worker 4 is starting!
    worker 3 is starting!
    worker 2 is finished! Time: 00:00:00
    worker 4 is finished! Time: 00:00:00
    worker 3 is finished! Time: 00:00:00

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on testMatrix (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.csv (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.csv (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running Standard CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.csv (1363 genes and 9 samples)

This is CoGAPS version 3.19.0 
Running genome-wide CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.mtx (1363 genes and 9 samples) with parameters:

-- Standard Parameters --
nPatterns            3 
nIterations          175 
seed                 42 
sparseOptimization   TRUE 
distributed          genome-wide 

-- Sparsity Parameters --
alpha          0.01 
maxGibbsMass   100 

-- Distributed CoGAPS Parameters -- 
nSets          5 
cut            4 
minNS          3 
maxNS          8 

Creating subsets...using provided indexed subsets
set sizes (min, mean, max): (200, 200, 200)
Running Final Stage...

Data Model: Sparse, Normal
Sampler Type: Sequential
Loading Data...Done! (00:00:00)
    worker 1 is starting!
    worker 3 is starting!
-- Equilibration Phase --
-- Sampling Phase --
    worker 1 is finished! Time: 00:00:00
    worker 5 is starting!
    worker 2 is starting!
    worker 2 is finished! Time: 00:00:00
    worker 5 is finished! Time: 00:00:00
    worker 3 is finished! Time: 00:00:00
    worker 4 is starting!
    worker 4 is finished! Time: 00:00:00

This is CoGAPS version 3.19.0 
Running genome-wide CoGAPS on /home/biocbuild/bbs-3.17-bioc/R/site-library/CoGAPS/extdata/GIST.mtx (1363 genes and 9 samples) with parameters:

-- Standard Parameters --
nPatterns            3 
nIterations          175 
seed                 42 
sparseOptimization   TRUE 
distributed          genome-wide 

-- Sparsity Parameters --
alpha          0.01 
maxGibbsMass   100 

-- Distributed CoGAPS Parameters -- 
nSets          5 
cut            4 
minNS          3 
maxNS          8 

Creating subsets...using provided indexed subsets
set sizes (min, mean, max): (200, 200, 200)
Running Final Stage...

Data Model: Sparse, Normal
Sampler Type: Sequential
Loading Data...Done! (00:00:00)
    worker 1 is starting!
    worker 3 is starting!
    worker 5 is starting!
    worker 3 is finished! Time: 00:00:00
    worker 5 is finished! Time: 00:00:00
    worker 4 is starting!
-- Equilibration Phase --
-- Sampling Phase --
    worker 1 is finished! Time: 00:00:00
    worker 4 is finished! Time: 00:00:00
    worker 2 is starting!
    worker 2 is finished! Time: 00:00:00
[ FAIL 0 | WARN 9 | SKIP 3 | PASS 58 ]

══ Skipped tests ═══════════════════════════════════════════════════════════════
• empty test (3)

[ FAIL 0 | WARN 9 | SKIP 3 | PASS 58 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
 44.117   7.086  45.131 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
</BODY>
</HTML>
