<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for fastreeR on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><B>[<A href="..#F">F</A>]</B></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-16 06:06:54 -0000 (Tue, 16 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4787</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for fastreeR on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the fastreeR package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/fastreeR.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>674</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">fastreeR</A>&nbsp;1.5.1</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/fastreeR">landing page</A>)</SPAN><BR>Anestis Gkanogiannis<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-13&nbsp;23:12:22&nbsp;-0000&nbsp;(Sat,&nbsp;13&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/fastreeR</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">44fe770</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-30&nbsp;17:59:15&nbsp;-0000&nbsp;(Sun,&nbsp;30&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: fastreeR</TD></TR>
<TR><TD><B>Version</B>: 1.5.1</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:fastreeR.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings fastreeR_1.5.1.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-15 03:16:10 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-15 03:17:41 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 91.2 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: fastreeR.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:fastreeR.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings fastreeR_1.5.1.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/fastreeR.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘fastreeR/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘fastreeR’ version ‘1.5.1’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘fastreeR’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking startup messages can be suppressed ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘spelling.R’
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘fastreeR_vignette.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: OK

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">fastreeR.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD INSTALL fastreeR
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0-bin/site-library’
* installing *source* package ‘fastreeR’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (fastreeR)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">fastreeR.Rcheck/tests/spelling.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; if(requireNamespace(&#x27;spelling&#x27;, quietly = TRUE))
+     spelling::spell_check_test(vignettes = TRUE, error = FALSE,
+                                                         skip_on_cran = TRUE)
NULL
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.200   0.036   0.216 
</PRE>
</DIV><P><SPAN class="filename">fastreeR.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(fastreeR)
&gt; 
&gt; test_check(&quot;fastreeR&quot;)
VCF2DIST : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:38 VCFManager: START READ
16777216
2023/05/15 03:16:38 VCFManager: END READ
Processed variants : 	       4
VCF2TREE : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:38 VCFManager: START READ
2023/05/15 03:16:38 VCFManager: END READ
Processed variants : 	       4
2023/05/15 03:16:38 Distances=3x3
hierarchical method=Complete
3 4
2023/05/15 03:16:38 Distances=3x3
hierarchical method=Complete
 ..cutHeight not given, setting it to 1.74  ===&gt;  99% of the (truncated) height range in dendro.
 ..done.
2023/05/15 03:16:39 Clusters=2

S2	0	1
S1	1	2
S3	1	2



3 4
2023/05/15 03:16:39 Distances=3x3
hierarchical method=Complete
 ..cutHeight not given, setting it to 1.74  ===&gt;  99% of the (truncated) height range in dendro.
 ..done.
2023/05/15 03:16:39 Clusters=2

S2	0	1
S1	1	2
S3	1	2



3 4
2023/05/15 03:16:39 Distances=3x3
hierarchical method=Complete
 ..cutHeight not given, setting it to 1.74  ===&gt;  99% of the (truncated) height range in dendro.
 ..done.
2023/05/15 03:16:39 Clusters=2

S2	0	1
S1	1	2
S3	1	2



3 4
2023/05/15 03:16:39 Distances=3x3
hierarchical method=Complete
 ..cutHeight not given, setting it to 1.74  ===&gt;  99% of the (truncated) height range in dendro.
 ..done.
2023/05/15 03:16:39 Clusters=2

S2	0	1
S1	1	2
S3	1	2



100 1000
2023/05/15 03:16:39 Distances=100x100
hierarchical method=Complete
 ..cutHeight not given, setting it to 0.0794  ===&gt;  99% of the (truncated) height range in dendro.
 ..done.
2023/05/15 03:16:39 Clusters=36

HG00126	1	5
HG00149	1	5
HG00182	1	5
HG00233	1	5
HG00262	1	5
HG00100	2	5
HG00101	2	5
HG00106	2	5
HG00107	2	5
HG00125	2	5
HG00096	3	5
HG00128	3	5
HG00154	3	5
HG00173	3	5
HG00242	3	5
HG00108	4	4
HG00129	4	4
HG00232	4	4
HG00253	4	4
HG00110	5	4
HG00121	5	4
HG00123	5	4
HG00256	5	4
HG00185	6	4
HG00189	6	4
HG00258	6	4
HG00261	6	4
HG00114	7	3
HG00133	7	3
HG00234	7	3
HG00120	8	3
HG00136	8	3
HG00236	8	3
HG00141	9	3
HG00187	9	3
HG00254	9	3
HG00142	10	3
HG00146	10	3
HG00160	10	3
HG00118	11	3
HG00138	11	3
HG00157	11	3
HG00158	12	3
HG00177	12	3
HG00178	12	3
HG00115	13	3
HG00130	13	3
HG00155	13	3
HG00111	14	3
HG00137	14	3
HG00150	14	3
HG00112	15	3
HG00132	15	3
HG00148	15	3
HG00140	16	3
HG00151	16	3
HG00237	16	3
HG00231	17	3
HG00238	17	3
HG00243	17	3
HG00180	18	3
HG00252	18	3
HG00260	18	3
HG00116	19	3
HG00122	19	3
HG00139	19	3
HG00097	20	2
HG00102	20	2
HG00145	21	2
HG00244	21	2
HG00113	22	2
HG00259	22	2
HG00099	23	2
HG00174	23	2
HG00171	24	2
HG00255	24	2
HG00176	25	2
HG00235	25	2
HG00179	26	2
HG00250	26	2
HG00103	27	2
HG00257	27	2
HG00188	28	2
HG00245	28	2
HG00127	29	2
HG00186	29	2
HG00105	30	2
HG00183	30	2
HG00119	31	2
HG00190	31	2
HG00131	32	2
HG00159	32	2
HG00109	33	2
HG00246	33	2
HG00117	34	2
HG00240	34	2
HG00143	35	2
HG00251	35	2
HG00181	36	2
HG00239	36	2



VCF2DIST : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:39 VCFManager: START READ
2023/05/15 03:16:39 VCFManager: END READ
Processed variants : 	       4
3 0
2023/05/15 03:16:39 Distances=3x3
hierarchical method=Complete
 ..cutHeight not given, setting it to 1.74  ===&gt;  99% of the (truncated) height range in dendro.
 ..done.
2023/05/15 03:16:39 Clusters=2

S2	0	1
S1	1	2
S3	1	2



3 4
2023/05/15 03:16:39 Distances=3x3
hierarchical method=Complete
100 1000
2023/05/15 03:16:39 Distances=100x100
hierarchical method=Complete
VCF2DIST : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:39 VCFManager: START READ
2023/05/15 03:16:39 VCFManager: END READ
Processed variants : 	       4
3 0
2023/05/15 03:16:39 Distances=3x3
hierarchical method=Complete
FASTA2DIST : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:40 FastaManager: START READ
CHUNK_SIZE=16777216
2023/05/15 03:16:40 FastaManager: Chunk with 7 lines.
2023/05/15 03:16:40 FastaManager: END READ
2023/05/15 03:16:40 FastaManager: FASTA
2	S1
2	S2
3	S3
FASTA2DIST : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:40 FastaManager: START READ
2023/05/15 03:16:40 FastaManager: Chunk with 7 lines.
2023/05/15 03:16:40 FastaManager: END READ
2023/05/15 03:16:40 FastaManager: FASTA
2	S1
2	S2
3	S3
FASTA2DIST : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:40 FastaManager: START READ
2023/05/15 03:16:40 FastaManager: Chunk with 7 lines.
2023/05/15 03:16:40 FastaManager: END READ
2023/05/15 03:16:40 FastaManager: FASTA
2	S1
2	S2
3	S3
FASTA2DIST : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:40 FastaManager: START READ
2023/05/15 03:16:40 FastaManager: Chunk with 7 lines.
1	S1
2023/05/15 03:16:40 FastaManager: END READ
2	S2
3	S3
2023/05/15 03:16:40 FastaManager: FASTA
FASTA2DIST : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:40 FastaManager: START READ
2023/05/15 03:16:40 FastaManager: Chunk with 96 lines.
2023/05/15 03:16:40 FastaManager: END READ
2023/05/15 03:16:40 FastaManager: FASTA
2	contig-202000258
3	contig-73001792
4	contig-13001800
4	contig-198000031
6	contig-366001068
7	contig-295001825
8	contig-100000567
8	contig-237001826
10	contig-15000592
11	contig-168000048
12	contig-620000331
13	contig-573001861
14	contig-30001369
15	contig-12001376
16	contig-201001879
17	contig-11000629
18	contig-25000365
19	contig-364
20	contig-127000661
20	contig-136001638
22	contig-16001175
22	contig-20000917
24	contig-448001924
25	contig-171000141
26	contig-5001683
27	contig-7001201
28	contig-5000427
29	contig-19001684
30	contig-218001938
31	contig-6001939
32	contig-60001206
33	contig-65000152
34	contig-263001217
35	contig-600000955
36	contig-261001227
37	contig-47000966
38	contig-16001959
39	contig-623000970
40	contig-98000731
41	contig-52000465
42	contig-118001491
43	contig-48000188
44	contig-53001003
45	contig-230000764
46	contig-40000768
47	contig-105000229
48	contig-19001776
48	contig-186000247
2023/05/15 03:16:41 CalculateD2ChildTask(51): 	50
 ..done.
2023/05/15 03:16:41 Clusters=2

S2	0	1
S1	1	2
S3	1	2



 ..cutHeight not given, setting it to 1.74  ===&gt;  99% of the (truncated) height range in dendro.
 ..done.
2023/05/15 03:16:41 Clusters=2

S2	0	1
S1	1	2
S3	1	2



 ..cutHeight not given, setting it to 1.74  ===&gt;  99% of the (truncated) height range in dendro.
 ..done.
2023/05/15 03:16:41 Clusters=2

S2	0	1
S1	1	2
S3	1	2



VCF2DIST : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:41 VCFManager: START READ
2023/05/15 03:16:41 VCFManager: END READ
Processed variants : 	       4
3 0
2023/05/15 03:16:41 Distances=3x3
hierarchical method=Complete
 ..cutHeight not given, setting it to 1.74  ===&gt;  99% of the (truncated) height range in dendro.
 ..done.
2023/05/15 03:16:41 Clusters=2

S2	0	1
S1	1	2
S3	1	2



VCF2DIST : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:41 VCFManager: START READ
2023/05/15 03:16:41 VCFManager: END READ
Processed variants : 	       4
3 0
2023/05/15 03:16:41 Distances=3x3
hierarchical method=Complete
 ..cutHeight not given, setting it to 1.74  ===&gt;  99% of the (truncated) height range in dendro.
 ..done.
2023/05/15 03:16:41 Clusters=2

S2	0	1
S1	1	2
S3	1	2



VCF2DIST : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:41 VCFManager: START READ
2023/05/15 03:16:41 VCFManager: END READ
Processed variants : 	       4
3 0
2023/05/15 03:16:41 Distances=3x3
hierarchical method=Complete
 ..cutHeight not given, setting it to 1.74  ===&gt;  99% of the (truncated) height range in dendro.
 ..done.
2023/05/15 03:16:41 Clusters=2

S2	0	1
S1	1	2
S3	1	2



VCF2DIST : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:41 VCFManager: START READ
2023/05/15 03:16:41 VCFManager: END READ
Processed variants : 	       4
3 0
2023/05/15 03:16:41 Distances=3x3
hierarchical method=Complete
 ..cutHeight not given, setting it to 1.74  ===&gt;  99% of the (truncated) height range in dendro.
 ..done.
2023/05/15 03:16:42 Clusters=2

S2	0	1
S1	1	2
S3	1	2



VCF2DIST : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:42 VCFManager: START READ
2023/05/15 03:16:42 VCFManager: END READ
Processed variants : 	    1000
2023/05/15 03:16:42 CalculateDistancesChildTask(88): 	50
2023/05/15 03:16:42 CalculateDistancesChildTask(53): 	100
100 0
2023/05/15 03:16:42 Distances=100x100
hierarchical method=Complete
 ..cutHeight not given, setting it to 0.0794  ===&gt;  99% of the (truncated) height range in dendro.
 ..done.
2023/05/15 03:16:42 Clusters=36

HG00126	1	5
HG00149	1	5
HG00182	1	5
HG00233	1	5
HG00262	1	5
HG00100	2	5
HG00101	2	5
HG00106	2	5
HG00107	2	5
HG00125	2	5
HG00096	3	5
HG00128	3	5
HG00154	3	5
HG00173	3	5
HG00242	3	5
HG00108	4	4
HG00129	4	4
HG00232	4	4
HG00253	4	4
HG00110	5	4
HG00121	5	4
HG00123	5	4
HG00256	5	4
HG00185	6	4
HG00189	6	4
HG00258	6	4
HG00261	6	4
HG00114	7	3
HG00133	7	3
HG00234	7	3
HG00120	8	3
HG00136	8	3
HG00236	8	3
HG00141	9	3
HG00187	9	3
HG00254	9	3
HG00142	10	3
HG00146	10	3
HG00160	10	3
HG00118	11	3
HG00138	11	3
HG00157	11	3
HG00158	12	3
HG00177	12	3
HG00178	12	3
HG00115	13	3
HG00130	13	3
HG00155	13	3
HG00111	14	3
HG00137	14	3
HG00150	14	3
HG00112	15	3
HG00132	15	3
HG00148	15	3
HG00140	16	3
HG00151	16	3
HG00237	16	3
HG00231	17	3
HG00238	17	3
HG00243	17	3
HG00180	18	3
HG00252	18	3
HG00260	18	3
HG00116	19	3
HG00122	19	3
HG00139	19	3
HG00097	20	2
HG00102	20	2
HG00145	21	2
HG00244	21	2
HG00113	22	2
HG00259	22	2
HG00099	23	2
HG00174	23	2
HG00171	24	2
HG00255	24	2
HG00176	25	2
HG00235	25	2
HG00179	26	2
HG00250	26	2
HG00103	27	2
HG00257	27	2
HG00188	28	2
HG00245	28	2
HG00127	29	2
HG00186	29	2
HG00105	30	2
HG00183	30	2
HG00119	31	2
HG00190	31	2
HG00131	32	2
HG00159	32	2
HG00109	33	2
HG00246	33	2
HG00117	34	2
HG00240	34	2
HG00143	35	2
HG00251	35	2
HG00181	36	2
HG00239	36	2



VCF2DIST : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:42 VCFManager: START READ
2023/05/15 03:16:42 VCFManager: END READ
Processed variants : 	       4
VCF2DIST : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:42 VCFManager: START READ
2023/05/15 03:16:42 VCFManager: END READ
Processed variants : 	       4
VCF2DIST : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:42 VCFManager: START READ
2023/05/15 03:16:42 VCFManager: END READ
Processed variants : 	       4
VCF2DIST : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:42 VCFManager: START READ
2023/05/15 03:16:42 VCFManager: END READ
Processed variants : 	       4
VCF2DIST : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:42 VCFManager: START READ
2023/05/15 03:16:42 VCFManager: END READ
Processed variants : 	    1000
2023/05/15 03:16:42 CalculateDistancesChildTask(36): 	50
2023/05/15 03:16:42 CalculateDistancesChildTask(64): 	100
VCF2ISTATS : Sun May 14 10:47:28 UTC 2023
Num of Ind = 3
VCF2ISTATS : Sun May 14 10:47:28 UTC 2023
Num of Ind = 3
VCF2ISTATS : Sun May 14 10:47:28 UTC 2023
Num of Ind = 3
VCF2TREE : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:43 VCFManager: START READ
2023/05/15 03:16:43 VCFManager: END READ
Processed variants : 	       4
2023/05/15 03:16:43 Distances=3x3
hierarchical method=Complete
VCF2TREE : Sun May 14 10:47:28 UTC 2023
cpus=32
using=2
2023/05/15 03:16:43 VCFManager: START READ
2023/05/15 03:16:43 VCFManager: END READ
Processed variants : 	    1000
2023/05/15 03:16:43 CalculateDistancesChildTask(91): 	50
2023/05/15 03:16:43 CalculateDistancesChildTask(70): 	100
2023/05/15 03:16:43 Distances=100x100
hierarchical method=Complete
[ FAIL 0 | WARN 0 | SKIP 0 | PASS 100 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
 11.739   0.719   6.697 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">fastreeR.Rcheck/fastreeR-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>dist2clusters</TD><TD style="text-align: right;">1.050</TD><TD style="text-align: right;">0.057</TD><TD style="text-align: right;">0.812</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dist2tree</TD><TD style="text-align: right;">0.161</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.096</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fasta2dist</TD><TD style="text-align: right;">3.010</TD><TD style="text-align: right;">0.175</TD><TD style="text-align: right;">1.355</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>tree2clusters</TD><TD style="text-align: right;">0.206</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.125</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>vcf2clusters</TD><TD style="text-align: right;">2.237</TD><TD style="text-align: right;">0.111</TD><TD style="text-align: right;">0.511</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>vcf2dist</TD><TD style="text-align: right;">0.944</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">0.240</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>vcf2istats</TD><TD style="text-align: right;">0.593</TD><TD style="text-align: right;">0.059</TD><TD style="text-align: right;">0.335</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>vcf2tree</TD><TD style="text-align: right;">0.220</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.108</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
