<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for NanoMethViz on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><B>[<A href="../index.html#N">N</A>]</B></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-13 06:49:44 -0000 (Mon, 13 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4222</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for NanoMethViz on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the NanoMethViz package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/NanoMethViz.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1310</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">NanoMethViz</A>&nbsp;2.5.2</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/NanoMethViz">landing page</A>)</SPAN><BR>Shian Su<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-11&nbsp;01:58:29&nbsp;-0000&nbsp;(Sat,&nbsp;11&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/NanoMethViz</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">27ea9dc</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-02-09&nbsp;01:48:45&nbsp;-0000&nbsp;(Thu,&nbsp;09&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: NanoMethViz</TD></TR>
<TR><TD><B>Version</B>: 2.5.2</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:NanoMethViz.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings NanoMethViz_2.5.2.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-11 18:39:34 -0000 (Sat, 11 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-11 18:52:38 -0000 (Sat, 11 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 784.3 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: NanoMethViz.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:NanoMethViz.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings NanoMethViz_2.5.2.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/NanoMethViz.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘NanoMethViz/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘NanoMethViz’ version ‘2.5.2’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘NanoMethViz’ can be installed ... OK
* used C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                         user system elapsed
get_exons              53.140  7.124  67.330
get_exons_mus_musculus 18.684  0.395  19.082
get_exons_homo_sapiens 16.884  0.404  17.337
plot_gene_heatmap      14.142  0.247  14.394
plot_region_heatmap     9.870  0.059   9.930
plot_agg_regions        7.865  0.100   7.965
NanoMethResult-class    5.760  0.361   6.205
plot_grange_heatmap     4.993  0.032   5.025
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  `plot_region(...)` produced warnings.
  ── Failure (&#x27;test-plot_region.R:16&#x27;): Plotting region works ────────────────────
  `plot_region(...)` produced warnings.
  ── Failure (&#x27;test-plot_region.R:16&#x27;): Plotting region works ────────────────────
  `plot_region(...)` produced warnings.
  ── Failure (&#x27;test-plot_region_heatmap.R:6&#x27;): Plotting region methylation heatmap works ──
  `p &lt;- plot_region_heatmap(nmr, &quot;chr7&quot;, 6703892, 6730431)` produced warnings.
  ── Failure (&#x27;test-plot_region_heatmap.R:9&#x27;): Plotting region methylation heatmap works ──
  `p &lt;- plot_region_heatmap(nmr, &quot;chr7&quot;, 6703892, 6730431, pos_style = &quot;compact&quot;)` produced warnings.
  ── Failure (&#x27;test-plot_region_heatmap.R:25&#x27;): Plotting region methylation works without exons ──
  `p &lt;- plot_region_heatmap(nmr, &quot;chr7&quot;, 6703892, 6730431)` produced warnings.
  
  [ FAIL 13 | WARN 0 | SKIP 0 | PASS 80 ]
  Error: Test failures
  Execution halted
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘DimensionalityReduction.Rmd’ using ‘UTF-8’... OK
  ‘ExonAnnotations.Rmd’ using ‘UTF-8’... OK
  ‘ImportingExportingData.Rmd’ using ‘UTF-8’... OK
  ‘Introduction.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... ERROR
Error(s) in re-building vignettes:
  ...
--- re-building ‘DimensionalityReduction.Rmd’ using rmarkdown
--- finished re-building ‘DimensionalityReduction.Rmd’

--- re-building ‘ExonAnnotations.Rmd’ using rmarkdown
--- finished re-building ‘ExonAnnotations.Rmd’

--- re-building ‘ImportingExportingData.Rmd’ using rmarkdown
--- finished re-building ‘ImportingExportingData.Rmd’

--- re-building ‘Introduction.Rmd’ using rmarkdown
Killed
SUMMARY: processing the following file failed:
  ‘Introduction.Rmd’

Error: Vignette re-building failed.
Execution halted

* checking PDF version of manual ... OK
* DONE

Status: 2 ERRORs, 1 NOTE
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/NanoMethViz.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">NanoMethViz.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL NanoMethViz
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘NanoMethViz’ ...
** using staged installation
** libs
using C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
using C++11
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c convert_methy_to_dss.cpp -o convert_methy_to_dss.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c utils.cpp -o utils.o
g++ -std=gnu++11 -shared -L/home/biocbuild/bbs-3.17-bioc/R/lib -L/usr/local/lib -o NanoMethViz.so RcppExports.o convert_methy_to_dss.o utils.o -L/home/biocbuild/bbs-3.17-bioc/R/lib -lR
installing to /home/biocbuild/bbs-3.17-bioc/R/library/00LOCK-NanoMethViz/00new/NanoMethViz/libs
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (NanoMethViz)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">NanoMethViz.Rcheck/tests/testthat.Rout.fail</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(NanoMethViz)
Loading required package: ggplot2
&gt; 
&gt; test_check(&quot;NanoMethViz&quot;)
Starting 2 test processes
[ FAIL 13 | WARN 0 | SKIP 0 | PASS 80 ]

══ Failed tests ════════════════════════════════════════════════════════════════
── Failure (&#x27;test-plot_gene_heatmap.R:6&#x27;): Plotting gene methylation heatmap works ──
`p1 &lt;- plot_gene_heatmap(nmr, &quot;Peg3&quot;)` produced warnings.
── Failure (&#x27;test-plot_granges.R:8&#x27;): Plotting GRanges works ───────────────────
`plot_grange_heatmap(nmr, GenomicRanges::GRanges(&quot;chr7:6703892-6730431&quot;))` produced warnings.
── Failure (&#x27;test-plot_gene.R:16&#x27;): Plotting gene works ────────────────────────
`plot_gene(nmr, &quot;Peg3&quot;, heatmap = params$heatmap[i], spaghetti = params$spaghetti[i])` produced warnings.
── Failure (&#x27;test-plot_gene.R:16&#x27;): Plotting gene works ────────────────────────
`plot_gene(nmr, &quot;Peg3&quot;, heatmap = params$heatmap[i], spaghetti = params$spaghetti[i])` produced warnings.
── Failure (&#x27;test-plot_gene.R:16&#x27;): Plotting gene works ────────────────────────
`plot_gene(nmr, &quot;Peg3&quot;, heatmap = params$heatmap[i], spaghetti = params$spaghetti[i])` produced warnings.
── Failure (&#x27;test-plot_gene.R:16&#x27;): Plotting gene works ────────────────────────
`plot_gene(nmr, &quot;Peg3&quot;, heatmap = params$heatmap[i], spaghetti = params$spaghetti[i])` produced warnings.
── Failure (&#x27;test-plot_region.R:16&#x27;): Plotting region works ────────────────────
`plot_region(...)` produced warnings.
── Failure (&#x27;test-plot_region.R:16&#x27;): Plotting region works ────────────────────
`plot_region(...)` produced warnings.
── Failure (&#x27;test-plot_region.R:16&#x27;): Plotting region works ────────────────────
`plot_region(...)` produced warnings.
── Failure (&#x27;test-plot_region.R:16&#x27;): Plotting region works ────────────────────
`plot_region(...)` produced warnings.
── Failure (&#x27;test-plot_region_heatmap.R:6&#x27;): Plotting region methylation heatmap works ──
`p &lt;- plot_region_heatmap(nmr, &quot;chr7&quot;, 6703892, 6730431)` produced warnings.
── Failure (&#x27;test-plot_region_heatmap.R:9&#x27;): Plotting region methylation heatmap works ──
`p &lt;- plot_region_heatmap(nmr, &quot;chr7&quot;, 6703892, 6730431, pos_style = &quot;compact&quot;)` produced warnings.
── Failure (&#x27;test-plot_region_heatmap.R:25&#x27;): Plotting region methylation works without exons ──
`p &lt;- plot_region_heatmap(nmr, &quot;chr7&quot;, 6703892, 6730431)` produced warnings.

[ FAIL 13 | WARN 0 | SKIP 0 | PASS 80 ]
Error: Test failures
Execution halted
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">NanoMethViz.Rcheck/NanoMethViz-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>NanoMethResult-class</TD><TD style="text-align: right;">5.760</TD><TD style="text-align: right;">0.361</TD><TD style="text-align: right;">6.205</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bsseq_to_edger</TD><TD style="text-align: right;">0.918</TD><TD style="text-align: right;">0.070</TD><TD style="text-align: right;">0.967</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bsseq_to_log_methy_ratio</TD><TD style="text-align: right;">1.463</TD><TD style="text-align: right;">0.089</TD><TD style="text-align: right;">1.460</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cluster_regions</TD><TD style="text-align: right;">4.414</TD><TD style="text-align: right;">0.031</TD><TD style="text-align: right;">4.450</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>create_tabix_file</TD><TD style="text-align: right;">0.235</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.243</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>exons</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>exons_to_genes</TD><TD style="text-align: right;">0.480</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.484</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filter_methy</TD><TD style="text-align: right;">4.133</TD><TD style="text-align: right;">0.108</TD><TD style="text-align: right;">4.239</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_example_exons_mus_musculus</TD><TD style="text-align: right;">0.386</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.395</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_exons</TD><TD style="text-align: right;">53.140</TD><TD style="text-align: right;"> 7.124</TD><TD style="text-align: right;">67.330</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_exons_homo_sapiens</TD><TD style="text-align: right;">16.884</TD><TD style="text-align: right;"> 0.404</TD><TD style="text-align: right;">17.337</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_exons_mus_musculus</TD><TD style="text-align: right;">18.684</TD><TD style="text-align: right;"> 0.395</TD><TD style="text-align: right;">19.082</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>load_example_nanomethresult</TD><TD style="text-align: right;">0.367</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.376</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>methy</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>methy_col_names</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>methy_to_bsseq</TD><TD style="text-align: right;">1.314</TD><TD style="text-align: right;">0.070</TD><TD style="text-align: right;">1.273</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>methy_to_edger</TD><TD style="text-align: right;">1.486</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">1.366</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_agg_genes</TD><TD style="text-align: right;">2.536</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">2.562</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_agg_regions</TD><TD style="text-align: right;">7.865</TD><TD style="text-align: right;">0.100</TD><TD style="text-align: right;">7.965</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_gene</TD><TD style="text-align: right;">3.309</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">3.337</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_gene_heatmap</TD><TD style="text-align: right;">14.142</TD><TD style="text-align: right;"> 0.247</TD><TD style="text-align: right;">14.394</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_grange</TD><TD style="text-align: right;">1.806</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">1.826</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_grange_heatmap</TD><TD style="text-align: right;">4.993</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">5.025</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_mds</TD><TD style="text-align: right;">2.017</TD><TD style="text-align: right;">0.050</TD><TD style="text-align: right;">1.844</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_pca</TD><TD style="text-align: right;">2.258</TD><TD style="text-align: right;">0.068</TD><TD style="text-align: right;">2.060</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_region</TD><TD style="text-align: right;">2.951</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">2.981</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_region_heatmap</TD><TD style="text-align: right;">9.870</TD><TD style="text-align: right;">0.059</TD><TD style="text-align: right;">9.930</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>query_methy</TD><TD style="text-align: right;">0.592</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.604</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>region_methy_stats</TD><TD style="text-align: right;">1.590</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">1.606</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>samples</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
