<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for annmap on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><B>[<A href="../index.html#A">A</A>]</B></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-08 01:14:50 -0000 (Wed, 08 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4164</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for annmap on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the annmap package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/annmap.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>53</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">annmap</A>&nbsp;1.41.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/annmap">landing page</A>)</SPAN><BR>Chris Wirth<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-06&nbsp;00:12:45&nbsp;-0000&nbsp;(Mon,&nbsp;06&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/annmap</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">5d59609</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:07:03&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: annmap</TD></TR>
<TR><TD><B>Version</B>: 1.41.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:annmap.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings annmap_1.41.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-06 17:12:06 -0000 (Mon, 06 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-06 17:15:22 -0000 (Mon, 06 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 195.2 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: annmap.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 1</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:annmap.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings annmap_1.41.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/annmap.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘annmap/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘annmap’ version ‘1.41.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .BBSoptions
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘annmap’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... NOTE
Malformed Title field: should not end in a period.
Malformed Description field: should contain one or more complete sentences.
* checking top-level files ... NOTE
File
  LICENSE
is not mentioned in the DESCRIPTION file.
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
&#x27;library&#x27; or &#x27;require&#x27; call to ‘rjson’ in package code.
  Please use :: or requireNamespace() instead.
  See section &#x27;Suggested packages&#x27; in the &#x27;Writing R Extensions&#x27; manual.
There are ::: calls to the package&#x27;s namespace in its code. A package
  almost never needs to use ::: for its own objects:
  ‘.make.hash’
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
.get.stats.cache: no visible global function definition for
  ‘flush.console’
.layout.genes: no visible global function definition for ‘strwidth’
.load.and.parse: no visible global function definition for ‘fromJSON’
.read.databases: no visible global function definition for ‘read.table’
.single.transcript.coords.to.genome: no visible global function
  definition for ‘tail’
.xmc.connect: no visible global function definition for ‘menu’
.xmcws.connect: no visible global function definition for ‘fromJSON’
.xmcws.connect: no visible global function definition for ‘menu’
annmapAddConnection: no visible global function definition for
  ‘write.table’
annmapConnect: no visible global function definition for
  ‘install.packages’
arrayType: no visible global function definition for ‘menu’
convertBamToRle: no visible global function definition for ‘RangedData’
convertBamToRle : &lt;anonymous&gt;: no visible global function definition
  for ‘Rle’
geneToGeneRegionTrack : generate: no visible global function definition
  for ‘GeneRegionTrack’
generateBridgeData: no visible global function definition for ‘rainbow’
genomicPlot : &lt;anonymous&gt;: no visible global function definition for
  ‘rgb’
genomicPlot: no visible global function definition for ‘rgb’
ngsTracePlotter : local.draw: no visible global function definition for
  ‘runValue’
ngsTracePlotter: no visible global function definition for ‘modifyList’
ngsTraceScale : &lt;anonymous&gt;: no visible global function definition for
  ‘runValue’
seqnameMapping: no visible global function definition for ‘seqlevels&lt;-’
seqnameMapping: no visible global function definition for ‘seqlevels’
Undefined global functions or variables:
  GeneRegionTrack RangedData Rle flush.console fromJSON
  install.packages menu modifyList rainbow read.table rgb runValue
  seqlevels seqlevels&lt;- strwidth tail write.table
Consider adding
  importFrom(&quot;grDevices&quot;, &quot;rainbow&quot;, &quot;rgb&quot;)
  importFrom(&quot;graphics&quot;, &quot;strwidth&quot;)
  importFrom(&quot;utils&quot;, &quot;flush.console&quot;, &quot;install.packages&quot;, &quot;menu&quot;,
             &quot;modifyList&quot;, &quot;read.table&quot;, &quot;tail&quot;, &quot;write.table&quot;)
to your NAMESPACE file.
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... WARNING
Missing link or links in documentation object &#x27;annmap.range.Rd&#x27;:
  ‘RangedData’

Missing link or links in documentation object &#x27;annmap.utils.Rd&#x27;:
  ‘RangedData’

Missing link or links in documentation object &#x27;plot.ngs.Rd&#x27;:
  ‘[IRanges]{Rle}’

See section &#x27;Cross-references&#x27; in the &#x27;Writing R Extensions&#x27; manual.
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in Makefiles ... OK
* checking for GNU extensions in Makefiles ... OK
* checking include directives in Makefiles ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘doRUnit.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘INSTALL.Rnw’... OK
  ‘annmap.Rnw’... OK
  ‘cookbook.Rnw’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 WARNING, 5 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/annmap.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">annmap.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL annmap
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘annmap’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
in method for ‘probesetInRange’ with signature ‘x=&quot;RangedData&quot;’: no definition for class “RangedData”
in method for ‘probeInRange’ with signature ‘x=&quot;RangedData&quot;’: no definition for class “RangedData”
in method for ‘proteinInRange’ with signature ‘x=&quot;RangedData&quot;’: no definition for class “RangedData”
in method for ‘domainInRange’ with signature ‘x=&quot;RangedData&quot;’: no definition for class “RangedData”
in method for ‘geneInRange’ with signature ‘x=&quot;RangedData&quot;’: no definition for class “RangedData”
in method for ‘transcriptInRange’ with signature ‘x=&quot;RangedData&quot;’: no definition for class “RangedData”
in method for ‘exonInRange’ with signature ‘x=&quot;RangedData&quot;’: no definition for class “RangedData”
in method for ‘estGeneInRange’ with signature ‘x=&quot;RangedData&quot;’: no definition for class “RangedData”
in method for ‘estTranscriptInRange’ with signature ‘x=&quot;RangedData&quot;’: no definition for class “RangedData”
in method for ‘estExonInRange’ with signature ‘x=&quot;RangedData&quot;’: no definition for class “RangedData”
in method for ‘predictionTranscriptInRange’ with signature ‘x=&quot;RangedData&quot;’: no definition for class “RangedData”
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (annmap)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">annmap.Rcheck/tests/doRUnit.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; if( require( &quot;RUnit&quot;, quietly=TRUE ) ) {
+   pkg &lt;- &quot;annmap&quot;
+  
+   if( Sys.getenv( &quot;RCMDCHECK&quot; ) == &quot;FALSE&quot; ) {
+     path &lt;- file.path( getwd(), &quot;..&quot;, &quot;inst&quot;, &quot;unitTests&quot; )
+   }
+   else {
+     path &lt;- system.file( package=pkg, &quot;unitTests&quot; )
+   }
+ 
+   cat( &quot;\nRunning unit tests\n&quot; )
+   print( list( pkg=pkg, getwd=getwd(), pathToUnitTests=path ) )
+   library( package=pkg, character.only=TRUE )
+ 
+   annmapClearCache()
+ 
+   # Run the old style RangedData for tim...
+   #annmapSetParam( oldstylekey=&#x27;P0fSpS&#x27; )
+   annmapSetParam( spacefn=if( annmap:::.usegranges() ) seqnames else space )
+   annmapSetParam( defaultclass=if( annmap:::.usegranges() ) &#x27;GRanges&#x27; else &#x27;RangedData&#x27; )
+ 
+   #Fail on warnings
+   options( warn=2 )
+ 
+   # Get the pattern (if there is one?)
+   patt = Sys.getenv( &quot;RUNITFILEPATTERN&quot; )
+   func = Sys.getenv( &#x27;RUNITFUNCPATTERN&#x27; )
+   if( is.null( patt ) || nchar( patt ) == 0 ) {
+     patt = &quot;^runit.+\\.[rR]$&quot;
+   }
+   else {
+     patt = paste( &quot;^runit\\.&quot;, patt, &quot;\\.[rR]$&quot;, sep=&quot;&quot; )
+   }
+   if( is.null( func ) || nchar( func ) == 0 ) {
+     func = &quot;^test.+&quot;
+   }
+   else {
+     func = paste( &quot;^test\\.&quot;, func, &quot;.+&quot;, sep=&quot;&quot; )
+   }
+   #testSuite &lt;- defineTestSuite( name=paste( pkg, &quot;unit testing&quot; ), testFileRegexp=patt, testFuncRegexp=func, dirs=path )
+   testSuite &lt;- defineTestSuite( name=paste( pkg, &quot;unit testing&quot; ), testFileRegexp=patt, testFuncRegexp=func, dirs=path, rngKind=&quot;Mersenne-Twister&quot; )
+   tests &lt;- runTestSuite( testSuite )
+  
+   pathReport &lt;- file.path( path, &quot;report&quot; )
+  
+   cat( &quot;------------------- UNIT TEST SUMMARY ---------------------\n\n&quot; )
+   printTextProtocol( tests, showDetails=FALSE )
+   printTextProtocol( tests, showDetails=FALSE, fileName=paste( pathReport, &quot;Summary.txt&quot;, sep=&quot;&quot; ) )
+   printTextProtocol( tests, showDetails=TRUE,  fileName=paste( pathReport, &quot;.txt&quot;, sep=&quot;&quot; ) )
+  
+   printHTMLProtocol( tests, fileName=paste( pathReport, &quot;.html&quot;, sep=&quot;&quot; ) )
+  
+   tmp &lt;- getErrors( tests )
+   if( tmp$nFail &gt; 0 | tmp$nErr &gt; 0 ) {
+     stop( paste( &quot;\n\nunit testing failed (#test failures: &quot;, tmp$nFail, &quot;, #R errors: &quot;,  tmp$nErr, &quot;)\n\n&quot;, sep=&quot;&quot; ) )
+   }
+ } else {
+   warning( &quot;cannot run unit tests -- package RUnit is not available&quot; )
+ }

Running unit tests
$pkg
[1] &quot;annmap&quot;

$getwd
[1] &quot;/home/biocbuild/bbs-3.17-bioc/meat/annmap.Rcheck/tests&quot;

$pathToUnitTests
[1] &quot;/home/biocbuild/bbs-3.17-bioc/R/library/annmap/unitTests&quot;

Loading required package: GenomicRanges
Loading required package: stats4
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
Using /home/biocbuild/.annmap as our configuration directory.


Executing test function test.template  ...  done successfully.



Executing test function test.all.queries  ... [1] &quot;\nSkipping allXXX tests, as not on a PICR machine&quot;
 done successfully.



Executing test function test.gotsomefunctions  ...  done successfully.



Executing test function test.cache  ...  done successfully.



Executing test function test.ANNMAP112  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.huisun.ANNMAP48  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.chaining  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27; so skipping this test.&quot;
 done successfully.



Executing test function test.addConnection  ... Using /home/biocbuild/.annmap as our configuration directory.
[1] &quot;Got conf.dir /home/biocbuild/.annmap&quot;
Connection a added. 
Connection b added. 
Connection a updated. 
[1] &quot;Resetting conf.dir to /home/biocbuild/.annmap&quot;
 done successfully.



Executing test function test.buildsql  ... Using /home/biocbuild/.annmap as our configuration directory.
[1] &quot;Got conf.dir /home/biocbuild/.annmap&quot;
[1] &quot;Resetting conf.dir to /home/biocbuild/.annmap&quot;
 done successfully.



Executing test function test.connection  ... Using /home/biocbuild/.annmap as our configuration directory.
[1] &quot;Got conf.dir /home/biocbuild/.annmap&quot;
[1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
[1] &quot;Resetting conf.dir to /home/biocbuild/.annmap&quot;
 done successfully.



Executing test function test.databases.txt  ... Using /home/biocbuild/.annmap as our configuration directory.
[1] &quot;Got conf.dir /home/biocbuild/.annmap&quot;
[1] &quot;Mocking annmapConnect()&quot;
[1] &quot;setting conf.dir to /home/biocbuild/bbs-3.17-bioc/R/library/annmap/unitTests/tabConfig&quot;
[1] &quot;setting conf.dir to /home/biocbuild/bbs-3.17-bioc/R/library/annmap/unitTests/commaConfig&quot;
[1] &quot;restoring connect method&quot;
[1] &quot;Resetting conf.dir to /home/biocbuild/.annmap&quot;
 done successfully.



Executing test function test.details.queries  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
[1] &quot;Cannot find datasource &#x27;mm-test&#x27;, so skipping these tests.&quot;
[1] &quot;Cannot find datasource &#x27;rn-test&#x27;, so skipping these tests.&quot;
[1] &quot;Cannot find datasource &#x27;pb-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.gotsomefunctions  ...  done successfully.



Executing test function test.filters  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.bug.ACBBUTIL33  ... [1] &quot;Cannot find datasource &#x27;mm-test&#x27;, so skipping this test.&quot;
 done successfully.



Executing test function test.bug.ANNMAP40  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping this test.&quot;
 done successfully.



Executing test function test.bug.ANNMAP44  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping this test.&quot;
 done successfully.



Executing test function test.gotsomefunctions  ...  done successfully.



Executing test function test.multi.queries  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.range.queries  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
[1] &quot;Cannot find datasource &#x27;mm-test&#x27;, so skipping these tests.&quot;
[1] &quot;Cannot find datasource &#x27;rn-test&#x27;, so skipping these tests.&quot;
[1] &quot;Cannot find datasource &#x27;pb-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.expr.query  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.gotsomefunctions  ...  done successfully.



Executing test function test.to.queries  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
[1] &quot;Cannot find datasource &#x27;mm-test&#x27;, so skipping these tests.&quot;
[1] &quot;Cannot find datasource &#x27;rn-test&#x27;, so skipping these tests.&quot;
[1] &quot;Cannot find datasource &#x27;pb-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.transcript.to.translatedprobes  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.ANNMAP.109  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.ANNMAP.97  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.ANNMAP.98  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.array.type  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.make.params  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.needs.array  ...  done successfully.



Executing test function test.rangeapply  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.reality  ...  done successfully.



Executing test function test.seqnames  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.set.get.params  ...  done successfully.



Executing test function test.ANNMAP118  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.michal.utr.bug  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.michal.utr.bug.three  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.michal.utr.bug.two  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.utr  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
 done successfully.



Executing test function test.yaoyong.utr.bug  ... [1] &quot;Cannot find datasource &#x27;hs-test&#x27;, so skipping these tests.&quot;
 done successfully.

------------------- UNIT TEST SUMMARY ---------------------

RUNIT TEST PROTOCOL -- Mon Feb  6 17:15:00 2023 
*********************************************** 
Number of test functions: 40 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
annmap unit testing - 40 test functions, 0 errors, 0 failures
&gt; 
&gt; proc.time()
   user  system elapsed 
  8.529   0.251   8.780 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">annmap.Rcheck/annmap-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>annmap.all</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annmap.coords</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annmap.details</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annmap.env</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annmap.filters</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annmap.range</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annmap.seqname</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annmap.spliceIndex</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annmap.to</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annmap.utils</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annmap.utr</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
