<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for COTAN on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><B>[<A href="..#C">C</A>]</B></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-16 06:06:52 -0000 (Tue, 16 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4787</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for COTAN on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the COTAN package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/COTAN.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>432</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">COTAN</A>&nbsp;2.1.2</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/COTAN">landing page</A>)</SPAN><BR>Galfrè Silvia Giulia<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-13&nbsp;23:12:22&nbsp;-0000&nbsp;(Sat,&nbsp;13&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/COTAN</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">122d07c</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-05-08&nbsp;13:44:41&nbsp;-0000&nbsp;(Mon,&nbsp;08&nbsp;May&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: COTAN</TD></TR>
<TR><TD><B>Version</B>: 2.1.2</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:COTAN.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings COTAN_2.1.2.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-15 00:58:26 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-15 01:35:14 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 2208.0 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: COTAN.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:COTAN.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings COTAN_2.1.2.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/COTAN.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘COTAN/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘COTAN’ version ‘2.1.2’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘COTAN’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... NOTE
Unexported object imported by a &#x27;:::&#x27; call: ‘ggplot2:::ggname’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
GDIPlot: no visible binding for global variable ‘sum.raw.norm’
GDIPlot: no visible binding for global variable ‘GDI’
UMAPPlot: no visible binding for global variable ‘x’
UMAPPlot: no visible binding for global variable ‘y’
calculateG: no visible binding for global variable ‘observedNN’
calculateG: no visible binding for global variable ‘observedNY’
calculateG: no visible binding for global variable ‘observedYN’
calculateG: no visible binding for global variable ‘observedYY’
calculateG: no visible binding for global variable ‘expectedNN’
calculateG: no visible binding for global variable ‘expectedNY’
calculateG: no visible binding for global variable ‘expectedYN’
calculateG: no visible binding for global variable ‘expectedYY’
cellsUniformClustering: no visible binding for global variable
  ‘objSeurat’
cellsUniformClustering: no visible binding for global variable
  ‘usedMaxResolution’
cleanPlots: no visible binding for global variable ‘PC1’
cleanPlots: no visible binding for global variable ‘PC2’
cleanPlots: no visible binding for global variable ‘n’
cleanPlots: no visible binding for global variable ‘means’
cleanPlots: no visible binding for global variable ‘nu’
clustersSummaryPlot: no visible binding for global variable ‘keys’
clustersSummaryPlot: no visible binding for global variable ‘values’
clustersSummaryPlot: no visible binding for global variable
  ‘CellNumber’
clustersSummaryPlot: no visible binding for global variable ‘MeanUDE’
clustersSummaryPlot: no visible binding for global variable
  ‘CellPercentage’
clustersSummaryPlot: no visible binding for global variable ‘Cluster’
clustersTreePlot: no visible binding for global variable ‘clusters’
establishGenesClusters: no visible binding for global variable
  ‘secondaryMarkers’
establishGenesClusters: no visible binding for global variable ‘GCS’
establishGenesClusters: no visible binding for global variable
  ‘rankGenes’
expectedContingencyTables: no visible binding for global variable
  ‘expectedN’
geom_flat_violin : &lt;anonymous&gt;: no visible binding for global variable
  ‘group’
geom_flat_violin : &lt;anonymous&gt;: no visible binding for global variable
  ‘y’
geom_flat_violin : &lt;anonymous&gt;: no visible binding for global variable
  ‘x’
geom_flat_violin : &lt;anonymous&gt;: no visible binding for global variable
  ‘width’
geom_flat_violin : &lt;anonymous&gt;: no visible binding for global variable
  ‘violinwidth’
geom_flat_violin : &lt;anonymous&gt;: no visible binding for global variable
  ‘xmax’
geom_flat_violin : &lt;anonymous&gt;: no visible binding for global variable
  ‘xminv’
geom_flat_violin : &lt;anonymous&gt;: no visible binding for global variable
  ‘xmaxv’
heatmapPlot: no visible binding for global variable ‘g2’
mitochondrialPercentagePlot: no visible binding for global variable
  ‘mit.percentage’
observedContingencyTables: no visible binding for global variable
  ‘observedY’
scatterPlot: no visible binding for global variable ‘.x’
calculateCoex,COTAN: no visible binding for global variable
  ‘expectedNN’
calculateCoex,COTAN: no visible binding for global variable
  ‘expectedNY’
calculateCoex,COTAN: no visible binding for global variable
  ‘expectedYN’
calculateCoex,COTAN: no visible binding for global variable
  ‘expectedYY’
calculateCoex,COTAN: no visible binding for global variable
  ‘observedYY’
calculateCoex,COTAN: no visible binding for global variable ‘.’
coerce,COTAN-scCOTAN: no visible binding for global variable ‘rawNorm’
coerce,COTAN-scCOTAN: no visible binding for global variable ‘nu’
coerce,COTAN-scCOTAN: no visible binding for global variable ‘lambda’
coerce,COTAN-scCOTAN: no visible binding for global variable ‘a’
coerce,COTAN-scCOTAN: no visible binding for global variable ‘hk’
coerce,COTAN-scCOTAN: no visible binding for global variable ‘clusters’
coerce,COTAN-scCOTAN: no visible binding for global variable
  ‘clusterData’
Undefined global functions or variables:
  . .x CellNumber CellPercentage Cluster GCS GDI MeanUDE PC1 PC2 a
  clusterData clusters expectedN expectedNN expectedNY expectedYN
  expectedYY g2 group hk keys lambda means mit.percentage n nu
  objSeurat observedNN observedNY observedY observedYN observedYY
  rankGenes rawNorm secondaryMarkers sum.raw.norm usedMaxResolution
  values violinwidth width x xmax xmaxv xminv y
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                           user system elapsed
UniformClusters         361.254  0.849 359.885
CalculatingCOEX          41.295  0.982  40.135
HeatmapPlots             38.780  0.852  37.768
ParametersEstimations    24.811  0.305  25.116
HandlingClusterizations  15.263  0.220  15.484
GenesCoexSpace           10.713  0.144  10.266
COTANObjectCreation       9.226  0.433   9.105
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘outputTestDatasetCreation.R’
  Running ‘spelling.R’
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘Guided_tutorial_v2.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 2 NOTEs
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/COTAN.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">COTAN.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD INSTALL COTAN
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0-bin/site-library’
* installing *source* package ‘COTAN’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (COTAN)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">COTAN.Rcheck/tests/outputTestDatasetCreation.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; 
&gt; # Creates the files to be reloaded by the tests for comparisons
&gt; 
&gt; outputTestDatasetCreation &lt;- function(testsDir = &quot;tests/testthat&quot;){
+   utils::data(&quot;test.dataset&quot;, package = &quot;COTAN&quot;)
+   options(parallelly.fork.enable = TRUE)
+ 
+   obj &lt;- COTAN(raw = test.dataset)
+   obj &lt;- initializeMetaDataset(obj, GEO = &quot; &quot;,
+                                sequencingMethod = &quot;artificial&quot;,
+                                sampleCondition = &quot;test&quot;)
+ 
+   obj &lt;- proceedToCoex(obj, cores = 12, saveObj = FALSE)
+   #saveRDS(obj, file = file.path(testsDir,&quot;temp.RDS&quot;))
+ 
+   cell.names.test  &lt;- getCells(obj)[c(1:10,591:610,991:1000)]
+   genes.names.test &lt;- getGenes(obj)[c(1:10,291:310,591:600)]
+   saveRDS(cell.names.test, file.path(testsDir, &quot;cell.names.test.RDS&quot;))
+   saveRDS(genes.names.test, file.path(testsDir, &quot;genes.names.test.RDS&quot;))
+ 
+   dispersion.test &lt;- getDispersion(obj)[genes.names.test]
+   saveRDS(dispersion.test, file.path(testsDir, &quot;dispersion.test.RDS&quot;))
+ 
+   raw.norm.test &lt;- getNormalizedData(obj)[genes.names.test, cell.names.test]
+   saveRDS(raw.norm.test, file.path(testsDir, &quot;raw.norm.test.RDS&quot;))
+ 
+   coex.test &lt;- getGenesCoex(obj, genes = genes.names.test, zeroDiagonal = FALSE)
+   saveRDS(coex.test, file.path(testsDir, &quot;coex.test.RDS&quot;))
+ 
+   lambda.test &lt;- getLambda(obj)[genes.names.test]
+   saveRDS(lambda.test, file.path(testsDir, &quot;lambda.test.RDS&quot;))
+ 
+   GDI.test &lt;- calculateGDI(obj)
+   GDI.test &lt;- GDI.test[genes.names.test, ]
+   saveRDS(GDI.test, file.path(testsDir, &quot;GDI.test.RDS&quot;))
+ 
+   nu.test &lt;- getNu(obj)[cell.names.test]
+   saveRDS(nu.test, file.path(testsDir, &quot;nu.test.RDS&quot;))
+ 
+   pval.test &lt;- calculatePValue(obj, geneSubsetCol = genes.names.test)
+   saveRDS(pval.test, file.path(testsDir, &quot;pval.test.RDS&quot;))
+ 
+   GDIThreshold &lt;- 1.5
+ 
+   clusters &lt;- cellsUniformClustering(obj, GDIThreshold = GDIThreshold,
+                                      cores = 12, saveObj = FALSE)
+   saveRDS(clusters, file.path(testsDir, &quot;clusters1.RDS&quot;))
+ 
+   c(coexDF, pvalDF) %&lt;-% DEAOnClusters(obj, clusters = clusters)
+   obj &lt;- addClusterization(obj, clName = &quot;clusters&quot;,
+                            clusters = clusters, coexDF = coexDF)
+ 
+   saveRDS(coexDF[genes.names.test, ],
+           file.path(testsDir, &quot;coex.test.cluster1.RDS&quot;))
+   saveRDS(pvalDF[genes.names.test, ],
+           file.path(testsDir, &quot;pval.test.cluster1.RDS&quot;))
+ 
+   c(mergedClusters, mCoexDF, mPValueDf) %&lt;-%
+     mergeUniformCellsClusters(objCOTAN = obj,
+                               clusters = NULL,
+                               GDIThreshold = GDIThreshold,
+                               cores = 12,
+                               distance = &quot;cosine&quot;,
+                               hclustMethod = &quot;ward.D2&quot;,
+                               saveObj = FALSE)
+ 
+   saveRDS(mergedClusters[genes.names.test],
+           file.path(testsDir, &quot;cluster_data_merged.RDS&quot;))
+ }
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.197   0.042   0.220 
</PRE>
</DIV><P><SPAN class="filename">COTAN.Rcheck/tests/spelling.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; if(requireNamespace(&#x27;spelling&#x27;, quietly = TRUE))
+   spelling::spell_check_test(vignettes = TRUE, error = FALSE,
+                              skip_on_cran = TRUE)
NULL
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.192   0.046   0.222 
</PRE>
</DIV><P><SPAN class="filename">COTAN.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; Sys.setenv(R_TESTS = &quot;&quot;)
&gt; library(testthat)
&gt; library(COTAN)
&gt; test_check(&quot;COTAN&quot;)
Setting new log level to 3
Initializing `COTAN` meta-data
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [10] genes and [20] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 0.9033203125 | max: 4.6796875 | % negative: 10
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0.181818181818182
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [10] genes and [20] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 5 genes batches from [1:2] to [9:10]
Estimate dispersion: DONE
dispersion | min: 0.9033203125 | max: 4.6796875 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 7 cells batches from [1:3] to [18:20]
Estimate nu: DONE
nu change (abs) | max: 1.75595238095238 | median:  1.07174634176587 | mean:  1.07174634176587
Estimate &#x27;dispersion&#x27;/&#x27;nu&#x27;: START
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 1.0362548828125 | max: 4.60986328125 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 1 cells batches from [1:20] to [1:20]
Estimate nu: DONE
nu change (abs) | max: 0.0265938895089288 | median:  0.0144680038331048 | mean:  0.0144680038331048
Nu mean: 1.69633192486233
Marginal errors | max: 1.95570586131367 | median 1.32068160171502 | mean: 1.33375826507259
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 0.058837890625 | max: 3.528076171875 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 1 cells batches from [1:20] to [1:20]
Estimate nu: DONE
nu change (abs) | max: 0.416683423613994 | median:  0.239880630367975 | mean:  0.239880630367975
Nu mean: 0.823197206753982
Marginal errors | max: 0.836359531101206 | median 0.703684202571891 | mean: 0.645537958989614
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 0.32879638671875 | max: 4.0302734375 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 1 cells batches from [1:20] to [1:20]
Estimate nu: DONE
nu change (abs) | max: 0.164237872898673 | median:  0.0955985184389135 | mean:  0.0955985184389135
Nu mean: 1.06863935445976
Marginal errors | max: 0.259872988828244 | median 0.213703042752633 | mean: 0.197386407582083
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 0.2294921875 | max: 3.8720703125 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 1 cells batches from [1:20] to [1:20]
Estimate nu: DONE
nu change (abs) | max: 0.055185575120883 | median:  0.0319991762044448 | mean:  0.0319991762044448
Nu mean: 0.976813601083562
Marginal errors | max: 0.0951586919577032 | median 0.0794297037094669 | mean: 0.0724140148396652
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 0.2637939453125 | max: 3.929443359375 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 1 cells batches from [1:20] to [1:20]
Estimate nu: DONE
nu change (abs) | max: 0.0196211148938294 | median:  0.01138609597457 | mean:  0.01138609597457
Nu mean: 1.00823501891926
Marginal errors | max: 0.0327747321002274 | median 0.0272104747849529 | mean: 0.0248963830312036
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 0.25177001953125 | max: 3.90966796875 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 1 cells batches from [1:20] to [1:20]
Estimate nu: DONE
nu change (abs) | max: 0.00670099066960717 | median:  0.00388888266671264 | mean:  0.00388888266671264
Nu mean: 0.997187891997105
Marginal errors | max: 0.0114324509186883 | median 0.00942326497706159 | mean: 0.00863113610779536
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 0.25592041015625 | max: 3.91650390625 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 1 cells batches from [1:20] to [1:20]
Estimate nu: DONE
nu change (abs) | max: 0.00230093811689414 | median:  0.00132529122122246 | mean:  0.00132529122122246
Nu mean: 1.00097564689567
Marginal errors | max: 0.00387133150664809 | median 0.0031091017608853 | mean: 0.00286071175800213
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 0.2545166015625 | max: 3.914306640625 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 1 cells batches from [1:20] to [1:20]
Estimate nu: DONE
nu change (abs) | max: 0.000837011646904529 | median:  0.000470837393363011 | mean:  0.000470837393363011
Nu mean: 0.999633825746458
Marginal errors | max: 0.00122501723202184 | median 0.00102126435760308 | mean: 0.000943992659051851
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 0.2550048828125 | max: 3.9150390625 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 1 cells batches from [1:20] to [1:20]
Estimate nu: DONE
nu change (abs) | max: 0.000209227351122054 | median:  0.000122070312500028 | mean:  0.000122070312500028
Nu mean: 1.00008715703862
Marginal errors | max: 0.000364602956583582 | median 0.000313956936819793 | mean: 0.000282899574318485
Estimate dispersion/nu: DONE
Estimate &#x27;dispersion&#x27;/&#x27;nu&#x27;: START
Initializing `COTAN` meta-data
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [10] genes and [20] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 0.9033203125 | max: 4.6796875 | % negative: 10
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0.181818181818182
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Calculate cells&#x27; coex: START
Retrieving expected cells&#x27; contingency table
calculating NN.. done
calculating YN..NY..YY.. done
Calculating cells&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0
Retrieving observed cells&#x27; yes/yes contingency table
calculating YY.. done
Estimating cells&#x27; coex
Calculate cells&#x27; coex: DONE
Initializing `COTAN` meta-data
Initializing `COTAN` meta-data
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [10] genes and [20] cells
Estimate &#x27;dispersion&#x27;/&#x27;nu&#x27;: START
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 0.903564453125 | max: 4.679443359375 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 1 cells batches from [1:20] to [1:20]
Estimate nu: DONE
nu change (abs) | max: 1.75719246031746 | median:  1.07229953342014 | mean:  1.07229953342014
Nu mean: 1.68489292689732
Marginal errors | max: 1.73564890252257 | median 1.37996360874076 | mean: 1.32180348113228
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 0.0655517578125 | max: 3.5439453125 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 1 cells batches from [1:20] to [1:20]
Estimate nu: DONE
nu change (abs) | max: 0.402649984216273 | median:  0.231868788425666 | mean:  0.231868788425666
Nu mean: 0.829218804209393
Marginal errors | max: 0.803213159865939 | median 0.677497553540579 | mean: 0.61937543089282
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 0.3260498046875 | max: 4.026123046875 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 1 cells batches from [1:20] to [1:20]
Estimate nu: DONE
nu change (abs) | max: 0.158004893526231 | median:  0.0919692884670312 | mean:  0.0919692884670312
Nu mean: 1.0660356050592
Marginal errors | max: 0.250724014302325 | median 0.206232152124435 | mean: 0.190425623677197
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 0.23040771484375 | max: 3.8736572265625 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 1 cells batches from [1:20] to [1:20]
Estimate nu: DONE
nu change (abs) | max: 0.0532774732102337 | median:  0.0308837890624999 | mean:  0.0308837890624999
Nu mean: 0.977606315852266
Marginal errors | max: 0.0916983669060105 | median 0.0765266929824948 | mean: 0.0697593208689693
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 0.26348876953125 | max: 3.928955078125 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 1 cells batches from [1:20] to [1:20]
Estimate nu: DONE
nu change (abs) | max: 0.0189966206463044 | median:  0.0110199320575908 | mean:  0.0110199320575908
Nu mean: 1.00797668858871
Marginal errors | max: 0.0317151207459254 | median 0.0262702142278233 | mean: 0.0240886952086955
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 0.2518310546875 | max: 3.9097900390625 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 1 cells batches from [1:20] to [1:20]
Estimate nu: DONE
nu change (abs) | max: 0.00670088501353994 | median:  0.00388888101583662 | mean:  0.00388888101583662
Nu mean: 0.997187996002297
Marginal errors | max: 0.011369331635624 | median 0.00939669372836338 | mean: 0.00860715734056949
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 0.2559814453125 | max: 3.9166259765625 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 1 cells batches from [1:20] to [1:20]
Estimate nu: DONE
nu change (abs) | max: 0.00251007446998996 | median:  0.00144735987374958 | mean:  0.00144735987374958
Nu mean: 1.00106271459624
Marginal errors | max: 0.00406746973787264 | median 0.00343393462175801 | mean: 0.00313496757119527
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 0.25445556640625 | max: 3.9140625 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 1 cells batches from [1:20] to [1:20]
Estimate nu: DONE
nu change (abs) | max: 0.000837027590858019 | median:  0.000488281249999889 | mean:  0.000488281249999889
Nu mean: 0.999651253659142
Marginal errors | max: 0.00143433714371355 | median 0.00116636244706836 | mean: 0.00109289166947804
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:10] to [1:10]
Estimate dispersion: DONE
dispersion | min: 0.2550048828125 | max: 3.9150390625 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 1 cells batches from [1:20] to [1:20]
Estimate nu: DONE
nu change (abs) | max: 0.000209227688885871 | median:  0.0001220703125 | mean:  0.0001220703125
Nu mean: 1.00008715737639
Marginal errors | max: 0.000379524846206181 | median 0.000325685250687435 | mean: 0.000295532844331703
Estimate dispersion/nu: DONE
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0.181818181818182
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Calculate cells&#x27; coex: START
Retrieving expected cells&#x27; contingency table
calculating NN.. done
calculating YN..NY..YY.. done
Calculating cells&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0
Retrieving observed cells&#x27; yes/yes contingency table
calculating YY.. done
Estimating cells&#x27; coex
Calculate cells&#x27; coex: DONE
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [10] genes and [20] cells
calculating YY.. done
calculating YY.. done
calculating YN..NY..NN.. done
Estimate dispersion: START
Effective number of cores used: 1
Executing 3 genes batches from [1:3] to [8:10]
Estimate dispersion: DONE
dispersion | min: 0.9033203125 | max: 4.6796875 | % negative: 10
calculating NN.. done
calculating NN.. done
calculating NY..YN..YY.. done
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0.181818181818182
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [10] genes and [20] cells
calculating YY.. done
calculating YY.. done
calculating NY..YN..NN.. done
Estimate &#x27;dispersion&#x27;/&#x27;nu&#x27;: START
Estimate dispersion: START
Effective number of cores used: 1
Executing 3 genes batches from [1:3] to [8:10]
Estimate dispersion: DONE
dispersion | min: 0.903564453125 | max: 4.679443359375 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 5 cells batches from [1:4] to [17:20]
Estimate nu: DONE
nu change (abs) | max: 1.75719246031746 | median:  1.07229953342014 | mean:  1.07229953342014
Nu mean: 1.68489292689732
Marginal errors | max: 0.255353289373158 | median 0.0807577993228143 | mean: 0.101980750205761
Estimate dispersion: START
Effective number of cores used: 1
Executing 3 genes batches from [1:3] to [8:10]
Estimate dispersion: DONE
dispersion | min: 1.037109375 | max: 4.6107177734375 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 5 cells batches from [1:4] to [17:20]
Estimate nu: DONE
nu change (abs) | max: 0.0273438105507502 | median:  0.0148852611818011 | mean:  0.0148852611818011
Nu mean: 1.69735147626627
Marginal errors | max: 0.00326864580272002 | median 0.00111524657842832 | mean: 0.00131556083122533
Estimate dispersion: START
Effective number of cores used: 1
Executing 3 genes batches from [1:3] to [8:10]
Estimate dispersion: DONE
dispersion | min: 1.03887939453125 | max: 4.6097412109375 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 5 cells batches from [1:4] to [17:20]
Estimate nu: DONE
nu change (abs) | max: 0 | median:  0 | mean:  0
Nu mean: 1.69735147626627
Marginal errors | max: 7.56328383637594e-05 | median 1.72948087246994e-05 | mean: 2.99252342141898e-05
Estimate dispersion/nu: DONE
calculating NN.. done
calculating NN.. done
calculating YN..NY..YY.. done
Calculate cells&#x27; coex: START
Retrieving expected cells&#x27; contingency table
calculating NN.. done
calculating YN..NY..YY.. done
Calculating cells&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0
Retrieving observed cells&#x27; yes/yes contingency table
calculating YY.. done
Estimating cells&#x27; coex
Calculate cells&#x27; coex: DONE
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [10] genes and [20] cells
Estimate &#x27;dispersion&#x27;/&#x27;nu&#x27;: START
Estimate dispersion: START
Effective number of cores used: 1
Executing 3 genes batches from [1:3] to [8:10]
Estimate dispersion: DONE
dispersion | min: 0.903564453125 | max: 4.679443359375 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 5 cells batches from [1:4] to [17:20]
Estimate nu: DONE
nu change (abs) | max: 1.75719246031746 | median:  1.07229953342014 | mean:  1.07229953342014
Nu mean: 1.68489292689732
Marginal errors | max: 0.255353289373158 | median 0.0807577993228143 | mean: 0.101980750205761
Estimate dispersion: START
Effective number of cores used: 1
Executing 3 genes batches from [1:3] to [8:10]
Estimate dispersion: DONE
dispersion | min: 1.037109375 | max: 4.6107177734375 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 5 cells batches from [1:4] to [17:20]
Estimate nu: DONE
nu change (abs) | max: 0.0273438105507502 | median:  0.0148852611818011 | mean:  0.0148852611818011
Nu mean: 1.69735147626627
Marginal errors | max: 0.00326864580272002 | median 0.00111524657842832 | mean: 0.00131556083122533
Estimate dispersion: START
Effective number of cores used: 1
Executing 3 genes batches from [1:3] to [8:10]
Estimate dispersion: DONE
dispersion | min: 1.03887939453125 | max: 4.6097412109375 | % negative: 10
Estimate nu: START
Effective number of cores used: 1
Executing 5 cells batches from [1:4] to [17:20]
Estimate nu: DONE
nu change (abs) | max: 0 | median:  0 | mean:  0
Nu mean: 1.69735147626627
Marginal errors | max: 7.56328383637594e-05 | median 1.72948087246994e-05 | mean: 2.99252342141898e-05
Estimate dispersion/nu: DONE
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0.181818181818182
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Calculating S: START
Calculating S: DONE
Calculating G: START
calculating YY.. done
calculating YN..NY..NN.. done
calculating NN.. done
calculating NY..YN..YY.. done
Estimating G
Calculating G: DONE
Using S
Calculating S: START
Calculating S: DONE
calculating PValues: START
Get p-values genome wide on columns and genome wide on rows
calculating PValues: DONE
Using G
Calculating G: START
calculating YY.. done
calculating YN..NY..NN.. done
calculating NN.. done
calculating NY..YN..YY.. done
Estimating G
Calculating G: DONE
calculating PValues: START
Get p-values on a set of genes on columns and on a set of genes on rows
calculating PValues: DONE
Using S
Calculating S: START
Calculating S: DONE
Calculate GDI dataframe: START
Calculate GDI dataframe: DONE
Using G
Calculating G: START
calculating YY.. done
calculating YN..NY..NN.. done
calculating NN.. done
calculating NY..YN..YY.. done
Estimating G
Calculating G: DONE
Calculate GDI dataframe: START
Calculate GDI dataframe: DONE
Initializing `COTAN` meta-data
Cotan analysis functions started
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [600] genes and [1200] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:600] to [1:600]
Estimate dispersion: DONE
dispersion | min: 0.2197265625 | max: 6.08056640625 | % negative: 0
Only analysis time 0.05214950243632
Cotan genes&#x27; coex estimation started
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Total time 0.15786802371343
Only genes&#x27; coex time 0.0970323443412781
Initializing `COTAN` meta-data
Condition test
n cells 1200
Cotan analysis functions started
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [600] genes and [1200] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:600] to [1:600]
Estimate dispersion: DONE
dispersion | min: 0.2197265625 | max: 6.08056640625 | % negative: 0
Only analysis time 0.0526100635528564
Cotan genes&#x27; coex estimation started
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Total time 0.155849866072337
Only genes&#x27; coex time 0.0945954362551371
Using S
Calculating S: START
Calculating S: DONE
calculating PValues: START
Get p-values on a set of genes on columns and genome wide on rows
calculating PValues: DONE
Using S
Calculating S: START
Calculating S: DONE
Calculate GDI dataframe: START
Calculate GDI dataframe: DONE
Initializing `COTAN` meta-data
Cotan analysis functions started
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [600] genes and [1200] cells
PCA: START
PCA: DONE
Hierarchical clustering: START
Hierarchical clustering: DONE
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:600] to [1:600]
Estimate dispersion: DONE
dispersion | min: 0.2197265625 | max: 6.08056640625 | % negative: 0
Only analysis time 0.053636105855306
Cotan genes&#x27; coex estimation started
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Total time 0.21229532957077
Only genes&#x27; coex time 0.0974710067113241
Saving elaborated data locally at: /tmp/RtmpSTOOq5/test.cotan.RDS
Creating cells&#x27; uniform clustering: START
In iteration 0 the number of cells to re-cluster is 1200 cells belonging to 0 clusters
Creating Seurat object: START
Performing log-normalization
0%   10   20   30   40   50   60   70   80   90   100%
[----|----|----|----|----|----|----|----|----|----|
**************************************************|
Calculating gene variances
0%   10   20   30   40   50   60   70   80   90   100%
[----|----|----|----|----|----|----|----|----|----|
**************************************************|
Calculating feature variances of standardized and clipped values
0%   10   20   30   40   50   60   70   80   90   100%
[----|----|----|----|----|----|----|----|----|----|
**************************************************|
Centering and scaling data matrix
  |                                                                              |                                                                      |   0%  |                                                                              |======================================================================| 100%
PC_ 1 
Positive:  g-000558, g-000570, g-000499, g-000504, g-000546, g-000506, g-000503, g-000517, g-000596, g-000528 
	   g-000527, g-000580, g-000592, g-000578, g-000509, g-000488, g-000555, g-000577, g-000534, g-000583 
	   g-000598, g-000535, g-000512, g-000554, g-000519, g-000525, g-000548, g-000544, g-000502, g-000541 
Negative:  g-000133, g-000007, g-000074, g-000141, g-000057, g-000235, g-000170, g-000019, g-000195, g-000140 
	   g-000183, g-000031, g-000046, g-000178, g-000177, g-000161, g-000157, g-000139, g-000011, g-000135 
	   g-000125, g-000208, g-000061, g-000085, g-000204, g-000104, g-000237, g-000004, g-000038, g-000128 
PC_ 2 
Positive:  g-000039, g-000050, g-000175, g-000078, g-000116, g-000189, g-000135, g-000047, g-000072, g-000087 
	   g-000063, g-000235, g-000066, g-000109, g-000018, g-000074, g-000231, g-000136, g-000034, g-000207 
	   g-000128, g-000167, g-000171, g-000049, g-000182, g-000013, g-000054, g-000062, g-000240, g-000158 
Negative:  g-000584, g-000583, g-000544, g-000519, g-000575, g-000516, g-000585, g-000486, g-000489, g-000539 
	   g-000484, g-000502, g-000523, g-000595, g-000305, g-000574, g-000599, g-000589, g-000509, g-000538 
	   g-000526, g-000551, g-000579, g-000590, g-000445, g-000556, g-000543, g-000501, g-000504, g-000570 
PC_ 3 
Positive:  g-000015, g-000575, g-000483, g-000316, g-000025, g-000364, g-000050, g-000278, g-000443, g-000360 
	   g-000332, g-000124, g-000212, g-000387, g-000536, g-000252, g-000251, g-000321, g-000501, g-000470 
	   g-000582, g-000106, g-000455, g-000368, g-000081, g-000104, g-000437, g-000288, g-000386, g-000317 
Negative:  g-000211, g-000337, g-000129, g-000185, g-000397, g-000403, g-000253, g-000098, g-000390, g-000303 
	   g-000052, g-000088, g-000463, g-000468, g-000236, g-000209, g-000005, g-000375, g-000342, g-000262 
	   g-000388, g-000091, g-000413, g-000285, g-000003, g-000095, g-000142, g-000205, g-000432, g-000241 
PC_ 4 
Positive:  g-000379, g-000193, g-000212, g-000434, g-000593, g-000513, g-000177, g-000223, g-000069, g-000131 
	   g-000162, g-000345, g-000462, g-000484, g-000448, g-000229, g-000365, g-000302, g-000010, g-000366 
	   g-000051, g-000535, g-000269, g-000270, g-000155, g-000529, g-000373, g-000008, g-000393, g-000306 
Negative:  g-000334, g-000398, g-000292, g-000095, g-000097, g-000202, g-000382, g-000195, g-000007, g-000079 
	   g-000086, g-000240, g-000263, g-000317, g-000576, g-000557, g-000160, g-000154, g-000214, g-000228 
	   g-000313, g-000053, g-000524, g-000374, g-000568, g-000188, g-000358, g-000528, g-000362, g-000150 
PC_ 5 
Positive:  g-000451, g-000339, g-000295, g-000328, g-000544, g-000061, g-000227, g-000391, g-000556, g-000237 
	   g-000067, g-000165, g-000449, g-000591, g-000087, g-000129, g-000197, g-000203, g-000487, g-000505 
	   g-000333, g-000029, g-000271, g-000064, g-000583, g-000156, g-000448, g-000153, g-000526, g-000393 
Negative:  g-000518, g-000108, g-000186, g-000170, g-000401, g-000337, g-000047, g-000599, g-000432, g-000578 
	   g-000042, g-000065, g-000493, g-000261, g-000533, g-000256, g-000560, g-000596, g-000368, g-000381 
	   g-000535, g-000338, g-000215, g-000159, g-000365, g-000234, g-000173, g-000387, g-000225, g-000272 
Computing nearest neighbor graph
Computing SNN
Modularity Optimizer version 1.3.0 by Ludo Waltman and Nees Jan van Eck

Number of nodes: 1200
Number of edges: 55489

Running Louvain algorithm with multilevel refinement...
0%   10   20   30   40   50   60   70   80   90   100%
[----|----|----|----|----|----|----|----|----|----|
**************************************************|
Maximum modularity in 10 random starts: 0.6846
Number of communities: 4
Elapsed time: 0 seconds
Used resolution for Seurat clusterization is: 0.5
01:13:04 UMAP embedding parameters a = 0.9922 b = 1.112
01:13:04 Read 1200 rows and found 50 numeric columns
01:13:04 Using Annoy for neighbor search, n_neighbors = 30
01:13:04 Building Annoy index with metric = cosine, n_trees = 50
0%   10   20   30   40   50   60   70   80   90   100%
[----|----|----|----|----|----|----|----|----|----|
**************************************************|
01:13:04 Writing NN index file to temp file /tmp/RtmpSTOOq5/file1ff4c97134dfbd
01:13:04 Searching Annoy index using 1 thread, search_k = 3000
01:13:04 Annoy recall = 100%
01:13:05 Commencing smooth kNN distance calibration using 1 thread with target n_neighbors = 30
01:13:06 Initializing from normalized Laplacian + noise (using RSpectra)
01:13:06 Commencing optimization for 500 epochs, with 42228 positive edges
Using method &#x27;umap&#x27;
0%   10   20   30   40   50   60   70   80   90   100%
[----|----|----|----|----|----|----|----|----|----|
**************************************************|
01:13:08 Optimization finished
Creating PDF UMAP in file:/tmp/RtmpSTOOq5/test/reclustering_0/pdf_umap.pdf
Creating Seurat object: DONE
* checking uniformity of cluster &#x27;0&#x27; of 4 clusters
Cotan analysis functions started
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [600] genes and [353] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:600] to [1:600]
Estimate dispersion: DONE
dispersion | min: -0.038818359375 | max: 11.109375 | % negative: 5
Only analysis time 0.0343083103497823
Cotan genes&#x27; coex estimation started
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0.000521353300055463
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Total time 0.132102354367574
Only genes&#x27; coex time 0.0884678204854329
Checking uniformity for the cluster &#x27;0&#x27; with 353 cells
Using S
Calculating S: START
Calculating S: DONE
Calculate GDI dataframe: START
Calculate GDI dataframe: DONE
PCA: START
PCA: DONE
Hierarchical clustering: START
Hierarchical clustering: DONE
GDI plot
Removed 0 low GDI genes (such as the fully-expressed) in GDI plot
cluster 0 is uniform
* checking uniformity of cluster &#x27;1&#x27; of 4 clusters
Cotan analysis functions started
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [600] genes and [315] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:600] to [1:600]
Estimate dispersion: DONE
dispersion | min: -0.05303955078125 | max: 52.125 | % negative: 32.6666666666667
Only analysis time 0.0331037322680155
Cotan genes&#x27; coex estimation started
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0.288990571270105
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Total time 0.131627623240153
Only genes&#x27; coex time 0.0889289180437724
Checking uniformity for the cluster &#x27;1&#x27; with 315 cells
Using S
Calculating S: START
Calculating S: DONE
Calculate GDI dataframe: START
Calculate GDI dataframe: DONE
PCA: START
PCA: DONE
Hierarchical clustering: START
Hierarchical clustering: DONE
GDI plot
Removed 0 low GDI genes (such as the fully-expressed) in GDI plot
cluster 1 is uniform
* checking uniformity of cluster &#x27;2&#x27; of 4 clusters
Cotan analysis functions started
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [600] genes and [311] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:600] to [1:600]
Estimate dispersion: DONE
dispersion | min: -0.0517578125 | max: 17.296875 | % negative: 8.66666666666667
Only analysis time 0.0333536545435588
Cotan genes&#x27; coex estimation started
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0.00206322795341098
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Total time 0.132310740152995
Only genes&#x27; coex time 0.0894862731297811
Checking uniformity for the cluster &#x27;2&#x27; with 311 cells
Using S
Calculating S: START
Calculating S: DONE
Calculate GDI dataframe: START
Calculate GDI dataframe: DONE
PCA: START
PCA: DONE
Hierarchical clustering: START
Hierarchical clustering: DONE
GDI plot
Removed 0 low GDI genes (such as the fully-expressed) in GDI plot
cluster 2 is uniform
* checking uniformity of cluster &#x27;3&#x27; of 4 clusters
Cotan analysis functions started
Genes/cells selection done: dropped [1] genes and [0] cells
Working on [599] genes and [221] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:599] to [1:599]
Estimate dispersion: DONE
dispersion | min: -0.06640625 | max: 79 | % negative: 35.0584307178631
Only analysis time 0.0316835562388102
Cotan genes&#x27; coex estimation started
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0.333583750695604
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Total time 0.129245861371358
Only genes&#x27; coex time 0.0880436460177104
Checking uniformity for the cluster &#x27;3&#x27; with 221 cells
Using S
Calculating S: START
Calculating S: DONE
Calculate GDI dataframe: START
Calculate GDI dataframe: DONE
PCA: START
PCA: DONE
Hierarchical clustering: START
Hierarchical clustering: DONE
GDI plot
Removed 0 low GDI genes (such as the fully-expressed) in GDI plot
cluster 3 is uniform

Found 4 uniform and  0 non-uniform clusters
NO new possible uniform clusters! Unclustered cell left: 0
The final raw clusterization contains [ 4 ] different clusters: 00_0000, 00_0001, 00_0002, 00_0003
Cluster, UMAP and Saving the Seurat dataset
Computing nearest neighbor graph
Computing SNN
Modularity Optimizer version 1.3.0 by Ludo Waltman and Nees Jan van Eck

Number of nodes: 1200
Number of edges: 55489

Running Louvain algorithm with multilevel refinement...
0%   10   20   30   40   50   60   70   80   90   100%
[----|----|----|----|----|----|----|----|----|----|
**************************************************|
Maximum modularity in 10 random starts: 0.6846
Number of communities: 4
Elapsed time: 0 seconds
01:14:06 UMAP embedding parameters a = 0.9922 b = 1.112
01:14:06 Read 1200 rows and found 25 numeric columns
01:14:06 Using Annoy for neighbor search, n_neighbors = 30
01:14:06 Building Annoy index with metric = cosine, n_trees = 50
0%   10   20   30   40   50   60   70   80   90   100%
[----|----|----|----|----|----|----|----|----|----|
**************************************************|
01:14:06 Writing NN index file to temp file /tmp/RtmpSTOOq5/file1ff4c92ee0abad
01:14:06 Searching Annoy index using 1 thread, search_k = 3000
01:14:06 Annoy recall = 100%
01:14:07 Commencing smooth kNN distance calibration using 1 thread with target n_neighbors = 30
01:14:08 Initializing from normalized Laplacian + noise (using RSpectra)
01:14:08 Commencing optimization for 500 epochs, with 43428 positive edges
Using method &#x27;umap&#x27;
0%   10   20   30   40   50   60   70   80   90   100%
[----|----|----|----|----|----|----|----|----|----|
**************************************************|
01:14:11 Optimization finished
Creating cells&#x27; uniform clustering: DONE
Cotan analysis functions started
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [600] genes and [315] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:600] to [1:600]
Estimate dispersion: DONE
dispersion | min: -0.05303955078125 | max: 52.125 | % negative: 32.6666666666667
Only analysis time 0.0330410401026408
Cotan genes&#x27; coex estimation started
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0.288990571270105
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Total time 0.131646351019541
Only genes&#x27; coex time 0.0890612880388896
Checking uniformity for the cluster &#x27;Cluster_2&#x27; with 315 cells
Using S
Calculating S: START
Calculating S: DONE
Calculate GDI dataframe: START
Calculate GDI dataframe: DONE
PCA: START
PCA: DONE
Hierarchical clustering: START
Hierarchical clustering: DONE
GDI plot
Removed 0 low GDI genes (such as the fully-expressed) in GDI plot
findClustersMarkers - START
Differential Expression Analysis - START
* analysis of cluster: &#x27;1&#x27; - START
* analysis of cluster: &#x27;1&#x27; - DONE
* analysis of cluster: &#x27;2&#x27; - START
* analysis of cluster: &#x27;2&#x27; - DONE
* analysis of cluster: &#x27;3&#x27; - START
* analysis of cluster: &#x27;3&#x27; - DONE
* analysis of cluster: &#x27;4&#x27; - START
* analysis of cluster: &#x27;4&#x27; - DONE

Differential Expression Analysis - DONE
clustersDeltaExpression - START
Handling cluster &#x27;1&#x27; with mean UDE 1.43530796540674
Handling cluster &#x27;2&#x27; with mean UDE 0.546546914955575
Handling cluster &#x27;3&#x27; with mean UDE 1.22034802329657
Handling cluster &#x27;4&#x27; with mean UDE 0.640931107489519
clustersDeltaExpression - DONE
findClustersMarkers - DONE
findClustersMarkers - START
Differential Expression Analysis - START
* analysis of cluster: &#x27;1&#x27; - START
* analysis of cluster: &#x27;1&#x27; - DONE
* analysis of cluster: &#x27;2&#x27; - START
* analysis of cluster: &#x27;2&#x27; - DONE
* analysis of cluster: &#x27;3&#x27; - START
* analysis of cluster: &#x27;3&#x27; - DONE
* analysis of cluster: &#x27;4&#x27; - START
* analysis of cluster: &#x27;4&#x27; - DONE

Differential Expression Analysis - DONE
clustersDeltaExpression - START
Handling cluster &#x27;1&#x27; with mean UDE 1.43530796540674
Handling cluster &#x27;2&#x27; with mean UDE 0.546546914955575
Handling cluster &#x27;3&#x27; with mean UDE 1.22034802329657
Handling cluster &#x27;4&#x27; with mean UDE 0.640931107489519
clustersDeltaExpression - DONE
findClustersMarkers - DONE
findClustersMarkers - START
Differential Expression Analysis - START
* analysis of cluster: &#x27;1&#x27; - START
* analysis of cluster: &#x27;1&#x27; - DONE
* analysis of cluster: &#x27;2&#x27; - START
* analysis of cluster: &#x27;2&#x27; - DONE
* analysis of cluster: &#x27;3&#x27; - START
* analysis of cluster: &#x27;3&#x27; - DONE
* analysis of cluster: &#x27;4&#x27; - START
* analysis of cluster: &#x27;4&#x27; - DONE

Differential Expression Analysis - DONE
clustersDeltaExpression - START
Handling cluster &#x27;1&#x27; with mean UDE 1.43530796540674
Handling cluster &#x27;2&#x27; with mean UDE 0.546546914955575
Handling cluster &#x27;3&#x27; with mean UDE 1.22034802329657
Handling cluster &#x27;4&#x27; with mean UDE 0.640931107489519
clustersDeltaExpression - DONE
findClustersMarkers - DONE
[1] &quot;4&quot;
Cotan analysis functions started
Genes/cells selection done: dropped [1] genes and [0] cells
Working on [599] genes and [221] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:599] to [1:599]
Estimate dispersion: DONE
dispersion | min: -0.06640625 | max: 79 | % negative: 35.0584307178631
Only analysis time 0.0322176893552144
Cotan genes&#x27; coex estimation started
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0.333583750695604
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Total time 0.131225927670797
Only genes&#x27; coex time 0.0893503705660502
Using S
Calculating S: START
Calculating S: DONE
Calculate GDI dataframe: START
Calculate GDI dataframe: DONE
[1] &quot;1&quot;
Cotan analysis functions started
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [600] genes and [353] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:600] to [1:600]
Estimate dispersion: DONE
dispersion | min: -0.038818359375 | max: 11.109375 | % negative: 5
Only analysis time 0.0339964509010315
Cotan genes&#x27; coex estimation started
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0.000521353300055463
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Total time 0.133610566457113
Only genes&#x27; coex time 0.0900969584782918
Using S
Calculating S: START
Calculating S: DONE
Calculate GDI dataframe: START
Calculate GDI dataframe: DONE
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [10] genes and [20] cells
Initializing `COTAN` meta-data
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [600] genes and [1200] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:600] to [1:600]
Estimate dispersion: DONE
dispersion | min: 0.2197265625 | max: 6.08056640625 | % negative: 0
Cotan analysis functions started
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [600] genes and [1200] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:600] to [1:600]
Estimate dispersion: DONE
dispersion | min: 0.2197265625 | max: 6.08056640625 | % negative: 0
Only analysis time 0.0533792575200399
Cotan genes&#x27; coex estimation started
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Total time 0.163134558995565
Only genes&#x27; coex time 0.100111854076385
Calculating gene coexpression space - START
Using S
Calculating S: START
Calculating S: DONE
calculating PValues: START
Get p-values on a set of genes on columns and genome wide on rows
calculating PValues: DONE
Number of selected secondary markers: 109
Calculating S: START
Calculating S: DONE
Number of columns (V set - secondary markers): 109
Number of rows (U set): 60
Calculating gene coexpression space - DONE
Establishing gene clusters - START
Calculating gene coexpression space - START
Using S
Calculating S: START
Calculating S: DONE
calculating PValues: START
Get p-values on a set of genes on columns and genome wide on rows
calculating PValues: DONE
Number of selected secondary markers: 109
Calculating S: START
Calculating S: DONE
Number of columns (V set - secondary markers): 109
Number of rows (U set): 60
Calculating gene coexpression space - DONE
Establishing gene clusters - DONE
Initializing `COTAN` meta-data
Cotan analysis functions started
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [600] genes and [1200] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:600] to [1:600]
Estimate dispersion: DONE
dispersion | min: 0.2197265625 | max: 6.08056640625 | % negative: 0
Only analysis time 0.0524200240770976
Cotan genes&#x27; coex estimation started
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Total time 0.163752273718516
Only genes&#x27; coex time 0.101512241363525
Differential Expression Analysis - START
* analysis of cluster: &#x27;1&#x27; - START
* analysis of cluster: &#x27;1&#x27; - DONE
* analysis of cluster: &#x27;2&#x27; - START
* analysis of cluster: &#x27;2&#x27; - DONE
* analysis of cluster: &#x27;3&#x27; - START
* analysis of cluster: &#x27;3&#x27; - DONE
* analysis of cluster: &#x27;4&#x27; - START
* analysis of cluster: &#x27;4&#x27; - DONE

Differential Expression Analysis - DONE
clustersDeltaExpression - START
Handling cluster &#x27;1&#x27; with mean UDE 1.43530796540674
Handling cluster &#x27;2&#x27; with mean UDE 0.546546914955575
Handling cluster &#x27;3&#x27; with mean UDE 1.22034802329657
Handling cluster &#x27;4&#x27; with mean UDE 0.640931107489519
clustersDeltaExpression - DONE
In group G1 there are 3 detected over 3 genes
In group G2 there are 2 detected over 2 genes
In group G3 there are 5 detected over 5 genes
Merging cells&#x27; uniform clustering: START
Start merging smallest clusters: iteration 1
Differential Expression Analysis - START
* analysis of cluster: &#x27;1&#x27; - START
* analysis of cluster: &#x27;1&#x27; - DONE
* analysis of cluster: &#x27;2&#x27; - START
* analysis of cluster: &#x27;2&#x27; - DONE
* analysis of cluster: &#x27;3&#x27; - START
* analysis of cluster: &#x27;3&#x27; - DONE
* analysis of cluster: &#x27;4&#x27; - START
* analysis of cluster: &#x27;4&#x27; - DONE

Differential Expression Analysis - DONE
Created leafs ID for merging: 1 4 2 3
*1_4-merge
Cotan analysis functions started
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [600] genes and [574] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:600] to [1:600]
Estimate dispersion: DONE
dispersion | min: -0.0340576171875 | max: 10.4375 | % negative: 4.33333333333333
Only analysis time 0.0400446613629659
Cotan genes&#x27; coex estimation started
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0.000105379922351636
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Total time 0.142012405395508
Only genes&#x27; coex time 0.0923430442810059
Checking uniformity for the cluster &#x27;1_4-merge&#x27; with 574 cells
Using S
Calculating S: START
Calculating S: DONE
Calculate GDI dataframe: START
Calculate GDI dataframe: DONE
PCA: START
PCA: DONE
Hierarchical clustering: START
Hierarchical clustering: DONE
GDI plot
Removed 0 low GDI genes (such as the fully-expressed) in GDI plot
Clusters 1 and 4 can be merged
*2_3-merge
Cotan analysis functions started
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [600] genes and [626] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:600] to [1:600]
Estimate dispersion: DONE
dispersion | min: -0.03839111328125 | max: 17.40625 | % negative: 6.66666666666667
Only analysis time 0.0395151893297831
Cotan genes&#x27; coex estimation started
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 5.54631170271769e-05
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Total time 0.142625188827515
Only genes&#x27; coex time 0.0935862302780151
Checking uniformity for the cluster &#x27;2_3-merge&#x27; with 626 cells
Using S
Calculating S: START
Calculating S: DONE
Calculate GDI dataframe: START
Calculate GDI dataframe: DONE
PCA: START
PCA: DONE
Hierarchical clustering: START
Hierarchical clustering: DONE
GDI plot
Removed 0 low GDI genes (such as the fully-expressed) in GDI plot
Clusters 2 and 3 can be merged
Start merging smallest clusters: iteration 2
Differential Expression Analysis - START
* analysis of cluster: &#x27;1_4-merge&#x27; - START
* analysis of cluster: &#x27;1_4-merge&#x27; - DONE
* analysis of cluster: &#x27;2_3-merge&#x27; - START
* analysis of cluster: &#x27;2_3-merge&#x27; - DONE

Differential Expression Analysis - DONE
Created leafs ID for merging: 1_4-merge 2_3-merge
*1_4-merge_2_3-merge-merge
No genes/cells where dropped
Cotan analysis functions started
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [600] genes and [1200] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:600] to [1:600]
Estimate dispersion: DONE
dispersion | min: 0.2197265625 | max: 6.08056640625 | % negative: 0
Only analysis time 0.0540987928708394
Cotan genes&#x27; coex estimation started
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Total time 0.165117267767588
Only genes&#x27; coex time 0.101299671332041
Checking uniformity for the cluster &#x27;1_4-merge_2_3-merge-merge&#x27; with 1200 cells
Using S
Calculating S: START
Calculating S: DONE
Calculate GDI dataframe: START
Calculate GDI dataframe: DONE
PCA: START
PCA: DONE
Hierarchical clustering: START
Hierarchical clustering: DONE
GDI plot
Removed 0 low GDI genes (such as the fully-expressed) in GDI plot
Merging clusters 1_4-merge and 2_3-merge results in a too high GDI
The final merged clusterization contains [ 2 ] different clusters: 1_4-merge, 2_3-merge
Merging cells&#x27; uniform clustering: DONE
Cotan analysis functions started
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [600] genes and [626] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:600] to [1:600]
Estimate dispersion: DONE
dispersion | min: -0.03839111328125 | max: 17.40625 | % negative: 6.66666666666667
Only analysis time 0.0404324611028035
Cotan genes&#x27; coex estimation started
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 5.54631170271769e-05
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Total time 0.144292287031809
Only genes&#x27; coex time 0.0941291968027751
Using S
Calculating S: START
Calculating S: DONE
Calculate GDI dataframe: START
Calculate GDI dataframe: DONE
Cotan analysis functions started
Genes/cells selection done: dropped [0] genes and [0] cells
Working on [600] genes and [574] cells
Estimate dispersion: START
Effective number of cores used: 1
Executing 1 genes batches from [1:600] to [1:600]
Estimate dispersion: DONE
dispersion | min: -0.0340576171875 | max: 10.4375 | % negative: 4.33333333333333
Only analysis time 0.0399521986643473
Cotan genes&#x27; coex estimation started
Calculate genes&#x27; coex: START
Retrieving expected genes&#x27; contingency table
calculating NN.. done
calculating NY..YN..YY.. done
Calculating genes&#x27; coex normalization factor
Fraction of genes with very low expected contingency tables: 0.000105379922351636
Retrieving observed genes&#x27; yes/yes contingency table
calculating YY.. done
Estimating genes&#x27; coex
Calculate genes&#x27; coex: DONE
Total time 0.142826243241628
Only genes&#x27; coex time 0.0931988596916199
Using S
Calculating S: START
Calculating S: DONE
Calculate GDI dataframe: START
Calculate GDI dataframe: DONE
[ FAIL 0 | WARN 1 | SKIP 0 | PASS 315 ]

[ FAIL 0 | WARN 1 | SKIP 0 | PASS 315 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
433.062   2.583 429.449 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">COTAN.Rcheck/COTAN-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>COTAN</TD><TD style="text-align: right;">0.666</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.683</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>COTANObjectCreation</TD><TD style="text-align: right;">9.226</TD><TD style="text-align: right;">0.433</TD><TD style="text-align: right;">9.105</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>CalculatingCOEX</TD><TD style="text-align: right;">41.295</TD><TD style="text-align: right;"> 0.982</TD><TD style="text-align: right;">40.135</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ClustersList</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GenesCoexSpace</TD><TD style="text-align: right;">10.713</TD><TD style="text-align: right;"> 0.144</TD><TD style="text-align: right;">10.266</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>HandleMetaData</TD><TD style="text-align: right;">0.096</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.096</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>HandlingClusterizations</TD><TD style="text-align: right;">15.263</TD><TD style="text-align: right;"> 0.220</TD><TD style="text-align: right;">15.484</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>HeatmapPlots</TD><TD style="text-align: right;">38.780</TD><TD style="text-align: right;"> 0.852</TD><TD style="text-align: right;">37.768</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>LegacyFastSymmMatrix</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>LoggingFunctions</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ParametersEstimations</TD><TD style="text-align: right;">24.811</TD><TD style="text-align: right;"> 0.305</TD><TD style="text-align: right;">25.116</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>RawDataCleaning</TD><TD style="text-align: right;">2.610</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">2.626</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>RawDataGetters</TD><TD style="text-align: right;">0.102</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.106</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>UniformClusters</TD><TD style="text-align: right;">361.254</TD><TD style="text-align: right;">  0.849</TD><TD style="text-align: right;">359.885</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cosineDissimilarity</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getColorsVector</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
