<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for RLSeq on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><B>[<A href="..#R">R</A>]</B></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-10 10:04:39 -0000 (Wed, 10 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-03-12&nbsp;r83975)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">6211</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for RLSeq on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the RLSeq package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/RLSeq.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1700</B>/2194</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">RLSeq</A>&nbsp;1.7.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/RLSeq">landing page</A>)</SPAN><BR>Henry Miller<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-08&nbsp;19:11:19&nbsp;-0000&nbsp;(Mon,&nbsp;08&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/RLSeq</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">b1c60a6</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;15:34:48&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: RLSeq</TD></TR>
<TR><TD><B>Version</B>: 1.7.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:RLSeq.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings RLSeq_1.7.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-10 02:35:04 -0000 (Wed, 10 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-10 03:05:17 -0000 (Wed, 10 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 1813.4 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: RLSeq.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:RLSeq.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings RLSeq_1.7.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/RLSeq.Rcheck’
* using R Under development (unstable) (2023-03-12 r83975)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘RLSeq/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘RLSeq’ version ‘1.7.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘RLSeq’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking startup messages can be suppressed ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                       user system elapsed
report               70.286  3.524  87.958
corrAnalyze          68.342  0.719  78.249
RLSeq                44.369  2.559  70.330
predictCondition     17.799  1.424  25.727
plotEnrichment       15.906  1.536  19.435
txFeatureOverlap     12.370  0.748  15.331
geneAnnotation       11.043  0.985  17.457
featureEnrich        11.060  0.688  13.952
rlRegionTest          7.182  4.392  14.016
corrHeatmap          10.412  0.764  13.502
plotRLRegionOverlap  10.086  0.828  18.024
plotTxFeatureOverlap  7.804  0.763  13.060
RLRangesFromRLBase    5.823  0.565  13.391
noiseComparisonPlot   5.035  0.777   8.998
analyzeRLFS           2.890  0.351  10.199
RLRanges              1.035  0.012   5.759
noiseAnalyze          0.875  0.056   8.130
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘RLSeq.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: OK

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">RLSeq.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL RLSeq
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library’
* installing *source* package ‘RLSeq’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (RLSeq)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">RLSeq.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(RLSeq)
&gt; 
&gt; test_check(&quot;RLSeq&quot;)
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

see ?RLHub and browseVignettes(&#x27;RLHub&#x27;) for documentation
see ?RLHub and browseVignettes(&#x27;RLHub&#x27;) for documentation
see ?RLHub and browseVignettes(&#x27;RLHub&#x27;) for documentation
see ?RLHub and browseVignettes(&#x27;RLHub&#x27;) for documentation
[1] &quot;Note: The minimum p-value with only 30 permutations is 0.032258064516129. You should consider increasing the number of permutations.&quot;
see ?RLHub and browseVignettes(&#x27;RLHub&#x27;) for documentation
see ?RLHub and browseVignettes(&#x27;RLHub&#x27;) for documentation
see ?RLHub and browseVignettes(&#x27;RLHub&#x27;) for documentation
see ?RLHub and browseVignettes(&#x27;RLHub&#x27;) for documentation
see ?RLHub and browseVignettes(&#x27;RLHub&#x27;) for documentation
Loading required package: GenomicFeatures
Loading required package: GenomicRanges
Loading required package: AnnotationDbi
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

  1690 genes were dropped because they have exons located on both strands
  of the same reference sequence or on more than one reference sequence,
  so cannot be represented by a single genomic range.
  Use &#x27;single.strand.genes.only=FALSE&#x27; to get all the genes in a
  GRangesList object, or use suppressMessages() to suppress this message.
see ?RLHub and browseVignettes(&#x27;RLHub&#x27;) for documentation
[1] &quot;Note: The minimum p-value with only 30 permutations is 0.032258064516129. You should consider increasing the number of permutations.&quot;
see ?RLHub and browseVignettes(&#x27;RLHub&#x27;) for documentation
see ?RLHub and browseVignettes(&#x27;RLHub&#x27;) for documentation
see ?RLHub and browseVignettes(&#x27;RLHub&#x27;) for documentation
see ?RLHub and browseVignettes(&#x27;RLHub&#x27;) for documentation
see ?RLHub and browseVignettes(&#x27;RLHub&#x27;) for documentation
  1690 genes were dropped because they have exons located on both strands
  of the same reference sequence or on more than one reference sequence,
  so cannot be represented by a single genomic range.
  Use &#x27;single.strand.genes.only=FALSE&#x27; to get all the genes in a
  GRangesList object, or use suppressMessages() to suppress this message.
see ?RLHub and browseVignettes(&#x27;RLHub&#x27;) for documentation
[ FAIL 0 | WARN 134 | SKIP 0 | PASS 8 ]

[ FAIL 0 | WARN 134 | SKIP 0 | PASS 8 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
473.930  21.696 628.097 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">RLSeq.Rcheck/RLSeq-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>RLRanges</TD><TD style="text-align: right;">1.035</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">5.759</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>RLRangesFromRLBase</TD><TD style="text-align: right;"> 5.823</TD><TD style="text-align: right;"> 0.565</TD><TD style="text-align: right;">13.391</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>RLResults</TD><TD style="text-align: right;">0.166</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.174</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>RLSeq</TD><TD style="text-align: right;">44.369</TD><TD style="text-align: right;"> 2.559</TD><TD style="text-align: right;">70.330</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>analyzeRLFS</TD><TD style="text-align: right;"> 2.890</TD><TD style="text-align: right;"> 0.351</TD><TD style="text-align: right;">10.199</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>auxdata</TD><TD style="text-align: right;">0.147</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.156</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>available_genomes</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.037</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>corrAnalyze</TD><TD style="text-align: right;">68.342</TD><TD style="text-align: right;"> 0.719</TD><TD style="text-align: right;">78.249</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>corrHeatmap</TD><TD style="text-align: right;">10.412</TD><TD style="text-align: right;"> 0.764</TD><TD style="text-align: right;">13.502</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>featureEnrich</TD><TD style="text-align: right;">11.060</TD><TD style="text-align: right;"> 0.688</TD><TD style="text-align: right;">13.952</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>geneAnnotation</TD><TD style="text-align: right;">11.043</TD><TD style="text-align: right;"> 0.985</TD><TD style="text-align: right;">17.457</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genomeMasks</TD><TD style="text-align: right;">0.121</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.126</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>noiseAnalyze</TD><TD style="text-align: right;">0.875</TD><TD style="text-align: right;">0.056</TD><TD style="text-align: right;">8.130</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>noiseComparisonPlot</TD><TD style="text-align: right;">5.035</TD><TD style="text-align: right;">0.777</TD><TD style="text-align: right;">8.998</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotEnrichment</TD><TD style="text-align: right;">15.906</TD><TD style="text-align: right;"> 1.536</TD><TD style="text-align: right;">19.435</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotFingerprint</TD><TD style="text-align: right;">0.382</TD><TD style="text-align: right;">0.056</TD><TD style="text-align: right;">0.438</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotRLFSRes</TD><TD style="text-align: right;">0.539</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">0.583</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotRLRegionOverlap</TD><TD style="text-align: right;">10.086</TD><TD style="text-align: right;"> 0.828</TD><TD style="text-align: right;">18.024</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotTxFeatureOverlap</TD><TD style="text-align: right;"> 7.804</TD><TD style="text-align: right;"> 0.763</TD><TD style="text-align: right;">13.060</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>predictCondition</TD><TD style="text-align: right;">17.799</TD><TD style="text-align: right;"> 1.424</TD><TD style="text-align: right;">25.727</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>randomWindows</TD><TD style="text-align: right;">0.070</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.071</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>report</TD><TD style="text-align: right;">70.286</TD><TD style="text-align: right;"> 3.524</TD><TD style="text-align: right;">87.958</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rlRegionTest</TD><TD style="text-align: right;"> 7.182</TD><TD style="text-align: right;"> 4.392</TD><TD style="text-align: right;">14.016</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rlbaseNoiseAnalyze</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.021</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rlsampleTxOl</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>txFeatureOverlap</TD><TD style="text-align: right;">12.370</TD><TD style="text-align: right;"> 0.748</TD><TD style="text-align: right;">15.331</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
