<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for ORFik on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><B>[<A href="../index.html#O">O</A>]</B></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-08 01:15:18 -0000 (Wed, 08 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4164</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for ORFik on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the ORFik package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/ORFik.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1402</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">ORFik</A>&nbsp;1.19.7</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/ORFik">landing page</A>)</SPAN><BR>Haakon Tjeldnes<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-06&nbsp;00:12:45&nbsp;-0000&nbsp;(Mon,&nbsp;06&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/ORFik</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">d3ee070</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-01-30&nbsp;08:23:09&nbsp;-0000&nbsp;(Mon,&nbsp;30&nbsp;Jan&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: ORFik</TD></TR>
<TR><TD><B>Version</B>: 1.19.7</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:ORFik.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings ORFik_1.19.7.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-07 10:28:46 -0000 (Tue, 07 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-07 10:39:23 -0000 (Tue, 07 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 636.7 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: ORFik.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:ORFik.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings ORFik_1.19.7.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/ORFik.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘ORFik/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘ORFik’ version ‘1.19.7’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘ORFik’ can be installed ... OK
* used C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... NOTE
  installed size is  6.2Mb
  sub-directories of 1Mb or more:
    libs   2.8Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
Unexported objects imported by &#x27;:::&#x27; calls:
  ‘GenomicFeatures:::.merge_seqinfo_and_infer_missing_seqlengths’
  ‘IRanges:::regroupBySupergroup’ ‘S4Vectors:::normarg_mcols’
  ‘biomartr:::getENSEMBL.Seq’ ‘biomartr:::getENSEMBL.gtf’
  See the note in ?`:::` about the use of this operator.
There are ::: calls to the package&#x27;s namespace in its code. A package
  almost never needs to use ::: for its own objects:
  ‘find_url_ebi’ ‘revElementsF’ ‘save.fstwig’ ‘trimming.table’
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
DEG.plot.static: no visible binding for global variable ‘Regulation’
DEG.plot.static: no visible binding for global variable ‘meanCounts’
DEG.plot.static: no visible binding for global variable ‘LFC’
DTEG.plot: no visible binding for global variable ‘Regulation’
DTEG.plot: no visible binding for global variable ‘rna’
DTEG.plot: no visible binding for global variable ‘rfp’
QCplots: no visible binding for global variable ‘leaders’
QCplots: no visible binding for global variable ‘trailers’
QCstats.plot: no visible binding for global variable ‘variable’
QCstats.plot: no visible binding for global variable ‘sample_total’
QCstats.plot: no visible binding for global variable ‘value’
QCstats.plot: no visible global function definition for ‘.’
QCstats.plot: no visible binding for global variable ‘sample_id’
QCstats.plot: no visible binding for global variable ‘percentage’
QCstats.plot: no visible binding for global variable
  ‘perc_of_counts_per_sample’
QCstats.plot: no visible binding for global variable ‘read length’
RiboQC.plot: no visible binding for global variable ‘variable’
RiboQC.plot: no visible binding for global variable ‘sample_total’
RiboQC.plot: no visible binding for global variable ‘value’
RiboQC.plot: no visible global function definition for ‘.’
RiboQC.plot: no visible binding for global variable ‘sample_id’
RiboQC.plot: no visible binding for global variable ‘percentage’
RiboQC.plot: no visible binding for global variable ‘percent’
RiboQC.plot: no visible binding for global variable ‘frame’
STAR.multiQC: no visible binding for global variable ‘sample_id’
STAR.multiQC: no visible binding for global variable ‘value’
TOP.Motif.ecdf: no visible binding for global variable ‘seq1’
TOP.Motif.ecdf: no visible binding for global variable ‘TOP’
alignmentFeatureStatistics : &lt;anonymous&gt;: no visible binding for global
  variable ‘percentage_mrna_aligned’
alignmentFeatureStatistics : &lt;anonymous&gt;: no visible binding for global
  variable ‘mRNA’
alignmentFeatureStatistics : &lt;anonymous&gt;: no visible binding for global
  variable ‘percentage_tx_aligned’
alignmentFeatureStatistics : &lt;anonymous&gt;: no visible binding for global
  variable ‘Transcript’
alignmentFeatureStatistics : &lt;anonymous&gt;: no visible binding for global
  variable ‘ratio_cds_mrna’
alignmentFeatureStatistics : &lt;anonymous&gt;: no visible binding for global
  variable ‘CDS’
alignmentFeatureStatistics : &lt;anonymous&gt;: no visible binding for global
  variable ‘ratio_cds_leader’
alignmentFeatureStatistics : &lt;anonymous&gt;: no visible binding for global
  variable ‘LEADERS’
allFeaturesHelper: no visible binding for global variable ‘te’
allFeaturesHelper: no visible binding for global variable ‘fpkmRFP’
allFeaturesHelper: no visible binding for global variable ‘fpkmRNA’
allFeaturesHelper: no visible binding for global variable ‘countRFP’
allFeaturesHelper: no visible binding for global variable ‘entropyRFP’
allFeaturesHelper: no visible binding for global variable
  ‘disengagementScores’
allFeaturesHelper: no visible binding for global variable ‘RRS’
allFeaturesHelper: no visible binding for global variable ‘RSS’
allFeaturesHelper: no visible binding for global variable ‘ORFScores’
allFeaturesHelper: no visible binding for global variable ‘ioScore’
allFeaturesHelper: no visible binding for global variable
  ‘startCodonCoverage’
allFeaturesHelper: no visible binding for global variable
  ‘startRegionRelative’
allFeaturesHelper: no visible binding for global variable ‘kozak’
allFeaturesHelper: no visible binding for global variable ‘StartCodons’
allFeaturesHelper: no visible binding for global variable ‘StopCodons’
allFeaturesHelper: no visible binding for global variable
  ‘fractionLengths’
allFeaturesHelper: no visible binding for global variable ‘distORFCDS’
allFeaturesHelper: no visible binding for global variable ‘inFrameCDS’
allFeaturesHelper: no visible binding for global variable
  ‘isOverlappingCds’
allFeaturesHelper: no visible binding for global variable ‘rankInTx’
appendZeroes: no visible binding for global variable ‘frame’
appendZeroes: no visible binding for global variable ‘position’
appendZeroes: no visible binding for global variable ‘count’
artificial.orfs: no visible binding for global variable ‘random’
artificial.orfs: no visible binding for global variable ‘pick’
artificial.orfs: no visible global function definition for ‘.’
codon_usage: no visible binding for global variable ‘merged’
codon_usage: no visible binding for global variable ‘AA’
codon_usage: no visible binding for global variable ‘codon’
codon_usage_plot: no visible binding for global variable ‘seqs’
collapse.by.scores: no visible global function definition for ‘.’
collapse.fastq.internal: no visible binding for global variable ‘N’
cor_plot: no visible binding for global variable ‘Var2’
cor_plot: no visible binding for global variable ‘Var1’
cor_plot: no visible binding for global variable ‘Cor’
cor_table: no visible binding for global variable ‘Var1’
cor_table: no visible binding for global variable ‘Var2’
cor_table: no visible binding for global variable ‘Cor’
coverageHeatMap: no visible binding for global variable ‘position’
coverageHeatMap: no visible binding for global variable ‘fraction’
coverageScorings: no visible binding for global variable ‘count’
coverageScorings: no visible binding for global variable ‘zscore’
coverageScorings: no visible binding for global variable ‘windowMean’
coverageScorings: no visible binding for global variable ‘windowSD’
coverageScorings: no visible global function definition for ‘.’
coverageScorings: no visible binding for global variable ‘gene_sum’
coverageScorings: no visible binding for global variable ‘fraction’
coverage_random_access_file: no visible binding for global variable
  ‘position’
coverage_random_access_file: no visible binding for global variable
  ‘frame’
coverage_to_dt: no visible binding for global variable ‘frame’
coverage_to_dt: no visible binding for global variable ‘position’
detectRibosomeShifts: no visible global function definition for ‘.’
detectRibosomeShifts: no visible binding for global variable ‘size’
detectRibosomeShifts: no visible binding for global variable ‘fraction’
detectRibosomeShifts: no visible binding for global variable ‘pShifted’
detectRibosomeShifts: no visible binding for global variable
  ‘sum.count’
detectRibosomeShifts: no visible binding for global variable ‘count’
detectRibosomeShifts: no visible binding for global variable
  ‘frac.score’
download.SRA.metadata: no visible binding for global variable ‘spots’
download.SRA.metadata: no visible binding for global variable ‘MONTH’
download.SRA.metadata: no visible binding for global variable
  ‘ReleaseDate’
download.SRA.metadata: no visible binding for global variable ‘YEAR’
entropy: no visible binding for global variable ‘Hx’
entropy: no visible binding for global variable ‘codonSums’
entropy: no visible global function definition for ‘.’
filterExtremePeakGenes: no visible binding for global variable ‘count’
filterExtremePeakGenes: no visible binding for global variable
  ‘median_per_gene’
filterExtremePeakGenes: no visible global function definition for ‘.’
filterTranscripts: no visible binding for global variable ‘utr5_len’
filterTranscripts: no visible binding for global variable ‘utr3_len’
findNGSPairs: no visible global function definition for ‘.’
findNGSPairs: no visible binding for global variable ‘forward’
findPeaksPerGene: no visible binding for global variable ‘sum_per_gene’
findPeaksPerGene: no visible binding for global variable ‘count’
findPeaksPerGene: no visible binding for global variable
  ‘mean_per_gene’
findPeaksPerGene: no visible binding for global variable ‘sd_per_gene’
findPeaksPerGene: no visible binding for global variable ‘zscore’
findPeaksPerGene: no visible binding for global variable ‘gene_id’
find_url_ebi_safe: no visible binding for global variable
  ‘run_accession’
flankPerGroup: no visible global function definition for ‘.’
flankPerGroup: no visible binding for global variable ‘group_name’
floss: no visible binding for global variable ‘ORFGrouping’
floss: no visible binding for global variable ‘widths’
floss: no visible global function definition for ‘.’
floss: no visible binding for global variable ‘CDSGrouping’
floss: no visible binding for global variable ‘fraction.x’
floss: no visible binding for global variable ‘fraction.y’
gSort: no visible binding for global variable ‘grnames’
geneToSymbol: no visible binding for global variable ‘ensembl_gene_id’
getNGenesCoverage: no visible global function definition for ‘.’
getNGenesCoverage: no visible binding for global variable ‘fraction’
get_phix_genome: no visible binding for global variable ‘phix.url’
initiationScore: no visible global function definition for ‘.’
initiationScore: no visible binding for global variable ‘dif’
initiationScore: no visible binding for global variable ‘fraction’
initiationScore: no visible binding for global variable ‘difPer’
isPeriodic: no visible binding for global variable ‘spec’
kozakHeatmap: no visible global function definition for ‘.’
kozakHeatmap: no visible binding for global variable ‘variable’
kozakHeatmap: no visible binding for global variable ‘value’
kozakHeatmap: no visible binding for global variable
  ‘count_seq_pos_with_count’
kozakHeatmap: no visible binding for global variable ‘median_score’
kozak_IR_ranking: no visible global function definition for ‘.’
kozak_IR_ranking: no visible binding for global variable ‘IR’
kozak_IR_ranking: no visible binding for global variable
  ‘upstream_kozak_strength’
kozak_IR_ranking: no visible binding for global variable ‘count’
kozak_IR_ranking: no visible binding for global variable ‘mean_IR’
list.genomes: no visible binding for global variable ‘STAR_index’
longestORFs: no visible global function definition for ‘.’
metaWindow: no visible binding for global variable ‘position’
metaWindow: no visible binding for global variable ‘frame’
metadata.autnaming: no visible binding for global variable
  ‘LIBRARYTYPE’
metadata.autnaming: no visible binding for global variable
  ‘LibraryStrategy’
orfFrameDistributions : &lt;anonymous&gt;: no visible binding for global
  variable ‘fraction’
orfFrameDistributions: no visible binding for global variable ‘percent’
orfFrameDistributions: no visible binding for global variable
  ‘fraction’
orfFrameDistributions: no visible binding for global variable
  ‘percent_length’
orfFrameDistributions: no visible global function definition for ‘.’
orfFrameDistributions: no visible binding for global variable
  ‘best_frame’
orfScore: no visible binding for global variable ‘frame’
orfScore: no visible binding for global variable ‘frame_one_RP’
orfScore: no visible binding for global variable ‘frame_two_RP’
pSitePlot: no visible binding for global variable ‘count’
pSitePlot: no visible binding for global variable ‘frame’
pSitePlot: no visible binding for global variable ‘position’
pcaExperiment: no visible binding for global variable ‘PC1’
pcaExperiment: no visible binding for global variable ‘PC2’
readLengthTable: no visible binding for global variable
  ‘counts_per_sample’
readLengthTable: no visible binding for global variable
  ‘perc_of_counts_per_sample’
regionPerReadLength : &lt;anonymous&gt;: no visible binding for global
  variable ‘fraction’
remakeTxdbExonIds: no visible global function definition for ‘.’
remakeTxdbExonIds: no visible binding for global variable ‘chr’
removeTxdbExons: no visible binding for global variable ‘exon_rank’
removeTxdbExons: no visible binding for global variable ‘ranks’
ribo_fft: no visible binding for global variable ‘fraction’
ribo_fft_plot: no visible binding for global variable ‘periods’
ribo_fft_plot: no visible binding for global variable ‘amplitude’
rnaNormalize: no visible binding for global variable ‘feature’
sample_info_append_SRA: no visible binding for global variable ‘Run’
scaledWindowPositions: no visible binding for global variable
  ‘scalingFactor’
scaledWindowPositions: no visible binding for global variable
  ‘position’
scaledWindowPositions: no visible global function definition for ‘.’
scoreSummarizedExperiment: no visible global function definition for
  ‘rowSums2’
seq_usage: no visible binding for global variable ‘variable’
seq_usage: no visible binding for global variable ‘codon_sum’
seq_usage: no visible global function definition for ‘.’
seq_usage: no visible binding for global variable ‘gene_sum’
seq_usage: no visible binding for global variable
  ‘N_AA_of_type_per_gene’
seq_usage: no visible binding for global variable ‘as_prob_normalized’
seq_usage: no visible binding for global variable ‘N_total’
seq_usage: no visible binding for global variable ‘N’
seq_usage: no visible binding for global variable ‘mean_txNorm’
seq_usage: no visible binding for global variable ‘sum_txNorm’
seq_usage: no visible binding for global variable ‘dispersion’
seq_usage: no visible binding for global variable ‘dispersion_txNorm’
seq_usage: no visible binding for global variable ‘var_txNorm’
seq_usage: no visible binding for global variable ‘mean_percentage’
seq_usage: no visible binding for global variable ‘mean_txNorm_prob’
seq_usage: no visible binding for global variable
  ‘mean_txNorm_percentage’
seq_usage: no visible binding for global variable
  ‘relative_to_max_score’
shiftPlots : &lt;anonymous&gt;: no visible binding for global variable
  ‘frame’
shiftPlots : &lt;anonymous&gt;: no visible binding for global variable
  ‘position’
te.plot: no visible global function definition for ‘rowMin’
te.plot: no visible binding for global variable ‘variable’
te.plot: no visible binding for global variable ‘LFC_TE’
te.plot: no visible binding for global variable ‘rfp_log2’
te.plot: no visible binding for global variable ‘rna_log2’
te.plot: no visible binding for global variable ‘rna_log10’
te.table: no visible global function definition for ‘rowMin’
te.table: no visible binding for global variable ‘variable’
te.table: no visible binding for global variable ‘TE_log2’
te.table: no visible binding for global variable ‘rfp_log2’
te.table: no visible binding for global variable ‘rna_log2’
te_rna.plot: no visible binding for global variable ‘subtitle’
te_rna.plot: no visible binding for global variable ‘rna_log10’
te_rna.plot: no visible binding for global variable ‘TE_log2’
topMotif: no visible binding for global variable ‘seq1’
topMotif: no visible binding for global variable ‘seq2’
topMotif: no visible binding for global variable ‘seq3’
topMotif: no visible binding for global variable ‘seq4’
topMotif: no visible binding for global variable ‘seq5’
transcriptWindow: no visible binding for global variable ‘fractions’
transcriptWindow: no visible binding for global variable ‘feature’
windowCoveragePlot: no visible binding for global variable ‘feature’
windowCoveragePlot: no visible binding for global variable ‘fraction’
windowCoveragePlot: no visible binding for global variable
  ‘fraction_min’
windowCoveragePlot: no visible binding for global variable ‘position’
collapseDuplicatedReads,GAlignmentPairs: no visible global function
  definition for ‘.’
collapseDuplicatedReads,GAlignmentPairs: no visible binding for global
  variable ‘start1’
collapseDuplicatedReads,GAlignmentPairs: no visible binding for global
  variable ‘start2’
collapseDuplicatedReads,GAlignmentPairs: no visible binding for global
  variable ‘cigar1’
collapseDuplicatedReads,GAlignmentPairs: no visible binding for global
  variable ‘cigar2’
collapseDuplicatedReads,GAlignments: no visible global function
  definition for ‘.’
collapseDuplicatedReads,GRanges: no visible binding for global variable
  ‘size’
collapseDuplicatedReads,GRanges: no visible global function definition
  for ‘.’
collapseDuplicatedReads,data.table: no visible global function
  definition for ‘.’
collapseDuplicatedReads,data.table: no visible binding for global
  variable ‘size’
design,experiment: no visible binding for global variable ‘..formula’
seqlevels,experiment: no visible binding for global variable ‘df’
show,covRleList: no visible global function definition for ‘head’
Undefined global functions or variables:
  . ..formula AA CDS CDSGrouping Cor Hx IR LEADERS LFC LFC_TE
  LIBRARYTYPE LibraryStrategy MONTH N N_AA_of_type_per_gene N_total
  ORFGrouping ORFScores PC1 PC2 RRS RSS Regulation ReleaseDate Run
  STAR_index StartCodons StopCodons TE_log2 TOP Transcript Var1 Var2
  YEAR amplitude as_prob_normalized best_frame chr cigar1 cigar2 codon
  codonSums codon_sum count countRFP count_seq_pos_with_count
  counts_per_sample df dif difPer disengagementScores dispersion
  dispersion_txNorm distORFCDS ensembl_gene_id entropyRFP exon_rank
  feature forward fpkmRFP fpkmRNA frac.score fraction fraction.x
  fraction.y fractionLengths fraction_min fractions frame frame_one_RP
  frame_two_RP gene_id gene_sum grnames group_name head inFrameCDS
  ioScore isOverlappingCds kozak leaders mRNA meanCounts mean_IR
  mean_per_gene mean_percentage mean_txNorm mean_txNorm_percentage
  mean_txNorm_prob median_per_gene median_score merged pShifted
  perc_of_counts_per_sample percent percent_length percentage
  percentage_mrna_aligned percentage_tx_aligned periods phix.url pick
  position random rankInTx ranks ratio_cds_leader ratio_cds_mrna read
  length relative_to_max_score rfp rfp_log2 rna rna_log10 rna_log2
  rowMin rowSums2 run_accession sample_id sample_total scalingFactor
  sd_per_gene seq1 seq2 seq3 seq4 seq5 seqs size spec spots start1
  start2 startCodonCoverage startRegionRelative subtitle sum.count
  sum_per_gene sum_txNorm te trailers upstream_kozak_strength utr3_len
  utr5_len value var_txNorm variable widths windowMean windowSD zscore
Consider adding
  importFrom(&quot;base&quot;, &quot;length&quot;)
  importFrom(&quot;graphics&quot;, &quot;frame&quot;)
  importFrom(&quot;stats&quot;, &quot;df&quot;)
  importFrom(&quot;utils&quot;, &quot;head&quot;)
to your NAMESPACE file.
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... WARNING
Killed
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... OK
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘ORFik-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: codon_usage_plot
&gt; ### Title: Plot codon_usage
&gt; ### Aliases: codon_usage_plot
&gt; 
&gt; ### ** Examples
&gt; 
&gt; df &lt;- ORFik.template.experiment()[9:10,] # Subset to 2 Ribo-seq libs
&gt; ## For multiple libs
&gt; res2 &lt;- codon_usage_exp(df, outputLibs(df, type = &quot;pshifted&quot;, output.mode = &quot;list&quot;),
+                  min_counts_cds_filter = 10)
Loading default 80S counts, update count.folder to pshifted if wanted?
Killed
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  The following object is masked from &#x27;package:base&#x27;:
  
      strsplit
  
  Loading required package: Rsamtools
  
  Attaching package: &#x27;ORFik&#x27;
  
  The following object is masked from &#x27;package:graphics&#x27;:
  
      symbols
  
  &gt; 
  &gt; test_check(&quot;ORFik&quot;)
  Killed
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘Annotation_Alignment.Rmd’ using ‘UTF-8’... OK
  ‘ORFikExperiment.Rmd’ using ‘UTF-8’... OK
  ‘ORFikOverview.Rmd’ using ‘UTF-8’... OK
  ‘Ribo-seq_pipeline.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... ERROR
Error(s) in re-building vignettes:
  ...
--- re-building ‘Annotation_Alignment.Rmd’ using rmarkdown
--- finished re-building ‘Annotation_Alignment.Rmd’

--- re-building ‘ORFikExperiment.Rmd’ using rmarkdown
The magick package is required to crop &quot;/home/biocbuild/bbs-3.17-bioc/meat/ORFik.Rcheck/vign_test/ORFik/vignettes/ORFikExperiment_files/figure-html/unnamed-chunk-21-1.png&quot; but not available.
The magick package is required to crop &quot;/home/biocbuild/bbs-3.17-bioc/meat/ORFik.Rcheck/vign_test/ORFik/vignettes/ORFikExperiment_files/figure-html/unnamed-chunk-21-2.png&quot; but not available.
--- finished re-building ‘ORFikExperiment.Rmd’

--- re-building ‘ORFikOverview.Rmd’ using rmarkdown
Killed
--- re-building ‘Ribo-seq_pipeline.Rmd’ using rmarkdown
--- finished re-building ‘Ribo-seq_pipeline.Rmd’

SUMMARY: processing the following file failed:
  ‘ORFikOverview.Rmd’

Error: Vignette re-building failed.
Execution halted

* checking PDF version of manual ... OK
* DONE

Status: 3 ERRORs, 1 WARNING, 4 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/ORFik.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">ORFik.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL ORFik
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘ORFik’ ...
** using staged installation
** libs
using C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
using C++11
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c findORFsHelpers.cpp -o findORFsHelpers.o
findORFsHelpers.cpp: In function ‘Rcpp::IntegerMatrix orfs_as_matrix(std::string&amp;, std::string, std::string, int)’:
findORFsHelpers.cpp:214:21: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
  214 |   for (int i = 0; i &lt; uorfSize/2; i++) {
      |                   ~~^~~~~~~~~~~~
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c findOrfs.cpp -o findOrfs.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c findOrfsFasta.cpp -o findOrfsFasta.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c pmapFromTranscripts.cpp -o pmapFromTranscripts.o
pmapFromTranscripts.cpp: In function ‘Rcpp::List pmapFromTranscriptsCPP(const std::vector&lt;int&gt;&amp;, const std::vector&lt;int&gt;&amp;, const std::vector&lt;int&gt;&amp;, const std::vector&lt;int&gt;&amp;, const std::vector&lt;int&gt;&amp;, const char&amp;, bool)’:
pmapFromTranscripts.cpp:151:20: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
  151 |   for(auto i = 0;i &lt; xSize; i++){
      |                  ~~^~~~~~~
pmapFromTranscripts.cpp:155:21: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
  155 |   for(auto i = 0; i &lt; indexSize; i++){
      |                   ~~^~~~~~~~~~~
pmapFromTranscripts.cpp:161:21: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
  161 |   for(auto i = 1; i &lt; indexSize; i++){
      |                   ~~^~~~~~~~~~~
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c pmapToTranscripts.cpp -o pmapToTranscripts.o
pmapToTranscripts.cpp: In function ‘void pmapToPositive(vi&amp;, const int&amp;, int&amp;, const vi&amp;, const vi&amp;, const vi&amp;, const vi&amp;, const vi&amp;, const vi&amp;, const vi&amp;, const vi&amp;)’:
pmapToTranscripts.cpp:22:7: warning: variable ‘currentWidth’ set but not used [-Wunused-but-set-variable]
   22 |   int currentWidth = 0;
      |       ^~~~~~~~~~~~
pmapToTranscripts.cpp: In function ‘void pmapToNegative(vi&amp;, const int&amp;, int&amp;, const vi&amp;, const vi&amp;, const vi&amp;, const vi&amp;, const vi&amp;, const vi&amp;, const vi&amp;, const vi&amp;)’:
pmapToTranscripts.cpp:68:7: warning: variable ‘currentWidth’ set but not used [-Wunused-but-set-variable]
   68 |   int currentWidth = 0;
      |       ^~~~~~~~~~~~
pmapToTranscripts.cpp: In function ‘Rcpp::List pmapToTranscriptsCPP(const std::vector&lt;int&gt;&amp;, const std::vector&lt;int&gt;&amp;, const std::vector&lt;int&gt;&amp;, const std::vector&lt;int&gt;&amp;, const std::vector&lt;int&gt;&amp;, const char&amp;, const std::vector&lt;int&gt;&amp;)’:
pmapToTranscripts.cpp:126:20: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
  126 |   for(auto i = 0;i &lt; xSize; i++){ // Width per exon in x
      |                  ~~^~~~~~~
pmapToTranscripts.cpp:130:21: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
  130 |   for(auto i = 0; i &lt; indexSize; i++){ // Width per exon in tx
      |                   ~~^~~~~~~~~~~
pmapToTranscripts.cpp:136:21: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
  136 |   for(auto i = 1; i &lt; indexSize; i++){
      |                   ~~^~~~~~~~~~~
pmapToTranscripts.cpp:147:22: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
  147 |   for (auto i = 1; i &lt; 2 * xSize; i = i + 2) {
      |                    ~~^~~~~~~~~~~
g++ -std=gnu++11 -shared -L/home/biocbuild/bbs-3.17-bioc/R/lib -L/usr/local/lib -o ORFik.so RcppExports.o findORFsHelpers.o findOrfs.o findOrfsFasta.o pmapFromTranscripts.o pmapToTranscripts.o -L/home/biocbuild/bbs-3.17-bioc/R/lib -lR
installing to /home/biocbuild/bbs-3.17-bioc/R/library/00LOCK-ORFik/00new/ORFik/libs
** R
** inst
** byte-compile and prepare package for lazy loading
Creating a new generic function for ‘symbols’ in package ‘ORFik’
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (ORFik)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">ORFik.Rcheck/tests/testthat.Rout.fail</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(ORFik)
Loading required package: IRanges
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: GenomicRanges
Loading required package: GenomeInfoDb
Loading required package: GenomicAlignments
Loading required package: SummarizedExperiment
Loading required package: MatrixGenerics
Loading required package: matrixStats

Attaching package: &#x27;MatrixGenerics&#x27;

The following objects are masked from &#x27;package:matrixStats&#x27;:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars

Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


Attaching package: &#x27;Biobase&#x27;

The following object is masked from &#x27;package:MatrixGenerics&#x27;:

    rowMedians

The following objects are masked from &#x27;package:matrixStats&#x27;:

    anyMissing, rowMedians

Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: Rsamtools

Attaching package: &#x27;ORFik&#x27;

The following object is masked from &#x27;package:graphics&#x27;:

    symbols

&gt; 
&gt; test_check(&quot;ORFik&quot;)
Killed
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">ORFik.Rcheck/ORFik-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>DEG.analysis</TD><TD style="text-align: right;">0.694</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.762</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DEG.plot.static</TD><TD style="text-align: right;">0.497</TD><TD style="text-align: right;">0.060</TD><TD style="text-align: right;">0.635</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DTEG.analysis</TD><TD style="text-align: right;">0.476</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.580</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DTEG.plot</TD><TD style="text-align: right;">0.473</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.585</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ORFik.template.experiment</TD><TD style="text-align: right;">0.517</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.043</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ORFik.template.experiment.zf</TD><TD style="text-align: right;">0.069</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.104</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ORFikQC</TD><TD style="text-align: right;">0.446</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.529</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>QCreport</TD><TD style="text-align: right;">0.535</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.591</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>QCstats</TD><TD style="text-align: right;">0.457</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.495</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>QCstats.plot</TD><TD style="text-align: right;">1.451</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">2.202</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>RiboQC.plot</TD><TD style="text-align: right;">0.445</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.598</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>STAR.align.folder</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>STAR.align.single</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>STAR.index</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>STAR.install</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>STAR.remove.crashed.genome</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TOP.Motif.ecdf</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>artificial.orfs</TD><TD style="text-align: right;">0.436</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.455</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>asTX</TD><TD style="text-align: right;">0.879</TD><TD style="text-align: right;">0.096</TD><TD style="text-align: right;">1.140</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>assignTSSByCage</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bamVarName</TD><TD style="text-align: right;">0.971</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">1.274</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>codon_usage</TD><TD style="text-align: right;">4.096</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">4.740</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>codon_usage_exp</TD><TD style="text-align: right;">11.014</TD><TD style="text-align: right;"> 0.393</TD><TD style="text-align: right;">15.790</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
