<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for TitanCNA on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><B>[<A href="../index.html#T">T</A>]</B></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-01-29 16:34:18 -0000 (Sun, 29 Jan 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4021</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for TitanCNA on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the TitanCNA package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/TitanCNA.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>2038</B>/2162</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">TitanCNA</A>&nbsp;1.37.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/TitanCNA">landing page</A>)</SPAN><BR>Gavin Ha<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-01-27&nbsp;13:09:27&nbsp;-0000&nbsp;(Fri,&nbsp;27&nbsp;Jan&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/TitanCNA</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">5c82ab3</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:09:35&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: TitanCNA</TD></TR>
<TR><TD><B>Version</B>: 1.37.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:TitanCNA.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings TitanCNA_1.37.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-01-29 10:25:49 -0000 (Sun, 29 Jan 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-01-29 10:32:34 -0000 (Sun, 29 Jan 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 404.4 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: TitanCNA.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:TitanCNA.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings TitanCNA_1.37.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/TitanCNA.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘TitanCNA/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘TitanCNA’ version ‘1.37.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘TitanCNA’ can be installed ... WARNING
Found the following significant warnings:
  Warning: replacing previous import ‘GenomicRanges::shift’ by ‘data.table::shift’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘data.table::first’ by ‘dplyr::first’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘data.table::between’ by ‘dplyr::between’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘IRanges::desc’ by ‘dplyr::desc’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘IRanges::slice’ by ‘dplyr::slice’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘BiocGenerics::combine’ by ‘dplyr::combine’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘data.table::last’ by ‘dplyr::last’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘GenomicRanges::setdiff’ by ‘dplyr::setdiff’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘GenomeInfoDb::intersect’ by ‘dplyr::intersect’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘IRanges::collapse’ by ‘dplyr::collapse’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘GenomicRanges::union’ by ‘dplyr::union’ when loading ‘TitanCNA’
  Warning: replacing previous import ‘dplyr::select’ by ‘VariantAnnotation::select’ when loading ‘TitanCNA’
See ‘/home/biocbuild/bbs-3.17-bioc/meat/TitanCNA.Rcheck/00install.out’ for details.
* used C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... WARNING
Invalid citation information in ‘inst/CITATION’:
  Warning message:
  In utils::packageDescription(basename(dir), dirname(dir)) :
    no package &#x27;TitanCNA&#x27; was found
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... WARNING
Error in library(TitanCNA, lib.loc = &quot;/home/biocbuild/bbs-3.17-bioc/R/library&quot;) : 
  there is no package called ‘TitanCNA’
Execution halted

It looks like this package has a loading problem when not on .libPaths:
see the messages for details.
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
computeSDbwIndex: no visible binding for global variable
  ‘ClonalCluster’
correctIntegerCN: no visible binding for global variable
  ‘Median_HaplotypeRatio’
correctIntegerCN: no visible binding for global variable ‘Chromosome’
correctIntegerCN: no visible binding for global variable ‘Copy_Number’
correctIntegerCN: no visible binding for global variable
  ‘logR_Copy_Number’
correctIntegerCN: no visible binding for global variable ‘Median_logR’
correctIntegerCN: no visible binding for global variable
  ‘Cellular_Prevalence’
correctIntegerCN: no visible binding for global variable ‘Chr’
correctIntegerCN: no visible binding for global variable ‘LogRatio’
correctIntegerCN: no visible binding for global variable
  ‘CellularPrevalence’
correctIntegerCN: no visible binding for global variable
  ‘Corrected_Ratio’
correctIntegerCN: no visible binding for global variable
  ‘Corrected_Copy_Number’
correctIntegerCN: no visible binding for global variable
  ‘Corrected_Call’
correctIntegerCN: no visible binding for global variable ‘TITAN_call’
correctIntegerCN: no visible binding for global variable
  ‘Corrected_MajorCN’
correctIntegerCN: no visible binding for global variable ‘MajorCN’
correctIntegerCN: no visible binding for global variable
  ‘Corrected_MinorCN’
correctIntegerCN: no visible binding for global variable ‘MinorCN’
correctIntegerCN: no visible binding for global variable ‘CopyNumber’
correctIntegerCN: no visible binding for global variable ‘TITANcall’
correctReadDepth: no visible global function definition for ‘queryHits’
correctReadcount: no visible global function definition for ‘loess’
correctReadcount: no visible global function definition for ‘predict’
correctReadcount: no visible global function definition for ‘approxfun’
correctReadcount: no visible global function definition for ‘lowess’
extendSegments: no visible binding for global variable ‘Start’
extendSegments: no visible binding for global variable ‘End’
extendSegments: no visible binding for global variable ‘Chromosome’
extendSegments: no visible binding for global variable ‘Start.snp’
extendSegments: no visible binding for global variable ‘End.snp’
extendSegments: no visible binding for global variable ‘Start.telo’
extendSegments: no visible binding for global variable ‘seq.info’
extractAlleleReadCounts: no visible global function definition for
  ‘PileupParam’
extractAlleleReadCounts: no visible global function definition for
  ‘BcfFile’
extractAlleleReadCounts: no visible global function definition for
  ‘scanBcf’
extractAlleleReadCounts: no visible global function definition for
  ‘ScanBamParam’
extractAlleleReadCounts: no visible global function definition for
  ‘scanBamFlag’
extractAlleleReadCounts: no visible global function definition for
  ‘BamFile’
extractAlleleReadCounts: no visible global function definition for
  ‘pileup’
extractAlleleReadCounts: no visible global function definition for
  ‘write.table’
getHaplotypesFromVCF: no visible global function definition for
  ‘rowRanges&lt;-’
getHaplotypesFromVCF: no visible global function definition for
  ‘rowRanges’
getHaplotypesFromVCF: no visible global function definition for
  ‘na.omit’
getHaplotypesFromVCF: no visible global function definition for
  ‘unstrsplit’
getHaplotypesFromVCF: no visible global function definition for
  ‘queryHits’
getHaplotypesFromVCF: no visible global function definition for
  ‘DataFrame’
getOverlap: no visible global function definition for ‘as’
getOverlap: no visible global function definition for ‘queryHits’
getOverlap: no visible global function definition for ‘subjectHits’
getPositionOverlap: no visible global function definition for ‘as’
getSubcloneProfiles: no visible global function definition for
  ‘read.delim’
getSubcloneProfiles: no visible binding for global variable
  ‘CopyNumber’
getSubcloneProfiles: no visible binding for global variable ‘TITANcall’
keepChr: no visible global function definition for ‘as’
loadAlleleCounts: no visible global function definition for
  ‘read.delim’
loadBXcountsFromBEDDir: no visible binding for global variable
  ‘BXcounts’
loadBXcountsFromBEDDir: no visible binding for global variable ‘BX’
loadBXcountsFromBEDDir: no visible global function definition for
  ‘RangedData’
loadHaplotypeAlleleCounts: no visible global function definition for
  ‘read.delim’
loadHaplotypeAlleleCounts: no visible global function definition for
  ‘subjectHits’
loadHaplotypeAlleleCounts: no visible global function definition for
  ‘as’
loadHaplotypeAlleleCounts: no visible binding for global variable
  ‘phasedAlleleFraction’
loadHaplotypeAlleleCounts: no visible binding for global variable
  ‘phasedCount’
loadHaplotypeAlleleCounts: no visible binding for global variable
  ‘depth’
loadHaplotypeAlleleCounts: no visible binding for global variable
  ‘SNPs’
loadHaplotypeAlleleCounts: no visible binding for global variable
  ‘HaplotypeFraction’
loadHaplotypeAlleleCounts: no visible binding for global variable
  ‘HaplotypeDepth.sum’
loadHaplotypeAlleleCounts: no visible binding for global variable
  ‘HaplotypeBinDepth.sum’
loadHaplotypeAlleleCounts: no visible binding for global variable
  ‘HaplotypeDepth.mean’
loadHaplotypeAlleleCounts: no visible binding for global variable
  ‘HaplotypeBinDepth.mean’
loadHaplotypeAlleleCounts: no visible binding for global variable
  ‘phaseSet’
loadHaplotypeAlleleCounts: no visible binding for global variable
  ‘haplotypeBin’
loadHaplotypeAlleleCounts: no visible binding for global variable
  ‘HaplotypeFraction.symmetric’
loadHaplotypeAlleleCounts: no visible binding for global variable
  ‘HaplotypeDepth.sum.symmetric’
loadHaplotypeAlleleCounts: no visible binding for global variable
  ‘HaplotypeDepth.mean.symmetric’
loadHaplotypeAlleleCounts: no visible global function definition for
  ‘.’
loadHaplotypeAlleleCounts: no visible global function definition for
  ‘na.omit’
loadHaplotypeAlleleCounts: no visible binding for global variable
  ‘phasedCount.haploSymmetric’
loadHaplotypeAlleleCounts: no visible binding for global variable
  ‘nonRef’
loadHaplotypeAlleleCounts: no visible binding for global variable
  ‘phaseSet.aggr’
loadHaplotypeAlleleCounts: no visible binding for global variable
  ‘HaplotypeRatio’
loadHaplotypeAlleleCounts: no visible binding for global variable
  ‘tumDepth’
loadReadCountsFromBed: no visible global function definition for
  ‘excludeCentromere’
loadReadCountsFromBed: no visible global function definition for
  ‘filterByTargetedSequences’
mergeSegsByCol: no visible binding for global variable ‘Median_Ratio’
mergeSegsByCol: no visible binding for global variable ‘Median_logR’
mergeSegsByCol: no visible binding for global variable ‘End’
mergeSegsByCol: no visible binding for global variable ‘Length.snp.’
outlierObslik: no visible global function definition for ‘dunif’
outputModelParameters: no visible global function definition for
  ‘write.table’
outputTitanResults: no visible global function definition for
  ‘write.table’
outputTitanSegments: no visible binding for global variable ‘Sample’
plotAllelicCN: no visible binding for global variable ‘Allele.1’
plotAllelicCN: no visible binding for global variable ‘LogRatio’
plotAllelicCN: no visible binding for global variable ‘Allele.2’
plotAllelicCN: no visible binding for global variable ‘Chr’
plotAllelicCN: no visible binding for global variable ‘TITANcall’
plotAllelicCN: no visible global function definition for ‘par’
plotAllelicCN: no visible binding for global variable ‘CopyNumber’
plotAllelicCN: no visible global function definition for ‘points’
plotAllelicCN: no visible global function definition for ‘lines’
plotAllelicRatio: no visible binding for global variable ‘Chr’
plotAllelicRatio: no visible binding for global variable ‘TITANcall’
plotAllelicRatio: no visible global function definition for ‘par’
plotAllelicRatio: no visible binding for global variable ‘AllelicRatio’
plotAllelicRatio: no visible global function definition for ‘lines’
plotCNlogRByChr: no visible binding for global variable ‘LogRatio’
plotCNlogRByChr: no visible binding for global variable ‘Median_logR’
plotCNlogRByChr: no visible binding for global variable ‘Chr’
plotCNlogRByChr: no visible binding for global variable ‘TITANcall’
plotCNlogRByChr: no visible global function definition for ‘par’
plotCNlogRByChr: no visible global function definition for ‘lines’
plotCNlogRByChr: no visible binding for global variable ‘Chromosome’
plotCNlogRByChr : &lt;anonymous&gt;: no visible global function definition
  for ‘lines’
plotCNlogRByChr: no visible binding for global variable
  ‘End_Position.bp.’
plotCNlogRByChr: no visible binding for global variable
  ‘Start_Position.bp.’
plotChrLines: no visible global function definition for ‘lines’
plotChrLines: no visible global function definition for ‘axis’
plotClonalFrequency: no visible binding for global variable
  ‘ClonalCluster’
plotClonalFrequency: no visible binding for global variable
  ‘CellularPrevalence’
plotClonalFrequency: no visible binding for global variable ‘TITANcall’
plotClonalFrequency: no visible binding for global variable ‘Chr’
plotClonalFrequency: no visible global function definition for ‘par’
plotClonalFrequency: no visible global function definition for ‘lines’
plotClonalFrequency: no visible global function definition for ‘mtext’
plotGeneAnnotation: no visible global function definition for ‘abline’
plotGeneAnnotation: no visible global function definition for ‘mtext’
plotHaplotypeFraction: no visible binding for global variable
  ‘HaplotypeRatio.1’
plotHaplotypeFraction: no visible binding for global variable
  ‘HaplotypeRatio’
plotHaplotypeFraction: no visible binding for global variable
  ‘HaplotypeRatio.2’
plotHaplotypeFraction: no visible binding for global variable ‘Chr’
plotHaplotypeFraction: no visible binding for global variable
  ‘TITANcall’
plotHaplotypeFraction: no visible global function definition for ‘par’
plotHaplotypeFraction: no visible global function definition for
  ‘points’
plotHaplotypeFraction: no visible binding for global variable
  ‘AllelicRatio’
plotHaplotypeFraction: no visible global function definition for
  ‘lines’
plotIdiogram.hg38: no visible global function definition for ‘par’
plotIdiogram.hg38: no visible binding for global variable ‘lsegments’
plotIdiogram.hg38: no visible binding for global variable ‘lpolygon’
plotIdiogram.hg38: no visible global function definition for ‘axis’
plotIdiogram.hg38: no visible global function definition for ‘text’
plotSegmentMedians: no visible binding for global variable ‘Chromosome’
plotSegmentMedians: no visible binding for global variable ‘TITAN_call’
plotSegmentMedians: no visible global function definition for ‘par’
plotSegmentMedians: no visible binding for global variable
  ‘End_Position.bp.’
plotSegmentMedians: no visible global function definition for ‘.’
plotSegmentMedians: no visible binding for global variable
  ‘Start_Position.bp.’
plotSegmentMedians: no visible binding for global variable ‘MajorCN’
plotSegmentMedians: no visible binding for global variable ‘MinorCN’
plotSegmentMedians : &lt;anonymous&gt;: no visible global function definition
  for ‘lines’
plotSegmentMedians: no visible global function definition for ‘lines’
plotSegmentMedians: no visible binding for global variable
  ‘Copy_Number’
plotSubcloneProfiles: no visible binding for global variable ‘Chr’
plotSubcloneProfiles: no visible global function definition for ‘par’
plotSubcloneProfiles: no visible binding for global variable
  ‘CopyNumber’
plotSubcloneProfiles: no visible global function definition for ‘axis’
plotSubcloneProfiles: no visible global function definition for
  ‘points’
plotSubcloneProfiles: no visible global function definition for ‘mtext’
plotSubcloneProfiles: no visible global function definition for ‘lines’
printSDbw: no visible global function definition for ‘write.table’
removeCentromereSegs: no visible binding for global variable
  ‘Chromosome’
removeCentromereSegs: no visible binding for global variable ‘Start’
removeCentromereSegs: no visible binding for global variable ‘End’
removeEmptyClusters: no visible global function definition for ‘tail’
runEMclonalCN: no visible binding for global variable ‘head’
updateParameters: no visible global function definition for ‘uniroot’
wigToRangedData: no visible global function definition for ‘RangedData’
Undefined global functions or variables:
  . Allele.1 Allele.2 AllelicRatio BX BXcounts BamFile BcfFile
  CellularPrevalence Cellular_Prevalence Chr Chromosome ClonalCluster
  CopyNumber Copy_Number Corrected_Call Corrected_Copy_Number
  Corrected_MajorCN Corrected_MinorCN Corrected_Ratio DataFrame End
  End.snp End_Position.bp. HaplotypeBinDepth.mean HaplotypeBinDepth.sum
  HaplotypeDepth.mean HaplotypeDepth.mean.symmetric HaplotypeDepth.sum
  HaplotypeDepth.sum.symmetric HaplotypeFraction
  HaplotypeFraction.symmetric HaplotypeRatio HaplotypeRatio.1
  HaplotypeRatio.2 Length.snp. LogRatio MajorCN Median_HaplotypeRatio
  Median_Ratio Median_logR MinorCN PileupParam RangedData SNPs Sample
  ScanBamParam Start Start.snp Start.telo Start_Position.bp. TITAN_call
  TITANcall abline approxfun as axis depth dunif excludeCentromere
  filterByTargetedSequences haplotypeBin head lines loess
  logR_Copy_Number lowess lpolygon lsegments mtext na.omit nonRef par
  phaseSet phaseSet.aggr phasedAlleleFraction phasedCount
  phasedCount.haploSymmetric pileup points predict queryHits read.delim
  rowRanges rowRanges&lt;- scanBamFlag scanBcf seq.info subjectHits tail
  text tumDepth uniroot unstrsplit write.table
Consider adding
  importFrom(&quot;graphics&quot;, &quot;abline&quot;, &quot;axis&quot;, &quot;lines&quot;, &quot;mtext&quot;, &quot;par&quot;,
             &quot;points&quot;, &quot;text&quot;)
  importFrom(&quot;methods&quot;, &quot;as&quot;)
  importFrom(&quot;stats&quot;, &quot;approxfun&quot;, &quot;dunif&quot;, &quot;loess&quot;, &quot;lowess&quot;, &quot;na.omit&quot;,
             &quot;predict&quot;, &quot;uniroot&quot;)
  importFrom(&quot;utils&quot;, &quot;head&quot;, &quot;read.delim&quot;, &quot;tail&quot;, &quot;write.table&quot;)
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains &#x27;methods&#x27;).
* checking Rd files ... NOTE
prepare_Rd: correctCN.Rd:51-53: Dropping empty section \details
* checking Rd metadata ... OK
* checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘list’
* checking for missing documentation entries ... WARNING
Undocumented code objects:
  ‘plotIdiogram.hg38’
All user-level objects in a package should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking compiled code ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... SKIPPED
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘TitanCNA.Rnw’ using ‘UTF-8’... failed to complete the test
 ERROR
Errors in running code in vignettes:
when running code in ‘TitanCNA.Rnw’
  ...
  found more than one best sequence renaming map compatible with seqname style &quot;NCBI&quot; for this object, using the first one
Correcting Tumour
Applying filter on data...
Correcting for GC bias...
Correcting for mappability bias...
Warning in regularize.values(x, y, ties, missing(ties), na.rm = na.rm) :
  collapsing to unique &#x27;x&#x27; values
Killed

... incomplete output.  Crash?

* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 4 WARNINGs, 3 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/TitanCNA.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">TitanCNA.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL TitanCNA
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/site-library’
* installing *source* package ‘TitanCNA’ ...
** using staged installation
** libs
using C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG   -I/usr/local/include    -fPIC  -g -O2  -Wall -c fwd_backC_clonalCN.c -o fwd_backC_clonalCN.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG   -I/usr/local/include    -fPIC  -g -O2  -Wall -c getPositionOverlapC.c -o getPositionOverlapC.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG   -I/usr/local/include    -fPIC  -g -O2  -Wall -c register.c -o register.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG   -I/usr/local/include    -fPIC  -g -O2  -Wall -c viterbiC_clonalCN.c -o viterbiC_clonalCN.o
gcc -shared -L/home/biocbuild/bbs-3.17-bioc/R/lib -L/usr/local/lib -o TitanCNA.so fwd_backC_clonalCN.o getPositionOverlapC.o register.o viterbiC_clonalCN.o -L/home/biocbuild/bbs-3.17-bioc/R/lib -lR
installing to /home/biocbuild/bbs-3.17-bioc/R/site-library/00LOCK-TitanCNA/00new/TitanCNA/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Warning: replacing previous import ‘GenomicRanges::shift’ by ‘data.table::shift’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::first’ by ‘dplyr::first’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::between’ by ‘dplyr::between’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::desc’ by ‘dplyr::desc’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::slice’ by ‘dplyr::slice’ when loading ‘TitanCNA’
Warning: replacing previous import ‘BiocGenerics::combine’ by ‘dplyr::combine’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::last’ by ‘dplyr::last’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomicRanges::setdiff’ by ‘dplyr::setdiff’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomeInfoDb::intersect’ by ‘dplyr::intersect’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::collapse’ by ‘dplyr::collapse’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomicRanges::union’ by ‘dplyr::union’ when loading ‘TitanCNA’
Warning: replacing previous import ‘dplyr::select’ by ‘VariantAnnotation::select’ when loading ‘TitanCNA’
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
Warning: replacing previous import ‘GenomicRanges::shift’ by ‘data.table::shift’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::first’ by ‘dplyr::first’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::between’ by ‘dplyr::between’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::desc’ by ‘dplyr::desc’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::slice’ by ‘dplyr::slice’ when loading ‘TitanCNA’
Warning: replacing previous import ‘BiocGenerics::combine’ by ‘dplyr::combine’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::last’ by ‘dplyr::last’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomicRanges::setdiff’ by ‘dplyr::setdiff’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomeInfoDb::intersect’ by ‘dplyr::intersect’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::collapse’ by ‘dplyr::collapse’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomicRanges::union’ by ‘dplyr::union’ when loading ‘TitanCNA’
Warning: replacing previous import ‘dplyr::select’ by ‘VariantAnnotation::select’ when loading ‘TitanCNA’
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
Warning: replacing previous import ‘GenomicRanges::shift’ by ‘data.table::shift’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::first’ by ‘dplyr::first’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::between’ by ‘dplyr::between’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::desc’ by ‘dplyr::desc’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::slice’ by ‘dplyr::slice’ when loading ‘TitanCNA’
Warning: replacing previous import ‘BiocGenerics::combine’ by ‘dplyr::combine’ when loading ‘TitanCNA’
Warning: replacing previous import ‘data.table::last’ by ‘dplyr::last’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomicRanges::setdiff’ by ‘dplyr::setdiff’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomeInfoDb::intersect’ by ‘dplyr::intersect’ when loading ‘TitanCNA’
Warning: replacing previous import ‘IRanges::collapse’ by ‘dplyr::collapse’ when loading ‘TitanCNA’
Warning: replacing previous import ‘GenomicRanges::union’ by ‘dplyr::union’ when loading ‘TitanCNA’
Warning: replacing previous import ‘dplyr::select’ by ‘VariantAnnotation::select’ when loading ‘TitanCNA’
** testing if installed package keeps a record of temporary installation path
* DONE (TitanCNA)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<HR>
<H3>Example timings</H3>
</BODY>
</HTML>
