<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for GUIDEseq on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><B>[<A href="../index.html#G">G</A>]</B></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-23 01:34:03 -0000 (Thu, 23 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4245</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for GUIDEseq on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the GUIDEseq package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/GUIDEseq.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>867</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">GUIDEseq</A>&nbsp;1.29.4</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/GUIDEseq">landing page</A>)</SPAN><BR>Lihua Julie Zhu<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-21&nbsp;12:29:53&nbsp;-0000&nbsp;(Tue,&nbsp;21&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/GUIDEseq</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">a1608c6</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-22&nbsp;04:35:38&nbsp;-0000&nbsp;(Tue,&nbsp;22&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: GUIDEseq</TD></TR>
<TR><TD><B>Version</B>: 1.29.4</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:GUIDEseq.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings GUIDEseq_1.29.4.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-22 06:18:20 -0000 (Wed, 22 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-22 06:28:57 -0000 (Wed, 22 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 636.9 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: GUIDEseq.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 1</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:GUIDEseq.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings GUIDEseq_1.29.4.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/GUIDEseq.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘GUIDEseq/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘GUIDEseq’ version ‘1.29.4’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘GUIDEseq’ can be installed ... OK
* checking installed package size ... NOTE
  installed size is 12.5Mb
  sub-directories of 1Mb or more:
    extdata  12.0Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
&#x27;:::&#x27; call which should be &#x27;::&#x27;: ‘CRISPRseek:::translatePattern’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
plotTracks: warning in scale_x_continuous(label =
  xaxis.lab.pos$chromosome, breaks = xaxis.lab.pos$chr.center): partial
  argument match of &#x27;label&#x27; to &#x27;labels&#x27;
.maskSubSeq: no visible global function definition for ‘.getMatchedInd’
.nucleotideSubstitutionMatrix: no visible binding for global variable
  ‘IUPAC_CODE_MAP’
.nucleotideSubstitutionMatrix: no visible binding for global variable
  ‘DNA_BASES’
GUIDEseqAnalysis: no visible binding for global variable ‘offTarget’
GUIDEseqAnalysis: no visible binding for global variable ‘peak_score’
GUIDEseqAnalysis: no visible binding for global variable
  ‘predicted_cleavage_score’
GUIDEseqAnalysis: no visible binding for global variable ‘gRNA.name’
GUIDEseqAnalysis: no visible binding for global variable ‘gRNAPlusPAM’
GUIDEseqAnalysis: no visible binding for global variable
  ‘offTarget_sequence’
GUIDEseqAnalysis: no visible binding for global variable
  ‘guideAlignment2OffTarget’
GUIDEseqAnalysis: no visible binding for global variable
  ‘offTargetStrand’
GUIDEseqAnalysis: no visible binding for global variable
  ‘mismatch.distance2PAM’
GUIDEseqAnalysis: no visible binding for global variable
  ‘n.guide.mismatch’
GUIDEseqAnalysis: no visible binding for global variable
  ‘offTarget_Start’
GUIDEseqAnalysis: no visible binding for global variable
  ‘offTarget_End’
GUIDEseqAnalysis: no visible binding for global variable ‘chromosome’
GUIDEseqAnalysis: no visible binding for global variable
  ‘gRNA.insertion’
GUIDEseqAnalysis: no visible binding for global variable
  ‘gRNA.deletion’
GUIDEseqAnalysis: no visible binding for global variable
  ‘pos.insertion’
GUIDEseqAnalysis: no visible binding for global variable ‘pos.deletion’
GUIDEseqAnalysis: no visible binding for global variable ‘n.insertion’
GUIDEseqAnalysis: no visible binding for global variable ‘n.deletion’
GUIDEseqAnalysis: no visible binding for global variable ‘n.RNA.bulge’
GUIDEseqAnalysis: no visible binding for global variable ‘n.DNA.bulge’
GUIDEseqAnalysis: no visible binding for global variable ‘feature’
GUIDEseqAnalysis: no visible binding for global variable
  ‘n.distinct.UMIs’
annotateOffTargets: no visible binding for global variable
  ‘offTarget_Start’
getAlnWithBulge : &lt;anonymous&gt;: no visible binding for global variable
  ‘pa.f1’
getAlnWithBulge : &lt;anonymous&gt;: no visible binding for global variable
  ‘pa.r2’
getPeaks: no visible binding for global variable ‘adjusted.p.value’
getPeaks: no visible binding for global variable ‘SNratio’
getUniqueCleavageEvents: no visible binding for global variable
  ‘width.first’
getUniqueCleavageEvents: no visible binding for global variable
  ‘width.last’
getUniqueCleavageEvents: no visible binding for global variable
  ‘qwidth.last’
getUniqueCleavageEvents: no visible binding for global variable
  ‘strand.last’
getUniqueCleavageEvents: no visible binding for global variable
  ‘qwidth.first’
getUniqueCleavageEvents: no visible binding for global variable
  ‘strand.first’
getUniqueCleavageEvents: no visible binding for global variable
  ‘readName’
getUniqueCleavageEvents: no visible binding for global variable
  ‘seqnames.last’
getUniqueCleavageEvents: no visible binding for global variable
  ‘seqnames.first’
getUniqueCleavageEvents: no visible binding for global variable
  ‘start.last’
getUniqueCleavageEvents: no visible binding for global variable
  ‘end.first’
getUniqueCleavageEvents: no visible binding for global variable ‘UMI’
getUniqueCleavageEvents: no visible binding for global variable
  ‘end.last’
getUniqueCleavageEvents: no visible binding for global variable
  ‘start.first’
offTargetAnalysisOfPeakRegions: no visible binding for global variable
  ‘thePeak’
offTargetAnalysisOfPeakRegions: no visible binding for global variable
  ‘gRNAPlusPAM’
offTargetAnalysisOfPeakRegions: no visible binding for global variable
  ‘offTarget’
plotAlignedOfftargets: no visible binding for global variable
  ‘total.mismatch.bulge’
plotAlignedOfftargets: no visible binding for global variable ‘RIR’
plotAlignedOfftargets: no visible binding for global variable
  ‘guideAlignment2OffTarget’
plotAlignedOfftargets: no visible binding for global variable
  ‘DNA.bulge’
plotAlignedOfftargets: no visible binding for global variable ‘y’
plotAlignedOfftargets: no visible binding for global variable ‘h’
plotAlignedOfftargets: no visible binding for global variable ‘IR’
plotHeatmapOfftargets: no visible binding for global variable
  ‘total.mismatch.bulge’
plotHeatmapOfftargets: no visible binding for global variable
  ‘Offtargets’
plotHeatmapOfftargets: no visible binding for global variable ‘IR’
plotHeatmapOfftargets: no visible binding for global variable
  ‘Ontarget’
plotHeatmapOfftargets: no visible binding for global variable ‘IR.max’
plotHeatmapOfftargets: no visible binding for global variable ‘Samples’
plotHeatmapOfftargets: no visible global function definition for
  ‘guides’
plotHeatmapOfftargets: no visible global function definition for
  ‘guide_legend’
plotHeatmapOfftargets: no visible global function definition for ‘unit’
plotTracks: no visible binding for global variable
  ‘total.mismatch.bulge’
plotTracks: no visible binding for global variable ‘n.PAM.mismatch’
plotTracks: no visible binding for global variable ‘offTargetStrand’
plotTracks: no visible binding for global variable ‘offTarget_Start’
plotTracks: no visible binding for global variable ‘offTarget_End’
plotTracks: no visible binding for global variable ‘n.distinct.UMIs’
plotTracks: no visible binding for global variable
  ‘predicted_cleavage_score’
plotTracks: no visible global function definition for ‘geom_smooth’
plotTracks: no visible binding for global variable ‘chromosome’
plotTracks: no visible binding for global variable ‘chr.max’
plotTracks: no visible binding for global variable ‘chr.offset’
plotTracks: no visible binding for global variable ‘.’
plotTracks: no visible binding for global variable
  ‘cum.cleavage.position’
Undefined global functions or variables:
  . .getMatchedInd DNA.bulge DNA_BASES IR IR.max IUPAC_CODE_MAP
  Offtargets Ontarget RIR SNratio Samples UMI adjusted.p.value chr.max
  chr.offset chromosome cum.cleavage.position end.first end.last
  feature gRNA.deletion gRNA.insertion gRNA.name gRNAPlusPAM
  geom_smooth guideAlignment2OffTarget guide_legend guides h
  mismatch.distance2PAM n.DNA.bulge n.PAM.mismatch n.RNA.bulge
  n.deletion n.distinct.UMIs n.guide.mismatch n.insertion offTarget
  offTargetStrand offTarget_End offTarget_Start offTarget_sequence
  pa.f1 pa.r2 peak_score pos.deletion pos.insertion
  predicted_cleavage_score qwidth.first qwidth.last readName
  seqnames.first seqnames.last start.first start.last strand.first
  strand.last thePeak total.mismatch.bulge unit width.first width.last
  y
* checking Rd files ... NOTE
prepare_Rd: annotateOffTargets.Rd:35-37: Dropping empty section \details
prepare_Rd: annotateOffTargets.Rd:63-65: Dropping empty section \references
prepare_Rd: createBarcodeFasta.Rd:56-58: Dropping empty section \references
prepare_Rd: getUsedBarcodes.Rd:53-55: Dropping empty section \references
checkRd: (-1) mergePlusMinusPeaks.Rd:72: Escaped LaTeX specials: \_
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... WARNING
Undocumented arguments in documentation object &#x27;plotTracks&#x27;
  ‘cleavage.position’

Functions with \usage entries need to have the appropriate \alias
entries, and all their arguments documented.
The \usage entries must correspond to syntactically valid R code.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                     user system elapsed
PEtagAnalysis      18.552  0.712  19.286
annotateOffTargets  7.504  0.221   7.726
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘GUIDEseq.Rnw’ using ‘UTF-8’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 WARNING, 4 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/GUIDEseq.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">GUIDEseq.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL GUIDEseq
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘GUIDEseq’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (GUIDEseq)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">GUIDEseq.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(GUIDEseq)
Loading required package: GenomicRanges
Loading required package: stats4
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
&gt; library(org.Hs.eg.db)
Loading required package: AnnotationDbi
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


&gt; 
&gt; test_check(&quot;GUIDEseq&quot;)
Loading required package: GenomicFeatures
Loading required package: BSgenome
Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: rtracklayer
GUIDEseqAnalysis without bulge. 
Remove duplicate reads ...

Peak calling ...

computing coverage for plus strand ...
computing coverage for minus strand ...
call peaks ...
finding local max for chromosome: chr13
combine plus and minus peaks ... 

keep peaks not in merged.gr but present in both peaks1 and peaks2

Find unmerged peaks with very high reads one-library protocol

offtarget analysis ...

search for gRNAs for input file1...
[1] &quot;Scoring ...&quot;
&gt;&gt;&gt; Finding all hits in sequence chr13+:27629413:27629420:chr13-:27629400:27629404 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13+:39262927:39262939:chr13-:39262918:39262920 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13+:30964712:30964730 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13+:67161908:67161923 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13-:29087136:29087137 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13-:115084360:115084375 ...
&gt;&gt;&gt; DONE searching
finish off-target search in sequence 2
finish off-target search in sequence 1
finish feature vector building
finish score calculation
[1] &quot;Done!&quot;
Extract PAM sequence and n.PAM.mismatch. 
Done with offtarget search!
Add gene and exon information to offTargets ....
Order offtargets. 
Add sequence depth information. 

Get the number of unique UMIs for each offtarget. 

Save offtargets. 
Please check output file in directory GUIDEseqTestResults

Remove duplicate reads ...

offtarget analysis ...

Done with offtarget search!
Add gene and exon information to offTargets ....
Order offtargets. 
Add sequence depth information. 

Get the number of unique UMIs for each offtarget. 

Save offtargets. 
Please check output file in directory GUIDEseqTestResults

GUIDEseqAnalysis with offtargets containing no bulge and with bulge. 
Remove duplicate reads ...

Peak calling ...

computing coverage for plus strand ...
computing coverage for minus strand ...
call peaks ...
finding local max for chromosome: chr13
combine plus and minus peaks ... 

keep peaks not in merged.gr but present in both peaks1 and peaks2

Find unmerged peaks with very high reads one-library protocol

offtarget analysis ...

search for gRNAs for input file1...
[1] &quot;Scoring ...&quot;
&gt;&gt;&gt; Finding all hits in sequence chr13+:27629413:27629420:chr13-:27629400:27629404 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13+:39262927:39262939:chr13-:39262918:39262920 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13+:30964712:30964730 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13+:67161908:67161923 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13-:29087136:29087137 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13-:115084360:115084375 ...
&gt;&gt;&gt; DONE searching
finish off-target search in sequence 2
finish off-target search in sequence 1
finish feature vector building
finish score calculation
[1] &quot;Done!&quot;
Extract PAM sequence and n.PAM.mismatch. 
Finding offtargets with bulges ...Done with offtarget search!
Add gene and exon information to offTargets ....
Order offtargets. 
Add sequence depth information. 

Get the number of unique UMIs for each offtarget. 

Save offtargets. 
Please check output file in directory GUIDEseqTestResults2

Remove duplicate reads ...

offtarget analysis ...

Finding offtargets with bulges ...Done with offtarget search!
Add gene and exon information to offTargets ....
Order offtargets. 
Add sequence depth information. 

Get the number of unique UMIs for each offtarget. 

Save offtargets. 
Please check output file in directory GUIDEseqTestResults2

GUIDEseqAnalysis with bulge containing offtargets where no offtargets contain no bulge. 
Remove duplicate reads ...

Peak calling ...

computing coverage for plus strand ...
computing coverage for minus strand ...
call peaks ...
finding local max for chromosome: chr13
combine plus and minus peaks ... 

keep peaks not in merged.gr but present in both peaks1 and peaks2

Find unmerged peaks with very high reads one-library protocol

offtarget analysis ...

search for gRNAs for input file1...
[1] &quot;Scoring ...&quot;
&gt;&gt;&gt; Finding all hits in sequence chr13+:27629413:27629420:chr13-:27629400:27629404 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13+:39262927:39262939:chr13-:39262918:39262920 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13+:30964712:30964730 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13+:66687218:66687225 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13+:67161908:67161923 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13+:98226906:98226924 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13-:29087136:29087137 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13-:97395652:97395666 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13-:115084360:115084375 ...
&gt;&gt;&gt; DONE searching
finish off-target search in sequence 2
finish off-target search in sequence 1
Remove duplicate reads ...

Peak calling ...

computing coverage for plus strand ...
computing coverage for minus strand ...
call peaks ...
finding local max for chromosome: chr13
combine plus and minus peaks ... 

keep peaks not in merged.gr but present in both peaks1 and peaks2

Find unmerged peaks with very high reads one-library protocol

offtarget analysis ...

search for gRNAs for input file1...
[1] &quot;Scoring ...&quot;
&gt;&gt;&gt; Finding all hits in sequence chr13+:27629413:27629420:chr13-:27629399:27629417 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13+:39262920:39262939:chr13-:39262918:39262918 ...
&gt;&gt;&gt; DONE searching
finish off-target search in sequence 2
finish off-target search in sequence 1
finish feature vector building
finish score calculation
[1] &quot;Done!&quot;
Extract PAM sequence and n.PAM.mismatch. 
Done with offtarget search!
Add gene and exon information to offTargets ....
Order offtargets. 
Add sequence depth information. 

Get the number of unique UMIs for each offtarget. 

Save offtargets. 
Please check output file in directory PEtagTestResults

search for gRNAs for input file1...
[1] &quot;Scoring ...&quot;
&gt;&gt;&gt; Finding all hits in sequence chr13+:27629413:27629420:chr13-:27629400:27629404 ...
&gt;&gt;&gt; DONE searching
&gt;&gt;&gt; Finding all hits in sequence chr13+:39262927:39262939:chr13-:39262918:39262920 ...
&gt;&gt;&gt; DONE searching
finish off-target search in sequence 2
finish off-target search in sequence 1
finish feature vector building
finish score calculation
[1] &quot;Done!&quot;
bulge on gRNA and offtarget on the minus strand  worksStart testing offtarget on the minus strand ...test sequence not long enough for allowing large stretch of bulge[ FAIL 0 | WARN 6 | SKIP 0 | PASS 259 ]

[ FAIL 0 | WARN 6 | SKIP 0 | PASS 259 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
144.726   2.810 147.536 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">GUIDEseq.Rcheck/GUIDEseq-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>GUIDEseq-package</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GUIDEseqAnalysis</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>PEtagAnalysis</TD><TD style="text-align: right;">18.552</TD><TD style="text-align: right;"> 0.712</TD><TD style="text-align: right;">19.286</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotateOffTargets</TD><TD style="text-align: right;">7.504</TD><TD style="text-align: right;">0.221</TD><TD style="text-align: right;">7.726</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>buildFeatureVectorForScoringBulge</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>combineOfftargets</TD><TD style="text-align: right;">0.106</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.112</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>createBarcodeFasta</TD><TD style="text-align: right;">0.027</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.027</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getPeaks</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getUniqueCleavageEvents</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getUsedBarcodes</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mergePlusMinusPeaks</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>offTargetAnalysisOfPeakRegions</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>offTargetAnalysisWithBulge</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>peaks.gr</TD><TD style="text-align: right;">0.030</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.033</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotAlignedOfftargets</TD><TD style="text-align: right;">0.935</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.963</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotHeatmapOfftargets</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotTracks</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>uniqueCleavageEvents</TD><TD style="text-align: right;">0.051</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.055</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
