<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for crlmm on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><B>[<A href="../index.html#C">C</A>]</B></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-10 01:25:29 -0000 (Fri, 10 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4219</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for crlmm on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the crlmm package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/crlmm.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>437</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">crlmm</A>&nbsp;1.57.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/crlmm">landing page</A>)</SPAN><BR>Benilton S Carvalho<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-09&nbsp;03:02:02&nbsp;-0000&nbsp;(Thu,&nbsp;09&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/crlmm</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">5bac5c8</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:04:22&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: crlmm</TD></TR>
<TR><TD><B>Version</B>: 1.57.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:crlmm.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings crlmm_1.57.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-09 13:50:23 -0000 (Thu, 09 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-09 14:00:50 -0000 (Thu, 09 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 627.2 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: crlmm.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:crlmm.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings crlmm_1.57.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/crlmm.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘crlmm/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘crlmm’ version ‘1.57.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘crlmm’ can be installed ... OK
* used C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... NOTE
  installed size is  5.0Mb
  sub-directories of 1Mb or more:
    R      1.2Mb
    data   2.7Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... NOTE
Versioned &#x27;LinkingTo&#x27; value for ‘preprocessCore’ is only usable in R &gt;= 3.0.2
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
&#x27;library&#x27; or &#x27;require&#x27; call to ‘RUnit’ in package code.
  Please use :: or requireNamespace() instead.
  See section &#x27;Suggested packages&#x27; in the &#x27;Writing R Extensions&#x27; manual.
Namespace in Imports field not imported from: ‘splines’
  All declared Imports should be used.
Unexported object imported by a &#x27;:::&#x27; call: ‘Biobase:::assayDataEnvLock’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
.test: no visible global function definition for ‘defineTestSuite’
.test: no visible global function definition for ‘runTestSuite’
.test: no visible global function definition for ‘printTextProtocol’
calculateRBafCNSet : processByChromosome: no visible global function
  definition for ‘position’
genotypeInf: no visible binding for global variable ‘anno’
krlmm: no visible binding for global variable ‘VGLMparameters’
Undefined global functions or variables:
  VGLMparameters anno defineTestSuite position printTextProtocol
  runTestSuite
* checking Rd files ... WARNING
checkRd: (5) CNSet-methods.Rd:39-41: \item in \describe must have non-empty label
checkRd: (5) CNSet-methods.Rd:65-67: \item in \describe must have non-empty label
checkRd: (5) CNSet-methods.Rd:69-71: \item in \describe must have non-empty label
checkRd: (-1) constructInf.Rd:35: Escaped LaTeX specials: \_ \_
checkRd: (-1) genotype.Illumina.Rd:48: Escaped LaTeX specials: \_ \_
checkRd: (-1) preprocessInf.Rd:55: Escaped LaTeX specials: \_ \_
checkRd: (-1) readIdatFiles.Rd:39: Escaped LaTeX specials: \_ \_
checkRd: (-1) readIdatFiles.Rd:55: Escaped LaTeX specials: \_ \_
checkRd: (-1) readIdatFiles.Rd:56: Escaped LaTeX specials: \_
checkRd: (-1) snprma.Rd:38: Escaped LaTeX specials: \_ \_
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking for GNU extensions in Makefiles ... OK
* checking include directives in Makefiles ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘crlmm-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: crlmm
&gt; ### Title: Genotype oligonucleotide arrays with CRLMM
&gt; ### Aliases: crlmm crlmm2
&gt; ### Keywords: classif
&gt; 
&gt; ### ** Examples
&gt; 
&gt; ## this can be slow
&gt; library(oligoClasses)
&gt; if (require(genomewidesnp6Crlmm) &amp; require(hapmapsnp6)){
+   path &lt;- system.file(&quot;celFiles&quot;, package=&quot;hapmapsnp6&quot;)
+ 
+   ## the filenames with full path...
+   ## very useful when genotyping samples not in the working directory
+   cels &lt;- list.celfiles(path, full.names=TRUE)
+   (crlmmOutput &lt;- crlmm(cels))
+   ## If gender is known, one should check that the assigned gender is
+   ## correct, or pass the integer coding of gender as an argument to the
+   ## crlmm function as done below
+ }
Loading required package: genomewidesnp6Crlmm
Annotation for genomewidesnp6Crlmm version 1.0.7 supports UCSC builds hg18 and hg19.
Loading required package: hapmapsnp6
/-------------------------------------------\
|         SAMPLE HAPMAP SNP 6.0             |
|-------------------------------------------|
| Data obtained from http://www.hapmap.org  |
| This package is meant to be used only for |
| demonstration of BioConductor packages.   |
| Access http://www.hapmap.org for details. |
|-------------------------------------------|
| The contents of this package are provided |
| in good faith and the maintainer does not |
| warrant their accuracy.                   |
\-------------------------------------------/
Loading annotations and mixture model parameters.
Processing 3 files.

  |                                                                            
  |                                                                      |   0%
  |                                                                            
  |=======================                                               |  33%Killed
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘crlmm_unit_tests.R’
 ERROR
Running the tests in ‘tests/crlmm_unit_tests.R’ failed.
Last 13 lines of output:
  | This package is meant to be used only for |
  | demonstration of BioConductor packages.   |
  | Access http://www.hapmap.org for details. |
  |-------------------------------------------|
  | The contents of this package are provided |
  | in good faith and the maintainer does not |
  | warrant their accuracy.                   |
  \-------------------------------------------/
  Loading annotations and mixture model parameters.
  Processing 3 files.
  
    |                                                                            
    |                                                                      |   0%
    |                                                                            
    |=======================                                               |  33%Killed
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘AffyGW.Rnw’... OK
  ‘CopyNumberOverview.Rnw’... OK
  ‘IlluminaPreprocessCN.Rnw’... OK
  ‘Infrastructure.Rnw’... OK
  ‘genotyping.Rnw’... failed to complete the test
  ‘gtypeDownstream.Rnw’... OK
 ERROR
Errors in running code in vignettes:
when running code in ‘genotyping.Rnw’
  ...

&gt; path &lt;- system.file(&quot;celFiles&quot;, package = &quot;hapmapsnp6&quot;)

&gt; celFiles &lt;- list.celfiles(path, full.names = TRUE)

&gt; system.time(crlmmResult &lt;- crlmm(celFiles, verbose = FALSE))
Annotation for genomewidesnp6Crlmm version 1.0.7 supports UCSC builds hg18 and hg19.
Killed

... incomplete output.  Crash?

* checking re-building of vignette outputs ... NOTE
Error(s) in re-building vignettes:
--- re-building ‘AffyGW.Rnw’ using Sweave
--- finished re-building ‘AffyGW.Rnw’

--- re-building ‘CopyNumberOverview.Rnw’ using Sweave
--- finished re-building ‘CopyNumberOverview.Rnw’

--- re-building ‘IlluminaPreprocessCN.Rnw’ using Sweave
--- finished re-building ‘IlluminaPreprocessCN.Rnw’

--- re-building ‘Infrastructure.Rnw’ using Sweave
--- finished re-building ‘Infrastructure.Rnw’

--- re-building ‘genotyping.Rnw’ using Sweave
Loading required package: oligoClasses
Welcome to oligoClasses version 1.61.0
Loading required package: preprocessCore
No methods found in package ‘oligoClasses’ for request: ‘mean’ when loading ‘crlmm’
Welcome to crlmm version 1.57.0
/-------------------------------------------\
|         SAMPLE HAPMAP SNP 6.0             |
|-------------------------------------------|
| Data obtained from http://www.hapmap.org  |
| This package is meant to be used only for |
| demonstration of BioConductor packages.   |
| Access http://www.hapmap.org for details. |
|-------------------------------------------|
| The contents of this package are provided |
| in good faith and the maintainer does not |
| warrant their accuracy.                   |
\-------------------------------------------/
Annotation for genomewidesnp6Crlmm version 1.0.7 supports UCSC builds hg18 and hg19.
Killed
--- re-building ‘gtypeDownstream.Rnw’ using Sweave
--- finished re-building ‘gtypeDownstream.Rnw’

cp ../inst/scripts/AffyGW.pdf .
cp ../inst/scripts/IlluminaPreprocessCN.pdf .
cp ../inst/scripts/Infrastructure.pdf .
cp ../inst/scripts/gtypeDownstream.pdf .
cp ../inst/scripts/crlmmIllumina.pdf .
&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD pdflatex genotyping.tex
This is pdfTeX, Version 3.141592653-2.6-1.40.22 (TeX Live 2022/dev/Debian) (preloaded format=pdflatex)
 restricted \write18 enabled.
entering extended mode
(./genotyping.tex
LaTeX2e &lt;2021-11-15&gt; patch level 1
L3 programming layer &lt;2022-01-21&gt;
(/usr/share/texlive/texmf-dist/tex/latex/base/article.cls
Document Class: article 2021/10/04 v1.4n Standard LaTeX document class
(/usr/share/texlive/texmf-dist/tex/latex/base/size10.clo))
(/home/biocbuild/bbs-3.17-bioc/R/share/texmf/tex/latex/Sweave.sty
(/usr/share/texlive/texmf-dist/tex/latex/base/ifthen.sty)
(/usr/share/texlive/texmf-dist/tex/generic/iftex/iftex.sty)
(/usr/share/texlive/texmf-dist/tex/latex/graphics/graphicx.sty
(/usr/share/texlive/texmf-dist/tex/latex/graphics/keyval.sty)
(/usr/share/texlive/texmf-dist/tex/latex/graphics/graphics.sty
(/usr/share/texlive/texmf-dist/tex/latex/graphics/trig.sty)
(/usr/share/texlive/texmf-dist/tex/latex/graphics-cfg/graphics.cfg)
(/usr/share/texlive/texmf-dist/tex/latex/graphics-def/pdftex.def)))
(/usr/share/texlive/texmf-dist/tex/latex/fancyvrb/fancyvrb.sty)
(/usr/share/texlive/texmf-dist/tex/latex/base/textcomp.sty)
(/usr/share/texlive/texmf-dist/tex/latex/base/fontenc.sty))
(/usr/share/texlive/texmf-dist/tex/latex/l3backend/l3backend-pdftex.def)
No file genotyping.aux.
(/usr/share/texlive/texmf-dist/tex/context/base/mkii/supp-pdf.mkii
[Loading MPS to PDF converter (version 2006.09.02).]
) (/usr/share/texlive/texmf-dist/tex/latex/epstopdf-pkg/epstopdf-base.sty
(/usr/share/texlive/texmf-dist/tex/generic/infwarerr/infwarerr.sty)
(/usr/share/texlive/texmf-dist/tex/latex/grfext/grfext.sty
(/usr/share/texlive/texmf-dist/tex/generic/kvdefinekeys/kvdefinekeys.sty))
(/usr/share/texlive/texmf-dist/tex/latex/kvoptions/kvoptions.sty
(/usr/share/texlive/texmf-dist/tex/generic/ltxcmds/ltxcmds.sty)
(/usr/share/texlive/texmf-dist/tex/generic/kvsetkeys/kvsetkeys.sty))
(/usr/share/texlive/texmf-dist/tex/generic/pdftexcmds/pdftexcmds.sty)
(/usr/share/texlive/texmf-dist/tex/latex/latexconfig/epstopdf-sys.cfg))
(/usr/share/texlive/texmf-dist/tex/latex/base/t1cmss.fd)
(/usr/share/texlive/texmf-dist/tex/latex/base/t1cmtt.fd))
Runaway argument?
! File ended while scanning use of \FancyVerbGetLine.
&lt;inserted text&gt; 
                \par 
&lt;*&gt; genotyping.tex
                  
? 
! Emergency stop.
&lt;inserted text&gt; 
                \par 
&lt;*&gt; genotyping.tex
                  
!  ==&gt; Fatal error occurred, no output PDF file produced!
Transcript written on genotyping.log.
make: *** [Makefile:10: vignettes] Error 1
Error in tools::buildVignettes(dir = &quot;/home/biocbuild/bbs-3.17-bioc/meat/crlmm.Rcheck/vign_test/crlmm&quot;,  : 
  running &#x27;make&#x27; failed
Execution halted

* checking PDF version of manual ... OK
* DONE

Status: 3 ERRORs, 1 WARNING, 6 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/crlmm.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">crlmm.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL crlmm
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘crlmm’ ...
** using staged installation
** libs
using C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/preprocessCore/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c gtypeCaller.c -o gtypeCaller.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/preprocessCore/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c init.c -o init.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/preprocessCore/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c trimmed.c -o trimmed.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/preprocessCore/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c utils.c -o utils.o
gcc -shared -L/home/biocbuild/bbs-3.17-bioc/R/lib -L/usr/local/lib -o crlmm.so gtypeCaller.o init.o trimmed.o utils.o -L/home/biocbuild/bbs-3.17-bioc/R/lib -lR
installing to /home/biocbuild/bbs-3.17-bioc/R/library/00LOCK-crlmm/00new/crlmm/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
No methods found in package ‘oligoClasses’ for request: ‘mean’ when loading ‘crlmm’
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
No methods found in package ‘oligoClasses’ for request: ‘mean’ when loading ‘crlmm’
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
No methods found in package ‘oligoClasses’ for request: ‘mean’ when loading ‘crlmm’
** testing if installed package keeps a record of temporary installation path
* DONE (crlmm)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">crlmm.Rcheck/tests/crlmm_unit_tests.Rout.fail</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; require(&quot;crlmm&quot;) || stop(&quot;unable to load crlmm package&quot;)
Loading required package: crlmm
Loading required package: oligoClasses
Welcome to oligoClasses version 1.61.0
Loading required package: preprocessCore
No methods found in package &#x27;oligoClasses&#x27; for request: &#x27;mean&#x27; when loading &#x27;crlmm&#x27;
Welcome to crlmm version 1.57.0
[1] TRUE
&gt; crlmm:::.test()
Loading required package: genomewidesnp6Crlmm
Annotation for genomewidesnp6Crlmm version 1.0.7 supports UCSC builds hg18 and hg19.
Loading required package: hapmapsnp6
/-------------------------------------------\
|         SAMPLE HAPMAP SNP 6.0             |
|-------------------------------------------|
| Data obtained from http://www.hapmap.org  |
| This package is meant to be used only for |
| demonstration of BioConductor packages.   |
| Access http://www.hapmap.org for details. |
|-------------------------------------------|
| The contents of this package are provided |
| in good faith and the maintainer does not |
| warrant their accuracy.                   |
\-------------------------------------------/
Loading annotations and mixture model parameters.
Processing 3 files.
  |                                                                              |                                                                      |   0%  |                                                                              |=======================                                               |  33%Killed
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">crlmm.Rcheck/crlmm-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>ListClassConstructors</TD><TD style="text-align: right;">1.912</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">1.950</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>PredictionRegion-class</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>batchStatisticAccessors</TD><TD style="text-align: right;">0.249</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.260</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>calculateRBaf</TD><TD style="text-align: right;">1.024</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">1.038</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>celfile-utils</TD><TD style="text-align: right;">1.791</TD><TD style="text-align: right;">0.355</TD><TD style="text-align: right;">2.242</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cnSetExample</TD><TD style="text-align: right;">1.155</TD><TD style="text-align: right;">0.083</TD><TD style="text-align: right;">1.245</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>constructInf</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>copynumberAccessors</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
