<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for methylumi on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><B>[<A href="../index.html#M">M</A>]</B></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-01-29 16:33:59 -0000 (Sun, 29 Jan 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4021</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for methylumi on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the methylumi package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/methylumi.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1166</B>/2162</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">methylumi</A>&nbsp;2.45.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/methylumi">landing page</A>)</SPAN><BR>Sean Davis<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-01-27&nbsp;13:09:27&nbsp;-0000&nbsp;(Fri,&nbsp;27&nbsp;Jan&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/methylumi</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">bd20f4e</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:04:28&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: methylumi</TD></TR>
<TR><TD><B>Version</B>: 2.45.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:methylumi.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings methylumi_2.45.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-01-29 03:41:00 -0000 (Sun, 29 Jan 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-01-29 03:49:09 -0000 (Sun, 29 Jan 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 488.2 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: methylumi.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:methylumi.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings methylumi_2.45.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/methylumi.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘methylumi/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘methylumi’ version ‘2.45.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Depends: includes the non-default packages:
  &#x27;Biobase&#x27;, &#x27;scales&#x27;, &#x27;reshape2&#x27;, &#x27;ggplot2&#x27;, &#x27;matrixStats&#x27;,
  &#x27;FDb.InfiniumMethylation.hg19&#x27;, &#x27;minfi&#x27;
Adding so many packages to the search path is excessive and importing
selectively is preferable.
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .travis.yml
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘methylumi’ can be installed ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... NOTE
Packages listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘Biobase’ ‘minfi’ ‘lattice’ ‘matrixStats’
A package should be listed in only one of these fields.
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... WARNING
Error in library(methylumi, lib.loc = &quot;/home/biocbuild/bbs-3.17-bioc/R/library&quot;) : 
  there is no package called ‘methylumi’
Execution halted

It looks like this package has a loading problem when not on .libPaths:
see the messages for details.
* checking dependencies in R code ... NOTE
Killed
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
getBarcodes: warning in list.files(path = path, patt = &quot;idat&quot;): partial
  argument match of &#x27;patt&#x27; to &#x27;pattern&#x27;
coerce,RangedSummarizedExperiment-GenomicMethylSet: warning in
  assays(from, withDim = F): partial argument match of &#x27;withDim&#x27; to
  &#x27;withDimnames&#x27;
.getFinalReportBlock: no visible global function definition for
  ‘read.table’
.mclapply: no visible global function definition for ‘mclapply’
.parallel : &lt;anonymous&gt;: no visible global function definition for
  ‘ecdf’
.readOldMethylationFile: no visible global function definition for
  ‘read.delim’
CSVtoDF: no visible global function definition for ‘read.csv’
SEtoGRset: no visible global function definition for ‘GenomicRatioSet’
beta.mme: no visible global function definition for ‘weighted.mean’
beta.transform: no visible global function definition for
  ‘weighted.mean’
cy3: no visible global function definition for ‘addColorChannelInfo’
cy5: no visible global function definition for ‘addColorChannelInfo’
gamma.get.xcs: no visible global function definition for
  ‘gamma.integral’
gamma.get.xs : &lt;anonymous&gt;: no visible global function definition for
  ‘gamma.mle’
gamma.get.xs : &lt;anonymous&gt;: no visible global function definition for
  ‘gamma.integral’
gamma.signal: no visible global function definition for
  ‘gamma.integral’
gammaM.get.xcs : &lt;anonymous&gt;: no visible binding for global variable
  ‘offset’
gammaM.get.xs : &lt;anonymous&gt;: no visible global function definition for
  ‘gamma.mode’
gammaM.get.xs : &lt;anonymous&gt;: no visible global function definition for
  ‘gamma.mle’
getAssayDataNameSubstitutions: no visible global function definition
  for ‘read.table’
getMethylationBeadMappers : &lt;anonymous&gt;: no visible global function
  definition for ‘data’
getMethylationBeadMappers : &lt;anonymous&gt;: no visible binding for global
  variable ‘hm27.controls’
getMethylationBeadMappers : &lt;anonymous&gt;: no visible binding for global
  variable ‘hm450.controls’
getMethylationBeadMappers : &lt;anonymous&gt;: no visible binding for global
  variable ‘hm27.ordering’
getMethylationBeadMappers : &lt;anonymous&gt;: no visible binding for global
  variable ‘hm450.ordering’
getPlatform: no visible binding for global variable
  ‘FDb.InfiniumMethylation.hg19’
getPlatform: no visible global function definition for ‘data’
getPlatform: no visible global function definition for ‘DNAStringSet’
illumina.get.xs: no visible global function definition for
  ‘colQuantiles’
median.get.xs: no visible global function definition for ‘colMedians’
methylumi.bgcorr: no visible binding for global variable
  ‘IlluminaHumanMethylation27kCOLORCHANNEL’
methylumi.bgcorr: no visible binding for global variable
  ‘IlluminaHumanMethylation450kCOLORCHANNEL’
methylumi.diagnostics: no visible binding for global variable
  ‘IlluminaHumanMethylation27kCOLORCHANNEL’
methylumi.diagnostics: no visible binding for global variable
  ‘IlluminaHumanMethylation450kCOLORCHANNEL’
methylumi.diagnostics: no visible global function definition for ‘par’
methylumi.diagnostics: no visible global function definition for
  ‘colorRampPalette’
methylumi.diagnostics: no visible global function definition for
  ‘lines’
methylumi.diagnostics: no visible global function definition for
  ‘title’
methylumi.diagnostics: no visible global function definition for
  ‘plot.density’
methylumi.diagnostics: no visible global function definition for
  ‘abline’
methylumiCSV: no visible global function definition for
  ‘DFsToNChannelSet’
methylumiR: no visible global function definition for ‘capture.output’
methylumiToMinfi: no visible global function definition for
  ‘RGChannelSet’
normalizeMethyLumiSet: no visible global function definition for
  ‘capture.output’
normexp.get.xs: no visible global function definition for ‘huber’
normexp.get.xs: no visible global function definition for ‘colSds’
normexp.signal: no visible global function definition for ‘dnorm’
normexp.signal: no visible global function definition for ‘pnorm’
plotNegOob: no visible global function definition for ‘par’
plotNegOob : &lt;anonymous&gt;: no visible global function definition for
  ‘melt’
plotNegOob: no visible global function definition for ‘ggplot’
plotNegOob: no visible global function definition for ‘aes’
plotNegOob: no visible binding for global variable ‘intensity’
plotNegOob: no visible binding for global variable ‘channel.probes’
plotNegOob: no visible global function definition for ‘geom_histogram’
plotNegOob: no visible binding for global variable ‘..density..’
plotNegOob: no visible global function definition for
  ‘position_identity’
plotNegOob: no visible global function definition for ‘facet_grid’
plotNegOob: no visible global function definition for
  ‘scale_x_continuous’
plotNegOob: no visible global function definition for
  ‘scale_y_continuous’
plotNegOob: no visible global function definition for
  ‘scale_fill_manual’
plotNegOob: no visible global function definition for ‘opts’
plotNegOob: no visible global function definition for ‘theme_bw’
psummary: no visible global function definition for ‘p.adjust’
qc.probe.plot: no visible global function definition for ‘log_trans’
qc.probe.plot: no visible global function definition for ‘melt’
qc.probe.plot: no visible binding for global variable ‘variable’
qc.probe.plot: no visible binding for global variable ‘value’
qc.probe.plot: no visible global function definition for ‘coord_flip’
qc.probe.plot: no visible global function definition for
  ‘scale_x_continuous’
qc.probe.plot: no visible global function definition for
  ‘scale_y_discrete’
qc.probe.plot: no visible global function definition for ‘facet_grid’
qc.probe.plot: no visible global function definition for
  ‘scale_colour_manual’
qc.probe.plot: no visible global function definition for
  ‘scale_shape_manual’
qc.probe.plot: no visible global function definition for ‘theme_bw’
[,MethyLumiM-ANY-ANY-ANY: no visible global function definition for
  ‘packageDescription’
coerce,MethyLumiM-MethylSet: no visible global function definition for
  ‘MethylSet’
coerce,MethyLumiM-MethylSet: no visible global function definition for
  ‘packageVersion’
coerce,MethyLumiSet-MethylSet: no visible global function definition
  for ‘MethylSet’
coerce,MethyLumiSet-MethylSet: no visible global function definition
  for ‘packageVersion’
coerce,RangedSummarizedExperiment-GenomicMethylSet: no visible global
  function definition for ‘GenomicMethylSet’
coerce,eSet-MethyLumiM: no visible global function definition for
  ‘capture.output’
coerce,eSet-MethyLumiM: no visible global function definition for
  ‘packageDescription’
combine27k450k,MethyLumiSet-MethyLumiSet: no visible global function
  definition for ‘subsetCommonProbes’
combine,MethyLumiM-MethyLumiM: no visible global function definition
  for ‘capture.output’
combine,MethyLumiM-MethyLumiM: no visible global function definition
  for ‘packageDescription’
corplot,MethyLumiSet: no visible global function definition for
  ‘hclust’
corplot,MethyLumiSet: no visible global function definition for
  ‘as.dist’
getProbesByChannel,methylData : &lt;anonymous&gt;: no visible binding for
  global variable ‘allele’
hist,MethyLumiQC: no visible global function definition for ‘par’
hist,MethyLumiSet: no visible global function definition for ‘par’
initialize,MethyLumiQC: ... may be used in an incorrect context:
  ‘assayDataNew(...)’
intensities.IB,MethyLumiSet-character: no visible binding for global
  variable ‘IlluminaHumanMethylation27kCOLORCHANNEL’
intensities.IB,MethyLumiSet-character: no visible binding for global
  variable ‘IlluminaHumanMethylation450kCOLORCHANNEL’
intensities.M,MethyLumiSet-character: no visible binding for global
  variable ‘IlluminaHumanMethylation27kCOLORCHANNEL’
intensities.M,MethyLumiSet-character: no visible binding for global
  variable ‘IlluminaHumanMethylation450kCOLORCHANNEL’
intensities.OOB.allelic,MethyLumiSet-character-character: no visible
  binding for global variable ‘IlluminaHumanMethylation27kCOLORCHANNEL’
intensities.OOB.allelic,MethyLumiSet-character-character: no visible
  binding for global variable
  ‘IlluminaHumanMethylation450kCOLORCHANNEL’
intensities.OOB,MethyLumiSet-character: no visible binding for global
  variable ‘IlluminaHumanMethylation27kCOLORCHANNEL’
intensities.OOB,MethyLumiSet-character: no visible binding for global
  variable ‘IlluminaHumanMethylation450kCOLORCHANNEL’
intensities.U,MethyLumiSet-character: no visible binding for global
  variable ‘IlluminaHumanMethylation27kCOLORCHANNEL’
intensities.U,MethyLumiSet-character: no visible binding for global
  variable ‘IlluminaHumanMethylation450kCOLORCHANNEL’
pairs,MethyLumiSet : upperPanel: no visible global function definition
  for ‘points’
pairs,MethyLumiSet : upperPanel: no visible global function definition
  for ‘abline’
pairs,MethyLumiSet : lowerPanel: no visible global function definition
  for ‘par’
pairs,MethyLumiSet : lowerPanel: no visible global function definition
  for ‘text’
pairs,MethyLumiSet : diagPanel: no visible global function definition
  for ‘par’
pairs,MethyLumiSet : diagPanel: no visible global function definition
  for ‘rect’
pairs,MethyLumiSet: no visible global function definition for ‘par’
plotNAs,methylData: no visible binding for global variable ‘index’
plotNAs,methylData: no visible binding for global variable ‘dropouts’
plotProbeNAs,methylData: no visible binding for global variable ‘mu’
plotProbeNAs,methylData: no visible binding for global variable ‘drops’
plotSampleIntensities,MethyLumiSet: no visible global function
  definition for ‘lines’
plotSampleIntensities,MethyLumiSet: no visible global function
  definition for ‘box’
plotSampleIntensities,MethyLumiSet: no visible global function
  definition for ‘axis’
pval.detect&lt;-,methylData-numeric : &lt;anonymous&gt; : &lt;anonymous&gt;: no
  visible global function definition for ‘ecdf’
pval.detect&lt;-,methylData-numeric: no visible global function definition
  for ‘rowMins’
Undefined global functions or variables:
  ..density.. DFsToNChannelSet DNAStringSet
  FDb.InfiniumMethylation.hg19 GenomicMethylSet GenomicRatioSet
  IlluminaHumanMethylation27kCOLORCHANNEL
  IlluminaHumanMethylation450kCOLORCHANNEL MethylSet RGChannelSet
  abline addColorChannelInfo aes allele as.dist axis box capture.output
  channel.probes colMedians colQuantiles colSds colorRampPalette
  coord_flip data dnorm dropouts drops ecdf facet_grid gamma.integral
  gamma.mle gamma.mode geom_histogram ggplot hclust hm27.controls
  hm27.ordering hm450.controls hm450.ordering huber index intensity
  lines log_trans mclapply melt mu offset opts p.adjust
  packageDescription packageVersion par plot.density pnorm points
  position_identity read.csv read.delim read.table rect rowMins
  scale_colour_manual scale_fill_manual scale_shape_manual
  scale_x_continuous scale_y_continuous scale_y_discrete
  subsetCommonProbes text theme_bw title value variable weighted.mean
Consider adding
  importFrom(&quot;grDevices&quot;, &quot;colorRampPalette&quot;)
  importFrom(&quot;graphics&quot;, &quot;abline&quot;, &quot;axis&quot;, &quot;box&quot;, &quot;lines&quot;, &quot;par&quot;,
             &quot;points&quot;, &quot;rect&quot;, &quot;text&quot;, &quot;title&quot;)
  importFrom(&quot;stats&quot;, &quot;as.dist&quot;, &quot;dnorm&quot;, &quot;ecdf&quot;, &quot;hclust&quot;, &quot;offset&quot;,
             &quot;p.adjust&quot;, &quot;pnorm&quot;, &quot;weighted.mean&quot;)
  importFrom(&quot;utils&quot;, &quot;capture.output&quot;, &quot;data&quot;, &quot;packageDescription&quot;,
             &quot;packageVersion&quot;, &quot;read.csv&quot;, &quot;read.delim&quot;, &quot;read.table&quot;)
to your NAMESPACE file.
* checking Rd files ... NOTE
prepare_Rd: estimateM.Rd:34-36: Dropping empty section \seealso
prepare_Rd: estimateM.Rd:37-39: Dropping empty section \examples
checkRd: (-1) extractBarcodeAndPosition.Rd:14: Escaped LaTeX specials: \_ \_
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... SKIPPED
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘methylumi.Rnw’ using ‘UTF-8’... OK
  ‘methylumi450k.Rnw’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... ERROR
Error(s) in re-building vignettes:
  ...
--- re-building ‘methylumi.Rnw’ using knitr
--- finished re-building ‘methylumi.Rnw’

--- re-building ‘methylumi450k.Rnw’ using knitr
Killed
SUMMARY: processing the following file failed:
  ‘methylumi450k.Rnw’

Error: Vignette re-building failed.
Execution halted

* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 1 WARNING, 6 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/methylumi.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">methylumi.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL methylumi
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/site-library’
* installing *source* package ‘methylumi’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (methylumi)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<HR>
<H3>Example timings</H3>
</BODY>
</HTML>
