<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for RnBeads on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.17</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><B>[<A href="..#R">R</A>]</B></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-03-01 07:38:49 -0000 (Wed, 01 Mar 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4266</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for RnBeads on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the RnBeads package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/RnBeads.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1702</B>/2171</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">RnBeads</A>&nbsp;2.17.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/RnBeads">landing page</A>)</SPAN><BR>Fabian Mueller<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-27&nbsp;07:53:22&nbsp;-0000&nbsp;(Mon,&nbsp;27&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/RnBeads</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">99cf51e</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-01-03&nbsp;12:42:18&nbsp;-0000&nbsp;(Tue,&nbsp;03&nbsp;Jan&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: RnBeads</TD></TR>
<TR><TD><B>Version</B>: 2.17.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:RnBeads.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings RnBeads_2.17.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-28 13:30:04 -0000 (Tue, 28 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-28 13:44:42 -0000 (Tue, 28 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 878.1 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: RnBeads.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:RnBeads.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings RnBeads_2.17.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/RnBeads.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘RnBeads/DESCRIPTION’ ... OK
* this is package ‘RnBeads’ version ‘2.17.0’
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Depends: includes the non-default packages:
  &#x27;BiocGenerics&#x27;, &#x27;S4Vectors&#x27;, &#x27;GenomicRanges&#x27;, &#x27;MASS&#x27;, &#x27;cluster&#x27;,
  &#x27;ff&#x27;, &#x27;fields&#x27;, &#x27;ggplot2&#x27;, &#x27;gplots&#x27;, &#x27;grid&#x27;, &#x27;gridExtra&#x27;, &#x27;limma&#x27;,
  &#x27;matrixStats&#x27;, &#x27;illuminaio&#x27;, &#x27;methylumi&#x27;, &#x27;plyr&#x27;
Adding so many packages to the search path is excessive and importing
selectively is preferable.
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .travis.yml
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘RnBeads’ can be installed ... OK
* checking installed package size ... NOTE
  installed size is  5.5Mb
  sub-directories of 1Mb or more:
    R     3.1Mb
    bin   1.0Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... NOTE
Package listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘qvalue’
A package should be listed in only one of these fields.
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... NOTE
Warning: no function found corresponding to methods exports from ‘RnBeads’ for: ‘samples’

A namespace must be able to be loaded with just the base namespace
loaded: otherwise if the namespace gets loaded by a saved object, the
session will be unable to start.

Probably some imports need to be declared in the NAMESPACE file.
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
Package in Depends field not imported from: &#x27;grid&#x27;
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.
Unexported objects imported by &#x27;:::&#x27; calls:
  &#x27;Gviz:::.getBMFeatureMap&#x27; &#x27;doParallel:::.options&#x27;
  &#x27;grDevices:::.smoothScatterCalcDensity&#x27;
  &#x27;minfi:::.default.450k.annotation&#x27; &#x27;minfi:::.extractFromRGSet450k&#x27;
  &#x27;minfi:::.normalizeFunnorm450k&#x27;
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
.stopImplicitCluster: no visible global function definition for
  ‘stopCluster’
MethyLumiSet2RnBeadSet: no visible global function definition for
  ‘phenoData’
MethyLumiSet2RnBeadSet: no visible global function definition for
  ‘assayDataElementNames’
MethyLumiSet2RnBeadSet: no visible global function definition for
  ‘featureNames’
MethyLumiSet2RnBeadSet: no visible global function definition for
  ‘varLabels’
MethyLumiSet2RnBeadSet: no visible global function definition for
  ‘featureData’
add.age.histogram: no visible binding for global variable ‘Age’
add.age.histogram: no visible binding for global variable ‘..count..’
add.age.histogram: no visible binding for global variable ‘..density..’
add.agecomparison.plot: no visible global function definition for
  ‘samples’
add.agecomparison.plot: no visible binding for global variable ‘Sample’
add.agecomparison.plot: no visible global function definition for
  ‘%dopar%’
add.agecomparison.plot: no visible global function definition for
  ‘foreach’
add.agecomparison.plot: no visible binding for global variable ‘i’
add.combination.plot: no visible global function definition for
  ‘samples’
add.combination.plot: no visible binding for global variable
  ‘Difference’
add.combination.plot: no visible binding for global variable ‘Density’
add.combination.plot: no visible binding for global variable ‘yint’
add.combination.plot: no visible binding for global variable ‘Measure’
add.error.plot: no visible binding for global variable ‘Value’
add.error.plot: no visible binding for global variable ‘Sample’
add.error.plot: no visible binding for global variable ‘Deviance’
add.error.plot: no visible binding for global variable ‘yint’
add.error.plot: no visible binding for global variable ‘Measure’
add.negative.control.boxplot: no visible global function definition for
  ‘samples’
add.profile.plots: no visible global function definition for ‘mclapply’
add.qc.barplots: no visible global function definition for ‘samples’
add.quantile.plot: no visible binding for global variable ‘Difference’
add.quantile.plot: no visible binding for global variable ‘..density..’
add.quantile.plot: no visible binding for global variable ‘Sample’
add.quantile.plot: no visible binding for global variable ‘Density’
add.seq.coverage.histograms: no visible global function definition for
  ‘samples’
add.seq.coverage.plot: no visible global function definition for
  ‘samples’
add.seq.coverage.violins: no visible global function definition for
  ‘samples’
add.stratification.plot: no visible global function definition for
  ‘%dopar%’
add.stratification.plot: no visible global function definition for
  ‘foreach’
add.stratification.plot: no visible binding for global variable ‘Group’
add.stratification.plot: no visible binding for global variable
  ‘Increase’
add.stratification.plot: no visible binding for global variable
  ‘Predicted’
add.stratification.plot: no visible binding for global variable
  ‘Annotated’
add.stratification.plot.immune: no visible global function definition
  for ‘%dopar%’
add.stratification.plot.immune: no visible global function definition
  for ‘foreach’
add.stratification.plot.immune: no visible binding for global variable
  ‘Group’
add.stratification.plot.immune: no visible binding for global variable
  ‘Immune’
addReportPlot.diffVar.volcano: no visible binding for global variable
  ‘var.diff’
addReportPlot.diffVar.volcano: no visible binding for global variable
  ‘log10P’
addReportPlot.diffVar.volcano: no visible binding for global variable
  ‘combinedRank.var’
addReportPlot.diffVar.volcano: no visible binding for global variable
  ‘log10FDR’
addReportPlots.diffMeth.bin.site.volcano: no visible binding for global
  variable ‘mean.diff’
addReportPlots.diffMeth.bin.site.volcano: no visible binding for global
  variable ‘diffmeth.p.val’
addReportPlots.diffMeth.bin.site.volcano: no visible binding for global
  variable ‘combinedRank’
addReportPlots.diffMeth.bin.site.volcano: no visible binding for global
  variable ‘mean.quot.log2’
addReportPlots.diffMeth.bin.site.volcano: no visible binding for global
  variable ‘diffmeth.p.adj.fdr’
agePredictorBiseq: no visible global function definition for
  ‘impute.knn’
agePredictorBiseq: no visible global function definition for ‘samples’
agePredictorChip: no visible global function definition for
  ‘impute.knn’
append.cpg.stats: no visible global function definition for ‘%dopar%’
append.cpg.stats: no visible global function definition for ‘foreach’
append.cpg.stats: no visible binding for global variable ‘chrom’
basic_combine: no visible global function definition for ‘samples’
computeDiffTab.default.region: no visible global function definition
  for ‘%dopar%’
computeDiffTab.default.region: no visible global function definition
  for ‘foreach’
computeDiffTab.default.region: no visible binding for global variable
  ‘i’
computeDiffTab.default.site: no visible binding for global variable
  ‘p.vals.t.na.adj’
computeDiffTab.default.site : &lt;anonymous&gt;: no visible binding for
  global variable ‘.inds.g2’
computeDiffVar.bin.site: no visible binding for global variable
  ‘p.vals.t.na.adj’
computeDiffVar.bin.site : &lt;anonymous&gt;: no visible binding for global
  variable ‘eps’
computeDiffVar.bin.site : &lt;anonymous&gt;: no visible binding for global
  variable ‘.inds.g2’
computeDiffVar.default.region: no visible global function definition
  for ‘%dopar%’
computeDiffVar.default.region: no visible global function definition
  for ‘foreach’
computeDiffVar.default.region: no visible binding for global variable
  ‘i’
create.densityScatter: no visible binding for global variable
  ‘..density..’
create.diffMeth.diffVar.subsample: no visible binding for global
  variable ‘..density..’
create.ucsc.track.hub: no visible global function definition for
  ‘samples’
createPredictor : ret: no visible global function definition for
  ‘impute.knn’
cv.array: no visible global function definition for ‘impute.knn’
data.frame2GRanges: no visible global function definition for
  ‘seqlevels&lt;-’
data.frame2GRanges: no visible global function definition for
  ‘genome&lt;-’
diffVar: no visible global function definition for ‘varFit’
diffVar: no visible binding for global variable ‘group1’
diffVar: no visible binding for global variable ‘group2’
estimateProportionsCP: no visible global function definition for ‘lme’
estimateProportionsCP: no visible global function definition for
  ‘getVarCov’
general.cv: no visible global function definition for ‘%dopar%’
general.cv: no visible global function definition for ‘foreach’
general.cv: no visible binding for global variable ‘i’
get.adjustment.variables: no visible global function definition for
  ‘samples’
get.comparison.info : rm.na.from.adj.tab: no visible global function
  definition for ‘samples’
get.components.isva: no visible global function definition for ‘DoISVA’
get.components.sva: no visible global function definition for ‘sva’
get.cpg.stats: no visible global function definition for
  ‘dinucleotideFrequency’
get.cpg.stats: no visible global function definition for
  ‘letterFrequency’
get.dataset.matrix: no visible global function definition for ‘samples’
get.diffmeth.tab.col.desc.list.txt: no visible binding for global
  variable ‘report’
get.diffmeth.tab.col.desc.list.txt: no visible binding for global
  variable ‘refText’
get.diffmeth.tab.col.desc.list.txt: no visible binding for global
  variable ‘diffmeth’
get.platform.tokens: no visible global function definition for
  ‘setNames’
getCGCounts: no visible global function definition for ‘seqlengths’
getCGCounts : &lt;anonymous&gt;: no visible global function definition for
  ‘ChrNumeric’
getGLADProfiles: no visible global function definition for ‘samples’
getGLADProfiles: no visible global function definition for ‘mclapply’
getGLADProfiles : &lt;anonymous&gt;: no visible global function definition
  for ‘as.profileCGH’
getGLADProfiles : &lt;anonymous&gt;: no visible global function definition
  for ‘daglad’
getMergeList: no visible global function definition for ‘samples’
groupPermutationP.site.parallel: no visible global function definition
  for ‘%dopar%’
groupPermutationP.site.parallel: no visible global function definition
  for ‘foreach’
iEVORA: no visible global function definition for ‘qvalue’
imputation.low.memory.cpgs: no visible global function definition for
  ‘samples’
inferWBCbyLme: no visible global function definition for ‘lme’
inferWBCbyLme: no visible global function definition for ‘getVarCov’
intensities.by.color: no visible binding for global variable
  ‘IlluminaHumanMethylationEPICmanifest’
intensities.by.color: no visible binding for global variable
  ‘IlluminaHumanMethylation450kmanifest’
intensities.by.color: no visible global function definition for
  ‘samples’
knn.imputation: no visible global function definition for ‘impute.knn’
limmaP: no visible binding for global variable ‘group1’
limmaP: no visible binding for global variable ‘group2’
loadLolaDbs: no visible global function definition for ‘loadRegionDB’
loadLolaDbs: no visible global function definition for ‘mergeRegionDBs’
locus.profile.get.base.tracks: no visible global function definition
  for ‘useMart’
locus.profile.get.base.tracks: no visible global function definition
  for ‘IdeogramTrack’
locus.profile.get.base.tracks: no visible global function definition
  for ‘GenomeAxisTrack’
locus.profile.get.base.tracks: no visible global function definition
  for ‘BiomartGeneRegionTrack’
locus.profile.get.base.tracks: no visible global function definition
  for ‘UcscTrack’
locus.profile.get.methylation.track.heatmap: no visible global function
  definition for ‘DataTrack’
locus.profile.get.methylation.track.smooth: no visible global function
  definition for ‘DataTrack’
methylumi.intensities.by.color: no visible global function definition
  for ‘featureNames’
methylumi.intensities.by.color: no visible binding for global variable
  ‘IlluminaHumanMethylation450kmanifest’
methylumi.intensities.by.color: no visible global function definition
  for ‘assayDataElement’
parallel.setup: no visible global function definition for
  ‘registerDoParallel’
parallel.setup: no visible global function definition for
  ‘getDoParWorkers’
performLolaEnrichment.diffMeth: no visible global function definition
  for ‘runLOLA’
performLolaEnrichment.diffVar: no visible global function definition
  for ‘runLOLA’
plot.heatmap.pc.correlations: no visible global function definition for
  ‘melt’
plot.heatmap.pc.correlations: no visible global function definition for
  ‘grid.newpage’
plot.heatmap.pc.correlations: no visible global function definition for
  ‘grid.draw’
plot.heatmap.pc.pvalues: no visible global function definition for
  ‘melt’
plot.heatmap.pc.pvalues: no visible global function definition for
  ‘grid.newpage’
plot.heatmap.pc.pvalues: no visible global function definition for
  ‘grid.draw’
plot.heatmap.rand: no visible global function definition for ‘melt’
plot.heatmap.symm: no visible global function definition for ‘melt’
plot.heatmap.symm: no visible global function definition for
  ‘grid.newpage’
plot.heatmap.symm: no visible global function definition for
  ‘grid.draw’
plotCGHProfile: no visible global function definition for ‘ChrNumeric’
prepareGEOSampleInfoTemplate: no visible global function definition for
  ‘samples’
projectWBC: no visible global function definition for ‘solve.QP’
read.GS.report: no visible global function definition for
  ‘featureNames’
read.idat.files: no visible binding for global variable ‘barcode’
rnb.RnBSet.to.GRangesList: no visible global function definition for
  ‘samples’
rnb.RnBSet.to.bed: no visible global function definition for ‘samples’
rnb.RnBSet.to.bedGraph: no visible global function definition for
  ‘samples’
rnb.bed.from.segmentation: no visible global function definition for
  ‘samples’
rnb.boxplot.from.segmentation: no visible global function definition
  for ‘samples’
rnb.boxplot.from.segmentation: no visible binding for global variable
  ‘Segment’
rnb.boxplot.from.segmentation: no visible binding for global variable
  ‘AvgMeth’
rnb.chromosome.lengths: no visible global function definition for
  ‘seqlengths’
rnb.combine.seq: no visible global function definition for ‘samples’
rnb.enmix.oob: no visible global function definition for ‘%dopar%’
rnb.enmix.oob: no visible global function definition for ‘foreach’
rnb.execute.dreduction: no visible global function definition for
  ‘samples’
rnb.execute.filter.summary.internal: no visible global function
  definition for ‘samples’
rnb.execute.high.coverage.removal.internal: no visible global function
  definition for ‘samples’
rnb.execute.high.dpval.masking.internal: no visible global function
  definition for ‘samples’
rnb.execute.imputation: no visible global function definition for
  ‘samples’
rnb.execute.low.coverage.masking.internal: no visible global function
  definition for ‘samples’
rnb.execute.na.removal.internal: no visible global function definition
  for ‘samples’
rnb.execute.normalization: no visible global function definition for
  ‘phenoData’
rnb.execute.normalization: no visible global function definition for
  ‘phenoData&lt;-’
rnb.execute.normalization: no visible global function definition for
  ‘RGChannelSet’
rnb.execute.normalization: no visible global function definition for
  ‘preprocessSWAN’
rnb.execute.normalization: no visible global function definition for
  ‘getMeth’
rnb.execute.normalization: no visible global function definition for
  ‘getUnmeth’
rnb.execute.normalization: no visible global function definition for
  ‘featureNames’
rnb.execute.normalization: no visible global function definition for
  ‘samples’
rnb.execute.normalization: no visible global function definition for
  ‘mapToGenome’
rnb.execute.normalization: no visible global function definition for
  ‘addSex’
rnb.execute.normalization: no visible global function definition for
  ‘getSex’
rnb.execute.normalization: no visible global function definition for
  ‘getCN’
rnb.execute.normalization.bmiq: no visible global function definition
  for ‘%dopar%’
rnb.execute.normalization.bmiq: no visible global function definition
  for ‘foreach’
rnb.execute.pOOBAH: no visible global function definition for ‘samples’
rnb.execute.pOOBAH: no visible global function definition for ‘SigDF’
rnb.execute.pOOBAH: no visible global function definition for ‘pOOBAH’
rnb.execute.pOOBAH: no visible binding for global variable ‘sigset.l’
rnb.execute.pOOBAH: no visible binding for global variable ‘pIG’
rnb.execute.pOOBAH: no visible binding for global variable ‘pII’
rnb.execute.pOOBAH: no visible binding for global variable ‘pIR’
rnb.execute.quality: no visible global function definition for
  ‘samples’
rnb.execute.quality : &lt;anonymous&gt;: no visible global function
  definition for ‘samples’
rnb.execute.segmentation: no visible binding for global variable
  ‘Hsapiens’
rnb.execute.segmentation: no visible binding for global variable
  ‘Mmusculus’
rnb.execute.segmentation: no visible global function definition for
  ‘seqlengths’
rnb.execute.segmentation: no visible global function definition for
  ‘browserSession’
rnb.execute.segmentation: no visible global function definition for
  ‘genome&lt;-’
rnb.execute.segmentation: no visible global function definition for
  ‘ucscTableQuery’
rnb.execute.segmentation: no visible global function definition for
  ‘getTable’
rnb.execute.segmentation: no visible global function definition for
  ‘samples’
rnb.execute.segmentation: no visible global function definition for
  ‘readMethylome’
rnb.execute.segmentation: no visible global function definition for
  ‘readSNPTable’
rnb.execute.segmentation: no visible global function definition for
  ‘removeSNPs’
rnb.execute.segmentation: no visible global function definition for
  ‘segmentPMDs’
rnb.execute.segmentation: no visible global function definition for
  ‘plotAlphaDistributionOneChr’
rnb.execute.segmentation: no visible global function definition for
  ‘calculateFDRs’
rnb.execute.segmentation: no visible global function definition for
  ‘segmentUMRsLMRs’
rnb.export.to.ewasher: no visible global function definition for
  ‘samples’
rnb.filter.dataset: no visible global function definition for ‘samples’
rnb.final.segmentation: no visible global function definition for
  ‘samples’
rnb.get.reliability.counts.per.sample: no visible global function
  definition for ‘samples’
rnb.options.description.table.fromRd: no visible global function
  definition for ‘Rd2HTML’
rnb.options.description.table.fromRd: no visible binding for global
  variable ‘xmlValue’
rnb.plot.beta.density.group: no visible binding for global variable
  ‘group’
rnb.plot.biseq.coverage: no visible global function definition for
  ‘samples’
rnb.plot.biseq.coverage.hist: no visible global function definition for
  ‘samples’
rnb.plot.control.barplot: no visible global function definition for
  ‘samples’
rnb.plot.control.barplot: no visible binding for global variable
  ‘Target’
rnb.plot.control.barplot: no visible binding for global variable
  ‘Index’
rnb.plot.control.barplot: no visible binding for global variable ‘ID’
rnb.plot.control.barplot: no visible binding for global variable
  ‘Address’
rnb.plot.control.barplot: no visible global function definition for
  ‘grid.newpage’
rnb.plot.control.barplot: no visible binding for global variable
  ‘Intensity’
rnb.plot.control.barplot: no visible global function definition for
  ‘grid.draw’
rnb.plot.control.boxplot: no visible global function definition for
  ‘grid.newpage’
rnb.plot.control.boxplot: no visible binding for global variable
  ‘Probe’
rnb.plot.control.boxplot: no visible binding for global variable
  ‘Intensity’
rnb.plot.control.boxplot: no visible global function definition for
  ‘grid.draw’
rnb.plot.ct.heatmap: no visible binding for global variable ‘x’
rnb.plot.ct.heatmap: no visible binding for global variable ‘y’
rnb.plot.ct.heatmap: no visible binding for global variable ‘v’
rnb.plot.ct.heatmap: no visible global function definition for
  ‘grid.newpage’
rnb.plot.ct.heatmap: no visible global function definition for
  ‘grid.draw’
rnb.plot.dreduction: no visible global function definition for ‘tsne’
rnb.plot.locus.profile: no visible global function definition for
  ‘plotTracks’
rnb.plot.marker.fstat: no visible binding for global variable ‘x’
rnb.plot.marker.fstat: no visible binding for global variable ‘y’
rnb.plot.marker.fstat: no visible global function definition for
  ‘muted’
rnb.plot.negative.boxplot: no visible global function definition for
  ‘samples’
rnb.plot.negative.boxplot: no visible binding for global variable
  ‘numeric.names’
rnb.plot.negative.boxplot: no visible binding for global variable
  ‘type’
rnb.plot.negative.boxplot: no visible binding for global variable
  ‘types’
rnb.plot.negative.boxplot: no visible global function definition for
  ‘grid.newpage’
rnb.plot.negative.boxplot: no visible binding for global variable
  ‘Sample’
rnb.plot.negative.boxplot: no visible binding for global variable
  ‘Intensity’
rnb.plot.negative.boxplot: no visible global function definition for
  ‘grid.draw’
rnb.plot.num.sites.covg: no visible global function definition for
  ‘samples’
rnb.plot.num.sites.covg : &lt;anonymous&gt;: no visible global function
  definition for ‘samples’
rnb.plot.num.sites.covg: no visible binding for global variable
  ‘numSites’
rnb.plot.num.sites.covg: no visible binding for global variable
  ‘covgMedian’
rnb.plot.num.sites.covg: no visible binding for global variable
  ‘covgPercLow’
rnb.plot.num.sites.covg: no visible binding for global variable
  ‘covgPercUp’
rnb.plot.num.sites.covg: no visible binding for global variable ‘comma’
rnb.plot.region.profile.density: no visible binding for global variable
  ‘relative.coord’
rnb.plot.region.profile.density: no visible binding for global variable
  ‘..density..’
rnb.plot.region.profiles : &lt;anonymous&gt;: no visible global function
  definition for ‘samples’
rnb.plot.region.profiles: no visible binding for global variable
  ‘relative.coord’
rnb.plot.region.profiles: no visible binding for global variable
  ‘group’
rnb.plot.region.site.density: no visible binding for global variable
  ‘relative.coord’
rnb.plot.segmentation.distributions: no visible binding for global
  variable ‘x2’
rnb.plot.segmentation.distributions: no visible binding for global
  variable ‘y2’
rnb.plot.segmentation.distributions: no visible binding for global
  variable ‘texthere’
rnb.plot.segmentation.final: no visible global function definition for
  ‘plotFinalSegmentation’
rnb.plot.sentrix.distributions: no visible global function definition
  for ‘samples’
rnb.plot.sentrix.distributions: no visible binding for global variable
  ‘Slide’
rnb.plot.snp.boxplot: no visible binding for global variable ‘SNP’
rnb.plot.snp.boxplot: no visible binding for global variable ‘bv’
rnb.run.example: no visible global function definition for ‘install’
rnb.run.preprocessing : logger.completed.filtering: no visible global
  function definition for ‘samples’
rnb.sample.summary.table: no visible global function definition for
  ‘samples’
rnb.section.ageprediction: no visible global function definition for
  ‘samples’
rnb.section.clustering : &lt;anonymous&gt; : &lt;anonymous&gt;: no visible global
  function definition for ‘samples’
rnb.section.clustering: no visible global function definition for
  ‘samples’
rnb.section.diffMeth.region: no visible global function definition for
  ‘%dopar%’
rnb.section.diffMeth.region: no visible global function definition for
  ‘foreach’
rnb.section.diffMeth.region: no visible binding for global variable ‘k’
rnb.section.diffMeth.region : do.enrichment.table: no visible global
  function definition for ‘sigCategories’
rnb.section.diffMeth.region : do.enrichment.table.var: no visible
  global function definition for ‘sigCategories’
rnb.section.diffMeth.site: no visible global function definition for
  ‘%dopar%’
rnb.section.diffMeth.site: no visible global function definition for
  ‘foreach’
rnb.section.diffVar: no visible global function definition for
  ‘%dopar%’
rnb.section.diffVar: no visible global function definition for
  ‘foreach’
rnb.section.diffVar.region: no visible global function definition for
  ‘%dopar%’
rnb.section.diffVar.region: no visible global function definition for
  ‘foreach’
rnb.section.diffVar.region: no visible binding for global variable ‘k’
rnb.section.import: no visible global function definition for ‘samples’
rnb.section.import: no visible global function definition for
  ‘phenoData’
rnb.section.imputation: no visible global function definition for
  ‘melt’
rnb.section.locus.profiles: no visible global function definition for
  ‘samples’
rnb.section.locus.profiles : do.plots: no visible global function
  definition for ‘plotTracks’
rnb.section.locus.profiles: no visible global function definition for
  ‘%dopar%’
rnb.section.locus.profiles: no visible global function definition for
  ‘foreach’
rnb.section.na.removal: no visible global function definition for
  ‘samples’
rnb.section.normalization: no visible global function definition for
  ‘samples’
rnb.section.region.description: no visible binding for global variable
  ‘size’
rnb.section.region.description: no visible binding for global variable
  ‘n.sites’
rnb.section.region.profiles: no visible global function definition for
  ‘samples’
rnb.section.region.subsegmentation : get.summary.df.from.list: no
  visible global function definition for ‘melt’
rnb.section.region.subsegmentation: no visible binding for global
  variable ‘region.size’
rnb.section.region.subsegmentation: no visible binding for global
  variable ‘is.subsegmentation’
rnb.section.region.subsegmentation: no visible binding for global
  variable ‘num.sites’
rnb.section.sex.prediction: no visible global function definition for
  ‘muted’
rnb.step.betadistribution.internal: no visible global function
  definition for ‘%dopar%’
rnb.step.betadistribution.internal: no visible global function
  definition for ‘foreach’
rnb.step.cell.types: no visible global function definition for
  ‘samples’
rnb.step.clustering.internal: no visible global function definition for
  ‘samples’
rnb.step.dreduction: no visible global function definition for
  ‘%dopar%’
rnb.step.dreduction: no visible global function definition for
  ‘foreach’
rnb.step.dreduction: no visible binding for global variable ‘target’
rnb.step.filter.summary.internal: no visible global function definition
  for ‘percent_format’
rnb.step.filter.summary.internal: no visible global function definition
  for ‘samples’
rnb.step.import: no visible global function definition for ‘samples’
rnb.step.import: no visible global function definition for ‘phenoData’
rnb.step.na.removal: no visible binding for global variable
  ‘sites2ignore’
rnb.step.na.removal.internal: no visible global function definition for
  ‘samples’
rnb.step.normalization: no visible global function definition for
  ‘phenoData’
robustHyperGResultSummary: no visible global function definition for
  ‘sigCategories’
robustHyperGResultSummary: no visible global function definition for
  ‘pvalues’
robustHyperGResultSummary: no visible global function definition for
  ‘oddsRatios’
robustHyperGResultSummary: no visible global function definition for
  ‘expectedCounts’
robustHyperGResultSummary: no visible global function definition for
  ‘geneCounts’
robustHyperGResultSummary: no visible global function definition for
  ‘universeCounts’
robustHyperGResultSummary: no visible binding for global variable
  ‘Term’
run.cross.validation: no visible global function definition for
  ‘samples’
run.cross.validation: no visible global function definition for ‘melt’
run.cross.validation: no visible binding for global variable ‘Measure’
run.cross.validation: no visible binding for global variable ‘Error’
sampleCovgApply: no visible global function definition for ‘samples’
set.covariates.ct: no visible global function definition for ‘samples’
simpleGlmnet: no visible global function definition for ‘impute.knn’
simpleGlmnet: no visible global function definition for ‘cv.glmnet’
simpleGlmnet: no visible global function definition for ‘glmnet’
simpleGlmnetBiseq: no visible global function definition for
  ‘cv.glmnet’
simpleGlmnetBiseq: no visible global function definition for ‘glmnet’
simpleGlmnetEvaluate: no visible global function definition for
  ‘cv.glmnet’
simpleGlmnetEvaluate: no visible global function definition for
  ‘glmnet’
symmetric.melt: no visible global function definition for ‘melt’
addDiffMethTable,RnBDiffMeth: no visible binding for global variable
  ‘reg.type’
coerce,RGChannelSet-RnBeadRawSet: no visible global function definition
  for ‘getManifest’
coerce,RGChannelSet-RnBeadRawSet: no visible global function definition
  for ‘getGreen’
coerce,RGChannelSet-RnBeadRawSet: no visible global function definition
  for ‘getRed’
coerce,RGChannelSet-RnBeadRawSet: no visible global function definition
  for ‘pData’
coerce,RnBeadRawSet-MethyLumiSet: no visible global function definition
  for ‘featureData&lt;-’
coerce,RnBeadRawSet-MethyLumiSet: no visible global function definition
  for ‘featureNames&lt;-’
coerce,RnBeadRawSet-RGChannelSet: no visible global function definition
  for ‘RGChannelSet’
join.diffMeth,RnBDiffMeth-RnBDiffMeth: no visible global function
  definition for ‘clone’
mask.sites.meth,RnBSet: no visible global function definition for
  ‘samples’
remove.samples,RnBSet: no visible global function definition for
  ‘samples’
remove.samples,RnBeadRawSet: no visible global function definition for
  ‘samples’
remove.samples,RnBeadSet: no visible global function definition for
  ‘samples’
remove.sites,RnBSet: no visible global function definition for
  ‘samples’
sampleCovgApply,RnBSet: no visible global function definition for
  ‘samples’
sampleMethApply,RnBSet: no visible global function definition for
  ‘samples’
summarize.regions,RnBSet: no visible global function definition for
  ‘samples’
summarize.regions,RnBSet: multiple local function definitions for
  ‘aggr.f’ with different formal arguments
updateMethylationSites,RnBSet: no visible global function definition
  for ‘samples’
Undefined global functions or variables:
  %dopar% ..count.. ..density.. .inds.g2 Address Age Annotated AvgMeth
  BiomartGeneRegionTrack ChrNumeric DataTrack Density Deviance
  Difference DoISVA Error GenomeAxisTrack Group Hsapiens ID
  IdeogramTrack IlluminaHumanMethylation450kmanifest
  IlluminaHumanMethylationEPICmanifest Immune Increase Index Intensity
  Measure Mmusculus Predicted Probe RGChannelSet Rd2HTML SNP Sample
  Segment SigDF Slide Target Term UcscTrack Value addSex as.profileCGH
  assayDataElement assayDataElementNames barcode browserSession bv
  calculateFDRs chrom clone combinedRank combinedRank.var comma
  covgMedian covgPercLow covgPercUp cv.glmnet daglad diffmeth
  diffmeth.p.adj.fdr diffmeth.p.val dinucleotideFrequency eps
  expectedCounts featureData featureData&lt;- featureNames featureNames&lt;-
  foreach geneCounts genome&lt;- getCN getDoParWorkers getGreen
  getManifest getMeth getRed getSex getTable getUnmeth getVarCov glmnet
  grid.draw grid.newpage group group1 group2 i impute.knn install
  is.subsegmentation k letterFrequency lme loadRegionDB log10FDR log10P
  mapToGenome mclapply mean.diff mean.quot.log2 melt mergeRegionDBs
  muted n.sites num.sites numSites numeric.names oddsRatios
  p.vals.t.na.adj pData pIG pII pIR pOOBAH percent_format phenoData
  phenoData&lt;- plotAlphaDistributionOneChr plotFinalSegmentation
  plotTracks preprocessSWAN pvalues qvalue readMethylome readSNPTable
  refText reg.type region.size registerDoParallel relative.coord
  removeSNPs report runLOLA samples segmentPMDs segmentUMRsLMRs
  seqlengths seqlevels&lt;- setNames sigCategories sigset.l sites2ignore
  size solve.QP stopCluster sva target texthere tsne type types
  ucscTableQuery universeCounts useMart v var.diff varFit varLabels x
  x2 xmlValue y y2 yint
Consider adding
  importFrom(&quot;stats&quot;, &quot;setNames&quot;)
to your NAMESPACE file.
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                     user system elapsed
rnb.execute.pOOBAH 21.198  0.907  24.462
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘RnBeads.Rnw’... OK
  ‘RnBeads_Annotations.Rnw’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 7 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/RnBeads.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">RnBeads.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL RnBeads
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘RnBeads’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (RnBeads)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">RnBeads.Rcheck/tests/runTests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; BiocGenerics:::testPackage(&quot;RnBeads&quot;)

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min


Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname


Attaching package: &#x27;bit&#x27;

The following object is masked from &#x27;package:base&#x27;:

    xor

Attaching package ff
- getOption(&quot;fftempdir&quot;)==&quot;/tmp/RtmpWA01dd/ff&quot;

- getOption(&quot;ffextension&quot;)==&quot;ff&quot;

- getOption(&quot;ffdrop&quot;)==TRUE

- getOption(&quot;fffinonexit&quot;)==TRUE

- getOption(&quot;ffpagesize&quot;)==65536

- getOption(&quot;ffcaching&quot;)==&quot;mmnoflush&quot;  -- consider &quot;ffeachflush&quot; if your system stalls on large writes

- getOption(&quot;ffbatchbytes&quot;)==16777216 -- consider a different value for tuning your system

- getOption(&quot;ffmaxbytes&quot;)==536870912 -- consider a different value for tuning your system


Attaching package: &#x27;ff&#x27;

The following objects are masked from &#x27;package:utils&#x27;:

    write.csv, write.csv2

The following objects are masked from &#x27;package:base&#x27;:

    is.factor, is.ordered

Spam version 2.9-1 (2022-08-07) is loaded.
Type &#x27;help( Spam)&#x27; or &#x27;demo( spam)&#x27; for a short introduction 
and overview of this package.
Help for individual functions is also obtained by adding the
suffix &#x27;.spam&#x27; to the function name, e.g. &#x27;help( chol.spam)&#x27;.

Attaching package: &#x27;spam&#x27;

The following object is masked from &#x27;package:stats4&#x27;:

    mle

The following objects are masked from &#x27;package:base&#x27;:

    backsolve, forwardsolve


Try help(fields) to get started.

Attaching package: &#x27;gplots&#x27;

The following object is masked from &#x27;package:IRanges&#x27;:

    space

The following object is masked from &#x27;package:S4Vectors&#x27;:

    space

The following object is masked from &#x27;package:stats&#x27;:

    lowess


Attaching package: &#x27;grid&#x27;

The following object is masked from &#x27;package:ff&#x27;:

    pattern


Attaching package: &#x27;gridExtra&#x27;

The following object is masked from &#x27;package:BiocGenerics&#x27;:

    combine


Attaching package: &#x27;limma&#x27;

The following object is masked from &#x27;package:BiocGenerics&#x27;:

    plotMA

Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


Attaching package: &#x27;Biobase&#x27;

The following objects are masked from &#x27;package:matrixStats&#x27;:

    anyMissing, rowMedians


Attaching package: &#x27;scales&#x27;

The following object is masked from &#x27;package:viridis&#x27;:

    viridis_pal


Attaching package: &#x27;AnnotationDbi&#x27;

The following object is masked from &#x27;package:MASS&#x27;:

    select



Attaching package: &#x27;MatrixGenerics&#x27;

The following object is masked from &#x27;package:Biobase&#x27;:

    rowMedians

The following objects are masked from &#x27;package:matrixStats&#x27;:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars


Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:grid&#x27;:

    pattern

The following objects are masked from &#x27;package:ff&#x27;:

    mismatch, pattern

The following object is masked from &#x27;package:base&#x27;:

    strsplit

locfit 1.5-9.7 	 2023-01-02
Setting options(&#x27;download.file.method.GEOquery&#x27;=&#x27;auto&#x27;)
Setting options(&#x27;GEOquery.inmemory.gpl&#x27;=FALSE)

Attaching package: &#x27;plyr&#x27;

The following object is masked from &#x27;package:XVector&#x27;:

    compact

The following object is masked from &#x27;package:matrixStats&#x27;:

    count

The following object is masked from &#x27;package:IRanges&#x27;:

    desc

The following object is masked from &#x27;package:S4Vectors&#x27;:

    rename

2023-02-28 13:38:51     1.1  STATUS STARTED Unit testing: differential
2023-02-28 13:38:51     1.1  STATUS     STARTED Testing function: rowWelchP
Loading required package: RnBeads.hg19
2023-02-28 13:38:51     1.1  STATUS     COMPLETED Testing function: rowWelchP
2023-02-28 13:38:51     1.1  STATUS     STARTED Testing function: limmaP
2023-02-28 13:38:51     1.1  STATUS     COMPLETED Testing function: limmaP
2023-02-28 13:38:51     1.1  STATUS     STARTED Testing function: computeDiffTab.extended.site
2023-02-28 13:38:51     1.1    INFO         Conducting differential analysis using limma
2023-02-28 13:38:51     1.1  STATUS     COMPLETED Testing function: computeDiffTab.extended.site
2023-02-28 13:38:51     1.1  STATUS     STARTED Testing function: computeDiffTab.default.region
2023-02-28 13:38:51     1.1    INFO         Conducting differential analysis using limma
2023-02-28 13:38:55     1.2  STATUS     COMPLETED Testing function: computeDiffTab.default.region
2023-02-28 13:38:55     1.2  STATUS     STARTED Testing function: combineTestPvalsMeth
2023-02-28 13:38:55     1.2  STATUS     COMPLETED Testing function: combineTestPvalsMeth
2023-02-28 13:38:55     1.2  STATUS     STARTED Testing function: get.adjustment.variables
2023-02-28 13:38:55     1.2  STATUS     COMPLETED Testing function: get.adjustment.variables
2023-02-28 13:38:55     1.2  STATUS     STARTED Testing function: get.comparison.info
2023-02-28 13:38:55     1.2  STATUS     COMPLETED Testing function: get.comparison.info
2023-02-28 13:38:55     1.2  STATUS     STARTED Testing function: rnb.execute.computeDiffMeth
2023-02-28 13:38:55     1.2  STATUS         STARTED Retrieving comparison info
2023-02-28 13:38:55     1.2  STATUS         COMPLETED Retrieving comparison info
2023-02-28 13:38:55     1.2  STATUS         STARTED Computing differential methylation tables
2023-02-28 13:38:55     1.2  STATUS             STARTED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:38:55     1.2  STATUS                 STARTED Computing Differential Methylation Table
2023-02-28 13:38:55     1.2    INFO                     Conducting differential analysis using limma
2023-02-28 13:38:55     1.2  STATUS                 COMPLETED Computing Differential Methylation Table
2023-02-28 13:38:55     1.2  STATUS                 STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:38:56     1.3  STATUS                     Computed table for tiling
2023-02-28 13:38:56     1.4  STATUS                     Computed table for genes
2023-02-28 13:38:57     1.4  STATUS                     Computed table for promoters
2023-02-28 13:38:57     1.4  STATUS                     Computed table for cpgislands
2023-02-28 13:38:57     1.4  STATUS                 COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:38:57     1.4  STATUS             COMPLETED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:38:57     1.4  STATUS             STARTED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:38:57     1.4  STATUS                 STARTED Computing Differential Methylation Table
2023-02-28 13:38:57     1.4    INFO                     Conducting differential analysis using limma
2023-02-28 13:38:57     1.4  STATUS                 COMPLETED Computing Differential Methylation Table
2023-02-28 13:38:57     1.4  STATUS                 STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:38:59     1.4  STATUS                     Computed table for tiling
2023-02-28 13:38:59     1.4  STATUS                     Computed table for genes
2023-02-28 13:39:00     1.4  STATUS                     Computed table for promoters
2023-02-28 13:39:00     1.4  STATUS                     Computed table for cpgislands
2023-02-28 13:39:00     1.4  STATUS                 COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:39:00     1.4  STATUS             COMPLETED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:39:00     1.4  STATUS         COMPLETED Computing differential methylation tables
2023-02-28 13:39:00     1.4  STATUS     COMPLETED Testing function: rnb.execute.computeDiffMeth
2023-02-28 13:39:00     1.4  STATUS     STARTED Testing function: diffVar
2023-02-28 13:39:00     1.4  STATUS         STARTED diffVar method
coef not specified. Using all columns of design matrix.
2023-02-28 13:40:15     4.1  STATUS         COMPLETED diffVar method
2023-02-28 13:40:16     4.1  STATUS         STARTED diffVar method
coef not specified. Using all columns of design matrix.
2023-02-28 13:40:16     4.1 WARNING             Could not compute p-values with diffVar, returning NA
2023-02-28 13:40:16     4.1    INFO             diffVar from missMethyl package not properly running
2023-02-28 13:40:16     4.1  STATUS         COMPLETED diffVar method
2023-02-28 13:40:16     4.1  STATUS         STARTED Testing function: apply.iEVORA
2023-02-28 13:40:16     4.1  STATUS             STARTED iEVORA method
2023-02-28 13:40:17     4.1    INFO                 No DVCs detected. All p-values set to 1.
2023-02-28 13:40:17     4.1  STATUS             COMPLETED iEVORA method
2023-02-28 13:40:17     4.1  STATUS             STARTED Testing function: rnb.execute.diffVar
2023-02-28 13:40:17     4.1  STATUS                 STARTED Differential Variability
2023-02-28 13:40:17     4.1  STATUS                     STARTED Retrieving comparison info
2023-02-28 13:40:17     4.1  STATUS                     COMPLETED Retrieving comparison info
2023-02-28 13:40:17     4.1    INFO                     No imputation method selected, &#x27;knn&#x27; method used.
2023-02-28 13:40:17     4.1  STATUS                     STARTED Imputation procedure knn 
2023-02-28 13:40:17     4.1  STATUS                     COMPLETED Imputation procedure knn 
2023-02-28 13:40:17     4.1  STATUS                     STARTED Comparing  hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:17     4.1    INFO                         Conducting differential variability using diffVar
2023-02-28 13:40:17     4.1  STATUS                         STARTED diffVar method
coef not specified. Using all columns of design matrix.
2023-02-28 13:40:17     4.1  STATUS                         COMPLETED diffVar method
2023-02-28 13:40:18     3.9  STATUS                         STARTED Computing Differential Variability Tables (Region Level)
2023-02-28 13:40:19     3.9  STATUS                             Computed table for tiling
2023-02-28 13:40:19     3.9  STATUS                             Computed table for genes
2023-02-28 13:40:19     3.9  STATUS                             Computed table for promoters
2023-02-28 13:40:19     3.9  STATUS                             Computed table for cpgislands
2023-02-28 13:40:19     3.9  STATUS                         COMPLETED Computing Differential Variability Tables (Region Level)
2023-02-28 13:40:19     3.9  STATUS                     COMPLETED Comparing  hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:19     3.9  STATUS                     STARTED Comparing  KOSR vs. TeSR (based on Treatment)
2023-02-28 13:40:19     3.9    INFO                         Conducting differential variability using diffVar
2023-02-28 13:40:19     3.9  STATUS                         STARTED diffVar method
coef not specified. Using all columns of design matrix.
2023-02-28 13:40:19     3.9  STATUS                         COMPLETED diffVar method
2023-02-28 13:40:22     3.9  STATUS                         STARTED Computing Differential Variability Tables (Region Level)
2023-02-28 13:40:22     3.9  STATUS                             Computed table for tiling
2023-02-28 13:40:23     3.9  STATUS                             Computed table for genes
2023-02-28 13:40:23     3.9  STATUS                             Computed table for promoters
2023-02-28 13:40:23     3.9  STATUS                             Computed table for cpgislands
2023-02-28 13:40:23     3.9  STATUS                         COMPLETED Computing Differential Variability Tables (Region Level)
2023-02-28 13:40:23     3.9  STATUS                     COMPLETED Comparing  KOSR vs. TeSR (based on Treatment)
2023-02-28 13:40:23     3.9  STATUS                 COMPLETED Differential Variability
2023-02-28 13:40:23     3.9  STATUS             COMPLETED Testing function: rnb.execute.diffVar
2023-02-28 13:40:23     3.9  STATUS             STARTED Testing function: rnb.execute.computeDiffMeth and Variability
2023-02-28 13:40:23     3.9  STATUS                 STARTED Retrieving comparison info
2023-02-28 13:40:23     3.3  STATUS                 COMPLETED Retrieving comparison info
2023-02-28 13:40:28     3.3  STATUS                 STARTED Computing differential methylation tables
2023-02-28 13:40:28     3.3  STATUS                     STARTED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:28     3.3  STATUS                         STARTED Computing Differential Methylation Table
2023-02-28 13:40:28     3.3    INFO                             Conducting differential analysis using limma
2023-02-28 13:40:28     3.3  STATUS                             STARTED Imputation procedure knn 
2023-02-28 13:40:28     3.3  STATUS                             COMPLETED Imputation procedure knn 
2023-02-28 13:40:28     3.3    INFO                             Conducting differential variability using diffVar
2023-02-28 13:40:28     3.3  STATUS                             STARTED diffVar method
coef not specified. Using all columns of design matrix.
2023-02-28 13:40:28     3.3  STATUS                             COMPLETED diffVar method
2023-02-28 13:40:29     3.3  STATUS                         COMPLETED Computing Differential Methylation Table
2023-02-28 13:40:29     3.3  STATUS                         STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:29     3.3  STATUS                             Computed table for tiling
2023-02-28 13:40:30     3.3  STATUS                             Computed table for genes
2023-02-28 13:40:30     3.4  STATUS                             Computed table for promoters
2023-02-28 13:40:31     3.5  STATUS                             Computed table for cpgislands
2023-02-28 13:40:31     3.5  STATUS                         COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:31     3.5  STATUS                     COMPLETED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:31     3.5  STATUS                     STARTED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:40:31     3.5  STATUS                         STARTED Computing Differential Methylation Table
2023-02-28 13:40:31     3.5    INFO                             Conducting differential analysis using limma
2023-02-28 13:40:31     3.5  STATUS                             STARTED Imputation procedure knn 
2023-02-28 13:40:31     3.5  STATUS                             COMPLETED Imputation procedure knn 
2023-02-28 13:40:31     3.5    INFO                             Conducting differential variability using diffVar
2023-02-28 13:40:31     3.5  STATUS                             STARTED diffVar method
coef not specified. Using all columns of design matrix.
2023-02-28 13:40:31     3.5  STATUS                             COMPLETED diffVar method
2023-02-28 13:40:31     3.5  STATUS                         COMPLETED Computing Differential Methylation Table
2023-02-28 13:40:31     3.5  STATUS                         STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:32     3.7  STATUS                             Computed table for tiling
2023-02-28 13:40:32     3.9  STATUS                             Computed table for genes
2023-02-28 13:40:34     3.9  STATUS                             Computed table for promoters
2023-02-28 13:40:34     3.9  STATUS                             Computed table for cpgislands
2023-02-28 13:40:34     3.9  STATUS                         COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:34     3.9  STATUS                     COMPLETED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:40:34     3.9  STATUS                 COMPLETED Computing differential methylation tables
2023-02-28 13:40:34     3.9  STATUS             COMPLETED Testing function: rnb.execute.computeDiffMeth and Variability
2023-02-28 13:40:34     3.9  STATUS             STARTED Testing class: RnBDiffMeth
2023-02-28 13:40:34     3.9  STATUS                 STARTED Retrieving comparison info
2023-02-28 13:40:34     3.9  STATUS                 COMPLETED Retrieving comparison info
2023-02-28 13:40:34     3.9  STATUS                 STARTED Computing differential methylation tables
2023-02-28 13:40:34     3.9  STATUS                     STARTED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:34     3.9  STATUS                         STARTED Computing Differential Methylation Table
2023-02-28 13:40:34     3.9    INFO                             Conducting differential analysis using limma
2023-02-28 13:40:34     3.9  STATUS                         COMPLETED Computing Differential Methylation Table
2023-02-28 13:40:34     3.9  STATUS                         STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:35     3.9  STATUS                             Computed table for tiling
2023-02-28 13:40:35     3.9  STATUS                             Computed table for genes
2023-02-28 13:40:36     3.9  STATUS                             Computed table for promoters
2023-02-28 13:40:36     3.9  STATUS                             Computed table for cpgislands
2023-02-28 13:40:36     3.9  STATUS                         COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:36     3.9  STATUS                     COMPLETED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:36     3.9  STATUS                     STARTED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:40:36     3.9  STATUS                         STARTED Computing Differential Methylation Table
2023-02-28 13:40:36     3.9    INFO                             Conducting differential analysis using limma
2023-02-28 13:40:36     3.9  STATUS                         COMPLETED Computing Differential Methylation Table
2023-02-28 13:40:36     3.9  STATUS                         STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:36     3.9  STATUS                             Computed table for tiling
2023-02-28 13:40:37     3.9  STATUS                             Computed table for genes
2023-02-28 13:40:37     3.9  STATUS                             Computed table for promoters
2023-02-28 13:40:37     3.9  STATUS                             Computed table for cpgislands
2023-02-28 13:40:37     3.9  STATUS                         COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:37     3.9  STATUS                     COMPLETED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:40:37     3.9  STATUS                 COMPLETED Computing differential methylation tables
2023-02-28 13:40:37     3.9  STATUS                 STARTED Retrieving comparison info
2023-02-28 13:40:37     3.9  STATUS                 COMPLETED Retrieving comparison info
2023-02-28 13:40:37     3.9  STATUS                 STARTED Computing differential methylation tables
2023-02-28 13:40:37     3.9  STATUS                     STARTED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:37     3.9  STATUS                         STARTED Computing Differential Methylation Table
2023-02-28 13:40:37     3.9    INFO                             Conducting differential analysis using limma
2023-02-28 13:40:37     3.9  STATUS                         COMPLETED Computing Differential Methylation Table
2023-02-28 13:40:37     3.9  STATUS                         STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:38     3.9  STATUS                             Computed table for genes
2023-02-28 13:40:39     3.9  STATUS                             Computed table for tiling
2023-02-28 13:40:39     3.9  STATUS                         COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:39     3.9  STATUS                     COMPLETED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:39     3.9  STATUS                 COMPLETED Computing differential methylation tables
2023-02-28 13:40:39     3.9  STATUS                 STARTED Retrieving comparison info
2023-02-28 13:40:39     3.9  STATUS                 COMPLETED Retrieving comparison info
2023-02-28 13:40:39     3.9  STATUS                 STARTED Computing differential methylation tables
2023-02-28 13:40:40     3.9  STATUS                     STARTED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:40     3.9  STATUS                         STARTED Computing Differential Methylation Table
2023-02-28 13:40:40     3.9    INFO                             Conducting differential analysis using limma
2023-02-28 13:40:40     3.9  STATUS                         COMPLETED Computing Differential Methylation Table
2023-02-28 13:40:40     3.9  STATUS                         STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:40     3.9  STATUS                             Computed table for genes
2023-02-28 13:40:40     3.9  STATUS                             Computed table for tiling
2023-02-28 13:40:40     3.9  STATUS                         COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:40     3.9  STATUS                     COMPLETED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:40     3.9  STATUS                 COMPLETED Computing differential methylation tables
2023-02-28 13:40:40     3.9  STATUS                 STARTED Retrieving comparison info
2023-02-28 13:40:40     3.9  STATUS                 COMPLETED Retrieving comparison info
2023-02-28 13:40:40     3.9  STATUS                 STARTED Computing differential methylation tables
2023-02-28 13:40:40     3.9  STATUS                     STARTED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:40     3.9  STATUS                         STARTED Computing Differential Methylation Table
2023-02-28 13:40:40     3.9    INFO                             Conducting differential analysis using limma
2023-02-28 13:40:40     3.9  STATUS                         COMPLETED Computing Differential Methylation Table
2023-02-28 13:40:40     3.9  STATUS                         STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:41     3.9  STATUS                             Computed table for promoters
2023-02-28 13:40:41     3.9  STATUS                         COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:41     3.9  STATUS                     COMPLETED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:41     3.9  STATUS                     STARTED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:40:41     3.9  STATUS                         STARTED Computing Differential Methylation Table
2023-02-28 13:40:41     3.9    INFO                             Conducting differential analysis using limma
2023-02-28 13:40:41     3.9  STATUS                         COMPLETED Computing Differential Methylation Table
2023-02-28 13:40:41     3.9  STATUS                         STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:41     3.9  STATUS                             Computed table for promoters
2023-02-28 13:40:41     3.9  STATUS                         COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:41     3.9  STATUS                     COMPLETED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:40:41     3.9  STATUS                 COMPLETED Computing differential methylation tables
2023-02-28 13:40:41     3.9  STATUS                 STARTED Retrieving comparison info
2023-02-28 13:40:41     3.9  STATUS                 COMPLETED Retrieving comparison info
2023-02-28 13:40:41     3.9  STATUS                 STARTED Computing differential methylation tables
2023-02-28 13:40:41     3.9  STATUS                     STARTED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:40:41     3.9  STATUS                         STARTED Computing Differential Methylation Table
2023-02-28 13:40:41     3.9    INFO                             Conducting differential analysis using limma
2023-02-28 13:40:41     3.9  STATUS                         COMPLETED Computing Differential Methylation Table
2023-02-28 13:40:41     3.9  STATUS                         STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:41     3.9  STATUS                             Computed table for genes
2023-02-28 13:40:42     3.9  STATUS                             Computed table for tiling
2023-02-28 13:40:42     3.9  STATUS                         COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:42     3.9  STATUS                     COMPLETED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:40:42     3.9  STATUS                 COMPLETED Computing differential methylation tables
2023-02-28 13:40:42     3.9  STATUS                 STARTED Testing function: get.region.types
2023-02-28 13:40:42     3.9  STATUS                 COMPLETED Testing function: get.region.types
2023-02-28 13:40:42     3.9  STATUS                 STARTED Testing function: get.comparisons
2023-02-28 13:40:42     3.9  STATUS                 COMPLETED Testing function: get.comparisons
2023-02-28 13:40:42     3.9  STATUS                 STARTED Testing function: get.comparison.grouplabels
2023-02-28 13:40:42     3.9  STATUS                 COMPLETED Testing function: get.comparison.grouplabels
2023-02-28 13:40:42     3.9  STATUS                 STARTED Testing function: get.site.test.method
2023-02-28 13:40:42     3.9  STATUS                 COMPLETED Testing function: get.site.test.method
2023-02-28 13:40:42     3.9  STATUS                 STARTED Testing function: get.table
2023-02-28 13:40:42     3.9  STATUS                 COMPLETED Testing function: get.table
2023-02-28 13:40:42     3.9  STATUS                 STARTED Testing function: addDiffMethTable
2023-02-28 13:40:42     3.9    INFO                     Conducting differential analysis using limma
2023-02-28 13:40:42     3.9  STATUS                 COMPLETED Testing function: addDiffMethTable
2023-02-28 13:40:42     3.9  STATUS                 STARTED Testing functions: join.diffMeth, is.valid
2023-02-28 13:40:43     3.9  STATUS                 COMPLETED Testing functions: join.diffMeth, is.valid
2023-02-28 13:40:43     3.9  STATUS                 STARTED Destructor
2023-02-28 13:40:43     3.9  STATUS                     STARTED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:40:43     3.9  STATUS                     COMPLETED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:40:43     3.9  STATUS                     STARTED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:40:43     3.9  STATUS                     COMPLETED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:40:43     3.9  STATUS                     STARTED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:40:43     3.9  STATUS                     COMPLETED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:40:43     3.9  STATUS                     STARTED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:40:43     3.9  STATUS                     COMPLETED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:40:43     3.9  STATUS                 COMPLETED Destructor
2023-02-28 13:40:43     3.9  STATUS             COMPLETED Testing class: RnBDiffMeth
2023-02-28 13:40:43     3.9  STATUS         COMPLETED Testing function: apply.iEVORA
2023-02-28 13:40:43     3.9  STATUS         STARTED Retrieving comparison info
2023-02-28 13:40:43     3.9  STATUS         COMPLETED Retrieving comparison info
2023-02-28 13:40:43     3.9  STATUS         STARTED Computing differential methylation tables
2023-02-28 13:40:43     3.9  STATUS             STARTED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:43     3.9  STATUS                 STARTED Computing Differential Methylation Table
2023-02-28 13:40:43     3.9    INFO                     Conducting differential analysis using limma
2023-02-28 13:40:43     3.9  STATUS                 COMPLETED Computing Differential Methylation Table
2023-02-28 13:40:43     3.9  STATUS                 STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:43     3.9  STATUS                     Computed table for tiling
2023-02-28 13:40:45     3.9  STATUS                     Computed table for genes
2023-02-28 13:40:45     3.9  STATUS                     Computed table for promoters
2023-02-28 13:40:45     3.9  STATUS                     Computed table for cpgislands
2023-02-28 13:40:45     3.9  STATUS                 COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:45     3.9  STATUS             COMPLETED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:45     3.9  STATUS             STARTED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:40:45     3.9  STATUS                 STARTED Computing Differential Methylation Table
2023-02-28 13:40:45     3.9    INFO                     Conducting differential analysis using limma
2023-02-28 13:40:45     3.9  STATUS                 COMPLETED Computing Differential Methylation Table
2023-02-28 13:40:45     3.9  STATUS                 STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:46     3.9  STATUS                     Computed table for tiling
2023-02-28 13:40:46     3.9  STATUS                     Computed table for genes
2023-02-28 13:40:47     3.9  STATUS                     Computed table for promoters
2023-02-28 13:40:47     3.9  STATUS                     Computed table for cpgislands
2023-02-28 13:40:47     3.9  STATUS                 COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:47     3.9  STATUS             COMPLETED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:40:47     3.9  STATUS         COMPLETED Computing differential methylation tables
2023-02-28 13:40:47     3.9  STATUS         STARTED Retrieving comparison info
2023-02-28 13:40:47     3.9  STATUS         COMPLETED Retrieving comparison info
2023-02-28 13:40:47     3.9  STATUS         STARTED Computing differential methylation tables
2023-02-28 13:40:47     3.9  STATUS             STARTED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:47     3.9  STATUS                 STARTED Computing Differential Methylation Table
2023-02-28 13:40:47     3.9    INFO                     Conducting differential analysis using limma
2023-02-28 13:40:47     3.9  STATUS                 COMPLETED Computing Differential Methylation Table
2023-02-28 13:40:47     3.9  STATUS                 STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:47     3.9  STATUS                     Computed table for genes
2023-02-28 13:40:48     3.9  STATUS                     Computed table for tiling
2023-02-28 13:40:48     3.9  STATUS                 COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:48     3.9  STATUS             COMPLETED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:48     3.9  STATUS         COMPLETED Computing differential methylation tables
2023-02-28 13:40:48     3.9  STATUS         STARTED Retrieving comparison info
2023-02-28 13:40:48     3.9  STATUS         COMPLETED Retrieving comparison info
2023-02-28 13:40:48     3.9  STATUS         STARTED Computing differential methylation tables
2023-02-28 13:40:48     3.9  STATUS             STARTED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:48     3.9  STATUS                 STARTED Computing Differential Methylation Table
2023-02-28 13:40:48     3.9    INFO                     Conducting differential analysis using limma
2023-02-28 13:40:48     3.9  STATUS                 COMPLETED Computing Differential Methylation Table
2023-02-28 13:40:48     3.9  STATUS                 STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:48     3.9  STATUS                     Computed table for genes
2023-02-28 13:40:49     3.9  STATUS                     Computed table for tiling
2023-02-28 13:40:49     3.9  STATUS                 COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:49     3.9  STATUS             COMPLETED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:49     3.9  STATUS         COMPLETED Computing differential methylation tables
2023-02-28 13:40:49     3.9  STATUS         STARTED Retrieving comparison info
2023-02-28 13:40:49     3.9  STATUS         COMPLETED Retrieving comparison info
2023-02-28 13:40:49     3.9  STATUS         STARTED Computing differential methylation tables
2023-02-28 13:40:49     3.9  STATUS             STARTED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:49     3.9  STATUS                 STARTED Computing Differential Methylation Table
2023-02-28 13:40:49     3.9    INFO                     Conducting differential analysis using limma
2023-02-28 13:40:49     3.9  STATUS                 COMPLETED Computing Differential Methylation Table
2023-02-28 13:40:49     3.9  STATUS                 STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:50     3.9  STATUS                     Computed table for promoters
2023-02-28 13:40:50     3.9  STATUS                 COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:50     3.9  STATUS             COMPLETED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:50     3.9  STATUS             STARTED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:40:50     3.9  STATUS                 STARTED Computing Differential Methylation Table
2023-02-28 13:40:50     3.9    INFO                     Conducting differential analysis using limma
2023-02-28 13:40:50     3.9  STATUS                 COMPLETED Computing Differential Methylation Table
2023-02-28 13:40:50     3.9  STATUS                 STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:51     3.9  STATUS                     Computed table for promoters
2023-02-28 13:40:51     3.9  STATUS                 COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:51     3.9  STATUS             COMPLETED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:40:51     3.9  STATUS         COMPLETED Computing differential methylation tables
2023-02-28 13:40:51     3.9  STATUS         STARTED Retrieving comparison info
2023-02-28 13:40:51     3.9  STATUS         COMPLETED Retrieving comparison info
2023-02-28 13:40:51     3.9  STATUS         STARTED Computing differential methylation tables
2023-02-28 13:40:51     3.9  STATUS             STARTED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:40:51     3.9  STATUS                 STARTED Computing Differential Methylation Table
2023-02-28 13:40:51     3.9    INFO                     Conducting differential analysis using limma
2023-02-28 13:40:51     3.9  STATUS                 COMPLETED Computing Differential Methylation Table
2023-02-28 13:40:51     3.9  STATUS                 STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:51     3.9  STATUS                     Computed table for genes
2023-02-28 13:40:51     3.9  STATUS                     Computed table for tiling
2023-02-28 13:40:51     3.9  STATUS                 COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:51     3.9  STATUS             COMPLETED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:40:51     3.9  STATUS         COMPLETED Computing differential methylation tables
2023-02-28 13:40:51     3.9  STATUS         STARTED Testing function: get.region.types
2023-02-28 13:40:51     3.9  STATUS         COMPLETED Testing function: get.region.types
2023-02-28 13:40:51     3.9  STATUS         STARTED Testing function: get.comparisons
2023-02-28 13:40:51     3.9  STATUS         COMPLETED Testing function: get.comparisons
2023-02-28 13:40:51     3.9  STATUS         STARTED Testing function: get.comparison.grouplabels
2023-02-28 13:40:51     3.9  STATUS         COMPLETED Testing function: get.comparison.grouplabels
2023-02-28 13:40:51     3.9  STATUS         STARTED Testing function: get.site.test.method
2023-02-28 13:40:51     3.9  STATUS         COMPLETED Testing function: get.site.test.method
2023-02-28 13:40:51     3.9  STATUS         STARTED Testing function: get.table
2023-02-28 13:40:51     3.9  STATUS         COMPLETED Testing function: get.table
2023-02-28 13:40:51     3.9  STATUS         STARTED Testing function: addDiffMethTable
2023-02-28 13:40:52     3.9    INFO             Conducting differential analysis using limma
2023-02-28 13:40:52     3.9  STATUS         COMPLETED Testing function: addDiffMethTable
2023-02-28 13:40:52     3.9  STATUS         STARTED Testing functions: join.diffMeth, is.valid
2023-02-28 13:40:52     3.9  STATUS         COMPLETED Testing functions: join.diffMeth, is.valid
2023-02-28 13:40:52     3.9  STATUS         STARTED Destructor
2023-02-28 13:40:52     3.9  STATUS             STARTED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:40:52     3.9  STATUS             COMPLETED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:40:52     3.9  STATUS             STARTED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:40:52     3.9  STATUS             COMPLETED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:40:52     3.9  STATUS             STARTED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:40:52     3.9  STATUS             COMPLETED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:40:52     3.9  STATUS             STARTED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:40:52     3.9  STATUS             COMPLETED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:40:52     3.9  STATUS         COMPLETED Destructor
2023-02-28 13:40:52     3.9    INFO         Conducting differential analysis using limma
2023-02-28 13:40:52     3.9    INFO         Conducting differential analysis using limma
2023-02-28 13:40:52     3.9  STATUS         STARTED diffVar method
coef not specified. Using all columns of design matrix.
2023-02-28 13:40:52     3.9  STATUS         COMPLETED diffVar method
2023-02-28 13:40:52     3.9  STATUS         STARTED diffVar method
coef not specified. Using all columns of design matrix.
2023-02-28 13:40:52     3.9 WARNING             Could not compute p-values with diffVar, returning NA
2023-02-28 13:40:52     3.9    INFO             diffVar from missMethyl package not properly running
2023-02-28 13:40:52     3.9  STATUS             STARTED diffVar method
coef not specified. Using all columns of design matrix.
2023-02-28 13:40:53     3.9  STATUS             COMPLETED diffVar method
2023-02-28 13:40:53     3.9  STATUS STARTED Unit testing: differential
2023-02-28 13:40:53     3.9  STATUS     STARTED Testing function: rowWelchP
2023-02-28 13:40:53     3.9  STATUS     COMPLETED Testing function: rowWelchP
2023-02-28 13:40:53     3.9  STATUS     STARTED Testing function: limmaP
2023-02-28 13:40:53     3.9  STATUS     COMPLETED Testing function: limmaP
2023-02-28 13:40:53     3.9  STATUS     STARTED Testing function: computeDiffTab.extended.site
2023-02-28 13:40:53     3.9    INFO         Conducting differential analysis using limma
2023-02-28 13:40:53     3.9  STATUS     COMPLETED Testing function: computeDiffTab.extended.site
2023-02-28 13:40:53     3.9  STATUS     STARTED Testing function: computeDiffTab.default.region
2023-02-28 13:40:53     3.9    INFO         Conducting differential analysis using limma
2023-02-28 13:40:53     3.9  STATUS     COMPLETED Testing function: computeDiffTab.default.region
2023-02-28 13:40:53     3.9  STATUS     STARTED Testing function: combineTestPvalsMeth
2023-02-28 13:40:53     3.9  STATUS     COMPLETED Testing function: combineTestPvalsMeth
2023-02-28 13:40:53     3.9  STATUS     STARTED Testing function: get.adjustment.variables
2023-02-28 13:40:53     3.9  STATUS     COMPLETED Testing function: get.adjustment.variables
2023-02-28 13:40:53     3.9  STATUS     STARTED Testing function: get.comparison.info
2023-02-28 13:40:53     3.9  STATUS     COMPLETED Testing function: get.comparison.info
2023-02-28 13:40:53     3.9  STATUS     STARTED Testing function: rnb.execute.computeDiffMeth
2023-02-28 13:40:53     3.9  STATUS         STARTED Retrieving comparison info
2023-02-28 13:40:53     3.9  STATUS         COMPLETED Retrieving comparison info
2023-02-28 13:40:53     3.9  STATUS         STARTED Computing differential methylation tables
2023-02-28 13:40:53     3.9  STATUS             STARTED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:53     3.9  STATUS                 STARTED Computing Differential Methylation Table
2023-02-28 13:40:53     3.9    INFO                     Conducting differential analysis using limma
2023-02-28 13:40:53     4.0  STATUS                 COMPLETED Computing Differential Methylation Table
2023-02-28 13:40:53     4.0  STATUS                 STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:54     4.0  STATUS                     Computed table for tiling
2023-02-28 13:40:54     4.0  STATUS                     Computed table for genes
2023-02-28 13:40:54     4.0  STATUS                     Computed table for promoters
2023-02-28 13:40:55     4.0  STATUS                     Computed table for cpgislands
2023-02-28 13:40:55     4.0  STATUS                 COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:55     4.0  STATUS             COMPLETED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:55     4.0  STATUS             STARTED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:40:55     4.0  STATUS                 STARTED Computing Differential Methylation Table
2023-02-28 13:40:55     4.0    INFO                     Conducting differential analysis using limma
2023-02-28 13:40:55     4.0  STATUS                 COMPLETED Computing Differential Methylation Table
2023-02-28 13:40:55     4.0  STATUS                 STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:57     4.0  STATUS                     Computed table for tiling
2023-02-28 13:40:57     4.0  STATUS                     Computed table for genes
2023-02-28 13:40:57     4.0  STATUS                     Computed table for promoters
2023-02-28 13:40:57     4.0  STATUS                     Computed table for cpgislands
2023-02-28 13:40:57     4.0  STATUS                 COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:40:57     4.0  STATUS             COMPLETED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:40:57     4.0  STATUS         COMPLETED Computing differential methylation tables
2023-02-28 13:40:57     4.0  STATUS     COMPLETED Testing function: rnb.execute.computeDiffMeth
2023-02-28 13:40:57     4.0  STATUS     STARTED Testing function: diffVar
2023-02-28 13:40:57     4.0  STATUS         STARTED diffVar method
coef not specified. Using all columns of design matrix.
2023-02-28 13:40:58     4.0  STATUS         COMPLETED diffVar method
2023-02-28 13:40:58     4.0  STATUS         STARTED diffVar method
coef not specified. Using all columns of design matrix.
2023-02-28 13:40:58     4.0 WARNING             Could not compute p-values with diffVar, returning NA
2023-02-28 13:40:58     4.0    INFO             diffVar from missMethyl package not properly running
2023-02-28 13:40:58     4.0  STATUS         COMPLETED diffVar method
2023-02-28 13:40:58     4.0  STATUS         STARTED Testing function: apply.iEVORA
2023-02-28 13:40:58     4.0  STATUS             STARTED iEVORA method
2023-02-28 13:40:59     4.0    INFO                 No DVCs detected. All p-values set to 1.
2023-02-28 13:40:59     4.0  STATUS             COMPLETED iEVORA method
2023-02-28 13:40:59     4.0  STATUS             STARTED Testing function: rnb.execute.diffVar
2023-02-28 13:40:59     4.0  STATUS                 STARTED Differential Variability
2023-02-28 13:40:59     4.0  STATUS                     STARTED Retrieving comparison info
2023-02-28 13:40:59     4.0  STATUS                     COMPLETED Retrieving comparison info
2023-02-28 13:40:59     4.0  STATUS                     STARTED Imputation procedure knn 
2023-02-28 13:40:59     4.0  STATUS                     COMPLETED Imputation procedure knn 
2023-02-28 13:40:59     4.0  STATUS                     STARTED Comparing  hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:40:59     4.0    INFO                         Conducting differential variability using diffVar
2023-02-28 13:40:59     4.0  STATUS                         STARTED diffVar method
coef not specified. Using all columns of design matrix.
2023-02-28 13:40:59     4.0  STATUS                         COMPLETED diffVar method
2023-02-28 13:40:59     4.0  STATUS                         STARTED Computing Differential Variability Tables (Region Level)
2023-02-28 13:40:59     4.0  STATUS                             Computed table for tiling
2023-02-28 13:40:59     4.0  STATUS                             Computed table for genes
2023-02-28 13:41:00     4.0  STATUS                             Computed table for promoters
2023-02-28 13:41:00     4.0  STATUS                             Computed table for cpgislands
2023-02-28 13:41:00     4.0  STATUS                         COMPLETED Computing Differential Variability Tables (Region Level)
2023-02-28 13:41:00     4.0  STATUS                     COMPLETED Comparing  hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:41:00     4.0  STATUS                     STARTED Comparing  KOSR vs. TeSR (based on Treatment)
2023-02-28 13:41:00     4.0    INFO                         Conducting differential variability using diffVar
2023-02-28 13:41:00     4.0  STATUS                         STARTED diffVar method
coef not specified. Using all columns of design matrix.
2023-02-28 13:41:00     4.0  STATUS                         COMPLETED diffVar method
2023-02-28 13:41:00     4.0  STATUS                         STARTED Computing Differential Variability Tables (Region Level)
2023-02-28 13:41:00     4.0  STATUS                             Computed table for tiling
2023-02-28 13:41:02     4.0  STATUS                             Computed table for genes
2023-02-28 13:41:02     4.0  STATUS                             Computed table for promoters
2023-02-28 13:41:02     4.0  STATUS                             Computed table for cpgislands
2023-02-28 13:41:02     4.0  STATUS                         COMPLETED Computing Differential Variability Tables (Region Level)
2023-02-28 13:41:02     4.0  STATUS                     COMPLETED Comparing  KOSR vs. TeSR (based on Treatment)
2023-02-28 13:41:02     4.0  STATUS                 COMPLETED Differential Variability
2023-02-28 13:41:02     4.0  STATUS             COMPLETED Testing function: rnb.execute.diffVar
2023-02-28 13:41:02     4.0  STATUS             STARTED Testing function: rnb.execute.computeDiffMeth and Variability
2023-02-28 13:41:02     4.0  STATUS                 STARTED Retrieving comparison info
2023-02-28 13:41:02     4.0  STATUS                 COMPLETED Retrieving comparison info
2023-02-28 13:41:02     4.0  STATUS                 STARTED Computing differential methylation tables
2023-02-28 13:41:02     4.0  STATUS                     STARTED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:41:02     4.0  STATUS                         STARTED Computing Differential Methylation Table
2023-02-28 13:41:02     4.0    INFO                             Conducting differential analysis using limma
2023-02-28 13:41:02     4.0  STATUS                             STARTED Imputation procedure knn 
2023-02-28 13:41:02     4.0  STATUS                             COMPLETED Imputation procedure knn 
2023-02-28 13:41:02     4.0    INFO                             Conducting differential variability using diffVar
2023-02-28 13:41:02     4.0  STATUS                             STARTED diffVar method
coef not specified. Using all columns of design matrix.
2023-02-28 13:41:02     4.0  STATUS                             COMPLETED diffVar method
2023-02-28 13:41:03     4.0  STATUS                         COMPLETED Computing Differential Methylation Table
2023-02-28 13:41:03     4.0  STATUS                         STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:03     4.0  STATUS                             Computed table for tiling
2023-02-28 13:41:04     4.0  STATUS                             Computed table for genes
2023-02-28 13:41:04     4.0  STATUS                             Computed table for promoters
2023-02-28 13:41:05     4.0  STATUS                             Computed table for cpgislands
2023-02-28 13:41:05     4.0  STATUS                         COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:05     4.0  STATUS                     COMPLETED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:41:05     4.0  STATUS                     STARTED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:41:05     4.0  STATUS                         STARTED Computing Differential Methylation Table
2023-02-28 13:41:05     4.0    INFO                             Conducting differential analysis using limma
2023-02-28 13:41:05     4.0  STATUS                             STARTED Imputation procedure knn 
2023-02-28 13:41:05     4.0  STATUS                             COMPLETED Imputation procedure knn 
2023-02-28 13:41:05     4.0    INFO                             Conducting differential variability using diffVar
2023-02-28 13:41:05     4.0  STATUS                             STARTED diffVar method
coef not specified. Using all columns of design matrix.
2023-02-28 13:41:05     4.0  STATUS                             COMPLETED diffVar method
2023-02-28 13:41:05     4.0  STATUS                         COMPLETED Computing Differential Methylation Table
2023-02-28 13:41:05     4.0  STATUS                         STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:06     4.0  STATUS                             Computed table for tiling
2023-02-28 13:41:07     4.0  STATUS                             Computed table for genes
2023-02-28 13:41:08     4.0  STATUS                             Computed table for promoters
2023-02-28 13:41:08     4.0  STATUS                             Computed table for cpgislands
2023-02-28 13:41:08     4.0  STATUS                         COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:08     4.0  STATUS                     COMPLETED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:41:08     4.0  STATUS                 COMPLETED Computing differential methylation tables
2023-02-28 13:41:08     4.0  STATUS             COMPLETED Testing function: rnb.execute.computeDiffMeth and Variability
2023-02-28 13:41:08     4.0  STATUS             STARTED Testing class: RnBDiffMeth
2023-02-28 13:41:08     4.0  STATUS                 STARTED Retrieving comparison info
2023-02-28 13:41:08     4.0  STATUS                 COMPLETED Retrieving comparison info
2023-02-28 13:41:08     4.0  STATUS                 STARTED Computing differential methylation tables
2023-02-28 13:41:08     4.0  STATUS                     STARTED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:41:08     4.0  STATUS                         STARTED Computing Differential Methylation Table
2023-02-28 13:41:08     4.0    INFO                             Conducting differential analysis using limma
2023-02-28 13:41:08     4.0  STATUS                         COMPLETED Computing Differential Methylation Table
2023-02-28 13:41:08     4.0  STATUS                         STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:09     4.0  STATUS                             Computed table for tiling
2023-02-28 13:41:09     4.0  STATUS                             Computed table for genes
2023-02-28 13:41:10     4.0  STATUS                             Computed table for promoters
2023-02-28 13:41:10     4.0  STATUS                             Computed table for cpgislands
2023-02-28 13:41:10     4.0  STATUS                         COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:10     4.0  STATUS                     COMPLETED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:41:10     4.0  STATUS                     STARTED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:41:10     4.0  STATUS                         STARTED Computing Differential Methylation Table
2023-02-28 13:41:10     4.0    INFO                             Conducting differential analysis using limma
2023-02-28 13:41:10     4.0  STATUS                         COMPLETED Computing Differential Methylation Table
2023-02-28 13:41:10     4.0  STATUS                         STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:10     4.0  STATUS                             Computed table for tiling
2023-02-28 13:41:11     4.0  STATUS                             Computed table for genes
2023-02-28 13:41:11     4.0  STATUS                             Computed table for promoters
2023-02-28 13:41:11     4.0  STATUS                             Computed table for cpgislands
2023-02-28 13:41:11     4.0  STATUS                         COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:11     4.0  STATUS                     COMPLETED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:41:11     4.0  STATUS                 COMPLETED Computing differential methylation tables
2023-02-28 13:41:11     4.0  STATUS                 STARTED Retrieving comparison info
2023-02-28 13:41:11     4.0  STATUS                 COMPLETED Retrieving comparison info
2023-02-28 13:41:11     4.0  STATUS                 STARTED Computing differential methylation tables
2023-02-28 13:41:11     4.0  STATUS                     STARTED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:41:11     4.0  STATUS                         STARTED Computing Differential Methylation Table
2023-02-28 13:41:11     4.0    INFO                             Conducting differential analysis using limma
2023-02-28 13:41:13     4.0  STATUS                         COMPLETED Computing Differential Methylation Table
2023-02-28 13:41:13     4.0  STATUS                         STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:13     4.0  STATUS                             Computed table for genes
2023-02-28 13:41:13     4.0  STATUS                             Computed table for tiling
2023-02-28 13:41:13     4.0  STATUS                         COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:13     4.0  STATUS                     COMPLETED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:41:13     4.0  STATUS                 COMPLETED Computing differential methylation tables
2023-02-28 13:41:13     4.0  STATUS                 STARTED Retrieving comparison info
2023-02-28 13:41:13     4.0  STATUS                 COMPLETED Retrieving comparison info
2023-02-28 13:41:13     4.0  STATUS                 STARTED Computing differential methylation tables
2023-02-28 13:41:13     4.0  STATUS                     STARTED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:41:13     4.0  STATUS                         STARTED Computing Differential Methylation Table
2023-02-28 13:41:13     4.0    INFO                             Conducting differential analysis using limma
2023-02-28 13:41:13     4.0  STATUS                         COMPLETED Computing Differential Methylation Table
2023-02-28 13:41:13     4.0  STATUS                         STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:14     4.0  STATUS                             Computed table for genes
2023-02-28 13:41:14     4.0  STATUS                             Computed table for tiling
2023-02-28 13:41:14     4.0  STATUS                         COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:14     4.0  STATUS                     COMPLETED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:41:14     4.0  STATUS                 COMPLETED Computing differential methylation tables
2023-02-28 13:41:14     4.0  STATUS                 STARTED Retrieving comparison info
2023-02-28 13:41:14     4.0  STATUS                 COMPLETED Retrieving comparison info
2023-02-28 13:41:14     4.0  STATUS                 STARTED Computing differential methylation tables
2023-02-28 13:41:14     4.0  STATUS                     STARTED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:41:14     4.0  STATUS                         STARTED Computing Differential Methylation Table
2023-02-28 13:41:14     4.0    INFO                             Conducting differential analysis using limma
2023-02-28 13:41:14     4.0  STATUS                         COMPLETED Computing Differential Methylation Table
2023-02-28 13:41:14     4.0  STATUS                         STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:15     4.0  STATUS                             Computed table for promoters
2023-02-28 13:41:15     4.0  STATUS                         COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:15     4.0  STATUS                     COMPLETED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:41:15     4.0  STATUS                     STARTED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:41:15     4.0  STATUS                         STARTED Computing Differential Methylation Table
2023-02-28 13:41:15     4.0    INFO                             Conducting differential analysis using limma
2023-02-28 13:41:15     4.0  STATUS                         COMPLETED Computing Differential Methylation Table
2023-02-28 13:41:15     4.0  STATUS                         STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:15     4.0  STATUS                             Computed table for promoters
2023-02-28 13:41:15     4.0  STATUS                         COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:15     4.0  STATUS                     COMPLETED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:41:15     4.0  STATUS                 COMPLETED Computing differential methylation tables
2023-02-28 13:41:15     4.0  STATUS                 STARTED Retrieving comparison info
2023-02-28 13:41:15     4.0  STATUS                 COMPLETED Retrieving comparison info
2023-02-28 13:41:15     4.0  STATUS                 STARTED Computing differential methylation tables
2023-02-28 13:41:15     4.0  STATUS                     STARTED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:41:15     4.0  STATUS                         STARTED Computing Differential Methylation Table
2023-02-28 13:41:15     4.0    INFO                             Conducting differential analysis using limma
2023-02-28 13:41:15     4.0  STATUS                         COMPLETED Computing Differential Methylation Table
2023-02-28 13:41:15     4.0  STATUS                         STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:15     4.0  STATUS                             Computed table for genes
2023-02-28 13:41:16     4.0  STATUS                             Computed table for tiling
2023-02-28 13:41:16     4.0  STATUS                         COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:16     4.0  STATUS                     COMPLETED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:41:16     4.0  STATUS                 COMPLETED Computing differential methylation tables
2023-02-28 13:41:16     4.0  STATUS                 STARTED Testing function: get.region.types
2023-02-28 13:41:16     4.0  STATUS                 COMPLETED Testing function: get.region.types
2023-02-28 13:41:16     4.0  STATUS                 STARTED Testing function: get.comparisons
2023-02-28 13:41:16     4.0  STATUS                 COMPLETED Testing function: get.comparisons
2023-02-28 13:41:16     4.0  STATUS                 STARTED Testing function: get.comparison.grouplabels
2023-02-28 13:41:16     4.0  STATUS                 COMPLETED Testing function: get.comparison.grouplabels
2023-02-28 13:41:16     4.0  STATUS                 STARTED Testing function: get.site.test.method
2023-02-28 13:41:16     4.0  STATUS                 COMPLETED Testing function: get.site.test.method
2023-02-28 13:41:16     4.0  STATUS                 STARTED Testing function: get.table
2023-02-28 13:41:16     4.0  STATUS                 COMPLETED Testing function: get.table
2023-02-28 13:41:16     4.0  STATUS                 STARTED Testing function: addDiffMethTable
2023-02-28 13:41:16     4.0    INFO                     Conducting differential analysis using limma
2023-02-28 13:41:16     4.0  STATUS                 COMPLETED Testing function: addDiffMethTable
2023-02-28 13:41:16     4.0  STATUS                 STARTED Testing functions: join.diffMeth, is.valid
2023-02-28 13:41:16     4.0  STATUS                 COMPLETED Testing functions: join.diffMeth, is.valid
2023-02-28 13:41:16     4.0  STATUS                 STARTED Destructor
2023-02-28 13:41:16     4.0  STATUS                     STARTED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:41:16     4.0  STATUS                     COMPLETED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:41:16     4.0  STATUS                     STARTED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:41:16     4.0  STATUS                     COMPLETED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:41:16     4.0  STATUS                     STARTED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:41:16     4.0  STATUS                     COMPLETED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:41:16     4.0  STATUS                     STARTED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:41:16     4.0  STATUS                     COMPLETED Deleting RnBDiffMeth disk dump files from disk
2023-02-28 13:41:16     4.0  STATUS                 COMPLETED Destructor
2023-02-28 13:41:16     4.0  STATUS             COMPLETED Testing class: RnBDiffMeth
2023-02-28 13:41:16     4.0  STATUS         COMPLETED Testing function: apply.iEVORA
2023-02-28 13:41:17     4.0  STATUS         STARTED iEVORA method
2023-02-28 13:41:19     4.0    INFO             No DVCs detected. All p-values set to 1.
Coefficients not estimable: x2x2 
2023-02-28 13:41:19     4.0  STATUS             STARTED Retrieving comparison info
2023-02-28 13:41:19     4.0  STATUS             COMPLETED Retrieving comparison info
2023-02-28 13:41:19     4.0  STATUS             STARTED Computing differential methylation tables
2023-02-28 13:41:19     4.0  STATUS                 STARTED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:41:19     4.0  STATUS                     STARTED Computing Differential Methylation Table
2023-02-28 13:41:19     4.0    INFO                         Conducting differential analysis using limma
2023-02-28 13:41:19     4.0  STATUS                     COMPLETED Computing Differential Methylation Table
2023-02-28 13:41:19     4.0  STATUS                     STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:20     4.0  STATUS                         Computed table for tiling
2023-02-28 13:41:20     4.0  STATUS                         Computed table for genes
2023-02-28 13:41:20     4.0  STATUS                         Computed table for promoters
2023-02-28 13:41:20     4.0  STATUS                         Computed table for cpgislands
2023-02-28 13:41:20     4.0  STATUS                     COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:20     4.0  STATUS                 COMPLETED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:41:20     4.0  STATUS                 STARTED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:41:20     4.0  STATUS                     STARTED Computing Differential Methylation Table
2023-02-28 13:41:20     4.0    INFO                         Conducting differential analysis using limma
2023-02-28 13:41:20     4.0  STATUS                     COMPLETED Computing Differential Methylation Table
2023-02-28 13:41:20     4.0  STATUS                     STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:21     4.0  STATUS                         Computed table for tiling
2023-02-28 13:41:21     4.0  STATUS                         Computed table for genes
2023-02-28 13:41:22     4.0  STATUS                         Computed table for promoters
2023-02-28 13:41:22     4.0  STATUS                         Computed table for cpgislands
2023-02-28 13:41:22     4.0  STATUS                     COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:22     4.0  STATUS                 COMPLETED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:41:22     4.0  STATUS             COMPLETED Computing differential methylation tables
2023-02-28 13:41:22     4.0  STATUS             STARTED Retrieving comparison info
2023-02-28 13:41:22     4.0  STATUS             COMPLETED Retrieving comparison info
2023-02-28 13:41:22     4.0  STATUS             STARTED Computing differential methylation tables
2023-02-28 13:41:22     4.0  STATUS                 STARTED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:41:22     4.0  STATUS                     STARTED Computing Differential Methylation Table
2023-02-28 13:41:22     4.0    INFO                         Conducting differential analysis using limma
2023-02-28 13:41:22     4.0  STATUS                         STARTED Imputation procedure knn 
2023-02-28 13:41:22     4.0  STATUS                         COMPLETED Imputation procedure knn 
2023-02-28 13:41:22     4.0    INFO                         Conducting differential variability using diffVar
2023-02-28 13:41:22     4.0  STATUS                         STARTED diffVar method
coef not specified. Using all columns of design matrix.
2023-02-28 13:41:22     4.0  STATUS                         COMPLETED diffVar method
2023-02-28 13:41:22     4.0  STATUS                     COMPLETED Computing Differential Methylation Table
2023-02-28 13:41:22     4.0  STATUS                     STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:24     4.0  STATUS                         Computed table for tiling
2023-02-28 13:41:25     4.0  STATUS                         Computed table for genes
2023-02-28 13:41:25     4.0  STATUS                         Computed table for promoters
2023-02-28 13:41:25     4.0  STATUS                         Computed table for cpgislands
2023-02-28 13:41:25     4.0  STATUS                     COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:25     4.0  STATUS                 COMPLETED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:41:25     4.0  STATUS                 STARTED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:41:25     4.0  STATUS                     STARTED Computing Differential Methylation Table
2023-02-28 13:41:25     4.0    INFO                         Conducting differential analysis using limma
2023-02-28 13:41:25     4.0  STATUS                         STARTED Imputation procedure knn 
2023-02-28 13:41:25     4.0  STATUS                         COMPLETED Imputation procedure knn 
2023-02-28 13:41:25     4.0    INFO                         Conducting differential variability using diffVar
2023-02-28 13:41:25     4.0  STATUS                         STARTED diffVar method
coef not specified. Using all columns of design matrix.
2023-02-28 13:41:25     4.0  STATUS                         COMPLETED diffVar method
2023-02-28 13:41:25     4.0  STATUS                     COMPLETED Computing Differential Methylation Table
2023-02-28 13:41:26     4.0  STATUS                     STARTED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:26     4.0  STATUS                         Computed table for tiling
2023-02-28 13:41:27     4.0  STATUS                         Computed table for genes
2023-02-28 13:41:27     4.0  STATUS                         Computed table for promoters
2023-02-28 13:41:28     4.0  STATUS                         Computed table for cpgislands
2023-02-28 13:41:28     4.0  STATUS                     COMPLETED Computing Differential Methylation Tables (Region Level)
2023-02-28 13:41:28     4.0  STATUS                 COMPLETED Comparing: KOSR vs. TeSR (based on Treatment)
2023-02-28 13:41:28     4.0  STATUS             COMPLETED Computing differential methylation tables
2023-02-28 13:41:28     4.0  STATUS             STARTED Differential Variability
2023-02-28 13:41:28     4.0  STATUS                 STARTED Retrieving comparison info
2023-02-28 13:41:28     4.0  STATUS                 COMPLETED Retrieving comparison info
2023-02-28 13:41:28     4.0  STATUS                 STARTED Imputation procedure knn 
2023-02-28 13:41:28     4.0  STATUS                 COMPLETED Imputation procedure knn 
2023-02-28 13:41:28     4.0  STATUS                 STARTED Comparing  hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:41:28     4.0    INFO                     Conducting differential variability using diffVar
2023-02-28 13:41:28     4.0  STATUS                     STARTED diffVar method
coef not specified. Using all columns of design matrix.
2023-02-28 13:41:28     4.0  STATUS                     COMPLETED diffVar method
2023-02-28 13:41:29     4.0  STATUS                     STARTED Computing Differential Variability Tables (Region Level)
2023-02-28 13:41:29     4.0  STATUS                         Computed table for tiling
2023-02-28 13:41:30     4.0  STATUS                         Computed table for genes
2023-02-28 13:41:30     4.0  STATUS                         Computed table for promoters
2023-02-28 13:41:30     4.0  STATUS                         Computed table for cpgislands
2023-02-28 13:41:30     4.0  STATUS                     COMPLETED Computing Differential Variability Tables (Region Level)
2023-02-28 13:41:30     4.0  STATUS                 COMPLETED Comparing  hESC vs. hiPSC (based on Sample_Group)
2023-02-28 13:41:30     4.0  STATUS                 STARTED Comparing  KOSR vs. TeSR (based on Treatment)
2023-02-28 13:41:30     4.0    INFO                     Conducting differential variability using diffVar
2023-02-28 13:41:30     4.0  STATUS                     STARTED diffVar method
coef not specified. Using all columns of design matrix.
2023-02-28 13:41:30     4.0  STATUS                     COMPLETED diffVar method
2023-02-28 13:41:30     4.0  STATUS                     STARTED Computing Differential Variability Tables (Region Level)
2023-02-28 13:41:31     4.0  STATUS                         Computed table for tiling
2023-02-28 13:41:31     4.0  STATUS                         Computed table for genes
2023-02-28 13:41:31     4.0  STATUS                         Computed table for promoters
2023-02-28 13:41:31     4.0  STATUS                         Computed table for cpgislands
2023-02-28 13:41:31     4.0  STATUS                     COMPLETED Computing Differential Variability Tables (Region Level)
2023-02-28 13:41:31     4.0  STATUS                 COMPLETED Comparing  KOSR vs. TeSR (based on Treatment)
2023-02-28 13:41:31     4.0  STATUS             COMPLETED Differential Variability
2023-02-28 13:41:31     4.0  STATUS STARTED Analysis
2023-02-28 13:41:31     4.0    INFO     Loaded information from data.RData
2023-02-28 13:41:31     4.0  STATUS     STARTED Processing Detection P-values
2023-02-28 13:41:31     4.0    INFO         Removed 3979 probes that overlap with SNPs
2023-02-28 13:41:31     4.0    INFO         Completed Greedycut on 510 samples
2023-02-28 13:41:31     4.0  STATUS     COMPLETED Processing Detection P-values
2023-02-28 13:41:31     4.0 WARNING     File not found: data2.RData
2023-02-28 13:41:31     4.0  STATUS COMPLETED Analysis

2023-02-28 13:41:31     4.0  STATUS STARTED Testing imputation
2023-02-28 13:41:31     4.0  STATUS     STARTED Imputation procedure knn 
2023-02-28 13:41:32     4.0  STATUS     COMPLETED Imputation procedure knn 
2023-02-28 13:41:32     4.0  STATUS     STARTED Imputation procedure mean.samples 
2023-02-28 13:41:34     4.0  STATUS     COMPLETED Imputation procedure mean.samples 
2023-02-28 13:41:34     4.0  STATUS     STARTED Imputation procedure mean.cpgs 
2023-02-28 13:41:35     4.0  STATUS     COMPLETED Imputation procedure mean.cpgs 
2023-02-28 13:41:35     4.0  STATUS     STARTED Imputation procedure random 
2023-02-28 13:41:35     4.0  STATUS     COMPLETED Imputation procedure random 
2023-02-28 13:41:35     4.0  STATUS COMPLETED Testing imputation

2023-02-28 13:41:35     4.0  STATUS STARTED Testing age prediction
2023-02-28 13:41:36     4.0  STATUS     STARTED Performing Age Prediction
2023-02-28 13:41:37     4.1  STATUS     COMPLETED Performing Age Prediction
2023-02-28 13:41:37     4.1  STATUS COMPLETED Testing age prediction

2023-02-28 13:41:37     4.1  STATUS STARTED Performing Age Prediction
2023-02-28 13:41:38     4.2  STATUS COMPLETED Performing Age Prediction

2023-02-28 13:41:38     4.2  STATUS STARTED Imputation procedure knn 
2023-02-28 13:41:39     4.2  STATUS COMPLETED Imputation procedure knn 

2023-02-28 13:41:39     4.2  STATUS STARTED Imputation procedure mean.samples 
2023-02-28 13:41:39     4.3  STATUS COMPLETED Imputation procedure mean.samples 

2023-02-28 13:41:39     4.3  STATUS STARTED Imputation procedure mean.cpgs 
2023-02-28 13:41:41     4.3  STATUS COMPLETED Imputation procedure mean.cpgs 

2023-02-28 13:41:41     4.3  STATUS STARTED Imputation procedure random 
2023-02-28 13:41:42     4.3  STATUS COMPLETED Imputation procedure random 

2023-02-28 13:41:42     4.3  STATUS STARTED Testing imputation
2023-02-28 13:41:42     4.3  STATUS     STARTED Imputation procedure knn 
2023-02-28 13:41:43     4.3  STATUS     COMPLETED Imputation procedure knn 
2023-02-28 13:41:43     4.3  STATUS     STARTED Imputation procedure mean.samples 
2023-02-28 13:41:43     4.3  STATUS     COMPLETED Imputation procedure mean.samples 
2023-02-28 13:41:43     4.3  STATUS     STARTED Imputation procedure mean.cpgs 
2023-02-28 13:41:44     4.3  STATUS     COMPLETED Imputation procedure mean.cpgs 
2023-02-28 13:41:44     4.3  STATUS     STARTED Imputation procedure random 
2023-02-28 13:41:44     4.3  STATUS     COMPLETED Imputation procedure random 
2023-02-28 13:41:44     4.3  STATUS COMPLETED Testing imputation

2023-02-28 13:41:44     4.3  STATUS STARTED Testing age prediction
2023-02-28 13:41:44     4.3  STATUS     STARTED Performing Age Prediction
2023-02-28 13:41:45     4.3  STATUS     COMPLETED Performing Age Prediction
2023-02-28 13:41:46     4.3  STATUS COMPLETED Testing age prediction



RUNIT TEST PROTOCOL -- Tue Feb 28 13:41:53 2023 
*********************************************** 
Number of test functions: 24 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
RnBeads RUnit Tests - 24 test functions, 0 errors, 0 failures
Number of test functions: 24 
Number of errors: 0 
Number of failures: 0 
There were 13 warnings (use warnings() to see them)
&gt; 
&gt; proc.time()
   user  system elapsed 
197.424   6.712 204.442 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">RnBeads.Rcheck/RnBeads-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>M-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>U-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addDiffMethTable-RnBDiffMeth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addPheno-RnBSet-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addRegionSubsegments</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotation-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>assembly-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>combine.rnb.sets-methods</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>combineTestPvalsMeth</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>computeDiffTab.region</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>computeDiffTab.site</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>covg-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>create.densityScatter</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>create.scatter.dens.points</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>createReport</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>createReportPlot</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>downloadLolaDbs</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dpval-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>exportDMRs2regionFile</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.adjustment.variables</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.comparison.grouplabels-RnBDiffMeth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.comparison.groupsizes-RnBDiffMeth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.comparison.info</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.comparisons-RnBDiffMeth-methods</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.covariates.sva</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.covg.thres-RnBDiffMeth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.files</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.region.types-RnBDiffMeth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.site.test.method-RnBDiffMeth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.table-RnBDiffMeth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getCellTypesFromLolaDb</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getNamesFromLolaDb</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getSubCmdTokens-ClusterArchitectureSGE-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getSubCmdTokens-ClusterArchitectureSLURM-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getTargetFromLolaDb</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>has.covariates.sva</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>hasCovg-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>includes.sites-RnBDiffMeth-methods</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>is.valid-RnBDiffMeth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>join.diffMeth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>limmaP</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loadLolaDbs</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>logger.argument</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>logger.getfiles</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>logger.isinitialized</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>logger.validate.file</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loggerManagement</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loggerMessages</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>lolaBarPlot</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>lolaBoxPlotPerTarget</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>lolaVolcanoPlot</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mergeSamples-methods</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>meth-methods</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mval-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nsites-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>parallel.getNumWorkers</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>parallel.isEnabled</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>parallel.setup</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>parallel.teardown</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>performGOEnrichment.diffVar</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>performGOenrichment.diffMeth.entrez</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>performGoEnrichment.diffMeth</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>performLolaEnrichment.diffMeth</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>performLolaEnrichment.diffVar</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pheno-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qc-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>read.sample.annotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>regionMapping-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>regions-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>reload-RnBDiffMeth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>remove.regions-methods</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>remove.samples-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>remove.sites-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.RnBSet.to.GRangesList</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.RnBSet.to.bed</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.RnBSet.to.bedGraph</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.add.list</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.add.paragraph</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.add.reference</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.add.section</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.annotation.size</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.annotation2data.frame</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.beta2mval</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.batcheffects</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.clustering</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.computeDiffMeth</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.context.removal</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.cross.reactive.removal</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.dreduction</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.export.csv</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.greedycut</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.import</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.na.removal</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.normalization</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.pOOBAH</TD><TD style="text-align: right;">21.198</TD><TD style="text-align: right;"> 0.907</TD><TD style="text-align: right;">24.462</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.sex.prediction</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.sex.removal</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.snp.removal</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.sva</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.tnt</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.variability.removal</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.export.all.annotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.export.annotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.export.to.ewasher</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.export.to.trackhub</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.get.annotation</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.get.assemblies</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.get.chromosomes</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.get.directory</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.get.mapping</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.get.reference</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.get.reliability.matrix</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.infinium.control.targets</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.initialize.reports</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.is.option</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.load.annotation.from.db</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.message.plot</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.mval2beta</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.options</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.options2xml</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.plot.betadistribution.probeCategories</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.plot.betadistribution.sampleGroups</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.plot.control.barplot</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.plot.control.boxplot</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.plot.dreduction</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.plot.locus.profile</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.plot.negative.boxplot</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.plot.region.profile.density</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.plot.region.profiles</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.plot.region.site.density</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.plot.sentrix.distribution</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.plot.snp.barplot</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.plot.snp.boxplot</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.plot.snp.heatmap</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.region.types</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.region.types.for.analysis</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.remove.annotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.run.example</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.runs</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.sample.groups</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.sample.summary.table</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.set.annotation</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.write.table</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.xml2options</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rowOneSampleTP</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rowWelchP</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>run-RnBClusterRun-methods</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>samples-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>save.tables-RnBDiffMeth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>set.covariates.sva</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sites-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>summarize.regions-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>summarized.regions-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
