<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for geneXtendeR on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.17</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><B>[<A href="..#G">G</A>]</B></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-03-27 05:50:34 -0000 (Mon, 27 Mar 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-03-12&nbsp;r83975)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">6083</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for geneXtendeR on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the geneXtendeR package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/geneXtendeR.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>781</B>/2195</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">geneXtendeR</A>&nbsp;1.25.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/geneXtendeR">landing page</A>)</SPAN><BR>Bohdan Khomtchouk<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-03-24&nbsp;01:40:07&nbsp;-0000&nbsp;(Fri,&nbsp;24&nbsp;Mar&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/geneXtendeR</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">f3f488f</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:14:35&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: geneXtendeR</TD></TR>
<TR><TD><B>Version</B>: 1.25.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:geneXtendeR.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings geneXtendeR_1.25.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-03-25 03:00:14 -0000 (Sat, 25 Mar 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-03-25 03:17:13 -0000 (Sat, 25 Mar 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 1019.4 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: geneXtendeR.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 2</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:geneXtendeR.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings geneXtendeR_1.25.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/geneXtendeR.Rcheck’
* using R Under development (unstable) (2023-03-12 r83975)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘geneXtendeR/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘geneXtendeR’ version ‘1.25.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘geneXtendeR’ can be installed ... WARNING
Found the following significant warnings:
  annotate.c:106:5: warning: ignoring return value of ‘fgets’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
  annotate.c:116:5: warning: ignoring return value of ‘fgets’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
  extract_number.c:98:5: warning: ignoring return value of ‘fgets’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
  extract_number.c:108:5: warning: ignoring return value of ‘fgets’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
  extract_peaks.c:107:5: warning: ignoring return value of ‘fgets’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
  extract_peaks.c:117:5: warning: ignoring return value of ‘fgets’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
See ‘/home/biocbuild/bbs-3.17-bioc/meat/geneXtendeR.Rcheck/00install.out’ for details.
* used C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... NOTE
  installed size is 11.5Mb
  sub-directories of 1Mb or more:
    data      5.8Mb
    doc       1.9Mb
    extdata   3.5Mb
* checking package directory ... OK
* checking DESCRIPTION meta-information ... NOTE
Package listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘rtracklayer’
A package should be listed in only one of these fields.
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... NOTE
Namespaces in Imports field not imported from:
  ‘BiocStyle’ ‘SnowballC’ ‘org.Rn.eg.db’
  All declared Imports should be used.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
.geneXtender: no visible binding for global variable ‘type’
.geneXtender: no visible binding for global variable ‘seqid’
.geneXtender: no visible binding for global variable ‘gene_id’
.geneXtender: no visible binding for global variable ‘gene_name’
annotate_n: no visible binding for global variable ‘..I’
annotate_n: no visible binding for global variable ‘seqid’
diffGO: no visible binding for global variable ‘rat’
gene_annotate: no visible global function definition for ‘.’
gene_annotate: no visible binding for global variable
  ‘Distance-of-Gene-to-Nearest-Peak’
gene_annotate: no visible global function definition for ‘sd’
gene_annotate: no visible binding for global variable ‘Chromosome’
gene_annotate: no visible binding for global variable ‘Gene-Start’
gene_annotate: no visible binding for global variable ‘Gene-End’
gene_annotate: no visible binding for global variable ‘Gene-ID’
gene_annotate: no visible binding for global variable ‘Gene-Name’
gene_annotate: no visible binding for global variable
  ‘Peaks-on-Gene-Body’
gene_annotate: no visible binding for global variable
  ‘Number-of-Peaks-Associated-with-Gene’
gene_lookup: no visible binding for global variable ‘gene_name_id’
gene_lookup: no visible binding for global variable ‘gene_id’
gene_lookup : internal_find: no visible binding for global variable
  ‘Chromosome’
gene_lookup : internal_find: no visible binding for global variable
  ‘distance’
gene_lookup : internal_find: no visible global function definition for
  ‘na.omit’
gene_lookup: no visible binding for global variable ‘..I’
gene_lookup: no visible binding for global variable ‘distance’
makeWordCloud: no visible binding for global variable ‘rat’
meanPeakLength: no visible binding for global variable ‘rat’
peaksInput: no visible binding for global variable ‘chr’
peaksInput: no visible global function definition for ‘na.omit’
peaksMerge: no visible binding for global variable ‘chr’
peaksMerge: no visible binding for global variable ‘g’
peaksMerge: no visible global function definition for ‘.’
plotWordFreq : geneXtender: no visible binding for global variable
  ‘type’
plotWordFreq : geneXtender: no visible binding for global variable
  ‘seqid’
plotWordFreq : geneXtender: no visible binding for global variable
  ‘gene_id’
plotWordFreq : geneXtender: no visible binding for global variable
  ‘gene_name’
Undefined global functions or variables:
  . ..I Chromosome Distance-of-Gene-to-Nearest-Peak Gene-End Gene-ID
  Gene-Name Gene-Start Number-of-Peaks-Associated-with-Gene
  Peaks-on-Gene-Body chr distance g gene_id gene_name gene_name_id
  na.omit rat sd seqid type
Consider adding
  importFrom(&quot;stats&quot;, &quot;na.omit&quot;, &quot;sd&quot;)
to your NAMESPACE file.
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking LazyData ... WARNING
  LazyData DB of 5.8 MB without LazyDataCompression set
  See §1.1.6 of &#x27;Writing R Extensions&#x27;
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking installed files from ‘inst/doc’ ... OK
* checking files in ‘vignettes’ ... NOTE
Files named as vignettes but with no recognized vignette engine:
   ‘vignettes/geneXtendeR.Rnw’
(Is a VignetteBuilder field missing?)
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                     user system elapsed
meanPeakLengthPlot 46.028  0.507  58.286
hotspotPlot        25.332  0.672  41.003
annotate           19.656  0.654  31.663
linePlot           19.672  0.396  43.730
cumlinePlot        18.819  0.372  32.069
barChart           17.827  0.384  30.378
diffGO             16.576  0.792  33.170
plotWordFreq       16.952  0.268  28.378
makeWordCloud      16.470  0.220  30.201
gene_annotate      15.160  0.280  28.375
meanPeakLength     15.221  0.204  27.388
makeNetwork        14.926  0.296  27.111
peakLengthBoxplot  14.475  0.208  25.508
distinct           14.059  0.305  27.047
gene_lookup        12.768  0.287  34.307
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking PDF version of manual ... OK
* DONE

Status: 2 WARNINGs, 6 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/geneXtendeR.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">geneXtendeR.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL geneXtendeR
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library’
* installing *source* package ‘geneXtendeR’ ...
** using staged installation
** libs
using C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG   -I/usr/local/include    -fPIC  -g -O2  -c annotate.c -o annotate.o
annotate.c: In function ‘annotate’:
annotate.c:106:5: warning: ignoring return value of ‘fgets’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
  106 |     fgets(varstr, 50000, varfp);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
annotate.c:116:5: warning: ignoring return value of ‘fgets’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
  116 |     fgets(vcfstr, 10000, vcffp);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG   -I/usr/local/include    -fPIC  -g -O2  -c extract_number.c -o extract_number.o
extract_number.c: In function ‘extractnumber’:
extract_number.c:98:5: warning: ignoring return value of ‘fgets’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
   98 |     fgets(varstr, 50000, varfp);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
extract_number.c:108:5: warning: ignoring return value of ‘fgets’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
  108 |     fgets(vcfstr, 10000, vcffp);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG   -I/usr/local/include    -fPIC  -g -O2  -c extract_peaks.c -o extract_peaks.o
extract_peaks.c: In function ‘extractpeaks’:
extract_peaks.c:107:5: warning: ignoring return value of ‘fgets’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
  107 |     fgets(varstr, 50000, varfp);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
extract_peaks.c:117:5: warning: ignoring return value of ‘fgets’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
  117 |     fgets(vcfstr, 10000, vcffp);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
gcc -shared -L/usr/local/lib -o geneXtendeR.so annotate.o extract_number.o extract_peaks.o
installing to /home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/00LOCK-geneXtendeR/00new/geneXtendeR/libs
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (geneXtendeR)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">geneXtendeR.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(geneXtendeR)
Loading required package: rtracklayer
Loading required package: GenomicRanges
Loading required package: stats4
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
Loading required package: GO.db
Loading required package: AnnotationDbi
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


&gt; 
&gt; test_check(&quot;geneXtendeR&quot;)
[ FAIL 0 | WARN 0 | SKIP 0 | PASS 86 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
208.428  12.788 221.116 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">geneXtendeR.Rcheck/geneXtendeR-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>allPeakLengths</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotate</TD><TD style="text-align: right;">19.656</TD><TD style="text-align: right;"> 0.654</TD><TD style="text-align: right;">31.663</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotate_n</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>barChart</TD><TD style="text-align: right;">17.827</TD><TD style="text-align: right;"> 0.384</TD><TD style="text-align: right;">30.378</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cumlinePlot</TD><TD style="text-align: right;">18.819</TD><TD style="text-align: right;"> 0.372</TD><TD style="text-align: right;">32.069</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>diffGO</TD><TD style="text-align: right;">16.576</TD><TD style="text-align: right;"> 0.792</TD><TD style="text-align: right;">33.170</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>distinct</TD><TD style="text-align: right;">14.059</TD><TD style="text-align: right;"> 0.305</TD><TD style="text-align: right;">27.047</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>gene_annotate</TD><TD style="text-align: right;">15.160</TD><TD style="text-align: right;"> 0.280</TD><TD style="text-align: right;">28.375</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>gene_lookup</TD><TD style="text-align: right;">12.768</TD><TD style="text-align: right;"> 0.287</TD><TD style="text-align: right;">34.307</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>hotspotPlot</TD><TD style="text-align: right;">25.332</TD><TD style="text-align: right;"> 0.672</TD><TD style="text-align: right;">41.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>linePlot</TD><TD style="text-align: right;">19.672</TD><TD style="text-align: right;"> 0.396</TD><TD style="text-align: right;">43.730</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeNetwork</TD><TD style="text-align: right;">14.926</TD><TD style="text-align: right;"> 0.296</TD><TD style="text-align: right;">27.111</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeWordCloud</TD><TD style="text-align: right;">16.470</TD><TD style="text-align: right;"> 0.220</TD><TD style="text-align: right;">30.201</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>meanPeakLength</TD><TD style="text-align: right;">15.221</TD><TD style="text-align: right;"> 0.204</TD><TD style="text-align: right;">27.388</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>meanPeakLengthPlot</TD><TD style="text-align: right;">46.028</TD><TD style="text-align: right;"> 0.507</TD><TD style="text-align: right;">58.286</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>peakLengthBoxplot</TD><TD style="text-align: right;">14.475</TD><TD style="text-align: right;"> 0.208</TD><TD style="text-align: right;">25.508</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>peaksInput</TD><TD style="text-align: right;">0.242</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.302</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>peaksMerge</TD><TD style="text-align: right;">0.153</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.156</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotWordFreq</TD><TD style="text-align: right;">16.952</TD><TD style="text-align: right;"> 0.268</TD><TD style="text-align: right;">28.378</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rat</TD><TD style="text-align: right;">3.877</TD><TD style="text-align: right;">0.192</TD><TD style="text-align: right;">4.070</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>samplepeaksinput</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
