<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for seqTools on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><B>[<A href="..#S">S</A>]</B></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-16 06:07:04 -0000 (Tue, 16 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4787</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for seqTools on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the seqTools package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/seqTools.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1866</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">seqTools</A>&nbsp;1.35.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/seqTools">landing page</A>)</SPAN><BR>Wolfgang Kaisers<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-13&nbsp;23:12:22&nbsp;-0000&nbsp;(Sat,&nbsp;13&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/seqTools</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">1840ca6</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:36:30&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: seqTools</TD></TR>
<TR><TD><B>Version</B>: 1.35.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:seqTools.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings seqTools_1.35.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-15 13:58:52 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-15 13:59:22 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 30.0 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: seqTools.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 3</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:seqTools.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings seqTools_1.35.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/seqTools.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘seqTools/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘seqTools’ version ‘1.35.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘seqTools’ can be installed ... OK
* used C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... WARNING
Undocumented code objects:
  ‘kmerSvd’
Undocumented S4 methods:
  generic &#x27;kmerSvd&#x27; and siglist &#x27;Fastqq&#x27;
All user-level objects in a package (including S4 classes and methods)
should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... OK
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... WARNING
Note: information on .o files is not available
File ‘/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/libs/seqTools.so’:
  Found ‘__sprintf_chk’, possibly from ‘sprintf’ (C)

Compiled code should not call entry points which might terminate R nor
write to stdout/stderr instead of to the console, nor use Fortran I/O
nor system RNGs nor [v]sprintf. The detected symbols are linked into
the code but might come from libraries and not actually be called.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
* checking files in ‘vignettes’ ... OK
* checking examples ... WARNING
Found the following significant warnings:
  Warning: working directory was changed to ‘/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata’, resetting
  Warning: working directory was changed to ‘/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata’, resetting
  Warning: working directory was changed to ‘/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata’, resetting
  Warning: working directory was changed to ‘/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata’, resetting
  Warning: working directory was changed to ‘/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata’, resetting
  Warning: working directory was changed to ‘/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata’, resetting
  Warning: working directory was changed to ‘/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata’, resetting
  Warning: working directory was changed to ‘/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata’, resetting
  Warning: working directory was changed to ‘/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata’, resetting
  Warning: working directory was changed to ‘/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata’, resetting
  Warning: working directory was changed to ‘/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata’, resetting
  Warning: working directory was changed to ‘/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata’, resetting
  Warning: working directory was changed to ‘/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata’, resetting
  Warning: working directory was changed to ‘/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata’, resetting
  Warning: working directory was changed to ‘/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata’, resetting
  Warning: working directory was changed to ‘/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata’, resetting
  Warning: working directory was changed to ‘/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata’, resetting
  Warning: working directory was changed to ‘/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata’, resetting
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘test-all.R’
  Running ‘test_seqTools.r’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘seqTools.Rnw’ using ‘UTF-8’... OK
  ‘seqTools_qual_report.Rnw’ using ‘UTF-8’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 3 WARNINGs
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/seqTools.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">seqTools.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD INSTALL seqTools
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0-bin/site-library’
* installing *source* package ‘seqTools’ ...
** using staged installation
** libs
using C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
gcc -I&quot;/home/biocbuild/R/R-4.3.0-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/R/R-4.3.0-bin/site-library/zlibbioc/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c seqTools.c -o seqTools.o
gcc -shared -L/home/biocbuild/R/R-4.3.0-bin/lib/R/lib -L/usr/local/lib -o seqTools.so seqTools.o -lm -lz -L/home/biocbuild/R/R-4.3.0-bin/lib/R/lib -lR
installing to /home/biocbuild/R/R-4.3.0-bin/site-library/00LOCK-seqTools/00new/seqTools/libs
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (seqTools)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">seqTools.Rcheck/tests/test-all.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; 
&gt; 
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; ## Load prerequisites
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; 
&gt; library(seqTools)
Loading required package: zlibbioc
&gt; 
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; ## Initialize example data
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; 
&gt; basedir&lt;-system.file(&quot;extdata&quot;,package=&quot;seqTools&quot;)
&gt; 
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; ## Run tests
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; 
&gt; filename &lt;- &quot;test_seqTools.R&quot;
&gt; basedir &lt;- system.file(&quot;extdata&quot;, package = &quot;seqTools&quot;)
&gt; load(file.path(basedir,&quot;test_res.RData&quot;))
&gt; 
&gt; 
&gt; 
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; ## kmerCount.fastqq
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; 
&gt; fq &lt;- fastqq(file.path(basedir, &quot;test_l5_N.fq&quot;), k = 2)
[fastqq] File ( 1/1) &#x27;/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata/test_l5_N.fq&#x27;	done.
&gt; if(!identical(kmerCount(fq), kmer_l5_N))
+     stop(&quot;[kmerCount.fastqq] Test 1 &#x27;&quot;, filename, &quot;&#x27; FAILED!&quot;)
&gt; 
&gt; fq&lt;-fastqq(file.path(basedir, &quot;test_l6.fq&quot;), k = 2)
[fastqq] File ( 1/1) &#x27;/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata/test_l6.fq&#x27;	done.
&gt; if(!identical(kmerCount(fq), kmer_l6))
+     stop(&quot;[kmerCount.fastqq] Test 2 &#x27;&quot;, filename, &quot;&#x27; FAILED!&quot;)
&gt; 
&gt; fq&lt;-fastqq(file.path(basedir, &quot;test_l6_multi_line.fq&quot;), k = 2)
[fastqq] File ( 1/1) &#x27;/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata/test_l6_multi_line.fq&#x27;	done.
&gt; if(!identical(kmerCount(fq), kmer_l6_ml))
+     stop(&quot;[kmerCount.fastqq] Test 3 &#x27;&quot;, filename, &quot;&#x27; FAILED!&quot;)
&gt; 
&gt; fq&lt;-fastqq(file.path(basedir, &quot;test_l10_20_40.fq&quot;),k = 2)
[fastqq] File ( 1/1) &#x27;/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata/test_l10_20_40.fq&#x27;	done.
&gt; if(!identical(kmerCount(fq), kmer_l10_20))
+     stop(&quot;[kmerCount.fastqq] Test 4 &#x27;&quot;, filename, &quot;&#x27; FAILED!&quot;)
&gt; 
&gt; fq&lt;-fastqq(file.path(basedir, &quot;test_l10_atcg.fq&quot;), k = 2)
[fastqq] File ( 1/1) &#x27;/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata/test_l10_atcg.fq&#x27;	done.
&gt; if(!identical(kmerCount(fq), kmer_l10_atcg))
+     stop(&quot;[kmerCount.fastqq] Test 5 &#x27;&quot;, filename, &quot;&#x27; FAILED!&quot;)
&gt; 
&gt; fq&lt;-fastqq(file.path(basedir, &quot;test_l10_ATCGN.fq&quot;), k = 2)
[fastqq] File ( 1/1) &#x27;/home/biocbuild/R/R-4.3.0-bin/site-library/seqTools/extdata/test_l10_ATCGN.fq&#x27;	done.
&gt; if(!identical(kmerCount(fq), kmer_l10_ATCGN))
+     stop(&quot;[kmerCount.fastqq] Test 6 &#x27;&quot;, filename, &quot;&#x27; FAILED!&quot;)
&gt; 
&gt; # Counting k-mers on linux (&#x27;\n&#x27;) and equal windows (&#x27;\r\n&#x27;)
&gt; # formatted FASTQ file should give equal results
&gt; # fq&lt;-fastqq(file.path(basedir, c(&quot;test_l4.fq&quot;, &quot;test_win.fq&quot;)), k = 2)
&gt; # kc &lt;- kmerCount(fq)
&gt; # if(!all(kc[,1]==kc[,2]))
&gt; #     stop(&quot;[kmerCount.fastqq] test_l4: kmerCount unequal to test_win.fq&quot;)
&gt; 
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; ## ascii2char, char2ascii
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; 
&gt; if(!identical(ascii2char(97:101, multiple = FALSE), &quot;abcde&quot;))
+     stop(&quot;[ascii2char] Test 1 &#x27;&quot;, filename, &quot;&#x27; FAILED!&quot;)
&gt; 
&gt; if(!identical(ascii2char(97:101, multiple = TRUE), letters[1:5]))
+     stop(&quot;[ascii2char] Test 2 &#x27;&quot;, filename, &quot;&#x27; FAILED!&quot;)
&gt; 
&gt; if(!identical(ascii2char(char2ascii(&quot;abcde&quot;)), &quot;abcde&quot;))
+     stop(&quot;[ascii2char] Test 3 &#x27;&quot;, filename, &quot;&#x27; FAILED!&quot;)
&gt; 
&gt; if(!identical(char2ascii(&quot;abcde&quot;), 97:101))
+     stop(&quot;[char2ascii] Test 1 &#x27;&quot;, filename, &quot;&#x27; FAILED!&quot;)
&gt; 
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; ## END OF FILE
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; 
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.281   0.052   0.316 
</PRE>
</DIV><P><SPAN class="filename">seqTools.Rcheck/tests/test_seqTools.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; 
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; ## Load prerequisites
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; 
&gt; library(seqTools)
Loading required package: zlibbioc
&gt; 
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; ## Initialize example data
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; 
&gt; basedir&lt;-system.file(&quot;extdata&quot;,package=&quot;seqTools&quot;)
&gt; 
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; ## Run tests
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; 
&gt; filename &lt;- &quot;test_seqTools.R&quot;
&gt; basedir &lt;- system.file(&quot;extdata&quot;, package = &quot;seqTools&quot;)
&gt; load(file.path(basedir,&quot;test_res.RData&quot;))
&gt; 
&gt; 
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; ## countDnaKmers
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; 
&gt; if(!identical(countDnaKmers(&quot;ACGT&quot;, k = 1, start = 3:1, width = 1), cdk_ACGT))
+     stop(&quot;[countDnaKmers] Test 1 &#x27;&quot;, filename, &quot;&#x27; FAILED!&quot;)
&gt; 
&gt; if(!identical(countDnaKmers(&quot;ACGT&quot;, k = 1, start = 3, width = 1), cdk_ACGT_one))
+     stop(&quot;[countDnaKmers] Test 2 &#x27;&quot;, filename, &quot;&#x27; FAILED!&quot;)
&gt; 
&gt; if(!identical(
+         countDnaKmers(&quot;ATTNAC&quot;, k = 2, start = 1:3, width = 1), cdk_ATTNAC))
+     stop(&quot;[countDnaKmers] Test 3 &#x27;&quot;, filename, &quot;&#x27; FAILED!&quot;)
&gt; 
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; ## revCountDnaKmers
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; 
&gt; if(!identical(
+         revCountDnaKmers(&quot;ACGTACGT&quot;, k = 2, start = 6:4, width = 2), rck_ACGT))
+     stop(&quot;[revCountDnaKmers] Test 1 &#x27;&quot;, filename, &quot;&#x27; FAILED!&quot;)
&gt; 
&gt; 
&gt; 
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; ## END OF FILE
&gt; ## + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + ##
&gt; 
&gt; proc.time()
   user  system elapsed 
  0.277   0.053   0.311 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">seqTools.Rcheck/seqTools-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>ascii2char</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cbDistMatrix</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>countDnaKmers</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>countFastaKmers</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>countGenomeKmers</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>countSpliceKmers</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>kMerIndex</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>phredTable</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>revCountDnaKmers</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>simFastqqRunTimes</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sim_fq</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>writeFai</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>writeSimContFastq</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>writeSimFastq</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
