<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for GenomicDataCommons on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><B>[<A href="..#G">G</A>]</B></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-16 06:06:55 -0000 (Tue, 16 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4787</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for GenomicDataCommons on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the GenomicDataCommons package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/GenomicDataCommons.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>792</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">GenomicDataCommons</A>&nbsp;1.25.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/GenomicDataCommons">landing page</A>)</SPAN><BR>Sean Davis<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-13&nbsp;23:12:22&nbsp;-0000&nbsp;(Sat,&nbsp;13&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/GenomicDataCommons</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">36a20cc</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:53:27&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: GenomicDataCommons</TD></TR>
<TR><TD><B>Version</B>: 1.25.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:GenomicDataCommons.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings GenomicDataCommons_1.25.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-15 04:01:54 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-15 04:06:04 -0000 (Mon, 15 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 249.3 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: GenomicDataCommons.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD check --install=check:GenomicDataCommons.install-out.txt --library=/home/biocbuild/R/R-4.3.0-bin/site-library --timings GenomicDataCommons_1.25.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/GenomicDataCommons.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘GenomicDataCommons/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘GenomicDataCommons’ version ‘1.25.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘GenomicDataCommons’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking R/sysdata.rda ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘GenomicDataCommons-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: gdcdata
&gt; ### Title: Download GDC files
&gt; ### Aliases: gdcdata
&gt; 
&gt; ### ** Examples
&gt; 
&gt; # get some example file uuids
&gt; uuids &lt;- files() %&gt;%
+     filter(~ access == &#x27;open&#x27; &amp; file_size &lt; 100000) %&gt;%
+     results(size = 3) %&gt;%
+     ids()
&gt; 
&gt; # and get the data, placing it into the gdc_cache() directory
&gt; gdcdata(uuids, use_cached=TRUE)
                                                                                                      0f8d8202-a1ca-4ea1-98b2-c20a6b08479a 
                           &quot;~/.cache/GenomicDataCommons/0f8d8202-a1ca-4ea1-98b2-c20a6b08479a/nationwidechildrens.org_ssf.TCGA-AN-A0FL.xml&quot; 
                                                                                                      b76f87b3-99c5-4297-b2df-8cbea8ecaf61 
&quot;~/.cache/GenomicDataCommons/b76f87b3-99c5-4297-b2df-8cbea8ecaf61/7c4e4c2a-a0b1-424f-97d8-359825674429.wxs.aliquot_ensemble_masked.maf.gz&quot; 
                                                                                                      0cc3d450-2c60-4cb0-a073-d92dc979fa5e 
                      &quot;~/.cache/GenomicDataCommons/0cc3d450-2c60-4cb0-a073-d92dc979fa5e/nationwidechildrens.org_clinical.TCGA-BH-A18F.xml&quot; 
&gt; 
&gt; # legacy data
&gt; exon &lt;- files(legacy = TRUE) %&gt;%
+     filter( ~ cases.project.project_id == &quot;TCGA-COAD&quot; &amp;
+         data_category == &quot;Gene expression&quot; &amp;
+         data_type == &quot;Exon quantification&quot;) %&gt;%
+     results(size = 1) %&gt;% ids()
Error in .gdc_post(entity_name(x), body = body, legacy = x$legacy, token = NULL,  : 
  Gone (HTTP 410).
Calls: %&gt;% ... stopifnot -&gt; is.response -&gt; .gdc_post -&gt; stop_for_status
Execution halted
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
    3. ├─GenomicDataCommons::results(., size = 10)
    4. └─GenomicDataCommons:::results.GDCQuery(., size = 10)
    5.   ├─GenomicDataCommons::results(response(x, ...))
    6.   ├─GenomicDataCommons::response(x, ...)
    7.   └─GenomicDataCommons:::response.GDCQuery(x, ...)
    8.     ├─jsonlite (local) response_handler(...)
    9.     ├─httr::content(...)
   10.     │ ├─base::stopifnot(is.response(x))
   11.     │ └─httr:::is.response(x)
   12.     └─GenomicDataCommons:::.gdc_post(...)
   13.       └─httr::stop_for_status(response)
  
  [ FAIL 1 | WARN 0 | SKIP 0 | PASS 46 ]
  Error: Test failures
  Execution halted
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘overview.Rmd’ using ‘UTF-8’... OK
  ‘questions-and-answers.Rmd’ using ‘UTF-8’... OK
  ‘somatic_mutations.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 2 ERRORs
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/GenomicDataCommons.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">GenomicDataCommons.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0-bin/bin/R CMD INSTALL GenomicDataCommons
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0-bin/site-library’
* installing *source* package ‘GenomicDataCommons’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (GenomicDataCommons)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">GenomicDataCommons.Rcheck/tests/testthat.Rout.fail</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(GenomicDataCommons)
Loading required package: magrittr

Attaching package: &#x27;magrittr&#x27;

The following objects are masked from &#x27;package:testthat&#x27;:

    equals, is_less_than, not


Attaching package: &#x27;GenomicDataCommons&#x27;

The following object is masked from &#x27;package:stats&#x27;:

    filter

&gt; 
&gt; test_check(&quot;GenomicDataCommons&quot;)
[ FAIL 1 | WARN 0 | SKIP 0 | PASS 46 ]

══ Failed tests ════════════════════════════════════════════════════════════════
── Error (&#x27;test_legacy.R:16:1&#x27;): (code run outside of `test_that()`) ───────────
&lt;http_410/http_400/http_error/error/condition&gt;
Error in `.gdc_post(entity_name(x), body = body, legacy = x$legacy, token = NULL, 
    ...)`: Gone (HTTP 410).
Backtrace:
     ▆
  1. ├─files(legacy = TRUE) %&gt;% results(size = 10) %&gt;% ids() at test_legacy.R:16:0
  2. ├─GenomicDataCommons::ids(.)
  3. ├─GenomicDataCommons::results(., size = 10)
  4. └─GenomicDataCommons:::results.GDCQuery(., size = 10)
  5.   ├─GenomicDataCommons::results(response(x, ...))
  6.   ├─GenomicDataCommons::response(x, ...)
  7.   └─GenomicDataCommons:::response.GDCQuery(x, ...)
  8.     ├─jsonlite (local) response_handler(...)
  9.     ├─httr::content(...)
 10.     │ ├─base::stopifnot(is.response(x))
 11.     │ └─httr:::is.response(x)
 12.     └─GenomicDataCommons:::.gdc_post(...)
 13.       └─httr::stop_for_status(response)

[ FAIL 1 | WARN 0 | SKIP 0 | PASS 46 ]
Error: Test failures
Execution halted
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">GenomicDataCommons.Rcheck/GenomicDataCommons-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>aggregations</TD><TD style="text-align: right;">0.191</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">1.692</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>available_expand</TD><TD style="text-align: right;">0.018</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.253</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>available_fields</TD><TD style="text-align: right;">0.042</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.740</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>available_values</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.061</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>constants</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>count</TD><TD style="text-align: right;">0.072</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">1.040</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>default_fields</TD><TD style="text-align: right;">0.042</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.742</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>entity_name</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.521</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>expand</TD><TD style="text-align: right;">0.108</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">1.566</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>faceting</TD><TD style="text-align: right;">0.075</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.816</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>field_description</TD><TD style="text-align: right;">0.153</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">1.354</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filtering</TD><TD style="text-align: right;">0.540</TD><TD style="text-align: right;">0.157</TD><TD style="text-align: right;">6.443</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>gdc_cache</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>gdc_client</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>gdc_clinical</TD><TD style="text-align: right;">0.945</TD><TD style="text-align: right;">0.227</TD><TD style="text-align: right;">3.012</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>gdc_token</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
