<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for Rbowtie on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><B>[<A href="../index.html#R">R</A>]</B></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-23 01:34:22 -0000 (Thu, 23 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4245</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for Rbowtie on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the Rbowtie package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/Rbowtie.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1593</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">Rbowtie</A>&nbsp;1.39.1</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/Rbowtie">landing page</A>)</SPAN><BR>Michael Stadler<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-21&nbsp;12:29:53&nbsp;-0000&nbsp;(Tue,&nbsp;21&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/Rbowtie</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">e24abdc</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-01-29&nbsp;16:48:45&nbsp;-0000&nbsp;(Sun,&nbsp;29&nbsp;Jan&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: Rbowtie</TD></TR>
<TR><TD><B>Version</B>: 1.39.1</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:Rbowtie.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings Rbowtie_1.39.1.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-22 10:57:44 -0000 (Wed, 22 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-22 10:58:13 -0000 (Wed, 22 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 29.1 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: Rbowtie.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:Rbowtie.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings Rbowtie_1.39.1.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/Rbowtie.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘Rbowtie/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘Rbowtie’ version ‘1.39.1’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘Rbowtie’ can be installed ... OK
* checking installed package size ... NOTE
  installed size is 18.3Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking for GNU extensions in Makefiles ... NOTE
GNU make is a SystemRequirements.
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking include directives in Makefiles ... OK
* checking compiled code ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘Rbowtie-Overview.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 2 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/Rbowtie.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">Rbowtie.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL Rbowtie
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘Rbowtie’ ...
** using staged installation
** libs
g++ -O3 -DCOMPILER_OPTIONS=&quot;\&quot;-O3 -Wl,--hash-style=both -g -O2  -Wall  -g -O2  -Wall  \&quot;&quot; -Wl,--hash-style=both -g -O2  -Wall  -g -O2  -Wall    \
	-fno-strict-aliasing -DBOWTIE_VERSION=&quot;\&quot;`cat VERSION`\&quot;&quot; -DBUILD_HOST=&quot;\&quot;`hostname`\&quot;&quot; -DBUILD_TIME=&quot;\&quot;2023-02-21T17:31:19\&quot;&quot; -DCOMPILER_VERSION=&quot;\&quot;`g++ -v 2&gt;&amp;1 | tail -1`\&quot;&quot; -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -D_GNU_SOURCE  -DPREFETCH_LOCALITY=2 -DBOWTIE_MM -DBOWTIE_SHARED_MEM -DNDEBUG -Wall -Wno-unused-parameter -Wno-reorder \
	 -I third_party \
	-o bowtie-build-s ebwt_build.cpp \
	ccnt_lut.cpp ref_read.cpp alphabet.cpp shmem.cpp edit.cpp ebwt.cpp bt2_locks.cpp tinythread.cpp  bowtie_build_main.cpp \
	-L/usr/local/lib  -lz -lpthread 
In file included from ebwt_build.cpp:8:
In member function ‘void EList&lt;T, S&gt;::push_back(const T&amp;) [with T = QSortRange; int S = 128]’,
    inlined from ‘void mkeyQSortSuf2(const T&amp;, size_t, TIndexOffU*, size_t, TIndexOffU*, int, size_t, size_t, size_t, size_t, EList&lt;long unsigned int&gt;*) [with T = S2bDnaString]’ at multikey_qsort.h:584:31:
ds.h:497:17: warning: ‘tmp2’ may be used uninitialized [-Wmaybe-uninitialized]
  497 |                 list_[cur_++] = el;
      |                 ^~~~~
In file included from diff_sample.h:14,
                 from blockwise_sa.h:17,
                 from ebwt.h:25,
                 from ebwt_build.cpp:10:
multikey_qsort.h: In function ‘void mkeyQSortSuf2(const T&amp;, size_t, TIndexOffU*, size_t, TIndexOffU*, int, size_t, size_t, size_t, size_t, EList&lt;long unsigned int&gt;*) [with T = S2bDnaString]’:
multikey_qsort.h:583:36: note: ‘tmp2’ declared here
  583 |                         QSortRange tmp2;
      |                                    ^~~~
In file included from ebwt_build.cpp:8:
In member function ‘void EList&lt;T, S&gt;::push_back(const T&amp;) [with T = QSortRange; int S = 128]’,
    inlined from ‘void mkeyQSortSuf2(const T&amp;, size_t, TIndexOffU*, size_t, TIndexOffU*, int, size_t, size_t, size_t, size_t, EList&lt;long unsigned int&gt;*) [with T = S2bDnaString]’ at multikey_qsort.h:593:31:
ds.h:497:17: warning: ‘tmp3’ may be used uninitialized [-Wmaybe-uninitialized]
  497 |                 list_[cur_++] = el;
      |                 ^~~~~
In file included from diff_sample.h:14,
                 from blockwise_sa.h:17,
                 from ebwt.h:25,
                 from ebwt_build.cpp:10:
multikey_qsort.h: In function ‘void mkeyQSortSuf2(const T&amp;, size_t, TIndexOffU*, size_t, TIndexOffU*, int, size_t, size_t, size_t, size_t, EList&lt;long unsigned int&gt;*) [with T = S2bDnaString]’:
multikey_qsort.h:592:36: note: ‘tmp3’ declared here
  592 |                         QSortRange tmp3;
      |                                    ^~~~
In file included from ebwt_build.cpp:8:
In member function ‘void EList&lt;T, S&gt;::push_back(const T&amp;) [with T = QSortRange; int S = 128]’,
    inlined from ‘void mkeyQSortSuf2(const T&amp;, size_t, TIndexOffU*, size_t, TIndexOffU*, int, size_t, size_t, size_t, size_t, EList&lt;long unsigned int&gt;*) [with T = S2bDnaString]’ at multikey_qsort.h:601:31:
ds.h:497:17: warning: ‘tmp4’ may be used uninitialized [-Wmaybe-uninitialized]
  497 |                 list_[cur_++] = el;
      |                 ^~~~~
In file included from diff_sample.h:14,
                 from blockwise_sa.h:17,
                 from ebwt.h:25,
                 from ebwt_build.cpp:10:
multikey_qsort.h: In function ‘void mkeyQSortSuf2(const T&amp;, size_t, TIndexOffU*, size_t, TIndexOffU*, int, size_t, size_t, size_t, size_t, EList&lt;long unsigned int&gt;*) [with T = S2bDnaString]’:
multikey_qsort.h:600:36: note: ‘tmp4’ declared here
  600 |                         QSortRange tmp4;
      |                                    ^~~~
In file included from ebwt_build.cpp:8:
In member function ‘void EList&lt;T, S&gt;::push_back(const T&amp;) [with T = QSortRange; int S = 128]’,
    inlined from ‘void mkeyQSortSuf2(const T&amp;, size_t, TIndexOffU*, size_t, TIndexOffU*, int, size_t, size_t, size_t, size_t, EList&lt;long unsigned int&gt;*) [with T = SString&lt;char&gt;]’ at multikey_qsort.h:584:31:
ds.h:497:17: warning: ‘tmp2’ may be used uninitialized [-Wmaybe-uninitialized]
  497 |                 list_[cur_++] = el;
      |                 ^~~~~
In file included from diff_sample.h:14,
                 from blockwise_sa.h:17,
                 from ebwt.h:25,
                 from ebwt_build.cpp:10:
multikey_qsort.h: In function ‘void mkeyQSortSuf2(const T&amp;, size_t, TIndexOffU*, size_t, TIndexOffU*, int, size_t, size_t, size_t, size_t, EList&lt;long unsigned int&gt;*) [with T = SString&lt;char&gt;]’:
multikey_qsort.h:583:36: note: ‘tmp2’ declared here
  583 |                         QSortRange tmp2;
      |                                    ^~~~
In file included from ebwt_build.cpp:8:
In member function ‘void EList&lt;T, S&gt;::push_back(const T&amp;) [with T = QSortRange; int S = 128]’,
    inlined from ‘void mkeyQSortSuf2(const T&amp;, size_t, TIndexOffU*, size_t, TIndexOffU*, int, size_t, size_t, size_t, size_t, EList&lt;long unsigned int&gt;*) [with T = SString&lt;char&gt;]’ at multikey_qsort.h:593:31:
ds.h:497:17: warning: ‘tmp3’ may be used uninitialized [-Wmaybe-uninitialized]
  497 |                 list_[cur_++] = el;
      |                 ^~~~~
In file included from diff_sample.h:14,
                 from blockwise_sa.h:17,
                 from ebwt.h:25,
                 from ebwt_build.cpp:10:
multikey_qsort.h: In function ‘void mkeyQSortSuf2(const T&amp;, size_t, TIndexOffU*, size_t, TIndexOffU*, int, size_t, size_t, size_t, size_t, EList&lt;long unsigned int&gt;*) [with T = SString&lt;char&gt;]’:
multikey_qsort.h:592:36: note: ‘tmp3’ declared here
  592 |                         QSortRange tmp3;
      |                                    ^~~~
In file included from ebwt_build.cpp:8:
In member function ‘void EList&lt;T, S&gt;::push_back(const T&amp;) [with T = QSortRange; int S = 128]’,
    inlined from ‘void mkeyQSortSuf2(const T&amp;, size_t, TIndexOffU*, size_t, TIndexOffU*, int, size_t, size_t, size_t, size_t, EList&lt;long unsigned int&gt;*) [with T = SString&lt;char&gt;]’ at multikey_qsort.h:601:31:
ds.h:497:17: warning: ‘tmp4’ may be used uninitialized [-Wmaybe-uninitialized]
  497 |                 list_[cur_++] = el;
      |                 ^~~~~
In file included from diff_sample.h:14,
                 from blockwise_sa.h:17,
                 from ebwt.h:25,
                 from ebwt_build.cpp:10:
multikey_qsort.h: In function ‘void mkeyQSortSuf2(const T&amp;, size_t, TIndexOffU*, size_t, TIndexOffU*, int, size_t, size_t, size_t, size_t, EList&lt;long unsigned int&gt;*) [with T = SString&lt;char&gt;]’:
multikey_qsort.h:600:36: note: ‘tmp4’ declared here
  600 |                         QSortRange tmp4;
      |                                    ^~~~
g++ -O3 -DCOMPILER_OPTIONS=&quot;\&quot;-O3 -Wl,--hash-style=both -g -O2  -Wall  -g -O2  -Wall  \&quot;&quot; -Wl,--hash-style=both -g -O2  -Wall  -g -O2  -Wall   \
	-fno-strict-aliasing -DBOWTIE_VERSION=&quot;\&quot;`cat VERSION`\&quot;&quot; -DBUILD_HOST=&quot;\&quot;`hostname`\&quot;&quot; -DBUILD_TIME=&quot;\&quot;2023-02-21T17:31:43\&quot;&quot; -DCOMPILER_VERSION=&quot;\&quot;`g++ -v 2&gt;&amp;1 | tail -1`\&quot;&quot; -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -D_GNU_SOURCE  -DPREFETCH_LOCALITY=2 -DBOWTIE_MM -DBOWTIE_SHARED_MEM -DNDEBUG -Wall -Wno-unused-parameter -Wno-reorder \
	 -I third_party \
	-o bowtie-align-s ebwt_search.cpp \
	ccnt_lut.cpp ref_read.cpp alphabet.cpp shmem.cpp edit.cpp ebwt.cpp bt2_locks.cpp tinythread.cpp qual.cpp pat.cpp ebwt_search_util.cpp ref_aligner.cpp log.cpp hit_set.cpp sam.cpp hit.cpp bowtie_main.cpp \
	-L/usr/local/lib  -lz -lpthread 
mkdir -p ../inst
cp bowtie-align-s ../inst/bowtie
cp bowtie-build-s ../inst/bowtie-build
cd SpliceMap; \
make -f Makefile; \
echo &quot;MAKE of SpliceMap DONE&quot;	
make[1]: Entering directory &#x27;/home/biocbuild/bbs-3.17-bioc/meat/Rbowtie/src/SpliceMap&#x27;
`&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config CXX` `&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config CXXFLAGS` -c SpliceMap_utils_QuasR.cpp
`&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config CXX` `&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config CXXFLAGS` -c params.cpp
`&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config CXX` `&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config CXXFLAGS` -c cfgfile.cpp
`&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config CXX` `&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config LDFLAGS` `&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config CXXFLAGS` -o runSpliceMap_QuasR SpliceMap_utils_QuasR.o cfgfile.o runSpliceMap_QuasR.cpp params.o
runSpliceMap_QuasR.cpp: In function ‘int main(int, char* const*)’:
runSpliceMap_QuasR.cpp:108:38: warning: variable ‘chr_file_list_it’ set but not used [-Wunused-but-set-variable]
  108 |             vector&lt;string&gt;::iterator chr_file_list_it = chr_file_list.begin();
      |                                      ^~~~~~~~~~~~~~~~
runSpliceMap_QuasR.cpp:537:47: warning: comparison of integer expressions of different signedness: ‘std::__cxx11::basic_string&lt;char&gt;::size_type’ {aka ‘long unsigned int’} and ‘int’ [-Wsign-compare]
  537 |                             if (line.length() &gt; head_clip_length) {
      |                                 ~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~
runSpliceMap_QuasR.cpp:595:47: warning: comparison of integer expressions of different signedness: ‘std::__cxx11::basic_string&lt;char&gt;::size_type’ {aka ‘long unsigned int’} and ‘int’ [-Wsign-compare]
  595 |                             if (line.length() &gt; head_clip_length) {
      |                                 ~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~
runSpliceMap_QuasR.cpp:716:47: warning: comparison of integer expressions of different signedness: ‘std::__cxx11::basic_string&lt;char&gt;::size_type’ {aka ‘long unsigned int’} and ‘int’ [-Wsign-compare]
  716 |                             if (line.length() &gt; head_clip_length) {
      |                                 ~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~
runSpliceMap_QuasR.cpp:841:43: warning: comparison of integer expressions of different signedness: ‘std::__cxx11::basic_string&lt;char&gt;::size_type’ {aka ‘long unsigned int’} and ‘int’ [-Wsign-compare]
  841 |                         if (line.length() &gt; head_clip_length) {
      |                             ~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~
runSpliceMap_QuasR.cpp: In function ‘void processbowtie(std::string&amp;, std::string&amp;)’:
runSpliceMap_QuasR.cpp:1327:9: warning: variable ‘min_mismatch’ set but not used [-Wunused-but-set-variable]
 1327 |     int min_mismatch;
      |         ^~~~~~~~~~~~
`&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config CXX` `&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config LDFLAGS` `&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config CXXFLAGS` -o sortsam sortsam.cpp SpliceMap_utils_QuasR.o params.o
sortsam.cpp: In function ‘int main(int, char* const*)’:
sortsam.cpp:189:27: warning: comparison of integer expressions of different signedness: ‘int’ and ‘const size_type’ {aka ‘const long unsigned int’} [-Wsign-compare]
  189 |                 if (nmloc == string::npos) { // unaligned, no NM tag
      |                     ~~~~~~^~~~~~~~~~~~~~~
sortsam.cpp:194:32: warning: comparison of integer expressions of different signedness: ‘int’ and ‘const size_type’ {aka ‘const long unsigned int’} [-Wsign-compare]
  194 |                     if (nmloc2 == string::npos) {
      |                         ~~~~~~~^~~~~~~~~~~~~~~
sortsam.cpp:231:35: warning: comparison of integer expressions of different signedness: ‘int’ and ‘const size_type’ {aka ‘const long unsigned int’} [-Wsign-compare]
  231 |                         if (nmloc == string::npos) { // unaligned, no NM tag
      |                             ~~~~~~^~~~~~~~~~~~~~~
sortsam.cpp:236:40: warning: comparison of integer expressions of different signedness: ‘int’ and ‘const size_type’ {aka ‘const long unsigned int’} [-Wsign-compare]
  236 |                             if (nmloc2 == string::npos) {
      |                                 ~~~~~~~^~~~~~~~~~~~~~~
`&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config CXX` `&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config LDFLAGS` `&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config CXXFLAGS` -o amalgamateSAM amalgamateSAM.cpp SpliceMap_utils_QuasR.o params.o
amalgamateSAM.cpp: In function ‘int extractId(const string&amp;)’:
amalgamateSAM.cpp:19:18: warning: comparison of integer expressions of different signedness: ‘int’ and ‘const size_type’ {aka ‘const long unsigned int’} [-Wsign-compare]
   19 |     if(start_pos != string::npos &amp;&amp; end_pos != string::npos)
      |        ~~~~~~~~~~^~~~~~~~~~~~~~~
amalgamateSAM.cpp:19:45: warning: comparison of integer expressions of different signedness: ‘int’ and ‘const size_type’ {aka ‘const long unsigned int’} [-Wsign-compare]
   19 |     if(start_pos != string::npos &amp;&amp; end_pos != string::npos)
      |                                     ~~~~~~~~^~~~~~~~~~~~~~~
amalgamateSAM.cpp: In function ‘void update_jun_coverage(std::map&lt;std::__cxx11::basic_string&lt;char&gt;, std::map&lt;int, std::map&lt;int, jun_store_ij&gt; &gt; &gt;&amp;, std::string, bool)’:
amalgamateSAM.cpp:695:31: warning: variable ‘front_clip’ set but not used [-Wunused-but-set-variable]
  695 |                 uint_fast32_t front_clip = 0;
      |                               ^~~~~~~~~~
`&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config CXX` `&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config LDFLAGS` `&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config CXXFLAGS` -o getSpliceMapUnmapped getSpliceMapUnmapped.cpp
getSpliceMapUnmapped.cpp: In function ‘int extractNextId(const string&amp;)’:
getSpliceMapUnmapped.cpp:88:18: warning: comparison of integer expressions of different signedness: ‘int’ and ‘const size_type’ {aka ‘const long unsigned int’} [-Wsign-compare]
   88 |     if(start_pos != string::npos &amp;&amp; end_pos != string::npos)
      |        ~~~~~~~~~~^~~~~~~~~~~~~~~
getSpliceMapUnmapped.cpp:88:45: warning: comparison of integer expressions of different signedness: ‘int’ and ‘const size_type’ {aka ‘const long unsigned int’} [-Wsign-compare]
   88 |     if(start_pos != string::npos &amp;&amp; end_pos != string::npos)
      |                                     ~~~~~~~~^~~~~~~~~~~~~~~
`&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config CXX` `&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config LDFLAGS` `&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config CXXFLAGS` -o fuseReorder fuseReorder.cpp
`&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config CXX` `&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config LDFLAGS` `&quot;/home/biocbuild/bbs-3.17-bioc/R/bin/R&quot; CMD config CXXFLAGS` -o SpliceMap main.cpp SpliceMap_utils_QuasR.o params.o cfgfile.o
make[1]: Leaving directory &#x27;/home/biocbuild/bbs-3.17-bioc/meat/Rbowtie/src/SpliceMap&#x27;
MAKE of SpliceMap DONE
mkdir -p ../inst
cp SpliceMap/SpliceMap ../inst/SpliceMap
cp SpliceMap/amalgamateSAM ../inst/amalgamateSAM
cp SpliceMap/getSpliceMapUnmapped ../inst/getSpliceMapUnmapped
cp SpliceMap/fuseReorder ../inst/fuseReorder
cp SpliceMap/runSpliceMap_QuasR ../inst/runSpliceMap_QuasR
cp SpliceMap/sortsam ../inst/sortsam
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (Rbowtie)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">Rbowtie.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(Rbowtie)
&gt; 
&gt; test_check(&quot;Rbowtie&quot;)

*** Warning ***
&#x27;bowtie2-build-s&#x27; was run directly.  It is recommended that you run the wrapper script &#x27;bowtie-build&#x27; instead.

 [1] &quot;Usage: bowtie2-build-s [options]* &lt;reference_in&gt; &lt;ebwt_outfile_base&gt;&quot;            
 [2] &quot;    reference_in            comma-separated list of files with ref sequences&quot;    
 [3] &quot;    ebwt_outfile_base       write Ebwt data to files with this dir/basename&quot;     
 [4] &quot;Options:&quot;                                                                        
 [5] &quot;    -f                      reference files are Fasta (default)&quot;                 
 [6] &quot;    -c                      reference sequences given on cmd line (as &lt;seq_in&gt;)&quot; 
 [7] &quot;    -a/--noauto             disable automatic -p/--bmax/--dcv memory-fitting&quot;    
 [8] &quot;    -p/--packed             use packed strings internally; slower, uses less mem&quot;
 [9] &quot;    --bmax &lt;int&gt;            max bucket sz for blockwise suffix-array builder&quot;    
[10] &quot;    --bmaxdivn &lt;int&gt;        max bucket sz as divisor of ref len (default: 4)&quot;    
[11] &quot;    --dcv &lt;int&gt;             diff-cover period for blockwise (default: 1024)&quot;     
[12] &quot;    --nodc                  disable diff-cover (algorithm becomes quadratic)&quot;    
[13] &quot;    -r/--noref              don&#x27;t build .3/.4.ebwt (packed reference) portion&quot;   
[14] &quot;    -3/--justref            just build .3/.4.ebwt (packed reference) portion&quot;    
[15] &quot;    -o/--offrate &lt;int&gt;      SA is sampled every 2^offRate BWT chars (default: 5)&quot;
[16] &quot;    -t/--ftabchars &lt;int&gt;    # of chars consumed in initial lookup (default: 10)&quot; 
[17] &quot;    --threads &lt;int&gt;         # of threads&quot;                                        
[18] &quot;    --ntoa                  convert Ns in reference to As&quot;                       
[19] &quot;    --seed &lt;int&gt;            seed for random number generator&quot;                    
[20] &quot;    -q/--quiet              verbose output (for debugging)&quot;                      
[21] &quot;    -h/--help               print detailed description of tool and its options&quot;  
[22] &quot;    --usage                 print this usage message&quot;                            
[23] &quot;    --version               print version information and quit&quot;                  

*** Warning ***
bowtie-align-s was run directly.  It is recommended that you run the wrapper script &#x27;bowtie&#x27; instead.

 [1] &quot;Usage: &quot;                                                                                           
 [2] &quot;bowtie-align-s [options]* -x &lt;ebwt&gt; {-1 &lt;m1&gt; -2 &lt;m2&gt; | --12 &lt;r&gt; | --interleaved &lt;i&gt; | &lt;s&gt;} [&lt;hit&gt;]&quot;
 [3] &quot;&quot;                                                                                                  
 [4] &quot;  &lt;ebwt&gt;  Index filename prefix (minus trailing .X.ebwt).&quot;                                         
 [5] &quot;  &lt;m1&gt;    Comma-separated list of files containing upstream mates (or the&quot;                         
 [6] &quot;          sequences themselves, if -c is set) paired with mates in &lt;m2&gt;&quot;                           
 [7] &quot;  &lt;m2&gt;    Comma-separated list of files containing downstream mates (or the&quot;                       
 [8] &quot;          sequences themselves if -c is set) paired with mates in &lt;m1&gt;&quot;                            
 [9] &quot;  &lt;r&gt;     Comma-separated list of files containing Crossbow-style reads.  Can be&quot;                  
[10] &quot;          a mixture of paired and unpaired.  Specify \&quot;-\&quot; for stdin.&quot;                             
[11] &quot;  &lt;i&gt;     Files with interleaved paired-end FASTQ reads.&quot;                                          
[12] &quot;  &lt;s&gt;     Comma-separated list of files containing unpaired reads, or the&quot;                         
[13] &quot;          sequences themselves, if -c is set.  Specify \&quot;-\&quot; for stdin.&quot;                           
[14] &quot;  &lt;hit&gt;   File to write hits to (default: stdout)&quot;                                                 
[15] &quot;Input:&quot;                                                                                            
[16] &quot;  -q                 query input files are FASTQ .fq/.fastq (default)&quot;                             
[17] &quot;  -f                 query input files are (multi-)FASTA .fa/.mfa&quot;                                 
[18] &quot;  -F k:&lt;int&gt;,i:&lt;int&gt; query input files are continuous FASTA where reads&quot;                           
[19] &quot;                     are substrings (k-mers) extracted from a FASTA file &lt;s&gt;&quot;                      
[20] &quot;                     and aligned at offsets 1, 1+i, 1+2i ... end of reference&quot;                     
[21] &quot;  -r                 query input files are raw one-sequence-per-line&quot;                              
[22] &quot;  -c                 query sequences given on cmd line (as &lt;mates&gt;, &lt;singles&gt;)&quot;                    
[23] &quot;  -Q/--quals &lt;file&gt;  QV file(s) corresponding to CSFASTA inputs; use with -f -C&quot;                   
[24] &quot;  --Q1/--Q2 &lt;file&gt;   same as -Q, but for mate files 1 and 2 respectively&quot;                          
[25] &quot;  -s/--skip &lt;int&gt;    skip the first &lt;int&gt; reads/pairs in the input&quot;                                
[26] &quot;  -u/--qupto &lt;int&gt;   stop after first &lt;int&gt; reads/pairs (excl. skipped reads)&quot;                     
[27] &quot;  -5/--trim5 &lt;int&gt;   trim &lt;int&gt; bases from 5&#x27; (left) end of reads&quot;                                 
[28] &quot;  -3/--trim3 &lt;int&gt;   trim &lt;int&gt; bases from 3&#x27; (right) end of reads&quot;                                
[29] &quot;  --phred33-quals    input quals are Phred+33 (default)&quot;                                           
[30] &quot;  --phred64-quals    input quals are Phred+64 (same as --solexa1.3-quals)&quot;                         
[31] &quot;  --solexa-quals     input quals are from GA Pipeline ver. &lt; 1.3&quot;                                  
[32] &quot;  --solexa1.3-quals  input quals are from GA Pipeline ver. &gt;= 1.3&quot;                                 
[33] &quot;  --integer-quals    qualities are given as space-separated integers (not ASCII)&quot;                  
[34] &quot;Alignment:&quot;                                                                                        
[35] &quot;  -v &lt;int&gt;           report end-to-end hits w/ &lt;=v mismatches; ignore qualities&quot;                   
[36] &quot;    or&quot;                                                                                            
[37] &quot;  -n/--seedmms &lt;int&gt; max mismatches in seed (can be 0-3, default: -n 2)&quot;                           
[38] &quot;  -e/--maqerr &lt;int&gt;  max sum of mismatch quals across alignment for -n (def: 70)&quot;                  
[39] &quot;  -l/--seedlen &lt;int&gt; seed length for -n (default: 28)&quot;                                             
[40] &quot;  --nomaqround       disable Maq-like quality rounding for -n (nearest 10 &lt;= 30)&quot;                  
[41] &quot;  -I/--minins &lt;int&gt;  minimum insert size for paired-end alignment (default: 0)&quot;                    
[42] &quot;  -X/--maxins &lt;int&gt;  maximum insert size for paired-end alignment (default: 250)&quot;                  
[43] &quot;  --fr/--rf/--ff     -1, -2 mates align fw/rev, rev/fw, fw/fw (default: --fr)&quot;                     
[44] &quot;  --nofw/--norc      do not align to forward/reverse-complement reference strand&quot;                  
[45] &quot;  --maxbts &lt;int&gt;     max # backtracks for -n 2/3 (default: 125, 800 for --best)&quot;                   
[46] &quot;  --pairtries &lt;int&gt;  max # attempts to find mate for anchor hit (default: 100)&quot;                    
[47] &quot;  -y/--tryhard       try hard to find valid alignments, at the expense of speed&quot;                   
[48] &quot;  --chunkmbs &lt;int&gt;   max megabytes of RAM for best-first search frames (def: 64)&quot;                  
[49] &quot; --reads-per-batch   # of reads to read from input file at once (default: 16)&quot;                     
[50] &quot;Reporting:&quot;                                                                                        
[51] &quot;  -k &lt;int&gt;           report up to &lt;int&gt; good alignments per read (default: 1)&quot;                     
[52] &quot;  -a/--all           report all alignments per read (much slower than low -k)&quot;                     
[53] &quot;  -m &lt;int&gt;           suppress all alignments if &gt; &lt;int&gt; exist (def: no limit)&quot;                     
[54] &quot;  -M &lt;int&gt;           like -m, but reports 1 random hit (MAPQ=0); requires --best&quot;                  
[55] &quot;  --best             hits guaranteed best stratum; ties broken by quality&quot;                         
[56] &quot;  --strata           hits in sub-optimal strata aren&#x27;t reported (requires --best)&quot;                 
[57] &quot;Output:&quot;                                                                                           
[58] &quot;  -t/--time          print wall-clock time taken by search phases&quot;                                 
[59] &quot;  -B/--offbase &lt;int&gt; leftmost ref offset = &lt;int&gt; in bowtie output (default: 0)&quot;                    
[60] &quot;  --quiet            print nothing but the alignments&quot;                                             
[61] &quot;  --refidx           refer to ref. seqs by 0-based index rather than name&quot;                         
[62] &quot;  --al &lt;fname&gt;       write aligned reads/pairs to file(s) &lt;fname&gt;&quot;                                 
[63] &quot;  --un &lt;fname&gt;       write unaligned reads/pairs to file(s) &lt;fname&gt;&quot;                               
[64] &quot;  --no-unal          suppress SAM records for unaligned reads&quot;                                     
[65] &quot;  --max &lt;fname&gt;      write reads/pairs over -m limit to file(s) &lt;fname&gt;&quot;                           
[66] &quot;  --suppress &lt;cols&gt;  suppresses given columns (comma-delim&#x27;ed) in default output&quot;                  
[67] &quot;  --fullref          write entire ref name (default: only up to 1st space)&quot;                        
[68] &quot;SAM:&quot;                                                                                              
[69] &quot;  -S/--sam           write hits in SAM format&quot;                                                     
[70] &quot;  --mapq &lt;int&gt;       default mapping quality (MAPQ) to print for SAM alignments&quot;                   
[71] &quot;  --sam-nohead       supppress header lines (starting with @) for SAM output&quot;                      
[72] &quot;  --sam-nosq         supppress @SQ header lines for SAM output&quot;                                    
[73] &quot;  --sam-RG &lt;text&gt;    add &lt;text&gt; (usually \&quot;lab=value\&quot;) to @RG line of SAM header&quot;                 
[74] &quot;Performance:&quot;                                                                                      
[75] &quot;  -o/--offrate &lt;int&gt; override offrate of index; must be &gt;= index&#x27;s offrate&quot;                        
[76] &quot;  -p/--threads &lt;int&gt; number of alignment threads to launch (default: 1)&quot;                           
[77] &quot;  --mm               use memory-mapped I/O for index; many &#x27;bowtie&#x27;s can share&quot;                    
[78] &quot;  --shmem            use shared mem for index; many &#x27;bowtie&#x27;s can share&quot;                           
[79] &quot;Other:&quot;                                                                                            
[80] &quot;  --seed &lt;int&gt;       seed for random number generator&quot;                                             
[81] &quot;  --verbose          verbose output (for debugging)&quot;                                               
[82] &quot;  --version          print version information and quit&quot;                                           
[83] &quot;  -h/--help          print this usage message&quot;                                                     
[1] &quot;/home/biocbuild/bbs-3.17-bioc/R/library/Rbowtie/bowtie version 1.3.0&quot;   
[2] &quot;64-bit&quot;                                                                 
[3] &quot;Built on kunpeng1&quot;                                                      
[4] &quot;2023-02-21T17:31:43&quot;                                                    
[5] &quot;Compiler: gcc version 11.3.0 (Ubuntu 11.3.0-1ubuntu1~22.04) &quot;           
[6] &quot;Options: -O3 -Wl,--hash-style=both -g -O2  -Wall  -g -O2  -Wall  &quot;      
[7] &quot;Sizeof {int, long, long long, void*, size_t, off_t}: {4, 8, 8, 8, 8, 8}&quot;
# reads processed: 3
# reads with at least one alignment: 2 (66.67%)
# reads that failed to align: 1 (33.33%)
Reported 2 alignments
# reads processed: 3
# reads with at least one alignment: 2 (66.67%)
# reads that failed to align: 1 (33.33%)
Reported 2 alignments
# reads processed: 3
# reads with at least one alignment: 0 (0.00%)
# reads that failed to align: 3 (100.00%)
No alignments
# reads processed: 3
# reads with at least one alignment: 2 (66.67%)
# reads that failed to align: 1 (33.33%)
Reported 2 alignments
# reads processed: 3
# reads with at least one alignment: 0 (0.00%)
# reads that failed to align: 3 (100.00%)
No alignments
[ FAIL 0 | WARN 0 | SKIP 0 | PASS 68 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
  1.711   0.444   3.450 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">Rbowtie.Rcheck/Rbowtie-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>Rbowtie</TD><TD style="text-align: right;">0.313</TD><TD style="text-align: right;">0.162</TD><TD style="text-align: right;">0.486</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SpliceMap</TD><TD style="text-align: right;">0.399</TD><TD style="text-align: right;">0.109</TD><TD style="text-align: right;">0.984</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
