<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for microbiomeMarker on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.17</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><B>[<A href="..#M">M</A>]</B></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-27 02:34:37 -0000 (Mon, 27 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4259</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for microbiomeMarker on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the microbiomeMarker package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/microbiomeMarker.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1183</B>/2169</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">microbiomeMarker</A>&nbsp;1.5.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/microbiomeMarker">landing page</A>)</SPAN><BR>Yang Cao<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-23&nbsp;09:40:21&nbsp;-0000&nbsp;(Thu,&nbsp;23&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/microbiomeMarker</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">9ef4bd2</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:25:50&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: microbiomeMarker</TD></TR>
<TR><TD><B>Version</B>: 1.5.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:microbiomeMarker.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings microbiomeMarker_1.5.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-24 14:10:58 -0000 (Fri, 24 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-24 14:24:48 -0000 (Fri, 24 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 830.2 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: microbiomeMarker.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:microbiomeMarker.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings microbiomeMarker_1.5.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/microbiomeMarker.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘microbiomeMarker/DESCRIPTION’ ... OK
* this is package ‘microbiomeMarker’ version ‘1.5.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘microbiomeMarker’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking R/sysdata.rda ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                       user system elapsed
run_ancombc          13.524  0.680  45.317
plot_cladogram       13.389  0.320  13.711
run_aldex            13.027  0.587  13.622
plot_heatmap          8.776  0.032   8.814
run_lefse             8.565  0.124   8.690
run_deseq2            8.283  0.236   8.520
plot_sl_roc           6.169  0.326   6.712
plot_abundance        5.965  0.048   6.014
run_test_two_groups   5.813  0.159   5.972
marker_table-methods  5.378  0.108   5.486
run_metagenomeseq     5.134  0.104   5.239
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘microbiomeMarker-vignette.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: OK

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">microbiomeMarker.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL microbiomeMarker
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘microbiomeMarker’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
*** copying figures
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (microbiomeMarker)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">microbiomeMarker.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(microbiomeMarker)
&gt; 
&gt; test_check(&quot;microbiomeMarker&quot;)
operating in serial mode
computing center with all features
New names:
* `` -&gt; `...1`
* `` -&gt; `...2`
* `` -&gt; `...3`
* `` -&gt; `...4`
* `` -&gt; `...5`
* `` -&gt; `...6`
* `` -&gt; `...7`
* `` -&gt; `...8`
* `` -&gt; `...9`
* `` -&gt; `...10`
* `` -&gt; `...11`
* `` -&gt; `...12`
New names:
* `` -&gt; `...1`
* `` -&gt; `...2`
* `` -&gt; `...3`
* `` -&gt; `...4`
* `` -&gt; `...5`
* `` -&gt; `...6`
* `` -&gt; `...7`
* `` -&gt; `...8`
* `` -&gt; `...9`
* `` -&gt; `...10`
* `` -&gt; `...11`
* `` -&gt; `...12`
operating in serial mode
New names:
* `` -&gt; `...1`
* `` -&gt; `...2`
* `` -&gt; `...3`
* `` -&gt; `...4`
* `` -&gt; `...5`
* `` -&gt; `...6`
* `` -&gt; `...7`
* `` -&gt; `...8`
* `` -&gt; `...9`
* `` -&gt; `...10`
* `` -&gt; `...11`
* `` -&gt; `...12`
New names:
* `` -&gt; `...1`
* `` -&gt; `...2`
* `` -&gt; `...3`
* `` -&gt; `...4`
* `` -&gt; `...5`
* `` -&gt; `...6`
* `` -&gt; `...7`
* `` -&gt; `...8`
* `` -&gt; `...9`
* `` -&gt; `...10`
* `` -&gt; `...11`
* `` -&gt; `...12`
New names:
* `` -&gt; `...1`
* `` -&gt; `...2`
* `` -&gt; `...3`
* `` -&gt; `...4`
* `` -&gt; `...5`
* `` -&gt; `...6`
* `` -&gt; `...7`
* `` -&gt; `...8`
* `` -&gt; `...9`
* `` -&gt; `...10`
* `` -&gt; `...11`
* `` -&gt; `...12`
New names:
* `` -&gt; `...1`
* `` -&gt; `...2`
* `` -&gt; `...3`
* `` -&gt; `...4`
* `` -&gt; `...5`
* `` -&gt; `...6`
* `` -&gt; `...7`
* `` -&gt; `...8`
* `` -&gt; `...9`
* `` -&gt; `...10`
* `` -&gt; `...11`
* `` -&gt; `...12`
`set.seed(2020)` was used to initialize repeatable random subsampling.
Please record this for your records so others can reproduce.
Try `set.seed(2020); .Random.seed` for the full vector
...
46OTUs were removed because they are no longer 
present in any sample after random subsampling

...
`set.seed(2020)` was used to initialize repeatable random subsampling.
Please record this for your records so others can reproduce.
Try `set.seed(2020); .Random.seed` for the full vector
...
46OTUs were removed because they are no longer 
present in any sample after random subsampling

...
`set.seed(2020)` was used to initialize repeatable random subsampling.
Please record this for your records so others can reproduce.
Try `set.seed(2020); .Random.seed` for the full vector
...
46OTUs were removed because they are no longer 
present in any sample after random subsampling

...
`set.seed(2020)` was used to initialize repeatable random subsampling.
Please record this for your records so others can reproduce.
Try `set.seed(2020); .Random.seed` for the full vector
...
46OTUs were removed because they are no longer 
present in any sample after random subsampling

...
Default value being used.
Default value being used.
Default value being used.
Default value being used.
converting counts to integer mode
Loading required package: ggplot2
Loading required package: lattice
[ FAIL 0 | WARN 43 | SKIP 8 | PASS 217 ]

══ Skipped tests ═══════════════════════════════════════════════════════════════
• On CRAN (7)
• empty test (1)

[ FAIL 0 | WARN 43 | SKIP 8 | PASS 217 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
106.911   2.578 109.488 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">microbiomeMarker.Rcheck/microbiomeMarker-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>abundances-methods</TD><TD style="text-align: right;">0.095</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.103</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>aggregate_taxa</TD><TD style="text-align: right;">0.734</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.750</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>assign-marker_table</TD><TD style="text-align: right;">4.252</TD><TD style="text-align: right;">0.196</TD><TD style="text-align: right;">4.449</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>confounder</TD><TD style="text-align: right;">2.743</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">2.768</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>effect_size-plot</TD><TD style="text-align: right;">4.581</TD><TD style="text-align: right;">0.101</TD><TD style="text-align: right;">4.682</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>extract_posthoc_res</TD><TD style="text-align: right;">0.156</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.172</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_dada2</TD><TD style="text-align: right;">0.098</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.099</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_picrust2</TD><TD style="text-align: right;">0.051</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.053</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_qiime2</TD><TD style="text-align: right;">0.329</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.356</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>marker_table-methods</TD><TD style="text-align: right;">5.378</TD><TD style="text-align: right;">0.108</TD><TD style="text-align: right;">5.486</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>microbiomeMarker</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.018</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nmarker-methods</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>normalize-methods</TD><TD style="text-align: right;">0.137</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.137</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>phyloseq2DESeq2</TD><TD style="text-align: right;">0.694</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.714</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>phyloseq2edgeR</TD><TD style="text-align: right;">0.179</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.203</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>phyloseq2metagenomeSeq</TD><TD style="text-align: right;">0.254</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.258</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_abundance</TD><TD style="text-align: right;">5.965</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">6.014</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_cladogram</TD><TD style="text-align: right;">13.389</TD><TD style="text-align: right;"> 0.320</TD><TD style="text-align: right;">13.711</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_heatmap</TD><TD style="text-align: right;">8.776</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">8.814</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_postHocTest</TD><TD style="text-align: right;">1.608</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">1.612</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plot_sl_roc</TD><TD style="text-align: right;">6.169</TD><TD style="text-align: right;">0.326</TD><TD style="text-align: right;">6.712</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>postHocTest</TD><TD style="text-align: right;">0.140</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">0.164</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>run_aldex</TD><TD style="text-align: right;">13.027</TD><TD style="text-align: right;"> 0.587</TD><TD style="text-align: right;">13.622</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>run_ancom</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>run_ancombc</TD><TD style="text-align: right;">13.524</TD><TD style="text-align: right;"> 0.680</TD><TD style="text-align: right;">45.317</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>run_deseq2</TD><TD style="text-align: right;">8.283</TD><TD style="text-align: right;">0.236</TD><TD style="text-align: right;">8.520</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>run_edger</TD><TD style="text-align: right;">4.075</TD><TD style="text-align: right;">0.100</TD><TD style="text-align: right;">4.182</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>run_lefse</TD><TD style="text-align: right;">8.565</TD><TD style="text-align: right;">0.124</TD><TD style="text-align: right;">8.690</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>run_limma_voom</TD><TD style="text-align: right;">4.254</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">4.282</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>run_metagenomeseq</TD><TD style="text-align: right;">5.134</TD><TD style="text-align: right;">0.104</TD><TD style="text-align: right;">5.239</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>run_posthoc_test</TD><TD style="text-align: right;">0.959</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.971</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>run_simple_stat</TD><TD style="text-align: right;">4.136</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">4.165</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>run_sl</TD><TD style="text-align: right;">2.359</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">2.367</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>run_test_multiple_groups</TD><TD style="text-align: right;">4.466</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">4.470</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>run_test_two_groups</TD><TD style="text-align: right;">5.813</TD><TD style="text-align: right;">0.159</TD><TD style="text-align: right;">5.972</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>subset_marker</TD><TD style="text-align: right;">3.859</TD><TD style="text-align: right;">0.096</TD><TD style="text-align: right;">3.955</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>summarize_taxa</TD><TD style="text-align: right;">0.527</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">0.592</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>transform_abundances</TD><TD style="text-align: right;">0.158</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.170</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
