<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for AnnotationHub on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.17</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><B>[<A href="..#A">A</A>]</B></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-03-01 07:38:35 -0000 (Wed, 01 Mar 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4266</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for AnnotationHub on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the AnnotationHub package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/AnnotationHub.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>58</B>/2171</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">AnnotationHub</A>&nbsp;3.7.1</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/AnnotationHub">landing page</A>)</SPAN><BR>Bioconductor Package Maintainer<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-27&nbsp;07:53:22&nbsp;-0000&nbsp;(Mon,&nbsp;27&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/AnnotationHub</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">57db729</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-02-06&nbsp;13:50:06&nbsp;-0000&nbsp;(Mon,&nbsp;06&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: AnnotationHub</TD></TR>
<TR><TD><B>Version</B>: 3.7.1</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:AnnotationHub.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings AnnotationHub_3.7.1.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-27 20:20:49 -0000 (Mon, 27 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-27 20:31:48 -0000 (Mon, 27 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 659.8 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: AnnotationHub.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:AnnotationHub.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings AnnotationHub_3.7.1.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/AnnotationHub.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘AnnotationHub/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘AnnotationHub’ version ‘3.7.1’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘AnnotationHub’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... WARNING
&#x27;::&#x27; or &#x27;:::&#x27; imports not declared from:
  ‘CompoundDb’ ‘ensembldb’ ‘keras’
Unexported objects imported by &#x27;:::&#x27; calls:
  ‘BiocFileCache:::.get_tbl_rid’ ‘S4Vectors:::selectSome’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... WARNING
checkRd: (5) AnnotationHub-class.Rd:134-142: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:151-154: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:155-158: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:159-162: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:163-166: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:167-171: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:172-178: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:179-213: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:215-217: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:219-221: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:223-225: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:234-238: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:239-243: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:244-250: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:251-272: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:273-281: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:282-287: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:288-293: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:300-304: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:305-309: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:310-321: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:322-325: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:326-329: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:330-337: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:338-345: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:346-351: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:358-362: \item in \describe must have non-empty label
checkRd: (5) AnnotationHub-class.Rd:363-367: \item in \describe must have non-empty label
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘AnnotationHub-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: AnnotationHub-objects
&gt; ### Title: AnnotationHub objects and their related methods and functions
&gt; ### Aliases: class:AnnotationHub AnnotationHub-class class:Hub Hub-class
&gt; ###   .Hub AnnotationHub refreshHub mcols,Hub-method cache cache,Hub-method
&gt; ###   cache,AnnotationHub-method cache&lt;- cache&lt;-,Hub-method hubUrl
&gt; ###   hubUrl,Hub-method hubCache hubCache,Hub-method hubDate
&gt; ###   hubDate,Hub-method package package,Hub-method removeCache isLocalHub
&gt; ###   isLocalHub,Hub-method isLocalHub&lt;- isLocalHub&lt;-,Hub-method
&gt; ###   possibleDates snapshotDate snapshotDate,Hub-method snapshotDate&lt;-
&gt; ###   snapshotDate&lt;-,Hub-method removeResources
&gt; ###   removeResources,missing-method removeResources,character-method
&gt; ###   dbconn,Hub-method dbfile,Hub-method .db_close recordStatus
&gt; ###   recordStatus,Hub-method length,Hub-method names,Hub-method
&gt; ###   fileName,Hub-method $,Hub-method [[,Hub,character,missing-method
&gt; ###   [[,Hub,numeric,missing-method [,Hub,character,missing-method
&gt; ###   [,Hub,logical,missing-method [,Hub,numeric,missing-method
&gt; ###   [&lt;-,Hub,character,missing,Hub-method
&gt; ###   [&lt;-,Hub,logical,missing,Hub-method [&lt;-,Hub,numeric,missing,Hub-method
&gt; ###   subset,Hub-method query query,Hub-method display display,Hub-method
&gt; ###   as.list.Hub as.list,Hub-method c,Hub-method show,Hub-method
&gt; ###   show,AnnotationHubResource-method
&gt; ### Keywords: classes methods
&gt; 
&gt; ### ** Examples
&gt; 
&gt;   ## create an AnnotationHub object
&gt;   library(AnnotationHub)
&gt;   ah = AnnotationHub()
snapshotDate(): 2022-12-16
&gt; 
&gt;   ## Summary of available records
&gt;   ah
AnnotationHub with 67944 records
# snapshotDate(): 2022-12-16
# $dataprovider: Ensembl, BroadInstitute, UCSC, ftp://ftp.ncbi.nlm.nih.gov/g...
# $species: Homo sapiens, Mus musculus, Drosophila melanogaster, Bos taurus,...
# $rdataclass: GRanges, TwoBitFile, BigWigFile, EnsDb, Rle, OrgDb, ChainFile...
# additional mcols(): taxonomyid, genome, description,
#   coordinate_1_based, maintainer, rdatadateadded, preparerclass, tags,
#   rdatapath, sourceurl, sourcetype 
# retrieve records with, e.g., &#x27;object[[&quot;AH5012&quot;]]&#x27; 

             title                                       
  AH5012   | Chromosome Band                             
  AH5013   | STS Markers                                 
  AH5014   | FISH Clones                                 
  AH5015   | Recomb Rate                                 
  AH5016   | ENCODE Pilot                                
  ...        ...                                         
  AH109476 | Ensembl 108 EnsDb for Xiphophorus couchianus
  AH109477 | Ensembl 108 EnsDb for Xiphophorus maculatus 
  AH109478 | Ensembl 108 EnsDb for Xenopus tropicalis    
  AH109479 | Ensembl 108 EnsDb for Zonotrichia albicollis
  AH109480 | Ensembl 108 EnsDb for Zalophus californianus
&gt; 
&gt;   ## Detail for a single record
&gt;   ah[1]
AnnotationHub with 1 record
# snapshotDate(): 2022-12-16
# names(): AH5012
# $dataprovider: UCSC
# $species: Homo sapiens
# $rdataclass: GRanges
# $rdatadateadded: 2013-03-26
# $title: Chromosome Band
# $description: GRanges object from UCSC track &#x27;Chromosome Band&#x27;
# $taxonomyid: 9606
# $genome: hg19
# $sourcetype: UCSC track
# $sourceurl: rtracklayer://hgdownload.cse.ucsc.edu/goldenpath/hg19/database...
# $sourcesize: NA
# $tags: c(&quot;cytoBand&quot;, &quot;UCSC&quot;, &quot;track&quot;, &quot;Gene&quot;, &quot;Transcript&quot;,
#   &quot;Annotation&quot;) 
# retrieve record with &#x27;object[[&quot;AH5012&quot;]]&#x27; 
&gt; 
&gt;   ## and what is the date we are using?
&gt;   snapshotDate(ah)
[1] &quot;2022-12-16&quot;
&gt; 
&gt;   ## how many resources?
&gt;   length(ah)
[1] 67944
&gt; 
&gt;   ## from which resources, is data available?
&gt;   head(sort(table(ah$dataprovider), decreasing=TRUE))

                                                                                                   Ensembl 
                                                                                                     33054 
                                                                                            BroadInstitute 
                                                                                                     18248 
                                                                                                      UCSC 
                                                                                                     11192 
                                                                     ftp://ftp.ncbi.nlm.nih.gov/gene/DATA/ 
                                                                                                      1871 
                                                                                                  Haemcode 
                                                                                                       945 
FANTOM5,DLRP,IUPHAR,HPRD,STRING,SWISSPROT,TREMBL,ENSEMBL,CELLPHONEDB,BADERLAB,SINGLECELLSIGNALR,HOMOLOGENE 
                                                                                                       501 
&gt; 
&gt;   ## from which species, is data available ?
&gt;   head(sort(table(ah$species),decreasing=TRUE))

           Homo sapiens            Mus musculus Drosophila melanogaster 
                  26552                    1796                     458 
             Bos taurus       Rattus norvegicus         Pan troglodytes 
                    331                     325                     317 
&gt; 
&gt;   ## what web service and local cache does this AnnotationHub point to?
&gt;   hubUrl(ah)
[1] &quot;https://annotationhub.bioconductor.org&quot;
&gt;   hubCache(ah)
[1] &quot;/home/biocbuild/.cache/R/AnnotationHub&quot;
&gt; 
&gt;   ### Examples ###
&gt; 
&gt;   ## One can  search the hub for multiple strings
&gt;   ahs2 &lt;- query(ah, c(&quot;GTF&quot;, &quot;77&quot;,&quot;Ensembl&quot;, &quot;Homo sapiens&quot;))
&gt; 
&gt;   ## information about the file can be retrieved using
&gt;   ahs2[1]
AnnotationHub with 1 record
# snapshotDate(): 2022-12-16
# names(): AH28812
# $dataprovider: Ensembl
# $species: Homo sapiens
# $rdataclass: GRanges
# $rdatadateadded: 2015-03-25
# $title: Homo_sapiens.GRCh38.77.gtf
# $description: Gene Annotation for Homo sapiens
# $taxonomyid: 9606
# $genome: GRCh38
# $sourcetype: GTF
# $sourceurl: ftp://ftp.ensembl.org/pub/release-77/gtf/homo_sapiens/Homo_sap...
# $sourcesize: 44454526
# $tags: c(&quot;GTF&quot;, &quot;ensembl&quot;, &quot;Gene&quot;, &quot;Transcript&quot;, &quot;Annotation&quot;) 
# retrieve record with &#x27;object[[&quot;AH28812&quot;]]&#x27; 
&gt; 
&gt;   ## one can further extract information from this show method
&gt;   ## like the sourceurl using:
&gt;   ahs2$sourceurl
[1] &quot;ftp://ftp.ensembl.org/pub/release-77/gtf/homo_sapiens/Homo_sapiens.GRCh38.77.gtf.gz&quot;
&gt;   ahs2$description
[1] &quot;Gene Annotation for Homo sapiens&quot;
&gt;   ahs2$title
[1] &quot;Homo_sapiens.GRCh38.77.gtf&quot;
&gt; 
&gt;   ## We can download a file by name like this (using a list semantic):
&gt;   gr &lt;- ahs2[[1]]
loading from cache
require(“GenomicRanges”)
Error: failed to load resource
  name: AH28812
  title: Homo_sapiens.GRCh38.77.gtf
  reason: error in evaluating the argument &#x27;x&#x27; in selecting a method for function &#x27;get&#x27;: error reading from connection
Execution halted
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘AnnotationHub-HOWTO.Rmd’... OK
  ‘AnnotationHub.Rmd’ using ‘UTF-8’... OK
  ‘TroubleshootingTheCache.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 2 WARNINGs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/AnnotationHub.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">AnnotationHub.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL AnnotationHub
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘AnnotationHub’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (AnnotationHub)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">AnnotationHub.Rcheck/tests/runTests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; BiocGenerics:::testPackage(&quot;AnnotationHub&quot;)

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

snapshotDate(): 2022-12-16
Using &#x27;localHub=TRUE&#x27;
  If offline, please also see BiocManager vignette section on offline use
snapshotDate(): 2022-12-16
snapshotDate(): 2022-12-16
snapshotDate(): 2022-12-16
snapshotDate(): 2022-12-16
snapshotDate(): 2022-12-16
snapshotDate(): 2022-12-16
snapshotDate(): 2022-12-16
snapshotDate(): 2022-12-16
snapshotDate(): 2022-12-16
snapshotDate(): 2022-12-16
Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

require(&quot;xxx_foo&quot;)


RUNIT TEST PROTOCOL -- Mon Feb 27 20:28:28 2023 
*********************************************** 
Number of test functions: 20 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
AnnotationHub RUnit Tests - 20 test functions, 0 errors, 0 failures
Number of test functions: 20 
Number of errors: 0 
Number of failures: 0 
Warning messages:
1: call dbDisconnect() when finished working with a connection 
2: download failed
  hub path: &#x27;bogus/url&#x27;
  cache resource: &#x27;AHInvalid : 5012&#x27;
  reason: all(rtype == &quot;web&quot; | file.exists(fpath)) is not TRUE 
&gt; 
&gt; proc.time()
   user  system elapsed 
 52.641   5.819 295.467 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">AnnotationHub.Rcheck/AnnotationHub-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
