<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for MACSr on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><B>[<A href="../index.html#M">M</A>]</B></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-03 02:36:05 -0000 (Fri, 03 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4039</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for MACSr on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the MACSr package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/MACSr.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1063</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">MACSr</A>&nbsp;1.7.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/MACSr">landing page</A>)</SPAN><BR>Qiang Hu<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-01&nbsp;03:13:00&nbsp;-0000&nbsp;(Wed,&nbsp;01&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/MACSr</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">63e7882</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:23:57&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: MACSr</TD></TR>
<TR><TD><B>Version</B>: 1.7.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:MACSr.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings MACSr_1.7.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-02 11:23:54 -0000 (Thu, 02 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-02 11:30:21 -0000 (Thu, 02 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 387.5 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: MACSr.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:MACSr.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings MACSr_1.7.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/MACSr.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘MACSr/DESCRIPTION’ ... OK
* this is package ‘MACSr’ version ‘1.7.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .BBSoptions
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘MACSr’ can be installed ... NOTE
Found the following notes/warnings:
  Non-staged installation was used
See ‘/home/biocbuild/bbs-3.17-bioc/meat/MACSr.Rcheck/00install.out’ for details.
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in shell scripts ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
               user system elapsed
bdgbroadcall 10.657  0.788  18.217
bdgdiff       7.555  0.177  12.853
refinepeak    6.395  0.091  11.661
cmbreps       5.726  0.168   9.717
bdgcmp        5.035  0.131  10.255
bdgopt        4.572  0.103   9.390
callpeak      4.054  0.136   8.813
predictd      1.876  0.068   5.465
filterdup     1.887  0.052   5.569
randsample    1.733  0.032   5.557
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘MACSr.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 2 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/MACSr.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">MACSr.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL MACSr
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘MACSr’ ...
** using non-staged installation via StagedInstall field
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded
* DONE (MACSr)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">MACSr.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(MACSr)
&gt; 
&gt; test_check(&quot;MACSr&quot;)
INFO  @ Thu, 02 Feb 2023 11:29:21: 
# Command line: 
# ARGUMENTS LIST:
# name = run_callpeak_narrow0
# format = BED
# ChIP-seq file = [&#x27;/home/biocbuild/.cache/R/ExperimentHub/18c47931cd3e22_4601&#x27;]
# control file = [&#x27;/home/biocbuild/.cache/R/ExperimentHub/18c479232b0844_4606&#x27;]
# effective genome size = 5.20e+07
# band width = 300
# model fold = [5.0, 50.0]
# qvalue cutoff = 5.00e-02
# The maximum gap between significant sites is assigned as the read length/tag size.
# The minimum length of peaks is assigned as the predicted fragment length &quot;d&quot;.
# Larger dataset will be scaled towards smaller dataset.
# Range for calculating regional lambda is: 1000 bps and 10000 bps
# Broad region calling is off
# Additional cutoff on fold-enrichment is: 0.10
# Paired-End mode is off
 
INFO  @ Thu, 02 Feb 2023 11:29:21: #1 read tag files... 
INFO  @ Thu, 02 Feb 2023 11:29:21: #1 read treatment tags... 
INFO  @ Thu, 02 Feb 2023 11:29:21: #1.2 read input tags... 
INFO  @ Thu, 02 Feb 2023 11:29:21: #1 tag size is determined as 101 bps 
INFO  @ Thu, 02 Feb 2023 11:29:21: #1 tag size = 101.0 
INFO  @ Thu, 02 Feb 2023 11:29:21: #1  total tags in treatment: 49622 
INFO  @ Thu, 02 Feb 2023 11:29:21: #1 user defined the maximum tags... 
INFO  @ Thu, 02 Feb 2023 11:29:21: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) 
INFO  @ Thu, 02 Feb 2023 11:29:21: #1  tags after filtering in treatment: 48047 
INFO  @ Thu, 02 Feb 2023 11:29:21: #1  Redundant rate of treatment: 0.03 
INFO  @ Thu, 02 Feb 2023 11:29:21: #1  total tags in control: 50837 
INFO  @ Thu, 02 Feb 2023 11:29:21: #1 user defined the maximum tags... 
INFO  @ Thu, 02 Feb 2023 11:29:21: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) 
INFO  @ Thu, 02 Feb 2023 11:29:21: #1  tags after filtering in control: 50783 
INFO  @ Thu, 02 Feb 2023 11:29:21: #1  Redundant rate of control: 0.00 
INFO  @ Thu, 02 Feb 2023 11:29:21: #1 finished! 
INFO  @ Thu, 02 Feb 2023 11:29:21: #2 Build Peak Model... 
INFO  @ Thu, 02 Feb 2023 11:29:21: #2 looking for paired plus/minus strand peaks... 
INFO  @ Thu, 02 Feb 2023 11:29:21: #2 Total number of paired peaks: 469 
INFO  @ Thu, 02 Feb 2023 11:29:21: #2 Model building with cross-correlation: Done 
INFO  @ Thu, 02 Feb 2023 11:29:21: #2 finished! 
INFO  @ Thu, 02 Feb 2023 11:29:21: #2 predicted fragment length is 228 bps 
INFO  @ Thu, 02 Feb 2023 11:29:21: #2 alternative fragment length(s) may be 228 bps 
INFO  @ Thu, 02 Feb 2023 11:29:21: #2.2 Generate R script for model : /tmp/Rtmp9JznNf/run_callpeak_narrow0_model.r 
INFO  @ Thu, 02 Feb 2023 11:29:21: #3 Call peaks... 
INFO  @ Thu, 02 Feb 2023 11:29:21: #3 Pre-compute pvalue-qvalue table... 
INFO  @ Thu, 02 Feb 2023 11:29:21: #3 Cutoff vs peaks called will be analyzed! 
INFO  @ Thu, 02 Feb 2023 11:29:22: #3 Analysis of cutoff vs num of peaks or total length has been saved in b&#x27;/tmp/Rtmp9JznNf/run_callpeak_narrow0_cutoff_analysis.txt&#x27; 
INFO  @ Thu, 02 Feb 2023 11:29:22: #3 In the peak calling step, the following will be performed simultaneously: 
INFO  @ Thu, 02 Feb 2023 11:29:22: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... run_callpeak_narrow0_treat_pileup.bdg 
INFO  @ Thu, 02 Feb 2023 11:29:22: #3   Write bedGraph files for control lambda (after scaling if necessary)... run_callpeak_narrow0_control_lambda.bdg 
INFO  @ Thu, 02 Feb 2023 11:29:22: #3   Pileup will be based on sequencing depth in treatment. 
INFO  @ Thu, 02 Feb 2023 11:29:22: #3 Call peaks for each chromosome... 
INFO  @ Thu, 02 Feb 2023 11:29:22: #4 Write output xls file... /tmp/Rtmp9JznNf/run_callpeak_narrow0_peaks.xls 
INFO  @ Thu, 02 Feb 2023 11:29:22: #4 Write peak in narrowPeak format file... /tmp/Rtmp9JznNf/run_callpeak_narrow0_peaks.narrowPeak 
INFO  @ Thu, 02 Feb 2023 11:29:22: #4 Write summits bed file... /tmp/Rtmp9JznNf/run_callpeak_narrow0_summits.bed 
INFO  @ Thu, 02 Feb 2023 11:29:22: Done! 
[ FAIL 0 | WARN 0 | SKIP 0 | PASS 14 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
 41.161   1.804  83.898 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">MACSr.Rcheck/MACSr-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>bdgbroadcall</TD><TD style="text-align: right;">10.657</TD><TD style="text-align: right;"> 0.788</TD><TD style="text-align: right;">18.217</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bdgcmp</TD><TD style="text-align: right;"> 5.035</TD><TD style="text-align: right;"> 0.131</TD><TD style="text-align: right;">10.255</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bdgdiff</TD><TD style="text-align: right;"> 7.555</TD><TD style="text-align: right;"> 0.177</TD><TD style="text-align: right;">12.853</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bdgopt</TD><TD style="text-align: right;">4.572</TD><TD style="text-align: right;">0.103</TD><TD style="text-align: right;">9.390</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bdgpeakcall</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>callpeak</TD><TD style="text-align: right;">4.054</TD><TD style="text-align: right;">0.136</TD><TD style="text-align: right;">8.813</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>callvar</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cmbreps</TD><TD style="text-align: right;">5.726</TD><TD style="text-align: right;">0.168</TD><TD style="text-align: right;">9.717</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filterdup</TD><TD style="text-align: right;">1.887</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">5.569</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pileup</TD><TD style="text-align: right;">1.738</TD><TD style="text-align: right;">0.067</TD><TD style="text-align: right;">4.873</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>predictd</TD><TD style="text-align: right;">1.876</TD><TD style="text-align: right;">0.068</TD><TD style="text-align: right;">5.465</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>randsample</TD><TD style="text-align: right;">1.733</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">5.557</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>refinepeak</TD><TD style="text-align: right;"> 6.395</TD><TD style="text-align: right;"> 0.091</TD><TD style="text-align: right;">11.661</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
