<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for TarSeqQC on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.17</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><B>[<A href="..#T">T</A>]</B></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-27 02:34:44 -0000 (Mon, 27 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4259</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for TarSeqQC on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the TarSeqQC package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/TarSeqQC.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>2015</B>/2169</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">TarSeqQC</A>&nbsp;1.29.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/TarSeqQC">landing page</A>)</SPAN><BR>Gabriela Merino<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-23&nbsp;09:40:21&nbsp;-0000&nbsp;(Thu,&nbsp;23&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/TarSeqQC</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">edd367f</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:12:16&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: TarSeqQC</TD></TR>
<TR><TD><B>Version</B>: 1.29.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:TarSeqQC.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings TarSeqQC_1.29.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-24 19:29:41 -0000 (Fri, 24 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-24 19:35:14 -0000 (Fri, 24 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 333.6 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: TarSeqQC.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:TarSeqQC.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings TarSeqQC_1.29.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/TarSeqQC.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘TarSeqQC/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘TarSeqQC’ version ‘1.29.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘TarSeqQC’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
plotNtdPercentage,TargetExperiment: no visible global function
  definition for ‘path’
plotRegion,TargetExperiment: no visible global function definition for
  ‘path’
readFrequencies,TargetExperiment: no visible global function definition
  for ‘path’
Undefined global functions or variables:
  path
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘TarSeqQC-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: TargetExperiment-class
&gt; ### Title: TargetExperiment S4 class implementation in R
&gt; ### Aliases: TargetExperiment-class
&gt; 
&gt; ### ** Examples
&gt; 
&gt; ## Defining bam file, bed file and fasta file names and paths
&gt; bamFile&lt;-system.file(&quot;extdata&quot;, &quot;mybam.bam&quot;, package=&quot;TarSeqQC&quot;, 
+     mustWork=TRUE)
&gt; bedFile&lt;-system.file(&quot;extdata&quot;, &quot;mybed.bed&quot;, package=&quot;TarSeqQC&quot;,
+     mustWork=TRUE)
&gt; fastaFile&lt;-system.file(&quot;extdata&quot;, &quot;myfasta.fa&quot;, package=&quot;TarSeqQC&quot;, 
+     mustWork=TRUE)
&gt; 
&gt; ## Creating a TargetExperiment object
&gt; 
&gt; # Defining feature parameter
&gt; feature&lt;-&quot;amplicon&quot;
&gt; # Defining attribute parameter
&gt; attribute&lt;-&quot;coverage&quot;
&gt; ampliPanel&lt;-TargetExperiment(bedFile, bamFile, fastaFile, attribute=attribute,
+     feature=feature)
&gt; 
&gt; ## Alternative object creation
&gt; # Creating the TargetExperiment object
&gt; ampliPanel&lt;-TargetExperiment(bedFile, bamFile, fastaFile)
Warning in .local(.Object, ...) : The &#x27;attribute&#x27; slot is empty
Warning in .local(.Object, ...) : The &#x27;feature&#x27; slot is empty
&gt; # Set feature slot value
&gt; setFeature(ampliPanel)&lt;-&quot;amplicon&quot;
&gt; # Set attribute slot value
&gt; setAttribute(ampliPanel)&lt;-&quot;coverage&quot;
&gt; # Set pileupP slot value in order to set the maximum depth at 1000
&gt; setPileupP(ampliPanel)&lt;-PileupParam(max_depth=1000)
&gt; # Set the featurePanel slot but now using the new pileupP definition
&gt; setFeaturePanel(ampliPanel)&lt;-buildFeaturePanel(ampliPanel)
&gt; ## Early exploration
&gt; # show/print
&gt; ampliPanel
TargetExperiment 
amplicon panel: 
	GRanges object with 3 ranges and 6 metadata columns:
        seqnames    ranges strand |        gene        gc  coverage sdCoverage
           &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;character&gt; &lt;numeric&gt; &lt;numeric&gt;  &lt;numeric&gt;
  AMPL1     chr1   463-551      * |       gene1     0.674       320         19
  AMPL2     chr1 1553-1603      * |       gene2     0.451       550         90
  AMPL3     chr1 3766-3814      * |       gene2     0.531       455         12
        medianCounts IQRCounts
           &lt;numeric&gt; &lt;numeric&gt;
  AMPL1          326        24
  AMPL2          574        14
  AMPL3          463        27
  -------
  seqinfo: 4 sequences from an unspecified genome; no seqlengths

gene panel: 
	GRanges object with 3 ranges and 4 metadata columns:
        seqnames    ranges strand | medianCounts IQRCounts  coverage sdCoverage
           &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; |    &lt;numeric&gt; &lt;numeric&gt; &lt;numeric&gt;  &lt;numeric&gt;
  gene1     chr1   463-551      * |          326         0       320          0
  gene2     chr1 1553-3814      * |          518        56       502         67
  gene3     chr3      1-59      * |            0         0         0          0
  -------
  seqinfo: 4 sequences from an unspecified genome; no seqlengths

selected attribute: 
	 coverage 
&gt; # summary
&gt; summary(ampliPanel)
         Min. 1st Qu. Median Mean 3rd Qu. Max.
gene        0     261    328  312     396  502
amplicon    0     143    288  316     472  931
&gt; # summary at feature level
&gt; summaryFeatureLev(ampliPanel)
         Min. 1st Qu. Median Mean 3rd Qu. Max.
amplicon    0     143    288  316     472  931
&gt; # summary at gene level
&gt; summaryGeneLev(ampliPanel)
     Min. 1st Qu. Median Mean 3rd Qu. Max.
gene    0     261    328  312     396  502
&gt; # attribute boxplot and density plot exploration
&gt; g&lt;-plotAttrExpl(ampliPanel,level=&quot;feature&quot;,join=TRUE, log=FALSE, color=&quot;blue&quot;)
Warning: The `&lt;scale&gt;` argument of `guides()` cannot be `FALSE`. Use &quot;none&quot; instead as
of ggplot2 3.3.4.
ℹ The deprecated feature was likely used in the TarSeqQC package.
  Please report the issue to the authors.
&gt; if(interactive()){
+ x11(type=&quot;cairo&quot;);g
+ }
&gt; # explore amplicon length distribution
&gt; g&lt;-plotMetaDataExpl(ampliPanel, &quot;length&quot;, log=FALSE, join=FALSE, color=
+ &quot;blueviolet&quot;)
&gt; if(interactive()){
+ g
+ }
&gt; # explore gene&#x27;s relative frequencies
&gt; g&lt;-plotMetaDataExpl(ampliPanel, &quot;gene&quot;, abs=FALSE)
&gt; if(interactive()){
+ g
+ }
&gt; ## Deep exploration and Quality Control
&gt; myfrequencies&lt;-readFrequencies(ampliPanel)
&gt; g&lt;-plotInOutFeatures(readFrequencies(ampliPanel))
&gt; if(interactive()){
+ g
+ }
&gt; # definition of the interval extreme values
&gt; attributeThres&lt;-c(0,1,50,200,500, Inf)
&gt; # plot panel overview
&gt; g&lt;-plot(ampliPanel, attributeThres, chrLabels =TRUE)
Error in aggregate.formula(formula = xmin ~ seqnames, data = df_panel,  : 
  argument &#x27;x&#x27; is  missing -- it has been renamed from &#x27;formula&#x27;
Calls: plot -&gt; plot -&gt; .local -&gt; aggregate -&gt; aggregate.formula
Execution halted
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘TarSeqQC-vignette.Rnw’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 1 NOTE
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/TarSeqQC.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">TarSeqQC.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL TarSeqQC
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘TarSeqQC’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (TarSeqQC)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">TarSeqQC.Rcheck/tests/runTests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; BiocGenerics:::testPackage(&quot;TarSeqQC&quot;)

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min


Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname


Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit


Attaching package: &#x27;plyr&#x27;

The following object is masked from &#x27;package:XVector&#x27;:

    compact

The following object is masked from &#x27;package:IRanges&#x27;:

    desc

The following object is masked from &#x27;package:S4Vectors&#x27;:

    rename



RUNIT TEST PROTOCOL -- Fri Feb 24 19:34:31 2023 
*********************************************** 
Number of test functions: 33 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
TarSeqQC RUnit Tests - 33 test functions, 0 errors, 0 failures
Number of test functions: 33 
Number of errors: 0 
Number of failures: 0 
Warning message:
The `&lt;scale&gt;` argument of `guides()` cannot be `FALSE`. Use &quot;none&quot; instead as
of ggplot2 3.3.4.
ℹ The deprecated feature was likely used in the TarSeqQC package.
  Please report the issue to the authors. 
&gt; 
&gt; proc.time()
   user  system elapsed 
 23.294   4.506  23.844 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">TarSeqQC.Rcheck/TarSeqQC-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>TargetExperiment-biasExploration</TD><TD style="text-align: right;">0.129</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.129</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TargetExperiment-buildFeaturePanel</TD><TD style="text-align: right;">2.108</TD><TD style="text-align: right;">0.131</TD><TD style="text-align: right;">2.248</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TargetExperiment-buildReport</TD><TD style="text-align: right;">1.542</TD><TD style="text-align: right;">0.096</TD><TD style="text-align: right;">1.638</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
