<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for easyRNASeq on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><B>[<A href="../index.html#E">E</A>]</B></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-13 06:49:25 -0000 (Mon, 13 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4222</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for easyRNASeq on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the easyRNASeq package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/easyRNASeq.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>577</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">easyRNASeq</A>&nbsp;2.35.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/easyRNASeq">landing page</A>)</SPAN><BR>Nicolas Delhomme<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-11&nbsp;01:58:29&nbsp;-0000&nbsp;(Sat,&nbsp;11&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/easyRNASeq</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">04012c4</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:06:37&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: easyRNASeq</TD></TR>
<TR><TD><B>Version</B>: 2.35.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:easyRNASeq.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings easyRNASeq_2.35.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-11 14:15:23 -0000 (Sat, 11 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-11 14:30:05 -0000 (Sat, 11 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 881.3 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: easyRNASeq.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:easyRNASeq.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings easyRNASeq_2.35.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/easyRNASeq.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘easyRNASeq/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘easyRNASeq’ version ‘2.35.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Files named as vignettes but with no recognized vignette engine:
   ‘inst/doc/01-Introduction.Rmd’
   ‘inst/doc/02-AnnotParam.Rmd’
   ‘inst/doc/03-SyntheticTranscripts.Rmd’
   ‘inst/doc/04-BamParam.Rmd’
   ‘inst/doc/05-RnaSeqParam.Rmd’
   ‘inst/doc/06-simpleRNASeq.Rmd’
   ‘inst/doc/07-cleanUp.Rmd’
   ‘inst/doc/08-Session-Info.Rmd’
   ‘inst/doc/09-Acknowledgments.Rmd’
   ‘inst/doc/10-Foonotes.Rmd’
   ‘inst/doc/11-Images.Rmd’
   ‘inst/doc/12-Appendix.Rmd’
(Is a VignetteBuilder field missing?)
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘easyRNASeq’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
Namespace in Imports field not imported from: ‘locfit’
  All declared Imports should be used.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
.parallelize: no visible global function definition for ‘clusterApply’
Undefined global functions or variables:
  clusterApply
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in Makefiles ... OK
* checking for GNU extensions in Makefiles ... OK
* checking include directives in Makefiles ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                                   user system elapsed
easyRNASeq-simpleRNASeq          95.356  2.011  98.357
easyRNASeq-package               54.584  2.683  57.767
easyRNASeq-synthetic-transcripts 41.460  0.488  42.352
BiocFileCache-methods             9.639  0.895  43.289
easyRNASeq-BamFileList            4.539  0.377   7.092
Rsamtools-methods                 3.947  0.324   6.784
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’
 ERROR
Running the tests in ‘tests/runTests.R’ failed.
Last 13 lines of output:
  3: In FUN(X[[i]], ...) :
    Bam file: 2baec1607f22_TTGCGA.bam is considered unstranded.
  4: In FUN(X[[i]], ...) :
    Bam file: 2baec1607f22_TTGCGA.bam Strandedness could not be determined using 19659 regions spanning 1381886 bp on either strand at a 90% cutoff; 73.37 percent appear to be stranded.
  5: In FUN(X[[i]], ...) :
    Bam file: 2baec164bab29e_ATGGCT.bam is considered unstranded.
  6: In FUN(X[[i]], ...) :
    Bam file: 2baec164bab29e_ATGGCT.bam Strandedness could not be determined using 18462 regions spanning 1280337 bp on either strand at a 90% cutoff; 74.26 percent appear to be stranded.
  7: In FUN(X[[i]], ...) :
    Bam file: 2baec175d93bd9_ACACTG.bam is considered unstranded.
  8: In FUN(X[[i]], ...) :
    Bam file: 2baec175d93bd9_ACACTG.bam Strandedness could not be determined using 18615 regions spanning 1300192 bp on either strand at a 90% cutoff; 73.67 percent appear to be stranded.
  9: In simpleRNASeq(bamFiles = bamFiles, param = param, verbose = FALSE) :
    As of version 2.15.5, easyRNASeq assumes that, if the data is strand specific, the sequencing was done using a protocol such as the Illumina TruSeq, where the reverse strand is quantified - i.e. the strandProtocol argument of the BamParam class defaults to &#x27;reverse&#x27;.
  Execution halted
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘simpleRNASeq.Rmd’ using ‘UTF-8’... OK
  ‘easyRNASeq.Rnw’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 3 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/easyRNASeq.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">easyRNASeq.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL easyRNASeq
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘easyRNASeq’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
No methods found in package ‘BiocGenerics’ for request: ‘clusterApply’ when loading ‘easyRNASeq’
Creating a generic function for ‘basename’ from package ‘base’ in package ‘easyRNASeq’
Creating a generic function for ‘file.exists’ from package ‘base’ in package ‘easyRNASeq’
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
No methods found in package ‘BiocGenerics’ for request: ‘clusterApply’ when loading ‘easyRNASeq’
** testing if installed package can be loaded from final location
No methods found in package ‘BiocGenerics’ for request: ‘clusterApply’ when loading ‘easyRNASeq’
** testing if installed package keeps a record of temporary installation path
* DONE (easyRNASeq)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">easyRNASeq.Rcheck/tests/runTests.Rout.fail</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; # get the example data
&gt; library(easyRNASeq)
No methods found in package &#x27;BiocGenerics&#x27; for request: &#x27;clusterApply&#x27; when loading &#x27;easyRNASeq&#x27;
&gt; tutorialData()
[1] &quot;/home/biocbuild/.cache/easyRNASeq&quot;
&gt; 
&gt; # set the env.var
&gt; #TUTORIAL.DATA &lt;- get(&quot;TUTORIAL.DATA&quot;,envir=as.environment(&quot;package:easyRNASeq&quot;))
&gt; 
&gt; # run the tests
&gt; BiocGenerics:::testPackage(&quot;easyRNASeq&quot;)
Loading required package: GenomicRanges
Loading required package: stats4
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following object is masked from &#x27;package:easyRNASeq&#x27;:

    basename

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
Timing stopped at: 0.07 0.007 13.18
Error in h(simpleError(msg, call)) : 
  error in evaluating the argument &#x27;table&#x27; in selecting a method for function &#x27;%in%&#x27;: Timeout was reached: [www.ensembl.org:443] Operation timed out after 10000 milliseconds with 0 bytes received
No validation performed at that stage
Validated a datasource of type biomaRt
No validation performed at that stage
Validated a datasource of type rda
Read 1000 records
Validated a datasource of type gtf
Read 999 records
Validated a datasource of type gff3


RUNIT TEST PROTOCOL -- Sat Feb 11 14:26:01 2023 
*********************************************** 
Number of test functions: 20 
Number of errors: 1 
Number of failures: 0 

 
1 Test Suite : 
easyRNASeq RUnit Tests - 20 test functions, 1 error, 0 failures
ERROR in test_getAnnotation_BiomaRt: Error in h(simpleError(msg, call)) : 
  error in evaluating the argument &#x27;table&#x27; in selecting a method for function &#x27;%in%&#x27;: Timeout was reached: [www.ensembl.org:443] Operation timed out after 10000 milliseconds with 0 bytes received

Test files with failing tests

   test_annotations.R 
     test_getAnnotation_BiomaRt 


Error in BiocGenerics:::testPackage(&quot;easyRNASeq&quot;) : 
  unit tests failed for package easyRNASeq
In addition: Warning messages:
1: In FUN(X[[i]], ...) :
  Bam file: 2baec124f7afe7_ACTAGC.bam is considered unstranded.
2: In FUN(X[[i]], ...) :
  Bam file: 2baec124f7afe7_ACTAGC.bam Strandedness could not be determined using 14772 regions spanning 1023473 bp on either strand at a 90% cutoff; 76.6 percent appear to be stranded.
3: In FUN(X[[i]], ...) :
  Bam file: 2baec1607f22_TTGCGA.bam is considered unstranded.
4: In FUN(X[[i]], ...) :
  Bam file: 2baec1607f22_TTGCGA.bam Strandedness could not be determined using 19659 regions spanning 1381886 bp on either strand at a 90% cutoff; 73.37 percent appear to be stranded.
5: In FUN(X[[i]], ...) :
  Bam file: 2baec164bab29e_ATGGCT.bam is considered unstranded.
6: In FUN(X[[i]], ...) :
  Bam file: 2baec164bab29e_ATGGCT.bam Strandedness could not be determined using 18462 regions spanning 1280337 bp on either strand at a 90% cutoff; 74.26 percent appear to be stranded.
7: In FUN(X[[i]], ...) :
  Bam file: 2baec175d93bd9_ACACTG.bam is considered unstranded.
8: In FUN(X[[i]], ...) :
  Bam file: 2baec175d93bd9_ACACTG.bam Strandedness could not be determined using 18615 regions spanning 1300192 bp on either strand at a 90% cutoff; 73.67 percent appear to be stranded.
9: In simpleRNASeq(bamFiles = bamFiles, param = param, verbose = FALSE) :
  As of version 2.15.5, easyRNASeq assumes that, if the data is strand specific, the sequencing was done using a protocol such as the Illumina TruSeq, where the reverse strand is quantified - i.e. the strandProtocol argument of the BamParam class defaults to &#x27;reverse&#x27;.
Execution halted
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">easyRNASeq.Rcheck/easyRNASeq-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>BiocFileCache-methods</TD><TD style="text-align: right;"> 9.639</TD><TD style="text-align: right;"> 0.895</TD><TD style="text-align: right;">43.289</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GenomicRanges-methods</TD><TD style="text-align: right;">0.413</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.449</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>IRanges-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Rsamtools-methods</TD><TD style="text-align: right;">3.947</TD><TD style="text-align: right;">0.324</TD><TD style="text-align: right;">6.784</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ShortRead-methods</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-AnnotParam-accessors</TD><TD style="text-align: right;">0.199</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.227</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-AnnotParam-class</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-AnnotParam</TD><TD style="text-align: right;">0.200</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.237</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-BamFileList</TD><TD style="text-align: right;">4.539</TD><TD style="text-align: right;">0.377</TD><TD style="text-align: right;">7.092</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-BamParam-accessors</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-BamParam-class</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-BamParam</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-RnaSeqParam-accessors</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-RnaSeqParam-class</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-RnaSeqParam</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-accessors</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-annotation-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-class</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-correction-methods</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-coverage-methods</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-easyRNASeq</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-island-methods</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-package</TD><TD style="text-align: right;">54.584</TD><TD style="text-align: right;"> 2.683</TD><TD style="text-align: right;">57.767</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-simpleRNASeq</TD><TD style="text-align: right;">95.356</TD><TD style="text-align: right;"> 2.011</TD><TD style="text-align: right;">98.357</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-summarization-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>easyRNASeq-synthetic-transcripts</TD><TD style="text-align: right;">41.460</TD><TD style="text-align: right;"> 0.488</TD><TD style="text-align: right;">42.352</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>edgeR-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genomeIntervals-methods</TD><TD style="text-align: right;">1.816</TD><TD style="text-align: right;">0.180</TD><TD style="text-align: right;">2.522</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>parallel-methods</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
