<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for tRanslatome on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><B>[<A href="../index.html#T">T</A>]</B></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-08 01:15:33 -0000 (Wed, 08 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4164</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for tRanslatome on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the tRanslatome package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/tRanslatome.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>2064</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">tRanslatome</A>&nbsp;1.37.6</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/tRanslatome">landing page</A>)</SPAN><BR>Toma Tebaldi <toma.tebaldi@unitn.it>, Erik Dassi<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-06&nbsp;00:12:45&nbsp;-0000&nbsp;(Mon,&nbsp;06&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/tRanslatome</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">59b77eb</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-02-03&nbsp;16:28:34&nbsp;-0000&nbsp;(Fri,&nbsp;03&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: tRanslatome</TD></TR>
<TR><TD><B>Version</B>: 1.37.6</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:tRanslatome.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings tRanslatome_1.37.6.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-07 18:01:20 -0000 (Tue, 07 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-07 18:08:52 -0000 (Tue, 07 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 451.9 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: tRanslatome.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:tRanslatome.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings tRanslatome_1.37.6.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/tRanslatome.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘tRanslatome/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘tRanslatome’ version ‘1.37.6’
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Depends: includes the non-default packages:
  &#x27;limma&#x27;, &#x27;anota&#x27;, &#x27;DESeq2&#x27;, &#x27;edgeR&#x27;, &#x27;RankProd&#x27;, &#x27;topGO&#x27;,
  &#x27;org.Hs.eg.db&#x27;, &#x27;GOSemSim&#x27;, &#x27;Heatplus&#x27;, &#x27;gplots&#x27;, &#x27;plotrix&#x27;,
  &#x27;Biobase&#x27;
Adding so many packages to the search path is excessive and importing
selectively is preferable.
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘tRanslatome’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
methodLimma: no visible global function definition for ‘lmFit’
methodTTest: no visible global function definition for ‘calcTStatFast’
GOEnrichment,DEGs: no visible global function definition for ‘toTable’
Undefined global functions or variables:
  calcTStatFast lmFit toTable
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
               user system elapsed
GOEnrichment 15.329  0.067  15.429
GOComparison 14.855  0.464  15.332
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘tRanslatome_package.Rnw’ using ‘UTF-8’... OK
 OK
* checking re-building of vignette outputs ... NOTE
Error(s) in re-building vignettes:
--- re-building ‘tRanslatome_package.Rnw’ using Sweave
Loading required package: limma
Loading required package: anota
Loading required package: qvalue
Loading required package: DESeq2
Loading required package: S4Vectors
Loading required package: stats4
Loading required package: BiocGenerics

Attaching package: ‘BiocGenerics’

The following object is masked from ‘package:limma’:

    plotMA

The following objects are masked from ‘package:stats’:

    IQR, mad, sd, var, xtabs

The following objects are masked from ‘package:base’:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min


Attaching package: ‘S4Vectors’

The following objects are masked from ‘package:base’:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomicRanges
Loading required package: GenomeInfoDb
Loading required package: SummarizedExperiment
Loading required package: MatrixGenerics
Loading required package: matrixStats

Attaching package: ‘MatrixGenerics’

The following objects are masked from ‘package:matrixStats’:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars

Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


Attaching package: ‘Biobase’

The following object is masked from ‘package:MatrixGenerics’:

    rowMedians

The following objects are masked from ‘package:matrixStats’:

    anyMissing, rowMedians

Loading required package: edgeR
Loading required package: RankProd
Loading required package: Rmpfr
Loading required package: gmp

Attaching package: ‘gmp’

The following objects are masked from ‘package:base’:

    %*%, apply, crossprod, matrix, tcrossprod

C code of R package &#x27;Rmpfr&#x27;: GMP using 64 bits per limb


Attaching package: ‘Rmpfr’

The following object is masked from ‘package:gmp’:

    outer

The following objects are masked from ‘package:SummarizedExperiment’:

    cbind, rbind

The following objects are masked from ‘package:IRanges’:

    cbind, pmax, pmin, rbind

The following objects are masked from ‘package:S4Vectors’:

    cbind, rbind

The following objects are masked from ‘package:BiocGenerics’:

    cbind, pmax, pmin, rbind

The following objects are masked from ‘package:stats’:

    dbinom, dgamma, dnbinom, dnorm, dpois, dt, pnorm

The following objects are masked from ‘package:base’:

    cbind, pmax, pmin, rbind

Loading required package: topGO
Loading required package: graph
Loading required package: GO.db
Loading required package: AnnotationDbi

Loading required package: SparseM

Attaching package: ‘SparseM’

The following object is masked from ‘package:base’:

    backsolve


groupGOTerms: 	GOBPTerm, GOMFTerm, GOCCTerm environments built.

Attaching package: ‘topGO’

The following object is masked from ‘package:IRanges’:

    members

Loading required package: org.Hs.eg.db

Loading required package: GOSemSim
GOSemSim v2.25.0  For help: https://yulab-smu.top/biomedical-knowledge-mining-book/

If you use GOSemSim in published research, please cite:
[36m-[39m Guangchuang Yu. Gene Ontology Semantic Similarity Analysis Using GOSemSim. In: Kidder B. (eds) Stem Cell Transcriptional Networks. Methods in Molecular Biology, 2020, 2117:207-215. Humana, New York, NY. doi:10.1007/978-1-0716-0301-7_11
[36m-[39m Guangchuang Yu, Fei Li, Yide Qin, Xiaochen Bo, Yibo Wu, Shengqi Wang. GOSemSim: an R package for measuring semantic similarity among GO terms and gene products Bioinformatics 2010, 26(7):976-978. doi:10.1093/bioinformatics/btq064


Loading required package: Heatplus
Loading required package: gplots

Attaching package: ‘gplots’

The following object is masked from ‘package:IRanges’:

    space

The following object is masked from ‘package:S4Vectors’:

    space

The following object is masked from ‘package:stats’:

    lowess

Loading required package: plotrix

Attaching package: ‘plotrix’

The following object is masked from ‘package:gplots’:

    plotCI


Building most specific GOs .....
	( 1836 GO terms found. )

Build GO DAG topology ..........
	( 2024 GO terms and 3404 relations. )

Annotating nodes ...............
	( 19869 genes annotated to the GO terms. )

			 -- Elim Algorithm -- 

		 the algorithm is scoring 178 nontrivial nodes
		 parameters: 
			 test statistic: fisher
			 cutOff: 0.01

	 Level 11:	3 nodes to be scored	(0 eliminated genes)

	 Level 10:	7 nodes to be scored	(0 eliminated genes)

	 Level 9:	13 nodes to be scored	(0 eliminated genes)

	 Level 8:	20 nodes to be scored	(23 eliminated genes)

	 Level 7:	30 nodes to be scored	(23 eliminated genes)

	 Level 6:	34 nodes to be scored	(3034 eliminated genes)

	 Level 5:	27 nodes to be scored	(3336 eliminated genes)

	 Level 4:	21 nodes to be scored	(7623 eliminated genes)

	 Level 3:	20 nodes to be scored	(7623 eliminated genes)

	 Level 2:	2 nodes to be scored	(7623 eliminated genes)

	 Level 1:	1 nodes to be scored	(7623 eliminated genes)

Building most specific GOs .....
	( 1836 GO terms found. )

Build GO DAG topology ..........
	( 2024 GO terms and 3404 relations. )

Annotating nodes ...............
	( 19869 genes annotated to the GO terms. )

			 -- Elim Algorithm -- 

		 the algorithm is scoring 181 nontrivial nodes
		 parameters: 
			 test statistic: fisher
			 cutOff: 0.01

	 Level 12:	1 nodes to be scored	(0 eliminated genes)

	 Level 11:	6 nodes to be scored	(0 eliminated genes)

	 Level 10:	11 nodes to be scored	(62 eliminated genes)

	 Level 9:	12 nodes to be scored	(62 eliminated genes)

	 Level 8:	23 nodes to be scored	(474 eliminated genes)

	 Level 7:	28 nodes to be scored	(523 eliminated genes)

	 Level 6:	29 nodes to be scored	(3040 eliminated genes)

	 Level 5:	25 nodes to be scored	(3295 eliminated genes)

	 Level 4:	23 nodes to be scored	(7500 eliminated genes)

	 Level 3:	20 nodes to be scored	(7500 eliminated genes)

	 Level 2:	2 nodes to be scored	(7500 eliminated genes)

	 Level 1:	1 nodes to be scored	(7500 eliminated genes)
preparing gene to GO mapping data...
Killed

* checking PDF version of manual ... OK
* DONE

Status: 3 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/tRanslatome.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">tRanslatome.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL tRanslatome
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘tRanslatome’ ...
** using staged installation
** R
** data
** byte-compile and prepare package for lazy loading

groupGOTerms: 	GOBPTerm, GOMFTerm, GOCCTerm environments built.
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location

groupGOTerms: 	GOBPTerm, GOMFTerm, GOCCTerm environments built.
** testing if installed package can be loaded from final location

groupGOTerms: 	GOBPTerm, GOMFTerm, GOCCTerm environments built.
** testing if installed package keeps a record of temporary installation path
* DONE (tRanslatome)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<HR>
<H3>Example timings</H3>
<P><SPAN class="filename">tRanslatome.Rcheck/tRanslatome-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>CVplot</TD><TD style="text-align: right;">0.091</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.105</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DEGs</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DEGs.table</TD><TD style="text-align: right;">0.067</TD><TD style="text-align: right;">0.068</TD><TD style="text-align: right;">0.135</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EnrichedSets</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>FC.threshold</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GOComparison</TD><TD style="text-align: right;">14.855</TD><TD style="text-align: right;"> 0.464</TD><TD style="text-align: right;">15.332</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GOEnrichment</TD><TD style="text-align: right;">15.329</TD><TD style="text-align: right;"> 0.067</TD><TD style="text-align: right;">15.429</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GOsets</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GOsims</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Heatmap</TD><TD style="text-align: right;">0.117</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.121</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Histogram</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.053</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>IdentityPlot</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>MAplot</TD><TD style="text-align: right;">0.061</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.074</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Radar</TD><TD style="text-align: right;">0.100</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.099</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>RegulatoryEnrichment</TD><TD style="text-align: right;">1.758</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">1.813</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SDplot</TD><TD style="text-align: right;">0.069</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.073</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Scatterplot</TD><TD style="text-align: right;">0.107</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.111</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SimilarityPlot</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TranslatomeDataset</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>average.similarity.scores</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>computeDEGs</TD><TD style="text-align: right;">0.347</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.358</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>enriched.table</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getConditionA</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getConditionB</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getConditionC</TD><TD style="text-align: right;">0.041</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getConditionD</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getConditionLabels</TD><TD style="text-align: right;">0.042</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getDEGs</TD><TD style="text-align: right;">0.041</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getDEGsMethod</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getDataType</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getExprMatrix</TD><TD style="text-align: right;">0.065</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.090</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getLevelLabels</TD><TD style="text-align: right;">0.042</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>identity.matrix</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>label.condition</TD><TD style="text-align: right;">0.037</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>label.level.DEGs</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>label.level.enriched</TD><TD style="text-align: right;">0.039</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.049</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>newTranslatomeDataset</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>significance.threshold</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>similarity.matrix</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.050</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>tRanslatomeSampleData</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
