<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for DMRcaller on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><B>[<A href="..#D">D</A>]</B></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-10 10:04:30 -0000 (Wed, 10 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-03-12&nbsp;r83975)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">6211</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for DMRcaller on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the DMRcaller package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/DMRcaller.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>562</B>/2194</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">DMRcaller</A>&nbsp;1.33.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/DMRcaller">landing page</A>)</SPAN><BR>Nicolae Radu Zabet<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-08&nbsp;19:11:19&nbsp;-0000&nbsp;(Mon,&nbsp;08&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/DMRcaller</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">3d695cb</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:38:50&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: DMRcaller</TD></TR>
<TR><TD><B>Version</B>: 1.33.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:DMRcaller.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings DMRcaller_1.33.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-09 14:41:11 -0000 (Tue, 09 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-09 14:47:33 -0000 (Tue, 09 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 381.5 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: DMRcaller.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:DMRcaller.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings DMRcaller_1.33.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/DMRcaller.Rcheck’
* using R Under development (unstable) (2023-03-12 r83975)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘DMRcaller/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘DMRcaller’ version ‘1.33.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘DMRcaller’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
extractGC: no visible global function definition for ‘getSeq’
extractGC: no visible global function definition for ‘seqlevels’
Undefined global functions or variables:
  getSeq seqlevels
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                       user system elapsed
mergeDMRsIteratively 13.697  0.024  13.722
computeDMRs          12.587  0.047  12.636
readBismarkPool       4.953  0.072   5.028
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘DMRcaller.Rnw’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 NOTE
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/DMRcaller.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">DMRcaller.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL DMRcaller
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library’
* installing *source* package ‘DMRcaller’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (DMRcaller)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">DMRcaller.Rcheck/tests/runTests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; BiocGenerics:::testPackage(&quot;DMRcaller&quot;)

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min


Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Parameters checking ...
Extract methylation in the corresponding context 
Computing DMRs 
Merge adjacent DMRs
Merge DMRs iteratively
Filter DMRs 
Parameters checking ...
Extract methylation in the corresponding context 
Computing DMRs at  Chr1:1..4976 
Calculating interpolations...
Identifying DMRs...
Analysed reads inside DMRs
Merge DMRs iteratively
Filter DMRs 
Parameters checking ...
Extract methylation in the corresponding context 
Computing DMRs at  Chr1:1..4976 
Selecting data...
Identifying DMRs...
Validating objects 
Finding overlaps 
Joining objects 
Parameters checking ...
Extract methylation in the corresponding context 
Computing DMRs at  Chr1:1..4976 
Count inside each bin...
Filter the bins...
Identifying DMRs...
Merge adjacent DMRs
Merge DMRs iteratively
Filter DMRs 
Parameters checking ...
Merge DMRs iteratively ...


RUNIT TEST PROTOCOL -- Tue May  9 14:46:06 2023 
*********************************************** 
Number of test functions: 6 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
DMRcaller RUnit Tests - 6 test functions, 0 errors, 0 failures
Number of test functions: 6 
Number of errors: 0 
Number of failures: 0 
Warning message:
In fisher.test(matrix(unlist(x), nrow = 2, byrow = TRUE), alternative = alternative) :
  &#x27;x&#x27; has been rounded to integer: Mean relative difference: 0.002857143
&gt; 
&gt; proc.time()
   user  system elapsed 
 131.04    1.98  133.01 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">DMRcaller.Rcheck/DMRcaller-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>DMRcaller</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>analyseReadsInsideRegionsForCondition</TD><TD style="text-align: right;">1.139</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">1.183</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>computeDMRs</TD><TD style="text-align: right;">12.587</TD><TD style="text-align: right;"> 0.047</TD><TD style="text-align: right;">12.636</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>computeDMRsReplicates</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>computeMethylationDataCoverage</TD><TD style="text-align: right;">0.880</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.904</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>computeMethylationDataSpatialCorrelation</TD><TD style="text-align: right;">0.628</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.644</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>computeMethylationProfile</TD><TD style="text-align: right;">1.640</TD><TD style="text-align: right;">0.031</TD><TD style="text-align: right;">1.672</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>computeOverlapProfile</TD><TD style="text-align: right;">2.008</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">2.029</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>extractGC</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filterDMRs</TD><TD style="text-align: right;">2.633</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">2.653</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getWholeChromosomes</TD><TD style="text-align: right;">0.584</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.592</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>joinReplicates</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mergeDMRsIteratively</TD><TD style="text-align: right;">13.697</TD><TD style="text-align: right;"> 0.024</TD><TD style="text-align: right;">13.722</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotLocalMethylationProfile</TD><TD style="text-align: right;">1.901</TD><TD style="text-align: right;">0.037</TD><TD style="text-align: right;">1.937</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotMethylationDataCoverage</TD><TD style="text-align: right;">1.396</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">1.421</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotMethylationDataSpatialCorrelation</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotMethylationProfile</TD><TD style="text-align: right;">1.601</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">1.618</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotMethylationProfileFromData</TD><TD style="text-align: right;">2.988</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">3.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotOverlapProfile</TD><TD style="text-align: right;">2.319</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">2.332</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>poolMethylationDatasets</TD><TD style="text-align: right;">1.363</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">1.407</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>poolTwoMethylationDatasets</TD><TD style="text-align: right;">1.342</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">1.366</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readBismark</TD><TD style="text-align: right;">2.091</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">2.136</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readBismarkPool</TD><TD style="text-align: right;">4.953</TD><TD style="text-align: right;">0.072</TD><TD style="text-align: right;">5.028</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>saveBismark</TD><TD style="text-align: right;">1.049</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">1.088</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
