<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for DECIPHER on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><B>[<A href="../index.html#D">D</A>]</B></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-13 06:49:23 -0000 (Mon, 13 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4222</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for DECIPHER on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the DECIPHER package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/DECIPHER.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard timeout">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>479</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">DECIPHER</A>&nbsp;2.27.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/DECIPHER">landing page</A>)</SPAN><BR>Erik Wright<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-11&nbsp;01:58:29&nbsp;-0000&nbsp;(Sat,&nbsp;11&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/DECIPHER</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">ffdccd0</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:06:21&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph TIMEOUT">&nbsp;&nbsp;TIMEOUT&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: DECIPHER</TD></TR>
<TR><TD><B>Version</B>: 2.27.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:DECIPHER.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings DECIPHER_2.27.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-11 13:32:36 -0000 (Sat, 11 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-11 14:12:37 -0000 (Sat, 11 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 2400.6 seconds</TD></TR>
<TR><TD><B>RetCode</B>: None</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph TIMEOUT">&nbsp;&nbsp;TIMEOUT&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: DECIPHER.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:DECIPHER.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings DECIPHER_2.27.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/DECIPHER.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘DECIPHER/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘DECIPHER’ version ‘2.27.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘DECIPHER’ can be installed ... OK
* used C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... NOTE
  installed size is 12.7Mb
  sub-directories of 1Mb or more:
    R         1.4Mb
    data      7.3Mb
    extdata   2.3Mb
    libs      1.3Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
DesignSignatures: no visible binding for global variable ‘deltaHrules’
FindGenes: no visible binding for global variable ‘deltaHrulesRNA’
FindNonCoding: no visible binding for global variable ‘deltaHrulesRNA’
LearnNonCoding: no visible binding for global variable ‘deltaHrulesRNA’
PredictDBN: no visible binding for global variable ‘deltaHrulesRNA’
TreeLine: multiple local function definitions for ‘.minimize’ with
  different formal arguments
Undefined global functions or variables:
  deltaHrules deltaHrulesRNA
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... OK
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                      user system elapsed
PredictDBN         147.197  0.440 147.645
MapCharacters      107.377  0.852 108.233
FindNonCoding       96.862  0.536  97.401
LearnNonCoding      73.709  3.264  76.977
BrowseSeqs          55.468  0.348  55.866
ExtractGenes        50.497  0.380  50.880
WriteGenes          49.767  0.096  49.864
Genes-class         47.923  0.160  48.085
FindGenes           46.885  0.124  47.013
AlignSeqs           26.899  0.588  27.489
CorrectFrameshifts  18.307  0.096  18.405
StaggerAlignment    16.547  0.196  16.743
LearnTaxa           15.176  0.672  15.849
AlignTranslation    14.414  0.140  14.562
Taxa-class          14.374  0.040  15.998
IdTaxa              12.953  0.024  12.977
DetectRepeats       12.576  0.058  12.635
TreeLine            10.436  0.096  11.021
Clusterize           7.578  0.076   7.655
ScoreAlignment       6.401  0.040   6.441
Array2Matrix         5.482  0.060   5.542
DesignArray          5.364  0.004   5.368
TileSeqs             5.123  0.024   5.147
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘ArtOfAlignmentInR.Rnw’... OK
  ‘ClassifySequences.Rnw’... OK
  ‘ClusteringSequences.Rnw’... OK
  ‘DECIPHERing.Rnw’... OK
  ‘DesignMicroarray.Rnw’... OK
  ‘DesignPrimers.Rnw’... OK
  ‘DesignProbes.Rnw’... OK
  ‘DesignSignatures.Rnw’... OK
  ‘FindChimeras.Rnw’... OK
  ‘FindingGenes.Rnw’... OK
  ‘FindingNonCodingRNAs.Rnw’... OK
  ‘GrowingTrees.Rnw’... OK
  ‘RepeatRepeat.Rnw’... OK
 OK
* checking re-building of vignette outputs ...</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">DECIPHER.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL DECIPHER
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘DECIPHER’ ...
** using staged installation
** libs
using C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c AlignProfiles.c -o AlignProfiles.o
AlignProfiles.c: In function ‘alignProfiles._omp_fn.0’:
AlignProfiles.c:424:37: warning: ‘lGp’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  424 |                                 lGp *= tot;
      |                                 ~~~~^~~~~~
AlignProfiles.c:61:46: note: ‘lGp’ was declared here
   61 |         double *pprofile, *sprofile, gp, gs, lGp, lGs, S, M, GP, GS, temp, avgM = 0;
      |                                              ^~~
AlignProfiles.c:426:37: warning: ‘lGs’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  426 |                                 lGs *= tot;
      |                                 ~~~~^~~~~~
AlignProfiles.c:61:51: note: ‘lGs’ was declared here
   61 |         double *pprofile, *sprofile, gp, gs, lGp, lGs, S, M, GP, GS, temp, avgM = 0;
      |                                                   ^~~
AlignProfiles.c: In function ‘alignProfilesAA._omp_fn.0’:
AlignProfiles.c:1266:37: warning: ‘lGp’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1266 |                                 lGp *= tot;
      |                                 ~~~~^~~~~~
AlignProfiles.c:808:46: note: ‘lGp’ was declared here
  808 |         double *pprofile, *sprofile, gp, gs, lGp, lGs, M, GP, GS, R, temp, avgM = 0;
      |                                              ^~~
AlignProfiles.c:1268:37: warning: ‘lGs’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1268 |                                 lGs *= tot;
      |                                 ~~~~^~~~~~
AlignProfiles.c:808:51: note: ‘lGs’ was declared here
  808 |         double *pprofile, *sprofile, gp, gs, lGp, lGs, M, GP, GS, R, temp, avgM = 0;
      |                                                   ^~~
AlignProfiles.c: In function ‘alignProfiles’:
AlignProfiles.c:378:25: warning: ‘subM’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  378 |                 #pragma omp parallel for private(i,j,gp,gs,S,M,GP,GS,tot,lGp,lGs,temp) reduction(+:totM,avgM) num_threads(nthreads)
      |                         ^~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c AssignIndels.c -o AssignIndels.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c Biostrings_stubs.c -o Biostrings_stubs.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c CalculateDeltaG.c -o CalculateDeltaG.o
CalculateDeltaG.c: In function ‘calculateHairpinDeltaG’:
CalculateDeltaG.c:463:49: warning: ‘s2’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  463 |                         if ((!((s1 == 4) &amp;&amp; (s2 == 4)) || j &gt;= a[i]) &amp;&amp; count &gt; 3) {
      |                                             ~~~~^~~~~
CalculateDeltaG.c:463:36: warning: ‘s1’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  463 |                         if ((!((s1 == 4) &amp;&amp; (s2 == 4)) || j &gt;= a[i]) &amp;&amp; count &gt; 3) {
      |                                ~~~~^~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c CalculateFISH.c -o CalculateFISH.o
CalculateFISH.c: In function ‘calculateFISH’:
CalculateFISH.c:25:30: warning: missing braces around initializer [-Wmissing-braces]
   25 |         double dH_DR[4][4] = {
      |                              ^
   26 |                 -11.5, -7.8, -7, -8.3,
      |                 {                    }
   27 |                 -10.4, -12.8, -16.3, -9.1,
      |                 {                        }
   28 |                 -8.6, -8, -9.3, -5.9,
      |                 {                   }
   29 |                 -7.8, -5.5, -9, -7.8
      |                 {
   30 |         };
      |         }
CalculateFISH.c:31:30: warning: missing braces around initializer [-Wmissing-braces]
   31 |         double dS_DR[4][4] = {
      |                              ^
   32 |                 -36.4, -21.6, -19.7, -23.9,
      |                 {                         }
   33 |                 -28.4, -31.9, -47.1, -23.5,
      |                 {                         }
   34 |                 -22.9, -17.1, -23.2, -12.3,
      |                 {                         }
   35 |                 -23.2, -13.5, -26.1, -21.9
      |                 {
   36 |         };
      |         }
CalculateFISH.c:37:30: warning: missing braces around initializer [-Wmissing-braces]
   37 |         double dH_DD[4][4] = {
      |                              ^
   38 |                 -7.9, -8.4, -7.8, -7.2,
      |                 {                     }
   39 |                 -8.5, -8, -10.6, -7.8,
      |                 {                    }
   40 |                 -8.2, -9.8, -8, -8.4,
      |                 {                   }
   41 |                 -7.2, -8.2, -8.5, -7.9
      |                 {
   42 |         };
      |         }
CalculateFISH.c:43:30: warning: missing braces around initializer [-Wmissing-braces]
   43 |         double dS_DD[4][4] = {
      |                              ^
   44 |                 -22.2, -22.4, -21, -20.4,
      |                 {                       }
   45 |                 -22.7, -19.9, -27.2, -21,
      |                 {                       }
   46 |                 -22.2, -24.4, -19.9, -22.4,
      |                 {                         }
   47 |                 -21.3, -22.2, -22.7, -22.2
      |                 {
   48 |         };
      |         }
CalculateFISH.c:49:30: warning: missing braces around initializer [-Wmissing-braces]
   49 |         double dH_RR[4][4] = {
      |                              ^
   50 |                 -6.6, -10.17, -7.65, -5.76,
      |                 {                         }
   51 |                 -10.56, -12.21, -7.95, -7.65,
      |                 {                           }
   52 |                 -13.37, -14.21, -12.21, -10.17,
      |                 {                             }
   53 |                 -8.11, -13.37, -10.56, -6.6
      |                 {
   54 |         };
      |         }
CalculateFISH.c:55:30: warning: missing braces around initializer [-Wmissing-braces]
   55 |         double dS_RR[4][4] = {
      |                              ^
   56 |                 -18.38, -26.03, -19.18, -15.67,
      |                 {                             }
   57 |                 -28.25, -30.02, -19.18, -19.18,
      |                 {                             }
   58 |                 -35.68, -34.85, -30.02, -26.03,
      |                 {                             }
   59 |                 -22.59, -35.68, -28.25, -18.38
      |                 {
   60 |         };
      |         }
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c ChainSegments.c -o ChainSegments.o
ChainSegments.c: In function ‘chainSegments’:
ChainSegments.c:500:49: warning: ‘upY’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  500 |                         if (minX == minY &amp;&amp; upX == upY) {
      |                                             ~~~~^~~~~~
ChainSegments.c:500:49: warning: ‘upX’ may be used uninitialized in this function [-Wmaybe-uninitialized]
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c Cluster.c -o Cluster.o
Cluster.c: In function ‘cluster._omp_fn.0’:
Cluster.c:403:50: warning: ‘minC’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  403 |                                         minCs[i] = minC;
      |                                         ~~~~~~~~~^~~~~~
Cluster.c:231:57: note: ‘minC’ was declared here
  231 |         int k, dobj, clusterNum, minRow, minCol, index, minC, met;
      |                                                         ^~~~
Cluster.c: In function ‘cluster._omp_fn.1’:
Cluster.c:427:72: warning: ‘minC’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  427 |                                                 minCols[rowIndices[i]] = minC;
      |                                                 ~~~~~~~~~~~~~~~~~~~~~~~^~~~~~
Cluster.c:231:57: note: ‘minC’ was declared here
  231 |         int k, dobj, clusterNum, minRow, minCol, index, minC, met;
      |                                                         ^~~~
Cluster.c: In function ‘cluster’:
Cluster.c:449:66: warning: ‘minC’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  449 |                 rans[1*(length - 1) + k] = *(colNums + colIndices[minCol]); // column merged
      |                                                                  ^
Cluster.c:739:37: warning: ‘nDiv’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  739 |                                 nDiv[i] = nDiv[i + 1];
      |                                     ^
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c ClusterML.c -o ClusterML.o
ClusterML.c:5666:13: warning: ‘L_unknown_5_SIMD’ defined but not used [-Wunused-function]
 5666 | static void L_unknown_5_SIMD(double *__restrict Ls, const int i3, const int i1, const int i2, const double *P1, const double *P2, const double epsilon, const double inv_epsilon, const int root)
      |             ^~~~~~~~~~~~~~~~
ClusterML.c:5064:13: warning: ‘L_unknown_SIMD’ defined but not used [-Wunused-function]
 5064 | static void L_unknown_SIMD(double *__restrict Ls, const int i3, const int i1, const int i2, const double *P1, const double *P2, const double epsilon, const double inv_epsilon, const int root)
      |             ^~~~~~~~~~~~~~
ClusterML.c: In function ‘clusterML’:
ClusterML.c:6905:17: warning: ‘node’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 6905 |         #pragma omp parallel for private(j,k,o,p,y_i,row) num_threads(nthreads)
      |                 ^~~
In file included from /home/biocbuild/bbs-3.17-bioc/R/include/Rdefines.h:38,
                 from ClusterML.c:11:
/home/biocbuild/bbs-3.17-bioc/R/include/R_ext/RS.h:55:25: warning: ‘I’ may be used uninitialized in this function [-Wmaybe-uninitialized]
   55 | #define Free(p)        (R_chk_free( (void *)(p) ), (p) = NULL)
      |                         ^~~~~~~~~~
ClusterML.c:6737:17: note: ‘I’ was declared here
 6737 |         double *I;
      |                 ^
In file included from /home/biocbuild/bbs-3.17-bioc/R/include/Rdefines.h:38,
                 from ClusterML.c:11:
/home/biocbuild/bbs-3.17-bioc/R/include/R_ext/RS.h:55:25: warning: ‘Up’ may be used uninitialized in this function [-Wmaybe-uninitialized]
   55 | #define Free(p)        (R_chk_free( (void *)(p) ), (p) = NULL)
      |                         ^~~~~~~~~~
ClusterML.c:6736:14: note: ‘Up’ was declared here
 6736 |         int *Up;
      |              ^~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c ClusterMP.c -o ClusterMP.o
ClusterMP.c: In function ‘clusterMP._omp_fn.0’:
ClusterMP.c:72:22: warning: ‘m’ may be used uninitialized in this function [-Wmaybe-uninitialized]
   72 |         int i, j, k, m, w;
      |                      ^
ClusterMP.c:125:30: warning: ‘P’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  125 |                         int *P;
      |                              ^
ClusterMP.c: In function ‘clusterMP’:
ClusterMP.c:540:17: warning: ‘Up’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  540 |                 free(Up);
      |                 ^~~~~~~~
ClusterMP.c:113:17: warning: ‘subM’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  113 |         #pragma omp parallel for private(i,j,k,m,w) num_threads(nthreads)
      |                 ^~~
ClusterMP.c:113:17: warning: ‘nodes’ may be used uninitialized in this function [-Wmaybe-uninitialized]
ClusterMP.c:113:17: warning: ‘lengths’ may be used uninitialized in this function [-Wmaybe-uninitialized]
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c CommonGaps.c -o CommonGaps.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c Compositions.c -o Compositions.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c Compression.c -o Compression.o
Compression.c: In function ‘nbit._omp_fn.0’:
Compression.c:976:52: warning: ‘k’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  976 |                                         p[c] = ((k - 1) &gt;&gt; 8) &amp; 0xFF; // length of run
      |                                                 ~~~^~~~
Compression.c:516:19: note: ‘k’ was declared here
  516 |         int i, j, k, pos;
      |                   ^
Compression.c:1010:54: warning: ‘count’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1010 |                                                 count++;
      |                                                 ~~~~~^~
Compression.c:542:43: note: ‘count’ was declared here
  542 |                 unsigned int *dict, word, count, lastHit, currHit, lastPos = 0;
      |                                           ^~~~~
Compression.c:1009:62: warning: ‘word’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1009 |                                                 word = (word &lt;&lt; 8) | (unsigned int)reorder(byte);
      |                                                        ~~~~~~^~~~~
Compression.c:542:37: note: ‘word’ was declared here
  542 |                 unsigned int *dict, word, count, lastHit, currHit, lastPos = 0;
      |                                     ^~~~
Compression.c:1212:56: warning: ‘rev’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1212 |                                                 p[c++] = rev==0 ? 254 : 255;
      |                                                 ~~~~~~~^~~~~~~~~~~~~~~~~~~~
Compression.c:543:41: note: ‘rev’ was declared here
  543 |                 int lastTemp, currTemp, rev, len, len2, thresh = 1;
      |                                         ^~~
Compression.c:556:21: warning: ‘lower’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  556 |                 int lower = 0;
      |                     ^~~~~
Compression.c:1239:71: warning: ‘lastTriplet’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1239 |                                 if (threeBitEnd &gt; threeBitBegin &amp;&amp; (j - lastTriplet) &gt; 20) {
      |                                                                    ~~~^~~~~~~~~~~~~~
Compression.c:629:26: note: ‘lastTriplet’ was declared here
  629 |                 int run, lastTriplet, lastCase;
      |                          ^~~~~~~~~~~
Compression.c:1029:79: warning: ‘dict’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1029 |                                                                 lastHit = dict[(word &gt;&gt; k) &amp; 0xFF];
      |                                                                               ^
Compression.c:542:31: note: ‘dict’ was declared here
  542 |                 unsigned int *dict, word, count, lastHit, currHit, lastPos = 0;
      |                               ^~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c ConsensusSequence.c -o ConsensusSequence.o
ConsensusSequence.c: In function ‘consensusProfile’:
ConsensusSequence.c:1578:17: warning: ‘DBN’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1578 |         double *DBN, *s;
      |                 ^~~
ConsensusSequence.c: In function ‘consensusProfileAA’:
ConsensusSequence.c:455:53: warning: ‘lastPos’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  455 |                                         *(runs + s) += weight;
      |                                                     ^~
ConsensusSequence.c:397:30: note: ‘lastPos’ was declared here
  397 |         int j, temp, length, lastPos, s = -1, value = -1, lastGap = start - 1;
      |                              ^~~~~~~
ConsensusSequence.c:1771:17: warning: ‘HEC’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1771 |         double *HEC, *s;
      |                 ^~~
ConsensusSequence.c: In function ‘colScores’:
ConsensusSequence.c:2040:48: warning: ‘curr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 2040 |                         *(rans + k) += GO*(curr*total);
      |                                           ~~~~~^~~~~~~
ConsensusSequence.c:2040:48: warning: ‘total’ may be used uninitialized in this function [-Wmaybe-uninitialized]
ConsensusSequence.c:1939:27: warning: ‘d’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1939 |         int do_DBN, n, l, d;
      |                           ^
ConsensusSequence.c:1938:17: warning: ‘DBN’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1938 |         double *DBN, *s;
      |                 ^~~
ConsensusSequence.c: In function ‘colScoresAA’:
ConsensusSequence.c:2177:48: warning: ‘curr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 2177 |                         *(rans + k) += GO*(curr*total);
      |                                           ~~~~~^~~~~~~
ConsensusSequence.c:2177:48: warning: ‘total’ may be used uninitialized in this function [-Wmaybe-uninitialized]
ConsensusSequence.c:2076:27: warning: ‘d’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 2076 |         int do_HEC, n, l, d;
      |                           ^
ConsensusSequence.c:2075:17: warning: ‘HEC’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 2075 |         double *HEC, *s;
      |                 ^~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c ConsolidateGaps.c -o ConsolidateGaps.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c DesignProbes.c -o DesignProbes.o
DesignProbes.c: In function ‘designProbes’:
DesignProbes.c:70:27: warning: missing braces around initializer [-Wmissing-braces]
   70 |         double NN[4][4] = {
      |                           ^
   71 |                 -0.816507461,-2.5401714,-1.647430026,-1.184658548
      |                 {
   72 |                 ,-1.854740485,-2.479102613,-2.826248182,-1.647430026
      |                 }{
   73 |                 ,-2.48761723,-4.694133177,-2.479102613,-2.5401714
      |                 }{
   74 |                 ,-0.495794417,-2.48761723,-1.854740485,-0.816507461
      |                 }{
   75 |         };
      |         }
DesignProbes.c:77:27: warning: missing braces around initializer [-Wmissing-braces]
   77 |         double PM[4][4] = {
      |                           ^
   78 |                 -0.141370102,-0.439805276,-0.285236035,-0.205111781
      |                 {
   79 |                 ,-0.321129768,-0.429231826,-0.48933661,-0.285236035
      |                 }{
   80 |                 ,-0.430706047,-0.812742218,-0.429231826,-0.439805276
      |                 }{
   81 |                 ,-0.085841845,-0.430706047,-0.321129768,-0.141370102
      |                 }{
   82 |         };
      |         }
DesignProbes.c:84:34: warning: missing braces around initializer [-Wmissing-braces]
   84 |         double sMM[4][5][5][4] = {
      |                                  ^
   85 |                 0,0,0,0
      |                 {{{
   86 |                 ,1.545032445,1.254355018,1.491691514,1.329138183
      |                 }{
   87 |                 ,1.150635633,0.582415494,1.075877275,1.187937642
      |                 }{
   88 |                 ,1.203555051,1.001540513,0.864287715,0.717125848
      |                 }{
   89 |                 ,0.75,0.65,0.69,0.78
      |                 }{
   90 |                 ,0.630005348,0.18553379,0.730763505,0.709272397
      |                 -
      |                 }},{{
   91 |                 ,0,0,0,0
      |                 }{
   92 |                 ,0.856582783,-0.143236405,0.716721488,0.603652831
      |                 }{
   93 |                 ,0.851622883,0.653168672,0.676545316,1.187937642
      |                 }{
   94 |                 ,0.75,0.65,0.69,0.78
      |                 }{
   95 |                 ,1.231861002,0.746214538,1.087821916,0.989140748
      |                 -
      |                 }},{{
   96 |                 ,1.822113278,1.270687029,1.336192565,1.364584949
      |                 }{
   97 |                 ,0,0,0,0
      |                 }{
   98 |                 ,1.443665704,1.385046493,1.256013166,1.329138183
      |                 }{
   99 |                 ,0.75,0.65,0.69,0.78
      |                 }{
  100 |                 ,1.478009492,0.882097231,1.20450984,1.061002478
      |                 -
      |                 }},{{
  101 |                 ,1.496720812,0.846496194,0.967868114,0.989140748
      |                 }{
  102 |                 ,0.766581547,-0.024857805,0.50754303,0.709272397
      |                 }{
  103 |                 ,0,0,0,0
      |                 }{
  104 |                 ,0.75,0.65,0.69,0.78
      |                 }{
  105 |                 ,0.75,0.65,0.69,0.78
      |                 -
      |                 }},{{
  106 |                 ,0.75,0.65,0.69,0.78
      |                 }{
  107 |                 ,0.75,0.65,0.69,0.78
      |                 }{
  108 |                 ,0.76,0.65,0.69,0.78
      |                 }{
  109 |                 ,0,0,0,0
      |                 }{
  110 |                 ,0,0,0,0
      |                 -
      |                 }}},{{{
  111 |                 ,1.295827995,0.84547091,0.91019099,1.256013166
      |                 }{
  112 |                 ,0.755889609,0.241428373,0.396379912,0.676545316
      |                 }{
  113 |                 ,0.99945386,0.740323132,0.435659206,0.864287715
      |                 }{
  114 |                 ,0.65,0.55,0.48,0.69
      |                 }{
  115 |                 ,0.843147406,0.101248351,0.49063599,0.50754303
      |                 -
      |                 }},{{
  116 |                 ,0,0,0,0
      |                 }{
  117 |                 ,1.0651638,0.249934344,0.699352949,0.716721488
      |                 }{
  118 |                 ,0.871921533,0.59458138,0.396379912,1.075877275
      |                 }{
  119 |                 ,0.65,0.56,0.49,0.69
      |                 }{
  120 |                 ,1.07531714,0.318907854,0.653287717,0.967868114
      |                 -
      |                 }},{{
  121 |                 ,1.099899195,0.730184613,0.661798984,1.336192565
      |                 }{
  122 |                 ,0,0,0,0
      |                 }{
  123 |                 ,1.45897431,1.318532145,0.91019099,1.491691514
      |                 }{
  124 |                 ,0.65,0.56,0.49,0.69
      |                 }{
  125 |                 ,1.242135174,0.894838095,1.108555445,1.20450984
      |                 -
      |                 }},{{
  126 |                 ,0.911428974,0.524430101,0.653287717,1.087821916
      |                 }{
  127 |                 ,0.503209827,0.274849491,0.49063599,0.730763505
      |                 }{
  128 |                 ,0,0,0,0
      |                 }{
  129 |                 ,0.65,0.55,0.48,0.69
      |                 }{
  130 |                 ,0.65,0.55,0.48,0.69
      |                 -
      |                 }},{{
  131 |                 ,0.65,0.56,0.49,0.69
      |                 }{
  132 |                 ,0.65,0.56,0.49,0.69
      |                 }{
  133 |                 ,0.65,0.55,0.48,0.69
      |                 }{
  134 |                 ,0,0,0,0
      |                 }{
  135 |                 ,0,0,0,0
      |                 -
      |                 }}},{{{
  136 |                 ,1.100661785,0.969784756,1.318532145,1.385046493
      |                 }{
  137 |                 ,0.565895968,-0.060347902,0.59458138,0.653168672
      |                 }{
  138 |                 ,0.782168488,0.788161238,0.740323132,1.001540513
      |                 }{
  139 |                 ,0.68,0.46,0.55,0.65
      |                 }{
  140 |                 ,0.468913405,-0.469855984,0.274849491,-0.024857805
      |                 -
      |                 }},{{
  141 |                 ,0,0,0,0
      |                 }{
  142 |                 ,0.258195131,-0.70438632,0.249934344,-0.143236405
      |                 }{
  143 |                 ,0.502914193,-0.060347902,0.241428373,0.582415494
      |                 }{
  144 |                 ,0.68,0.47,0.56,0.65
      |                 }{
  145 |                 ,0.584083861,0.258975454,0.524430101,0.846496194
      |                 -
      |                 }},{{
  146 |                 ,0.968040559,0.797499702,0.730184613,1.270687029
      |                 }{
  147 |                 ,0,0,0,0
      |                 }{
  148 |                 ,1.081040749,0.969784756,0.84547091,1.254355018
      |                 }{
  149 |                 ,0.68,0.47,0.56,0.65
      |                 }{
  150 |                 ,1.048553951,0.728354541,0.894838095,0.882097231
      |                 -
      |                 }},{{
  151 |                 ,0.88611252,0.258975454,0.318907854,0.746214538
      |                 }{
  152 |                 ,0.239520858,-0.469855984,0.101248351,0.18553379
      |                 }{
  153 |                 ,0,0,0,0
      |                 }{
  154 |                 ,0.68,0.46,0.55,0.65
      |                 }{
  155 |                 ,0.68,0.46,0.55,0.65
      |                 -
      |                 }},{{
  156 |                 ,0.68,0.47,0.56,0.65
      |                 }{
  157 |                 ,0.68,0.47,0.56,0.65
      |                 }{
  158 |                 ,0.68,0.46,0.55,0.65
      |                 }{
  159 |                 ,0,0,0,0
      |                 }{
  160 |                 ,0,0,0,0
      |                 -
      |                 }}},{{{
  161 |                 ,1.566899704,1.081040749,1.45897431,1.443665704
      |                 }{
  162 |                 ,0.976725675,0.502914193,0.871921533,0.851622883
      |                 }{
  163 |                 ,1.482046826,0.782168488,0.99945386,1.203555051
      |                 }{
  164 |                 ,0.85,0.68,0.65,0.76
      |                 }{
  165 |                 ,0.798628781,0.239520858,0.503209827,0.766581547
      |                 -
      |                 }},{{
  166 |                 ,0,0,0,0
      |                 }{
  167 |                 ,1.141098246,0.258195131,1.0651638,0.856582783
      |                 }{
  168 |                 ,0.976725675,0.565895968,0.755889609,1.150635633
      |                 }{
  169 |                 ,0.85,0.68,0.65,0.75
      |                 }{
  170 |                 ,1.125403302,0.88611252,0.911428974,1.496720812
      |                 -
      |                 }},{{
  171 |                 ,1.68169282,0.968040559,1.099899195,1.822113278
      |                 }{
  172 |                 ,0,0,0,0
      |                 }{
  173 |                 ,1.566899704,1.100661785,1.295827995,1.545032445
      |                 }{
  174 |                 ,0.85,0.68,0.65,0.75
      |                 }{
  175 |                 ,1.35948517,1.048553951,1.242135174,1.478009492
      |                 -
      |                 }},{{
  176 |                 ,1.125403302,0.584083861,1.07531714,1.231861002
      |                 }{
  177 |                 ,0.798628781,0.468913405,0.843147406,0.630005348
      |                 }{
  178 |                 ,0,0,0,0
      |                 }{
  179 |                 ,0.85,0.68,0.65,0.75
      |                 }{
  180 |                 ,0.85,0.68,0.65,0.75
      |                 -
      |                 }},{{
  181 |                 ,0.85,0.68,0.65,0.75
      |                 }{
  182 |                 ,0.85,0.68,0.65,0.75
      |                 }{
  183 |                 ,0.85,0.68,0.65,0.75
      |                 }{
  184 |                 ,0,0,0,0
      |                 }{
  185 |         };
      |         }}}
DesignProbes.c: In function ‘designProbes._omp_fn.0’:
DesignProbes.c:834:85: warning: ‘lastCycle’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  834 |                                                                 cycles += lastCycle - thisCycle;
      |                                                                           ~~~~~~~~~~^~~~~~~~~~~
DesignProbes.c:267:58: note: ‘lastCycle’ was declared here
  267 |                         int MM, num, thisStart, thisEnd, lastCycle, thisCycle, cycles;
      |                                                          ^~~~~~~~~
DesignProbes.c:834:85: warning: ‘thisCycle’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  834 |                                                                 cycles += lastCycle - thisCycle;
      |                                                                           ~~~~~~~~~~^~~~~~~~~~~
DesignProbes.c:267:69: note: ‘thisCycle’ was declared here
  267 |                         int MM, num, thisStart, thisEnd, lastCycle, thisCycle, cycles;
      |                                                                     ^~~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c Diff.c -o Diff.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c DistanceMatrix.c -o DistanceMatrix.o
DistanceMatrix.c: In function ‘firstSeqsPosEqual’:
DistanceMatrix.c:756:17: warning: this ‘if’ clause does not guard... [-Wmisleading-indentation]
  756 |                 if (!ci)
      |                 ^~
DistanceMatrix.c:759:25: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘if’
  759 |                         while (i &lt; ex) {
      |                         ^~~~~
DistanceMatrix.c:778:17: warning: this ‘if’ clause does not guard... [-Wmisleading-indentation]
  778 |                 if (!cj)
      |                 ^~
DistanceMatrix.c:781:25: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘if’
  781 |                         while (j &lt; ey) {
      |                         ^~~~~
DistanceMatrix.c: In function ‘similarities._omp_fn.0’:
DistanceMatrix.c:995:45: warning: ‘OV’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  995 |                                 (double)(OV + g)/(double)w1 &lt; coverage &amp;&amp;
      |                                         ~~~~^~~~
DistanceMatrix.c:843:45: note: ‘OV’ was declared here
  843 |         int i, j, n, s, p1, p2, t1, t2, ov, OV, g1, g2, g, o, count, *r;
      |                                             ^~
DistanceMatrix.c:1015:82: warning: ‘ov’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1015 |                                                 rans[i] = (double)s/((double)(ov + count + g2));
      |                                                                               ~~~^~~~~~~
DistanceMatrix.c:843:41: note: ‘ov’ was declared here
  843 |         int i, j, n, s, p1, p2, t1, t2, ov, OV, g1, g2, g, o, count, *r;
      |                                         ^~
DistanceMatrix.c:1000:44: warning: ‘o’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1000 |                                         if (o == 1) {
      |                                            ^
DistanceMatrix.c:843:60: note: ‘o’ was declared here
  843 |         int i, j, n, s, p1, p2, t1, t2, ov, OV, g1, g2, g, o, count, *r;
      |                                                            ^
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c EnumerateSequence.c -o EnumerateSequence.o
EnumerateSequence.c: In function ‘pop’:
EnumerateSequence.c:315:15: warning: suggest parentheses around ‘+’ in operand of ‘&amp;’ [-Wparentheses]
  315 |         x = x + (x &gt;&gt; 4) &amp; 0xF0F0F0F;
      |             ~~^~~~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c ExpandAmbiguities.c -o ExpandAmbiguities.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c FindFrameshifts.c -o FindFrameshifts.o
FindFrameshifts.c: In function ‘findFrameshifts’:
FindFrameshifts.c:381:35: warning: ‘K’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  381 |                         } else if (k==2) {
      |                                   ^
FindFrameshifts.c:318:47: warning: ‘J’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  318 |                                 if (C[k*rc + j*r + i] &gt;= 0) {
      |                                              ~^~
FindFrameshifts.c:370:43: warning: ‘I’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  370 |                                         i += B[k*rc + j*r + i];
      |                                         ~~^~~~~~~~~~~~~~~~~~~~
In file included from /home/biocbuild/bbs-3.17-bioc/R/include/Rdefines.h:41,
                 from FindFrameshifts.c:11:
/home/biocbuild/bbs-3.17-bioc/R/include/Rinternals.h:893:33: warning: ‘utilsPackage’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  893 | #define eval                    Rf_eval
      |                                 ^~~~~~~
FindFrameshifts.c:162:31: note: ‘utilsPackage’ was declared here
  162 |         SEXP percentComplete, utilsPackage;
      |                               ^~~~~~~~~~~~
In file included from /home/biocbuild/bbs-3.17-bioc/R/include/Rdefines.h:41,
                 from FindFrameshifts.c:11:
/home/biocbuild/bbs-3.17-bioc/R/include/Rinternals.h:893:33: warning: ‘percentComplete’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  893 | #define eval                    Rf_eval
      |                                 ^~~~~~~
FindFrameshifts.c:162:14: note: ‘percentComplete’ was declared here
  162 |         SEXP percentComplete, utilsPackage;
      |              ^~~~~~~~~~~~~~~
FindFrameshifts.c:468:40: warning: ‘rPercentComplete’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  468 |                                 before = *rPercentComplete;
      |                                 ~~~~~~~^~~~~~~~~~~~~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c GeneFinding.c -o GeneFinding.o
GeneFinding.c: In function ‘scoreCodonModel’:
GeneFinding.c:422:59: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  422 |                                 val += 4*getBaseRC(x_i.ptr[j++]);
      |                                                           ^
GeneFinding.c:432:64: warning: ‘lastVal’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  432 |                                         score += codons[lastVal*64 + val];
      |                                                         ~~~~~~~^~~
GeneFinding.c: In function ‘startCodonModel’:
GeneFinding.c:791:48: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  791 |                         val = getBaseRC(x_i.ptr[j++]);
      |                                                ^
GeneFinding.c: In function ‘scoreStartCodonModel’:
GeneFinding.c:892:48: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  892 |                         val = getBaseRC(x_i.ptr[j++]);
      |                                                ^
GeneFinding.c: In function ‘initialCodonModel’:
GeneFinding.c:973:59: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  973 |                                 val += 4*getBaseRC(x_i.ptr[j++]);
      |                                                           ^
GeneFinding.c: In function ‘scoreInitialCodonModel’:
GeneFinding.c:1059:59: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1059 |                                 val += 4*getBaseRC(x_i.ptr[j++]);
      |                                                           ^
GeneFinding.c: In function ‘terminationCodonModel’:
GeneFinding.c:1131:59: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1131 |                                 val += 4*getBaseRC(x_i.ptr[j++]);
      |                                                           ^
GeneFinding.c: In function ‘scoreTerminationCodonModel’:
GeneFinding.c:1216:59: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1216 |                                 val += 4*getBaseRC(x_i.ptr[j++]);
      |                                                           ^
GeneFinding.c: In function ‘getRegion’:
GeneFinding.c:1286:49: warning: ‘x_i.length’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1286 |                                 (s==0 &amp;&amp; j &gt;= 0 &amp;&amp; j + w &lt;= x_i.length)) {
      |                                 ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~
GeneFinding.c:1289:81: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1289 |                                                 seq[k] = getBaseLetterRC(x_i.ptr[j--]);
      |                                                                                 ^
GeneFinding.c: In function ‘autocorrelationModel’:
GeneFinding.c:1379:59: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1379 |                                 val += 4*getBaseRC(x_i.ptr[j++]);
      |                                                           ^
GeneFinding.c: In function ‘scoreAutocorrelationModel’:
GeneFinding.c:1496:59: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1496 |                                 val += 4*getBaseRC(x_i.ptr[j++]);
      |                                                           ^
GeneFinding.c: In function ‘couplingModel’:
GeneFinding.c:1598:59: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1598 |                                 val += 4*getBaseRC(x_i.ptr[j++]);
      |                                                           ^
GeneFinding.c: In function ‘scoreCouplingModel’:
GeneFinding.c:1712:59: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1712 |                                 val += 4*getBaseRC(x_i.ptr[j++]);
      |                                                           ^
GeneFinding.c: In function ‘nucleotideBiasModel’:
GeneFinding.c:1826:56: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1826 |                                 val = getBaseRC(x_i.ptr[j++]);
      |                                                        ^
GeneFinding.c: In function ‘scoreNucleotideBiasModel’:
GeneFinding.c:1913:56: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1913 |                                 val = getBaseRC(x_i.ptr[j++]);
      |                                                        ^
GeneFinding.c: In function ‘upstreamMotifModel’:
GeneFinding.c:1990:77: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1990 |                                         val += mult[k - 1]*getBaseRC(x_i.ptr[j + k - 1]);
      |                                                                             ^
GeneFinding.c: In function ‘scoreUpstreamMotifModel’:
GeneFinding.c:2090:77: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 2090 |                                         val += mult[k - 1]*getBaseRC(x_i.ptr[j + k - 1]);
      |                                                                             ^
GeneFinding.c: In function ‘scoreRunLengthModel’:
GeneFinding.c:2307:56: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 2307 |                                 val = getBaseRC(x_i.ptr[j++]);
      |                                                        ^
GeneFinding.c: In function ‘stopCodonModel’:
GeneFinding.c:2419:48: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 2419 |                         val = getBaseRC(x_i.ptr[j++]);
      |                                                ^
GeneFinding.c: In function ‘scoreStopCodonModel’:
GeneFinding.c:2520:48: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 2520 |                         val = getBaseRC(x_i.ptr[j++]);
      |                                                ^
GeneFinding.c: In function ‘codonFrequencies’:
GeneFinding.c:2578:59: warning: ‘x_i.ptr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 2578 |                                 val += 4*getBaseRC(x_i.ptr[j++]);
      |                                                           ^
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c GetPools.c -o GetPools.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c Import.c -o Import.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c InformationContent.c -o InformationContent.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c InsertGaps.c -o InsertGaps.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c IntDist.c -o IntDist.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c MeltPolymer.c -o MeltPolymer.o
MeltPolymer.c: In function ‘meltPolymer’:
MeltPolymer.c:79:27: warning: missing braces around initializer [-Wmissing-braces]
   79 |         double dH[4][4] = {
      |                           ^
   80 |                 -7.9,-8.4,-7.8,-7.2
      |                 {
   81 |                 ,-8.5,-8.0,-10.6,-7.8
      |                 }{
   82 |                 ,-8.2,-9.8,-8.0,-8.4
      |                 }{
   83 |                 ,-7.2,-8.2,-8.5,-7.9
      |                 }{
   84 |         };
      |         }
MeltPolymer.c:88:27: warning: missing braces around initializer [-Wmissing-braces]
   88 |         double dS[4][4] = {
      |                           ^
   89 |                 -22.2,-22.4,-21.0,-20.4
      |                 {
   90 |                 ,-22.7,-19.9,-27.2,-21.0
      |                 }{
   91 |                 ,-22.2,-24.4,-19.9,-22.4
      |                 }{
   92 |                 ,-21.3,-22.2,-22.7,-22.2
      |                 }{
   93 |         };
      |         }
MeltPolymer.c:358:59: warning: ‘rans’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  358 |                                         *(rans + k + l*s) += 1;
      |                                                           ^~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c MovingAverage.c -o MovingAverage.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c MultiMatch.c -o MultiMatch.o
MultiMatch.c: In function ‘intMatchSelfOnce’:
MultiMatch.c:871:28: warning: variable ‘start’ set but not used [-Wunused-but-set-variable]
  871 |         int i, j, k, temp, start = 0;
      |                            ^~~~~
MultiMatch.c: In function ‘matchOverlap._omp_fn.0’:
MultiMatch.c:1033:45: warning: ‘one’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1033 |                                         two != one) {
      |                                         ~~~~^~~~~~
MultiMatch.c:1008:21: note: ‘one’ was declared here
 1008 |                 int one, two;
      |                     ^~~
In file included from /home/biocbuild/bbs-3.17-bioc/R/include/Rdefines.h:41,
                 from MultiMatch.c:11:
MultiMatch.c: In function ‘matchLists’:
/home/biocbuild/bbs-3.17-bioc/R/include/Rinternals.h:893:33: warning: ‘utilsPackage’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  893 | #define eval                    Rf_eval
      |                                 ^~~~~~~
MultiMatch.c:247:31: note: ‘utilsPackage’ was declared here
  247 |         SEXP percentComplete, utilsPackage;
      |                               ^~~~~~~~~~~~
In file included from /home/biocbuild/bbs-3.17-bioc/R/include/Rdefines.h:41,
                 from MultiMatch.c:11:
/home/biocbuild/bbs-3.17-bioc/R/include/Rinternals.h:893:33: warning: ‘percentComplete’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  893 | #define eval                    Rf_eval
      |                                 ^~~~~~~
MultiMatch.c:247:14: note: ‘percentComplete’ was declared here
  247 |         SEXP percentComplete, utilsPackage;
      |              ^~~~~~~~~~~~~~~
MultiMatch.c:330:40: warning: ‘rPercentComplete’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  330 |                                 before = *rPercentComplete;
      |                                 ~~~~~~~^~~~~~~~~~~~~~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c NNLS.c -o NNLS.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c Order.c -o Order.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c PairwiseAlignment.c -o PairwiseAlignment.o
PairwiseAlignment.c: In function ‘alignPair._omp_fn.0’:
PairwiseAlignment.c:469:39: warning: ‘p2’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  469 |                                 P2[i] = p2;
      |                                 ~~~~~~^~~~
PairwiseAlignment.c:173:19: note: ‘p2’ was declared here
  173 |         int *p1, *p2, *p3, *p4;
      |                   ^~
PairwiseAlignment.c:468:39: warning: ‘p1’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  468 |                                 P1[i] = p1;
      |                                 ~~~~~~^~~~
PairwiseAlignment.c:173:14: note: ‘p1’ was declared here
  173 |         int *p1, *p2, *p3, *p4;
      |              ^~
PairwiseAlignment.c:473:39: warning: ‘p4’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  473 |                                 P4[i] = p4;
      |                                 ~~~~~~^~~~
PairwiseAlignment.c:173:29: note: ‘p4’ was declared here
  173 |         int *p1, *p2, *p3, *p4;
      |                             ^~
PairwiseAlignment.c:472:39: warning: ‘p3’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  472 |                                 P3[i] = p3;
      |                                 ~~~~~~^~~~
PairwiseAlignment.c:173:24: note: ‘p3’ was declared here
  173 |         int *p1, *p2, *p3, *p4;
      |                        ^~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c PredictDBN.c -o PredictDBN.o
PredictDBN.c: In function ‘predictDBN’:
PredictDBN.c:869:85: warning: ‘prev’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  869 |                                                                 range2[0] = nucs[pos[prev]];// + 1;
      |                                                                                     ^
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c PredictHEC.c -o PredictHEC.o
PredictHEC.c: In function ‘predictHEC’:
PredictHEC.c:255:25: warning: ‘ans’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  255 |                         SET_VECTOR_ELT(ret, i, ans);
      |                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~
PredictHEC.c:233:51: warning: ‘states’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  233 |                                         states[j] = &#x27;C&#x27;;
      |                                         ~~~~~~~~~~^~~~~
PredictHEC.c:237:46: warning: ‘rans’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  237 |                                 *(rans + 3*j + 1) = E;
      |                                  ~~~~~~~~~~~~^~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c R_init_decipher.c -o R_init_decipher.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c RemoveGaps.c -o RemoveGaps.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c ReplaceChars.c -o ReplaceChars.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c S4Vectors_stubs.c -o S4Vectors_stubs.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c SphericalKmeans.c -o SphericalKmeans.o
SphericalKmeans.c: In function ‘sphericalKmeans’:
SphericalKmeans.c:117:25: warning: ‘b’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  117 |                         free(b);
      |                         ^~~~~~~
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c TerminalMismatch.c -o TerminalMismatch.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c Translate.c -o Translate.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c VectorSums.c -o VectorSums.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I/usr/local/include   -fopenmp -fPIC  -g -O2  -Wall -c XVector_stubs.c -o XVector_stubs.o
gcc -shared -L/home/biocbuild/bbs-3.17-bioc/R/lib -L/usr/local/lib -o DECIPHER.so AlignProfiles.o AssignIndels.o Biostrings_stubs.o CalculateDeltaG.o CalculateFISH.o ChainSegments.o Cluster.o ClusterML.o ClusterMP.o CommonGaps.o Compositions.o Compression.o ConsensusSequence.o ConsolidateGaps.o DesignProbes.o Diff.o DistanceMatrix.o EnumerateSequence.o ExpandAmbiguities.o FindFrameshifts.o GeneFinding.o GetPools.o Import.o InformationContent.o InsertGaps.o IntDist.o MeltPolymer.o MovingAverage.o MultiMatch.o NNLS.o Order.o PairwiseAlignment.o PredictDBN.o PredictHEC.o R_init_decipher.o RemoveGaps.o ReplaceChars.o S4Vectors_stubs.o SphericalKmeans.o TerminalMismatch.o Translate.o VectorSums.o XVector_stubs.o -fopenmp -L/home/biocbuild/bbs-3.17-bioc/R/lib -lR
installing to /home/biocbuild/bbs-3.17-bioc/R/library/00LOCK-DECIPHER/00new/DECIPHER/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (DECIPHER)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<HR>
<H3>Example timings</H3>
<P><SPAN class="filename">DECIPHER.Rcheck/DECIPHER-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>AA_REDUCED</TD><TD style="text-align: right;">0.06</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.06</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Add2DB</TD><TD style="text-align: right;">0.310</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.330</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>AdjustAlignment</TD><TD style="text-align: right;">0.257</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.266</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>AlignDB</TD><TD style="text-align: right;">1.253</TD><TD style="text-align: right;">0.061</TD><TD style="text-align: right;">1.316</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>AlignProfiles</TD><TD style="text-align: right;">0.825</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.838</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>AlignSeqs</TD><TD style="text-align: right;">26.899</TD><TD style="text-align: right;"> 0.588</TD><TD style="text-align: right;">27.489</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>AlignSynteny</TD><TD style="text-align: right;">3.432</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">3.481</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>AlignTranslation</TD><TD style="text-align: right;">14.414</TD><TD style="text-align: right;"> 0.140</TD><TD style="text-align: right;">14.562</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>AmplifyDNA</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Array2Matrix</TD><TD style="text-align: right;">5.482</TD><TD style="text-align: right;">0.060</TD><TD style="text-align: right;">5.542</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>BrowseDB</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>BrowseSeqs</TD><TD style="text-align: right;">55.468</TD><TD style="text-align: right;"> 0.348</TD><TD style="text-align: right;">55.866</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>CalculateEfficiencyArray</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>CalculateEfficiencyFISH</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>CalculateEfficiencyPCR</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Clusterize</TD><TD style="text-align: right;">7.578</TD><TD style="text-align: right;">0.076</TD><TD style="text-align: right;">7.655</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Codec</TD><TD style="text-align: right;">1.332</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">1.336</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ConsensusSequence</TD><TD style="text-align: right;">0.210</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.215</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Cophenetic</TD><TD style="text-align: right;">0.21</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.21</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>CorrectFrameshifts</TD><TD style="text-align: right;">18.307</TD><TD style="text-align: right;"> 0.096</TD><TD style="text-align: right;">18.405</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>CreateChimeras</TD><TD style="text-align: right;">0.922</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.926</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DB2Seqs</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DesignArray</TD><TD style="text-align: right;">5.364</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">5.368</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DesignPrimers</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DesignProbes</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DesignSignatures</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DetectRepeats</TD><TD style="text-align: right;">12.576</TD><TD style="text-align: right;"> 0.058</TD><TD style="text-align: right;">12.635</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DigestDNA</TD><TD style="text-align: right;">0.174</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">0.188</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Disambiguate</TD><TD style="text-align: right;">0.059</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.063</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>DistanceMatrix</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ExtractGenes</TD><TD style="text-align: right;">50.497</TD><TD style="text-align: right;"> 0.380</TD><TD style="text-align: right;">50.880</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>FindChimeras</TD><TD style="text-align: right;">0.087</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.087</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>FindGenes</TD><TD style="text-align: right;">46.885</TD><TD style="text-align: right;"> 0.124</TD><TD style="text-align: right;">47.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>FindNonCoding</TD><TD style="text-align: right;">96.862</TD><TD style="text-align: right;"> 0.536</TD><TD style="text-align: right;">97.401</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>FindSynteny</TD><TD style="text-align: right;">1.843</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">1.847</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>FormGroups</TD><TD style="text-align: right;">0.062</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.061</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Genes-class</TD><TD style="text-align: right;">47.923</TD><TD style="text-align: right;"> 0.160</TD><TD style="text-align: right;">48.085</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>HEC_MI</TD><TD style="text-align: right;">0.235</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.235</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>IdConsensus</TD><TD style="text-align: right;">0.878</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.897</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>IdLengths</TD><TD style="text-align: right;">0.025</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.025</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>IdTaxa</TD><TD style="text-align: right;">12.953</TD><TD style="text-align: right;"> 0.024</TD><TD style="text-align: right;">12.977</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>IdentifyByRank</TD><TD style="text-align: right;">0.031</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.031</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>LearnNonCoding</TD><TD style="text-align: right;">73.709</TD><TD style="text-align: right;"> 3.264</TD><TD style="text-align: right;">76.977</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>LearnTaxa</TD><TD style="text-align: right;">15.176</TD><TD style="text-align: right;"> 0.672</TD><TD style="text-align: right;">15.849</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>MIQS</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.051</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>MODELS</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>MapCharacters</TD><TD style="text-align: right;">107.377</TD><TD style="text-align: right;">  0.852</TD><TD style="text-align: right;">108.233</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>MaskAlignment</TD><TD style="text-align: right;">0.607</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.627</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>MeltDNA</TD><TD style="text-align: right;">0.054</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.055</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>NNLS</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>NonCoding-class</TD><TD style="text-align: right;">0.056</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.059</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>NonCodingRNA</TD><TD style="text-align: right;">0.103</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.104</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>OrientNucleotides</TD><TD style="text-align: right;">1.000</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">1.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>PFASUM</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>PredictDBN</TD><TD style="text-align: right;">147.197</TD><TD style="text-align: right;">  0.440</TD><TD style="text-align: right;">147.645</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>PredictHEC</TD><TD style="text-align: right;">0.266</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.265</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>RESTRICTION_ENZYMES</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ReadDendrogram</TD><TD style="text-align: right;">0.098</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.097</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>RemoveGaps</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ScoreAlignment</TD><TD style="text-align: right;">6.401</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">6.441</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SearchDB</TD><TD style="text-align: right;">0.037</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.041</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Seqs2DB</TD><TD style="text-align: right;">0.106</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.106</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>StaggerAlignment</TD><TD style="text-align: right;">16.547</TD><TD style="text-align: right;"> 0.196</TD><TD style="text-align: right;">16.743</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Synteny-class</TD><TD style="text-align: right;">2.760</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">2.771</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Taxa-class</TD><TD style="text-align: right;">14.374</TD><TD style="text-align: right;"> 0.040</TD><TD style="text-align: right;">15.998</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TerminalChar</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TileSeqs</TD><TD style="text-align: right;">5.123</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">5.147</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TrainingSet_16S</TD><TD style="text-align: right;">2.607</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">2.619</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TreeLine</TD><TD style="text-align: right;">10.436</TD><TD style="text-align: right;"> 0.096</TD><TD style="text-align: right;">11.021</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TrimDNA</TD><TD style="text-align: right;">0.149</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.154</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>WriteDendrogram</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>WriteGenes</TD><TD style="text-align: right;">49.767</TD><TD style="text-align: right;"> 0.096</TD><TD style="text-align: right;">49.864</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>deltaGrules</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>deltaHrules</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>deltaHrulesRNA</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.018</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>deltaSrules</TD><TD style="text-align: right;">0.062</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.061</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>deltaSrulesRNA</TD><TD style="text-align: right;">0.018</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.017</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
