<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for biovizBase on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.17</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><B>[<A href="..#B">B</A>]</B></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-03-01 07:38:36 -0000 (Wed, 01 Mar 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4266</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for biovizBase on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the biovizBase package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/biovizBase.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>203</B>/2171</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">biovizBase</A>&nbsp;1.47.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/biovizBase">landing page</A>)</SPAN><BR>Michael Lawrence<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-27&nbsp;07:53:22&nbsp;-0000&nbsp;(Mon,&nbsp;27&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/biovizBase</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">73f68c1</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:06:19&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: biovizBase</TD></TR>
<TR><TD><B>Version</B>: 1.47.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:biovizBase.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings biovizBase_1.47.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-27 22:04:11 -0000 (Mon, 27 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-27 22:11:35 -0000 (Mon, 27 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 443.3 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: biovizBase.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 2</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:biovizBase.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings biovizBase_1.47.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/biovizBase.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘biovizBase/DESCRIPTION’ ... OK
* this is package ‘biovizBase’ version ‘1.47.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘biovizBase’ can be installed ... WARNING
Found the following significant warnings:
  bin_offsets.c:57:15: warning: pointer targets in passing argument 1 of ‘strncmp’ differ in signedness [-Wpointer-sign]
See ‘/home/biocbuild/bbs-3.17-bioc/meat/biovizBase.Rcheck/00install.out’ for details.
* used C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... NOTE
Malformed Title field: should not end in a period.
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
&#x27;library&#x27; or &#x27;require&#x27; calls in package code:
  &#x27;BSgenome&#x27; &#x27;rtracklayer&#x27;
  Please use :: or requireNamespace() instead.
  See section &#x27;Suggested packages&#x27; in the &#x27;Writing R Extensions&#x27; manual.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
genSymbols: no visible binding for global variable &#x27;start_location&#x27;
genSymbols: no visible binding for global variable &#x27;end_location&#x27;
genSymbols: no visible binding for global variable &#x27;Chromosome&#x27;
genSymbols: no visible binding for global variable &#x27;symbol&#x27;
transformToLinkInCircle: no visible binding for global variable
  &#x27;.circle.x&#x27;
transformToLinkInCircle: no visible binding for global variable
  &#x27;.circle.y&#x27;
transformToLinkInCircle : &lt;anonymous&gt;: no visible binding for global
  variable &#x27;from.x&#x27;
transformToLinkInCircle : &lt;anonymous&gt;: no visible binding for global
  variable &#x27;from.y&#x27;
transformToLinkInCircle : &lt;anonymous&gt;: no visible binding for global
  variable &#x27;to.x&#x27;
transformToLinkInCircle : &lt;anonymous&gt;: no visible binding for global
  variable &#x27;to.y&#x27;
mold,ExpressionSet: no visible global function definition for &#x27;exprs&#x27;
mold,ExpressionSet: no visible global function definition for &#x27;pData&#x27;
mold,RleList: no visible binding for global variable &#x27;xRleList&#x27;
mold,eSet: no visible global function definition for &#x27;phenoData&#x27;
mold,eSet: no visible global function definition for &#x27;melt&#x27;
mold,eSet: no visible global function definition for &#x27;varLabels&#x27;
Undefined global functions or variables:
  .circle.x .circle.y Chromosome end_location exprs from.x from.y melt
  pData phenoData start_location symbol to.x to.y varLabels xRleList
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking LazyData ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... WARNING
Found the following significant warnings:

  Warning: &#x27;GenenameFilter&#x27; is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
Examples with CPU (user + system) or elapsed time &gt; 5s
                user system elapsed
crunch-method 20.067  0.604  20.837
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘intro.Rnw’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 2 WARNINGs, 4 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/biovizBase.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">biovizBase.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL biovizBase
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘biovizBase’ ...
** using staged installation
** libs
using C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG   -I/usr/local/include    -fPIC  -g -O2  -Wall -c R_init_biovizBase.c -o R_init_biovizBase.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG   -I/usr/local/include    -fPIC  -g -O2  -Wall -c bin_offsets.c -o bin_offsets.o
bin_offsets.c: In function ‘scan_bam_bin_offsets’:
bin_offsets.c:57:15: warning: pointer targets in passing argument 1 of ‘strncmp’ differ in signedness [-Wpointer-sign]
   57 |   if (strncmp(b, &quot;BAI\1&quot;, 4))
      |               ^
      |               |
      |               Rbyte * {aka unsigned char *}
In file included from /home/biocbuild/bbs-3.17-bioc/R/include/R_ext/RS.h:34,
                 from /home/biocbuild/bbs-3.17-bioc/R/include/Rdefines.h:38,
                 from bin_offsets.h:1,
                 from bin_offsets.c:3:
/usr/include/string.h:159:33: note: expected ‘const char *’ but argument is of type ‘Rbyte *’ {aka ‘unsigned char *’}
  159 | extern int strncmp (const char *__s1, const char *__s2, size_t __n)
      |                     ~~~~~~~~~~~~^~~~
gcc -shared -L/home/biocbuild/bbs-3.17-bioc/R/lib -L/usr/local/lib -o biovizBase.so R_init_biovizBase.o bin_offsets.o -L/home/biocbuild/bbs-3.17-bioc/R/lib -lR
installing to /home/biocbuild/bbs-3.17-bioc/R/library/00LOCK-biovizBase/00new/biovizBase/libs
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (biovizBase)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">biovizBase.Rcheck/tests/runTests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; BiocGenerics:::testPackage(&quot;biovizBase&quot;)
Loading required package: ensembldb
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: GenomicRanges
Loading required package: stats4
Loading required package: S4Vectors

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
Loading required package: GenomicFeatures
Loading required package: AnnotationDbi
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

Loading required package: AnnotationFilter

Attaching package: &#x27;ensembldb&#x27;

The following object is masked from &#x27;package:stats&#x27;:

    filter

Fetching data...OK
Parsing exons...OK
Defining introns...OK
Defining UTRs...OK
Defining CDS...OK
aggregating...
Done
Fetching data...OK
Parsing exons...OK
Defining introns...OK
Defining UTRs...OK
Defining CDS...OK
aggregating...
Done
Fetching data...Fetching data...OK
Parsing exons...OK
Defining introns...OK
Defining UTRs...OK
Defining CDS...OK
aggregating...
Done
Fetching data...OK
Parsing exons...OK
Defining introns...OK
Defining UTRs...OK
Defining CDS...OK
aggregating...
Done
Fetching data...OK
Parsing exons...OK
Defining introns...OK
Defining UTRs...OK
Defining CDS...OK
aggregating...
Done
Fetching data...OK
Parsing exons...OK
Defining introns...OK
Defining UTRs...OK
Defining CDS...OK
aggregating...
Done


RUNIT TEST PROTOCOL -- Mon Feb 27 22:10:44 2023 
*********************************************** 
Number of test functions: 1 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
biovizBase RUnit Tests - 1 test function, 0 errors, 0 failures
Number of test functions: 1 
Number of errors: 0 
Number of failures: 0 
Warning message:
&#x27;GenenameFilter&#x27; is deprecated.
Use &#x27;GeneNameFilter&#x27; instead.
See help(&quot;Deprecated&quot;) 
&gt; 
&gt; proc.time()
   user  system elapsed 
 24.269   1.133  25.509 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">biovizBase.Rcheck/biovizBase-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>CRC</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GCcontent</TD><TD style="text-align: right;">0.492</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.493</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addStepping-method</TD><TD style="text-align: right;">0.966</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">1.009</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>aes-utils</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>colorBlindSafePal</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>containLetters</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>crc1.GeRL</TD><TD style="text-align: right;">0.066</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.066</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>crunch-method</TD><TD style="text-align: right;">20.067</TD><TD style="text-align: right;"> 0.604</TD><TD style="text-align: right;">20.837</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>darned_hg19_subset500</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>flatGrl</TD><TD style="text-align: right;">0.267</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.279</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genesymbol</TD><TD style="text-align: right;">0.058</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.059</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getBioColor</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getFormalNames</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getGaps</TD><TD style="text-align: right;">1.402</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">1.409</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getIdeoGR</TD><TD style="text-align: right;">0.230</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.247</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getIdeogram</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>hg19Ideogram</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>hg19IdeogramCyto</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.017</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ideo</TD><TD style="text-align: right;">0.056</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.056</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ideoCyto</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>isIdeogram</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>isMatchedWithModel</TD><TD style="text-align: right;">0.353</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.359</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>isSimpleIdeogram</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>maxGap-method</TD><TD style="text-align: right;">0.393</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.396</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pileupAsGRanges</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pileupGRangesAsVariantTable</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotColorLegend</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>scale</TD><TD style="text-align: right;">0.305</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">0.337</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>showColor</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>shrinkageFun-method</TD><TD style="text-align: right;">0.240</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.256</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>splitByFacets-method</TD><TD style="text-align: right;">0.547</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.559</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>strip_formula_dots</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>subsetArgsByFormals</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>transform</TD><TD style="text-align: right;">2.008</TD><TD style="text-align: right;">0.088</TD><TD style="text-align: right;">2.096</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>transformGRangesForEvenSpace</TD><TD style="text-align: right;">0.329</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.330</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
