<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for SeqVarTools on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><B>[<A href="..#S">S</A>]</B></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-31 05:44:44 -0000 (Wed, 31 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4219</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for SeqVarTools on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the SeqVarTools package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/SeqVarTools.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1867</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">SeqVarTools</A>&nbsp;1.39.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/SeqVarTools">landing page</A>)</SPAN><BR>Stephanie M. Gogarten<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-29&nbsp;10:19:22&nbsp;-0000&nbsp;(Mon,&nbsp;29&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/SeqVarTools</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">fa2a136</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:28:13&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: SeqVarTools</TD></TR>
<TR><TD><B>Version</B>: 1.39.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:SeqVarTools.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings SeqVarTools_1.39.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-30 18:59:50 -0000 (Tue, 30 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-30 19:04:32 -0000 (Tue, 30 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 282.4 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: SeqVarTools.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:SeqVarTools.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings SeqVarTools_1.39.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/SeqVarTools.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘SeqVarTools/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘SeqVarTools’ version ‘1.39.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘SeqVarTools’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking LazyData ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘test.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘Iterators.Rnw’... OK
  ‘SeqVarTools.Rnw’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: OK

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">SeqVarTools.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD INSTALL SeqVarTools
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0/site-library’
* installing *source* package ‘SeqVarTools’ ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (SeqVarTools)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">SeqVarTools.Rcheck/tests/test.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(GenomicRanges)
Loading required package: stats4
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors

Attaching package: &#x27;S4Vectors&#x27;

The following object is masked from &#x27;package:utils&#x27;:

    findMatches

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
&gt; BiocGenerics:::testPackage(&quot;SeqVarTools&quot;)
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

# of selected samples: 5
# of selected variants: 5
# of selected samples: 5
# of selected variants: 10
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 3
# of selected samples: 3
matching samples... 2 pairs identified!
matching variants... 1346 non-overlapping variant matches identified!
# of selected samples: 2
# of selected variants: 1,346
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 2
# of selected variants: 50
# of selected samples: 2
# of selected variants: 51
matching samples... 2 pairs identified!
matching variants... 26 non-overlapping variant matches identified!
# of selected variants: 26
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 5
# of selected variants: 10
# of selected samples: 5
# of selected variants: 10
File: /home/biocbuild/R/R-4.3.0/site-library/SeqArray/extdata/CEU_Exon.gds (287.6K)
+    [  ] *
|--+ description   [  ] *
|--+ sample.id   { Str8 90 LZMA_ra(34.7%), 257B } *
|--+ variant.id   { Int32 1348 LZMA_ra(16.7%), 905B } *
|--+ position   { Int32 1348 LZMA_ra(64.4%), 3.4K } *
|--+ chromosome   { Str8 1348 LZMA_ra(4.39%), 157B } *
|--+ allele   { Str8 1348 LZMA_ra(16.6%), 901B } *
|--+ genotype   [  ] *
|  |--+ data   { Bit2 2x90x1348 LZMA_ra(26.3%), 15.6K } *
|  |--+ ~data   { Bit2 2x1348x90 LZMA_ra(29.2%), 17.3K } *
|  |--+ extra.index   { Int32 3x0 LZMA_ra, 18B } *
|  \--+ extra   { Int16 0 LZMA_ra, 18B }
|--+ phase   [  ]
|  |--+ data   { Bit1 90x1348 LZMA_ra(0.86%), 137B } *
|  |--+ ~data   { Bit1 1348x90 LZMA_ra(0.86%), 137B } *
|  |--+ extra.index   { Int32 3x0 LZMA_ra, 18B } *
|  \--+ extra   { Bit1 0 LZMA_ra, 18B }
|--+ annotation   [  ]
|  |--+ id   { Str8 1348 LZMA_ra(38.3%), 5.5K } *
|  |--+ qual   { Float32 1348 LZMA_ra(2.11%), 121B } *
|  |--+ filter   { Int32,factor 1348 LZMA_ra(2.11%), 121B } *
|  |--+ info   [  ]
|  |  |--+ AA   { Str8 1328 LZMA_ra(22.1%), 593B } *
|  |  |--+ AC   { Int32 1348 LZMA_ra(24.1%), 1.3K } *
|  |  |--+ AN   { Int32 1348 LZMA_ra(19.6%), 1.0K } *
|  |  |--+ DP   { Int32 1348 LZMA_ra(47.7%), 2.5K } *
|  |  |--+ HM2   { Bit1 1348 LZMA_ra(145.6%), 253B } *
|  |  |--+ HM3   { Bit1 1348 LZMA_ra(145.6%), 253B } *
|  |  |--+ OR   { Str8 1348 LZMA_ra(19.6%), 341B } *
|  |  |--+ GP   { Str8 1348 LZMA_ra(24.3%), 3.8K } *
|  |  \--+ BN   { Int32 1348 LZMA_ra(20.7%), 1.1K } *
|  \--+ format   [  ]
|     \--+ DP   [  ] *
|        |--+ data   { VL_Int 90x1348 LZMA_ra(70.8%), 115.2K } *
|        \--+ ~data   { VL_Int 1348x90 LZMA_ra(65.1%), 105.9K } *
\--+ sample.annotation   [  ]
   \--+ family   { Str8 90 LZMA_ra(55.0%), 221B } *
# of selected variants: 142
# of selected samples: 90
# of selected variants: 1,348
# of selected variants: 5
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 5
# of selected variants: 5
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 3
# of selected samples: 3
matching samples... 2 pairs identified!
matching variants... 1346 variant matches identified!
matching samples... 2 pairs identified!
matching variants... 1346 variant matches identified!
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 2
# of selected variants: 50
# of selected samples: 2
# of selected variants: 51
matching samples... 2 pairs identified!
matching variants... 26 variant matches identified!
matching samples... 2 pairs identified!
matching variants... 26 variant matches identified!
# of selected variants: 26
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 3
# of selected samples: 3
matching samples... 2 pairs identified!
matching variants... 1346 variant matches identified!
matching samples... 2 pairs identified!
matching variants... 1346 variant matches identified!
matching samples... 2 pairs identified!
matching variants... 1346 variant matches identified!
matching samples... 2 pairs identified!
matching variants... 1346 variant matches identified!
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 2
# of selected variants: 50
# of selected samples: 2
# of selected variants: 51
matching samples... 2 pairs identified!
matching variants... 26 variant matches identified!
matching samples... 2 pairs identified!
matching variants... 26 variant matches identified!
# of selected variants: 26
Warning in SeqVarTools:::.samplePairs1(samples) :
  More than two samples for subject 4
Selecting first two samples: samp7, samp8
# of selected samples: 5
# of selected variants: 10
# of selected samples: 5
# of selected variants: 10
# of selected samples: 5
# of selected variants: 10
# of selected samples: 5
# of selected variants: 10
# of selected samples: 5
# of selected variants: 10
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 5
# of selected variants: 10
# of selected samples: 5
# of selected variants: 10
# of selected samples: 5
# of selected variants: 10
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 5
# of selected variants: 10
# of selected samples: 5
# of selected variants: 10
# of selected variants: 1,346
# of selected variants: 1,330
# of selected samples: 5
# of selected variants: 10
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 5
# of selected variants: 10
# of selected samples: 5
# of selected variants: 10
# of selected samples: 5
# of selected variants: 10
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 5
# of selected variants: 10
# of selected samples: 3
# of selected variants: 10
# of selected samples: 3
# of selected samples: 90
# of selected variants: 1,348
# of selected variants: 10
# of selected samples: 3
# of selected samples: 5
# of selected variants: 10
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 5
# of selected variants: 10
# of selected samples: 5
# of selected variants: 10
# of selected samples: 5
# of selected variants: 10
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 5
# of selected variants: 10
# of selected variants: 903
# of selected samples: 59
# of selected samples: 58
# of selected samples: 32
# of selected samples: 5
# of selected variants: 10
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 5
# of selected variants: 10
# of selected samples: 5
# of selected variants: 10
# of selected samples: 90
# of selected variants: 1,348
# of selected samples: 5
# of selected variants: 10
# of selected samples: 10
# of selected samples: 10
# of selected variants: 10
# of selected samples: 10


RUNIT TEST PROTOCOL -- Tue May 30 19:03:09 2023 
*********************************************** 
Number of test functions: 152 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
SeqVarTools RUnit Tests - 152 test functions, 0 errors, 0 failures
Number of test functions: 152 
Number of errors: 0 
Number of failures: 0 
&gt; 
&gt; proc.time()
   user  system elapsed 
 31.002  11.420  35.385 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">SeqVarTools.Rcheck/SeqVarTools-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>Iterator-class</TD><TD style="text-align: right;">0.801</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">0.842</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SeqVarData-class</TD><TD style="text-align: right;">0.050</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.058</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>allele-methods</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>alleleFrequency</TD><TD style="text-align: right;">0.022</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.021</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>alternateAlleleDetection</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>applyMethod</TD><TD style="text-align: right;">0.239</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.247</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>countSingletons</TD><TD style="text-align: right;">0.030</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.034</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>duplicateDiscordance</TD><TD style="text-align: right;">0.168</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.172</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getGenotype</TD><TD style="text-align: right;">0.042</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getVariableLengthData</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>heterozygosity</TD><TD style="text-align: right;">0.253</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.269</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>hwe</TD><TD style="text-align: right;">0.085</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.086</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>imputedDosage</TD><TD style="text-align: right;">0.112</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.129</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>inbreedCoeff</TD><TD style="text-align: right;">0.176</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.175</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>isSNV</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>isVariant</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>meanBySample</TD><TD style="text-align: right;">0.031</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.035</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mendelErr</TD><TD style="text-align: right;">0.018</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.018</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>missingGenotypeRate</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pca</TD><TD style="text-align: right;">1.021</TD><TD style="text-align: right;">0.060</TD><TD style="text-align: right;">1.081</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pedigree</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>refFrac</TD><TD style="text-align: right;">0.061</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.065</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>regression</TD><TD style="text-align: right;">0.087</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.086</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>setVariantID</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>titv</TD><TD style="text-align: right;">0.220</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.228</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>variantInfo</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
