<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for MungeSumstats on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><B>[<A href="../index.html#M">M</A>]</B></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-13 06:49:43 -0000 (Mon, 13 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4222</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for MungeSumstats on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the MungeSumstats package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/MungeSumstats.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1298</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">MungeSumstats</A>&nbsp;1.7.17</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/MungeSumstats">landing page</A>)</SPAN><BR>Alan Murphy<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-11&nbsp;01:58:29&nbsp;-0000&nbsp;(Sat,&nbsp;11&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/MungeSumstats</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">2678c7e</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-02-07&nbsp;11:03:29&nbsp;-0000&nbsp;(Tue,&nbsp;07&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: MungeSumstats</TD></TR>
<TR><TD><B>Version</B>: 1.7.17</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:MungeSumstats.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings MungeSumstats_1.7.17.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-11 18:35:50 -0000 (Sat, 11 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-11 18:47:50 -0000 (Sat, 11 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 720.1 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: MungeSumstats.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:MungeSumstats.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings MungeSumstats_1.7.17.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/MungeSumstats.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘MungeSumstats/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘MungeSumstats’ version ‘1.7.17’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘MungeSumstats’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking R/sysdata.rda ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                    user system elapsed
get_genome_builds 85.649  6.131  92.941
format_sumstats   49.521  3.830  54.580
liftover           1.248  0.140   7.243
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
      duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
      lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
      pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
      tapply, union, unique, unsplit, which.max, which.min
  
  Loading required package: S4Vectors
  Loading required package: stats4
  
  Attaching package: &#x27;S4Vectors&#x27;
  
  The following objects are masked from &#x27;package:base&#x27;:
  
      I, expand.grid, unname
  
  Killed
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘MungeSumstats.Rmd’ using ‘UTF-8’... OK
  ‘OpenGWAS.Rmd’ using ‘UTF-8’... OK
  ‘docker.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/MungeSumstats.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">MungeSumstats.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL MungeSumstats
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘MungeSumstats’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (MungeSumstats)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">MungeSumstats.Rcheck/tests/testthat.Rout.fail</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(MungeSumstats)
&gt; 
&gt; test_check(&quot;MungeSumstats&quot;)
Collecting metadata from Open GWAS.
Filtering metadata by substring criteria.
Found 3 GWAS datasets matching search criteria across:
   - 3 trait(s)
   - 1 population(s)
   - 2 category(ies)
   - 2 subcategory(ies)
   - 2 publication(s)
   - 2 consortia(ium)
   - 1 genome build(s)
Collecting metadata from Open GWAS.
Filtering metadata by substring criteria.
Filtering metadata by sample/case/control/SNP size criteria.
Excluding sample/case/control size with NAs.
Found 3 GWAS datasets matching search criteria across:
   - 3 trait(s)
   - 1 population(s)
   - 2 category(ies)
   - 2 subcategory(ies)
   - 2 publication(s)
   - 2 consortia(ium)
   - 1 genome build(s)
Collecting metadata from Open GWAS.
Filtering metadata by substring criteria.
Found 45 GWAS datasets matching search criteria across:
   - 42 trait(s)
   - 3 population(s)
   - 2 category(ies)
   - 2 subcategory(ies)
   - 7 publication(s)
   - 5 consortia(ium)
   - 1 genome build(s)
Downloading VCF ==&gt; /tmp/RtmpfFLdOp/ieu-a-298.vcf.gz
Downloading with download.file.
trying URL &#x27;https://gwas.mrcieu.ac.uk/files/ieu-a-298/ieu-a-298.vcf.gz&#x27;
Content type &#x27;application/gzip&#x27; length 234480 bytes (228 KB)
==================================================
downloaded 228 KB

Downloading VCF index ==&gt; https://gwas.mrcieu.ac.uk/files/ieu-a-298/ieu-a-298.vcf.gz.tbi
Downloading with download.file.
trying URL &#x27;https://gwas.mrcieu.ac.uk/files/ieu-a-298/ieu-a-298.vcf.gz.tbi&#x27;
Content type &#x27;application/gzip&#x27; length 37803 bytes (36 KB)
==================================================
downloaded 36 KB

Processing 1 datasets from Open GWAS.

========== Processing dataset : a-fake-id ==========

Downloading VCF ==&gt; /tmp/RtmpfFLdOp/a-fake-id.vcf.gz
Downloading with download.file.
trying URL &#x27;https://gwas.mrcieu.ac.uk/files/a-fake-id/a-fake-id.vcf.gz&#x27;
Processing 1 datasets from Open GWAS.

========== Processing dataset : ieu-a-298 ==========

Using previously downloaded VCF.
Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpfFLdOp/ieu-a-298/ieu-a-298.tsv.gz
Loading required namespace: GenomicFiles
Using local VCF.
File already tabix-indexed.
Finding empty VCF columns based on first 10,000 rows.
1 sample detected: ieu-a-298
Constructing ScanVcfParam object.
VCF contains: 10,684 variant(s) x 1 sample(s)
Reading VCF file: single-threaded
Converting VCF to data.table.
Expanding VCF first, so number of rows may increase.
Checking for empty columns.
Unlisting 4 columns.
Time difference of 1.1 secs
VCF data.table contains: 10,684 rows x 11 columns.
Time difference of 2.1 secs
Renaming ID as SNP.
VCF file has -log10 P-values; these will be converted to unadjusted p-values in the &#x27;P&#x27; column.
No INFO (SI) column detected.
Standardising column headers.
First line of summary statistics file: 
SNP	chr	BP	end	REF	ALT	FILTER	ES	SE	LP	SS	P	
Summary statistics report:
   - 10,684 rows
   - 10,684 unique variants
   - 553 genome-wide significant variants (P&lt;5e-8)
   - 22 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Reordering so first three column headers are SNP, CHR and BP in this order.
Reordering so the fourth and fifth columns are A1 and A2.
Checking for missing data.
Checking for duplicate columns.
Ensuring that the N column is all integers.
The sumstats N column is not all integers, this could effect downstream analysis. These will be converted to integers.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
N already exists within sumstats_dt.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpfFLdOp/ieu-a-298/ieu-a-298.tsv.gz
Summary statistics report:
   - 10,684 rows (100% of original 10,684 rows)
   - 10,684 unique variants
   - 553 genome-wide significant variants (P&lt;5e-8)
   - 22 chromosomes
Done munging in 0.094 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR      BP A1 A2     END FILTER   BETA     SE      LP     N
1: rs76805690   1 2256288  C  A 2256288   PASS 0.0389 0.0175 1.58536 74046
2: rs75379543   1 2261983  C  A 2261983   PASS 0.0427 0.0167 1.96738 74046
3: rs75273719   1 2263666  G  A 2263666   PASS 0.0502 0.0171 2.47353 74046
4:   rs903904   1 2263888  C  T 2263888   PASS 0.0413 0.0169 1.82769 74046
             P
1: 0.025980051
2: 0.010780031
3: 0.003361012
4: 0.014869967
Returning path to saved data.

ieu-a-298 : Done in 0.094 minutes.

Done with all processing in 0.09 minutes.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpfFLdOp/file3b28192ff84fea.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpfFLdOp/file3b281948b75003
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A0	A1	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpfFLdOp/file3b28192ff84fea.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpfFLdOp/file3b28193144a96d.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpfFLdOp/file3b281948b75003
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A1	A2	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for missing data.
Checking for duplicate columns.
Checking for duplicated rows.
INFO column not available. Skipping INFO score filtering step.
Filtering SNPs, ensuring SE&gt;0.
Ensuring all SNPs have N&lt;5 std dev above mean.
Removing &#x27;chr&#x27; prefix from CHR.
Making X/Y/MT CHR uppercase.
Warning: When method is an integer, must be &gt;0.
47 SNPs (50.5%) have FRQ values &gt; 0.5. Conventionally the FRQ column is intended to show the minor/effect allele frequency.
The FRQ column was mapped from one of the following from the inputted  summary statistics file:
FRQ, EAF, FREQUENCY, FRQ_U, F_U, MAF, FREQ, FREQ_TESTED_ALLELE, FRQ_TESTED_ALLELE, FREQ_EFFECT_ALLELE, FRQ_EFFECT_ALLELE, EFFECT_ALLELE_FREQUENCY, EFFECT_ALLELE_FREQ, EFFECT_ALLELE_FRQ, A1FREQ, A1FRQ, A2FREQ, A2FRQ, ALLELE_FREQUENCY, ALLELE_FREQ, ALLELE_FRQ, AF, MINOR_AF, EFFECT_AF, A2_AF, EFF_AF, ALT_AF, ALTERNATIVE_AF, INC_AF, A_2_AF, TESTED_AF, AF1, ALLELEFREQ, ALT_FREQ, EAF_HRC, EFFECTALLELEFREQ, FREQ.A1.1000G.EUR, FREQ.A1.ESP.EUR, FREQ.ALLELE1.HAPMAPCEU, FREQ.B, FREQ1, FREQ1.HAPMAP, FREQ_EUROPEAN_1000GENOMES, FREQ_HAPMAP, FREQ_TESTED_ALLELE_IN_HRS, FRQ_A1, FRQ_U_113154, FRQ_U_31358, FRQ_U_344901, FRQ_U_43456, POOLED_ALT_AF, AF_ALT, AF.ALT, AF-ALT, ALT.AF, ALT-AF, A2.AF, A2-AF, AF.EFF, AF_EFF, AF_EFF
As frq_is_maf=TRUE, the FRQ column will not be renamed. If the FRQ values were intended to represent major allele frequency,
set frq_is_maf=FALSE to rename the column as MAJOR_ALLELE_FRQ and differentiate it from minor/effect allele frequency.
Sorting coordinates with &#x27;data.table&#x27;.
Writing in tabular format ==&gt; /tmp/RtmpfFLdOp/file3b28193144a96d.tsv.gz
Summary statistics report:
   - 93 rows (100% of original 93 rows)
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Done munging in 0.001 minutes.
Successfully finished preparing sumstats file, preview:
Reading header.
          SNP CHR       BP A1 A2     FRQ   BETA    SE         P
1:   rs301800   1  8490603  T  C 0.17910  0.019 0.003 1.794e-08
2: rs11210860   1 43982527  A  G 0.36940  0.017 0.003 2.359e-10
3: rs34305371   1 72733610  A  G 0.08769  0.035 0.005 3.762e-14
4:  rs2568955   1 72762169  T  C 0.23690 -0.017 0.003 1.797e-08
Returning path to saved data.


******::NOTE::******
 - Formatted results will be saved to `tempdir()` by default.
 - This means all formatted summary stats will be deleted upon ending the R session.
 - To keep formatted summary stats, change `save_path`  ( e.g. `save_path=file.path(&#x27;./formatted&#x27;,basename(path))` ),   or make sure to copy files elsewhere after processing  ( e.g. `file.copy(save_path, &#x27;./formatted/&#x27; )`.
 ******************** 

Formatted summary statistics will be saved to ==&gt;  /tmp/RtmpfFLdOp/file3b2819496d1e42.tsv.gz
Reading header.
Tabular format detected.
Importing tabular file: /tmp/RtmpfFLdOp/file3b28194d45466c
Checking for empty columns.
Standardising column headers.
First line of summary statistics file: 
MarkerName	CHR	POS	A2	A1	EAF	Beta	SE	Pval	
Summary statistics report:
   - 93 rows
   - 93 unique variants
   - 70 genome-wide significant variants (P&lt;5e-8)
   - 20 chromosomes
Checking for multi-GWAS.
Checking for multiple RSIDs on one row.
Checking SNP RSIDs.
Checking for merged allele column.
Checking A1 is uppercase
Checking A2 is uppercase
Checking for correct direction of A1 (reference) and A2 (alternative allele).
Loading SNPlocs data.
Loading reference genome data.
Preprocessing RSIDs.
Validating RSIDs of 93 SNPs using BSgenome::snpsById...
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Killed
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">MungeSumstats.Rcheck/MungeSumstats-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>compute_nsize</TD><TD style="text-align: right;">0.059</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.069</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>download_vcf</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>find_sumstats</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>format_sumstats</TD><TD style="text-align: right;">49.521</TD><TD style="text-align: right;"> 3.830</TD><TD style="text-align: right;">54.580</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>formatted_example</TD><TD style="text-align: right;">0.030</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_genome_builds</TD><TD style="text-align: right;">85.649</TD><TD style="text-align: right;"> 6.131</TD><TD style="text-align: right;">92.941</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>import_sumstats</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>index_tabular</TD><TD style="text-align: right;">0.063</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>index_vcf</TD><TD style="text-align: right;">0.054</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.053</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>liftover</TD><TD style="text-align: right;">1.248</TD><TD style="text-align: right;">0.140</TD><TD style="text-align: right;">7.243</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>list_sumstats</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>load_snp_loc_data</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>parse_logs</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>read_header</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>read_sumstats</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>read_vcf</TD><TD style="text-align: right;">2.264</TD><TD style="text-align: right;">0.068</TD><TD style="text-align: right;">2.341</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>standardise_header</TD><TD style="text-align: right;">0.022</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>vcf2df</TD><TD style="text-align: right;">0.714</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.721</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>write_sumstats</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
