<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for RnBeads on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.17</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><B>[<A href="..#R">R</A>]</B></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-27 02:34:42 -0000 (Mon, 27 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4259</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for RnBeads on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the RnBeads package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/RnBeads.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard timeout">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1701</B>/2169</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">RnBeads</A>&nbsp;2.17.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/RnBeads">landing page</A>)</SPAN><BR>Fabian Mueller<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-23&nbsp;09:40:21&nbsp;-0000&nbsp;(Thu,&nbsp;23&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/RnBeads</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">99cf51e</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-01-03&nbsp;12:42:18&nbsp;-0000&nbsp;(Tue,&nbsp;03&nbsp;Jan&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph TIMEOUT">&nbsp;&nbsp;TIMEOUT&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: RnBeads</TD></TR>
<TR><TD><B>Version</B>: 2.17.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:RnBeads.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings RnBeads_2.17.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-24 17:34:11 -0000 (Fri, 24 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-24 19:34:11 -0000 (Fri, 24 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 7200.5 seconds</TD></TR>
<TR><TD><B>RetCode</B>: None</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph TIMEOUT">&nbsp;&nbsp;TIMEOUT&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: RnBeads.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:RnBeads.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings RnBeads_2.17.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/RnBeads.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘RnBeads/DESCRIPTION’ ... OK
* this is package ‘RnBeads’ version ‘2.17.0’
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Depends: includes the non-default packages:
  &#x27;BiocGenerics&#x27;, &#x27;S4Vectors&#x27;, &#x27;GenomicRanges&#x27;, &#x27;MASS&#x27;, &#x27;cluster&#x27;,
  &#x27;ff&#x27;, &#x27;fields&#x27;, &#x27;ggplot2&#x27;, &#x27;gplots&#x27;, &#x27;grid&#x27;, &#x27;gridExtra&#x27;, &#x27;limma&#x27;,
  &#x27;matrixStats&#x27;, &#x27;illuminaio&#x27;, &#x27;methylumi&#x27;, &#x27;plyr&#x27;
Adding so many packages to the search path is excessive and importing
selectively is preferable.
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .travis.yml
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘RnBeads’ can be installed ... OK
* checking installed package size ... NOTE
  installed size is  5.5Mb
  sub-directories of 1Mb or more:
    R     3.1Mb
    bin   1.0Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... NOTE
Package listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘qvalue’
A package should be listed in only one of these fields.
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... NOTE
Warning: no function found corresponding to methods exports from ‘RnBeads’ for: ‘samples’

A namespace must be able to be loaded with just the base namespace
loaded: otherwise if the namespace gets loaded by a saved object, the
session will be unable to start.

Probably some imports need to be declared in the NAMESPACE file.
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
Package in Depends field not imported from: &#x27;grid&#x27;
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.
Unexported objects imported by &#x27;:::&#x27; calls:
  &#x27;Gviz:::.getBMFeatureMap&#x27; &#x27;doParallel:::.options&#x27;
  &#x27;grDevices:::.smoothScatterCalcDensity&#x27;
  &#x27;minfi:::.default.450k.annotation&#x27; &#x27;minfi:::.extractFromRGSet450k&#x27;
  &#x27;minfi:::.normalizeFunnorm450k&#x27;
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
.stopImplicitCluster: no visible global function definition for
  ‘stopCluster’
MethyLumiSet2RnBeadSet: no visible global function definition for
  ‘phenoData’
MethyLumiSet2RnBeadSet: no visible global function definition for
  ‘assayDataElementNames’
MethyLumiSet2RnBeadSet: no visible global function definition for
  ‘featureNames’
MethyLumiSet2RnBeadSet: no visible global function definition for
  ‘varLabels’
MethyLumiSet2RnBeadSet: no visible global function definition for
  ‘featureData’
add.age.histogram: no visible binding for global variable ‘Age’
add.age.histogram: no visible binding for global variable ‘..count..’
add.age.histogram: no visible binding for global variable ‘..density..’
add.agecomparison.plot: no visible global function definition for
  ‘samples’
add.agecomparison.plot: no visible binding for global variable ‘Sample’
add.agecomparison.plot: no visible global function definition for
  ‘%dopar%’
add.agecomparison.plot: no visible global function definition for
  ‘foreach’
add.agecomparison.plot: no visible binding for global variable ‘i’
add.combination.plot: no visible global function definition for
  ‘samples’
add.combination.plot: no visible binding for global variable
  ‘Difference’
add.combination.plot: no visible binding for global variable ‘Density’
add.combination.plot: no visible binding for global variable ‘yint’
add.combination.plot: no visible binding for global variable ‘Measure’
add.error.plot: no visible binding for global variable ‘Value’
add.error.plot: no visible binding for global variable ‘Sample’
add.error.plot: no visible binding for global variable ‘Deviance’
add.error.plot: no visible binding for global variable ‘yint’
add.error.plot: no visible binding for global variable ‘Measure’
add.negative.control.boxplot: no visible global function definition for
  ‘samples’
add.profile.plots: no visible global function definition for ‘mclapply’
add.qc.barplots: no visible global function definition for ‘samples’
add.quantile.plot: no visible binding for global variable ‘Difference’
add.quantile.plot: no visible binding for global variable ‘..density..’
add.quantile.plot: no visible binding for global variable ‘Sample’
add.quantile.plot: no visible binding for global variable ‘Density’
add.seq.coverage.histograms: no visible global function definition for
  ‘samples’
add.seq.coverage.plot: no visible global function definition for
  ‘samples’
add.seq.coverage.violins: no visible global function definition for
  ‘samples’
add.stratification.plot: no visible global function definition for
  ‘%dopar%’
add.stratification.plot: no visible global function definition for
  ‘foreach’
add.stratification.plot: no visible binding for global variable ‘Group’
add.stratification.plot: no visible binding for global variable
  ‘Increase’
add.stratification.plot: no visible binding for global variable
  ‘Predicted’
add.stratification.plot: no visible binding for global variable
  ‘Annotated’
add.stratification.plot.immune: no visible global function definition
  for ‘%dopar%’
add.stratification.plot.immune: no visible global function definition
  for ‘foreach’
add.stratification.plot.immune: no visible binding for global variable
  ‘Group’
add.stratification.plot.immune: no visible binding for global variable
  ‘Immune’
addReportPlot.diffVar.volcano: no visible binding for global variable
  ‘var.diff’
addReportPlot.diffVar.volcano: no visible binding for global variable
  ‘log10P’
addReportPlot.diffVar.volcano: no visible binding for global variable
  ‘combinedRank.var’
addReportPlot.diffVar.volcano: no visible binding for global variable
  ‘log10FDR’
addReportPlots.diffMeth.bin.site.volcano: no visible binding for global
  variable ‘mean.diff’
addReportPlots.diffMeth.bin.site.volcano: no visible binding for global
  variable ‘diffmeth.p.val’
addReportPlots.diffMeth.bin.site.volcano: no visible binding for global
  variable ‘combinedRank’
addReportPlots.diffMeth.bin.site.volcano: no visible binding for global
  variable ‘mean.quot.log2’
addReportPlots.diffMeth.bin.site.volcano: no visible binding for global
  variable ‘diffmeth.p.adj.fdr’
agePredictorBiseq: no visible global function definition for
  ‘impute.knn’
agePredictorBiseq: no visible global function definition for ‘samples’
agePredictorChip: no visible global function definition for
  ‘impute.knn’
append.cpg.stats: no visible global function definition for ‘%dopar%’
append.cpg.stats: no visible global function definition for ‘foreach’
append.cpg.stats: no visible binding for global variable ‘chrom’
basic_combine: no visible global function definition for ‘samples’
computeDiffTab.default.region: no visible global function definition
  for ‘%dopar%’
computeDiffTab.default.region: no visible global function definition
  for ‘foreach’
computeDiffTab.default.region: no visible binding for global variable
  ‘i’
computeDiffTab.default.site: no visible binding for global variable
  ‘p.vals.t.na.adj’
computeDiffTab.default.site : &lt;anonymous&gt;: no visible binding for
  global variable ‘.inds.g2’
computeDiffVar.bin.site: no visible binding for global variable
  ‘p.vals.t.na.adj’
computeDiffVar.bin.site : &lt;anonymous&gt;: no visible binding for global
  variable ‘eps’
computeDiffVar.bin.site : &lt;anonymous&gt;: no visible binding for global
  variable ‘.inds.g2’
computeDiffVar.default.region: no visible global function definition
  for ‘%dopar%’
computeDiffVar.default.region: no visible global function definition
  for ‘foreach’
computeDiffVar.default.region: no visible binding for global variable
  ‘i’
create.densityScatter: no visible binding for global variable
  ‘..density..’
create.diffMeth.diffVar.subsample: no visible binding for global
  variable ‘..density..’
create.ucsc.track.hub: no visible global function definition for
  ‘samples’
createPredictor : ret: no visible global function definition for
  ‘impute.knn’
cv.array: no visible global function definition for ‘impute.knn’
data.frame2GRanges: no visible global function definition for
  ‘seqlevels&lt;-’
data.frame2GRanges: no visible global function definition for
  ‘genome&lt;-’
diffVar: no visible global function definition for ‘varFit’
diffVar: no visible binding for global variable ‘group1’
diffVar: no visible binding for global variable ‘group2’
estimateProportionsCP: no visible global function definition for ‘lme’
estimateProportionsCP: no visible global function definition for
  ‘getVarCov’
general.cv: no visible global function definition for ‘%dopar%’
general.cv: no visible global function definition for ‘foreach’
general.cv: no visible binding for global variable ‘i’
get.adjustment.variables: no visible global function definition for
  ‘samples’
get.comparison.info : rm.na.from.adj.tab: no visible global function
  definition for ‘samples’
get.components.isva: no visible global function definition for ‘DoISVA’
get.components.sva: no visible global function definition for ‘sva’
get.cpg.stats: no visible global function definition for
  ‘dinucleotideFrequency’
get.cpg.stats: no visible global function definition for
  ‘letterFrequency’
get.dataset.matrix: no visible global function definition for ‘samples’
get.diffmeth.tab.col.desc.list.txt: no visible binding for global
  variable ‘report’
get.diffmeth.tab.col.desc.list.txt: no visible binding for global
  variable ‘refText’
get.diffmeth.tab.col.desc.list.txt: no visible binding for global
  variable ‘diffmeth’
get.platform.tokens: no visible global function definition for
  ‘setNames’
getCGCounts: no visible global function definition for ‘seqlengths’
getCGCounts : &lt;anonymous&gt;: no visible global function definition for
  ‘ChrNumeric’
getGLADProfiles: no visible global function definition for ‘samples’
getGLADProfiles: no visible global function definition for ‘mclapply’
getGLADProfiles : &lt;anonymous&gt;: no visible global function definition
  for ‘as.profileCGH’
getGLADProfiles : &lt;anonymous&gt;: no visible global function definition
  for ‘daglad’
getMergeList: no visible global function definition for ‘samples’
groupPermutationP.site.parallel: no visible global function definition
  for ‘%dopar%’
groupPermutationP.site.parallel: no visible global function definition
  for ‘foreach’
iEVORA: no visible global function definition for ‘qvalue’
imputation.low.memory.cpgs: no visible global function definition for
  ‘samples’
inferWBCbyLme: no visible global function definition for ‘lme’
inferWBCbyLme: no visible global function definition for ‘getVarCov’
intensities.by.color: no visible binding for global variable
  ‘IlluminaHumanMethylationEPICmanifest’
intensities.by.color: no visible binding for global variable
  ‘IlluminaHumanMethylation450kmanifest’
intensities.by.color: no visible global function definition for
  ‘samples’
knn.imputation: no visible global function definition for ‘impute.knn’
limmaP: no visible binding for global variable ‘group1’
limmaP: no visible binding for global variable ‘group2’
loadLolaDbs: no visible global function definition for ‘loadRegionDB’
loadLolaDbs: no visible global function definition for ‘mergeRegionDBs’
locus.profile.get.base.tracks: no visible global function definition
  for ‘useMart’
locus.profile.get.base.tracks: no visible global function definition
  for ‘IdeogramTrack’
locus.profile.get.base.tracks: no visible global function definition
  for ‘GenomeAxisTrack’
locus.profile.get.base.tracks: no visible global function definition
  for ‘BiomartGeneRegionTrack’
locus.profile.get.base.tracks: no visible global function definition
  for ‘UcscTrack’
locus.profile.get.methylation.track.heatmap: no visible global function
  definition for ‘DataTrack’
locus.profile.get.methylation.track.smooth: no visible global function
  definition for ‘DataTrack’
methylumi.intensities.by.color: no visible global function definition
  for ‘featureNames’
methylumi.intensities.by.color: no visible binding for global variable
  ‘IlluminaHumanMethylation450kmanifest’
methylumi.intensities.by.color: no visible global function definition
  for ‘assayDataElement’
parallel.setup: no visible global function definition for
  ‘registerDoParallel’
parallel.setup: no visible global function definition for
  ‘getDoParWorkers’
performLolaEnrichment.diffMeth: no visible global function definition
  for ‘runLOLA’
performLolaEnrichment.diffVar: no visible global function definition
  for ‘runLOLA’
plot.heatmap.pc.correlations: no visible global function definition for
  ‘melt’
plot.heatmap.pc.correlations: no visible global function definition for
  ‘grid.newpage’
plot.heatmap.pc.correlations: no visible global function definition for
  ‘grid.draw’
plot.heatmap.pc.pvalues: no visible global function definition for
  ‘melt’
plot.heatmap.pc.pvalues: no visible global function definition for
  ‘grid.newpage’
plot.heatmap.pc.pvalues: no visible global function definition for
  ‘grid.draw’
plot.heatmap.rand: no visible global function definition for ‘melt’
plot.heatmap.symm: no visible global function definition for ‘melt’
plot.heatmap.symm: no visible global function definition for
  ‘grid.newpage’
plot.heatmap.symm: no visible global function definition for
  ‘grid.draw’
plotCGHProfile: no visible global function definition for ‘ChrNumeric’
prepareGEOSampleInfoTemplate: no visible global function definition for
  ‘samples’
projectWBC: no visible global function definition for ‘solve.QP’
read.GS.report: no visible global function definition for
  ‘featureNames’
read.idat.files: no visible binding for global variable ‘barcode’
rnb.RnBSet.to.GRangesList: no visible global function definition for
  ‘samples’
rnb.RnBSet.to.bed: no visible global function definition for ‘samples’
rnb.RnBSet.to.bedGraph: no visible global function definition for
  ‘samples’
rnb.bed.from.segmentation: no visible global function definition for
  ‘samples’
rnb.boxplot.from.segmentation: no visible global function definition
  for ‘samples’
rnb.boxplot.from.segmentation: no visible binding for global variable
  ‘Segment’
rnb.boxplot.from.segmentation: no visible binding for global variable
  ‘AvgMeth’
rnb.chromosome.lengths: no visible global function definition for
  ‘seqlengths’
rnb.combine.seq: no visible global function definition for ‘samples’
rnb.enmix.oob: no visible global function definition for ‘%dopar%’
rnb.enmix.oob: no visible global function definition for ‘foreach’
rnb.execute.dreduction: no visible global function definition for
  ‘samples’
rnb.execute.filter.summary.internal: no visible global function
  definition for ‘samples’
rnb.execute.high.coverage.removal.internal: no visible global function
  definition for ‘samples’
rnb.execute.high.dpval.masking.internal: no visible global function
  definition for ‘samples’
rnb.execute.imputation: no visible global function definition for
  ‘samples’
rnb.execute.low.coverage.masking.internal: no visible global function
  definition for ‘samples’
rnb.execute.na.removal.internal: no visible global function definition
  for ‘samples’
rnb.execute.normalization: no visible global function definition for
  ‘phenoData’
rnb.execute.normalization: no visible global function definition for
  ‘phenoData&lt;-’
rnb.execute.normalization: no visible global function definition for
  ‘RGChannelSet’
rnb.execute.normalization: no visible global function definition for
  ‘preprocessSWAN’
rnb.execute.normalization: no visible global function definition for
  ‘getMeth’
rnb.execute.normalization: no visible global function definition for
  ‘getUnmeth’
rnb.execute.normalization: no visible global function definition for
  ‘featureNames’
rnb.execute.normalization: no visible global function definition for
  ‘samples’
rnb.execute.normalization: no visible global function definition for
  ‘mapToGenome’
rnb.execute.normalization: no visible global function definition for
  ‘addSex’
rnb.execute.normalization: no visible global function definition for
  ‘getSex’
rnb.execute.normalization: no visible global function definition for
  ‘getCN’
rnb.execute.normalization.bmiq: no visible global function definition
  for ‘%dopar%’
rnb.execute.normalization.bmiq: no visible global function definition
  for ‘foreach’
rnb.execute.pOOBAH: no visible global function definition for ‘samples’
rnb.execute.pOOBAH: no visible global function definition for ‘SigDF’
rnb.execute.pOOBAH: no visible global function definition for ‘pOOBAH’
rnb.execute.pOOBAH: no visible binding for global variable ‘sigset.l’
rnb.execute.pOOBAH: no visible binding for global variable ‘pIG’
rnb.execute.pOOBAH: no visible binding for global variable ‘pII’
rnb.execute.pOOBAH: no visible binding for global variable ‘pIR’
rnb.execute.quality: no visible global function definition for
  ‘samples’
rnb.execute.quality : &lt;anonymous&gt;: no visible global function
  definition for ‘samples’
rnb.execute.segmentation: no visible binding for global variable
  ‘Hsapiens’
rnb.execute.segmentation: no visible binding for global variable
  ‘Mmusculus’
rnb.execute.segmentation: no visible global function definition for
  ‘seqlengths’
rnb.execute.segmentation: no visible global function definition for
  ‘browserSession’
rnb.execute.segmentation: no visible global function definition for
  ‘genome&lt;-’
rnb.execute.segmentation: no visible global function definition for
  ‘ucscTableQuery’
rnb.execute.segmentation: no visible global function definition for
  ‘getTable’
rnb.execute.segmentation: no visible global function definition for
  ‘samples’
rnb.execute.segmentation: no visible global function definition for
  ‘readMethylome’
rnb.execute.segmentation: no visible global function definition for
  ‘readSNPTable’
rnb.execute.segmentation: no visible global function definition for
  ‘removeSNPs’
rnb.execute.segmentation: no visible global function definition for
  ‘segmentPMDs’
rnb.execute.segmentation: no visible global function definition for
  ‘plotAlphaDistributionOneChr’
rnb.execute.segmentation: no visible global function definition for
  ‘calculateFDRs’
rnb.execute.segmentation: no visible global function definition for
  ‘segmentUMRsLMRs’
rnb.export.to.ewasher: no visible global function definition for
  ‘samples’
rnb.filter.dataset: no visible global function definition for ‘samples’
rnb.final.segmentation: no visible global function definition for
  ‘samples’
rnb.get.reliability.counts.per.sample: no visible global function
  definition for ‘samples’
rnb.options.description.table.fromRd: no visible global function
  definition for ‘Rd2HTML’
rnb.options.description.table.fromRd: no visible binding for global
  variable ‘xmlValue’
rnb.plot.beta.density.group: no visible binding for global variable
  ‘group’
rnb.plot.biseq.coverage: no visible global function definition for
  ‘samples’
rnb.plot.biseq.coverage.hist: no visible global function definition for
  ‘samples’
rnb.plot.control.barplot: no visible global function definition for
  ‘samples’
rnb.plot.control.barplot: no visible binding for global variable
  ‘Target’
rnb.plot.control.barplot: no visible binding for global variable
  ‘Index’
rnb.plot.control.barplot: no visible binding for global variable ‘ID’
rnb.plot.control.barplot: no visible binding for global variable
  ‘Address’
rnb.plot.control.barplot: no visible global function definition for
  ‘grid.newpage’
rnb.plot.control.barplot: no visible binding for global variable
  ‘Intensity’
rnb.plot.control.barplot: no visible global function definition for
  ‘grid.draw’
rnb.plot.control.boxplot: no visible global function definition for
  ‘grid.newpage’
rnb.plot.control.boxplot: no visible binding for global variable
  ‘Probe’
rnb.plot.control.boxplot: no visible binding for global variable
  ‘Intensity’
rnb.plot.control.boxplot: no visible global function definition for
  ‘grid.draw’
rnb.plot.ct.heatmap: no visible binding for global variable ‘x’
rnb.plot.ct.heatmap: no visible binding for global variable ‘y’
rnb.plot.ct.heatmap: no visible binding for global variable ‘v’
rnb.plot.ct.heatmap: no visible global function definition for
  ‘grid.newpage’
rnb.plot.ct.heatmap: no visible global function definition for
  ‘grid.draw’
rnb.plot.dreduction: no visible global function definition for ‘tsne’
rnb.plot.locus.profile: no visible global function definition for
  ‘plotTracks’
rnb.plot.marker.fstat: no visible binding for global variable ‘x’
rnb.plot.marker.fstat: no visible binding for global variable ‘y’
rnb.plot.marker.fstat: no visible global function definition for
  ‘muted’
rnb.plot.negative.boxplot: no visible global function definition for
  ‘samples’
rnb.plot.negative.boxplot: no visible binding for global variable
  ‘numeric.names’
rnb.plot.negative.boxplot: no visible binding for global variable
  ‘type’
rnb.plot.negative.boxplot: no visible binding for global variable
  ‘types’
rnb.plot.negative.boxplot: no visible global function definition for
  ‘grid.newpage’
rnb.plot.negative.boxplot: no visible binding for global variable
  ‘Sample’
rnb.plot.negative.boxplot: no visible binding for global variable
  ‘Intensity’
rnb.plot.negative.boxplot: no visible global function definition for
  ‘grid.draw’
rnb.plot.num.sites.covg: no visible global function definition for
  ‘samples’
rnb.plot.num.sites.covg : &lt;anonymous&gt;: no visible global function
  definition for ‘samples’
rnb.plot.num.sites.covg: no visible binding for global variable
  ‘numSites’
rnb.plot.num.sites.covg: no visible binding for global variable
  ‘covgMedian’
rnb.plot.num.sites.covg: no visible binding for global variable
  ‘covgPercLow’
rnb.plot.num.sites.covg: no visible binding for global variable
  ‘covgPercUp’
rnb.plot.num.sites.covg: no visible binding for global variable ‘comma’
rnb.plot.region.profile.density: no visible binding for global variable
  ‘relative.coord’
rnb.plot.region.profile.density: no visible binding for global variable
  ‘..density..’
rnb.plot.region.profiles : &lt;anonymous&gt;: no visible global function
  definition for ‘samples’
rnb.plot.region.profiles: no visible binding for global variable
  ‘relative.coord’
rnb.plot.region.profiles: no visible binding for global variable
  ‘group’
rnb.plot.region.site.density: no visible binding for global variable
  ‘relative.coord’
rnb.plot.segmentation.distributions: no visible binding for global
  variable ‘x2’
rnb.plot.segmentation.distributions: no visible binding for global
  variable ‘y2’
rnb.plot.segmentation.distributions: no visible binding for global
  variable ‘texthere’
rnb.plot.segmentation.final: no visible global function definition for
  ‘plotFinalSegmentation’
rnb.plot.sentrix.distributions: no visible global function definition
  for ‘samples’
rnb.plot.sentrix.distributions: no visible binding for global variable
  ‘Slide’
rnb.plot.snp.boxplot: no visible binding for global variable ‘SNP’
rnb.plot.snp.boxplot: no visible binding for global variable ‘bv’
rnb.run.example: no visible global function definition for ‘install’
rnb.run.preprocessing : logger.completed.filtering: no visible global
  function definition for ‘samples’
rnb.sample.summary.table: no visible global function definition for
  ‘samples’
rnb.section.ageprediction: no visible global function definition for
  ‘samples’
rnb.section.clustering : &lt;anonymous&gt; : &lt;anonymous&gt;: no visible global
  function definition for ‘samples’
rnb.section.clustering: no visible global function definition for
  ‘samples’
rnb.section.diffMeth.region: no visible global function definition for
  ‘%dopar%’
rnb.section.diffMeth.region: no visible global function definition for
  ‘foreach’
rnb.section.diffMeth.region: no visible binding for global variable ‘k’
rnb.section.diffMeth.region : do.enrichment.table: no visible global
  function definition for ‘sigCategories’
rnb.section.diffMeth.region : do.enrichment.table.var: no visible
  global function definition for ‘sigCategories’
rnb.section.diffMeth.site: no visible global function definition for
  ‘%dopar%’
rnb.section.diffMeth.site: no visible global function definition for
  ‘foreach’
rnb.section.diffVar: no visible global function definition for
  ‘%dopar%’
rnb.section.diffVar: no visible global function definition for
  ‘foreach’
rnb.section.diffVar.region: no visible global function definition for
  ‘%dopar%’
rnb.section.diffVar.region: no visible global function definition for
  ‘foreach’
rnb.section.diffVar.region: no visible binding for global variable ‘k’
rnb.section.import: no visible global function definition for ‘samples’
rnb.section.import: no visible global function definition for
  ‘phenoData’
rnb.section.imputation: no visible global function definition for
  ‘melt’
rnb.section.locus.profiles: no visible global function definition for
  ‘samples’
rnb.section.locus.profiles : do.plots: no visible global function
  definition for ‘plotTracks’
rnb.section.locus.profiles: no visible global function definition for
  ‘%dopar%’
rnb.section.locus.profiles: no visible global function definition for
  ‘foreach’
rnb.section.na.removal: no visible global function definition for
  ‘samples’
rnb.section.normalization: no visible global function definition for
  ‘samples’
rnb.section.region.description: no visible binding for global variable
  ‘size’
rnb.section.region.description: no visible binding for global variable
  ‘n.sites’
rnb.section.region.profiles: no visible global function definition for
  ‘samples’
rnb.section.region.subsegmentation : get.summary.df.from.list: no
  visible global function definition for ‘melt’
rnb.section.region.subsegmentation: no visible binding for global
  variable ‘region.size’
rnb.section.region.subsegmentation: no visible binding for global
  variable ‘is.subsegmentation’
rnb.section.region.subsegmentation: no visible binding for global
  variable ‘num.sites’
rnb.section.sex.prediction: no visible global function definition for
  ‘muted’
rnb.step.betadistribution.internal: no visible global function
  definition for ‘%dopar%’
rnb.step.betadistribution.internal: no visible global function
  definition for ‘foreach’
rnb.step.cell.types: no visible global function definition for
  ‘samples’
rnb.step.clustering.internal: no visible global function definition for
  ‘samples’
rnb.step.dreduction: no visible global function definition for
  ‘%dopar%’
rnb.step.dreduction: no visible global function definition for
  ‘foreach’
rnb.step.dreduction: no visible binding for global variable ‘target’
rnb.step.filter.summary.internal: no visible global function definition
  for ‘percent_format’
rnb.step.filter.summary.internal: no visible global function definition
  for ‘samples’
rnb.step.import: no visible global function definition for ‘samples’
rnb.step.import: no visible global function definition for ‘phenoData’
rnb.step.na.removal: no visible binding for global variable
  ‘sites2ignore’
rnb.step.na.removal.internal: no visible global function definition for
  ‘samples’
rnb.step.normalization: no visible global function definition for
  ‘phenoData’
robustHyperGResultSummary: no visible global function definition for
  ‘sigCategories’
robustHyperGResultSummary: no visible global function definition for
  ‘pvalues’
robustHyperGResultSummary: no visible global function definition for
  ‘oddsRatios’
robustHyperGResultSummary: no visible global function definition for
  ‘expectedCounts’
robustHyperGResultSummary: no visible global function definition for
  ‘geneCounts’
robustHyperGResultSummary: no visible global function definition for
  ‘universeCounts’
robustHyperGResultSummary: no visible binding for global variable
  ‘Term’
run.cross.validation: no visible global function definition for
  ‘samples’
run.cross.validation: no visible global function definition for ‘melt’
run.cross.validation: no visible binding for global variable ‘Measure’
run.cross.validation: no visible binding for global variable ‘Error’
sampleCovgApply: no visible global function definition for ‘samples’
set.covariates.ct: no visible global function definition for ‘samples’
simpleGlmnet: no visible global function definition for ‘impute.knn’
simpleGlmnet: no visible global function definition for ‘cv.glmnet’
simpleGlmnet: no visible global function definition for ‘glmnet’
simpleGlmnetBiseq: no visible global function definition for
  ‘cv.glmnet’
simpleGlmnetBiseq: no visible global function definition for ‘glmnet’
simpleGlmnetEvaluate: no visible global function definition for
  ‘cv.glmnet’
simpleGlmnetEvaluate: no visible global function definition for
  ‘glmnet’
symmetric.melt: no visible global function definition for ‘melt’
addDiffMethTable,RnBDiffMeth: no visible binding for global variable
  ‘reg.type’
coerce,RGChannelSet-RnBeadRawSet: no visible global function definition
  for ‘getManifest’
coerce,RGChannelSet-RnBeadRawSet: no visible global function definition
  for ‘getGreen’
coerce,RGChannelSet-RnBeadRawSet: no visible global function definition
  for ‘getRed’
coerce,RGChannelSet-RnBeadRawSet: no visible global function definition
  for ‘pData’
coerce,RnBeadRawSet-MethyLumiSet: no visible global function definition
  for ‘featureData&lt;-’
coerce,RnBeadRawSet-MethyLumiSet: no visible global function definition
  for ‘featureNames&lt;-’
coerce,RnBeadRawSet-RGChannelSet: no visible global function definition
  for ‘RGChannelSet’
join.diffMeth,RnBDiffMeth-RnBDiffMeth: no visible global function
  definition for ‘clone’
mask.sites.meth,RnBSet: no visible global function definition for
  ‘samples’
remove.samples,RnBSet: no visible global function definition for
  ‘samples’
remove.samples,RnBeadRawSet: no visible global function definition for
  ‘samples’
remove.samples,RnBeadSet: no visible global function definition for
  ‘samples’
remove.sites,RnBSet: no visible global function definition for
  ‘samples’
sampleCovgApply,RnBSet: no visible global function definition for
  ‘samples’
sampleMethApply,RnBSet: no visible global function definition for
  ‘samples’
summarize.regions,RnBSet: no visible global function definition for
  ‘samples’
summarize.regions,RnBSet: multiple local function definitions for
  ‘aggr.f’ with different formal arguments
updateMethylationSites,RnBSet: no visible global function definition
  for ‘samples’
Undefined global functions or variables:
  %dopar% ..count.. ..density.. .inds.g2 Address Age Annotated AvgMeth
  BiomartGeneRegionTrack ChrNumeric DataTrack Density Deviance
  Difference DoISVA Error GenomeAxisTrack Group Hsapiens ID
  IdeogramTrack IlluminaHumanMethylation450kmanifest
  IlluminaHumanMethylationEPICmanifest Immune Increase Index Intensity
  Measure Mmusculus Predicted Probe RGChannelSet Rd2HTML SNP Sample
  Segment SigDF Slide Target Term UcscTrack Value addSex as.profileCGH
  assayDataElement assayDataElementNames barcode browserSession bv
  calculateFDRs chrom clone combinedRank combinedRank.var comma
  covgMedian covgPercLow covgPercUp cv.glmnet daglad diffmeth
  diffmeth.p.adj.fdr diffmeth.p.val dinucleotideFrequency eps
  expectedCounts featureData featureData&lt;- featureNames featureNames&lt;-
  foreach geneCounts genome&lt;- getCN getDoParWorkers getGreen
  getManifest getMeth getRed getSex getTable getUnmeth getVarCov glmnet
  grid.draw grid.newpage group group1 group2 i impute.knn install
  is.subsegmentation k letterFrequency lme loadRegionDB log10FDR log10P
  mapToGenome mclapply mean.diff mean.quot.log2 melt mergeRegionDBs
  muted n.sites num.sites numSites numeric.names oddsRatios
  p.vals.t.na.adj pData pIG pII pIR pOOBAH percent_format phenoData
  phenoData&lt;- plotAlphaDistributionOneChr plotFinalSegmentation
  plotTracks preprocessSWAN pvalues qvalue readMethylome readSNPTable
  refText reg.type region.size registerDoParallel relative.coord
  removeSNPs report runLOLA samples segmentPMDs segmentUMRsLMRs
  seqlengths seqlevels&lt;- setNames sigCategories sigset.l sites2ignore
  size solve.QP stopCluster sva target texthere tsne type types
  ucscTableQuery universeCounts useMart v var.diff varFit varLabels x
  x2 xmlValue y y2 yint
Consider adding
  importFrom(&quot;stats&quot;, &quot;setNames&quot;)
to your NAMESPACE file.
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ...</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">RnBeads.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL RnBeads
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘RnBeads’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (RnBeads)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<HR>
<H3>Example timings</H3>
<P><SPAN class="filename">RnBeads.Rcheck/RnBeads-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>M-methods</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>U-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addDiffMethTable-RnBDiffMeth-methods</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addPheno-RnBSet-methods</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addRegionSubsegments</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotation-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>assembly-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>combine.rnb.sets-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>combineTestPvalsMeth</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>computeDiffTab.region</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>computeDiffTab.site</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>covg-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>create.densityScatter</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>create.scatter.dens.points</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>createReport</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>createReportPlot</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>downloadLolaDbs</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dpval-methods</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>exportDMRs2regionFile</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.adjustment.variables</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.comparison.grouplabels-RnBDiffMeth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.comparison.groupsizes-RnBDiffMeth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.comparison.info</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.comparisons-RnBDiffMeth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.covariates.sva</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.covg.thres-RnBDiffMeth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.files</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.region.types-RnBDiffMeth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.site.test.method-RnBDiffMeth-methods</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get.table-RnBDiffMeth-methods</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getCellTypesFromLolaDb</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getNamesFromLolaDb</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getSubCmdTokens-ClusterArchitectureSGE-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getSubCmdTokens-ClusterArchitectureSLURM-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getTargetFromLolaDb</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>has.covariates.sva</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>hasCovg-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>includes.sites-RnBDiffMeth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>is.valid-RnBDiffMeth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>join.diffMeth-methods</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>limmaP</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loadLolaDbs</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>logger.argument</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>logger.getfiles</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>logger.isinitialized</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>logger.validate.file</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loggerManagement</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loggerMessages</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>lolaBarPlot</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>lolaBoxPlotPerTarget</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>lolaVolcanoPlot</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mergeSamples-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>meth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mval-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nsites-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>parallel.getNumWorkers</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>parallel.isEnabled</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>parallel.setup</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>parallel.teardown</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>performGOEnrichment.diffVar</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>performGOenrichment.diffMeth.entrez</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>performGoEnrichment.diffMeth</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>performLolaEnrichment.diffMeth</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>performLolaEnrichment.diffVar</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pheno-methods</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qc-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>read.sample.annotation</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>regionMapping-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>regions-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>reload-RnBDiffMeth-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>remove.regions-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>remove.samples-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>remove.sites-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.RnBSet.to.GRangesList</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.RnBSet.to.bed</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.RnBSet.to.bedGraph</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.add.list</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.add.paragraph</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.add.reference</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.add.section</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.annotation.size</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.annotation2data.frame</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.beta2mval</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.batcheffects</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.clustering</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.computeDiffMeth</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.context.removal</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.cross.reactive.removal</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.dreduction</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.export.csv</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.greedycut</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.import</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.na.removal</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rnb.execute.normalization</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
