<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for RnBeads on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><B>[<A href="../index.html#R">R</A>]</B></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-01-29 16:34:11 -0000 (Sun, 29 Jan 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4021</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for RnBeads on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the RnBeads package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/RnBeads.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1696</B>/2162</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">RnBeads</A>&nbsp;2.17.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/RnBeads">landing page</A>)</SPAN><BR>Fabian Mueller<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-01-27&nbsp;13:09:27&nbsp;-0000&nbsp;(Fri,&nbsp;27&nbsp;Jan&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/RnBeads</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">99cf51e</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-01-03&nbsp;12:42:18&nbsp;-0000&nbsp;(Tue,&nbsp;03&nbsp;Jan&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: RnBeads</TD></TR>
<TR><TD><B>Version</B>: 2.17.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:RnBeads.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings RnBeads_2.17.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-01-29 07:52:47 -0000 (Sun, 29 Jan 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-01-29 08:03:47 -0000 (Sun, 29 Jan 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 660.0 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: RnBeads.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:RnBeads.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings RnBeads_2.17.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/RnBeads.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘RnBeads/DESCRIPTION’ ... OK
* this is package ‘RnBeads’ version ‘2.17.0’
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Depends: includes the non-default packages:
  &#x27;BiocGenerics&#x27;, &#x27;S4Vectors&#x27;, &#x27;GenomicRanges&#x27;, &#x27;MASS&#x27;, &#x27;cluster&#x27;,
  &#x27;ff&#x27;, &#x27;fields&#x27;, &#x27;ggplot2&#x27;, &#x27;gplots&#x27;, &#x27;grid&#x27;, &#x27;gridExtra&#x27;, &#x27;limma&#x27;,
  &#x27;matrixStats&#x27;, &#x27;illuminaio&#x27;, &#x27;methylumi&#x27;, &#x27;plyr&#x27;
Adding so many packages to the search path is excessive and importing
selectively is preferable.
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .travis.yml
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘RnBeads’ can be installed ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... NOTE
Package listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘qvalue’
A package should be listed in only one of these fields.
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... WARNING
Invalid citation information in ‘inst/CITATION’:
  Warning message:
  In utils::packageDescription(basename(dir), dirname(dir)) :
    no package &#x27;RnBeads&#x27; was found
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... NOTE
Warning: no function found corresponding to methods exports from ‘RnBeads’ for: ‘samples’

A namespace must be able to be loaded with just the base namespace
loaded: otherwise if the namespace gets loaded by a saved object, the
session will be unable to start.

Probably some imports need to be declared in the NAMESPACE file.
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... WARNING
Error in library(RnBeads, lib.loc = &quot;/home/biocbuild/bbs-3.17-bioc/R/library&quot;) : 
  there is no package called ‘RnBeads’
Execution halted

It looks like this package has a loading problem when not on .libPaths:
see the messages for details.
* checking dependencies in R code ... NOTE
Package in Depends field not imported from: &#x27;grid&#x27;
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.
Unexported objects imported by &#x27;:::&#x27; calls:
  &#x27;Gviz:::.getBMFeatureMap&#x27; &#x27;doParallel:::.options&#x27;
  &#x27;grDevices:::.smoothScatterCalcDensity&#x27;
  &#x27;minfi:::.default.450k.annotation&#x27; &#x27;minfi:::.extractFromRGSet450k&#x27;
  &#x27;minfi:::.normalizeFunnorm450k&#x27;
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
Killed
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... WARNING
Killed
All user-level objects in a package should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... SKIPPED
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’
 ERROR
Running the tests in ‘tests/runTests.R’ failed.
Last 13 lines of output:
  2023-01-29 08:00:24     1.4  STATUS                 STARTED Computing Differential Methylation Table
  2023-01-29 08:00:24     1.4    INFO                     Conducting differential analysis using limma
  2023-01-29 08:00:24     1.4  STATUS                 COMPLETED Computing Differential Methylation Table
  2023-01-29 08:00:24     1.4  STATUS                 STARTED Computing Differential Methylation Tables (Region Level)
  2023-01-29 08:00:26     1.4  STATUS                     Computed table for tiling
  2023-01-29 08:00:26     1.4  STATUS                     Computed table for genes
  2023-01-29 08:00:27     1.4  STATUS                     Computed table for promoters
  2023-01-29 08:00:27     1.4  STATUS                     Computed table for cpgislands
  2023-01-29 08:00:27     1.4  STATUS                 COMPLETED Computing Differential Methylation Tables (Region Level)
  2023-01-29 08:00:27     1.4  STATUS             COMPLETED Comparing: KOSR vs. TeSR (based on Treatment)
  2023-01-29 08:00:27     1.4  STATUS         COMPLETED Computing differential methylation tables
  2023-01-29 08:00:27     1.4  STATUS     COMPLETED Testing function: rnb.execute.computeDiffMeth
  2023-01-29 08:00:27     1.4  STATUS     STARTED Testing function: diffVar
  2023-01-29 08:00:27     1.4  STATUS         STARTED diffVar method
  Killed
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘RnBeads.Rnw’... failed to complete the test
  ‘RnBeads_Annotations.Rnw’... OK
 ERROR
Errors in running code in vignettes:
when running code in ‘RnBeads.Rnw’
  ...

&gt; f()
2023-01-29 08:01:24     1.6  STATUS     STARTED Another tedious task
2023-01-29 08:01:26     1.6  STATUS         STARTED Some tedious subtask
2023-01-29 08:01:28     1.6  STATUS         COMPLETED Some tedious subtask
2023-01-29 08:01:28     1.6  STATUS         STARTED Another, more tedious subtask
2023-01-29 08:01:31     1.6  STATUS             STARTED Some tedious subsubtask
Killed

... incomplete output.  Crash?

* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 2 ERRORs, 3 WARNINGs, 6 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/RnBeads.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">RnBeads.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL RnBeads
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/site-library’
* installing *source* package ‘RnBeads’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (RnBeads)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">RnBeads.Rcheck/tests/runTests.Rout.fail</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; BiocGenerics:::testPackage(&quot;RnBeads&quot;)

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min


Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname


Attaching package: &#x27;bit&#x27;

The following object is masked from &#x27;package:base&#x27;:

    xor

Attaching package ff
- getOption(&quot;fftempdir&quot;)==&quot;/tmp/RtmpiIJPiN/ff&quot;

- getOption(&quot;ffextension&quot;)==&quot;ff&quot;

- getOption(&quot;ffdrop&quot;)==TRUE

- getOption(&quot;fffinonexit&quot;)==TRUE

- getOption(&quot;ffpagesize&quot;)==65536

- getOption(&quot;ffcaching&quot;)==&quot;mmnoflush&quot;  -- consider &quot;ffeachflush&quot; if your system stalls on large writes

- getOption(&quot;ffbatchbytes&quot;)==16777216 -- consider a different value for tuning your system

- getOption(&quot;ffmaxbytes&quot;)==536870912 -- consider a different value for tuning your system


Attaching package: &#x27;ff&#x27;

The following objects are masked from &#x27;package:utils&#x27;:

    write.csv, write.csv2

The following objects are masked from &#x27;package:base&#x27;:

    is.factor, is.ordered

Spam version 2.9-1 (2022-08-07) is loaded.
Type &#x27;help( Spam)&#x27; or &#x27;demo( spam)&#x27; for a short introduction 
and overview of this package.
Help for individual functions is also obtained by adding the
suffix &#x27;.spam&#x27; to the function name, e.g. &#x27;help( chol.spam)&#x27;.

Attaching package: &#x27;spam&#x27;

The following object is masked from &#x27;package:stats4&#x27;:

    mle

The following objects are masked from &#x27;package:base&#x27;:

    backsolve, forwardsolve


Try help(fields) to get started.

Attaching package: &#x27;gplots&#x27;

The following object is masked from &#x27;package:IRanges&#x27;:

    space

The following object is masked from &#x27;package:S4Vectors&#x27;:

    space

The following object is masked from &#x27;package:stats&#x27;:

    lowess


Attaching package: &#x27;grid&#x27;

The following object is masked from &#x27;package:ff&#x27;:

    pattern


Attaching package: &#x27;gridExtra&#x27;

The following object is masked from &#x27;package:BiocGenerics&#x27;:

    combine


Attaching package: &#x27;limma&#x27;

The following object is masked from &#x27;package:BiocGenerics&#x27;:

    plotMA

Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


Attaching package: &#x27;Biobase&#x27;

The following objects are masked from &#x27;package:matrixStats&#x27;:

    anyMissing, rowMedians


Attaching package: &#x27;scales&#x27;

The following object is masked from &#x27;package:viridis&#x27;:

    viridis_pal


Attaching package: &#x27;AnnotationDbi&#x27;

The following object is masked from &#x27;package:MASS&#x27;:

    select



Attaching package: &#x27;MatrixGenerics&#x27;

The following object is masked from &#x27;package:Biobase&#x27;:

    rowMedians

The following objects are masked from &#x27;package:matrixStats&#x27;:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars


Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:grid&#x27;:

    pattern

The following objects are masked from &#x27;package:ff&#x27;:

    mismatch, pattern

The following object is masked from &#x27;package:base&#x27;:

    strsplit

locfit 1.5-9.7 	 2023-01-02
Setting options(&#x27;download.file.method.GEOquery&#x27;=&#x27;auto&#x27;)
Setting options(&#x27;GEOquery.inmemory.gpl&#x27;=FALSE)

Attaching package: &#x27;plyr&#x27;

The following object is masked from &#x27;package:XVector&#x27;:

    compact

The following object is masked from &#x27;package:matrixStats&#x27;:

    count

The following object is masked from &#x27;package:IRanges&#x27;:

    desc

The following object is masked from &#x27;package:S4Vectors&#x27;:

    rename

2023-01-29 08:00:18     1.1  STATUS STARTED Unit testing: differential
2023-01-29 08:00:18     1.1  STATUS     STARTED Testing function: rowWelchP
Loading required package: RnBeads.hg19
2023-01-29 08:00:18     1.1  STATUS     COMPLETED Testing function: rowWelchP
2023-01-29 08:00:18     1.1  STATUS     STARTED Testing function: limmaP
2023-01-29 08:00:18     1.1  STATUS     COMPLETED Testing function: limmaP
2023-01-29 08:00:18     1.1  STATUS     STARTED Testing function: computeDiffTab.extended.site
2023-01-29 08:00:18     1.1    INFO         Conducting differential analysis using limma
2023-01-29 08:00:18     1.1  STATUS     COMPLETED Testing function: computeDiffTab.extended.site
2023-01-29 08:00:18     1.1  STATUS     STARTED Testing function: computeDiffTab.default.region
2023-01-29 08:00:18     1.1    INFO         Conducting differential analysis using limma
2023-01-29 08:00:22     1.2  STATUS     COMPLETED Testing function: computeDiffTab.default.region
2023-01-29 08:00:22     1.2  STATUS     STARTED Testing function: combineTestPvalsMeth
2023-01-29 08:00:22     1.2  STATUS     COMPLETED Testing function: combineTestPvalsMeth
2023-01-29 08:00:22     1.2  STATUS     STARTED Testing function: get.adjustment.variables
2023-01-29 08:00:22     1.2  STATUS     COMPLETED Testing function: get.adjustment.variables
2023-01-29 08:00:22     1.2  STATUS     STARTED Testing function: get.comparison.info
2023-01-29 08:00:22     1.2  STATUS     COMPLETED Testing function: get.comparison.info
2023-01-29 08:00:22     1.2  STATUS     STARTED Testing function: rnb.execute.computeDiffMeth
2023-01-29 08:00:22     1.2  STATUS         STARTED Retrieving comparison info
2023-01-29 08:00:22     1.2  STATUS         COMPLETED Retrieving comparison info
2023-01-29 08:00:22     1.2  STATUS         STARTED Computing differential methylation tables
2023-01-29 08:00:22     1.2  STATUS             STARTED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-01-29 08:00:22     1.2  STATUS                 STARTED Computing Differential Methylation Table
2023-01-29 08:00:22     1.2    INFO                     Conducting differential analysis using limma
2023-01-29 08:00:22     1.2  STATUS                 COMPLETED Computing Differential Methylation Table
2023-01-29 08:00:22     1.2  STATUS                 STARTED Computing Differential Methylation Tables (Region Level)
2023-01-29 08:00:23     1.3  STATUS                     Computed table for tiling
2023-01-29 08:00:24     1.4  STATUS                     Computed table for genes
2023-01-29 08:00:24     1.4  STATUS                     Computed table for promoters
2023-01-29 08:00:24     1.4  STATUS                     Computed table for cpgislands
2023-01-29 08:00:24     1.4  STATUS                 COMPLETED Computing Differential Methylation Tables (Region Level)
2023-01-29 08:00:24     1.4  STATUS             COMPLETED Comparing: hESC vs. hiPSC (based on Sample_Group)
2023-01-29 08:00:24     1.4  STATUS             STARTED Comparing: KOSR vs. TeSR (based on Treatment)
2023-01-29 08:00:24     1.4  STATUS                 STARTED Computing Differential Methylation Table
2023-01-29 08:00:24     1.4    INFO                     Conducting differential analysis using limma
2023-01-29 08:00:24     1.4  STATUS                 COMPLETED Computing Differential Methylation Table
2023-01-29 08:00:24     1.4  STATUS                 STARTED Computing Differential Methylation Tables (Region Level)
2023-01-29 08:00:26     1.4  STATUS                     Computed table for tiling
2023-01-29 08:00:26     1.4  STATUS                     Computed table for genes
2023-01-29 08:00:27     1.4  STATUS                     Computed table for promoters
2023-01-29 08:00:27     1.4  STATUS                     Computed table for cpgislands
2023-01-29 08:00:27     1.4  STATUS                 COMPLETED Computing Differential Methylation Tables (Region Level)
2023-01-29 08:00:27     1.4  STATUS             COMPLETED Comparing: KOSR vs. TeSR (based on Treatment)
2023-01-29 08:00:27     1.4  STATUS         COMPLETED Computing differential methylation tables
2023-01-29 08:00:27     1.4  STATUS     COMPLETED Testing function: rnb.execute.computeDiffMeth
2023-01-29 08:00:27     1.4  STATUS     STARTED Testing function: diffVar
2023-01-29 08:00:27     1.4  STATUS         STARTED diffVar method
Killed
</PRE>
</DIV><HR>
<H3>Example timings</H3>
</BODY>
</HTML>
