<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for wateRmelon on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><B>[<A href="../index.html#W">W</A>]</B></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-09 02:50:58 -0000 (Thu, 09 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4165</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for wateRmelon on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the wateRmelon package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/wateRmelon.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>2139</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">wateRmelon</A>&nbsp;2.5.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/wateRmelon">landing page</A>)</SPAN><BR>Leo C Schalkwyk<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-08&nbsp;03:21:00&nbsp;-0000&nbsp;(Wed,&nbsp;08&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/wateRmelon</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">7cac6c6</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:07:41&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: wateRmelon</TD></TR>
<TR><TD><B>Version</B>: 2.5.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:wateRmelon.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings wateRmelon_2.5.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-09 01:30:55 -0000 (Thu, 09 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-09 01:39:26 -0000 (Thu, 09 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 510.2 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: wateRmelon.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:wateRmelon.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings wateRmelon_2.5.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/wateRmelon.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘wateRmelon/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘wateRmelon’ version ‘2.5.0’
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Depends: includes the non-default packages:
  &#x27;Biobase&#x27;, &#x27;limma&#x27;, &#x27;matrixStats&#x27;, &#x27;methylumi&#x27;, &#x27;lumi&#x27;, &#x27;ROC&#x27;,
  &#x27;IlluminaHumanMethylation450kanno.ilmn12.hg19&#x27;, &#x27;illuminaio&#x27;
Adding so many packages to the search path is excessive and importing
selectively is preferable.
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘wateRmelon’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... NOTE
Package listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘Biobase’
A package should be listed in only one of these fields.
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... NOTE
Problems with news in ‘NEWS’:
  Cannot process chunk/lines:
    1.19.1 Updated normalization methods to account for change in minfi MethylSet structure
           and methods will return &#x27;MethylSet&#x27; instead of beta matrix.
  Cannot process chunk/lines:
    1.3.1	Changed annotation method for tost() (minfi and methylumi methods.
    	thanks to several users for bug reports!
  Cannot process chunk/lines:
    0.99.16	replaced dependency on ROCR with ROC (bioconductor) package
    	seabird() no longer takes a vector for the stop argument
  Cannot process chunk/lines:
    0.99.15	codoc corrections.  
    	temporarily removed dependency on ROCR (and thus gdata)
  Cannot process chunk/lines:
    0.99.14	MethylSet methods [dn]a[st]e[nt] now return objects containing 
    	normalized intensities as well as betas
    	fixed a bug in as.methylumi
  Cannot process chunk/lines:
    0.99.13	fixed swan wrapper generic and MethyLumiSet method bugs 
  Cannot process chunk/lines:
    0.99.12	methods d**** now have ... argument so that data without
    	Sentrix IDs should work (thanks to Elodie Portales-Casamar
    	for bug report).
  Cannot process chunk/lines:
    0.99.11	RGChannelSetExtended pfilter method bug fixed
  Cannot process chunk/lines:
    0.99.10	BioConductor submission
  Cannot process chunk/lines:
    0.9.8	MethyLumiSet methods now update history slot
    	new MethyLumiSet method for pfilter
    	RGtoy2 removed from data(minfitoy).  This is because minfi 
    	now expects the manifest to be a package.
    	swan function patched so that it can process subsets of array data
    	(provided by Jovana Maksimovic).  Now works with melon data set.
    	as.methylumi generic function added. Gets fData from 
    	package:IlluminaHumanMethylation450k.db . The MethyLumiSet method 
    	is useful for ensuring methylumi objects contain this data.
    	Andrew Teschendorff&#x27;s BMIQ function and a MethyLumiSet method
    	added 
    	minfi 1.4.0 has a changed manifest structure resulting in
    	SNP probes being left out of the MethylSet.  This breaks genki()
    	for the minfi objects and also the minfitoy data() set. 
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
Packages in Depends field not imported from:
  ‘IlluminaHumanMethylation450kanno.ilmn12.hg19’ ‘matrixStats’
  ‘methylumi’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.
Unexported objects imported by &#x27;:::&#x27; calls:
  ‘minfi:::.annoGet’ ‘minfi:::.availableAnnotation’
  ‘minfi:::.isRGOrStop’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
.adjusted_normalizeFunnorm450k : unbiased_normalizeQuantiles: no
  visible global function definition for ‘colQuantiles’
.adjusted_normalizeFunnorm450k: no visible global function definition
  for ‘getMeth’
.adjusted_normalizeFunnorm450k: no visible global function definition
  for ‘getUnmeth’
.adjusted_normalizeFunnorm450k: no visible global function definition
  for ‘assay&lt;-’
.buildControlMatrix450k: no visible global function definition for
  ‘colMeans2’
.extractFromRGSet450k: no visible global function definition for
  ‘getProbeInfo’
.extractFromRGSet450k: no visible global function definition for
  ‘getRed’
.extractFromRGSet450k: no visible global function definition for
  ‘getGreen’
.extractFromRGSet450k: no visible global function definition for
  ‘getOOB’
.extractFromRGSet450k: no visible global function definition for
  ‘colQuantiles’
.getFunnormIndices: no visible global function definition for
  ‘getProbeType’
.getFunnormIndices: no visible global function definition for
  ‘seqnames’
.isMatrixBacked: no visible global function definition for ‘assays’
.regularizeQuantiles: no visible global function definition for
  ‘colCummaxs’
.returnFit: no visible global function definition for ‘rowMeans2’
IDATsToMatrices2: no visible global function definition for ‘.mclapply’
NChannelSetToMethyLumiSet2: no visible global function definition for
  ‘betas’
NChannelSetToMethyLumiSet2: no visible global function definition for
  ‘methylated’
NChannelSetToMethyLumiSet2: no visible global function definition for
  ‘unmethylated’
NChannelSetToMethyLumiSet2: no visible global function definition for
  ‘pval.detect&lt;-’
adjustedFunnorm: no visible global function definition for
  ‘preprocessNoob’
adjustedFunnorm: no visible global function definition for
  ‘mapToGenome’
adjustedFunnorm: no visible global function definition for ‘addSex’
adjustedFunnorm: no visible global function definition for ‘getSex’
adjustedFunnorm: no visible global function definition for ‘getCN’
adjustedFunnorm: no visible global function definition for ‘assay&lt;-’
beadcount: no visible global function definition for ‘getNBeads’
beadcount: no visible global function definition for ‘getProbeInfo’
beadcount: no visible global function definition for ‘getManifestInfo’
bgIntensitySwan.methylumi: no visible global function definition for
  ‘colMedians’
bscon_methy: no visible global function definition for
  ‘intensitiesByChannel’
bscon_methy: no visible global function definition for ‘QCdata’
bscon_minfi : csp.green: no visible global function definition for
  ‘getRed’
bscon_minfi : csp.green: no visible global function definition for
  ‘getGreen’
bscon_minfi : csp.green : &lt;anonymous&gt;: no visible global function
  definition for ‘getControlAddress’
bscon_minfi : csp.red: no visible global function definition for
  ‘getRed’
bscon_minfi : csp.red: no visible global function definition for
  ‘getGreen’
bscon_minfi : csp.red : &lt;anonymous&gt;: no visible global function
  definition for ‘getControlAddress’
combo : &lt;anonymous&gt;: no visible global function definition for ‘QCdata’
combo: no visible global function definition for ‘QCdata&lt;-’
dfort: no visible binding for global variable
  ‘IlluminaHumanMethylation450kanno.ilmn12.hg19’
estimateSex: no visible binding for global variable ‘sex_coef’
estimateSex: no visible global function definition for ‘colSds’
genall: no visible global function definition for ‘betas’
generateManifest: no visible global function definition for
  ‘getAnnotationObject’
generateManifest: no visible global function definition for
  ‘getAnnotation’
generateManifest: no visible global function definition for
  ‘getProbeInfo’
getMethylationBeadMappers2 : &lt;anonymous&gt;: no visible binding for global
  variable ‘hm27.ordering’
getMethylationBeadMappers2 : &lt;anonymous&gt;: no visible binding for global
  variable ‘hm450.ordering’
getMethylationBeadMappers2 : &lt;anonymous&gt;: no visible binding for &#x27;&lt;&lt;-&#x27;
  assignment to ‘epic.ordering’
getMethylationBeadMappers2 : &lt;anonymous&gt;: no visible binding for global
  variable ‘epic.ordering’
getMethylationBeadMappers2 : &lt;anonymous&gt;: no visible binding for global
  variable ‘hm27.controls’
getMethylationBeadMappers2 : &lt;anonymous&gt;: no visible binding for global
  variable ‘hm450.controls’
getMethylationBeadMappers2 : &lt;anonymous&gt;: no visible binding for global
  variable ‘epic.controls’
getMethylumiBeta: no visible global function definition for
  ‘unmethylated’
getMethylumiBeta: no visible global function definition for
  ‘methylated’
got: no visible global function definition for ‘getProbeInfo’
lumiMethyR2: no visible global function definition for ‘methylumiR’
oxyscale: no visible global function definition for ‘betas’
oxyscale: no visible global function definition for ‘betas&lt;-’
p_dfsfit: no visible global function definition for ‘mclapply’
pipelineIlluminaMethylation.batch: no visible binding for global
  variable ‘PATH_RES’
pipelineIlluminaMethylation.batch: no visible binding for global
  variable ‘probeAnnotationsCategory’
preprocessIlluminaMethylation: no visible binding for global variable
  ‘projectName’
readEPIC: no visible binding for global variable ‘n.sd’
sort_order: no visible binding for global variable ‘i’
sort_order: no visible binding for global variable ‘n_1’
uSexQNengine: no visible global function definition for ‘mclapply’
BMIQ,MethyLumiSet : &lt;anonymous&gt;: no visible global function definition
  for ‘warn’
BMIQ,MethyLumiSet: no visible global function definition for ‘betas&lt;-’
BMIQ,MethylSet: no visible global function definition for ‘getBeta’
BMIQ,MethylSet : &lt;anonymous&gt;: no visible global function definition for
  ‘warn’
adjustedDasen,MethyLumiSet: no visible global function definition for
  ‘methylated’
adjustedDasen,MethyLumiSet: no visible global function definition for
  ‘unmethylated’
adjustedDasen,MethyLumiSet: no visible binding for global variable ‘ds’
adjustedDasen,MethyLumiSet: no visible global function definition for
  ‘betas&lt;-’
adjustedDasen,MethyLumiSet: no visible global function definition for
  ‘methylated&lt;-’
adjustedDasen,MethyLumiSet: no visible global function definition for
  ‘unmethylated&lt;-’
adjustedDasen,MethylSet: no visible global function definition for
  ‘getMeth’
adjustedDasen,MethylSet: no visible global function definition for
  ‘getUnmeth’
adjustedDasen,MethylSet: no visible global function definition for
  ‘getAnnotation’
adjustedDasen,MethylSet: no visible global function definition for
  ‘MethylSet’
adjustedDasen,MethylSet: no visible global function definition for
  ‘colData’
adjustedDasen,MethylSet: no visible global function definition for
  ‘metadata’
adjustedDasen,RGChannelSet: no visible global function definition for
  ‘preprocessRaw’
agep,MethylSet: no visible global function definition for ‘getBeta’
as.methylumi,MethyLumiSet: no visible global function definition for
  ‘methylated’
as.methylumi,MethyLumiSet: no visible global function definition for
  ‘unmethylated’
as.methylumi,MethyLumiSet: no visible global function definition for
  ‘betas’
as.methylumi,MethyLumiSet: no visible global function definition for
  ‘pvals’
as.methylumi,MethyLumiSet: no visible global function definition for
  ‘QCdata’
as.methylumi,MethylSet: no visible global function definition for
  ‘getMeth’
as.methylumi,MethylSet: no visible global function definition for
  ‘getUnmeth’
as.methylumi,MethylSet: no visible global function definition for
  ‘getBeta’
as.methylumi,MethylSet: no visible global function definition for
  ‘getAnnotation’
betaqn,MethyLumiSet: no visible global function definition for
  ‘betas&lt;-’
betaqn,MethyLumiSet: no visible global function definition for ‘betas’
betaqn,MethylSet: no visible global function definition for ‘getBeta’
betaqn,RGChannelSet: no visible global function definition for
  ‘getBeta’
colnames,MethyLumiSet: no visible global function definition for
  ‘methylated’
danen,MethyLumiSet: no visible global function definition for
  ‘methylated’
danen,MethyLumiSet: no visible global function definition for
  ‘unmethylated’
danen,MethyLumiSet: no visible global function definition for ‘betas&lt;-’
danen,MethyLumiSet: no visible global function definition for
  ‘methylated&lt;-’
danen,MethyLumiSet: no visible global function definition for
  ‘unmethylated&lt;-’
danen,MethylSet: no visible global function definition for ‘getMeth’
danen,MethylSet: no visible global function definition for ‘getUnmeth’
danen,MethylSet: no visible global function definition for ‘MethylSet’
danen,MethylSet: no visible global function definition for ‘colData’
danen,MethylSet: no visible global function definition for ‘metadata’
danen,RGChannelSet: no visible global function definition for
  ‘preprocessRaw’
danes,MethyLumiSet: no visible global function definition for
  ‘methylated’
danes,MethyLumiSet: no visible global function definition for
  ‘unmethylated’
danes,MethyLumiSet: no visible global function definition for ‘betas&lt;-’
danes,MethyLumiSet: no visible global function definition for
  ‘methylated&lt;-’
danes,MethyLumiSet: no visible global function definition for
  ‘unmethylated&lt;-’
danes,MethylSet: no visible global function definition for ‘getMeth’
danes,MethylSet: no visible global function definition for ‘getUnmeth’
danes,MethylSet: no visible global function definition for ‘MethylSet’
danes,MethylSet: no visible global function definition for ‘colData’
danes,MethylSet: no visible global function definition for ‘metadata’
danes,RGChannelSet: no visible global function definition for
  ‘preprocessRaw’
danet,MethyLumiSet: no visible global function definition for
  ‘methylated’
danet,MethyLumiSet: no visible global function definition for
  ‘unmethylated’
danet,MethyLumiSet: no visible global function definition for ‘betas&lt;-’
danet,MethyLumiSet: no visible global function definition for
  ‘methylated&lt;-’
danet,MethyLumiSet: no visible global function definition for
  ‘unmethylated&lt;-’
danet,MethylSet: no visible global function definition for ‘getMeth’
danet,MethylSet: no visible global function definition for ‘getUnmeth’
danet,MethylSet: no visible global function definition for ‘MethylSet’
danet,MethylSet: no visible global function definition for ‘colData’
danet,MethylSet: no visible global function definition for ‘metadata’
danet,RGChannelSet: no visible global function definition for
  ‘preprocessRaw’
dasen,MethyLumiSet: no visible global function definition for
  ‘methylated’
dasen,MethyLumiSet: no visible global function definition for
  ‘unmethylated’
dasen,MethyLumiSet: no visible global function definition for ‘betas&lt;-’
dasen,MethyLumiSet: no visible global function definition for
  ‘methylated&lt;-’
dasen,MethyLumiSet: no visible global function definition for
  ‘unmethylated&lt;-’
dasen,MethylSet: no visible global function definition for ‘getMeth’
dasen,MethylSet: no visible global function definition for ‘getUnmeth’
dasen,MethylSet: no visible global function definition for ‘MethylSet’
dasen,MethylSet: no visible global function definition for ‘colData’
dasen,MethylSet: no visible global function definition for ‘metadata’
dasen,RGChannelSet: no visible global function definition for
  ‘preprocessRaw’
daten1,MethyLumiSet: no visible global function definition for
  ‘methylated’
daten1,MethyLumiSet: no visible global function definition for
  ‘unmethylated’
daten1,MethyLumiSet: no visible global function definition for
  ‘betas&lt;-’
daten1,MethyLumiSet: no visible global function definition for
  ‘methylated&lt;-’
daten1,MethyLumiSet: no visible global function definition for
  ‘unmethylated&lt;-’
daten1,MethylSet: no visible global function definition for ‘getMeth’
daten1,MethylSet: no visible global function definition for ‘getUnmeth’
daten1,MethylSet: no visible global function definition for ‘MethylSet’
daten1,MethylSet: no visible global function definition for ‘colData’
daten1,MethylSet: no visible global function definition for ‘metadata’
daten1,RGChannelSet: no visible global function definition for
  ‘preprocessRaw’
daten2,MethyLumiSet: no visible global function definition for
  ‘methylated’
daten2,MethyLumiSet: no visible global function definition for
  ‘unmethylated’
daten2,MethyLumiSet: no visible global function definition for
  ‘betas&lt;-’
daten2,MethyLumiSet: no visible global function definition for
  ‘methylated&lt;-’
daten2,MethyLumiSet: no visible global function definition for
  ‘unmethylated&lt;-’
daten2,MethylSet: no visible global function definition for ‘getMeth’
daten2,MethylSet: no visible global function definition for ‘getUnmeth’
daten2,MethylSet: no visible global function definition for ‘MethylSet’
daten2,MethylSet: no visible global function definition for ‘colData’
daten2,MethylSet: no visible global function definition for ‘metadata’
daten2,RGChannelSet: no visible global function definition for
  ‘preprocessRaw’
dmrse,MethylSet: no visible global function definition for ‘getBeta’
dmrse,RGChannelSet: no visible global function definition for ‘getBeta’
dmrse_col,MethylSet: no visible global function definition for
  ‘getBeta’
dmrse_col,RGChannelSet: no visible global function definition for
  ‘getBeta’
dmrse_row,MethylSet: no visible global function definition for
  ‘getBeta’
dmrse_row,RGChannelSet: no visible global function definition for
  ‘getBeta’
estimateCellCounts.wateRmelon,MethylSet: no visible global function
  definition for ‘getManifest’
estimateCellCounts.wateRmelon,MethylSet: no visible global function
  definition for ‘getMeth’
estimateCellCounts.wateRmelon,MethylSet: no visible global function
  definition for ‘getUnmeth’
estimateCellCounts.wateRmelon,MethylSet: no visible global function
  definition for ‘getBeta’
estimateCellCounts.wateRmelon,RGChannelSet: no visible global function
  definition for ‘getManifest’
estimateCellCounts.wateRmelon,RGChannelSet: no visible global function
  definition for ‘preprocessRaw’
estimateCellCounts.wateRmelon,RGChannelSet: no visible global function
  definition for ‘getMeth’
estimateCellCounts.wateRmelon,RGChannelSet: no visible global function
  definition for ‘getUnmeth’
estimateCellCounts.wateRmelon,RGChannelSet: no visible global function
  definition for ‘getBeta’
fuks,MethyLumiSet: no visible global function definition for ‘betas&lt;-’
fuks,MethyLumiSet: no visible global function definition for ‘betas’
fuks,MethylSet: no visible global function definition for ‘getBeta’
fuks,RGChannelSet: no visible global function definition for
  ‘preprocessRaw’
fuks,RGChannelSet: no visible global function definition for ‘getBeta’
genki,MethyLumiSet: no visible global function definition for ‘betas’
genki,MethylSet: no visible global function definition for ‘getBeta’
genki,RGChannelSet: no visible global function definition for
  ‘getSnpBeta’
nanes,MethyLumiSet: no visible global function definition for
  ‘methylated’
nanes,MethyLumiSet: no visible global function definition for
  ‘unmethylated’
nanes,MethyLumiSet: no visible global function definition for ‘betas&lt;-’
nanes,MethyLumiSet: no visible global function definition for
  ‘methylated&lt;-’
nanes,MethyLumiSet: no visible global function definition for
  ‘unmethylated&lt;-’
nanes,MethylSet: no visible global function definition for ‘getMeth’
nanes,MethylSet: no visible global function definition for ‘getUnmeth’
nanes,MethylSet: no visible global function definition for ‘MethylSet’
nanes,MethylSet: no visible global function definition for ‘colData’
nanes,MethylSet: no visible global function definition for ‘metadata’
nanes,RGChannelSet: no visible global function definition for
  ‘preprocessRaw’
nanet,MethyLumiSet: no visible global function definition for
  ‘methylated’
nanet,MethyLumiSet: no visible global function definition for
  ‘unmethylated’
nanet,MethyLumiSet: no visible global function definition for ‘betas&lt;-’
nanet,MethyLumiSet: no visible global function definition for
  ‘methylated&lt;-’
nanet,MethyLumiSet: no visible global function definition for
  ‘unmethylated&lt;-’
nanet,MethylSet: no visible global function definition for ‘getMeth’
nanet,MethylSet: no visible global function definition for ‘getUnmeth’
nanet,MethylSet: no visible global function definition for ‘MethylSet’
nanet,MethylSet: no visible global function definition for ‘colData’
nanet,MethylSet: no visible global function definition for ‘metadata’
nanet,RGChannelSet: no visible global function definition for
  ‘preprocessRaw’
nasen,MethyLumiSet: no visible global function definition for
  ‘methylated’
nasen,MethyLumiSet: no visible global function definition for
  ‘unmethylated’
nasen,MethyLumiSet: no visible global function definition for ‘betas&lt;-’
nasen,MethyLumiSet: no visible global function definition for
  ‘methylated&lt;-’
nasen,MethyLumiSet: no visible global function definition for
  ‘unmethylated&lt;-’
nasen,MethylSet: no visible global function definition for ‘getMeth’
nasen,MethylSet: no visible global function definition for ‘getUnmeth’
nasen,MethylSet: no visible global function definition for ‘MethylSet’
nasen,MethylSet: no visible global function definition for ‘colData’
nasen,MethylSet: no visible global function definition for ‘metadata’
nasen,RGChannelSet: no visible global function definition for
  ‘preprocessRaw’
naten,MethyLumiSet: no visible global function definition for
  ‘methylated’
naten,MethyLumiSet: no visible global function definition for
  ‘unmethylated’
naten,MethyLumiSet: no visible global function definition for ‘betas&lt;-’
naten,MethyLumiSet: no visible global function definition for
  ‘methylated&lt;-’
naten,MethyLumiSet: no visible global function definition for
  ‘unmethylated&lt;-’
naten,MethylSet: no visible global function definition for ‘getMeth’
naten,MethylSet: no visible global function definition for ‘getUnmeth’
naten,MethylSet: no visible global function definition for ‘MethylSet’
naten,MethylSet: no visible global function definition for ‘colData’
naten,MethylSet: no visible global function definition for ‘metadata’
naten,RGChannelSet: no visible global function definition for
  ‘preprocessRaw’
outlyx,MethyLumiSet: no visible global function definition for ‘betas’
outlyx,MethylSet: no visible global function definition for ‘getBeta’
outlyx,RGChannelSet: no visible global function definition for
  ‘getBeta’
pfilter,MethyLumiSet: no visible global function definition for ‘betas’
pfilter,MethyLumiSet: no visible global function definition for
  ‘methylated’
pfilter,MethyLumiSet: no visible global function definition for
  ‘unmethylated’
pfilter,MethyLumiSet: no visible global function definition for ‘pvals’
pfilter,RGChannelSetExtended: no visible binding for global variable
  ‘RGChannelSetExtended’
pfilter,RGChannelSetExtended: no visible global function definition for
  ‘preprocessRaw’
pfilter,RGChannelSetExtended: no visible global function definition for
  ‘detectionP’
pfilter,RGChannelSetExtended: no visible global function definition for
  ‘subsetByLoci’
pwod,MethyLumiSet: no visible global function definition for ‘betas’
pwod,MethyLumiSet: no visible global function definition for ‘betas&lt;-’
pwod,MethylSet: no visible global function definition for ‘getBeta’
pwod,RGChannelSet: no visible global function definition for ‘getBeta’
seabi,MethyLumiSet: no visible global function definition for ‘betas’
seabi,MethylSet: no visible global function definition for ‘getBeta’
seabi,RGChannelSet: no visible global function definition for ‘getBeta’
swan,MethyLumiSet: no visible global function definition for
  ‘methylated’
swan,MethyLumiSet: no visible global function definition for
  ‘unmethylated’
swan,MethyLumiSet: no visible global function definition for
  ‘intensitiesByChannel’
swan,MethyLumiSet: no visible global function definition for ‘QCdata’
swan,MethyLumiSet: no visible global function definition for ‘getBeta’
swan,MethyLumiSet: no visible global function definition for ‘getMeth’
swan,MethyLumiSet: no visible global function definition for
  ‘getUnmeth’
swan,MethyLumiSet: no visible global function definition for ‘betas’
swan,RGChannelSet: no visible global function definition for
  ‘preprocessSWAN’
tost,MethyLumiSet: no visible global function definition for
  ‘methylated’
tost,MethyLumiSet: no visible global function definition for ‘betas&lt;-’
tost,MethyLumiSet: no visible global function definition for
  ‘unmethylated’
tost,MethyLumiSet: no visible global function definition for ‘pvals’
tost,RGChannelSet: no visible global function definition for
  ‘detectionP’
tost,RGChannelSet: no visible global function definition for
  ‘preprocessRaw’
tost,RGChannelSet: no visible global function definition for ‘getMeth’
tost,RGChannelSet: no visible global function definition for
  ‘getUnmeth’
uSexQN,MethyLumiSet: no visible global function definition for
  ‘methylated’
uSexQN,MethyLumiSet: no visible global function definition for
  ‘unmethylated’
uSexQN,MethyLumiSet: no visible global function definition for
  ‘betas&lt;-’
uSexQN,MethyLumiSet: no visible global function definition for
  ‘methylated&lt;-’
uSexQN,MethyLumiSet: no visible global function definition for
  ‘unmethylated&lt;-’
uSexQN,MethylSet: no visible global function definition for ‘getMeth’
uSexQN,MethylSet: no visible global function definition for ‘getUnmeth’
uSexQN,MethylSet: no visible global function definition for
  ‘getAnnotation’
uSexQN,MethylSet: no visible global function definition for ‘MethylSet’
uSexQN,MethylSet: no visible global function definition for ‘colData’
uSexQN,MethylSet: no visible global function definition for ‘metadata’
uSexQN,RGChannelSet: no visible global function definition for
  ‘preprocessRaw’
Undefined global functions or variables:
  .mclapply IlluminaHumanMethylation450kanno.ilmn12.hg19 MethylSet
  PATH_RES QCdata QCdata&lt;- RGChannelSetExtended addSex assay&lt;- assays
  betas betas&lt;- colCummaxs colData colMeans2 colMedians colQuantiles
  colSds detectionP ds epic.controls epic.ordering getAnnotation
  getAnnotationObject getBeta getCN getControlAddress getGreen
  getManifest getManifestInfo getMeth getNBeads getOOB getProbeInfo
  getProbeType getRed getSex getSnpBeta getUnmeth hm27.controls
  hm27.ordering hm450.controls hm450.ordering i intensitiesByChannel
  mapToGenome mclapply metadata methylated methylated&lt;- methylumiR n.sd
  n_1 preprocessNoob preprocessRaw preprocessSWAN
  probeAnnotationsCategory projectName pval.detect&lt;- pvals rowMeans2
  seqnames sex_coef subsetByLoci unmethylated unmethylated&lt;- warn
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
          user system elapsed
sextest 10.584  0.092  10.676
seabi   10.088  0.132  10.220
metrics  6.254  0.132   6.386
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘wateRmelon.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 5 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/wateRmelon.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">wateRmelon.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL wateRmelon
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘wateRmelon’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
No methods found in package ‘RSQLite’ for request: ‘dbListFields’ when loading ‘lumi’
Creating a generic function from function ‘as.methylumi’ in package ‘wateRmelon’
Creating a generic function from function ‘adjustedDasen’ in package ‘wateRmelon’
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
No methods found in package ‘RSQLite’ for request: ‘dbListFields’ when loading ‘lumi’
** testing if installed package can be loaded from final location
No methods found in package ‘RSQLite’ for request: ‘dbListFields’ when loading ‘lumi’
** testing if installed package keeps a record of temporary installation path
* DONE (wateRmelon)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<HR>
<H3>Example timings</H3>
<P><SPAN class="filename">wateRmelon.Rcheck/wateRmelon-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>BMIQ</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>adjustedDasen</TD><TD style="text-align: right;">3.439</TD><TD style="text-align: right;">0.133</TD><TD style="text-align: right;">3.572</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>adjustedFunnorm</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>agep</TD><TD style="text-align: right;">0.247</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">0.295</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bscon</TD><TD style="text-align: right;">0.128</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.132</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>combo</TD><TD style="text-align: right;">0.440</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.452</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dasen</TD><TD style="text-align: right;">1.717</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">1.749</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dmrse</TD><TD style="text-align: right;">1.734</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">1.758</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>estimateSex</TD><TD style="text-align: right;">1.594</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">1.610</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genki</TD><TD style="text-align: right;">1.911</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">1.916</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>iDMR</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>melon</TD><TD style="text-align: right;">0.168</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.175</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>metrics</TD><TD style="text-align: right;">6.254</TD><TD style="text-align: right;">0.132</TD><TD style="text-align: right;">6.386</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>outlyx</TD><TD style="text-align: right;">0.797</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.817</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pfilter</TD><TD style="text-align: right;">0.248</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.256</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pwod</TD><TD style="text-align: right;">0.473</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.485</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>qual</TD><TD style="text-align: right;">2.060</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">2.076</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readEPIC</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>seabi</TD><TD style="text-align: right;">10.088</TD><TD style="text-align: right;"> 0.132</TD><TD style="text-align: right;">10.220</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>sextest</TD><TD style="text-align: right;">10.584</TD><TD style="text-align: right;"> 0.092</TD><TD style="text-align: right;">10.676</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
