<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for GenVisR on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><B>[<A href="..#G">G</A>]</B></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-10 10:04:32 -0000 (Wed, 10 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-03-12&nbsp;r83975)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">6211</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for GenVisR on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the GenVisR package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/GenVisR.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>803</B>/2194</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">GenVisR</A>&nbsp;1.33.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/GenVisR">landing page</A>)</SPAN><BR>Zachary Skidmore<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-08&nbsp;19:11:19&nbsp;-0000&nbsp;(Mon,&nbsp;08&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/GenVisR</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">1f55f5d</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:43:51&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: GenVisR</TD></TR>
<TR><TD><B>Version</B>: 1.33.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:GenVisR.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings GenVisR_1.33.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-09 17:03:25 -0000 (Tue, 09 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-09 17:29:37 -0000 (Tue, 09 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 1572.0 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: GenVisR.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:GenVisR.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings GenVisR_1.33.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/GenVisR.Rcheck’
* using R Under development (unstable) (2023-03-12 r83975)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘GenVisR/DESCRIPTION’ ... OK
* this is package ‘GenVisR’ version ‘1.33.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘GenVisR’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
Namespace in Imports field not imported from: ‘reshape2’
  All declared Imports should be used.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
setTierTwo,data.table : a: no visible binding for global variable ‘tmp’
toLolliplot,GMS: no visible binding for global variable ‘missingINdex’
Undefined global functions or variables:
  missingINdex tmp
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking LazyData ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘GenVisR-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: Lolliplot-class
&gt; ### Title: Class Lolliplot
&gt; ### Aliases: Lolliplot-class Lolliplot
&gt; 
&gt; ### ** Examples
&gt; 
&gt; # Load a pre-existing data set
&gt; dataset &lt;- PIK3CA
&gt; 
&gt; # mode 1, amino acid changes are not present
&gt; 
&gt; library(TxDb.Hsapiens.UCSC.hg38.knownGene)
Loading required package: GenomicFeatures
Loading required package: BiocGenerics

Attaching package: ‘BiocGenerics’

The following objects are masked from ‘package:stats’:

    IQR, mad, sd, var, xtabs

The following objects are masked from ‘package:base’:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: ‘S4Vectors’

The following objects are masked from ‘package:base’:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
Loading required package: GenomicRanges
Loading required package: AnnotationDbi
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

&gt; library(BSgenome.Hsapiens.UCSC.hg38)
Loading required package: BSgenome
Loading required package: Biostrings
Loading required package: XVector

Attaching package: ‘Biostrings’

The following object is masked from ‘package:base’:

    strsplit

Loading required package: rtracklayer
&gt; txdb &lt;- TxDb.Hsapiens.UCSC.hg38.knownGene
&gt; BSgenome &lt;- BSgenome.Hsapiens.UCSC.hg38
&gt; 
&gt; keep &lt;- c(&quot;Chromosome&quot;, &quot;Start_Position&quot;, &quot;End_Position&quot;, &quot;Reference_Allele&quot;,
+           &quot;Tumor_Seq_Allele2&quot;, &quot;Tumor_Sample_Barcode&quot;, &quot;Gene&quot;, &quot;Variant_Classification&quot;)
&gt; dataset.mode1 &lt;- dataset[,keep]
&gt; colnames(dataset.mode1) &lt;- c(&quot;chromosome&quot;, &quot;start&quot;, &quot;stop&quot;, &quot;reference&quot;, &quot;variant&quot;,
+                              &quot;sample&quot;, &quot;gene&quot;, &quot;consequence&quot;)
&gt; 
&gt; 
&gt; # mode 2, amino acid changes are present
&gt; 
&gt; keep &lt;- c(&quot;Chromosome&quot;, &quot;Start_Position&quot;, &quot;End_Position&quot;, &quot;Reference_Allele&quot;,
+           &quot;Tumor_Seq_Allele2&quot;, &quot;Tumor_Sample_Barcode&quot;, &quot;Gene&quot;, &quot;Variant_Classification&quot;,
+           &quot;Transcript_ID&quot;, &quot;HGVSp&quot;)
&gt; dataset.mode2 &lt;- dataset[,keep]
&gt; colnames(dataset.mode2) &lt;- c(&quot;chromosome&quot;, &quot;start&quot;, &quot;stop&quot;, &quot;reference&quot;, &quot;variant&quot;,
+                              &quot;sample&quot;, &quot;gene&quot;, &quot;consequence&quot;, &quot;transcript&quot;, &quot;proteinCoord&quot;)
&gt; 
&gt; # run Lolliplot
&gt; 
&gt; object &lt;- Lolliplot(dataset.mode1, transcript=&quot;ENST00000263967&quot;,
+                     species=&quot;hsapiens&quot;, txdb=txdb, BSgenome=BSgenome)
This function is part of the new S4 feature and is under active development, did you mean to use lolliplot() with a lowercase l?
Warning: Ensembl will soon enforce the use of https.
Ensure the &#x27;host&#x27; argument includes &quot;https://&quot;
Warning in valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
  GRanges object contains 3780 out-of-bound ranges located on sequence
  48099. Note that ranges located on a sequence whose length is unknown
  (NA) or on a circular sequence are not considered out-of-bound (use
  seqlengths() and isCircular() to get the lengths and circularity flags
  of the underlying sequences). You can use trim() to trim these ranges.
  See ?`trim,GenomicRanges-method` for more information.
Warning in .predictCodingGRangesList(query, cache[[&quot;cdsbytx&quot;]], seqSource,  :
  &#x27;varAllele&#x27; values with &#x27;N&#x27;, &#x27;.&#x27;, &#x27;+&#x27; or &#x27;-&#x27; were not translated
&#x27;select()&#x27; returned many:1 mapping between keys and columns
Error in curl::curl_fetch_memory(url, handle = handle) : 
  Timeout was reached: [www.ensembl.org] SSL connection timeout
Calls: Lolliplot ... request_fetch -&gt; request_fetch.write_memory -&gt; &lt;Anonymous&gt;
Execution halted
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  • MutSpectra-class/mutspectra-frequency-plot-custom-pallette.svg
  • MutSpectra-class/mutspectra-frequency-plot.svg
  • MutSpectra-class/mutspectra-proportion-plot-add-layer.svg
  • MutSpectra-class/mutspectra-proportion-plot-custom-pallette.svg
  • MutSpectra-class/mutspectra-proportion-plot.svg
  • Rainfall-class/density-main-plot.svg
  • Rainfall-class/density-plot-add-layer.svg
  • Rainfall-class/drawplot-rainfall.svg
  • Rainfall-class/final-rainfall-alter-section-hieghts.svg
  • Rainfall-class/final-rainfall-base.svg
  • Rainfall-class/rainfall-main-plot.svg
  • Rainfall-class/rainfall-plot-add-layer.svg
  • Rainfall-class/rainfall-plot-aesthetic-options.svg
  Error: Test failures
  Execution halted
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘Intro.Rmd’ using ‘UTF-8’... OK
  ‘waterfall_introduction.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 2 ERRORs, 2 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/GenVisR.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">GenVisR.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL GenVisR
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library’
* installing *source* package ‘GenVisR’ ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (GenVisR)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">GenVisR.Rcheck/tests/testthat.Rout.fail</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(GenVisR)
&gt; 
&gt; test_check(&quot;GenVisR&quot;)
[ FAIL 1 | WARN 17 | SKIP 54 | PASS 601 ]

══ Skipped tests ═══════════════════════════════════════════════════════════════
• On Bioconductor (54)

══ Failed tests ════════════════════════════════════════════════════════════════
── Failure (&#x27;test-Lolliplot-class.R:526:5&#x27;): setDomainHeights domains are nested correctly ──
`expected` not equivalent to `actual`.
1/1 mismatches
[1] 2 - 9 == -7

[ FAIL 1 | WARN 17 | SKIP 54 | PASS 601 ]
Deleting unused snapshots:
• Lolliplot-class/drawplot-lolliplot.svg
• Lolliplot-class/final-lolliplot-alter-section-height.svg
• Lolliplot-class/final-lolliplot-base.svg
• Lolliplot-class/lolliplot-density-plot-layer.svg
• Lolliplot-class/lolliplot-density-plot.svg
• Lolliplot-class/lolliplot-plot-base-add-domain-palette.svg
• Lolliplot-class/lolliplot-plot-base-add-labels.svg
• Lolliplot-class/lolliplot-plot-base-add-layer.svg
• Lolliplot-class/lolliplot-plot-base-add-mutation-palette.svg
• Lolliplot-class/lolliplot-plot-base.svg
• MutSpectra-class/drawplot-mutspectra.svg
• MutSpectra-class/final-mutspectra-alter-section-heights.svg
• MutSpectra-class/final-mutspectra-base.svg
• MutSpectra-class/mutspectra-clinical.svg
• MutSpectra-class/mutspectra-frequency-plot-add-layer.svg
• MutSpectra-class/mutspectra-frequency-plot-custom-pallette.svg
• MutSpectra-class/mutspectra-frequency-plot.svg
• MutSpectra-class/mutspectra-proportion-plot-add-layer.svg
• MutSpectra-class/mutspectra-proportion-plot-custom-pallette.svg
• MutSpectra-class/mutspectra-proportion-plot.svg
• Rainfall-class/density-main-plot.svg
• Rainfall-class/density-plot-add-layer.svg
• Rainfall-class/drawplot-rainfall.svg
• Rainfall-class/final-rainfall-alter-section-hieghts.svg
• Rainfall-class/final-rainfall-base.svg
• Rainfall-class/rainfall-main-plot.svg
• Rainfall-class/rainfall-plot-add-layer.svg
• Rainfall-class/rainfall-plot-aesthetic-options.svg
Error: Test failures
Execution halted
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">GenVisR.Rcheck/GenVisR-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
