<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for ensembldb on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><B>[<A href="../index.html#E">E</A>]</B></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-10 01:25:33 -0000 (Fri, 10 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4219</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for ensembldb on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the ensembldb package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/ensembldb.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>605</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">ensembldb</A>&nbsp;2.23.1</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/ensembldb">landing page</A>)</SPAN><BR>Johannes Rainer<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-09&nbsp;03:02:02&nbsp;-0000&nbsp;(Thu,&nbsp;09&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/ensembldb</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">8995853</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-09&nbsp;10:50:04&nbsp;-0000&nbsp;(Wed,&nbsp;09&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: ensembldb</TD></TR>
<TR><TD><B>Version</B>: 2.23.1</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:ensembldb.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings ensembldb_2.23.1.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-09 14:37:55 -0000 (Thu, 09 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-09 14:54:10 -0000 (Thu, 09 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 975.5 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: ensembldb.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:ensembldb.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings ensembldb_2.23.1.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/ensembldb.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘ensembldb/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘ensembldb’ version ‘2.23.1’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘ensembldb’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                     user system elapsed
proteinToGenome     3.716 34.180 101.884
EnsDb-AnnotationDbi 3.901  0.814   6.255
Filter-classes      2.652  1.634   7.105
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
   Common transcripts with defined CDS: (3996) common, (0) only in x, (0) only in y.
   CDS start coordinates: (3996) identical, (0) different.
   CDS end coordinates: (3996) identical, (0) different.
   Associated gene IDs: (4045) identical, (0) different.
  Done. Result: OK
  
  Comparing exon data:
   exon IDs: (4045) common, (0) only in x, (0) only in y.
   Exon start coordinates: (4045) identical, (0) different.
   Exon end coordinates: (4045) identical, (0) different.
   Exon index in transcript models: (4045) identical, (0) different.
  Done. Result: OK
  Error in readLines(curl::curl(paste0(ensembldb:::.ENSEMBL_URL, &quot;current_README&quot;))) : 
    Given file does not exist
  Killed
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘MySQL-backend.Rmd’ using ‘UTF-8’... OK
  ‘coordinate-mapping-use-cases.Rmd’ using ‘UTF-8’... OK
  ‘coordinate-mapping.Rmd’ using ‘UTF-8’... OK
  ‘ensembldb.Rmd’ using ‘UTF-8’... OK
  ‘proteins.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/ensembldb.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">ensembldb.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL ensembldb
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘ensembldb’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (ensembldb)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">ensembldb.Rcheck/tests/testthat.Rout.fail</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(ensembldb)
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: GenomicRanges
Loading required package: stats4
Loading required package: S4Vectors

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
Loading required package: GenomicFeatures
Loading required package: AnnotationDbi
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

Loading required package: AnnotationFilter

Attaching package: &#x27;AnnotationFilter&#x27;

The following object is masked from &#x27;package:testthat&#x27;:

    not


Attaching package: &#x27;ensembldb&#x27;

The following object is masked from &#x27;package:stats&#x27;:

    filter

&gt; library(EnsDb.Hsapiens.v86)
&gt; edb &lt;- EnsDb.Hsapiens.v86
&gt; 
&gt; test_check(&quot;ensembldb&quot;)
Creating package in /tmp/RtmpXEIX5h/EnsDb.Hsapiens.v75 
Error in readLines(curl(my_url)) : 
  Timeout was reached: [ftp.ensemblgenomes.org] Connection time-out
In addition: Warning message:
call dbDisconnect() when finished working with a connection 

Comparing chromosome data:
 Sequence names: (180) common, (0) only in x, (0) only in y.
 Sequence lengths: (180) identical, (0) different.
Done. Result: OK

Comparing gene data:
 gene IDs: (4045) common, (0) only in x, (0) only in y.
 Sequence names: (4045) identical, (0) different.
 Gene start coordinates: (4045) identical, (0) different.
 Gene end coordinates: (4045) identical, (0) different.
 Gene strand: (4045) identical, (0) different.
 Gene names: (132) identical, (3913) different.
 Entrezgene IDs: (4045) identical, (0) different.
 Gene biotypes: (4045) identical, (0) different.
Done. Result: WARN

Comparing transcript data:
 transcript IDs: (4045) common, (0) only in x, (0) only in y.
 Transcript start coordinates: (4045) identical, (0) different.
 Transcript end coordinates: (4045) identical, (0) different.
 Transcript biotypes: (4045) identical, (0) different.
 Common transcripts with defined CDS: (3996) common, (0) only in x, (0) only in y.
 CDS start coordinates: (3996) identical, (0) different.
 CDS end coordinates: (3996) identical, (0) different.
 Associated gene IDs: (4045) identical, (0) different.
Done. Result: OK

Comparing exon data:
 exon IDs: (4045) common, (0) only in x, (0) only in y.
 Exon start coordinates: (4045) identical, (0) different.
 Exon end coordinates: (4045) identical, (0) different.
 Exon index in transcript models: (4045) identical, (0) different.
Done. Result: OK

Comparing metadata:
 Ensembl versions match.
 Genome builds match.
 All differences: &lt;name&gt;: &lt;value x&gt; != &lt;value y&gt;
  -  Creation time : Thu Feb  9 14:49:13 2023  !=  Thu Feb  9 14:48:24 2023 
  -  source_file : Devosia_geojensis.ASM96941v1.32.gtf.gz  !=  Devosia_geojensis.ASM96941v1.32.gff3.gz 
Done. Result: NOTE

Comparing chromosome data:
 Sequence names: (180) common, (0) only in x, (0) only in y.
 Sequence lengths: (180) identical, (0) different.
Done. Result: OK

Comparing gene data:
 gene IDs: (4045) common, (0) only in x, (0) only in y.
 Sequence names: (4045) identical, (0) different.
 Gene start coordinates: (4045) identical, (0) different.
 Gene end coordinates: (4045) identical, (0) different.
 Gene strand: (4045) identical, (0) different.
 Gene names: (132) identical, (3913) different.
 Entrezgene IDs: (4045) identical, (0) different.
 Gene biotypes: (4045) identical, (0) different.
Done. Result: WARN

Comparing transcript data:
 transcript IDs: (4045) common, (0) only in x, (0) only in y.
 Transcript start coordinates: (4045) identical, (0) different.
 Transcript end coordinates: (4045) identical, (0) different.
 Transcript biotypes: (4045) identical, (0) different.
 Common transcripts with defined CDS: (3996) common, (0) only in x, (0) only in y.
 CDS start coordinates: (3996) identical, (0) different.
 CDS end coordinates: (3996) identical, (0) different.
 Associated gene IDs: (4045) identical, (0) different.
Done. Result: OK

Comparing exon data:
 exon IDs: (4045) common, (0) only in x, (0) only in y.
 Exon start coordinates: (4045) identical, (0) different.
 Exon end coordinates: (4045) identical, (0) different.
 Exon index in transcript models: (4045) identical, (0) different.
Done. Result: OK
Error in readLines(curl::curl(paste0(ensembldb:::.ENSEMBL_URL, &quot;current_README&quot;))) : 
  Given file does not exist
Killed
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">ensembldb.Rcheck/ensembldb-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>EnsDb-AnnotationDbi</TD><TD style="text-align: right;">3.901</TD><TD style="text-align: right;">0.814</TD><TD style="text-align: right;">6.255</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EnsDb-class</TD><TD style="text-align: right;">1.000</TD><TD style="text-align: right;">0.101</TD><TD style="text-align: right;">1.105</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EnsDb-exonsBy</TD><TD style="text-align: right;">2.644</TD><TD style="text-align: right;">0.459</TD><TD style="text-align: right;">4.523</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EnsDb-lengths</TD><TD style="text-align: right;">1.333</TD><TD style="text-align: right;">0.125</TD><TD style="text-align: right;">1.459</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EnsDb-seqlevels</TD><TD style="text-align: right;">0.070</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.079</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EnsDb-sequences</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.017</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EnsDb-utils</TD><TD style="text-align: right;">0.344</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.352</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EnsDb</TD><TD style="text-align: right;">1.244</TD><TD style="text-align: right;">0.119</TD><TD style="text-align: right;">1.362</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Filter-classes</TD><TD style="text-align: right;">2.652</TD><TD style="text-align: right;">1.634</TD><TD style="text-align: right;">7.105</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ProteinFunctionality</TD><TD style="text-align: right;">0.072</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.108</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cdsToTranscript</TD><TD style="text-align: right;">2.131</TD><TD style="text-align: right;">0.097</TD><TD style="text-align: right;">2.232</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>convertFilter</TD><TD style="text-align: right;">0.033</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.033</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genomeToProtein</TD><TD style="text-align: right;">2.662</TD><TD style="text-align: right;">0.148</TD><TD style="text-align: right;">2.810</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genomeToTranscript</TD><TD style="text-align: right;">2.158</TD><TD style="text-align: right;">0.181</TD><TD style="text-align: right;">2.342</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>global-filters</TD><TD style="text-align: right;">0.139</TD><TD style="text-align: right;">0.023</TD><TD style="text-align: right;">0.217</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>hasProteinData-EnsDb-method</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>listEnsDbs</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeEnsemblDbPackage</TD><TD style="text-align: right;">0.969</TD><TD style="text-align: right;">0.059</TD><TD style="text-align: right;">1.047</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>proteinToGenome</TD><TD style="text-align: right;">  3.716</TD><TD style="text-align: right;"> 34.180</TD><TD style="text-align: right;">101.884</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>proteinToTranscript</TD><TD style="text-align: right;">1.128</TD><TD style="text-align: right;">0.211</TD><TD style="text-align: right;">1.972</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>transcriptToCds</TD><TD style="text-align: right;">0.787</TD><TD style="text-align: right;">0.039</TD><TD style="text-align: right;">0.840</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>transcriptToGenome</TD><TD style="text-align: right;">1.360</TD><TD style="text-align: right;">0.093</TD><TD style="text-align: right;">1.461</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>transcriptToProtein</TD><TD style="text-align: right;">2.441</TD><TD style="text-align: right;">0.061</TD><TD style="text-align: right;">2.503</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>useMySQL-EnsDb-method</TD><TD style="text-align: right;">0.017</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.017</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
