<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for diffHic on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><B>[<A href="..#D">D</A>]</B></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-31 05:44:33 -0000 (Wed, 31 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>4.3.0&nbsp;(2023-04-21)&nbsp;--&nbsp;"Already&nbsp;Tomorrow"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4219</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for diffHic on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the diffHic package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/diffHic.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>547</B>/2197</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">diffHic</A>&nbsp;1.33.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/diffHic">landing page</A>)</SPAN><BR>Aaron Lun<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-29&nbsp;10:19:22&nbsp;-0000&nbsp;(Mon,&nbsp;29&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/diffHic</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">8da9340</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:39:29&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: diffHic</TD></TR>
<TR><TD><B>Version</B>: 1.33.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:diffHic.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings diffHic_1.33.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-30 05:49:57 -0000 (Tue, 30 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-30 05:57:26 -0000 (Tue, 30 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 448.6 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: diffHic.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 2</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD check --install=check:diffHic.install-out.txt --library=/home/biocbuild/R/R-4.3.0/site-library --timings diffHic_1.33.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.18-bioc/meat/diffHic.Rcheck’
* using R version 4.3.0 (2023-04-21)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘diffHic/DESCRIPTION’ ... OK
* this is package ‘diffHic’ version ‘1.33.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘diffHic’ can be installed ... OK
* used C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04.1) 11.3.0’
* checking C++ specification ... NOTE
  Specified C++11: please drop specification unless essential
* checking installed package size ... NOTE
  installed size is 12.8Mb
  sub-directories of 1Mb or more:
    doc    1.3Mb
    libs  10.2Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... WARNING
checkRd: (5) pairParam.Rd:48-51: \item in \describe must have non-empty label
checkRd: (5) pairParam.Rd:58-60: \item in \describe must have non-empty label
checkRd: (5) pairParam.Rd:67-69: \item in \describe must have non-empty label
checkRd: (5) pairParam.Rd:70-73: \item in \describe must have non-empty label
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... NOTE
GNU make is a SystemRequirements.
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... WARNING
Note: information on .o files is not available
File ‘/home/biocbuild/R/R-4.3.0/site-library/diffHic/libs/diffHic.so’:
  Found ‘__sprintf_chk’, possibly from ‘sprintf’ (C)
  Found ‘abort’, possibly from ‘abort’ (C)
  Found ‘exit’, possibly from ‘exit’ (C)
  Found ‘stderr’, possibly from ‘stderr’ (C)
  Found ‘stdout’, possibly from ‘stdout’ (C)

Compiled code should not call entry points which might terminate R nor
write to stdout/stderr instead of to the console, nor use Fortran I/O
nor system RNGs nor [v]sprintf. The detected symbols are linked into
the code but might come from libraries and not actually be called.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
* checking installed files from ‘inst/doc’ ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
               user system elapsed
normalizeCNV 10.690  0.012  10.702
cutGenome     5.927  0.108   6.064
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘test-basic.R’
  Comparing ‘test-basic.Rout’ to ‘test-basic.Rout.save’ ...208,210d207
&lt; Warning message:
&lt; In DGEList.default(group = &quot;a&quot;, lib.size = 17L, norm.factors = 2,  :
&lt;   norm factors don&#x27;t multiply to 1
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘diffHic.Rnw’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 2 WARNINGs, 3 NOTEs
See
  ‘/home/biocbuild/bbs-3.18-bioc/meat/diffHic.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">diffHic.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/R/R-4.3.0/bin/R CMD INSTALL diffHic
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/R/R-4.3.0/site-library’
* installing *source* package ‘diffHic’ ...
** using staged installation
** libs
using C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04.1) 11.3.0’
using C++11
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c binner.cpp -o binner.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c check_input.cpp -o check_input.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c cluster_2d.cpp -o cluster_2d.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c count_background.cpp -o count_background.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c count_connect.cpp -o count_connect.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c count_patch.cpp -o count_patch.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c directionality.cpp -o directionality.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c init.cpp -o init.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c iterative_correction.cpp -o iterative_correction.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c neighbors.cpp -o neighbors.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c pair_stats.cpp -o pair_stats.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c quadrant_bg.cpp -o quadrant_bg.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c report_hic_pairs.cpp -o report_hic_pairs.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c trended_filter.cpp -o trended_filter.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/R/R-4.3.0/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rhtslib/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/zlibbioc/include&#x27; -I&#x27;/home/biocbuild/R/R-4.3.0/site-library/Rcpp/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c utils.cpp -o utils.o
g++ -std=gnu++11 -shared -L/home/biocbuild/R/R-4.3.0/lib -L/usr/local/lib -o diffHic.so binner.o check_input.o cluster_2d.o count_background.o count_connect.o count_patch.o directionality.o init.o iterative_correction.o neighbors.o pair_stats.o quadrant_bg.o report_hic_pairs.o trended_filter.o utils.o /home/biocbuild/R/R-4.3.0/site-library/Rhtslib/usrlib/libhts.a -lcurl -L/home/biocbuild/R/R-4.3.0/lib -lR
installing to /home/biocbuild/R/R-4.3.0/site-library/00LOCK-diffHic/00new/diffHic/libs
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (diffHic)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">diffHic.Rcheck/tests/test-basic.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; # These are just placeholders for the real things in inst/tests.
&gt; 
&gt; suppressWarnings(suppressPackageStartupMessages(require(diffHic)))
&gt; 
&gt; hic.file &lt;- system.file(&quot;exdata&quot;, &quot;hic_sort.bam&quot;, package=&quot;diffHic&quot;)
&gt; cuts &lt;- readRDS(system.file(&quot;exdata&quot;, &quot;cuts.rds&quot;, package=&quot;diffHic&quot;))
&gt; param &lt;- pairParam(fragments=cuts)
&gt; 
&gt; # Setting up the parameters
&gt; fout &lt;- &quot;output.h5&quot;
&gt; preparePairs(hic.file, param, file=fout)
$pairs
   total   marked filtered   mapped 
      32        7        3       22 

$same.id
   dangling self.circle 
          4           1 

$singles
[1] 2

$chimeras
  total  mapped   multi invalid 
     12       8       7       5 

&gt; head(getPairData(fout, param))
  length orientation insert
1     40           1     40
2     80           1     80
3     60           2     54
4     40           3    160
5     20           1     66
6     80           0    114
&gt; 
&gt; loadChromos(fout)
  anchor1 anchor2
1    chrA    chrA
2    chrB    chrA
3    chrB    chrB
&gt; head(loadData(fout, &quot;chrA&quot;, &quot;chrA&quot;))
  anchor1.id anchor2.id anchor1.pos anchor2.pos anchor1.len anchor2.len
1          2          1          49          14          -5          10
2          2          1          79           9         -10          10
3          2          1          65          21          10         -10
4          4          1         156           6         -10         -10
5          4          2         141          80          -5          10
6          4          2         154          50          10          10
&gt; head(loadData(fout, &quot;chrA&quot;, &quot;chrB&quot;))
  anchor1.id anchor2.id anchor1.pos anchor2.pos anchor1.len anchor2.len
1          5          2          19          44          10         -10
2          5          3          14         105          10         -10
3          6          1          65          19           5          10
4          6          2          24          90          -5           5
5          6          3          24         100          -5         -10
6          6          3          24          95          -5         -10
Warning message:
In value[[3L]](cond) : anchor definitions are reversed
&gt; 
&gt; # Loading the counts.
&gt; data &lt;- squareCounts(fout, param, width=50, filter=1)
&gt; data
class: InteractionSet 
dim: 10 1 
metadata(2): param width
assays(1): counts
rownames: NULL
rowData names(0):
colnames: NULL
colData names(1): totals
type: ReverseStrictGInteractions
regions: 6
&gt; 
&gt; margins &lt;- marginCounts(fout, param, width=50)
&gt; margins
class: RangedSummarizedExperiment 
dim: 6 1 
metadata(1): param
assays(1): counts
rownames: NULL
rowData names(1): nfrags
colnames: NULL
colData names(1): totals
&gt; totalCounts(fout, param)
[1] 17
&gt; 
&gt; regions &lt;- GRanges(&quot;chrA&quot;, IRanges(c(1, 100, 150), c(20, 140, 160)))
&gt; connectCounts(fout, param, regions=regions, filter=1L)
class: InteractionSet 
dim: 2 1 
metadata(1): param
assays(1): counts
rownames: NULL
rowData names(0):
colnames: NULL
colData names(1): totals
type: ReverseStrictGInteractions
regions: 3
&gt; 
&gt; # Checking some values.
&gt; head(getArea(data))
[1] 2208 2304 2208 2208 3312 3174
&gt; head(pairdist(data))
[1]  47 140  93  47  NA  NA
&gt; 
&gt; anchors(data, type=&quot;first&quot;)
GRanges object with 10 ranges and 1 metadata column:
       seqnames    ranges strand |    nfrags
          &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt;
   [1]     chrA     49-94      * |         1
   [2]     chrA   141-188      * |         1
   [3]     chrA   141-188      * |         1
   [4]     chrA   141-188      * |         1
   [5]     chrB      1-69      * |         2
   [6]     chrB      1-69      * |         2
   [7]     chrB      1-69      * |         2
   [8]     chrB      1-69      * |         2
   [9]     chrB     70-92      * |         1
  [10]     chrB     70-92      * |         1
  -------
  seqinfo: 2 sequences from an unspecified genome
&gt; anchors(data, type=&quot;second&quot;)
GRanges object with 10 ranges and 1 metadata column:
       seqnames    ranges strand |    nfrags
          &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt;
   [1]     chrA      1-48      * |         1
   [2]     chrA      1-48      * |         1
   [3]     chrA     49-94      * |         1
   [4]     chrA    95-140      * |         1
   [5]     chrA      1-48      * |         1
   [6]     chrA     49-94      * |         1
   [7]     chrA    95-140      * |         1
   [8]     chrB      1-69      * |         2
   [9]     chrB      1-69      * |         2
  [10]     chrB     70-92      * |         1
  -------
  seqinfo: 2 sequences from an unspecified genome
&gt; assay(data)
      [,1]
 [1,]    3
 [2,]    1
 [3,]    2
 [4,]    1
 [5,]    1
 [6,]    2
 [7,]    3
 [8,]    1
 [9,]    2
[10,]    1
&gt; regions(data)
GRanges object with 6 ranges and 1 metadata column:
      seqnames    ranges strand |    nfrags
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt;
  [1]     chrA      1-48      * |         1
  [2]     chrA     49-94      * |         1
  [3]     chrA    95-140      * |         1
  [4]     chrA   141-188      * |         1
  [5]     chrB      1-69      * |         2
  [6]     chrB     70-92      * |         1
  -------
  seqinfo: 2 sequences from an unspecified genome
&gt; 
&gt; data$totals
[1] 17
&gt; colData(data)
DataFrame with 1 row and 1 column
     totals
  &lt;integer&gt;
1        17
&gt; metadata(data)
$param
Genome contains 7 restriction fragments across 2 chromosomes
No discard regions are specified
No limits on chromosomes for read extraction
No cap on the read pairs per pair of restriction fragments

$width
[1] 50

&gt; 
&gt; asDGEList(data)
An object of class &quot;DGEList&quot;
$counts
   Sample1
1        3
2        1
3        2
4        1
5        1
6        2
7        3
8        1
9        2
10       1

$samples
        group lib.size norm.factors
Sample1     1       17            1

&gt; asDGEList(data, lib.size=20)$samples
        group lib.size norm.factors
Sample1     1       20            1
&gt; asDGEList(data, norm.factors=2, group=&quot;a&quot;)$samples
        group lib.size norm.factors
Sample1     a       17            2
Warning message:
In DGEList.default(group = &quot;a&quot;, lib.size = 17L, norm.factors = 2,  :
  norm factors don&#x27;t multiply to 1
&gt; 
&gt; # Playing around with some bin counts.
&gt; stuff &lt;- correctedContact(data)
&gt; head(stuff$truth)
[1] 1.517032e+04 6.123523e-01 6.123523e-01 8.162749e-01 2.041633e-01
[6] 2.041633e-01
&gt; 
&gt; data.large &lt;- squareCounts(fout, param, width=100, filter=1)
&gt; boxed &lt;- boxPairs(larger=data.large, smaller=data)
&gt; head(boxed$indices$larger)
[1] 1 2 3 4 5 6
&gt; head(boxed$indices$smaller)
[1] 1 2 2 3 4 4
&gt; 
&gt; head(enrichedPairs(data))
class: InteractionSet 
dim: 6 1 
metadata(2): param width
assays(5): counts quadrant vertical horizontal surrounding
rownames: NULL
rowData names(4): N.quadrant N.vertical N.horizontal N.surrounding
colnames: NULL
colData names(1): totals
type: ReverseStrictGInteractions
regions: 6
&gt; head(clusterPairs(data, tol=10)$indices[[1]])
[1] 1 2 2 2 3 3
&gt; 
&gt; # End.
&gt; 
&gt; unlink(fout)
&gt; 
&gt; proc.time()
   user  system elapsed 
 16.664   0.681  17.330 
</PRE>
</DIV><P><SPAN class="filename">diffHic.Rcheck/tests/test-basic.Rout.save</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 3.5.0 Patched (2018-04-30 r74679) -- &quot;Joy in Playing&quot;
Copyright (C) 2018 The R Foundation for Statistical Computing
Platform: x86_64-pc-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

  Natural language support but running in an English locale

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; # These are just placeholders for the real things in inst/tests.
&gt; 
&gt; suppressWarnings(suppressPackageStartupMessages(require(diffHic)))
&gt; 
&gt; hic.file &lt;- system.file(&quot;exdata&quot;, &quot;hic_sort.bam&quot;, package=&quot;diffHic&quot;)
&gt; cuts &lt;- readRDS(system.file(&quot;exdata&quot;, &quot;cuts.rds&quot;, package=&quot;diffHic&quot;))
&gt; param &lt;- pairParam(fragments=cuts)
&gt; 
&gt; # Setting up the parameters
&gt; fout &lt;- &quot;output.h5&quot;
&gt; preparePairs(hic.file, param, file=fout)
$pairs
   total   marked filtered   mapped 
      32        7        3       22 

$same.id
   dangling self.circle 
          4           1 

$singles
[1] 2

$chimeras
  total  mapped   multi invalid 
     12       8       7       5 

&gt; head(getPairData(fout, param))
  length orientation insert
1     40           1     40
2     80           1     80
3     60           2     54
4     40           3    160
5     20           1     66
6     80           0    114
&gt; 
&gt; loadChromos(fout)
  anchor1 anchor2
1    chrA    chrA
2    chrB    chrA
3    chrB    chrB
&gt; head(loadData(fout, &quot;chrA&quot;, &quot;chrA&quot;))
  anchor1.id anchor2.id anchor1.pos anchor2.pos anchor1.len anchor2.len
1          2          1          49          14          -5          10
2          2          1          79           9         -10          10
3          2          1          65          21          10         -10
4          4          1         156           6         -10         -10
5          4          2         141          80          -5          10
6          4          2         154          50          10          10
&gt; head(loadData(fout, &quot;chrA&quot;, &quot;chrB&quot;))
  anchor1.id anchor2.id anchor1.pos anchor2.pos anchor1.len anchor2.len
1          5          2          19          44          10         -10
2          5          3          14         105          10         -10
3          6          1          65          19           5          10
4          6          2          24          90          -5           5
5          6          3          24         100          -5         -10
6          6          3          24          95          -5         -10
Warning message:
In value[[3L]](cond) : anchor definitions are reversed
&gt; 
&gt; # Loading the counts.
&gt; data &lt;- squareCounts(fout, param, width=50, filter=1)
&gt; data
class: InteractionSet 
dim: 10 1 
metadata(2): param width
assays(1): counts
rownames: NULL
rowData names(0):
colnames: NULL
colData names(1): totals
type: ReverseStrictGInteractions
regions: 6
&gt; 
&gt; margins &lt;- marginCounts(fout, param, width=50)
&gt; margins
class: RangedSummarizedExperiment 
dim: 6 1 
metadata(1): param
assays(1): counts
rownames: NULL
rowData names(1): nfrags
colnames: NULL
colData names(1): totals
&gt; totalCounts(fout, param)
[1] 17
&gt; 
&gt; regions &lt;- GRanges(&quot;chrA&quot;, IRanges(c(1, 100, 150), c(20, 140, 160)))
&gt; connectCounts(fout, param, regions=regions, filter=1L)
class: InteractionSet 
dim: 2 1 
metadata(1): param
assays(1): counts
rownames: NULL
rowData names(0):
colnames: NULL
colData names(1): totals
type: ReverseStrictGInteractions
regions: 3
&gt; 
&gt; # Checking some values.
&gt; head(getArea(data))
[1] 2208 2304 2208 2208 3312 3174
&gt; head(pairdist(data))
[1]  47 140  93  47  NA  NA
&gt; 
&gt; anchors(data, type=&quot;first&quot;)
GRanges object with 10 ranges and 1 metadata column:
       seqnames    ranges strand |    nfrags
          &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt;
   [1]     chrA     49-94      * |         1
   [2]     chrA   141-188      * |         1
   [3]     chrA   141-188      * |         1
   [4]     chrA   141-188      * |         1
   [5]     chrB      1-69      * |         2
   [6]     chrB      1-69      * |         2
   [7]     chrB      1-69      * |         2
   [8]     chrB      1-69      * |         2
   [9]     chrB     70-92      * |         1
  [10]     chrB     70-92      * |         1
  -------
  seqinfo: 2 sequences from an unspecified genome
&gt; anchors(data, type=&quot;second&quot;)
GRanges object with 10 ranges and 1 metadata column:
       seqnames    ranges strand |    nfrags
          &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt;
   [1]     chrA      1-48      * |         1
   [2]     chrA      1-48      * |         1
   [3]     chrA     49-94      * |         1
   [4]     chrA    95-140      * |         1
   [5]     chrA      1-48      * |         1
   [6]     chrA     49-94      * |         1
   [7]     chrA    95-140      * |         1
   [8]     chrB      1-69      * |         2
   [9]     chrB      1-69      * |         2
  [10]     chrB     70-92      * |         1
  -------
  seqinfo: 2 sequences from an unspecified genome
&gt; assay(data)
      [,1]
 [1,]    3
 [2,]    1
 [3,]    2
 [4,]    1
 [5,]    1
 [6,]    2
 [7,]    3
 [8,]    1
 [9,]    2
[10,]    1
&gt; regions(data)
GRanges object with 6 ranges and 1 metadata column:
      seqnames    ranges strand |    nfrags
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt;
  [1]     chrA      1-48      * |         1
  [2]     chrA     49-94      * |         1
  [3]     chrA    95-140      * |         1
  [4]     chrA   141-188      * |         1
  [5]     chrB      1-69      * |         2
  [6]     chrB     70-92      * |         1
  -------
  seqinfo: 2 sequences from an unspecified genome
&gt; 
&gt; data$totals
[1] 17
&gt; colData(data)
DataFrame with 1 row and 1 column
     totals
  &lt;integer&gt;
1        17
&gt; metadata(data)
$param
Genome contains 7 restriction fragments across 2 chromosomes
No discard regions are specified
No limits on chromosomes for read extraction
No cap on the read pairs per pair of restriction fragments

$width
[1] 50

&gt; 
&gt; asDGEList(data)
An object of class &quot;DGEList&quot;
$counts
   Sample1
1        3
2        1
3        2
4        1
5        1
6        2
7        3
8        1
9        2
10       1

$samples
        group lib.size norm.factors
Sample1     1       17            1

&gt; asDGEList(data, lib.size=20)$samples
        group lib.size norm.factors
Sample1     1       20            1
&gt; asDGEList(data, norm.factors=2, group=&quot;a&quot;)$samples
        group lib.size norm.factors
Sample1     a       17            2
&gt; 
&gt; # Playing around with some bin counts.
&gt; stuff &lt;- correctedContact(data)
&gt; head(stuff$truth)
[1] 1.517032e+04 6.123523e-01 6.123523e-01 8.162749e-01 2.041633e-01
[6] 2.041633e-01
&gt; 
&gt; data.large &lt;- squareCounts(fout, param, width=100, filter=1)
&gt; boxed &lt;- boxPairs(larger=data.large, smaller=data)
&gt; head(boxed$indices$larger)
[1] 1 2 3 4 5 6
&gt; head(boxed$indices$smaller)
[1] 1 2 2 3 4 4
&gt; 
&gt; head(enrichedPairs(data))
class: InteractionSet 
dim: 6 1 
metadata(2): param width
assays(5): counts quadrant vertical horizontal surrounding
rownames: NULL
rowData names(4): N.quadrant N.vertical N.horizontal N.surrounding
colnames: NULL
colData names(1): totals
type: ReverseStrictGInteractions
regions: 6
&gt; head(clusterPairs(data, tol=10)$indices[[1]])
[1] 1 2 2 2 3 3
&gt; 
&gt; # End.
&gt; 
&gt; unlink(fout)
&gt; 
&gt; proc.time()
   user  system elapsed 
  5.571   0.168   5.757 
</PRE>
</DIV><P><SPAN class="filename">diffHic.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R version 4.3.0 (2023-04-21) -- &quot;Already Tomorrow&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(diffHic)
Loading required package: GenomicRanges
Loading required package: stats4
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors

Attaching package: &#x27;S4Vectors&#x27;

The following object is masked from &#x27;package:utils&#x27;:

    findMatches

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
Loading required package: InteractionSet
Loading required package: SummarizedExperiment
Loading required package: MatrixGenerics
Loading required package: matrixStats

Attaching package: &#x27;MatrixGenerics&#x27;

The following objects are masked from &#x27;package:matrixStats&#x27;:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars

Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


Attaching package: &#x27;Biobase&#x27;

The following object is masked from &#x27;package:MatrixGenerics&#x27;:

    rowMedians

The following objects are masked from &#x27;package:matrixStats&#x27;:

    anyMissing, rowMedians

&gt; 
&gt; test_check(&quot;diffHic&quot;)
[ FAIL 0 | WARN 0 | SKIP 0 | PASS 427 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
 61.256   1.556  62.800 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">diffHic.Rcheck/diffHic-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>DNaseHiC</TD><TD style="text-align: right;">0.191</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.208</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotatePairs</TD><TD style="text-align: right;">0.519</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.531</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>boxPairs</TD><TD style="text-align: right;">0.891</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.903</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>clusterPairs</TD><TD style="text-align: right;">0.912</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.916</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>compartmentalize</TD><TD style="text-align: right;">1.790</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">1.799</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>connectCounts</TD><TD style="text-align: right;">1.898</TD><TD style="text-align: right;">0.096</TD><TD style="text-align: right;">2.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>consolidatePairs</TD><TD style="text-align: right;">0.480</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.484</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>correctedContact</TD><TD style="text-align: right;">0.526</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.527</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cutGenome</TD><TD style="text-align: right;">5.927</TD><TD style="text-align: right;">0.108</TD><TD style="text-align: right;">6.064</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>diClusters</TD><TD style="text-align: right;">2.021</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">2.021</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>diffHicUsersGuide</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>domainDirections</TD><TD style="text-align: right;">0.143</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.142</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>enrichedPairs</TD><TD style="text-align: right;">0.531</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.531</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>extractPatch</TD><TD style="text-align: right;">0.581</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.585</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filterDiag</TD><TD style="text-align: right;">0.136</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.136</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filterPeaks</TD><TD style="text-align: right;">0.249</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.249</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filters</TD><TD style="text-align: right;">0.306</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.306</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getArea</TD><TD style="text-align: right;">0.181</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.182</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getPairData</TD><TD style="text-align: right;">0.066</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.069</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>loadData</TD><TD style="text-align: right;">0.073</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.073</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>marginCounts</TD><TD style="text-align: right;">0.686</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.694</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mergeCMs</TD><TD style="text-align: right;">0.548</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.559</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mergePairs</TD><TD style="text-align: right;">0.211</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.215</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>neighborCounts</TD><TD style="text-align: right;">0.147</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.154</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>normalizeCNV</TD><TD style="text-align: right;">10.690</TD><TD style="text-align: right;"> 0.012</TD><TD style="text-align: right;">10.702</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pairParam</TD><TD style="text-align: right;">0.128</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.128</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotDI</TD><TD style="text-align: right;">0.579</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.583</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotPlaid</TD><TD style="text-align: right;">1.672</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">1.683</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>preparePairs</TD><TD style="text-align: right;">0.209</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.216</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>prunePairs</TD><TD style="text-align: right;">0.255</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.275</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readMTX2IntSet</TD><TD style="text-align: right;">0.395</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.406</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>savePairs</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>squareCounts</TD><TD style="text-align: right;">0.821</TD><TD style="text-align: right;">0.030</TD><TD style="text-align: right;">0.851</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>totalCounts</TD><TD style="text-align: right;">0.546</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.559</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
