<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for tidybulk on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><B>[<A href="..#T">T</A>]</B></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-10 10:04:42 -0000 (Wed, 10 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-03-12&nbsp;r83975)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">6211</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for tidybulk on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the tidybulk package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/tidybulk.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>2057</B>/2194</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">tidybulk</A>&nbsp;1.13.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/tidybulk">landing page</A>)</SPAN><BR>Stefano Mangiola<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-08&nbsp;19:11:19&nbsp;-0000&nbsp;(Mon,&nbsp;08&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/tidybulk</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">726fc15</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;15:16:05&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: tidybulk</TD></TR>
<TR><TD><B>Version</B>: 1.13.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:tidybulk.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings tidybulk_1.13.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-10 06:59:05 -0000 (Wed, 10 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-10 07:13:24 -0000 (Wed, 10 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 859.2 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: tidybulk.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:tidybulk.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings tidybulk_1.13.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/tidybulk.Rcheck’
* using R Under development (unstable) (2023-03-12 r83975)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘tidybulk/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘tidybulk’ version ‘1.13.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘tidybulk’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking startup messages can be suppressed ... OK
* checking dependencies in R code ... WARNING
&#x27;::&#x27; or &#x27;:::&#x27; import not declared from: ‘pkgconfig’
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
.adjust_abundance_se: no visible binding for global variable ‘.’
.adjust_abundance_se: no visible binding for global variable ‘x’
.aggregate_duplicates_se: no visible binding for global variable ‘.’
.aggregate_duplicates_se: no visible binding for global variable
  ‘group_name’
.aggregate_duplicates_se: no visible binding for global variable
  ‘group’
.as_SummarizedExperiment: no visible binding for global variable ‘.’
.cluster_elements_se: no visible binding for global variable ‘.’
.deconvolve_cellularity_se: no visible binding for global variable
  ‘X_cibersort’
.deconvolve_cellularity_se: no visible binding for global variable ‘.’
.describe_transcript: no visible binding for global variable ‘.’
.describe_transcript_SE: no visible binding for global variable ‘.’
.describe_transcript_SE: no visible binding for global variable
  ‘transcript’
.describe_transcript_SE: no visible binding for global variable
  ‘description’
.get_bibliography: no visible binding for global variable ‘.’
.identify_abundant_se: no visible binding for global variable ‘.’
.impute_missing_abundance_se: no visible binding for global variable
  ‘.’
.keep_abundant: no visible binding for global variable ‘.abundant’
.keep_variable_se: no visible binding for global variable ‘.’
.pivot_sample: no visible binding for global variable ‘.’
.pivot_transcript: no visible binding for global variable ‘.’
.reduce_dimensions_se: no visible binding for global variable ‘.’
.rotate_dimensions_se: no visible binding for global variable ‘.’
.scale_abundance: no visible binding for global variable ‘x’
.scale_abundance: no visible binding for global variable ‘multiplier’
.scale_abundance_se: no visible binding for global variable ‘.’
.scale_abundance_se: no visible binding for global variable ‘x’
.test_differential_abundance_se: no visible binding for global variable
  ‘.contrasts’
.test_differential_abundance_se: no visible binding for global variable
  ‘.’
.test_differential_cellularity: no visible binding for global variable
  ‘X_cibersort’
.test_differential_cellularity: no visible binding for global variable
  ‘.’
.test_differential_cellularity_se: no visible binding for global
  variable ‘X_cibersort’
.test_differential_cellularity_se: no visible binding for global
  variable ‘cell_type’
.test_differential_cellularity_se: no visible binding for global
  variable ‘prop’
.test_differential_cellularity_se: no visible binding for global
  variable ‘.cell_type’
.test_gene_enrichment_SE: no visible global function definition for
  ‘buildCustomIdx’
.test_gene_enrichment_SE: no visible global function definition for
  ‘buildIdx’
.test_gene_enrichment_SE: no visible global function definition for
  ‘egsea’
.test_gene_enrichment_SE: no visible binding for global variable
  ‘pathway’
.test_gene_enrichment_SE: no visible binding for global variable
  ‘data_base’
.test_gene_enrichment_SE: no visible binding for global variable
  ‘web_page’
.test_stratification_cellularity: no visible binding for global
  variable ‘X_cibersort’
.test_stratification_cellularity: no visible binding for global
  variable ‘.’
.test_stratification_cellularity_SE: no visible binding for global
  variable ‘X_cibersort’
.test_stratification_cellularity_SE: no visible binding for global
  variable ‘.’
.test_stratification_cellularity_SE: no visible binding for global
  variable ‘.cell_type’
.tidybulk_se: no visible binding for global variable ‘.’
add_scaled_counts_bulk.calcNormFactor: no visible binding for global
  variable ‘transcript’
add_scaled_counts_bulk.get_low_expressed: no visible binding for global
  variable ‘transcript’
add_scaled_counts_bulk.get_low_expressed: no visible binding for global
  variable ‘.’
aggregate_duplicated_transcripts_DT: no visible binding for global
  variable ‘.abundance_scaled’
aggregate_duplicated_transcripts_bulk: no visible binding for global
  variable ‘.abundance_scaled’
aggregate_duplicated_transcripts_bulk: no visible binding for global
  variable ‘n_aggr’
as_matrix: no visible binding for global variable ‘variable’
call_core: no visible binding for global variable ‘nulldist’
change_reserved_column_names: no visible binding for global variable
  ‘.’
check_if_duplicated_genes: no visible binding for global variable
  ‘transcript’
check_if_duplicated_genes: no visible binding for global variable ‘read
  count’
counts_scaled_exist_SE: no visible binding for global variable
  ‘tt_columns’
counts_scaled_exist_SE: no visible binding for global variable ‘.’
create_tt_from_bam_sam_bulk: no visible binding for global variable ‘.’
create_tt_from_bam_sam_bulk: no visible binding for global variable
  ‘temp’
create_tt_from_bam_sam_bulk: no visible binding for global variable
  ‘Status’
create_tt_from_bam_sam_bulk: no visible binding for global variable
  ‘counts’
create_tt_from_bam_sam_bulk: no visible binding for global variable
  ‘GeneID’
create_tt_from_bam_sam_bulk: no visible binding for global variable
  ‘genes’
create_tt_from_bam_sam_bulk: no visible binding for global variable
  ‘samples’
create_tt_from_bam_sam_bulk: no visible binding for global variable
  ‘transcript’
eliminate_sparse_transcripts: no visible binding for global variable
  ‘my_n’
entrez_over_to_gsea: no visible binding for global variable ‘gs_cat’
entrez_over_to_gsea: no visible binding for global variable ‘test’
entrez_over_to_gsea: no visible binding for global variable ‘geneID’
entrez_rank_to_gsea: no visible binding for global variable ‘gs_cat’
entrez_rank_to_gsea: no visible binding for global variable ‘fit’
error_if_duplicated_genes: no visible binding for global variable
  ‘transcript’
error_if_duplicated_genes: no visible binding for global variable ‘read
  count’
error_if_log_transformed: no visible binding for global variable ‘m’
fill_NA_matrix_with_factor_colwise: no visible binding for global
  variable ‘.’
fill_NA_using_formula: no visible binding for global variable ‘NUL’
fill_NA_using_formula: no visible binding for global variable ‘ct_data’
fill_NA_using_formula: no visible binding for global variable
  ‘library_size__’
fill_NA_using_formula: no visible binding for global variable
  ‘cov_data’
filter_genes_on_condition: no visible binding for global variable
  ‘.feature’
get_abundance_norm_if_exists: no visible binding for global variable
  ‘.abundance_scaled’
get_adjusted_counts_for_unwanted_variation_bulk: no visible binding for
  global variable ‘.’
get_assay_scaled_if_exists_SE: no visible binding for global variable
  ‘tt_columns’
get_assay_scaled_if_exists_SE: no visible binding for global variable
  ‘.abundance_scaled’
get_cell_type_proportions: no visible binding for global variable ‘.’
get_clusters_SNN_bulk: no visible binding for global variable
  ‘seurat_clusters’
get_clusters_SNN_bulk_SE: no visible binding for global variable ‘.’
get_clusters_SNN_bulk_SE: no visible binding for global variable
  ‘seurat_clusters’
get_clusters_kmeans_bulk: no visible binding for global variable ‘.’
get_clusters_kmeans_bulk: no visible binding for global variable
  ‘cluster’
get_clusters_kmeans_bulk: no visible binding for global variable
  ‘cluster kmeans’
get_clusters_kmeans_bulk_SE: no visible binding for global variable ‘.’
get_clusters_kmeans_bulk_SE: no visible binding for global variable
  ‘cluster’
get_differential_transcript_abundance_bulk: no visible binding for
  global variable ‘.’
get_differential_transcript_abundance_bulk_SE: no visible binding for
  global variable ‘.’
get_differential_transcript_abundance_bulk_voom: no visible binding for
  global variable ‘.’
get_differential_transcript_abundance_bulk_voom_SE: no visible binding
  for global variable ‘.’
get_differential_transcript_abundance_deseq2: no visible binding for
  global variable ‘counts’
get_differential_transcript_abundance_deseq2: no visible binding for
  global variable ‘.’
get_differential_transcript_abundance_deseq2_SE: no visible binding for
  global variable ‘.’
get_reduced_dimensions_MDS_bulk: no visible binding for global variable
  ‘Component’
get_reduced_dimensions_MDS_bulk: no visible binding for global variable
  ‘Component value’
get_reduced_dimensions_MDS_bulk_SE: no visible binding for global
  variable ‘Component’
get_reduced_dimensions_MDS_bulk_SE: no visible binding for global
  variable ‘Component value’
get_reduced_dimensions_PCA_bulk: no visible binding for global variable
  ‘sdev’
get_reduced_dimensions_PCA_bulk: no visible binding for global variable
  ‘name’
get_reduced_dimensions_PCA_bulk: no visible binding for global variable
  ‘value’
get_reduced_dimensions_PCA_bulk: no visible binding for global variable
  ‘x’
get_reduced_dimensions_PCA_bulk_SE: no visible binding for global
  variable ‘sdev’
get_reduced_dimensions_PCA_bulk_SE: no visible binding for global
  variable ‘name’
get_reduced_dimensions_PCA_bulk_SE: no visible binding for global
  variable ‘value’
get_reduced_dimensions_PCA_bulk_SE: no visible binding for global
  variable ‘x’
get_reduced_dimensions_TSNE_bulk: no visible binding for global
  variable ‘Y’
get_reduced_dimensions_TSNE_bulk_SE: no visible binding for global
  variable ‘Y’
get_reduced_dimensions_UMAP_bulk_SE: no visible binding for global
  variable ‘x’
get_rotated_dimensions: no visible binding for global variable ‘value’
get_rotated_dimensions: no visible binding for global variable ‘rotated
  dimensions’
get_scaled_counts_bulk: no visible binding for global variable ‘med’
get_scaled_counts_bulk: no visible binding for global variable
  ‘tot_filt’
get_scaled_counts_bulk: no visible binding for global variable ‘nf’
get_scaled_counts_bulk: no visible binding for global variable ‘.’
get_scaled_counts_bulk: no visible binding for global variable ‘tot’
get_symbol_from_ensembl: no visible binding for global variable
  ‘ensembl_id’
get_symbol_from_ensembl: no visible binding for global variable
  ‘transcript’
get_symbol_from_ensembl: no visible binding for global variable
  ‘ref_genome’
get_tt_columns: no visible binding for global variable ‘tt_columns’
initialise_tt_internals: no visible binding for global variable ‘.’
memorise_methods_used: no visible binding for global variable ‘.’
multivariable_differential_tissue_composition: no visible binding for
  global variable ‘.’
multivariable_differential_tissue_composition: no visible binding for
  global variable ‘.cell_type’
multivariable_differential_tissue_composition: no visible binding for
  global variable ‘term’
multivariable_differential_tissue_composition_SE: no visible binding
  for global variable ‘.’
multivariable_differential_tissue_composition_SE: no visible binding
  for global variable ‘.cell_type’
multivariable_differential_tissue_composition_SE: no visible binding
  for global variable ‘term’
remove_redundancy_elements_though_reduced_dimensions: no visible
  binding for global variable ‘sample b’
remove_redundancy_elements_though_reduced_dimensions: no visible
  binding for global variable ‘sample a’
remove_redundancy_elements_though_reduced_dimensions: no visible
  binding for global variable ‘sample 1’
remove_redundancy_elements_though_reduced_dimensions: no visible
  binding for global variable ‘sample 2’
remove_redundancy_elements_though_reduced_dimensions_SE: no visible
  binding for global variable ‘sample b’
remove_redundancy_elements_though_reduced_dimensions_SE: no visible
  binding for global variable ‘sample a’
remove_redundancy_elements_though_reduced_dimensions_SE: no visible
  binding for global variable ‘sample 1’
remove_redundancy_elements_though_reduced_dimensions_SE: no visible
  binding for global variable ‘sample 2’
remove_redundancy_elements_through_correlation: no visible binding for
  global variable ‘rc’
remove_redundancy_elements_through_correlation: no visible binding for
  global variable ‘transcript’
remove_redundancy_elements_through_correlation: no visible binding for
  global variable ‘correlation’
remove_redundancy_elements_through_correlation: no visible binding for
  global variable ‘item1’
remove_redundancy_elements_through_correlation_SE: no visible binding
  for global variable ‘abundance’
remove_redundancy_elements_through_correlation_SE: no visible binding
  for global variable ‘transcript’
remove_redundancy_elements_through_correlation_SE: no visible binding
  for global variable ‘element’
remove_redundancy_elements_through_correlation_SE: no visible binding
  for global variable ‘feature’
remove_redundancy_elements_through_correlation_SE: no visible binding
  for global variable ‘rc’
remove_redundancy_elements_through_correlation_SE: no visible binding
  for global variable ‘correlation’
remove_redundancy_elements_through_correlation_SE: no visible binding
  for global variable ‘item1’
run_epic: no visible global function definition for ‘EPIC’
run_llsr: no visible binding for global variable ‘X_cibersort’
scale_design: no visible binding for global variable ‘value’
scale_design: no visible binding for global variable ‘sample_idx’
scale_design: no visible binding for global variable ‘(Intercept)’
select_closest_pairs: no visible binding for global variable ‘sample 1’
select_closest_pairs: no visible binding for global variable ‘sample 2’
subset_tibble_output: no visible binding for global variable ‘.’
symbol_to_entrez: no visible binding for global variable
  ‘transcript_upper’
symbol_to_entrez: no visible binding for global variable ‘.’
symbol_to_entrez: no visible binding for global variable ‘entrez’
test_differential_cellularity: no visible binding for global variable
  ‘X_cibersort’
test_differential_cellularity_: no visible binding for global variable
  ‘cell_type’
test_differential_cellularity_: no visible binding for global variable
  ‘prop’
test_differential_cellularity_: no visible binding for global variable
  ‘.cell_type’
test_gene_enrichment_bulk_EGSEA: no visible global function definition
  for ‘buildCustomIdx’
test_gene_enrichment_bulk_EGSEA: no visible global function definition
  for ‘buildIdx’
test_gene_enrichment_bulk_EGSEA: no visible global function definition
  for ‘egsea’
test_gene_enrichment_bulk_EGSEA: no visible binding for global variable
  ‘pathway’
test_gene_enrichment_bulk_EGSEA: no visible binding for global variable
  ‘data_base’
test_gene_enrichment_bulk_EGSEA: no visible binding for global variable
  ‘web_page’
test_stratification_cellularity: no visible binding for global variable
  ‘X_cibersort’
test_stratification_cellularity_: no visible binding for global
  variable ‘.cell_type’
tidybulk_to_SummarizedExperiment: no visible binding for global
  variable ‘.’
univariable_differential_tissue_composition: no visible binding for
  global variable ‘.proportion’
univariable_differential_tissue_composition: no visible binding for
  global variable ‘.cell_type’
univariable_differential_tissue_composition: no visible binding for
  global variable ‘cell_type_proportions’
univariable_differential_tissue_composition: no visible binding for
  global variable ‘surv_test’
univariable_differential_tissue_composition_SE: no visible binding for
  global variable ‘.proportion’
univariable_differential_tissue_composition_SE: no visible binding for
  global variable ‘.cell_type’
univariable_differential_tissue_composition_SE: no visible binding for
  global variable ‘cell_type_proportions’
univariable_differential_tissue_composition_SE: no visible binding for
  global variable ‘surv_test’
univariable_differential_tissue_stratification: no visible binding for
  global variable ‘.cell_type’
univariable_differential_tissue_stratification: no visible binding for
  global variable ‘cell_type_proportions’
univariable_differential_tissue_stratification: no visible binding for
  global variable ‘surv_test’
univariable_differential_tissue_stratification_SE: no visible binding
  for global variable ‘.cell_type’
univariable_differential_tissue_stratification_SE: no visible binding
  for global variable ‘cell_type_proportions’
univariable_differential_tissue_stratification_SE: no visible binding
  for global variable ‘surv_test’
adjust_abundance,RangedSummarizedExperiment: no visible binding for
  global variable ‘.’
adjust_abundance,RangedSummarizedExperiment: no visible binding for
  global variable ‘x’
adjust_abundance,SummarizedExperiment: no visible binding for global
  variable ‘.’
adjust_abundance,SummarizedExperiment: no visible binding for global
  variable ‘x’
aggregate_duplicates,RangedSummarizedExperiment: no visible binding for
  global variable ‘.’
aggregate_duplicates,RangedSummarizedExperiment: no visible binding for
  global variable ‘group_name’
aggregate_duplicates,RangedSummarizedExperiment: no visible binding for
  global variable ‘group’
aggregate_duplicates,SummarizedExperiment: no visible binding for
  global variable ‘.’
aggregate_duplicates,SummarizedExperiment: no visible binding for
  global variable ‘group_name’
aggregate_duplicates,SummarizedExperiment: no visible binding for
  global variable ‘group’
as_SummarizedExperiment,spec_tbl_df: no visible binding for global
  variable ‘.’
as_SummarizedExperiment,tbl_df: no visible binding for global variable
  ‘.’
as_SummarizedExperiment,tidybulk: no visible binding for global
  variable ‘.’
cluster_elements,RangedSummarizedExperiment: no visible binding for
  global variable ‘.’
cluster_elements,SummarizedExperiment: no visible binding for global
  variable ‘.’
deconvolve_cellularity,RangedSummarizedExperiment: no visible binding
  for global variable ‘.’
deconvolve_cellularity,SummarizedExperiment: no visible binding for
  global variable ‘.’
describe_transcript,RangedSummarizedExperiment: no visible binding for
  global variable ‘.’
describe_transcript,RangedSummarizedExperiment: no visible binding for
  global variable ‘transcript’
describe_transcript,RangedSummarizedExperiment: no visible binding for
  global variable ‘description’
describe_transcript,SummarizedExperiment: no visible binding for global
  variable ‘.’
describe_transcript,SummarizedExperiment: no visible binding for global
  variable ‘transcript’
describe_transcript,SummarizedExperiment: no visible binding for global
  variable ‘description’
describe_transcript,spec_tbl_df: no visible binding for global variable
  ‘.’
describe_transcript,tbl_df: no visible binding for global variable ‘.’
describe_transcript,tidybulk: no visible binding for global variable
  ‘.’
get_bibliography,RangedSummarizedExperiment: no visible binding for
  global variable ‘.’
get_bibliography,SummarizedExperiment: no visible binding for global
  variable ‘.’
get_bibliography,spec_tbl_df: no visible binding for global variable
  ‘.’
get_bibliography,tbl: no visible binding for global variable ‘.’
get_bibliography,tbl_df: no visible binding for global variable ‘.’
get_bibliography,tidybulk: no visible binding for global variable ‘.’
identify_abundant,RangedSummarizedExperiment: no visible binding for
  global variable ‘.’
identify_abundant,SummarizedExperiment: no visible binding for global
  variable ‘.’
impute_missing_abundance,RangedSummarizedExperiment: no visible binding
  for global variable ‘.’
impute_missing_abundance,SummarizedExperiment: no visible binding for
  global variable ‘.’
keep_abundant,spec_tbl_df: no visible binding for global variable
  ‘.abundant’
keep_abundant,tbl_df: no visible binding for global variable
  ‘.abundant’
keep_abundant,tidybulk: no visible binding for global variable
  ‘.abundant’
keep_variable,RangedSummarizedExperiment: no visible binding for global
  variable ‘.’
keep_variable,SummarizedExperiment: no visible binding for global
  variable ‘.’
pivot_sample,RangedSummarizedExperiment: no visible binding for global
  variable ‘.’
pivot_sample,SummarizedExperiment: no visible binding for global
  variable ‘.’
pivot_transcript,RangedSummarizedExperiment: no visible binding for
  global variable ‘.’
pivot_transcript,SummarizedExperiment: no visible binding for global
  variable ‘.’
reduce_dimensions,RangedSummarizedExperiment: no visible binding for
  global variable ‘.’
reduce_dimensions,SummarizedExperiment: no visible binding for global
  variable ‘.’
rotate_dimensions,RangedSummarizedExperiment: no visible binding for
  global variable ‘.’
rotate_dimensions,SummarizedExperiment: no visible binding for global
  variable ‘.’
scale_abundance,RangedSummarizedExperiment: no visible binding for
  global variable ‘.’
scale_abundance,RangedSummarizedExperiment: no visible binding for
  global variable ‘x’
scale_abundance,SummarizedExperiment: no visible binding for global
  variable ‘.’
scale_abundance,SummarizedExperiment: no visible binding for global
  variable ‘x’
scale_abundance,spec_tbl_df: no visible binding for global variable ‘x’
scale_abundance,spec_tbl_df: no visible binding for global variable
  ‘multiplier’
scale_abundance,tbl_df: no visible binding for global variable ‘x’
scale_abundance,tbl_df: no visible binding for global variable
  ‘multiplier’
scale_abundance,tidybulk: no visible binding for global variable ‘x’
scale_abundance,tidybulk: no visible binding for global variable
  ‘multiplier’
test_differential_abundance,RangedSummarizedExperiment: no visible
  binding for global variable ‘.’
test_differential_abundance,SummarizedExperiment: no visible binding
  for global variable ‘.’
test_differential_cellularity,RangedSummarizedExperiment: no visible
  binding for global variable ‘X_cibersort’
test_differential_cellularity,RangedSummarizedExperiment: no visible
  binding for global variable ‘cell_type’
test_differential_cellularity,RangedSummarizedExperiment: no visible
  binding for global variable ‘prop’
test_differential_cellularity,RangedSummarizedExperiment: no visible
  binding for global variable ‘.cell_type’
test_differential_cellularity,SummarizedExperiment: no visible binding
  for global variable ‘X_cibersort’
test_differential_cellularity,SummarizedExperiment: no visible binding
  for global variable ‘cell_type’
test_differential_cellularity,SummarizedExperiment: no visible binding
  for global variable ‘prop’
test_differential_cellularity,SummarizedExperiment: no visible binding
  for global variable ‘.cell_type’
test_differential_cellularity,spec_tbl_df: no visible binding for
  global variable ‘X_cibersort’
test_differential_cellularity,spec_tbl_df: no visible binding for
  global variable ‘.’
test_differential_cellularity,tbl_df: no visible binding for global
  variable ‘X_cibersort’
test_differential_cellularity,tbl_df: no visible binding for global
  variable ‘.’
test_differential_cellularity,tidybulk: no visible binding for global
  variable ‘X_cibersort’
test_differential_cellularity,tidybulk: no visible binding for global
  variable ‘.’
test_gene_enrichment,RangedSummarizedExperiment: no visible global
  function definition for ‘buildCustomIdx’
test_gene_enrichment,RangedSummarizedExperiment: no visible global
  function definition for ‘buildIdx’
test_gene_enrichment,RangedSummarizedExperiment: no visible global
  function definition for ‘egsea’
test_gene_enrichment,RangedSummarizedExperiment: no visible binding for
  global variable ‘pathway’
test_gene_enrichment,RangedSummarizedExperiment: no visible binding for
  global variable ‘data_base’
test_gene_enrichment,RangedSummarizedExperiment: no visible binding for
  global variable ‘web_page’
test_gene_enrichment,SummarizedExperiment: no visible global function
  definition for ‘buildCustomIdx’
test_gene_enrichment,SummarizedExperiment: no visible global function
  definition for ‘buildIdx’
test_gene_enrichment,SummarizedExperiment: no visible global function
  definition for ‘egsea’
test_gene_enrichment,SummarizedExperiment: no visible binding for
  global variable ‘pathway’
test_gene_enrichment,SummarizedExperiment: no visible binding for
  global variable ‘data_base’
test_gene_enrichment,SummarizedExperiment: no visible binding for
  global variable ‘web_page’
test_stratification_cellularity,RangedSummarizedExperiment: no visible
  binding for global variable ‘X_cibersort’
test_stratification_cellularity,RangedSummarizedExperiment: no visible
  binding for global variable ‘.’
test_stratification_cellularity,RangedSummarizedExperiment: no visible
  binding for global variable ‘.cell_type’
test_stratification_cellularity,SummarizedExperiment: no visible
  binding for global variable ‘X_cibersort’
test_stratification_cellularity,SummarizedExperiment: no visible
  binding for global variable ‘.’
test_stratification_cellularity,SummarizedExperiment: no visible
  binding for global variable ‘.cell_type’
test_stratification_cellularity,spec_tbl_df: no visible binding for
  global variable ‘X_cibersort’
test_stratification_cellularity,spec_tbl_df: no visible binding for
  global variable ‘.’
test_stratification_cellularity,tbl_df: no visible binding for global
  variable ‘X_cibersort’
test_stratification_cellularity,tbl_df: no visible binding for global
  variable ‘.’
test_stratification_cellularity,tidybulk: no visible binding for global
  variable ‘X_cibersort’
test_stratification_cellularity,tidybulk: no visible binding for global
  variable ‘.’
tidybulk,RangedSummarizedExperiment: no visible binding for global
  variable ‘.’
tidybulk,SummarizedExperiment: no visible binding for global variable
  ‘.’
Undefined global functions or variables:
  (Intercept) . .abundance_scaled .abundant .cell_type .contrasts
  .feature .proportion Component Component value EPIC GeneID NUL Status
  X_cibersort Y abundance buildCustomIdx buildIdx cell_type
  cell_type_proportions cluster cluster kmeans correlation counts
  cov_data ct_data data_base description egsea element ensembl_id
  entrez feature fit geneID genes group group_name gs_cat item1
  library_size__ m med multiplier my_n n_aggr name nf nulldist pathway
  prop rc read count ref_genome rotated dimensions sample 1 sample 2
  sample a sample b sample_idx samples sdev seurat_clusters surv_test
  temp term test tot tot_filt transcript transcript_upper tt_columns
  value variable web_page x
Consider adding
  importFrom(&quot;base&quot;, &quot;sample&quot;)
  importFrom(&quot;stats&quot;, &quot;kmeans&quot;)
to your NAMESPACE file.
* checking Rd files ... NOTE
prepare_Rd: remove_redundancy-methods.Rd:144-146: Dropping empty section \details
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking LazyData ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                                          user system elapsed
test_differential_cellularity-methods   16.102  0.076  16.178
test_differential_abundance-methods     12.955  0.100  13.055
test_stratification_cellularity-methods  9.361  0.080   9.442
deconvolve_cellularity-methods           7.073  0.036   7.108
adjust_abundance-methods                 6.834  0.164   6.999
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  Fitting performed in 1.06 seconds.
  [ FAIL 1 | WARN 40 | SKIP 2 | PASS 210 ]
  
  ══ Skipped tests ═══════════════════════════════════════════════════════════════
  • empty test (2)
  
  ══ Failed tests ════════════════════════════════════════════════════════════════
  ── Failure (&#x27;test-bulk_methods.R:1405:3&#x27;): Add reduced dimensions UMAP - no object ──
  magrittr::extract2(pull(res, UMAP1), 1) not equal to -2.12.
  1/1 mismatches
  [1] -1.91 - -2.12 == 0.209
  
  [ FAIL 1 | WARN 40 | SKIP 2 | PASS 210 ]
  Error: Test failures
  Execution halted
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘comparison_with_base_R.Rmd’ using ‘UTF-8’... OK
  ‘introduction.Rmd’ using ‘UTF-8’... OK
  ‘manuscript_differential_transcript_abundance.Rmd’ using ‘UTF-8’... OK
  ‘manuscript_transcriptional_signatures.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 1 WARNING, 2 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/tidybulk.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">tidybulk.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL tidybulk
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library’
* installing *source* package ‘tidybulk’ ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Note: wrong number of arguments to &#x27;floor&#x27; 
Note: wrong number of arguments to &#x27;floor&#x27; 
** help
*** installing help indices
*** copying figures
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (tidybulk)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">tidybulk.Rcheck/tests/testthat.Rout.fail</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(tidybulk)
========================================
tidybulk version 1.13.0
If you use TIDYBULK in published research, please cite:

Mangiola et al. tidybulk: an R tidy framework for modular 
transcriptomic data analysis. Genome Biology 2021.

This message can be suppressed by:
  suppressPackageStartupMessages(library(tidybulk))
========================================


Attaching package: &#x27;tidybulk&#x27;

The following object is masked from &#x27;package:stats&#x27;:

    filter

&gt; 
&gt; test_check(&quot;tidybulk&quot;)
Coefficients not estimable: conditionTRUE 
Coefficients not estimable: conditionTRUE 
Performing PCA
Read the 251 x 50 data matrix successfully!
OpenMP is working. 1 threads.
Using no_dims = 2, perplexity = 30.000000, and theta = 0.500000
Computing input similarities...
Building tree...
Done in 0.03 seconds (sparsity = 0.490437)!
Learning embedding...
Iteration 50: error is 55.873758 (50 iterations in 0.06 seconds)
Iteration 100: error is 58.369755 (50 iterations in 0.07 seconds)
Iteration 150: error is 56.408562 (50 iterations in 0.08 seconds)
Iteration 200: error is 58.858164 (50 iterations in 0.08 seconds)
Iteration 250: error is 58.519817 (50 iterations in 0.07 seconds)
Iteration 300: error is 1.151620 (50 iterations in 0.06 seconds)
Iteration 350: error is 0.852145 (50 iterations in 0.05 seconds)
Iteration 400: error is 0.806480 (50 iterations in 0.04 seconds)
Iteration 450: error is 0.803961 (50 iterations in 0.05 seconds)
Iteration 500: error is 0.800740 (50 iterations in 0.05 seconds)
Iteration 550: error is 0.800665 (50 iterations in 0.05 seconds)
Iteration 600: error is 0.800197 (50 iterations in 0.05 seconds)
Iteration 650: error is 0.800794 (50 iterations in 0.05 seconds)
Iteration 700: error is 0.801818 (50 iterations in 0.05 seconds)
Iteration 750: error is 0.800719 (50 iterations in 0.05 seconds)
Iteration 800: error is 0.795559 (50 iterations in 0.05 seconds)
Iteration 850: error is 0.786386 (50 iterations in 0.05 seconds)
Iteration 900: error is 0.782877 (50 iterations in 0.05 seconds)
Iteration 950: error is 0.777578 (50 iterations in 0.05 seconds)
Iteration 1000: error is 0.778071 (50 iterations in 0.05 seconds)
Fitting performed in 1.06 seconds.
[ FAIL 1 | WARN 40 | SKIP 2 | PASS 210 ]

══ Skipped tests ═══════════════════════════════════════════════════════════════
• empty test (2)

══ Failed tests ════════════════════════════════════════════════════════════════
── Failure (&#x27;test-bulk_methods.R:1405:3&#x27;): Add reduced dimensions UMAP - no object ──
magrittr::extract2(pull(res, UMAP1), 1) not equal to -2.12.
1/1 mismatches
[1] -1.91 - -2.12 == 0.209

[ FAIL 1 | WARN 40 | SKIP 2 | PASS 210 ]
Error: Test failures
Execution halted
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">tidybulk.Rcheck/tidybulk-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>adjust_abundance-methods</TD><TD style="text-align: right;">6.834</TD><TD style="text-align: right;">0.164</TD><TD style="text-align: right;">6.999</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>aggregate_duplicates-methods</TD><TD style="text-align: right;">0.127</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.130</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>arrange-methods</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>as_matrix</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bind-methods</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cluster_elements-methods</TD><TD style="text-align: right;">0.084</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.083</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>deconvolve_cellularity-methods</TD><TD style="text-align: right;">7.073</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">7.108</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>describe_transcript-methods</TD><TD style="text-align: right;">1.274</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">1.324</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>distinct-methods</TD><TD style="text-align: right;">0.160</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.168</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dplyr-methods</TD><TD style="text-align: right;">3.007</TD><TD style="text-align: right;">0.072</TD><TD style="text-align: right;">3.079</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ensembl_to_symbol-methods</TD><TD style="text-align: right;">2.256</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">2.277</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>fill_missing_abundance-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filter-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>get_bibliography-methods</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>group_by-methods</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>identify_abundant-methods</TD><TD style="text-align: right;">0.021</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.021</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>impute_missing_abundance-methods</TD><TD style="text-align: right;">0.094</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.094</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>join-methods</TD><TD style="text-align: right;">3.962</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">4.010</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>keep_abundant-methods</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.045</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>keep_variable-methods</TD><TD style="text-align: right;">0.04</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.04</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>log10_reverse_trans</TD><TD style="text-align: right;">0.255</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.263</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>logit_trans</TD><TD style="text-align: right;">0.172</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.175</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mutate-methods</TD><TD style="text-align: right;">0.083</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.082</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>nest-methods</TD><TD style="text-align: right;">0.533</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">0.573</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pivot_sample-methods</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pivot_transcript-methods</TD><TD style="text-align: right;">0.026</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.031</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>reduce_dimensions-methods</TD><TD style="text-align: right;">0.258</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">0.298</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>remove_redundancy-methods</TD><TD style="text-align: right;">1.436</TD><TD style="text-align: right;">0.108</TD><TD style="text-align: right;">1.567</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rename-methods</TD><TD style="text-align: right;">0.035</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.035</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rotate_dimensions-methods</TD><TD style="text-align: right;">0.16</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.16</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>rowwise-methods</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>scale_abundance-methods</TD><TD style="text-align: right;">0.094</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.094</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>summarise-methods</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>symbol_to_entrez</TD><TD style="text-align: right;">0.619</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.627</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>test_differential_abundance-methods</TD><TD style="text-align: right;">12.955</TD><TD style="text-align: right;"> 0.100</TD><TD style="text-align: right;">13.055</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>test_differential_cellularity-methods</TD><TD style="text-align: right;">16.102</TD><TD style="text-align: right;"> 0.076</TD><TD style="text-align: right;">16.178</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>test_gene_enrichment-methods</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>test_gene_overrepresentation-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>test_gene_rank-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>test_stratification_cellularity-methods</TD><TD style="text-align: right;">9.361</TD><TD style="text-align: right;">0.080</TD><TD style="text-align: right;">9.442</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>tidybulk-methods</TD><TD style="text-align: right;">0.197</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.196</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
