<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for VariantAnnotation on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><B>[<A href="../index.html#V">V</A>]</B></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-13 06:50:04 -0000 (Mon, 13 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4222</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for VariantAnnotation on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the VariantAnnotation package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/VariantAnnotation.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>2115</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">VariantAnnotation</A>&nbsp;1.45.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/VariantAnnotation">landing page</A>)</SPAN><BR>Bioconductor Package Maintainer<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-11&nbsp;01:58:29&nbsp;-0000&nbsp;(Sat,&nbsp;11&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/VariantAnnotation</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">3fc786d</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:06:07&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: VariantAnnotation</TD></TR>
<TR><TD><B>Version</B>: 1.45.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:VariantAnnotation.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings VariantAnnotation_1.45.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-11 23:46:41 -0000 (Sat, 11 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-12 00:01:11 -0000 (Sun, 12 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 870.6 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: VariantAnnotation.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 2</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:VariantAnnotation.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings VariantAnnotation_1.45.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/VariantAnnotation.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘VariantAnnotation/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘VariantAnnotation’ version ‘1.45.0’
* checking package namespace information ... OK
* checking package dependencies ... NOTE
Depends: includes the non-default packages:
  &#x27;BiocGenerics&#x27;, &#x27;MatrixGenerics&#x27;, &#x27;GenomeInfoDb&#x27;, &#x27;GenomicRanges&#x27;,
  &#x27;SummarizedExperiment&#x27;, &#x27;Rsamtools&#x27;
Adding so many packages to the search path is excessive and importing
selectively is preferable.
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘VariantAnnotation’ can be installed ... OK
* used C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking installed package size ... NOTE
  installed size is  6.9Mb
  sub-directories of 1Mb or more:
    R         2.1Mb
    extdata   1.2Mb
    libs      3.1Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
Unexported objects imported by &#x27;:::&#x27; calls:
  ‘BiocGenerics:::replaceSlots’ ‘BiocGenerics:::testPackage’
  ‘Rsamtools:::.RsamtoolsFile’ ‘Rsamtools:::.RsamtoolsFileList’
  ‘Rsamtools:::.io_check_exists’ ‘S4Vectors:::expandByColumnSet’
  ‘S4Vectors:::labeledLine’ ‘S4Vectors:::recycleVector’
  ‘S4Vectors:::selectSome’
  ‘SummarizedExperiment:::.SummarizedExperiment.charbound’
  ‘SummarizedExperiment:::.cbind.DataFrame’
  ‘rtracklayer:::checkArgFormat’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
VRangesForMatching: no visible binding for global variable ‘REF’
VRangesForMatching: no visible binding for global variable ‘ALT’
Undefined global functions or variables:
  ALT REF
* checking Rd files ... WARNING
checkRd: (5) PROVEANDb-class.Rd:24-27: \item in \describe must have non-empty label
checkRd: (5) PROVEANDb-class.Rd:28-32: \item in \describe must have non-empty label
checkRd: (5) PROVEANDb-class.Rd:33-37: \item in \describe must have non-empty label
checkRd: (5) PROVEANDb-class.Rd:38-42: \item in \describe must have non-empty label
checkRd: (5) PROVEANDb-class.Rd:43-49: \item in \describe must have non-empty label
checkRd: (5) PolyPhenDb-class.Rd:25-28: \item in \describe must have non-empty label
checkRd: (5) PolyPhenDb-class.Rd:29-33: \item in \describe must have non-empty label
checkRd: (5) PolyPhenDb-class.Rd:34-38: \item in \describe must have non-empty label
checkRd: (5) PolyPhenDb-class.Rd:39-45: \item in \describe must have non-empty label
checkRd: (5) PolyPhenDb-class.Rd:46-52: \item in \describe must have non-empty label
checkRd: (5) SIFTDb-class.Rd:24-27: \item in \describe must have non-empty label
checkRd: (5) SIFTDb-class.Rd:28-32: \item in \describe must have non-empty label
checkRd: (5) SIFTDb-class.Rd:33-37: \item in \describe must have non-empty label
checkRd: (5) SIFTDb-class.Rd:38-44: \item in \describe must have non-empty label
checkRd: (5) VCF-class.Rd:106-108: \item in \describe must have non-empty label
checkRd: (5) VCF-class.Rd:109-120: \item in \describe must have non-empty label
checkRd: (5) VCF-class.Rd:128-169: \item in \describe must have non-empty label
checkRd: (5) VCF-class.Rd:170-176: \item in \describe must have non-empty label
checkRd: (5) VCF-class.Rd:177-182: \item in \describe must have non-empty label
checkRd: (5) VCF-class.Rd:183-187: \item in \describe must have non-empty label
checkRd: (5) VCF-class.Rd:188-196: \item in \describe must have non-empty label
checkRd: (5) VCF-class.Rd:197-203: \item in \describe must have non-empty label
checkRd: (5) VCF-class.Rd:204-210: \item in \describe must have non-empty label
checkRd: (5) VCF-class.Rd:211-215: \item in \describe must have non-empty label
checkRd: (5) VCF-class.Rd:216-220: \item in \describe must have non-empty label
checkRd: (5) VCF-class.Rd:221-225: \item in \describe must have non-empty label
checkRd: (5) VCF-class.Rd:226-239: \item in \describe must have non-empty label
checkRd: (5) VCF-class.Rd:240-245: \item in \describe must have non-empty label
checkRd: (5) VCF-class.Rd:253-257: \item in \describe must have non-empty label
checkRd: (5) VCF-class.Rd:258-265: \item in \describe must have non-empty label
checkRd: (5) VCF-class.Rd:266-283: \item in \describe must have non-empty label
checkRd: (5) VCF-class.Rd:290-303: \item in \describe must have non-empty label
checkRd: (5) VCFHeader-class.Rd:47-51: \item in \describe must have non-empty label
checkRd: (5) VCFHeader-class.Rd:58-61: \item in \describe must have non-empty label
checkRd: (5) VCFHeader-class.Rd:62-66: \item in \describe must have non-empty label
checkRd: (5) VCFHeader-class.Rd:67-86: \item in \describe must have non-empty label
checkRd: (5) VCFHeader-class.Rd:87-94: \item in \describe must have non-empty label
checkRd: (5) VCFHeader-class.Rd:95-100: \item in \describe must have non-empty label
checkRd: (5) VCFHeader-class.Rd:101-106: \item in \describe must have non-empty label
checkRd: (5) VCFHeader-class.Rd:107-111: \item in \describe must have non-empty label
checkRd: (5) VCFHeader-class.Rd:112-118: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:146-178: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:179-224: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:232-265: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:266-268: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:269-281: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:290-292: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:293-295: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:296-299: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:300-303: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:304-307: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:308-310: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:311-314: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:315-319: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:320-322: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:323-326: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:327-330: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:336-339: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:340-344: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:345-349: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:355-358: \item in \describe must have non-empty label
checkRd: (5) VRanges-class.Rd:359-372: \item in \describe must have non-empty label
checkRd: (5) VRangesList-class.Rd:35-38: \item in \describe must have non-empty label
checkRd: (5) VRangesList-class.Rd:44-47: \item in \describe must have non-empty label
checkRd: (5) VRangesList-class.Rd:48-52: \item in \describe must have non-empty label
checkRd: (5) VRangesList-class.Rd:58-62: \item in \describe must have non-empty label
checkRd: (5) VariantType-class.Rd:134-138: \item in \describe must have non-empty label
checkRd: (5) VariantType-class.Rd:139-143: \item in \describe must have non-empty label
checkRd: (5) VariantType-class.Rd:144-149: \item in \describe must have non-empty label
checkRd: (5) VariantType-class.Rd:150-154: \item in \describe must have non-empty label
checkRd: (5) VariantType-class.Rd:155-159: \item in \describe must have non-empty label
checkRd: (5) VcfFile-class.Rd:31-34: \item in \describe must have non-empty label
checkRd: (5) VcfFile-class.Rd:35-37: \item in \describe must have non-empty label
checkRd: (5) VcfFile-class.Rd:42-43: \item in \describe must have non-empty label
checkRd: (5) VcfFile-class.Rd:44-45: \item in \describe must have non-empty label
checkRd: (5) VcfFile-class.Rd:46-47: \item in \describe must have non-empty label
checkRd: (5) VcfFile-class.Rd:48-49: \item in \describe must have non-empty label
checkRd: (5) VcfFile-class.Rd:50-51: \item in \describe must have non-empty label
checkRd: (5) VcfFile-class.Rd:52-54: \item in \describe must have non-empty label
checkRd: (5) VcfFile-class.Rd:59-60: \item in \describe must have non-empty label
checkRd: (5) VcfFile-class.Rd:61-62: \item in \describe must have non-empty label
checkRd: (5) VcfFile-class.Rd:67-69: \item in \describe must have non-empty label
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... NOTE
GNU make is a SystemRequirements.
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... WARNING
Note: information on .o files is not available
File ‘/home/biocbuild/bbs-3.17-bioc/R/library/VariantAnnotation/libs/VariantAnnotation.so’:
  Found ‘__sprintf_chk’, possibly from ‘sprintf’ (C)
  Found ‘abort’, possibly from ‘abort’ (C)
  Found ‘exit’, possibly from ‘exit’ (C)
  Found ‘stderr’, possibly from ‘stderr’ (C)
  Found ‘stdout’, possibly from ‘stdout’ (C)
File ‘VariantAnnotation/libs/VariantAnnotation.so’:
  Found non-API calls to R: ‘R_GetConnection’, ‘R_WriteConnection’

Compiled code should not call entry points which might terminate R nor
write to stdout/stderr instead of to the console, nor use Fortran I/O
nor system RNGs nor [v]sprintf. The detected symbols are linked into
the code but might come from libraries and not actually be called.
Compiled code should not call non-API entry points in R.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                            user system elapsed
locateVariants-methods    27.138  1.212  28.354
predictCoding-methods     22.717  0.516  23.239
summarizeVariants-methods  6.775  0.064   6.839
PROVEANDb-class            4.779  0.634   8.809
filterVcf-methods          3.403  0.252  10.270
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘VariantAnnotation_unit_tests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘VariantAnnotation.Rnw’... OK
  ‘filterVcf.Rnw’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 2 WARNINGs, 5 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/VariantAnnotation.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">VariantAnnotation.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL VariantAnnotation
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘VariantAnnotation’ ...
** using staged installation
** libs
using C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c Biostrings_stubs.c -o Biostrings_stubs.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c IRanges_stubs.c -o IRanges_stubs.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c R_init_VariantAnnotation.c -o R_init_VariantAnnotation.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c XVector_stubs.c -o XVector_stubs.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c dna_hash.c -o dna_hash.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c rle.c -o rle.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c strhash.c -o strhash.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c utilities.c -o utilities.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c vcffile.c -o vcffile.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c vcftype.c -o vcftype.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -D_FILE_OFFSET_BITS=64 -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/S4Vectors/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/IRanges/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/XVector/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Biostrings/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c writevcf.c -o writevcf.o
gcc -shared -L/home/biocbuild/bbs-3.17-bioc/R/lib -L/usr/local/lib -o VariantAnnotation.so Biostrings_stubs.o IRanges_stubs.o R_init_VariantAnnotation.o XVector_stubs.o dna_hash.o rle.o strhash.o utilities.o vcffile.o vcftype.o writevcf.o /home/biocbuild/bbs-3.17-bioc/R/library/Rhtslib/usrlib/libhts.a -lcurl -L/home/biocbuild/bbs-3.17-bioc/R/lib -lR
installing to /home/biocbuild/bbs-3.17-bioc/R/library/00LOCK-VariantAnnotation/00new/VariantAnnotation/libs
** R
** inst
** byte-compile and prepare package for lazy loading
Creating a new generic function for ‘tabulate’ in package ‘VariantAnnotation’
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (VariantAnnotation)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">VariantAnnotation.Rcheck/tests/VariantAnnotation_unit_tests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; require(&quot;VariantAnnotation&quot;) || stop(&quot;unable to load VariantAnnotation package&quot;)
Loading required package: VariantAnnotation
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: MatrixGenerics
Loading required package: matrixStats

Attaching package: &#x27;MatrixGenerics&#x27;

The following objects are masked from &#x27;package:matrixStats&#x27;:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars

Loading required package: GenomeInfoDb
Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomicRanges
Loading required package: SummarizedExperiment
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


Attaching package: &#x27;Biobase&#x27;

The following object is masked from &#x27;package:MatrixGenerics&#x27;:

    rowMedians

The following objects are masked from &#x27;package:matrixStats&#x27;:

    anyMissing, rowMedians

Loading required package: Rsamtools
Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit


Attaching package: &#x27;VariantAnnotation&#x27;

The following object is masked from &#x27;package:base&#x27;:

    tabulate

[1] TRUE
&gt; VariantAnnotation:::.test()
Loading required package: RSQLite
[W::tbx_parse1] VCF INFO/END=2827680 is smaller than POS at 1:2827692
This tag will be ignored. Note: only one invalid END tag will be reported.
[W::bcf_hdr_register_hrec] The definition of Flag &quot;INFO/TS&quot; is invalid, forcing Number=0
[W::bcf_hdr_register_hrec] The definition of Flag &quot;INFO/TS&quot; is invalid, forcing Number=0
starting prefilter
prefiltering 10376 records
prefiltered to /tmp/RtmpOkGNxY/file9e12660767b4c
compressing and indexing &#x27;/tmp/RtmpOkGNxY/file9e12660767b4c&#x27;
starting filter
filtering 10376 records
completed filtering
compressing and indexing &#x27;/tmp/RtmpOkGNxY/file9e12652dea406&#x27;
Loading required package: survival
Loading required package: Matrix

Attaching package: &#x27;Matrix&#x27;

The following object is masked from &#x27;package:VariantAnnotation&#x27;:

    expand

The following object is masked from &#x27;package:S4Vectors&#x27;:

    expand

[W::bcf_hdr_check_sanity] GL should be declared as Number=G
non-single nucleotide variations are set to NA
non-single nucleotide variations are set to NA
non-single nucleotide variations are set to NA
non-single nucleotide variations are set to NA
non-single nucleotide variations are set to NA
non-single nucleotide variations are set to NA
Loading required package: GenomicFeatures
Loading required package: AnnotationDbi
&#x27;select()&#x27; returned 1:1 mapping between keys and columns
Loading required package: BSgenome
Loading required package: rtracklayer
&#x27;select()&#x27; returned many:1 mapping between keys and columns
[W::bcf_hdr_check_sanity] PL should be declared as Number=G


RUNIT TEST PROTOCOL -- Sat Feb 11 23:56:08 2023 
*********************************************** 
Number of test functions: 100 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
VariantAnnotation RUnit Tests - 100 test functions, 0 errors, 0 failures
Number of test functions: 100 
Number of errors: 0 
Number of failures: 0 
Warning messages:
1: info fields with no header: noMatch 
2: In .bcfHeaderAsSimpleList(header) :
  duplicate keys in header will be forced to unique rownames
3: In DataFrame(..., check.names = FALSE) : NAs introduced by coercion
4: In valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
  GRanges object contains 7 out-of-bound ranges located on sequence
  70477. Note that ranges located on a sequence whose length is unknown
  (NA) or on a circular sequence are not considered out-of-bound (use
  seqlengths() and isCircular() to get the lengths and circularity flags
  of the underlying sequences). You can use trim() to trim these ranges.
  See ?`trim,GenomicRanges-method` for more information.
5: In valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
  GRanges object contains 6 out-of-bound ranges located on sequence
  70477. Note that ranges located on a sequence whose length is unknown
  (NA) or on a circular sequence are not considered out-of-bound (use
  seqlengths() and isCircular() to get the lengths and circularity flags
  of the underlying sequences). You can use trim() to trim these ranges.
  See ?`trim,GenomicRanges-method` for more information.
&gt; 
&gt; proc.time()
   user  system elapsed 
147.787   3.020 153.308 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">VariantAnnotation.Rcheck/VariantAnnotation-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>GLtoGP</TD><TD style="text-align: right;">1.523</TD><TD style="text-align: right;">0.233</TD><TD style="text-align: right;">1.759</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>PROVEANDb-class</TD><TD style="text-align: right;">4.779</TD><TD style="text-align: right;">0.634</TD><TD style="text-align: right;">8.809</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>PolyPhenDb-class</TD><TD style="text-align: right;">2.328</TD><TD style="text-align: right;">0.149</TD><TD style="text-align: right;">4.374</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SIFTDb-class</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ScanVcfParam-class</TD><TD style="text-align: right;">1.192</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">1.201</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>VCF-class</TD><TD style="text-align: right;">2.024</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">2.073</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>VCFHeader-class</TD><TD style="text-align: right;">0.104</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.108</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>VRanges-class</TD><TD style="text-align: right;">0.601</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.602</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>VRangesList-class</TD><TD style="text-align: right;">0.568</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.568</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>VariantType-class</TD><TD style="text-align: right;">0.006</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.009</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>VcfFile-class</TD><TD style="text-align: right;">0.775</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.778</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filterVcf-methods</TD><TD style="text-align: right;"> 3.403</TD><TD style="text-align: right;"> 0.252</TD><TD style="text-align: right;">10.270</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genotypeToSnpMatrix-methods</TD><TD style="text-align: right;">1.657</TD><TD style="text-align: right;">0.300</TD><TD style="text-align: right;">1.957</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getTranscriptSeqs-methods</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>indexVcf-method</TD><TD style="text-align: right;">0.010</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>isSNV-methods</TD><TD style="text-align: right;">1.256</TD><TD style="text-align: right;">0.100</TD><TD style="text-align: right;">1.357</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>locateVariants-methods</TD><TD style="text-align: right;">27.138</TD><TD style="text-align: right;"> 1.212</TD><TD style="text-align: right;">28.354</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>predictCoding-methods</TD><TD style="text-align: right;">22.717</TD><TD style="text-align: right;"> 0.516</TD><TD style="text-align: right;">23.239</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>probabilityToSnpMatrix</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>readVcf-methods</TD><TD style="text-align: right;">4.768</TD><TD style="text-align: right;">0.124</TD><TD style="text-align: right;">4.893</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>scanVcf-methods</TD><TD style="text-align: right;">0.336</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.346</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>seqinfo-method</TD><TD style="text-align: right;">0.090</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.093</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>snpSummary</TD><TD style="text-align: right;">0.414</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.418</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>summarizeVariants-methods</TD><TD style="text-align: right;">6.775</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">6.839</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>writeVcf-methods</TD><TD style="text-align: right;">2.314</TD><TD style="text-align: right;">0.048</TD><TD style="text-align: right;">2.362</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
