<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for infercnv on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><B>[<A href="../index.html#I">I</A>]</B></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-10 01:25:41 -0000 (Fri, 10 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4219</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for infercnv on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the infercnv package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/infercnv.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>960</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">infercnv</A>&nbsp;1.15.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/infercnv">landing page</A>)</SPAN><BR>Christophe Georgescu<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-09&nbsp;03:02:02&nbsp;-0000&nbsp;(Thu,&nbsp;09&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/infercnv</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">6753002</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-02-02&nbsp;00:04:25&nbsp;-0000&nbsp;(Thu,&nbsp;02&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: infercnv</TD></TR>
<TR><TD><B>Version</B>: 1.15.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:infercnv.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings infercnv_1.15.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-09 16:03:46 -0000 (Thu, 09 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-09 16:17:03 -0000 (Thu, 09 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 797.1 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: infercnv.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:infercnv.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings infercnv_1.15.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/infercnv.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘infercnv/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘infercnv’ version ‘1.15.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘infercnv’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
Unexported object imported by a &#x27;:::&#x27; call: ‘HiddenMarkov:::makedensity’
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... WARNING
Undocumented arguments in documentation object &#x27;run&#x27;
  ‘per_chr_hmm_subclusters_references’

Functions with \usage entries need to have the appropriate \alias
entries, and all their arguments documented.
The \usage entries must correspond to syntactically valid R code.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking LazyData ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘infercnv-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: inferCNVBayesNet
&gt; ### Title: inferCNVBayesNet: Run Bayesian Network Mixture Model To Obtain
&gt; ###   Posterior Probabilities For HMM Predicted States
&gt; ### Aliases: inferCNVBayesNet
&gt; 
&gt; ### ** Examples
&gt; 
&gt; data(infercnv_data_example)
&gt; data(infercnv_annots_example)
&gt; data(infercnv_genes_example)
&gt; data(HMM_states)
&gt; 
&gt; infercnv_object_example &lt;- infercnv::CreateInfercnvObject(raw_counts_matrix=infercnv_data_example, 
+                                                           gene_order_file=infercnv_genes_example,
+                                                           annotations_file=infercnv_annots_example,
+                                                           ref_group_names=c(&quot;normal&quot;))
INFO [2023-02-09 16:08:48] ::order_reduce:Start.
INFO [2023-02-09 16:08:48] .order_reduce(): expr and order match.
INFO [2023-02-09 16:08:48] ::process_data:order_reduce:Reduction from positional data, new dimensions (r,c) = 8252,20 Total=2817798 Min=0 Max=6927.
INFO [2023-02-09 16:08:48] num genes removed taking into account provided gene ordering list: 2308 = 27.9689772176442% removed.
INFO [2023-02-09 16:08:48] -filtering out cells &lt; 100 or &gt; Inf, removing 0 % of cells
WARN [2023-02-09 16:08:48] Please use &quot;options(scipen = 100)&quot; before running infercnv if you are using the analysis_mode=&quot;subclusters&quot; option or you may encounter an error while the hclust is being generated.
INFO [2023-02-09 16:08:48] validating infercnv_obj
&gt;           
&gt; out_dir = tempfile()
&gt; infercnv_object_example &lt;- infercnv::run(infercnv_object_example,
+                                          cutoff=1,
+                                          out_dir=out_dir, 
+                                          cluster_by_groups=TRUE,
+                                          analysis_mode=&quot;samples&quot;,
+                                          denoise=TRUE,
+                                          HMM=TRUE,
+                                          num_threads=2,
+                                          no_plot=TRUE)
INFO [2023-02-09 16:08:48] ::process_data:Start
INFO [2023-02-09 16:08:48] Creating output path /tmp/Rtmp8eSgjD/file3acab73ed4197f
INFO [2023-02-09 16:08:48] Checking for saved results.
INFO [2023-02-09 16:08:48] 

	STEP 1: incoming data

INFO [2023-02-09 16:08:48] 

	STEP 02: Removing lowly expressed genes

INFO [2023-02-09 16:08:48] ::above_min_mean_expr_cutoff:Start
INFO [2023-02-09 16:08:48] Removing 1161 genes from matrix as below mean expr threshold: 1
INFO [2023-02-09 16:08:48] validating infercnv_obj
INFO [2023-02-09 16:08:48] There are 4783 genes and 20 cells remaining in the expr matrix.
INFO [2023-02-09 16:08:48] Removed 170 genes having fewer than 3 min cells per gene = 3.55425 % genes removed here
INFO [2023-02-09 16:08:48] validating infercnv_obj
INFO [2023-02-09 16:08:48] 

	STEP 03: normalization by sequencing depth

INFO [2023-02-09 16:08:48] normalizing counts matrix by depth
INFO [2023-02-09 16:08:48] Computed total sum normalization factor as median libsize: 102587.500000
INFO [2023-02-09 16:08:48] Adding h-spike
INFO [2023-02-09 16:08:48] -hspike modeling of normal
INFO [2023-02-09 16:10:07] validating infercnv_obj
INFO [2023-02-09 16:10:07] normalizing counts matrix by depth
INFO [2023-02-09 16:10:07] Using specified normalization factor: 102587.500000
INFO [2023-02-09 16:10:08] 

	STEP 04: log transformation of data

INFO [2023-02-09 16:10:08] transforming log2xplus1()
INFO [2023-02-09 16:10:08] -mirroring for hspike
INFO [2023-02-09 16:10:08] transforming log2xplus1()
INFO [2023-02-09 16:10:09] 

	STEP 08: removing average of reference data (before smoothing)

INFO [2023-02-09 16:10:09] ::subtract_ref_expr_from_obs:Start inv_log=FALSE, use_bounds=TRUE
INFO [2023-02-09 16:10:09] subtracting mean(normal) per gene per cell across all data
INFO [2023-02-09 16:10:12] -subtracting expr per gene, use_bounds=TRUE
INFO [2023-02-09 16:10:12] -mirroring for hspike
INFO [2023-02-09 16:10:12] ::subtract_ref_expr_from_obs:Start inv_log=FALSE, use_bounds=TRUE
INFO [2023-02-09 16:10:12] subtracting mean(normal) per gene per cell across all data
INFO [2023-02-09 16:10:15] -subtracting expr per gene, use_bounds=TRUE
INFO [2023-02-09 16:10:18] 

	STEP 09: apply max centered expression threshold: 3

INFO [2023-02-09 16:10:18] ::process_data:setting max centered expr, threshold set to: +/-:  3
INFO [2023-02-09 16:10:18] -mirroring for hspike
INFO [2023-02-09 16:10:18] ::process_data:setting max centered expr, threshold set to: +/-:  3
INFO [2023-02-09 16:10:20] 

	STEP 10: Smoothing data per cell by chromosome

INFO [2023-02-09 16:10:20] smooth_by_chromosome: chr: chr1
INFO [2023-02-09 16:10:20] smooth_by_chromosome: chr: chr2
INFO [2023-02-09 16:10:20] smooth_by_chromosome: chr: chr3
INFO [2023-02-09 16:10:20] smooth_by_chromosome: chr: chr4
INFO [2023-02-09 16:10:20] smooth_by_chromosome: chr: chr5
INFO [2023-02-09 16:10:20] smooth_by_chromosome: chr: chr6
INFO [2023-02-09 16:10:20] smooth_by_chromosome: chr: chr7
INFO [2023-02-09 16:10:20] smooth_by_chromosome: chr: chr8
INFO [2023-02-09 16:10:20] smooth_by_chromosome: chr: chr9
INFO [2023-02-09 16:10:20] smooth_by_chromosome: chr: chr10
INFO [2023-02-09 16:10:20] smooth_by_chromosome: chr: chr11
INFO [2023-02-09 16:10:20] smooth_by_chromosome: chr: chr12
INFO [2023-02-09 16:10:21] smooth_by_chromosome: chr: chr13
INFO [2023-02-09 16:10:21] smooth_by_chromosome: chr: chr14
INFO [2023-02-09 16:10:21] smooth_by_chromosome: chr: chr15
INFO [2023-02-09 16:10:21] smooth_by_chromosome: chr: chr16
INFO [2023-02-09 16:10:21] smooth_by_chromosome: chr: chr17
INFO [2023-02-09 16:10:21] smooth_by_chromosome: chr: chr18
INFO [2023-02-09 16:10:21] smooth_by_chromosome: chr: chr19
INFO [2023-02-09 16:10:21] smooth_by_chromosome: chr: chr20
INFO [2023-02-09 16:10:21] smooth_by_chromosome: chr: chr21
INFO [2023-02-09 16:10:21] smooth_by_chromosome: chr: chr22
INFO [2023-02-09 16:10:21] -mirroring for hspike
INFO [2023-02-09 16:10:21] smooth_by_chromosome: chr: chrA
INFO [2023-02-09 16:10:22] smooth_by_chromosome: chr: chr_0
INFO [2023-02-09 16:10:22] smooth_by_chromosome: chr: chr_B
INFO [2023-02-09 16:10:23] smooth_by_chromosome: chr: chr_0pt5
INFO [2023-02-09 16:10:23] smooth_by_chromosome: chr: chr_C
INFO [2023-02-09 16:10:24] smooth_by_chromosome: chr: chr_1pt5
INFO [2023-02-09 16:10:25] smooth_by_chromosome: chr: chr_D
INFO [2023-02-09 16:10:25] smooth_by_chromosome: chr: chr_2pt0
INFO [2023-02-09 16:10:26] smooth_by_chromosome: chr: chr_E
INFO [2023-02-09 16:10:26] smooth_by_chromosome: chr: chr_3pt0
INFO [2023-02-09 16:10:26] smooth_by_chromosome: chr: chr_F
INFO [2023-02-09 16:10:29] 

	STEP 11: re-centering data across chromosome after smoothing

INFO [2023-02-09 16:10:29] ::center_smooth across chromosomes per cell
INFO [2023-02-09 16:10:29] -mirroring for hspike
INFO [2023-02-09 16:10:29] ::center_smooth across chromosomes per cell
INFO [2023-02-09 16:10:30] 

	STEP 12: removing average of reference data (after smoothing)

INFO [2023-02-09 16:10:31] ::subtract_ref_expr_from_obs:Start inv_log=FALSE, use_bounds=TRUE
INFO [2023-02-09 16:10:31] subtracting mean(normal) per gene per cell across all data
INFO [2023-02-09 16:10:33] -subtracting expr per gene, use_bounds=TRUE
INFO [2023-02-09 16:10:33] -mirroring for hspike
INFO [2023-02-09 16:10:33] ::subtract_ref_expr_from_obs:Start inv_log=FALSE, use_bounds=TRUE
INFO [2023-02-09 16:10:33] subtracting mean(normal) per gene per cell across all data
INFO [2023-02-09 16:10:36] -subtracting expr per gene, use_bounds=TRUE
INFO [2023-02-09 16:10:38] 

	STEP 14: invert log2(FC) to FC

INFO [2023-02-09 16:10:38] invert_log2(), computing 2^x
INFO [2023-02-09 16:10:38] -mirroring for hspike
INFO [2023-02-09 16:10:38] invert_log2(), computing 2^x
INFO [2023-02-09 16:10:40] 

	STEP 15: Clustering samples (not defining tumor subclusters)

INFO [2023-02-09 16:10:40] define_signif_tumor_subclusters(p_val=0.1
INFO [2023-02-09 16:10:40] define_signif_tumor_subclusters(), tumor: tumor
INFO [2023-02-09 16:10:40] cut tree into: 1 groups
INFO [2023-02-09 16:10:40] -processing tumor,tumor_s1
INFO [2023-02-09 16:10:40] define_signif_tumor_subclusters(), tumor: normal
INFO [2023-02-09 16:10:40] cut tree into: 1 groups
INFO [2023-02-09 16:10:40] -processing normal,normal_s1
INFO [2023-02-09 16:10:40] -mirroring for hspike
INFO [2023-02-09 16:10:40] define_signif_tumor_subclusters(p_val=0.1
INFO [2023-02-09 16:10:40] define_signif_tumor_subclusters(), tumor: spike_tumor_cell_normal
INFO [2023-02-09 16:10:41] cut tree into: 1 groups
INFO [2023-02-09 16:10:41] -processing spike_tumor_cell_normal,spike_tumor_cell_normal_s1
INFO [2023-02-09 16:10:41] define_signif_tumor_subclusters(), tumor: simnorm_cell_normal
INFO [2023-02-09 16:10:41] cut tree into: 1 groups
INFO [2023-02-09 16:10:41] -processing simnorm_cell_normal,simnorm_cell_normal_s1
INFO [2023-02-09 16:10:43] 

	STEP 17: HMM-based CNV prediction

INFO [2023-02-09 16:10:43] predict_CNV_via_HMM_on_whole_tumor_samples
INFO [2023-02-09 16:10:46] -done predicting CNV based on initial tumor subclusters
INFO [2023-02-09 16:10:46] get_predicted_CNV_regions(subcluster)
INFO [2023-02-09 16:10:46] -processing cell_group_name: tumor.tumor_s1, size: 10
INFO [2023-02-09 16:10:53] -processing cell_group_name: normal.normal_s1, size: 10
INFO [2023-02-09 16:11:02] -writing cell clusters file: /tmp/Rtmp8eSgjD/file3acab73ed4197f/17_HMM_predHMMi6.hmm_mode-samples.cell_groupings
INFO [2023-02-09 16:11:02] -writing cnv regions file: /tmp/Rtmp8eSgjD/file3acab73ed4197f/17_HMM_predHMMi6.hmm_mode-samples.pred_cnv_regions.dat
INFO [2023-02-09 16:11:02] -writing per-gene cnv report: /tmp/Rtmp8eSgjD/file3acab73ed4197f/17_HMM_predHMMi6.hmm_mode-samples.pred_cnv_genes.dat
INFO [2023-02-09 16:11:02] -writing gene ordering info: /tmp/Rtmp8eSgjD/file3acab73ed4197f/17_HMM_predHMMi6.hmm_mode-samples.genes_used.dat
INFO [2023-02-09 16:11:11] 

	STEP 18: Run Bayesian Network Model on HMM predicted CNVs

INFO [2023-02-09 16:11:12] Creating the following Directory:  /tmp/Rtmp8eSgjD/file3acab73ed4197f/BayesNetOutput.HMMi6.hmm_mode-samples
INFO [2023-02-09 16:11:12] Initializing new MCM InferCNV Object.
INFO [2023-02-09 16:11:12] validating infercnv_obj
INFO [2023-02-09 16:11:12] Total CNV&#x27;s:  14
INFO [2023-02-09 16:11:12] Loading BUGS Model.
INFO [2023-02-09 16:11:12] Running Sampling Using Parallel with  2 Cores
Killed
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘testthat.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘inferCNV.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 1 WARNING, 1 NOTE
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/infercnv.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">infercnv.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL infercnv
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘infercnv’ ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (infercnv)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">infercnv.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; #!/usr/bin/env Rscript
&gt; 
&gt; library(testthat)
&gt; library(infercnv)
&gt; 
&gt; test_check(&quot;infercnv&quot;)
INFO [2023-02-09 16:14:34] ::subtract_ref_expr_from_obs:Start inv_log=FALSE, use_bounds=TRUE
INFO [2023-02-09 16:14:34] subtracting mean(normal) per gene per cell across all data
INFO [2023-02-09 16:14:34] -subtracting expr per gene, use_bounds=TRUE
INFO [2023-02-09 16:14:34] ::subtract_ref_expr_from_obs:Start inv_log=FALSE, use_bounds=TRUE
INFO [2023-02-09 16:14:34] subtracting mean(normal) per gene per cell across all data
INFO [2023-02-09 16:14:34] -subtracting expr per gene, use_bounds=TRUE
INFO [2023-02-09 16:14:34] ::subtract_ref_expr_from_obs:Start inv_log=FALSE, use_bounds=TRUE
INFO [2023-02-09 16:14:34] subtracting mean(normal) per gene per cell across all data
INFO [2023-02-09 16:14:34] -subtracting expr per gene, use_bounds=TRUE
INFO [2023-02-09 16:14:34] ::subtract_ref_expr_from_obs:Start inv_log=FALSE, use_bounds=TRUE
INFO [2023-02-09 16:14:34] subtracting mean(normal) per gene per cell across all data
INFO [2023-02-09 16:14:34] -subtracting expr per gene, use_bounds=TRUE
INFO [2023-02-09 16:14:34] ::subtract_ref_expr_from_obs:Start inv_log=FALSE, use_bounds=TRUE
INFO [2023-02-09 16:14:34] subtracting mean(normal) per gene per cell across all data
INFO [2023-02-09 16:14:34] -subtracting expr per gene, use_bounds=TRUE
INFO [2023-02-09 16:14:34] ::subtract_ref_expr_from_obs:Start inv_log=FALSE, use_bounds=TRUE
INFO [2023-02-09 16:14:34] subtracting mean(normal) per gene per cell across all data
INFO [2023-02-09 16:14:34] -subtracting expr per gene, use_bounds=TRUE
WARN [2023-02-09 16:14:34] window length &lt; 2, returning original unmodified data
WARN [2023-02-09 16:14:34] window length &lt; 2, returning original unmodified data
INFO [2023-02-09 16:14:48] ::remove_outlier_norm:Start out_method: average_bound lower_bound: -1 upper_bound: 30
INFO [2023-02-09 16:14:48] ::remove_outlier_norm: using hard thresholds:  lower_bound: -1 upper_bound: 30
INFO [2023-02-09 16:14:48] ::remove_outlier_norm:Start out_method: average_bound lower_bound: 5 upper_bound: 15
INFO [2023-02-09 16:14:48] ::remove_outlier_norm: using hard thresholds:  lower_bound: 5 upper_bound: 15
INFO [2023-02-09 16:14:48] ::remove_outlier_norm:Start out_method: average_bound lower_bound: NA upper_bound: NA
INFO [2023-02-09 16:14:48] ::remove_outlier_norm using method: average_bound for defining outliers.
INFO [2023-02-09 16:14:48] outlier bounds defined between: -0.5 - 17.75
INFO [2023-02-09 16:14:48] ::order_reduce:Start.
INFO [2023-02-09 16:14:48] ::order_reduce:Start.
INFO [2023-02-09 16:14:48] .order_reduce(): expr and order match.
INFO [2023-02-09 16:14:48] ::process_data:order_reduce:Reduction from positional data, new dimensions (r,c) = 10,2 Total=110 Min=1 Max=10.
INFO [2023-02-09 16:14:48] ::order_reduce:Start.
INFO [2023-02-09 16:14:48] .order_reduce(): expr and order match.
INFO [2023-02-09 16:14:49] ::process_data:order_reduce:Reduction from positional data, new dimensions (r,c) = 10,2 Total=110 Min=1 Max=10.
INFO [2023-02-09 16:14:49] ::order_reduce:Start.
INFO [2023-02-09 16:14:49] ::process_data:order_reduce:The position file  and the expression file row (gene) names do not match.
[ FAIL 0 | WARN 0 | SKIP 3 | PASS 39 ]

══ Skipped tests ═══════════════════════════════════════════════════════════════
• empty test (3)

[ FAIL 0 | WARN 0 | SKIP 3 | PASS 39 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
 15.675  19.659  99.778 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">infercnv.Rcheck/infercnv-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>CreateInfercnvObject</TD><TD style="text-align: right;">0.913</TD><TD style="text-align: right;">0.005</TD><TD style="text-align: right;">0.918</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>apply_median_filtering</TD><TD style="text-align: right;">6.728</TD><TD style="text-align: right;">0.156</TD><TD style="text-align: right;">6.884</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>color.palette</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>filterHighPNormals</TD><TD style="text-align: right;">0.017</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.018</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
