<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for bcSeq on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><B>[<A href="..#B">B</A>]</B></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-10 10:04:26 -0000 (Wed, 10 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-03-12&nbsp;r83975)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">6211</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for bcSeq on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the bcSeq package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/bcSeq.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>140</B>/2194</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">bcSeq</A>&nbsp;1.23.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/bcSeq">landing page</A>)</SPAN><BR>Jiaxing Lin<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-08&nbsp;19:11:19&nbsp;-0000&nbsp;(Mon,&nbsp;08&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/bcSeq</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">8c2f02d</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-25&nbsp;14:58:15&nbsp;-0000&nbsp;(Tue,&nbsp;25&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: bcSeq</TD></TR>
<TR><TD><B>Version</B>: 1.23.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:bcSeq.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings bcSeq_1.23.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-09 09:21:05 -0000 (Tue, 09 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-09 09:23:15 -0000 (Tue, 09 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 130.2 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: bcSeq.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:bcSeq.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings bcSeq_1.23.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/bcSeq.Rcheck’
* using R Under development (unstable) (2023-03-12 r83975)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘bcSeq/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘bcSeq’ version ‘1.23.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘bcSeq’ can be installed ... OK
* used C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* used C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking C++ specification ... NOTE
  Specified C++11: please drop specification unless essential
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... OK
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... NOTE
Note: information on .o files is not available
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘test_bcSeq.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘bcSeq.Rnw’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 2 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/bcSeq.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">bcSeq.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL bcSeq
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library’
* installing *source* package ‘bcSeq’ ...
** using staged installation
** libs
using C compiler: ‘gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
using C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
using C++11
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/library/Matrix/include&#x27; -I/usr/local/include   -pthread -fPIC  -g -O2  -c CRISPR_matching.cpp -o CRISPR_matching.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/library/Matrix/include&#x27; -I/usr/local/include   -pthread -fPIC  -g -O2  -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/library/Matrix/include&#x27; -I/usr/local/include   -pthread -fPIC  -g -O2  -c Trie_edit.cpp -o Trie_edit.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/library/Matrix/include&#x27; -I/usr/local/include   -pthread -fPIC  -g -O2  -c Trie_hamming.cpp -o Trie_hamming.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/library/Matrix/include&#x27; -I/usr/local/include   -pthread -fPIC  -g -O2  -c Trie_util.cpp -o Trie_util.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/library/Matrix/include&#x27; -I/usr/local/include   -pthread -fPIC  -g -O2  -c alignment.cpp -o alignment.o
gcc -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/library/Matrix/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -c init.c -o init.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/library/Matrix/include&#x27; -I/usr/local/include   -pthread -fPIC  -g -O2  -c trimRead.cpp -o trimRead.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/include&quot; -DNDEBUG  -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/library/Rcpp/include&#x27; -I&#x27;/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/lib/R/library/Matrix/include&#x27; -I/usr/local/include   -pthread -fPIC  -g -O2  -c uniqueBar.cpp -o uniqueBar.o
g++ -std=gnu++11 -shared -L/usr/local/lib -o bcSeq.so CRISPR_matching.o RcppExports.o Trie_edit.o Trie_hamming.o Trie_util.o alignment.o init.o trimRead.o uniqueBar.o
installing to /home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library/00LOCK-bcSeq/00new/bcSeq/libs
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (bcSeq)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">bcSeq.Rcheck/tests/test_bcSeq.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(bcSeq)
&gt; #devtools::load_all(&quot;../&quot;)
&gt; #### Set the seed
&gt; set.seed(4523)
&gt; 
&gt; #### Generate barcode
&gt; lFName    &lt;- &quot;./libFile.fasta&quot;
&gt; bases     &lt;- c(rep(&#x27;A&#x27;, 4), rep(&#x27;C&#x27;,4), rep(&#x27;G&#x27;,4), rep(&#x27;T&#x27;,4))
&gt; numOfBars &lt;- 7
&gt; Barcodes  &lt;- rep(NA, numOfBars*2)
&gt; for (i in 1:numOfBars){
+     Barcodes[2*i-1] &lt;- paste0(&quot;&gt;barcode_ID: &quot;, i)
+     Barcodes[2*i]   &lt;- paste(sample(bases, length(bases)), collapse = &#x27;&#x27;)
+ }
&gt; write(Barcodes, lFName)
&gt; 
&gt; #### Generate reads and phred score
&gt; rFName     &lt;- &quot;./readFile.fastq&quot;
&gt; numOfReads &lt;- 8
&gt; Reads      &lt;- rep(NA, numOfReads*4)
&gt; for (i in 1:numOfReads){
+     Reads[4*i-3] &lt;- paste0(&quot;@read_ID_&quot;,i)
+     Reads[4*i-2] &lt;- Barcodes[2*sample(1:numOfBars,1,
+                   replace=TRUE, prob=seq(1:numOfBars))]
+     Reads[4*i-1] &lt;- &quot;+&quot;
+     Reads[4*i]   &lt;- paste(rawToChar(as.raw(
+                   33+sample(20:30, length(bases),replace=TRUE))),
+                   collapse=&#x27;&#x27;)
+ }
&gt; write(Reads, rFName)
&gt; 
&gt; #### perform alignment 
&gt; ReadFile &lt;- &quot;./readFile.fastq&quot;
&gt; BarFile  &lt;- &quot;./libFile.fasta&quot;
&gt; outFile  &lt;- &quot;./countH.csv&quot;
&gt; 
&gt; #### with default output for bcSeq_hamming
&gt; #res &lt;- bcSeq_hamming(ReadFile, BarFile, outFile, misMatch = 2,
&gt; #    tMat = NULL, numThread = 2, count_only = TRUE )
&gt; #res &lt;- read.csv(outFile, header=FALSE)
&gt; #res
&gt; 
&gt; #### with return of alignment probability matrix to R
&gt; #outFile  &lt;- &quot;./countH2.csv&quot;
&gt; #res &lt;- bcSeq_hamming(ReadFile, BarFile, outFile, misMatch = 2,
&gt; #    tMat = NULL, numThread = 2, count_only = FALSE )
&gt; #res 
&gt; 
&gt; #### with default output for bcSeq_edit
&gt; outFile  &lt;- &quot;./countE.csv&quot;
&gt; #res &lt;- bcSeq_edit(ReadFile, BarFile, outFile, misMatch = 2,
&gt; #    tMat = NULL, numThread = 2, count_only = TRUE,
&gt; #    gap_left = 2, ext_left = 1, gap_right = 2, ext_right = 1,
&gt; #    pen_max = 7)
&gt; #res &lt;- read.csv(outFile, header=FALSE)
&gt; #res
&gt; 
&gt; #### with return of alignment probability matrix to R
&gt; #outFile  &lt;- &quot;./countE2.csv&quot;
&gt; #res &lt;- bcSeq_edit(ReadFile, BarFile, outFile, misMatch = 2,
&gt; #    tMat = NULL, numThread = 2, count_only = FALSE,
&gt; #    gap_left = 2, ext_left = 1, gap_right = 2, ext_right = 1,
&gt; #    pen_max = 7)
&gt; #res
&gt; 
&gt; #### user-defined probability model
&gt; comstomizeP &lt;- function(m, x, y)
+ {
+     x * (1 - log(2) + log(1 + m / (m + y) ) )
+ }
&gt; outFile = &quot;comstomizeP.csv&quot;
&gt; #bcSeq_edit(ReadFile, BarFile, outFile, misMatch = 2,
&gt; #    tMat = NULL, numThread = 2, count_only = TRUE,
&gt; #    gap_left = 2, ext_left = 1, gap_right = 2, ext_right = 1,
&gt; #    pen_max = 7, userProb = comstomizeP)
&gt; 
&gt; proc.time()
   user  system elapsed 
  4.932   0.384   5.296 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">bcSeq.Rcheck/bcSeq-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>bcSeq-package</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.038</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bcSeq_edit</TD><TD style="text-align: right;">0.076</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.088</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bcSeq_hamming</TD><TD style="text-align: right;">0.072</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.088</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>trimRead</TD><TD style="text-align: right;">0.067</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.095</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>uniqueBar</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
