<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for metagene on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><B>[<A href="../index.html#M">M</A>]</B></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-01-29 16:33:58 -0000 (Sun, 29 Jan 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4021</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for metagene on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the metagene package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/metagene.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1133</B>/2162</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">metagene</A>&nbsp;2.31.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/metagene">landing page</A>)</SPAN><BR>Charles Joly Beauparlant<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-01-27&nbsp;13:09:27&nbsp;-0000&nbsp;(Fri,&nbsp;27&nbsp;Jan&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/metagene</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">7597c31</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:10:04&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: metagene</TD></TR>
<TR><TD><B>Version</B>: 2.31.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:metagene.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings metagene_2.31.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-01-29 03:16:42 -0000 (Sun, 29 Jan 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-01-29 03:25:46 -0000 (Sun, 29 Jan 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 543.9 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: metagene.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:metagene.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings metagene_2.31.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/metagene.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘metagene/DESCRIPTION’ ... OK
* this is package ‘metagene’ version ‘2.31.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘metagene’ can be installed ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... WARNING
Error in library(metagene, lib.loc = &quot;/home/biocbuild/bbs-3.17-bioc/R/library&quot;) : 
  there is no package called ‘metagene’
Execution halted

It looks like this package has a loading problem when not on .libPaths:
see the messages for details.
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
NCIS.internal: no visible global function definition for
  &#x27;est.norm.med.search&#x27;
avoid_gaps_update: no visible binding for global variable &#x27;value&#x27;
avoid_gaps_update: no visible binding for global variable &#x27;bam&#x27;
avoid_gaps_update: no visible binding for global variable &#x27;tab&#x27;
avoid_gaps_update: no visible binding for global variable &#x27;nuc&#x27;
bin.data: no visible global function definition for &#x27;hist&#x27;
permutation_test: no visible global function definition for &#x27;.&#x27;
permutation_test: no visible binding for global variable &#x27;value&#x27;
plot_metagene: no visible binding for global variable &#x27;bin&#x27;
plot_metagene: no visible binding for global variable &#x27;value&#x27;
plot_metagene: no visible binding for global variable &#x27;qinf&#x27;
plot_metagene: no visible binding for global variable &#x27;qsup&#x27;
plot_metagene: no visible binding for global variable &#x27;group&#x27;
plot_metagene: no visible binding for global variable &#x27;nuc&#x27;
plot_metagene: no visible binding for global variable &#x27;design&#x27;
plot_metagene: no visible binding for global variable &#x27;nuctot&#x27;
Undefined global functions or variables:
  . bam bin design est.norm.med.search group hist nuc nuctot qinf qsup
  tab value
Consider adding
  importFrom(&quot;graphics&quot;, &quot;hist&quot;)
to your NAMESPACE file.
* checking Rd files ... WARNING
checkRd: (5) Bam_Handler.Rd:22: \item in \describe must have non-empty label
checkRd: (5) Bam_Handler.Rd:42: \item in \describe must have non-empty label
checkRd: (5) Bam_Handler.Rd:46: \item in \describe must have non-empty label
checkRd: (5) Bam_Handler.Rd:50: \item in \describe must have non-empty label
checkRd: (5) Bam_Handler.Rd:54: \item in \describe must have non-empty label
checkRd: (5) Bam_Handler.Rd:58: \item in \describe must have non-empty label
checkRd: (5) Bam_Handler.Rd:61-62: \item in \describe must have non-empty label
checkRd: (5) Bam_Handler.Rd:71-72: \item in \describe must have non-empty label
checkRd: (5) Bam_Handler.Rd:81: \item in \describe must have non-empty label
checkRd: (5) metagene.Rd:23-26: \item in \describe must have non-empty label
checkRd: (5) metagene.Rd:61-62: \item in \describe must have non-empty label
checkRd: (5) metagene.Rd:77-78: \item in \describe must have non-empty label
checkRd: (5) metagene.Rd:102-103: \item in \describe must have non-empty label
checkRd: (5) metagene.Rd:116: \item in \describe must have non-empty label
checkRd: (5) metagene.Rd:119: \item in \describe must have non-empty label
checkRd: (5) metagene.Rd:122: \item in \describe must have non-empty label
checkRd: (5) metagene.Rd:127: \item in \describe must have non-empty label
checkRd: (5) metagene.Rd:130: \item in \describe must have non-empty label
checkRd: (5) metagene.Rd:133: \item in \describe must have non-empty label
checkRd: (5) metagene.Rd:144: \item in \describe must have non-empty label
checkRd: (5) metagene.Rd:147: \item in \describe must have non-empty label
checkRd: (5) metagene.Rd:155: \item in \describe must have non-empty label
checkRd: (5) metagene.Rd:164: \item in \describe must have non-empty label
checkRd: (5) metagene.Rd:170: \item in \describe must have non-empty label
checkRd: (5) metagene.Rd:179: \item in \describe must have non-empty label
checkRd: (5) metagene.Rd:183: \item in \describe must have non-empty label
checkRd: (5) metagene.Rd:186: \item in \describe must have non-empty label
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking LazyData ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking R/sysdata.rda ... OK
* checking line endings in Makefiles ... OK
* checking for GNU extensions in Makefiles ... OK
* checking include directives in Makefiles ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... SKIPPED
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’
 ERROR
Running the tests in ‘tests/runTests.R’ failed.
Last 13 lines of output:
   
  FAILURE in test.parallel_job_set_core_count_valid_multiple_core_biocparallelparam: Error in checkIdentical(obs, exp) : FALSE 
   
  
  Test files with failing tests
  
     test_parallel_job.R 
       test.parallel_job_get_core_count_valid_multiple_core_biocparallelparam 
       test.parallel_job_set_core_count_valid_multiple_core_biocparallelparam 
  
  
  Error in BiocGenerics:::testPackage(&quot;metagene&quot;) : 
    unit tests failed for package metagene
  In addition: There were 50 or more warnings (use warnings() to see the first 50)
  Execution halted
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘metagene.Rmd’ using ‘UTF-8’... OK
  ‘metagene_rnaseq.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 2 WARNINGs, 1 NOTE
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/metagene.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">metagene.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL metagene
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/site-library’
* installing *source* package ‘metagene’ ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (metagene)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename fail">metagene.Rcheck/tests/runTests.Rout.fail</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; ## Run all tests in the metagene package
&gt; BiocGenerics:::testPackage(&quot;metagene&quot;)

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min


Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

produce data table : ChIP-Seq
produce data frame : ChIP-Seq
produce data table : ChIP-Seq
produce data frame : ChIP-Seq
produce data table : ChIP-Seq
produce data frame : ChIP-Seq
produce data table : ChIP-Seq
produce data frame : ChIP-Seq
produce data table : ChIP-Seq
produce data frame : ChIP-Seq
produce data table : ChIP-Seq
produce data frame : ChIP-Seq
produce data table : ChIP-Seq
produce data frame : ChIP-Seq
produce data table : ChIP-Seq
produce data frame : ChIP-Seq
produce data table : ChIP-Seq
produce data frame : ChIP-Seq
produce data table : ChIP-Seq
produce data frame : ChIP-Seq
produce data table : ChIP-Seq
produce data frame : ChIP-Seq
produce data table : ChIP-Seq
produce data frame : ChIP-Seq
produce data table : ChIP-Seq
produce data table : ChIP-Seq
produce data table : ChIP-Seq
produce data table : ChIP-Seq
produce data frame : ChIP-Seq
[1] TRUE
produce data table : ChIP-Seq
produce data table : ChIP-Seq
produce data table : ChIP-Seq
produce data table : ChIP-Seq
produce data table : ChIP-Seq
produce data table : ChIP-Seq
ChIP-Seq flip/unflip
Normalization done
produce data table : ChIP-Seq
[1] TRUE
produce data table : ChIP-Seq
[1] TRUE
produce data table : ChIP-Seq
[1] TRUE
produce data table : ChIP-Seq
[1] TRUE
produce data table : ChIP-Seq
[1] TRUE
produce data table : ChIP-Seq
produce data table : ChIP-Seq
ChIP-Seq flip/unflip
ChIP-Seq flip/unflip
Timing stopped at: 0.934 0 0.934
Error in checkIdentical(obs, exp) : FALSE 
 
In addition: There were 50 or more warnings (use warnings() to see the first 50)
Timing stopped at: 0.929 0 0.93
Error in checkIdentical(obs, exp) : FALSE 
 
produce data table : ChIP-Seq
produce data table : ChIP-Seq
produce data table : ChIP-Seq
produce data table : ChIP-Seq
produce data table : ChIP-Seq
produce data table : ChIP-Seq
produce data table : ChIP-Seq
produce data table : ChIP-Seq
produce data table : ChIP-Seq
produce data table : ChIP-Seq
[1] TRUE


RUNIT TEST PROTOCOL -- Sun Jan 29 03:23:22 2023 
*********************************************** 
Number of test functions: 218 
Number of errors: 0 
Number of failures: 2 

 
1 Test Suite : 
metagene RUnit Tests - 218 test functions, 0 errors, 2 failures
FAILURE in test.parallel_job_get_core_count_valid_multiple_core_biocparallelparam: Error in checkIdentical(obs, exp) : FALSE 
 
FAILURE in test.parallel_job_set_core_count_valid_multiple_core_biocparallelparam: Error in checkIdentical(obs, exp) : FALSE 
 

Test files with failing tests

   test_parallel_job.R 
     test.parallel_job_get_core_count_valid_multiple_core_biocparallelparam 
     test.parallel_job_set_core_count_valid_multiple_core_biocparallelparam 


Error in BiocGenerics:::testPackage(&quot;metagene&quot;) : 
  unit tests failed for package metagene
In addition: There were 50 or more warnings (use warnings() to see the first 50)
Execution halted
</PRE>
</DIV><HR>
<H3>Example timings</H3>
</BODY>
</HTML>
