<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for ReUseData on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.17</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><B>[<A href="..#R">R</A>]</B></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-03-27 05:50:42 -0000 (Mon, 27 Mar 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-03-12&nbsp;r83975)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">6083</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for ReUseData on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the ReUseData package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/ReUseData.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1669</B>/2195</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">ReUseData</A>&nbsp;0.99.38</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/ReUseData">landing page</A>)</SPAN><BR>Qian Liu<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-03-24&nbsp;01:40:07&nbsp;-0000&nbsp;(Fri,&nbsp;24&nbsp;Mar&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/ReUseData</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">ef32d88</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-03-07&nbsp;18:48:48&nbsp;-0000&nbsp;(Tue,&nbsp;07&nbsp;Mar&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: ReUseData</TD></TR>
<TR><TD><B>Version</B>: 0.99.38</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:ReUseData.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings ReUseData_0.99.38.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-03-25 10:34:13 -0000 (Sat, 25 Mar 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-03-25 10:38:38 -0000 (Sat, 25 Mar 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 264.4 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: ReUseData.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:ReUseData.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings ReUseData_0.99.38.tar.gz
###
##############################################################################
##############################################################################


* using log directory â€˜/home/biocbuild/bbs-3.17-bioc/meat/ReUseData.Rcheckâ€™
* using R Under development (unstable) (2023-03-12 r83975)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file â€˜ReUseData/DESCRIPTIONâ€™ ... OK
* this is package â€˜ReUseDataâ€™ version â€˜0.99.38â€™
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .BBSoptions
These were most likely included in error. See section â€˜Package
structureâ€™ in the â€˜Writing R Extensionsâ€™ manual.
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package â€˜ReUseDataâ€™ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking â€˜buildâ€™ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in â€˜vignettesâ€™ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                user system elapsed
dataUpdate    16.210  1.658  18.447
getCloudData  10.340  1.024  14.939
dataHub-class  8.649  0.392   9.045
* checking for unstated dependencies in â€˜testsâ€™ ... OK
* checking tests ...
  Running â€˜testthat.Râ€™
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in â€˜inst/docâ€™ ... OK
* checking running R code from vignettes ...
  â€˜ReUseData_data.Rmdâ€™ using â€˜UTF-8â€™... OK
  â€˜ReUseData_quickStart.Rmdâ€™ using â€˜UTF-8â€™... OK
  â€˜ReUseData_recipe.Rmdâ€™ using â€˜UTF-8â€™... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 NOTE
See
  â€˜/home/biocbuild/bbs-3.17-bioc/meat/ReUseData.Rcheck/00check.logâ€™
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">ReUseData.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL ReUseData
###
##############################################################################
##############################################################################


* installing to library â€˜/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-libraryâ€™
* installing *source* package â€˜ReUseDataâ€™ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (ReUseData)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">ReUseData.Rcheck/tests/testthat.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; library(testthat)
&gt; library(ReUseData)
&gt; 
&gt; test_check(&quot;ReUseData&quot;)
Note: you need to assign a name for the recipe: rcpName &lt;- recipeLoad(&#x27;xx&#x27;)
Data recipe loaded!
Use inputs() to check required input parameters before evaluation.
Check here: https://rcwl.org/dataRecipes/echo_out.html
for user instructions (e.g., eligible input values, data source, etc.)

}[1;30mINFO[0m Final process status is success

Updating data record...
outfile.txt added

Updating data record...
2b3de34afb8eba_GRCh38.primary_assembly.genome.fa.1.bt2 added
2b3de3b3a487d_GRCh38.primary_assembly.genome.fa.2.bt2 added
2b3de37973139c_GRCh38.primary_assembly.genome.fa.3.bt2 added
2b3de31c829a2_GRCh38.primary_assembly.genome.fa.4.bt2 added
2b3de37c60930_GRCh38.primary_assembly.genome.fa.rev.1.bt2 added
2b3de330a9939e_GRCh38.primary_assembly.genome.fa.rev.2.bt2 added
2b3de33b3c39c7_outfile.txt added
2b3de3715619a8_GRCh37_to_GRCh38.chain added
2b3de3731efa89_GRCh37_to_NCBI34.chain added
2b3de33a7be4a1_GRCh37_to_NCBI35.chain added
2b3de32dc6cda1_GRCh37_to_NCBI36.chain added
2b3de35cbc17f2_GRCh38_to_GRCh37.chain added
2b3de36488a156_GRCh38_to_NCBI34.chain added
2b3de348e1b70c_GRCh38_to_NCBI35.chain added
2b3de32d0c9c60_GRCh38_to_NCBI36.chain added
2b3de34faec925_NCBI34_to_GRCh37.chain added
2b3de3681f08b_NCBI34_to_GRCh38.chain added
2b3de3121131e3_NCBI35_to_GRCh37.chain added
2b3de358ba77b1_NCBI35_to_GRCh38.chain added
2b3de37c299077_NCBI36_to_GRCh37.chain added
2b3de34c62ff91_NCBI36_to_GRCh38.chain added
2b3de31a47231a_GRCm38_to_NCBIM36.chain added
2b3de347a220cd_GRCm38_to_NCBIM37.chain added
2b3de31b75fbf_NCBIM36_to_GRCm38.chain added
2b3de33b199736_NCBIM37_to_GRCm38.chain added
2b3de315a1e978_1000G_omni2.5.b37.vcf.gz added
2b3de372a00c5c_1000G_omni2.5.b37.vcf.gz.tbi added
2b3de3285ea42e_Mills_and_1000G_gold_standard.indels.b37.vcf.gz added
2b3de36f9dedac_Mills_and_1000G_gold_standard.indels.b37.vcf.gz.tbi added
2b3de3552c7b1d_1000G_omni2.5.hg38.vcf.gz added
2b3de36dfe9fd5_1000G_omni2.5.hg38.vcf.gz.tbi added
2b3de33a997c66_Mills_and_1000G_gold_standard.indels.hg38.vcf.gz added
2b3de36066c39a_Mills_and_1000G_gold_standard.indels.hg38.vcf.gz.tbi added
2b3de36771b371_af-only-gnomad.raw.sites.vcf added
2b3de33c61a608_af-only-gnomad.raw.sites.vcf.idx added
2b3de3682cccca_Mutect2-exome-panel.vcf.idx added
2b3de3181b470f_Mutect2-WGS-panel-b37.vcf added
2b3de3779ddfcf_Mutect2-WGS-panel-b37.vcf.idx added
2b3de35982e672_small_exac_common_3.vcf added
2b3de3b3a4198_small_exac_common_3.vcf.idx added
2b3de33219c470_1000g_pon.hg38.vcf.gz added
2b3de3749b414_1000g_pon.hg38.vcf.gz.tbi added
2b3de367f6598a_af-only-gnomad.hg38.vcf.gz added
2b3de316a265c6_af-only-gnomad.hg38.vcf.gz.tbi added
2b3de3502b6b20_small_exac_common_3.hg38.vcf.gz added
2b3de31502f5ea_small_exac_common_3.hg38.vcf.gz.tbi added
2b3de366512eec_gencode.v41.annotation.gtf added
2b3de356ad5bab_gencode.v42.annotation.gtf added
2b3de3271427cd_gencode.vM30.annotation.gtf added
2b3de33f0ba69d_gencode.vM31.annotation.gtf added
2b3de352d6ec23_gencode.v41.transcripts.fa added
2b3de37377275f_gencode.v41.transcripts.fa.fai added
2b3de35952c9b8_gencode.v42.transcripts.fa added
2b3de31a790cf0_gencode.v42.transcripts.fa.fai added
2b3de3752e871e_gencode.vM30.pc_transcripts.fa added
2b3de3146c60ee_gencode.vM30.pc_transcripts.fa.fai added
2b3de3301af668_gencode.vM31.pc_transcripts.fa added
2b3de367ce937a_gencode.vM31.pc_transcripts.fa.fai added
2b3de33ccb051c_GRCh38.primary_assembly.genome.fa.1.ht2 added
2b3de31fb8e415_GRCh38.primary_assembly.genome.fa.2.ht2 added
2b3de33cfb0e97_GRCh38.primary_assembly.genome.fa.3.ht2 added
2b3de32ac9a4f1_GRCh38.primary_assembly.genome.fa.4.ht2 added
2b3de35a52607b_GRCh38.primary_assembly.genome.fa.5.ht2 added
2b3de31d61d232_GRCh38.primary_assembly.genome.fa.6.ht2 added
2b3de3123b5862_GRCh38.primary_assembly.genome.fa.7.ht2 added
2b3de316b40684_GRCh38.primary_assembly.genome.fa.8.ht2 added
2b3de358e9efc_GRCh38_full_analysis_set_plus_decoy_hla.fa.fai added
2b3de32a569f71_GRCh38_full_analysis_set_plus_decoy_hla.fa.amb added
2b3de3e51e653_GRCh38_full_analysis_set_plus_decoy_hla.fa.ann added
2b3de35f11856f_GRCh38_full_analysis_set_plus_decoy_hla.fa.bwt added
2b3de33590e109_GRCh38_full_analysis_set_plus_decoy_hla.fa.pac added
2b3de3406baac4_GRCh38_full_analysis_set_plus_decoy_hla.fa.sa added
2b3de3665b3983_GRCh38_full_analysis_set_plus_decoy_hla.fa added
2b3de31d873a94_GRCh38.primary_assembly.genome.fa.fai added
2b3de3570e108a_GRCh38.primary_assembly.genome.fa.amb added
2b3de33686a4a3_GRCh38.primary_assembly.genome.fa.ann added
2b3de3328a307e_GRCh38.primary_assembly.genome.fa.bwt added
2b3de33d5f3f76_GRCh38.primary_assembly.genome.fa.pac added
2b3de3d34004f_GRCh38.primary_assembly.genome.fa.sa added
2b3de3599e584c_GRCh38.primary_assembly.genome.fa added
2b3de37c6ae614_hs37d5.fa.fai added
2b3de3600aec72_hs37d5.fa.amb added
2b3de34d157fab_hs37d5.fa.ann added
2b3de355bdafcc_hs37d5.fa.bwt added
2b3de37a83f962_hs37d5.fa.pac added
2b3de3424406c9_hs37d5.fa.sa added
2b3de36a2a10ba_hs37d5.fa added
2b3de32a9eefca_complete_ref_lens.bin added
2b3de32a129a44_ctable.bin added
2b3de326f515d6_ctg_offsets.bin added
2b3de34a57d3df_duplicate_clusters.tsv added
2b3de3670da8db_info.json added
2b3de351bebac7_mphf.bin added
2b3de324aa345b_pos.bin added
2b3de346f7b0d_pre_indexing.log added
2b3de363fa1329_rank.bin added
2b3de33b5e3adf_ref_indexing.log added
2b3de39fe1a0a_refAccumLengths.bin added
2b3de3e50b29a_reflengths.bin added
2b3de349b02132_refseq.bin added
2b3de3690f9f79_seq.bin added
2b3de343e193a3_versionInfo.json added
2b3de3a1bcbf6_salmon_index added
2b3de34f6ad8fc_chrLength.txt added
2b3de36168ce37_chrName.txt added
2b3de36129dc81_chrNameLength.txt added
2b3de35f17d9f_chrStart.txt added
2b3de313f2feb6_exonGeTrInfo.tab added
2b3de31e891bf7_exonInfo.tab added
2b3de313257dee_geneInfo.tab added
2b3de36d915702_Genome added
2b3de31af4020b_genomeParameters.txt added
2b3de373306a60_Log.out added
2b3de33aa6d6ad_SA added
2b3de370b1b1d7_SAindex added
2b3de36db463c2_sjdbInfo.txt added
2b3de37ceadd76_sjdbList.fromGTF.out.tab added
2b3de35adbc291_sjdbList.out.tab added
2b3de31853538d_transcriptInfo.tab added
2b3de326fd77ba_GRCh38.GENCODE.v42_100 added
2b3de31d0d867_knownGene_hg38.sql added
2b3de362ab276c_knownGene_hg38.txt added
2b3de3e0b2096_refGene_hg38.sql added
2b3de3538f932e_refGene_hg38.txt added
2b3de37555bc7_knownGene_mm39.sql added
2b3de3127a9ba3_knownGene_mm39.txt added
2b3de33789a657_refGene_mm39.sql added
2b3de342b396a6_refGene_mm39.txt added
trying URL &#x27;https://storage.googleapis.com/reusedata/echo/outfile.txt&#x27;
Content type &#x27;text/plain&#x27; length 30 bytes
==================================================
downloaded 30 bytes

trying URL &#x27;https://storage.googleapis.com/reusedata/echo/echo_out_Hello_World!_outfile.yml&#x27;
Content type &#x27;application/x-yaml&#x27; length 151 bytes
==================================================
downloaded 151 bytes

trying URL &#x27;https://storage.googleapis.com/reusedata/echo/echo_out_Hello_World!_outfile.cwl&#x27;
Content type &#x27;application/octet-stream&#x27; length 1152 bytes
==================================================
downloaded 1152 bytes

trying URL &#x27;https://storage.googleapis.com/reusedata/echo/echo_out_Hello_World!_outfile.md5&#x27;
Content type &#x27;application/octet-stream&#x27; length 45 bytes
==================================================
downloaded 45 bytes

trying URL &#x27;https://storage.googleapis.com/reusedata/echo/echo_out_Hello_World!_outfile.sh&#x27;
Content type &#x27;application/x-shellscript&#x27; length 68 bytes
==================================================
downloaded 68 bytes

Data is downloaded: 
/tmp/Rtmp9J1Hli/test_gcpData/outfile.txt
}[1;30mINFO[0m Final process status is success
NOTE: existing caches will be removed and regenerated!
Updating recipes...
STAR_index.R added
bowtie2_index.R added
echo_out.R added
ensembl_liftover.R added
gcp_broad_gatk_hg19.R added
gcp_broad_gatk_hg38.R added
gcp_gatk_mutect2_b37.R added
gcp_gatk_mutect2_hg38.R added
gencode_annotation.R added
gencode_genome_grch38.R added
gencode_transcripts.R added
hisat2_index.R added
reference_genome.R added
salmon_index.R added
ucsc_database.R added

Data recipe loaded!
Use inputs(ensembl_liftover) to check required input parameters before evaluation.
Check here: https://rcwl.org/dataRecipes/ensembl_liftover.html
for user instructions (e.g., eligible input values, data source, etc.)

[ FAIL 0 | WARN 0 | SKIP 0 | PASS 44 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
 31.129   1.968  34.633 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">ReUseData.Rcheck/ReUseData-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>dataHub-class</TD><TD style="text-align: right;">8.649</TD><TD style="text-align: right;">0.392</TD><TD style="text-align: right;">9.045</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dataSearch</TD><TD style="text-align: right;">1.389</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">1.401</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>dataUpdate</TD><TD style="text-align: right;">16.210</TD><TD style="text-align: right;"> 1.658</TD><TD style="text-align: right;">18.447</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getCloudData</TD><TD style="text-align: right;">10.340</TD><TD style="text-align: right;"> 1.024</TD><TD style="text-align: right;">14.939</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getData</TD><TD style="text-align: right;">4.532</TD><TD style="text-align: right;">0.218</TD><TD style="text-align: right;">4.744</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>meta_data</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">0.00</TD><TD style="text-align: right;">0.01</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeHub-class</TD><TD style="text-align: right;">0.233</TD><TD style="text-align: right;">0.019</TD><TD style="text-align: right;">0.253</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeLoad</TD><TD style="text-align: right;">1.395</TD><TD style="text-align: right;">0.084</TD><TD style="text-align: right;">1.482</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeMake</TD><TD style="text-align: right;">4.315</TD><TD style="text-align: right;">0.202</TD><TD style="text-align: right;">4.516</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeSearch</TD><TD style="text-align: right;">0.655</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.663</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>recipeUpdate</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
