<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for compEpiTools on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><B>[<A href="../index.html#C">C</A>]</B></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-10 01:25:28 -0000 (Fri, 10 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4219</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for compEpiTools on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the compEpiTools package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/compEpiTools.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>389</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">compEpiTools</A>&nbsp;1.33.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/compEpiTools">landing page</A>)</SPAN><BR>Kamal Kishore<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-09&nbsp;03:02:02&nbsp;-0000&nbsp;(Thu,&nbsp;09&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/compEpiTools</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">5de0197</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:10:09&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: compEpiTools</TD></TR>
<TR><TD><B>Version</B>: 1.33.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:compEpiTools.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings compEpiTools_1.33.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-09 13:35:57 -0000 (Thu, 09 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-09 13:48:17 -0000 (Thu, 09 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 740.6 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: compEpiTools.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:compEpiTools.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings compEpiTools_1.33.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/compEpiTools.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘compEpiTools/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘compEpiTools’ version ‘1.33.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘compEpiTools’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
TSS: no visible global function definition for &#x27;end&lt;-&#x27;
TSS: no visible global function definition for &#x27;start&lt;-&#x27;
findLncRNA: no visible global function definition for &#x27;start&lt;-&#x27;
findLncRNA: no visible global function definition for &#x27;end&lt;-&#x27;
findLncRNA: no visible global function definition for &#x27;IRanges&#x27;
stallingIndex : getBound: no visible global function definition for
  &#x27;start&lt;-&#x27;
stallingIndex : getBound: no visible global function definition for
  &#x27;end&lt;-&#x27;
stallingIndex: no visible global function definition for &#x27;IRanges&#x27;
topGOres : &lt;anonymous&gt;: no visible global function definition for
  &#x27;ggplot&#x27;
topGOres : &lt;anonymous&gt;: no visible global function definition for &#x27;aes&#x27;
topGOres : &lt;anonymous&gt;: no visible binding for global variable
  &#x27;Significant&#x27;
topGOres : &lt;anonymous&gt;: no visible binding for global variable &#x27;P_val&#x27;
topGOres : &lt;anonymous&gt;: no visible global function definition for
  &#x27;geom_bar&#x27;
topGOres : &lt;anonymous&gt;: no visible global function definition for
  &#x27;coord_flip&#x27;
topGOres : &lt;anonymous&gt;: no visible global function definition for
  &#x27;ylab&#x27;
ucsc2GRanges: no visible global function definition for &#x27;IRanges&#x27;
GR2fasta,GRanges: no visible global function definition for
  &#x27;seqlengths&#x27;
GR2fasta,GRanges: no visible global function definition for
  &#x27;seqlengths&lt;-&#x27;
GR2fasta,GRanges: no visible global function definition for &#x27;trim&#x27;
GRangesInPromoters,GRanges: no visible global function definition for
  &#x27;start&lt;-&#x27;
GRannotate,GRanges: no visible global function definition for &#x27;start&lt;-&#x27;
GRannotate,GRanges: no visible global function definition for &#x27;end&lt;-&#x27;
GRcoverageSummit,GRanges: no visible global function definition for
  &#x27;start&lt;-&#x27;
GRcoverageSummit,GRanges: no visible global function definition for
  &#x27;end&lt;-&#x27;
GRmidpoint,GRanges: no visible global function definition for &#x27;start&lt;-&#x27;
GRmidpoint,GRanges: no visible global function definition for &#x27;end&lt;-&#x27;
GRsetwidth,GRanges: no visible global function definition for &#x27;start&lt;-&#x27;
GRsetwidth,GRanges: no visible global function definition for &#x27;end&lt;-&#x27;
countOverlapsInBins,GRanges: no visible global function definition for
  &#x27;IRanges&#x27;
featuresLength,TxDb: no visible global function definition for &#x27;reduce&#x27;
getPromoterClass,TxDb: no visible binding for global variable &#x27;txdb&#x27;
getPromoterClass,TxDb: no visible global function definition for
  &#x27;start&lt;-&#x27;
makeGtfFromDb,TxDb: no visible global function definition for &#x27;reduce&#x27;
matchEnhancers,GRanges: no visible global function definition for
  &#x27;distanceToNearest&#x27;
matchEnhancers,GRanges: no visible global function definition for
  &#x27;IRanges&#x27;
Undefined global functions or variables:
  IRanges P_val Significant aes coord_flip distanceToNearest end&lt;-
  geom_bar ggplot reduce seqlengths seqlengths&lt;- start&lt;- trim txdb ylab
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘compEpiTools-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: getPromoterClass
&gt; ### Title: Determining the CpG promoter class and the average CpG content
&gt; ### Aliases: getPromoterClass getPromoterClass-methods
&gt; ###   getPromoterClass,TxDb-method
&gt; ### Keywords: methods
&gt; 
&gt; ### ** Examples
&gt; 
&gt; require(BSgenome.Mmusculus.UCSC.mm9)
Loading required package: BSgenome.Mmusculus.UCSC.mm9
Loading required package: BSgenome
Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:graph&#x27;:

    complement

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: rtracklayer
&gt; require(TxDb.Mmusculus.UCSC.mm9.knownGene)
Loading required package: TxDb.Mmusculus.UCSC.mm9.knownGene
Loading required package: GenomicFeatures

Attaching package: &#x27;GenomicFeatures&#x27;

The following object is masked from &#x27;package:topGO&#x27;:

    genes

&gt; txdb &lt;- TxDb.Mmusculus.UCSC.mm9.knownGene
&gt; isActiveSeq(txdb) &lt;- c(rep(FALSE,20), TRUE, rep(FALSE, 14))
&gt; allpromoter &lt;- getPromoterClass(txdb, Nproc=1, org=Mmusculus)
Killed
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘compEpiTools.rnw’... OK
 NONE
* checking re-building of vignette outputs ... ERROR
Error(s) in re-building vignettes:
  ...
--- re-building ‘compEpiTools.rnw’ using knitr
Killed

* checking PDF version of manual ... OK
* DONE

Status: 2 ERRORs, 1 NOTE
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/compEpiTools.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">compEpiTools.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL compEpiTools
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘compEpiTools’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading

groupGOTerms: 	GOBPTerm, GOMFTerm, GOCCTerm environments built.
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location

groupGOTerms: 	GOBPTerm, GOMFTerm, GOCCTerm environments built.
** testing if installed package can be loaded from final location

groupGOTerms: 	GOBPTerm, GOMFTerm, GOCCTerm environments built.
** testing if installed package keeps a record of temporary installation path
* DONE (compEpiTools)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<HR>
<H3>Example timings</H3>
<P><SPAN class="filename">compEpiTools.Rcheck/compEpiTools-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>GR2fasta-methods</TD><TD style="text-align: right;">0.596</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">0.626</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GRanges2ucsc-methods</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GRangesInPromoters-methods</TD><TD style="text-align: right;">2.845</TD><TD style="text-align: right;">0.105</TD><TD style="text-align: right;">2.969</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GRannotate-methods</TD><TD style="text-align: right;">6.143</TD><TD style="text-align: right;">0.144</TD><TD style="text-align: right;">6.306</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GRannotateSimple</TD><TD style="text-align: right;">0.892</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.912</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GRbaseCoverage-methods</TD><TD style="text-align: right;">0.093</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.099</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GRcoverage-methods</TD><TD style="text-align: right;">0.159</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.164</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GRcoverageSummit-methods</TD><TD style="text-align: right;">0.085</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.085</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GRenrichment-methods</TD><TD style="text-align: right;">0.1</TD><TD style="text-align: right;">0.0</TD><TD style="text-align: right;">0.1</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GRmidpoint-methods</TD><TD style="text-align: right;">0.039</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GRsetwidth</TD><TD style="text-align: right;">0.054</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.054</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TSS</TD><TD style="text-align: right;">1.777</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">1.805</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>countOverlapsInBins-methods</TD><TD style="text-align: right;">0.177</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.185</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>distanceFromTSS-methods</TD><TD style="text-align: right;">4.109</TD><TD style="text-align: right;">0.304</TD><TD style="text-align: right;">4.415</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>enhancers</TD><TD style="text-align: right;">0.497</TD><TD style="text-align: right;">0.046</TD><TD style="text-align: right;">0.544</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findLncRNA</TD><TD style="text-align: right;">0.951</TD><TD style="text-align: right;">0.069</TD><TD style="text-align: right;">1.028</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
