<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.18 - CHECK results for ChIPpeakAnno on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.18</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><B>[<A href="..#C">C</A>]</B></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-05-10 10:04:28 -0000 (Wed, 10 May 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-03-12&nbsp;r83975)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">6211</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for ChIPpeakAnno on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the ChIPpeakAnno package:<BR>
- Allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/ChIPpeakAnno.git to reflect on this report. See <A href="%s">Troubleshooting Build Report</A> for more information.<BR><BR>
- Use the following <A href="../Renviron.bioc">Renviron settings</A> to reproduce errors and warnings.<BR><BR>
Note: If "R CMD check" recently failed on the Linux builder over a missing dependency, add the missing dependency to "Suggests" in your DESCRIPTION file. See the <A href="../Renviron.bioc">Renviron.bioc</A> for details.
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>323</B>/2194</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">ChIPpeakAnno</A>&nbsp;3.35.1</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.18/ChIPpeakAnno">landing page</A>)</SPAN><BR>Jianhong Ou<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-05-08&nbsp;19:11:19&nbsp;-0000&nbsp;(Mon,&nbsp;08&nbsp;May&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/ChIPpeakAnno</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">71289ee</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2023-04-26&nbsp;22:35:48&nbsp;-0000&nbsp;(Wed,&nbsp;26&nbsp;Apr&nbsp;2023)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: ChIPpeakAnno</TD></TR>
<TR><TD><B>Version</B>: 3.35.1</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:ChIPpeakAnno.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings ChIPpeakAnno_3.35.1.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-05-09 11:26:22 -0000 (Tue, 09 May 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-05-09 11:47:46 -0000 (Tue, 09 May 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 1284.6 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: ChIPpeakAnno.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:ChIPpeakAnno.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings ChIPpeakAnno_3.35.1.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/ChIPpeakAnno.Rcheck’
* using R Under development (unstable) (2023-03-12 r83975)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘ChIPpeakAnno/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘ChIPpeakAnno’ version ‘3.35.1’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘ChIPpeakAnno’ can be installed ... OK
* checking installed package size ... NOTE
  installed size is 19.1Mb
  sub-directories of 1Mb or more:
    data     12.6Mb
    extdata   5.2Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
Namespace in Imports field not imported from: ‘ensembldb’
  All declared Imports should be used.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... NOTE
checkRd: (-1) assignChromosomeRegion.Rd:133: Escaped LaTeX specials: \_
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking LazyData ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
Examples with CPU (user + system) or elapsed time &gt; 5s
                           user system elapsed
findEnhancers            33.605  0.636  34.243
findMotifsInPromoterSeqs 18.755  0.316  19.074
annotatePeakInBatch      16.153  0.948  17.332
summarizeOverlapsByBins   6.051  0.479   6.178
annoPeaks                 4.130  0.464  10.054
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘ChIPpeakAnno.Rmd’ using ‘UTF-8’... OK
  ‘FAQs.Rmd’ using ‘UTF-8’... OK
  ‘pipeline.Rmd’ using ‘UTF-8’... OK
  ‘quickStart.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 3 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/ChIPpeakAnno.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">ChIPpeakAnno.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL ChIPpeakAnno
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library’
* installing *source* package ‘ChIPpeakAnno’ ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
in method for ‘annoGR’ with signature ‘&quot;EnsDb&quot;’: no definition for class “EnsDb”
in method for ‘toGRanges’ with signature ‘&quot;EnsDb&quot;’: no definition for class “EnsDb”
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (ChIPpeakAnno)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">ChIPpeakAnno.Rcheck/tests/runTests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-03-12 r83975) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; require(&quot;ChIPpeakAnno&quot;) || stop(&quot;unable to load Package:ChIPpeakAnno&quot;)
Loading required package: ChIPpeakAnno
Loading required package: IRanges
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: GenomicRanges
Loading required package: GenomeInfoDb
[1] TRUE
&gt; require(&quot;BSgenome.Drerio.UCSC.danRer7&quot;) || stop(&quot;unable to load Package:BSgenome.Drerio.UCSC.danRer7&quot;)
Loading required package: BSgenome.Drerio.UCSC.danRer7
Loading required package: BSgenome
Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: rtracklayer
[1] TRUE
&gt; require(&quot;org.Hs.eg.db&quot;) || stop(&quot;unable to load Package:org.Hs.eg.db&quot;)
Loading required package: org.Hs.eg.db
Loading required package: AnnotationDbi
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.


[1] TRUE
&gt; require(&quot;org.Ce.eg.db&quot;) || stop(&quot;unable to load Package:org.Ce.eg.db&quot;)
Loading required package: org.Ce.eg.db

[1] TRUE
&gt; require(&quot;BSgenome.Celegans.UCSC.ce10&quot;) || stop(&quot;unable to load Package:BSgenome.Celegans.UCSC.ce10&quot;)
Loading required package: BSgenome.Celegans.UCSC.ce10
[1] TRUE
&gt; require(&quot;EnsDb.Hsapiens.v79&quot;) || stop(&quot;unable to load Package:EnsDb.Hsapiens.v79&quot;)
Loading required package: EnsDb.Hsapiens.v79
Loading required package: ensembldb
Loading required package: GenomicFeatures
Loading required package: AnnotationFilter

Attaching package: &#x27;ensembldb&#x27;

The following object is masked from &#x27;package:stats&#x27;:

    filter

[1] TRUE
&gt; require(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;) || stop(&quot;unable to load TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
Loading required package: TxDb.Hsapiens.UCSC.hg38.knownGene
[1] TRUE
&gt; require(&quot;biomaRt&quot;) || stop(&quot;unable to load biomaRt&quot;)
Loading required package: biomaRt
[1] TRUE
&gt; require(&quot;reactome.db&quot;) || stop(&quot;unable to load reactome.db&quot;)
Loading required package: reactome.db
[1] TRUE
&gt; require(&quot;testthat&quot;) || stop(&quot;unable to load testthat&quot;)
Loading required package: testthat

Attaching package: &#x27;testthat&#x27;

The following object is masked from &#x27;package:AnnotationFilter&#x27;:

    not

[1] TRUE
&gt; test_check(&quot;ChIPpeakAnno&quot;)
INFO [2023-05-09 11:38:43] $cat.cex
INFO [2023-05-09 11:38:43] [1] 1
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:43] $cat.col
INFO [2023-05-09 11:38:43] [1] &quot;black&quot;
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:43] $cat.fontface
INFO [2023-05-09 11:38:43] [1] &quot;plain&quot;
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:43] $cat.fontfamily
INFO [2023-05-09 11:38:43] [1] &quot;serif&quot;
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:43] $x
INFO [2023-05-09 11:38:43] $x$TF1
INFO [2023-05-09 11:38:43] [1] 3 4 5
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:43] $x$TF2
INFO [2023-05-09 11:38:43] [1] 1 2 3 4 5
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:43] $filename
INFO [2023-05-09 11:38:43] NULL
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:43] $disable.logging
INFO [2023-05-09 11:38:43] [1] TRUE
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:43] $cat.cex
INFO [2023-05-09 11:38:43] [1] 1
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:43] $cat.col
INFO [2023-05-09 11:38:43] [1] &quot;black&quot;
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:43] $cat.fontface
INFO [2023-05-09 11:38:43] [1] &quot;plain&quot;
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:43] $cat.fontfamily
INFO [2023-05-09 11:38:43] [1] &quot;serif&quot;
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:43] $x
INFO [2023-05-09 11:38:43] $x$TF1
INFO [2023-05-09 11:38:43] [1] 3 4 5
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:43] $x$TF2
INFO [2023-05-09 11:38:43] [1] 1 2 4 5
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:43] $x$TF3
INFO [2023-05-09 11:38:43] [1] 3 4 5
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:43] $x$TF4
INFO [2023-05-09 11:38:43] [1] 1 2 4 5
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:43] $filename
INFO [2023-05-09 11:38:43] NULL
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:43] $disable.logging
INFO [2023-05-09 11:38:43] [1] TRUE
INFO [2023-05-09 11:38:43] 
INFO [2023-05-09 11:38:44] $scaled
INFO [2023-05-09 11:38:44] [1] FALSE
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] $euler.d
INFO [2023-05-09 11:38:44] [1] FALSE
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] $cat.cex
INFO [2023-05-09 11:38:44] [1] 1
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] $cat.col
INFO [2023-05-09 11:38:44] [1] &quot;black&quot;
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] $cat.fontface
INFO [2023-05-09 11:38:44] [1] &quot;plain&quot;
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] $cat.fontfamily
INFO [2023-05-09 11:38:44] [1] &quot;serif&quot;
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] $x
INFO [2023-05-09 11:38:44] $x$TF1
INFO [2023-05-09 11:38:44] [1] 1 2 3
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] $x$TF2
INFO [2023-05-09 11:38:44] [1] 1 2 3
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] $filename
INFO [2023-05-09 11:38:44] NULL
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] $disable.logging
INFO [2023-05-09 11:38:44] [1] TRUE
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] $scaled
INFO [2023-05-09 11:38:44] [1] FALSE
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] $euler.d
INFO [2023-05-09 11:38:44] [1] FALSE
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] $cat.cex
INFO [2023-05-09 11:38:44] [1] 1
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] $cat.col
INFO [2023-05-09 11:38:44] [1] &quot;black&quot;
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] $cat.fontface
INFO [2023-05-09 11:38:44] [1] &quot;plain&quot;
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] $cat.fontfamily
INFO [2023-05-09 11:38:44] [1] &quot;serif&quot;
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] $x
INFO [2023-05-09 11:38:44] $x$TF1
INFO [2023-05-09 11:38:44] [1] 4 5 6
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] $x$TF2
INFO [2023-05-09 11:38:44] [1] 1 2 3
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] $filename
INFO [2023-05-09 11:38:44] NULL
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:44] $disable.logging
INFO [2023-05-09 11:38:44] [1] TRUE
INFO [2023-05-09 11:38:44] 
INFO [2023-05-09 11:38:46] $scaled
INFO [2023-05-09 11:38:46] [1] FALSE
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $euler.d
INFO [2023-05-09 11:38:46] [1] FALSE
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $cat.cex
INFO [2023-05-09 11:38:46] [1] 1
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $cat.col
INFO [2023-05-09 11:38:46] [1] &quot;black&quot;
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $cat.fontface
INFO [2023-05-09 11:38:46] [1] &quot;plain&quot;
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $cat.fontfamily
INFO [2023-05-09 11:38:46] [1] &quot;serif&quot;
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $x
INFO [2023-05-09 11:38:46] $x$TF1
INFO [2023-05-09 11:38:46] [1] 4 5 6
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $x$TF2
INFO [2023-05-09 11:38:46] [1] 1 2 3
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $filename
INFO [2023-05-09 11:38:46] NULL
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $disable.logging
INFO [2023-05-09 11:38:46] [1] TRUE
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $scaled
INFO [2023-05-09 11:38:46] [1] FALSE
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $euler.d
INFO [2023-05-09 11:38:46] [1] FALSE
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $cat.cex
INFO [2023-05-09 11:38:46] [1] 1
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $cat.col
INFO [2023-05-09 11:38:46] [1] &quot;black&quot;
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $cat.fontface
INFO [2023-05-09 11:38:46] [1] &quot;plain&quot;
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $cat.fontfamily
INFO [2023-05-09 11:38:46] [1] &quot;serif&quot;
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $x
INFO [2023-05-09 11:38:46] $x$TF1
INFO [2023-05-09 11:38:46] [1] 4 5 6
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $x$TF2
INFO [2023-05-09 11:38:46] [1] 1 2 3
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $x$TF3
INFO [2023-05-09 11:38:46] [1] 2 3 6
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $filename
INFO [2023-05-09 11:38:46] NULL
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:46] $disable.logging
INFO [2023-05-09 11:38:46] [1] TRUE
INFO [2023-05-09 11:38:46] 
INFO [2023-05-09 11:38:47] $scaled
INFO [2023-05-09 11:38:47] [1] FALSE
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $euler.d
INFO [2023-05-09 11:38:47] [1] FALSE
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $cat.cex
INFO [2023-05-09 11:38:47] [1] 1
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $cat.col
INFO [2023-05-09 11:38:47] [1] &quot;black&quot;
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $cat.fontface
INFO [2023-05-09 11:38:47] [1] &quot;plain&quot;
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $cat.fontfamily
INFO [2023-05-09 11:38:47] [1] &quot;serif&quot;
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $x
INFO [2023-05-09 11:38:47] $x$TF1
INFO [2023-05-09 11:38:47] [1] 3 4 5
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $x$TF2
INFO [2023-05-09 11:38:47] [1] 1 2 5
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $x$TF3
INFO [2023-05-09 11:38:47] [1] 1 2 5
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $filename
INFO [2023-05-09 11:38:47] NULL
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $disable.logging
INFO [2023-05-09 11:38:47] [1] TRUE
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $scaled
INFO [2023-05-09 11:38:47] [1] FALSE
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $euler.d
INFO [2023-05-09 11:38:47] [1] FALSE
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $cat.cex
INFO [2023-05-09 11:38:47] [1] 1
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $cat.col
INFO [2023-05-09 11:38:47] [1] &quot;black&quot;
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $cat.fontface
INFO [2023-05-09 11:38:47] [1] &quot;plain&quot;
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $cat.fontfamily
INFO [2023-05-09 11:38:47] [1] &quot;serif&quot;
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $x
INFO [2023-05-09 11:38:47] $x$TF1
INFO [2023-05-09 11:38:47] [1] 3 4 5
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $x$TF2
INFO [2023-05-09 11:38:47] [1] 1 2 5
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $x$TF3
INFO [2023-05-09 11:38:47] [1] 1 2 5
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $x$TF4
INFO [2023-05-09 11:38:47] [1] 1 2 5
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $filename
INFO [2023-05-09 11:38:47] NULL
INFO [2023-05-09 11:38:47] 
INFO [2023-05-09 11:38:47] $disable.logging
INFO [2023-05-09 11:38:47] [1] TRUE
INFO [2023-05-09 11:38:47] 
[ FAIL 0 | WARN 4 | SKIP 6 | PASS 316 ]

══ Skipped tests ═══════════════════════════════════════════════════════════════
• empty test (6)

[ FAIL 0 | WARN 4 | SKIP 6 | PASS 316 ]
&gt; 
&gt; proc.time()
   user  system elapsed 
219.360   5.631 239.819 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">ChIPpeakAnno.Rcheck/ChIPpeakAnno-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>ChIPpeakAnno-package</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>ExonPlusUtr.human.GRCh37</TD><TD style="text-align: right;">3.531</TD><TD style="text-align: right;">0.150</TD><TD style="text-align: right;">3.680</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>HOT.spots</TD><TD style="text-align: right;">0.117</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.129</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>IDRfilter</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Peaks.Ste12.Replicate1</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.066</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Peaks.Ste12.Replicate2</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.017</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Peaks.Ste12.Replicate3</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TSS.human.GRCh37</TD><TD style="text-align: right;">0.200</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.212</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TSS.human.GRCh38</TD><TD style="text-align: right;">0.136</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.139</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TSS.human.NCBI36</TD><TD style="text-align: right;">0.105</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.108</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TSS.mouse.GRCm38</TD><TD style="text-align: right;">0.095</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.102</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TSS.mouse.NCBIM37</TD><TD style="text-align: right;">0.091</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.092</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TSS.rat.RGSC3.4</TD><TD style="text-align: right;">0.073</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.077</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TSS.rat.Rnor_5.0</TD><TD style="text-align: right;">0.068</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.068</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TSS.zebrafish.Zv8</TD><TD style="text-align: right;">0.064</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.073</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>TSS.zebrafish.Zv9</TD><TD style="text-align: right;">0.075</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.087</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addAncestors</TD><TD style="text-align: right;">1.521</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">1.533</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addGeneIDs</TD><TD style="text-align: right;">2.077</TD><TD style="text-align: right;">0.216</TD><TD style="text-align: right;">2.294</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>addMetadata</TD><TD style="text-align: right;">1.787</TD><TD style="text-align: right;">0.082</TD><TD style="text-align: right;">1.868</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annoGR</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annoPeaks</TD><TD style="text-align: right;"> 4.130</TD><TD style="text-align: right;"> 0.464</TD><TD style="text-align: right;">10.054</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotatePeakInBatch</TD><TD style="text-align: right;">16.153</TD><TD style="text-align: right;"> 0.948</TD><TD style="text-align: right;">17.332</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotatedPeak</TD><TD style="text-align: right;">0.056</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.060</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>assignChromosomeRegion</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bdp</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>binOverFeature</TD><TD style="text-align: right;">1.007</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">1.027</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>binOverGene</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>binOverRegions</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>condenseMatrixByColnames</TD><TD style="text-align: right;">0.014</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>convert2EntrezID</TD><TD style="text-align: right;">0.392</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.393</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>countPatternInSeqs</TD><TD style="text-align: right;">0.188</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.195</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>cumulativePercentage</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>downstreams</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.029</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>egOrgMap</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>enrichedGO</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>enrichmentPlot</TD><TD style="text-align: right;">0.664</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.663</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>estFragmentLength</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>estLibSize</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>featureAlignedDistribution</TD><TD style="text-align: right;">0.289</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.288</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>featureAlignedExtendSignal</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>featureAlignedHeatmap</TD><TD style="text-align: right;">0.496</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.496</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>featureAlignedSignal</TD><TD style="text-align: right;">0.226</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.253</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findEnhancers</TD><TD style="text-align: right;">33.605</TD><TD style="text-align: right;"> 0.636</TD><TD style="text-align: right;">34.243</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findMotifsInPromoterSeqs</TD><TD style="text-align: right;">18.755</TD><TD style="text-align: right;"> 0.316</TD><TD style="text-align: right;">19.074</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findOverlappingPeaks</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>findOverlapsOfPeaks</TD><TD style="text-align: right;">2.332</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">2.348</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genomicElementDistribution</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>genomicElementUpSetR</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getAllPeakSequence</TD><TD style="text-align: right;">0.742</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.750</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getAnnotation</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getEnrichedGO</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.011</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getEnrichedPATH</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getGO</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getGeneSeq</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getUniqueGOidCount</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getVennCounts</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>hyperGtest</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>makeVennDiagram</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mergePlusMinusPeaks</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>metagenePlot</TD><TD style="text-align: right;">2.433</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">2.467</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>myPeakList</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>oligoFrequency</TD><TD style="text-align: right;">0.328</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.344</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>oligoSummary</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>peakPermTest</TD><TD style="text-align: right;">0.002</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>peaksNearBDP</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pie1</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.007</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotBinOverRegions</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>preparePool</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>reCenterPeaks</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.033</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>summarizeOverlapsByBins</TD><TD style="text-align: right;">6.051</TD><TD style="text-align: right;">0.479</TD><TD style="text-align: right;">6.178</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>summarizePatternInPeaks</TD><TD style="text-align: right;">0.696</TD><TD style="text-align: right;">0.076</TD><TD style="text-align: right;">0.773</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>tileCount</TD><TD style="text-align: right;">0.264</TD><TD style="text-align: right;">0.228</TD><TD style="text-align: right;">0.418</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>tileGRanges</TD><TD style="text-align: right;">0.148</TD><TD style="text-align: right;">0.163</TD><TD style="text-align: right;">0.053</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>toGRanges</TD><TD style="text-align: right;">0.107</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.136</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>translatePattern</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>wgEncodeTfbsV3</TD><TD style="text-align: right;">0.192</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.208</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>write2FASTA</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>xget</TD><TD style="text-align: right;">0.134</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.158</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
