<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for ViSEAGO on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.17</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><A href="..#S">S</A></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><B>[<A href="..#V">V</A>]</B></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-27 02:34:45 -0000 (Mon, 27 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4259</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for ViSEAGO on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the ViSEAGO package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/ViSEAGO.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>2135</B>/2169</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">ViSEAGO</A>&nbsp;1.13.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/ViSEAGO">landing page</A>)</SPAN><BR>Aurelien Brionne<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-23&nbsp;09:40:21&nbsp;-0000&nbsp;(Thu,&nbsp;23&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/ViSEAGO</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">54a13c4</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:20:11&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: ViSEAGO</TD></TR>
<TR><TD><B>Version</B>: 1.13.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:ViSEAGO.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings ViSEAGO_1.13.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-24 20:11:07 -0000 (Fri, 24 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-24 20:15:55 -0000 (Fri, 24 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 288.5 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: ViSEAGO.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 3</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:ViSEAGO.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings ViSEAGO_1.13.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/ViSEAGO.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘ViSEAGO/DESCRIPTION’ ... OK
* this is package ‘ViSEAGO’ version ‘1.13.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘ViSEAGO’ can be installed ... WARNING
Found the following significant warnings:
  Warning: replacing previous import ‘data.table::set’ by ‘dendextend::set’ when loading ‘ViSEAGO’
See ‘/home/biocbuild/bbs-3.17-bioc/meat/ViSEAGO.Rcheck/00install.out’ for details.
* checking installed package size ... NOTE
  installed size is 12.3Mb
  sub-directories of 1Mb or more:
    extdata  10.6Mb
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... WARNING
&#x27;library&#x27; or &#x27;require&#x27; call not declared from: ‘topGO’
&#x27;library&#x27; or &#x27;require&#x27; call to ‘topGO’ in package code.
  Please use :: or requireNamespace() instead.
  See section &#x27;Suggested packages&#x27; in the &#x27;Writing R Extensions&#x27; manual.
Namespace in Imports field not imported from: ‘processx’
  All declared Imports should be used.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
annotate,character-genomic_ressource: warning in getBM(attributes =
  c(&quot;ensembl_gene_id&quot;, &quot;go_id&quot;, &quot;go_linkage_type&quot;, &quot;namespace_1003&quot;),
  filters = unlist(go_filter[grep(&quot;with GO ID&quot;, ignore.case = TRUE,
  go_filter$description), &quot;name&quot;, with = FALSE]), value = TRUE, mart =
  myspecies): partial argument match of &#x27;value&#x27; to &#x27;values&#x27;
compute_SS_distances,ANY-character: warning in select(GO.db, keys =
  onto$values, column = &quot;TERM&quot;): partial argument match of &#x27;column&#x27; to
  &#x27;columns&#x27;
merge_enrich_terms,list : genes_symbols_add: warning in
  getBM(attributes = c(&quot;ensembl_gene_id&quot;, &quot;external_gene_name&quot;), value
  = TRUE, mart = myspecies): partial argument match of &#x27;value&#x27; to
  &#x27;values&#x27;
GOterms_heatmap,GO_SS: no visible binding for global variable ‘GO.ID’
MDSplot,ANY: no visible binding for global variable ‘text’
MDSplot,ANY: no visible binding for global variable ‘GO.cluster’
Upset,ANY: no visible global function definition for ‘png’
Upset,ANY: no visible global function definition for ‘dev.off’
annotate,character-genomic_ressource: no visible global function
  definition for ‘.’
annotate,character-genomic_ressource: no visible binding for global
  variable ‘ENTREZID’
annotate,character-genomic_ressource: no visible binding for global
  variable ‘GO’
annotate,character-genomic_ressource: no visible binding for global
  variable ‘EVIDENCE’
annotate,character-genomic_ressource: no visible binding for global
  variable ‘ONTOLOGY’
compute_SS_distances,ANY-character: no visible binding for global
  variable ‘N’
compute_SS_distances,ANY-character: no visible binding for global
  variable ‘IC’
merge_enrich_terms,list : genes_symbols_add : esummary : &lt;anonymous&gt;:
  no visible binding for global variable ‘start’
merge_enrich_terms,list : genes_symbols_add : esummary : &lt;anonymous&gt;:
  no visible binding for global variable ‘end’
merge_enrich_terms,list : &lt;anonymous&gt;: no visible binding for global
  variable ‘pval’
merge_enrich_terms,list : &lt;anonymous&gt;: no visible binding for global
  variable ‘pathway’
merge_enrich_terms,list : &lt;anonymous&gt;: no visible binding for global
  variable ‘Id’
merge_enrich_terms,list : &lt;anonymous&gt;: no visible global function
  definition for ‘.’
merge_enrich_terms,list : &lt;anonymous&gt;: no visible binding for global
  variable ‘padj’
merge_enrich_terms,list : &lt;anonymous&gt;: no visible binding for global
  variable ‘nMoreExtreme’
merge_enrich_terms,list : &lt;anonymous&gt;: no visible binding for global
  variable ‘ES’
merge_enrich_terms,list : &lt;anonymous&gt;: no visible binding for global
  variable ‘NES’
merge_enrich_terms,list : &lt;anonymous&gt;: no visible binding for global
  variable ‘size’
merge_enrich_terms,list : &lt;anonymous&gt;: no visible binding for global
  variable ‘genes_frequency’
merge_enrich_terms,list : &lt;anonymous&gt;: no visible binding for global
  variable ‘log2err’
merge_enrich_terms,list : &lt;anonymous&gt;: no visible binding for global
  variable ‘leadingEdge’
merge_enrich_terms,list : &lt;anonymous&gt;: no visible binding for global
  variable ‘Significant_genes’
merge_enrich_terms,list : &lt;anonymous&gt;: no visible binding for global
  variable ‘GO.ID’
runfgsea,ANY-gene2GO-character: no visible binding for global variable
  ‘genes_frequency’
runfgsea,ANY-gene2GO-character: no visible binding for global variable
  ‘leadingEdge’
runfgsea,ANY-gene2GO-character: no visible binding for global variable
  ‘size’
runfgsea,ANY-gene2GO-character: no visible binding for global variable
  ‘pathway’
Undefined global functions or variables:
  . ENTREZID ES EVIDENCE GO GO.ID GO.cluster IC Id N NES ONTOLOGY
  Significant_genes dev.off end genes_frequency leadingEdge log2err
  nMoreExtreme padj pathway png pval size start text
Consider adding
  importFrom(&quot;grDevices&quot;, &quot;dev.off&quot;, &quot;png&quot;)
  importFrom(&quot;graphics&quot;, &quot;text&quot;)
  importFrom(&quot;stats&quot;, &quot;end&quot;, &quot;start&quot;)
to your NAMESPACE file.
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking data for ASCII and uncompressed saves ... WARNING
  
  Note: significantly better compression could be obtained
        by using R CMD build --resave-data
              old_size new_size compress
  myGOs.RData    876Kb    517Kb       xz
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘SS_choice.Rmd’ using ‘UTF-8’... OK
  ‘ViSEAGO.Rmd’ using ‘UTF-8’... OK
  ‘fgsea_alternative.Rmd’ using ‘UTF-8’... OK
  ‘mouse_bioconductor.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 3 WARNINGs, 2 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/ViSEAGO.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">ViSEAGO.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL ViSEAGO
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘ViSEAGO’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Warning: replacing previous import ‘data.table::set’ by ‘dendextend::set’ when loading ‘ViSEAGO’
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
Warning: replacing previous import ‘data.table::set’ by ‘dendextend::set’ when loading ‘ViSEAGO’
** testing if installed package can be loaded from final location
Warning: replacing previous import ‘data.table::set’ by ‘dendextend::set’ when loading ‘ViSEAGO’
** testing if installed package keeps a record of temporary installation path
* DONE (ViSEAGO)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<HR>
<H3>Example timings</H3>
<P><SPAN class="filename">ViSEAGO.Rcheck/ViSEAGO-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>Bioconductor2GO</TD><TD style="text-align: right;">0.042</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Custom2GO</TD><TD style="text-align: right;">0.193</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.194</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Ensembl2GO</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EntrezGene2GO</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>EntrezGene_orthologs</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GOclusters_heatmap-methods</TD><TD style="text-align: right;">0.039</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GOcount-methods</TD><TD style="text-align: right;">0.988</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">1.021</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>GOterms_heatmap-methods</TD><TD style="text-align: right;">0.042</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.042</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>MDSplot-methods</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Uniprot2GO</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>Upset-methods</TD><TD style="text-align: right;">1.658</TD><TD style="text-align: right;">0.160</TD><TD style="text-align: right;">1.774</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotate-methods</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>available_organisms-methods</TD><TD style="text-align: right;">0.105</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.131</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>build_GO_SS-methods</TD><TD style="text-align: right;">0.042</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.042</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>clusters_cor-methods</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>compare_clusters-methods</TD><TD style="text-align: right;">0.040</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>compute_SS_distances-methods</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>create_topGOdata-methods</TD><TD style="text-align: right;">0.013</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.015</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>merge_enrich_terms-methods</TD><TD style="text-align: right;">0.089</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.116</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>overLapper</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">0</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>pkgdiagram</TD><TD style="text-align: right;">0.106</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">0.149</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>runfgsea-methods</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>show_heatmap-methods</TD><TD style="text-align: right;">0.044</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.043</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>show_table-methods</TD><TD style="text-align: right;">0.232</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.258</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>taxonomy</TD><TD style="text-align: right;">0.158</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">1.760</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
