<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for HIBAG on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><B>[<A href="../index.html#H">H</A>]</B></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-01-29 16:33:52 -0000 (Sun, 29 Jan 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4021</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for HIBAG on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the HIBAG package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/HIBAG.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard warnings">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>890</B>/2162</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">HIBAG</A>&nbsp;1.35.2</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/HIBAG">landing page</A>)</SPAN><BR>Xiuwen Zheng<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-01-27&nbsp;13:09:27&nbsp;-0000&nbsp;(Fri,&nbsp;27&nbsp;Jan&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/HIBAG</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">d609964</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-12-17&nbsp;00:02:26&nbsp;-0000&nbsp;(Sat,&nbsp;17&nbsp;Dec&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: HIBAG</TD></TR>
<TR><TD><B>Version</B>: 1.35.2</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:HIBAG.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings HIBAG_1.35.2.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-01-29 00:47:54 -0000 (Sun, 29 Jan 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-01-29 00:49:52 -0000 (Sun, 29 Jan 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 118.2 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph WARNINGS">&nbsp;&nbsp;WARNINGS&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: HIBAG.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 2</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:HIBAG.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings HIBAG_1.35.2.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/HIBAG.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘HIBAG/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘HIBAG’ version ‘1.35.2’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘HIBAG’ can be installed ... OK
* used C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... WARNING
Invalid citation information in ‘inst/CITATION’:
  Warning message:
  In utils::packageDescription(basename(dir), dirname(dir)) :
    no package &#x27;HIBAG&#x27; was found
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... WARNING
Error in library(HIBAG, lib.loc = &quot;/home/biocbuild/bbs-3.17-bioc/R/library&quot;) : 
  there is no package called ‘HIBAG’
Execution halted

It looks like this package has a loading problem when not on .libPaths:
see the messages for details.
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking LazyData ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking line endings in Makefiles ... OK
* checking compilation flags in Makevars ... OK
* checking for GNU extensions in Makefiles ... NOTE
GNU make is a SystemRequirements.
* checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... OK
* checking use of PKG_*FLAGS in Makefiles ... OK
* checking compiled code ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... SKIPPED
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘HIBAG.Rmd’... OK
  ‘HLA_Association.Rmd’... OK
  ‘Implementation.Rmd’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 2 WARNINGs, 1 NOTE
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/HIBAG.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">HIBAG.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL HIBAG
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/site-library’
* installing *source* package ‘HIBAG’ ...
** using staged installation
** libs
using C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
using C++11
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -I../inst/include -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c HIBAG.cpp -o HIBAG.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -I../inst/include -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c LibHLA.cpp -o LibHLA.o
LibHLA.cpp: In static member function ‘static void HLA_LIB::CAlg_Prediction::Init_Target_IFunc(const char*)’:
LibHLA.cpp:1241:14: warning: variable ‘cpu_auto_avx2’ set but not used [-Wunused-but-set-variable]
 1241 |         bool cpu_auto_avx2, cpu_max;
      |              ^~~~~~~~~~~~~
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -I../inst/include -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c LibHLA_ext_avx.cpp -o LibHLA_ext_avx.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -I../inst/include -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c LibHLA_ext_avx2.cpp -o LibHLA_ext_avx2.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -I../inst/include -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c LibHLA_ext_avx512bw.cpp -o LibHLA_ext_avx512bw.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -I../inst/include -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c LibHLA_ext_avx512f.cpp -o LibHLA_ext_avx512f.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -I../inst/include -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c LibHLA_ext_avx512vpopcnt.cpp -o LibHLA_ext_avx512vpopcnt.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -I../inst/include -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c LibHLA_ext_sse2.cpp -o LibHLA_ext_sse2.o
g++ -std=gnu++11 -I&quot;/home/biocbuild/bbs-3.17-bioc/R/include&quot; -DNDEBUG -I../inst/include -I&#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/RcppParallel/include&#x27; -I/usr/local/include    -fPIC  -g -O2  -Wall -c LibHLA_ext_sse4_2.cpp -o LibHLA_ext_sse4_2.o
g++ -std=gnu++11 -shared -L/home/biocbuild/bbs-3.17-bioc/R/lib -L/usr/local/lib -o HIBAG.so HIBAG.o LibHLA.o LibHLA_ext_avx.o LibHLA_ext_avx2.o LibHLA_ext_avx512bw.o LibHLA_ext_avx512f.o LibHLA_ext_avx512vpopcnt.o LibHLA_ext_sse2.o LibHLA_ext_sse4_2.o -lgcc -L/home/biocbuild/bbs-3.17-bioc/R/lib -lR
installing to /home/biocbuild/bbs-3.17-bioc/R/site-library/00LOCK-HIBAG/00new/HIBAG/libs
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (HIBAG)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">HIBAG.Rcheck/tests/runTests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; #############################################################
&gt; #
&gt; # DESCRIPTION: Unit tests in the HIBAG package
&gt; #
&gt; 
&gt; # load the HIBAG package
&gt; library(HIBAG)
HIBAG (HLA Genotype Imputation with Attribute Bagging)
Kernel Version: v1.5 (64-bit)
&gt; 
&gt; 
&gt; #############################################################
&gt; 
&gt; # a list of HLA genes
&gt; hla.list &lt;- c(&quot;A&quot;, &quot;B&quot;, &quot;C&quot;, &quot;DQA1&quot;, &quot;DQB1&quot;, &quot;DRB1&quot;)
&gt; 
&gt; # pre-defined lower bound of prediction accuracy
&gt; hla.acc &lt;- c(0.9, 0.8, 0.8, 0.8, 0.8, 0.7)
&gt; 
&gt; 
&gt; for (hla.idx in seq_along(hla.list))
+ {
+ 	hla.id &lt;- hla.list[hla.idx]
+ 
+ 	# make a &quot;hlaAlleleClass&quot; object
+ 	hla &lt;- hlaAllele(HLA_Type_Table$sample.id,
+ 		H1 = HLA_Type_Table[, paste(hla.id, &quot;.1&quot;, sep=&quot;&quot;)],
+ 		H2 = HLA_Type_Table[, paste(hla.id, &quot;.2&quot;, sep=&quot;&quot;)],
+ 		locus=hla.id, assembly=&quot;hg19&quot;)
+ 
+ 	# divide HLA types randomly
+ 	set.seed(100)
+ 	hlatab &lt;- hlaSplitAllele(hla, train.prop=0.5)
+ 
+ 	# SNP predictors within the flanking region on each side
+ 	region &lt;- 500	# kb
+ 	snpid &lt;- hlaFlankingSNP(HapMap_CEU_Geno$snp.id,
+ 		HapMap_CEU_Geno$snp.position,
+ 		hla.id, region*1000, assembly=&quot;hg19&quot;)
+ 
+ 	# training and validation genotypes
+ 	train.geno &lt;- hlaGenoSubset(HapMap_CEU_Geno,
+ 		snp.sel=match(snpid, HapMap_CEU_Geno$snp.id),
+ 		samp.sel=match(hlatab$training$value$sample.id,
+ 		HapMap_CEU_Geno$sample.id))
+ 	test.geno &lt;- hlaGenoSubset(HapMap_CEU_Geno,
+ 		samp.sel=match(hlatab$validation$value$sample.id,
+ 		HapMap_CEU_Geno$sample.id))
+ 
+ 
+ 	# train a HIBAG model
+ 	set.seed(100)
+ 	model &lt;- hlaAttrBagging(hlatab$training, train.geno, nclassifier=10)
+ 	summary(model)
+ 
+ 	# validation
+ 	pred &lt;- hlaPredict(model, test.geno, type=&quot;response&quot;)
+ 	summary(pred)
+ 
+ 	# compare
+ 	comp &lt;- hlaCompareAllele(hlatab$validation, pred, allele.limit=model,
+ 		call.threshold=0)
+ 	print(comp$overall)
+ 
+ 	# check
+ 	if (comp$overall$acc.haplo &lt; hla.acc[hla.idx])
+ 		stop(&quot;HLA - &quot;, hla.id, &quot;, &#x27;acc.haplo&#x27; should be &gt;= &quot;, hla.acc[hla.idx], &quot;.&quot;)
+ 
+ 	cat(&quot;\n\n&quot;)
+ }
Build a HIBAG model with 10 individual classifiers:
    MAF threshold: NaN
    excluding 11 monomorphic SNPs
    # of SNPs randomly sampled as candidates for each selection: 17
    # of SNPs: 264
    # of samples: 34
    # of unique HLA alleles: 14
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:31
=== building individual classifier 1, out-of-bag (11/32.4%) ===
[1] 2023-01-29 00:48:31, oob acc: 77.27%, # of SNPs: 13, # of haplo: 23
=== building individual classifier 2, out-of-bag (13/38.2%) ===
[2] 2023-01-29 00:48:32, oob acc: 88.46%, # of SNPs: 11, # of haplo: 48
=== building individual classifier 3, out-of-bag (14/41.2%) ===
[3] 2023-01-29 00:48:32, oob acc: 85.71%, # of SNPs: 11, # of haplo: 14
=== building individual classifier 4, out-of-bag (10/29.4%) ===
[4] 2023-01-29 00:48:32, oob acc: 75.00%, # of SNPs: 13, # of haplo: 23
=== building individual classifier 5, out-of-bag (17/50.0%) ===
[5] 2023-01-29 00:48:32, oob acc: 79.41%, # of SNPs: 13, # of haplo: 34
=== building individual classifier 6, out-of-bag (11/32.4%) ===
[6] 2023-01-29 00:48:32, oob acc: 100.00%, # of SNPs: 19, # of haplo: 72
=== building individual classifier 7, out-of-bag (9/26.5%) ===
[7] 2023-01-29 00:48:32, oob acc: 100.00%, # of SNPs: 17, # of haplo: 37
=== building individual classifier 8, out-of-bag (13/38.2%) ===
[8] 2023-01-29 00:48:32, oob acc: 84.62%, # of SNPs: 14, # of haplo: 56
=== building individual classifier 9, out-of-bag (14/41.2%) ===
[9] 2023-01-29 00:48:32, oob acc: 89.29%, # of SNPs: 13, # of haplo: 34
=== building individual classifier 10, out-of-bag (13/38.2%) ===
[10] 2023-01-29 00:48:32, oob acc: 80.77%, # of SNPs: 14, # of haplo: 24
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0004220405 0.0004263127 0.0004647623 0.0055051726 0.0109090365 0.0235725089 
        Max.         Mean           SD 
0.4696060704 0.0448179210 0.1063804198 
Accuracy with training data: 98.53%
Out-of-bag accuracy: 86.05%
Gene: HLA-A
Training dataset: 34 samples X 264 SNPs
    # of HLA alleles: 14
    # of individual classifiers: 10
    total # of SNPs used: 92
    avg. # of SNPs in an individual classifier: 13.80
        (sd: 2.49, min: 11, max: 19, median: 13.00)
    avg. # of haplotypes in an individual classifier: 36.50
        (sd: 17.68, min: 14, max: 72, median: 34.00)
    avg. out-of-bag accuracy: 86.05%
        (sd: 8.68%, min: 75.00%, max: 100.00%, median: 85.16%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0004220405 0.0004263127 0.0004647623 0.0055051726 0.0109090365 0.0235725089 
        Max.         Mean           SD 
0.4696060704 0.0448179210 0.1063804198 
Genome assembly: hg19
HIBAG model for HLA-A:
    10 individual classifiers
    264 SNPs
    14 unique HLA alleles: 01:01, 02:01, 02:06, ...
Prediction:
    based on the averaged posterior probabilities
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 26
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:32)	0%
Predicting (2023-01-29 00:48:32)	100%
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 26
# of unique HLA alleles: 12
# of unique HLA genotypes: 14
Posterior probability:
  [0,0.25) [0.25,0.5) [0.5,0.75)   [0.75,1] 
  1 (3.8%)  3 (11.5%)  4 (15.4%) 18 (69.2%) 
Matching proportion of SNP haplotype:
    Min.  1st Qu.   Median     Mean  3rd Qu.     Max. 
0.000000 0.002746 0.007371 0.031621 0.024572 0.469606 
  total.num.ind crt.num.ind crt.num.haplo   acc.ind acc.haplo call.threshold
1            26          25            51 0.9615385 0.9807692              0
  n.call call.rate
1     26         1


Build a HIBAG model with 10 individual classifiers:
    MAF threshold: NaN
    excluding 1 monomorphic SNP
    # of SNPs randomly sampled as candidates for each selection: 19
    # of SNPs: 340
    # of samples: 28
    # of unique HLA alleles: 22
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:32
=== building individual classifier 1, out-of-bag (12/42.9%) ===
[1] 2023-01-29 00:48:32, oob acc: 58.33%, # of SNPs: 17, # of haplo: 52
=== building individual classifier 2, out-of-bag (11/39.3%) ===
[2] 2023-01-29 00:48:32, oob acc: 63.64%, # of SNPs: 18, # of haplo: 51
=== building individual classifier 3, out-of-bag (13/46.4%) ===
[3] 2023-01-29 00:48:32, oob acc: 50.00%, # of SNPs: 15, # of haplo: 29
=== building individual classifier 4, out-of-bag (11/39.3%) ===
[4] 2023-01-29 00:48:33, oob acc: 59.09%, # of SNPs: 12, # of haplo: 57
=== building individual classifier 5, out-of-bag (11/39.3%) ===
[5] 2023-01-29 00:48:33, oob acc: 63.64%, # of SNPs: 15, # of haplo: 86
=== building individual classifier 6, out-of-bag (12/42.9%) ===
[6] 2023-01-29 00:48:33, oob acc: 79.17%, # of SNPs: 18, # of haplo: 66
=== building individual classifier 7, out-of-bag (12/42.9%) ===
[7] 2023-01-29 00:48:33, oob acc: 70.83%, # of SNPs: 15, # of haplo: 86
=== building individual classifier 8, out-of-bag (9/32.1%) ===
[8] 2023-01-29 00:48:33, oob acc: 77.78%, # of SNPs: 16, # of haplo: 117
=== building individual classifier 9, out-of-bag (9/32.1%) ===
[9] 2023-01-29 00:48:33, oob acc: 77.78%, # of SNPs: 18, # of haplo: 92
=== building individual classifier 10, out-of-bag (9/32.1%) ===
[10] 2023-01-29 00:48:34, oob acc: 66.67%, # of SNPs: 16, # of haplo: 91
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
7.972868e-05 8.070947e-05 8.953662e-05 2.940413e-03 7.764710e-03 1.211147e-02 
        Max.         Mean           SD 
1.254185e-01 1.298944e-02 2.365315e-02 
Accuracy with training data: 100.00%
Out-of-bag accuracy: 66.69%
Gene: HLA-B
Training dataset: 28 samples X 340 SNPs
    # of HLA alleles: 22
    # of individual classifiers: 10
    total # of SNPs used: 119
    avg. # of SNPs in an individual classifier: 16.00
        (sd: 1.89, min: 12, max: 18, median: 16.00)
    avg. # of haplotypes in an individual classifier: 72.70
        (sd: 26.09, min: 29, max: 117, median: 76.00)
    avg. out-of-bag accuracy: 66.69%
        (sd: 9.68%, min: 50.00%, max: 79.17%, median: 65.15%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
7.972868e-05 8.070947e-05 8.953662e-05 2.940413e-03 7.764710e-03 1.211147e-02 
        Max.         Mean           SD 
1.254185e-01 1.298944e-02 2.365315e-02 
Genome assembly: hg19
HIBAG model for HLA-B:
    10 individual classifiers
    340 SNPs
    22 unique HLA alleles: 07:02, 08:01, 13:02, ...
Prediction:
    based on the averaged posterior probabilities
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 15
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:34)	0%
Predicting (2023-01-29 00:48:34)	100%
Gene: HLA-B
Range: [31321649bp, 31324989bp] on hg19
# of samples: 15
# of unique HLA alleles: 9
# of unique HLA genotypes: 12
Posterior probability:
  [0,0.25) [0.25,0.5) [0.5,0.75)   [0.75,1] 
 3 (20.0%)  5 (33.3%)  3 (20.0%)  4 (26.7%) 
Matching proportion of SNP haplotype:
     Min.   1st Qu.    Median      Mean   3rd Qu.      Max. 
2.000e-08 4.068e-05 2.934e-03 1.840e-02 5.152e-03 1.367e-01 
  total.num.ind crt.num.ind crt.num.haplo   acc.ind acc.haplo call.threshold
1            15          11            25 0.7333333 0.8333333              0
  n.call call.rate
1     15         1


Build a HIBAG model with 10 individual classifiers:
    MAF threshold: NaN
    excluding 2 monomorphic SNPs
    # of SNPs randomly sampled as candidates for each selection: 19
    # of SNPs: 354
    # of samples: 36
    # of unique HLA alleles: 17
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:34
=== building individual classifier 1, out-of-bag (13/36.1%) ===
[1] 2023-01-29 00:48:34, oob acc: 80.77%, # of SNPs: 19, # of haplo: 40
=== building individual classifier 2, out-of-bag (11/30.6%) ===
[2] 2023-01-29 00:48:34, oob acc: 90.91%, # of SNPs: 32, # of haplo: 32
=== building individual classifier 3, out-of-bag (14/38.9%) ===
[3] 2023-01-29 00:48:34, oob acc: 89.29%, # of SNPs: 19, # of haplo: 43
=== building individual classifier 4, out-of-bag (13/36.1%) ===
[4] 2023-01-29 00:48:34, oob acc: 84.62%, # of SNPs: 19, # of haplo: 72
=== building individual classifier 5, out-of-bag (10/27.8%) ===
[5] 2023-01-29 00:48:34, oob acc: 90.00%, # of SNPs: 19, # of haplo: 66
=== building individual classifier 6, out-of-bag (10/27.8%) ===
[6] 2023-01-29 00:48:34, oob acc: 95.00%, # of SNPs: 21, # of haplo: 59
=== building individual classifier 7, out-of-bag (16/44.4%) ===
[7] 2023-01-29 00:48:35, oob acc: 90.62%, # of SNPs: 18, # of haplo: 25
=== building individual classifier 8, out-of-bag (14/38.9%) ===
[8] 2023-01-29 00:48:35, oob acc: 89.29%, # of SNPs: 23, # of haplo: 57
=== building individual classifier 9, out-of-bag (13/36.1%) ===
[9] 2023-01-29 00:48:35, oob acc: 84.62%, # of SNPs: 18, # of haplo: 39
=== building individual classifier 10, out-of-bag (14/38.9%) ===
[10] 2023-01-29 00:48:35, oob acc: 89.29%, # of SNPs: 35, # of haplo: 62
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0007653736 0.0007682927 0.0007945643 0.0017428621 0.0044732464 0.0093175730 
        Max.         Mean           SD 
0.0703539734 0.0088728477 0.0132051834 
Accuracy with training data: 100.00%
Out-of-bag accuracy: 88.44%
Gene: HLA-C
Training dataset: 36 samples X 354 SNPs
    # of HLA alleles: 17
    # of individual classifiers: 10
    total # of SNPs used: 135
    avg. # of SNPs in an individual classifier: 22.30
        (sd: 6.13, min: 18, max: 35, median: 19.00)
    avg. # of haplotypes in an individual classifier: 49.50
        (sd: 15.74, min: 25, max: 72, median: 50.00)
    avg. out-of-bag accuracy: 88.44%
        (sd: 4.04%, min: 80.77%, max: 95.00%, median: 89.29%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0007653736 0.0007682927 0.0007945643 0.0017428621 0.0044732464 0.0093175730 
        Max.         Mean           SD 
0.0703539734 0.0088728477 0.0132051834 
Genome assembly: hg19
HIBAG model for HLA-C:
    10 individual classifiers
    354 SNPs
    17 unique HLA alleles: 01:02, 02:02, 03:03, ...
Prediction:
    based on the averaged posterior probabilities
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 24
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:35)	0%
Predicting (2023-01-29 00:48:35)	100%
Gene: HLA-C
Range: [31236526bp, 31239913bp] on hg19
# of samples: 24
# of unique HLA alleles: 14
# of unique HLA genotypes: 19
Posterior probability:
  [0,0.25) [0.25,0.5) [0.5,0.75)   [0.75,1] 
  2 (8.3%)  3 (12.5%)  6 (25.0%) 13 (54.2%) 
Matching proportion of SNP haplotype:
     Min.   1st Qu.    Median      Mean   3rd Qu.      Max. 
0.0000000 0.0000000 0.0002058 0.0058893 0.0035911 0.0468290 
  total.num.ind crt.num.ind crt.num.haplo   acc.ind acc.haplo call.threshold
1            24          16            39 0.6666667    0.8125              0
  n.call call.rate
1     24         1


Build a HIBAG model with 10 individual classifiers:
    MAF threshold: NaN
    excluding 4 monomorphic SNPs
    # of SNPs randomly sampled as candidates for each selection: 19
    # of SNPs: 345
    # of samples: 31
    # of unique HLA alleles: 7
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:35
=== building individual classifier 1, out-of-bag (11/35.5%) ===
[1] 2023-01-29 00:48:35, oob acc: 95.45%, # of SNPs: 11, # of haplo: 22
=== building individual classifier 2, out-of-bag (11/35.5%) ===
[2] 2023-01-29 00:48:35, oob acc: 100.00%, # of SNPs: 13, # of haplo: 22
=== building individual classifier 3, out-of-bag (15/48.4%) ===
[3] 2023-01-29 00:48:35, oob acc: 83.33%, # of SNPs: 15, # of haplo: 23
=== building individual classifier 4, out-of-bag (14/45.2%) ===
[4] 2023-01-29 00:48:35, oob acc: 82.14%, # of SNPs: 8, # of haplo: 14
=== building individual classifier 5, out-of-bag (13/41.9%) ===
[5] 2023-01-29 00:48:35, oob acc: 88.46%, # of SNPs: 11, # of haplo: 34
=== building individual classifier 6, out-of-bag (10/32.3%) ===
[6] 2023-01-29 00:48:35, oob acc: 90.00%, # of SNPs: 11, # of haplo: 21
=== building individual classifier 7, out-of-bag (13/41.9%) ===
[7] 2023-01-29 00:48:35, oob acc: 92.31%, # of SNPs: 14, # of haplo: 23
=== building individual classifier 8, out-of-bag (13/41.9%) ===
[8] 2023-01-29 00:48:35, oob acc: 96.15%, # of SNPs: 11, # of haplo: 16
=== building individual classifier 9, out-of-bag (14/45.2%) ===
[9] 2023-01-29 00:48:35, oob acc: 89.29%, # of SNPs: 12, # of haplo: 19
=== building individual classifier 10, out-of-bag (11/35.5%) ===
[10] 2023-01-29 00:48:35, oob acc: 86.36%, # of SNPs: 8, # of haplo: 13
Calculating matching proportion:
       Min.    0.1% Qu.      1% Qu.     1st Qu.      Median     3rd Qu. 
0.001972961 0.001998819 0.002231547 0.005363515 0.008831104 0.018431530 
       Max.        Mean          SD 
0.537093886 0.028877632 0.094687228 
Accuracy with training data: 96.77%
Out-of-bag accuracy: 90.35%
Gene: HLA-DQA1
Training dataset: 31 samples X 345 SNPs
    # of HLA alleles: 7
    # of individual classifiers: 10
    total # of SNPs used: 80
    avg. # of SNPs in an individual classifier: 11.40
        (sd: 2.27, min: 8, max: 15, median: 11.00)
    avg. # of haplotypes in an individual classifier: 20.70
        (sd: 5.96, min: 13, max: 34, median: 21.50)
    avg. out-of-bag accuracy: 90.35%
        (sd: 5.72%, min: 82.14%, max: 100.00%, median: 89.64%)
Matching proportion:
       Min.    0.1% Qu.      1% Qu.     1st Qu.      Median     3rd Qu. 
0.001972961 0.001998819 0.002231547 0.005363515 0.008831104 0.018431530 
       Max.        Mean          SD 
0.537093886 0.028877632 0.094687228 
Genome assembly: hg19
HIBAG model for HLA-DQA1:
    10 individual classifiers
    345 SNPs
    7 unique HLA alleles: 01:01, 01:02, 01:03, ...
Prediction:
    based on the averaged posterior probabilities
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 29
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:35)	0%
Predicting (2023-01-29 00:48:35)	100%
Gene: HLA-DQA1
Range: [32605169bp, 32612152bp] on hg19
# of samples: 29
# of unique HLA alleles: 6
# of unique HLA genotypes: 14
Posterior probability:
  [0,0.25) [0.25,0.5) [0.5,0.75)   [0.75,1] 
 5 (17.2%)  5 (17.2%)   2 (6.9%) 17 (58.6%) 
Matching proportion of SNP haplotype:
     Min.   1st Qu.    Median      Mean   3rd Qu.      Max. 
0.0000001 0.0019253 0.0069908 0.0532601 0.0167536 0.5404845 
  total.num.ind crt.num.ind crt.num.haplo   acc.ind acc.haplo call.threshold
1            29          21            49 0.7241379 0.8448276              0
  n.call call.rate
1     29         1


Build a HIBAG model with 10 individual classifiers:
    MAF threshold: NaN
    excluding 6 monomorphic SNPs
    # of SNPs randomly sampled as candidates for each selection: 19
    # of SNPs: 350
    # of samples: 34
    # of unique HLA alleles: 12
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:35
=== building individual classifier 1, out-of-bag (11/32.4%) ===
[1] 2023-01-29 00:48:35, oob acc: 86.36%, # of SNPs: 13, # of haplo: 34
=== building individual classifier 2, out-of-bag (13/38.2%) ===
[2] 2023-01-29 00:48:36, oob acc: 76.92%, # of SNPs: 21, # of haplo: 42
=== building individual classifier 3, out-of-bag (13/38.2%) ===
[3] 2023-01-29 00:48:36, oob acc: 80.77%, # of SNPs: 10, # of haplo: 17
=== building individual classifier 4, out-of-bag (13/38.2%) ===
[4] 2023-01-29 00:48:36, oob acc: 92.31%, # of SNPs: 22, # of haplo: 78
=== building individual classifier 5, out-of-bag (13/38.2%) ===
[5] 2023-01-29 00:48:36, oob acc: 92.31%, # of SNPs: 11, # of haplo: 40
=== building individual classifier 6, out-of-bag (14/41.2%) ===
[6] 2023-01-29 00:48:36, oob acc: 71.43%, # of SNPs: 8, # of haplo: 22
=== building individual classifier 7, out-of-bag (14/41.2%) ===
[7] 2023-01-29 00:48:36, oob acc: 71.43%, # of SNPs: 14, # of haplo: 53
=== building individual classifier 8, out-of-bag (11/32.4%) ===
[8] 2023-01-29 00:48:36, oob acc: 86.36%, # of SNPs: 14, # of haplo: 40
=== building individual classifier 9, out-of-bag (14/41.2%) ===
[9] 2023-01-29 00:48:36, oob acc: 100.00%, # of SNPs: 16, # of haplo: 56
=== building individual classifier 10, out-of-bag (13/38.2%) ===
[10] 2023-01-29 00:48:36, oob acc: 88.46%, # of SNPs: 14, # of haplo: 34
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0003282346 0.0003687353 0.0007332412 0.0038570393 0.0073528147 0.0148594626 
        Max.         Mean           SD 
0.3073781820 0.0225078064 0.0573939534 
Accuracy with training data: 98.53%
Out-of-bag accuracy: 84.64%
Gene: HLA-DQB1
Training dataset: 34 samples X 350 SNPs
    # of HLA alleles: 12
    # of individual classifiers: 10
    total # of SNPs used: 99
    avg. # of SNPs in an individual classifier: 14.30
        (sd: 4.45, min: 8, max: 22, median: 14.00)
    avg. # of haplotypes in an individual classifier: 41.60
        (sd: 17.55, min: 17, max: 78, median: 40.00)
    avg. out-of-bag accuracy: 84.64%
        (sd: 9.41%, min: 71.43%, max: 100.00%, median: 86.36%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0003282346 0.0003687353 0.0007332412 0.0038570393 0.0073528147 0.0148594626 
        Max.         Mean           SD 
0.3073781820 0.0225078064 0.0573939534 
Genome assembly: hg19
HIBAG model for HLA-DQB1:
    10 individual classifiers
    350 SNPs
    12 unique HLA alleles: 02:01, 02:02, 03:01, ...
Prediction:
    based on the averaged posterior probabilities
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 26
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:36)	0%
Predicting (2023-01-29 00:48:36)	100%
Gene: HLA-DQB1
Range: [32627241bp, 32634466bp] on hg19
# of samples: 26
# of unique HLA alleles: 10
# of unique HLA genotypes: 17
Posterior probability:
  [0,0.25) [0.25,0.5) [0.5,0.75)   [0.75,1] 
 3 (11.5%)  7 (26.9%)  5 (19.2%) 11 (42.3%) 
Matching proportion of SNP haplotype:
     Min.   1st Qu.    Median      Mean   3rd Qu.      Max. 
0.0000000 0.0002253 0.0018486 0.0308488 0.0099906 0.4023552 
  total.num.ind crt.num.ind crt.num.haplo   acc.ind acc.haplo call.threshold
1            26          21            46 0.8076923 0.8846154              0
  n.call call.rate
1     26         1


Build a HIBAG model with 10 individual classifiers:
    MAF threshold: NaN
    excluding 5 monomorphic SNPs
    # of SNPs randomly sampled as candidates for each selection: 18
    # of SNPs: 322
    # of samples: 35
    # of unique HLA alleles: 20
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:36
=== building individual classifier 1, out-of-bag (15/42.9%) ===
[1] 2023-01-29 00:48:37, oob acc: 66.67%, # of SNPs: 21, # of haplo: 132
=== building individual classifier 2, out-of-bag (13/37.1%) ===
[2] 2023-01-29 00:48:37, oob acc: 61.54%, # of SNPs: 19, # of haplo: 91
=== building individual classifier 3, out-of-bag (15/42.9%) ===
[3] 2023-01-29 00:48:37, oob acc: 70.00%, # of SNPs: 10, # of haplo: 33
=== building individual classifier 4, out-of-bag (13/37.1%) ===
[4] 2023-01-29 00:48:37, oob acc: 73.08%, # of SNPs: 17, # of haplo: 121
=== building individual classifier 5, out-of-bag (12/34.3%) ===
[5] 2023-01-29 00:48:38, oob acc: 79.17%, # of SNPs: 17, # of haplo: 153
=== building individual classifier 6, out-of-bag (12/34.3%) ===
[6] 2023-01-29 00:48:39, oob acc: 66.67%, # of SNPs: 20, # of haplo: 154
=== building individual classifier 7, out-of-bag (11/31.4%) ===
[7] 2023-01-29 00:48:39, oob acc: 63.64%, # of SNPs: 15, # of haplo: 38
=== building individual classifier 8, out-of-bag (11/31.4%) ===
[8] 2023-01-29 00:48:39, oob acc: 68.18%, # of SNPs: 19, # of haplo: 115
=== building individual classifier 9, out-of-bag (12/34.3%) ===
[9] 2023-01-29 00:48:40, oob acc: 79.17%, # of SNPs: 23, # of haplo: 126
=== building individual classifier 10, out-of-bag (11/31.4%) ===
[10] 2023-01-29 00:48:40, oob acc: 81.82%, # of SNPs: 22, # of haplo: 146
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
7.654289e-05 1.001066e-04 3.121801e-04 1.848734e-03 2.693889e-03 5.721191e-03 
        Max.         Mean           SD 
3.820492e-01 3.558536e-02 1.056328e-01 
Accuracy with training data: 94.29%
Out-of-bag accuracy: 70.99%
Gene: HLA-DRB1
Training dataset: 35 samples X 322 SNPs
    # of HLA alleles: 20
    # of individual classifiers: 10
    total # of SNPs used: 116
    avg. # of SNPs in an individual classifier: 18.30
        (sd: 3.80, min: 10, max: 23, median: 19.00)
    avg. # of haplotypes in an individual classifier: 110.90
        (sd: 44.01, min: 33, max: 154, median: 123.50)
    avg. out-of-bag accuracy: 70.99%
        (sd: 7.03%, min: 61.54%, max: 81.82%, median: 69.09%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
7.654289e-05 1.001066e-04 3.121801e-04 1.848734e-03 2.693889e-03 5.721191e-03 
        Max.         Mean           SD 
3.820492e-01 3.558536e-02 1.056328e-01 
Genome assembly: hg19
HIBAG model for HLA-DRB1:
    10 individual classifiers
    322 SNPs
    20 unique HLA alleles: 01:01, 01:03, 03:01, ...
Prediction:
    based on the averaged posterior probabilities
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 25
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:40)	0%
Predicting (2023-01-29 00:48:40)	100%
Gene: HLA-DRB1
Range: [32546546bp, 32557613bp] on hg19
# of samples: 25
# of unique HLA alleles: 9
# of unique HLA genotypes: 16
Posterior probability:
  [0,0.25) [0.25,0.5) [0.5,0.75)   [0.75,1] 
 4 (16.0%)  6 (24.0%)  6 (24.0%)  9 (36.0%) 
Matching proportion of SNP haplotype:
     Min.   1st Qu.    Median      Mean   3rd Qu.      Max. 
0.0000000 0.0003674 0.0015170 0.0053025 0.0024490 0.0880370 
  total.num.ind crt.num.ind crt.num.haplo acc.ind acc.haplo call.threshold
1            25          15            39     0.6      0.78              0
  n.call call.rate
1     25         1


&gt; 
&gt; 
&gt; 
&gt; #############################################################
&gt; 
&gt; {
+ 	function.list &lt;- readRDS(
+ 		system.file(&quot;Meta&quot;, &quot;Rd.rds&quot;, package=&quot;HIBAG&quot;))$Name
+ 
+ 	sapply(function.list, FUN = function(func.name)
+ 		{
+ 			args &lt;- list(
+ 				topic   = func.name,
+ 				package = &quot;HIBAG&quot;,
+ 				echo = FALSE,
+ 				verbose = FALSE,
+ 				ask = FALSE
+ 			)
+ 			suppressWarnings(do.call(example, args))
+ 			NULL
+ 		})
+ 	invisible()
+ }
SNP genotypes: 
    60 samples X 1564 SNPs
    SNPs range from 25769023bp to 33421576bp on hg19
Missing rate per SNP:
    min: 0, max: 0.0666667, mean: 0.0651215, median: 0.0666667, sd: 0.00968287
Missing rate per sample:
    min: 0, max: 0.969949, mean: 0.0651215, median: 0.000639386, sd: 0.243737
Minor allele frequency:
    min: 0, max: 0.5, mean: 0.227582, median: 0.205357, sd: 0.1389
Allelic information:
A/G C/T G/T A/C 
655 632 141 136 
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 34
# of unique HLA alleles: 14
# of unique HLA genotypes: 23
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 26
# of unique HLA alleles: 12
# of unique HLA genotypes: 14
Build a HIBAG model with 4 individual classifiers:
    MAF threshold: NaN
    excluding 11 monomorphic SNPs
    # of SNPs randomly sampled as candidates for each selection: 17
    # of SNPs: 264
    # of samples: 34
    # of unique HLA alleles: 14
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:40
=== building individual classifier 1, out-of-bag (11/32.4%) ===
     1, SNP: 211, loss: 196.4, oob acc: 54.55%, # of haplo: 13
     2, SNP: 66, loss: 173.548, oob acc: 63.64%, # of haplo: 13
     3, SNP: 177, loss: 136.352, oob acc: 68.18%, # of haplo: 13
     4, SNP: 108, loss: 95.8359, oob acc: 72.73%, # of haplo: 13
     5, SNP: 127, loss: 67.3216, oob acc: 77.27%, # of haplo: 13
     6, SNP: 95, loss: 47.5888, oob acc: 77.27%, # of haplo: 13
     7, SNP: 33, loss: 37.2631, oob acc: 77.27%, # of haplo: 16
     8, SNP: 6, loss: 29.7419, oob acc: 77.27%, # of haplo: 18
     9, SNP: 208, loss: 25.6913, oob acc: 77.27%, # of haplo: 19
    10, SNP: 225, loss: 25.3087, oob acc: 77.27%, # of haplo: 21
    11, SNP: 11, loss: 24.8356, oob acc: 77.27%, # of haplo: 23
    12, SNP: 151, loss: 19.4134, oob acc: 77.27%, # of haplo: 23
    13, SNP: 199, loss: 17.011, oob acc: 77.27%, # of haplo: 23
[1] 2023-01-29 00:48:40, oob acc: 77.27%, # of SNPs: 13, # of haplo: 23
=== building individual classifier 2, out-of-bag (13/38.2%) ===
     1, SNP: 160, loss: 221.236, oob acc: 76.92%, # of haplo: 17
     2, SNP: 145, loss: 173.538, oob acc: 80.77%, # of haplo: 23
     3, SNP: 177, loss: 128.58, oob acc: 84.62%, # of haplo: 31
     4, SNP: 111, loss: 79.6877, oob acc: 84.62%, # of haplo: 31
     5, SNP: 207, loss: 52.5557, oob acc: 88.46%, # of haplo: 32
     6, SNP: 245, loss: 41.8731, oob acc: 88.46%, # of haplo: 34
     7, SNP: 230, loss: 31.7937, oob acc: 88.46%, # of haplo: 38
     8, SNP: 151, loss: 20.4566, oob acc: 88.46%, # of haplo: 36
     9, SNP: 14, loss: 19.5805, oob acc: 88.46%, # of haplo: 42
    10, SNP: 129, loss: 19.5104, oob acc: 88.46%, # of haplo: 44
    11, SNP: 62, loss: 19.4853, oob acc: 88.46%, # of haplo: 48
[2] 2023-01-29 00:48:41, oob acc: 88.46%, # of SNPs: 11, # of haplo: 48
=== building individual classifier 3, out-of-bag (14/41.2%) ===
     1, SNP: 191, loss: 193.067, oob acc: 57.14%, # of haplo: 11
     2, SNP: 264, loss: 150.427, oob acc: 64.29%, # of haplo: 12
     3, SNP: 132, loss: 93.4067, oob acc: 67.86%, # of haplo: 12
     4, SNP: 128, loss: 39.8353, oob acc: 71.43%, # of haplo: 12
     5, SNP: 160, loss: 28.2998, oob acc: 75.00%, # of haplo: 12
     6, SNP: 144, loss: 13.635, oob acc: 75.00%, # of haplo: 12
     7, SNP: 111, loss: 6.04609, oob acc: 75.00%, # of haplo: 12
     8, SNP: 40, loss: 6.04583, oob acc: 82.14%, # of haplo: 14
     9, SNP: 141, loss: 6.04583, oob acc: 85.71%, # of haplo: 14
    10, SNP: 73, loss: 2.9038, oob acc: 85.71%, # of haplo: 14
    11, SNP: 199, loss: 2.20025, oob acc: 85.71%, # of haplo: 14
[3] 2023-01-29 00:48:41, oob acc: 85.71%, # of SNPs: 11, # of haplo: 14
=== building individual classifier 4, out-of-bag (10/29.4%) ===
     1, SNP: 147, loss: 158.631, oob acc: 50.00%, # of haplo: 12
     2, SNP: 152, loss: 140.375, oob acc: 55.00%, # of haplo: 13
     3, SNP: 78, loss: 115.887, oob acc: 60.00%, # of haplo: 16
     4, SNP: 115, loss: 77.8082, oob acc: 60.00%, # of haplo: 18
     5, SNP: 148, loss: 62.6831, oob acc: 65.00%, # of haplo: 18
     6, SNP: 13, loss: 46.5657, oob acc: 75.00%, # of haplo: 20
     7, SNP: 109, loss: 31.0312, oob acc: 75.00%, # of haplo: 20
     8, SNP: 176, loss: 22.5073, oob acc: 75.00%, # of haplo: 21
     9, SNP: 145, loss: 20.9122, oob acc: 75.00%, # of haplo: 21
    10, SNP: 128, loss: 20.6728, oob acc: 75.00%, # of haplo: 21
    11, SNP: 73, loss: 14.6217, oob acc: 75.00%, # of haplo: 22
    12, SNP: 151, loss: 10.2879, oob acc: 75.00%, # of haplo: 23
    13, SNP: 199, loss: 8.74645, oob acc: 75.00%, # of haplo: 23
[4] 2023-01-29 00:48:41, oob acc: 75.00%, # of SNPs: 13, # of haplo: 23
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0002440856 0.0002467397 0.0002706268 0.0043752100 0.0092453327 0.0260848720 
        Max.         Mean           SD 
0.4759517521 0.0445726720 0.1064934602 
Accuracy with training data: 97.06%
Out-of-bag accuracy: 81.61%
Gene: HLA-A
Training dataset: 34 samples X 264 SNPs
    # of HLA alleles: 14
    # of individual classifiers: 4
    total # of SNPs used: 38
    avg. # of SNPs in an individual classifier: 12.00
        (sd: 1.15, min: 11, max: 13, median: 12.00)
    avg. # of haplotypes in an individual classifier: 27.00
        (sd: 14.63, min: 14, max: 48, median: 23.00)
    avg. out-of-bag accuracy: 81.61%
        (sd: 6.49%, min: 75.00%, max: 88.46%, median: 81.49%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0002440856 0.0002467397 0.0002706268 0.0043752100 0.0092453327 0.0260848720 
        Max.         Mean           SD 
0.4759517521 0.0445726720 0.1064934602 
Genome assembly: hg19
HIBAG model for HLA-A:
    4 individual classifiers
    264 SNPs
    14 unique HLA alleles: 01:01, 02:01, 02:06, ...
Prediction:
    based on the averaged posterior probabilities
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 26
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:41)	0%
Predicting (2023-01-29 00:48:41)	100%
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 26
# of unique HLA alleles: 11
# of unique HLA genotypes: 13
Posterior probability:
  [0,0.25) [0.25,0.5) [0.5,0.75)   [0.75,1] 
  2 (7.7%)   2 (7.7%)  5 (19.2%) 17 (65.4%) 
Matching proportion of SNP haplotype:
    Min.  1st Qu.   Median     Mean  3rd Qu.     Max. 
0.000000 0.002097 0.007711 0.032476 0.025017 0.475952 
Dosages:
$dosage - num [1:14, 1:26] 2.50e-01 2.17e-04 8.33e-02 7.31e-01 3.94e-15 ...
 - attr(*, &quot;dimnames&quot;)=List of 2
  ..$ : chr [1:14] &quot;01:01&quot; &quot;02:01&quot; &quot;02:06&quot; &quot;03:01&quot; ...
  ..$ : chr [1:26] &quot;NA11881&quot; &quot;NA11992&quot; &quot;NA11994&quot; &quot;NA12249&quot; ...
HIBAG model for HLA-A:
    4 individual classifiers
    264 SNPs
    14 unique HLA alleles: 01:01, 02:01, 02:06, ...
Prediction:
    based on the averaged posterior probabilities
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 26
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:41)	0%
Predicting (2023-01-29 00:48:41)	100%
Open &#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/HIBAG/extdata/HapMap_CEU.bed&#x27; (the individual-major mode)
Open &#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/HIBAG/extdata/HapMap_CEU.fam&#x27;
Open &#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/HIBAG/extdata/HapMap_CEU.bim&#x27;
Import 3932 SNPs within the xMHC region on chromosome 6
HIBAG model for HLA-A:
    4 individual classifiers
    264 SNPs
    14 unique HLA alleles: 01:01, 02:01, 02:06, ...
Prediction:
    based on the averaged posterior probabilities
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 90
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:41)	0%
Predicting (2023-01-29 00:48:41)	100%
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 60
# of unique HLA alleles: 14
# of unique HLA genotypes: 29
using the default genome assembly (assembly=&quot;hg19&quot;)
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 60
# of unique HLA alleles: 14
# of unique HLA genotypes: 29
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 60
# of unique HLA alleles: 12
# of unique HLA genotypes: 28
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 60
# of unique HLA alleles: 14
# of unique HLA genotypes: 29
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 100
# of unique HLA alleles: 14
# of unique HLA genotypes: 29
Build a HIBAG model with 2 individual classifiers:
    MAF threshold: NaN
    excluding 32 monomorphic SNPs
    # of SNPs randomly sampled as candidates for each selection: 40
    # of SNPs: 1532
    # of samples: 60
    # of unique HLA alleles: 14
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:41
=== building individual classifier 1, out-of-bag (23/38.3%) ===
[1] 2023-01-29 00:48:42, oob acc: 76.09%, # of SNPs: 17, # of haplo: 85
=== building individual classifier 2, out-of-bag (23/38.3%) ===
[2] 2023-01-29 00:48:43, oob acc: 95.65%, # of SNPs: 27, # of haplo: 97
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
2.083333e-19 5.773691e-15 5.773504e-14 1.609970e-04 9.072180e-04 3.299599e-03 
        Max.         Mean           SD 
9.323132e-02 6.976053e-03 2.016301e-02 
Accuracy with training data: 98.33%
Out-of-bag accuracy: 85.87%
Gene: HLA-A
Training dataset: 60 samples X 1532 SNPs
    # of HLA alleles: 14
    # of individual classifiers: 2
    total # of SNPs used: 41
    avg. # of SNPs in an individual classifier: 22.00
        (sd: 7.07, min: 17, max: 27, median: 22.00)
    avg. # of haplotypes in an individual classifier: 91.00
        (sd: 8.49, min: 85, max: 97, median: 91.00)
    avg. out-of-bag accuracy: 85.87%
        (sd: 13.83%, min: 76.09%, max: 95.65%, median: 85.87%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
2.083333e-19 5.773691e-15 5.773504e-14 1.609970e-04 9.072180e-04 3.299599e-03 
        Max.         Mean           SD 
9.323132e-02 6.976053e-03 2.016301e-02 
Genome assembly: hg19
HIBAG model for HLA-A:
    2 individual classifiers
    1532 SNPs
    14 unique HLA alleles: 01:01, 02:01, 02:06, ...
Prediction:
    based on the averaged posterior probabilities
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 60
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:43)	0%
Predicting (2023-01-29 00:48:43)	100%
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 60
# of unique HLA alleles: 13
# of unique HLA genotypes: 28
Posterior probability:
  [0,0.25) [0.25,0.5) [0.5,0.75)   [0.75,1] 
  0 (0.0%)  8 (13.3%)  6 (10.0%) 46 (76.7%) 
Matching proportion of SNP haplotype:
     Min.   1st Qu.    Median      Mean   3rd Qu.      Max. 
0.0000000 0.0001610 0.0009072 0.0069761 0.0032996 0.0932313 
Dosages:
$dosage - num [1:14, 1:60] 1.00 5.25e-06 1.88e-12 1.44e-10 6.88e-07 ...
 - attr(*, &quot;dimnames&quot;)=List of 2
  ..$ : chr [1:14] &quot;01:01&quot; &quot;02:01&quot; &quot;02:06&quot; &quot;03:01&quot; ...
  ..$ : chr [1:60] &quot;NA11882&quot; &quot;NA11881&quot; &quot;NA11993&quot; &quot;NA11992&quot; ...
Convert to dosage VCF format:
    # of samples: 4
    # of unique HLA alleles: 5
    output: &lt;connection&gt;
##fileformat=VCFv4.0
##fileDate=20230129
##source=HIBAG
##FILTER=&lt;ID=PASS,Description=&quot;All filters passed&quot;&gt;
##FORMAT=&lt;ID=GT,Number=1,Type=String,Description=&quot;Genotype&quot;&gt;
##FORMAT=&lt;ID=DS,Number=1,Type=Float,Description=&quot;Dosage of HLA allele&quot;&gt;
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA11882	NA11881	NA11993	NA11992
6	29911954	HLA-A*01:01	A	P_0101	.	PASS	.	GT:DS	1/0:9.9999e-01	0/0:1.1430e-15	0/0:5.6614e-12	1/0:1.0000e+00
6	29911954	HLA-A*02:01	A	P_0201	.	PASS	.	GT:DS	0/0:5.2508e-06	0/0:1.0973e-16	0/0:1.3208e-01	0/1:1.0000e+00
6	29911954	HLA-A*03:01	A	P_0301	.	PASS	.	GT:DS	0/0:1.4438e-10	1/0:1.0000e+00	0/0:3.9622e-11	0/0:4.4399e-25
6	29911954	HLA-A*26:01	A	P_2601	.	PASS	.	GT:DS	0/0:1.8751e-12	0/1:5.0000e-01	1/0:5.7076e-01	0/0:7.1785e-26
6	29911954	HLA-A*29:02	A	P_2902	.	PASS	.	GT:DS	0/1:5.0000e-01	0/0:2.5000e-21	0/1:5.0000e-01	0/0:3.1934e-26
dominant model:
      [-/-] [-/h,h/h] %.[-/-] %.[-/h,h/h] chisq.st chisq.p fisher.p
24:02    49        11    42.9        81.8   4.0074  0.045*   0.042*
-----                                                              
01:01    36        24    50.0        50.0   0.0000  1.000    1.000 
02:01    25        35    52.0        48.6   0.0000  1.000    1.000 
02:06    59         1    50.8         0.0   0.0000  1.000    1.000 
03:01    51         9    49.0        55.6   0.0000  1.000    1.000 
11:01    55         5    50.9        40.0   0.0000  1.000    1.000 
23:01    58         2    50.0        50.0   0.0000  1.000    1.000 
24:03    59         1    50.8         0.0   0.0000  1.000    1.000 
25:01    55         5    52.7        20.0   0.8727  0.350    0.353 
26:01    57         3    52.6         0.0   1.4035  0.236    0.237 
29:02    56         4    51.8        25.0   0.2679  0.605    0.612 
31:01    57         3    49.1        66.7   0.0000  1.000    1.000 
32:01    56         4    46.4       100.0   2.4107  0.121    0.112 
68:01    57         3    52.6         0.0   1.4035  0.236    0.237 
additive model:
      [-] [h] %.[-] %.[h] chisq.st chisq.p fisher.p
01:01  95  25  50.5  48.0   0.0000  1.000    1.000 
02:01  77  43  48.1  53.5   0.1450  0.703    0.704 
02:06 119   1  50.4   0.0   0.0000  1.000    1.000 
03:01 111   9  49.5  55.6   0.0000  1.000    1.000 
11:01 115   5  50.4  40.0   0.0000  1.000    1.000 
23:01 117   3  50.4  33.3   0.0000  1.000    1.000 
24:02 109  11  46.8  81.8   3.6030  0.058    0.053 
24:03 119   1  50.4   0.0   0.0000  1.000    1.000 
25:01 115   5  51.3  20.0   0.8348  0.361    0.364 
26:01 117   3  51.3   0.0   1.3675  0.242    0.244 
29:02 116   4  50.9  25.0   0.2586  0.611    0.619 
31:01 117   3  49.6  66.7   0.0000  1.000    1.000 
32:01 116   4  48.3 100.0   2.3276  0.127    0.119 
68:01 117   3  51.3   0.0   1.3675  0.242    0.244 
recessive model:
      [-/-,-/h] [h/h] %.[-/-,-/h] %.[h/h] chisq.st chisq.p fisher.p
01:01        59     1        50.8       0    0.000  1.000    1.000 
02:01        52     8        46.2      75    1.298  0.255    0.254 
02:06        60     0        50.0       .        .       .        .
03:01        60     0        50.0       .        .       .        .
11:01        60     0        50.0       .        .       .        .
23:01        59     1        50.8       0    0.000  1.000    1.000 
24:02        60     0        50.0       .        .       .        .
24:03        60     0        50.0       .        .       .        .
25:01        60     0        50.0       .        .       .        .
26:01        60     0        50.0       .        .       .        .
29:02        60     0        50.0       .        .       .        .
31:01        60     0        50.0       .        .       .        .
32:01        60     0        50.0       .        .       .        .
68:01        60     0        50.0       .        .       .        .
genotype model:
      [-/-] [-/h] [h/h] %.[-/-] %.[-/h] %.[h/h] chisq.st chisq.p fisher.p
24:02    49    11     0    42.9    81.8       .   4.0074  0.045*   0.042*
-----                                                                    
01:01    36    23     1    50.0    52.2       0   1.0435  0.593    1.000 
02:01    25    27     8    52.0    40.7      75   2.9659  0.227    0.271 
02:06    59     1     0    50.8     0.0       .   0.0000  1.000    1.000 
03:01    51     9     0    49.0    55.6       .   0.0000  1.000    1.000 
11:01    55     5     0    50.9    40.0       .   0.0000  1.000    1.000 
23:01    58     1     1    50.0   100.0       0   2.0000  0.368    1.000 
24:03    59     1     0    50.8     0.0       .   0.0000  1.000    1.000 
25:01    55     5     0    52.7    20.0       .   0.8727  0.350    0.353 
26:01    57     3     0    52.6     0.0       .   1.4035  0.236    0.237 
29:02    56     4     0    51.8    25.0       .   0.2679  0.605    0.612 
31:01    57     3     0    49.1    66.7       .   0.0000  1.000    1.000 
32:01    56     4     0    46.4   100.0       .   2.4107  0.121    0.112 
68:01    57     3     0    52.6     0.0       .   1.4035  0.236    0.237 
dominant model:
      [-/-] [-/h,h/h] avg.[-/-] avg.[-/h,h/h] ttest.p
01:01    36        24  -0.14684     -0.117427  0.909 
02:01    25        35  -0.32331     -0.000618  0.190 
02:06    59         1  -0.14024      0.170057       .
03:01    51         9  -0.05600     -0.583178  0.147 
11:01    55         5  -0.19188      0.489815  0.287 
23:01    58         2  -0.15400      0.413687  0.281 
24:02    49        11  -0.10486     -0.269664  0.537 
24:03    59         1  -0.11409     -1.373118       .
25:01    55         5  -0.12237     -0.274749  0.742 
26:01    57         3  -0.12473     -0.331558  0.690 
29:02    56         4  -0.13044     -0.199941  0.789 
31:01    57         3  -0.10097     -0.783003  0.607 
32:01    56         4  -0.07702     -0.947791  0.092 
68:01    57         3  -0.16915      0.512457  0.196 
genotype model:
      [-/-] [-/h] [h/h] avg.[-/-] avg.[-/h] avg.[h/h] anova.p
01:01    36    23     1  -0.14684  -0.08833  -0.78655  0.784 
02:01    25    27     8  -0.32331  -0.02341   0.07631  0.446 
02:06    59     1     0  -0.14024   0.17006         .  0.756 
03:01    51     9     0  -0.05600  -0.58318         .  0.138 
11:01    55     5     0  -0.19188   0.48981         .  0.137 
23:01    58     1     1  -0.15400   0.10762   0.71975  0.663 
24:02    49    11     0  -0.10486  -0.26966         .  0.618 
24:03    59     1     0  -0.11409  -1.37312         .  0.205 
25:01    55     5     0  -0.12237  -0.27475         .  0.742 
26:01    57     3     0  -0.12473  -0.33156         .  0.725 
29:02    56     4     0  -0.13044  -0.19994         .  0.892 
31:01    57     3     0  -0.10097  -0.78300         .  0.243 
32:01    56     4     0  -0.07702  -0.94779         .  0.086 
68:01    57     3     0  -0.16915   0.51246         .  0.243 
Logistic regression (dominant model) with 60 individuals:
  glm(case ~ h, family = binomial, data = data)
      [-/-] [-/h,h/h] %.[-/-] %.[-/h,h/h] chisq.st chisq.p fisher.p      h.est
24:02    49        11    42.9        81.8   4.0074  0.045*   0.042*  1.792e+00
-----                                                                         
01:01    36        24    50.0        50.0   0.0000  1.000    1.000  -5.523e-16
02:01    25        35    52.0        48.6   0.0000  1.000    1.000  -1.372e-01
02:06    59         1    50.8         0.0   0.0000  1.000    1.000  -1.560e+01
03:01    51         9    49.0        55.6   0.0000  1.000    1.000   2.624e-01
11:01    55         5    50.9        40.0   0.0000  1.000    1.000  -4.418e-01
23:01    58         2    50.0        50.0   0.0000  1.000    1.000   2.226e-15
24:03    59         1    50.8         0.0   0.0000  1.000    1.000  -1.560e+01
25:01    55         5    52.7        20.0   0.8727  0.350    0.353  -1.495e+00
26:01    57         3    52.6         0.0   1.4035  0.236    0.237  -1.667e+01
29:02    56         4    51.8        25.0   0.2679  0.605    0.612  -1.170e+00
31:01    57         3    49.1        66.7   0.0000  1.000    1.000   7.282e-01
32:01    56         4    46.4       100.0   2.4107  0.121    0.112   1.771e+01
68:01    57         3    52.6         0.0   1.4035  0.236    0.237  -1.667e+01
          h.2.5%   h.97.5% h.pval
24:02     0.1585    3.4251 0.032*
-----                            
01:01    -1.0330    1.0330 1.000 
02:01    -1.1643    0.8899 0.793 
02:06 -2868.1268 2836.9268 0.991 
03:01    -1.1624    1.6872 0.718 
11:01    -2.3074    1.4237 0.643 
23:01    -2.8192    2.8192 1.000 
24:03 -2868.1268 2836.9268 0.991 
25:01    -3.7498    0.7588 0.194 
26:01 -2731.9621 2698.6192 0.990 
29:02    -3.4931    1.1530 0.324 
31:01    -1.7277    3.1842 0.561 
32:01 -3859.2763 3894.6947 0.993 
68:01 -2731.9621 2698.6192 0.990 
Logistic regression (dominant model) with 60 individuals:
  glm(case ~ h + pc1, family = binomial, data = data)
      [-/-] [-/h,h/h] %.[-/-] %.[-/h,h/h] chisq.st chisq.p fisher.p      h.est
24:02    49        11    42.9        81.8   4.0074  0.045*   0.042*  1.793e+00
-----                                                                         
01:01    36        24    50.0        50.0   0.0000  1.000    1.000  -2.268e-04
02:01    25        35    52.0        48.6   0.0000  1.000    1.000  -1.370e-01
02:06    59         1    50.8         0.0   0.0000  1.000    1.000  -1.562e+01
03:01    51         9    49.0        55.6   0.0000  1.000    1.000   2.686e-01
11:01    55         5    50.9        40.0   0.0000  1.000    1.000  -4.451e-01
23:01    58         2    50.0        50.0   0.0000  1.000    1.000  -3.062e-03
24:03    59         1    50.8         0.0   0.0000  1.000    1.000  -1.560e+01
25:01    55         5    52.7        20.0   0.8727  0.350    0.353  -1.501e+00
26:01    57         3    52.6         0.0   1.4035  0.236    0.237  -1.667e+01
29:02    56         4    51.8        25.0   0.2679  0.605    0.612  -1.189e+00
31:01    57         3    49.1        66.7   0.0000  1.000    1.000   7.289e-01
32:01    56         4    46.4       100.0   2.4107  0.121    0.112   1.781e+01
68:01    57         3    52.6         0.0   1.4035  0.236    0.237  -1.673e+01
          h.2.5%   h.97.5% h.pval   pc1.est pc1.2.5% pc1.97.5% pc1.pval
24:02     0.1587    3.4264 0.032*  0.011111  -0.5249    0.5471   0.968 
-----                                                                  
01:01    -1.0334    1.0330 1.000  -0.005807  -0.5126    0.5010   0.982 
02:01    -1.1652    0.8913 0.794  -0.002618  -0.5102    0.5049   0.992 
02:06 -2868.1460 2836.9076 0.991  -0.028534  -0.5374    0.4803   0.912 
03:01    -1.1813    1.7185 0.717   0.011958  -0.5044    0.5283   0.964 
11:01    -2.3225    1.4322 0.642   0.008025  -0.5026    0.5186   0.975 
23:01    -2.8348    2.8287 0.998  -0.005857  -0.5148    0.5031   0.982 
24:03 -2868.1286 2836.9250 0.991  -0.011249  -0.5182    0.4957   0.965 
25:01    -3.7579    0.7568 0.193  -0.025685  -0.5490    0.4976   0.923 
26:01 -2731.8901 2698.5450 0.990  -0.014069  -0.5297    0.5015   0.957 
29:02    -3.5309    1.1526 0.320   0.033234  -0.4796    0.5461   0.899 
31:01    -1.7274    3.1851 0.561  -0.008320  -0.5153    0.4987   0.974 
32:01 -3845.6317 3881.2510 0.993  -0.125426  -0.6671    0.4162   0.650 
68:01 -2721.2124 2687.7497 0.990  -0.086589  -0.6512    0.4781   0.764 
Logistic regression (dominant model) with 60 individuals:
  glm(case ~ h + pc1, family = binomial, data = data)
      [-/-] [-/h,h/h] %.[-/-] %.[-/h,h/h] chisq.st chisq.p fisher.p  h.est_OR
24:02    49        11    42.9        81.8   4.0074  0.045*   0.042* 6.005e+00
-----                                                                        
01:01    36        24    50.0        50.0   0.0000  1.000    1.000  9.998e-01
02:01    25        35    52.0        48.6   0.0000  1.000    1.000  8.720e-01
02:06    59         1    50.8         0.0   0.0000  1.000    1.000  1.647e-07
03:01    51         9    49.0        55.6   0.0000  1.000    1.000  1.308e+00
11:01    55         5    50.9        40.0   0.0000  1.000    1.000  6.407e-01
23:01    58         2    50.0        50.0   0.0000  1.000    1.000  9.969e-01
24:03    59         1    50.8         0.0   0.0000  1.000    1.000  1.676e-07
25:01    55         5    52.7        20.0   0.8727  0.350    0.353  2.230e-01
26:01    57         3    52.6         0.0   1.4035  0.236    0.237  5.744e-08
29:02    56         4    51.8        25.0   0.2679  0.605    0.612  3.045e-01
31:01    57         3    49.1        66.7   0.0000  1.000    1.000  2.073e+00
32:01    56         4    46.4       100.0   2.4107  0.121    0.112  5.428e+07
68:01    57         3    52.6         0.0   1.4035  0.236    0.237  5.416e-08
      h.2.5%_OR h.97.5%_OR h.pval   pc1.est pc1.2.5% pc1.97.5% pc1.pval
24:02   1.17200     30.766 0.032*  0.011111  -0.5249    0.5471   0.968 
-----                                                                  
01:01   0.35579      2.809 1.000  -0.005807  -0.5126    0.5010   0.982 
02:01   0.31185      2.438 0.794  -0.002618  -0.5102    0.5049   0.992 
02:06   0.00000        Inf 0.991  -0.028534  -0.5374    0.4803   0.912 
03:01   0.30687      5.576 0.717   0.011958  -0.5044    0.5283   0.964 
11:01   0.09803      4.188 0.642   0.008025  -0.5026    0.5186   0.975 
23:01   0.05873     16.923 0.998  -0.005857  -0.5148    0.5031   0.982 
24:03   0.00000        Inf 0.991  -0.011249  -0.5182    0.4957   0.965 
25:01   0.02333      2.131 0.193  -0.025685  -0.5490    0.4976   0.923 
26:01   0.00000        Inf 0.990  -0.014069  -0.5297    0.5015   0.957 
29:02   0.02928      3.167 0.320   0.033234  -0.4796    0.5461   0.899 
31:01   0.17774     24.171 0.561  -0.008320  -0.5153    0.4987   0.974 
32:01   0.00000        Inf 0.993  -0.125426  -0.6671    0.4162   0.650 
68:01   0.00000        Inf 0.990  -0.086589  -0.6512    0.4781   0.764 
Linear regression (dominant model) with 60 individuals:
  glm(y ~ h, data = data)
      [-/-] [-/h,h/h] avg.[-/-] avg.[-/h,h/h] ttest.p    h.est  h.2.5% h.97.5%
01:01    36        24  -0.14684     -0.117427  0.909   0.02941 -0.4805  0.5393
02:01    25        35  -0.32331     -0.000618  0.190   0.32269 -0.1772  0.8226
02:06    59         1  -0.14024      0.170057       .  0.31030 -1.6397  2.2603
03:01    51         9  -0.05600     -0.583178  0.147  -0.52718 -1.2136  0.1592
11:01    55         5  -0.19188      0.489815  0.287   0.68170 -0.2051  1.5685
23:01    58         2  -0.15400      0.413687  0.281   0.56768 -0.8165  1.9518
24:02    49        11  -0.10486     -0.269664  0.537  -0.16481 -0.8091  0.4795
24:03    59         1  -0.11409     -1.373118       . -1.25903 -3.1835  0.6655
25:01    55         5  -0.12237     -0.274749  0.742  -0.15237 -1.0555  0.7507
26:01    57         3  -0.12473     -0.331558  0.690  -0.20683 -1.3519  0.9383
29:02    56         4  -0.13044     -0.199941  0.789  -0.06950 -1.0709  0.9319
31:01    57         3  -0.10097     -0.783003  0.607  -0.68203 -1.8149  0.4508
32:01    56         4  -0.07702     -0.947791  0.092  -0.87077 -1.8470  0.1054
68:01    57         3  -0.16915      0.512457  0.196   0.68161 -0.4512  1.8145
      h.pval
01:01 0.910 
02:01 0.211 
02:06 0.756 
03:01 0.138 
11:01 0.137 
23:01 0.425 
24:02 0.618 
24:03 0.205 
25:01 0.742 
26:01 0.725 
29:02 0.892 
31:01 0.243 
32:01 0.086 
68:01 0.243 
Linear regression (dominant model) with 60 individuals:
  glm(y ~ h + pc1, data = data)
      [-/-] [-/h,h/h] avg.[-/-] avg.[-/h,h/h] ttest.p    h.est  h.2.5%
01:01    36        24  -0.14684     -0.117427  0.909   0.03377 -0.4773
02:01    25        35  -0.32331     -0.000618  0.190   0.31273 -0.1891
02:06    59         1  -0.14024      0.170057       .  0.38821 -1.5722
03:01    51         9  -0.05600     -0.583178  0.147  -0.48613 -1.1884
11:01    55         5  -0.19188      0.489815  0.287   0.64430 -0.2520
23:01    58         2  -0.15400      0.413687  0.281   0.63150 -0.7598
24:02    49        11  -0.10486     -0.269664  0.537  -0.15742 -0.8034
24:03    59         1  -0.11409     -1.373118       . -1.24145 -3.1708
25:01    55         5  -0.12237     -0.274749  0.742  -0.13241 -1.0388
26:01    57         3  -0.12473     -0.331558  0.690  -0.19823 -1.3460
29:02    56         4  -0.13044     -0.199941  0.789  -0.13606 -1.1496
31:01    57         3  -0.10097     -0.783003  0.607  -0.69057 -1.8254
32:01    56         4  -0.07702     -0.947791  0.092  -0.99595 -1.9862
68:01    57         3  -0.16915      0.512457  0.196   0.76795 -0.3749
        h.97.5% h.pval pc1.est pc1.2.5% pc1.97.5% pc1.pval
01:01  0.544844 0.897  0.11172  -0.1390    0.3624   0.386 
02:01  0.814606 0.227  0.10412  -0.1436    0.3519   0.414 
02:06  2.348616 0.699  0.11570  -0.1356    0.3670   0.371 
03:01  0.216142 0.180  0.07919  -0.1719    0.3303   0.539 
11:01  1.540569 0.164  0.09117  -0.1569    0.3392   0.474 
23:01  2.022811 0.377  0.12207  -0.1280    0.3721   0.343 
24:02  0.488543 0.635  0.10982  -0.1404    0.3601   0.393 
24:03  0.687920 0.212  0.10809  -0.1392    0.3554   0.395 
25:01  0.773943 0.776  0.10956  -0.1413    0.3604   0.396 
26:01  0.949529 0.736  0.11067  -0.1398    0.3611   0.390 
29:02  0.877431 0.793  0.11626  -0.1369    0.3694   0.372 
31:01  0.444260 0.238  0.11387  -0.1338    0.3615   0.371 
32:01 -0.005739 0.054  0.16001  -0.0873    0.4073   0.210 
68:01  1.910822 0.193  0.13482  -0.1146    0.3842   0.294 
Linear regression (dominant model) with 60 individuals:
  glm(y ~ h + pc1, data = data)
      [-/-] [-/h,h/h] avg.[-/-] avg.[-/h,h/h] ttest.p    h.est  h.2.5%
01:01    36        24  -0.14684     -0.117427  0.909   0.03377 -0.4773
02:01    25        35  -0.32331     -0.000618  0.190   0.31273 -0.1891
02:06    59         1  -0.14024      0.170057       .  0.38821 -1.5722
03:01    51         9  -0.05600     -0.583178  0.147  -0.48613 -1.1884
11:01    55         5  -0.19188      0.489815  0.287   0.64430 -0.2520
23:01    58         2  -0.15400      0.413687  0.281   0.63150 -0.7598
24:02    49        11  -0.10486     -0.269664  0.537  -0.15742 -0.8034
24:03    59         1  -0.11409     -1.373118       . -1.24145 -3.1708
25:01    55         5  -0.12237     -0.274749  0.742  -0.13241 -1.0388
26:01    57         3  -0.12473     -0.331558  0.690  -0.19823 -1.3460
29:02    56         4  -0.13044     -0.199941  0.789  -0.13606 -1.1496
31:01    57         3  -0.10097     -0.783003  0.607  -0.69057 -1.8254
32:01    56         4  -0.07702     -0.947791  0.092  -0.99595 -1.9862
68:01    57         3  -0.16915      0.512457  0.196   0.76795 -0.3749
        h.97.5% h.pval pc1.est pc1.2.5% pc1.97.5% pc1.pval
01:01  0.544844 0.897  0.11172  -0.1390    0.3624   0.386 
02:01  0.814606 0.227  0.10412  -0.1436    0.3519   0.414 
02:06  2.348616 0.699  0.11570  -0.1356    0.3670   0.371 
03:01  0.216142 0.180  0.07919  -0.1719    0.3303   0.539 
11:01  1.540569 0.164  0.09117  -0.1569    0.3392   0.474 
23:01  2.022811 0.377  0.12207  -0.1280    0.3721   0.343 
24:02  0.488543 0.635  0.10982  -0.1404    0.3601   0.393 
24:03  0.687920 0.212  0.10809  -0.1392    0.3554   0.395 
25:01  0.773943 0.776  0.10956  -0.1413    0.3604   0.396 
26:01  0.949529 0.736  0.11067  -0.1398    0.3611   0.390 
29:02  0.877431 0.793  0.11626  -0.1369    0.3694   0.372 
31:01  0.444260 0.238  0.11387  -0.1338    0.3615   0.371 
32:01 -0.005739 0.054  0.16001  -0.0873    0.4073   0.210 
68:01  1.910822 0.193  0.13482  -0.1146    0.3842   0.294 
Logistic regression (additive model) with 60 individuals:
  glm(case ~ h, family = binomial, data = data)
      [-] [h] %.[-] %.[h] chisq.st chisq.p fisher.p    h.est     h.2.5%
24:02 109  11  46.8  81.8   3.6030  0.058    0.053    1.7918     0.1585
-----                                                                  
01:01  95  25  50.5  48.0   0.0000  1.000    1.000   -0.1207    -1.0843
02:01  77  43  48.1  53.5   0.1450  0.703    0.704    0.2137    -0.5289
02:06 119   1  50.4   0.0   0.0000  1.000    1.000  -15.6000 -2868.1268
03:01 111   9  49.5  55.6   0.0000  1.000    1.000    0.2624    -1.1624
11:01 115   5  50.4  40.0   0.0000  1.000    1.000   -0.4418    -2.3074
23:01 117   3  50.4  33.3   0.0000  1.000    1.000   -0.4323    -2.3435
24:03 119   1  50.4   0.0   0.0000  1.000    1.000  -15.6000 -2868.1268
25:01 115   5  51.3  20.0   0.8348  0.361    0.364   -1.4955    -3.7498
26:01 117   3  51.3   0.0   1.3675  0.242    0.244  -16.6714 -2731.9621
29:02 116   4  50.9  25.0   0.2586  0.611    0.619   -1.1701    -3.4931
31:01 117   3  49.6  66.7   0.0000  1.000    1.000    0.7282    -1.7277
32:01 116   4  48.3 100.0   2.3276  0.127    0.119   17.7092 -3859.2763
68:01 117   3  51.3   0.0   1.3675  0.242    0.244  -16.6714 -2731.9621
        h.97.5% h.pval
24:02    3.4251 0.032*
-----                 
01:01    0.8430 0.806 
02:01    0.9563 0.573 
02:06 2836.9268 0.991 
03:01    1.6872 0.718 
11:01    1.4237 0.643 
23:01    1.4789 0.658 
24:03 2836.9268 0.991 
25:01    0.7588 0.194 
26:01 2698.6192 0.990 
29:02    1.1530 0.324 
31:01    3.1842 0.561 
32:01 3894.6947 0.993 
68:01 2698.6192 0.990 
Logistic regression (recessive model) with 60 individuals:
  glm(case ~ h, family = binomial, data = data)
      [-/-,-/h] [h/h] %.[-/-,-/h] %.[h/h] chisq.st chisq.p fisher.p   h.est
01:01        59     1        50.8       0    0.000  1.000    1.000  -15.600
02:01        52     8        46.2      75    1.298  0.255    0.254    1.253
02:06        60     0        50.0       .        .       .        .       .
03:01        60     0        50.0       .        .       .        .       .
11:01        60     0        50.0       .        .       .        .       .
23:01        59     1        50.8       0    0.000  1.000    1.000  -15.600
24:02        60     0        50.0       .        .       .        .       .
24:03        60     0        50.0       .        .       .        .       .
25:01        60     0        50.0       .        .       .        .       .
26:01        60     0        50.0       .        .       .        .       .
29:02        60     0        50.0       .        .       .        .       .
31:01        60     0        50.0       .        .       .        .       .
32:01        60     0        50.0       .        .       .        .       .
68:01        60     0        50.0       .        .       .        .       .
          h.2.5%  h.97.5% h.pval
01:01 -2868.1268 2836.927 0.991 
02:01    -0.4379    2.943 0.146 
02:06          .        .      .
03:01          .        .      .
11:01          .        .      .
23:01 -2868.1268 2836.927 0.991 
24:02          .        .      .
24:03          .        .      .
25:01          .        .      .
26:01          .        .      .
29:02          .        .      .
31:01          .        .      .
32:01          .        .      .
68:01          .        .      .
Logistic regression (genotype model) with 60 individuals:
  glm(case ~ h, family = binomial, data = data)
      [-/-] [-/h] [h/h] %.[-/-] %.[-/h] %.[h/h] chisq.st chisq.p fisher.p
24:02    49    11     0    42.9    81.8       .   4.0074  0.045*   0.042*
-----                                                                    
01:01    36    23     1    50.0    52.2       0   1.0435  0.593    1.000 
02:01    25    27     8    52.0    40.7      75   2.9659  0.227    0.271 
02:06    59     1     0    50.8     0.0       .   0.0000  1.000    1.000 
03:01    51     9     0    49.0    55.6       .   0.0000  1.000    1.000 
11:01    55     5     0    50.9    40.0       .   0.0000  1.000    1.000 
23:01    58     1     1    50.0   100.0       0   2.0000  0.368    1.000 
24:03    59     1     0    50.8     0.0       .   0.0000  1.000    1.000 
25:01    55     5     0    52.7    20.0       .   0.8727  0.350    0.353 
26:01    57     3     0    52.6     0.0       .   1.4035  0.236    0.237 
29:02    56     4     0    51.8    25.0       .   0.2679  0.605    0.612 
31:01    57     3     0    49.1    66.7       .   0.0000  1.000    1.000 
32:01    56     4     0    46.4   100.0       .   2.4107  0.121    0.112 
68:01    57     3     0    52.6     0.0       .   1.4035  0.236    0.237 
         h1.est    h1.2.5%  h1.97.5% h1.pval  h2.est    h2.2.5% h2.97.5%
24:02   1.79176     0.1585    3.4251  0.032*       .          .        .
-----                                                                   
01:01   0.08701    -0.9600    1.1340  0.871  -15.566 -2868.0929 2836.961
02:01  -0.45474    -1.5524    0.6430  0.417    1.019    -0.7637    2.801
02:06 -15.59997 -2868.1268 2836.9268  0.991        .          .        .
03:01   0.26236    -1.1624    1.6872  0.718        .          .        .
11:01  -0.44183    -2.3074    1.4237  0.643        .          .        .
23:01  16.56607 -4686.4552 4719.5873  0.994  -16.566 -4719.5873 4686.455
24:03 -15.59997 -2868.1268 2836.9268  0.991        .          .        .
25:01  -1.49549    -3.7498    0.7588  0.194        .          .        .
26:01 -16.67143 -2731.9621 2698.6192  0.990        .          .        .
29:02  -1.17007    -3.4931    1.1530  0.324        .          .        .
31:01   0.72824    -1.7277    3.1842  0.561        .          .        .
32:01  17.70917 -3859.2763 3894.6947  0.993        .          .        .
68:01 -16.67143 -2731.9621 2698.6192  0.990        .          .        .
      h2.pval
24:02       .
-----        
01:01  0.991 
02:01  0.263 
02:06       .
03:01       .
11:01       .
23:01  0.994 
24:03       .
25:01       .
26:01       .
29:02       .
31:01       .
32:01       .
68:01       .
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 34
# of unique HLA alleles: 14
# of unique HLA genotypes: 23
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 26
# of unique HLA alleles: 12
# of unique HLA genotypes: 14
Build a HIBAG model with 4 individual classifiers:
    MAF threshold: NaN
    excluding 11 monomorphic SNPs
    # of SNPs randomly sampled as candidates for each selection: 17
    # of SNPs: 264
    # of samples: 34
    # of unique HLA alleles: 14
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:44
=== building individual classifier 1, out-of-bag (11/32.4%) ===
     1, SNP: 211, loss: 196.4, oob acc: 54.55%, # of haplo: 13
     2, SNP: 66, loss: 173.548, oob acc: 63.64%, # of haplo: 13
     3, SNP: 177, loss: 136.352, oob acc: 68.18%, # of haplo: 13
     4, SNP: 108, loss: 95.8359, oob acc: 72.73%, # of haplo: 13
     5, SNP: 127, loss: 67.3216, oob acc: 77.27%, # of haplo: 13
     6, SNP: 95, loss: 47.5888, oob acc: 77.27%, # of haplo: 13
     7, SNP: 33, loss: 37.2631, oob acc: 77.27%, # of haplo: 16
     8, SNP: 6, loss: 29.7419, oob acc: 77.27%, # of haplo: 18
     9, SNP: 208, loss: 25.6913, oob acc: 77.27%, # of haplo: 19
    10, SNP: 225, loss: 25.3087, oob acc: 77.27%, # of haplo: 21
    11, SNP: 11, loss: 24.8356, oob acc: 77.27%, # of haplo: 23
    12, SNP: 151, loss: 19.4134, oob acc: 77.27%, # of haplo: 23
    13, SNP: 199, loss: 17.011, oob acc: 77.27%, # of haplo: 23
[1] 2023-01-29 00:48:44, oob acc: 77.27%, # of SNPs: 13, # of haplo: 23
=== building individual classifier 2, out-of-bag (13/38.2%) ===
     1, SNP: 160, loss: 221.236, oob acc: 76.92%, # of haplo: 17
     2, SNP: 145, loss: 173.538, oob acc: 80.77%, # of haplo: 23
     3, SNP: 177, loss: 128.58, oob acc: 84.62%, # of haplo: 31
     4, SNP: 111, loss: 79.6877, oob acc: 84.62%, # of haplo: 31
     5, SNP: 207, loss: 52.5557, oob acc: 88.46%, # of haplo: 32
     6, SNP: 245, loss: 41.8731, oob acc: 88.46%, # of haplo: 34
     7, SNP: 230, loss: 31.7937, oob acc: 88.46%, # of haplo: 38
     8, SNP: 151, loss: 20.4566, oob acc: 88.46%, # of haplo: 36
     9, SNP: 14, loss: 19.5805, oob acc: 88.46%, # of haplo: 42
    10, SNP: 129, loss: 19.5104, oob acc: 88.46%, # of haplo: 44
    11, SNP: 62, loss: 19.4853, oob acc: 88.46%, # of haplo: 48
[2] 2023-01-29 00:48:44, oob acc: 88.46%, # of SNPs: 11, # of haplo: 48
=== building individual classifier 3, out-of-bag (14/41.2%) ===
     1, SNP: 191, loss: 193.067, oob acc: 57.14%, # of haplo: 11
     2, SNP: 264, loss: 150.427, oob acc: 64.29%, # of haplo: 12
     3, SNP: 132, loss: 93.4067, oob acc: 67.86%, # of haplo: 12
     4, SNP: 128, loss: 39.8353, oob acc: 71.43%, # of haplo: 12
     5, SNP: 160, loss: 28.2998, oob acc: 75.00%, # of haplo: 12
     6, SNP: 144, loss: 13.635, oob acc: 75.00%, # of haplo: 12
     7, SNP: 111, loss: 6.04609, oob acc: 75.00%, # of haplo: 12
     8, SNP: 40, loss: 6.04583, oob acc: 82.14%, # of haplo: 14
     9, SNP: 141, loss: 6.04583, oob acc: 85.71%, # of haplo: 14
    10, SNP: 73, loss: 2.9038, oob acc: 85.71%, # of haplo: 14
    11, SNP: 199, loss: 2.20025, oob acc: 85.71%, # of haplo: 14
[3] 2023-01-29 00:48:44, oob acc: 85.71%, # of SNPs: 11, # of haplo: 14
=== building individual classifier 4, out-of-bag (10/29.4%) ===
     1, SNP: 147, loss: 158.631, oob acc: 50.00%, # of haplo: 12
     2, SNP: 152, loss: 140.375, oob acc: 55.00%, # of haplo: 13
     3, SNP: 78, loss: 115.887, oob acc: 60.00%, # of haplo: 16
     4, SNP: 115, loss: 77.8082, oob acc: 60.00%, # of haplo: 18
     5, SNP: 148, loss: 62.6831, oob acc: 65.00%, # of haplo: 18
     6, SNP: 13, loss: 46.5657, oob acc: 75.00%, # of haplo: 20
     7, SNP: 109, loss: 31.0312, oob acc: 75.00%, # of haplo: 20
     8, SNP: 176, loss: 22.5073, oob acc: 75.00%, # of haplo: 21
     9, SNP: 145, loss: 20.9122, oob acc: 75.00%, # of haplo: 21
    10, SNP: 128, loss: 20.6728, oob acc: 75.00%, # of haplo: 21
    11, SNP: 73, loss: 14.6217, oob acc: 75.00%, # of haplo: 22
    12, SNP: 151, loss: 10.2879, oob acc: 75.00%, # of haplo: 23
    13, SNP: 199, loss: 8.74645, oob acc: 75.00%, # of haplo: 23
[4] 2023-01-29 00:48:44, oob acc: 75.00%, # of SNPs: 13, # of haplo: 23
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0002440856 0.0002467397 0.0002706268 0.0043752100 0.0092453327 0.0260848720 
        Max.         Mean           SD 
0.4759517521 0.0445726720 0.1064934602 
Accuracy with training data: 97.06%
Out-of-bag accuracy: 81.61%
Gene: HLA-A
Training dataset: 34 samples X 264 SNPs
    # of HLA alleles: 14
    # of individual classifiers: 4
    total # of SNPs used: 38
    avg. # of SNPs in an individual classifier: 12.00
        (sd: 1.15, min: 11, max: 13, median: 12.00)
    avg. # of haplotypes in an individual classifier: 27.00
        (sd: 14.63, min: 14, max: 48, median: 23.00)
    avg. out-of-bag accuracy: 81.61%
        (sd: 6.49%, min: 75.00%, max: 88.46%, median: 81.49%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0002440856 0.0002467397 0.0002706268 0.0043752100 0.0092453327 0.0260848720 
        Max.         Mean           SD 
0.4759517521 0.0445726720 0.1064934602 
Genome assembly: hg19
HIBAG model for HLA-A:
    4 individual classifiers
    264 SNPs
    14 unique HLA alleles: 01:01, 02:01, 02:06, ...
Prediction:
    based on the averaged posterior probabilities
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 26
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:44)	0%
Predicting (2023-01-29 00:48:44)	100%
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 26
# of unique HLA alleles: 11
# of unique HLA genotypes: 13
Posterior probability:
  [0,0.25) [0.25,0.5) [0.5,0.75)   [0.75,1] 
  2 (7.7%)   2 (7.7%)  5 (19.2%) 17 (65.4%) 
Matching proportion of SNP haplotype:
    Min.  1st Qu.   Median     Mean  3rd Qu.     Max. 
0.000000 0.002097 0.007711 0.032476 0.025017 0.475952 
Dosages:
$dosage - num [1:14, 1:26] 2.50e-01 2.17e-04 8.33e-02 7.31e-01 3.94e-15 ...
 - attr(*, &quot;dimnames&quot;)=List of 2
  ..$ : chr [1:14] &quot;01:01&quot; &quot;02:01&quot; &quot;02:06&quot; &quot;03:01&quot; ...
  ..$ : chr [1:26] &quot;NA11881&quot; &quot;NA11992&quot; &quot;NA11994&quot; &quot;NA12249&quot; ...
HIBAG model for HLA-A:
    4 individual classifiers
    264 SNPs
    14 unique HLA alleles: 01:01, 02:01, 02:06, ...
Prediction:
    based on the averaged posterior probabilities
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 26
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:45)	0%
Predicting (2023-01-29 00:48:45)	100%
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 26
# of unique HLA alleles: 11
# of unique HLA genotypes: 13
Posterior probability:
  [0,0.25) [0.25,0.5) [0.5,0.75)   [0.75,1] 
  2 (7.7%)   2 (7.7%)  5 (19.2%) 17 (65.4%) 
Matching proportion of SNP haplotype:
    Min.  1st Qu.   Median     Mean  3rd Qu.     Max. 
0.000000 0.002097 0.007711 0.032476 0.025017 0.475952 
Open &#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/HIBAG/extdata/HapMap_CEU.bed&#x27; (the individual-major mode)
Open &#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/HIBAG/extdata/HapMap_CEU.fam&#x27;
Open &#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/HIBAG/extdata/HapMap_CEU.bim&#x27;
Import 3932 SNPs within the xMHC region on chromosome 6
SNP genotypes: 
    90 samples X 3932 SNPs
    SNPs range from 28694391bp to 33426848bp on hg19
Missing rate per SNP:
    min: 0, max: 0.1, mean: 0.0888861, median: 0.1, sd: 0.0298489
Missing rate per sample:
    min: 0, max: 0.869786, mean: 0.0888861, median: 0.00101729, sd: 0.261554
Minor allele frequency:
    min: 0, max: 0.5, mean: 0.210453, median: 0.191358, sd: 0.155144
Allelic information:
 A/G  C/T  G/T  A/C  C/G  A/T 
1567 1510  348  332  111   64 
Open &#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/HIBAG/extdata/HapMap_CEU.bed&#x27; (the individual-major mode)
Open &#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/HIBAG/extdata/HapMap_CEU.fam&#x27;
Open &#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/HIBAG/extdata/HapMap_CEU.bim&#x27;
Import 5316 SNPs from chromosome 6
SNP genotypes: 
    90 samples X 5316 SNPs
    SNPs range from 25651262bp to 33426848bp on hg19
Missing rate per SNP:
    min: 0, max: 0.1, mean: 0.0882054, median: 0.1, sd: 0.030674
Missing rate per sample:
    min: 0, max: 0.863619, mean: 0.0882054, median: 0.00131678, sd: 0.259735
Minor allele frequency:
    min: 0, max: 0.5, mean: 0.201867, median: 0.179012, sd: 0.155475
Allelic information:
 A/G  C/T  G/T  A/C  C/G  A/T 
2102 2046  480  471  134   83 
Build a HIBAG model with 2 individual classifiers:
    MAF threshold: NaN
    excluding 1 monomorphic SNP
    # of SNPs randomly sampled as candidates for each selection: 9
    # of SNPs: 77
    # of samples: 60
    # of unique HLA alleles: 12
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:45
=== building individual classifier 1, out-of-bag (25/41.7%) ===
[1] 2023-01-29 00:48:45, oob acc: 98.00%, # of SNPs: 13, # of haplo: 20
=== building individual classifier 2, out-of-bag (22/36.7%) ===
[2] 2023-01-29 00:48:45, oob acc: 90.91%, # of SNPs: 15, # of haplo: 21
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
7.678603e-08 2.465759e-05 2.458848e-04 5.196909e-03 1.419916e-02 2.996040e-02 
        Max.         Mean           SD 
4.735980e-01 4.413724e-02 1.070518e-01 
Accuracy with training data: 95.00%
Out-of-bag accuracy: 94.45%
Gene: HLA-DQB1
Training dataset: 60 samples X 77 SNPs
    # of HLA alleles: 12
    # of individual classifiers: 2
    total # of SNPs used: 20
    avg. # of SNPs in an individual classifier: 14.00
        (sd: 1.41, min: 13, max: 15, median: 14.00)
    avg. # of haplotypes in an individual classifier: 20.50
        (sd: 0.71, min: 20, max: 21, median: 20.50)
    avg. out-of-bag accuracy: 94.45%
        (sd: 5.01%, min: 90.91%, max: 98.00%, median: 94.45%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
7.678603e-08 2.465759e-05 2.458848e-04 5.196909e-03 1.419916e-02 2.996040e-02 
        Max.         Mean           SD 
4.735980e-01 4.413724e-02 1.070518e-01 
Genome assembly: hg19
The HIBAG model:
	There are 77 SNP predictors in total.
	There are 2 individual classifiers.
Summarize the missing fractions of SNP predictors per classifier:
   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
      0       0       0       0       0       0 
Gene: HLA-C
Range: [31236526bp, 31239913bp] on hg19
# of samples: 60
# of unique HLA alleles: 17
# of unique HLA genotypes: 35
Gene: HLA-C
Range: [31236526bp, 31239913bp] on hg19
# of samples: 100
# of unique HLA alleles: 17
# of unique HLA genotypes: 35
Gene: HLA-C
Range: [31236526bp, 31239913bp] on hg19
# of samples: 100
# of unique HLA alleles: 0
# of unique HLA genotypes: 0
Gene: HLA-C
Range: [31236526bp, 31239913bp] on hg19
# of samples: 200
# of unique HLA alleles: 17
# of unique HLA genotypes: 35
Build a HIBAG model with 1 individual classifier:
    MAF threshold: NaN
    excluding 9 monomorphic SNPs
    # of SNPs randomly sampled as candidates for each selection: 17
    # of SNPs: 266
    # of samples: 60
    # of unique HLA alleles: 14
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:45
=== building individual classifier 1, out-of-bag (23/38.3%) ===
[1] 2023-01-29 00:48:45, oob acc: 86.96%, # of SNPs: 12, # of haplo: 32
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
4.166789e-14 4.261245e-14 5.111347e-14 2.589270e-03 1.608934e-02 5.868848e-02 
        Max.         Mean           SD 
6.267394e-01 6.664806e-02 1.405453e-01 
Accuracy with training data: 94.17%
Out-of-bag accuracy: 86.96%
Build a HIBAG model with 1 individual classifier:
    MAF threshold: NaN
    excluding 9 monomorphic SNPs
    # of SNPs randomly sampled as candidates for each selection: 17
    # of SNPs: 266
    # of samples: 60
    # of unique HLA alleles: 14
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:45
=== building individual classifier 1, out-of-bag (24/40.0%) ===
[1] 2023-01-29 00:48:45, oob acc: 87.50%, # of SNPs: 15, # of haplo: 40
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
7.894066e-24 9.219565e-20 9.218854e-19 2.189685e-03 7.704546e-03 2.406258e-02 
        Max.         Mean           SD 
2.755151e-01 2.949891e-02 6.162169e-02 
Accuracy with training data: 95.00%
Out-of-bag accuracy: 87.50%
Gene: HLA-A
Training dataset: 60 samples X 266 SNPs
    # of HLA alleles: 14
    # of individual classifiers: 2
    total # of SNPs used: 24
    avg. # of SNPs in an individual classifier: 13.50
        (sd: 2.12, min: 12, max: 15, median: 13.50)
    avg. # of haplotypes in an individual classifier: 36.00
        (sd: 5.66, min: 32, max: 40, median: 36.00)
    avg. out-of-bag accuracy: 87.23%
        (sd: 0.38%, min: 86.96%, max: 87.50%, median: 87.23%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
9.233104e-13 5.204084e-10 5.195775e-09 2.309655e-03 1.448839e-02 3.746431e-02 
        Max.         Mean           SD 
4.511273e-01 4.807348e-02 1.006148e-01 
Genome assembly: hg19
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 34
# of unique HLA alleles: 14
# of unique HLA genotypes: 23
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 26
# of unique HLA alleles: 12
# of unique HLA genotypes: 14
Build a HIBAG model with 4 individual classifiers:
    MAF threshold: NaN
    excluding 11 monomorphic SNPs
    # of SNPs randomly sampled as candidates for each selection: 17
    # of SNPs: 264
    # of samples: 34
    # of unique HLA alleles: 14
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:45
=== building individual classifier 1, out-of-bag (11/32.4%) ===
     1, SNP: 211, loss: 196.4, oob acc: 54.55%, # of haplo: 13
     2, SNP: 66, loss: 173.548, oob acc: 63.64%, # of haplo: 13
     3, SNP: 177, loss: 136.352, oob acc: 68.18%, # of haplo: 13
     4, SNP: 108, loss: 95.8359, oob acc: 72.73%, # of haplo: 13
     5, SNP: 127, loss: 67.3216, oob acc: 77.27%, # of haplo: 13
     6, SNP: 95, loss: 47.5888, oob acc: 77.27%, # of haplo: 13
     7, SNP: 33, loss: 37.2631, oob acc: 77.27%, # of haplo: 16
     8, SNP: 6, loss: 29.7419, oob acc: 77.27%, # of haplo: 18
     9, SNP: 208, loss: 25.6913, oob acc: 77.27%, # of haplo: 19
    10, SNP: 225, loss: 25.3087, oob acc: 77.27%, # of haplo: 21
    11, SNP: 11, loss: 24.8356, oob acc: 77.27%, # of haplo: 23
    12, SNP: 151, loss: 19.4134, oob acc: 77.27%, # of haplo: 23
    13, SNP: 199, loss: 17.011, oob acc: 77.27%, # of haplo: 23
[1] 2023-01-29 00:48:45, oob acc: 77.27%, # of SNPs: 13, # of haplo: 23
=== building individual classifier 2, out-of-bag (13/38.2%) ===
     1, SNP: 160, loss: 221.236, oob acc: 76.92%, # of haplo: 17
     2, SNP: 145, loss: 173.538, oob acc: 80.77%, # of haplo: 23
     3, SNP: 177, loss: 128.58, oob acc: 84.62%, # of haplo: 31
     4, SNP: 111, loss: 79.6877, oob acc: 84.62%, # of haplo: 31
     5, SNP: 207, loss: 52.5557, oob acc: 88.46%, # of haplo: 32
     6, SNP: 245, loss: 41.8731, oob acc: 88.46%, # of haplo: 34
     7, SNP: 230, loss: 31.7937, oob acc: 88.46%, # of haplo: 38
     8, SNP: 151, loss: 20.4566, oob acc: 88.46%, # of haplo: 36
     9, SNP: 14, loss: 19.5805, oob acc: 88.46%, # of haplo: 42
    10, SNP: 129, loss: 19.5104, oob acc: 88.46%, # of haplo: 44
    11, SNP: 62, loss: 19.4853, oob acc: 88.46%, # of haplo: 48
[2] 2023-01-29 00:48:45, oob acc: 88.46%, # of SNPs: 11, # of haplo: 48
=== building individual classifier 3, out-of-bag (14/41.2%) ===
     1, SNP: 191, loss: 193.067, oob acc: 57.14%, # of haplo: 11
     2, SNP: 264, loss: 150.427, oob acc: 64.29%, # of haplo: 12
     3, SNP: 132, loss: 93.4067, oob acc: 67.86%, # of haplo: 12
     4, SNP: 128, loss: 39.8353, oob acc: 71.43%, # of haplo: 12
     5, SNP: 160, loss: 28.2998, oob acc: 75.00%, # of haplo: 12
     6, SNP: 144, loss: 13.635, oob acc: 75.00%, # of haplo: 12
     7, SNP: 111, loss: 6.04609, oob acc: 75.00%, # of haplo: 12
     8, SNP: 40, loss: 6.04583, oob acc: 82.14%, # of haplo: 14
     9, SNP: 141, loss: 6.04583, oob acc: 85.71%, # of haplo: 14
    10, SNP: 73, loss: 2.9038, oob acc: 85.71%, # of haplo: 14
    11, SNP: 199, loss: 2.20025, oob acc: 85.71%, # of haplo: 14
[3] 2023-01-29 00:48:45, oob acc: 85.71%, # of SNPs: 11, # of haplo: 14
=== building individual classifier 4, out-of-bag (10/29.4%) ===
     1, SNP: 147, loss: 158.631, oob acc: 50.00%, # of haplo: 12
     2, SNP: 152, loss: 140.375, oob acc: 55.00%, # of haplo: 13
     3, SNP: 78, loss: 115.887, oob acc: 60.00%, # of haplo: 16
     4, SNP: 115, loss: 77.8082, oob acc: 60.00%, # of haplo: 18
     5, SNP: 148, loss: 62.6831, oob acc: 65.00%, # of haplo: 18
     6, SNP: 13, loss: 46.5657, oob acc: 75.00%, # of haplo: 20
     7, SNP: 109, loss: 31.0312, oob acc: 75.00%, # of haplo: 20
     8, SNP: 176, loss: 22.5073, oob acc: 75.00%, # of haplo: 21
     9, SNP: 145, loss: 20.9122, oob acc: 75.00%, # of haplo: 21
    10, SNP: 128, loss: 20.6728, oob acc: 75.00%, # of haplo: 21
    11, SNP: 73, loss: 14.6217, oob acc: 75.00%, # of haplo: 22
    12, SNP: 151, loss: 10.2879, oob acc: 75.00%, # of haplo: 23
    13, SNP: 199, loss: 8.74645, oob acc: 75.00%, # of haplo: 23
[4] 2023-01-29 00:48:45, oob acc: 75.00%, # of SNPs: 13, # of haplo: 23
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0002440856 0.0002467397 0.0002706268 0.0043752100 0.0092453327 0.0260848720 
        Max.         Mean           SD 
0.4759517521 0.0445726720 0.1064934602 
Accuracy with training data: 97.06%
Out-of-bag accuracy: 81.61%
Gene: HLA-A
Training dataset: 34 samples X 264 SNPs
    # of HLA alleles: 14
    # of individual classifiers: 4
    total # of SNPs used: 38
    avg. # of SNPs in an individual classifier: 12.00
        (sd: 1.15, min: 11, max: 13, median: 12.00)
    avg. # of haplotypes in an individual classifier: 27.00
        (sd: 14.63, min: 14, max: 48, median: 23.00)
    avg. out-of-bag accuracy: 81.61%
        (sd: 6.49%, min: 75.00%, max: 88.46%, median: 81.49%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0002440856 0.0002467397 0.0002706268 0.0043752100 0.0092453327 0.0260848720 
        Max.         Mean           SD 
0.4759517521 0.0445726720 0.1064934602 
Genome assembly: hg19
HIBAG model for HLA-A:
    4 individual classifiers
    264 SNPs
    14 unique HLA alleles: 01:01, 02:01, 02:06, ...
Prediction:
    based on the averaged posterior probabilities
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 26
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:45)	0%
Predicting (2023-01-29 00:48:45)	100%
Allelic ambiguity: 01:01, 02:02
Allelic ambiguity: 01:01, 02:02
Allelic ambiguity: 09:01
Allelic ambiguity: 09:01
Allelic ambiguity: 05:01, 06:01
Allelic ambiguity: 05:01, 06:01
Allelic ambiguity: 01:01, 03:01, 26:01, 02:01, 23:01, 02:06, 11:01, 25:01, 31:01, 24:02, 29:02, 68:01, 24:03, 32:01
Pos Num   *   -   A   D   E   F   G   H   I   K   L   M   N   Q   R   S   T   V   W   Y 
  1 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
  9 120   .  81   .   .   .   .   .   .   .   .   .   .   .   .   .  15   7   .   .  17 
 44 120   .  25   .   .   .   .   .   .   .   .   .   .   .   .  95   .   .   .   .   . 
 56 120   . 117   .   .   .   .   .   .   .   .   .   .   .   .   3   .   .   .   .   . 
 62 120   .  46   .   .  15   .  44   .   .   .   4   .   .   .  11   .   .   .   .   . 
 63 120   . 105   .   .   .   .   .   .   .   .   .   .  11   4   .   .   .   .   .   . 
 65 120   . 105   .   .   .   .  15   .   .   .   .   .   .   .   .   .   .   .   .   . 
 66 120   .  61   .   .   .   .   .   .   .  59   .   .   .   .   .   .   .   .   .   . 
 67 120   .  25   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .  95   .   . 
 70 120   .  99   .   .   .   .   .   .   .   .   .   .   .  21   .   .   .   .   .   . 
 73 120   . 117   .   .   .   .   .   .   3   .   .   .   .   .   .   .   .   .   .   . 
 74 120   .  76   .   .   .   .   .  44   .   .   .   .   .   .   .   .   .   .   .   . 
 76 120   .  32   .   .  24   .   .   .   .   .   .   .   .   .   .   .   .  64   .   . 
 77 120   .  47   .  64   .   .   .   .   .   .   .   .   .   .   .   9   .   .   .   . 
 79 120   .  96   .   .   .   .   .   .   .   .   .   .   .   .  24   .   .   .   .   . 
 80 120   .  96   .   .   .   .   .   .  24   .   .   .   .   .   .   .   .   .   .   . 
 81 120   .  96  24   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 82 120   .  96   .   .   .   .   .   .   .   .  24   .   .   .   .   .   .   .   .   . 
 83 120   .  96   .   .   .   .   .   .   .   .   .   .   .   .  24   .   .   .   .   . 
 90 120   .  38  82   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 95 120   .  61   .   .   .   .   .   .   .   .  15   .   .   .   .   .   .  44   .   . 
 97 120   .  39   .   .   .   .   .   .   .   .   .  29   .   .  52   .   .   .   .   . 
 99 120   . 105   .   .   .  15   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
105 120   .  42   .   .   .   .   .   .   .   .   .   .   .   .   .  78   .   .   .   . 
107 120   .  76   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .  44   . 
109 120   . 116   .   .   .   .   .   .   .   .   4   .   .   .   .   .   .   .   .   . 
114 120   .  46   .   .   .   .   .  59   .   .   .   .   .  15   .   .   .   .   .   . 
116 120   .  61   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .  59 
127 120   .  58   .   .   .   .   .   .   .  62   .   .   .   .   .   .   .   .   .   . 
142 120   .  73   .   .   .   .   .   .   .   .   .   .   .   .   .   .  47   .   .   . 
144 120   .  98   .   .   .   .   .   .   .   .   .   .   .  22   .   .   .   .   .   . 
145 120   .  73   .   .   .   .   .  47   .   .   .   .   .   .   .   .   .   .   .   . 
149 120   . 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   8   .   .   . 
150 120   .  25  95   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
151 120   . 106   .   .   .   .   .   .   .   .   .   .   .   .  14   .   .   .   .   . 
152 120   .  30   .   .  17   .   .   .   .   .   .   .   .   .   .   .   .  73   .   . 
156 120   .  25   .   .   .   .   .   .   .   .  67   .   .  17   .   .   .   .  11   . 
158 120   .  25  95   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
161 120   . 111   .   9   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
163 120   .  38   .   .   .   .   .   .   .   .   .   .   .   .   .   .  82   .   .   . 
166 120   .  39   .   .  81   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
167 120   .  39   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .  81   . 
183 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
Allelic ambiguity: 01:01, 03:01, 26:01, 02:01, 23:01, 02:06, 11:01, 25:01, 31:01, 24:02, 29:02, 68:01, 24:03, 32:01
Pos Num   *   -   A   D   E   F   G   H   I   K   L   M   N   Q   R   S   T   V   W   Y 
-23 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-22 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-21 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-20 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-19 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-18 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-17 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-16 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-15 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-14 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-13 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-12 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-11 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-10 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 -9 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 -8 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 -7 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 -6 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 -5 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 -4 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 -3 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 -2 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 -1 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
  . 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
  1 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
  9 120   .  81   .   .   .   .   .   .   .   .   .   .   .   .   .  15   7   .   .  17 
 44 120   .  25   .   .   .   .   .   .   .   .   .   .   .   .  95   .   .   .   .   . 
 56 120   . 117   .   .   .   .   .   .   .   .   .   .   .   .   3   .   .   .   .   . 
 62 120   .  46   .   .  15   .  44   .   .   .   4   .   .   .  11   .   .   .   .   . 
 63 120   . 105   .   .   .   .   .   .   .   .   .   .  11   4   .   .   .   .   .   . 
 65 120   . 105   .   .   .   .  15   .   .   .   .   .   .   .   .   .   .   .   .   . 
 66 120   .  61   .   .   .   .   .   .   .  59   .   .   .   .   .   .   .   .   .   . 
 67 120   .  25   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .  95   .   . 
 70 120   .  99   .   .   .   .   .   .   .   .   .   .   .  21   .   .   .   .   .   . 
 73 120   . 117   .   .   .   .   .   .   3   .   .   .   .   .   .   .   .   .   .   . 
 74 120   .  76   .   .   .   .   .  44   .   .   .   .   .   .   .   .   .   .   .   . 
 76 120   .  32   .   .  24   .   .   .   .   .   .   .   .   .   .   .   .  64   .   . 
 77 120   .  47   .  64   .   .   .   .   .   .   .   .   .   .   .   9   .   .   .   . 
 79 120   .  96   .   .   .   .   .   .   .   .   .   .   .   .  24   .   .   .   .   . 
 80 120   .  96   .   .   .   .   .   .  24   .   .   .   .   .   .   .   .   .   .   . 
 81 120   .  96  24   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 82 120   .  96   .   .   .   .   .   .   .   .  24   .   .   .   .   .   .   .   .   . 
 83 120   .  96   .   .   .   .   .   .   .   .   .   .   .   .  24   .   .   .   .   . 
 90 120   .  38  82   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 95 120   .  61   .   .   .   .   .   .   .   .  15   .   .   .   .   .   .  44   .   . 
 97 120   .  39   .   .   .   .   .   .   .   .   .  29   .   .  52   .   .   .   .   . 
 99 120   . 105   .   .   .  15   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
105 120   .  42   .   .   .   .   .   .   .   .   .   .   .   .   .  78   .   .   .   . 
107 120   .  76   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .  44   . 
109 120   . 116   .   .   .   .   .   .   .   .   4   .   .   .   .   .   .   .   .   . 
114 120   .  46   .   .   .   .   .  59   .   .   .   .   .  15   .   .   .   .   .   . 
116 120   .  61   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .  59 
127 120   .  58   .   .   .   .   .   .   .  62   .   .   .   .   .   .   .   .   .   . 
142 120   .  73   .   .   .   .   .   .   .   .   .   .   .   .   .   .  47   .   .   . 
144 120   .  98   .   .   .   .   .   .   .   .   .   .   .  22   .   .   .   .   .   . 
145 120   .  73   .   .   .   .   .  47   .   .   .   .   .   .   .   .   .   .   .   . 
149 120   . 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   8   .   .   . 
150 120   .  25  95   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
151 120   . 106   .   .   .   .   .   .   .   .   .   .   .   .  14   .   .   .   .   . 
152 120   .  30   .   .  17   .   .   .   .   .   .   .   .   .   .   .   .  73   .   . 
156 120   .  25   .   .   .   .   .   .   .   .  67   .   .  17   .   .   .   .  11   . 
158 120   .  25  95   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
161 120   . 111   .   9   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
163 120   .  38   .   .   .   .   .   .   .   .   .   .   .   .   .   .  82   .   .   . 
166 120   .  39   .   .  81   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
167 120   .  39   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .  81   . 
183 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
184 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
185 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
186 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
187 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
188 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
189 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
190 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
191 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
192 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
193 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
194 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
195 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
196 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
197 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
198 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
199 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
200 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
201 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
202 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
203 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
204 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
205 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
206 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
207 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
208 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
209 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
210 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
211 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
212 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
213 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
214 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
215 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
216 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
217 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
218 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
219 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
220 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
221 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
222 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
223 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
224 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
225 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
226 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
227 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
228 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
229 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
230 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
231 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
232 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
233 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
234 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
235 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
236 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
237 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
238 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
239 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
240 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
241 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
242 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
243 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
244 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
245 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
246 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
247 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
248 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
249 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
250 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
251 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
252 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
253 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
254 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
255 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
256 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
257 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
258 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
259 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
260 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
261 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
262 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
263 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
264 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
265 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
266 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
267 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
268 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
269 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
270 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
271 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
272 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
273 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
274 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
275 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
276 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
277 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
278 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
279 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
280 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
281 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
282 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
283 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
284 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
285 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
286 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
287 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
288 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
289 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
290 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
291 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
292 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
293 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
294 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
295 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
296 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
297 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
298 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
299 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
300 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
301 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
302 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
303 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
304 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
305 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
306 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
307 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
308 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
309 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
310 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
311 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
312 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
313 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
314 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
315 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
316 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
317 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
318 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
319 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
320 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
321 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
322 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
323 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
324 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
325 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
326 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
327 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
328 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
329 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
330 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
331 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
332 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
333 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
334 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
335 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
336 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
337 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
338 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
339 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
340 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
341 120 120   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
Allelic ambiguity: 03:02, 06:02, 05:01, 02:01, 05:03, 03:03, 03:01, 04:02, 06:03, 06:04, 05:02
Pos Num   *   -   A   D   E   F   G   I   K   L   M   N   P   Q   R   S   T   Y 
  5 120 112   .   .   .   .   .   .   .   .   .   .   8   .   .   .   .   .   . 
  6 120  20  92   8   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
  7 112  20  92   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
  8 112  20  92   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
  9 112   3  76   .   .   .  33   .   .   .   .   .   .   .   .   .   .   .   . 
 10 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 11 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 12 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 13 112   3  93  16   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 14 112   3  14   .   .   .   .   .   .   .   .  95   .   .   .   .   .   .   . 
 15 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 16 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 17 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 18 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 19 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 20 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 26 112   .  20   .   .   .   .   .   .   .  76   .   .   .   .   .   .   .  16 
 28 112   . 100   .   .   .   .   .   .   .   .   .   .   .   .   .  12   .   . 
 30 112   .  24   .   .   .   .   .   .   .   .   .   .   .   .   .  12   .  76 
 37 112   . 100   .   .   .   .   .  12   .   .   .   .   .   .   .   .   .   . 
 38 112   .  29  83   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 45 112   .  96   .   .  16   .   .   .   .   .   .   .   .   .   .   .   .   . 
 46 112   . 100   .   .  12   .   .   .   .   .   .   .   .   .   .   .   .   . 
 47 112   . 100   .   .   .  12   .   .   .   .   .   .   .   .   .   .   .   . 
 52 112   . 100   .   .   .   .   .   .   .  12   .   .   .   .   .   .   .   . 
 53 112   .  54   .   .   .   .   .   .   .  58   .   .   .   .   .   .   .   . 
 55 112   .  57   .   .   .   .   .   .   .  12   .   .  43   .   .   .   .   . 
 56 112   . 109   .   .   .   .   .   .   .   3   .   .   .   .   .   .   .   . 
 57 112   .  14  33  64   .   .   .   .   .   .   .   .   .   .   .   1   .   . 
 66 112   .  97   .  15   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 67 112   .  97   .   .   .   .   .  15   .   .   .   .   .   .   .   .   .   . 
 70 112   3  50   .   .   3   .   .   .   .   .   .   .   .   .  56   .   .   . 
 71 112   3  14   .   3   .   .   .   .  12   .   .   .   .   .   .   .  80   . 
 72 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 73 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 74 112   3  17  12   .  80   .   .   .   .   .   .   .   .   .   .   .   .   . 
 75 112   3  29   .   .   .   .   .   .   .  80   .   .   .   .   .   .   .   . 
 76 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 77 112   3  26   .   .   .   .   .   .   .   .   .   .   .   .   .   .  83   . 
 78 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 79 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 80 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 81 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 82 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 83 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 84 112   3  51   .   .   .   .   .   .   .   .   .   .   .  58   .   .   .   . 
 85 112   3  51   .   .   .   .   .   .   .  58   .   .   .   .   .   .   .   . 
 86 112   3  50   .   .  58   .   1   .   .   .   .   .   .   .   .   .   .   . 
 87 112   3  15   .   .   .  36   .   .   .  58   .   .   .   .   .   .   .   . 
 88 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 89 112   3  51   .   .   .   .   .   .   .   .   .   .   .   .   .   .  58   . 
 90 112   3  51   .   .   .   .   .   .   .   .   .   .   .   .   .   .  58   . 
 91 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 92 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 93 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 94 112  17  95   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 95 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
Allelic ambiguity: 03:02, 06:02, 05:01, 02:01, 05:03, 03:03, 03:01, 04:02, 06:03, 06:04, 05:02
Pos Num   *   -   A   D   E   F   G   I   K   L   M   N   P   Q   R   S   T   Y 
-31 120 112   .   .   .   .   .   .   .   .   .   .   8   .   .   .   .   .   . 
-30 120 112   .   8   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-29 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-28 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-27 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-26 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-25 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-24 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-23 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-22 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-21 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-20 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-19 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-18 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-17 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-16 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-15 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-14 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-13 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-12 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-11 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
-10 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 -9 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 -8 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 -7 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 -6 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 -5 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 -4 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 -3 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 -2 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 -1 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
  . 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
  1 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
  2 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
  3 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
  4 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
  5 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
  6 112  20  92   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
  7 112  20  92   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
  8 112  20  92   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
  9 112   3  76   .   .   .  33   .   .   .   .   .   .   .   .   .   .   .   . 
 10 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 11 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 12 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 13 112   3  93  16   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 14 112   3  14   .   .   .   .   .   .   .   .  95   .   .   .   .   .   .   . 
 15 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 16 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 17 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 18 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 19 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 20 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 26 112   .  20   .   .   .   .   .   .   .  76   .   .   .   .   .   .   .  16 
 28 112   . 100   .   .   .   .   .   .   .   .   .   .   .   .   .  12   .   . 
 30 112   .  24   .   .   .   .   .   .   .   .   .   .   .   .   .  12   .  76 
 37 112   . 100   .   .   .   .   .  12   .   .   .   .   .   .   .   .   .   . 
 38 112   .  29  83   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 45 112   .  96   .   .  16   .   .   .   .   .   .   .   .   .   .   .   .   . 
 46 112   . 100   .   .  12   .   .   .   .   .   .   .   .   .   .   .   .   . 
 47 112   . 100   .   .   .  12   .   .   .   .   .   .   .   .   .   .   .   . 
 52 112   . 100   .   .   .   .   .   .   .  12   .   .   .   .   .   .   .   . 
 53 112   .  54   .   .   .   .   .   .   .  58   .   .   .   .   .   .   .   . 
 55 112   .  57   .   .   .   .   .   .   .  12   .   .  43   .   .   .   .   . 
 56 112   . 109   .   .   .   .   .   .   .   3   .   .   .   .   .   .   .   . 
 57 112   .  14  33  64   .   .   .   .   .   .   .   .   .   .   .   1   .   . 
 66 112   .  97   .  15   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 67 112   .  97   .   .   .   .   .  15   .   .   .   .   .   .   .   .   .   . 
 70 112   3  50   .   .   3   .   .   .   .   .   .   .   .   .  56   .   .   . 
 71 112   3  14   .   3   .   .   .   .  12   .   .   .   .   .   .   .  80   . 
 72 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 73 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 74 112   3  17  12   .  80   .   .   .   .   .   .   .   .   .   .   .   .   . 
 75 112   3  29   .   .   .   .   .   .   .  80   .   .   .   .   .   .   .   . 
 76 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 77 112   3  26   .   .   .   .   .   .   .   .   .   .   .   .   .   .  83   . 
 78 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 79 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 80 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 81 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 82 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 83 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 84 112   3  51   .   .   .   .   .   .   .   .   .   .   .  58   .   .   .   . 
 85 112   3  51   .   .   .   .   .   .   .  58   .   .   .   .   .   .   .   . 
 86 112   3  50   .   .  58   .   1   .   .   .   .   .   .   .   .   .   .   . 
 87 112   3  15   .   .   .  36   .   .   .  58   .   .   .   .   .   .   .   . 
 88 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 89 112   3  51   .   .   .   .   .   .   .   .   .   .   .   .   .   .  58   . 
 90 112   3  51   .   .   .   .   .   .   .   .   .   .   .   .   .   .  58   . 
 91 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 92 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 93 112   3 109   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 94 112  17  95   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 95 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 96 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 97 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 98 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
 99 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
100 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
101 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
102 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
103 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
104 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
105 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
106 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
107 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
108 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
109 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
110 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
111 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
112 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
113 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
114 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
115 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
116 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
117 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
118 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
119 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
120 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
121 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
122 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
123 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
124 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
125 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
126 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
127 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
128 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
129 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
130 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
131 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
132 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
133 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
134 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
135 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
136 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
137 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
138 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
139 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
140 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
141 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
142 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
143 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
144 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
145 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
146 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
147 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
148 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
149 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
150 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
151 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
152 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
153 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
154 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
155 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
156 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
157 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
158 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
159 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
160 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
161 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
162 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
163 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
164 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
165 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
166 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
167 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
168 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
169 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
170 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
171 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
172 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
173 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
174 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
175 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
176 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
177 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
178 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
179 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
180 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
181 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
182 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
183 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
184 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
185 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
186 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
187 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
188 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
189 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
190 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
191 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
192 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
193 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
194 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
195 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
196 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
197 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
198 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
199 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
200 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
201 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
202 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
203 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
204 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
205 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
206 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
207 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
208 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
209 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
210 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
211 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
212 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
213 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
214 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
215 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
216 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
217 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
218 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
219 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
220 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
221 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
222 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
223 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
224 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
225 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
226 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
227 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
228 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
229 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
230 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
231 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
232 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
233 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
234 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
235 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
236 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
237 112 112   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   . 
using the default genome assembly (assembly=&quot;hg19&quot;)
SNP genotypes: 
    60 samples X 275 SNPs
    SNPs range from 29417816bp to 30410205bp on hg19
Missing rate per SNP:
    min: 0, max: 0.0666667, mean: 0.0652727, median: 0.0666667, sd: 0.00939558
Missing rate per sample:
    min: 0, max: 0.974545, mean: 0.0652727, median: 0, sd: 0.245066
Minor allele frequency:
    min: 0, max: 0.491071, mean: 0.215181, median: 0.1875, sd: 0.139271
Allelic information:
C/T A/G G/T A/C 
125  97  32  21 
Build a HIBAG model with 10 individual classifiers:
    MAF threshold: NaN
    excluding 9 monomorphic SNPs
    # of SNPs randomly sampled as candidates for each selection: 17
    # of SNPs: 266
    # of samples: 60
    # of unique HLA alleles: 14
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:48
=== building individual classifier 1, out-of-bag (23/38.3%) ===
[1] 2023-01-29 00:48:49, oob acc: 86.96%, # of SNPs: 12, # of haplo: 32
=== building individual classifier 2, out-of-bag (24/40.0%) ===
[2] 2023-01-29 00:48:49, oob acc: 87.50%, # of SNPs: 15, # of haplo: 40
=== building individual classifier 3, out-of-bag (24/40.0%) ===
[3] 2023-01-29 00:48:49, oob acc: 97.92%, # of SNPs: 14, # of haplo: 21
=== building individual classifier 4, out-of-bag (22/36.7%) ===
[4] 2023-01-29 00:48:49, oob acc: 95.45%, # of SNPs: 14, # of haplo: 25
=== building individual classifier 5, out-of-bag (19/31.7%) ===
[5] 2023-01-29 00:48:49, oob acc: 78.95%, # of SNPs: 14, # of haplo: 21
=== building individual classifier 6, out-of-bag (24/40.0%) ===
[6] 2023-01-29 00:48:49, oob acc: 93.75%, # of SNPs: 16, # of haplo: 22
=== building individual classifier 7, out-of-bag (24/40.0%) ===
[7] 2023-01-29 00:48:49, oob acc: 93.75%, # of SNPs: 24, # of haplo: 81
=== building individual classifier 8, out-of-bag (21/35.0%) ===
[8] 2023-01-29 00:48:49, oob acc: 92.86%, # of SNPs: 20, # of haplo: 45
=== building individual classifier 9, out-of-bag (19/31.7%) ===
[9] 2023-01-29 00:48:50, oob acc: 94.74%, # of SNPs: 16, # of haplo: 45
=== building individual classifier 10, out-of-bag (19/31.7%) ===
[10] 2023-01-29 00:48:50, oob acc: 97.37%, # of SNPs: 15, # of haplo: 40
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0001493079 0.0001617044 0.0002732730 0.0039571951 0.0150509810 0.0326242837 
        Max.         Mean           SD 
0.3657388922 0.0410332850 0.0799788450 
Accuracy with training data: 98.33%
Out-of-bag accuracy: 91.92%
Gene: HLA-A
Training dataset: 60 samples X 266 SNPs
    # of HLA alleles: 14
    # of individual classifiers: 10
    total # of SNPs used: 95
    avg. # of SNPs in an individual classifier: 16.00
        (sd: 3.50, min: 12, max: 24, median: 15.00)
    avg. # of haplotypes in an individual classifier: 37.20
        (sd: 18.22, min: 21, max: 81, median: 36.00)
    avg. out-of-bag accuracy: 91.92%
        (sd: 5.83%, min: 78.95%, max: 97.92%, median: 93.75%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0001493079 0.0001617044 0.0002732730 0.0039571951 0.0150509810 0.0326242837 
        Max.         Mean           SD 
0.3657388922 0.0410332850 0.0799788450 
Genome assembly: hg19
SNP genotypes: 
    60 samples X 275 SNPs
    SNPs range from 29417816bp to 30410205bp on hg19
Missing rate per SNP:
    min: 0, max: 0.0666667, mean: 0.0652727, median: 0.0666667, sd: 0.00939558
Missing rate per sample:
    min: 0, max: 0.974545, mean: 0.0652727, median: 0, sd: 0.245066
Minor allele frequency:
    min: 0, max: 0.491071, mean: 0.215181, median: 0.1875, sd: 0.139271
Allelic information:
C/T A/G G/T A/C 
125  97  32  21 
using the default genome assembly (assembly=&quot;hg19&quot;)
SNP genotypes: 
    60 samples X 275 SNPs
    SNPs range from 29417816bp to 30410205bp on hg19
Missing rate per SNP:
    min: 0, max: 0.0666667, mean: 0.0652727, median: 0.0666667, sd: 0.00939558
Missing rate per sample:
    min: 0, max: 0.974545, mean: 0.0652727, median: 0, sd: 0.245066
Minor allele frequency:
    min: 0, max: 0.491071, mean: 0.215181, median: 0.1875, sd: 0.139271
Allelic information:
C/T A/G G/T A/C 
125  97  32  21 
Loading required namespace: gdsfmt
Loading required namespace: SNPRelate
Open &#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/HIBAG/extdata/HapMap_CEU_Chr6.gds&#x27;
Import 1668 SNPs within the xMHC region on chromosome 6
2 SNPs with invalid alleles have been removed.
SNP genotypes: 
    165 samples X 1666 SNPs
    SNPs range from 28837960bp to 33524089bp on hg18
Missing rate per SNP:
    min: 0, max: 0.0484848, mean: 0.00175707, median: 0, sd: 0.00515153
Missing rate per sample:
    min: 0, max: 0.0120048, mean: 0.00175707, median: 0.00120048, sd: 0.00210091
Minor allele frequency:
    min: 0, max: 0.5, mean: 0.19767, median: 0.175758, sd: 0.150469
Allelic information:
A/G T/C G/A C/T T/G A/C C/A G/T A/T C/G G/C T/A 
412 318 299 285  79  76  75  56  20  19  16  11 
SNP genotypes: 
    60 samples X 1564 SNPs
    SNPs range from 25769023bp to 33421576bp on hg19
Missing rate per SNP:
    min: 0, max: 0.0666667, mean: 0.0651215, median: 0.0666667, sd: 0.00968287
Missing rate per sample:
    min: 0, max: 0.969949, mean: 0.0651215, median: 0.000639386, sd: 0.243737
Minor allele frequency:
    min: 0, max: 0.5, mean: 0.227582, median: 0.205357, sd: 0.1389
Allelic information:
A/G C/T G/T A/C 
655 632 141 136 
Open &#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/HIBAG/extdata/HapMap_CEU.bed&#x27; (the individual-major mode)
Open &#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/HIBAG/extdata/HapMap_CEU.fam&#x27;
Open &#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/HIBAG/extdata/HapMap_CEU.bim&#x27;
Import 3932 SNPs within the xMHC region on chromosome 6
No allelic strand or A/B allele is flipped.
SNP genotypes: 
    150 samples X 1214 SNPs
    SNPs range from 28695148bp to 33421576bp on hg19
Missing rate per SNP:
    min: 0, max: 0.0866667, mean: 0.0844646, median: 0.0866667, sd: 0.0128841
Missing rate per sample:
    min: 0, max: 0.968699, mean: 0.0844646, median: 0.000823723, sd: 0.273119
Minor allele frequency:
    min: 0, max: 0.5, mean: 0.234168, median: 0.218978, sd: 0.137855
Allelic information:
A/G C/T G/T A/C 
505 496 109 104 
SNP genotypes: 
    60 samples X 1564 SNPs
    SNPs range from 25769023bp to 33421576bp on hg19
Missing rate per SNP:
    min: 0, max: 0.0666667, mean: 0.0651215, median: 0.0666667, sd: 0.00968287
Missing rate per sample:
    min: 0, max: 0.969949, mean: 0.0651215, median: 0.000639386, sd: 0.243737
Minor allele frequency:
    min: 0, max: 0.5, mean: 0.227582, median: 0.205357, sd: 0.1389
Allelic information:
A/G C/T G/T A/C 
655 632 141 136 
SNP genotypes: 
    60 samples X 1564 SNPs
    SNPs range from 25769023bp to 33421576bp on hg19
Missing rate per SNP:
    min: 0, max: 0.0666667, mean: 0.0651215, median: 0.0666667, sd: 0.00968287
Missing rate per sample:
    min: 0, max: 0.969949, mean: 0.0651215, median: 0.000639386, sd: 0.243737
Minor allele frequency:
    min: 0, max: 0.5, mean: 0.227582, median: 0.205357, sd: 0.1389
Allelic information:
A/G C/T G/T A/C 
655 632 141 136 
SNP genotypes: 
    60 samples X 1564 SNPs
    SNPs range from 25769023bp to 33421576bp on hg19
Missing rate per SNP:
    min: 0, max: 0.0666667, mean: 0.0651215, median: 0.0666667, sd: 0.00968287
Missing rate per sample:
    min: 0, max: 0.969949, mean: 0.0651215, median: 0.000639386, sd: 0.243737
Minor allele frequency:
    min: 0, max: 0.5, mean: 0.227582, median: 0.205357, sd: 0.1389
Allelic information:
A/G C/T G/T A/C 
655 632 141 136 
SNP genotypes: 
    60 samples X 1564 SNPs
    SNPs range from 25769023bp to 33421576bp on hg19
Missing rate per SNP:
    min: 0, max: 0.0666667, mean: 0.0651215, median: 0.0666667, sd: 0.00968287
Missing rate per sample:
    min: 0, max: 0.969949, mean: 0.0651215, median: 0.000639386, sd: 0.243737
Minor allele frequency:
    min: 0, max: 0.5, mean: 0.227582, median: 0.205357, sd: 0.1389
Allelic information:
A/G C/T G/T A/C 
655 632 141 136 
SNP genotypes: 
    60 samples X 1197 SNPs
    SNPs range from 25769023bp to 33421576bp on hg19
Missing rate per SNP:
    min: 0, max: 0.0666667, mean: 0.0657059, median: 0.0666667, sd: 0.00757446
Missing rate per sample:
    min: 0, max: 0.978279, mean: 0.0657059, median: 0.000835422, sd: 0.245786
Minor allele frequency:
    min: 0.101695, max: 0.5, mean: 0.278734, median: 0.267857, sd: 0.117338
Allelic information:
A/G C/T A/C G/T 
511 476 105 105 
SNP genotypes: 
    60 samples X 1564 SNPs
    SNPs range from 25769023bp to 33421576bp on hg19
Missing rate per SNP:
    min: 0, max: 0.0666667, mean: 0.0651215, median: 0.0666667, sd: 0.00968287
Missing rate per sample:
    min: 0, max: 0.969949, mean: 0.0651215, median: 0.000639386, sd: 0.243737
Minor allele frequency:
    min: 0, max: 0.5, mean: 0.227582, median: 0.205357, sd: 0.1389
Allelic information:
A/G C/T G/T A/C 
655 632 141 136 
Open &#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/HIBAG/extdata/HapMap_CEU.bed&#x27; (the individual-major mode)
Open &#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/HIBAG/extdata/HapMap_CEU.fam&#x27;
Open &#x27;/home/biocbuild/bbs-3.17-bioc/R/site-library/HIBAG/extdata/HapMap_CEU.bim&#x27;
Import 3932 SNPs within the xMHC region on chromosome 6
SNP genotypes: 
    90 samples X 3932 SNPs
    SNPs range from 28694391bp to 33426848bp on hg19
Missing rate per SNP:
    min: 0, max: 0.1, mean: 0.0888861, median: 0.1, sd: 0.0298489
Missing rate per sample:
    min: 0, max: 0.869786, mean: 0.0888861, median: 0.00101729, sd: 0.261554
Minor allele frequency:
    min: 0, max: 0.5, mean: 0.210453, median: 0.191358, sd: 0.155144
Allelic information:
 A/G  C/T  G/T  A/C  C/G  A/T 
1567 1510  348  332  111   64 
No allelic strand or A/B allele is flipped.
SNP genotypes: 
    60 samples X 1214 SNPs
    SNPs range from 28695148bp to 33421576bp on hg19
Missing rate per SNP:
    min: 0, max: 0.0666667, mean: 0.0650879, median: 0.0666667, sd: 0.0097381
Missing rate per sample:
    min: 0, max: 0.968699, mean: 0.0650879, median: 0.000823723, sd: 0.243373
Minor allele frequency:
    min: 0, max: 0.5, mean: 0.234476, median: 0.223214, sd: 0.13833
Allelic information:
A/G C/T G/T A/C 
505 496 109 104 
using the default genome assembly (assembly=&quot;hg19&quot;)
SNP genotypes: 
    60 samples X 275 SNPs
    SNPs range from 29417816bp to 30410205bp on hg19
Missing rate per SNP:
    min: 0, max: 0.0666667, mean: 0.0652727, median: 0.0666667, sd: 0.00939558
Missing rate per sample:
    min: 0, max: 0.974545, mean: 0.0652727, median: 0, sd: 0.245066
Minor allele frequency:
    min: 0, max: 0.491071, mean: 0.215181, median: 0.1875, sd: 0.139271
Allelic information:
C/T A/G G/T A/C 
125  97  32  21 
MAF filter (&gt;=0.01), excluding 9 SNP(s)
using the default genome assembly (assembly=&quot;hg19&quot;)
SNP genotypes: 
    60 samples X 1564 SNPs
    SNPs range from 25769023bp to 33421576bp on hg19
Missing rate per SNP:
    min: 0, max: 0.0666667, mean: 0.0651215, median: 0.0666667, sd: 0.00968287
Missing rate per sample:
    min: 0, max: 0.969949, mean: 0.0651215, median: 0.000639386, sd: 0.243737
Minor allele frequency:
    min: 0, max: 0.5, mean: 0.227582, median: 0.205357, sd: 0.1389
Allelic information:
A/G C/T G/T A/C 
655 632 141 136 
SNP genotypes: 
    60 samples X 1564 SNPs
    SNPs range from 25769023bp to 33421576bp on hg19
Missing rate per SNP:
    min: 0, max: 0.0666667, mean: 0.0651215, median: 0.0666667, sd: 0.00968287
Missing rate per sample:
    min: 0, max: 0.969949, mean: 0.0651215, median: 0.000639386, sd: 0.243737
Minor allele frequency:
    min: 0, max: 0.5, mean: 0.227582, median: 0.205357, sd: 0.1389
Allelic information:
A/G C/T G/T A/C 
655 632 141 136 
Build a HIBAG model with 1 individual classifier:
    MAF threshold: NaN
    # of SNPs randomly sampled as candidates for each selection: 10
    # of SNPs: 83
    # of samples: 60
    # of unique HLA alleles: 17
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:53
=== building individual classifier 1, out-of-bag (25/41.7%) ===
[1] 2023-01-29 00:48:53, oob acc: 92.00%, # of SNPs: 24, # of haplo: 29
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
2.222247e-28 1.128571e-24 1.128371e-23 6.944660e-04 8.333349e-03 3.673611e-02 
        Max.         Mean           SD 
9.105734e-02 2.054649e-02 2.598603e-02 
Accuracy with training data: 96.67%
Out-of-bag accuracy: 92.00%
Build a HIBAG model with 1 individual classifier:
    MAF threshold: NaN
    # of SNPs randomly sampled as candidates for each selection: 10
    # of SNPs: 83
    # of samples: 60
    # of unique HLA alleles: 17
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:53
=== building individual classifier 1, out-of-bag (20/33.3%) ===
[1] 2023-01-29 00:48:53, oob acc: 97.50%, # of SNPs: 18, # of haplo: 34
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
5.014366e-13 4.671716e-10 4.667203e-09 1.640727e-03 7.504546e-03 2.126745e-02 
        Max.         Mean           SD 
9.784316e-02 1.490504e-02 1.947399e-02 
Accuracy with training data: 97.50%
Out-of-bag accuracy: 97.50%
Build a HIBAG model with 1 individual classifier:
    MAF threshold: NaN
    # of SNPs randomly sampled as candidates for each selection: 10
    # of SNPs: 83
    # of samples: 60
    # of unique HLA alleles: 17
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:53
=== building individual classifier 1, out-of-bag (18/30.0%) ===
[1] 2023-01-29 00:48:54, oob acc: 88.89%, # of SNPs: 14, # of haplo: 38
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
2.222223e-18 6.603163e-16 6.583163e-15 1.944468e-03 1.020834e-02 4.122739e-02 
        Max.         Mean           SD 
1.808372e-01 2.422083e-02 3.699146e-02 
Accuracy with training data: 95.83%
Out-of-bag accuracy: 88.89%
Gene: HLA-C
Training dataset: 60 samples X 83 SNPs
    # of HLA alleles: 17
    # of individual classifiers: 3
    total # of SNPs used: 40
    avg. # of SNPs in an individual classifier: 18.67
        (sd: 5.03, min: 14, max: 24, median: 18.00)
    avg. # of haplotypes in an individual classifier: 33.67
        (sd: 4.51, min: 29, max: 38, median: 34.00)
    avg. out-of-bag accuracy: 92.80%
        (sd: 4.36%, min: 88.89%, max: 97.50%, median: 92.00%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
1.708707e-13 1.229313e-05 1.229313e-04 1.860746e-03 9.050936e-03 3.332722e-02 
        Max.         Mean           SD 
1.210500e-01 1.989079e-02 2.507466e-02 
Genome assembly: hg19
Build a HIBAG model with 2 individual classifiers:
    MAF threshold: NaN
    excluding 1 monomorphic SNP
    # of SNPs randomly sampled as candidates for each selection: 9
    # of SNPs: 77
    # of samples: 60
    # of unique HLA alleles: 12
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:54
=== building individual classifier 1, out-of-bag (25/41.7%) ===
[1] 2023-01-29 00:48:54, oob acc: 98.00%, # of SNPs: 13, # of haplo: 20
=== building individual classifier 2, out-of-bag (22/36.7%) ===
[2] 2023-01-29 00:48:54, oob acc: 90.91%, # of SNPs: 15, # of haplo: 21
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
7.678603e-08 2.465759e-05 2.458848e-04 5.196909e-03 1.419916e-02 2.996040e-02 
        Max.         Mean           SD 
4.735980e-01 4.413724e-02 1.070518e-01 
Accuracy with training data: 95.00%
Out-of-bag accuracy: 94.45%
Gene: HLA-DQB1
Training dataset: 60 samples X 77 SNPs
    # of HLA alleles: 12
    # of individual classifiers: 2
    total # of SNPs used: 20
    avg. # of SNPs in an individual classifier: 14.00
        (sd: 1.41, min: 13, max: 15, median: 14.00)
    avg. # of haplotypes in an individual classifier: 20.50
        (sd: 0.71, min: 20, max: 21, median: 20.50)
    avg. out-of-bag accuracy: 94.45%
        (sd: 5.01%, min: 90.91%, max: 98.00%, median: 94.45%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
7.678603e-08 2.465759e-05 2.458848e-04 5.196909e-03 1.419916e-02 2.996040e-02 
        Max.         Mean           SD 
4.735980e-01 4.413724e-02 1.070518e-01 
Genome assembly: hg19
Build a HIBAG model with 4 individual classifiers:
    MAF threshold: NaN
    excluding 9 monomorphic SNPs
    # of SNPs randomly sampled as candidates for each selection: 17
    # of SNPs: 266
    # of samples: 60
    # of unique HLA alleles: 14
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:54
=== building individual classifier 1, out-of-bag (23/38.3%) ===
[1] 2023-01-29 00:48:54, oob acc: 86.96%, # of SNPs: 12, # of haplo: 32
=== building individual classifier 2, out-of-bag (24/40.0%) ===
[2] 2023-01-29 00:48:54, oob acc: 87.50%, # of SNPs: 15, # of haplo: 40
=== building individual classifier 3, out-of-bag (24/40.0%) ===
[3] 2023-01-29 00:48:54, oob acc: 97.92%, # of SNPs: 14, # of haplo: 21
=== building individual classifier 4, out-of-bag (22/36.7%) ===
[4] 2023-01-29 00:48:54, oob acc: 95.45%, # of SNPs: 14, # of haplo: 25
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0002130783 0.0002168959 0.0002512539 0.0038875845 0.0168866292 0.0328690777 
        Max.         Mean           SD 
0.3658111951 0.0404459574 0.0794719104 
Accuracy with training data: 99.17%
Out-of-bag accuracy: 91.96%
Gene: HLA-A
Training dataset: 60 samples X 266 SNPs
    # of HLA alleles: 14
    # of individual classifiers: 4
    total # of SNPs used: 42
    avg. # of SNPs in an individual classifier: 13.75
        (sd: 1.26, min: 12, max: 15, median: 14.00)
    avg. # of haplotypes in an individual classifier: 29.50
        (sd: 8.35, min: 21, max: 40, median: 28.50)
    avg. out-of-bag accuracy: 91.96%
        (sd: 5.56%, min: 86.96%, max: 97.92%, median: 91.48%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0002130783 0.0002168959 0.0002512539 0.0038875845 0.0168866292 0.0328690777 
        Max.         Mean           SD 
0.3658111951 0.0404459574 0.0794719104 
Genome assembly: hg19
Gene: HLA-A
Training dataset: 60 samples X 266 SNPs
    # of HLA alleles: 14
    # of individual classifiers: 4
    total # of SNPs used: 42
    avg. # of SNPs in an individual classifier: 13.75
        (sd: 1.26, min: 12, max: 15, median: 14.00)
    avg. # of haplotypes in an individual classifier: 29.50
        (sd: 8.35, min: 21, max: 40, median: 28.50)
    avg. out-of-bag accuracy: 91.96%
        (sd: 5.56%, min: 86.96%, max: 97.92%, median: 91.48%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0002130783 0.0002168959 0.0002512539 0.0038875845 0.0168866292 0.0328690777 
        Max.         Mean           SD 
0.3658111951 0.0404459574 0.0794719104 
Genome assembly: hg19
Sun Jan 29 00:48:54 2023, passing the 1/4 classifiers.
Sun Jan 29 00:48:54 2023, passing the 2/4 classifiers.
Sun Jan 29 00:48:54 2023, passing the 3/4 classifiers.
Sun Jan 29 00:48:54 2023, passing the 4/4 classifiers.
Allele	Num.	Freq.	CR	ACC	SEN	SPE	PPV	NPV	Miscall
	Valid.	Valid.	(%)	(%)	(%)	(%)	(%)	(%)	(%)
----
Overall accuracy: 92.0%, Call rate: 100.0%
01:01 25 0.2083 100.0 100.0 100.0 100.0 100.0 100.0 --
02:01 43 0.3583 100.0 96.7 100.0 95.1 92.5 100.0 --
02:06 1 0.0083 25.0 97.7 0.0 100.0 -- 97.7 02:01 (100)
03:01 9 0.0750 100.0 100.0 100.0 100.0 100.0 100.0 --
11:01 5 0.0417 100.0 100.0 100.0 100.0 100.0 100.0 --
23:01 3 0.0250 100.0 98.4 75.0 100.0 100.0 98.4 24:02 (100)
24:02 11 0.0917 100.0 97.3 100.0 97.1 76.2 100.0 --
24:03 1 0.0083 100.0 97.8 0.0 100.0 -- 97.8 24:02 (75)
25:01 5 0.0417 100.0 98.4 100.0 98.3 84.7 100.0 --
26:01 3 0.0250 100.0 98.4 62.5 100.0 100.0 98.4 25:01 (83)
29:02 4 0.0333 100.0 97.8 75.0 100.0 100.0 97.8 02:01 (75)
31:01 3 0.0250 75.0 100.0 100.0 100.0 100.0 100.0 --
32:01 4 0.0333 100.0 100.0 100.0 100.0 100.0 100.0 --
68:01 3 0.0250 100.0 100.0 100.0 100.0 100.0 100.0 --
\title{Imputation Evaluation}

\documentclass[12pt]{article}

\usepackage{fullpage}
\usepackage{longtable}

\begin{document}

\maketitle

\setlength{\LTcapwidth}{6.5in}

% -------- BEGIN TABLE --------
\begin{longtable}{rrr | rrrrrrl}
\caption{The sensitivity (SEN), specificity (SPE), positive predictive value (PPV), negative predictive value (NPV) and call rate (CR).}
\label{tab:accuracy} \\
Allele &amp; Num. &amp; Freq. &amp; CR &amp; ACC &amp; SEN &amp; SPE &amp; PPV &amp; NPV &amp; Miscall \\
 &amp; Valid. &amp; Valid. &amp; (\%) &amp; (\%) &amp; (\%) &amp; (\%) &amp; (\%) &amp; (\%) &amp; (\%) \\
\hline\hline
\endfirsthead
\multicolumn{10}{c}{{\normalsize \tablename\ \thetable{} -- Continued from previous page}} \\
Allele &amp; Num. &amp; Freq. &amp; CR &amp; ACC &amp; SEN &amp; SPE &amp; PPV &amp; NPV &amp; Miscall \\
 &amp; Valid. &amp; Valid. &amp; (\%) &amp; (\%) &amp; (\%) &amp; (\%) &amp; (\%) &amp; (\%) &amp; (\%) \\
\hline\hline
\endhead
\hline
\multicolumn{10}{r}{Continued on next page ...} \\
\hline
\endfoot
\hline\hline
\endlastfoot
\multicolumn{10}{l}{\it Overall accuracy: 92.0\%, Call rate: 100.0\%} \\
01:01 &amp; 25 &amp; 0.2083 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; -- \\
02:01 &amp; 43 &amp; 0.3583 &amp; 100.0 &amp; 96.7 &amp; 100.0 &amp; 95.1 &amp; 92.5 &amp; 100.0 &amp; -- \\
02:06 &amp; 1 &amp; 0.0083 &amp; 25.0 &amp; 97.7 &amp; 0.0 &amp; 100.0 &amp; -- &amp; 97.7 &amp; 02:01 (100) \\
03:01 &amp; 9 &amp; 0.0750 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; -- \\
11:01 &amp; 5 &amp; 0.0417 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; -- \\
23:01 &amp; 3 &amp; 0.0250 &amp; 100.0 &amp; 98.4 &amp; 75.0 &amp; 100.0 &amp; 100.0 &amp; 98.4 &amp; 24:02 (100) \\
24:02 &amp; 11 &amp; 0.0917 &amp; 100.0 &amp; 97.3 &amp; 100.0 &amp; 97.1 &amp; 76.2 &amp; 100.0 &amp; -- \\
24:03 &amp; 1 &amp; 0.0083 &amp; 100.0 &amp; 97.8 &amp; 0.0 &amp; 100.0 &amp; -- &amp; 97.8 &amp; 24:02 (75) \\
25:01 &amp; 5 &amp; 0.0417 &amp; 100.0 &amp; 98.4 &amp; 100.0 &amp; 98.3 &amp; 84.7 &amp; 100.0 &amp; -- \\
26:01 &amp; 3 &amp; 0.0250 &amp; 100.0 &amp; 98.4 &amp; 62.5 &amp; 100.0 &amp; 100.0 &amp; 98.4 &amp; 25:01 (83) \\
29:02 &amp; 4 &amp; 0.0333 &amp; 100.0 &amp; 97.8 &amp; 75.0 &amp; 100.0 &amp; 100.0 &amp; 97.8 &amp; 02:01 (75) \\
31:01 &amp; 3 &amp; 0.0250 &amp; 75.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; -- \\
32:01 &amp; 4 &amp; 0.0333 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; -- \\
68:01 &amp; 3 &amp; 0.0250 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; -- \\
\end{longtable}
% -------- END TABLE --------

\end{document}
&lt;!DOCTYPE html&gt;
&lt;html&gt;
&lt;head&gt;
  &lt;title&gt;Imputation Evaluation&lt;/title&gt;
&lt;/head&gt;
&lt;body&gt;
&lt;h1&gt;Imputation Evaluation&lt;/h1&gt;
&lt;p&gt;&lt;/p&gt;
&lt;h3&gt;&lt;b&gt;Table 1L:&lt;/b&gt; The sensitivity (SEN), specificity (SPE),
positive predictive value (PPV), negative predictive value (NPV)
and call rate (CR).&lt;/h3&gt;
&lt;table id=&quot;TB-Acc&quot; class=&quot;tabular&quot; border=&quot;1&quot;  CELLSPACING=&quot;1&quot;&gt;
&lt;tr&gt;
&lt;th&gt;Allele &lt;/th&gt; &lt;th&gt;Num. Valid.&lt;/th&gt; &lt;th&gt;Freq. Valid.&lt;/th&gt; &lt;th&gt;CR (%)&lt;/th&gt; &lt;th&gt;ACC (%)&lt;/th&gt; &lt;th&gt;SEN (%)&lt;/th&gt; &lt;th&gt;SPE (%)&lt;/th&gt; &lt;th&gt;PPV (%)&lt;/th&gt; &lt;th&gt;NPV (%)&lt;/th&gt; &lt;th&gt;Miscall (%)&lt;/th&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td colspan=&quot;10&quot;&gt;
&lt;i&gt; Overall accuracy: 92.0%, Call rate: 100.0% &lt;/i&gt;
&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;01:01&lt;/td&gt; &lt;td&gt;25&lt;/td&gt; &lt;td&gt;0.2083&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;02:01&lt;/td&gt; &lt;td&gt;43&lt;/td&gt; &lt;td&gt;0.3583&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;96.7&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;95.1&lt;/td&gt; &lt;td&gt;92.5&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;02:06&lt;/td&gt; &lt;td&gt;1&lt;/td&gt; &lt;td&gt;0.0083&lt;/td&gt; &lt;td&gt;25.0&lt;/td&gt; &lt;td&gt;97.7&lt;/td&gt; &lt;td&gt;0.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt; &lt;td&gt;97.7&lt;/td&gt; &lt;td&gt;02:01 (100)&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;03:01&lt;/td&gt; &lt;td&gt;9&lt;/td&gt; &lt;td&gt;0.0750&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;11:01&lt;/td&gt; &lt;td&gt;5&lt;/td&gt; &lt;td&gt;0.0417&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;23:01&lt;/td&gt; &lt;td&gt;3&lt;/td&gt; &lt;td&gt;0.0250&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;98.4&lt;/td&gt; &lt;td&gt;75.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;98.4&lt;/td&gt; &lt;td&gt;24:02 (100)&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;24:02&lt;/td&gt; &lt;td&gt;11&lt;/td&gt; &lt;td&gt;0.0917&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;97.3&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;97.1&lt;/td&gt; &lt;td&gt;76.2&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;24:03&lt;/td&gt; &lt;td&gt;1&lt;/td&gt; &lt;td&gt;0.0083&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;97.8&lt;/td&gt; &lt;td&gt;0.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt; &lt;td&gt;97.8&lt;/td&gt; &lt;td&gt;24:02 (75)&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;25:01&lt;/td&gt; &lt;td&gt;5&lt;/td&gt; &lt;td&gt;0.0417&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;98.4&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;98.3&lt;/td&gt; &lt;td&gt;84.7&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;26:01&lt;/td&gt; &lt;td&gt;3&lt;/td&gt; &lt;td&gt;0.0250&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;98.4&lt;/td&gt; &lt;td&gt;62.5&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;98.4&lt;/td&gt; &lt;td&gt;25:01 (83)&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;29:02&lt;/td&gt; &lt;td&gt;4&lt;/td&gt; &lt;td&gt;0.0333&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;97.8&lt;/td&gt; &lt;td&gt;75.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;97.8&lt;/td&gt; &lt;td&gt;02:01 (75)&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;31:01&lt;/td&gt; &lt;td&gt;3&lt;/td&gt; &lt;td&gt;0.0250&lt;/td&gt; &lt;td&gt;75.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;32:01&lt;/td&gt; &lt;td&gt;4&lt;/td&gt; &lt;td&gt;0.0333&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;68:01&lt;/td&gt; &lt;td&gt;3&lt;/td&gt; &lt;td&gt;0.0250&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt;
&lt;/tr&gt;
&lt;/table&gt;

&lt;/body&gt;
&lt;/html&gt;
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 34
# of unique HLA alleles: 14
# of unique HLA genotypes: 23
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 26
# of unique HLA alleles: 12
# of unique HLA genotypes: 14
Building a HIBAG model:
    4 individual classifiers
    run in parallel with 1 compute node
Build a HIBAG model with 4 individual classifiers:
    MAF threshold: NaN
    excluding 11 monomorphic SNPs
    # of SNPs randomly sampled as candidates for each selection: 17
    # of SNPs: 264
    # of samples: 34
    # of unique HLA alleles: 14
CPU flags: 64-bit
# of threads: 2
[-] 2023-01-29 00:48:54
=== building individual classifier 1, out-of-bag (11/32.4%) ===
[1] 2023-01-29 00:48:54, oob acc: 77.27%, # of SNPs: 13, # of haplo: 23
=== building individual classifier 2, out-of-bag (13/38.2%) ===
[2] 2023-01-29 00:48:55, oob acc: 88.46%, # of SNPs: 11, # of haplo: 48
=== building individual classifier 3, out-of-bag (14/41.2%) ===
[3] 2023-01-29 00:48:55, oob acc: 85.71%, # of SNPs: 11, # of haplo: 14
=== building individual classifier 4, out-of-bag (10/29.4%) ===
[4] 2023-01-29 00:48:55, oob acc: 75.00%, # of SNPs: 13, # of haplo: 23
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0002440856 0.0002467397 0.0002706268 0.0043752100 0.0092453327 0.0260848720 
        Max.         Mean           SD 
0.4759517521 0.0445726720 0.1064934602 
Accuracy with training data: 97.06%
Out-of-bag accuracy: 81.61%
Building a HIBAG model:
    4 individual classifiers
    run in parallel with 2 compute nodes
    autosave to &#x27;tmp_model.RData&#x27;
[-] 2023-01-29 00:48:55
[1] 2023-01-29 00:48:55, worker  2, # of SNPs: 12, # of haplo: 53, oob acc: 90.9%
==Saved== #1, avg oob acc: 90.91%, sd: NA%, min: 90.91%, max: 90.91%
[2] 2023-01-29 00:48:55, worker  1, # of SNPs: 14, # of haplo: 70, oob acc: 90.9%
==Saved== #2, avg oob acc: 90.91%, sd: 0.00%, min: 90.91%, max: 90.91%
[3] 2023-01-29 00:48:56, worker  2, # of SNPs: 14, # of haplo: 26, oob acc: 90.9%
Stop &quot;job 2&quot;.
==Saved== #3, avg oob acc: 90.91%, sd: 0.00%, min: 90.91%, max: 90.91%
[4] 2023-01-29 00:48:56, worker  1, # of SNPs: 14, # of haplo: 21, oob acc: 84.6%
Stop &quot;job 1&quot;.
==Saved== #4, avg oob acc: 89.34%, sd: 3.15%, min: 84.62%, max: 90.91%
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0003244030 0.0003397173 0.0004775462 0.0024647016 0.0132103293 0.0413575687 
        Max.         Mean           SD 
0.4198946752 0.0474033545 0.1009720547 
Accuracy with training data: 98.53%
Out-of-bag accuracy: 89.34%
Gene: HLA-A
Training dataset: 34 samples X 264 SNPs
    # of HLA alleles: 14
    # of individual classifiers: 4
    total # of SNPs used: 44
    avg. # of SNPs in an individual classifier: 13.50
        (sd: 1.00, min: 12, max: 14, median: 14.00)
    avg. # of haplotypes in an individual classifier: 42.50
        (sd: 23.10, min: 21, max: 70, median: 39.50)
    avg. out-of-bag accuracy: 89.34%
        (sd: 3.15%, min: 84.62%, max: 90.91%, median: 90.91%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0003244030 0.0003397173 0.0004775462 0.0024647016 0.0132103293 0.0413575687 
        Max.         Mean           SD 
0.4198946752 0.0474033545 0.1009720547 
Genome assembly: hg19
HIBAG model for HLA-A:
    4 individual classifiers
    264 SNPs
    14 unique HLA alleles: 01:01, 02:01, 02:06, ...
Prediction:
    based on the averaged posterior probabilities
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 26
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:56)	0%
Predicting (2023-01-29 00:48:56)	100%
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 26
# of unique HLA alleles: 11
# of unique HLA genotypes: 14
Posterior probability:
  [0,0.25) [0.25,0.5) [0.5,0.75)   [0.75,1] 
  1 (3.8%)   2 (7.7%)  4 (15.4%) 19 (73.1%) 
Matching proportion of SNP haplotype:
    Min.  1st Qu.   Median     Mean  3rd Qu.     Max. 
0.000000 0.001710 0.006045 0.031433 0.035496 0.419895 
Dosages:
$dosage - num [1:14, 1:26] 2.31e-10 5.07e-09 3.75e-12 9.93e-01 2.08e-15 ...
 - attr(*, &quot;dimnames&quot;)=List of 2
  ..$ : chr [1:14] &quot;01:01&quot; &quot;02:01&quot; &quot;02:06&quot; &quot;03:01&quot; ...
  ..$ : chr [1:26] &quot;NA11881&quot; &quot;NA11992&quot; &quot;NA11994&quot; &quot;NA12249&quot; ...
HIBAG model for HLA-A:
    4 individual classifiers
    264 SNPs
    14 unique HLA alleles: 01:01, 02:01, 02:06, ...
Prediction:
    based on the averaged posterior probabilities
    run in parallel with 2 compute nodes
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 26
CPU flags: 64-bit
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 26
# of unique HLA alleles: 11
# of unique HLA genotypes: 14
Posterior probability:
  [0,0.25) [0.25,0.5) [0.5,0.75)   [0.75,1] 
  1 (3.8%)   2 (7.7%)  4 (15.4%) 19 (73.1%) 
Matching proportion of SNP haplotype:
    Min.  1st Qu.   Median     Mean  3rd Qu.     Max. 
0.000000 0.001710 0.006045 0.031433 0.035496 0.419895 
Dosages:
$dosage - num [1:14, 1:26] 2.31e-10 5.07e-09 3.75e-12 9.93e-01 2.08e-15 ...
 - attr(*, &quot;dimnames&quot;)=List of 2
  ..$ : chr [1:14] &quot;01:01&quot; &quot;02:01&quot; &quot;02:06&quot; &quot;03:01&quot; ...
  ..$ : chr [1:26] &quot;NA11881&quot; &quot;NA11992&quot; &quot;NA11994&quot; &quot;NA12249&quot; ...
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 34
# of unique HLA alleles: 14
# of unique HLA genotypes: 21
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 26
# of unique HLA alleles: 12
# of unique HLA genotypes: 17
Build a HIBAG model with 2 individual classifiers:
    MAF threshold: NaN
    excluding 11 monomorphic SNPs
    # of SNPs randomly sampled as candidates for each selection: 17
    # of SNPs: 264
    # of samples: 34
    # of unique HLA alleles: 14
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:56
=== building individual classifier 1, out-of-bag (11/32.4%) ===
     1, SNP: 235, loss: 208.912, oob acc: 59.09%, # of haplo: 14
     2, SNP: 136, loss: 139.316, oob acc: 63.64%, # of haplo: 14
     3, SNP: 126, loss: 96.148, oob acc: 72.73%, # of haplo: 14
     4, SNP: 89, loss: 76.2917, oob acc: 77.27%, # of haplo: 15
     5, SNP: 102, loss: 61.3783, oob acc: 86.36%, # of haplo: 15
     6, SNP: 105, loss: 49.1567, oob acc: 90.91%, # of haplo: 15
     7, SNP: 117, loss: 43.0927, oob acc: 95.45%, # of haplo: 15
     8, SNP: 259, loss: 26.6243, oob acc: 95.45%, # of haplo: 17
     9, SNP: 60, loss: 17.6253, oob acc: 95.45%, # of haplo: 19
    10, SNP: 236, loss: 9.50329, oob acc: 95.45%, # of haplo: 20
    11, SNP: 94, loss: 7.27191, oob acc: 95.45%, # of haplo: 21
    12, SNP: 58, loss: 6.70503, oob acc: 95.45%, # of haplo: 27
    13, SNP: 243, loss: 2.87079, oob acc: 95.45%, # of haplo: 30
    14, SNP: 5, loss: 2.77321, oob acc: 95.45%, # of haplo: 31
[1] 2023-01-29 00:48:56, oob acc: 95.45%, # of SNPs: 14, # of haplo: 31
=== building individual classifier 2, out-of-bag (9/26.5%) ===
     1, SNP: 149, loss: 171.797, oob acc: 66.67%, # of haplo: 13
     2, SNP: 176, loss: 120.459, oob acc: 72.22%, # of haplo: 14
     3, SNP: 97, loss: 80.1731, oob acc: 83.33%, # of haplo: 14
     4, SNP: 56, loss: 51.5193, oob acc: 94.44%, # of haplo: 16
     5, SNP: 182, loss: 34.5643, oob acc: 94.44%, # of haplo: 18
     6, SNP: 121, loss: 23.0259, oob acc: 94.44%, # of haplo: 18
     7, SNP: 234, loss: 15.0596, oob acc: 94.44%, # of haplo: 20
     8, SNP: 148, loss: 9.66757, oob acc: 94.44%, # of haplo: 20
     9, SNP: 19, loss: 4.29975, oob acc: 94.44%, # of haplo: 27
    10, SNP: 226, loss: 0.481093, oob acc: 94.44%, # of haplo: 27
    11, SNP: 64, loss: 0.447483, oob acc: 94.44%, # of haplo: 28
    12, SNP: 240, loss: 0.365545, oob acc: 94.44%, # of haplo: 37
    13, SNP: 57, loss: 0.365132, oob acc: 94.44%, # of haplo: 38
[2] 2023-01-29 00:48:56, oob acc: 94.44%, # of SNPs: 13, # of haplo: 38
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
1.275953e-07 1.742509e-05 1.731025e-04 2.811482e-03 8.650597e-03 1.989621e-02 
        Max.         Mean           SD 
5.990492e-02 1.464043e-02 1.658610e-02 
Accuracy with training data: 100.00%
Out-of-bag accuracy: 94.95%
Build a HIBAG model with 2 individual classifiers:
    MAF threshold: NaN
    excluding 11 monomorphic SNPs
    # of SNPs randomly sampled as candidates for each selection: 17
    # of SNPs: 264
    # of samples: 34
    # of unique HLA alleles: 14
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:56
=== building individual classifier 1, out-of-bag (14/41.2%) ===
     1, SNP: 118, loss: 190.304, oob acc: 53.57%, # of haplo: 13
     2, SNP: 175, loss: 157.208, oob acc: 60.71%, # of haplo: 15
     3, SNP: 103, loss: 128.429, oob acc: 64.29%, # of haplo: 15
     4, SNP: 182, loss: 66.6054, oob acc: 71.43%, # of haplo: 15
     5, SNP: 152, loss: 58.8041, oob acc: 78.57%, # of haplo: 15
     6, SNP: 111, loss: 30.086, oob acc: 82.14%, # of haplo: 15
     7, SNP: 130, loss: 15.3177, oob acc: 89.29%, # of haplo: 19
     8, SNP: 229, loss: 9.99758, oob acc: 89.29%, # of haplo: 28
     9, SNP: 185, loss: 7.40712, oob acc: 89.29%, # of haplo: 29
    10, SNP: 199, loss: 6.21341, oob acc: 89.29%, # of haplo: 29
    11, SNP: 217, loss: 1.38739, oob acc: 89.29%, # of haplo: 30
[1] 2023-01-29 00:48:57, oob acc: 89.29%, # of SNPs: 11, # of haplo: 30
=== building individual classifier 2, out-of-bag (13/38.2%) ===
     1, SNP: 101, loss: 154.355, oob acc: 46.15%, # of haplo: 16
     2, SNP: 102, loss: 139.148, oob acc: 61.54%, # of haplo: 22
     3, SNP: 132, loss: 95.2502, oob acc: 73.08%, # of haplo: 23
     4, SNP: 147, loss: 76.9692, oob acc: 76.92%, # of haplo: 34
     5, SNP: 53, loss: 68.3851, oob acc: 88.46%, # of haplo: 51
     6, SNP: 186, loss: 41.8787, oob acc: 88.46%, # of haplo: 53
     7, SNP: 128, loss: 33.5437, oob acc: 92.31%, # of haplo: 53
     8, SNP: 14, loss: 23.3103, oob acc: 92.31%, # of haplo: 55
     9, SNP: 219, loss: 18.3628, oob acc: 92.31%, # of haplo: 57
    10, SNP: 149, loss: 17.9413, oob acc: 92.31%, # of haplo: 89
    11, SNP: 73, loss: 16.3172, oob acc: 92.31%, # of haplo: 90
    12, SNP: 70, loss: 16.1056, oob acc: 92.31%, # of haplo: 90
    13, SNP: 199, loss: 12.3057, oob acc: 92.31%, # of haplo: 90
    14, SNP: 203, loss: 12.2013, oob acc: 92.31%, # of haplo: 90
    15, SNP: 151, loss: 11.1795, oob acc: 92.31%, # of haplo: 90
[2] 2023-01-29 00:48:57, oob acc: 92.31%, # of SNPs: 15, # of haplo: 90
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0002703521 0.0002971139 0.0005379705 0.0036521203 0.0131584084 0.0415528465 
        Max.         Mean           SD 
0.5087413114 0.0420589840 0.0891771528 
Accuracy with training data: 97.06%
Out-of-bag accuracy: 90.80%
HIBAG model for HLA-A:
    2 individual classifiers
    264 SNPs
    14 unique HLA alleles: 01:01, 02:01, 02:06, ...
Prediction:
    by voting from all individual classifiers
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 26
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:57)	0%
Predicting (2023-01-29 00:48:57)	100%
HIBAG model for HLA-A:
    2 individual classifiers
    264 SNPs
    14 unique HLA alleles: 01:01, 02:01, 02:06, ...
Prediction:
    by voting from all individual classifiers
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 26
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:57)	0%
Predicting (2023-01-29 00:48:57)	100%
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 34
# of unique HLA alleles: 14
# of unique HLA genotypes: 21
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 26
# of unique HLA alleles: 12
# of unique HLA genotypes: 17
Build a HIBAG model with 4 individual classifiers:
    MAF threshold: NaN
    excluding 11 monomorphic SNPs
    # of SNPs randomly sampled as candidates for each selection: 17
    # of SNPs: 264
    # of samples: 34
    # of unique HLA alleles: 14
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:57
=== building individual classifier 1, out-of-bag (11/32.4%) ===
     1, SNP: 235, loss: 208.912, oob acc: 59.09%, # of haplo: 14
     2, SNP: 136, loss: 139.316, oob acc: 63.64%, # of haplo: 14
     3, SNP: 126, loss: 96.148, oob acc: 72.73%, # of haplo: 14
     4, SNP: 89, loss: 76.2917, oob acc: 77.27%, # of haplo: 15
     5, SNP: 102, loss: 61.3783, oob acc: 86.36%, # of haplo: 15
     6, SNP: 105, loss: 49.1567, oob acc: 90.91%, # of haplo: 15
     7, SNP: 117, loss: 43.0927, oob acc: 95.45%, # of haplo: 15
     8, SNP: 259, loss: 26.6243, oob acc: 95.45%, # of haplo: 17
     9, SNP: 60, loss: 17.6253, oob acc: 95.45%, # of haplo: 19
    10, SNP: 236, loss: 9.50329, oob acc: 95.45%, # of haplo: 20
    11, SNP: 94, loss: 7.27191, oob acc: 95.45%, # of haplo: 21
    12, SNP: 58, loss: 6.70503, oob acc: 95.45%, # of haplo: 27
    13, SNP: 243, loss: 2.87079, oob acc: 95.45%, # of haplo: 30
    14, SNP: 5, loss: 2.77321, oob acc: 95.45%, # of haplo: 31
[1] 2023-01-29 00:48:57, oob acc: 95.45%, # of SNPs: 14, # of haplo: 31
=== building individual classifier 2, out-of-bag (9/26.5%) ===
     1, SNP: 149, loss: 171.797, oob acc: 66.67%, # of haplo: 13
     2, SNP: 176, loss: 120.459, oob acc: 72.22%, # of haplo: 14
     3, SNP: 97, loss: 80.1731, oob acc: 83.33%, # of haplo: 14
     4, SNP: 56, loss: 51.5193, oob acc: 94.44%, # of haplo: 16
     5, SNP: 182, loss: 34.5643, oob acc: 94.44%, # of haplo: 18
     6, SNP: 121, loss: 23.0259, oob acc: 94.44%, # of haplo: 18
     7, SNP: 234, loss: 15.0596, oob acc: 94.44%, # of haplo: 20
     8, SNP: 148, loss: 9.66757, oob acc: 94.44%, # of haplo: 20
     9, SNP: 19, loss: 4.29975, oob acc: 94.44%, # of haplo: 27
    10, SNP: 226, loss: 0.481093, oob acc: 94.44%, # of haplo: 27
    11, SNP: 64, loss: 0.447483, oob acc: 94.44%, # of haplo: 28
    12, SNP: 240, loss: 0.365545, oob acc: 94.44%, # of haplo: 37
    13, SNP: 57, loss: 0.365132, oob acc: 94.44%, # of haplo: 38
[2] 2023-01-29 00:48:57, oob acc: 94.44%, # of SNPs: 13, # of haplo: 38
=== building individual classifier 3, out-of-bag (14/41.2%) ===
     1, SNP: 118, loss: 190.304, oob acc: 53.57%, # of haplo: 13
     2, SNP: 175, loss: 157.208, oob acc: 60.71%, # of haplo: 15
     3, SNP: 103, loss: 128.429, oob acc: 64.29%, # of haplo: 15
     4, SNP: 182, loss: 66.6054, oob acc: 71.43%, # of haplo: 15
     5, SNP: 152, loss: 58.8041, oob acc: 78.57%, # of haplo: 15
     6, SNP: 111, loss: 30.086, oob acc: 82.14%, # of haplo: 15
     7, SNP: 130, loss: 15.3177, oob acc: 89.29%, # of haplo: 19
     8, SNP: 229, loss: 9.99758, oob acc: 89.29%, # of haplo: 28
     9, SNP: 185, loss: 7.40712, oob acc: 89.29%, # of haplo: 29
    10, SNP: 199, loss: 6.21341, oob acc: 89.29%, # of haplo: 29
    11, SNP: 217, loss: 1.38739, oob acc: 89.29%, # of haplo: 30
[3] 2023-01-29 00:48:57, oob acc: 89.29%, # of SNPs: 11, # of haplo: 30
=== building individual classifier 4, out-of-bag (13/38.2%) ===
     1, SNP: 101, loss: 154.355, oob acc: 46.15%, # of haplo: 16
     2, SNP: 102, loss: 139.148, oob acc: 61.54%, # of haplo: 22
     3, SNP: 132, loss: 95.2502, oob acc: 73.08%, # of haplo: 23
     4, SNP: 147, loss: 76.9692, oob acc: 76.92%, # of haplo: 34
     5, SNP: 53, loss: 68.3851, oob acc: 88.46%, # of haplo: 51
     6, SNP: 186, loss: 41.8787, oob acc: 88.46%, # of haplo: 53
     7, SNP: 128, loss: 33.5437, oob acc: 92.31%, # of haplo: 53
     8, SNP: 14, loss: 23.3103, oob acc: 92.31%, # of haplo: 55
     9, SNP: 219, loss: 18.3628, oob acc: 92.31%, # of haplo: 57
    10, SNP: 149, loss: 17.9413, oob acc: 92.31%, # of haplo: 89
    11, SNP: 73, loss: 16.3172, oob acc: 92.31%, # of haplo: 90
    12, SNP: 70, loss: 16.1056, oob acc: 92.31%, # of haplo: 90
    13, SNP: 199, loss: 12.3057, oob acc: 92.31%, # of haplo: 90
    14, SNP: 203, loss: 12.2013, oob acc: 92.31%, # of haplo: 90
    15, SNP: 151, loss: 11.1795, oob acc: 92.31%, # of haplo: 90
[4] 2023-01-29 00:48:57, oob acc: 92.31%, # of SNPs: 15, # of haplo: 90
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0004055642 0.0004189435 0.0005393566 0.0035332670 0.0110835407 0.0312705424 
        Max.         Mean           SD 
0.5148772297 0.0357753361 0.0879935706 
Accuracy with training data: 97.06%
Out-of-bag accuracy: 92.87%
Gene: HLA-A
Training dataset: 34 samples X 264 SNPs
    # of HLA alleles: 14
    # of individual classifiers: 4
    total # of SNPs used: 49
    avg. # of SNPs in an individual classifier: 13.25
        (sd: 1.71, min: 11, max: 15, median: 13.50)
    avg. # of haplotypes in an individual classifier: 47.25
        (sd: 28.72, min: 30, max: 90, median: 34.50)
    avg. out-of-bag accuracy: 92.87%
        (sd: 2.73%, min: 89.29%, max: 95.45%, median: 93.38%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0004055642 0.0004189435 0.0005393566 0.0035332670 0.0110835407 0.0312705424 
        Max.         Mean           SD 
0.5148772297 0.0357753361 0.0879935706 
Genome assembly: hg19
HIBAG model for HLA-A:
    4 individual classifiers
    264 SNPs
    14 unique HLA alleles: 01:01, 02:01, 02:06, ...
Prediction:
    based on the averaged posterior probabilities
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 26
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:58)	0%
Predicting (2023-01-29 00:48:58)	100%
Build a HIBAG model with 2 individual classifiers:
    MAF threshold: NaN
    excluding 1 monomorphic SNP
    # of SNPs randomly sampled as candidates for each selection: 13
    # of SNPs: 158
    # of samples: 60
    # of unique HLA alleles: 14
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:58
=== building individual classifier 1, out-of-bag (24/40.0%) ===
     1, SNP: 141, loss: 378.06, oob acc: 52.08%, # of haplo: 14
     2, SNP: 74, loss: 262.497, oob acc: 58.33%, # of haplo: 15
     3, SNP: 78, loss: 162.497, oob acc: 68.75%, # of haplo: 19
     4, SNP: 118, loss: 70.0426, oob acc: 72.92%, # of haplo: 23
     5, SNP: 82, loss: 45.8279, oob acc: 83.33%, # of haplo: 23
     6, SNP: 95, loss: 35.4414, oob acc: 89.58%, # of haplo: 27
     7, SNP: 89, loss: 32.6134, oob acc: 89.58%, # of haplo: 35
     8, SNP: 83, loss: 31.7921, oob acc: 89.58%, # of haplo: 51
     9, SNP: 151, loss: 31.0653, oob acc: 89.58%, # of haplo: 55
    10, SNP: 94, loss: 31.0246, oob acc: 89.58%, # of haplo: 55
    11, SNP: 111, loss: 18.9027, oob acc: 89.58%, # of haplo: 56
    12, SNP: 139, loss: 18.4248, oob acc: 89.58%, # of haplo: 59
    13, SNP: 93, loss: 17.0195, oob acc: 91.67%, # of haplo: 58
    14, SNP: 15, loss: 14.1692, oob acc: 91.67%, # of haplo: 60
[1] 2023-01-29 00:48:58, oob acc: 91.67%, # of SNPs: 14, # of haplo: 60
=== building individual classifier 2, out-of-bag (19/31.7%) ===
     1, SNP: 94, loss: 403.365, oob acc: 63.16%, # of haplo: 15
     2, SNP: 82, loss: 294.053, oob acc: 71.05%, # of haplo: 18
     3, SNP: 57, loss: 226.142, oob acc: 76.32%, # of haplo: 23
     4, SNP: 155, loss: 197.199, oob acc: 84.21%, # of haplo: 29
     5, SNP: 44, loss: 132.804, oob acc: 86.84%, # of haplo: 40
     6, SNP: 30, loss: 122.507, oob acc: 92.11%, # of haplo: 40
     7, SNP: 109, loss: 72.0179, oob acc: 92.11%, # of haplo: 41
     8, SNP: 72, loss: 59.3281, oob acc: 92.11%, # of haplo: 41
     9, SNP: 36, loss: 54.939, oob acc: 94.74%, # of haplo: 43
    10, SNP: 127, loss: 48.1392, oob acc: 94.74%, # of haplo: 43
    11, SNP: 53, loss: 44.7676, oob acc: 94.74%, # of haplo: 43
    12, SNP: 148, loss: 43.047, oob acc: 94.74%, # of haplo: 44
    13, SNP: 152, loss: 40.2104, oob acc: 94.74%, # of haplo: 45
    14, SNP: 125, loss: 39.8862, oob acc: 94.74%, # of haplo: 45
    15, SNP: 78, loss: 39.5652, oob acc: 94.74%, # of haplo: 45
    16, SNP: 3, loss: 39.0621, oob acc: 94.74%, # of haplo: 47
    17, SNP: 141, loss: 37.6822, oob acc: 94.74%, # of haplo: 47
    18, SNP: 1, loss: 36.5022, oob acc: 94.74%, # of haplo: 50
[2] 2023-01-29 00:48:58, oob acc: 94.74%, # of SNPs: 18, # of haplo: 50
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
8.333397e-09 5.342641e-08 4.592635e-07 2.372425e-03 1.515625e-02 5.190505e-02 
        Max.         Mean           SD 
4.790185e-01 5.479747e-02 1.101559e-01 
Accuracy with training data: 96.67%
Out-of-bag accuracy: 93.20%
Gene: HLA-A
Training dataset: 60 samples X 158 SNPs
    # of HLA alleles: 14
    # of individual classifiers: 2
    total # of SNPs used: 28
    avg. # of SNPs in an individual classifier: 16.00
        (sd: 2.83, min: 14, max: 18, median: 16.00)
    avg. # of haplotypes in an individual classifier: 55.00
        (sd: 7.07, min: 50, max: 60, median: 55.00)
    avg. out-of-bag accuracy: 93.20%
        (sd: 2.17%, min: 91.67%, max: 94.74%, median: 93.20%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
8.333397e-09 5.342641e-08 4.592635e-07 2.372425e-03 1.515625e-02 5.190505e-02 
        Max.         Mean           SD 
4.790185e-01 5.479747e-02 1.101559e-01 
Genome assembly: hg19
Remove 130 unused SNPs.
Gene: HLA-A
Training dataset: 60 samples X 28 SNPs
    # of HLA alleles: 14
    # of individual classifiers: 2
    total # of SNPs used: 28
    avg. # of SNPs in an individual classifier: 16.00
        (sd: 2.83, min: 14, max: 18, median: 16.00)
    avg. # of haplotypes in an individual classifier: 55.00
        (sd: 7.07, min: 50, max: 60, median: 55.00)
    avg. out-of-bag accuracy: 93.20%
        (sd: 2.17%, min: 91.67%, max: 94.74%, median: 93.20%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
8.333397e-09 5.342641e-08 4.592635e-07 2.372425e-03 1.515625e-02 5.190505e-02 
        Max.         Mean           SD 
4.790185e-01 5.479747e-02 1.101559e-01 
Genome assembly: hg19
Platform: Illumina 1M Duo 
Information: Training set -- HapMap Phase II 
HIBAG model for HLA-A:
    2 individual classifiers
    158 SNPs
    14 unique HLA alleles: 01:01, 02:01, 02:06, ...
Prediction:
    based on the averaged posterior probabilities
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 60
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:58)	0%
Predicting (2023-01-29 00:48:58)	100%
HIBAG model for HLA-A:
    2 individual classifiers
    28 SNPs
    14 unique HLA alleles: 01:01, 02:01, 02:06, ...
Prediction:
    based on the averaged posterior probabilities
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: Illumina 1M Duo
No allelic strand or A/B allele is flipped.
# of samples: 60
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:58)	0%
Predicting (2023-01-29 00:48:58)	100%
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 34
# of unique HLA alleles: 14
# of unique HLA genotypes: 21
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 26
# of unique HLA alleles: 12
# of unique HLA genotypes: 17
Build a HIBAG model with 4 individual classifiers:
    MAF threshold: NaN
    excluding 11 monomorphic SNPs
    # of SNPs randomly sampled as candidates for each selection: 17
    # of SNPs: 264
    # of samples: 34
    # of unique HLA alleles: 14
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:58
=== building individual classifier 1, out-of-bag (11/32.4%) ===
     1, SNP: 235, loss: 208.912, oob acc: 59.09%, # of haplo: 14
     2, SNP: 136, loss: 139.316, oob acc: 63.64%, # of haplo: 14
     3, SNP: 126, loss: 96.148, oob acc: 72.73%, # of haplo: 14
     4, SNP: 89, loss: 76.2917, oob acc: 77.27%, # of haplo: 15
     5, SNP: 102, loss: 61.3783, oob acc: 86.36%, # of haplo: 15
     6, SNP: 105, loss: 49.1567, oob acc: 90.91%, # of haplo: 15
     7, SNP: 117, loss: 43.0927, oob acc: 95.45%, # of haplo: 15
     8, SNP: 259, loss: 26.6243, oob acc: 95.45%, # of haplo: 17
     9, SNP: 60, loss: 17.6253, oob acc: 95.45%, # of haplo: 19
    10, SNP: 236, loss: 9.50329, oob acc: 95.45%, # of haplo: 20
    11, SNP: 94, loss: 7.27191, oob acc: 95.45%, # of haplo: 21
    12, SNP: 58, loss: 6.70503, oob acc: 95.45%, # of haplo: 27
    13, SNP: 243, loss: 2.87079, oob acc: 95.45%, # of haplo: 30
    14, SNP: 5, loss: 2.77321, oob acc: 95.45%, # of haplo: 31
[1] 2023-01-29 00:48:58, oob acc: 95.45%, # of SNPs: 14, # of haplo: 31
=== building individual classifier 2, out-of-bag (9/26.5%) ===
     1, SNP: 149, loss: 171.797, oob acc: 66.67%, # of haplo: 13
     2, SNP: 176, loss: 120.459, oob acc: 72.22%, # of haplo: 14
     3, SNP: 97, loss: 80.1731, oob acc: 83.33%, # of haplo: 14
     4, SNP: 56, loss: 51.5193, oob acc: 94.44%, # of haplo: 16
     5, SNP: 182, loss: 34.5643, oob acc: 94.44%, # of haplo: 18
     6, SNP: 121, loss: 23.0259, oob acc: 94.44%, # of haplo: 18
     7, SNP: 234, loss: 15.0596, oob acc: 94.44%, # of haplo: 20
     8, SNP: 148, loss: 9.66757, oob acc: 94.44%, # of haplo: 20
     9, SNP: 19, loss: 4.29975, oob acc: 94.44%, # of haplo: 27
    10, SNP: 226, loss: 0.481093, oob acc: 94.44%, # of haplo: 27
    11, SNP: 64, loss: 0.447483, oob acc: 94.44%, # of haplo: 28
    12, SNP: 240, loss: 0.365545, oob acc: 94.44%, # of haplo: 37
    13, SNP: 57, loss: 0.365132, oob acc: 94.44%, # of haplo: 38
[2] 2023-01-29 00:48:58, oob acc: 94.44%, # of SNPs: 13, # of haplo: 38
=== building individual classifier 3, out-of-bag (14/41.2%) ===
     1, SNP: 118, loss: 190.304, oob acc: 53.57%, # of haplo: 13
     2, SNP: 175, loss: 157.208, oob acc: 60.71%, # of haplo: 15
     3, SNP: 103, loss: 128.429, oob acc: 64.29%, # of haplo: 15
     4, SNP: 182, loss: 66.6054, oob acc: 71.43%, # of haplo: 15
     5, SNP: 152, loss: 58.8041, oob acc: 78.57%, # of haplo: 15
     6, SNP: 111, loss: 30.086, oob acc: 82.14%, # of haplo: 15
     7, SNP: 130, loss: 15.3177, oob acc: 89.29%, # of haplo: 19
     8, SNP: 229, loss: 9.99758, oob acc: 89.29%, # of haplo: 28
     9, SNP: 185, loss: 7.40712, oob acc: 89.29%, # of haplo: 29
    10, SNP: 199, loss: 6.21341, oob acc: 89.29%, # of haplo: 29
    11, SNP: 217, loss: 1.38739, oob acc: 89.29%, # of haplo: 30
[3] 2023-01-29 00:48:58, oob acc: 89.29%, # of SNPs: 11, # of haplo: 30
=== building individual classifier 4, out-of-bag (13/38.2%) ===
     1, SNP: 101, loss: 154.355, oob acc: 46.15%, # of haplo: 16
     2, SNP: 102, loss: 139.148, oob acc: 61.54%, # of haplo: 22
     3, SNP: 132, loss: 95.2502, oob acc: 73.08%, # of haplo: 23
     4, SNP: 147, loss: 76.9692, oob acc: 76.92%, # of haplo: 34
     5, SNP: 53, loss: 68.3851, oob acc: 88.46%, # of haplo: 51
     6, SNP: 186, loss: 41.8787, oob acc: 88.46%, # of haplo: 53
     7, SNP: 128, loss: 33.5437, oob acc: 92.31%, # of haplo: 53
     8, SNP: 14, loss: 23.3103, oob acc: 92.31%, # of haplo: 55
     9, SNP: 219, loss: 18.3628, oob acc: 92.31%, # of haplo: 57
    10, SNP: 149, loss: 17.9413, oob acc: 92.31%, # of haplo: 89
    11, SNP: 73, loss: 16.3172, oob acc: 92.31%, # of haplo: 90
    12, SNP: 70, loss: 16.1056, oob acc: 92.31%, # of haplo: 90
    13, SNP: 199, loss: 12.3057, oob acc: 92.31%, # of haplo: 90
    14, SNP: 203, loss: 12.2013, oob acc: 92.31%, # of haplo: 90
    15, SNP: 151, loss: 11.1795, oob acc: 92.31%, # of haplo: 90
[4] 2023-01-29 00:48:58, oob acc: 92.31%, # of SNPs: 15, # of haplo: 90
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0004055642 0.0004189435 0.0005393566 0.0035332670 0.0110835407 0.0312705424 
        Max.         Mean           SD 
0.5148772297 0.0357753361 0.0879935706 
Accuracy with training data: 97.06%
Out-of-bag accuracy: 92.87%
Gene: HLA-A
Training dataset: 34 samples X 264 SNPs
    # of HLA alleles: 14
    # of individual classifiers: 4
    total # of SNPs used: 49
    avg. # of SNPs in an individual classifier: 13.25
        (sd: 1.71, min: 11, max: 15, median: 13.50)
    avg. # of haplotypes in an individual classifier: 47.25
        (sd: 28.72, min: 30, max: 90, median: 34.50)
    avg. out-of-bag accuracy: 92.87%
        (sd: 2.73%, min: 89.29%, max: 95.45%, median: 93.38%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0004055642 0.0004189435 0.0005393566 0.0035332670 0.0110835407 0.0312705424 
        Max.         Mean           SD 
0.5148772297 0.0357753361 0.0879935706 
Genome assembly: hg19
HIBAG model for HLA-A:
    4 individual classifiers
    264 SNPs
    14 unique HLA alleles: 01:01, 02:01, 02:06, ...
Prediction:
    based on the averaged posterior probabilities
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 26
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:58)	0%
Predicting (2023-01-29 00:48:58)	100%
Allele	Num.	Freq.	Num.	Freq.	CR	ACC	SEN	SPE	PPV	NPV	Miscall
	Train	Train	Valid.	Valid.	(%)	(%)	(%)	(%)	(%)	(%)	(%)
----
Overall accuracy: 88.5%, Call rate: 100.0%
01:01 13 0.1912 12 0.2308 100.0 96.2 100.0 95.0 85.7 100.0 --
02:01 25 0.3676 18 0.3462 100.0 98.1 94.4 100.0 100.0 97.1 01:01 (100)
02:06 1 0.0147 0 0 -- -- -- -- -- -- --
03:01 4 0.0588 5 0.0962 100.0 98.1 100.0 97.9 83.3 100.0 --
11:01 2 0.0294 3 0.0577 100.0 100.0 100.0 100.0 100.0 100.0 --
23:01 1 0.0147 2 0.0385 100.0 96.2 0.0 100.0 -- 96.2 24:02 (100)
24:02 6 0.0882 5 0.0962 100.0 92.3 60.0 95.7 60.0 95.7 01:01 (50)
24:03 1 0.0147 0 0 -- -- -- -- -- -- --
25:01 4 0.0588 1 0.0192 100.0 100.0 100.0 100.0 100.0 100.0 --
26:01 2 0.0294 1 0.0192 100.0 100.0 100.0 100.0 100.0 100.0 --
29:02 3 0.0441 1 0.0192 100.0 98.1 0.0 100.0 -- 98.1 03:01 (50)
31:01 2 0.0294 1 0.0192 100.0 100.0 100.0 100.0 100.0 100.0 --
32:01 2 0.0294 2 0.0385 100.0 100.0 100.0 100.0 100.0 100.0 --
68:01 2 0.0294 1 0.0192 100.0 98.1 100.0 98.0 50.0 100.0 --
\title{Imputation Evaluation}

\documentclass[12pt]{article}

\usepackage{fullpage}
\usepackage{longtable}

\begin{document}

\maketitle

\setlength{\LTcapwidth}{6.5in}

% -------- BEGIN TABLE --------
\begin{longtable}{rrrrr | rrrrrrl}
\caption{The sensitivity (SEN), specificity (SPE), positive predictive value (PPV), negative predictive value (NPV) and call rate (CR).}
\label{tab:accuracy} \\
Allele &amp; Num. &amp; Freq. &amp; Num. &amp; Freq. &amp; CR &amp; ACC &amp; SEN &amp; SPE &amp; PPV &amp; NPV &amp; Miscall \\
 &amp; Train &amp; Train &amp; Valid. &amp; Valid. &amp; (\%) &amp; (\%) &amp; (\%) &amp; (\%) &amp; (\%) &amp; (\%) &amp; (\%) \\
\hline\hline
\endfirsthead
\multicolumn{12}{c}{{\normalsize \tablename\ \thetable{} -- Continued from previous page}} \\
Allele &amp; Num. &amp; Freq. &amp; Num. &amp; Freq. &amp; CR &amp; ACC &amp; SEN &amp; SPE &amp; PPV &amp; NPV &amp; Miscall \\
 &amp; Train &amp; Train &amp; Valid. &amp; Valid. &amp; (\%) &amp; (\%) &amp; (\%) &amp; (\%) &amp; (\%) &amp; (\%) &amp; (\%) \\
\hline\hline
\endhead
\hline
\multicolumn{12}{r}{Continued on next page ...} \\
\hline
\endfoot
\hline\hline
\endlastfoot
\multicolumn{12}{l}{\it Overall accuracy: 88.5\%, Call rate: 100.0\%} \\
01:01 &amp; 13 &amp; 0.1912 &amp; 12 &amp; 0.2308 &amp; 100.0 &amp; 96.2 &amp; 100.0 &amp; 95.0 &amp; 85.7 &amp; 100.0 &amp; -- \\
02:01 &amp; 25 &amp; 0.3676 &amp; 18 &amp; 0.3462 &amp; 100.0 &amp; 98.1 &amp; 94.4 &amp; 100.0 &amp; 100.0 &amp; 97.1 &amp; 01:01 (100) \\
02:06 &amp; 1 &amp; 0.0147 &amp; 0 &amp; 0 &amp; -- &amp; -- &amp; -- &amp; -- &amp; -- &amp; -- &amp; -- \\
03:01 &amp; 4 &amp; 0.0588 &amp; 5 &amp; 0.0962 &amp; 100.0 &amp; 98.1 &amp; 100.0 &amp; 97.9 &amp; 83.3 &amp; 100.0 &amp; -- \\
11:01 &amp; 2 &amp; 0.0294 &amp; 3 &amp; 0.0577 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; -- \\
23:01 &amp; 1 &amp; 0.0147 &amp; 2 &amp; 0.0385 &amp; 100.0 &amp; 96.2 &amp; 0.0 &amp; 100.0 &amp; -- &amp; 96.2 &amp; 24:02 (100) \\
24:02 &amp; 6 &amp; 0.0882 &amp; 5 &amp; 0.0962 &amp; 100.0 &amp; 92.3 &amp; 60.0 &amp; 95.7 &amp; 60.0 &amp; 95.7 &amp; 01:01 (50) \\
24:03 &amp; 1 &amp; 0.0147 &amp; 0 &amp; 0 &amp; -- &amp; -- &amp; -- &amp; -- &amp; -- &amp; -- &amp; -- \\
25:01 &amp; 4 &amp; 0.0588 &amp; 1 &amp; 0.0192 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; -- \\
26:01 &amp; 2 &amp; 0.0294 &amp; 1 &amp; 0.0192 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; -- \\
29:02 &amp; 3 &amp; 0.0441 &amp; 1 &amp; 0.0192 &amp; 100.0 &amp; 98.1 &amp; 0.0 &amp; 100.0 &amp; -- &amp; 98.1 &amp; 03:01 (50) \\
31:01 &amp; 2 &amp; 0.0294 &amp; 1 &amp; 0.0192 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; -- \\
32:01 &amp; 2 &amp; 0.0294 &amp; 2 &amp; 0.0385 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; 100.0 &amp; -- \\
68:01 &amp; 2 &amp; 0.0294 &amp; 1 &amp; 0.0192 &amp; 100.0 &amp; 98.1 &amp; 100.0 &amp; 98.0 &amp; 50.0 &amp; 100.0 &amp; -- \\
\end{longtable}
% -------- END TABLE --------

\end{document}
&lt;!DOCTYPE html&gt;
&lt;html&gt;
&lt;head&gt;
  &lt;title&gt;Imputation Evaluation&lt;/title&gt;
&lt;/head&gt;
&lt;body&gt;
&lt;h1&gt;Imputation Evaluation&lt;/h1&gt;
&lt;p&gt;&lt;/p&gt;
&lt;h3&gt;&lt;b&gt;Table 1L:&lt;/b&gt; The sensitivity (SEN), specificity (SPE),
positive predictive value (PPV), negative predictive value (NPV)
and call rate (CR).&lt;/h3&gt;
&lt;table id=&quot;TB-Acc&quot; class=&quot;tabular&quot; border=&quot;1&quot;  CELLSPACING=&quot;1&quot;&gt;
&lt;tr&gt;
&lt;th&gt;Allele &lt;/th&gt; &lt;th&gt;Num. Train&lt;/th&gt; &lt;th&gt;Freq. Train&lt;/th&gt; &lt;th&gt;Num. Valid.&lt;/th&gt; &lt;th&gt;Freq. Valid.&lt;/th&gt; &lt;th&gt;CR (%)&lt;/th&gt; &lt;th&gt;ACC (%)&lt;/th&gt; &lt;th&gt;SEN (%)&lt;/th&gt; &lt;th&gt;SPE (%)&lt;/th&gt; &lt;th&gt;PPV (%)&lt;/th&gt; &lt;th&gt;NPV (%)&lt;/th&gt; &lt;th&gt;Miscall (%)&lt;/th&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td colspan=&quot;12&quot;&gt;
&lt;i&gt; Overall accuracy: 88.5%, Call rate: 100.0% &lt;/i&gt;
&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;01:01&lt;/td&gt; &lt;td&gt;13&lt;/td&gt; &lt;td&gt;0.1912&lt;/td&gt; &lt;td&gt;12&lt;/td&gt; &lt;td&gt;0.2308&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;96.2&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;95.0&lt;/td&gt; &lt;td&gt;85.7&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;02:01&lt;/td&gt; &lt;td&gt;25&lt;/td&gt; &lt;td&gt;0.3676&lt;/td&gt; &lt;td&gt;18&lt;/td&gt; &lt;td&gt;0.3462&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;98.1&lt;/td&gt; &lt;td&gt;94.4&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;97.1&lt;/td&gt; &lt;td&gt;01:01 (100)&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;02:06&lt;/td&gt; &lt;td&gt;1&lt;/td&gt; &lt;td&gt;0.0147&lt;/td&gt; &lt;td&gt;0&lt;/td&gt; &lt;td&gt;0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt; &lt;td&gt;--&lt;/td&gt; &lt;td&gt;--&lt;/td&gt; &lt;td&gt;--&lt;/td&gt; &lt;td&gt;--&lt;/td&gt; &lt;td&gt;--&lt;/td&gt; &lt;td&gt;--&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;03:01&lt;/td&gt; &lt;td&gt;4&lt;/td&gt; &lt;td&gt;0.0588&lt;/td&gt; &lt;td&gt;5&lt;/td&gt; &lt;td&gt;0.0962&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;98.1&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;97.9&lt;/td&gt; &lt;td&gt;83.3&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;11:01&lt;/td&gt; &lt;td&gt;2&lt;/td&gt; &lt;td&gt;0.0294&lt;/td&gt; &lt;td&gt;3&lt;/td&gt; &lt;td&gt;0.0577&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;23:01&lt;/td&gt; &lt;td&gt;1&lt;/td&gt; &lt;td&gt;0.0147&lt;/td&gt; &lt;td&gt;2&lt;/td&gt; &lt;td&gt;0.0385&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;96.2&lt;/td&gt; &lt;td&gt;0.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt; &lt;td&gt;96.2&lt;/td&gt; &lt;td&gt;24:02 (100)&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;24:02&lt;/td&gt; &lt;td&gt;6&lt;/td&gt; &lt;td&gt;0.0882&lt;/td&gt; &lt;td&gt;5&lt;/td&gt; &lt;td&gt;0.0962&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;92.3&lt;/td&gt; &lt;td&gt;60.0&lt;/td&gt; &lt;td&gt;95.7&lt;/td&gt; &lt;td&gt;60.0&lt;/td&gt; &lt;td&gt;95.7&lt;/td&gt; &lt;td&gt;01:01 (50)&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;24:03&lt;/td&gt; &lt;td&gt;1&lt;/td&gt; &lt;td&gt;0.0147&lt;/td&gt; &lt;td&gt;0&lt;/td&gt; &lt;td&gt;0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt; &lt;td&gt;--&lt;/td&gt; &lt;td&gt;--&lt;/td&gt; &lt;td&gt;--&lt;/td&gt; &lt;td&gt;--&lt;/td&gt; &lt;td&gt;--&lt;/td&gt; &lt;td&gt;--&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;25:01&lt;/td&gt; &lt;td&gt;4&lt;/td&gt; &lt;td&gt;0.0588&lt;/td&gt; &lt;td&gt;1&lt;/td&gt; &lt;td&gt;0.0192&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;26:01&lt;/td&gt; &lt;td&gt;2&lt;/td&gt; &lt;td&gt;0.0294&lt;/td&gt; &lt;td&gt;1&lt;/td&gt; &lt;td&gt;0.0192&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;29:02&lt;/td&gt; &lt;td&gt;3&lt;/td&gt; &lt;td&gt;0.0441&lt;/td&gt; &lt;td&gt;1&lt;/td&gt; &lt;td&gt;0.0192&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;98.1&lt;/td&gt; &lt;td&gt;0.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt; &lt;td&gt;98.1&lt;/td&gt; &lt;td&gt;03:01 (50)&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;31:01&lt;/td&gt; &lt;td&gt;2&lt;/td&gt; &lt;td&gt;0.0294&lt;/td&gt; &lt;td&gt;1&lt;/td&gt; &lt;td&gt;0.0192&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;32:01&lt;/td&gt; &lt;td&gt;2&lt;/td&gt; &lt;td&gt;0.0294&lt;/td&gt; &lt;td&gt;2&lt;/td&gt; &lt;td&gt;0.0385&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt;
&lt;/tr&gt;
&lt;tr&gt;
&lt;td&gt;68:01&lt;/td&gt; &lt;td&gt;2&lt;/td&gt; &lt;td&gt;0.0294&lt;/td&gt; &lt;td&gt;1&lt;/td&gt; &lt;td&gt;0.0192&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;98.1&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;98.0&lt;/td&gt; &lt;td&gt;50.0&lt;/td&gt; &lt;td&gt;100.0&lt;/td&gt; &lt;td&gt;--&lt;/td&gt;
&lt;/tr&gt;
&lt;/table&gt;

&lt;/body&gt;
&lt;/html&gt;
**Overall accuracy: 88.5%, Call rate: 100.0%**

| Allele  | # Train | Freq. Train | # Valid. | Freq. Valid. | CR (%) | ACC (%) | SEN (%) | SPE (%) | PPV (%) | NPV (%) | Miscall (%) |
|:--|--:|--:|--:|--:|--:|--:|--:|--:|--:|--:|:--|
| 01:01 | 13 | 0.1912 | 12 | 0.2308 | 100.0 | 96.2 | 100.0 | 95.0 | 85.7 | 100.0 | -- |
| 02:01 | 25 | 0.3676 | 18 | 0.3462 | 100.0 | 98.1 | 94.4 | 100.0 | 100.0 | 97.1 | 01:01 (100) |
| 02:06 |  1 | 0.0147 |  0 | 0 | -- | -- | -- | -- | -- | -- | -- |
| 03:01 |  4 | 0.0588 |  5 | 0.0962 | 100.0 | 98.1 | 100.0 | 97.9 | 83.3 | 100.0 | -- |
| 11:01 |  2 | 0.0294 |  3 | 0.0577 | 100.0 | 100.0 | 100.0 | 100.0 | 100.0 | 100.0 | -- |
| 23:01 |  1 | 0.0147 |  2 | 0.0385 | 100.0 | 96.2 | 0.0 | 100.0 | -- | 96.2 | 24:02 (100) |
| 24:02 |  6 | 0.0882 |  5 | 0.0962 | 100.0 | 92.3 | 60.0 | 95.7 | 60.0 | 95.7 | 01:01 (50) |
| 24:03 |  1 | 0.0147 |  0 | 0 | -- | -- | -- | -- | -- | -- | -- |
| 25:01 |  4 | 0.0588 |  1 | 0.0192 | 100.0 | 100.0 | 100.0 | 100.0 | 100.0 | 100.0 | -- |
| 26:01 |  2 | 0.0294 |  1 | 0.0192 | 100.0 | 100.0 | 100.0 | 100.0 | 100.0 | 100.0 | -- |
| 29:02 |  3 | 0.0441 |  1 | 0.0192 | 100.0 | 98.1 | 0.0 | 100.0 | -- | 98.1 | 03:01 (50) |
| 31:01 |  2 | 0.0294 |  1 | 0.0192 | 100.0 | 100.0 | 100.0 | 100.0 | 100.0 | 100.0 | -- |
| 32:01 |  2 | 0.0294 |  2 | 0.0385 | 100.0 | 100.0 | 100.0 | 100.0 | 100.0 | 100.0 | -- |
| 68:01 |  2 | 0.0294 |  1 | 0.0192 | 100.0 | 98.1 | 100.0 | 98.0 | 50.0 | 100.0 | -- |
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 34
# of unique HLA alleles: 14
# of unique HLA genotypes: 21
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 26
# of unique HLA alleles: 12
# of unique HLA genotypes: 17
Build a HIBAG model with 4 individual classifiers:
    MAF threshold: NaN
    excluding 11 monomorphic SNPs
    # of SNPs randomly sampled as candidates for each selection: 17
    # of SNPs: 264
    # of samples: 34
    # of unique HLA alleles: 14
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:58
=== building individual classifier 1, out-of-bag (11/32.4%) ===
     1, SNP: 235, loss: 208.912, oob acc: 59.09%, # of haplo: 14
     2, SNP: 136, loss: 139.316, oob acc: 63.64%, # of haplo: 14
     3, SNP: 126, loss: 96.148, oob acc: 72.73%, # of haplo: 14
     4, SNP: 89, loss: 76.2917, oob acc: 77.27%, # of haplo: 15
     5, SNP: 102, loss: 61.3783, oob acc: 86.36%, # of haplo: 15
     6, SNP: 105, loss: 49.1567, oob acc: 90.91%, # of haplo: 15
     7, SNP: 117, loss: 43.0927, oob acc: 95.45%, # of haplo: 15
     8, SNP: 259, loss: 26.6243, oob acc: 95.45%, # of haplo: 17
     9, SNP: 60, loss: 17.6253, oob acc: 95.45%, # of haplo: 19
    10, SNP: 236, loss: 9.50329, oob acc: 95.45%, # of haplo: 20
    11, SNP: 94, loss: 7.27191, oob acc: 95.45%, # of haplo: 21
    12, SNP: 58, loss: 6.70503, oob acc: 95.45%, # of haplo: 27
    13, SNP: 243, loss: 2.87079, oob acc: 95.45%, # of haplo: 30
    14, SNP: 5, loss: 2.77321, oob acc: 95.45%, # of haplo: 31
[1] 2023-01-29 00:48:58, oob acc: 95.45%, # of SNPs: 14, # of haplo: 31
=== building individual classifier 2, out-of-bag (9/26.5%) ===
     1, SNP: 149, loss: 171.797, oob acc: 66.67%, # of haplo: 13
     2, SNP: 176, loss: 120.459, oob acc: 72.22%, # of haplo: 14
     3, SNP: 97, loss: 80.1731, oob acc: 83.33%, # of haplo: 14
     4, SNP: 56, loss: 51.5193, oob acc: 94.44%, # of haplo: 16
     5, SNP: 182, loss: 34.5643, oob acc: 94.44%, # of haplo: 18
     6, SNP: 121, loss: 23.0259, oob acc: 94.44%, # of haplo: 18
     7, SNP: 234, loss: 15.0596, oob acc: 94.44%, # of haplo: 20
     8, SNP: 148, loss: 9.66757, oob acc: 94.44%, # of haplo: 20
     9, SNP: 19, loss: 4.29975, oob acc: 94.44%, # of haplo: 27
    10, SNP: 226, loss: 0.481093, oob acc: 94.44%, # of haplo: 27
    11, SNP: 64, loss: 0.447483, oob acc: 94.44%, # of haplo: 28
    12, SNP: 240, loss: 0.365545, oob acc: 94.44%, # of haplo: 37
    13, SNP: 57, loss: 0.365132, oob acc: 94.44%, # of haplo: 38
[2] 2023-01-29 00:48:59, oob acc: 94.44%, # of SNPs: 13, # of haplo: 38
=== building individual classifier 3, out-of-bag (14/41.2%) ===
     1, SNP: 118, loss: 190.304, oob acc: 53.57%, # of haplo: 13
     2, SNP: 175, loss: 157.208, oob acc: 60.71%, # of haplo: 15
     3, SNP: 103, loss: 128.429, oob acc: 64.29%, # of haplo: 15
     4, SNP: 182, loss: 66.6054, oob acc: 71.43%, # of haplo: 15
     5, SNP: 152, loss: 58.8041, oob acc: 78.57%, # of haplo: 15
     6, SNP: 111, loss: 30.086, oob acc: 82.14%, # of haplo: 15
     7, SNP: 130, loss: 15.3177, oob acc: 89.29%, # of haplo: 19
     8, SNP: 229, loss: 9.99758, oob acc: 89.29%, # of haplo: 28
     9, SNP: 185, loss: 7.40712, oob acc: 89.29%, # of haplo: 29
    10, SNP: 199, loss: 6.21341, oob acc: 89.29%, # of haplo: 29
    11, SNP: 217, loss: 1.38739, oob acc: 89.29%, # of haplo: 30
[3] 2023-01-29 00:48:59, oob acc: 89.29%, # of SNPs: 11, # of haplo: 30
=== building individual classifier 4, out-of-bag (13/38.2%) ===
     1, SNP: 101, loss: 154.355, oob acc: 46.15%, # of haplo: 16
     2, SNP: 102, loss: 139.148, oob acc: 61.54%, # of haplo: 22
     3, SNP: 132, loss: 95.2502, oob acc: 73.08%, # of haplo: 23
     4, SNP: 147, loss: 76.9692, oob acc: 76.92%, # of haplo: 34
     5, SNP: 53, loss: 68.3851, oob acc: 88.46%, # of haplo: 51
     6, SNP: 186, loss: 41.8787, oob acc: 88.46%, # of haplo: 53
     7, SNP: 128, loss: 33.5437, oob acc: 92.31%, # of haplo: 53
     8, SNP: 14, loss: 23.3103, oob acc: 92.31%, # of haplo: 55
     9, SNP: 219, loss: 18.3628, oob acc: 92.31%, # of haplo: 57
    10, SNP: 149, loss: 17.9413, oob acc: 92.31%, # of haplo: 89
    11, SNP: 73, loss: 16.3172, oob acc: 92.31%, # of haplo: 90
    12, SNP: 70, loss: 16.1056, oob acc: 92.31%, # of haplo: 90
    13, SNP: 199, loss: 12.3057, oob acc: 92.31%, # of haplo: 90
    14, SNP: 203, loss: 12.2013, oob acc: 92.31%, # of haplo: 90
    15, SNP: 151, loss: 11.1795, oob acc: 92.31%, # of haplo: 90
[4] 2023-01-29 00:48:59, oob acc: 92.31%, # of SNPs: 15, # of haplo: 90
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0004055642 0.0004189435 0.0005393566 0.0035332670 0.0110835407 0.0312705424 
        Max.         Mean           SD 
0.5148772297 0.0357753361 0.0879935706 
Accuracy with training data: 97.06%
Out-of-bag accuracy: 92.87%
Gene: HLA-A
Training dataset: 34 samples X 264 SNPs
    # of HLA alleles: 14
    # of individual classifiers: 4
    total # of SNPs used: 49
    avg. # of SNPs in an individual classifier: 13.25
        (sd: 1.71, min: 11, max: 15, median: 13.50)
    avg. # of haplotypes in an individual classifier: 47.25
        (sd: 28.72, min: 30, max: 90, median: 34.50)
    avg. out-of-bag accuracy: 92.87%
        (sd: 2.73%, min: 89.29%, max: 95.45%, median: 93.38%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
0.0004055642 0.0004189435 0.0005393566 0.0035332670 0.0110835407 0.0312705424 
        Max.         Mean           SD 
0.5148772297 0.0357753361 0.0879935706 
Genome assembly: hg19
HIBAG model for HLA-A:
    4 individual classifiers
    264 SNPs
    14 unique HLA alleles: 01:01, 02:01, 02:06, ...
Prediction:
    based on the averaged posterior probabilities
Model assembly: hg19, SNP assembly: hg19
Matching the SNPs between the model and the test data:
    match.type=&quot;--&quot;   missing SNPs #                
           Position         0 (0.0%) *being used [1]
         Pos+Allele         0 (0.0%)             [2]
    RefSNP+Position         0 (0.0%)                
             RefSNP         0 (0.0%)                
      [1]: useful if ambiguous strands on array-based platforms
      [2]: suggested if the model and test data have been matched to the same reference genome
    Model platform: not applicable
No allelic strand or A/B allele is flipped.
# of samples: 26
CPU flags: 64-bit
# of threads: 1
Predicting (2023-01-29 00:48:59)	0%
Predicting (2023-01-29 00:48:59)	100%
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 60
# of unique HLA alleles: 14
# of unique HLA genotypes: 29
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 34
# of unique HLA alleles: 14
# of unique HLA genotypes: 21
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 26
# of unique HLA alleles: 12
# of unique HLA genotypes: 17
Build a HIBAG model with 2 individual classifiers:
    MAF threshold: NaN
    # of SNPs randomly sampled as candidates for each selection: 8
    # of SNPs: 51
    # of samples: 60
    # of unique HLA alleles: 17
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:59
=== building individual classifier 1, out-of-bag (24/40.0%) ===
     1, SNP: 13, loss: 391.274, oob acc: 41.67%, # of haplo: 17
     2, SNP: 2, loss: 321.685, oob acc: 52.08%, # of haplo: 18
     3, SNP: 36, loss: 232.846, oob acc: 58.33%, # of haplo: 19
     4, SNP: 28, loss: 178.077, oob acc: 62.50%, # of haplo: 20
     5, SNP: 35, loss: 107.151, oob acc: 68.75%, # of haplo: 20
     6, SNP: 3, loss: 72.2736, oob acc: 72.92%, # of haplo: 23
     7, SNP: 19, loss: 50.8439, oob acc: 77.08%, # of haplo: 25
     8, SNP: 4, loss: 47.2744, oob acc: 83.33%, # of haplo: 29
     9, SNP: 42, loss: 47.0092, oob acc: 85.42%, # of haplo: 37
    10, SNP: 33, loss: 41.5486, oob acc: 85.42%, # of haplo: 41
    11, SNP: 5, loss: 39.769, oob acc: 85.42%, # of haplo: 51
    12, SNP: 10, loss: 34.0977, oob acc: 85.42%, # of haplo: 51
    13, SNP: 37, loss: 32.3969, oob acc: 85.42%, # of haplo: 52
    14, SNP: 7, loss: 28.1492, oob acc: 85.42%, # of haplo: 52
    15, SNP: 15, loss: 27.2163, oob acc: 85.42%, # of haplo: 55
[1] 2023-01-29 00:48:59, oob acc: 85.42%, # of SNPs: 15, # of haplo: 55
=== building individual classifier 2, out-of-bag (17/28.3%) ===
     1, SNP: 18, loss: 453.852, oob acc: 44.12%, # of haplo: 17
     2, SNP: 4, loss: 358.517, oob acc: 50.00%, # of haplo: 18
     3, SNP: 49, loss: 258.495, oob acc: 52.94%, # of haplo: 18
     4, SNP: 5, loss: 172.555, oob acc: 67.65%, # of haplo: 21
     5, SNP: 42, loss: 144.905, oob acc: 76.47%, # of haplo: 21
     6, SNP: 38, loss: 98.7462, oob acc: 79.41%, # of haplo: 21
     7, SNP: 36, loss: 83.4743, oob acc: 82.35%, # of haplo: 24
     8, SNP: 19, loss: 60.2385, oob acc: 88.24%, # of haplo: 24
     9, SNP: 46, loss: 49.1775, oob acc: 88.24%, # of haplo: 24
    10, SNP: 20, loss: 42.3205, oob acc: 88.24%, # of haplo: 24
    11, SNP: 12, loss: 41.1299, oob acc: 91.18%, # of haplo: 25
    12, SNP: 1, loss: 33.8332, oob acc: 91.18%, # of haplo: 25
    13, SNP: 37, loss: 32.8313, oob acc: 91.18%, # of haplo: 26
    14, SNP: 7, loss: 38.8398, oob acc: 94.12%, # of haplo: 27
    15, SNP: 15, loss: 35.0817, oob acc: 94.12%, # of haplo: 32
    16, SNP: 39, loss: 33.7063, oob acc: 94.12%, # of haplo: 30
[2] 2023-01-29 00:48:59, oob acc: 94.12%, # of SNPs: 16, # of haplo: 30
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
3.464062e-10 4.686489e-10 1.568833e-09 3.197938e-03 1.266674e-02 3.773631e-02 
        Max.         Mean           SD 
9.739941e-02 2.429599e-02 2.696412e-02 
Accuracy with training data: 95.83%
Out-of-bag accuracy: 89.77%
Gene: HLA-C
Training dataset: 60 samples X 51 SNPs
    # of HLA alleles: 17
    # of individual classifiers: 2
    total # of SNPs used: 23
    avg. # of SNPs in an individual classifier: 15.50
        (sd: 0.71, min: 15, max: 16, median: 15.50)
    avg. # of haplotypes in an individual classifier: 42.50
        (sd: 17.68, min: 30, max: 55, median: 42.50)
    avg. out-of-bag accuracy: 89.77%
        (sd: 6.15%, min: 85.42%, max: 94.12%, median: 89.77%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
3.464062e-10 4.686489e-10 1.568833e-09 3.197938e-03 1.266674e-02 3.773631e-02 
        Max.         Mean           SD 
9.739941e-02 2.429599e-02 2.696412e-02 
Genome assembly: hg19
Gene: HLA-C
Training dataset: 60 samples X 51 SNPs
    # of HLA alleles: 17
    # of individual classifiers: 1
    total # of SNPs used: 15
    avg. # of SNPs in an individual classifier: 15.00
        (sd: NA, min: 15, max: 15, median: 15.00)
    avg. # of haplotypes in an individual classifier: 55.00
        (sd: NA, min: 55, max: 55, median: 55.00)
    avg. out-of-bag accuracy: 85.42%
        (sd: NA%, min: 85.42%, max: 85.42%, median: 85.42%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
3.464062e-10 4.686489e-10 1.568833e-09 3.197938e-03 1.266674e-02 3.773631e-02 
        Max.         Mean           SD 
9.739941e-02 2.429599e-02 2.696412e-02 
Genome assembly: hg19
Gene: HLA-A
Range: [29910247bp, 29913661bp] on hg19
# of samples: 60
# of unique HLA alleles: 14
# of unique HLA genotypes: 29
Build a HIBAG model with 2 individual classifiers:
    MAF threshold: NaN
    # of SNPs randomly sampled as candidates for each selection: 10
    # of SNPs: 83
    # of samples: 60
    # of unique HLA alleles: 17
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:48:59
=== building individual classifier 1, out-of-bag (24/40.0%) ===
     1, SNP: 44, loss: 396.506, oob acc: 41.67%, # of haplo: 17
     2, SNP: 58, loss: 291.148, oob acc: 50.00%, # of haplo: 17
     3, SNP: 80, loss: 211.469, oob acc: 56.25%, # of haplo: 20
     4, SNP: 18, loss: 138.615, oob acc: 60.42%, # of haplo: 20
     5, SNP: 29, loss: 111.977, oob acc: 62.50%, # of haplo: 22
     6, SNP: 62, loss: 90.976, oob acc: 68.75%, # of haplo: 24
     7, SNP: 13, loss: 70.2962, oob acc: 72.92%, # of haplo: 24
     8, SNP: 14, loss: 54.5685, oob acc: 77.08%, # of haplo: 22
     9, SNP: 72, loss: 35.1951, oob acc: 77.08%, # of haplo: 24
    10, SNP: 3, loss: 23.2868, oob acc: 79.17%, # of haplo: 24
    11, SNP: 70, loss: 21.089, oob acc: 79.17%, # of haplo: 28
    12, SNP: 5, loss: 20.9664, oob acc: 79.17%, # of haplo: 29
    13, SNP: 40, loss: 20.9662, oob acc: 83.33%, # of haplo: 37
    14, SNP: 24, loss: 20.2385, oob acc: 85.42%, # of haplo: 38
    15, SNP: 2, loss: 20.2383, oob acc: 87.50%, # of haplo: 39
    16, SNP: 74, loss: 20.0601, oob acc: 87.50%, # of haplo: 40
    17, SNP: 57, loss: 17.8846, oob acc: 87.50%, # of haplo: 41
    18, SNP: 56, loss: 14.7377, oob acc: 89.58%, # of haplo: 43
    19, SNP: 27, loss: 10.7709, oob acc: 89.58%, # of haplo: 43
[1] 2023-01-29 00:48:59, oob acc: 89.58%, # of SNPs: 19, # of haplo: 43
=== building individual classifier 2, out-of-bag (17/28.3%) ===
     1, SNP: 66, loss: 434.369, oob acc: 44.12%, # of haplo: 19
     2, SNP: 28, loss: 337.451, oob acc: 58.82%, # of haplo: 21
     3, SNP: 30, loss: 302.194, oob acc: 73.53%, # of haplo: 21
     4, SNP: 59, loss: 209.932, oob acc: 73.53%, # of haplo: 21
     5, SNP: 69, loss: 146.631, oob acc: 82.35%, # of haplo: 21
     6, SNP: 73, loss: 96.4111, oob acc: 91.18%, # of haplo: 21
     7, SNP: 70, loss: 81.5466, oob acc: 91.18%, # of haplo: 21
     8, SNP: 3, loss: 71.8294, oob acc: 91.18%, # of haplo: 22
     9, SNP: 5, loss: 66.5825, oob acc: 94.12%, # of haplo: 23
    10, SNP: 27, loss: 46.6959, oob acc: 94.12%, # of haplo: 23
    11, SNP: 72, loss: 39.0572, oob acc: 94.12%, # of haplo: 23
    12, SNP: 6, loss: 35.0674, oob acc: 94.12%, # of haplo: 24
    13, SNP: 78, loss: 34.8741, oob acc: 94.12%, # of haplo: 32
    14, SNP: 82, loss: 33.4558, oob acc: 94.12%, # of haplo: 38
    15, SNP: 57, loss: 30.709, oob acc: 94.12%, # of haplo: 41
    16, SNP: 32, loss: 26.6513, oob acc: 94.12%, # of haplo: 42
    17, SNP: 23, loss: 26.6236, oob acc: 94.12%, # of haplo: 46
    18, SNP: 2, loss: 25.7938, oob acc: 94.12%, # of haplo: 56
[2] 2023-01-29 00:49:00, oob acc: 94.12%, # of SNPs: 18, # of haplo: 56
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
1.736136e-08 5.138167e-06 5.122542e-05 1.574056e-03 6.457411e-03 3.142989e-02 
        Max.         Mean           SD 
8.812257e-02 1.848522e-02 2.222954e-02 
Accuracy with training data: 96.67%
Out-of-bag accuracy: 91.85%
Build a HIBAG model with 2 individual classifiers:
    MAF threshold: NaN
    # of SNPs randomly sampled as candidates for each selection: 10
    # of SNPs: 83
    # of samples: 60
    # of unique HLA alleles: 17
CPU flags: 64-bit
# of threads: 1
[-] 2023-01-29 00:49:00
=== building individual classifier 1, out-of-bag (24/40.0%) ===
     1, SNP: 44, loss: 396.506, oob acc: 41.67%, # of haplo: 17
     2, SNP: 58, loss: 291.148, oob acc: 50.00%, # of haplo: 17
     3, SNP: 80, loss: 211.469, oob acc: 56.25%, # of haplo: 20
     4, SNP: 18, loss: 138.615, oob acc: 60.42%, # of haplo: 20
     5, SNP: 29, loss: 111.977, oob acc: 62.50%, # of haplo: 22
     6, SNP: 62, loss: 90.976, oob acc: 68.75%, # of haplo: 24
     7, SNP: 13, loss: 70.2962, oob acc: 72.92%, # of haplo: 24
     8, SNP: 14, loss: 54.5685, oob acc: 77.08%, # of haplo: 22
     9, SNP: 72, loss: 35.1951, oob acc: 77.08%, # of haplo: 24
    10, SNP: 3, loss: 23.2868, oob acc: 79.17%, # of haplo: 24
    11, SNP: 70, loss: 21.089, oob acc: 79.17%, # of haplo: 28
    12, SNP: 5, loss: 20.9664, oob acc: 79.17%, # of haplo: 29
    13, SNP: 40, loss: 20.9662, oob acc: 83.33%, # of haplo: 37
    14, SNP: 24, loss: 20.2385, oob acc: 85.42%, # of haplo: 38
    15, SNP: 2, loss: 20.2383, oob acc: 87.50%, # of haplo: 39
    16, SNP: 74, loss: 20.0601, oob acc: 87.50%, # of haplo: 40
    17, SNP: 57, loss: 17.8846, oob acc: 87.50%, # of haplo: 41
    18, SNP: 56, loss: 14.7377, oob acc: 89.58%, # of haplo: 43
    19, SNP: 27, loss: 10.7709, oob acc: 89.58%, # of haplo: 43
[1] 2023-01-29 00:49:00, oob acc: 89.58%, # of SNPs: 19, # of haplo: 43
=== building individual classifier 2, out-of-bag (17/28.3%) ===
     1, SNP: 66, loss: 434.369, oob acc: 44.12%, # of haplo: 19
     2, SNP: 28, loss: 337.451, oob acc: 58.82%, # of haplo: 21
     3, SNP: 30, loss: 302.194, oob acc: 73.53%, # of haplo: 21
     4, SNP: 59, loss: 209.932, oob acc: 73.53%, # of haplo: 21
     5, SNP: 69, loss: 146.631, oob acc: 82.35%, # of haplo: 21
     6, SNP: 73, loss: 96.4111, oob acc: 91.18%, # of haplo: 21
     7, SNP: 70, loss: 81.5466, oob acc: 91.18%, # of haplo: 21
     8, SNP: 3, loss: 71.8294, oob acc: 91.18%, # of haplo: 22
     9, SNP: 5, loss: 66.5825, oob acc: 94.12%, # of haplo: 23
    10, SNP: 27, loss: 46.6959, oob acc: 94.12%, # of haplo: 23
    11, SNP: 72, loss: 39.0572, oob acc: 94.12%, # of haplo: 23
    12, SNP: 6, loss: 35.0674, oob acc: 94.12%, # of haplo: 24
    13, SNP: 78, loss: 34.8741, oob acc: 94.12%, # of haplo: 32
    14, SNP: 82, loss: 33.4558, oob acc: 94.12%, # of haplo: 38
    15, SNP: 57, loss: 30.709, oob acc: 94.12%, # of haplo: 41
    16, SNP: 32, loss: 26.6513, oob acc: 94.12%, # of haplo: 42
    17, SNP: 23, loss: 26.6236, oob acc: 94.12%, # of haplo: 46
    18, SNP: 2, loss: 25.7938, oob acc: 94.12%, # of haplo: 56
[2] 2023-01-29 00:49:00, oob acc: 94.12%, # of SNPs: 18, # of haplo: 56
Calculating matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
1.736136e-08 5.138167e-06 5.122542e-05 1.574056e-03 6.457411e-03 3.142989e-02 
        Max.         Mean           SD 
8.812257e-02 1.848522e-02 2.222954e-02 
Accuracy with training data: 96.67%
Out-of-bag accuracy: 91.85%
Gene: HLA-C
Training dataset: 60 samples X 83 SNPs
    # of HLA alleles: 17
    # of individual classifiers: 2
    total # of SNPs used: 30
    avg. # of SNPs in an individual classifier: 18.50
        (sd: 0.71, min: 18, max: 19, median: 18.50)
    avg. # of haplotypes in an individual classifier: 49.50
        (sd: 9.19, min: 43, max: 56, median: 49.50)
    avg. out-of-bag accuracy: 91.85%
        (sd: 3.21%, min: 89.58%, max: 94.12%, median: 91.85%)
Matching proportion:
        Min.     0.1% Qu.       1% Qu.      1st Qu.       Median      3rd Qu. 
1.736136e-08 5.138167e-06 5.122542e-05 1.574056e-03 6.457411e-03 3.142989e-02 
        Max.         Mean           SD 
8.812257e-02 1.848522e-02 2.222954e-02 
Genome assembly: hg19
SNP genotypes: 
    60 samples X 1564 SNPs
    SNPs range from 25769023bp to 33421576bp on hg19
Missing rate per SNP:
    min: 0, max: 0.0666667, mean: 0.0651215, median: 0.0666667, sd: 0.00968287
Missing rate per sample:
    min: 0, max: 0.969949, mean: 0.0651215, median: 0.000639386, sd: 0.243737
Minor allele frequency:
    min: 0, max: 0.5, mean: 0.227582, median: 0.205357, sd: 0.1389
Allelic information:
A/G C/T G/T A/C 
655 632 141 136 
&gt; 
&gt; proc.time()
   user  system elapsed 
 26.171   0.407  28.652 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
</BODY>
</HTML>
