<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for SNPhood on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I>Back to <B>Build/check report for BioC 3.17</B>:&nbsp;&nbsp;&nbsp;<A href="../">simplified</A>&nbsp;&nbsp;&nbsp;<A href="../long-report.html">long</A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="..#A">A</A></TD><TD><A href="..#B">B</A></TD><TD><A href="..#C">C</A></TD><TD><A href="..#D">D</A></TD><TD><A href="..#E">E</A></TD><TD><A href="..#F">F</A></TD><TD><A href="..#G">G</A></TD><TD><A href="..#H">H</A></TD><TD><A href="..#I">I</A></TD><TD><A href="..#J">J</A></TD><TD><A href="..#K">K</A></TD><TD><A href="..#L">L</A></TD><TD><A href="..#M">M</A></TD><TD><A href="..#N">N</A></TD><TD><A href="..#O">O</A></TD><TD><A href="..#P">P</A></TD><TD><A href="..#Q">Q</A></TD><TD><A href="..#R">R</A></TD><TD><B>[<A href="..#S">S</A>]</B></TD><TD><A href="..#T">T</A></TD><TD><A href="..#U">U</A></TD><TD><A href="..#V">V</A></TD><TD><A href="..#W">W</A></TD><TD><A href="..#X">X</A></TD><TD><A href="..#Y">Y</A></TD><TD><A href="..#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-03-27 05:50:44 -0000 (Mon, 27 Mar 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-03-12&nbsp;r83975)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">6083</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for SNPhood on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the SNPhood package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/SNPhood.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1927</B>/2195</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">SNPhood</A>&nbsp;1.29.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/SNPhood">landing page</A>)</SPAN><BR>Christian Arnold<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-03-24&nbsp;01:40:07&nbsp;-0000&nbsp;(Fri,&nbsp;24&nbsp;Mar&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/SNPhood</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">devel</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">4ae9902</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:12:26&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: SNPhood</TD></TR>
<TR><TD><B>Version</B>: 1.29.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:SNPhood.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings SNPhood_1.29.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-03-25 12:57:43 -0000 (Sat, 25 Mar 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-03-25 13:12:52 -0000 (Sat, 25 Mar 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 909.1 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: SNPhood.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:SNPhood.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/site-library --timings SNPhood_1.29.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/SNPhood.Rcheck’
* using R Under development (unstable) (2023-03-12 r83975)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.2 LTS
* using session charset: UTF-8
* checking for file ‘SNPhood/DESCRIPTION’ ... OK
* this is package ‘SNPhood’ version ‘1.29.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘SNPhood’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... NOTE
BugReports field is not a suitable URL but appears to contain an email address
  not specified by mailto: nor contained in &lt; &gt;
   use the Contact field instead
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking startup messages can be suppressed ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
.calcBinomTestVector: no visible binding for global variable ‘pp’
Undefined global functions or variables:
  pp
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of ‘data’ directory ... OK
* checking data for non-ASCII characters ... OK
* checking LazyData ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘SNPhood-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: plotAndSummarizeAllelicBiasTest
&gt; ### Title: Summarize the allelic bias analysis across SNP regions and bins
&gt; ###   and visualize some of the results.
&gt; ### Aliases: plotAndSummarizeAllelicBiasTest
&gt; 
&gt; ### ** Examples
&gt; 
&gt; data(SNPhood, package=&quot;SNPhood&quot;)
Warning in data(SNPhood, package = &quot;SNPhood&quot;) :
  data set ‘SNPhood’ not found
&gt; SNPhood.o = testForAllelicBiases (SNPhood.o, readGroups = c(&quot;maternal&quot;, &quot;paternal&quot;))
Warning in value[[3L]](cond) :
  Could not obtain chromosome sizes using GenomeInfoDb, try fallback implementation...

Test allelic bias for dataset GM10847
 Calculating read overlaps for the background distribution... This may take a while
 Read group maternal
 Read group paternal
 Calculate background distribution with 100 repetitions. This may take a while.
Test allelic bias for dataset GM12890
 Calculating read overlaps for the background distribution... This may take a while
 Read group maternal
 Read group paternal
 Calculate background distribution with 100 repetitions. This may take a while.


FINISHED SUCCESSFULLY WITH ALLELIC BIAS TEST.

 Execution time: 35.4 secs
&gt; SNPhood.o = plotAndSummarizeAllelicBiasTest(SNPhood.o)
Warning in plotAndSummarizeAllelicBiasTest(SNPhood.o) :
  Multiple plots will be produced but no PDf filename has been specified (fileToPlot). It might possible that only the last graph is shown.
No id variables; using all as measure variables
Warning: Removed 305 rows containing non-finite values (`stat_bin()`).
Warning: Removed 4 rows containing missing values (`geom_bar()`).
Error in xtfrm.data.frame(x) : cannot xtfrm data frames
Calls: plotAndSummarizeAllelicBiasTest ... lapply -&gt; FUN -&gt; as.vector -&gt; xtfrm -&gt; xtfrm.data.frame
Execution halted
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘IntroductionToSNPhood.Rmd’ using ‘UTF-8’... OK
  ‘workflow.Rmd’ using ‘UTF-8’... OK
 NONE
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 ERROR, 2 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/SNPhood.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">SNPhood.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL SNPhood
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R-devel_2023-03-12_r83975-bin/site-library’
* installing *source* package ‘SNPhood’ ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (SNPhood)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<HR>
<H3>Example timings</H3>
<P><SPAN class="filename">SNPhood.Rcheck/SNPhood-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>analyzeSNPhood</TD><TD style="text-align: right;">131.865</TD><TD style="text-align: right;">  5.004</TD><TD style="text-align: right;">137.813</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotation-methods</TD><TD style="text-align: right;">0.369</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">0.405</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotationBins</TD><TD style="text-align: right;">0.255</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">0.275</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotationBins2</TD><TD style="text-align: right;">11.346</TD><TD style="text-align: right;"> 0.060</TD><TD style="text-align: right;">11.407</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotationDatasets</TD><TD style="text-align: right;">0.248</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.260</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotationReadGroups</TD><TD style="text-align: right;">0.257</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.266</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>annotationRegions</TD><TD style="text-align: right;">1.260</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">1.277</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>associateGenotypes</TD><TD style="text-align: right;">6.464</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">6.481</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bins-methods</TD><TD style="text-align: right;">0.308</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.332</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>changeObjectIntegrityChecking</TD><TD style="text-align: right;">0.268</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.276</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>collectFiles</TD><TD style="text-align: right;">0.032</TD><TD style="text-align: right;">0.003</TD><TD style="text-align: right;">0.035</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>convertToAllelicFractions</TD><TD style="text-align: right;">0.299</TD><TD style="text-align: right;">0.004</TD><TD style="text-align: right;">0.303</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>counts-method</TD><TD style="text-align: right;">0.372</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.379</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>datasets-methods</TD><TD style="text-align: right;">1.240</TD><TD style="text-align: right;">0.052</TD><TD style="text-align: right;">1.292</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>deleteDatasets</TD><TD style="text-align: right;">0.270</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.286</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>deleteReadGroups</TD><TD style="text-align: right;">0.273</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.273</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>deleteRegions</TD><TD style="text-align: right;">0.286</TD><TD style="text-align: right;">0.028</TD><TD style="text-align: right;">0.313</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>enrichment-methods</TD><TD style="text-align: right;">0.263</TD><TD style="text-align: right;">0.008</TD><TD style="text-align: right;">0.270</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>getDefaultParameterList</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>mergeReadGroups</TD><TD style="text-align: right;">1.289</TD><TD style="text-align: right;">0.020</TD><TD style="text-align: right;">1.309</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>parameters-methods</TD><TD style="text-align: right;">0.246</TD><TD style="text-align: right;">0.016</TD><TD style="text-align: right;">0.262</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotAllelicBiasResults</TD><TD style="text-align: right;">37.957</TD><TD style="text-align: right;"> 0.100</TD><TD style="text-align: right;">37.934</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotAllelicBiasResultsOverview</TD><TD style="text-align: right;">1.399</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">1.435</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotAndCalculateCorrelationDatasets</TD><TD style="text-align: right;">1.513</TD><TD style="text-align: right;">0.047</TD><TD style="text-align: right;">1.561</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotAndCalculateWeakAndStrongGenotype</TD><TD style="text-align: right;">0.923</TD><TD style="text-align: right;">0.024</TD><TD style="text-align: right;">0.948</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>plotAndClusterMatrix</TD><TD style="text-align: right;">0.796</TD><TD style="text-align: right;">0.012</TD><TD style="text-align: right;">0.808</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
