<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for SplicingGraphs on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><A href="../index.html#P">P</A></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><B>[<A href="../index.html#S">S</A>]</B></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-13 06:50:00 -0000 (Mon, 13 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4222</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for SplicingGraphs on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the SplicingGraphs package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/SplicingGraphs.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard error">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1934</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">SplicingGraphs</A>&nbsp;1.39.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/SplicingGraphs">landing page</A>)</SPAN><BR>H. Pagès<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-11&nbsp;01:58:29&nbsp;-0000&nbsp;(Sat,&nbsp;11&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/SplicingGraphs</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">547459a</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:08:13&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: SplicingGraphs</TD></TR>
<TR><TD><B>Version</B>: 1.39.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:SplicingGraphs.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings SplicingGraphs_1.39.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-11 22:43:21 -0000 (Sat, 11 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-11 22:56:18 -0000 (Sat, 11 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 777.1 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 1</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph ERROR">&nbsp;&nbsp;ERROR&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: SplicingGraphs.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: NA</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:SplicingGraphs.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings SplicingGraphs_1.39.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/SplicingGraphs.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘SplicingGraphs/DESCRIPTION’ ... OK
* this is package ‘SplicingGraphs’ version ‘1.39.0’
* package encoding: UTF-8
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘SplicingGraphs’ can be installed ... WARNING
Found the following significant warnings:
  Warning: replacing previous import ‘IRanges::from’ by ‘Rgraphviz::from’ when loading ‘SplicingGraphs’
  Warning: replacing previous import ‘IRanges::to’ by ‘Rgraphviz::to’ when loading ‘SplicingGraphs’
See ‘/home/biocbuild/bbs-3.17-bioc/meat/SplicingGraphs.Rcheck/00install.out’ for details.
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... NOTE
Packages listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘GenomicFeatures’ ‘GenomicAlignments’ ‘Rgraphviz’ ‘igraph’
A package should be listed in only one of these fields.
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... NOTE
&#x27;library&#x27; or &#x27;require&#x27; call to &#x27;igraph&#x27; in package code.
  Please use :: or requireNamespace() instead.
  See section &#x27;Suggested packages&#x27; in the &#x27;Writing R Extensions&#x27; manual.
&#x27;:::&#x27; calls which should be &#x27;::&#x27;:
  &#x27;S4Vectors:::matchIntegerPairs&#x27; &#x27;S4Vectors:::orderIntegerPairs&#x27;
  &#x27;S4Vectors:::selfmatchIntegerPairs&#x27;
  See the note in ?`:::` about the use of this operator.
Unexported objects imported by &#x27;:::&#x27; calls:
  &#x27;BiocGenerics:::testPackage&#x27; &#x27;GenomicAlignments:::fillJunctionGaps&#x27;
  &#x27;IRanges:::newCompressedList0&#x27; &#x27;IRanges:::regroupBySupergroup&#x27;
  &#x27;IRanges:::unlist_as_integer&#x27; &#x27;S4Vectors:::setPrototypeFromObject&#x27;
  See the note in ?`:::` about the use of this operator.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... WARNING
Undocumented code objects:
  ‘reportReads’
Undocumented S4 methods:
  generic &#x27;[&#x27; and siglist &#x27;SplicingGraphs,ANY,ANY,ANY&#x27;
  generic &#x27;reportReads&#x27; and siglist &#x27;SplicingGraphs&#x27;
  generic &#x27;updateObject&#x27; and siglist &#x27;SplicingGraphs&#x27;
All user-level objects in a package (including S4 classes and methods)
should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... WARNING
Objects in \usage without \alias in documentation object &#x27;countReads-methods&#x27;:
  ‘reportReads’

Functions with \usage entries need to have the appropriate \alias
entries, and all their arguments documented.
The \usage entries must correspond to syntactically valid R code.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... ERROR
Running examples in ‘SplicingGraphs-Ex.R’ failed
The error most likely occurred in:

&gt; base::assign(&quot;.ptime&quot;, proc.time(), pos = &quot;CheckExEnv&quot;)
&gt; ### Name: plotTranscripts-methods
&gt; ### Title: Plot a set of transcripts along genomic coordinates.
&gt; ### Aliases: plotTranscripts-methods plotTranscripts
&gt; ###   plotTranscripts,GRangesList-method plotTranscripts,TxDb-method
&gt; ###   plotTranscripts,SplicingGraphs-method
&gt; 
&gt; ### ** Examples
&gt; 
&gt; ## ---------------------------------------------------------------------
&gt; ## A. PLOT TRANSCRIPTS
&gt; ## ---------------------------------------------------------------------
&gt; example(SplicingGraphs)  # create SplicingGraphs object &#x27;sg&#x27;

SplcnG&gt; ## ---------------------------------------------------------------------
SplcnG&gt; ## 1. Load a toy gene model as a TxDb object
SplcnG&gt; ## ---------------------------------------------------------------------
SplcnG&gt; 
SplcnG&gt; library(GenomicFeatures)

SplcnG&gt; suppressWarnings(
SplcnG+   toy_genes_txdb &lt;- makeTxDbFromGFF(toy_genes_gff())
SplcnG+ )
Import genomic features from the file as a GRanges object ... OK
Prepare the &#x27;metadata&#x27; data frame ... OK
Make the TxDb object ... OK

SplcnG&gt; ## ---------------------------------------------------------------------
SplcnG&gt; ## 2. Compute all the splicing graphs (1 graph per gene) and return them
SplcnG&gt; ##    in a SplicingGraphs object
SplcnG&gt; ## ---------------------------------------------------------------------
SplcnG&gt; 
SplcnG&gt; ## Extract the exons grouped by transcript:
SplcnG&gt; ex_by_tx &lt;- exonsBy(toy_genes_txdb, by=&quot;tx&quot;, use.names=TRUE)

SplcnG&gt; ## Extract the transcripts grouped by gene:
SplcnG&gt; tx_by_gn &lt;- transcriptsBy(toy_genes_txdb, by=&quot;gene&quot;)

SplcnG&gt; sg &lt;- SplicingGraphs(ex_by_tx, tx_by_gn)

SplcnG&gt; sg
SplicingGraphs object with 5 gene(s) and 13 transcript(s)

SplcnG&gt; ## Alternatively &#x27;sg&#x27; can be constructed directly from the TxDb
SplcnG&gt; ## object:
SplcnG&gt; sg2 &lt;- SplicingGraphs(toy_genes_txdb)  # same as &#x27;sg&#x27;

SplcnG&gt; sg2
SplicingGraphs object with 5 gene(s) and 13 transcript(s)

SplcnG&gt; ## Note that because SplicingGraphs objects have a slot that is an
SplcnG&gt; ## environment (for caching the bubbles), they cannot be compared with
SplcnG&gt; ## &#x27;identical()&#x27; (will always return FALSE). &#x27;all.equal()&#x27; should be
SplcnG&gt; ## used instead:
SplcnG&gt; stopifnot(isTRUE(all.equal(sg2, sg)))

SplcnG&gt; ## &#x27;sg&#x27; has 1 element per gene and &#x27;names(sg)&#x27; gives the gene ids:
SplcnG&gt; length(sg)
[1] 5

SplcnG&gt; names(sg)
[1] &quot;geneA&quot; &quot;geneB&quot; &quot;geneC&quot; &quot;geneD&quot; &quot;geneE&quot;

SplcnG&gt; ## ---------------------------------------------------------------------
SplcnG&gt; ## 3. Basic manipulation of a SplicingGraphs object
SplcnG&gt; ## ---------------------------------------------------------------------
SplcnG&gt; 
SplcnG&gt; ## Basic accessors:
SplcnG&gt; seqnames(sg)
geneA geneB geneC geneD geneE 
 chrX  chrX  chrX  chrX  chrX 
Levels: chrX

SplcnG&gt; strand(sg)
geneA geneB geneC geneD geneE 
    +     -     +     +     + 
Levels: + - *

SplcnG&gt; seqinfo(sg)
Seqinfo object with 1 sequence from an unspecified genome; no seqlengths:
  seqnames seqlengths isCircular genome
  chrX             NA         NA   &lt;NA&gt;

SplcnG&gt; ## Number of transcripts per gene:
SplcnG&gt; elementNROWS(sg)
geneA geneB geneC geneD geneE 
    2     2     3     4     2 

SplcnG&gt; ## The transcripts of a given gene can be extracted with [[. The result
SplcnG&gt; ## is an *unnamed* GRangesList object containing the exons grouped by
SplcnG&gt; ## transcript:
SplcnG&gt; sg[[&quot;geneD&quot;]]
GRangesList object of length 4:
[[1]]
GRanges object with 2 ranges and 5 metadata columns:
      seqnames    ranges strand |   exon_id   exon_name exon_rank start_SSid
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt; &lt;character&gt; &lt;integer&gt;  &lt;integer&gt;
  [1]     chrX   601-630      + |        10         Dx2         1          1
  [2]     chrX   666-675      + |        12         Dx4         2          5
       end_SSid
      &lt;integer&gt;
  [1]         3
  [2]         6
  -------
  seqinfo: 1 sequence from an unspecified genome; no seqlengths

[[2]]
GRanges object with 2 ranges and 5 metadata columns:
      seqnames    ranges strand |   exon_id   exon_name exon_rank start_SSid
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt; &lt;character&gt; &lt;integer&gt;  &lt;integer&gt;
  [1]     chrX   601-620      + |         9         Dx1         1          1
  [2]     chrX   651-700      + |        11         Dx3         2          4
       end_SSid
      &lt;integer&gt;
  [1]         2
  [2]         8
  -------
  seqinfo: 1 sequence from an unspecified genome; no seqlengths

[[3]]
GRanges object with 3 ranges and 5 metadata columns:
      seqnames    ranges strand |   exon_id   exon_name exon_rank start_SSid
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt; &lt;character&gt; &lt;integer&gt;  &lt;integer&gt;
  [1]     chrX   601-620      + |         9         Dx1         1          1
  [2]     chrX   666-675      + |        12         Dx4         2          5
  [3]     chrX   691-700      + |        13         Dx5         3          7
       end_SSid
      &lt;integer&gt;
  [1]         2
  [2]         6
  [3]         8
  -------
  seqinfo: 1 sequence from an unspecified genome; no seqlengths

[[4]]
GRanges object with 2 ranges and 5 metadata columns:
      seqnames    ranges strand |   exon_id   exon_name exon_rank start_SSid
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt; &lt;character&gt; &lt;integer&gt;  &lt;integer&gt;
  [1]     chrX   601-630      + |        10         Dx2         1          1
  [2]     chrX   651-700      + |        11         Dx3         2          4
       end_SSid
      &lt;integer&gt;
  [1]         3
  [2]         8
  -------
  seqinfo: 1 sequence from an unspecified genome; no seqlengths


SplcnG&gt; ## See &#x27;?plotTranscripts&#x27; for how to plot those transcripts.
SplcnG&gt; 
SplcnG&gt; ## The transcripts of all the genes can be extracted with unlist(). The
SplcnG&gt; ## result is a *named* GRangesList object containing the exons grouped
SplcnG&gt; ## by transcript. The names on the object are the gene ids:
SplcnG&gt; ex_by_tx &lt;- unlist(sg)

SplcnG&gt; ex_by_tx
GRangesList object of length 13:
$geneA
GRanges object with 1 range and 5 metadata columns:
      seqnames    ranges strand |   exon_id   exon_name exon_rank start_SSid
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt; &lt;character&gt; &lt;integer&gt;  &lt;integer&gt;
  [1]     chrX     11-50      + |         2         Ax2         1          1
       end_SSid
      &lt;integer&gt;
  [1]         3
  -------
  seqinfo: 1 sequence from an unspecified genome; no seqlengths

$geneA
GRanges object with 2 ranges and 5 metadata columns:
      seqnames    ranges strand |   exon_id   exon_name exon_rank start_SSid
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt; &lt;character&gt; &lt;integer&gt;  &lt;integer&gt;
  [1]     chrX     11-40      + |         1         Ax1         1          1
  [2]     chrX    71-100      + |         3         Ax3         2          4
       end_SSid
      &lt;integer&gt;
  [1]         2
  [2]         5
  -------
  seqinfo: 1 sequence from an unspecified genome; no seqlengths

$geneB
GRanges object with 2 ranges and 5 metadata columns:
      seqnames    ranges strand |   exon_id   exon_name exon_rank start_SSid
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt; &lt;character&gt; &lt;integer&gt;  &lt;integer&gt;
  [1]     chrX   251-300      - |        23         Bx1         1          3
  [2]     chrX   201-230      - |        20         Bx2         2          6
       end_SSid
      &lt;integer&gt;
  [1]         1
  [2]         4
  -------
  seqinfo: 1 sequence from an unspecified genome; no seqlengths

...
&lt;10 more elements&gt;
&gt; sg
SplicingGraphs object with 5 gene(s) and 13 transcript(s)
&gt; 
&gt; ## &#x27;sg&#x27; has 1 element per gene and &#x27;names(sg)&#x27; gives the gene ids.
&gt; names(sg)
[1] &quot;geneA&quot; &quot;geneB&quot; &quot;geneC&quot; &quot;geneD&quot; &quot;geneE&quot;
&gt; 
&gt; ## The transcripts of a given gene can be extracted with [[. The result
&gt; ## is an *unnamed* GRangesList object containing the exons grouped by
&gt; ## transcript:
&gt; sg[[&quot;geneD&quot;]]
GRangesList object of length 4:
[[1]]
GRanges object with 2 ranges and 5 metadata columns:
      seqnames    ranges strand |   exon_id   exon_name exon_rank start_SSid
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt; &lt;character&gt; &lt;integer&gt;  &lt;integer&gt;
  [1]     chrX   601-630      + |        10         Dx2         1          1
  [2]     chrX   666-675      + |        12         Dx4         2          5
       end_SSid
      &lt;integer&gt;
  [1]         3
  [2]         6
  -------
  seqinfo: 1 sequence from an unspecified genome; no seqlengths

[[2]]
GRanges object with 2 ranges and 5 metadata columns:
      seqnames    ranges strand |   exon_id   exon_name exon_rank start_SSid
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt; &lt;character&gt; &lt;integer&gt;  &lt;integer&gt;
  [1]     chrX   601-620      + |         9         Dx1         1          1
  [2]     chrX   651-700      + |        11         Dx3         2          4
       end_SSid
      &lt;integer&gt;
  [1]         2
  [2]         8
  -------
  seqinfo: 1 sequence from an unspecified genome; no seqlengths

[[3]]
GRanges object with 3 ranges and 5 metadata columns:
      seqnames    ranges strand |   exon_id   exon_name exon_rank start_SSid
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt; &lt;character&gt; &lt;integer&gt;  &lt;integer&gt;
  [1]     chrX   601-620      + |         9         Dx1         1          1
  [2]     chrX   666-675      + |        12         Dx4         2          5
  [3]     chrX   691-700      + |        13         Dx5         3          7
       end_SSid
      &lt;integer&gt;
  [1]         2
  [2]         6
  [3]         8
  -------
  seqinfo: 1 sequence from an unspecified genome; no seqlengths

[[4]]
GRanges object with 2 ranges and 5 metadata columns:
      seqnames    ranges strand |   exon_id   exon_name exon_rank start_SSid
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt; &lt;character&gt; &lt;integer&gt;  &lt;integer&gt;
  [1]     chrX   601-630      + |        10         Dx2         1          1
  [2]     chrX   651-700      + |        11         Dx3         2          4
       end_SSid
      &lt;integer&gt;
  [1]         3
  [2]         8
  -------
  seqinfo: 1 sequence from an unspecified genome; no seqlengths

&gt; plotTranscripts(sg[[&quot;geneD&quot;]])  # requires the Gviz package
  - plotting genomic range: from=591.1, to=709.9
&gt; 
&gt; ## The transcripts of all the genes can be extracted with unlist(). The
&gt; ## result is a *named* GRangesList object containing the exons grouped
&gt; ## by transcript. The names on the object are the gene ids:
&gt; ex_by_tx &lt;- unlist(sg)
&gt; ex_by_tx
GRangesList object of length 13:
$geneA
GRanges object with 1 range and 5 metadata columns:
      seqnames    ranges strand |   exon_id   exon_name exon_rank start_SSid
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt; &lt;character&gt; &lt;integer&gt;  &lt;integer&gt;
  [1]     chrX     11-50      + |         2         Ax2         1          1
       end_SSid
      &lt;integer&gt;
  [1]         3
  -------
  seqinfo: 1 sequence from an unspecified genome; no seqlengths

$geneA
GRanges object with 2 ranges and 5 metadata columns:
      seqnames    ranges strand |   exon_id   exon_name exon_rank start_SSid
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt; &lt;character&gt; &lt;integer&gt;  &lt;integer&gt;
  [1]     chrX     11-40      + |         1         Ax1         1          1
  [2]     chrX    71-100      + |         3         Ax3         2          4
       end_SSid
      &lt;integer&gt;
  [1]         2
  [2]         5
  -------
  seqinfo: 1 sequence from an unspecified genome; no seqlengths

$geneB
GRanges object with 2 ranges and 5 metadata columns:
      seqnames    ranges strand |   exon_id   exon_name exon_rank start_SSid
         &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt; &lt;character&gt; &lt;integer&gt;  &lt;integer&gt;
  [1]     chrX   251-300      - |        23         Bx1         1          3
  [2]     chrX   201-230      - |        20         Bx2         2          6
       end_SSid
      &lt;integer&gt;
  [1]         1
  [2]         4
  -------
  seqinfo: 1 sequence from an unspecified genome; no seqlengths

...
&lt;10 more elements&gt;
&gt; plotTranscripts(ex_by_tx)
  - plotting genomic range: from=-77.9, to=988.9
Killed
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘run_unitTests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘SplicingGraphs.Rnw’ using ‘UTF-8’... failed to complete the test
 ERROR
Errors in running code in vignettes:
when running code in ‘SplicingGraphs.Rnw’
  ...
...
&lt;1 more element&gt;

&gt; pdf(&quot;3183-transcripts.pdf&quot;, width = 6, height = 3)

&gt; plotTranscripts(sg[[&quot;3183&quot;]])
  - plotting genomic range: from=21671261.8, to=21743672.2
Killed

... incomplete output.  Crash?

* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 2 ERRORs, 3 WARNINGs, 2 NOTEs
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/SplicingGraphs.Rcheck/00check.log’
for details.

</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">SplicingGraphs.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL SplicingGraphs
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘SplicingGraphs’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Warning: replacing previous import ‘IRanges::from’ by ‘Rgraphviz::from’ when loading ‘SplicingGraphs’
Warning: replacing previous import ‘IRanges::to’ by ‘Rgraphviz::to’ when loading ‘SplicingGraphs’
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
Warning: replacing previous import ‘IRanges::from’ by ‘Rgraphviz::from’ when loading ‘SplicingGraphs’
Warning: replacing previous import ‘IRanges::to’ by ‘Rgraphviz::to’ when loading ‘SplicingGraphs’
** testing if installed package can be loaded from final location
Warning: replacing previous import ‘IRanges::from’ by ‘Rgraphviz::from’ when loading ‘SplicingGraphs’
Warning: replacing previous import ‘IRanges::to’ by ‘Rgraphviz::to’ when loading ‘SplicingGraphs’
** testing if installed package keeps a record of temporary installation path
* DONE (SplicingGraphs)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">SplicingGraphs.Rcheck/tests/run_unitTests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; require(&quot;SplicingGraphs&quot;) || stop(&quot;unable to load SplicingGraphs package&quot;)
Loading required package: SplicingGraphs
Loading required package: GenomicFeatures
Loading required package: BiocGenerics

Attaching package: &#x27;BiocGenerics&#x27;

The following objects are masked from &#x27;package:stats&#x27;:

    IQR, mad, sd, var, xtabs

The following objects are masked from &#x27;package:base&#x27;:

    Filter, Find, Map, Position, Reduce, anyDuplicated, aperm, append,
    as.data.frame, basename, cbind, colnames, dirname, do.call,
    duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
    lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
    pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
    tapply, union, unique, unsplit, which.max, which.min

Loading required package: S4Vectors
Loading required package: stats4

Attaching package: &#x27;S4Vectors&#x27;

The following objects are masked from &#x27;package:base&#x27;:

    I, expand.grid, unname

Loading required package: IRanges
Loading required package: GenomeInfoDb
Loading required package: GenomicRanges
Loading required package: AnnotationDbi
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#x27;browseVignettes()&#x27;. To cite Bioconductor, see
    &#x27;citation(&quot;Biobase&quot;)&#x27;, and for packages &#x27;citation(&quot;pkgname&quot;)&#x27;.

Loading required package: GenomicAlignments
Loading required package: SummarizedExperiment
Loading required package: MatrixGenerics
Loading required package: matrixStats

Attaching package: &#x27;matrixStats&#x27;

The following objects are masked from &#x27;package:Biobase&#x27;:

    anyMissing, rowMedians


Attaching package: &#x27;MatrixGenerics&#x27;

The following objects are masked from &#x27;package:matrixStats&#x27;:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars

The following object is masked from &#x27;package:Biobase&#x27;:

    rowMedians

Loading required package: Biostrings
Loading required package: XVector

Attaching package: &#x27;Biostrings&#x27;

The following object is masked from &#x27;package:base&#x27;:

    strsplit

Loading required package: Rsamtools
Loading required package: Rgraphviz
Loading required package: graph

Attaching package: &#x27;graph&#x27;

The following object is masked from &#x27;package:Biostrings&#x27;:

    complement

Loading required package: grid

Attaching package: &#x27;grid&#x27;

The following object is masked from &#x27;package:Biostrings&#x27;:

    pattern


Attaching package: &#x27;Rgraphviz&#x27;

The following objects are masked from &#x27;package:IRanges&#x27;:

    from, to

The following objects are masked from &#x27;package:S4Vectors&#x27;:

    from, to

[1] TRUE
Warning messages:
1: replacing previous import &#x27;IRanges::from&#x27; by &#x27;Rgraphviz::from&#x27; when loading &#x27;SplicingGraphs&#x27; 
2: replacing previous import &#x27;IRanges::to&#x27; by &#x27;Rgraphviz::to&#x27; when loading &#x27;SplicingGraphs&#x27; 
&gt; SplicingGraphs:::.test()
Import genomic features from the file as a GRanges object ... OK
Prepare the &#x27;metadata&#x27; data frame ... OK
Make the TxDb object ... OK


RUNIT TEST PROTOCOL -- Sat Feb 11 22:51:59 2023 
*********************************************** 
Number of test functions: 1 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
SplicingGraphs RUnit Tests - 1 test function, 0 errors, 0 failures
Number of test functions: 1 
Number of errors: 0 
Number of failures: 0 
&gt; 
&gt; proc.time()
   user  system elapsed 
 23.769   3.258  30.122 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">SplicingGraphs.Rcheck/SplicingGraphs-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>SplicingGraphs-class</TD><TD style="text-align: right;">7.264</TD><TD style="text-align: right;">1.258</TD><TD style="text-align: right;">7.505</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>SplicingGraphs-package</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">0.001</TD><TD style="text-align: right;">0.000</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>assignReads</TD><TD style="text-align: right;">5.759</TD><TD style="text-align: right;">1.064</TD><TD style="text-align: right;">5.845</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>bubbles-methods</TD><TD style="text-align: right;">4.246</TD><TD style="text-align: right;">1.027</TD><TD style="text-align: right;">4.346</TD><TD style="text-align: right;">
</TD><TR>
<TR><TD>countReads-methods</TD><TD style="text-align: right;"> 9.334</TD><TD style="text-align: right;"> 2.646</TD><TD style="text-align: right;">12.711</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
