<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<HEAD>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>Build/check report for BioC 3.17 - CHECK results for PathNet on kunpeng1</TITLE>
<LINK rel="stylesheet" href="../report.css" type="text/css">
<SCRIPT type="text/javascript" src="../report.js"></SCRIPT>
</HEAD>
<BODY onLoad="initialize();">
<TABLE class="grid_layout" style="width: 100%; background: #EEE;"><TR><TD style="text-align: left; padding: 5px; vertical-align: middle;"><I><A href="../index.html">Back to <B>Build/check report for BioC 3.17</B></A></I></TD><TD><TABLE class="abc_dispatcher"><TR><TD><A href="../index.html#A">A</A></TD><TD><A href="../index.html#B">B</A></TD><TD><A href="../index.html#C">C</A></TD><TD><A href="../index.html#D">D</A></TD><TD><A href="../index.html#E">E</A></TD><TD><A href="../index.html#F">F</A></TD><TD><A href="../index.html#G">G</A></TD><TD><A href="../index.html#H">H</A></TD><TD><A href="../index.html#I">I</A></TD><TD><A href="../index.html#J">J</A></TD><TD><A href="../index.html#K">K</A></TD><TD><A href="../index.html#L">L</A></TD><TD><A href="../index.html#M">M</A></TD><TD><A href="../index.html#N">N</A></TD><TD><A href="../index.html#O">O</A></TD><TD><B>[<A href="../index.html#P">P</A>]</B></TD><TD><A href="../index.html#Q">Q</A></TD><TD><A href="../index.html#R">R</A></TD><TD><A href="../index.html#S">S</A></TD><TD><A href="../index.html#T">T</A></TD><TD><A href="../index.html#U">U</A></TD><TD><A href="../index.html#V">V</A></TD><TD><A href="../index.html#W">W</A></TD><TD><A href="../index.html#X">X</A></TD><TD><A href="../index.html#Y">Y</A></TD><TD><A href="../index.html#Z">Z</A></TD></TR></TABLE></TD></TR></TABLE>
<P class="time_stamp">
This page was generated on 2023-02-03 02:36:14 -0000 (Fri, 03 Feb 2023).
</P>
<TABLE class="node_specs">
<TR><TH>Hostname</TH><TH>OS</TH><TH>Arch&nbsp;(*)</TH><TH>R&nbsp;version</TH><TH style="text-align: right;">Installed&nbsp;pkgs</TH></TR>
<TR class="kunpeng1"><TD><B><A href="../kunpeng1-NodeInfo.html"><B>kunpeng1</B></A></B></TD><TD>Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)</TD><TD>aarch64</TD><TD>R&nbsp;Under&nbsp;development&nbsp;(unstable)&nbsp;(2023-01-14&nbsp;r83615)&nbsp;--&nbsp;"Unsuffered&nbsp;Consequences"
</TD><TD style="text-align: right;"><A href="../kunpeng1-R-instpkgs.html">4039</A></TD></TR>
<TR><TD COLSPAN="5" style="font-size: smaller;"><I>Click on any hostname to see more info about the system (e.g. compilers) &nbsp;&nbsp;&nbsp;&nbsp; (*) as reported by 'uname -p', except on Windows and Mac OS X</I></TD></TR>
</TABLE>
<BR>
<H2><SPAN class="kunpeng1">CHECK results for PathNet on kunpeng1</SPAN></H2>
<BR>
<DIV class="motd">
<TABLE><TR><TD>
To the developers/maintainers of the PathNet package:<BR>
- Please allow up to 24 hours (and sometimes 48 hours) for your latest push to git@git.bioconductor.org:packages/PathNet.git to<BR>reflect on this report. See <A href="https://bioconductor.org/developers/how-to/troubleshoot-build-report/">How and When does the builder pull? When will my changes propagate?</A> for more information.<BR>
- Make sure to <A href="../Renviron.bioc">use the following settings</A> in order to reproduce any error or warning you see on this page.<BR>
</TD></TR></TABLE>
</DIV>
<P style="text-align: center;"><A href="raw-results/">raw results</A><P>
<TABLE class="gcard_list">
<TBODY class="gcard ok">
<TR class="header"><TD class="leftmost top_left_corner"></TD><TD>Package <B>1431</B>/2164</TD><TD style="width: 75px;">Hostname</TD><TD style="width: 225px;">OS&nbsp;/&nbsp;Arch</TD><TD class="STAGE install">INSTALL</TD><TD class="STAGE buildsrc">BUILD</TD><TD class="STAGE checksrc selected">CHECK</TD><TD class="STAGE buildbin">BUILD BIN</TD><TD style="width: 12px;"></TD><TD class="rightmost top_right_corner"></TD></TR>
<TR class="selected_row"><TD ROWSPAN="2" class="leftmost bottom_left_corner"></TD><TD ROWSPAN="1" style="vertical-align: top;"><B><A href="./">PathNet</A>&nbsp;1.39.0</B>&nbsp;&nbsp;<SPAN style="font-size: smaller; font-style: italic;">(<A href="/packages/3.17/PathNet">landing page</A>)</SPAN><BR>Ludwig Geistlinger<BR><TABLE class="svn_info">
<TR><TD class="svn_info">Snapshot&nbsp;Date:&nbsp;<SPAN class="svn_info">2023-02-01&nbsp;03:13:00&nbsp;-0000&nbsp;(Wed,&nbsp;01&nbsp;Feb&nbsp;2023)</SPAN></TD></TR>
<TR><TD class="svn_info">git_url:&nbsp;<SPAN class="svn_info">https://git.bioconductor.org/packages/PathNet</SPAN></TD></TR>
<TR><TD class="svn_info">git_branch:&nbsp;<SPAN class="svn_info">master</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit:&nbsp;<SPAN class="svn_info">fc892d8</SPAN></TD></TR>
<TR><TD class="svn_info">git_last_commit_date:&nbsp;<SPAN class="svn_info">2022-11-01&nbsp;15:21:04&nbsp;-0000&nbsp;(Tue,&nbsp;01&nbsp;Nov&nbsp;2022)</SPAN></TD></TR>
</TABLE>
</TD><TD class="kunpeng1 selected"><B>kunpeng1</B></TD><TD class="kunpeng1"><SPAN style="font-size: smaller;">Linux&nbsp;(Ubuntu&nbsp;22.04.1&nbsp;LTS)&nbsp;/&nbsp;aarch64</SPAN></TD><TD class="status kunpeng1 install"><A href="./kunpeng1-install.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 buildsrc"><A href="./kunpeng1-buildsrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1 checksrc selected"><A href="./kunpeng1-checksrc.html" onmouseover="add_class_mouseover(this);" onmouseout="remove_class_mouseover(this);"><SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></A></TD><TD class="status kunpeng1"></TD><TD class="status kunpeng1"></TD><TD ROWSPAN="2" class="rightmost bottom_right_corner"></TD></TR>
<TR class="footer"><TD COLSPAN="8"></TD></TR>
</TBODY>
</TABLE>
<HR>
<H3>Summary</H3>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD><B>Package</B>: PathNet</TD></TR>
<TR><TD><B>Version</B>: 1.39.0</TD></TR>
<TR><TD><B>Command</B>: /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:PathNet.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings PathNet_1.39.0.tar.gz</TD></TR>
<TR><TD><B>StartedAt</B>: 2023-02-02 15:20:26 -0000 (Thu, 02 Feb 2023)</TD></TR>
<TR><TD><B>EndedAt</B>: 2023-02-02 15:24:14 -0000 (Thu, 02 Feb 2023)</TD></TR>
<TR><TD><B>EllapsedTime</B>: 228.3 seconds</TD></TR>
<TR><TD><B>RetCode</B>: 0</TD></TR>
<TR><TD><B>Status</B>: <SPAN class="glyph OK">&nbsp;&nbsp;OK&nbsp;&nbsp;</SPAN></TD></TR>
<TR><TD><B>CheckDir</B>: PathNet.Rcheck</TD></TR>
<TR><TD><B>Warnings</B>: 0</TD></TR>
</TABLE>
</DIV>
<HR>
<H3>Command output</H3>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD check --install=check:PathNet.install-out.txt --library=/home/biocbuild/bbs-3.17-bioc/R/library --timings PathNet_1.39.0.tar.gz
###
##############################################################################
##############################################################################


* using log directory ‘/home/biocbuild/bbs-3.17-bioc/meat/PathNet.Rcheck’
* using R Under development (unstable) (2023-01-14 r83615)
* using platform: aarch64-unknown-linux-gnu (64-bit)
* R was compiled by
    gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
    GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
* running under: Ubuntu 22.04.1 LTS
* using session charset: UTF-8
* checking for file ‘PathNet/DESCRIPTION’ ... OK
* checking extension type ... Package
* this is package ‘PathNet’ version ‘1.39.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘PathNet’ can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking ‘build’ directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
construct.DirectEvidence: no visible global function definition for
  ‘write.table’
do.contextual.analysis: no visible global function definition for
  ‘phyper’
do.enrichment.analysis: no visible global function definition for
  ‘phyper’
do.enrichment.analysis: no visible global function definition for
  ‘p.adjust’
Undefined global functions or variables:
  p.adjust phyper write.table
Consider adding
  importFrom(&quot;stats&quot;, &quot;p.adjust&quot;, &quot;phyper&quot;)
  importFrom(&quot;utils&quot;, &quot;write.table&quot;)
to your NAMESPACE file.
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking files in ‘vignettes’ ... OK
* checking examples ... OK
* checking for unstated dependencies in ‘tests’ ... OK
* checking tests ...
  Running ‘runTests.R’
 OK
* checking for unstated dependencies in vignettes ... OK
* checking package vignettes in ‘inst/doc’ ... OK
* checking running R code from vignettes ...
  ‘PathNet.Rnw’... OK
 OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* DONE

Status: 1 NOTE
See
  ‘/home/biocbuild/bbs-3.17-bioc/meat/PathNet.Rcheck/00check.log’
for details.


</PRE>
</DIV><HR>
<H3>Installation output</H3>
<P><SPAN class="filename">PathNet.Rcheck/00install.out</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.17-bioc/R/bin/R CMD INSTALL PathNet
###
##############################################################################
##############################################################################


* installing to library ‘/home/biocbuild/bbs-3.17-bioc/R/library’
* installing *source* package ‘PathNet’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (PathNet)
</PRE>
</DIV><HR>
<H3>Tests output</H3>
<P><SPAN class="filename">PathNet.Rcheck/tests/runTests.Rout</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<PRE style="padding: 3px;">

R Under development (unstable) (2023-01-14 r83615) -- &quot;Unsuffered Consequences&quot;
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: aarch64-unknown-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type &#x27;license()&#x27; or &#x27;licence()&#x27; for distribution details.

R is a collaborative project with many contributors.
Type &#x27;contributors()&#x27; for more information and
&#x27;citation()&#x27; on how to cite R or R packages in publications.

Type &#x27;demo()&#x27; for some demos, &#x27;help()&#x27; for on-line help, or
&#x27;help.start()&#x27; for an HTML browser interface to help.
Type &#x27;q()&#x27; to quit R.

&gt; BiocGenerics:::testPackage(&quot;PathNet&quot;)
Number of genes present in the DirectEvidence File:
100
Starting contextual analysis...
Number of significant pathways to use for contextual analysis:130
Performing contextual analysis of ABC transporters pathway
Performing contextual analysis of Acute myeloid leukemia pathway
Performing contextual analysis of Adherens junction pathway
Performing contextual analysis of Adipocytokine signaling pathway pathway
Performing contextual analysis of Aldosterone-regulated sodium reabsorption pathway
Performing contextual analysis of Allograft rejection pathway
Performing contextual analysis of Alzheimers disease pathway
Performing contextual analysis of Amoebiasis pathway
Performing contextual analysis of Amyotrophic lateral sclerosis (ALS) pathway
Performing contextual analysis of Antigen processing and presentation pathway
Performing contextual analysis of Apoptosis pathway
Performing contextual analysis of Arrhythmogenic right ventricular cardiomyopathy (ARVC) pathway
Performing contextual analysis of Asthma pathway
Performing contextual analysis of Autoimmune thyroid disease pathway
Performing contextual analysis of Axon guidance pathway
Performing contextual analysis of B cell receptor signaling pathway pathway
Performing contextual analysis of Bacterial invasion of epithelial cells pathway
Performing contextual analysis of Basal cell carcinoma pathway
Performing contextual analysis of Basal transcription factors pathway
Performing contextual analysis of Base excision repair pathway
Performing contextual analysis of Bladder cancer pathway
Performing contextual analysis of Calcium signaling pathway pathway
Performing contextual analysis of Carbohydrate digestion and absorption pathway
Performing contextual analysis of Cardiac muscle contraction pathway
Performing contextual analysis of Cell adhesion molecules (CAMs) pathway
Performing contextual analysis of Cell cycle pathway
Performing contextual analysis of Chagas disease pathway
Performing contextual analysis of Chemokine signaling pathway pathway
Performing contextual analysis of Chronic myeloid leukemia pathway
Performing contextual analysis of Circadian rhythm - mammal pathway
Performing contextual analysis of Collecting duct acid secretion pathway
Performing contextual analysis of Colorectal cancer pathway
Performing contextual analysis of Complement and coagulation cascades pathway
Performing contextual analysis of Cytokine-cytokine receptor interaction pathway
Performing contextual analysis of Cytosolic DNA-sensing pathway pathway
Performing contextual analysis of Dilated cardiomyopathy pathway
Performing contextual analysis of DNA replication pathway
Performing contextual analysis of Dorso-ventral axis formation pathway
Performing contextual analysis of ECM-receptor interaction pathway
Performing contextual analysis of Endocytosis pathway
Performing contextual analysis of Endometrial cancer pathway
Performing contextual analysis of Epithelial cell signaling in Helicobacter pylori infection pathway
Performing contextual analysis of ErbB signaling pathway pathway
Performing contextual analysis of Fc epsilon RI signaling pathway pathway
Performing contextual analysis of Fc gamma R-mediated phagocytosis pathway
Performing contextual analysis of Focal adhesion pathway
Performing contextual analysis of Gap junction pathway
Performing contextual analysis of Gastric acid secretion pathway
Performing contextual analysis of Glioma pathway
Performing contextual analysis of GnRH signaling pathway pathway
Performing contextual analysis of Graft-versus-host disease pathway
Performing contextual analysis of Hedgehog signaling pathway pathway
Performing contextual analysis of Hematopoietic cell lineage pathway
Performing contextual analysis of Homologous recombination pathway
Performing contextual analysis of Huntingtons disease pathway
Performing contextual analysis of Hypertrophic cardiomyopathy (HCM) pathway
Performing contextual analysis of Insulin signaling pathway pathway
Performing contextual analysis of Intestinal immune network for IgA production pathway
Performing contextual analysis of Jak-STAT signaling pathway pathway
Performing contextual analysis of Leishmaniasis pathway
Performing contextual analysis of Leukocyte transendothelial migration pathway
Performing contextual analysis of Long-term depression pathway
Performing contextual analysis of Long-term potentiation pathway
Performing contextual analysis of Lysosome pathway
Performing contextual analysis of Malaria pathway
Performing contextual analysis of MAPK signaling pathway pathway
Performing contextual analysis of Maturity onset diabetes of the young pathway
Performing contextual analysis of Melanogenesis pathway
Performing contextual analysis of Melanoma pathway
Performing contextual analysis of Mismatch repair pathway
Performing contextual analysis of mTOR signaling pathway pathway
Performing contextual analysis of Natural killer cell mediated cytotoxicity pathway
Performing contextual analysis of Neuroactive ligand-receptor interaction pathway
Performing contextual analysis of Neurotrophin signaling pathway pathway
Performing contextual analysis of NOD-like receptor signaling pathway pathway
Performing contextual analysis of Non-homologous end-joining pathway
Performing contextual analysis of Non-small cell lung cancer pathway
Performing contextual analysis of Notch signaling pathway pathway
Performing contextual analysis of Nucleotide excision repair pathway
Performing contextual analysis of Olfactory transduction pathway
Performing contextual analysis of Oocyte meiosis pathway
Performing contextual analysis of p53 signaling pathway pathway
Performing contextual analysis of Pancreatic cancer pathway
Performing contextual analysis of Pancreatic secretion pathway
Performing contextual analysis of Parkinsons disease pathway
Performing contextual analysis of Pathogenic Escherichia coli infection pathway
Performing contextual analysis of Pathways in cancer pathway
Performing contextual analysis of Peroxisome pathway
Performing contextual analysis of Phagosome pathway
Performing contextual analysis of Phosphatidylinositol signaling system pathway
Performing contextual analysis of Phototransduction pathway
Performing contextual analysis of PPAR signaling pathway pathway
Performing contextual analysis of Primary immunodeficiency pathway
Performing contextual analysis of Prion diseases pathway
Performing contextual analysis of Progesterone-mediated oocyte maturation pathway
Performing contextual analysis of Prostate cancer pathway
Performing contextual analysis of Proteasome pathway
Performing contextual analysis of Protein export pathway
Performing contextual analysis of Protein processing in endoplasmic reticulum pathway
Performing contextual analysis of Proximal tubule bicarbonate reclamation pathway
Performing contextual analysis of Regulation of actin cytoskeleton pathway
Performing contextual analysis of Regulation of autophagy pathway
Performing contextual analysis of Renal cell carcinoma pathway
Performing contextual analysis of Renin-angiotensin system pathway
Performing contextual analysis of Ribosome pathway
Performing contextual analysis of RIG-I-like receptor signaling pathway pathway
Performing contextual analysis of RNA degradation pathway
Performing contextual analysis of RNA polymerase pathway
Performing contextual analysis of Salivary secretion pathway
Performing contextual analysis of Shigellosis pathway
Performing contextual analysis of Small cell lung cancer pathway
Performing contextual analysis of SNARE interactions in vesicular transport pathway
Performing contextual analysis of Spliceosome pathway
Performing contextual analysis of Systemic lupus erythematosus pathway
Performing contextual analysis of T cell receptor signaling pathway pathway
Performing contextual analysis of Taste transduction pathway
Performing contextual analysis of TGF-beta signaling pathway pathway
Performing contextual analysis of Thyroid cancer pathway
Performing contextual analysis of Tight junction pathway
Performing contextual analysis of Toll-like receptor signaling pathway pathway
Performing contextual analysis of Toxoplasmosis pathway
Performing contextual analysis of Type I diabetes mellitus pathway
Performing contextual analysis of Type II diabetes mellitus pathway
Performing contextual analysis of Ubiquitin mediated proteolysis pathway
Performing contextual analysis of Vascular smooth muscle contraction pathway
Performing contextual analysis of Vasopressin-regulated water reabsorption pathway
Performing contextual analysis of VEGF signaling pathway pathway
Performing contextual analysis of Vibrio cholerae infection pathway
Performing contextual analysis of Viral myocarditis pathway
Performing contextual analysis of Wnt signaling pathway pathway
Number of genes present in the DirectEvidence File:
20077
Number of genes present in the DirectEvidence File:
401
Number of genes present in the DirectEvidence File:
100
Starting enrichment analysis
Processing  130 pathways
Completed enrichment analysis of ABC transporters pathway
Completed enrichment analysis of Acute myeloid leukemia pathway
Completed enrichment analysis of Adherens junction pathway
Completed enrichment analysis of Adipocytokine signaling pathway pathway
Completed enrichment analysis of Aldosterone-regulated sodium reabsorption pathway
Completed enrichment analysis of Allograft rejection pathway
Completed enrichment analysis of Alzheimers disease pathway
Completed enrichment analysis of Amoebiasis pathway
Completed enrichment analysis of Amyotrophic lateral sclerosis (ALS) pathway
Completed enrichment analysis of Antigen processing and presentation pathway
Completed enrichment analysis of Apoptosis pathway
Completed enrichment analysis of Arrhythmogenic right ventricular cardiomyopathy (ARVC) pathway
Completed enrichment analysis of Asthma pathway
Completed enrichment analysis of Autoimmune thyroid disease pathway
Completed enrichment analysis of Axon guidance pathway
Completed enrichment analysis of B cell receptor signaling pathway pathway
Completed enrichment analysis of Bacterial invasion of epithelial cells pathway
Completed enrichment analysis of Basal cell carcinoma pathway
Completed enrichment analysis of Basal transcription factors pathway
Completed enrichment analysis of Base excision repair pathway
Completed enrichment analysis of Bladder cancer pathway
Completed enrichment analysis of Calcium signaling pathway pathway
Completed enrichment analysis of Carbohydrate digestion and absorption pathway
Completed enrichment analysis of Cardiac muscle contraction pathway
Completed enrichment analysis of Cell adhesion molecules (CAMs) pathway
Completed enrichment analysis of Cell cycle pathway
Completed enrichment analysis of Chagas disease pathway
Completed enrichment analysis of Chemokine signaling pathway pathway
Completed enrichment analysis of Chronic myeloid leukemia pathway
Completed enrichment analysis of Circadian rhythm - mammal pathway
Completed enrichment analysis of Collecting duct acid secretion pathway
Completed enrichment analysis of Colorectal cancer pathway
Completed enrichment analysis of Complement and coagulation cascades pathway
Completed enrichment analysis of Cytokine-cytokine receptor interaction pathway
Completed enrichment analysis of Cytosolic DNA-sensing pathway pathway
Completed enrichment analysis of Dilated cardiomyopathy pathway
Completed enrichment analysis of DNA replication pathway
Completed enrichment analysis of Dorso-ventral axis formation pathway
Completed enrichment analysis of ECM-receptor interaction pathway
Completed enrichment analysis of Endocytosis pathway
Completed enrichment analysis of Endometrial cancer pathway
Completed enrichment analysis of Epithelial cell signaling in Helicobacter pylori infection pathway
Completed enrichment analysis of ErbB signaling pathway pathway
Completed enrichment analysis of Fc epsilon RI signaling pathway pathway
Completed enrichment analysis of Fc gamma R-mediated phagocytosis pathway
Completed enrichment analysis of Focal adhesion pathway
Completed enrichment analysis of Gap junction pathway
Completed enrichment analysis of Gastric acid secretion pathway
Completed enrichment analysis of Glioma pathway
Completed enrichment analysis of GnRH signaling pathway pathway
Completed enrichment analysis of Graft-versus-host disease pathway
Completed enrichment analysis of Hedgehog signaling pathway pathway
Completed enrichment analysis of Hematopoietic cell lineage pathway
Completed enrichment analysis of Homologous recombination pathway
Completed enrichment analysis of Huntingtons disease pathway
Completed enrichment analysis of Hypertrophic cardiomyopathy (HCM) pathway
Completed enrichment analysis of Insulin signaling pathway pathway
Completed enrichment analysis of Intestinal immune network for IgA production pathway
Completed enrichment analysis of Jak-STAT signaling pathway pathway
Completed enrichment analysis of Leishmaniasis pathway
Completed enrichment analysis of Leukocyte transendothelial migration pathway
Completed enrichment analysis of Long-term depression pathway
Completed enrichment analysis of Long-term potentiation pathway
Completed enrichment analysis of Lysosome pathway
Completed enrichment analysis of Malaria pathway
Completed enrichment analysis of MAPK signaling pathway pathway
Completed enrichment analysis of Maturity onset diabetes of the young pathway
Completed enrichment analysis of Melanogenesis pathway
Completed enrichment analysis of Melanoma pathway
Completed enrichment analysis of Mismatch repair pathway
Completed enrichment analysis of mTOR signaling pathway pathway
Completed enrichment analysis of Natural killer cell mediated cytotoxicity pathway
Completed enrichment analysis of Neuroactive ligand-receptor interaction pathway
Completed enrichment analysis of Neurotrophin signaling pathway pathway
Completed enrichment analysis of NOD-like receptor signaling pathway pathway
Completed enrichment analysis of Non-homologous end-joining pathway
Completed enrichment analysis of Non-small cell lung cancer pathway
Completed enrichment analysis of Notch signaling pathway pathway
Completed enrichment analysis of Nucleotide excision repair pathway
Completed enrichment analysis of Olfactory transduction pathway
Completed enrichment analysis of Oocyte meiosis pathway
Completed enrichment analysis of p53 signaling pathway pathway
Completed enrichment analysis of Pancreatic cancer pathway
Completed enrichment analysis of Pancreatic secretion pathway
Completed enrichment analysis of Parkinsons disease pathway
Completed enrichment analysis of Pathogenic Escherichia coli infection pathway
Completed enrichment analysis of Pathways in cancer pathway
Completed enrichment analysis of Peroxisome pathway
Completed enrichment analysis of Phagosome pathway
Completed enrichment analysis of Phosphatidylinositol signaling system pathway
Completed enrichment analysis of Phototransduction pathway
Completed enrichment analysis of PPAR signaling pathway pathway
Completed enrichment analysis of Primary immunodeficiency pathway
Completed enrichment analysis of Prion diseases pathway
Completed enrichment analysis of Progesterone-mediated oocyte maturation pathway
Completed enrichment analysis of Prostate cancer pathway
Completed enrichment analysis of Proteasome pathway
Completed enrichment analysis of Protein export pathway
Completed enrichment analysis of Protein processing in endoplasmic reticulum pathway
Completed enrichment analysis of Proximal tubule bicarbonate reclamation pathway
Completed enrichment analysis of Regulation of actin cytoskeleton pathway
Completed enrichment analysis of Regulation of autophagy pathway
Completed enrichment analysis of Renal cell carcinoma pathway
Completed enrichment analysis of Renin-angiotensin system pathway
Completed enrichment analysis of Ribosome pathway
Completed enrichment analysis of RIG-I-like receptor signaling pathway pathway
Completed enrichment analysis of RNA degradation pathway
Completed enrichment analysis of RNA polymerase pathway
Completed enrichment analysis of Salivary secretion pathway
Completed enrichment analysis of Shigellosis pathway
Completed enrichment analysis of Small cell lung cancer pathway
Completed enrichment analysis of SNARE interactions in vesicular transport pathway
Completed enrichment analysis of Spliceosome pathway
Completed enrichment analysis of Systemic lupus erythematosus pathway
Completed enrichment analysis of T cell receptor signaling pathway pathway
Completed enrichment analysis of Taste transduction pathway
Completed enrichment analysis of TGF-beta signaling pathway pathway
Completed enrichment analysis of Thyroid cancer pathway
Completed enrichment analysis of Tight junction pathway
Completed enrichment analysis of Toll-like receptor signaling pathway pathway
Completed enrichment analysis of Toxoplasmosis pathway
Completed enrichment analysis of Type I diabetes mellitus pathway
Completed enrichment analysis of Type II diabetes mellitus pathway
Completed enrichment analysis of Ubiquitin mediated proteolysis pathway
Completed enrichment analysis of Vascular smooth muscle contraction pathway
Completed enrichment analysis of Vasopressin-regulated water reabsorption pathway
Completed enrichment analysis of VEGF signaling pathway pathway
Completed enrichment analysis of Vibrio cholerae infection pathway
Completed enrichment analysis of Viral myocarditis pathway
Completed enrichment analysis of Wnt signaling pathway pathway
Number of significant genes from direct evidence
14
Number of significant genes from combined evidence
13
Processing  130 pathways
Number of genes present in the DirectEvidence File:
20077
[1] &quot;Using significant pathways requires enrichment analysis to be performed, run PathNet with Enrichment_Analysis=TRUE&quot;
[1] &quot;Using significant pathways requires enrichment analysis to be performed, run PathNet with Enrichment_Analysis=TRUE&quot;


RUNIT TEST PROTOCOL -- Thu Feb  2 15:21:32 2023 
*********************************************** 
Number of test functions: 9 
Number of errors: 0 
Number of failures: 0 

 
1 Test Suite : 
PathNet RUnit Tests - 9 test functions, 0 errors, 0 failures
Number of test functions: 9 
Number of errors: 0 
Number of failures: 0 
&gt; 
&gt; proc.time()
   user  system elapsed 
 41.648   0.263  43.743 
</PRE>
</DIV><HR>
<H3>Example timings</H3>
<P><SPAN class="filename">PathNet.Rcheck/PathNet-Ex.timings</SPAN></P>
<DIV class="kunpeng1 hscrollable">
<TABLE>
<TR><TD>name</TD><TD style="text-align: right;">user</TD><TD style="text-align: right;">system</TD><TD style="text-align: right;">elapsed
</TD><TR>
<TR><TD>PathNet</TD><TD style="text-align: right;">0.894</TD><TD style="text-align: right;">0.036</TD><TD style="text-align: right;">1.275</TD><TD style="text-align: right;">
</TD><TR>
</TABLE>
</DIV></BODY>
</HTML>
